compose: TestComposeCompare failed

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2453713&tab=buildLog) on [master@06944cb1996ed9ccfa036118b810e46673fe0ff4](https://github.com/cockroachdb/cockroach/commits/06944cb1996ed9ccfa036118b810e46673fe0ff4):

```
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_4, col2_5 ASC, col2_0 ASC);
        test_1        |             CREATE INDEX ON table2 (col2_7 DESC, col2_2 DESC, col2_6 ASC, col2_1, col2_5 DESC, col2_0, col2_3 ASC, col2_4 ASC);
        test_1        |             CREATE INDEX ON table2 (col2_7, col2_5 ASC, col2_1, col2_6 ASC, col2_4, col2_3 ASC, col2_2, col2_0 DESC);
        test_1        |             CREATE INDEX ON table2 (col2_4, col2_2 DESC) INCLUDE (col2_0);
        test_1        |             CREATE INDEX ON table2 (col2_1 ASC, col2_4 DESC, col2_7 DESC, col2_6 DESC);
        test_1        |             CREATE TABLE table3 (col3_0 INT8 NOT NULL, col3_1 CHAR NOT NULL, col3_2 BYTEA NOT NULL, col3_3 BOOL NOT NULL, col3_4 REGCLASS NOT NULL, col3_5 FLOAT8, col3_6 INT8[] NULL, col3_7 TIMETZ NULL, col3_8 DATE NOT NULL, col3_9 FLOAT8 NOT NULL, col3_10 BYTEA NOT NULL, col3_11 UUID NULL, col3_12 DECIMAL, col3_13 FLOAT8 NOT NULL, col3_14 REGPROCEDURE NOT NULL, col3_15 REGNAMESPACE NOT NULL, PRIMARY KEY (col3_14, col3_13, col3_8, col3_2, col3_10, col3_4, col3_15, col3_1, col3_9));
        test_1        |             CREATE TABLE table4 (col3_14 REGPROCEDURE NOT NULL, col3_13 FLOAT8 NOT NULL, col3_8 DATE NOT NULL, col3_2 BYTEA NOT NULL, col3_10 BYTEA NOT NULL, col3_4 REGCLASS NOT NULL, col3_15 REGNAMESPACE NOT NULL, col3_1 CHAR NOT NULL, col3_9 FLOAT8 NOT NULL, col4_9 INTERVAL NULL, PRIMARY KEY (col3_14, col3_13, col3_8, col3_2, col3_10, col3_4, col3_15, col3_1, col3_9));
        test_1        |             CREATE UNIQUE INDEX ON table4 (col3_2 ASC);
        test_1        |             CREATE INDEX ON table4 (col3_1, col4_9 ASC, col3_9);
        test_1        |             CREATE INDEX ON table4 (col3_9 ASC, col3_1 DESC, col3_2, col3_10, col3_8 DESC, col3_14 DESC, col3_15 DESC, col4_9 DESC, col3_13);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col3_15, col3_2, col4_9 ASC, col3_10 ASC, col3_1 DESC, col3_9 DESC, col3_14, col3_8, col3_13 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_9);
        test_1        |             CREATE INDEX ON table4 (col3_13 ASC, col3_14 ASC, col3_15 DESC, col3_2) INCLUDE (col4_9);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col3_1, col3_2 ASC, col3_13 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col3_2, col3_13 DESC, col3_9 ASC, col3_10 DESC, col3_8, col3_14 ASC, col3_4 ASC, col3_1 DESC, col3_15 ASC, col4_9);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col3_4, col3_2 DESC, col3_8 ASC, col3_9, col3_13 DESC, col3_10, col3_14, col3_15 ASC, col3_1);
        test_1        |             CREATE TABLE table5 (col5_0 TEXT[] NULL, col5_1 BOOL NULL, col5_2 REGCLASS NULL, col5_3 TIMESTAMP, col5_4 INT8, col5_5 INT8, col5_6 VARCHAR, col5_7 TEXT NULL, col5_8 BIT(21), col5_9 OID, col5_10 BIT(49), col5_11 BOOL, col5_12 "char" NULL, col5_13 INTERVAL NULL, col5_14 INTERVAL NOT NULL, col5_15 TEXT, col5_16 BOOL, col5_17 BOX2D);
        test_1        |             CREATE INDEX ON table5 (col5_8, col5_14, col5_5 ASC, col5_10 ASC, col5_1 ASC, col5_11, col5_9 ASC, col5_4 DESC, col5_15, col5_13 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_2, col5_10 DESC, col5_7 DESC, col5_13 DESC, col5_15 ASC, col5_1 DESC, col5_5 DESC, col5_6 DESC, col5_4, col5_3 ASC, col5_12 ASC, col5_16 ASC, col5_9 ASC) INCLUDE (col5_0, col5_8, col5_11, col5_14, col5_17);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_1 ASC, col5_16 DESC, col5_17 ASC, col5_4, col5_2 DESC, col5_15 DESC, col5_11 ASC, col5_12 ASC, col5_5 DESC, col5_6) INCLUDE (col5_0, col5_8, col5_9, col5_10);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_10 ASC, col5_7, col5_17 DESC, col5_1);
        test_1        |             CREATE INDEX ON table5 (col5_13 DESC, col5_16 ASC, col5_12 ASC, col5_14 DESC, col5_7, col5_8 DESC, col5_4 ASC, col5_1 DESC, col5_5 ASC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('vurvfm', 'st', 'zwxky');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (0.78s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 INTERVAL NOT NULL, col1_1 FLOAT4 NULL, col1_2 BYTES NOT NULL, col1_3 DATE NOT NULL, col1_4 REGNAMESPACE NOT NULL, col1_5 STRING NOT NULL, PRIMARY KEY (col1_0 DESC, col1_4 DESC, col1_2 DESC), INDEX (col1_1, col1_5 ASC, col1_4 DESC, col1_0) WHERE table1.col1_2 > e'\\x27':::BYTES, INDEX (col1_4, col1_2 ASC, col1_0 DESC) WHERE (((table1.col1_1 >= 0.0:::FLOAT8) OR (table1.col1_2 > e'\\x27':::BYTES)) OR (table1.col1_3 <= '4714-11-24 BC':::DATE)) AND (table1.col1_5 = e'\U00002603':::STRING), UNIQUE (col1_5 DESC, col1_2 ASC, col1_1 ASC, col1_4, col1_0 DESC) STORING (col1_3) WHERE (table1.col1_3 >= '1970-01-01':::DATE) OR (table1.col1_1 >= 0.0:::FLOAT8), INDEX (col1_3 ASC, col1_2, col1_4 DESC, col1_5, col1_1) WHERE (table1.col1_3 != '1970-01-01':::DATE) OR (table1.col1_1 > 1.401298464324817e-45:::FLOAT8), INDEX (col1_1 ASC) WHERE (table1.col1_1 >= 0.0) AND (table1.col1_2 <= '\x27'), INDEX (col1_4 ASC, col1_5 DESC) STORING (col1_3) WHERE (((table1.col1_1 < -Inf) AND (table1.col1_3 = '4714-11-24 BC')) AND (table1.col1_5 < e'\x00')) OR (table1.col1_2 = '\x00'), INDEX (col1_5 DESC, col1_3 ASC, col1_1) WHERE (((table1.col1_2 < '\xe29883') OR (table1.col1_1 <= 1.0)) OR (table1.col1_3 != '-infinity')) OR (table1.col1_5 < e'\U00002603'), UNIQUE (col1_5) STORING (col1_3) WHERE ((table1.col1_1 > '-Inf':::FLOAT8) AND (table1.col1_2 < e'\\x00':::BYTES)) AND (table1.col1_3 >= '4714-11-24 BC':::DATE), INDEX (col1_1 DESC, col1_0, col1_5 DESC, col1_2 ASC, col1_3, col1_4 DESC), FAMILY (col1_3), FAMILY (col1_0), FAMILY (col1_2, col1_1), FAMILY (col1_4, col1_5));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 297, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": "-0.8077356219291687"}, {"distinct_range": 4884620.975677108, "num_eq": 100000000, "num_range": 6000000, "upper_bound": "-0.04603995382785797"}, {"distinct_range": 1610565044583911200, "num_eq": 4658770328225459898, "num_range": 6783238225420596526, "upper_bound": "0.0"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 664, "row_count": 800}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 220, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 784, "histo_buckets": [{"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "1970-01-01"}, {"distinct_range": 700, "num_eq": 8705294959235858768, "num_range": 700, "upper_bound": "1972-09-27"}, {"distinct_range": 0, "num_eq": 5000000000, "num_range": 7558013972544563136, "upper_bound": "1974-08-27"}, {"distinct_range": 700, "num_eq": 2807331542370169869, "num_range": 700, "upper_bound": "1989-06-28"}, {"distinct_range": 2486909850895121400, "num_eq": 3276077972441879336, "num_range": 5864369844849934535, "upper_bound": "1994-04-02"}, {"distinct_range": 1041633211.3044033, "num_eq": 7510958036383694846, "num_range": 4000000000, "upper_bound": "1994-05-29"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 305, "histo_buckets": [{"distinct_range": 0, "num_eq": 4362656743032084859, "num_range": 0, "upper_bound": "51513504"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "76099314"}, {"distinct_range": 3882606063.593031, "num_eq": 7668195071049074503, "num_range": 6000000000, "upper_bound": "1672935595"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 745, "histo_buckets": [{"distinct_range": 0, "num_eq": 8505440873109504670, "num_range": 0, "upper_bound": "\\u000eR~:\\u0019dq"}, {"distinct_range": 10000000000, "num_eq": 900000, "num_range": 10000000000, "upper_bound": "\\u001eZ\\u0016YZ7)w!"}, {"distinct_range": 10, "num_eq": 1242421543958886454, "num_range": 10, "upper_bound": "#b"}, {"distinct_range": 2026442198714444800, "num_eq": 40000, "num_range": 2555940793393464550, "upper_bound": "?bKN\'"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 285, "histo_buckets": [{"distinct_range": 0, "num_eq": 2119523058634866312, "num_range": 0, "upper_bound": "-61 years -117 days -03:04:32.961583"}, {"distinct_range": 3000, "num_eq": 0, "num_range": 3000, "upper_bound": "-11 years -481 days -13:47:22.017953"}, {"distinct_range": 6831891354833113000, "num_eq": 300000000, "num_range": 6831891354833113566, "upper_bound": "65 years 7 mons 444 days 03:35:58.003542"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 800}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('chbbwb', 'skobcv', 'uspu', 'khx', 'polqw', 'f');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('b', 'fcews', 'snzz', 'mzxsoz', 'ryxiq', 't');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('ymj', 'c', 'hpvo');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('f', 'oanm', 'nwwkz', 'qqwz', 'xccst');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('h');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1838333920963923692, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 8710796370276273160, "num_range": 772474106717724496, "upper_bound": "\\u0000"}, {"distinct_range": 50000, "num_eq": 5000000, "num_range": 50000, "upper_bound": "\'\\r"}, {"distinct_range": 0, "num_eq": 30000000, "num_range": 300, "upper_bound": "I[xQ<W\\u0006\\u0007X"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 5805222229389395114}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2830039195221570006, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000, "num_range": 0, "upper_bound": "-50 years -2 mons -535 days -18:26:49.488444"}, {"distinct_range": 5562626094198901000, "num_eq": 60000000, "num_range": 8654107938860693580, "upper_bound": "-22 years -4 mons -664 days -00:35:50.30695"}, {"distinct_range": 2676856898960741400, "num_eq": 1609683175858932668, "num_range": 2676856898960741200, "upper_bound": "-22 years -11 mons -441 days -05:16:42.467298"}, {"distinct_range": 237.54749490406928, "num_eq": 40000000, "num_range": 1000, "upper_bound": "-18 years -5 mons -129 days -21:56:00.705187"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 8000, "upper_bound": "8 years 8 mons 142 days 16:11:20.469481"}, {"distinct_range": 7243118906100415000, "num_eq": 800000000, "num_range": 8542866341923151972, "upper_bound": "12 years 651 days 20:43:33.94171"}, {"distinct_range": 19.761520728232075, "num_eq": 900000000, "num_range": 80, "upper_bound": "46 years 7 mons 501 days 05:17:39.388726"}, {"distinct_range": 4143343947830002700, "num_eq": 900, "num_range": 7789910068949277606, "upper_bound": "70 years 2 mons 676 days 07:40:06.269296"}, {"distinct_range": 70850974.0693859, "num_eq": 80000000, "num_range": 100000000, "upper_bound": "75 years 3 mons 605 days 23:17:36.137675"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 5805222229389395114}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3344941051893330634, "histo_buckets": [{"distinct_range": 0, "num_eq": 6527010983162974654, "num_range": 0, "upper_bound": "-0.2042299509048462"}, {"distinct_range": 0, "num_eq": 5051212583938548709, "num_range": 8591114541933225755, "upper_bound": "0.2463379055261612"}, {"distinct_range": 0, "num_eq": 0, "num_range": 3060858986414540261, "upper_bound": "0.7044917941093445"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 3105631888214498083, "row_count": 5805222229389395114}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2138003543833275446, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5805222229389395114}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2705653004774982736, "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 5805222229389395114}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2341020466715409802, "histo_buckets": [{"distinct_range": 0, "num_eq": 1343988266138674422, "num_range": 0, "upper_bound": "117598067"}, {"distinct_range": 187.8855961455652, "num_eq": 7079972733172286630, "num_range": 400, "upper_bound": "1325195945"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 5805222229389395114}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2370684656036170201, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "501373729"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 6010714191886569819}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1314185067688028832, "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 6010714191886569819}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1043215994213195324, "histo_buckets": [{"distinct_range": 0, "num_eq": 7921821122121141582, "num_range": 0, "upper_bound": "-34 years -3 mons -849 days -06:50:29.240699"}, {"distinct_range": 0, "num_eq": 30, "num_range": 10000, "upper_bound": "74 years 8 mons 643 days 19:04:46.373705"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 6010714191886569819}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2544902280184549321, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "-0.4242779314517975"}, {"distinct_range": 664614790337116400, "num_eq": 3000000, "num_range": 2697376229200832815, "upper_bound": "0.4465687870979309"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 1723940467247944265, "row_count": 6010714191886569819}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5166093138798763660, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6010714191886569819}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1618784862613990158, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000, "num_range": 0, "upper_bound": "1976-11-10"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 6010714191886569819}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "inf" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2457323&tab=buildLog) on [master@8ff64f5d897a9c7b39ca437193a11a7300cecbc1](https://github.com/cockroachdb/cockroach/commits/8ff64f5d897a9c7b39ca437193a11a7300cecbc1):

```
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        test_1        | --- FAIL: TestCompare (2.68s)
        test_1        |     --- FAIL: TestCompare/postgres (1.80s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.88s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 FLOAT4 NOT NULL, PRIMARY KEY (col1_0), UNIQUE (col1_0 DESC), UNIQUE (col1_0) WHERE table1.col1_0 != 0.0, INDEX (col1_0 DESC) WHERE table1.col1_0 > -Inf, UNIQUE (col1_0));
        test_1        |             CREATE TABLE table2 (col1_0 FLOAT4 NOT NULL, col2_1 FLOAT4, col2_2 DATE NOT NULL, col2_3 DECIMAL, col2_4 REGPROCEDURE, col2_5 INT2 NULL, col2_6 DECIMAL NOT NULL, col2_7 OID NOT NULL, col2_8 TIMESTAMP NULL, col2_9 BYTES NULL, col2_10 CHAR NULL, col2_11 REGPROCEDURE, col2_12 TIME NOT NULL, col2_13 FLOAT8, col2_14 DECIMAL NULL, PRIMARY KEY (col1_0, col2_14 DESC, col2_1 DESC, col2_9, col2_4, col2_2, col2_11 ASC, col2_3, col2_10, col2_5 DESC, col2_8 ASC, col2_6 DESC, col2_13 ASC), INDEX (col2_9 ASC, col2_12 ASC, col2_1 DESC, col2_5 DESC, col2_11 ASC, col2_10, col2_6, col2_3 DESC, col2_8, col2_4 DESC, col2_13 ASC, col2_14) WHERE (((((table2.col2_8 > '0001-01-01 00:00:00') AND (table2.col2_3 != NaN)) AND (table2.col2_10 > e'\x00')) OR (table2.col2_5 = 0)) AND (table2.col2_13 <= NaN)) OR (table2.col2_1 != 0.0), INDEX (col2_12, col2_4 ASC, col2_14 DESC, col2_5 DESC, col2_3 DESC, col2_7 DESC, col2_8, col2_13 DESC, col1_0 DESC, col2_2 ASC) WHERE ((((((((table2.col2_6 < 0:::DECIMAL) OR (table2.col2_14 != (-1):::DECIMAL)) AND (table2.col2_1 > 0.0:::FLOAT8)) AND (table2.col2_13 != 'NaN':::FLOAT8)) OR (table2.col2_10 < e'\'':::STRING)) OR (table2.col2_8 >= '294276-12-31 23:59:59.999999':::TIMESTAMP)) AND (table2.col2_3 != 'Infinity':::DECIMAL)) OR (table2.col2_12 = '00:00:00':::TIME)) OR (table2.col1_0 != 'NaN':::FLOAT8), INDEX (col2_5, col2_14 ASC, col2_3, col2_12 ASC, col1_0, col2_2, col2_13, col2_7 ASC) WHERE (((((((table2.col2_9 = '\x00') OR (table2.col2_2 >= '-infinity')) AND (table2.col1_0 = 1.0)) AND (table2.col2_10 != 'X')) OR (table2.col2_1 = 1.401298464324817e-45)) OR (table2.col2_3 <= 1)) AND (table2.col2_13 <= 0.0)) AND (table2.col2_5 >= 1), FAMILY (col2_5, col2_14, col2_10, col2_8, col2_1), FAMILY (col2_7), FAMILY (col2_11, col1_0), FAMILY (col2_4), FAMILY (col2_13), FAMILY (col2_6, col2_12, col2_2, col2_3), FAMILY (col2_9)) INTERLEAVE IN PARENT table1 (col1_0);
        test_1        |             CREATE TABLE table3 (col3_0 INT8 NOT NULL, col3_1 DECIMAL NOT NULL, col3_2 INT4, col3_3 UUID NOT NULL, col3_4 TIMETZ NOT NULL, col3_5 REGNAMESPACE NOT NULL, col3_6 INT2 NULL, col3_7 "char", col3_8 TIMESTAMPTZ, col3_9 STRING NULL, col3_10 BOOL NOT NULL, col3_11 TIMESTAMP NULL, col3_12 REGNAMESPACE, col3_13 REGPROC NOT NULL, col3_14 REGPROC NOT NULL, col3_15 STRING NOT NULL, col3_16 REGPROCEDURE NOT NULL, col3_17 BOX2D NOT NULL, col3_18 INT2 NULL, PRIMARY KEY (col3_15 ASC, col3_10, col3_16 DESC), UNIQUE (col3_2 DESC, col3_0 DESC, col3_17, col3_15 ASC, col3_1 ASC, col3_9 ASC, col3_4, col3_11 ASC, col3_8 DESC, col3_10, col3_7 ASC, col3_18 ASC, col3_5 DESC, col3_14 ASC, col3_12 ASC, col3_13) WHERE (((((((((((table3.col3_18 > (-1):::INT8) AND (table3.col3_0 <= (-128):::INT8)) AND (table3.col3_2 = 1:::INT8)) OR (table3.col3_4 > '24:00:00-15:59:00':::TIMETZ)) AND (table3.col3_15 > '"':::STRING)) AND (table3.col3_8 != '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (table3.col3_1 <= (-1.234E+401):::DECIMAL)) AND (table3.col3_7 != '':::STRING)) OR (table3.col3_11 >= '3000-01-01 00:00:00':::TIMESTAMP)) AND (table3.col3_6 >= (-1):::INT8)) AND (NOT table3.col3_10)) AND (table3.col3_9 <= e'\x00':::STRING), UNIQUE (col3_6 DESC, col3_9 ASC, col3_7 DESC, col3_12 ASC, col3_17, col3_3 ASC, col3_15, col3_14, col3_11 ASC, col3_0 DESC, col3_10, col3_8 DESC, col3_16 DESC, col3_13 DESC, col3_2, col3_1 ASC, col3_18 DESC, col3_5, col3_4 DESC) WHERE (((((((((((table3.col3_4 >= '24:00:00-15:59:00') AND (table3.col3_0 = 9223372036854775807)) AND (table3.col3_15 != e'\x00')) AND (table3.col3_8 = '294276-12-31 23:59:59.999999+00:00')) AND (NOT table3.col3_10)) AND (table3.col3_11 != '-4713-11-24 00:00:00')) AND (table3.col3_18 > 127)) AND (table3.col3_1 <= 0)) OR (table3.col3_2 > 127)) OR (table3.col3_9 >= 'X')) AND (table3.col3_6 >= -1)) AND (table3.col3_7 >= '"'), UNIQUE (col3_17 DESC, col3_0, col3_13 DESC, col3_2 ASC, col3_4 ASC, col3_18 DESC, col3_15 DESC, col3_7) STORING (col3_1, col3_5, col3_8, col3_9, col3_11, col3_12) WHERE (((((((((((table3.col3_7 <= '':::STRING) OR (table3.col3_1 < (-1.234E+401):::DECIMAL)) OR (table3.col3_9 > '':::STRING)) AND (table3.col3_0 = 2147483647:::INT8)) AND table3.col3_10) AND (table3.col3_15 < e'\'':::STRING)) AND (table3.col3_11 <= '3000-01-01 00:00:00':::TIMESTAMP)) AND (table3.col3_8 > '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table3.col3_2 <= (-2147483648):::INT8)) AND (table3.col3_4 >= '00:00:00+15:59:00':::TIMETZ)) OR (table3.col3_18 < 127:::INT8)) AND (table3.col3_6 < 1:::INT8), UNIQUE (col3_1 ASC) STORING (col3_0, col3_3, col3_5, col3_7, col3_11, col3_14, col3_17) WHERE ((((NOT table3.col3_10) AND (table3.col3_0 <= 32767:::INT8)) AND (table3.col3_9 != '"':::STRING)) AND (table3.col3_2 <= 32767:::INT8)) OR (table3.col3_15 != 'X':::STRING), INDEX (col3_17, col3_4 ASC, col3_2, col3_9, col3_12), FAMILY (col3_13), FAMILY (col3_9, col3_15), FAMILY (col3_16), FAMILY (col3_14, col3_7), FAMILY (col3_3), FAMILY (col3_8, col3_18, col3_2, col3_1), FAMILY (col3_4, col3_17), FAMILY (col3_0, col3_5), FAMILY (col3_10, col3_12), FAMILY (col3_11), FAMILY (col3_6));
        test_1        |             CREATE TABLE table4 (col1_0 FLOAT4 NOT NULL, col2_14 DECIMAL NULL, col2_1 FLOAT4, col2_9 BYTES NULL, col2_4 REGPROCEDURE, col2_2 DATE NOT NULL, col2_11 REGPROCEDURE, col2_3 DECIMAL, col2_10 CHAR NULL, col2_5 INT2 NULL, col2_8 TIMESTAMP NULL, col2_6 DECIMAL NOT NULL, col2_13 FLOAT8, col4_13 REGPROC NULL, col4_14 BOOL NOT NULL, col4_15 NAME NULL, col4_16 FLOAT8 NULL, col4_17 REGPROCEDURE NOT NULL, col4_18 CHAR, col4_19 BOOL NOT NULL, col4_20 "char" NULL, col4_21 FLOAT8 NULL, col4_22 FLOAT4 NOT NULL, col4_23 DATE, col4_24 BOOL NULL, col4_25 INT8 NOT NULL, col4_26 REGPROC, col4_27 BOOL NULL, col4_28 OID, PRIMARY KEY (col1_0, col2_14 DESC, col2_1 DESC, col2_9, col2_4, col2_2, col2_11 ASC, col2_3, col2_10, col2_5 DESC, col2_8 ASC, col2_6 DESC, col2_13 ASC), INDEX (col2_10 DESC, col2_6 DESC, col1_0, col4_24, col4_18, col2_14, col2_11, col4_13) STORING (col4_14, col4_16, col4_17, col4_19, col4_20, col4_21, col4_22, col4_23, col4_25, col4_26, col4_27) WHERE ((((((((((((table4.col2_9 <= e'\\x22':::BYTES) AND (table4.col2_3 > 'Infinity':::DECIMAL)) AND table4.col4_27) AND (table4.col4_15 > '"':::STRING)) OR (table4.col2_5 <= 0:::INT8)) AND (table4.col4_25 >= 9223372036854775807:::INT8)) AND (NOT table4.col4_19)) OR (table4.col4_22 = 0.0:::FLOAT8)) OR (NOT table4.col4_14)) AND (table4.col2_6 >= 'Infinity':::DECIMAL)) OR (table4.col4_16 > 1.401298464324817e-45:::FLOAT8)) AND (table4.col4_21 != (-1.0):::FLOAT8)) OR (table4.col4_20 < e'\x00':::STRING), INDEX (col2_3, col4_25 DESC, col1_0 DESC, col4_18, col2_2 ASC, col4_13 ASC, col2_11, col4_21 ASC, col4_23 DESC, col4_16) STORING (col4_15, col4_20, col4_22, col4_24, col4_26, col4_27) WHERE (table4.col4_20 = e'\U00002603') OR (table4.col2_2 = 'infinity'), UNIQUE (col4_23 DESC, col4_20 ASC, col2_8 ASC, col4_19 ASC, col4_28 DESC, col4_16 DESC, col2_6 DESC, col2_5, col2_13 DESC, col4_14 DESC, col4_15 ASC, col4_17, col2_4, col4_13 DESC) STORING (col4_22, col4_27) WHERE (((((((((((((((((table4.col2_1 < 1.401298464324817e-45) OR table4.col4_19) OR (table4.col2_10 <= e'\'')) OR (table4.col2_8 = '-2000-01-01 00:00:00')) AND (table4.col4_15 >= e'\'')) OR (table4.col2_5 >= 1)) AND (table4.col1_0 = 0.0)) OR (NOT table4.col4_14)) OR (table4.col4_21 != 1.7976931348623157e+308)) OR (NOT table4.col4_24)) OR (table4.col4_22 = NaN)) OR (table4.col2_6 != NaN)) OR (table4.col4_25 <= 2147483647)) AND (table4.col4_23 >= '5874897-12-31')) AND (table4.col2_14 < -1)) AND (table4.col4_16 != -Inf)) OR (NOT table4.col4_27)) OR (table4.col2_3 != Infinity), UNIQUE (col2_11 ASC, col4_27 ASC, col4_24 DESC, col4_20, col4_13 DESC, col2_9, col4_16 DESC, col2_1 ASC, col4_17 DESC, col2_8 DESC, col4_26, col2_14 DESC, col2_3, col2_13 DESC, col2_6 ASC, col2_10 DESC, col4_14, col4_28 DESC, col4_25 DESC, col4_19, col4_18 ASC, col2_4 ASC, col4_15 DESC, col4_22 DESC, col2_2, col2_5, col1_0, col4_23 ASC, col4_21 DESC), INDEX (col1_0 ASC, col2_5, col2_11, col2_13, col4_15, col4_17, col4_18 ASC, col4_13 ASC, col4_22 DESC, col4_28 ASC, col4_21 ASC, col4_16 DESC, col4_19, col4_27, col4_23 ASC, col2_9, col2_4 DESC), INDEX (col4_20 ASC, col4_27, col4_24, col2_2, col2_10 DESC, col4_19, col4_17 ASC, col2_9 DESC, col4_16 ASC) STORING (col4_13, col4_15, col4_21, col4_22, col4_28), FAMILY (col4_20), FAMILY (col2_14), FAMILY (col4_17, col4_27), FAMILY (col4_21, col4_22), FAMILY (col4_28), FAMILY (col2_5), FAMILY (col4_24), FAMILY (col2_4), FAMILY (col2_8), FAMILY (col2_3, col2_9, col2_13), FAMILY (col4_15, col2_10, col2_2, col2_6), FAMILY (col4_23, col4_25), FAMILY (col4_16, col1_0), FAMILY (col4_19), FAMILY (col4_13), FAMILY (col4_14), FAMILY (col2_1, col4_26), FAMILY (col2_11), FAMILY (col4_18)) INTERLEAVE IN PARENT table2 (col1_0, col2_14, col2_1, col2_9, col2_4, col2_2, col2_11, col2_3, col2_10, col2_5, col2_8, col2_6, col2_13);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9540, "histo_buckets": [{"distinct_range": 0, "num_eq": 7312134471633092331, "num_range": 0, "upper_bound": "-1.7265839576721191"}, {"distinct_range": 1057901165901726500, "num_eq": 2905474211069122234, "num_range": 1650197635507804839, "upper_bound": "-0.8312184810638428"}, {"distinct_range": 0, "num_eq": 5000, "num_range": 400000000, "upper_bound": "-0.7478808760643005"}, {"distinct_range": 0, "num_eq": 0, "num_range": 60000000000, "upper_bound": "-0.08107120543718338"}, {"distinct_range": 0, "num_eq": 700, "num_range": 4000000, "upper_bound": "-0.07263533025979996"}, {"distinct_range": 0, "num_eq": 200000, "num_range": 10000000, "upper_bound": "0.1269967257976532"}, {"distinct_range": 161256152.46249038, "num_eq": 0, "num_range": 200000000, "upper_bound": "0.6396889090538025"}, {"distinct_range": 0, "num_eq": 6930350039582283283, "num_range": 80000, "upper_bound": "1.4522517919540405"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 30000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28667, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000000, "num_range": 0, "upper_bound": "-1.0"}, {"distinct_range": 49341.58027577746, "num_eq": 20, "num_range": 300000, "upper_bound": "0.035340119153261185"}, {"distinct_range": 20000000000, "num_eq": 70000000, "num_range": 20000000000, "upper_bound": "1.2205716371536255"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16726, "histo_col_type": "", "name": "__auto__", "null_count": 15925, "row_count": 30000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20279, "histo_col_type": "", "name": "__auto__", "null_count": 23384, "row_count": 30000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21355, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9021, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "18:13:22.633334"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21881, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12793, "histo_col_type": "", "name": "__auto__", "null_count": 10295, "row_count": 30000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28673, "histo_buckets": [{"distinct_range": 0, "num_eq": 8806759440709754679, "num_range": 0, "upper_bound": "-18310"}, {"distinct_range": 4136441802687563000, "num_eq": 100, "num_range": 4136441802687562666, "upper_bound": "-7715"}, {"distinct_range": 1000000, "num_eq": 30000000, "num_range": 1000000, "upper_bound": "1"}, {"distinct_range": 305010705566013100, "num_eq": 400000000, "num_range": 305010705566013132, "upper_bound": "26787"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 8872, "row_count": 30000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24262, "histo_col_type": "", "name": "__auto__", "null_count": 15277, "row_count": 30000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22162, "histo_col_type": "", "name": "__auto__", "null_count": 14102, "row_count": 30000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18791, "histo_col_type": "", "name": "__auto__", "null_count": 11930, "row_count": 30000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29647, "histo_col_type": "", "name": "__auto__", "null_count": 2250, "row_count": 30000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1178, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15318, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 8725028203184981000, "num_eq": 70000000, "num_range": 8725028203184980481, "upper_bound": "\\\\x22aa056730"}, {"distinct_range": 40, "num_eq": 4080640436462101679, "num_range": 40, "upper_bound": "\\\\x2ed40e96"}, {"distinct_range": 0, "num_eq": 100, "num_range": 8534500217281915608, "upper_bound": "\\\\x9d"}, {"distinct_range": 0, "num_eq": 3980732871579365691, "num_range": 600000, "upper_bound": "\\\\xdf854e431a"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 8535, "row_count": 30000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25857, "histo_col_type": "", "name": "__auto__", "null_count": 8106, "row_count": 30000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1559330094663732694, "histo_col_type": "", "name": "__auto__", "null_count": 5676210844834153765, "row_count": 8304929289153834324}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6177863659300527319, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8304929289153834324}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4948716466527363110, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8304929289153834324}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5587176181019321966, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8304929289153834324}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 669465964055024883, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8304929289153834324}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7032802777069231403, "histo_buckets": [{"distinct_range": 0, "num_eq": 5212998760742589775, "num_range": 0, "upper_bound": "BOX(-1.735591848808881 -1.9191803251482862,-0.8523760864549557 0.06760750530268207)"}, {"distinct_range": 1216426455037815300, "num_eq": 90000, "num_range": 1216426455037815324, "upper_bound": "BOX(-0.8336305577034111 0.07236459278946672,-0.4992025599950395 0.22700668500625198)"}, {"distinct_range": 1173997910143510500, "num_eq": 90000000000, "num_range": 1986444595413921073, "upper_bound": "BOX(-0.7758395575902685 0.5159713113595014,1.6816782857012607 1.117497990566917)"}, {"distinct_range": 0, "num_eq": 20, "num_range": 0, "upper_bound": "BOX(-0.7515237128795654 -0.14463040213174216,-0.6589743131259982 -0.03086302563704668)"}, {"distinct_range": 0, "num_eq": 6050642920598623789, "num_range": 4758738431002144233, "upper_bound": "BOX(-0.35364010447551253 0.5136467657389281,-0.1705775218987 0.9266133537284775)"}, {"distinct_range": 0, "num_eq": 60000, "num_range": 2705452879129655688, "upper_bound": "BOX(0.905676923017944 -1.043141018818801,0.9667066414581527 0.20790073086776284)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 8304929289153834324}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4786221075737871413, "histo_col_type": "", "name": "__auto__", "null_count": 6856703321411901572, "row_count": 8304929289153834324}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4835513453843741129, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8304929289153834324}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5765961175396820211, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8304929289153834324}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7009539787848605491, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8304929289153834324}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7289196631322006146, "histo_buckets": [{"distinct_range": 0, "num_eq": 7361491867187443682, "num_range": 0, "upper_bound": "7998"}, {"distinct_range": 1234839013409787400, "num_eq": 100000, "num_range": 4012247209079072869, "upper_bound": "24407"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 4042851795441777768, "row_count": 8304929289153834324}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 914157224679398458, "histo_col_type": "", "name": "__auto__", "null_count": 3793113005153501364, "row_count": 8304929289153834324}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3429681349696735758, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8304929289153834324}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4655293549401590944, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "-6.434112558166808942E+32"}, {"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-3.146752638714278216E+25"}, {"distinct_range": 0, "num_eq": 6871856063636419300, "num_range": 8802589990806969847, "upper_bound": "-716936531389777.9352"}, {"distinct_range": 9141533196536635000, "num_eq": 900, "num_range": 9141533196536635550, "upper_bound": "-44034465924093.6191"}, {"distinct_range": 5806698993776583000, "num_eq": 100000, "num_range": 6088786426574470807, "upper_bound": "-25.8679966538443911"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 3571901352668842445, "upper_bound": "1"}, {"distinct_range": 0, "num_eq": 100, "num_range": 900, "upper_bound": "4.791890851464046289E+19"}, {"distinct_range": 70000, "num_eq": 400000, "num_range": 70000, "upper_bound": "2.742001713224424989E+32"}, {"distinct_range": 9074210844840581000, "num_eq": 80, "num_range": 9074210844840580827, "upper_bound": "Infinity"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 8304929289153834324}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5460491931137596452, "histo_buckets": [{"distinct_range": 0, "num_eq": 3752316201481162957, "num_range": 0, "upper_bound": "-1987679004"}, {"distinct_range": 0, "num_eq": 7586098864407160152, "num_range": 0, "upper_bound": "-80250732"}, {"distinct_range": 7244422397260057000, "num_eq": 3701088404173433661, "num_range": 8530144834189915517, "upper_bound": "127"}, {"distinct_range": 1993362308157066500, "num_eq": 3169764409855465834, "num_range": 1993362308157066452, "upper_bound": "396848345"}, {"distinct_range": 0, "num_eq": 4408604218177535580, "num_range": 3187705642605777588, "upper_bound": "1072848046"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 2627375935661404675, "row_count": 8304929289153834324}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2789738480503522722, "histo_col_type": "", "name": "__auto__", "null_count": 721424881907152153, "row_count": 8304929289153834324}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4604132327661802831, "histo_col_type": "", "name": "__auto__", "null_count": 976506412679921306, "row_count": 8304929289153834324}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1789630152641032210, "histo_col_type": "", "name": "__auto__", "null_count": 1459683450552085809, "row_count": 8304929289153834324}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6842843489355607262, "histo_buckets": [{"distinct_range": 0, "num_eq": 900, "num_range": 0, "upper_bound": "\\u0012X\\u0019B"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 3000, "upper_bound": "\\u0014nl\\u001a1N"}, {"distinct_range": 0, "num_eq": 60000000000, "num_range": 6636773752159454737, "upper_bound": "!e\\u0014"}, {"distinct_range": 200, "num_eq": 4868995780970491822, "num_range": 200, "upper_bound": "J\\u001br\\u000e@Gz\\u001e"}, {"distinct_range": 20000, "num_eq": 7688760201858410836, "num_range": 20000, "upper_bound": "g"}, {"distinct_range": 129029949.34020747, "num_eq": 4918316273128734602, "num_range": 300000000, "upper_bound": "qM\\u0017/2`"}, {"distinct_range": 226198.00023557493, "num_eq": 7000000, "num_range": 1000000, "upper_bound": "z"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 8304929289153834324}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7077373496738347621, "histo_col_type": "", "name": "__auto__", "null_count": 7227705705105592613, "row_count": 7484770741498181613}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6997502506873240500, "histo_col_type": "", "name": "__auto__", "null_count": 860356411190857288, "row_count": 7484770741498181613}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3475413118889021914, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000000, "num_range": 0, "upper_bound": "\\r\\u001b\\u0017"}, {"distinct_range": 0, "num_eq": 1526865569965916430, "num_range": 5093153873305409258, "upper_bound": "\\u001d1\\u0010X}x"}, {"distinct_range": 4499670095921951000, "num_eq": 6480578466198645886, "num_range": 4499670095921951370, "upper_bound": ":"}, {"distinct_range": 4347712843.252035, "num_eq": 179987217686775229, "num_range": 9000000000, "upper_bound": "D\\u0004}ng"}, {"distinct_range": 1390634974306586400, "num_eq": 2091005220971668088, "num_range": 2787906380236502422, "upper_bound": "Id\\u001e\\u000bK"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 100000, "upper_bound": "d\\u0010v7"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 2852197790291174040, "row_count": 7484770741498181613}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2632595939130489790, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7484770741498181613}, {"columns": ["col4_26"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5582206224424591190, "histo_col_type": "", "name": "__auto__", "null_count": 4117202670293777273, "row_count": 7484770741498181613}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6244843751837109941, "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 5124840268516926769, "row_count": 7484770741498181613}, {"columns": ["col4_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3642259982803944249, "histo_col_type": "", "name": "__auto__", "null_count": 955123103709340836, "row_count": 7484770741498181613}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4312645118395566495, "histo_col_type": "", "name": "__auto__", "null_count": 4896514710175484677, "row_count": 7484770741498181613}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5107354027194888940, "histo_col_type": "", "name": "__auto__", "null_count": 3438144643502501558, "row_count": 7484770741498181613}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5051720873703881137, "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 7484770741498181613}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3684777318124458242, "histo_col_type": "", "name": "__auto__", "null_count": 4481582135368463625, "row_count": 7484770741498181613}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1157457343382211746, "histo_col_type": "", "name": "__auto__", "null_count": 1416709996820394753, "row_count": 7484770741498181613}, {"columns": ["col4_25"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4310515708325509930, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7484770741498181613}, {"columns": ["col4_28"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7060015766176535810, "histo_col_type": "", "name": "__auto__", "null_count": 1175124052093650953, "row_count": 7484770741498181613}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6746943576890038309, "histo_col_type": "", "name": "__auto__", "null_count": 5503343740876983426, "row_count": 7484770741498181613}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6325998682497093969, "histo_col_type": "", "name": "__auto__", "null_count": 7120388023407957829, "row_count": 7484770741498181613}, {"columns": ["col4_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6858456163199518968, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7484770741498181613}, {"columns": ["col4_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3248342123901716057, "histo_col_type": "", "name": "__auto__", "null_count": 6810556371463127205, "row_count": 7484770741498181613}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1216605098619612432, "histo_col_type": "", "name": "__auto__", "null_count": 5852927832814802219, "row_count": 7484770741498181613}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2786944961526242160, "histo_col_type": "", "name": "__auto__", "null_count": 6743679525550078783, "row_count": 7484770741498181613}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7259065408463700537, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7484770741498181613}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6459393396723911703, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7484770741498181613}, {"columns": ["col4_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6686399822305908933, "histo_col_type": "", "name": "__auto__", "null_count": 2681569049904546281, "row_count": 7484770741498181613}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1149366890283724743, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7484770741498181613}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3286051550895517535, "histo_buckets": [{"distinct_range": 0, "num_eq": 1009881677710439450, "num_range": 0, "upper_bound": "476451718"}, {"distinct_range": 70000, "num_eq": 0, "num_range": 70000, "upper_bound": "2247480099"}, {"distinct_range": 0, "num_eq": 3349892476743789127, "num_range": 3041814687020579964, "upper_bound": "2307549816"}, {"distinct_range": 700000000, "num_eq": 30000, "num_range": 700000000, "upper_bound": "3006668047"}, {"distinct_range": 61952.967794342636, "num_eq": 0, "num_range": 200000, "upper_bound": "4036475337"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 792485943822154385, "row_count": 7484770741498181613}, {"columns": ["col4_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6865230085596451853, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7484770741498181613}, {"columns": ["col4_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1536469238850986450, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000, "num_range": 0, "upper_bound": ")f\\u0006dd#P"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 0, "upper_bound": ",)\\u0012R!ux"}, {"distinct_range": 513463.65166750853, "num_eq": 20, "num_range": 1000000, "upper_bound": "4O\\u0019"}, {"distinct_range": 86450.67015420599, "num_eq": 30, "num_range": 100000, "upper_bound": "8y1\\u0012VY`Of"}, {"distinct_range": 6445517035916859000, "num_eq": 5000, "num_range": 6445517035916859080, "upper_bound": "O\\u000f\\u0012\\u000b"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 1016856456978928340, "row_count": 7484770741498181613}, {"columns": ["col4_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5805218653817303009, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": "1995-05-15"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 1324261702506301211, "row_count": 7484770741498181613}, {"columns": ["col4_27"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5864469347311719267, "histo_col_type": "", "name": "__auto__", "null_count": 961896295191727276, "row_count": 7484770741498181613}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('ebg', 'u', 'vkuz', 'l');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('tmyipb', 'pyoxmu', 'izvk', 'zt', 'lnkit');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('dzi', 'sjgikd', 'jlffah', 't');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('vfuur', 'yykbil', 'roq', 'kgqmgh');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39782, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000, "num_range": 0, "upper_bound": "-0.6843520402908325"}, {"distinct_range": 0, "num_eq": 200, "num_range": 2729575494504791371, "upper_bound": "-0.6788941025733948"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 5000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 161484971022283053, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 897266681909199521}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 711640162231820540, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000, "num_range": 0, "upper_bound": "01:55:02.832872"}, {"distinct_range": 386863466184542700, "num_eq": 100000000000, "num_range": 887915898506907605, "upper_bound": "02:10:32.024088"}, {"distinct_range": 6983644.207928703, "num_eq": 60, "num_range": 7000000, "upper_bound": "03:02:38.733928"}, {"distinct_range": 700000, "num_eq": 2978285980646864016, "num_range": 700000, "upper_bound": "03:22:20.578949"}, {"distinct_range": 0, "num_eq": 7704553713151079832, "num_range": 200000, "upper_bound": "06:27:36.131696"}, {"distinct_range": 10000000, "num_eq": 2963209127718623632, "num_range": 10000000, "upper_bound": "09:42:48.663469"}, {"distinct_range": 6463407330864338000, "num_eq": 20000000000, "num_range": 6821708893350050413, "upper_bound": "10:39:51.286365"}, {"distinct_range": 0, "num_eq": 301184777289975033, "num_range": 0, "upper_bound": "13:28:24.127022"}, {"distinct_range": 50000000000, "num_eq": 4147230518404331241, "num_range": 50000000000, "upper_bound": "22:57:14.999728"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 897266681909199521}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 162805977113743453, "histo_col_type": "", "name": "__auto__", "null_count": 763484229234572674, "row_count": 897266681909199521}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 843921034229506219, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000000, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 5435614.914595542, "num_eq": 300000, "num_range": 7000000, "upper_bound": "\\\\x3491124b71011f82"}, {"distinct_range": 2761033575847266300, "num_eq": 500000, "num_range": 5299558840917874343, "upper_bound": "\\\\x4da7e804905ef0"}, {"distinct_range": 800000, "num_eq": 300, "num_range": 800000, "upper_bound": "\\\\x58"}, {"distinct_range": 41729487786676030, "num_eq": 6000000, "num_range": 8912751160361734554, "upper_bound": "\\\\x5d9a21"}, {"distinct_range": 0, "num_eq": 3413713736966073395, "num_range": 0, "upper_bound": "\\\\x68f1b4c4f7a1d3"}, {"distinct_range": 8000000000, "num_eq": 5471724936017404932, "num_range": 8000000000, "upper_bound": "\\\\xa2"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 5580783915708348674, "upper_bound": "\\\\xecd6"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 308242427399768404, "row_count": 897266681909199521}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 260609183338195696, "histo_col_type": "", "name": "__auto__", "null_count": 414392041089141305, "row_count": 897266681909199521}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 308362668715982929, "histo_col_type": "", "name": "__auto__", "null_count": 583396697930261255, "row_count": 897266681909199521}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 234957637726089486, "histo_col_type": "", "name": "__auto__", "null_count": 162815753592660685, "row_count": 897266681909199521}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 797562168533344685, "histo_buckets": [{"distinct_range": 0, "num_eq": 8491645720204277425, "num_range": 0, "upper_bound": "-0.14009760320186615"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 897266681909199521}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 807910645033746851, "histo_col_type": "", "name": "__auto__", "null_count": 459144942962934736, "row_count": 897266681909199521}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 724449965531772979, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 897266681909199521}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 579933835028681901, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 897266681909199521}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 839175062301521440, "histo_col_type": "", "name": "__auto__", "null_count": 697519958865733464, "row_count": 897266681909199521}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 869902186233176056, "histo_col_type": "", "name": "__auto__", "null_count": 750797014600556757, "row_count": 897266681909199521}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 365050811337662712, "histo_col_type": "", "name": "__auto__", "null_count": 178524451433913683, "row_count": 897266681909199521}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 701978582309725266, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000000, "num_range": 0, "upper_bound": "-10524"}, {"distinct_range": 0, "num_eq": 7867976562450482022, "num_range": 8611394780211804291, "upper_bound": "-1"}, {"distinct_range": 37737442656.78595, "num_eq": 80000, "num_range": 70000000000, "upper_bound": "2588"}, {"distinct_range": 4320921627071478000, "num_eq": 1928498827629296966, "num_range": 4320921627071477524, "upper_bound": "3709"}, {"distinct_range": 9162486669386982000, "num_eq": 7132526799595002156, "num_range": 9162486669386982713, "upper_bound": "9799"}, {"distinct_range": 50785104932508810, "num_eq": 4789626675926422955, "num_range": 114338146348530215, "upper_bound": "12372"}, {"distinct_range": 9047905631689630000, "num_eq": 10000000000, "num_range": 9047905631689629527, "upper_bound": "26409"}, {"distinct_range": 0, "num_eq": 3500974158515567365, "num_range": 70, "upper_bound": "32290"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 253512148945368074, "row_count": 897266681909199521}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3949090738195004899, "histo_col_type": "", "name": "__auto__", "null_count": 3726313392701050067, "row_count": 6071806906798989573}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5176320891577103655, "histo_buckets": [{"distinct_range": 0, "num_eq": 1458522016007589835, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 200927138156703859, "num_range": 3000000, "upper_bound": "\\u0001\\u0002y\\t\\tU\\u0019"}, {"distinct_range": 0, "num_eq": 1134043717349206216, "num_range": 5494070631220152216, "upper_bound": "\\u00103?\\u001ebp2 "}, {"distinct_range": 493958907003251700, "num_eq": 100000, "num_range": 5334855326638142095, "upper_bound": "\\u0015\\u0013\\u0002\\""}, {"distinct_range": 702767819935329800, "num_eq": 1978291934671533162, "num_range": 702767819935329806, "upper_bound": "904"}, {"distinct_range": 705295178706687500, "num_eq": 8211344656883735631, "num_range": 881911528563117796, "upper_bound": "D \\u0019N"}, {"distinct_range": 10, "num_eq": 7882717323614726396, "num_range": 10, "upper_bound": "_/y\x7f\\u001b\\u0019X\\u000b"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 2458289003665426567, "upper_bound": "zf?g"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 6071806906798989573}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4739728138162218727, "histo_col_type": "", "name": "__auto__", "null_count": 5545368227554602694, "row_count": 6071806906798989573}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3377582938039688751, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6071806906798989573}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3905368470302498259, "histo_col_type": "", "name": "__auto__", "null_count": 5916779430570996108, "row_count": 6071806906798989573}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5181409462649842935, "histo_col_type": "", "name": "__auto__", "null_count": 508647362428093432, "row_count": 6071806906798989573}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2499984129387036174, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6071806906798989573}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2306098219760720004, "histo_col_type": "", "name": "__auto__", "null_count": 4822213470842765524, "row_count": 6071806906798989573}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3120924519563172434, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6071806906798989573}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5934629413696492194, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6071806906798989573}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 673025081004428971, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6071806906798989573}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 199410495674041225, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000000, "num_range": 0, "upper_bound": "BOX(-2.4908333742541533 -1.728889458042011,0.32400249209158793 0.2860665362198091)"}, {"distinct_range": 70, "num_eq": 1936756785340738022, "num_range": 70, "upper_bound": "BOX(-1.7925339109041216 -0.4733465534663769,-0.05465209028730178 0.22336052303360465)"}, {"distinct_range": 3000, "num_eq": 6973524576934023758, "num_range": 3000, "upper_bound": "BOX(-0.41507408440803034 -1.36810284095828,0.6009360265979424 -0.6525029211527034)"}, {"distinct_range": 0, "num_eq": 400000, "num_range": 1000000, "upper_bound": "BOX(-0.4068189140204852 -0.8413978675534587,0.31914816023214976 0.8823110069386663)"}, {"distinct_range": 0, "num_eq": 200, "num_range": 6693002155436433346, "upper_bound": "BOX(-0.3178249888784783 0.22647951417758866,1.2725108119985173 1.4316392522045343)"}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 1379032747963395655, "upper_bound": "BOX(0.3810248604555676 -0.770123228894562,1.5185565050200855 -0.0958446724968906)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 6071806906798989573}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2331417781462415747, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6071806906798989573}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2091881808544478183, "histo_buckets": [{"distinct_range": 0, "num_eq": 2210782367099915024, "num_range": 0, "upper_bound": "-27593"}, {"distinct_range": 2254598.237693676, "num_eq": 100000, "num_range": 10000000, "upper_bound": "-14063"}, {"distinct_range": 0, "num_eq": 2765616991745732861, "num_range": 50000000, "upper_bound": "-4094"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 70, "upper_bound": "19258"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 5053492057129613243, "row_count": 6071806906798989573}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 983044869409561188, "histo_col_type": "", "name": "__auto__", "null_count": 1161828238054456725, "row_count": 6071806906798989573}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1191015824991649659, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6071806906798989573}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4405796118923861097, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6071806906798989573}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3320812762257420053, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "-54815163899080811.23"}, {"distinct_range": 7000966634568361000, "num_eq": 2776787774969173092, "num_range": 7000966634568361234, "upper_bound": "0.4222301013165482824"}, {"distinct_range": 6446205000882804000, "num_eq": 2548287419778971770, "num_range": 6446205000882803255, "upper_bound": "6.259709636743106469"}, {"distinct_range": 2000000, "num_eq": 60000000000, "num_range": 2000000, "upper_bound": "6212985423602.178119"}, {"distinct_range": 80000000, "num_eq": 800, "num_range": 80000000, "upper_bound": "672901406232094359.2"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 6071806906798989573}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2033711066708082251, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": "-1710744820"}, {"distinct_range": 0, "num_eq": 3290727371002762333, "num_range": 927659698425052407, "upper_bound": "-918304636"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 1516896716668446716, "row_count": 6071806906798989573}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2408814722201835488, "histo_col_type": "", "name": "__auto__", "null_count": 2994890759781359629, "row_count": 3433017785340887737}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1729097706181858124, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3433017785340887737}, {"columns": ["col4_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1552461142016827857, "histo_col_type": "", "name": "__auto__", "null_count": 1466877591189428491, "row_count": 3433017785340887737}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3308483318442146610, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3433017785340887737}, {"columns": ["col4_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2976350831377000804, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3433017785340887737}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 227557456123189766, "histo_col_type": "", "name": "__auto__", "null_count": 3096587150327855881, "row_count": 3433017785340887737}, {"columns": ["col4_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2199049062783096352, "histo_col_type": "", "name": "__auto__", "null_count": 1385630587873834549, "row_count": 3433017785340887737}, {"columns": ["col4_28"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 766661285379399708, "histo_col_type": "", "name": "__auto__", "null_count": 180730233050507289, "row_count": 3433017785340887737}, {"columns": ["col4_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 689173157025469994, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000000, "num_range": 0, "upper_bound": "1971-05-31"}, {"distinct_range": 609609389071607900, "num_eq": 8000000000, "num_range": 609609389071607899, "upper_bound": "1973-10-02"}, {"distinct_range": 0, "num_eq": 300000000, "num_range": 50, "upper_bound": "1977-03-23"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 2067634971130413647, "row_count": 3433017785340887737}, {"columns": ["col4_25"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1267878407279127897, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3433017785340887737}, {"columns": ["col4_27"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1843258951749183229, "histo_col_type": "", "name": "__auto__", "null_count": 344720933183756505, "row_count": 3433017785340887737}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2649639490044036830, "histo_col_type": "", "name": "__auto__", "null_count": 292801106365446320, "row_count": 3433017785340887737}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2485628964756465963, "histo_col_type": "", "name": "__auto__", "null_count": 711796514431697760, "row_count": 3433017785340887737}, {"columns": ["col4_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1142338611312984659, "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 1373291776163597526, "row_count": 3433017785340887737}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1729965849792277706, "histo_col_type": "", "name": "__auto__", "null_count": 2714094775398950571, "row_count": 3433017785340887737}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2042374431774499371, "histo_col_type": "", "name": "__auto__", "null_count": 82503457489855999, "row_count": 3433017785340887737}, {"columns": ["col4_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 282601984147321161, "histo_col_type": "", "name": "__auto__", "null_count": 2596245347978505366, "row_count": 3433017785340887737}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1541586766630208464, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3433017785340887737}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 890355263664654963, "histo_col_type": "", "name": "__auto__", "null_count": 2787667980591604038, "row_count": 3433017785340887737}, {"columns": ["col4_26"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 71997139257950308, "histo_col_type": "", "name": "__auto__", "null_count": 2908542403505489621, "row_count": 3433017785340887737}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2619977455409187233, "histo_col_type": "", "name": "__auto__", "null_count": 2663308396949495362, "row_count": 3433017785340887737}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2163221334360290220, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3433017785340887737}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3027650204682429105, "histo_col_type": "", "name": "__auto__", "null_count": 1346851537278868576, "row_count": 3433017785340887737}, {"columns": ["col4_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 588257367789950332, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3433017785340887737}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 475764007264900769, "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 528176207531198436, "row_count": 3433017785340887737}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3046512282225256319, "histo_col_type": "CHAR", "name": "__auto__", "null_count": 3195757709441742359, "row_count": 3433017785340887737}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 123342226275059204, "histo_buckets": [{"distinct_range": 0, "num_eq": 70, "num_range": 0, "upper_bound": "-0.4416610896587372"}, {"distinct_range": 0, "num_eq": 6157161011317680333, "num_range": 100000000000, "upper_bound": "0.2482217699289322"}, {"distinct_range": 52470216.50942551, "num_eq": 1000, "num_range": 100000000, "upper_bound": "0.9327520728111267"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 3433017785340887737}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1374882712703375105, "histo_col_type": "", "name": "__auto__", "null_count": 1411827957769349190, "row_count": 3433017785340887737}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3205804619500101503, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000000, "num_range": 0, "upper_bound": "107700281"}, {"distinct_range": 2010612094425545500, "num_eq": 50000000000, "num_range": 2010612094425545557, "upper_bound": "2197183644"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 1003096474457914762, "row_count": 3433017785340887737}]';
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_13) REFERENCES table1 (col1_0) ON DELETE RESTRICT;
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 41740, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "-0.25647690892219543"}, {"distinct_range": 0, "num_eq": 20000000, "num_range": 4492197639791406420, "upper_bound": "0.25490161776542664"}, {"distinct_range": 2125336.090751459, "num_eq": 2000000000, "num_range": 5000000, "upper_bound": "1.443442940711975"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 90000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 985145, "histo_col_type": "", "name": "__auto__", "null_count": 2402057, "row_count": 10000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 340761, "histo_col_type": "", "name": "__auto__", "null_count": 8043397, "row_count": 10000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7879079, "histo_col_type": "", "name": "__auto__", "null_count": 9739252, "row_count": 10000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7986655, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000, "num_range": 0, "upper_bound": "21:23:05.148036"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 705648, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "-1.4524073600769043"}, {"distinct_range": 100, "num_eq": 30000, "num_range": 100, "upper_bound": "0.0"}, {"distinct_range": 0, "num_eq": 70, "num_range": 600000, "upper_bound": "1.2561839818954468"}, {"distinct_range": 3541011519223475700, "num_eq": 90000000000, "num_range": 3541011519223475904, "upper_bound": "1.8262289762496948"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9123451, "histo_buckets": [{"distinct_range": 0, "num_eq": 70, "num_range": 0, "upper_bound": "-27761"}, {"distinct_range": 6927812383792971000, "num_eq": 5307627573124854394, "num_range": 6927812383792971195, "upper_bound": "-11651"}, {"distinct_range": 7370964266825698000, "num_eq": 10000, "num_range": 7370964266825698001, "upper_bound": "-272"}, {"distinct_range": 800, "num_eq": 100000, "num_range": 800, "upper_bound": "127"}, {"distinct_range": 100, "num_eq": 4936579362316569877, "num_range": 100, "upper_bound": "13274"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 9000000000, "upper_bound": "19069"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 3950143, "row_count": 10000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5265519, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8559079, "histo_col_type": "", "name": "__auto__", "null_count": 5209935, "row_count": 10000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4305939, "histo_col_type": "", "name": "__auto__", "null_count": 6167903, "row_count": 10000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1886913, "histo_col_type": "", "name": "__auto__", "null_count": 6926622, "row_count": 10000000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4619319, "histo_col_type": "", "name": "__auto__", "null_count": 7734796, "row_count": 10000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4538793, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5903112, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4056024, "histo_col_type": "", "name": "__auto__", "null_count": 9805532, "row_count": 10000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4345668, "histo_buckets": [{"distinct_range": 0, "num_eq": 8133996784083765931, "num_range": 0, "upper_bound": "\\\\x436b62c98bb99c87"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 700000, "upper_bound": "\\\\x4b671133a768"}, {"distinct_range": 396450411521413200, "num_eq": 90000000000, "num_range": 2851139444858771089, "upper_bound": "\\\\x68"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 9814203, "row_count": 10000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 123, "histo_col_type": "", "name": "__auto__", "null_count": 77, "row_count": 200}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 91, "histo_col_type": "", "name": "__auto__", "null_count": 150, "row_count": 200}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 96, "histo_col_type": "", "name": "__auto__", "null_count": 80, "row_count": 200}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 115, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 92, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 187, "histo_buckets": [{"distinct_range": 0, "num_eq": 2421079801061585908, "num_range": 0, "upper_bound": "-3.06836143741718718E+23"}, {"distinct_range": 0, "num_eq": 8919864591777740602, "num_range": 265179318091614495, "upper_bound": "648.4454526645814217"}, {"distinct_range": 5270356443090739000, "num_eq": 100, "num_range": 5270356443090739398, "upper_bound": "3.651120087820779466E+37"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 200}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 73, "histo_col_type": "", "name": "__auto__", "null_c

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2460638&tab=buildLog) on [master@609cad52785f49338f7ceff289486b87db4cecb6](https://github.com/cockroachdb/cockroach/commits/609cad52785f49338f7ceff289486b87db4cecb6):

```
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('jnfb', 'unznq', 'qoyya');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('oary', 'bm');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('wkt', 'l', 'kkgfd', 'nr');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('g', 'rok');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('bhqt', 'ukf', 'enapxm', 'siupsf', 'yawken');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (1.89s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 FLOAT8 NOT NULL, col1_1 FLOAT8 NOT NULL, col1_2 "char" NOT NULL, PRIMARY KEY (col1_0 ASC, col1_1 DESC, col1_2 ASC), INDEX (col1_0) WHERE ((table1.col1_2 < 'X') AND (table1.col1_1 != +Inf)) AND (table1.col1_0 = NaN), UNIQUE (col1_2 DESC, col1_0) WHERE (table1.col1_2 <= e'\U00002603':::STRING) AND (table1.col1_1 != (-1.0):::FLOAT8), INDEX (col1_0 ASC, col1_1, col1_2 ASC) WHERE table1.col1_2 > '"':::STRING, INDEX (col1_2 ASC) WHERE ((table1.col1_1 > NaN) AND (table1.col1_2 <= '')) OR (table1.col1_0 < 1.7976931348623157e+308), INDEX (col1_1, col1_0 DESC), FAMILY (col1_0), FAMILY (col1_2), FAMILY (col1_1));
        test_1        |             CREATE TABLE table2 (col2_0 INTERVAL NOT NULL, col2_1 STRING NULL, col2_2 STRING NULL, col2_3 FLOAT8 NULL, col2_4 BOX2D, col2_5 DECIMAL, col2_6 BOOL NOT NULL, col2_7 "char" NOT NULL, col2_8 INTERVAL, col2_9 VARCHAR[] NULL, col2_10 TIMESTAMP NULL, col2_11 INTERVAL NOT NULL, col2_12 UUID NOT NULL, col2_13 INET NOT NULL, col2_14 TIMESTAMP NOT NULL, col2_15 TIMETZ NOT NULL, col2_16 UUID NOT NULL, col2_17 REGTYPE NULL, UNIQUE (col2_5 ASC, col2_13 DESC, col2_10 ASC, col2_15 ASC, col2_3 ASC, col2_4 DESC, col2_16 ASC, col2_2, col2_0 DESC, col2_12 ASC) STORING (col2_1, col2_7, col2_8, col2_14), INDEX (col2_8 DESC, col2_3 ASC, col2_14, col2_15 DESC, col2_17 DESC, col2_7 DESC, col2_5, col2_6, col2_16 DESC, col2_12 DESC) WHERE ((((((((table2.col2_7 > e'\'') OR (table2.col2_2 < e'\'')) OR (table2.col2_1 < e'\x00')) AND (table2.col2_10 = '0001-01-01 00:00:00')) AND (NOT table2.col2_6)) AND (table2.col2_14 < '-4713-11-24 00:00:00')) OR (table2.col2_3 > 1.401298464324817e-45)) AND (table2.col2_5 != Infinity)) AND (table2.col2_15 <= '24:00:00-15:59:00'), UNIQUE (col2_2, col2_17, col2_11 ASC, col2_15, col2_1 ASC, col2_16 DESC, col2_0, col2_8 ASC) STORING (col2_3, col2_4, col2_5, col2_10, col2_12, col2_13, col2_14) WHERE table2.col2_15 >= '24:00:00-15:59:00', FAMILY (col2_11, col2_13), FAMILY (col2_0), FAMILY (col2_1, col2_10, col2_9, col2_3), FAMILY (col2_17), FAMILY (col2_12, col2_7, col2_5, col2_4), FAMILY (col2_15), FAMILY (col2_14, col2_6), FAMILY (col2_2), FAMILY (col2_16, col2_8));
        test_1        |             CREATE TABLE table3 (col3_0 REGPROCEDURE NOT NULL, col3_1 INT8 NOT NULL, col3_2 DECIMAL NOT NULL, col3_3 BOX2D NOT NULL, col3_4 INT4 NOT NULL, col3_5 INT2 NOT NULL, col3_6 BIT(35) NOT NULL, col3_7 DATE NOT NULL, col3_8 STRING NOT NULL, col3_9 STRING NULL, col3_10 BOOL[], col3_11 INET NOT NULL, col3_12 TIMESTAMPTZ NOT NULL, col3_13 REGCLASS NOT NULL, col3_14 INT2 NOT NULL, col3_15 VARCHAR NOT NULL, col3_16 BYTES NOT NULL, col3_17 BIT(45) NOT NULL, PRIMARY KEY (col3_15 DESC, col3_4 DESC, col3_16 DESC, col3_11 DESC, col3_14, col3_7 ASC, col3_3 ASC, col3_17 DESC, col3_8, col3_5 ASC, col3_13, col3_0 ASC, col3_2 DESC, col3_1 ASC), UNIQUE (col3_6 DESC, col3_14 DESC, col3_2 DESC, col3_13 ASC, col3_17 DESC, col3_12 ASC, col3_5, col3_1 ASC, col3_15 DESC, col3_9 DESC, col3_0 DESC, col3_16 ASC, col3_11, col3_8 ASC) WHERE ((((((((((table3.col3_9 <= e'\'') OR (table3.col3_7 != '-infinity')) OR (table3.col3_4 <= 127)) AND (table3.col3_5 = 1)) OR (table3.col3_1 <= 1)) AND (table3.col3_8 >= '"')) OR (table3.col3_15 <= e'\U00002603')) AND (table3.col3_2 != -1)) AND (table3.col3_16 < '\x22')) OR (table3.col3_14 <= -1)) OR (table3.col3_12 <= '294276-12-31 23:59:59.999999+00:00'), UNIQUE (col3_11, col3_8 ASC, col3_1, col3_12 ASC, col3_15 ASC, col3_14 ASC, col3_6 DESC, col3_5 ASC, col3_17 ASC, col3_0, col3_4 ASC, col3_16 DESC, col3_7 ASC, col3_3 DESC, col3_2 DESC, col3_13 DESC, col3_9), UNIQUE (col3_9, col3_14 DESC, col3_0 ASC, col3_8, col3_16 ASC, col3_6, col3_1, col3_4 ASC, col3_13) STORING (col3_12) WHERE ((((((((((table3.col3_1 != 2147483647:::INT8) AND (table3.col3_9 != e'\'':::STRING)) OR (table3.col3_5 != (-1):::INT8)) OR (table3.col3_8 < 'X':::STRING)) AND (table3.col3_4 <= (-1):::INT8)) AND (table3.col3_12 != '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) AND (table3.col3_2 < (-1.234E+401):::DECIMAL)) AND (table3.col3_16 > e'\\x22':::BYTES)) AND (table3.col3_7 < '5874897-12-31':::DATE)) OR (table3.col3_15 != e'\x00':::STRING)) OR (table3.col3_14 > 1:::INT8), UNIQUE (col3_9 DESC, col3_0 DESC, col3_8 ASC, col3_11, col3_15, col3_5 ASC, col3_14 DESC, col3_6 DESC, col3_12, col3_16 ASC, col3_4, col3_7 ASC, col3_13 ASC, col3_1 ASC, col3_17, col3_2, col3_3 DESC), INDEX (col3_12 ASC, col3_2 ASC, col3_15, col3_11, col3_14, col3_5 DESC, col3_0, col3_9, col3_3 ASC) STORING (col3_6), UNIQUE (col3_7 DESC, col3_13, col3_14 DESC, col3_12, col3_8 DESC, col3_2, col3_4 ASC, col3_1 DESC, col3_5, col3_9, col3_3 DESC, col3_6 DESC, col3_17 DESC, col3_0) STORING (col3_10) WHERE ((((((((table3.col3_8 > e'\U00002603':::STRING) OR (table3.col3_9 > '':::STRING)) AND (table3.col3_1 <= (-1):::INT8)) OR (table3.col3_16 > e'\\x':::BYTES)) AND (table3.col3_7 = '1970-01-01':::DATE)) OR (table3.col3_14 > (-1):::INT8)) AND (table3.col3_12 < '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table3.col3_4 >= 1:::INT8)) AND (table3.col3_15 = 'X':::STRING), FAMILY (col3_2, col3_10), FAMILY (col3_3, col3_6), FAMILY (col3_12), FAMILY (col3_14), FAMILY (col3_8), FAMILY (col3_17, col3_7), FAMILY (col3_11, col3_9, col3_1), FAMILY (col3_13), FAMILY (col3_4, col3_15, col3_0, col3_5), FAMILY (col3_16));
        test_1        |             CREATE TABLE table4 (col1_0 FLOAT8 NOT NULL, col1_1 FLOAT8 NOT NULL, col1_2 "char" NOT NULL, col4_3 REGPROC NOT NULL, col4_4 REGTYPE, col4_5 INT2, PRIMARY KEY (col1_0 ASC, col1_1 DESC, col1_2 ASC), INDEX (col4_4 DESC, col1_2 DESC, col4_3 ASC, col1_1) WHERE (table4.col1_1 != (-1.0):::FLOAT8) OR (table4.col4_5 != 127:::INT8), UNIQUE (col4_5), UNIQUE (col4_5, col4_3), FAMILY (col1_2, col4_3, col4_4, col4_5), FAMILY (col1_0, col1_1)) INTERLEAVE IN PARENT table1 (col1_0, col1_1, col1_2);
        test_1        |             CREATE TABLE table5 (col1_0 FLOAT8 NOT NULL, col1_1 FLOAT8 NOT NULL, col1_2 "char" NOT NULL, col5_3 TIME, col5_4 REGTYPE NULL, col5_5 REGNAMESPACE, col5_6 "char" NULL, col5_7 STRING NOT NULL, col5_8 REGPROC NULL, col5_9 INT8, col5_10 INT8 NOT NULL, col5_11 TIMESTAMPTZ, col5_12 INTERVAL NOT NULL, col5_13 TIMETZ NULL, col5_14 BYTES, col5_15 DECIMAL NOT NULL, col5_16 INT4 NULL, col5_17 REGPROC, col5_18 OID, col5_19 STRING, col5_20 REGPROCEDURE, col5_21 CHAR, PRIMARY KEY (col1_0 ASC, col1_1 DESC, col1_2 ASC, col5_9 DESC, col5_16 ASC, col5_20, col5_13 ASC, col5_3 ASC, col5_15 DESC, col5_7 ASC, col5_8 DESC, col5_11 ASC, col5_19, col5_14 DESC), UNIQUE (col5_15, col5_14 ASC, col5_19 ASC, col5_5 ASC) WHERE ((((((((((table5.col5_9 != (-9223372036854775807):::INT8) OR (table5.col5_15 <= '-Infinity':::DECIMAL)) AND (table5.col5_21 <= e'\x00':::STRING)) OR (table5.col5_14 <= e'\\x00':::BYTES)) AND (table5.col5_13 = '24:00:00-15:59:00':::TIMETZ)) OR (table5.col5_10 >= (-2147483648):::INT8)) OR (table5.col5_11 = '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table5.col1_2 >= '':::STRING)) AND (table5.col5_6 = e'\x00':::STRING)) AND (table5.col5_7 != '':::STRING)) OR (table5.col1_0 = 'NaN':::FLOAT8), INDEX (col5_20 DESC, col1_2, col5_9, col5_16 ASC) STORING (col5_6, col5_17, col5_18) WHERE ((((((((((((((table5.col5_3 >= '24:00:00') AND (table5.col1_1 > 1.0)) OR (table5.col1_2 >= '')) OR (table5.col5_7 >= '')) AND (table5.col5_9 = -2147483648)) OR (table5.col5_13 > '24:00:00-15:59:00')) AND (table5.col5_10 < 9223372036854775807)) AND (table5.col5_16 <= 1)) OR (table5.col5_15 >= NaN)) OR (table5.col1_0 != +Inf)) AND (table5.col5_19 >= e'\'')) OR (table5.col5_6 >= e'\U00002603')) AND (table5.col5_14 = '\xe29883')) OR (table5.col5_11 != '-4713-11-24 00:00:00+00:00')) AND (table5.col5_21 > e'\U00002603'), UNIQUE (col5_10 ASC, col5_5 DESC, col5_4, col1_1 ASC, col5_19, col5_18 ASC, col5_16 ASC, col5_15 ASC) STORING (col5_6, col5_17), UNIQUE (col5_12 ASC, col5_11, col5_6 ASC, col5_18, col5_7 ASC, col5_13) STORING (col5_4, col5_5, col5_10, col5_21) WHERE (((table5.col5_15 <= 0:::DECIMAL) AND (table5.col1_1 <= '+Inf':::FLOAT8)) AND (table5.col5_11 >= '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table5.col1_2 < 'X':::STRING), INDEX (col5_13 DESC, col5_16, col5_5 DESC, col5_18 DESC, col5_9 DESC, col5_14 DESC, col5_11, col1_0 ASC, col1_1 DESC, col5_19 DESC) STORING (col5_4, col5_6, col5_10, col5_17) WHERE ((((table5.col5_19 < 'X':::STRING) AND (table5.col1_0 > 3.4028234663852886e+38:::FLOAT8)) OR (table5.col5_16 != (-32768):::INT8)) OR (table5.col5_21 = e'\x00':::STRING)) OR (table5.col5_14 >= e'\\x':::BYTES), FAMILY (col5_11), FAMILY (col5_20, col1_0, col5_7, col5_5, col5_15), FAMILY (col1_1), FAMILY (col5_4, col5_19, col5_14), FAMILY (col5_13), FAMILY (col5_9), FAMILY (col5_8), FAMILY (col5_21), FAMILY (col5_16, col5_17, col5_3), FAMILY (col5_10), FAMILY (col5_18, col5_6), FAMILY (col5_12, col1_2)) INTERLEAVE IN PARENT table4 (col1_0, col1_1, col1_2);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 174119744202358028, "histo_buckets": [{"distinct_range": 0, "num_eq": 535201969307022756, "num_range": 0, "upper_bound": "-0.4783078644505018"}, {"distinct_range": 9000000000, "num_eq": 3000, "num_range": 9000000000, "upper_bound": "-0.2557540881051106"}, {"distinct_range": 0, "num_eq": 700, "num_range": 4000, "upper_bound": "-0.048314668281708495"}, {"distinct_range": 0, "num_eq": 200000000, "num_range": 6000, "upper_bound": "0.04832914053655246"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "0.3304088075643227"}, {"distinct_range": 3325173708303562000, "num_eq": 5778306592091596051, "num_range": 3325173708303562278, "upper_bound": "0.43709204836242893"}, {"distinct_range": 4695426771268536000, "num_eq": 5657164127020660301, "num_range": 4695426771268536034, "upper_bound": "2.1201822291798758"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 2820250772356432703}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1417375109207209975, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 10000000000, "num_eq": 1362495020170246076, "num_range": 10000000000, "upper_bound": "\\u000e"}, {"distinct_range": 8549565643029944000, "num_eq": 8540713039595251435, "num_range": 8549565643029943984, "upper_bound": "\\u000fl@M\\u0011"}, {"distinct_range": 90871.9346583568, "num_eq": 500000000, "num_range": 100000, "upper_bound": "A"}, {"distinct_range": 6838200253475760000, "num_eq": 500000, "num_range": 6838200253475760470, "upper_bound": "B"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 2820250772356432703}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2581502796643043453, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "NaN"}, {"distinct_range": 3672918573.760999, "num_eq": 7614619602711381205, "num_range": 10000000000, "upper_bound": "-0.5467342974721476"}, {"distinct_range": 0, "num_eq": 600, "num_range": 7539860807717507311, "upper_bound": "0.28686408142868336"}, {"distinct_range": 0, "num_eq": 5980657290632190161, "num_range": 5566575372558457327, "upper_bound": "0.8246459214361512"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 2820250772356432703}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 82157195, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 312774024, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 104150093, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 372587092, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 49911187, "histo_col_type": "", "name": "__auto__", "null_count": 393468579, "row_count": 400000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 169602693, "histo_col_type": "", "name": "__auto__", "null_count": 298907250, "row_count": 400000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 274254584, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4273774, "histo_col_type": "", "name": "__auto__", "null_count": 164535606, "row_count": 400000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 268108309, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 336586822, "histo_col_type": "", "name": "__auto__", "null_count": 9798949, "row_count": 400000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 253699921, "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 74460214, "row_count": 400000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 189521352, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "-22 years -2 mons -417 days -11:17:23.010042"}, {"distinct_range": 0, "num_eq": 900, "num_range": 0, "upper_bound": "-12 years -4 mons -983 days -13:37:09.092421"}, {"distinct_range": 2580.9858111385365, "num_eq": 600000000, "num_range": 4000, "upper_bound": "13 years 3 mons 409 days 20:21:57.430138"}, {"distinct_range": 0, "num_eq": 5043142262647409309, "num_range": 1012459815132119928, "upper_bound": "38 years 5 mons 741 days 11:24:53.170257"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 264188515, "row_count": 400000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 31234575, "histo_col_type": "", "name": "__auto__", "null_count": 391229847, "row_count": 400000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67811258, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 108824554, "histo_buckets": [{"distinct_range": 0, "num_eq": 80, "num_range": 0, "upper_bound": ""}, {"distinct_range": 1056670418675836700, "num_eq": 7626719223995162718, "num_range": 1955064529532303327, "upper_bound": "\\u0008$\x7f~k$Km\\u0011"}, {"distinct_range": 0, "num_eq": 500, "num_range": 0, "upper_bound": "\\u001c<&lP\\u0016"}, {"distinct_range": 7000000000, "num_eq": 8000000, "num_range": 7000000000, "upper_bound": "\'"}, {"distinct_range": 3894434988370337000, "num_eq": 400000000, "num_range": 8020542118854493830, "upper_bound": "H\\u000b\\u0017"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 49253185, "row_count": 400000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 191260550, "histo_col_type": "", "name": "__auto__", "null_count": 6348437, "row_count": 400000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 307904151, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 272600226, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4050, "histo_col_type": "", "name": "__auto__", "null_count": 18830, "row_count": 20000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12735, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1308, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8499, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18304, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19145, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000, "num_range": 0, "upper_bound": "00010100000110100000110100100010111"}, {"distinct_range": 7901.077917348442, "num_eq": 6366250263183583277, "num_range": 9000, "upper_bound": "00011111000011000000100011000011010"}, {"distinct_range": 0, "num_eq": 0, "num_range": 1000, "upper_bound": "01000110100000110101010111000100110"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 70000000, "upper_bound": "01011110011010011100100111001000111"}], "histo_col_type": "BIT(35)", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4375, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14510, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9047, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14751, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18299, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14724, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15984, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": "\\tXo\\tk"}, {"distinct_range": 96.7209375956227, "num_eq": 70000, "num_range": 100, "upper_bound": "\\u0019\\r8\\u0015"}, {"distinct_range": 2551880132633705500, "num_eq": 700, "num_range": 2551880132633705429, "upper_bound": "\',"}, {"distinct_range": 20000000000, "num_eq": 8482391788131785068, "num_range": 20000000000, "upper_bound": "U;"}, {"distinct_range": 4688469767559462000, "num_eq": 10000000000, "num_range": 4688469767559462043, "upper_bound": "n\\u0010~\\u0007\\u000f;:v"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 3360, "row_count": 20000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14759, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000, "num_range": 0, "upper_bound": "137.210.53.154/16"}, {"distinct_range": 0, "num_eq": 3858726341703127815, "num_range": 0, "upper_bound": "8f20:c41d:9802:cb12:c67d:e55:806a:16a4/55"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3173, "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15375, "histo_buckets": [{"distinct_range": 0, "num_eq": 1351713949401614917, "num_range": 0, "upper_bound": "\\u00105\\u0008"}, {"distinct_range": 3889754138678968000, "num_eq": 100000000000, "num_range": 3889754138678967972, "upper_bound": "=-\\u0017Yn\\tL\\u0006"}, {"distinct_range": 0, "num_eq": 90000, "num_range": 10000000000, "upper_bound": "@`\\u001eKp"}, {"distinct_range": 70000000000, "num_eq": 5900237043082808669, "num_range": 70000000000, "upper_bound": "O0&Qt8:"}, {"distinct_range": 1000000000, "num_eq": 6000000, "num_range": 1000000000, "upper_bound": "UU"}, {"distinct_range": 8107618218.937955, "num_eq": 1000, "num_range": 10000000000, "upper_bound": "W~{{\\u001c\\u0016c\\u001e"}, {"distinct_range": 45.36814750479981, "num_eq": 7000000, "num_range": 800, "upper_bound": "`8T rM"}, {"distinct_range": 4485489.95889762, "num_eq": 6594701806677452337, "num_range": 7000000, "upper_bound": "c_"}, {"distinct_range": 32865.1308949322, "num_eq": 50000, "num_range": 70000, "upper_bound": "vl7i"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15469, "histo_buckets": [{"distinct_range": 0, "num_eq": 9043036913318641870, "num_range": 0, "upper_bound": "1977-07-01"}, {"distinct_range": 0, "num_eq": 8187406374427680378, "num_range": 4000, "upper_bound": "1987-09-15"}, {"distinct_range": 1483737579966622500, "num_eq": 8000000, "num_range": 2739404697426246364, "upper_bound": "1992-04-05"}, {"distinct_range": 0, "num_eq": 2751828236026796054, "num_range": 10000, "upper_bound": "5874897-12-31"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2733, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 84, "histo_col_type": "", "name": "__auto__", "null_count": 84, "row_count": 200}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24, "histo_buckets": [{"distinct_range": 0, "num_eq": 300000000, "num_range": 0, "upper_bound": "-1.6943330546657398"}, {"distinct_range": 0, "num_eq": 5212660434184597188, "num_range": 4000, "upper_bound": "-0.8729446573150577"}, {"distinct_range": 145991723619484300, "num_eq": 3096194966419144743, "num_range": 499763554862131055, "upper_bound": "-0.6892445576836301"}, {"distinct_range": 20868.338845974547, "num_eq": 40000000000, "num_range": 30000, "upper_bound": "-0.5562629222847043"}, {"distinct_range": 3329.038957184103, "num_eq": 500000, "num_range": 5000, "upper_bound": "-0.5389995217755333"}, {"distinct_range": 204.66832964859, "num_eq": 9000000000, "num_range": 3000, "upper_bound": "0.7893301767646314"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 200}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 185, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 194, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 54, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "35929011"}, {"distinct_range": 32473495.97415807, "num_eq": 2809400557992193992, "num_range": 60000000, "upper_bound": "2406874426"}, {"distinct_range": 0, "num_eq": 5502069122382947452, "num_range": 2819800871420352760, "upper_bound": "2464986006"}, {"distinct_range": 2064158904528390000, "num_eq": 385959912551947432, "num_range": 4159630918916812036, "upper_bound": "3771044162"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 150, "row_count": 200}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 239889, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000}, {"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12308, "histo_col_type": "", "name": "__auto__", "null_count": 20943, "row_count": 1000000}, {"columns": ["col5_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 631014, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000}, {"columns": ["col5_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 301857, "histo_col_type": "", "name": "__auto__", "null_count": 663882, "row_count": 1000000}, {"columns": ["col5_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 565729, "histo_col_type": "", "name": "__auto__", "null_count": 798082, "row_count": 1000000}, {"columns": ["col5_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 68125, "histo_col_type": "", "name": "__auto__", "null_count": 622337, "row_count": 1000000}, {"columns": ["col5_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 857152, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 8798371626888315192, "num_range": 10000000000, "upper_bound": "1509324924"}, {"distinct_range": 263298746822695260, "num_eq": 4437533739177767842, "num_range": 675910711336973698, "upper_bound": "3452556712"}, {"distinct_range": 0, "num_eq": 2468844578006758969, "num_range": 1000000000, "upper_bound": "3601589662"}, {"distinct_range": 0, "num_eq": 4247273052650836124, "num_range": 8000000000, "upper_bound": "3742490211"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 70763, "row_count": 1000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 324450, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000, "num_range": 0, "upper_bound": "-1.3350707924041267"}, {"distinct_range": 5860614942793072000, "num_eq": 6774684223784897989, "num_range": 7353553787167915718, "upper_bound": "-0.37712355853186175"}, {"distinct_range": 3048544447406423000, "num_eq": 100000000, "num_range": 3383876500096381360, "upper_bound": "5e-324"}, {"distinct_range": 323620.6706875304, "num_eq": 3211515763577382931, "num_range": 400000, "upper_bound": "0.6850656881663476"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 1000000}, {"columns": ["col5_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 727399, "histo_col_type": "", "name": "__auto__", "null_count": 598626, "row_count": 1000000}, {"columns": ["col5_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 325665, "histo_buckets": [{"distinct_range": 0, "num_eq": 80, "num_range": 0, "upper_bound": "-2636485830764942654"}, {"distinct_range": 0, "num_eq": 800, "num_range": 0, "upper_bound": "-1833977389147278911"}, {"distinct_range": 389805.8042722763, "num_eq": 168253426164864016, "num_range": 700000, "upper_bound": "127"}, {"distinct_range": 0, "num_eq": 4995364619705899230, "num_range": 800000, "upper_bound": "870510494151598330"}, {"distinct_range": 0, "num_eq": 8000, "num_range": 70, "upper_bound": "1170358402497584801"}, {"distinct_range": 4984759562268451000, "num_eq": 1283236478723051365, "num_range": 4984759562268450577, "upper_bound": "5314921714191141513"}, {"distinct_range": 700000, "num_eq": 800000000, "num_range": 700000, "upper_bound": "6877865775340802647"}, {"distinct_range": 1634141736262986800, "num_eq": 4122479307824469518, "num_range": 5814224006571365985, "upper_bound": "7569874782595982005"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 1000000}, {"columns": ["col5_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 986614, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "07:57:25.973431+08:27:00"}, {"distinct_range": 0, "num_eq": 4720813740642420229, "num_range": 70000, "upper_bound": "08:46:15.804591+00:14:00"}, {"distinct_range": 98553034432295550, "num_eq": 6000, "num_range": 894591285736692483, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 590146, "row_count": 1000000}, {"columns": ["col5_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 705251, "histo_buckets": [{"distinct_range": 0, "num_eq": 8741859264297588960, "num_range": 0, "upper_bound": "-Infinity"}, {"distinct_range": 0, "num_eq": 5685600260931741833, "num_range": 4000000, "upper_bound": "-5.18867667926720465E+29"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 2072439314572082136, "upper_bound": "-8.824759951234151703E+20"}, {"distinct_range": 287.01596305397976, "num_eq": 7195809761703783369, "num_range": 800, "upper_bound": "-2279289069007048222"}, {"distinct_range": 0, "num_eq": 8350461899164426698, "num_range": 2000000, "upper_bound": "-2817927.286679948826"}, {"distinct_range": 17.06213586343087, "num_eq": 20000000, "num_range": 700, "upper_bound": "27.70473731857237464"}, {"distinct_range": 380613309158682900, "num_eq": 300, "num_range": 380613309158682894, "upper_bound": "504.7920835739288196"}, {"distinct_range": 7000, "num_eq": 7757298170399338694, "num_range": 7000, "upper_bound": "795.779305077296935"}, {"distinct_range": 0, "num_eq": 5000000000, "num_range": 0, "upper_bound": "1.663218561897193204E+32"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 1000000}, {"columns": ["col5_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 182557, "histo_col_type": "", "name": "__auto__", "null_count": 400984, "row_count": 1000000}, {"columns": ["col5_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 85698, "histo_col_type": "", "name": "__auto__", "null_count": 228042, "row_count": 1000000}, {"columns": ["col5_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 285084, "histo_col_type": "", "name": "__auto__", "null_count": 764146, "row_count": 1000000}, {"columns": ["col5_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 477545, "histo_col_type": "", "name": "__auto__", "null_count": 573170, "row_count": 1000000}, {"columns": ["col5_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 783313, "histo_col_type": "", "name": "__auto__", "null_count": 885000, "row_count": 1000000}, {"columns": ["col5_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 304000, "histo_col_type": "", "name": "__auto__", "null_count": 985851, "row_count": 1000000}, {"columns": ["col5_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 206438, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "-64 years -9 mons -553 days -01:47:16.971684"}, {"distinct_range": 0, "num_eq": 6114272729015348644, "num_range": 70000000000, "upper_bound": "-33 years -3 mons -375 days -17:39:46.06804"}, {"distinct_range": 3001765847720039000, "num_eq": 6159336251193007547, "num_range": 3001765847720038679, "upper_bound": "-16 years -2 mons -561 days -12:39:11.917916"}, {"distinct_range": 0, "num_eq": 1913266798269713643, "num_range": 90000, "upper_bound": "-4 years -5 mons -594 days -01:02:39.481212"}, {"distinct_range": 1691.399187265369, "num_eq": 200, "num_range": 7000, "upper_bound": "-2 mons -290 days -09:21:27.25112"}, {"distinct_range": 0, "num_eq": 500, "num_range": 3623731039488639879, "upper_bound": "00:00:00"}, {"distinct_range": 6476398266520846000, "num_eq": 1881082636597043127, "num_range": 6476398266520845879, "upper_bound": "35 years 1 mon 297 days 17:50:15.94829"}, {"distinct_range": 50000000, "num_eq": 3171735606307956602, "num_range": 50000000, "upper_bound": "43 years 8 mons 105 days 17:52:00.604377"}, {"distinct_range": 0, "num_eq": 8000, "num_range": 80000000, "upper_bound": "63 years 6 mons 756 days 10:17:30.897364"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 1000000}, {"columns": ["col5_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 614327, "histo_col_type": "", "name": "__auto__", "null_count": 773831, "row_count": 1000000}, {"columns": ["col5_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 130461, "histo_col_type": "", "name": "__auto__", "null_count": 427387, "row_count": 1000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 845623, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col3_14) REFERENCES table4 (col4_5) ON DELETE RESTRICT ON UPDATE RESTRICT;
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col3_1, col3_13) REFERENCES table4 (col4_5, col4_3);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('mxcrmh', 'vvjlod', 'gkzzv', 'bvjkkv', 'vd', 'vc');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('i', 's', 'mddthh', 'fjgn', 'ct', 'ihg');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('bjb');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('el', 'yv', 'pxozao', 'l', 'he', 'kymmpj');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('cdwjqn', 'zpmyes', 'dh', 'clkcxy');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5439899743698914238, "histo_buckets": [{"distinct_range": 0, "num_eq": 7525737356841425962, "num_range": 0, "upper_bound": "-0.19484756393648917"}, {"distinct_range": 0, "num_eq": 500, "num_range": 10000, "upper_bound": "0.4709246035112473"}, {"distinct_range": 0, "num_eq": 2775081249233121653, "num_range": 0, "upper_bound": "0.6945983848437012"}, {"distinct_range": 5802372164368860000, "num_eq": 4106346134450037175, "num_range": 8435701315989389387, "upper_bound": "0.7178713630922131"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 1000000, "upper_bound": "0.8839294713080953"}, {"distinct_range": 0, "num_eq": 0, "num_range": 6216648879050586023, "upper_bound": "1.7976931348623157e+308"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 6208899133208191830}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3655348331878104768, "histo_buckets": [{"distinct_range": 0, "num_eq": 4264172735511917871, "num_range": 0, "upper_bound": "\\u0013{!)"}, {"distinct_range": 0, "num_eq": 1229221573949393287, "num_range": 30, "upper_bound": "\\u001e\\u0005=\\u000b\\u0016Q"}, {"distinct_range": 0, "num_eq": 100, "num_range": 800, "upper_bound": "KX\\u0003\\u0019\\u0005]Qu="}, {"distinct_range": 2256211269625515300, "num_eq": 2400427246013269100, "num_range": 2256211269625515194, "upper_bound": "Nx"}, {"distinct_range": 769.0508991199787, "num_eq": 900, "num_range": 1000, "upper_bound": "Rc\\u0011lGPb\\u000e"}, {"distinct_range": 0, "num_eq": 2381772352642331385, "num_range": 1000000, "upper_bound": "f:Pq& V0"}, {"distinct_range": 31.81094773648675, "num_eq": 7468939494537841191, "num_range": 50, "upper_bound": "w/D,p[\\u000bD"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 6208899133208191830}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1960030090565945751, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "-2.0331211483174347"}, {"distinct_range": 200000, "num_eq": 7113418036154221526, "num_range": 200000, "upper_bound": "-1.2840906549461584"}, {"distinct_range": 774313834756038100, "num_eq": 4198192584129603036, "num_range": 6742823700967908870, "upper_bound": "-0.17588129339619557"}, {"distinct_range": 20, "num_eq": 20000, "num_range": 20, "upper_bound": "0.2191750258327223"}, {"distinct_range": 0, "num_eq": 50000000, "num_range": 300000, "upper_bound": "0.8437936426989874"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 10000, "upper_bound": "0.8438540459379862"}, {"distinct_range": 12.784056699319846, "num_eq": 1683480501935923052, "num_range": 100, "upper_bound": "1.4677310519776068"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 6208899133208191830}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6709038, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2222118, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3969803, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2300359, "histo_col_type": "", "name": "__auto__", "null_count": 2531052, "row_count": 10000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5781034, "histo_col_type": "", "name": "__auto__", "null_count": 346014, "row_count": 10000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2080041, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3763134, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000, "num_range": 0, "upper_bound": "-2.342434824752317118E+33"}, {"distinct_range": 0, "num_eq": 7386768659732393984, "num_range": 7246715761345280255, "upper_bound": "-6.508497086042259937E+21"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 70, "upper_bound": "0.2661607594136270979"}, {"distinct_range": 40000, "num_eq": 40000000000, "num_range": 40000, "upper_bound": "7.750170716443459233E+24"}, {"distinct_range": 30000000, "num_eq": 1777441218826344946, "num_range": 30000000, "upper_bound": "5.360403645140017779E+25"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 8087367, "row_count": 10000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7616681, "histo_col_type": "", "name": "__auto__", "null_count": 626630, "row_count": 10000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6485293, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4621716, "histo_col_type": "", "name": "__auto__", "null_count": 6688173, "row_count": 10000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 768849, "histo_col_type": "", "name": "__auto__", "null_count": 175802, "row_count": 10000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8816333, "histo_buckets": [{"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "\\u0002a\\u0019~l"}, {"distinct_range": 24250663.701557495, "num_eq": 5000000, "num_range": 30000000, "upper_bound": ".j$"}, {"distinct_range": 2311376186426843600, "num_eq": 2000000000, "num_range": 8392244820176317841, "upper_bound": "=EpI\\u0015\\u001e\\u0013"}, {"distinct_range": 0, "num_eq": 5367228144730737790, "num_range": 40, "upper_bound": "Va}5/\\u000ep"}, {"distinct_range": 500, "num_eq": 5959790950908219393, "num_range": 500, "upper_bound": "X"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 60, "upper_bound": "r"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 7132593, "row_count": 10000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4869950, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4381040, "histo_buckets": [{"distinct_range": 0, "num_eq": 3954033800142443274, "num_range": 0, "upper_bound": "-35 years -8 mons -984 days -14:52:12.285263"}, {"distinct_range": 0, "num_eq": 800000000, "num_range": 90000, "upper_bound": "1 mon 1 day"}, {"distinct_range": 0, "num_eq": 2665082554551512123, "num_range": 5563119455204684613, "upper_bound": "17 years 11 mons 281 days 11:45:20.726207"}, {"distinct_range": 0, "num_eq": 7972938407705596359, "num_range": 5000, "upper_bound": "24 years 1 mon 120 days 07:20:32.240106"}, {"distinct_range": 8813445007142958000, "num_eq": 800, "num_range": 8813445007142957611, "upper_bound": "25 years 7 mons 442 days 24:39:03.433666"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 606136, "row_count": 10000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4019178, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9610751, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9201771, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 888387, "histo_col_type": "", "name": "__auto__", "null_count": 3953308, "row_count": 10000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32306838373457671, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 594725010190504921}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 256987308958299842, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 594725010190504921}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 285468876033270893, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 594725010190504921}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 86064426426922099, "histo_col_type": "", "name": "__auto__", "null_count": 290103070946332487, "row_count": 594725010190504921}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 480257490004878634, "histo_buckets": [{"distinct_range": 0, "num_eq": 9187221426407884241, "num_range": 0, "upper_bound": "217.72.249.85/2"}, {"distinct_range": 30000000, "num_eq": 4443472111684595356, "num_range": 30000000, "upper_bound": "946d:e6f9:b76:2ae4:4c34:4a0c:b150:efa7/113"}, {"distinct_range": 8794987457938773000, "num_eq": 5211223748769363961, "num_range": 8932286993669517566, "upper_bound": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 594725010190504921}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 554833394628296182, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000, "num_range": 0, "upper_bound": "1980-07-22 18:52:56.000953+00:00"}, {"distinct_range": 0, "num_eq": 60000, "num_range": 0, "upper_bound": "1984-02-18 21:26:01.000757+00:00"}, {"distinct_range": 874815803.1957792, "num_eq": 4810619998922204437, "num_range": 10000000000, "upper_bound": "1991-04-10 09:40:54.000067+00:00"}, {"distinct_range": 2114928326120328200, "num_eq": 6781966462241523567, "num_range": 3391701886063354549, "upper_bound": "2011-12-29 01:20:46.000213+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 594725010190504921}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 210611696337156233, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 594725010190504921}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 557269361156332130, "histo_buckets": [{"distinct_range": 0, "num_eq": 7083671454242908958, "num_range": 0, "upper_bound": "X"}, {"distinct_range": 400000, "num_eq": 7000, "num_range": 400000, "upper_bound": "]w+\\u0002qU5c"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 594725010190504921}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 93948854033475366, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 594725010190504921}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 269113745735531661, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 594725010190504921}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 578374777065355786, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 594725010190504921}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 289788444328972520, "histo_buckets": [{"distinct_range": 0, "num_eq": 7926266161344488736, "num_range": 0, "upper_bound": "00111001110110110011101100000001101"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 90000000, "upper_bound": "01111010000111011101101011110100000"}, {"distinct_range": 0, "num_eq": 6000000, "num_range": 1882906299054673070, "upper_bound": "10011110011100110000110100111010101"}, {"distinct_range": 9000000, "num_eq": 1162532589548372739, "num_range": 9000000, "upper_bound": "11000100000110010010010000000000001"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 1000000, "upper_bound": "11010011111110100000000110010011100"}], "histo_col_type": "BIT(35)", "name": "__auto__", "null_count": 0, "row_count": 594725010190504921}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 145985425949961384, "histo_buckets": [{"distinct_range": 0, "num_eq": 4962361821048729677, "num_range": 0, "upper_bound": "\\u0015NFU"}, {"distinct_range": 3695192421042476500, "num_eq": 2920962057297231570, "num_range": 4857296786224723663, "upper_bound": "/\\u000b5\\u0003"}, {"distinct_range": 1000000000, "num_eq": 20000000000, "num_range": 1000000000, "upper_bound": "~\\n"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 75447243622067018, "row_count": 594725010190504921}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 374008163232928880, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 594725010190504921}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 464440405853971897, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 594725010190504921}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 80887946202346068, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 594725010190504921}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 344963863570137548, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000000, "num_range": 0, "upper_bound": "-infinity"}, {"distinct_range": 0, "num_eq": 7000000, "num_range": 6931594520619171472, "upper_bound": "1970-01-21"}, {"distinct_range": 15620078503.479395, "num_eq": 0, "num_range": 100000000000, "upper_bound": "1977-02-15"}, {"distinct_range": 0, "num_eq": 4581847746906401044, "num_range": 10000, "upper_bound": "1977-09-09"}, {"distinct_range": 1694614062237712600, "num_eq": 200, "num_range": 1694614062237712703, "upper_bound": "1978-09-30"}, {"distinct_range": 311836773183911800, "num_eq": 5000, "num_range": 7576269849819570195, "upper_bound": "1982-03-29"}, {"distinct_range": 80, "num_eq": 30, "num_range": 80, "upper_bound": "1997-04-12"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 594725010190504921}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 200907422013766208, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 594725010190504921}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5702, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000, "num_range": 0, "upper_bound": "-28804"}, {"distinct_range": 5000000, "num_eq": 40, "num_range": 5000000, "upper_bound": "-24663"}, {"distinct_range": 0, "num_eq": 4000000, "num_range": 600000, "upper_bound": "-15100"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 6231769694229805770, "upper_bound": "-10690"}, {"distinct_range": 0, "num_eq": 3880303378689485372, "num_range": 900000, "upper_bound": "1699"}, {"distinct_range": 2048639446589245700, "num_eq": 5918590490179461422, "num_range": 2338229887731534137, "upper_bound": "2918"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 1934, "row_count": 10000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6995, "histo_buckets": [{"distinct_range": 0, "num_eq": 4602372830347514781, "num_range": 0, "upper_bound": "-1.124623449807376"}, {"distinct_range": 864.1343589568482, "num_eq": 7523541153219461932, "num_range": 3000, "upper_bound": "-0.8782211503762842"}, {"distinct_range": 5071830239391838000, "num_eq": 6467232323437358361, "num_range": 6185535775952089084, "upper_bound": "-0.18888410588759724"}, {"distinct_range": 2641024772636965000, "num_eq": 1224101210725458509, "num_range": 3287270976195751762, "upper_bound": "1.6612290186607304"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7064, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6407, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4888, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2888, "histo_buckets": [{"distinct_range": 0, "num_eq": 350102554336118980, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 7799521989904158000, "num_eq": 5646626795764174614, "num_range": 7799521989904157543, "upper_bound": "151002036"}, {"distinct_range": 3000000000, "num_eq": 8000000000, "num_range": 3000000000, "upper_bound": "793843143"}, {"distinct_range": 0, "num_eq": 5941913555818799253, "num_range": 40000000, "upper_bound": "1734980427"}, {"distinct_range": 2608658836779643400, "num_eq": 8488267715500721565, "num_range": 8755882950230437695, "upper_bound": "2783506681"}, {"distinct_range": 657244.9467457118, "num_eq": 700, "num_range": 800000, "upper_bound": "2837352327"}, {"distinct_range": 4411451282167389700, "num_eq": 90000000000, "num_range": 4411451282167389585, "upper_bound": "3474533267"}, {"distinct_range": 0, "num_eq": 7643158462347859687, "num_range": 6000000, "upper_bound": "3602322901"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 1391418983941144370, "upper_bound": "4077729057"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 1271, "row_count": 10000}]';
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col5_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "11:21:19.783713+09:32:00"}, {"distinct_range": 3000, "num_eq": 900000, "num_range": 3000, "upper_bound": "10:37:15.406379+06:54:00"}, {"distinct_range": 745658645.3754683, "num_eq": 9134399491002832789, "num_range": 900000000, "upper_bound": "07:35:07.194456+02:13:00"}, {"distinct_range": 1000, "num_eq": 90, "num_range": 1000, "upper_bound": "07:37:23.021303-02:25:00"}, {"distinct_range": 0, "num_eq": 1011457792211034026, "num_range": 6241351805954679535, "upper_bound": "21:11:40.00794+07:19:00"}, {"distinct_range": 0, "num_eq": 8213698160786447028, "num_range": 391334285955194401, "upper_bound": "23:46:28.832713+07:10:00"}, {"distinct_range": 4000, "num_eq": 7698082140752919391, "num_range": 4000, "upper_bound": "18:09:01.853475-13:36:00"}, {"distinct_range": 6110040971385742000, "num_eq": 2710476083601641557, "num_range": 7596537254529340937, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 3710897058866963717, "num_range": 0, "upper_bound": "0.3932988371713163"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 6885000958564257839, "num_range": 0, "upper_bound": "-80 years -4 mons -329 days -00:10:11.267134"}, {"distinct_range": 10000000, "num_eq": 20000000, "num_range": 10000000, "upper_bound": "-26 years -8 mons -957 days -02:39:40.111744"}, {"distinct_range": 20000000, "num_eq": 90000000, "num_range": 20000000, "upper_bound": "-28 years -5 mons -316 days -02:17:46.307027"}, {"distinct_range": 2407044685859283000, "num_eq": 5691261550923952618, "num_range": 8485202118387553423, "upper_bound": "00:00:00"}, {"distinct_range": 7925648910734583000, "num_eq": 1645336941988737314, "num_range": 8761776326862657527, "upper_bound": "74 years 2 mons 57 days 23:32:56.338195"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 900, "num_range": 0, "upper_bound": "-2.664261617661198995E+24"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "720039844"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 900, "upper_bound": "1018237493"}, {"distinct_range": 7725296301632572000, "num_eq": 4332810719686897849, "num_range": 7725296301632572699, "upper_bound": "1184298965"}, {"distinct_range": 5115605764364632000, "num_eq": 3041238551171943864, "num_range": 8220536050841559479, "upper_bound": "2242902170"}, {"distinct_range": 10000000000, "num_eq": 20000000, "num_range": 10000000000, "upper_bound": "2526947283"}, {"distinct_range": 20, "num_eq": 8879280636075558082, "num_range": 20, "upper_bound": "2854501664"}, {"distinct_range": 6158417180240840000, "num_eq": 3458975134257174648, "num_range": 6796296416246294818, "upper_bound": "3192406580"}, {"distinct_range": 335662350.83166033, "num_eq": 200, "num_range": 400000000, "upper_bound": "3263200870"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1909254669984582324"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_cou

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2463324&tab=buildLog) on [master@f5f0f11d6e31a6667fe1cafa46cd1a472df38f76](https://github.com/cockroachdb/cockroach/commits/f5f0f11d6e31a6667fe1cafa46cd1a472df38f76):

```
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        test_1        | --- FAIL: TestCompare (2.12s)
        test_1        |     --- FAIL: TestCompare/postgres (0.93s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.2:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (1.20s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 BIT(42) NULL, col1_1 BIT(3) NOT NULL, col1_2 REGTYPE NULL, col1_3 BOOL NULL, col1_4 INT2 NOT NULL, col1_5 INT4 NOT NULL, col1_6 REGCLASS NOT NULL, col1_7 NAME, PRIMARY KEY (col1_4 ASC), UNIQUE (col1_1 ASC, col1_6, col1_3 DESC, col1_5 ASC) WHERE (((table1.col1_5 <= (-2147483648):::INT8) OR (table1.col1_7 != e'\'':::STRING)) OR (table1.col1_4 != (-1):::INT8)) OR (NOT table1.col1_3), UNIQUE (col1_3, col1_5, col1_0 DESC, col1_2, col1_6, col1_1, col1_7 ASC), UNIQUE (col1_4 ASC, col1_1 DESC) STORING (col1_5, col1_6), UNIQUE (col1_5 DESC, col1_0 ASC, col1_1) WHERE (((table1.col1_5 != 1) AND (NOT table1.col1_3)) AND (table1.col1_7 > e'\U00002603')) AND (table1.col1_4 = -1), UNIQUE (col1_1, col1_4, col1_7, col1_3 ASC, col1_0 DESC, col1_5) STORING (col1_2, col1_6), INDEX (col1_6 ASC, col1_4 DESC, col1_7 ASC), UNIQUE (col1_6 ASC, col1_3 ASC, col1_0, col1_1, col1_4 ASC, col1_7 DESC) STORING (col1_2, col1_5) WHERE (((table1.col1_7 <= '"':::STRING) OR (table1.col1_4 = (-1):::INT8)) OR (table1.col1_5 = 127:::INT8)) AND table1.col1_3, INDEX (col1_0 ASC, col1_5 DESC, col1_7) STORING (col1_2, col1_3) WHERE ((table1.col1_4 <= 0:::INT8) OR table1.col1_3) OR (table1.col1_7 >= e'\'':::STRING), INDEX (col1_3, col1_4, col1_6 DESC, col1_2, col1_1 DESC) STORING (col1_5), FAMILY (col1_2), FAMILY (col1_7), FAMILY (col1_3), FAMILY (col1_4), FAMILY (col1_6, col1_1), FAMILY (col1_5, col1_0));
        test_1        |             CREATE TABLE table2 (col2_0 DECIMAL NOT NULL, col2_1 TIMESTAMP NOT NULL, col2_2 TIMETZ NOT NULL, col2_3 "char" NOT NULL, col2_4 FLOAT8 NOT NULL, PRIMARY KEY (col2_4 DESC, col2_0, col2_2 ASC, col2_3), INDEX (col2_2, col2_0 DESC, col2_3 DESC, col2_1 DESC, col2_4 ASC), UNIQUE (col2_4), UNIQUE (col2_3 DESC, col2_4 ASC, col2_2 DESC) STORING (col2_1) WHERE ((((table2.col2_3 = e'\U00002603') AND (table2.col2_2 <= '00:00:00+15:59:00')) AND (table2.col2_4 > 1.0)) OR (table2.col2_1 = '3000-01-01 00:00:00')) AND (table2.col2_0 <= -Infinity), UNIQUE (col2_0, col2_4, col2_1) WHERE (table2.col2_2 <= '00:00:00+15:59:00') AND (table2.col2_3 = ''), INDEX (col2_1), UNIQUE (col2_3) WHERE (table2.col2_4 != 1.7976931348623157e+308:::FLOAT8) OR (table2.col2_3 <= e'\'':::STRING), UNIQUE (col2_1 DESC, col2_4 ASC, col2_2) WHERE table2.col2_3 >= '"':::STRING, UNIQUE (col2_1, col2_3 DESC, col2_4 ASC) WHERE (((table2.col2_3 >= '"') OR (table2.col2_1 >= '0001-01-01 00:00:00')) OR (table2.col2_4 < +Inf)) OR (table2.col2_2 < '24:00:00-15:59:00'), FAMILY (col2_4, col2_2), FAMILY (col2_0, col2_3), FAMILY (col2_1));
        test_1        |             CREATE TABLE table3 (col3_0 REGTYPE NOT NULL, col3_1 DECIMAL, col3_2 "char" NOT NULL, col3_3 TIMESTAMP NOT NULL, col3_4 TIMESTAMPTZ, col3_5 OID, col3_6 DATE NOT NULL, PRIMARY KEY (col3_6, col3_3 ASC, col3_2 DESC, col3_0), INDEX (col3_3 DESC, col3_2, col3_4 DESC, col3_1) STORING (col3_5) WHERE ((((table3.col3_1 = 1:::DECIMAL) AND (table3.col3_4 != '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (table3.col3_2 >= '"':::STRING)) AND (table3.col3_3 < '0001-01-01 00:00:00':::TIMESTAMP)) AND (table3.col3_6 > '-infinity':::DATE), INDEX (col3_6 ASC, col3_3, col3_0 DESC, col3_5 DESC, col3_2 DESC, col3_1 DESC) WHERE table3.col3_3 > '-2000-01-01 00:00:00', INDEX (col3_0, col3_2) STORING (col3_1), UNIQUE (col3_5 DESC, col3_0 DESC, col3_3 DESC, col3_1, col3_6 DESC) WHERE (table3.col3_4 <= '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ) OR (table3.col3_3 != '-2000-01-01 00:00:00':::TIMESTAMP), FAMILY (col3_2, col3_6), FAMILY (col3_0), FAMILY (col3_1), FAMILY (col3_3), FAMILY (col3_5, col3_4));
        test_1        |             CREATE TABLE table4 (col2_4 FLOAT8 NOT NULL, col2_0 DECIMAL NOT NULL, col2_2 TIMETZ NOT NULL, col2_3 "char" NOT NULL, col4_4 BIT(43), col4_5 CHAR NOT NULL, col4_6 "char" NOT NULL, col4_7 NAME, col4_8 INTERVAL NULL, col4_9 STRING NOT NULL, col4_10 TIMESTAMP NULL, col4_11 REGPROCEDURE, col4_12 FLOAT4 NULL, col4_13 NAME NOT NULL, col4_14 REGCLASS, col4_15 REGPROCEDURE, col4_16 UUID NOT NULL, col4_17 INT2 NOT NULL, col4_18 CHAR NULL, col4_19 TIMETZ NULL, PRIMARY KEY (col2_4 DESC, col2_0, col2_2 ASC, col2_3, col4_16 DESC, col4_13 ASC, col4_11, col4_19, col4_9, col4_10 ASC, col4_14 DESC), INDEX (col4_19, col4_14, col4_16, col4_7 DESC, col4_12 ASC, col4_13 DESC, col4_17, col2_0 DESC, col2_3 DESC, col4_10, col4_15 ASC, col2_2, col2_4 ASC, col4_18 DESC, col4_8, col4_4, col4_11 DESC), FAMILY (col4_7, col4_8), FAMILY (col4_10, col4_17), FAMILY (col4_12), FAMILY (col4_9), FAMILY (col4_6, col4_5), FAMILY (col2_0, col4_11, col2_3, col4_19, col4_16), FAMILY (col4_4), FAMILY (col4_14), FAMILY (col2_2), FAMILY (col4_15), FAMILY (col4_18), FAMILY (col2_4), FAMILY (col4_13), UNIQUE (col2_4)) INTERLEAVE IN PARENT table2 (col2_4, col2_0, col2_2, col2_3);
        test_1        |             CREATE TABLE table5 (col3_6 DATE NOT NULL, col3_3 TIMESTAMP NOT NULL, col3_2 "char" NOT NULL, col3_0 REGTYPE NOT NULL, col5_4 CHAR, col5_5 DATE NULL, col5_6 DATE NULL, col5_7 "char" NOT NULL, col5_8 REGPROCEDURE NULL, PRIMARY KEY (col3_6, col3_3 ASC, col3_2 DESC, col3_0), UNIQUE (col3_3 DESC, col5_4, col5_6 ASC) STORING (col5_8) WHERE ((((table5.col3_3 > '294276-12-31 23:59:59.999999') AND (table5.col5_4 < '"')) OR (table5.col3_2 = e'\'')) AND (table5.col5_7 <= 'X')) AND (table5.col5_6 >= '-infinity'), FAMILY (col3_6, col5_4, col3_3), FAMILY (col5_7, col3_0, col5_5, col5_6), FAMILY (col5_8, col3_2)) INTERLEAVE IN PARENT table3 (col3_6, col3_3, col3_2, col3_0);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38452, "histo_buckets": [{"distinct_range": 0, "num_eq": 303551273487993178, "num_range": 0, "upper_bound": "011100100011111100010001100111011000111010"}], "histo_col_type": "BIT(42)", "name": "__auto__", "null_count": 26218, "row_count": 50000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 43506, "histo_buckets": [{"distinct_range": 0, "num_eq": 533135609756892290, "num_range": 0, "upper_bound": "0000000000000000000000000000000000000000000000000000000000000000"}, {"distinct_range": 5388776156205611000, "num_eq": 3658257848248991411, "num_range": 5717485656126655661, "upper_bound": "011"}], "histo_col_type": "BIT(3)", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28245, "histo_col_type": "", "name": "__auto__", "null_count": 14671, "row_count": 50000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45941, "histo_buckets": [{"distinct_range": 0, "num_eq": 5723277026761009567, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 1000000, "num_eq": 300000000, "num_range": 1000000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 30044, "row_count": 50000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38139, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000000000, "num_range": 0, "upper_bound": "-27577"}, {"distinct_range": 90, "num_eq": 9100884134336471594, "num_range": 90, "upper_bound": "-17252"}, {"distinct_range": 0, "num_eq": 276260383061079723, "num_range": 5000000, "upper_bound": "-4676"}, {"distinct_range": 9000000000, "num_eq": 6000000000, "num_range": 9000000000, "upper_bound": "7273"}, {"distinct_range": 200000, "num_eq": 5422735253955741436, "num_range": 200000, "upper_bound": "15615"}, {"distinct_range": 1179.055559837885, "num_eq": 4000000000, "num_range": 2000, "upper_bound": "20529"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45062, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000000, "num_range": 0, "upper_bound": "-2147483648"}, {"distinct_range": 35158166352.68976, "num_eq": 100, "num_range": 60000000000, "upper_bound": "-2050248412"}, {"distinct_range": 47545041.0078088, "num_eq": 1050824073474820642, "num_range": 100000000, "upper_bound": "-1806703783"}, {"distinct_range": 3959458921219536400, "num_eq": 4530201850622996056, "num_range": 3959458921219536310, "upper_bound": "-1175824059"}, {"distinct_range": 8109422266646277000, "num_eq": 30, "num_range": 8109422266646276949, "upper_bound": "-1144636010"}, {"distinct_range": 0, "num_eq": 40000000000, "num_range": 500000, "upper_bound": "377758535"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30379, "histo_buckets": [{"distinct_range": 0, "num_eq": 3250791081144293343, "num_range": 0, "upper_bound": "1335902993"}, {"distinct_range": 150630282989266940, "num_eq": 0, "num_range": 165261195274510437, "upper_bound": "3345405991"}, {"distinct_range": 0, "num_eq": 800, "num_range": 0, "upper_bound": "3446486071"}, {"distinct_range": 32343287.103251234, "num_eq": 4277560488002641783, "num_range": 60000000, "upper_bound": "3536068871"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38997, "histo_col_type": "", "name": "__auto__", "null_count": 33967, "row_count": 50000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 2612509484325623486, "num_range": 0, "upper_bound": "580220203.927155029"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 1936126331171517224, "num_range": 0, "upper_bound": "1995-01-12 01:34:50.000208"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000000, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 300000, "num_eq": 5620102111051315519, "num_range": 300000, "upper_bound": "06:33:58.504205+00:22:00"}, {"distinct_range": 546947.0145287862, "num_eq": 5000000, "num_range": 1000000, "upper_bound": "07:03:02.65145-01:42:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 8003114069639856029, "num_range": 0, "upper_bound": "\\u001c\\nW*\\u000b\\r\\u001cx"}, {"distinct_range": 4059460390937346600, "num_eq": 4748903860721992053, "num_range": 4059460390937346559, "upper_bound": "\\u001d\\u00176Di:t"}, {"distinct_range": 1316523814942096600, "num_eq": 30000, "num_range": 3669058835385977361, "upper_bound": "6cF{:\\rG*"}, {"distinct_range": 3325754385222862300, "num_eq": 2000, "num_range": 9150563507179638882, "upper_bound": "9r"}, {"distinct_range": 2154589674233771800, "num_eq": 4258077004970294405, "num_range": 2154589674233771794, "upper_bound": "<\\u0006</I\\u00172"}, {"distinct_range": 0, "num_eq": 80, "num_range": 8338213247894189025, "upper_bound": "fU65"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000000, "num_range": 0, "upper_bound": "-2.2494579923371747"}, {"distinct_range": 0, "num_eq": 300, "num_range": 0, "upper_bound": "-0.2929639828584653"}, {"distinct_range": 159395649.8506394, "num_eq": 10000, "num_range": 600000000, "upper_bound": "0.02687538623804414"}, {"distinct_range": 1470489824887965200, "num_eq": 3245338260237846049, "num_range": 1470489824887965152, "upper_bound": "0.2796726077446061"}, {"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": "0.3008827926449631"}, {"distinct_range": 94539750.01018149, "num_eq": 100000, "num_range": 100000000, "upper_bound": "0.3263759067436274"}, {"distinct_range": 946022300165728400, "num_eq": 4000, "num_range": 946022300165728344, "upper_bound": "0.8044672412048702"}, {"distinct_range": 1225.441282117332, "num_eq": 8715977589085100670, "num_range": 6000, "upper_bound": "1.0"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67, "histo_col_type": "", "name": "__auto__", "null_count": 2007, "row_count": 4000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2142, "histo_col_type": "OID", "name": "__auto__", "null_count": 2995, "row_count": 4000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3624, "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1307, "histo_buckets": [{"distinct_range": 0, "num_eq": 6141098608837332511, "num_range": 0, "upper_bound": "14009460"}, {"distinct_range": 4836430903069677000, "num_eq": 400000000, "num_range": 4836430903069676198, "upper_bound": "151413897"}, {"distinct_range": 9090106508618605000, "num_eq": 5000, "num_range": 9090106508618604164, "upper_bound": "160455972"}, {"distinct_range": 7510968295616809000, "num_eq": 9000000, "num_range": 7510968295616809445, "upper_bound": "1152915366"}, {"distinct_range": 0, "num_eq": 90000000000, "num_range": 591363531298616273, "upper_bound": "1576975713"}, {"distinct_range": 2074822040893910800, "num_eq": 6178931636980758715, "num_range": 2074822040893910782, "upper_bound": "2367613253"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 50000000000, "upper_bound": "3609526275"}, {"distinct_range": 800000000, "num_eq": 6000, "num_range": 800000000, "upper_bound": "3707367097"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1380, "histo_col_type": "", "name": "__auto__", "null_count": 1886, "row_count": 4000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 261, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3705, "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 4000}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000, "num_range": 0, "upper_bound": "-0.5829612546276719"}, {"distinct_range": 35947.23475589447, "num_eq": 2572548069704145681, "num_range": 40000, "upper_bound": "0.035217661004476986"}, {"distinct_range": 0, "num_eq": 1825517593489879264, "num_range": 30000000, "upper_bound": "0.13672950431310074"}, {"distinct_range": 3219675907706397700, "num_eq": 0, "num_range": 4055591110490822793, "upper_bound": "0.141573480056297"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 70}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67, "histo_col_type": "", "name": "__auto__", "null_count": 5, "row_count": 70}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 58, "histo_col_type": "", "name": "__auto__", "null_count": 8, "row_count": 70}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14, "histo_col_type": "", "name": "__auto__", "null_count": 32, "row_count": 70}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 63, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 52, "histo_col_type": "", "name": "__auto__", "null_count": 2, "row_count": 70}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70}, {"columns": ["col4_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 64, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "03:08:40.938935-01:25:00"}, {"distinct_range": 363026869034449800, "num_eq": 0, "num_range": 363026869034449814, "upper_bound": "08:53:06.675209+02:36:00"}, {"distinct_range": 500, "num_eq": 20000000, "num_range": 500, "upper_bound": "05:51:55.552169-12:20:00"}, {"distinct_range": 0, "num_eq": 2590816180971387117, "num_range": 2791241722740147484, "upper_bound": "17:09:25.347153-04:04:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 65, "row_count": 70}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32, "histo_col_type": "", "name": "__auto__", "null_count": 16, "row_count": 70}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22, "histo_col_type": "", "name": "__auto__", "null_count": 63, "row_count": 70}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 40, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 62, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 43, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24, "histo_col_type": "", "name": "__auto__", "null_count": 22, "row_count": 70}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 69, "histo_col_type": "", "name": "__auto__", "null_count": 40, "row_count": 70}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 59, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70}, {"columns": ["col4_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 68, "histo_col_type": "", "name": "__auto__", "null_count": 11, "row_count": 70}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4843, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000, "num_range": 0, "upper_bound": "1976-04-07 00:13:41.000328"}, {"distinct_range": 0, "num_eq": 6444966221124258919, "num_range": 0, "upper_bound": "1991-05-25 16:00:31.000748"}, {"distinct_range": 4155009859706443000, "num_eq": 80, "num_range": 8522120416740425146, "upper_bound": "2005-03-04 07:48:24.000106"}, {"distinct_range": 0, "num_eq": 2590271069293729760, "num_range": 400000000, "upper_bound": "2015-11-09 02:28:16.000852"}, {"distinct_range": 7356029075063162000, "num_eq": 500000000, "num_range": 7356029075063162072, "upper_bound": "2017-12-26 21:51:21.000028"}, {"distinct_range": 360468881263092300, "num_eq": 10000, "num_range": 446175522294603910, "upper_bound": "2023-05-18 19:29:42.000605"}, {"distinct_range": 100000000, "num_eq": 6000000, "num_range": 100000000, "upper_bound": "2024-11-20 12:14:54.000957"}, {"distinct_range": 0, "num_eq": 8613087452167969637, "num_range": 9000000000, "upper_bound": "2032-01-23 10:26:36.000025"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 509, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col5_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7, "histo_col_type": "", "name": "__auto__", "null_count": 1964, "row_count": 5000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4678, "histo_buckets": [{"distinct_range": 0, "num_eq": 30, "num_range": 0, "upper_bound": "1974-06-03"}, {"distinct_range": 0, "num_eq": 6386368615238804340, "num_range": 4511675670214227852, "upper_bound": "1976-03-28"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 0, "upper_bound": "1978-03-18"}, {"distinct_range": 109987947854289900, "num_eq": 8756321179968425046, "num_range": 109987947854289911, "upper_bound": "1986-06-28"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 104, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col5_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4559, "histo_col_type": "", "name": "__auto__", "null_count": 2221, "row_count": 5000}, {"columns": ["col5_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 169, "histo_col_type": "", "name": "__auto__", "null_count": 3037, "row_count": 5000}, {"columns": ["col5_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 783, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col5_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2345, "histo_col_type": "", "name": "__auto__", "null_count": 1608, "row_count": 5000}]':::JSONB;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_4) REFERENCES table4 (col2_4);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('kgq', 'llolpe');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('blmxe', 'kj', 'esf');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('xm', 'o', 'wzh', 'xpwhao');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('s', 'bmu', 'gtigr');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('lsvrf', 'a', 'eahh', 'w', 'i', 'io');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1099944009257850034, "histo_col_type": "", "name": "__auto__", "null_count": 466070534678250573, "row_count": 2233620062230594975}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 189192266457435155, "histo_buckets": [{"distinct_range": 0, "num_eq": 7304770746641611933, "num_range": 0, "upper_bound": "000100000010110100011111110101110010010010"}, {"distinct_range": 692100850417721900, "num_eq": 900000, "num_range": 1213008003788081893, "upper_bound": "001011000100111010010001110111001000111110"}, {"distinct_range": 0, "num_eq": 8707286934405866496, "num_range": 1000, "upper_bound": "111101010010110111000000000011100010101011"}], "histo_col_type": "BIT(42)", "name": "__auto__", "null_count": 649214031144242028, "row_count": 2233620062230594975}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 757910240829905187, "histo_col_type": "BIT(3)", "name": "__auto__", "null_count": 0, "row_count": 2233620062230594975}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1094189898482354124, "histo_col_type": "", "name": "__auto__", "null_count": 151573552455237481, "row_count": 2233620062230594975}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1015570486178589640, "histo_buckets": [{"distinct_range": 0, "num_eq": 710788293677509290, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 7166809033620931332, "num_range": 2573105778421937131, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 151416529239884861, "row_count": 2233620062230594975}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 962709964851194130, "histo_buckets": [{"distinct_range": 0, "num_eq": 3495612748423271909, "num_range": 0, "upper_bound": "-22290"}, {"distinct_range": 0, "num_eq": 4702580014198783944, "num_range": 500000, "upper_bound": "-21110"}, {"distinct_range": 5000000000, "num_eq": 6749716021206829877, "num_range": 5000000000, "upper_bound": "-17075"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 6966173794345262970, "upper_bound": "-13530"}, {"distinct_range": 1036379603737948900, "num_eq": 6000000, "num_range": 1036379603737948901, "upper_bound": "11544"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 2471132994451900280, "upper_bound": "12277"}, {"distinct_range": 6000000000, "num_eq": 300000000, "num_range": 6000000000, "upper_bound": "19336"}, {"distinct_range": 73.62655302673498, "num_eq": 8000000, "num_range": 400, "upper_bound": "32767"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 2233620062230594975}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 732485261811284045, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "-1874259005"}, {"distinct_range": 0, "num_eq": 30000, "num_range": 0, "upper_bound": "-1663165268"}, {"distinct_range": 0, "num_eq": 5130315619840114112, "num_range": 7972523573283431006, "upper_bound": "-1228639090"}, {"distinct_range": 2203391322339545300, "num_eq": 70, "num_range": 8733366621120900526, "upper_bound": "1"}, {"distinct_range": 5000000000, "num_eq": 5539219137535546428, "num_range": 5000000000, "upper_bound": "1524180293"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 2233620062230594975}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1888854220967864727, "histo_buckets": [{"distinct_range": 0, "num_eq": 3209739761549800469, "num_range": 0, "upper_bound": "719348476"}, {"distinct_range": 14.26477521195636, "num_eq": 30000, "num_range": 80, "upper_bound": "1237020253"}, {"distinct_range": 0, "num_eq": 6650999795873694812, "num_range": 400, "upper_bound": "2903920191"}, {"distinct_range": 0, "num_eq": 5024861784970742459, "num_range": 0, "upper_bound": "3161616295"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 5071349441768862361, "upper_bound": "3298842988"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 2233620062230594975}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5892781, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "-7.7255787861289737E+32"}, {"distinct_range": 6021.772047352625, "num_eq": 3252231864460006824, "num_range": 10000, "upper_bound": "-5.487151265741460496E+21"}, {"distinct_range": 4218019278563033000, "num_eq": 100, "num_range": 7207946646899195458, "upper_bound": "-51866693191.54071705"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 100000000000, "upper_bound": "4.724041589729500908E+21"}, {"distinct_range": 0, "num_eq": 5000000, "num_range": 0, "upper_bound": "4.266339962845769814E+31"}, {"distinct_range": 0, "num_eq": 4535981187749140176, "num_range": 20000000, "upper_bound": "8.635131601489353133E+32"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3442280, "histo_buckets": [{"distinct_range": 0, "num_eq": 1799820115210435175, "num_range": 0, "upper_bound": "1979-10-27 10:49:05.000562"}, {"distinct_range": 1093462309022101000, "num_eq": 400, "num_range": 5938821899551663741, "upper_bound": "1982-02-27 03:13:13.000635"}, {"distinct_range": 39071098.14120978, "num_eq": 200000, "num_range": 70000000, "upper_bound": "2004-02-29 18:33:09.000067"}, {"distinct_range": 7185151046332185000, "num_eq": 9043621721455979637, "num_range": 7185151046332184805, "upper_bound": "2015-04-08 12:45:10.000642"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "2020-08-03 13:35:44.000988"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 177309, "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 786498, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000000, "num_range": 0, "upper_bound": "\\u0008^\\u000b\\u001b<F"}, {"distinct_range": 120162657200917600, "num_eq": 3283908604232665462, "num_range": 5004731002707346598, "upper_bound": "&nC=5\\u00055[C"}, {"distinct_range": 2424199171390108000, "num_eq": 332474490879933478, "num_range": 2424199171390108081, "upper_bound": "\'arT\\u0019P!"}, {"distinct_range": 400000000, "num_eq": 500000000, "num_range": 400000000, "upper_bound": "G\\u0007\\u0006!"}, {"distinct_range": 0, "num_eq": 0, "num_range": 7911863206366399294, "upper_bound": "II3\\u0016n|W\\u0018"}, {"distinct_range": 675719.2972717041, "num_eq": 7000000000, "num_range": 4000000, "upper_bound": "K]tK\\u0013|\\u0007"}, {"distinct_range": 1147563621.7777784, "num_eq": 4170360576027034348, "num_range": 2000000000, "upper_bound": "f`v"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7609724, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "-1.3159051267493327"}, {"distinct_range": 1092693590987515300, "num_eq": 2000000000, "num_range": 1092693590987515261, "upper_bound": "-0.5668487579073066"}, {"distinct_range": 8339360733869269000, "num_eq": 2000, "num_range": 9165747779002002751, "upper_bound": "-0.17710922870355517"}, {"distinct_range": 60000000, "num_eq": 0, "num_range": 60000000, "upper_bound": "0.5069482096281768"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 50000, "upper_bound": "0.9875987075672422"}, {"distinct_range": 0, "num_eq": 40, "num_range": 10000, "upper_bound": "1.1118603107693255"}, {"distinct_range": 0, "num_eq": 8547089779937647263, "num_range": 0, "upper_bound": "1.287902604999581"}, {"distinct_range": 100000, "num_eq": 4651557781478150779, "num_range": 100000, "upper_bound": "3.707282882905137"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 8000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 116, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "225933698"}, {"distinct_range": 1286077.3068469192, "num_eq": 50000, "num_range": 2000000, "upper_bound": "4226961309"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 107, "histo_col_type": "", "name": "__auto__", "null_count": 145, "row_count": 400}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 283, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 272, "histo_buckets": [{"distinct_range": 0, "num_eq": 8482160877373991097, "num_range": 0, "upper_bound": "-2000-01-01 00:00:00"}, {"distinct_range": 2365002336703926300, "num_eq": 20000000000, "num_range": 4737391636290575545, "upper_bound": "1975-07-14 15:44:11.000508"}, {"distinct_range": 700000, "num_eq": 3875308607780084138, "num_range": 700000, "upper_bound": "1996-04-24 06:41:15.00054"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 5001678442238137616, "upper_bound": "2000-07-05 00:39:40.000249"}, {"distinct_range": 1792828518546783500, "num_eq": 6661573383979815285, "num_range": 1853318285259078595, "upper_bound": "3000-01-01 00:00:00"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 384, "histo_col_type": "", "name": "__auto__", "null_count": 272, "row_count": 400}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 358, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 900000, "num_eq": 80000000, "num_range": 900000, "upper_bound": "640527113"}, {"distinct_range": 499339638899130300, "num_eq": 0, "num_range": 499339638899130329, "upper_bound": "1219694003"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 319, "row_count": 400}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 327, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": "1977-07-15"}, {"distinct_range": 686953628557331800, "num_eq": 8000000000, "num_range": 3307082795120620465, "upper_bound": "1980-08-12"}, {"distinct_range": 0, "num_eq": 5000000, "num_range": 20000000000, "upper_bound": "1990-10-10"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 400}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 655304570, "histo_col_type": "", "name": "__auto__", "null_count": 854522336, "row_count": 3000000000}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1377148982, "histo_col_type": "", "name": "__auto__", "null_count": 132224632, "row_count": 3000000000}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1740741126, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000000}, {"columns": ["col4_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 347586572, "histo_buckets": [{"distinct_range": 0, "num_eq": 5337689464116666147, "num_range": 0, "upper_bound": "02:30:48.775701+10:41:00"}, {"distinct_range": 1000000, "num_eq": 1497735787371142854, "num_range": 1000000, "upper_bound": "17:22:54.125693+10:01:00"}, {"distinct_range": 0, "num_eq": 6354959204744039774, "num_range": 60000000, "upper_bound": "01:46:27.702501-07:41:00"}, {"distinct_range": 0, "num_eq": 7493134566260768570, "num_range": 1000000, "upper_bound": "07:17:33.425195-09:02:00"}, {"distinct_range": 4587247704517583400, "num_eq": 4648680966681172261, "num_range": 4587247704517583487, "upper_bound": "23:34:17.238686+01:23:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 1779366240, "row_count": 3000000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1405815993, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1062070994, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2586711659, "histo_col_type": "", "name": "__auto__", "null_count": 1417884276, "row_count": 3000000000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1993020817, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000000}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22029342, "histo_col_type": "", "name": "__auto__", "null_count": 2587965466, "row_count": 3000000000}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1196381337, "histo_col_type": "", "name": "__auto__", "null_count": 2504683912, "row_count": 3000000000}, {"columns": ["col4_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2353062591, "histo_col_type": "", "name": "__auto__", "null_count": 1364007759, "row_count": 3000000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1470126985, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 921551173, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000000}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2373968389, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000000}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2658127785, "histo_col_type": "", "name": "__auto__", "null_count": 1864476911, "row_count": 3000000000}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 276697974, "histo_col_type": "", "name": "__auto__", "null_count": 2544592624, "row_count": 3000000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1991375793, "histo_buckets": [{"distinct_range": 0, "num_eq": 300000000, "num_range": 0, "upper_bound": "-1.8154579327375315"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "-0.7738341461679968"}, {"distinct_range": 80000, "num_eq": 5652536838182760664, "num_range": 80000, "upper_bound": "-0.6717511688203686"}, {"distinct_range": 2986.647335640384, "num_eq": 900, "num_range": 5000, "upper_bound": "-0.6692684898785672"}, {"distinct_range": 6333058543608799000, "num_eq": 2134566064688846562, "num_range": 6333058543608798849, "upper_bound": "-0.39605021457882716"}, {"distinct_range": 8000000000, "num_eq": 3217893895423190449, "num_range": 8000000000, "upper_bound": "-0.008337386543390557"}, {"distinct_range": 0, "num_eq": 50000000, "num_range": 0, "upper_bound": "1.401298464324817e-45"}, {"distinct_range": 221696161173521920, "num_eq": 60000, "num_range": 249506028583251737, "upper_bound": "0.07332095066076949"}, {"distinct_range": 0, "num_eq": 3000000, "num_range": 3000000000, "upper_bound": "0.5301642826737087"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 3000000000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 494404923, "histo_col_type": "", "name": "__auto__", "null_count": 2433361342, "row_count": 3000000000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1674832336, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000000}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2625166870, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000000}]';
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 146201152118480176, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 703918821622408502}, {"columns": ["col5_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 396794174602770100, "histo_col_type": "", "name": "__auto__", "null_count": 521867829125208671, "row_count": 703918821622408502}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 385186913430378094, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 703918821622408502}, {"columns": ["col5_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 355812093562222228, "histo_col_type": "", "name": "__auto__", "null_count": 234745263031344463, "row_count": 703918821622408502}, {"columns": ["col5_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 555354944365137609, "histo_col_type": "", "name": "__auto__", "null_count": 169377616660884906, "row_count": 703918821622408502}, {"columns": ["col5_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 304005326671759826, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 703918821622408502}, {"columns": ["col5_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 151687691607612424, "histo_col_type": "", "name": "__auto__", "null_count": 283996177163495057, "row_count": 703918821622408502}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 640508498775965192, "histo_buckets": [{"distinct_range": 0, "num_eq": 6260928086094894246, "num_range": 0, "upper_bound": "1971-08-01"}, {"distinct_range": 0, "num_eq": 2164851199248014171, "num_range": 90000000000, "upper_bound": "1972-02-13"}, {"distinct_range": 5169673486024617000, "num_eq": 6000000000, "num_range": 5416275082676083036, "upper_bound": "1972-03-19"}, {"distinct_range": 852819.066936008, "num_eq": 3000000000, "num_range": 1000000, "upper_bound": "1986-09-21"}, {"distinct_range": 0, "num_eq": 100, "num_range": 10000000, "upper_bound": "1989-06-02"}, {"distinct_range": 3000, "num_eq": 5007920438759834109, "num_range": 3000, "upper_bound": "1993-10-18"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 703918821622408502}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1556992518940796, "histo_buckets": [{"distinct_range": 0, "num_eq": 3876542269425977151, "num_range": 0, "upper_bound": "0001-01-01 00:00:00"}, {"distinct_range": 30378208.613791898, "num_eq": 300000000, "num_range": 50000000, "upper_bound": "1974-12-14 10:05:30.000765"}, {"distinct_range": 0, "num_eq": 4309343927247513159, "num_range": 0, "upper_bound": "1986-06-05 06:32:15.000322"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 1181443008609491539, "upper_bound": "1994-11-10 19:27:53.000926"}, {"distinct_range": 1109443810820214300, "num_eq": 6409627966027355006, "num_range": 5078376317081293100, "upper_bound": "2021-01-09 01:08:42.000658"}, {"distinct_range": 400, "num_eq": 7000000, "num_range": 400, "upper_bound": "294276-12-31 23:59:59.999999"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 703918821622408502}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 300000000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 8000000, "num_eq": 10000000000, "num_range": 8000000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-819"}, {"distinct_range": 2782045371818389000, "num_eq": 60000, "num_range": 2782045371818389163, "upper_bound": "2049"}, {"distinct_range": 3000, "num_eq": 2000, "num_range": 3000, "upper_bound": "4492"}, {"distinct_range": 3179159398439377000, "num_eq": 10000000, "num_range": 7605638960958855144, "upper_bound": "12458"}, {"distinct_range": 10000, "num_eq": 500, "num_range": 10000, "upper_bound": "19168"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000000, "num_range": 0, "upper_bound": "-2147483648"}, {"distinct_range": 4091704351473357300, "num_eq": 611765452983040965, "num_range": 6790118553181660439, "upper_bound": "-1937895032"}, {"distinct_range": 7000, "num_eq": 30000, "num_range": 7000, "upper_bound": "-1189582037"}, {"distinct_range": 0, "num_eq": 5041711026249427506, "num_range": 0, "upper_bound": "-1037825613"}, {"distinct_range": 1660810010503215900, "num_eq": 7400439610924614999, "num_range": 1660810010503215770, "upper_bound": "-695918027"}, {"distinct_range": 112384882363779230, "num_eq": 30000000, "num_range": 3703774697268798247, "upper_bound": "-41609182"}, {"distinct_range": 0, "num_eq": 7782800155893970802, "num_range": 7000, "upper_bound": "32767"}, {"distinct_range": 3678052393631018000, "num_eq": 2937552354558109550, "num_range": 3678052393631017998, "upper_bound": "676500056"}, {"distinct_range": 0, "num_eq": 5471408444896526070, "num_range": 200000, "upper_bound": "1936161192"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000, "num_range": 0, "upper_bound": "2695601970"}, {"distinct_range": 1233349768512650000, "num_eq": 6000000000, "num_range": 1233349768512650046, "upper_bound": "4128692735"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 7348692534238353117, "num_range": 0, "upper_bound": "011110100111010010011001110100010111011111"}], "histo_col_type": "BIT(42)", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 7884482893522308404, "num_range": 0, "upper_bound": "001"}, {"distinct_range": 1995.1680524849983, "num_eq": 40000, "num_range": 7000, "upper_bound": "101"}, {"distinct_range": 1000000, "num_eq": 700000, "num_range": 1000000, "upper_bound": "110"}, {"distinct_range": 1117782285768769800, "num_eq": 800000000, "num_range": 1117782285768769846, "upper_bound": "111"}], "histo_col_type": "BIT(3)", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3271166, "histo_buckets": [{"distinct_range": 0, "num_eq": 6016850351900742183, "num_range": 0, "upper_bound": "-8.368988903140693305E+36"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 100000, "upper_bound": "-5.169557482020200159E+26"}, {"distinct_range": 2285353109601282000, "num_eq": 8982206231276776606, "num_range": 2285353109601282153, "upper_bound": "-51025.6536374852664"}, {"distinct_range": 5905066060278297000, "num_eq": 4217017864689241220, "num_range": 8546526229475102115, "upper_bound": "-662.9080117440477124"}, {"distinct_range": 879713061890604500, "num_eq": 225644307262484402, "num_range": 4204579775097447434, "upper_bound": "72211.75660004742551"}, {"distinct_range": 8413092599924103000, "num_eq": 2573326671849497294, "num_range": 8413092599924102876, "upper_bound": "5329609314823598.714"}, {"distinct_range": 361581807.8300372, "num_eq": 7000000000, "num_range": 400000000, "upper_bound": "7.513472389773854651E+27"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 881835212404806005, "upper_bound": "3.22435506344907355E+36"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1543714, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000000, "num_range": 0, "upper_bound": "1973-06-14 10:01:52.00093"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6675293, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "00:03:28.491175+11:14:00"}, {"distinct_range": 0, "num_eq": 200, "num_range": 30, "upper_bound": "03:27:35.017854+12:53:00"}, {"distinct_range": 0, "num_eq": 7548082541885130895, "num_range": 8755256064839905003, "upper_bound": "10:12:50.831586+09:02:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2386361, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "5\\u000bAH2"}, {"distinct_range": 4868.271579860402, "num_eq": 1066781995618309153, "num_range": 50000, "upper_bound": "N2=\\u001f|I"}, {"distinct_range": 1695659430107752200, "num_eq": 100000, "num_range": 4723298453862516722, "upper_bound": "QzA\\u0014\\u0002&"}, {"distinct_range": 100000, "num_eq": 1000, "num_range": 100000, "upper_bound": "U\\u0001="}, {"distinct_range": 500, "num_eq": 5149265232888828469, "num_range": 500, "upper_bound": "\U00002603"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2588274, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "0.7061132051343645"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 7000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1530741688191461211, "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 2791825764275095220}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1434070041532908208, "histo_col_type": "", "name": "__auto__", "null_count": 198233129536867269, "row_count": 2791825764275095220}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2637717833894769682, "histo_buckets": [{"distinct_range": 0, "num_eq": 5267774203709367488, "num_range": 0, "upper_bound": "70195573"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 5000, "upper_bound": "1786069915"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 2349918573175331205, "row_count": 2791825764275095220}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1460735288070562823, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000, "num_range": 0, "upper_bound": "4714-11-24 BC"}, {"distinct_range": 7930059605961801000, "num_eq": 1505603542018211331, "num_range": 7930059605961800648, "upper_bound": "1983-01-07"}, {"distinct_range": 5111688109767084000, "num_eq": 7018264818001487421, "num_range": 5111688109767083676, "upper_bound": "1984-10-24"}, {"distinct_range": 1890925421183887600, "num_eq": 3856721941627832769, "num_range": 1890925421183887683, "upper_bound": "1985-11-02"}, {"distinct_range": 826934751.3159742, "num_eq": 400000000, "num_range": 900000000, "upper_bound": "1990-10-31"}, {"distinct_range": 6.780604938604418, "num_eq": 1910073293204217415, "num_range": 300, "upper_bound": "1991-04-10"}, {"distinct_range": 0, "num_eq": 8407768461734034615, "num_range": 4668485717180661867, "upper_bound": "1996-11-14"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 2791825764275095220}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1560245006378508806, "histo_buckets": [{"distinct_range": 0, "num_eq": 327815987180523074, "num_range": 0, "upper_bound": "424387942"}, {"distinct_range": 400000000, "num_eq": 4554283857390239349, "num_range": 400000000, "upper_bound": "476768155"}, {"distinct_range": 3968501052800293400, "num_eq": 8272176848194715703, "num_range": 3968501052800293397, "upper_bound": "992950431"}, {"distinct_range": 471307229634244740, "num_eq": 5609477197209552288, "num_range": 1255423889486206404, "upper_bound": "2935868846"}, {"distinct_range": 0, "num_eq": 400000, "num_range": 50000000000, "upper_bound": "3391366988"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 2791825764275095220}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2479109876535193323, "histo_col_type": "", "name": "__auto__", "null_count": 681837149427608418, "row_count": 2791825764275095220}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1963786928311915575, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2791825764275095220}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 287580, "histo_col_type": "", "name": "__auto__", "null_count": 360608, "row_count": 400000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 218093, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 284296, "histo_col_type": "", "name": "__auto__", "null_count": 381066, "row_count": 400000}, {"columns": ["col4_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 334780, "histo_col_type": "", "name": "__auto__", "null_count": 131256, "row_count": 400000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 103072, "histo_buckets": [{"distinct_range": 0, "num_eq": 500000, "num_range": 0, "upper_bound": "-1.3906589055915117"}, {"distinct_range": 23660.017970047356, "num_eq": 4385487230158331124, "num_range": 80000, "upper_bound": "-0.7404195463466915"}, {"distinct_range": 4344098356938539500, "num_eq": 10000000, "num_range": 4344098356938539741, "upper_bound": "-0.6630541830027257"}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 4222461728320885760, "upper_bound": "-0.6395153671917714"}, {"distinct_range": 188104.458531967, "num_eq": 30000, "num_range": 600000, "upper_bound": "0.13851277490448818"}, {"distinct_range": 0, "num_eq": 2000000000, "num_range": 1059235211188882121, "upper_bound": "0.1670002860944504"}, {"distinct_range": 3426793001867138600, "num_eq": 3058492922699962254, "num_range": 3426793001867138497, "upper_bound": "0.29572140798986873"}, {"distinct_range": 5419706578726236000, "num_eq": 4260290978747193725, "num_range": 8308679552429901390, "upper_bound": "1.0818304663434128"}, {"distinct_range": 0, "num_eq": 60000000000, "num_range": 800, "upper_bound": "1.1457445370686714"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 400000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 108815, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 145322, "histo_col_type": "", "name": "__auto__", "null_count": 294967, "row_count": 400000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 362469, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 340006, "histo_col_type": "", "name": "__auto__", "null_count": 185343, "row_count": 400000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 154928, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 73231, "histo_col_type": "", "name": "__auto__", "null_count": 247366, "row_count": 400000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 355619, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 396858, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 380986, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 147848, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 165935, "histo_col_type": "", "name": "__auto__", "null_count": 69523, "row_count": 400000}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 78744, "histo_col_type": "", "name": "__auto__", "null_count": 208662, "row_count": 400000}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13201, "histo_col_type": "", "name": "__auto__", "null_count": 211149, "row_count": 400000}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 219000, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000}, {"columns": ["col4_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 137044, "histo_buckets": [{"distinct_range": 0, "num_eq": 5189974718065799548, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 0, "upper_bound": "09:34:32.150715+13:11:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 235075, "row_count": 400000}]';
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 245471890409588846, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "4714-11-24 BC"}, {"distinct_range": 189197821666347200, "num_eq": 6795896909676096935, "num_range": 927527193234157845, "upper_bound": "1985-09-23"}, {"distinct_range": 100000000000, "num_eq": 0, "num_range": 100000000000, "upper_bound": "1989-08-17"}, {"distinct_range": 148.37203484283333, "num_eq": 22946262523

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2465497&tab=buildLog) on [master@c8e6237257a15f294cfb3a2676528f19e17f829f](https://github.com/cockroachdb/cockroach/commits/c8e6237257a15f294cfb3a2676528f19e17f829f):

```
        test_1        | --- FAIL: TestCompare (31.19s)
        test_1        |     --- FAIL: TestCompare/postgres (30.16s)
        test_1        |         compare_test.go:155: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |               	float64(9),
        test_1        |             - 	string("9.0A"),
        test_1        |             + 	string("9A"),
        test_1        |               }))
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT 9.0::FLOAT8 AS col_44980, (9.0::FLOAT8::FLOAT8 || 'A'::TEXT::TEXT)::TEXT AS col_44981) AS tab_23631
        test_1        |             ORDER BY
        test_1        |             	col_44980 NULLS FIRST, col_44981 NULLS FIRST;
        test_1        |             
        test_1        |             cockroach1: [same as previous]
        test_1        |             
        test_1        |     --- FAIL: TestCompare/mutators (1.03s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 REGCLASS NOT NULL, col1_1 TIMESTAMP NOT NULL, col1_2 DECIMAL NOT NULL, col1_3 TIME, col1_4 BIT(21) NOT NULL, col1_5 TIMESTAMPTZ NOT NULL, col1_6 BOOL, col1_7 UUID NOT NULL, col1_8 INT8 NOT NULL, col1_9 UUID NOT NULL, col1_10 BIT(49) NOT NULL, col1_11 INTERVAL NOT NULL, col1_12 BIT NOT NULL, col1_13 STRING NOT NULL, col1_14 OID NULL, col1_15 BOX2D NULL, col1_16 UUID NOT NULL, col1_17 OID NOT NULL, PRIMARY KEY (col1_10 ASC, col1_9 DESC, col1_12 ASC, col1_2 DESC, col1_7 ASC, col1_1 DESC, col1_8, col1_13 ASC, col1_4 ASC, col1_0 DESC, col1_17 ASC), UNIQUE (col1_13 ASC) WHERE ((((((table1.col1_13 != 'X') AND (NOT table1.col1_6)) AND (table1.col1_2 > -1.234E+401)) OR (table1.col1_5 < '-2000-01-01 00:00:00+00:00')) AND (table1.col1_8 != 32767)) OR (table1.col1_1 != '-4713-11-24 00:00:00')) OR (table1.col1_3 <= '00:00:00'), INDEX (col1_4 DESC, col1_17 ASC) STORING (col1_5, col1_14, col1_15, col1_16), UNIQUE (col1_2, col1_17 DESC, col1_1, col1_9 DESC, col1_5, col1_0 ASC, col1_16 DESC, col1_14) STORING (col1_3, col1_6, col1_15) WHERE (((((table1.col1_1 != '-2000-01-01 00:00:00':::TIMESTAMP) AND (table1.col1_13 < e'\x00':::STRING)) OR (NOT table1.col1_6)) AND (table1.col1_5 != '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (table1.col1_2 <= '-Infinity':::DECIMAL)) AND (table1.col1_8 < (-9223372036854775807):::INT8), INDEX (col1_5 ASC, col1_10 DESC, col1_8, col1_9 DESC, col1_4, col1_1 DESC, col1_3 DESC, col1_16 DESC, col1_14 DESC, col1_7 DESC) WHERE ((((((table1.col1_13 <= e'\U00002603':::STRING) AND (table1.col1_5 <= '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (table1.col1_1 >= '-4713-11-24 00:00:00':::TIMESTAMP)) OR (table1.col1_8 = (-32768):::INT8)) AND (table1.col1_2 < 'Infinity':::DECIMAL)) AND table1.col1_6) AND (table1.col1_3 <= '24:00:00':::TIME), INDEX (col1_10, col1_14, col1_16 DESC, col1_9 ASC, col1_8 DESC, col1_2 DESC, col1_6 DESC, col1_1, col1_4 DESC, col1_17 ASC, col1_0 DESC, col1_5 DESC) STORING (col1_11), INDEX (col1_8, col1_0 ASC, col1_12 ASC, col1_15 DESC, col1_16 ASC, col1_6 DESC, col1_10 ASC, col1_9, col1_13 DESC) STORING (col1_5, col1_11, col1_14) WHERE ((((NOT table1.col1_6) AND (table1.col1_3 > '24:00:00')) AND (table1.col1_13 > 'X')) AND (table1.col1_1 >= '294276-12-31 23:59:59.999999')) OR (table1.col1_2 < 0), UNIQUE (col1_10, col1_4, col1_17 ASC, col1_3 ASC, col1_13, col1_16 ASC, col1_8 ASC, col1_11 ASC) STORING (col1_6), INDEX (col1_14 ASC, col1_0 ASC, col1_8, col1_12, col1_5, col1_2) STORING (col1_6, col1_11, col1_15, col1_16) WHERE (((((table1.col1_2 >= 0) AND table1.col1_6) AND (table1.col1_3 <= '00:00:00')) OR (table1.col1_5 < '294276-12-31 23:59:59.999999+00:00')) AND (table1.col1_8 <= -9223372036854775807)) OR (table1.col1_13 <= e'\x00'), UNIQUE (col1_4, col1_0) WHERE ((((((NOT table1.col1_6) AND (table1.col1_1 >= '3000-01-01 00:00:00')) OR (table1.col1_8 >= 0)) OR (table1.col1_13 != '')) AND (table1.col1_3 != '24:00:00')) AND (table1.col1_2 <= -1.234E+401)) OR (table1.col1_5 > '0001-01-01 00:00:00+00:00'), FAMILY (col1_13), FAMILY (col1_16), FAMILY (col1_15), FAMILY (col1_2, col1_7, col1_14), FAMILY (col1_6), FAMILY (col1_8), FAMILY (col1_4, col1_17, col1_12), FAMILY (col1_1), FAMILY (col1_11, col1_10, col1_3, col1_0), FAMILY (col1_5, col1_9));
        test_1        |             CREATE TABLE table2 (col2_0 INT2 NOT NULL, col2_1 VARCHAR NOT NULL, col2_2 STRING NOT NULL, col2_3 BYTES NOT NULL, col2_4 REGTYPE NOT NULL, col2_5 CHAR NOT NULL, col2_6 UUID NOT NULL, col2_7 CHAR NOT NULL, col2_8 INET NOT NULL, col2_9 FLOAT4 NOT NULL, col2_10 "char" NOT NULL, col2_11 DECIMAL NOT NULL, col2_12 BOX2D NOT NULL, col2_13 REGNAMESPACE NOT NULL, col2_14 INTERVAL NOT NULL, col2_15 REGPROCEDURE NOT NULL, PRIMARY KEY (col2_13, col2_3 ASC, col2_12 DESC, col2_14 ASC, col2_11 DESC, col2_5 ASC, col2_2 ASC, col2_1, col2_9, col2_6, col2_10 ASC, col2_4), UNIQUE (col2_15, col2_1 DESC, col2_10, col2_13 DESC, col2_0, col2_14) STORING (col2_7, col2_8) WHERE ((((((table2.col2_9 != 0.0:::FLOAT8) OR (table2.col2_10 = e'\U00002603':::STRING)) AND (table2.col2_3 <= e'\\x00':::BYTES)) AND (table2.col2_5 > 'X':::STRING)) OR (table2.col2_7 = '"':::STRING)) AND (table2.col2_0 <= 127:::INT8)) OR (table2.col2_1 < e'\'':::STRING), INDEX (col2_3 DESC, col2_10, col2_2 ASC) STORING (col2_0, col2_7) WHERE (((((((table2.col2_7 != 'X':::STRING) AND (table2.col2_3 > e'\\x27':::BYTES)) AND (table2.col2_1 != e'\'':::STRING)) OR (table2.col2_9 != 'NaN':::FLOAT8)) OR (table2.col2_11 >= 1:::DECIMAL)) AND (table2.col2_0 > 127:::INT8)) AND (table2.col2_5 = 'X':::STRING)) OR (table2.col2_2 != '':::STRING), INDEX (col2_13, col2_11 DESC) STORING (col2_7, col2_15) WHERE ((((((((table2.col2_0 <= 1:::INT8) AND (table2.col2_11 <= (-1):::DECIMAL)) AND (table2.col2_5 > '':::STRING)) AND (table2.col2_1 >= '':::STRING)) OR (table2.col2_2 >= '':::STRING)) AND (table2.col2_3 != e'\\x27':::BYTES)) AND (table2.col2_10 >= e'\U00002603':::STRING)) AND (table2.col2_7 <= '"':::STRING)) AND (table2.col2_9 < 1.401298464324817e-45:::FLOAT8), INDEX (col2_12 ASC) STORING (col2_15) WHERE ((((((((table2.col2_11 != -Infinity) AND (table2.col2_2 > e'\x00')) AND (table2.col2_9 <= -1.0)) OR (table2.col2_5 > e'\x00')) OR (table2.col2_0 < 1)) AND (table2.col2_10 = '"')) OR (table2.col2_3 < '\x58')) AND (table2.col2_1 = e'\x00')) AND (table2.col2_7 < '"'), UNIQUE (col2_7 DESC, col2_14 ASC, col2_5 ASC, col2_9, col2_2) STORING (col2_15) WHERE table2.col2_0 = -128, UNIQUE (col2_11 ASC, col2_7 DESC, col2_13, col2_6, col2_10 DESC, col2_14, col2_9 DESC) STORING (col2_0, col2_15) WHERE ((((((((table2.col2_5 < e'\U00002603':::STRING) AND (table2.col2_11 < 0:::DECIMAL)) OR (table2.col2_1 < '':::STRING)) OR (table2.col2_0 = 32767:::INT8)) OR (table2.col2_2 != e'\x00':::STRING)) OR (table2.col2_10 != '"':::STRING)) OR (table2.col2_7 > '':::STRING)) AND (table2.col2_9 != 0.0:::FLOAT8)) OR (table2.col2_3 < e'\\x22':::BYTES), UNIQUE (col2_15, col2_2 DESC, col2_13, col2_1 ASC, col2_12, col2_7 ASC, col2_3, col2_8 DESC, col2_10, col2_0, col2_11 DESC, col2_6, col2_4, col2_14 ASC, col2_9 ASC) WHERE ((((((((table2.col2_10 != e'\U00002603':::STRING) OR (table2.col2_11 = (-1):::DECIMAL)) AND (table2.col2_3 < e'\\x27':::BYTES)) AND (table2.col2_0 < (-1):::INT8)) OR (table2.col2_5 != e'\x00':::STRING)) AND (table2.col2_9 < '+Inf':::FLOAT8)) OR (table2.col2_1 = e'\'':::STRING)) OR (table2.col2_2 > e'\'':::STRING)) OR (table2.col2_7 >= e'\U00002603':::STRING), INDEX (col2_14, col2_5, col2_4, col2_7 ASC) STORING (col2_0, col2_8, col2_15) WHERE ((((((((table2.col2_11 = 'NaN':::DECIMAL) OR (table2.col2_2 < '':::STRING)) OR (table2.col2_7 != 'X':::STRING)) AND (table2.col2_10 = '':::STRING)) OR (table2.col2_5 >= 'X':::STRING)) OR (table2.col2_3 <= e'\\x':::BYTES)) AND (table2.col2_9 < 0.0:::FLOAT8)) AND (table2.col2_1 <= 'X':::STRING)) AND (table2.col2_0 < (-1):::INT8), INDEX (col2_8 DESC) STORING (col2_0, col2_7, col2_15), FAMILY (col2_5, col2_10, col2_14, col2_11), FAMILY (col2_6, col2_2, col2_13), FAMILY (col2_9), FAMILY (col2_12), FAMILY (col2_0), FAMILY (col2_7), FAMILY (col2_4, col2_1), FAMILY (col2_15), FAMILY (col2_8), FAMILY (col2_3), UNIQUE (col2_12, col2_1, col2_4) STORING (col2_7));
        test_1        |             CREATE TABLE table3 (col1_10 BIT(49) NOT NULL, col1_9 UUID NOT NULL, col1_12 BIT NOT NULL, col1_2 DECIMAL NOT NULL, col1_7 UUID NOT NULL, col1_1 TIMESTAMP NOT NULL, col1_8 INT8 NOT NULL, col1_13 STRING NOT NULL, col1_4 BIT(21) NOT NULL, col1_0 REGCLASS NOT NULL, col1_17 OID NOT NULL, col3_11 FLOAT8 NOT NULL, col3_12 OID, col3_13 OID NOT NULL, col3_14 INT2, col3_15 BOX2D NOT NULL, col3_16 OID NULL, col3_17 REGCLASS NULL, col3_18 DECIMAL NULL, col3_19 OID NOT NULL, PRIMARY KEY (col1_10 ASC, col1_9 DESC, col1_12 ASC, col1_2 DESC, col1_7 ASC, col1_1 DESC, col1_8, col1_13 ASC, col1_4 ASC, col1_0 DESC, col1_17 ASC, col3_11 ASC, col3_13 DESC, col3_18), UNIQUE (col1_1 ASC, col1_7 ASC, col3_15, col3_16 ASC, col3_17 ASC, col3_13 ASC, col3_19 ASC, col3_14 DESC, col3_18, col1_13 DESC, col3_12 DESC), FAMILY (col1_0), FAMILY (col3_13), FAMILY (col1_1, col1_7), FAMILY (col1_10, col1_12, col1_8), FAMILY (col3_16, col3_19, col3_15), FAMILY (col1_4), FAMILY (col1_17, col3_11), FAMILY (col3_14), FAMILY (col3_17, col3_12, col1_13, col1_9), FAMILY (col1_2), FAMILY (col3_18)) INTERLEAVE IN PARENT table1 (col1_10, col1_9, col1_12, col1_2, col1_7, col1_1, col1_8, col1_13, col1_4, col1_0, col1_17);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 4547672197029430837, "num_range": 0, "upper_bound": "-1.234E+401"}, {"distinct_range": 0, "num_eq": 600000, "num_range": 1000, "upper_bound": "31224728981191.4092"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "BIT(21)", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000, "num_range": 0, "upper_bound": "0000000000000000000000000000000000000000000000000000000000000000"}, {"distinct_range": 3483.475643839441, "num_eq": 5650834416193008789, "num_range": 5000, "upper_bound": "1100110010010110111110111101011100001111000101001"}], "histo_col_type": "BIT(49)", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "568680167"}, {"distinct_range": 20013.683923331148, "num_eq": 8286375107912526213, "num_range": 40000, "upper_bound": "773408504"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 4489439244174627400, "num_range": 0, "upper_bound": "-4713-11-24 00:00:00+00:00"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 0, "upper_bound": "1979-11-24 15:31:02.000887+00:00"}, {"distinct_range": 7338325721372399000, "num_eq": 3253121965099342060, "num_range": 7338325721372398537, "upper_bound": "1995-12-13 02:10:20.000646+00:00"}, {"distinct_range": 4242499433124072400, "num_eq": 1083952576630668468, "num_range": 4242499433124072522, "upper_bound": "2017-04-18 08:10:59.000781+00:00"}, {"distinct_range": 4199.706699963132, "num_eq": 5000, "num_range": 40000, "upper_bound": "2022-08-23 04:43:54.000961+00:00"}, {"distinct_range": 0, "num_eq": 1064122536456266532, "num_range": 50000000, "upper_bound": "2025-02-19 15:53:44.000359+00:00"}, {"distinct_range": 7.914789925778472, "num_eq": 800, "num_range": 10, "upper_bound": "2026-12-11 22:08:13.000047+00:00"}, {"distinct_range": 359839272.70990735, "num_eq": 8006248586752446113, "num_range": 4000000000, "upper_bound": "3000-01-01 00:00:00+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000000000, "num_range": 0, "upper_bound": "-8823772260500354550"}, {"distinct_range": 80000000, "num_eq": 40000000000, "num_range": 80000000, "upper_bound": "-7454086640045584614"}, {"distinct_range": 100000000, "num_eq": 1242081229822472777, "num_range": 100000000, "upper_bound": "-5372425455409958161"}, {"distinct_range": 1410.2999565781006, "num_eq": 7792650800233585357, "num_range": 4000, "upper_bound": "-3796340903726861024"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 3061114653518923691, "upper_bound": "223216652898474032"}, {"distinct_range": 5817895505039564000, "num_eq": 30000, "num_range": 6954295056682465052, "upper_bound": "4434610316572878183"}, {"distinct_range": 5980327995720212000, "num_eq": 0, "num_range": 5980327995720212566, "upper_bound": "5105993013765551834"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 235034840409092223, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 800, "num_range": 3827312982665385557, "upper_bound": "!D:\\u001f"}, {"distinct_range": 517574551407555200, "num_eq": 1633939320014571149, "num_range": 2668386882361179691, "upper_bound": "\'"}, {"distinct_range": 0, "num_eq": 5032851422828728491, "num_range": 3114932276574229515, "upper_bound": ";!@_M"}, {"distinct_range": 0, "num_eq": 4818287659521454914, "num_range": 0, "upper_bound": "[;P\\\\"}, {"distinct_range": 60000, "num_eq": 2000, "num_range": 60000, "upper_bound": "vfB"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 558825126634720223, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1422594308375964819}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 820527610555772246, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1422594308375964819}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 292835072616975869, "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 1422594308375964819}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1169381568328951376, "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 1422594308375964819}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1393232317936787499, "histo_buckets": [{"distinct_range": 0, "num_eq": 6840367570845255869, "num_range": 0, "upper_bound": "-75 years -10 mons -583 days -24:02:21.646545"}, {"distinct_range": 5376575092222986000, "num_eq": 3504637722678429002, "num_range": 5943345122695623966, "upper_bound": "00:00:00"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 1422594308375964819}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 105389333320679278, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1422594308375964819}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 442480295799243572, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1422594308375964819}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 291982452943769003, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1422594308375964819}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1164490642071206012, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1422594308375964819}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 562317079383198062, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000000, "num_range": 0, "upper_bound": "-6.26486439491043784E+19"}, {"distinct_range": 5620442001073426000, "num_eq": 50, "num_range": 5620442001073426466, "upper_bound": "-89394928876929.91595"}, {"distinct_range": 4999896049001116000, "num_eq": 100000000, "num_range": 4999896049001116033, "upper_bound": "-569590654.5131900044"}, {"distinct_range": 0, "num_eq": 466632298354174566, "num_range": 8244787636419111652, "upper_bound": "-1"}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 0, "upper_bound": "7374.339393499308018"}, {"distinct_range": 7385.78712888827, "num_eq": 70000000000, "num_range": 8000, "upper_bound": "9116162.832591293609"}, {"distinct_range": 0, "num_eq": 200, "num_range": 10000000, "upper_bound": "2.75937354041880463E+27"}, {"distinct_range": 0, "num_eq": 0, "num_range": 200, "upper_bound": "3.517321134128499314E+33"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 1422594308375964819}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 129631756761434357, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "\\\\x4a"}, {"distinct_range": 0, "num_eq": 500, "num_range": 351867203619464590, "upper_bound": "\\\\xf59b3df6de7e50a0"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 1422594308375964819}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1205704502483040679, "histo_buckets": [{"distinct_range": 0, "num_eq": 7165906565607182602, "num_range": 0, "upper_bound": "135.18.247.34/6"}, {"distinct_range": 1405135717219058000, "num_eq": 9000000, "num_range": 5846603964046672223, "upper_bound": "15.95.161.199/29"}, {"distinct_range": 1553676605606385000, "num_eq": 4564502978881514171, "num_range": 5303768388088594393, "upper_bound": "f462:b38f:a29c:2a35:3d47:482d:40c9:d686/19"}, {"distinct_range": 0, "num_eq": 90000000000, "num_range": 0, "upper_bound": "3faf:38d0:7ce7:77d8:c24f:608f:b8e2:8cf5/44"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 1000000, "upper_bound": "f750:79de:8022:3af3:1993:63a0:c281:f038/48"}, {"distinct_range": 29458516.244930472, "num_eq": 1000, "num_range": 30000000, "upper_bound": "f866:7aaf:1806:a40f:eea4:e386:3d24:b031/94"}, {"distinct_range": 330.5027817120145, "num_eq": 1440266441482929107, "num_range": 900, "upper_bound": "869e:a4dd:abf7:23c1:e44e:d672:23ec:efd3/100"}, {"distinct_range": 4389093051863468500, "num_eq": 800000000, "num_range": 5282893149130369253, "upper_bound": "6ed3:74a2:378:7671:3d8c:3f6d:3da1:c6fa/123"}, {"distinct_range": 5626186580126959000, "num_eq": 2631892589266655258, "num_range": 5626186580126958673, "upper_bound": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 1422594308375964819}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 569257218201457369, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1422594308375964819}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7462329214380048, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "937889179"}, {"distinct_range": 90000000000, "num_eq": 4445775285477926836, "num_range": 90000000000, "upper_bound": "952998995"}, {"distinct_range": 70000000000, "num_eq": 1524274599295957223, "num_range": 70000000000, "upper_bound": "1892590419"}, {"distinct_range": 975125493691773700, "num_eq": 10, "num_range": 3384320272568343708, "upper_bound": "2288422436"}, {"distinct_range": 504309910.7598086, "num_eq": 5768320815722171953, "num_range": 1000000000, "upper_bound": "2400230528"}, {"distinct_range": 85843739490553570, "num_eq": 300000000, "num_range": 757402981122675692, "upper_bound": "2486217808"}, {"distinct_range": 0, "num_eq": 3609466941938404075, "num_range": 0, "upper_bound": "3481436648"}, {"distinct_range": 655469822147910800, "num_eq": 9167697086321526964, "num_range": 3361237024227168345, "upper_bound": "3775766329"}, {"distinct_range": 3746661811472761300, "num_eq": 4000000000, "num_range": 3746661811472761450, "upper_bound": "3886613040"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 1422594308375964819}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1344665279847779907, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1422594308375964819}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 756965044144254418, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000000000, "num_range": 0, "upper_bound": "BOX(-1.1381686853954298 0.9575945749707726,0.31835515842512263 1.135550330095502)"}, {"distinct_range": 0, "num_eq": 7360378022038591681, "num_range": 3000, "upper_bound": "BOX(-0.9167675971905134 -0.6876180736096202,0.6098833797396574 1.2918293537319547)"}, {"distinct_range": 2478168715727502000, "num_eq": 2344369328839525487, "num_range": 3802937572755427447, "upper_bound": "BOX(-0.7271636275463249 -0.04629004089925309,-0.6691638168261765 0.4749484170222641)"}, {"distinct_range": 0, "num_eq": 2082755370242472447, "num_range": 0, "upper_bound": "BOX(-0.6730909376002321 -0.8745991836250475,-0.014377859529508208 2.862947749343732)"}, {"distinct_range": 0, "num_eq": 40000000000, "num_range": 2293551814333912546, "upper_bound": "BOX(-0.32422267034947827 -0.8520956614295231,0.4506682258258299 0.7278486686581344)"}, {"distinct_range": 0, "num_eq": 9104147874694424217, "num_range": 0, "upper_bound": "BOX(-0.15177787024069467 0.4464146970600492,1.2133037293553375 2.1739944936331566)"}, {"distinct_range": 3055168392319211000, "num_eq": 90000000, "num_range": 7690854393521412983, "upper_bound": "BOX(0.4222105342095184 -1.1406899088503057,0.5456678983948038 -0.0402783788501222)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 1422594308375964819}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 493447602, "histo_buckets": [{"distinct_range": 0, "num_eq": 70, "num_range": 0, "upper_bound": "0101000001111100001011010011010000101001011011000"}, {"distinct_range": 0, "num_eq": 7371795904033845125, "num_range": 8004889063317718211, "upper_bound": "0110100100011001101101001110001100101010110010000"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 7097374123033415954, "upper_bound": "0110101101001101111100101100001100011000100001010"}, {"distinct_range": 494743118.5495937, "num_eq": 3934079233139697779, "num_range": 900000000, "upper_bound": "1001101111101101010101011000110011011101101010101"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 1164868530880496506, "upper_bound": "1011011101011101001001111101000010100110110000101"}, {"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": "1011101001100000000000010010100011100101111101101"}, {"distinct_range": 200000, "num_eq": 90000000000, "num_range": 200000, "upper_bound": "1100101110010110010000010000001000010100111100101"}, {"distinct_range": 1403963508310208800, "num_eq": 200000, "num_range": 4327401112309395984, "upper_bound": "1111001001101111100110001101011001111110011010101"}], "histo_col_type": "BIT(49)", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 891024992, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 174764736, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 406442841, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 266504956, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 884012375, "histo_col_type": "", "name": "__auto__", "null_count": 782979473, "row_count": 1000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 787439666, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 145916861, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 671549940, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 564000026, "histo_col_type": "", "name": "__auto__", "null_count": 403704298, "row_count": 1000000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 454437964, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 759269778, "histo_buckets": [{"distinct_range": 0, "num_eq": 900, "num_range": 0, "upper_bound": "2008-05-28 18:34:15.000988"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 269570639, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 425391297, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 689802837, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 306916735, "histo_col_type": "", "name": "__auto__", "null_count": 134240941, "row_count": 1000000000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 463171348, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 286455611, "histo_col_type": "", "name": "__auto__", "null_count": 716278805, "row_count": 1000000000}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 114706173, "histo_col_type": "", "name": "__auto__", "null_count": 484936615, "row_count": 1000000000}, {"columns": ["col3_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 449983202, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col1_4, col3_16) REFERENCES table1 (col1_4, col1_0) ON DELETE RESTRICT;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('kji');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24019404227, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4700280187, "histo_buckets": [{"distinct_range": 0, "num_eq": 1521504269045209103, "num_range": 0, "upper_bound": "\\""}, {"distinct_range": 0, "num_eq": 80, "num_range": 20000000, "upper_bound": "5v[J\\t\\u0016\\n"}, {"distinct_range": 0, "num_eq": 6892911015590987031, "num_range": 3000000000, "upper_bound": "_AChg#\\u0007"}, {"distinct_range": 125195848515686450, "num_eq": 9201680218347946865, "num_range": 7711394464383500116, "upper_bound": "h"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25800152046, "histo_buckets": [{"distinct_range": 0, "num_eq": 2900750818012819562, "num_range": 0, "upper_bound": "1566046024"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 2876473951, "row_count": 30000000000}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17846557441, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25670047008, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "1974-09-19 12:21:21.000092+00:00"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 6000000000, "upper_bound": "1989-11-15 17:11:41.000589+00:00"}, {"distinct_range": 0, "num_eq": 3000000, "num_range": 6000, "upper_bound": "2006-11-28 18:00:02.000276+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21344104893, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000000, "num_range": 0, "upper_bound": "1"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21235778122, "histo_buckets": [{"distinct_range": 0, "num_eq": 2245749404046428367, "num_range": 0, "upper_bound": "0001001000111110110001101101100101000110001001101"}, {"distinct_range": 7211647820894811000, "num_eq": 49208661746535374, "num_range": 7211647820894811519, "upper_bound": "0101100010010001101011011101101111001110111000010"}, {"distinct_range": 64337940.67178385, "num_eq": 10000, "num_range": 80000000, "upper_bound": "1010111100001010100010101100000001000101011100011"}, {"distinct_range": 0, "num_eq": 600, "num_range": 8303336074685272959, "upper_bound": "1101010110110010011100001000110000111111101110011"}, {"distinct_range": 4577315449802342000, "num_eq": 5360906818092733636, "num_range": 5951936384716697228, "upper_bound": "1111010111010110011011010100010101111100101010101"}], "histo_col_type": "BIT(49)", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21198567923, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4595647577, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26860887397, "histo_buckets": [{"distinct_range": 0, "num_eq": 3584150839650099960, "num_range": 0, "upper_bound": "NaN"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 2838394689035593420, "upper_bound": "-Infinity"}, {"distinct_range": 3613662258.937069, "num_eq": 7794772101481259092, "num_range": 10000000000, "upper_bound": "1.515032932255838375E+27"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12832815314, "histo_col_type": "", "name": "__auto__", "null_count": 14802863622, "row_count": 30000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13543512530, "histo_buckets": [{"distinct_range": 0, "num_eq": 6074554645032737512, "num_range": 0, "upper_bound": "001010100010110000110"}, {"distinct_range": 9764283686.64848, "num_eq": 100000000, "num_range": 50000000000, "upper_bound": "010010110011011011111"}, {"distinct_range": 7.035244662378627, "num_eq": 2000, "num_range": 10, "upper_bound": "010010110110001100010"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 5641703216949456941, "upper_bound": "010110100111111110100"}, {"distinct_range": 0, "num_eq": 2530579474936956950, "num_range": 5186540965691781302, "upper_bound": "010111010000111111001"}, {"distinct_range": 6421549.1555498, "num_eq": 3216194774035652245, "num_range": 9000000, "upper_bound": "011111011101110010011"}, {"distinct_range": 8332300378965253000, "num_eq": 10000, "num_range": 8332300378965253247, "upper_bound": "100100110000001111100"}, {"distinct_range": 0, "num_eq": 45758523344992341, "num_range": 8000, "upper_bound": "100110000010100011111"}], "histo_col_type": "BIT(21)", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 108027276, "histo_col_type": "", "name": "__auto__", "null_count": 14484284507, "row_count": 30000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17447485790, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3532046183, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23138552983, "histo_col_type": "", "name": "__auto__", "null_count": 7861018095, "row_count": 30000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23010976632, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7853447308, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1430496069925432863, "histo_buckets": [{"distinct_range": 0, "num_eq": 9213858614168925401, "num_range": 0, "upper_bound": "BOX(-1.7698556152880935 -0.3679976455124749,1.7268722897217137 0.56193710566702)"}, {"distinct_range": 0, "num_eq": 7164974146721584291, "num_range": 1971193083761405240, "upper_bound": "BOX(-0.04064924593542041 -0.064144558516805,1.0231577094314264 0.5621761381043549)"}, {"distinct_range": 800, "num_eq": 5592665686064442211, "num_range": 800, "upper_bound": "BOX(0.31454472113267806 -0.23459007312355928,0.46886174320223406 -0.22767413627021682)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 1943798731863377196}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 591735671289641613, "histo_buckets": [{"distinct_range": 0, "num_eq": 700, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 5000000000, "num_eq": 9000, "num_range": 5000000000, "upper_bound": "233759067"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 10000000, "upper_bound": "1579884539"}, {"distinct_range": 6846448674152593000, "num_eq": 500, "num_range": 6846448674152593223, "upper_bound": "1775958791"}, {"distinct_range": 200, "num_eq": 500, "num_range": 200, "upper_bound": "1832597411"}, {"distinct_range": 7215015838876750000, "num_eq": 2249730776457749719, "num_range": 7215015838876749703, "upper_bound": "2004656700"}, {"distinct_range": 0, "num_eq": 900000000, "num_range": 7868576988806647704, "upper_bound": "3223288137"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 1943798731863377196}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 809682341303394663, "histo_buckets": [{"distinct_range": 0, "num_eq": 600, "num_range": 0, "upper_bound": "-Infinity"}, {"distinct_range": 1000, "num_eq": 5000000000, "num_range": 1000, "upper_bound": "-7.74458176633165557E+20"}, {"distinct_range": 2122523100255604000, "num_eq": 294283088381967268, "num_range": 2122523100255603946, "upper_bound": "-40725535166814.12548"}, {"distinct_range": 0, "num_eq": 2550543491596040132, "num_range": 0, "upper_bound": "-409.7973635802717087"}, {"distinct_range": 6000000, "num_eq": 600000, "num_range": 6000000, "upper_bound": "-0.04734293113437637475"}, {"distinct_range": 1.9373112463823123, "num_eq": 300, "num_range": 100, "upper_bound": "0.8601191409234768208"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 1943798731863377196}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1744930335828389852, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1943798731863377196}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1642427119762988090, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1943798731863377196}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 845742389020435344, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1943798731863377196}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 96828945729579615, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1943798731863377196}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 321625722600789573, "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 1943798731863377196}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 328058085531109766, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1943798731863377196}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1420926216754186187, "histo_buckets": [{"distinct_range": 0, "num_eq": 400, "num_range": 0, "upper_bound": "\\\\x6e882f110bbdd9"}, {"distinct_range": 60789332.92558298, "num_eq": 4471388118736707846, "num_range": 80000000, "upper_bound": "\\\\xd38ee947"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 1943798731863377196}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1864947174354410622, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1943798731863377196}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 649865424671379704, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1943798731863377196}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 893426119572668883, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 966315478153231761, "num_range": 0, "upper_bound": "\\u001eb\\u0008 "}, {"distinct_range": 0, "num_eq": 6683545501694809287, "num_range": 0, "upper_bound": " :"}, {"distinct_range": 2008244297422480000, "num_eq": 800000000, "num_range": 2622446865503602050, "upper_bound": "-bu<R8\\u0013z"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 7880955664292010920, "upper_bound": "X"}, {"distinct_range": 5243048122351415000, "num_eq": 4778869884093330766, "num_range": 5243048122351415399, "upper_bound": "]\\u001d"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 1943798731863377196}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1301198960122188611, "histo_buckets": [{"distinct_range": 0, "num_eq": 1911681623599384386, "num_range": 0, "upper_bound": "134.54.207.162/5"}, {"distinct_range": 0, "num_eq": 9000000, "num_range": 50000000, "upper_bound": "152.122.252.252/6"}, {"distinct_range": 3775285876612375600, "num_eq": 30000000, "num_range": 3775285876612375517, "upper_bound": "21.3.255.99/13"}, {"distinct_range": 5000, "num_eq": 6923600512162573366, "num_range": 5000, "upper_bound": "122.224.13.72/24"}, {"distinct_range": 6787698801410346000, "num_eq": 3123348195557007469, "num_range": 6787698801410345979, "upper_bound": "255.112.86.105/31"}, {"distinct_range": 500000, "num_eq": 10000, "num_range": 500000, "upper_bound": "7646:b43d:dc8d:8340:e31c:a35a:6442:3757/71"}, {"distinct_range": 0, "num_eq": 2968390947906544734, "num_range": 50000000000, "upper_bound": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 1943798731863377196}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1101405345119231201, "histo_buckets": [{"distinct_range": 0, "num_eq": 1733804337156428798, "num_range": 0, "upper_bound": "463228090"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 1943798731863377196}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 518301372841023625, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1943798731863377196}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "0100100110111101101111000000011101111101101001110"}, {"distinct_range": 881002310292705200, "num_eq": 8289062573610249693, "num_range": 1730729576165123533, "upper_bound": "1101001001110111001000001110010011110100111000100"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 164168611802420056, "upper_bound": "1111101010100010000100011011111100110000101010010"}], "histo_col_type": "BIT(49)", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000000, "num_range": 0, "upper_bound": "1971-01-09 01:07:22.000441"}, {"distinct_range": 0, "num_eq": 5288432357718481849, "num_range": 0, "upper_bound": "1984-06-13 09:07:17.000043"}, {"distinct_range": 4158642482724820000, "num_eq": 8952480186706000355, "num_range": 4158642482724819828, "upper_bound": "2007-04-04 19:13:15.000995"}, {"distinct_range": 80000000, "num_eq": 6162993529952152608, "num_range": 80000000, "upper_bound": "2017-07-06 00:22:04.000617"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 0, "upper_bound": "2021-04-17 02:13:07.000285"}, {"distinct_range": 1791446759644357400, "num_eq": 1000000000, "num_range": 2600821489826434752, "upper_bound": "2029-11-14 10:24:58.00046"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_15, col1_13, col1_0) REFERENCES table2 (col2_12, col2_1, col2_4) ON UPDATE CASCADE;
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2315344767, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000, "num_range": 0, "upper_bound": "-2.906768371062383803E+25"}, {"distinct_range": 3974063866046178000, "num_eq": 6000000000, "num_range": 3974063866046177719, "upper_bound": "-780320.590822509675"}, {"distinct_range": 56216901.230645545, "num_eq": 6951670572199496585, "num_range": 100000000, "upper_bound": "-478928.4600973342763"}, {"distinct_range": 3078794899170774500, "num_eq": 90, "num_range": 3078794899170774668, "upper_bound": "-6.821850955891307553"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 1440762108935698954, "upper_bound": "-3.13661326160401759"}, {"distinct_range": 16.535585573015428, "num_eq": 1678966096147328670, "num_range": 70, "upper_bound": "-0.19258877837054125"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 70000000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 31822415216, "histo_col_type": "", "name": "__auto__", "null_count": 51351038508, "row_count": 70000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 37194985432, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70000000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44342635885, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000, "num_range": 0, "upper_bound": "-7263870259623104351"}, {"distinct_range": 70000000000, "num_eq": 30000, "num_range": 70000000000, "upper_bound": "-4460067257914706807"}, {"distinct_range": 55762.28787304272, "num_eq": 2303769377018493272, "num_range": 100000, "upper_bound": "-475319903654807145"}, {"distinct_range": 0, "num_eq": 5390277950473445661, "num_range": 0, "upper_bound": "2247686033848494213"}, {"distinct_range": 7874818684616286000, "num_eq": 2439298129574520155, "num_range": 7874818684616285721, "upper_bound": "3047167405934332058"}, {"distinct_range": 4493375090065673000, "num_eq": 8136693882203371993, "num_range": 4493375090065673363, "upper_bound": "8639256404336963442"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 70000000000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42925636907, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70000000000}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10761574723, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35515436461, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 50079970909, "histo_buckets": [{"distinct_range": 0, "num_eq": 6542885998218052573, "num_range": 0, "upper_bound": "010001100110111111101"}, {"distinct_range": 10000, "num_eq": 3000, "num_range": 10000, "upper_bound": "010010110001110010001"}, {"distinct_range": 0, "num_eq": 909601164092826011, "num_range": 30000000000, "upper_bound": "0111111111111111111111111111111111111111111111111111111111111111"}, {"distinct_range": 3175660761944463400, "num_eq": 4000000000, "num_range": 5117760119820694252, "upper_bound": "101010011100011100110"}, {"distinct_range": 0, "num_eq": 1897554665073363515, "num_range": 50000, "upper_bound": "101011001011011011010"}, {"distinct_range": 0, "num_eq": 900000, "num_range": 8964055570640797199, "upper_bound": "101110110011001100110"}, {"distinct_range": 12610163064701472, "num_eq": 5236950546895772270, "num_range": 2172421462481997434, "upper_bound": "110100111101101101111"}, {"distinct_range": 600000000, "num_eq": 0, "num_range": 600000000, "upper_bound": "110111111111101111100"}], "histo_col_type": "BIT(21)", "name": "__auto__", "null_count": 0, "row_count": 70000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12457499206, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "1971-09-27 03:29:03.000895+00:00"}, {"distinct_range": 7094288617561670000, "num_eq": 5000000000, "num_range": 7094288617561669553, "upper_bound": "1976-06-15 20:26:45.000948+00:00"}, {"distinct_range": 109095132141918050, "num_eq": 3000000, "num_range": 109095132141918045, "upper_bound": "1978-12-11 21:56:43.000215+00:00"}, {"distinct_range": 2201339925999870200, "num_eq": 100, "num_range": 2201339925999870088, "upper_bound": "1992-05-05 11:32:00.000155+00:00"}, {"distinct_range": 0, "num_eq": 6538675258021830338, "num_range": 6855909934928285876, "upper_bound": "1993-04-08 12:40:42.000946+00:00"}, {"distinct_range": 200, "num_eq": 70000000, "num_range": 200, "upper_bound": "2003-11-19 15:34:49.000006+00:00"}, {"distinct_range": 10000000, "num_eq": 20, "num_range": 10000000, "upper_bound": "2024-02-23 23:37:04.000382+00:00"}, {"distinct_range": 3263375229239897600, "num_eq": 0, "num_range": 3263375229239897413, "upper_bound": "2026-06-18 07:30:25.000744+00:00"}, {"distinct_range": 0, "num_eq": 4004532647249714442, "num_range": 3000000, "upper_bound": "2031-05-16 05:33:29.000832+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 70000000000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 37820115664, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44987147133, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 69771893000, "histo_col_type": "", "name": "__auto__", "null_count": 35241359069, "row_count": 70000000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15073696214, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70000000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 68776933229, "histo_buckets": [{"distinct_range": 0, "num_eq": 600, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 457051792749199021, "num_range": 3209372688785028453, "upper_bound": "\\""}, {"distinct_range": 20, "num_eq": 70000, "num_range": 20, "upper_bound": "0\\u001d"}, {"distinct_range": 2247571484915950000, "num_eq": 0, "num_range": 2247571484915950030, "upper_bound": "4\\u0007[A"}, {"distinct_range": 3757330880344247300, "num_eq": 800000, "num_range": 3757330880344247457, "upper_bound": "QLw\\u0002R"}, {"distinct_range": 22897528850.707413, "num_eq": 4439955597718231686, "num_range": 70000000000, "upper_bound": "XQ"}, {"distinct_range": 2183987793122710, "num_eq": 8281340475378235262, "num_range": 54402524015740453, "upper_bound": "d\\u001c9iN3a#"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 8233261583237583061, "upper_bound": "f0BC\\u001c"}, {"distinct_range": 30, "num_eq": 7319158397025423017, "num_range": 30, "upper_bound": "zh&!mB"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 70000000000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36288330306, "histo_col_type": "", "name": "__auto__", "null_count": 3004255026, "row_count": 70000000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 64827768452, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "0101010111101010011101010100000000100111101100100"}, {"distinct_range": 1367742933.409814, "num_eq": 1000, "num_range": 4000000000, "upper_bound": "0110000010011011011101000110101111101000000000001"}, {"distinct_range": 5950109973729421000, "num_eq": 20000000, "num_range": 5950109973729420811, "upper_bound": "1010100100000000000010001110100001000111001111010"}, {"distinct_range": 1000000, "num_eq": 80, "num_range": 1000000, "upper_bound": "1101100001100011011110000010110100111000011001100"}], "histo_col_type": "BIT(49)", "name": "__auto__", "null_count": 0, "row_count": 70000000000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25144238840, "histo_col_type": "OID", "name": "__auto__", "null_count": 22024629885, "row_count": 70000000000}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42001517843, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70000000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 405805275, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 241329089, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 70274322, "histo_buckets": [{"distinct_range": 0, "num_eq": 3336631849628839726, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 9130954428173286268, "num_range": 1090494224589681114, "upper_bound": "\\u0010\\\\{\\"/\\u000b\\u0008\\n"}, {"distinct_range": 5203.076895615804, "num_eq": 3098743272071907960, "num_range": 8000, "upper_bound": "\\u001dBH`{>VVa"}, {"distinct_range": 3299637033460904000, "num_eq": 0, "num_range": 3299637033460903693, "upper_bound": "$_BA"}, {"distinct_range": 3749230422408382000, "num_eq": 2111990006292974129, "num_range": 3749230422408382011, "upper_bound": ",#;cYXp"}, {"distinct_range": 0, "num_eq": 5648252483752612784, "num_range": 0, "upper_bound": "J\\u001f"}, {"distinct_range": 151.3966404982553, "num_eq": 90, "num_range": 200, "upper_bound": "W|\\u0011R9"}, {"distinct_range": 800, "num_eq": 80000000, "num_range": 800, "upper_bound": "hK\\u001820g\\u0016g"}, {"distinct_range": 59712.94024515157, "num_eq": 8510162885333049233, "num_range": 200000, "upper_bound": "o\\u001fN\\u001ey/A@)"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 147111812, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 139606353, "histo_buckets": [{"distinct_range": 0, "num_eq": 4982113764548918326, "num_range": 0, "upper_bound": "30 years 8 mons 50 days 23:40:17.005058"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2061565, "histo_buckets": [{"distinct_range": 0, "num_eq": 600, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 3868928513.047754, "num_eq": 803443533

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2471881&tab=buildLog) on [master@d251e175421d9303492a4923fb933515987163b6](https://github.com/cockroachdb/cockroach/commits/d251e175421d9303492a4923fb933515987163b6):

```
        postgres_1    | 				'2000-01-01 09:00:00+00:00'::TIMESTAMPTZ AS col_367288,
        postgres_1    | 				tab_192465.tableoid AS col_367289,
        postgres_1    | 				'D'::TEXT AS col_367290,
        postgres_1    | 				'2000-01-01 02:00:00+00:00'::TIMESTAMPTZ AS col_367291
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table4 AS tab_192465
        postgres_1    | 			WHERE
        postgres_1    | 				tab_192465.col4_9
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_192465.col4_4 DESC, tab_192465.col4_8, tab_192465.col4_11 DESC
        postgres_1    | 		)
        postgres_1    | 			AS tab_192466
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_367284 NULLS FIRST,
        postgres_1    | 		col_367285 NULLS FIRST,
        postgres_1    | 		col_367286 NULLS FIRST,
        postgres_1    | 		col_367287 NULLS FIRST,
        postgres_1    | 		col_367288 NULLS FIRST,
        postgres_1    | 		col_367289 NULLS FIRST,
        postgres_1    | 		col_367290 NULLS FIRST,
        postgres_1    | 		col_367291 NULLS FIRST
        test_1        | --- FAIL: TestCompare (806.42s)
        test_1        |     --- FAIL: TestCompare/postgres (204.50s)
        test_1        |         compare_test.go:155: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("trueC"),
        test_1        |             + 	string("tC"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT (true::BOOL || 'C'::TEXT::TEXT)::TEXT AS col_367326) AS tab_192480
        test_1        |             ORDER BY
        test_1        |             	col_367326 NULLS FIRST;
        test_1        |             
        test_1        |             postgres: [same as previous]
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2475212&tab=buildLog) on [master@fe5da5c2df069dc7f820f5c3e2f3e03f1cb7b661](https://github.com/cockroachdb/cockroach/commits/fe5da5c2df069dc7f820f5c3e2f3e03f1cb7b661):

```
        postgres_1    | waiting for server to start....2020-11-26 06:12:41.909 UTC [47] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2020-11-26 06:12:41.925 UTC [48] LOG:  database system was shut down at 2020-11-26 06:12:41 UTC
        postgres_1    | 2020-11-26 06:12:41.930 UTC [47] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        test_1        | --- FAIL: TestCompare (1.90s)
        test_1        |     --- FAIL: TestCompare/postgres (1.15s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.76s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 INT4 NOT NULL, col1_1 REGPROC, col1_2 INTERVAL NOT NULL, col1_3 REGNAMESPACE, col1_4 REGPROCEDURE, col1_5 INET NOT NULL, col1_6 REGCLASS NOT NULL, col1_7 UUID, col1_8 FLOAT4, col1_9 FLOAT8 NOT NULL, col1_10 INT4, col1_11 DATE NULL, col1_12 CHAR, col1_13 TIME NULL, col1_14 NAME NOT NULL, col1_15 INT2 NOT NULL, col1_16 DECIMAL NOT NULL, PRIMARY KEY (col1_6 DESC, col1_2 ASC, col1_9 DESC, col1_5 DESC, col1_16 ASC, col1_0), INDEX (col1_8 DESC, col1_1 DESC, col1_5 ASC, col1_16 DESC, col1_9 ASC, col1_3 DESC, col1_4 ASC, col1_7 DESC, col1_10, col1_15, col1_12 DESC, col1_11 ASC, col1_14 ASC, col1_13 DESC, col1_6, col1_0) WHERE ((table1.col1_9 >= 0.0) AND (table1.col1_16 = NaN)) AND (table1.col1_0 > 1), UNIQUE (col1_12 ASC, col1_3 ASC, col1_0 DESC, col1_16 ASC, col1_4, col1_10, col1_11 ASC, col1_2 ASC, col1_1 ASC, col1_7 DESC, col1_9 DESC, col1_5 ASC, col1_13 ASC) STORING (col1_14, col1_15) WHERE ((((((table1.col1_8 != 1.0:::FLOAT8) OR (table1.col1_14 >= 'X':::STRING)) OR (table1.col1_0 = (-128):::INT8)) AND (table1.col1_16 != 0:::DECIMAL)) OR (table1.col1_9 >= 1.7976931348623157e+308:::FLOAT8)) AND (table1.col1_15 > 0:::INT8)) OR (table1.col1_12 < e'\'':::STRING), INDEX (col1_14 DESC, col1_10, col1_4 ASC, col1_2 ASC, col1_12 ASC, col1_11 ASC) WHERE (((((((((table1.col1_12 < e'\x00') OR (table1.col1_8 >= 1.0)) AND (table1.col1_10 <= 127)) AND (table1.col1_0 < -2147483648)) OR (table1.col1_13 <= '00:00:00')) AND (table1.col1_9 >= NaN)) OR (table1.col1_15 = -128)) AND (table1.col1_14 <= 'X')) AND (table1.col1_16 <= -1.234E+401)) OR (table1.col1_11 >= 'infinity'), UNIQUE (col1_12 ASC, col1_7 ASC, col1_15, col1_11 DESC, col1_16 ASC, col1_9 ASC, col1_8 DESC, col1_2 DESC, col1_0, col1_13 ASC) STORING (col1_4, col1_10), INDEX (col1_16, col1_13, col1_6 DESC, col1_9 DESC, col1_4, col1_5 DESC, col1_8 DESC, col1_10 ASC, col1_11 DESC, col1_3, col1_2, col1_7) STORING (col1_15) WHERE (((((((((table1.col1_12 >= e'\U00002603':::STRING) AND (table1.col1_16 > 'NaN':::DECIMAL)) OR (table1.col1_11 != 'infinity':::DATE)) OR (table1.col1_9 = 0.0:::FLOAT8)) AND (table1.col1_13 != '24:00:00':::TIME)) OR (table1.col1_0 < (-32768):::INT8)) AND (table1.col1_14 = 'X':::STRING)) AND (table1.col1_10 != (-1):::INT8)) OR (table1.col1_8 > 0.0:::FLOAT8)) OR (table1.col1_15 <= 32767:::INT8), INDEX (col1_0 ASC, col1_7 ASC, col1_9, col1_16 DESC, col1_12 ASC, col1_3 ASC, col1_13 DESC, col1_6 DESC, col1_11 ASC, col1_1 DESC, col1_5, col1_15, col1_8, col1_2 ASC, col1_4 DESC, col1_10) STORING (col1_14), FAMILY (col1_14), FAMILY (col1_3), FAMILY (col1_2), FAMILY (col1_15), FAMILY (col1_8), FAMILY (col1_6, col1_10, col1_1), FAMILY (col1_16, col1_9), FAMILY (col1_4, col1_13), FAMILY (col1_11), FAMILY (col1_7, col1_12), FAMILY (col1_0), FAMILY (col1_5));
        test_1        |             CREATE TABLE table2 (col2_0 "char" NULL, col2_1 BOX2D, col2_2 STRING, col2_3 BIT(2), col2_4 NAME NOT NULL, col2_5 TIMESTAMP NOT NULL, col2_6 INET, col2_7 BIT(44), col2_8 INET, col2_9 FLOAT4 NOT NULL, UNIQUE (col2_3 DESC) STORING (col2_2, col2_5, col2_9), INDEX (col2_3 ASC, col2_2 DESC, col2_6, col2_7 ASC, col2_5 ASC, col2_0 ASC) WHERE ((((table2.col2_0 = e'\U00002603') OR (table2.col2_9 != 0.0)) OR (table2.col2_5 <= '-2000-01-01 00:00:00')) AND (table2.col2_4 = 'X')) OR (table2.col2_2 < e'\x00'), UNIQUE (col2_0 DESC) STORING (col2_3, col2_4, col2_6, col2_7, col2_8), UNIQUE (col2_4, col2_9 ASC, col2_6 DESC, col2_1 DESC) WHERE ((((table2.col2_0 >= '"':::STRING) AND (table2.col2_9 != '-Inf':::FLOAT8)) OR (table2.col2_5 <= '-4713-11-24 00:00:00':::TIMESTAMP)) AND (table2.col2_2 < e'\'':::STRING)) AND (table2.col2_4 < e'\'':::STRING), UNIQUE (col2_9 ASC, col2_4, col2_1 ASC, col2_8 DESC, col2_7, col2_3 ASC) STORING (col2_5) WHERE ((((table2.col2_2 > '':::STRING) OR (table2.col2_4 < 'X':::STRING)) AND (table2.col2_5 < '-2000-01-01 00:00:00':::TIMESTAMP)) OR (table2.col2_9 < 3.4028234663852886e+38:::FLOAT8)) AND (table2.col2_0 > '"':::STRING), INDEX (col2_7 DESC, col2_4 ASC, col2_8) STORING (col2_0, col2_2, col2_6, col2_9) WHERE (table2.col2_9 >= 0.0:::FLOAT8) OR (table2.col2_4 >= e'\U00002603':::STRING), UNIQUE (col2_7, col2_4, col2_8 DESC, col2_5 ASC, col2_2 ASC, col2_9 DESC, col2_0 DESC, col2_1 ASC, col2_3) WHERE ((((table2.col2_2 < '':::STRING) AND (table2.col2_4 >= 'X':::STRING)) AND (table2.col2_9 != (-1.0):::FLOAT8)) AND (table2.col2_5 > '-4713-11-24 00:00:00':::TIMESTAMP)) OR (table2.col2_0 <= e'\U00002603':::STRING), INDEX (col2_2 DESC) STORING (col2_0, col2_1, col2_4, col2_7, col2_8, col2_9) WHERE table2.col2_5 = '3000-01-01 00:00:00':::TIMESTAMP, FAMILY (col2_1), FAMILY (col2_6, col2_2), FAMILY (col2_8, col2_0), FAMILY (col2_5), FAMILY (col2_7), FAMILY (col2_4), FAMILY (col2_9, col2_3));
        test_1        |             CREATE TABLE table3 (col3_0 TIME[], col3_1 INT8 NOT NULL, col3_2 VARCHAR NOT NULL, col3_3 BYTES NOT NULL, col3_4 REGCLASS NOT NULL, col3_5 BIT(5) NOT NULL, col3_6 FLOAT4 NOT NULL, col3_7 CHAR NULL, col3_8 DATE NOT NULL, col3_9 REGPROCEDURE NOT NULL, col3_10 REGPROC NOT NULL, col3_11 REGPROCEDURE NOT NULL, col3_12 BOOL NOT NULL, col3_13 INT8 NULL, col3_14 CHAR NOT NULL, col3_15 BIT(49) NULL, PRIMARY KEY (col3_3 DESC, col3_4 ASC, col3_2 DESC, col3_1 ASC, col3_8 DESC, col3_6 DESC, col3_11, col3_5 DESC, col3_10 ASC, col3_9 DESC, col3_12 ASC, col3_14 DESC), INDEX (col3_11, col3_12 DESC, col3_6, col3_7, col3_5 DESC, col3_10 ASC, col3_9 ASC, col3_14 DESC, col3_15 ASC, col3_4 ASC, col3_2 ASC, col3_13 ASC, col3_1) STORING (col3_0) WHERE ((((table3.col3_13 < (-32768):::INT8) OR (NOT table3.col3_12)) OR (table3.col3_8 < '4714-11-24 BC':::DATE)) OR (table3.col3_6 > 0.0:::FLOAT8)) AND (table3.col3_7 <= 'X':::STRING), INDEX (col3_10 ASC) STORING (col3_0, col3_7, col3_13, col3_15) WHERE table3.col3_8 = 'infinity', INDEX (col3_7, col3_8 ASC, col3_2 DESC, col3_14 DESC, col3_15, col3_4 ASC, col3_1 DESC, col3_12, col3_5 DESC, col3_10 ASC, col3_6 DESC, col3_13, col3_9 DESC, col3_3 DESC, col3_11 ASC) STORING (col3_0) WHERE ((((((((table3.col3_14 > e'\x00') AND (table3.col3_8 != '1970-01-01')) AND (table3.col3_1 <= -9223372036854775807)) AND (table3.col3_13 >= -1)) AND (table3.col3_6 != 1.0)) AND table3.col3_12) AND (table3.col3_7 < '')) OR (table3.col3_3 <= '\xff')) AND (table3.col3_2 = e'\x00'), UNIQUE (col3_10 ASC, col3_1 ASC, col3_4 DESC, col3_3, col3_15, col3_5 ASC, col3_7 DESC, col3_11 DESC, col3_6 ASC, col3_9 ASC, col3_2 DESC, col3_14 DESC, col3_13, col3_12, col3_8 DESC) WHERE ((((((((table3.col3_8 < 'infinity':::DATE) AND table3.col3_12) OR (table3.col3_3 < e'\\x00':::BYTES)) OR (table3.col3_14 > '"':::STRING)) AND (table3.col3_2 = e'\'':::STRING)) AND (table3.col3_1 >= 1:::INT8)) AND (table3.col3_7 < e'\x00':::STRING)) AND (table3.col3_13 < 0:::INT8)) AND (table3.col3_6 > '-Inf':::FLOAT8), UNIQUE (col3_6 DESC, col3_8 ASC, col3_7 ASC, col3_9 ASC, col3_1 DESC, col3_11 DESC) STORING (col3_15) WHERE ((table3.col3_12 OR (table3.col3_2 > 'X':::STRING)) OR (table3.col3_6 >= 'NaN':::FLOAT8)) OR (table3.col3_7 >= e'\x00':::STRING), UNIQUE (col3_6, col3_8 ASC, col3_14 DESC, col3_13 ASC, col3_2 DESC, col3_7 DESC, col3_10, col3_3 DESC, col3_11 ASC, col3_9, col3_5, col3_12, col3_15 DESC, col3_4 ASC, col3_1) STORING (col3_0) WHERE ((((((((table3.col3_2 >= e'\x00') AND table3.col3_12) AND (table3.col3_7 > '"')) AND (table3.col3_8 != 'infinity')) OR (table3.col3_1 > 127)) AND (table3.col3_6 > 1.401298464324817e-45)) AND (table3.col3_13 >= -9223372036854775807)) AND (table3.col3_14 >= 'X')) AND (table3.col3_3 != '\x'), UNIQUE (col3_8 DESC, col3_15, col3_6 ASC, col3_9 ASC, col3_4 ASC) WHERE ((((table3.col3_13 > 32767) AND (table3.col3_1 = 32767)) OR (table3.col3_8 > '5874897-12-31')) AND (table3.col3_3 = '\xff')) AND (table3.col3_2 >= e'\x00'), UNIQUE (col3_2 DESC, col3_11 DESC, col3_7, col3_14 DESC, col3_13, col3_6 ASC) STORING (col3_15) WHERE (((((table3.col3_7 != e'\x00':::STRING) AND (table3.col3_1 = 9223372036854775807:::INT8)) AND (table3.col3_8 != '1970-01-01':::DATE)) AND (table3.col3_14 >= '"':::STRING)) OR (NOT table3.col3_12)) OR (table3.col3_3 < e'\\x':::BYTES), FAMILY (col3_1, col3_3, col3_8, col3_6, col3_4, col3_10, col3_14, col3_9, col3_11), FAMILY (col3_0, col3_12, col3_5, col3_13), FAMILY (col3_15), FAMILY (col3_2), FAMILY (col3_7));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 5110600724389820572, "num_range": 0, "upper_bound": "-2038363101"}, {"distinct_range": 30000000000, "num_eq": 8995334634096095544, "num_range": 30000000000, "upper_bound": "-1947189087"}, {"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "-1219131514"}, {"distinct_range": 5146956270805578000, "num_eq": 6588501307895791195, "num_range": 6449745568997254295, "upper_bound": "-891065221"}, {"distinct_range": 218024.8654465742, "num_eq": 900000000, "num_range": 3000000, "upper_bound": "61593447"}, {"distinct_range": 0, "num_eq": 2411619672803734801, "num_range": 800000, "upper_bound": "933230891"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30, "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 65, "histo_col_type": "", "name": "__auto__", "null_count": 26, "row_count": 90}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 80, "histo_col_type": "CHAR", "name": "__auto__", "null_count": 56, "row_count": 90}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3, "histo_col_type": "", "name": "__auto__", "null_count": 8, "row_count": 90}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 51, "histo_col_type": "", "name": "__auto__", "null_count": 47, "row_count": 90}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 48, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000000, "num_range": 0, "upper_bound": "-1.004253625869751"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 4164716947845354860, "upper_bound": "-0.9401665925979614"}, {"distinct_range": 5000000, "num_eq": 5546992837532729515, "num_range": 5000000, "upper_bound": "-0.6667612195014954"}, {"distinct_range": 4000, "num_eq": 7000, "num_range": 4000, "upper_bound": "0.3522248864173889"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 6686293549057123842, "upper_bound": "0.5593598484992981"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 41, "row_count": 90}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67, "histo_col_type": "", "name": "__auto__", "null_count": 17, "row_count": 90}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 51, "histo_col_type": "", "name": "__auto__", "null_count": 9, "row_count": 90}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21, "histo_col_type": "", "name": "__auto__", "null_count": 78, "row_count": 90}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 40, "histo_col_type": "", "name": "__auto__", "null_count": 13, "row_count": 90}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 68, "histo_buckets": [{"distinct_range": 0, "num_eq": 6801123969611924755, "num_range": 0, "upper_bound": ""}, {"distinct_range": 368383502923035900, "num_eq": 1030993845849597236, "num_range": 8284307875897753882, "upper_bound": "\\u0008"}, {"distinct_range": 1487382338472064000, "num_eq": 4483636271412458890, "num_range": 1487382338472064015, "upper_bound": "\\u0008o"}, {"distinct_range": 10000000, "num_eq": 100, "num_range": 10000000, "upper_bound": "G"}, {"distinct_range": 7564631484276782000, "num_eq": 8844627936978839028, "num_range": 8293258141697644697, "upper_bound": "L"}, {"distinct_range": 1400717688578795500, "num_eq": 80000000000, "num_range": 4090639497057995717, "upper_bound": "[\\u001d\\t\\u00107#5"}, {"distinct_range": 0, "num_eq": 60, "num_range": 936564920230108911, "upper_bound": "x\\u0012YgT"}, {"distinct_range": 503220729547579200, "num_eq": 70000000, "num_range": 510677740179440525, "upper_bound": "}\\u0002U["}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42, "histo_buckets": [{"distinct_range": 0, "num_eq": 2770816837348781903, "num_range": 0, "upper_bound": "-Infinity"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 600000000, "upper_bound": "-1.234E+401"}, {"distinct_range": 8061044824819084000, "num_eq": 6000000000, "num_range": 8061044824819084025, "upper_bound": "-68500675235781.1055"}, {"distinct_range": 1333970417086968800, "num_eq": 4201709367894377026, "num_range": 6318408373875127385, "upper_bound": "-0.0375192454719310767"}, {"distinct_range": 64811767462.771484, "num_eq": 90000000000, "num_range": 90000000000, "upper_bound": "1.980636408087017412E+27"}, {"distinct_range": 50, "num_eq": 1408136605613996460, "num_range": 50, "upper_bound": "9.15200891877644644E+29"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 90}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9037598, "histo_col_type": "BIT(2)", "name": "__auto__", "null_count": 47040451, "row_count": 60000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11266899, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18292359, "histo_buckets": [{"distinct_range": 0, "num_eq": 1948663989369991306, "num_range": 0, "upper_bound": "X"}, {"distinct_range": 4000, "num_eq": 5000, "num_range": 4000, "upper_bound": "[>\\u0017\\n\\u0006\\u0010!\\u0003"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 56694265, "row_count": 60000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39963815, "histo_col_type": "", "name": "__auto__", "null_count": 23923434, "row_count": 60000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23336023, "histo_buckets": [{"distinct_range": 0, "num_eq": 1417271219381297178, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 8696367463906105945, "num_range": 312324443585181041, "upper_bound": " "}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 1447462664536877573, "upper_bound": "8E@z "}, {"distinct_range": 5356323248794985000, "num_eq": 900000, "num_range": 5356323248794985626, "upper_bound": ":6\\u000fg=$UY"}, {"distinct_range": 0, "num_eq": 6287088347683111225, "num_range": 7987328094403243759, "upper_bound": "?ay~\'LI"}, {"distinct_range": 4382128023713124400, "num_eq": 7000000000, "num_range": 4382128023713124548, "upper_bound": "S:?\\u0012($ni"}, {"distinct_range": 1823693349185747500, "num_eq": 60000000, "num_range": 7758021228235081720, "upper_bound": "h"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32410742, "histo_col_type": "", "name": "__auto__", "null_count": 29097567, "row_count": 60000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30717647, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "00001110000110010010001010001101100111110110"}, {"distinct_range": 809117707577449700, "num_eq": 50000000, "num_range": 3174877300045278528, "upper_bound": "01101010010010000101001011101111101111110010"}, {"distinct_range": 70, "num_eq": 0, "num_range": 70, "upper_bound": "10001000110100101000011111010000001111100111"}, {"distinct_range": 3411554913406760400, "num_eq": 100, "num_range": 5959197834092764119, "upper_bound": "10100010101010100101100101011101110111110011"}, {"distinct_range": 0, "num_eq": 6874078191384079984, "num_range": 7000000, "upper_bound": "11000110111011000000000101001111100010101011"}], "histo_col_type": "BIT(44)", "name": "__auto__", "null_count": 48684740, "row_count": 60000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27349567, "histo_col_type": "", "name": "__auto__", "null_count": 8870044, "row_count": 60000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7057380, "histo_buckets": [{"distinct_range": 0, "num_eq": 4641419084116945073, "num_range": 0, "upper_bound": "-1.5297725200653076"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 3580741299203186856, "upper_bound": "-1.140720248222351"}, {"distinct_range": 10000000000, "num_eq": 2154345767704637264, "num_range": 10000000000, "upper_bound": "-0.09569627046585083"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 5318716116843947486, "upper_bound": "0.04730211943387985"}, {"distinct_range": 30000000, "num_eq": 20000, "num_range": 30000000, "upper_bound": "0.16552291810512543"}, {"distinct_range": 200000, "num_eq": 6000000, "num_range": 200000, "upper_bound": "0.4083329141139984"}, {"distinct_range": 727460107.2873309, "num_eq": 10000, "num_range": 1000000000, "upper_bound": "0.4290940761566162"}, {"distinct_range": 0, "num_eq": 8002437738081228126, "num_range": 6258572973219906684, "upper_bound": "1.3261831998825073"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9059777, "histo_buckets": [{"distinct_range": 0, "num_eq": 6330320400340303506, "num_range": 0, "upper_bound": ""}, {"distinct_range": 647730942361006000, "num_eq": 4219005091432352012, "num_range": 3762929953307063039, "upper_bound": "\\u0002\\u0008Edm\\u0012hAB"}, {"distinct_range": 500, "num_eq": 160432346379321760, "num_range": 500, "upper_bound": "\\u0015\\u0013"}, {"distinct_range": 1175123205.1193302, "num_eq": 3000000000, "num_range": 7000000000, "upper_bound": "\\u0015\\u0017T5\\u0001`3"}, {"distinct_range": 10000000, "num_eq": 5000000, "num_range": 10000000, "upper_bound": "]\\u0014ECu-"}, {"distinct_range": 0, "num_eq": 500000, "num_range": 80, "upper_bound": "aEeA+"}, {"distinct_range": 9000000, "num_eq": 10000, "num_range": 9000000, "upper_bound": "\x7f\x7frXw9\\u0016"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 40422737, "row_count": 60000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_col_type": "", "name": "__auto__", "null_count": 10, "row_count": 20}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 14, "row_count": 20}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10, "histo_buckets": [{"distinct_range": 0, "num_eq": 6517699922340543678, "num_range": 0, "upper_bound": "-0.7411007881164551"}, {"distinct_range": 58434535.70280526, "num_eq": 0, "num_range": 60000000, "upper_bound": "0.11981840431690216"}, {"distinct_range": 400000, "num_eq": 3132927982553486342, "num_range": 400000, "upper_bound": "0.18212167918682098"}, {"distinct_range": 76362965.52933873, "num_eq": 700000, "num_range": 80000000, "upper_bound": "0.6438499093055725"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 885505246760133847, "upper_bound": "1.1683107614517212"}, {"distinct_range": 6000, "num_eq": 700, "num_range": 6000, "upper_bound": "1.2514727115631104"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 20}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13, "histo_buckets": [{"distinct_range": 0, "num_eq": 7799688193300768663, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 944672171247834900, "num_eq": 1000000, "num_range": 6929601391460357478, "upper_bound": "314567564"}, {"distinct_range": 4794522917410002000, "num_eq": 6000000000, "num_range": 6632835993223559499, "upper_bound": "2404697856"}, {"distinct_range": 0, "num_eq": 800000, "num_range": 2160793825690665214, "upper_bound": "2559313321"}, {"distinct_range": 200000, "num_eq": 40000, "num_range": 200000, "upper_bound": "2667401080"}, {"distinct_range": 80000000, "num_eq": 80000000, "num_range": 80000000, "upper_bound": "3876062816"}, {"distinct_range": 0, "num_eq": 0, "num_range": 139486197514690442, "upper_bound": "3986845762"}, {"distinct_range": 576.7856969481527, "num_eq": 500000000, "num_range": 800, "upper_bound": "4108719010"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 20}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15, "histo_col_type": "", "name": "__auto__", "null_count": 18, "row_count": 20}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4, "histo_buckets": [{"distinct_range": 0, "num_eq": 6915293481658684683, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 20000000000, "num_range": 300000000, "upper_bound": "422732015"}, {"distinct_range": 0, "num_eq": 7310910608709405095, "num_range": 10000000000, "upper_bound": "555068177"}, {"distinct_range": 0, "num_eq": 4000000, "num_range": 5756230651451418142, "upper_bound": "607336669"}, {"distinct_range": 5342829506074376000, "num_eq": 700, "num_range": 5342829506074376050, "upper_bound": "914762481"}, {"distinct_range": 0, "num_eq": 8242303522160463151, "num_range": 0, "upper_bound": "3416693631"}, {"distinct_range": 7418289396945792000, "num_eq": 200, "num_range": 7418289396945792508, "upper_bound": "3918935464"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 20}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9, "histo_buckets": [{"distinct_range": 0, "num_eq": 177781913376099492, "num_range": 0, "upper_bound": "P"}, {"distinct_range": 1000, "num_eq": 9052428982697780721, "num_range": 1000, "upper_bound": "X"}, {"distinct_range": 24172279.996378068, "num_eq": 30000000000, "num_range": 40000000, "upper_bound": "}3+}\\u0018"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 20}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3, "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 20}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3, "histo_col_type": "CHAR", "name": "__auto__", "null_count": 16, "row_count": 20}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10, "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 20}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('kfply', 'g', 'pbc', 'x');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('gretgf', 'knbf');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('hko', 'gcu', 'ijc');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('hovmp', 'ewmecz', 'zp');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('jxlzsx', 'gpu', 'cqlrmv');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 198009, "histo_buckets": [{"distinct_range": 0, "num_eq": 4554706772738851227, "num_range": 0, "upper_bound": "1421811737"}, {"distinct_range": 14.415489185142992, "num_eq": 8988465036956854580, "num_range": 20, "upper_bound": "2116849966"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 459897, "histo_col_type": "", "name": "__auto__", "null_count": 495091, "row_count": 500000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 315353, "histo_col_type": "", "name": "__auto__", "null_count": 120768, "row_count": 500000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 402010, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33048, "histo_buckets": [{"distinct_range": 0, "num_eq": 40, "num_range": 0, "upper_bound": "-4894432242398277.575"}, {"distinct_range": 1000000, "num_eq": 4160504794702012360, "num_range": 1000000, "upper_bound": "-275942590356272.2601"}, {"distinct_range": 50000000, "num_eq": 2000, "num_range": 50000000, "upper_bound": "-7223351.072237656557"}, {"distinct_range": 293.27425497837356, "num_eq": 4000000, "num_range": 500, "upper_bound": "0"}, {"distinct_range": 6468704151072943000, "num_eq": 4000, "num_range": 6468704151072942942, "upper_bound": "77963.5068681137346"}, {"distinct_range": 0, "num_eq": 3990637736465700362, "num_range": 600, "upper_bound": "2.710009455569767732E+33"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 342141, "histo_col_type": "", "name": "__auto__", "null_count": 420150, "row_count": 500000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15871, "histo_col_type": "", "name": "__auto__", "null_count": 429505, "row_count": 500000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45053, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 76995, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 284701.39218328183, "num_eq": 0, "num_range": 300000, "upper_bound": "493721667"}, {"distinct_range": 0, "num_eq": 4759817833411683754, "num_range": 10000000, "upper_bound": "758121730"}, {"distinct_range": 0, "num_eq": 600000, "num_range": 7000000, "upper_bound": "2016721414"}, {"distinct_range": 3136.9082446563625, "num_eq": 5000000, "num_range": 6000, "upper_bound": "2083860715"}, {"distinct_range": 0, "num_eq": 3282309954596640897, "num_range": 50000000, "upper_bound": "2094389956"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 0, "upper_bound": "3946347795"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 267082, "histo_col_type": "", "name": "__auto__", "null_count": 167678, "row_count": 500000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 84239, "histo_buckets": [{"distinct_range": 0, "num_eq": 6730537511061419182, "num_range": 0, "upper_bound": ":f<[phE"}, {"distinct_range": 930570305173349100, "num_eq": 8000000, "num_range": 1000114554937043905, "upper_bound": "Z\\u001ea>I"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 464978, "row_count": 500000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 140755, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 200677, "histo_buckets": [{"distinct_range": 0, "num_eq": 624239835291914778, "num_range": 0, "upper_bound": "-0.4684862792491913"}, {"distinct_range": 0, "num_eq": 900000, "num_range": 400000000, "upper_bound": "-0.15660405158996582"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 241846, "row_count": 500000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 122032, "histo_col_type": "", "name": "__auto__", "null_count": 253277, "row_count": 500000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 62539, "histo_col_type": "", "name": "__auto__", "null_count": 242956, "row_count": 500000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 420785, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 290280, "histo_buckets": [{"distinct_range": 0, "num_eq": 8842593417102175467, "num_range": 0, "upper_bound": ""}, {"distinct_range": 3856.755593693023, "num_eq": 7099588278785136454, "num_range": 4000, "upper_bound": "\\u0018\\u000bG\\u000c"}, {"distinct_range": 2331382713739228700, "num_eq": 0, "num_range": 4067093381456287693, "upper_bound": " k(\\u0019\\u000cod"}, {"distinct_range": 14.400459635241303, "num_eq": 100000000, "num_range": 90, "upper_bound": "%u\\u001akD\\u001b"}, {"distinct_range": 0, "num_eq": 3140251010909198227, "num_range": 10000, "upper_bound": "\'(&"}, {"distinct_range": 0, "num_eq": 2424593923493942110, "num_range": 7655028214547127016, "upper_bound": "Tj\\u001bM*\\u000eF68"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 40000000, "upper_bound": "uOp\\u001ar^]\\u000b"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 500000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4806383519997800709, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000000, "num_range": 0, "upper_bound": "0000000000000000000000000000000000000000000000000000000000000000"}, {"distinct_range": 691732.709086111, "num_eq": 9000000000, "num_range": 1000000, "upper_bound": "00011100101011110011100000111101101010011001"}, {"distinct_range": 30000000, "num_eq": 10000, "num_range": 30000000, "upper_bound": "01110010110111010010111001100010100101111101"}, {"distinct_range": 369689804.8363962, "num_eq": 1017145340771254333, "num_range": 1000000000, "upper_bound": "0111111111111111111111111111111111111111111111111111111111111111"}, {"distinct_range": 0, "num_eq": 6967342026901495198, "num_range": 525215804638202605, "upper_bound": "11001101001001100001011111101010100100111001"}], "histo_col_type": "BIT(44)", "name": "__auto__", "null_count": 584630914762865842, "row_count": 5241463974187804673}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4613037154787925452, "histo_col_type": "", "name": "__auto__", "null_count": 1293632307826081978, "row_count": 5241463974187804673}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4714838606756769743, "histo_col_type": "", "name": "__auto__", "null_count": 262257185036128605, "row_count": 5241463974187804673}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4675904168895680591, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000000, "num_range": 0, "upper_bound": ";_\\u0011#U69N"}, {"distinct_range": 0, "num_eq": 5000000, "num_range": 10000000000, "upper_bound": "?]"}, {"distinct_range": 3325528366305917000, "num_eq": 100000, "num_range": 7251290771846046664, "upper_bound": "@g\\r"}, {"distinct_range": 50000000000, "num_eq": 6937336404565669070, "num_range": 50000000000, "upper_bound": "Le33w\\u000cr"}, {"distinct_range": 600000000, "num_eq": 859755885894125910, "num_range": 600000000, "upper_bound": "W\\tq"}, {"distinct_range": 1681507874812641000, "num_eq": 8582263906004992011, "num_range": 9128157337407715247, "upper_bound": "`8va"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 9001818286573855175, "upper_bound": "d\\\\i\\u0014"}, {"distinct_range": 0, "num_eq": 7842169712958405479, "num_range": 5716318188679077243, "upper_bound": "\x7fc!M\\u000bf\\u0006\\u0017+"}, {"distinct_range": 256956646564570400, "num_eq": 8944731344787385387, "num_range": 256956646564570402, "upper_bound": "\U00002603"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 2315182204140367485, "row_count": 5241463974187804673}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4283357867663777449, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "00"}, {"distinct_range": 0, "num_eq": 7272548175186924107, "num_range": 5000000000, "upper_bound": "01"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 0, "upper_bound": "10"}, {"distinct_range": 0, "num_eq": 9000000000, "num_range": 60000, "upper_bound": "11"}], "histo_col_type": "BIT(2)", "name": "__auto__", "null_count": 3988315667080951560, "row_count": 5241463974187804673}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1356503714264991054, "histo_col_type": "", "name": "__auto__", "null_count": 2769069944890885273, "row_count": 5241463974187804673}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3422200361899364666, "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 5241463974187804673}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 690592959262552659, "histo_buckets": [{"distinct_range": 0, "num_eq": 8063587464778553883, "num_range": 0, "upper_bound": "\\""}, {"distinct_range": 7593732583697416000, "num_eq": 8836562124363502469, "num_range": 7593732583697415827, "upper_bound": "fg\\"!Y"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 1334675544270041789, "row_count": 5241463974187804673}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5098277735488877921, "histo_buckets": [{"distinct_range": 0, "num_eq": 9129241272690377618, "num_range": 0, "upper_bound": "\\u0004"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 6880971571394619177, "upper_bound": "\\r"}, {"distinct_range": 3670357680513538600, "num_eq": 3489537907328391601, "num_range": 5960225627674813961, "upper_bound": "\\u0018"}, {"distinct_range": 5302009845023131000, "num_eq": 20, "num_range": 7977416789591637539, "upper_bound": "2"}, {"distinct_range": 0, "num_eq": 60000, "num_range": 500000, "upper_bound": "<%5("}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 0, "upper_bound": "te7\\u0017rX\\u0014\\u001aP"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 5241463974187804673}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 91908640019005433, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5241463974187804673}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1998404528382226581, "histo_col_type": "", "name": "__auto__", "null_count": 1714041799156117487, "row_count": 2383537918066375815}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1434467022412732807, "histo_buckets": [{"distinct_range": 0, "num_eq": 3930332677901136725, "num_range": 0, "upper_bound": "/n"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "9\\u0013k\\u001d\\u0018CD"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 2383537918066375815}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 547785025348390438, "histo_buckets": [{"distinct_range": 0, "num_eq": 4792580805581636999, "num_range": 0, "upper_bound": "\\\\x2db1"}, {"distinct_range": 543657612825015100, "num_eq": 7000, "num_range": 5034107267402355026, "upper_bound": "\\\\x7ed72b7125a4a4f7"}, {"distinct_range": 974232666976383000, "num_eq": 1000000, "num_range": 2317479737116340496, "upper_bound": "\\\\xa1ede6708e9d58"}, {"distinct_range": 1177326885517026300, "num_eq": 2550497103312136720, "num_range": 2499592892636991986, "upper_bound": "\\\\xba31806630"}, {"distinct_range": 4229308200664876000, "num_eq": 40, "num_range": 4229308200664875992, "upper_bound": "\\\\xe356f992"}, {"distinct_range": 500000000, "num_eq": 9020830957473682379, "num_range": 500000000, "upper_bound": "\\\\xe3e8f2e6"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 2383537918066375815}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 992138353140388524, "histo_col_type": "", "name": "__auto__", "null_count": 304550545864315855, "row_count": 2383537918066375815}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1780604919602876430, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2383537918066375815}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 401967271907195939, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2383537918066375815}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 433953818733664349, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 300000000, "num_range": 7000000, "upper_bound": "1711655200"}, {"distinct_range": 6014857333059414000, "num_eq": 8000, "num_range": 6904927638956085282, "upper_bound": "1992433914"}, {"distinct_range": 6031341974370263000, "num_eq": 4156748482230534958, "num_range": 6031341974370263189, "upper_bound": "2486076739"}, {"distinct_range": 2608448538.896036, "num_eq": 100, "num_range": 3000000000, "upper_bound": "3007432153"}, {"distinct_range": 500000, "num_eq": 1094101936435423969, "num_range": 500000, "upper_bound": "3219153395"}, {"distinct_range": 4303693363033723400, "num_eq": 6856013341107043388, "num_range": 5341670024251267774, "upper_bound": "3985352571"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 2383537918066375815}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 779981903959677170, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000000, "num_range": 0, "upper_bound": "3341849764"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 2383537918066375815}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2094685192406685336, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2383537918066375815}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1606203103718751908, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2383537918066375815}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1466558624267992664, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": "-0.9080224633216858"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 2383537918066375815}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1044093541936642114, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000000, "num_range": 0, "upper_bound": "\\u0011J>u@"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 8000, "upper_bound": "\\""}, {"distinct_range": 1883922026922888400, "num_eq": 472421051761450242, "num_range": 2802896293128603351, "upper_bound": "\'#wdv2"}, {"distinct_range": 3638615505406934000, "num_eq": 6073480184071294427, "num_range": 3638615505406933914, "upper_bound": "V\\u0004"}, {"distinct_range": 1158.0253854486918, "num_eq": 30000000000, "num_range": 2000, "upper_bound": "W\\u001fsj\\u0017#"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 802305001896157008, "row_count": 2383537918066375815}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1777674046233623104, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000000, "num_range": 0, "upper_bound": "-infinity"}, {"distinct_range": 3026193354279865300, "num_eq": 549985985453590622, "num_range": 3026193354279865199, "upper_bound": "1972-05-01"}, {"distinct_range": 292285665.958292, "num_eq": 30, "num_range": 400000000, "upper_bound": "1975-12-11"}, {"distinct_range": 3431567759487999000, "num_eq": 3152474385530089866, "num_range": 3431567759487998773, "upper_bound": "1983-06-25"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 2383537918066375815}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1894221829307689660, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2383537918066375815}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1087969798545644135, "histo_col_type": "", "name": "__auto__", "null_count": 943173200560331032, "row_count": 2383537918066375815}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2311175916546541814, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2383537918066375815}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3634524830384446404, "histo_col_type": "", "name": "__auto__", "null_count": 3169418954440352811, "row_count": 8536039505070130330}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6466050288128009466, "histo_buckets": [{"distinct_range": 0, "num_eq": 818815164344935296, "num_range": 0, "upper_bound": "-1.234E+401"}, {"distinct_range": 1755769649242521300, "num_eq": 9000000, "num_range": 8320384023903603837, "upper_bound": "-3.295288369186306608E+37"}, {"distinct_range": 4039805334127094000, "num_eq": 80000000000, "num_range": 4039805334127093754, "upper_bound": "-1"}, {"distinct_range": 0, "num_eq": 5667016723191658688, "num_range": 60000000, "upper_bound": "-0.5289364970705241322"}, {"distinct_range": 0, "num_eq": 900000, "num_range": 80000000000, "upper_bound": "-0.4047882278867247064"}, {"distinct_range": 0, "num_eq": 3502974648526739481, "num_range": 30000000000, "upper_bound": "4.127323609754587974E+21"}, {"distinct_range": 700000, "num_eq": 113918737803638437, "num_range": 700000, "upper_bound": "3.958063771485068188E+36"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 8536039505070130330}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5691761029028513518, "histo_col_type": "", "name": "__auto__", "null_count": 1657029218903807713, "row_count": 8536039505070130330}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8523416715722126658, "histo_col_type": "", "name": "__auto__", "null_count": 5763288273220827161, "row_count": 8536039505070130330}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5033897101915325702, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8536039505070130330}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 622006482601142325, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000, "num_range": 0, "upper_bound": "1672335420"}, {"distinct_range": 0, "num_eq": 20, "num_range": 7799576681249123607, "upper_bound": "4205767526"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 8536039505070130330}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8244610783160251862, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 20000, "upper_bound": "330287748"}, {"distinct_range": 366.3395294362518, "num_eq": 10000000, "num_range": 1000, "upper_bound": "441541913"}, {"distinct_range": 1894950583825048000, "num_eq": 200000, "num_range": 6445553877720801339, "upper_bound": "573034921"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 8536039505070130330}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4446780442202947894, "histo_col_type": "", "name": "__auto__", "null_count": 5891851422361827283, "row_count": 8536039505070130330}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7177017293180117357, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000000, "num_range": 0, "upper_bound": "-0.1270720660686493"}, {"distinct_range": 800000000, "num_eq": 4000000, "num_range": 800000000, "upper_bound": "0.48151659965515137"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 2056258322268967362, "upper_bound": "0.9458360075950623"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 7573265857281001129, "row_count": 8536039505070130330}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7776030055338298936, "histo_buckets": [{"distinct_range": 0, "num_eq": 4661176473528560338, "num_range": 0, "upper_bound": ""}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 8536039505070130330}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7223216189032025350, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8536039505070130330}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5614201299665768778, "histo_col_type": "", "name": "__auto__", "null_count": 4996068051271955613, "row_count": 8536039505070130330}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1382291803837653314, "histo_col_type": "", "name": "__auto__", "null_count": 7940533118660179691, "row_count": 8536039505070130330}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3056783045818141710, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 10.570473494994234, "num_eq": 90, "num_range": 30, "upper_bound": "\\u0014mQ\\u0016Q"}, {"distinct_range": 9928.77200901896, "num_eq": 1000, "num_range": 10000, "upper_bound": "1\x7f\\u001e\\u0008"}, {"distinct_range": 93695.84803314363, "num_eq": 6221223594192401163, "num_range": 800000, "upper_bound": "<?\\u000bF/[=#"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 4238936348112448301, "upper_bound": "EUNU"}, {"distinct_range": 0, "num_eq": 6000000, "num_range": 239462928895238011, "upper_bound": "J<\\u000e"}, {"distinct_range": 669315.1946475842, "num_eq": 80000000000, "num_range": 1000000, "upper_bound": "X"}, {"distinct_range": 4171993178973052400, "num_eq": 1097632947266587053, "num_range": 4171993178973052256, "upper_bound": "Y\\u0006C"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 2033331288770414515, "row_count": 8536039505070130330}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3887533779056087295, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8536039505070130330}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8126816876257232812, "histo_col_type": "", "name": "__auto__", "null_count": 7528888055659437526, "row_count": 8536039505070130330}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2353014848922166771, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8536039505070130330}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 320719716007072574, "histo_buckets": [{"distinct_range": 0, "num_eq": 2983121218323062981, "num_range": 0, "upper_bound": "-0.17355069518089294"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 2527621723300496747}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 201444347009494575, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": ""}, {"distinct_range": 33540869224.591396, "num_eq": 40, "num_range": 50000000000, "upper_bound": "\\u000elq60WW"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 0, "upper_bound": "*;L\\n\\""}, {"distinct_range": 980688.6491515834, "num_eq": 100000, "num_range": 2000000, "upper_bound": "+g\\u0010g9"}, {"distinct_range": 5933893768651976000, "num_eq": 0, "num_range": 5933893768651976149, "upper_bound": "N|"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 243757403045189883, "row_count": 2527621723300496747}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1001513482352539366, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2527621723300496747}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1317101222842275490, "histo_col_type": "", "name": "__auto__", "null_count": 1667732786955377680, "row_count": 2527621723300496747}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 732542743174285823, "histo_buckets": [{"distinct_range": 0, "num_eq": 5049431262399237873, "num_range": 0, "upper_bound": "01010100100010100000100010011001100001010101"}, {"distinct_range": 1938461731428245500, "num_eq": 8820438255841205349, "num_range": 5122763138198264608, "upper_bound": "01100111111110110001001110000110011111000000"}, {"distinct_range": 0, "num_eq": 7281934086621146458, "num_range": 1850802724286484265, "upper_bound": "10010000101000110000101100100011101001101110"}, {"distinct_range": 1207375958927434200, "num_eq": 8580658349944259160, "num_range": 1207375958927434153, "upper_bound": "10010011101001111110110111000001100010110011"}, {"distinct_range": 0, "num_eq": 100, "num_range": 2400124518102337361, "upper_bound": "11000011000010100111011100110111000110000101"}, {"distinct_range": 0, "num_eq": 5529017120165387636, "num_range": 6000000000, "upper_bound": "11010011000101110000100111100110011101110010"}, {"distinct_range": 600000, "num_eq": 1388583403432394294, "num_range": 600000, "upper_bound": "11111100001000001011111011000000111011011000"}], "histo_col_type": "BIT(44)", "name": "__auto__", "null_count": 505251351115794046, "row_count": 2527621723300496747}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2507589582656764996, "histo_col_type": "", "name": "__auto__", "null_count": 565632159323872006, "row_count": 2527621723300496747}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 705785768086689946, "histo_buckets": [{"distinct_range": 0, "num_eq": 10, "num_range": 0, "upper_bound": "\\t("}, {"distinct_range": 336078362.3080659, "num_eq": 2000000, "num_range": 400000000, "upper_bound": "\\""}, {"distinct_range": 0, "num_eq": 200000000, "num_range": 0, "upper_bound": "4vp&W 6"}, {"distinct_range": 982117218918448500, "num_eq": 600, "num_range": 3431910217805363053, "upper_bound": ":Jo\\u0015O#"}, {"distinct_range": 7205101586867725000, "num_eq": 200, "num_range": 7205101586867725098, "upper_bound": "Tv"}, {"distinct_range": 0, "num_eq": 2392308799972826572, "num_range": 8284663111562763972, "upper_bound": "[\\u0001\\u0004"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 835987204477534674, "row_count": 2527621723300496747}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 361874861185381958, "histo_col_type": "", "name": "__auto__", "null_count": 2139837999260886053, "row_count": 2527621723300496747}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 274121897933850611, "histo_buckets": [{"distinct_range": 0, "num_eq": 8571657824282088403, "num_range": 0, "upper_bound": "00"}], "histo_col_type": "BIT(2)", "name": "__auto__", "null_count": 1650255922095056876, "row_count": 2527621723300496747}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1369535744660645969, "histo_buckets": [{"distinct_range": 0, "num_eq": 9196613045468207469, "num_range": 0, "upper_bound": ""}, {"distinct_range": 1664701187.9894855, "num_eq": 70000000, "num_range": 8000000000, "upper_bound": "+\\u000fq\\u0014\\u0017"}, {"distinct_range": 0, "num_eq": 7678095938235754153, "num_range": 500000000, "upper_bound": "Cc\\u0010cW\\u0002HE"}, {"distinct_range": 596362884887616500, "num_eq": 5000000000, "num_range": 1696590205413819712, "upper_bound": "G\\u0011{"}, {"distinct_range": 999144.3883519508, "num_eq": 20000, "num_range": 1000000, "upper_bound": "H^J;"}, {"distinct_range": 1432420070300475000, "num_eq": 5648225651855835759, "num_range": 2169723788126569339, "upper_bound": "IwGf1t^X"}, {"distinct_range": 0, "num_eq": 0, "num_range": 1415037306529415226, "upper_bound": "X"}, {"distinct_range": 0, "num_eq": 8205441179029647342, "num_range": 700000000, "upper_bound": "u\\u000f#.~w"}, {"distinct_range": 5210622538857327000, "num_eq": 50000, "num_range": 5210622538857326490, "upper_bound": "|\\nV\\u0010"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 2527621723300496747}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 71, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000000, "num_range": 0, "upper_bound": "1979-12-28"}, {"distinct_range": 126401875052385330, "num_eq": 300000000, "num_range": 8035582890316588175, "upper_bound": "1981-05-10"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 72, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000, "num_range": 0, "upper_bound": "3363032623"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9, "histo_col_type": "", "name": "__auto__", "null_count": 61, "row_count": 90}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2, "histo_col_type": "", "name": "__auto__", "null_count": 83, "row_count": 90}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 68, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "\\\\x5bae"}, {"distinct_range": 0, "num_eq": 0, "num_range": 2148109360352551117, "upper_bound": "\\\\xd93b6bcf"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 71, "histo_buckets": [{"distinct_range": 0, "num_eq": 3335439654133462285, "num_range": 0, "upper_bound": ""}, {"distinct_range": 976967.7834789685, "num_eq": 7989226087620136667, "num_range": 1000000, "upper_bound": "\\n\\u0016\\u0001pA"}, {"distinct_range": 0, "num_eq": 500, "num_range": 451049660751112851, "upper_bound": "BL\\u0010,o\\u0001\\""}, {"distinct_range": 0, "num_eq": 700000, "num_range": 80000000, "upper_bound": "i\\u0012\\n\\u001fJ\\u0017SM"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 37, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 76, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col3_6"], "created_at": "2000-01-01

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2477208&tab=buildLog) on [master@361215163c597bd1460bba65ca3298f37e29aacc](https://github.com/cockroachdb/cockroach/commits/361215163c597bd1460bba65ca3298f37e29aacc):

```
        cockroach1_1  | nodeID:              1
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        test_1        | --- FAIL: TestCompare (2.19s)
        test_1        |     --- FAIL: TestCompare/postgres (1.12s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (1.07s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 REGNAMESPACE NOT NULL, col1_1 INTERVAL, col1_2 INT2 NOT NULL, col1_3 INET, col1_4 FLOAT4 NOT NULL, col1_5 DECIMAL NULL, col1_6 REGPROCEDURE NOT NULL, col1_7 INTERVAL NULL, col1_8 BYTES, col1_9 INT4 NULL, col1_10 REGCLASS NOT NULL, col1_11 NAME, col1_12 DATE NULL, col1_13 VARCHAR NOT NULL, PRIMARY KEY (col1_6 DESC, col1_10 DESC, col1_13 ASC, col1_0 ASC), FAMILY (col1_2), FAMILY (col1_0), FAMILY (col1_1, col1_4, col1_11, col1_8, col1_3, col1_7), FAMILY (col1_10), FAMILY (col1_13), FAMILY (col1_5), FAMILY (col1_6, col1_9, col1_12), UNIQUE (col1_4) WHERE (((((table1.col1_5 <= -1) AND (table1.col1_2 >= 32767)) AND (table1.col1_8 = '\xe29883')) OR (table1.col1_11 >= e'\x00')) AND (table1.col1_9 >= 2147483647)) AND (table1.col1_4 != +Inf), UNIQUE (col1_0));
        test_1        |             CREATE TABLE table2 (col2_0 TIMESTAMPTZ NOT NULL, col2_1 OID NOT NULL, col2_2 FLOAT8 NOT NULL, col2_3 REGCLASS NOT NULL, col2_4 TIMESTAMPTZ NOT NULL, col2_5 UUID, PRIMARY KEY (col2_0 ASC, col2_2, col2_4 ASC), UNIQUE (col2_5, col2_0 ASC, col2_2, col2_4 DESC, col2_1 DESC, col2_3) WHERE table2.col2_0 < '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ, INDEX (col2_2, col2_3 DESC, col2_0 DESC, col2_4 ASC) STORING (col2_5) WHERE ((table2.col2_0 = '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ) OR (table2.col2_2 > 1.7976931348623157e+308:::FLOAT8)) AND (table2.col2_4 >= '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ), UNIQUE (col2_2 DESC, col2_4, col2_5 ASC, col2_1, col2_3 DESC) WHERE ((table2.col2_0 > '-2000-01-01 00:00:00+00:00') AND (table2.col2_2 >= 1.0)) OR (table2.col2_4 > '3000-01-01 00:00:00+00:00'), FAMILY (col2_1), FAMILY (col2_3, col2_0), FAMILY (col2_5), FAMILY (col2_2), FAMILY (col2_4));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1717674552263503527, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3402068488254635103}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2392584643304495551, "histo_col_type": "", "name": "__auto__", "null_count": 1779980373125102398, "row_count": 3402068488254635103}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 705533796950414014, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000000, "num_range": 0, "upper_bound": "7382807"}, {"distinct_range": 0, "num_eq": 4000000000, "num_range": 400, "upper_bound": "933824725"}, {"distinct_range": 0, "num_eq": 7272210657335392385, "num_range": 4000, "upper_bound": "967584545"}, {"distinct_range": 5673673155460612000, "num_eq": 4023658811025771041, "num_range": 5673673155460612020, "upper_bound": "1598205638"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 8355256021013873557, "upper_bound": "1612159746"}, {"distinct_range": 500, "num_eq": 971881222053142179, "num_range": 500, "upper_bound": "2799382068"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 3402068488254635103}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 930166831536112142, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3402068488254635103}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1159972447899706939, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3402068488254635103}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3145163995903959197, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3402068488254635103}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3097395355529049168, "histo_col_type": "", "name": "__auto__", "null_count": 1609999617601266487, "row_count": 3402068488254635103}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1067864816311296851, "histo_col_type": "", "name": "__auto__", "null_count": 893795077920739159, "row_count": 3402068488254635103}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 232999266467671220, "histo_col_type": "", "name": "__auto__", "null_count": 1912303145831581851, "row_count": 3402068488254635103}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3386269260844581295, "histo_col_type": "", "name": "__auto__", "null_count": 2422661394994784758, "row_count": 3402068488254635103}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1821149085976265143, "histo_col_type": "", "name": "__auto__", "null_count": 1182852656292808624, "row_count": 3402068488254635103}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2895211621847313196, "histo_col_type": "", "name": "__auto__", "null_count": 1559766658044443882, "row_count": 3402068488254635103}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 938922418461526229, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3402068488254635103}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2563009438689217150, "histo_col_type": "", "name": "__auto__", "null_count": 2320484754451053781, "row_count": 3402068488254635103}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5865388, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "7054f157-b001-448c-ac50-b3a399469e9f"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 1632568, "row_count": 10000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5833102, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000000, "num_range": 0, "upper_bound": "1999-04-06 07:30:27.000445+00:00"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 70000, "upper_bound": "2005-10-07 07:19:47.00011+00:00"}, {"distinct_range": 4192.824096867166, "num_eq": 80, "num_range": 5000, "upper_bound": "2008-06-08 18:27:02.000273+00:00"}, {"distinct_range": 0, "num_eq": 2238351446341281900, "num_range": 0, "upper_bound": "2021-03-30 11:47:30.00073+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5223145, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 614315, "histo_buckets": [{"distinct_range": 0, "num_eq": 1754455829113423261, "num_range": 0, "upper_bound": "-2.718090142420091"}, {"distinct_range": 0, "num_eq": 900000000, "num_range": 200, "upper_bound": "-0.5725093094551388"}, {"distinct_range": 5408220601867965000, "num_eq": 3000000, "num_range": 6864860195783014004, "upper_bound": "-0.20099852565431697"}, {"distinct_range": 0, "num_eq": 0, "num_range": 3614837596461853045, "upper_bound": "0.015119189188637139"}, {"distinct_range": 9186342162909427000, "num_eq": 600000000, "num_range": 9186342162909426946, "upper_bound": "0.029772108567161615"}, {"distinct_range": 53027814.05848147, "num_eq": 8000000, "num_range": 70000000, "upper_bound": "1.0078316080927552"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8310564, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 385865, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}]':::JSONB;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_2) REFERENCES table1 (col1_4);
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_1) REFERENCES table1 (col1_0) ON DELETE CASCADE;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('bcaseu', 'knvf', 'zxzv', 'y');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('gzlkgx', 'jxjuly', 'yytfn', 'r', 'fshsod');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('cyeph');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('p', 'x', 'w', 'ev', 'hdek');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('zzenp', 'eijsk', 'dwkfio', 'ui', 'wzllw');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1184141327334301657, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000, "num_range": 0, "upper_bound": "312302656"}, {"distinct_range": 0, "num_eq": 3267922016683120978, "num_range": 80000000, "upper_bound": "2303918721"}, {"distinct_range": 6249727115228189000, "num_eq": 60000000000, "num_range": 6249727115228188688, "upper_bound": "3079620635"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 1186122113717285531}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 467415942080137837, "histo_col_type": "", "name": "__auto__", "null_count": 373790266748441766, "row_count": 1186122113717285531}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 774365354474894849, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1186122113717285531}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 446936410115902214, "histo_col_type": "", "name": "__auto__", "null_count": 741437645193899094, "row_count": 1186122113717285531}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 117102804000140809, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1186122113717285531}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1086459507848517294, "histo_col_type": "", "name": "__auto__", "null_count": 422500119574626596, "row_count": 1186122113717285531}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1106218689704727452, "histo_col_type": "", "name": "__auto__", "null_count": 37317293716968104, "row_count": 1186122113717285531}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 816282551571458161, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000, "num_range": 0, "upper_bound": "-1.3904199600219727"}, {"distinct_range": 6173103647472195000, "num_eq": 5401575261290444207, "num_range": 7788111115888576807, "upper_bound": "-0.5251715183258057"}, {"distinct_range": 253227442258417600, "num_eq": 10000, "num_range": 7694835111233489314, "upper_bound": "-0.4531900882720947"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 1186122113717285531}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3659510736197249, "histo_col_type": "", "name": "__auto__", "null_count": 1031862384691697345, "row_count": 1186122113717285531}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 666660923281977197, "histo_col_type": "", "name": "__auto__", "null_count": 1029532819797819951, "row_count": 1186122113717285531}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 860076378486445031, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1186122113717285531}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 176092835232161462, "histo_col_type": "", "name": "__auto__", "null_count": 368030847538555044, "row_count": 1186122113717285531}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 73904715442681403, "histo_buckets": [{"distinct_range": 0, "num_eq": 50, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 7893883777765859000, "num_eq": 10000, "num_range": 8300772304649367809, "upper_bound": "43154430"}, {"distinct_range": 70000000, "num_eq": 80000000, "num_range": 70000000, "upper_bound": "641513439"}, {"distinct_range": 0, "num_eq": 700, "num_range": 90, "upper_bound": "1450251332"}, {"distinct_range": 19603137.232744753, "num_eq": 10000000000, "num_range": 20000000, "upper_bound": "1968936617"}, {"distinct_range": 66474567.96867537, "num_eq": 500000, "num_range": 80000000, "upper_bound": "2216963529"}, {"distinct_range": 2294084682341404200, "num_eq": 8233259659475823464, "num_range": 2294084682341404235, "upper_bound": "2411781573"}, {"distinct_range": 0, "num_eq": 8124502304637529010, "num_range": 1905674212087386847, "upper_bound": "2873846425"}, {"distinct_range": 5817315267703174000, "num_eq": 6793002307265007300, "num_range": 5817315267703174401, "upper_bound": "3476600499"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 1186122113717285531}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 125246001220315668, "histo_col_type": "", "name": "__auto__", "null_count": 693923255467478065, "row_count": 1186122113717285531}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3559889566053487447, "histo_buckets": [{"distinct_range": 0, "num_eq": 20, "num_range": 0, "upper_bound": "1987-09-24 09:49:42.000689+00:00"}, {"distinct_range": 70000000000, "num_eq": 700000, "num_range": 70000000000, "upper_bound": "1989-11-15 09:38:10.000473+00:00"}, {"distinct_range": 0, "num_eq": 6530646201124174867, "num_range": 0, "upper_bound": "2000-03-18 21:32:48.000966+00:00"}, {"distinct_range": 40491623.56067833, "num_eq": 50000000, "num_range": 70000000, "upper_bound": "2006-02-27 20:12:42.000821+00:00"}, {"distinct_range": 90000, "num_eq": 755559006767244876, "num_range": 90000, "upper_bound": "2008-01-04 02:11:47.000251+00:00"}, {"distinct_range": 40000000, "num_eq": 1422312783602741441, "num_range": 40000000, "upper_bound": "2011-06-17 10:00:38.00066+00:00"}, {"distinct_range": 7354265966581920000, "num_eq": 40000000, "num_range": 7354265966581920253, "upper_bound": "2023-03-17 03:49:08.000644+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 7896003599538049586}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6053369711735037362, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7896003599538049586}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3034240937788668343, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "-1.2087694875891868"}, {"distinct_range": 0, "num_eq": 1092017472033522992, "num_range": 2000000, "upper_bound": "0.06383604148162703"}, {"distinct_range": 0, "num_eq": 5860947753358202444, "num_range": 0, "upper_bound": "1.7976931348623157e+308"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 7896003599538049586}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1734977587957926111, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7896003599538049586}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6171319403281149013, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7896003599538049586}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2239886646057262608, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "07475ff6-5b8f-431d-a345-732102fcd416"}, {"distinct_range": 0, "num_eq": 20000000000, "num_range": 100, "upper_bound": "5fc3a728-f4d8-4e2d-b7c6-419d656bf681"}, {"distinct_range": 1015410001779607700, "num_eq": 4674080344902848598, "num_range": 1015410001779607711, "upper_bound": "a9d6e1d9-e7d3-483d-ba54-ed12ff412b29"}, {"distinct_range": 0, "num_eq": 6509727070331095574, "num_range": 6893236689294126112, "upper_bound": "b4732a09-e5e9-48fc-8c20-056edc65974f"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 3000, "upper_bound": "b5f5036c-38fe-4d6d-9956-1e7632bf38a3"}, {"distinct_range": 289.9212275314686, "num_eq": 153144039082986877, "num_range": 1000, "upper_bound": "fa76c3c6-d1c9-45f5-bcaf-12e04a428938"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 7529760920598465140, "row_count": 7896003599538049586}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 899665331711203429, "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 1425896502927862055}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 198568555563342716, "histo_col_type": "", "name": "__auto__", "null_count": 298680658228240595, "row_count": 1425896502927862055}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1041940949754884531, "histo_col_type": "", "name": "__auto__", "null_count": 15612103101698972, "row_count": 1425896502927862055}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 491382716973595711, "histo_buckets": [{"distinct_range": 0, "num_eq": 2810379066164149104, "num_range": 0, "upper_bound": "1407923458"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 1425896502927862055}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 57223288275674758, "histo_buckets": [{"distinct_range": 0, "num_eq": 9015593327550648371, "num_range": 0, "upper_bound": "201337524"}, {"distinct_range": 1000000, "num_eq": 300000, "num_range": 1000000, "upper_bound": "287980368"}, {"distinct_range": 75960.45738230456, "num_eq": 9191422431777678954, "num_range": 90000, "upper_bound": "307298408"}, {"distinct_range": 2000000000, "num_eq": 85696991191552107, "num_range": 2000000000, "upper_bound": "528941464"}, {"distinct_range": 761424835767424400, "num_eq": 7956249683098876940, "num_range": 761424835767424385, "upper_bound": "760505345"}, {"distinct_range": 0, "num_eq": 9051015100700817355, "num_range": 2348179723793389792, "upper_bound": "1656113311"}, {"distinct_range": 80, "num_eq": 642548031345365235, "num_range": 80, "upper_bound": "1872012488"}, {"distinct_range": 0, "num_eq": 80, "num_range": 4085874743554811989, "upper_bound": "2172294631"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 1425896502927862055}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1070881741863874561, "histo_col_type": "", "name": "__auto__", "null_count": 1378931872240124766, "row_count": 1425896502927862055}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1387040232947654826, "histo_col_type": "", "name": "__auto__", "null_count": 1297931949978975876, "row_count": 1425896502927862055}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 379629743845568690, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1425896502927862055}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 141973020058199413, "histo_col_type": "", "name": "__auto__", "null_count": 169377727817232358, "row_count": 1425896502927862055}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 388358695098991640, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1425896502927862055}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 126022397896972441, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1425896502927862055}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 329057683793202049, "histo_col_type": "", "name": "__auto__", "null_count": 101216036874957682, "row_count": 1425896502927862055}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 335816280670556747, "histo_col_type": "", "name": "__auto__", "null_count": 1292993153494793999, "row_count": 1425896502927862055}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1305834944702414104, "histo_col_type": "", "name": "__auto__", "null_count": 1092194759675867569, "row_count": 1425896502927862055}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1856464229347712256, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7996659797832968230}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6097905671415488869, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000000, "num_range": 0, "upper_bound": "ffffffff-ffff-ffff-ffff-ffffffffffff"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 508937779740171665, "row_count": 7996659797832968230}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4822063987131292847, "histo_buckets": [{"distinct_range": 0, "num_eq": 30, "num_range": 0, "upper_bound": "1974-12-06 09:11:23.000561+00:00"}, {"distinct_range": 757625588882918500, "num_eq": 9000000000, "num_range": 3086299418054144316, "upper_bound": "1976-03-08 23:15:09.000248+00:00"}, {"distinct_range": 401060167629689300, "num_eq": 1480772205803583773, "num_range": 401060167629689274, "upper_bound": "1982-08-31 10:40:28.00022+00:00"}, {"distinct_range": 0, "num_eq": 7186153912302343590, "num_range": 700, "upper_bound": "1994-09-04 13:37:30.000585+00:00"}, {"distinct_range": 9097331847515717000, "num_eq": 3817160225986716993, "num_range": 9097331847515717103, "upper_bound": "2009-10-15 01:16:39.000671+00:00"}, {"distinct_range": 0, "num_eq": 4151912293956540793, "num_range": 0, "upper_bound": "2029-06-13 14:54:39.000732+00:00"}, {"distinct_range": 50000000000, "num_eq": 4000, "num_range": 50000000000, "upper_bound": "2030-02-23 08:38:42.000984+00:00"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 4212111283226173032, "upper_bound": "2031-11-04 01:27:01.000928+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 7996659797832968230}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3800697323138576533, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7996659797832968230}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 104779809872687227, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000000, "num_range": 0, "upper_bound": "-0.011735068413898653"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 7996659797832968230}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5943825906591388599, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7996659797832968230}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "inf" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2020-11-27 06:11:56.392 UTC [47] LOG:  received fast shutdown request
        postgres_1    | .2020-11-27 06:11:56.395 UTC [47] LOG:  aborting any active transactions
        postgres_1    | 2020-11-27 06:11:56.396 UTC [47] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
        postgres_1    | 2020-11-27 06:11:56.402 UTC [49] LOG:  shutting down
        postgres_1    | 2020-11-27 06:11:56.522 UTC [47] LOG:  database system is shut down
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2479008&tab=buildLog) on [master@54cdc5fec0e7dc835af7d2fc4231b52d49a71bf8](https://github.com/cockroachdb/cockroach/commits/54cdc5fec0e7dc835af7d2fc4231b52d49a71bf8):

```
        postgres_1    | 				JOIN postgres.public.table2 AS tab_149995 ON
        postgres_1    | 						(tab_149994.tableoid) = (tab_149995.col2_14)
        postgres_1    | 						AND (tab_149994.tableoid) = (tab_149995.col2_10)
        postgres_1    | 						AND (tab_149994.crdb_internal_mvcc_timestamp) = (tab_149995.crdb_internal_mvcc_timestamp)
        postgres_1    | 						AND (tab_149994.crdb_internal_mvcc_timestamp) = (tab_149995.col2_1)
        postgres_1    | 			WHERE
        postgres_1    | 				tab_149995.col2_7
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_149995.col2_13 DESC, tab_149995.col2_8 ASC
        postgres_1    | 		)
        postgres_1    | 			AS tab_149996
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_252552 NULLS FIRST,
        postgres_1    | 		col_252553 NULLS FIRST,
        postgres_1    | 		col_252554 NULLS FIRST,
        postgres_1    | 		col_252555 NULLS FIRST,
        postgres_1    | 		col_252556 NULLS FIRST,
        postgres_1    | 		col_252557 NULLS FIRST,
        postgres_1    | 		col_252558 NULLS FIRST,
        postgres_1    | 		col_252559 NULLS FIRST,
        postgres_1    | 		col_252560 NULLS FIRST
        test_1        | --- FAIL: TestCompare (760.22s)
        test_1        |     --- FAIL: TestCompare/postgres (158.18s)
        test_1        |         compare_test.go:155: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("J6"),
        test_1        |             + 	string("J6.0"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('J'::TEXT::TEXT || 6.0::FLOAT8::FLOAT8)::TEXT AS col_252561) AS tab_149997
        test_1        |             ORDER BY
        test_1        |             	col_252561 NULLS FIRST;
        test_1        |             
        test_1        |             postgres: [same as previous]
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2480415&tab=buildLog) on [master@76935fd62c5a76f88b754cd3f9a5bfb3ccf1d8c2](https://github.com/cockroachdb/cockroach/commits/76935fd62c5a76f88b754cd3f9a5bfb3ccf1d8c2):

```
        test_1        | --- FAIL: TestCompare (607.77s)
        test_1        |     --- FAIL: TestCompare/postgres (4.54s)
        test_1        |         compare_test.go:129: CREATE TABLE table1 (col1_0 BOOL NOT NULL, col1_1 TIME NULL, col1_2 BOOL, col1_3 TIMESTAMP NOT NULL, col1_4 BYTEA NULL, col1_5 INT8 NULL, col1_6 TIMESTAMPTZ NOT NULL, col1_7 DECIMAL NULL, col1_8 VARCHAR NOT NULL, col1_9 DECIMAL, col1_10 DATE[], col1_11 TIMESTAMPTZ, col1_12 TIMESTAMPTZ, PRIMARY KEY (col1_8, col1_6, col1_3));
        test_1        |             CREATE INDEX ON table1 (col1_12, col1_1 DESC, col1_8 ASC) INCLUDE (col1_0, col1_2, col1_4, col1_5, col1_11);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_1 DESC, col1_3, col1_9 ASC, col1_7, col1_4 DESC, col1_8 ASC, col1_12 DESC, col1_6 ASC) INCLUDE (col1_5);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_9, col1_11, col1_5 DESC, col1_8 ASC, col1_0) INCLUDE (col1_1, col1_4, col1_7, col1_10, col1_12);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_0 DESC, col1_1 ASC, col1_3 DESC, col1_12, col1_8, col1_7 DESC, col1_11 ASC, col1_4 DESC, col1_5, col1_6 ASC, col1_2, col1_9 DESC);
        test_1        |             CREATE TABLE table2 (col1_8 VARCHAR NOT NULL, col1_6 TIMESTAMPTZ NOT NULL, col1_3 TIMESTAMP NOT NULL, col2_3 FLOAT8, col2_4 INT8, col2_5 TEXT, col2_6 INTERVAL, col2_7 REGPROCEDURE, col2_8 TEXT, col2_9 REGCLASS NOT NULL, col2_10 OID NOT NULL, col2_11 "char" NOT NULL, col2_12 DECIMAL, col2_13 BOX2D NULL, col2_14 INT8 NOT NULL, col2_15 TIMESTAMP NOT NULL, col2_16 REGPROCEDURE NOT NULL, col2_17 TIMESTAMPTZ, col2_18 REGCLASS NULL, col2_19 INTERVAL NULL, col2_20 TIMESTAMP NOT NULL, PRIMARY KEY (col1_8, col1_6, col1_3));
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_3 DESC, col2_12, col2_20, col2_18 ASC, col2_14 ASC, col2_9 ASC, col1_6 DESC, col2_6, col2_7 ASC, col2_8, col2_16, col2_13, col2_10 DESC, col1_3, col2_4 ASC, col1_8 DESC, col2_17 DESC, col2_15, col2_5 DESC, col2_11, col2_19);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_18);
        test_1        |             CREATE TABLE table3 (col1_8 VARCHAR NOT NULL, col1_6 TIMESTAMPTZ NOT NULL, col1_3 TIMESTAMP NOT NULL, col3_3 FLOAT8 NULL, col3_4 CHAR NOT NULL, col3_5 INT8 NOT NULL, col3_6 OID NOT NULL, col3_7 DECIMAL NULL, col3_8 OID NULL, col3_9 INT8 NOT NULL, col3_10 TIMESTAMPTZ, col3_11 REGPROCEDURE NULL, col3_12 REGTYPE NOT NULL, col3_13 BYTEA, col3_14 TEXT NULL, col3_15 BIT(6) NOT NULL, PRIMARY KEY (col1_8, col1_6, col1_3, col3_8, col3_15, col3_4, col3_6, col3_7, col3_5));
        test_1        |             CREATE INDEX ON table3 (col3_4, col3_15, col3_6, col3_12 ASC, col3_14, col3_7 ASC, col1_6 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_7 ASC, col3_11 ASC, col1_3 DESC, col3_10 ASC, col3_13 ASC, col3_4 ASC, col3_6 DESC, col3_5, col3_12 ASC, col3_15 DESC, col3_8, col3_14 ASC);
        test_1        |             CREATE INDEX ON table3 (col3_10, col3_7, col3_6 DESC, col3_15 DESC, col3_3, col3_12) INCLUDE (col3_9, col3_11);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_7 ASC, col3_9 DESC, col3_8 ASC, col3_15 DESC, col3_5 ASC, col1_3 DESC, col3_11, col1_8, col3_10 DESC, col3_13 ASC, col3_3 ASC, col3_12, col3_14 ASC, col3_6 ASC, col1_6, col3_4 DESC);
        test_1        |             CREATE INDEX ON table3 (col3_12 ASC, col3_4, col3_15 DESC, col3_11 ASC, col3_14 DESC, col3_13, col3_6 ASC, col3_3 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col1_8 DESC, col3_12, col3_9 DESC, col3_15 DESC, col1_6, col3_10 DESC, col3_5, col3_14, col3_6 DESC, col1_3 DESC, col3_4 DESC, col3_3, col3_7 ASC, col3_8, col3_13 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_14 DESC, col3_9 ASC, col3_3 ASC, col1_6, col3_6 DESC, col3_7 DESC, col3_15 ASC, col3_11 ASC, col1_3 DESC, col3_10, col1_8, col3_13 DESC, col3_12 ASC, col3_4 ASC, col3_8 ASC, col3_5 DESC);
        test_1        |             CREATE TABLE table4 (col1_8 VARCHAR NOT NULL, col1_6 TIMESTAMPTZ NOT NULL, col1_3 TIMESTAMP NOT NULL, col4_3 "char", col4_4 BIT(42) NULL, col4_5 "char" NOT NULL, col4_6 DECIMAL, col4_7 OID NOT NULL, col4_8 REGNAMESPACE NOT NULL, col4_9 BOX2D NULL, col4_10 BIT(4) NULL, col4_11 INET NOT NULL, col4_12 FLOAT8 NOT NULL, col4_13 REGPROC NULL, PRIMARY KEY (col1_8, col1_6, col1_3, col4_12, col4_10, col4_11, col4_3));
        test_1        |             CREATE INDEX ON table4 (col1_3, col4_10 ASC, col4_4 DESC) INCLUDE (col4_5, col4_13);
        test_1        |             CREATE INDEX ON table4 (col4_4 ASC, col4_9 DESC, col1_3 ASC) INCLUDE (col4_5, col4_6, col4_7, col4_8);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col1_8 ASC, col4_11, col4_9 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_11 DESC, col1_6 DESC, col4_4, col4_6 DESC, col1_3 DESC, col4_10 ASC, col4_12 ASC, col4_7, col4_5 DESC, col4_8 DESC, col4_13);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col1_6, col4_11, col1_3 ASC, col4_9 DESC, col4_10 ASC, col4_12 ASC, col4_3 DESC) INCLUDE (col4_5, col4_7);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_6, col4_5, col4_9 ASC, col4_4 ASC, col1_3 DESC, col4_7 ASC, col4_12, col4_10, col1_6 DESC, col4_11) INCLUDE (col4_8);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col1_6 DESC, col4_5 ASC, col4_7, col4_13 DESC, col4_11 DESC, col4_9 ASC, col4_12 DESC, col4_8 DESC) INCLUDE (col4_6);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_5, col4_4, col1_3 DESC, col4_9);
        test_1        |             CREATE INDEX ON table4 (col4_9 ASC, col4_7 ASC, col4_8 DESC, col4_10 DESC, col4_13 ASC, col1_8, col1_6, col1_3 DESC, col4_5, col4_12, col4_4 ASC);
        test_1        |             CREATE TABLE table5 (col5_0 INT8[], col5_1 BYTEA, col5_2 TIME NOT NULL, col5_3 INTERVAL NOT NULL, col5_4 NAME NOT NULL, col5_5 CHAR NOT NULL, col5_6 BOX2D NOT NULL, col5_7 REGPROCEDURE NULL, col5_8 INT8 NOT NULL, col5_9 DATE NOT NULL, col5_10 VARCHAR NOT NULL, col5_11 OID NULL, col5_12 REGCLASS NOT NULL, col5_13 TEXT NOT NULL, col5_14 OID NOT NULL, PRIMARY KEY (col5_4, col5_12, col5_9, col5_2, col5_5, col5_8, col5_10, col5_6, col5_3, col5_13, col5_14));
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_9 ASC, col5_10 ASC, col5_2 DESC, col5_7 ASC, col5_14 DESC, col5_6 DESC, col5_13 ASC, col5_11 ASC, col5_12 ASC, col5_5, col5_1, col5_3 DESC, col5_4 DESC);
        test_1        |             CREATE INDEX ON table5 (col5_3 DESC, col5_4 DESC, col5_5 DESC, col5_12 ASC, col5_11, col5_14 ASC, col5_1 ASC, col5_10 DESC, col5_9, col5_13 ASC);
        test_1        |             CREATE INDEX ON table5 (col5_14 ASC, col5_8 ASC, col5_9 ASC, col5_10, col5_1 ASC, col5_3 ASC, col5_5 ASC, col5_7 DESC, col5_6) INCLUDE (col5_0, col5_11);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_8 DESC, col5_9 ASC, col5_12 ASC, col5_2, col5_13, col5_4, col5_1 DESC, col5_5 ASC, col5_7 ASC, col5_10) INCLUDE (col5_11);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_6 ASC, col5_8, col5_13, col5_1 ASC, col5_11 DESC, col5_7 ASC, col5_10, col5_14, col5_3 DESC, col5_5 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_3, col5_14 DESC, col5_5, col5_12 ASC, col5_7 DESC, col5_2);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('lq', 'qpb');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('vjwbj', 'esskn', 'urws', 'arzerk', 'tk');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2483917&tab=buildLog) on [master@22906f72e795f9f2c69828e65194f4177833ffbb](https://github.com/cockroachdb/cockroach/commits/22906f72e795f9f2c69828e65194f4177833ffbb):

```
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('aqi', 'x');
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('ea');
        postgres_1    | 	CREATE TYPE rand_typ_2 AS ENUM ('f', 'slvzd', 'o', 'rdgu', 'bwvn', 'iro');
        postgres_1    | 	CREATE TYPE rand_typ_3 AS ENUM ('blgc');
        postgres_1    | 	CREATE TYPE rand_typ_4 AS ENUM ('xc', 'lnqn', 'vayctr', 'i');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (605.36s)
        test_1        |     --- FAIL: TestCompare/postgres (3.63s)
        test_1        |         compare_test.go:129: CREATE TABLE table1 (col1_0 REGPROCEDURE NOT NULL, col1_1 TIMETZ NOT NULL, col1_2 REGTYPE NOT NULL, col1_3 REGTYPE NOT NULL, col1_4 UUID, col1_5 FLOAT8, col1_6 UUID NOT NULL, col1_7 FLOAT8 NOT NULL, col1_8 REGNAMESPACE NULL, col1_9 REGNAMESPACE NOT NULL, col1_10 DECIMAL NOT NULL, col1_11 FLOAT8 NOT NULL, PRIMARY KEY (col1_7, col1_0, col1_11, col1_9));
        test_1        |             CREATE INDEX ON table1 (col1_7 DESC, col1_3 DESC, col1_8 ASC, col1_2 DESC, col1_9 ASC, col1_6, col1_10, col1_1, col1_0 DESC, col1_11 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_8 ASC);
        test_1        |             CREATE INDEX ON table1 (col1_0, col1_3 ASC, col1_8, col1_2 DESC, col1_7 DESC, col1_9, col1_4, col1_11 ASC) INCLUDE (col1_5, col1_6, col1_10);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_1, col1_3 DESC, col1_8) INCLUDE (col1_2, col1_4);
        test_1        |             CREATE INDEX ON table1 (col1_8 DESC, col1_7 DESC, col1_9 DESC, col1_11 DESC, col1_2, col1_1 DESC);
        test_1        |             CREATE INDEX ON table1 (col1_5) INCLUDE (col1_3, col1_4, col1_6, col1_8);
        test_1        |             CREATE INDEX ON table1 (col1_5, col1_6 DESC, col1_3 ASC, col1_0, col1_2 ASC, col1_10 ASC, col1_11, col1_9 DESC) INCLUDE (col1_4);
        test_1        |             CREATE INDEX ON table1 (col1_8 ASC, col1_10 ASC, col1_3 DESC, col1_1 DESC, col1_11 ASC, col1_9 ASC, col1_4 ASC);
        test_1        |             CREATE INDEX ON table1 (col1_5, col1_1 ASC, col1_10, col1_2 ASC, col1_4, col1_11, col1_7 DESC, col1_6 DESC);
        test_1        |             CREATE TABLE table2 (col1_7 FLOAT8 NOT NULL, col1_0 REGPROCEDURE NOT NULL, col1_11 FLOAT8 NOT NULL, col1_9 REGNAMESPACE NOT NULL, col2_4 TIME NOT NULL, col2_5 BYTEA NOT NULL, col2_6 BIT(26) NULL, col2_7 BIT(9), col2_8 INTERVAL NOT NULL, col2_9 TEXT NULL, col2_10 "char" NULL, col2_11 REGTYPE NOT NULL, col2_12 TEXT NOT NULL, col2_13 REGCLASS NULL, col2_14 BYTEA NULL, PRIMARY KEY (col1_7, col1_0, col1_11, col1_9, col2_4, col2_5));
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_7, col1_11 ASC) INCLUDE (col2_6, col2_9, col2_10, col2_11, col2_12, col2_13, col2_14);
        test_1        |             CREATE TABLE table3 (col3_0 TIMETZ NULL, col3_1 REGPROC NULL, col3_2 TEXT[] NULL, col3_3 TIMETZ, col3_4 TIMESTAMP NOT NULL, col3_5 BOX2D NOT NULL, col3_6 "char" NULL, col3_7 TIMESTAMPTZ, col3_8 "char" NOT NULL, col3_9 REGCLASS NOT NULL, PRIMARY KEY (col3_4));
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_4 DESC, col3_6 DESC, col3_1) INCLUDE (col3_0, col3_2, col3_3, col3_5, col3_7, col3_8);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_9, col3_4, col3_1 DESC, col3_8 DESC, col3_3 ASC, col3_7 DESC, col3_5 DESC) INCLUDE (col3_6);
        test_1        |             CREATE INDEX ON table3 (col3_0) INCLUDE (col3_2, col3_6, col3_7, col3_9);
        test_1        |             CREATE INDEX ON table3 (col3_5) INCLUDE (col3_7, col3_9);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_3 DESC) INCLUDE (col3_2, col3_6, col3_8, col3_9);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_5 ASC, col3_4 ASC, col3_3, col3_1 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 ASC);
        test_1        |             CREATE INDEX ON table3 (col3_8 DESC, col3_4 ASC, col3_5 DESC, col3_7 DESC, col3_0 ASC, col3_6 ASC, col3_1 DESC, col3_3, col3_9);
        test_1        |             CREATE TABLE table4 (col4_0 INET NULL, col4_1 INT8);
        test_1        |             CREATE INDEX ON table4 (col4_0 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_1 DESC, col4_0 DESC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('aqi', 'x');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('ea');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('f', 'slvzd', 'o', 'rdgu', 'bwvn', 'iro');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('blgc');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('xc', 'lnqn', 'vayctr', 'i');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2488194&tab=buildLog) on [master@5af065eeb4f520ee93901e75fbef5d877f06585c](https://github.com/cockroachdb/cockroach/commits/5af065eeb4f520ee93901e75fbef5d877f06585c):

```
        postgres_1    | 2020-12-02 06:13:58.696 UTC [92] ERROR:  column tab_1419.crdb_internal_mvcc_timestamp does not exist at character 54
        postgres_1    | 2020-12-02 06:13:58.696 UTC [92] STATEMENT:  SELECT
        postgres_1    | 		*
        postgres_1    | 	FROM
        postgres_1    | 		(
        postgres_1    | 			SELECT
        postgres_1    | 				6::OID AS col_2462,
        postgres_1    | 				tab_1419.crdb_internal_mvcc_timestamp AS col_2463,
        postgres_1    | 				tab_1419.col2_0 AS col_2464,
        postgres_1    | 				9.0::FLOAT8 AS col_2465
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table4 AS tab_1419
        postgres_1    | 			WHERE
        postgres_1    | 				true
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_1419.tableoid
        postgres_1    | 		)
        postgres_1    | 			AS tab_1420
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_2462 NULLS FIRST, col_2463 NULLS FIRST, col_2464 NULLS FIRST, col_2465 NULLS FIRST
        test_1        | --- FAIL: TestCompare (605.34s)
        test_1        |     --- FAIL: TestCompare/postgres (4.60s)
        test_1        |         compare_test.go:155: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("2000-01-01 06:00:00A"),
        test_1        |             + 	string("2000-01-01 06:00:00+00A"),
        test_1        |               }))
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('2000-01-01 06:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ || 'A'::TEXT::TEXT)::TEXT AS col_2466)
        test_1        |             		AS tab_1421
        test_1        |             ORDER BY
        test_1        |             	col_2466 NULLS FIRST;
        test_1        |             
        test_1        |             cockroach1: [same as previous]
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2491304&tab=buildLog) on [master@677f6f89f97d492f5eb443fcdb326d5695ddc5d7](https://github.com/cockroachdb/cockroach/commits/677f6f89f97d492f5eb443fcdb326d5695ddc5d7):

```
        cockroach1_1  | webui:               http://cockroach1:8080
        cockroach1_1  | sql:                 postgresql://root@cockroach1:26257?sslmode=disable
        cockroach1_1  | RPC client flags:    /cockroach/cockroach <client cmd> --host=cockroach1:26257 --insecure
        cockroach1_1  | logs:                /cockroach-data/logs
        cockroach1_1  | temp dir:            /cockroach-data/cockroach-temp273898687
        cockroach1_1  | external I/O path:   /cockroach-data/extern
        cockroach1_1  | store[0]:            path=/cockroach-data
        cockroach1_1  | storage engine:      pebble
        cockroach1_1  | status:              initialized new cluster
        cockroach1_1  | clusterID:           cff1f60e-6d90-490a-8ac1-1ec17bcee970
        cockroach1_1  | nodeID:              1
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down....2020-12-03 06:14:43.780 UTC [48] LOG:  received fast shutdown request
        postgres_1    | 2020-12-03 06:14:43.782 UTC [48] LOG:  aborting any active transactions
        postgres_1    | 2020-12-03 06:14:43.784 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2020-12-03 06:14:43.790 UTC [50] LOG:  shutting down
        postgres_1    | 2020-12-03 06:14:43.919 UTC [48] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2020-12-03 06:14:43.991 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2020-12-03 06:14:43.991 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2020-12-03 06:14:43.996 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2020-12-03 06:14:44.009 UTC [85] LOG:  database system was shut down at 2020-12-03 06:14:43 UTC
        postgres_1    | 2020-12-03 06:14:44.015 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (602.70s)
        test_1        |     --- FAIL: TestCompare/postgres (1.81s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.3:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2495097&tab=buildLog) on [master@aa69b42032f89d1bb777986ff7a8d4f753b71d83](https://github.com/cockroachdb/cockroach/commits/aa69b42032f89d1bb777986ff7a8d4f753b71d83):

```
        postgres_1    | You can change this by editing pg_hba.conf or using the option -A, or
        postgres_1    | --auth-local and --auth-host, the next time you run initdb.
        postgres_1    | waiting for server to start....2020-12-04 06:15:02.620 UTC [47] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2020-12-04 06:15:02.633 UTC [48] LOG:  database system was shut down at 2020-12-04 06:15:02 UTC
        postgres_1    | 2020-12-04 06:15:02.637 UTC [47] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2020-12-04 06:15:05.489 UTC [47] LOG:  received fast shutdown request
        postgres_1    | .2020-12-04 06:15:05.490 UTC [47] LOG:  aborting any active transactions
        postgres_1    | 2020-12-04 06:15:05.493 UTC [47] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
        postgres_1    | 2020-12-04 06:15:05.500 UTC [49] LOG:  shutting down
        postgres_1    | 2020-12-04 06:15:05.610 UTC [47] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2020-12-04 06:15:05.700 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2020-12-04 06:15:05.700 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2020-12-04 06:15:05.705 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2020-12-04 06:15:05.719 UTC [84] LOG:  database system was shut down at 2020-12-04 06:15:05 UTC
        postgres_1    | 2020-12-04 06:15:05.724 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (603.10s)
        test_1        |     --- FAIL: TestCompare/postgres (1.41s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2498146&tab=buildLog) on [master@ce3f9b29fee565a2994ca84d2ecd20db7fe59d0b](https://github.com/cockroachdb/cockroach/commits/ce3f9b29fee565a2994ca84d2ecd20db7fe59d0b):

```
        cockroach1_1  | webui:               http://cockroach1:8080
        cockroach1_1  | sql:                 postgresql://root@cockroach1:26257?sslmode=disable
        cockroach1_1  | RPC client flags:    /cockroach/cockroach <client cmd> --host=cockroach1:26257 --insecure
        cockroach1_1  | logs:                /cockroach-data/logs
        cockroach1_1  | temp dir:            /cockroach-data/cockroach-temp172442576
        cockroach1_1  | external I/O path:   /cockroach-data/extern
        cockroach1_1  | store[0]:            path=/cockroach-data
        cockroach1_1  | storage engine:      pebble
        cockroach1_1  | status:              initialized new cluster
        cockroach1_1  | clusterID:           e4e050d2-9c00-4ed5-bdef-f5c743921dd0
        cockroach1_1  | nodeID:              1
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down....2020-12-05 06:13:23.224 UTC [47] LOG:  received fast shutdown request
        postgres_1    | 2020-12-05 06:13:23.226 UTC [47] LOG:  aborting any active transactions
        postgres_1    | 2020-12-05 06:13:23.227 UTC [47] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
        postgres_1    | 2020-12-05 06:13:23.234 UTC [49] LOG:  shutting down
        postgres_1    | 2020-12-05 06:13:23.357 UTC [47] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2020-12-05 06:13:23.434 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2020-12-05 06:13:23.434 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2020-12-05 06:13:23.438 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2020-12-05 06:13:23.449 UTC [84] LOG:  database system was shut down at 2020-12-05 06:13:23 UTC
        postgres_1    | 2020-12-05 06:13:23.454 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (602.99s)
        test_1        |     --- FAIL: TestCompare/postgres (1.80s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.3:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2500107&tab=buildLog) on [master@1ed669de1c3b77798fade7ad9f056edc0bf27c36](https://github.com/cockroachdb/cockroach/commits/1ed669de1c3b77798fade7ad9f056edc0bf27c36):

```
        postgres_1    | You can change this by editing pg_hba.conf or using the option -A, or
        postgres_1    | --auth-local and --auth-host, the next time you run initdb.
        postgres_1    | ok
        postgres_1    | 
        postgres_1    | Success. You can now start the database server using:
        postgres_1    | 
        postgres_1    |     pg_ctl -D /var/lib/postgresql/data -l logfile start
        postgres_1    | 
        postgres_1    | waiting for server to start....2020-12-07 06:14:25.440 UTC [47] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2020-12-07 06:14:25.453 UTC [48] LOG:  database system was shut down at 2020-12-07 06:14:25 UTC
        postgres_1    | 2020-12-07 06:14:25.458 UTC [47] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        test_1        | --- FAIL: TestCompare (1.34s)
        test_1        |     --- FAIL: TestCompare/postgres (0.84s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.3:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.50s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 REGNAMESPACE NOT NULL, col1_1 REGNAMESPACE NOT NULL, col1_2 FLOAT8 NOT NULL, col1_3 BIT(5) NOT NULL, col1_4 BOX2D NOT NULL, col1_5 DECIMAL NOT NULL, col1_6 FLOAT4 NOT NULL, col1_7 INTERVAL NOT NULL, col1_8 INT2 NOT NULL, col1_9 REGTYPE NOT NULL, col1_10 TIMESTAMPTZ NOT NULL, col1_11 BIT(39) NOT NULL, col1_12 TIMESTAMP NOT NULL, col1_13 INT8 NOT NULL, col1_14 REGCLASS NOT NULL, col1_15 INET NOT NULL, col1_16 BYTES NOT NULL, col1_17 UUID NOT NULL, col1_18 BOOL NOT NULL, PRIMARY KEY (col1_5 ASC, col1_18 DESC, col1_14 ASC, col1_8 DESC, col1_1, col1_16 ASC, col1_0, col1_7, col1_9 DESC, col1_13, col1_6 ASC, col1_11 DESC, col1_12 DESC, col1_10 DESC, col1_4 ASC, col1_15 DESC, col1_3 DESC, col1_2, col1_17 ASC), UNIQUE (col1_3 ASC, col1_7 ASC, col1_12 ASC, col1_9 DESC, col1_1, col1_5 DESC, col1_16 DESC, col1_2 DESC, col1_4, col1_10, col1_17 DESC, col1_14), UNIQUE (col1_17 ASC, col1_12 ASC, col1_3 ASC, col1_0 ASC, col1_4 DESC, col1_6 DESC, col1_8, col1_1 ASC, col1_11 ASC, col1_18 DESC) WHERE ((((((((table1.col1_13 <= -1) OR (table1.col1_16 > '\x00')) OR (table1.col1_6 < 1.401298464324817e-45)) OR (table1.col1_12 != '3000-01-01 00:00:00')) OR (table1.col1_10 > '3000-01-01 00:00:00+00:00')) OR (table1.col1_8 > 127)) AND table1.col1_18) OR (table1.col1_2 >= 1.7976931348623157e+308)) OR (table1.col1_5 <= NaN), UNIQUE (col1_11 DESC, col1_4, col1_1 DESC, col1_15 ASC, col1_13, col1_7 DESC, col1_17 ASC, col1_14 DESC, col1_0, col1_5, col1_9, col1_16 ASC, col1_2 ASC) WHERE ((((((((table1.col1_2 = 'NaN':::FLOAT8) AND (table1.col1_12 > '-2000-01-01 00:00:00':::TIMESTAMP)) OR (table1.col1_5 <= '-Infinity':::DECIMAL)) AND (table1.col1_8 < (-1):::INT8)) OR (NOT table1.col1_18)) OR (table1.col1_16 > e'\\xe29883':::BYTES)) AND (table1.col1_6 < '-Inf':::FLOAT8)) OR (table1.col1_10 <= '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table1.col1_13 < (-2147483648):::INT8), UNIQUE (col1_6 DESC, col1_16 ASC, col1_10, col1_9 DESC, col1_17 DESC, col1_13 DESC, col1_2 ASC, col1_15 ASC, col1_0 DESC, col1_7 DESC, col1_1, col1_3 ASC, col1_5 ASC, col1_18 DESC, col1_12, col1_14 DESC) WHERE (((((table1.col1_10 > '0001-01-01 00:00:00+00:00') OR (table1.col1_6 >= 3.4028234663852886e+38)) AND (table1.col1_2 >= -1.0)) OR (table1.col1_12 <= '-2000-01-01 00:00:00')) AND (table1.col1_8 > -1)) OR table1.col1_18, INDEX (col1_5 ASC, col1_16, col1_13 ASC, col1_18 ASC, col1_10 DESC, col1_3, col1_1, col1_6 ASC, col1_8 DESC, col1_17 ASC, col1_0, col1_4 ASC) WHERE ((((((((table1.col1_13 > 1:::INT8) OR (table1.col1_5 >= (-1.234E+401):::DECIMAL)) OR (table1.col1_16 <= e'\\x27':::BYTES)) AND (NOT table1.col1_18)) OR (table1.col1_8 != 32767:::INT8)) AND (table1.col1_6 != '+Inf':::FLOAT8)) AND (table1.col1_10 != '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table1.col1_12 = '294276-12-31 23:59:59.999999':::TIMESTAMP)) AND (table1.col1_2 != '-Inf':::FLOAT8), FAMILY (col1_8, col1_17, col1_9, col1_6), FAMILY (col1_5, col1_1), FAMILY (col1_4), FAMILY (col1_11, col1_12), FAMILY (col1_13), FAMILY (col1_18, col1_7, col1_0, col1_16), FAMILY (col1_15, col1_10, col1_3, col1_2, col1_14));
        test_1        |             CREATE TABLE table2 (col2_0 FLOAT8 NOT NULL, col2_1 REGPROCEDURE NULL, col2_2 DATE, col2_3 REGNAMESPACE NULL, col2_4 BOX2D NOT NULL, col2_5 VARCHAR NULL, col2_6 INT4, col2_7 INT4, col2_8 REGCLASS NOT NULL, col2_9 INT2, col2_10 BOX2D NULL, col2_11 BOX2D NULL, col2_12 REGCLASS, col2_13 INT2 NULL, col2_14 REGPROCEDURE, col2_15 INT2, col2_16 STRING, INDEX (col2_9, col2_2 DESC, col2_8 DESC, col2_16 DESC, col2_13, col2_0 ASC, col2_4 ASC, col2_11 DESC, col2_6, col2_7 ASC, col2_1, col2_15 ASC, col2_3 DESC, col2_12, col2_14 DESC, col2_5, col2_10) WHERE table2.col2_16 > e'\U00002603', FAMILY (col2_3, col2_13), FAMILY (col2_14), FAMILY (col2_8), FAMILY (col2_11), FAMILY (col2_5), FAMILY (col2_0), FAMILY (col2_15), FAMILY (col2_1), FAMILY (col2_7, col2_12, col2_2), FAMILY (col2_10), FAMILY (col2_6, col2_9), FAMILY (col2_4), FAMILY (col2_16));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 550, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 37, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 426, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 125, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 90, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 673, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 362, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "40718a77-ce41-498c-87b3-90caf06371aa"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 537, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 452, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000000, "num_range": 0, "upper_bound": "-1.0633761882781982"}, {"distinct_range": 0, "num_eq": 7070295410477950058, "num_range": 1000000, "upper_bound": "-0.35531970858573914"}, {"distinct_range": 0, "num_eq": 933241670259528630, "num_range": 117882968585153253, "upper_bound": "0.6834092140197754"}, {"distinct_range": 0, "num_eq": 2275641429907948161, "num_range": 9000, "upper_bound": "1.022737741470337"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 114, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 50, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "011011001001110100001011011000011100110"}], "histo_col_type": "BIT(39)", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 513, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 452, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 237, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 283, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 133, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000000, "num_range": 0, "upper_bound": "00000"}, {"distinct_range": 227299661206333020, "num_eq": 8603169105451399349, "num_range": 1055290414030659304, "upper_bound": "00011"}, {"distinct_range": 232043565300534370, "num_eq": 7265132281110170234, "num_range": 504259297498446483, "upper_bound": "10011"}], "histo_col_type": "BIT(5)", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 562, "histo_buckets": [{"distinct_range": 0, "num_eq": 600, "num_range": 0, "upper_bound": "-Infinity"}, {"distinct_range": 759605.490078298, "num_eq": 70000000000, "num_range": 2000000, "upper_bound": "551137257851359149.4"}, {"distinct_range": 0, "num_eq": 600000, "num_range": 7206151939353439718, "upper_bound": "9.159173867188882041E+35"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 626, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1726, "histo_col_type": "", "name": "__auto__", "null_count": 2373, "row_count": 4000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1508, "histo_col_type": "", "name": "__auto__", "null_count": 832, "row_count": 4000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 497, "histo_col_type": "", "name": "__auto__", "null_count": 3473, "row_count": 4000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1655, "histo_col_type": "", "name": "__auto__", "null_count": 3768, "row_count": 4000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 126, "histo_col_type": "", "name": "__auto__", "null_count": 3902, "row_count": 4000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1884, "histo_col_type": "", "name": "__auto__", "null_count": 1044, "row_count": 4000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2176, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2013, "histo_col_type": "", "name": "__auto__", "null_count": 168, "row_count": 4000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2573, "histo_buckets": [{"distinct_range": 0, "num_eq": 4833013496368488536, "num_range": 0, "upper_bound": "-22869"}, {"distinct_range": 0, "num_eq": 6806698657198820915, "num_range": 2891051510084960248, "upper_bound": "-21710"}, {"distinct_range": 167.59746686740942, "num_eq": 50000, "num_range": 1000, "upper_bound": "-14286"}, {"distinct_range": 9000000, "num_eq": 6041995942024279264, "num_range": 9000000, "upper_bound": "-1"}, {"distinct_range": 5046770553995163000, "num_eq": 7497277914087576815, "num_range": 5046770553995162970, "upper_bound": "15932"}, {"distinct_range": 0, "num_eq": 1177347732255393013, "num_range": 5422755729144871955, "upper_bound": "21879"}, {"distinct_range": 5000000000, "num_eq": 60000000000, "num_range": 5000000000, "upper_bound": "32767"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 2156, "row_count": 4000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2410, "histo_col_type": "", "name": "__auto__", "null_count": 81, "row_count": 4000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 634, "histo_col_type": "", "name": "__auto__", "null_count": 3763, "row_count": 4000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3591, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1084, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2215, "histo_col_type": "", "name": "__auto__", "null_count": 343, "row_count": 4000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 666, "histo_col_type": "", "name": "__auto__", "null_count": 2941, "row_count": 4000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1473, "histo_col_type": "", "name": "__auto__", "null_count": 2774, "row_count": 4000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3475, "histo_col_type": "", "name": "__auto__", "null_count": 715, "row_count": 4000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('lzytx', 'rrssjp', 'dgspgk', 'bz', 'duppvj');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('ynkobu', 'fz', 'qxasuc', 'umj');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('dkm', 'ntnj', 'hv', 'rmv');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4930346533417632538, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7208779371351735135, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 797867529326688048, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 248523844625920451, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2756936261989000684, "histo_col_type": "BIT(5)", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8893492606020942088, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": "-7.301312452032320856E+25"}, {"distinct_range": 242010462410386300, "num_eq": 100, "num_range": 464915720832725377, "upper_bound": "6.4759771523456687E+17"}, {"distinct_range": 755477287186628700, "num_eq": 70, "num_range": 1994178025161888239, "upper_bound": "7.084308546436682111E+19"}, {"distinct_range": 1009644.8185036313, "num_eq": 6580188778183236808, "num_range": 9000000, "upper_bound": "8.82421444875805445E+26"}, {"distinct_range": 2982246188752960000, "num_eq": 8000, "num_range": 6609538837751393883, "upper_bound": "Infinity"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8713434913543561703, "histo_buckets": [{"distinct_range": 0, "num_eq": 3987014012079856818, "num_range": 0, "upper_bound": "-0.07302670925855637"}, {"distinct_range": 200000000, "num_eq": 2297481714169194276, "num_range": 200000000, "upper_bound": "-0.05358833819627762"}, {"distinct_range": 0, "num_eq": 2510881001842864663, "num_range": 3000, "upper_bound": "0.0"}, {"distinct_range": 7000000, "num_eq": 8237198305741176445, "num_range": 7000000, "upper_bound": "0.13262300193309784"}, {"distinct_range": 0, "num_eq": 0, "num_range": 300, "upper_bound": "0.33270972967147827"}, {"distinct_range": 0, "num_eq": 4753808207750904022, "num_range": 5393244118930165477, "upper_bound": "1.4219046831130981"}, {"distinct_range": 1370351430787064300, "num_eq": 2624405866361445044, "num_range": 1469118196151029047, "upper_bound": "1.4298930168151855"}, {"distinct_range": 8000000, "num_eq": 4247977125313319008, "num_range": 8000000, "upper_bound": "+Inf"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7294897870837579521, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2400846514342408105, "histo_buckets": [{"distinct_range": 0, "num_eq": 5849196649837449686, "num_range": 0, "upper_bound": "000011001001000000110110011100100000110"}, {"distinct_range": 1491206834045356500, "num_eq": 4000000, "num_range": 4827194198323208819, "upper_bound": "000100010001101110100100000100110010100"}, {"distinct_range": 0, "num_eq": 800, "num_range": 3000000000, "upper_bound": "001101011111111111000111011000000111000"}, {"distinct_range": 8250058328353134000, "num_eq": 800, "num_range": 8250058328353133636, "upper_bound": "100110110011111001111011011111100010000"}], "histo_col_type": "BIT(39)", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1903835033638576724, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5050139770188731951, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1584760275670111779, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2488836027591111403, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4661956857878476124, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2670961024030741972, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5223927400232158390, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000, "num_range": 0, "upper_bound": "00000000-0000-0000-0000-000000000000"}, {"distinct_range": 329.1227075535551, "num_eq": 1000000000, "num_range": 500, "upper_bound": "35af3c62-6908-49da-ae2f-a21220cb9e18"}, {"distinct_range": 10770079863.98416, "num_eq": 669829772284712890, "num_range": 30000000000, "upper_bound": "5752aed1-9860-44a5-9409-3cdb78e4325d"}, {"distinct_range": 65966.19835289022, "num_eq": 6000, "num_range": 200000, "upper_bound": "bd3d4b2e-c8e0-4e4e-be2b-0892dd479c00"}, {"distinct_range": 1584861903266554600, "num_eq": 7481749533628715014, "num_range": 2453314907077534851, "upper_bound": "e82bcd0f-5ec6-41b9-b3d0-74f6cbbca7c4"}, {"distinct_range": 608229.34024546, "num_eq": 2000000000, "num_range": 4000000, "upper_bound": "e88ebc9d-32d6-4f34-a0ad-c060d4eb7c87"}, {"distinct_range": 4000000, "num_eq": 1000000000, "num_range": 4000000, "upper_bound": "f44b34d5-28e7-47bd-acf0-798794c978d6"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8194431039879503911, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3020569476432159505, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7603223765439868296, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8920373053588799025}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 142, "histo_col_type": "", "name": "__auto__", "null_count": 163, "row_count": 200}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7, "histo_col_type": "", "name": "__auto__", "null_count": 187, "row_count": 200}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 184, "histo_col_type": "", "name": "__auto__", "null_count": 31, "row_count": 200}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 104, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23, "histo_col_type": "", "name": "__auto__", "null_count": 54, "row_count": 200}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 183, "histo_col_type": "INT2", "name": "__auto__", "null_count": 141, "row_count": 200}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 193, "histo_col_type": "", "name": "__auto__", "null_count": 191, "row_count": 200}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 141, "histo_col_type": "", "name": "__auto__", "null_count": 104, "row_count": 200}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 179, "histo_col_type": "", "name": "__auto__", "null_count": 175, "row_count": 200}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 192, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11, "histo_col_type": "", "name": "__auto__", "null_count": 45, "row_count": 200}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 178, "histo_col_type": "", "name": "__auto__", "null_count": 21, "row_count": 200}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 85, "histo_col_type": "", "name": "__auto__", "null_count": 198, "row_count": 200}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 103, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 175, "row_count": 200}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 75, "histo_col_type": "", "name": "__auto__", "null_count": 11, "row_count": 200}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 75, "histo_col_type": "", "name": "__auto__", "null_count": 3, "row_count": 200}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 56, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23, "histo_buckets": [{"distinct_range": 0, "num_eq": 2123742015048237910, "num_range": 0, "upper_bound": "110000000111010010001011011001110010000"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 50000000000, "upper_bound": "110101001101010000010100011100000001000"}], "histo_col_type": "BIT(39)", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 71, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1a1c0622-5a21-4b2f-ba03-1a59ce4ac8aa"}, {"distinct_range": 0, "num_eq": 1950624964696554201, "num_range": 895271215657578089, "upper_bound": "d0ff1189-c941-4bee-8bae-9e723f784e8d"}, {"distinct_range": 1853005372357974800, "num_eq": 800000, "num_range": 1853005372357974718, "upper_bound": "ffffffff-ffff-ffff-ffff-ffffffffffff"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 40, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "00110"}, {"distinct_range": 0, "num_eq": 291469459063981725, "num_range": 2290508793656156966, "upper_bound": "01000"}, {"distinct_range": 200000, "num_eq": 700000, "num_range": 200000, "upper_bound": "10001"}, {"distinct_range": 1177635.3568751856, "num_eq": 766803585331416666, "num_range": 2000000, "upper_bound": "11100"}, {"distinct_range": 0, "num_eq": 8342498817629431958, "num_range": 60, "upper_bound": "11111"}], "histo_col_type": "BIT(5)", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18, "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 73, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000, "num_range": 0, "upper_bound": "-3.551574445346471128E+23"}, {"distinct_range": 8039969943256010000, "num_eq": 800000, "num_range": 8039969943256009528, "upper_bound": "-9.21222527394248348E+18"}, {"distinct_range": 4446752375555848000, "num_eq": 20000000, "num_range": 8754062216304343116, "upper_bound": "84517.86625204583327"}, {"distinct_range": 3883864239330435600, "num_eq": 900000, "num_range": 5982950007126018982, "upper_bound": "9021869.683270853452"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 69, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25, "histo_col_type": "", "name": "__auto__", "null_count": 42, "row_count": 70}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 57, "histo_col_type": "", "name": "__auto__", "null_count": 2, "row_count": 70}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 31, "histo_buckets": [{"distinct_range": 0, "num_eq": 4719972871250049625, "num_range": 0, "upper_bound": "-30731"}, {"distinct_range": 3674075987399301600, "num_eq": 500000, "num_range": 5581870455046091408, "upper_bound": "-21504"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 4000, "upper_bound": "-21195"}, {"distinct_range": 0, "num_eq": 9000, "num_range": 200000000, "upper_bound": "-13140"}, {"distinct_range": 0, "num_eq": 4000000000, "num_range": 585621257755573561, "upper_bound": "8840"}, {"distinct_range": 1000000000, "num_eq": 90000000000, "num_range": 1000000000, "upper_bound": "26328"}, {"distinct_range": 356994292.74753743, "num_eq": 300000000, "num_range": 800000000, "upper_bound": "30197"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 3, "row_count": 70}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 64, "histo_col_type": "", "name": "__auto__", "null_count": 18, "row_count": 70}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 69, "histo_col_type": "", "name": "__auto__", "null_count": 24, "row_count": 70}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 59, "histo_col_type": "", "name": "__auto__", "null_count": 14, "row_count": 70}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10, "histo_col_type": "", "name": "__auto__", "null_count": 63, "row_count": 70}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35, "histo_col_type": "", "name": "__auto__", "null_count": 63, "row_count": 70}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67, "histo_col_type": "", "name": "__auto__", "null_count": 41, "row_count": 70}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_col_type": "", "name": "__auto__", "null_count": 63, "row_count": 70}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34, "histo_col_type": "", "name": "__auto__", "null_count": 44, "row_count": 70}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23, "histo_col_type": "", "name": "__auto__", "null_count": 60, "row_count": 70}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19, "histo_col_type": "", "name": "__auto__", "null_count": 41, "row_count": 70}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16, "histo_col_type": "", "name": "__auto__", "null_count": 35, "row_count": 70}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "nan" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2506256&tab=buildLog) on [master@258ef5765d0205f487b1481a9c26059db2a70362](https://github.com/cockroachdb/cockroach/commits/258ef5765d0205f487b1481a9c26059db2a70362):

```
        postgres_1    | 
        postgres_1    | 2020-12-09 06:13:46.887 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2020-12-09 06:13:46.887 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2020-12-09 06:13:46.893 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2020-12-09 06:13:46.906 UTC [88] LOG:  database system was shut down at 2020-12-09 06:13:46 UTC
        postgres_1    | 2020-12-09 06:13:46.912 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (2.84s)
        test_1        |     --- FAIL: TestCompare/postgres (1.34s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (1.50s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 TIMESTAMPTZ NOT NULL, col1_1 REGCLASS, col1_2 TIME NOT NULL, col1_3 FLOAT4, col1_4 TIMESTAMPTZ NULL, PRIMARY KEY (col1_2 DESC), UNIQUE (col1_3, col1_1 DESC, col1_0, col1_4 ASC) WHERE (((table1.col1_4 > '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ) OR (table1.col1_3 = '+Inf':::FLOAT8)) OR (table1.col1_2 < '00:00:00':::TIME)) OR (table1.col1_0 > '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ), UNIQUE (col1_2, col1_1) STORING (col1_3, col1_4) WHERE ((table1.col1_0 <= '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ) OR (table1.col1_3 >= '+Inf':::FLOAT8)) AND (table1.col1_4 > '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ), UNIQUE (col1_4 DESC, col1_1 DESC, col1_2) WHERE (((table1.col1_0 < '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ) AND (table1.col1_2 <= '24:00:00':::TIME)) OR (table1.col1_3 != '-Inf':::FLOAT8)) OR (table1.col1_4 <= '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ), INDEX (col1_2, col1_3 ASC), UNIQUE (col1_3 ASC, col1_4 ASC, col1_1 ASC, col1_0 ASC, col1_2 ASC) WHERE ((table1.col1_4 < '0001-01-01 00:00:00+00:00') OR (table1.col1_2 <= '24:00:00')) OR (table1.col1_0 >= '-4713-11-24 00:00:00+00:00'), INDEX (col1_2 DESC, col1_1 ASC, col1_4 ASC) STORING (col1_0, col1_3) WHERE (((table1.col1_0 < '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ) AND (table1.col1_4 = '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table1.col1_3 > 'NaN':::FLOAT8)) AND (table1.col1_2 = '24:00:00':::TIME), INDEX (col1_4) STORING (col1_1, col1_3), UNIQUE (col1_0, col1_3 ASC) STORING (col1_4), FAMILY (col1_3), FAMILY (col1_2, col1_1, col1_0), FAMILY (col1_4));
        test_1        |             CREATE TABLE table2 (col1_2 TIME NOT NULL, col2_1 DATE, col2_2 UUID NOT NULL, col2_3 UUID NULL, col2_4 DATE, col2_5 TIME NOT NULL, col2_6 REGPROC, col2_7 TIME, col2_8 FLOAT8 NOT NULL, col2_9 INT8 NULL, col2_10 NAME NOT NULL, col2_11 INT2, col2_12 TIMESTAMP NULL, col2_13 CHAR NULL, col2_14 VARCHAR NULL, PRIMARY KEY (col1_2 DESC, col2_14 DESC, col2_4 DESC, col2_12 DESC, col2_5 ASC, col2_1 DESC, col2_8 DESC, col2_11 DESC, col2_10 DESC), UNIQUE (col1_2 DESC, col2_10 DESC, col2_7 ASC, col2_6 DESC, col2_2 DESC, col2_12 DESC, col2_5 ASC, col2_11, col2_14 ASC) STORING (col2_3, col2_13), UNIQUE (col2_7 ASC, col2_12 DESC, col2_10, col2_4 DESC, col2_13, col2_8 DESC, col2_1 DESC, col2_11, col2_2 DESC, col2_6 ASC, col1_2 ASC) STORING (col2_3, col2_9), UNIQUE (col2_3 ASC, col2_7, col2_5, col1_2 ASC) STORING (col2_9), UNIQUE (col2_9) STORING (col2_2, col2_6, col2_7), UNIQUE (col1_2) WHERE ((((table2.col2_9 < -32768) OR (table2.col2_12 >= '0001-01-01 00:00:00')) OR (table2.col2_5 < '24:00:00')) AND (table2.col2_8 > 0.0)) OR (table2.col2_1 <= 'infinity'), FAMILY (col2_12, col2_3), FAMILY (col1_2, col2_9), FAMILY (col2_2), FAMILY (col2_5), FAMILY (col2_7), FAMILY (col2_11), FAMILY (col2_8, col2_6, col2_10, col2_13), FAMILY (col2_1), FAMILY (col2_14), FAMILY (col2_4)) INTERLEAVE IN PARENT table1 (col1_2);
        test_1        |             CREATE TABLE table3 (col1_2 TIME NOT NULL, col3_1 TIMESTAMPTZ NOT NULL, col3_2 INT4 NULL, col3_3 CHAR, col3_4 FLOAT4 NOT NULL, col3_5 REGPROCEDURE NULL, col3_6 STRING NULL, col3_7 INET NOT NULL, col3_8 BIT(17), col3_9 TIME, col3_10 REGPROCEDURE, col3_11 INT2 NOT NULL, col3_12 "char" NULL, col3_13 FLOAT8 NULL, col3_14 DATE NULL, col3_15 VARCHAR, PRIMARY KEY (col1_2 DESC), UNIQUE (col3_4 DESC, col3_6 DESC, col3_8, col3_1 ASC, col3_11 DESC, col3_7 ASC, col3_10 ASC, col1_2 DESC, col3_14 DESC) WHERE (((((((((((table3.col3_13 = 1.0:::FLOAT8) AND (table3.col3_6 <= '"':::STRING)) OR (table3.col3_2 <= 2147483647:::INT8)) OR (table3.col1_2 = '24:00:00':::TIME)) AND (table3.col3_3 != '':::STRING)) AND (table3.col3_14 > '5874897-12-31':::DATE)) OR (table3.col3_1 <= '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) AND (table3.col3_11 <= (-1):::INT8)) OR (table3.col3_9 > '24:00:00':::TIME)) OR (table3.col3_15 > e'\x00':::STRING)) AND (table3.col3_12 >= e'\x00':::STRING)) AND (table3.col3_4 > 1.0:::FLOAT8), INDEX (col3_2 DESC, col3_8 ASC, col3_12 DESC, col3_10 ASC, col3_7 ASC, col3_11 DESC, col3_13, col3_15, col3_3 DESC) STORING (col3_5, col3_14) WHERE ((table3.col3_3 = e'\'':::STRING) OR (table3.col3_4 < 0.0:::FLOAT8)) OR (table3.col3_6 = '':::STRING), INDEX (col3_5, col3_4, col1_2 ASC, col3_14 DESC, col3_12, col3_2, col3_6, col3_13, col3_9 DESC, col3_15, col3_7) STORING (col3_11), UNIQUE (col3_11 ASC, col3_13 ASC, col3_14 DESC, col3_2 DESC, col1_2 DESC, col3_5, col3_9, col3_4, col3_6 ASC, col3_1 DESC, col3_3, col3_8) WHERE (((table3.col3_15 <= '':::STRING) OR (table3.col3_2 < (-1):::INT8)) OR (table3.col3_6 > 'X':::STRING)) AND (table3.col3_4 <= 1.401298464324817e-45:::FLOAT8), INDEX (col3_10 ASC, col1_2 DESC, col3_14, col3_13 DESC, col3_5 DESC, col3_6, col3_4 DESC, col3_11 ASC) STORING (col3_1, col3_2, col3_7, col3_12), UNIQUE (col3_5 ASC, col3_14 ASC, col3_7 DESC, col3_12, col3_10 DESC, col3_6, col3_15 ASC, col3_13 DESC, col3_3 DESC, col3_4 ASC, col1_2) STORING (col3_1, col3_9), INDEX (col3_10 DESC, col3_15 ASC, col3_5 ASC, col3_1 ASC, col3_13 DESC, col3_3 ASC, col3_6 ASC, col3_4 DESC, col3_12 ASC) STORING (col3_2, col3_7, col3_8, col3_14), UNIQUE (col3_13 DESC, col3_4, col3_5 DESC, col3_8, col3_14 ASC, col3_11 ASC, col3_1 DESC, col3_2, col3_12) STORING (col3_6, col3_9, col3_15) WHERE (((((((((((table3.col3_15 < e'\x00') OR (table3.col3_3 > '"')) OR (table3.col3_14 != '1970-01-01')) AND (table3.col3_6 > '')) AND (table3.col3_12 >= '"')) AND (table3.col3_1 <= '3000-01-01 00:00:00+00:00')) AND (table3.col3_2 <= 1)) AND (table3.col3_13 = NaN)) OR (table3.col3_9 != '24:00:00')) OR (table3.col3_11 = 1)) AND (table3.col3_4 != 0.0)) AND (table3.col1_2 < '24:00:00'), UNIQUE (col3_1) WHERE ((table3.col3_4 < NaN) AND (table3.col3_2 = 0)) AND (table3.col3_14 = '1970-01-01'), UNIQUE (col3_4, col3_5), FAMILY (col3_6), FAMILY (col3_4, col3_11, col3_1), FAMILY (col3_15), FAMILY (col1_2), FAMILY (col3_3), FAMILY (col3_2), FAMILY (col3_7, col3_13), FAMILY (col3_12, col3_9), FAMILY (col3_10), FAMILY (col3_14), FAMILY (col3_8), FAMILY (col3_5)) INTERLEAVE IN PARENT table1 (col1_2);
        test_1        |             CREATE TABLE table4 (col4_0 INT2 NOT NULL, col4_1 BOOL NOT NULL, col4_2 BOOL NOT NULL, col4_3 UUID NOT NULL, col4_4 DATE NOT NULL, col4_5 BIT(34), col4_6 CHAR NOT NULL, col4_7 FLOAT4 NOT NULL, col4_8 "char" NOT NULL, col4_9 STRING NOT NULL, col4_10 BOX2D NULL, col4_11 STRING NOT NULL, col4_12 BIT(27) NOT NULL, col4_13 INET NULL, col4_14 FLOAT8 NOT NULL, col4_15 REGNAMESPACE NOT NULL, PRIMARY KEY (col4_0 DESC, col4_3, col4_7 ASC, col4_15, col4_12, col4_11 DESC, col4_9 DESC, col4_6 DESC, col4_2 ASC, col4_8, col4_1 ASC, col4_4 DESC, col4_14 DESC), UNIQUE (col4_11 ASC, col4_6 ASC, col4_15, col4_4 DESC, col4_0, col4_1 ASC, col4_5, col4_8 DESC, col4_10 ASC, col4_14 DESC, col4_9 DESC, col4_7 ASC, col4_3 DESC, col4_12, col4_13 DESC, col4_2) WHERE ((((table4.col4_9 <= '"':::STRING) AND (table4.col4_8 != '"':::STRING)) AND (table4.col4_0 = 0:::INT8)) OR (table4.col4_7 >= 0.0:::FLOAT8)) AND (table4.col4_4 <= '-infinity':::DATE), UNIQUE (col4_13 ASC, col4_2 DESC, col4_4 ASC, col4_8, col4_0 ASC, col4_11 DESC, col4_6 ASC) STORING (col4_5) WHERE (((((((((table4.col4_0 >= 1:::INT8) AND (table4.col4_11 >= 'X':::STRING)) AND (table4.col4_6 != '"':::STRING)) AND (NOT table4.col4_1)) AND (table4.col4_8 >= '"':::STRING)) AND (table4.col4_7 <= 0.0:::FLOAT8)) OR (NOT table4.col4_2)) AND (table4.col4_14 < 5e-324:::FLOAT8)) AND (table4.col4_9 > '':::STRING)) OR (table4.col4_4 >= '5874897-12-31':::DATE), UNIQUE (col4_1 ASC, col4_8 ASC) STORING (col4_5, col4_10, col4_13), INDEX (col4_0 DESC, col4_5, col4_3 ASC, col4_1 DESC, col4_11 DESC, col4_6 ASC, col4_12 ASC) STORING (col4_10, col4_13) WHERE (((((((((table4.col4_14 != 5e-324:::FLOAT8) OR (NOT table4.col4_2)) OR (table4.col4_8 <= '"':::STRING)) OR (NOT table4.col4_1)) OR (table4.col4_7 < 0.0:::FLOAT8)) OR (table4.col4_4 = '5874897-12-31':::DATE)) AND (table4.col4_11 = e'\'':::STRING)) OR (table4.col4_0 != 0:::INT8)) OR (table4.col4_9 = e'\'':::STRING)) AND (table4.col4_6 > e'\'':::STRING), UNIQUE (col4_12 DESC, col4_8 ASC, col4_15, col4_13 ASC, col4_4 DESC, col4_10, col4_5, col4_3 DESC, col4_11 ASC, col4_6 ASC, col4_14 ASC), UNIQUE (col4_13, col4_9 DESC, col4_12 ASC, col4_6 ASC, col4_14 DESC, col4_5, col4_2 DESC, col4_11 DESC, col4_10, col4_1 DESC, col4_4, col4_0 DESC, col4_7 ASC, col4_8 DESC), UNIQUE (col4_5 ASC, col4_3, col4_11 DESC, col4_8, col4_14, col4_12 ASC, col4_0 DESC, col4_6, col4_13 DESC, col4_4) STORING (col4_10) WHERE ((((((table4.col4_9 < e'\U00002603':::STRING) OR table4.col4_2) OR (table4.col4_8 = '':::STRING)) AND (table4.col4_7 != '+Inf':::FLOAT8)) AND (table4.col4_14 = 'NaN':::FLOAT8)) AND (table4.col4_4 >= '4714-11-24 BC':::DATE)) OR table4.col4_1, FAMILY (col4_14, col4_11, col4_4), FAMILY (col4_0), FAMILY (col4_10), FAMILY (col4_1), FAMILY (col4_9, col4_8), FAMILY (col4_5), FAMILY (col4_12, col4_13), FAMILY (col4_6), FAMILY (col4_15, col4_7, col4_2), FAMILY (col4_3));
        test_1        |             CREATE TABLE table5 (col1_2 TIME NOT NULL, col2_14 VARCHAR NULL, col2_4 DATE, col2_12 TIMESTAMP NULL, col2_5 TIME NOT NULL, col2_1 DATE, col2_8 FLOAT8 NOT NULL, col2_11 INT2, col2_10 NAME NOT NULL, col5_9 REGTYPE NOT NULL, col5_10 INT4 NULL, PRIMARY KEY (col1_2 DESC, col2_14 DESC, col2_4 DESC, col2_12 DESC, col2_5 ASC, col2_1 DESC, col2_8 DESC, col2_11 DESC, col2_10 DESC, col5_10 DESC), UNIQUE (col2_14 ASC, col5_10 DESC) WHERE table5.col2_8 > 3.4028234663852886e+38:::FLOAT8, INDEX (col2_10 DESC, col2_12 DESC, col5_9 ASC, col2_4 ASC, col2_8, col1_2 ASC, col2_11 ASC, col5_10 ASC, col2_5 ASC, col2_1 DESC, col2_14) WHERE ((((table5.col2_12 >= '0001-01-01 00:00:00') AND (table5.col2_5 != '24:00:00')) OR (table5.col2_1 > '4714-11-24 BC')) AND (table5.col2_4 > '-infinity')) AND (table5.col2_10 = ''), INDEX (col2_11, col2_4, col5_10 DESC, col2_10) STORING (col5_9) WHERE ((table5.col2_5 <= '00:00:00':::TIME) OR (table5.col2_4 = '4714-11-24 BC':::DATE)) AND (table5.col2_11 < 32767:::INT8), FAMILY (col2_10, col2_5, col2_11, col2_4, col5_10), FAMILY (col2_12, col2_14), FAMILY (col2_1), FAMILY (col1_2, col2_8, col5_9)) INTERLEAVE IN PARENT table2 (col1_2, col2_14, col2_4, col2_12, col2_5, col2_1, col2_8, col2_11, col2_10);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4846670353860857953, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "-2000-01-01 00:00:00+00:00"}, {"distinct_range": 0, "num_eq": 500, "num_range": 9000000000, "upper_bound": "1974-11-11 15:57:45.000432+00:00"}, {"distinct_range": 0, "num_eq": 5827841046700764406, "num_range": 700000000, "upper_bound": "1980-10-17 09:04:29.000726+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 5866021801982246326}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2486747811543351642, "histo_col_type": "", "name": "__auto__", "null_count": 2552258107397426144, "row_count": 5866021801982246326}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4489229598627880654, "histo_buckets": [{"distinct_range": 0, "num_eq": 4929906398611415438, "num_range": 0, "upper_bound": "07:44:12.242513"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 5866021801982246326}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3140094383591085403, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000000, "num_range": 0, "upper_bound": "-1.0797587633132935"}, {"distinct_range": 114110356671983280, "num_eq": 4000000000, "num_range": 125806075477142251, "upper_bound": "0.2179243415594101"}, {"distinct_range": 2112536341470458600, "num_eq": 9000000, "num_range": 2333900033619348472, "upper_bound": "0.38012993335723877"}, {"distinct_range": 4856211559062267000, "num_eq": 70000000, "num_range": 4856211559062266930, "upper_bound": "0.4559743106365204"}, {"distinct_range": 0, "num_eq": 900000000, "num_range": 50000000, "upper_bound": "1.084549069404602"}, {"distinct_range": 667181710566137100, "num_eq": 5596584262105144331, "num_range": 1598158882549599026, "upper_bound": "1.2585570812225342"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 164625947300635420, "row_count": 5866021801982246326}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1742882992267851214, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "-2000-01-01 00:00:00+00:00"}, {"distinct_range": 6441419470301082000, "num_eq": 7000000, "num_range": 6441419470301081669, "upper_bound": "1992-10-01 11:25:44.000882+00:00"}, {"distinct_range": 70, "num_eq": 5000000000, "num_range": 70, "upper_bound": "1997-01-16 02:40:11.000687+00:00"}, {"distinct_range": 0, "num_eq": 0, "num_range": 90000000, "upper_bound": "2005-06-05 08:28:46.000426+00:00"}, {"distinct_range": 496.94602118516775, "num_eq": 800000000, "num_range": 500, "upper_bound": "2007-03-06 20:45:08.000336+00:00"}, {"distinct_range": 600000, "num_eq": 7641171669216872711, "num_range": 600000, "upper_bound": "2014-02-24 13:52:40.000933+00:00"}, {"distinct_range": 0, "num_eq": 1739884572152992110, "num_range": 2291310359949848870, "upper_bound": "2028-11-08 19:34:02.000349+00:00"}, {"distinct_range": 748044791067183000, "num_eq": 6849995884921923698, "num_range": 1144183595269543233, "upper_bound": "2028-12-02 04:33:38.000048+00:00"}, {"distinct_range": 0, "num_eq": 4937890304887898374, "num_range": 6532592080838176052, "upper_bound": "2033-05-12 05:43:44.0006+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 4175712840683896174, "row_count": 5866021801982246326}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 634820, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 703722, "histo_col_type": "", "name": "__auto__", "null_count": 170673, "row_count": 800000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 173419, "histo_col_type": "", "name": "__auto__", "null_count": 161643, "row_count": 800000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 278009, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 757858, "histo_col_type": "", "name": "__auto__", "null_count": 125887, "row_count": 800000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 580454, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "04:00:03.707824"}, {"distinct_range": 7394673528853967000, "num_eq": 800000000, "num_range": 7394673528853966615, "upper_bound": "08:22:48.057122"}, {"distinct_range": 0, "num_eq": 4967043275336616446, "num_range": 6000, "upper_bound": "11:05:35.992087"}, {"distinct_range": 0, "num_eq": 50, "num_range": 6467693351881545117, "upper_bound": "17:04:30.013671"}, {"distinct_range": 40000000, "num_eq": 7529231967944306387, "num_range": 40000000, "upper_bound": "18:05:05.537009"}, {"distinct_range": 0, "num_eq": 30, "num_range": 7708124406041054621, "upper_bound": "18:12:44.549444"}, {"distinct_range": 8821707343065020000, "num_eq": 90, "num_range": 8821707343065020726, "upper_bound": "21:06:00.436654"}, {"distinct_range": 4000, "num_eq": 700000, "num_range": 4000, "upper_bound": "22:46:53.080118"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26146, "histo_col_type": "", "name": "__auto__", "null_count": 413618, "row_count": 800000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 764581, "histo_buckets": [{"distinct_range": 0, "num_eq": 1047636220094561159, "num_range": 0, "upper_bound": "01:28:00.539337"}, {"distinct_range": 49420458.340552814, "num_eq": 5983345470077424268, "num_range": 80000000, "upper_bound": "03:29:00.009438"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 700000000, "upper_bound": "03:31:04.540324"}, {"distinct_range": 2285.9684733193794, "num_eq": 10000, "num_range": 3000, "upper_bound": "09:13:39.637127"}, {"distinct_range": 4656614654111539000, "num_eq": 100000000000, "num_range": 6904482665335812171, "upper_bound": "12:14:23.270638"}, {"distinct_range": 0, "num_eq": 7000000, "num_range": 900, "upper_bound": "14:56:48.391736"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 251166, "row_count": 800000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 366162, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 533994, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 111352, "histo_col_type": "", "name": "__auto__", "null_count": 727151, "row_count": 800000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 591367, "histo_col_type": "", "name": "__auto__", "null_count": 47840, "row_count": 800000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 301106, "histo_col_type": "", "name": "__auto__", "null_count": 57929, "row_count": 800000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18494, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000, "num_range": 0, "upper_bound": "2f861701-2a00-4b82-bc13-8867d8168394"}, {"distinct_range": 4000000000, "num_eq": 283331112208904359, "num_range": 4000000000, "upper_bound": "5054b6a6-a2aa-40a9-9801-64a77ba3afc6"}, {"distinct_range": 0, "num_eq": 9000000, "num_range": 7219544367644696309, "upper_bound": "7e8d9105-4709-4829-9ebd-d0685faaa0d0"}, {"distinct_range": 20000000000, "num_eq": 1000000, "num_range": 20000000000, "upper_bound": "cf4b5840-99b1-4c3e-981e-10488e068b0c"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 80000000, "upper_bound": "e9af2f67-6135-4abc-9afe-8c96540bf01c"}, {"distinct_range": 50000000, "num_eq": 8467336469007358992, "num_range": 50000000, "upper_bound": "ec32af3b-5c4b-4e3c-8ee1-73df2f45e9ae"}, {"distinct_range": 100, "num_eq": 0, "num_range": 100, "upper_bound": "ef2cf9d0-6fe9-4fe9-88be-cd5c0f5ad017"}, {"distinct_range": 5285341232191742000, "num_eq": 30, "num_range": 5285341232191741750, "upper_bound": "f38f590b-3998-4d3a-a964-5a3b563b17e2"}, {"distinct_range": 0, "num_eq": 20, "num_range": 5817221768743704477, "upper_bound": "ffffffff-ffff-ffff-ffff-ffffffffffff"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 229525, "row_count": 800000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 454150, "histo_col_type": "", "name": "__auto__", "null_count": 133761, "row_count": 800000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8816, "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 70000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 49064, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18712, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-1245448783"}, {"distinct_range": 452.798697235877, "num_eq": 300000, "num_range": 1000, "upper_bound": "-883922699"}, {"distinct_range": 616904693399191000, "num_eq": 300, "num_range": 2518341323754280432, "upper_bound": "-810162591"}, {"distinct_range": 1923097003460467200, "num_eq": 2581810261086540980, "num_range": 5062549426864006113, "upper_bound": "-658134732"}, {"distinct_range": 4220036001053473300, "num_eq": 0, "num_range": 4220036001053473452, "upper_bound": "854061721"}, {"distinct_range": 0, "num_eq": 30000000, "num_range": 461187315091984512, "upper_bound": "1822619552"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 52162, "row_count": 70000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15271, "histo_col_type": "", "name": "__auto__", "null_count": 50478, "row_count": 70000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 63692, "histo_col_type": "", "name": "__auto__", "null_count": 41351, "row_count": 70000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 60359, "histo_buckets": [{"distinct_range": 0, "num_eq": 6411396728916985059, "num_range": 0, "upper_bound": "-0.9567985534667969"}, {"distinct_range": 2770.51454817674, "num_eq": 100000, "num_range": 10000, "upper_bound": "-0.6092013120651245"}, {"distinct_range": 0, "num_eq": 5364586096058250363, "num_range": 1000000, "upper_bound": "1.7878080606460571"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 70000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 63182, "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 10927, "row_count": 70000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 41705, "histo_col_type": "", "name": "__auto__", "null_count": 52318, "row_count": 70000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 41589, "histo_col_type": "", "name": "__auto__", "null_count": 40727, "row_count": 70000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3278, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 48131, "histo_buckets": [{"distinct_range": 0, "num_eq": 4137081452246199997, "num_range": 0, "upper_bound": "-28111"}, {"distinct_range": 3436780205913382000, "num_eq": 3950948142077467905, "num_range": 5927289057515743169, "upper_bound": "-26409"}, {"distinct_range": 9000, "num_eq": 300000, "num_range": 9000, "upper_bound": "-14385"}, {"distinct_range": 2028748428669990700, "num_eq": 10000000000, "num_range": 2028748428669990649, "upper_bound": "-10038"}, {"distinct_range": 0, "num_eq": 3000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 100000000, "num_eq": 900, "num_range": 100000000, "upper_bound": "8900"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 40000, "upper_bound": "19426"}, {"distinct_range": 2415.1807723894462, "num_eq": 6534153586136699273, "num_range": 10000, "upper_bound": "25594"}, {"distinct_range": 0, "num_eq": 3751589636783216049, "num_range": 3850561631919739153, "upper_bound": "29634"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 70000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12680, "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 36771, "row_count": 70000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28378, "histo_col_type": "", "name": "__auto__", "null_count": 27806, "row_count": 70000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13680, "histo_buckets": [{"distinct_range": 0, "num_eq": 3084741332251044790, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 561630591757083400, "num_eq": 9172700535704011756, "num_range": 8004545430349537767, "upper_bound": "90260765"}, {"distinct_range": 608374437656484400, "num_eq": 300000, "num_range": 608374437656484370, "upper_bound": "666257111"}, {"distinct_range": 3000000000, "num_eq": 1007088462462539887, "num_range": 3000000000, "upper_bound": "909725148"}, {"distinct_range": 0, "num_eq": 700, "num_range": 5555045757066264355, "upper_bound": "2104771127"}, {"distinct_range": 6000000, "num_eq": 4576176790963696487, "num_range": 6000000, "upper_bound": "2180417183"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 13141, "row_count": 70000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 56300, "histo_col_type": "", "name": "__auto__", "null_count": 26941, "row_count": 70000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16971, "histo_col_type": "", "name": "__auto__", "null_count": 9757, "row_count": 70000}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 257137, "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1287492, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16207, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1127419, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1166585, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "180.253.153.27/3"}, {"distinct_range": 30000000000, "num_eq": 20, "num_range": 30000000000, "upper_bound": "213.55.61.18/3"}, {"distinct_range": 1963484833735380500, "num_eq": 5447742913454459682, "num_range": 1963484833735380416, "upper_bound": "1.55.71.132/12"}, {"distinct_range": 0, "num_eq": 9204087415992354994, "num_range": 5522230541762105152, "upper_bound": "214.80.230.222/24"}, {"distinct_range": 2000, "num_eq": 193021141762305126, "num_range": 2000, "upper_bound": "22.90.114.14/29"}, {"distinct_range": 80000000, "num_eq": 3000000, "num_range": 80000000, "upper_bound": "42e0:46cc:3d93:bc29:e0f7:259a:fbcf:ee9b/80"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 272895, "row_count": 2000000}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1419620, "histo_buckets": [{"distinct_range": 0, "num_eq": 9205291978412992440, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 7175033712083398627, "num_range": 40000000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 565947, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1359086, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 777170, "histo_col_type": "", "name": "__auto__", "null_count": 1901976, "row_count": 2000000}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1585027, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000, "num_range": 0, "upper_bound": "000000000000000000000000000"}, {"distinct_range": 4156697158647147500, "num_eq": 4400117201248654896, "num_range": 8972313650207810717, "upper_bound": "000111110100101110101100100"}, {"distinct_range": 5509877351643054000, "num_eq": 2000, "num_range": 5509877351643054488, "upper_bound": "001111010101001011011101001"}, {"distinct_range": 7000000000, "num_eq": 5298344172730485076, "num_range": 7000000000, "upper_bound": "101000010010110011011010001"}, {"distinct_range": 30000, "num_eq": 1000000, "num_range": 30000, "upper_bound": "110100111011111110101010101"}], "histo_col_type": "BIT(27)", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 858276, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1706771, "histo_col_type": "BIT(34)", "name": "__auto__", "null_count": 1247609, "row_count": 2000000}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1149457, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 400123, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1882886, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 9000, "num_range": 10000, "upper_bound": "(}\\u001bu\\u0007?\\u0001\\u0003"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 7939350003661749662, "upper_bound": "X"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 542430, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 55049920352, "histo_buckets": [{"distinct_range": 0, "num_eq": 30, "num_range": 0, "upper_bound": "01:17:49.840633"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 2000000000, "upper_bound": "05:26:23.452129"}, {"distinct_range": 0, "num_eq": 4044210073194850264, "num_range": 90000000000, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27820073661, "histo_col_type": "", "name": "__auto__", "null_count": 57651689284, "row_count": 90000000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9406923776, "histo_buckets": [{"distinct_range": 0, "num_eq": 8888126506052622899, "num_range": 0, "upper_bound": "-32304"}, {"distinct_range": 20000, "num_eq": 700000000, "num_range": 20000, "upper_bound": "-31905"}, {"distinct_range": 880.2653111432552, "num_eq": 10000000000, "num_range": 8000, "upper_bound": "-29723"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 10000000, "upper_bound": "-1"}, {"distinct_range": 1000, "num_eq": 5069314899715799251, "num_range": 1000, "upper_bound": "1"}, {"distinct_range": 700000, "num_eq": 10000000000, "num_range": 700000, "upper_bound": "16305"}, {"distinct_range": 123229206185656720, "num_eq": 5767696100562292384, "num_range": 7170477611472837300, "upper_bound": "26853"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 25006761123, "row_count": 90000000000}, {"columns": ["col5_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15240215027, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col5_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18058325193, "histo_col_type": "", "name": "__auto__", "null_count": 5625446478, "row_count": 90000000000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35452979605, "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 22991232542, "row_count": 90000000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 88946708257, "histo_col_type": "", "name": "__auto__", "null_count": 17169169075, "row_count": 90000000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11169447881, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 88938119059, "histo_col_type": "", "name": "__auto__", "null_count": 7109884148, "row_count": 90000000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5045127347, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1202111597, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 6107241816484013357, "upper_bound": "\\u000bO\\u0016hu\\\\;"}, {"distinct_range": 6729461138769905000, "num_eq": 5353833663363602738, "num_range": 6729461138769904387, "upper_bound": ".#"}, {"distinct_range": 43493115802.543785, "num_eq": 8332905879080286360, "num_range": 60000000000, "upper_bound": "NG\\u001d\\nGi1g"}, {"distinct_range": 300000, "num_eq": 5052345324623140309, "num_range": 300000, "upper_bound": "v"}, {"distinct_range": 80000, "num_eq": 500, "num_range": 80000, "upper_bound": "~/\\u0002IpH"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 90000000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col3_11) REFERENCES table2 (col2_9) ON UPDATE CASCADE;
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_2) REFERENCES table2 (col1_2);
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_0) REFERENCES table3 (col3_1) ON DELETE RESTRICT ON UPDATE RESTRICT;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('erhyw', 'w', 'jmze');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 129017874452159150, "histo_buckets": [{"distinct_range": 0, "num_eq": 4195875004166604600, "num_range": 0, "upper_bound": "1974-04-17 18:13:23.000161+00:00"}, {"distinct_range": 20000000000, "num_eq": 7360312413149887559, "num_range": 20000000000, "upper_bound": "1976-06-28 17:33:18.000563+00:00"}, {"distinct_range": 2155670145664191500, "num_eq": 4060513804587758809, "num_range": 5716331800605213941, "upper_bound": "1981-02-10 01:09:26.000022+00:00"}, {"distinct_range": 0, "num_eq": 50, "num_range": 700000, "upper_bound": "2013-01-14 15:37:43.000386+00:00"}, {"distinct_range": 0, "num_eq": 6000000, "num_range": 2916215618535808513, "upper_bound": "2028-02-22 11:54:29.000214+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 85626137613124441, "row_count": 218534130614147977}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 190624739468490380, "histo_buckets": [{"distinct_range": 0, "num_eq": 878197904522181090, "num_range": 0, "upper_bound": "1983-09-24 09:08:22.000962+00:00"}, {"distinct_range": 5613496208.038565, "num_eq": 300, "num_range": 6000000000, "upper_bound": "2000-02-12 23:09:05.0009+00:00"}, {"distinct_range": 2078064431868723000, "num_eq": 4000000, "num_range": 2078064431868722859, "upper_bound": "2010-05-06 12:27:38.000458+00:00"}, {"distinct_range": 1043.3713698342383, "num_eq": 8609708296083786882, "num_range": 2000, "upper_bound": "2011-07-12 21:14:35.000064+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 218534130614147977}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 180174404084233987, "histo_col_type": "", "name": "__auto__", "null_count": 138452635354474576, "row_count": 218534130614147977}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 194184848989573365, "histo_buckets": [{"distinct_range": 0, "num_eq": 8274320454295232827, "num_range": 0, "upper_bound": "12:08:39.900372"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 218534130614147977}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10995320355704010, "histo_buckets": [{"distinct_range": 0, "num_eq": 20, "num_range": 0, "upper_bound": "-2.7516517639160156"}, {"distinct_range": 7103936429715027000, "num_eq": 3626377068722954570, "num_range": 7103936429715026596, "upper_bound": "-1.9647129774093628"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 6000000, "upper_bound": "-1.3399288654327393"}, {"distinct_range": 4138221032323849700, "num_eq": 1000, "num_range": 4138221032323849711, "upper_bound": "-0.7831025719642639"}, {"distinct_range": 0, "num_eq": 40, "num_range": 8641835475688815875, "upper_bound": "-0.14899791777133942"}, {"distinct_range": 0, "num_eq": 2243156331760962663, "num_range": 8019776903093722014, "upper_bound": "0.0721757784485817"}, {"distinct_range": 0, "num_eq": 8362342517312690960, "num_range": 6773755288480967148, "upper_bound": "1.0101137161254883"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 141272320288250200, "row_count": 218534130614147977}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7796047062239991333, "histo_col_type": "", "name": "__auto__", "null_count": 279909610593354976, "row_count": 8085280832740461711}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1024539594473430686, "histo_buckets": [{"distinct_range": 0, "num_eq": 6672254827564624681, "num_range": 0, "upper_bound": "0e0310ec-9cc7-42d0-8443-8e203701adf0"}, {"distinct_range": 80000000, "num_eq": 1000000000, "num_range": 80000000, "upper_bound": "4889a30d-0f9c-4a23-bc2e-2251b5e6df3b"}, {"distinct_range": 80000, "num_eq": 70000000, "num_range": 80000, "upper_bound": "6e561080-7c08-4c7a-992f-4382fd16cf65"}, {"distinct_range": 16.83353628670128, "num_eq": 5165419676059501145, "num_range": 40, "upper_bound": "74fe6642-8823-4c47-ab34-877eab98e5e4"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 8024710028829469888, "upper_bound": "7bbdc776-e61e-4a70-b540-64bf49f03149"}, {"distinct_range": 0, "num_eq": 6133302491646713367, "num_range": 0, "upper_bound": "ad5a5685-923d-4350-86c7-ea262ed8248c"}, {"distinct_range": 227136270.38441187, "num_eq": 100000000000, "num_range": 300000000, "upper_bound": "d7bb3acf-e4c6-4380-a456-d5bfd6a5e8a9"}, {"distinct_range": 0, "num_eq": 30000, "num_range": 400, "upper_bound": "f37b6c94-768e-45f9-9738-d39b884f4f9a"}, {"distinct_range": 444058420542734460, "num_eq": 20, "num_range": 553453662561726930, "upper_bound": "ffffffff-ffff-ffff-ffff-ffffffffffff"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 4275932187220757062, "row_count": 8085280832740461711}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6287214142138472710, "histo_col_type": "", "name": "__auto__", "null_count": 1010011920939823192, "row_count": 8085280832740461711}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2317730063778215848, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "02:48:36.560619"}, {"distinct_range": 100, "num_eq": 1000000, "num_range": 100, "upper_bound": "05:25:54.051878"}, {"distinct_range": 0, "num_eq": 500000, "num_range": 1319036944677322051, "upper_bound": "07:22:49.651355"}, {"distinct_range": 0, "num_eq": 90000000000, "num_range": 0, "upper_bound": "12:15:37.253787"}, {"distinct_range": 1165463582061380000, "num_eq": 2409534059234358743, "num_range": 1295824339179708070, "upper_bound": "12:41:52.921273"}, {"distinct_range": 700, "num_eq": 3241152307489400868, "num_range": 700, "upper_bound": "17:54:08.067226"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 7161787003329267793, "upper_bound": "18:50:04.014958"}, {"distinct_range": 0, "num_eq": 5287920030673929852, "num_range": 200, "upper_bound": "21:11:51.098171"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 7097602869577804895, "row_count": 8085280832740461711}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3772702703300020768, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8085280832740461711}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1139044344655721678, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8085280832740461711}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2926965541681608421, "histo_col_type": "", "name": "__auto__", "null_count": 6856476180293057643, "row_count": 8085280832740461711}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1373234306611229619, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8085280832740461711}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3384735130900406707, "histo_col_type": "INT8", "name": "__auto__", "null_count": 567105893584243935, "row_count": 8085280832740461711}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5820589467873887158, "histo_col_type": "", "name": "__auto__", "null_count": 5510231370291190335, "row_count": 8085280832740461711}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1104044605072020661, "histo_buckets": [{"distinct_range": 0, "num_eq": 2101485344731375064, "num_range": 0, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 8085280832740461711}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1587049163642926673, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8085280832740461711}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7704553931977382856, "histo_col_type": "", "name": "__auto__", "null_count": 4437834571003472231, "row_count": 8085280832740461711}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5614378034651509103, "histo_col_type": "", "name": "__auto__", "null_count": 7724588390665379000, "row_count": 8085280832740461711}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6343981046379737003, "histo_col_type": "", "name": "__auto__", "null_count": 116750979012924594, "row_count": 8085280832740461711}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6811117, "histo_col_type": "", "name": "__auto__", "null_count": 6974658, "row_count": 10000000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7694680, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000, "num_range": 0, "upper_bound": "-10982"}, {"distinct_range": 0, "num_eq": 60000000000, "num_range": 6654620339174384611, "upper_bound": "1189"}, {"distinct_range": 0, "num_eq": 2945525134888112408, "num_range": 70, "upper_bound": "17826"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1144282, "histo_col_type": "", "name": "__auto__", "null_count": 7947550, "row_count": 10000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6526984, "histo_col_type": "", "name": "__auto__", "null_count": 4040760, "row_count": 10000000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9333100, "histo_col_type": "", "name": "__auto__", "null_count": 6934520, "row_count": 10000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7997256, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000, "num_range": 0, "upper_bound": "00:00:00"}, {"distinct_range": 296585251548979460, "num_eq": 0, "num_range": 296585251548979479, "upper_bound": "02:33:08.166196"}, {"distinct_range": 0, "num_eq": 8869052938962667261, "num_range": 0, "upper_bound": "04:50:07.94252"}, {"distinct_range": 2062121150120384000, "num_eq": 50000000000, "num_range": 3228871078089708485, "upper_bound": "06:56:36.045973"}, {"distinct_range": 0, "num_eq": 4393554090345918025, "num_range": 0, "upper_bound": "12:32:59.718335"}, {"distinct_range": 0, "num_eq": 900, "num_range": 5000, "upper_bound": "13:17:50.450337"}, {"distinct_range": 0, "num_eq": 3420382200907784402, "num_range": 40000000000, "upper_bound": "14:00:22.805949"}, {"distinct_range": 7340397422809292000, "num_eq": 4359669082239385496, "num_range": 7340397422809291567, "upper_bound": "16:24:44.865123"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2342862, "histo_col_type": "", "name": "__auto__", "null_count": 4170235, "row_count": 10000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5961973, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000, "num_range": 0, "upper_bound": "-1.0046364068984985"}, {"distinct_range": 400, "num_eq": 10000, "num_range": 400, "upper_bound": "-0.6442777514457703"}, {"distinct_range": 4423539919468773400, "num_eq": 7000000, "num_range": 7156610061850636636, "upper_bound": "-0.1681525558233261"}, {"distinct_range": 30, "num_eq": 6697749272188878349, "num_range": 30, "upper_bound": "-0.10320442914962769"}, {"distinct_range": 400, "num_eq": 6452537397562995667, "num_range": 400, "upper_bound": "0.0"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 6290632751445662679, "upper_bound": "0.46677926182746887"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9992458, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2681217, "histo_buckets": [{"distinct_range": 0, "num_eq": 10, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 506990.8562942566, "num_eq": 1538791338283235799, "num_range": 700000, "upper_bound": "1971023313"}, {"distinct_range": 1241535438019121400, "num_eq": 2824285769137627949, "num_range": 8216093479899150998, "upper_bound": "2659028489"}, {"distinct_range": 3631467183.072005, "num_eq": 3059714287771608717, "num_range": 40000000000, "upper_bound": "2938725680"}, {"distinct_range": 0, "num_eq": 0, "num_range": 8907281049003040604, "upper_bound": "3264872368"}, {"distinct_range": 2277029374695776500, "num_eq": 9000000000, "num_range": 2381363074877489647, "upper_bound": "4012135693"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 8042408, "row_count": 10000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20758, "histo_col_type": "INT4", "name": "__auto__", "null_count": 970500, "row_count": 10000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2772214, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "1050264157"}, {"distinct_range": 0, "num_eq": 3720953701523184306, "num_range": 10000000, "upper_bound": "2167000572"}, {"distinct_range": 0, "num_eq": 3059596145758257730, "num_range": 0, "upper_bound": "2414483222"}, {"distinct_range": 0, "num_eq": 200, "num_range": 30000000000, "upper_bound": "2919008595"}, {"distinct_range": 0, "num_eq": 7468719734084787046, "num_range": 30000, "upper_bound": "3232408484"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 10000000, "upper_bound": "3952609777"}, {"distinct_range": 543.9410541680032, "num_eq": 800, "num_range": 800, "upper_bound": "4236401643"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 5516514, "row_count": 10000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2967033, "histo_col_type": "", "name": "__auto__", "null_count": 3682479, "row_count": 10000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4057649, "histo_buckets": [{"distinct_range": 0, "num_eq": 1486683267722404148, "num_range": 0, "upper_bound": "1980-11-17 06:05:54.000869+00:00"}, {"distinct_range": 20952905748046090, "num_eq": 6730192290783842278, "num_range": 55815227711438419, "upper_bound": "1983-06-21 19:33:17.000998+00:00"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 6694115564825911318, "upper_bound": "1988-06-06 08:34:46.000026+00:00"}, {"distinct_range": 196728.32444383483, "num_eq": 7043925568234768786, "num_range": 400000, "upper_bound": "2009-08-11 21:50:51.000331+00:00"}, {"distinct_range": 0, "num_eq": 90000000000, "num_range": 5326573707066318201, "upper_bound": "2011-05-19 11:30:13.000222+00:00"}, {"distinct_range": 46071882651188056, "num_eq": 5687751306812703300, "num_range": 5278644905706803696, "upper_bound": "2025-09-11 19:14:23.000427+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3387104, "histo_buckets": [{"distinct_range": 0, "num_eq": 316619039846745942, "num_range": 0, "upper_bound": "NaN"}, {"distinct_range": 7806390253698415000, "num_eq": 5733779287373944635, "num_range": 8145111692484607437, "upper_bound": "-1.1862609322392714"}, {"distinct_range": 1984484432263309300, "num_eq": 3501849902957966511, "num_range": 2775209879419956665, "upper_bound": "-0.9854380501059765"}, {"distinct_range": 66218.1731338895, "num_eq": 934194201259109683, "num_range": 400000, "upper_bound": "-0.9759078545571565"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 20000, "upper_bound": "0.22546475350955628"}, {"distinct_range": 3000, "num_eq": 7569178477072641242, "num_range": 3000, "upper_bound": "0.29736168351786196"}, {"distinct_range": 1670326.6838743559, "num_eq": 2000000, "num_range": 8000000, "upper_bound": "1.0"}, {"distinct_range": 2951873620138953700, "num_eq": 6796099116889710660, "num_range": 4495421257014999470, "upper_bound": "1.2227423223511549"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 7653835, "row_count": 10000000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4247452, "histo_col_type": "", "name": "__auto__", "null_count": 2428255, "row_count": 10000000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3762972354046909766, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 26943.07234307297, "num_eq": 2000, "num_range": 30000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 6409993052140120120}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1392334789576824020, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6409993052140120120}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1120387964224354206, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6409993052140120120}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1221028595678977837, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6409993052140120120}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5524741789185292484, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000, "num_range": 0, "upper_bound": "22.52.78.135/0"}, {"distinct_range": 600000000, "num_eq": 395822988790541401, "num_range": 600000000, "upper_bound": "223.254.178.102/11"}, {"distinct_range": 0, "num_eq": 1560746015845101858, "num_range": 600000, "upper_bound": "160.53.156.211/14"}, {"distinct_range": 131810777709454380, "num_eq": 300000, "num_range": 448221105792236707, "upper_bound": "39.215.32.47/16"}, {"distinct_range": 7980784520798885000, "num_eq": 600000000, "num_range": 7980784520798884359, "upper_bound": "147.168.189.172/21"}, {"distinct_range": 0, "num_eq": 100, "num_range": 8216004710420656644, "upper_bound": "3252:252d:d77b:271d:67c8:8ee1:bf9b:5ec3/14"}, {"distinct_range": 533691102.95063233, "num_eq": 5789348914352624387, "num_range": 800000000, "upper_bound": "98ef:2a68:efdc:5f9a:dbfa:7f3b:7e23:7549/121"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 4131230139674747605, "row_count": 6409993052140120120}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2206440298202930018, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6409993052140120120}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5692763503809738177, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "0001010001000101000001100010011001"}, {"distinct_range": 0, "num_eq": 6172250915367711614, "num_range": 5147856114356333109, "upper_bound": "0110100110010111000011110110000010"}, {"distinct_range": 46.250505896957634, "num_eq": 30000, "num_range": 100, "upper_bound": "1000101001111011010011010001011100"}, {"distinct_range": 0, "num_eq": 3995989715379567990, "num_range": 944459466075631977, "upper_bound": "1001111011000111000000001001101100"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 6241008869290183553, "upper_bound": "1010111100111011111111000000011111"}, {"distinct_range": 10000000, "num_eq": 70, "num_range": 10000000, "upper_bound": "1111000110110000101000100111111000"}], "histo_col_type": "BIT(34)", "name": "__auto__", "null_count": 4686241501443381877, "row_count": 6409993052140120120}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5676368147308726268, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000000, "num_range": 0, "upper_bound": "000000000000000000000000000"}, {"distinct_range": 5716328653080846000, "num_eq": 7157405566334927035, "num_range": 5716328653080846356, "upper_bound": "001000101101011111101011101"}, {"distinct_range": 0, "num_eq": 5000000, "num_range": 100, "upper_bound": "010100010000000010001001100"}, {"distinct_range": 7127209739021981000, "num_eq": 3393041246690970083, "num_range": 7127209739021980350, "upper_bound": "011111111111111111111111111"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 10000000, "upper_bound": "100111110010100110011011001"}], "histo_col_type": "BIT(27)", "name": "__auto__", "null_count": 0, "row_count": 6409993052140120120}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2325170561003600883, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6409993052140120120}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4872192570319717206, "histo_col_type": "", "name": "__auto__", "null_count": 4431758557001249614, "row_count": 6409993052140120120}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 847071413962727708, "histo_buckets": [{"distinct_range": 0, "num_eq": 5468452856865582958, "num_range": 0, "upper_bound": "(t6ai|Mj"}, {"distinct_range": 0, "num_eq": 3483338236800419715, "num_range": 4406169708324014692, "upper_bound": ",R.[ "}, {"distinct_range": 0, "num_eq": 487113959889911154, "num_range": 8952177815269430824, "upper_bound": "Pv"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 6409993052140120120}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5695906086785315117, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "-26772"}, {"distinct_range": 0, "num_eq": 7000000, "num_range": 0, "upper_bound": "-23762"}, {"distinct_range": 0, "num_eq": 4706580636549360377, "num_range": 8000, "upper_bound": "-14856"}, {"distinct_range": 0, "num_eq": 7062106029001026749, "num_range": 0, "upper_bound": "5902"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 6409993052140120120}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5265715788398416363, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6409993052140120120}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3707389947298159675, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6409993052140120120}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6141781835921401583, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6409993052140120120}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2699469491890641232, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6409993052140120120}]';
        test_1        |             ALTER TABLE table5 INJECT STATISTICS e'[{"columns": ["col5_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2610266047030463975, "histo_col_type": "", "name": "__auto__", "null_count": 1284970062380947147, "row_count": 5131891129973892064}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 514651421384812054, "histo_col_type": "", "name": "__auto__", "null_count": 1580982451196938109, "row_count": 5131891129973892064}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 606793632799541802, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5131891129973892064}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 167889369389788199, "histo_col_type": "INT2", "name": "__auto__", "null_count": 3946

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2509113&tab=buildLog) on [master@c41a2ed503cd31d95bc3ee4365663811772c8bd3](https://github.com/cockroachdb/cockroach/commits/c41a2ed503cd31d95bc3ee4365663811772c8bd3):

```
        test_1        |             CREATE TABLE table4 (col3_13 REGTYPE NOT NULL, col4_1 INT8 NULL, col4_2 BYTEA, col4_3 VARCHAR NOT NULL, col4_4 BOOL, col4_5 REGPROC NULL, col4_6 BIT(23) NULL, col4_7 TIMESTAMP NOT NULL, col4_8 REGTYPE, col4_9 BYTEA NOT NULL, col4_10 REGTYPE NOT NULL, col4_11 VARCHAR NOT NULL, col4_12 CHAR, col4_13 CHAR NULL, col4_14 BIT(49) NULL, col4_15 INT8 NOT NULL, col4_16 BIT(19), col4_17 TIMESTAMP, PRIMARY KEY (col3_13, col4_5, col4_7, col4_14, col4_11, col4_15, col4_17, col4_3, col4_13, col4_10, col4_4, col4_8, col4_16, col4_1, col4_9));
        test_1        |             CREATE INDEX ON table4 (col4_12 DESC, col4_6 ASC, col4_17, col4_5 DESC, col4_11, col4_7 DESC, col4_8 DESC, col4_4 ASC, col4_13 DESC, col4_15, col4_10, col4_16, col4_3 ASC, col4_2 ASC, col3_13 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_14, col4_15, col4_2) INCLUDE (col4_6, col4_12);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_14 DESC, col4_6 DESC, col4_7, col3_13 DESC) INCLUDE (col4_12);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_10 ASC, col4_3 DESC, col4_6, col3_13 ASC, col4_4 ASC, col4_16 ASC, col4_12 DESC, col4_7, col4_2, col4_11 DESC, col4_17, col4_5, col4_9, col4_15, col4_14 ASC, col4_1 ASC, col4_8);
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_9, col1_7) REFERENCES table2 (col2_6, col2_4);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('rocnf', 'eq', 'iea', 'oq', 'tqv', 'kfucx');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('t', 'pdpgp', 'kbpv', 'rtsf', 'qmj');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('yh', 'jh', 'o', 'jier', 'xs');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('yua');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('frbtk', 'nu', 'sinn');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (1.46s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 REGTYPE NOT NULL, col1_1 TIMETZ NULL, col1_2 TIMETZ NOT NULL, col1_3 DATE NULL, col1_4 "char", col1_5 REGPROC, col1_6 FLOAT4 NOT NULL, col1_7 STRING NULL, col1_8 TIMETZ NULL, col1_9 OID NOT NULL, col1_10 INTERVAL, col1_11 INT4[], col1_12 STRING NOT NULL, col1_13 CHAR NOT NULL, col1_14 BOOL, PRIMARY KEY (col1_0 DESC, col1_13, col1_9), UNIQUE (col1_12 ASC) STORING (col1_2, col1_3, col1_4, col1_5, col1_6, col1_7, col1_10, col1_11) WHERE (((((table1.col1_8 > '24:00:00-15:59:00') OR (table1.col1_4 > e'\U00002603')) AND (table1.col1_2 > '00:00:00+15:59:00')) AND (table1.col1_13 != '"')) OR (table1.col1_7 > '')) OR (table1.col1_3 != '1970-01-01'), FAMILY (col1_5, col1_13), FAMILY (col1_6, col1_3, col1_8, col1_12), FAMILY (col1_4, col1_11, col1_14), FAMILY (col1_2, col1_7), FAMILY (col1_9), FAMILY (col1_0, col1_1), FAMILY (col1_10));
        test_1        |             CREATE TABLE table2 (col1_0 REGTYPE NOT NULL, col1_13 CHAR NOT NULL, col1_9 OID NOT NULL, col2_3 INET, col2_4 CHAR NOT NULL, col2_5 REGPROCEDURE NOT NULL, col2_6 NAME NULL, col2_7 UUID NULL, col2_8 BOX2D NULL, col2_9 OID NOT NULL, col2_10 INET NULL, col2_11 OID NOT NULL, col2_12 INET, col2_13 INET, col2_14 TIME, col2_15 STRING NOT NULL, col2_16 INET, col2_17 INTERVAL NOT NULL, PRIMARY KEY (col1_0 DESC, col1_13, col1_9, col2_10, col2_6), UNIQUE (col2_9, col2_3, col2_11, col2_12, col1_13 DESC, col1_9 DESC, col2_8 DESC, col2_16 ASC, col2_13 DESC, col2_17, col1_0) STORING (col2_4, col2_5, col2_7, col2_14, col2_15) WHERE ((((table2.col1_13 > e'\U00002603':::STRING) AND (table2.col2_6 <= '"':::STRING)) AND (table2.col2_4 != e'\'':::STRING)) AND (table2.col2_15 >= 'X':::STRING)) OR (table2.col2_14 <= '24:00:00':::TIME), INDEX (col2_16 DESC, col2_10, col2_8 ASC, col2_5, col2_3) STORING (col2_4, col2_7, col2_9, col2_11, col2_12, col2_13, col2_17) WHERE (((table2.col2_15 <= e'\'':::STRING) OR (table2.col2_14 < '00:00:00':::TIME)) OR (table2.col1_13 != '':::STRING)) OR (table2.col2_6 < 'X':::STRING), INDEX (col2_5 ASC, col1_13 DESC, col2_6 ASC, col2_10 DESC, col2_13 DESC, col2_4, col2_7 DESC, col2_9 DESC, col2_12) STORING (col2_16, col2_17), FAMILY (col1_9, col2_16, col2_4, col2_3, col1_13, col2_14, col2_8), FAMILY (col2_6, col2_10, col2_15), FAMILY (col2_5), FAMILY (col1_0), FAMILY (col2_12, col2_9), FAMILY (col2_7, col2_11), FAMILY (col2_13, col2_17)) INTERLEAVE IN PARENT table1 (col1_0, col1_13, col1_9);
        test_1        |             CREATE TABLE table3 (col1_0 REGTYPE NOT NULL, col1_13 CHAR NOT NULL, col1_9 OID NOT NULL, col2_10 INET NULL, col2_6 NAME NULL, col3_5 CHAR, col3_6 INT8 NULL, col3_7 INT8 NULL, col3_8 FLOAT4 NULL, col3_9 CHAR NULL, col3_10 INT4 NULL, col3_11 INET NOT NULL, col3_12 INT4 NULL, col3_13 UUID NOT NULL, col3_14 FLOAT8 NULL, col3_15 REGCLASS, col3_16 TIME, col3_17 OID NOT NULL, col3_18 INT4, PRIMARY KEY (col1_0 DESC, col1_13, col1_9, col2_10, col2_6, col3_16 ASC), UNIQUE (col1_0 DESC, col3_16 DESC) STORING (col3_5, col3_7, col3_8, col3_9, col3_10, col3_11, col3_13, col3_14, col3_17, col3_18), INDEX (col3_17 DESC, col2_6) STORING (col3_6, col3_8, col3_9, col3_11, col3_14, col3_15, col3_18), FAMILY (col3_10, col2_6), FAMILY (col3_18), FAMILY (col3_17, col3_15, col3_16, col3_5, col3_8, col1_0), FAMILY (col1_9), FAMILY (col3_12), FAMILY (col3_7, col1_13), FAMILY (col3_11), FAMILY (col2_10), FAMILY (col3_13, col3_9), FAMILY (col3_14), FAMILY (col3_6), UNIQUE (col3_13) WHERE (((((table3.col1_13 != '') OR (table3.col3_10 >= 1)) AND (table3.col3_5 <= e'\U00002603')) OR (table3.col3_16 >= '24:00:00')) AND (table3.col3_9 <= '')) AND (table3.col2_6 <= e'\U00002603')) INTERLEAVE IN PARENT table2 (col1_0, col1_13, col1_9, col2_10, col2_6);
        test_1        |             CREATE TABLE table4 (col4_0 NAME NOT NULL, col4_1 UUID NULL, col4_2 INTERVAL NOT NULL, col4_3 BYTES NOT NULL, col4_4 STRING, col4_5 DECIMAL NULL, col4_6 BIT(48) NULL, col4_7 INTERVAL, col4_8 TIMETZ NOT NULL, col4_9 INT8 NOT NULL, col4_10 REGPROCEDURE NOT NULL, col4_11 REGPROCEDURE, col4_12 REGPROC, col4_13 FLOAT4 NULL, col4_14 NAME NOT NULL, col4_15 "char" NOT NULL, col4_16 VARCHAR NOT NULL, col4_17 INT4 NULL, PRIMARY KEY (col4_9 ASC, col4_15 ASC, col4_0 ASC, col4_8 ASC, col4_16 ASC, col4_3 ASC), UNIQUE (col4_13 DESC) WHERE ((((((((table4.col4_13 <= 3.4028234663852886e+38) OR (table4.col4_0 <= '')) OR (table4.col4_3 <= '\x')) OR (table4.col4_9 <= 32767)) OR (table4.col4_16 < e'\x00')) AND (table4.col4_5 <= -1)) AND (table4.col4_14 <= e'\'')) AND (table4.col4_17 != 32767)) OR (table4.col4_8 = '00:00:00+15:59:00'), UNIQUE (col4_15, col4_14, col4_12 DESC, col4_1 ASC, col4_4 ASC, col4_13 ASC) STORING (col4_2, col4_6, col4_7, col4_10) WHERE ((((((((((table4.col4_15 >= '"':::STRING) AND (table4.col4_16 <= '"':::STRING)) OR (table4.col4_8 != '00:00:00+15:59:00':::TIMETZ)) OR (table4.col4_17 < 2147483647:::INT8)) OR (table4.col4_13 = 1.401298464324817e-45:::FLOAT8)) AND (table4.col4_3 != e'\\x22':::BYTES)) OR (table4.col4_9 = 0:::INT8)) OR (table4.col4_5 = 1:::DECIMAL)) OR (table4.col4_14 > 'X':::STRING)) OR (table4.col4_4 >= e'\'':::STRING)) OR (table4.col4_0 != e'\'':::STRING), INDEX (col4_2 DESC, col4_0 DESC) STORING (col4_1, col4_4, col4_5, col4_7, col4_14, col4_17) WHERE (((((table4.col4_15 > '"') AND (table4.col4_3 != '\xff')) AND (table4.col4_17 = -2147483648)) OR (table4.col4_5 >= 1)) OR (table4.col4_14 <= e'\x00')) OR (table4.col4_13 >= 0.0), INDEX (col4_3 ASC, col4_9, col4_7 DESC, col4_2 ASC) STORING (col4_4, col4_6, col4_11, col4_12) WHERE ((((((table4.col4_13 > '+Inf':::FLOAT8) AND (table4.col4_4 != '"':::STRING)) AND (table4.col4_9 != (-1):::INT8)) AND (table4.col4_15 != '':::STRING)) AND (table4.col4_3 >= e'\\xff':::BYTES)) OR (table4.col4_5 != (-1.234E+401):::DECIMAL)) OR (table4.col4_16 < 'X':::STRING), FAMILY (col4_11, col4_4, col4_6, col4_12, col4_17), FAMILY (col4_10, col4_7), FAMILY (col4_3), FAMILY (col4_5), FAMILY (col4_8), FAMILY (col4_9), FAMILY (col4_14, col4_0), FAMILY (col4_15), FAMILY (col4_1, col4_13, col4_16), FAMILY (col4_2));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 193586, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 102102, "histo_col_type": "", "name": "__auto__", "null_count": 518376, "row_count": 800000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24194, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 157017, "histo_col_type": "", "name": "__auto__", "null_count": 45213, "row_count": 800000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 654153, "histo_col_type": "", "name": "__auto__", "null_count": 591270, "row_count": 800000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 701368, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 602571, "histo_col_type": "", "name": "__auto__", "null_count": 792469, "row_count": 800000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 127553, "histo_col_type": "", "name": "__auto__", "null_count": 256066, "row_count": 800000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 562219, "histo_buckets": [{"distinct_range": 0, "num_eq": 3508845474140787431, "num_range": 0, "upper_bound": "148585424"}, {"distinct_range": 0, "num_eq": 7875483296763114941, "num_range": 3475089803942820871, "upper_bound": "2602022314"}, {"distinct_range": 800, "num_eq": 60000000, "num_range": 800, "upper_bound": "3485983942"}, {"distinct_range": 0, "num_eq": 600, "num_range": 2770047965265172178, "upper_bound": "3486232296"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 601075, "histo_col_type": "", "name": "__auto__", "null_count": 754999, "row_count": 800000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 458324, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 535584, "histo_col_type": "", "name": "__auto__", "null_count": 234288, "row_count": 800000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 448740, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 8435457553184442332, "num_range": 500000, "upper_bound": "\\u0015dIq"}, {"distinct_range": 0, "num_eq": 100, "num_range": 6197695544208984969, "upper_bound": "+0~g.j"}, {"distinct_range": 6.51152654056078, "num_eq": 60, "num_range": 50, "upper_bound": "n_2+3\\u0005a"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 215898, "histo_col_type": "", "name": "__auto__", "null_count": 735833, "row_count": 800000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 288561, "histo_col_type": "", "name": "__auto__", "null_count": 783040, "row_count": 800000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 353086244, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 644214747, "histo_col_type": "", "name": "__auto__", "null_count": 257551993, "row_count": 700000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 85476693, "histo_col_type": "", "name": "__auto__", "null_count": 358953297, "row_count": 700000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 514481416, "histo_col_type": "", "name": "__auto__", "null_count": 390315347, "row_count": 700000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 245621691, "histo_col_type": "", "name": "__auto__", "null_count": 410734439, "row_count": 700000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 377885868, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 41194451, "histo_col_type": "", "name": "__auto__", "null_count": 579588179, "row_count": 700000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 47938377, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 387832800, "histo_col_type": "", "name": "__auto__", "null_count": 485522111, "row_count": 700000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 117758016, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 238711318, "histo_col_type": "", "name": "__auto__", "null_count": 60162711, "row_count": 700000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 101181288, "histo_buckets": [{"distinct_range": 0, "num_eq": 800, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 132408837116064720, "num_eq": 800, "num_range": 132408837116064717, "upper_bound": "99823457"}, {"distinct_range": 0, "num_eq": 7016006127055756750, "num_range": 9000, "upper_bound": "392400687"}, {"distinct_range": 41.60210843255375, "num_eq": 900, "num_range": 100, "upper_bound": "1182548107"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 1849472667462184694, "upper_bound": "2530739103"}, {"distinct_range": 9.811231605167121, "num_eq": 1000000000, "num_range": 30, "upper_bound": "3066493796"}, {"distinct_range": 4815900964987001000, "num_eq": 1000000000, "num_range": 5956288549032938982, "upper_bound": "3643036670"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 492785554, "histo_col_type": "", "name": "__auto__", "null_count": 256680274, "row_count": 700000000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 422804620, "histo_col_type": "INET", "name": "__auto__", "null_count": 427023360, "row_count": 700000000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 349733587, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 446293908, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 3427514.4362611864, "num_eq": 1773679110562791573, "num_range": 5000000, "upper_bound": "252540503"}, {"distinct_range": 0, "num_eq": 8504870573122211394, "num_range": 0, "upper_bound": "1368560372"}, {"distinct_range": 0, "num_eq": 500000, "num_range": 0, "upper_bound": "1880709338"}, {"distinct_range": 6471497982200170000, "num_eq": 3823077116838154663, "num_range": 6471497982200170030, "upper_bound": "2520632750"}, {"distinct_range": 0, "num_eq": 585513334114932805, "num_range": 8188636128456576497, "upper_bound": "3590495284"}, {"distinct_range": 35.84464591517695, "num_eq": 7420220051123120573, "num_range": 60, "upper_bound": "4270277012"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 497234070, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 296210506, "histo_buckets": [{"distinct_range": 0, "num_eq": 500000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 5864713083564979000, "num_eq": 500000, "num_range": 5864713083564979062, "upper_bound": "974629202"}, {"distinct_range": 1000000000, "num_eq": 10000, "num_range": 1000000000, "upper_bound": "2782661181"}, {"distinct_range": 0, "num_eq": 1285346412875333975, "num_range": 5643119810643413766, "upper_bound": "2984775620"}, {"distinct_range": 80000, "num_eq": 4988387132798207680, "num_range": 80000, "upper_bound": "3196616372"}, {"distinct_range": 0, "num_eq": 70, "num_range": 80000000000, "upper_bound": "3833933243"}, {"distinct_range": 100000, "num_eq": 8000, "num_range": 100000, "upper_bound": "4142449567"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 700000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 121, "histo_col_type": "", "name": "__auto__", "null_count": 98, "row_count": 300}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 230, "histo_col_type": "", "name": "__auto__", "null_count": 70, "row_count": 300}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 179, "histo_col_type": "", "name": "__auto__", "null_count": 250, "row_count": 300}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 58, "histo_col_type": "", "name": "__auto__", "null_count": 249, "row_count": 300}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 81, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 292, "histo_col_type": "", "name": "__auto__", "null_count": 162, "row_count": 300}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 103, "histo_col_type": "", "name": "__auto__", "null_count": 44, "row_count": 300}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 296, "histo_col_type": "", "name": "__auto__", "null_count": 191, "row_count": 300}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 194, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 61, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 188, "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 300}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 204, "histo_buckets": [{"distinct_range": 0, "num_eq": 800, "num_range": 0, "upper_bound": "1534309030"}, {"distinct_range": 15613825086.67105, "num_eq": 500000, "num_range": 20000000000, "upper_bound": "2417018763"}, {"distinct_range": 0, "num_eq": 30000000, "num_range": 3164807564076545824, "upper_bound": "2428650108"}, {"distinct_range": 76.29128521563098, "num_eq": 7166410955990895401, "num_range": 80, "upper_bound": "3950112122"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 300}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 153, "histo_col_type": "", "name": "__auto__", "null_count": 49, "row_count": 300}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 60, "histo_col_type": "", "name": "__auto__", "null_count": 105, "row_count": 300}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 61, "histo_col_type": "", "name": "__auto__", "null_count": 38, "row_count": 300}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22, "histo_col_type": "", "name": "__auto__", "null_count": 297, "row_count": 300}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32, "histo_col_type": "", "name": "__auto__", "null_count": 153, "row_count": 300}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 184, "histo_col_type": "", "name": "__auto__", "null_count": 127, "row_count": 300}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 99, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3864163524705336993, "histo_col_type": "", "name": "__auto__", "null_count": 3136520796414920112, "row_count": 5199628374650278054}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2969030442774947328, "histo_col_type": "", "name": "__auto__", "null_count": 1657242573655363835, "row_count": 5199628374650278054}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4241137187567252501, "histo_col_type": "", "name": "__auto__", "null_count": 676950451438912783, "row_count": 5199628374650278054}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4996183654745827699, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5199628374650278054}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3211186445365494234, "histo_col_type": "", "name": "__auto__", "null_count": 3807462346066770386, "row_count": 5199628374650278054}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 694545060638955532, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5199628374650278054}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4699556738755759395, "histo_col_type": "", "name": "__auto__", "null_count": 2079746103110025132, "row_count": 5199628374650278054}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 661474274246233664, "histo_col_type": "", "name": "__auto__", "null_count": 2853278094128972003, "row_count": 5199628374650278054}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4080624256564803878, "histo_buckets": [{"distinct_range": 0, "num_eq": 6864326871233640742, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 6000000000, "num_eq": 8000000000, "num_range": 6000000000, "upper_bound": "\\r"}, {"distinct_range": 7000, "num_eq": 9000, "num_range": 7000, "upper_bound": "$"}, {"distinct_range": 0, "num_eq": 7000000, "num_range": 6987455288463724967, "upper_bound": ")"}, {"distinct_range": 8380504854299974000, "num_eq": 90000, "num_range": 9043430211428650364, "upper_bound": ","}, {"distinct_range": 10, "num_eq": 2769466880209328946, "num_range": 10, "upper_bound": "@"}, {"distinct_range": 2695383251723764000, "num_eq": 2972130180554443630, "num_range": 6503224168699593537, "upper_bound": "G"}, {"distinct_range": 0, "num_eq": 800, "num_range": 1008734831943060154, "upper_bound": "H"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 5199628374650278054}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3243778990592860980, "histo_buckets": [{"distinct_range": 0, "num_eq": 3340357142947607728, "num_range": 0, "upper_bound": "-0.996345043182373"}, {"distinct_range": 0, "num_eq": 318269413938382587, "num_range": 400, "upper_bound": "0.22846195101737976"}, {"distinct_range": 7510381195608619000, "num_eq": 4329379104429284712, "num_range": 7510381195608618932, "upper_bound": "1.5561668872833252"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 0, "upper_bound": "1.909018635749817"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 3321167507102283624, "row_count": 5199628374650278054}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2858653224794659774, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5199628374650278054}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2926292250638897296, "histo_col_type": "", "name": "__auto__", "null_count": 1126148562158090317, "row_count": 5199628374650278054}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3743378145890368588, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "-21 years -3 mons -606 days -19:33:39.703507"}, {"distinct_range": 400, "num_eq": 80, "num_range": 400, "upper_bound": "32 years 820 days 08:07:25.460106"}, {"distinct_range": 400000000, "num_eq": 800000000, "num_range": 400000000, "upper_bound": "46 years 2 mons 147 days 10:21:25.822362"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 5199628374650278054}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2125097173369670002, "histo_buckets": [{"distinct_range": 0, "num_eq": 4536596720535466297, "num_range": 0, "upper_bound": "\\\\x560472ec"}, {"distinct_range": 3571303579309428000, "num_eq": 8729607126078628685, "num_range": 3571303579309428112, "upper_bound": "\\\\x73170ad6"}, {"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "\\\\xda0ebdd0"}, {"distinct_range": 1566453204192398000, "num_eq": 40000, "num_range": 3359953608694962586, "upper_bound": "\\\\xe29883"}, {"distinct_range": 0, "num_eq": 2102716452193664585, "num_range": 7733901634521322132, "upper_bound": "\\\\xf3cf02c1eb00a7"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 5199628374650278054}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2182926211716635728, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000, "num_range": 0, "upper_bound": "-6933678914393572793"}, {"distinct_range": 0, "num_eq": 40000000000, "num_range": 500000, "upper_bound": "-4137696948812116949"}, {"distinct_range": 0, "num_eq": 1224387302429206869, "num_range": 4853240458479118244, "upper_bound": "-718306023057935152"}, {"distinct_range": 2540237732.6871767, "num_eq": 400, "num_range": 5000000000, "upper_bound": "2907389234408758432"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 5199628374650278054}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1332669008943452277, "histo_col_type": "", "name": "__auto__", "null_count": 5037044998341801366, "row_count": 5199628374650278054}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2112473813313308044, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5199628374650278054}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3768942300789952948, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5199628374650278054}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('od', 'wtgo');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('ycmhl', 'scesva');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('qn');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('lj', 'p');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3105844698189257312, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5210799177531854025}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2052785725486824324, "histo_col_type": "", "name": "__auto__", "null_count": 784296999578187439, "row_count": 5210799177531854025}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5025610404746397292, "histo_col_type": "", "name": "__auto__", "null_count": 2426987385187261253, "row_count": 5210799177531854025}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2381070842600051963, "histo_col_type": "", "name": "__auto__", "null_count": 3116985316265817833, "row_count": 5210799177531854025}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2069539835355419255, "histo_col_type": "", "name": "__auto__", "null_count": 3147139023514529167, "row_count": 5210799177531854025}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 524855357534571297, "histo_col_type": "", "name": "__auto__", "null_count": 281722621942793818, "row_count": 5210799177531854025}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4831988643370887959, "histo_col_type": "", "name": "__auto__", "null_count": 433043971175510831, "row_count": 5210799177531854025}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 303585976422498835, "histo_buckets": [{"distinct_range": 0, "num_eq": 400, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 6545776365324089000, "num_eq": 8612223428004965893, "num_range": 6545776365324089817, "upper_bound": "236300856"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 1179014318621486839, "upper_bound": "878401008"}, {"distinct_range": 1732307927251011800, "num_eq": 80000000000, "num_range": 1732307927251011811, "upper_bound": "1808827871"}, {"distinct_range": 0, "num_eq": 8660180440600715677, "num_range": 9000000, "upper_bound": "2461886612"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "3777584469"}, {"distinct_range": 0, "num_eq": 186650845008029323, "num_range": 800, "upper_bound": "3928002197"}, {"distinct_range": 3000000, "num_eq": 30000000, "num_range": 3000000, "upper_bound": "4037838762"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 5210799177531854025}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3837209359423280284, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5210799177531854025}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1565828100433650748, "histo_col_type": "", "name": "__auto__", "null_count": 1862928352668608239, "row_count": 5210799177531854025}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3462385298020905278, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5210799177531854025}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4510482671692550020, "histo_col_type": "", "name": "__auto__", "null_count": 3498714792096086217, "row_count": 5210799177531854025}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27623436308428316, "histo_col_type": "", "name": "__auto__", "null_count": 1835411345817759114, "row_count": 5210799177531854025}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3182337206137042336, "histo_buckets": [{"distinct_range": 0, "num_eq": 40, "num_range": 0, "upper_bound": "K\\u0008M\\u0003,Nu\\u0011"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 5210799177531854025}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2871468642777014058, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5210799177531854025}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29, "histo_col_type": "", "name": "__auto__", "null_count": 21, "row_count": 50}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 49, "histo_col_type": "", "name": "__auto__", "null_count": 34, "row_count": 50}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46, "histo_buckets": [{"distinct_range": 0, "num_eq": 3917429160026904108, "num_range": 0, "upper_bound": "160.106.28.162/5"}, {"distinct_range": 43520400382387620, "num_eq": 1210838881280910485, "num_range": 43520400382387620, "upper_bound": "244.209.117.32/5"}, {"distinct_range": 2777524802741401600, "num_eq": 3000000000, "num_range": 2777524802741401596, "upper_bound": "86.163.111.204/19"}, {"distinct_range": 1364588187039300600, "num_eq": 3081557716790379377, "num_range": 2965074655199408982, "upper_bound": "226.22.208.198/19"}, {"distinct_range": 80000, "num_eq": 3316402422590680633, "num_range": 80000, "upper_bound": "b25e:b0d7:58df:5fe5:e8f5:d5d9:5b1b:8d18/35"}, {"distinct_range": 7000000000, "num_eq": 8683834739334170465, "num_range": 7000000000, "upper_bound": "e4c6:65ea:6317:5163:8e40:f328:e6ef:4ddc/50"}, {"distinct_range": 0, "num_eq": 7413730205567588575, "num_range": 616173054237806220, "upper_bound": "bcc8:36e1:7c56:e804:c2f9:7a59:b0ce:3f49/124"}, {"distinct_range": 4165279624649792000, "num_eq": 3000, "num_range": 4165279624649791766, "upper_bound": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 40, "row_count": 50}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34, "histo_col_type": "", "name": "__auto__", "null_count": 36, "row_count": 50}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 31, "histo_col_type": "", "name": "__auto__", "null_count": 14, "row_count": 50}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18, "histo_col_type": "", "name": "__auto__", "null_count": 10, "row_count": 50}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26, "histo_col_type": "", "name": "__auto__", "null_count": 6, "row_count": 50}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14, "histo_col_type": "", "name": "__auto__", "null_count": 45, "row_count": 50}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 47, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1687065469"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 2144506021042626046, "upper_bound": "2933560310"}, {"distinct_range": 347263647854186900, "num_eq": 3884645023552614417, "num_range": 433423142178448114, "upper_bound": "3898017228"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 41, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34, "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "907002861"}, {"distinct_range": 493809.54088524455, "num_eq": 895312170646976413, "num_range": 1000000, "upper_bound": "1212830665"}, {"distinct_range": 0, "num_eq": 9169307801085413332, "num_range": 7637674365235846659, "upper_bound": "1375781025"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 47, "histo_col_type": "", "name": "__auto__", "null_count": 11, "row_count": 50}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 228104, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 272659, "histo_col_type": "", "name": "__auto__", "null_count": 607828, "row_count": 700000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 199010, "histo_col_type": "", "name": "__auto__", "null_count": 545612, "row_count": 700000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 122978, "histo_col_type": "", "name": "__auto__", "null_count": 415963, "row_count": 700000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19272, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 295808, "histo_col_type": "", "name": "__auto__", "null_count": 253556, "row_count": 700000}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 359155, "histo_col_type": "", "name": "__auto__", "null_count": 73386, "row_count": 700000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 354407, "histo_col_type": "", "name": "__auto__", "null_count": 304073, "row_count": 700000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 364671, "histo_col_type": "", "name": "__auto__", "null_count": 245500, "row_count": 700000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 403183, "histo_col_type": "", "name": "__auto__", "null_count": 34938, "row_count": 700000}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 652458, "histo_col_type": "", "name": "__auto__", "null_count": 342129, "row_count": 700000}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 345285, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 6026707592557398000, "num_eq": 7663606455560514520, "num_range": 6026707592557398078, "upper_bound": "351159044"}, {"distinct_range": 1446254945092297200, "num_eq": 7580097202282775762, "num_range": 3226991563528893807, "upper_bound": "875257229"}, {"distinct_range": 2619177925612460500, "num_eq": 90000, "num_range": 2619177925612460512, "upper_bound": "886869728"}, {"distinct_range": 700371086431750300, "num_eq": 5000, "num_range": 1225821942833881414, "upper_bound": "1310788416"}, {"distinct_range": 800000, "num_eq": 8000, "num_range": 800000, "upper_bound": "1933025044"}, {"distinct_range": 0, "num_eq": 7676726870155734510, "num_range": 900000000, "upper_bound": "3270054035"}, {"distinct_range": 0, "num_eq": 7022354399346596716, "num_range": 6992603168693949327, "upper_bound": "3858746940"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 52900, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "1959734652"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 358173, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 184874, "histo_col_type": "", "name": "__auto__", "null_count": 582432, "row_count": 700000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 218009, "histo_col_type": "", "name": "__auto__", "null_count": 541860, "row_count": 700000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 114675, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 424937, "histo_col_type": "", "name": "__auto__", "null_count": 4377, "row_count": 700000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 79936, "histo_col_type": "", "name": "__auto__", "null_count": 266128, "row_count": 700000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "-4300175675840645749"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 80000000, "upper_bound": "6311355126670962916"}, {"distinct_range": 3221655232902710000, "num_eq": 1825465387706285893, "num_range": 3221655232902709555, "upper_bound": "7844918945890700468"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 5191227085302201781, "num_range": 0, "upper_bound": "-1.800526738166809"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 0, "upper_bound": "-1.4348584413528442"}, {"distinct_range": 1000000, "num_eq": 7766174369418226376, "num_range": 1000000, "upper_bound": "-1.2732161283493042"}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 7782363801169605701, "upper_bound": "-0.4371127486228943"}, {"distinct_range": 0, "num_eq": 7413609950510324286, "num_range": 9221476367997660399, "upper_bound": "-0.19088159501552582"}, {"distinct_range": 0, "num_eq": 0, "num_range": 4706876985833377179, "upper_bound": "0.05055299028754234"}, {"distinct_range": 7376861956021766000, "num_eq": 0, "num_range": 7376861956021765746, "upper_bound": "0.5553134083747864"}, {"distinct_range": 0, "num_eq": 60, "num_range": 4531659836346184706, "upper_bound": "0.5761203169822693"}, {"distinct_range": 0, "num_eq": 20000000, "num_range": 70, "upper_bound": "1.4382203817367554"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 7189641831151523998, "num_range": 0, "upper_bound": "\\u000f"}, {"distinct_range": 139425107468411650, "num_eq": 1000000, "num_range": 3306511791898268628, "upper_bound": "\\u001e"}, {"distinct_range": 693008797135485700, "num_eq": 90000000000, "num_range": 4101894741101822774, "upper_bound": "("}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 20, "upper_bound": ","}, {"distinct_range": 579944.1461018992, "num_eq": 3819667481550828968, "num_range": 1000000, "upper_bound": "2"}, {"distinct_range": 3790975792290349000, "num_eq": 2433044522317061686, "num_range": 3790975792290348914, "upper_bound": "6"}, {"distinct_range": 0, "num_eq": 50000000, "num_range": 7000, "upper_bound": "B"}, {"distinct_range": 2972870437768804400, "num_eq": 965668007750480321, "num_range": 2972870437768804316, "upper_bound": "^"}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 400, "upper_bound": "q"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "\\\\x31"}, {"distinct_range": 7.963245767691492, "num_eq": 90000, "num_range": 10, "upper_bound": "\\\\x58"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000, "num_range": 0, "upper_bound": "-70 years -6 mons -521 days -08:34:24.451758"}, {"distinct_range": 68279.9078051028, "num_eq": 50, "num_range": 600000, "upper_bound": "-59 years -8 mons -803 days -21:02:03.870465"}, {"distinct_range": 5000000, "num_eq": 700000, "num_range": 5000000, "upper_bound": "-56 years -5 mons -585 days -08:50:30.14414"}, {"distinct_range": 0, "num_eq": 20, "num_range": 4000, "upper_bound": "-15 years -9 mons -533 days -12:59:35.067513"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 3078167126550242017, "upper_bound": "-7 years -10 mons -96 days -23:30:59.078506"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 10, "upper_bound": "1 mon 1 day"}, {"distinct_range": 3114503033088425500, "num_eq": 3359781873147761877, "num_range": 6093163123775594901, "upper_bound": "40 years 2 mons 797 days 08:03:39.898451"}, {"distinct_range": 90700.6919737249, "num_eq": 8254388348214622449, "num_range": 5000000, "upper_bound": "68 years 9 mons 666 days 11:39:59.767677"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_7) REFERENCES table3 (col3_13) ON DELETE CASCADE;
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2656389, "histo_col_type": "", "name": "__auto__", "null_count": 2247296, "row_count": 7000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 386459, "histo_buckets": [{"distinct_range": 0, "num_eq": 4123103456628929094, "num_range": 0, "upper_bound": "883342996"}, {"distinct_range": 3103709151762958300, "num_eq": 800000, "num_range": 3103709151762958289, "upper_bound": "1841020764"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 104288110897131280, "upper_bound": "2895022305"}, {"distinct_range": 2112.7459530971764, "num_eq": 580398850124513662, "num_range": 100000, "upper_bound": "3232606739"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 232021, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 499207, "histo_col_type": "", "name": "__auto__", "null_count": 2343812, "row_count": 7000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1264663, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3739498, "histo_col_type": "", "name": "__auto__", "null_count": 4317947, "row_count": 7000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6854588, "histo_col_type": "", "name": "__auto__", "null_count": 2481279, "row_count": 7000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1944265, "histo_col_type": "", "name": "__auto__", "null_count": 5111056, "row_count": 7000000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4443815, "histo_col_type": "", "name": "__auto__", "null_count": 5014211, "row_count": 7000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1802297, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1802059, "histo_col_type": "", "name": "__auto__", "null_count": 2987612, "row_count": 7000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1737513, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 181328, "histo_col_type": "", "name": "__auto__", "null_count": 5269420, "row_count": 7000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4904557, "histo_col_type": "", "name": "__auto__", "null_count": 488154, "row_count": 7000000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67938, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "\\u000cC;\\u0004"}, {"distinct_range": 9091590506753330000, "num_eq": 844766662799790499, "num_range": 9091590506753330056, "upper_bound": "\\u000e[)\\u000b\\"ZZ\\u0005\\u0010"}, {"distinct_range": 0, "num_eq": 90000, "num_range": 1450623657369868825, "upper_bound": "@9"}, {"distinct_range": 913852170110507800, "num_eq": 8921192559492211164, "num_range": 913852170110507752, "upper_bound": "Wh\\u0005\\u0013)L-T "}, {"distinct_range": 0, "num_eq": 200000000, "num_range": 5892124824012194198, "upper_bound": "cf\\u001a\\u001d]\\u0008 |Z"}, {"distinct_range": 2446970.3713247855, "num_eq": 0, "num_range": 20000000, "upper_bound": "g"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 7000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2262913895, "histo_col_type": "", "name": "__auto__", "null_count": 788995001, "row_count": 6000000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 618680966, "histo_col_type": "", "name": "__auto__", "null_count": 5798730877, "row_count": 6000000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4528992449, "histo_col_type": "", "name": "__auto__", "null_count": 3459057817, "row_count": 6000000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3571380342, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4809738915, "histo_buckets": [{"distinct_range": 0, "num_eq": 1879771782562152978, "num_range": 0, "upper_bound": "146093408"}, {"distinct_range": 3778507858511263000, "num_eq": 400, "num_range": 4786697231338647741, "upper_bound": "227964229"}, {"distinct_range": 3000000000, "num_eq": 1460135118386265113, "num_range": 3000000000, "upper_bound": "437839566"}, {"distinct_range": 0, "num_eq": 700, "num_range": 2860703894933453155, "upper_bound": "2877272769"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 6000000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4886417907, "histo_col_type": "", "name": "__auto__", "null_count": 4813203274, "row_count": 6000000000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5257851130, "histo_buckets": [{"distinct_range": 0, "num_eq": 3780728054013309592, "num_range": 0, "upper_bound": "101.24.172.35/5"}, {"distinct_range": 4536361036214279000, "num_eq": 1745438348958074456, "num_range": 4536361036214279148, "upper_bound": "174.127.161.99/5"}, {"distinct_range": 6787968120634250000, "num_eq": 200000, "num_range": 6787968120634249982, "upper_bound": "88.154.102.98/12"}, {"distinct_range": 0, "num_eq": 345412384011578197, "num_range": 80000000, "upper_bound": "f078:bb34:c198:b98e:7ddc:4d05:a256:b6a/5"}, {"distinct_range": 4454.569711103344, "num_eq": 900000000, "num_range": 9000, "upper_bound": "b901:a361:91db:b419:6c6b:fb31:b7c3:7e58/12"}, {"distinct_range": 16.663922883004826, "num_eq": 80, "num_range": 800, "upper_bound": "7c4c:4ae9:49:bcc9:f749:9589:5c78:3ffe/91"}, {"distinct_range": 1535798177732588000, "num_eq": 100000000, "num_range": 3295450010984182787, "upper_bound": "507:9229:9ed5:568e:f9af:9ff:e089:1bce/102"}, {"distinct_range": 0, "num_eq": 70, "num_range": 1000, "upper_bound": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 5245344734, "row_count": 6000000000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4554515418, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1437033768, "histo_buckets": [{"distinct_range": 0, "num_eq": 60, "num_range": 0, "upper_bound": "109896694"}, {"distinct_range": 0, "num_eq": 1387939743712978943, "num_range": 700000, "upper_bound": "136090738"}, {"distinct_range": 3000000000, "num_eq": 4603820733826638467, "num_range": 3000000000, "upper_bound": "340072793"}, {"distinct_range": 633282086425310600, "num_eq": 6172397043542388473, "num_range": 1512037716992040817, "upper_bound": "420297050"}, {"distinct_range": 3979202949028119600, "num_eq": 400, "num_range": 3979202949028119577, "upper_bound": "581247694"}, {"distinct_range": 0, "num_eq": 1209033941513777327, "num_range": 4384582945079902882, "upper_bound": "3489850836"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 6000000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 972365060, "histo_col_type": "", "name": "__auto__", "null_count": 1266130369, "row_count": 6000000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3085278289, "histo_col_type": "", "name": "__auto__", "null_count": 3589194387, "row_count": 6000000000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3293604098, "histo_col_type": "", "name": "__auto__", "null_count": 3868815891, "row_count": 6000000000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5638321632, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 285851589, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1405561913, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000000, "num_range": 0, "upper_bound": "2105993303"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 6000000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5681361804, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3718762701, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4101146296, "histo_col_type": "", "name": "__auto__", "null_count": 590836537, "row_count": 6000000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3045384441069141974, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "00000000-0000-0000-0000-000000000000"}, {"distinct_range": 1076897215532515100, "num_eq": 8000000000, "num_range": 3801277233886133133, "upper_bound": "475064f1-d189-4642-82ba-7fb6c89c6bd4"}, {"distinct_range": 2212518541697542000, "num_eq": 80, "num_range": 2212518541697541898, "upper_bound": "581f2ed8-3d0e-44d3-8e44-28126874ff5a"}, {"distinct_range": 0, "num_eq": 30, "num_range": 8000000000, "upper_bound": "6e5ede61-47b5-409d-8d29-428babc0e993"}, {"distinct_range": 214475732184254700, "num_eq": 700, "num_range": 2681581744162904262, "upper_bound": "6fc64af9-197b-459f-88a9-153279b12432"}, {"distinct_range": 70000, "num_eq": 0, "num_range": 70000, "upper_bound": "7410d951-4bc6-4974-a5ac-061446a50f89"}, {"distinct_range": 5402335893784356000, "num_eq": 400, "num_range": 8617198378518160143, "upper_bound": "aac950d1-12d2-4c33-8c3a-357bef91772c"}, {"distinct_range"

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2511953&tab=buildLog) on [master@5d9026c85bb9b762d7d02e2710ec36c31c641a81](https://github.com/cockroachdb/cockroach/commits/5d9026c85bb9b762d7d02e2710ec36c31c641a81):

```
        cockroach2_1  | webui:               http://cockroach2:8080
        cockroach2_1  | sql:                 postgresql://root@cockroach2:26257?sslmode=disable
        cockroach2_1  | RPC client flags:    /cockroach/cockroach <client cmd> --host=cockroach2:26257 --insecure
        cockroach2_1  | logs:                /cockroach-data/logs
        cockroach2_1  | temp dir:            /cockroach-data/cockroach-temp217673188
        cockroach2_1  | external I/O path:   /cockroach-data/extern
        cockroach2_1  | store[0]:            path=/cockroach-data
        cockroach2_1  | storage engine:      pebble
        cockroach2_1  | status:              initialized new cluster
        cockroach2_1  | clusterID:           5e8ab714-489b-4b6d-b362-b216cc46845e
        cockroach2_1  | nodeID:              1
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | 2020-12-11 06:13:56.102 UTC [48] LOG:  received fast shutdown request
        postgres_1    | waiting for server to shut down....2020-12-11 06:13:56.105 UTC [48] LOG:  aborting any active transactions
        postgres_1    | 2020-12-11 06:13:56.106 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2020-12-11 06:13:56.112 UTC [50] LOG:  shutting down
        postgres_1    | 2020-12-11 06:13:56.232 UTC [48] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2020-12-11 06:13:56.313 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2020-12-11 06:13:56.313 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2020-12-11 06:13:56.318 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2020-12-11 06:13:56.332 UTC [85] LOG:  database system was shut down at 2020-12-11 06:13:56 UTC
        postgres_1    | 2020-12-11 06:13:56.337 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (601.46s)
        test_1        |     --- FAIL: TestCompare/postgres (0.81s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2515124&tab=buildLog) on [master@960b4cfc54c8b78df56e62f07d7a07b986ceacff](https://github.com/cockroachdb/cockroach/commits/960b4cfc54c8b78df56e62f07d7a07b986ceacff):

```
        postgres_1    | 	CREATE INDEX ON table5 (col5_1 ASC, col5_6) INCLUDE (col5_0, col5_2, col5_7, col5_8, col5_10, col5_11, col5_13, col5_14, col5_17);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('u', 'akh', 'nv', 'il');
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('xcwuh', 'le', 'oaotz');
        postgres_1    | 	CREATE TYPE rand_typ_2 AS ENUM ('u', 'hbcpln');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (605.14s)
        test_1        |     --- FAIL: TestCompare/postgres (3.71s)
        test_1        |         compare_test.go:129: CREATE TABLE table1 (col1_0 FLOAT8 NOT NULL, col1_1 DECIMAL, col1_2 BOX2D NOT NULL, col1_3 REGPROC NOT NULL, col1_4 INT8 NOT NULL, col1_5 REGTYPE NOT NULL, col1_6 VARCHAR NOT NULL, col1_7 UUID NOT NULL, col1_8 REGTYPE NULL, col1_9 FLOAT8 NOT NULL, col1_10 TEXT, col1_11 BYTEA, PRIMARY KEY (col1_3, col1_5, col1_0, col1_4, col1_9));
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_6 ASC, col1_1, col1_2, col1_10 ASC, col1_7 ASC, col1_3, col1_5 ASC, col1_9 ASC, col1_0 ASC, col1_8, col1_11 ASC, col1_4 ASC);
        test_1        |             CREATE INDEX ON table1 (col1_5 DESC, col1_3 ASC, col1_8 ASC, col1_1 ASC, col1_2 ASC, col1_9 DESC, col1_10 ASC, col1_11, col1_0, col1_7 ASC, col1_4 DESC) INCLUDE (col1_6);
        test_1        |             CREATE INDEX ON table1 (col1_11, col1_2 DESC, col1_0 DESC, col1_10 DESC, col1_5) INCLUDE (col1_6, col1_7);
        test_1        |             CREATE INDEX ON table1 (col1_1 ASC, col1_8);
        test_1        |             CREATE INDEX ON table1 (col1_4 DESC, col1_6 ASC, col1_0, col1_2 DESC, col1_9);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_11, col1_5 DESC, col1_3 ASC, col1_1 ASC, col1_0 DESC, col1_10 DESC, col1_6 DESC, col1_4 DESC, col1_9, col1_8 ASC, col1_2) INCLUDE (col1_7);
        test_1        |             CREATE INDEX ON table1 (col1_2, col1_7 DESC, col1_6);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_1 DESC, col1_10 DESC, col1_0 DESC, col1_9, col1_6 ASC, col1_7 ASC, col1_8 DESC);
        test_1        |             CREATE INDEX ON table1 (col1_7 DESC, col1_8 DESC, col1_5, col1_9 DESC, col1_6 DESC, col1_4 ASC);
        test_1        |             CREATE TABLE table2 (col1_3 REGPROC NOT NULL, col1_5 REGTYPE NOT NULL, col1_0 FLOAT8 NOT NULL, col1_4 INT8 NOT NULL, col1_9 FLOAT8 NOT NULL, col2_5 BYTEA NULL, col2_6 REGPROC, col2_7 FLOAT8 NOT NULL, col2_8 REGCLASS, col2_9 TIMESTAMP, col2_10 "char" NOT NULL, col2_11 CHAR, col2_12 DATE, col2_13 DATE NULL, col2_14 TIMESTAMP, col2_15 INET, col2_16 REGNAMESPACE NOT NULL, col2_17 BIT(2), col2_18 REGPROC NOT NULL, col2_19 INET NULL, col2_20 REGPROCEDURE, PRIMARY KEY (col1_3, col1_5, col1_0, col1_4, col1_9, col2_16, col2_17, col2_18, col2_5, col2_11, col2_13, col2_15, col2_10, col2_12, col2_9, col2_14, col2_20));
        test_1        |             CREATE INDEX ON table2 (col2_10 DESC, col2_8 ASC, col2_17 DESC, col1_0 ASC, col2_6, col1_9 ASC, col2_7 DESC, col2_18 DESC, col2_16 DESC, col2_9, col2_14 DESC, col1_4 ASC, col2_20 DESC, col2_19);
        test_1        |             CREATE INDEX ON table2 (col2_13 DESC, col1_3 ASC, col1_0 ASC, col2_12, col2_15, col2_11, col2_5, col2_14 DESC, col2_10 ASC, col2_18 ASC, col1_9, col2_19);
        test_1        |             CREATE INDEX ON table2 (col2_13 ASC, col2_19 DESC, col1_0 DESC, col2_16 ASC, col2_20 ASC, col1_9 ASC, col2_5, col2_17 ASC) INCLUDE (col2_7);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_7 ASC, col2_8 DESC, col2_5 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_7);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_19 ASC, col1_0, col1_5 ASC, col2_16, col1_9, col2_15 DESC, col2_13, col2_10 ASC, col2_8 ASC, col1_4, col2_11 DESC, col2_9 DESC, col2_12, col2_20 ASC, col2_18 DESC, col2_14, col1_3 DESC, col2_6 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_3 DESC, col2_19, col1_4 ASC, col2_17 ASC, col2_20 ASC, col2_10, col2_18, col2_6 ASC);
        test_1        |             CREATE INDEX ON table2 (col2_11 ASC, col2_10 ASC, col2_16 DESC, col2_9 ASC, col1_4 DESC, col2_13 ASC, col1_0, col2_8, col2_7, col1_9 DESC, col2_5 DESC, col1_3 ASC, col2_12, col2_15, col2_19 ASC, col2_18 ASC, col2_17 DESC, col1_5 DESC, col2_20);
        test_1        |             CREATE INDEX ON table2 (col2_11, col1_9 ASC, col2_6 ASC) INCLUDE (col2_19);
        test_1        |             CREATE TABLE table3 (col1_3 REGPROC NOT NULL, col1_5 REGTYPE NOT NULL, col1_0 FLOAT8 NOT NULL, col1_4 INT8 NOT NULL, col1_9 FLOAT8 NOT NULL, col2_16 REGNAMESPACE NOT NULL, col2_17 BIT(2), col2_18 REGPROC NOT NULL, col2_5 BYTEA NULL, col2_11 CHAR, col2_13 DATE NULL, col2_15 INET, col2_10 "char" NOT NULL, col2_12 DATE, col2_9 TIMESTAMP, col2_14 TIMESTAMP, col2_20 REGPROCEDURE, col3_17 TIMESTAMPTZ NOT NULL, PRIMARY KEY (col1_3, col1_5, col1_0, col1_4, col1_9, col2_16, col2_17, col2_18, col2_5, col2_11, col2_13, col2_15, col2_10, col2_12, col2_9, col2_14, col2_20));
        test_1        |             CREATE UNIQUE INDEX ON table3 (col2_11, col2_17);
        test_1        |             CREATE INDEX ON table3 (col1_9 ASC) INCLUDE (col3_17);
        test_1        |             CREATE TABLE table4 (col4_0 "char" NOT NULL, col4_1 OID, col4_2 INTERVAL NULL, col4_3 DATE NOT NULL, col4_4 TEXT NULL, col4_5 INT8 NULL);
        test_1        |             CREATE TABLE table5 (col5_0 REGNAMESPACE NULL, col5_1 DATE NULL, col5_2 REGTYPE NULL, col5_3 INT8 NULL, col5_4 TIME, col5_5 BOX2D NULL, col5_6 TIME, col5_7 TIME, col5_8 NAME, col5_9 VARCHAR NOT NULL, col5_10 "char" NULL, col5_11 VARCHAR NULL, col5_12 TIMESTAMP NULL, col5_13 BOOL NULL, col5_14 BOX2D NOT NULL, col5_15 DECIMAL NOT NULL, col5_16 TIMESTAMPTZ NULL, col5_17 REGNAMESPACE);
        test_1        |             CREATE INDEX ON table5 (col5_12 ASC, col5_15) INCLUDE (col5_5, col5_6, col5_10, col5_14, col5_17);
        test_1        |             CREATE INDEX ON table5 (col5_1 ASC, col5_6) INCLUDE (col5_0, col5_2, col5_7, col5_8, col5_10, col5_11, col5_13, col5_14, col5_17);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('u', 'akh', 'nv', 'il');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('xcwuh', 'le', 'oaotz');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('u', 'hbcpln');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2516759&tab=buildLog) on [master@53b08837cf5e76504e437bad80c97e05989d2c60](https://github.com/cockroachdb/cockroach/commits/53b08837cf5e76504e437bad80c97e05989d2c60):

```
        postgres_1    | You can change this by editing pg_hba.conf or using the option -A, or
        postgres_1    | --auth-local and --auth-host, the next time you run initdb.
        postgres_1    | waiting for server to start....2020-12-14 06:14:39.209 UTC [49] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2020-12-14 06:14:39.224 UTC [50] LOG:  database system was shut down at 2020-12-14 06:14:39 UTC
        postgres_1    | 2020-12-14 06:14:39.229 UTC [49] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | 2020-12-14 06:14:42.086 UTC [49] LOG:  received fast shutdown request
        postgres_1    | waiting for server to shut down....2020-12-14 06:14:42.088 UTC [49] LOG:  aborting any active transactions
        postgres_1    | 2020-12-14 06:14:42.090 UTC [49] LOG:  background worker "logical replication launcher" (PID 56) exited with exit code 1
        postgres_1    | 2020-12-14 06:14:42.096 UTC [51] LOG:  shutting down
        postgres_1    | 2020-12-14 06:14:42.217 UTC [49] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2020-12-14 06:14:42.297 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2020-12-14 06:14:42.298 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2020-12-14 06:14:42.304 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2020-12-14 06:14:42.317 UTC [86] LOG:  database system was shut down at 2020-12-14 06:14:42 UTC
        postgres_1    | 2020-12-14 06:14:42.322 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (602.68s)
        test_1        |     --- FAIL: TestCompare/postgres (1.47s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.2:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2522962&tab=buildLog) on [master@de91557c0634ab3797356d445ba39e37d45d8205](https://github.com/cockroachdb/cockroach/commits/de91557c0634ab3797356d445ba39e37d45d8205):

```
        postgres_1    | 2020-12-16 06:13:43.772 UTC [49] LOG:  database system was shut down at 2020-12-16 06:13:43 UTC
        postgres_1    | 2020-12-16 06:13:43.777 UTC [48] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        test_1        | --- FAIL: TestCompare (2.39s)
        test_1        |     --- FAIL: TestCompare/postgres (1.66s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.2:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.74s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 OID NOT NULL, col1_1 TIMESTAMP, col1_2 INT4 NULL, col1_3 "char" NOT NULL, col1_4 "char" NULL, col1_5 VARCHAR, col1_6 INT4, col1_7 DECIMAL NOT NULL, col1_8 NAME NOT NULL, col1_9 TIMESTAMPTZ, col1_10 DATE NOT NULL, col1_11 REGPROCEDURE NOT NULL, col1_12 BIT(16) NULL, col1_13 BIT(24) NULL, PRIMARY KEY (col1_11 ASC, col1_7 DESC, col1_8, col1_10, col1_0 ASC), FAMILY (col1_13), FAMILY (col1_11), FAMILY (col1_4), FAMILY (col1_6, col1_0, col1_5, col1_2, col1_1), FAMILY (col1_10, col1_9, col1_12), FAMILY (col1_8, col1_7, col1_3));
        test_1        |             CREATE TABLE table2 (col1_11 REGPROCEDURE NOT NULL, col1_7 DECIMAL NOT NULL, col1_8 NAME NOT NULL, col1_10 DATE NOT NULL, col1_0 OID NOT NULL, col2_5 REGNAMESPACE NOT NULL, col2_6 BYTES NOT NULL, col2_7 REGPROCEDURE, col2_8 DATE NULL, col2_9 BIT(49) NULL, col2_10 BYTES NULL, col2_11 TIMESTAMP NOT NULL, col2_12 DECIMAL NOT NULL, col2_13 DATE NULL, col2_14 UUID NULL, col2_15 FLOAT8 NOT NULL, col2_16 BIT(28), col2_17 STRING NULL, col2_18 TIMESTAMP NULL, col2_19 INT8 NOT NULL, col2_20 DATE NOT NULL, col2_21 REGTYPE NULL, col2_22 TIMETZ, PRIMARY KEY (col1_11 ASC, col1_7 DESC, col1_8, col1_10, col1_0 ASC, col2_19 DESC, col2_13 DESC, col2_21 ASC), INDEX (col1_7 ASC) WHERE (((table2.col2_15 < 1.0:::FLOAT8) AND (table2.col2_18 = '-4713-11-24 00:00:00':::TIMESTAMP)) AND (table2.col1_10 < '-infinity':::DATE)) OR (table2.col2_12 >= (-1.234E+401):::DECIMAL), INDEX (col2_15 ASC, col2_16, col1_8 DESC, col1_7, col2_10 ASC, col2_12 ASC, col2_11 ASC, col2_5, col2_6 ASC, col2_18 DESC, col1_10, col1_11 DESC, col2_9 DESC, col2_21, col2_7, col2_20, col1_0 ASC, col2_13, col2_14 DESC, col2_8, col2_22 DESC, col2_17 DESC, col2_19 DESC) WHERE (((table2.col2_8 <= 'infinity') AND (table2.col2_17 > e'\x00')) OR (table2.col1_7 >= -Infinity)) OR (table2.col2_22 >= '24:00:00-15:59:00'), INDEX (col2_16 DESC, col2_6) STORING (col2_8, col2_12, col2_14, col2_17, col2_18, col2_22) WHERE ((table2.col2_13 > '1970-01-01':::DATE) OR (table2.col2_10 >= e'\\x27':::BYTES)) OR (table2.col2_19 != (-9223372036854775807):::INT8), UNIQUE (col2_17 DESC, col2_21, col2_11 ASC, col2_6 ASC, col2_15 ASC, col2_16 DESC) STORING (col2_8, col2_9, col2_14, col2_18, col2_20, col2_22) WHERE ((((((((((((((table2.col2_8 != '1970-01-01':::DATE) AND (table2.col2_6 <= e'\\x00':::BYTES)) OR (table2.col2_19 != 127:::INT8)) AND (table2.col2_11 > '0001-01-01 00:00:00':::TIMESTAMP)) OR (table2.col1_7 != (-1.234E+401):::DECIMAL)) AND (table2.col2_17 = '"':::STRING)) OR (table2.col1_10 != '1970-01-01':::DATE)) OR (table2.col2_20 <= '4714-11-24 BC':::DATE)) AND (table2.col2_12 <= 'Infinity':::DECIMAL)) OR (table2.col1_8 >= e'\U00002603':::STRING)) AND (table2.col2_18 < '-2000-01-01 00:00:00':::TIMESTAMP)) OR (table2.col2_10 <= e'\\x22':::BYTES)) AND (table2.col2_13 > '4714-11-24 BC':::DATE)) OR (table2.col2_22 = '24:00:00-15:59:00':::TIMETZ)) OR (table2.col2_15 < 0.0:::FLOAT8), INDEX (col2_5 ASC, col2_15 ASC, col2_17 DESC, col2_9 DESC, col2_7 ASC, col1_8 ASC, col2_19, col1_10 ASC, col2_6 DESC, col1_0, col2_18, col2_14, col2_12 DESC, col2_10, col2_11) STORING (col2_8, col2_22) WHERE ((((((((((((((table2.col2_11 = '-4713-11-24 00:00:00':::TIMESTAMP) AND (table2.col2_15 != '-Inf':::FLOAT8)) AND (table2.col1_7 != 'NaN':::DECIMAL)) OR (table2.col2_6 <= e'\\x58':::BYTES)) AND (table2.col1_8 < e'\U00002603':::STRING)) AND (table2.col2_8 <= '1970-01-01':::DATE)) AND (table2.col2_12 < 'NaN':::DECIMAL)) AND (table2.col2_17 != e'\x00':::STRING)) OR (table2.col1_10 != '4714-11-24 BC':::DATE)) OR (table2.col2_22 > '24:00:00-15:59:00':::TIMETZ)) AND (table2.col2_13 > '5874897-12-31':::DATE)) AND (table2.col2_20 <= '-infinity':::DATE)) OR (table2.col2_10 >= e'\\x00':::BYTES)) OR (table2.col2_18 < '-2000-01-01 00:00:00':::TIMESTAMP)) OR (table2.col2_19 != (-9223372036854775807):::INT8), UNIQUE (col1_10 DESC, col2_12 DESC, col2_20 ASC, col2_5, col1_8, col2_11 DESC, col2_9 DESC, col2_15 DESC, col2_13 DESC, col1_0 DESC) STORING (col2_8, col2_14, col2_16, col2_18, col2_22) WHERE table2.col1_7 <= 0:::DECIMAL, FAMILY (col2_11, col2_12, col2_20), FAMILY (col2_6, col2_17), FAMILY (col2_9), FAMILY (col2_18, col2_19, col2_13, col2_14), FAMILY (col1_0, col1_7), FAMILY (col1_10), FAMILY (col2_5), FAMILY (col1_11), FAMILY (col2_15, col2_16, col2_22), FAMILY (col2_10), FAMILY (col2_7, col1_8, col2_21, col2_8)) INTERLEAVE IN PARENT table1 (col1_11, col1_7, col1_8, col1_10, col1_0);
        test_1        |             CREATE TABLE table3 (col1_11 REGPROCEDURE NOT NULL, col1_7 DECIMAL NOT NULL, col1_8 NAME NOT NULL, col1_10 DATE NOT NULL, col1_0 OID NOT NULL, col3_5 TIMESTAMP NULL, PRIMARY KEY (col1_11 ASC, col1_7 DESC, col1_8, col1_10, col1_0 ASC), UNIQUE (col1_10 ASC, col1_7, col1_0, col1_8, col3_5, col1_11 DESC) WHERE (((table3.col1_8 != e'\'') AND (table3.col1_7 <= 0)) OR (table3.col1_10 = '1970-01-01')) OR (table3.col3_5 <= '294276-12-31 23:59:59.999999'), UNIQUE (col1_11 DESC, col1_10 ASC, col1_8, col1_0, col3_5 DESC, col1_7 DESC) WHERE table3.col1_7 < 1:::DECIMAL, UNIQUE (col1_10, col1_0, col1_8 ASC), UNIQUE (col1_8 ASC, col1_0) WHERE (((table3.col3_5 = '0001-01-01 00:00:00') AND (table3.col1_7 >= NaN)) AND (table3.col1_8 > '')) AND (table3.col1_10 = '-infinity'), UNIQUE (col1_11 ASC, col1_10 ASC, col1_0 DESC, col3_5 ASC) WHERE (((table3.col3_5 >= '-2000-01-01 00:00:00') AND (table3.col1_8 <= '"')) OR (table3.col1_7 > -1)) OR (table3.col1_10 >= 'infinity'), INDEX (col1_0 DESC, col1_11, col1_8, col1_10 ASC, col3_5) WHERE (table3.col1_7 != '-Infinity':::DECIMAL) OR (table3.col1_8 < '"':::STRING), UNIQUE (col1_8, col1_11 ASC) WHERE (((table3.col1_7 = 0) OR (table3.col3_5 >= '0001-01-01 00:00:00')) OR (table3.col1_10 <= 'infinity')) AND (table3.col1_8 <= ''), FAMILY (col1_10, col1_8, col3_5, col1_11), FAMILY (col1_0, col1_7)) INTERLEAVE IN PARENT table1 (col1_11, col1_7, col1_8, col1_10, col1_0);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 8036459888293242401, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 6917109217884090000, "num_eq": 6000, "num_range": 6917109217884090459, "upper_bound": "453910582"}, {"distinct_range": 0, "num_eq": 727154959791150425, "num_range": 2950241818429107490, "upper_bound": "1037730769"}, {"distinct_range": 10000000, "num_eq": 3461763593405147847, "num_range": 10000000, "upper_bound": "2021128297"}, {"distinct_range": 0, "num_eq": 700878624607314533, "num_range": 7533040708818477843, "upper_bound": "2207093267"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 64, "histo_col_type": "", "name": "__auto__", "null_count": 29, "row_count": 100}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 6997135201592973605, "num_range": 0, "upper_bound": "1110001010101000010001101100"}], "histo_col_type": "BIT(28)", "name": "__auto__", "null_count": 52, "row_count": 100}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 91, "histo_buckets": [{"distinct_range": 0, "num_eq": 2066692957319723849, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 0, "num_range": 8000000, "upper_bound": "\\n\\r\\u0012\\u000e?"}, {"distinct_range": 1968261036.817643, "num_eq": 6335723663294860463, "num_range": 7000000000, "upper_bound": "\\u0019.q"}, {"distinct_range": 50000000000, "num_eq": 400, "num_range": 50000000000, "upper_bound": "\'"}, {"distinct_range": 8060227302509070000, "num_eq": 1985353329394541352, "num_range": 8060227302509070404, "upper_bound": ";\\"\\n\\u000c\\u0019~X"}, {"distinct_range": 8636779839583613000, "num_eq": 500000000, "num_range": 8636779839583613177, "upper_bound": "=6V+\\u0019!"}, {"distinct_range": 0, "num_eq": 5000000000, "num_range": 600000, "upper_bound": "RS.>"}, {"distinct_range": 2168725959412066600, "num_eq": 9197076923214913856, "num_range": 3655223188261661699, "upper_bound": "z{H[\\u001aZ"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 3, "row_count": 100}, {"columns": ["col2_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 59, "histo_col_type": "", "name": "__auto__", "null_count": 42, "row_count": 100}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 61, "histo_buckets": [{"distinct_range": 0, "num_eq": 5976513543747090288, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 1907046075403185200, "num_eq": 40000000000, "num_range": 1907046075403185075, "upper_bound": "744600430"}, {"distinct_range": 0, "num_eq": 2152304620393918985, "num_range": 400000000, "upper_bound": "1166631286"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 51, "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39, "histo_col_type": "", "name": "__auto__", "null_count": 24, "row_count": 100}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19, "histo_col_type": "", "name": "__auto__", "null_count": 29, "row_count": 100}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 90, "histo_col_type": "", "name": "__auto__", "null_count": 99, "row_count": 100}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 77, "histo_col_type": "", "name": "__auto__", "null_count": 75, "row_count": 100}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 73, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 31, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "1973-07-27"}, {"distinct_range": 3074507818878155000, "num_eq": 8000000000, "num_range": 7350114761302582242, "upper_bound": "1977-03-08"}, {"distinct_range": 2032648233457322200, "num_eq": 7365843414150568885, "num_range": 2032648233457322271, "upper_bound": "1977-03-20"}, {"distinct_range": 715828308507341200, "num_eq": 5000, "num_range": 715828308507341190, "upper_bound": "1982-11-17"}, {"distinct_range": 7511401180325010000, "num_eq": 5000000000, "num_range": 7511401180325010755, "upper_bound": "1989-07-04"}, {"distinct_range": 100, "num_eq": 6000, "num_range": 100, "upper_bound": "1990-01-25"}, {"distinct_range": 400, "num_eq": 7000, "num_range": 400, "upper_bound": "1996-11-07"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 82, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18, "histo_col_type": "", "name": "__auto__", "null_count": 80, "row_count": 100}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 70, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 79, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67, "histo_buckets": [{"distinct_range": 0, "num_eq": 4574556078965823835, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 157265116339972930, "num_eq": 40, "num_range": 157265116339972942, "upper_bound": "1127715467"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 1178665919046340630, "upper_bound": "1288828014"}, {"distinct_range": 910477490350365600, "num_eq": 0, "num_range": 5630499297577407346, "upper_bound": "2023142766"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 400, "upper_bound": "3242287654"}, {"distinct_range": 2877439.822699517, "num_eq": 1000000, "num_range": 10000000, "upper_bound": "3720944708"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 68, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 84, "histo_col_type": "", "name": "__auto__", "null_count": 26, "row_count": 100}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 56, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2, "histo_col_type": "", "name": "__auto__", "null_count": 43, "row_count": 100}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 94, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000, "num_range": 0, "upper_bound": "1.0"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 100}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22429, "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20493, "histo_col_type": "", "name": "__auto__", "null_count": 20576, "row_count": 50000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7742, "histo_buckets": [{"distinct_range": 0, "num_eq": 8775254093223116758, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 9000000, "num_eq": 900, "num_range": 9000000, "upper_bound": "584806106"}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 0, "upper_bound": "678211544"}, {"distinct_range": 50000, "num_eq": 8505169656560523606, "num_range": 50000, "upper_bound": "1578312330"}, {"distinct_range": 400000, "num_eq": 7458888096730686014, "num_range": 400000, "upper_bound": "2275521768"}, {"distinct_range": 0, "num_eq": 1923780521579769761, "num_range": 3894422305202222292, "upper_bound": "3226408436"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25994, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27603, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 18820.387479862824, "num_eq": 6773515366670627694, "num_range": 1000000, "upper_bound": "\\u0004=D"}, {"distinct_range": 0, "num_eq": 2000000000, "num_range": 400000000, "upper_bound": "\\""}, {"distinct_range": 7737708251008981000, "num_eq": 4596694025555419258, "num_range": 7737708251008980842, "upper_bound": "J\\u0018Xi\\u000c"}, {"distinct_range": 2399.82282000504, "num_eq": 1785485723840775200, "num_range": 3000, "upper_bound": "\U00002603"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 40369, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000, "num_range": 0, "upper_bound": "-infinity"}, {"distinct_range": 0, "num_eq": 6222116575260822368, "num_range": 2932562551279989542, "upper_bound": "1971-09-21"}, {"distinct_range": 0, "num_eq": 0, "num_range": 300000, "upper_bound": "1980-08-20"}, {"distinct_range": 77114534.47444135, "num_eq": 0, "num_range": 80000000, "upper_bound": "1984-04-24"}, {"distinct_range": 17722027.002710674, "num_eq": 900000, "num_range": 30000000, "upper_bound": "1987-05-06"}, {"distinct_range": 3806129710493515000, "num_eq": 90000, "num_range": 6690078365807290794, "upper_bound": "1994-01-16"}, {"distinct_range": 100, "num_eq": 2221539879822885464, "num_range": 100, "upper_bound": "1994-07-12"}, {"distinct_range": 9811384.386337416, "num_eq": 8101020295446569702, "num_range": 10000000, "upper_bound": "infinity"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 50000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('oimqtk', 'jjnpp', 'qjxgf', 'i', 'wylf', 'dgsfal');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('qxcumx', 'aq');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('akq', 'cg', 'yrebov', 'sxdlj', 'ommeg', 'btpd');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('tryod', 'dt', 'rvk');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('y');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1955487896265159565, "histo_col_type": "", "name": "__auto__", "null_count": 3062650810536020451, "row_count": 5166184528567486064}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2957103579215945354, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5166184528567486064}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4246070317401832973, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5166184528567486064}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2823917273688401836, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5166184528567486064}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4201773785054317650, "histo_col_type": "", "name": "__auto__", "null_count": 3356394156062182294, "row_count": 5166184528567486064}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3029653466531004738, "histo_buckets": [{"distinct_range": 0, "num_eq": 582855521411344029, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 153031221.28978842, "num_eq": 6347246290260350918, "num_range": 5000000000, "upper_bound": "636037008"}, {"distinct_range": 2061102750675015700, "num_eq": 2000, "num_range": 6368613914447477371, "upper_bound": "1343700112"}, {"distinct_range": 3669406906.038599, "num_eq": 4000000, "num_range": 6000000000, "upper_bound": "2042180822"}, {"distinct_range": 90000, "num_eq": 675475137625964806, "num_range": 90000, "upper_bound": "2237505063"}, {"distinct_range": 198.77418993616374, "num_eq": 300000, "num_range": 200, "upper_bound": "2347557381"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 5166184528567486064}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2746745787404898779, "histo_col_type": "", "name": "__auto__", "null_count": 3931161595539650707, "row_count": 5166184528567486064}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 854263942253157661, "histo_col_type": "", "name": "__auto__", "null_count": 2667840721671093984, "row_count": 5166184528567486064}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 205737863288280688, "histo_col_type": "", "name": "__auto__", "null_count": 2634415426127603400, "row_count": 5166184528567486064}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4572098302070725398, "histo_col_type": "", "name": "__auto__", "null_count": 2214623596758582339, "row_count": 5166184528567486064}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4905410892753608159, "histo_col_type": "", "name": "__auto__", "null_count": 4683559144071770098, "row_count": 5166184528567486064}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2068403793099383608, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5166184528567486064}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2464308379653052176, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5166184528567486064}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3174614376616019056, "histo_col_type": "", "name": "__auto__", "null_count": 1812815187408203673, "row_count": 5166184528567486064}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3222333219265715938, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "-4.254298760862531487E+34"}, {"distinct_range": 0, "num_eq": 6974718091446164259, "num_range": 40, "upper_bound": "-1833567195.90121283"}, {"distinct_range": 4115765068858016300, "num_eq": 0, "num_range": 4115765068858016353, "upper_bound": "-0.8224352423773615722"}, {"distinct_range": 0, "num_eq": 8628579138166659820, "num_range": 8407796026361198664, "upper_bound": "113004.803258306527"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 70000000000, "upper_bound": "5.94194138180695364E+31"}, {"distinct_range": 0, "num_eq": 6751682371396381957, "num_range": 60000000, "upper_bound": "Infinity"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 6938749918561325818}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2080957113164491870, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6938749918561325818}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3504411092831367472, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6938749918561325818}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6497844865933898149, "histo_col_type": "", "name": "__auto__", "null_count": 863347753201448986, "row_count": 6938749918561325818}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3262597132978197633, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6938749918561325818}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 915778280761067699, "histo_buckets": [{"distinct_range": 0, "num_eq": 968240271037626193, "num_range": 0, "upper_bound": "0000000000000000000000000000"}, {"distinct_range": 4679285228492117000, "num_eq": 0, "num_range": 8942732008215451522, "upper_bound": "1000000010110001111111001101"}, {"distinct_range": 1878405593450404000, "num_eq": 853439270437848171, "num_range": 1878405593450404043, "upper_bound": "1010000110101111000000010101"}, {"distinct_range": 5579750718012533000, "num_eq": 8560360214637991921, "num_range": 5579750718012532870, "upper_bound": "1110000000100010100100111011"}], "histo_col_type": "BIT(28)", "name": "__auto__", "null_count": 4214943207305400203, "row_count": 6938749918561325818}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1135796730433484177, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000000000, "num_range": 0, "upper_bound": "376624259"}, {"distinct_range": 0, "num_eq": 800, "num_range": 8652721396332701328, "upper_bound": "423240452"}, {"distinct_range": 0, "num_eq": 6980202182139271162, "num_range": 300000000, "upper_bound": "1242477165"}, {"distinct_range": 9035625838454678000, "num_eq": 503279255630103786, "num_range": 9214926295535876728, "upper_bound": "1551900671"}, {"distinct_range": 0, "num_eq": 60000, "num_range": 2000, "upper_bound": "3027628281"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 6938749918561325818}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3333351414295919002, "histo_col_type": "", "name": "__auto__", "null_count": 6461062366122246349, "row_count": 6938749918561325818}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4751327406307002844, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6938749918561325818}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1365832246500232197, "histo_col_type": "", "name": "__auto__", "null_count": 4322881514883095157, "row_count": 6938749918561325818}, {"columns": ["col2_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4805227984963841173, "histo_col_type": "", "name": "__auto__", "null_count": 3915191254425275906, "row_count": 6938749918561325818}, {"columns": ["col2_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1871538839248870898, "histo_col_type": "", "name": "__auto__", "null_count": 1245148843689068106, "row_count": 6938749918561325818}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3525253472752142744, "histo_col_type": "", "name": "__auto__", "null_count": 4084567802093211185, "row_count": 6938749918561325818}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5939483050734564241, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6938749918561325818}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5117114531288672382, "histo_col_type": "", "name": "__auto__", "null_count": 4215569375957242610, "row_count": 6938749918561325818}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5142834602600948433, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6938749918561325818}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1955745672384177240, "histo_buckets": [{"distinct_range": 0, "num_eq": 7409555475779794945, "num_range": 0, "upper_bound": "4714-11-24 BC"}, {"distinct_range": 0, "num_eq": 1831220331426972344, "num_range": 900000, "upper_bound": "1971-04-27"}, {"distinct_range": 8000, "num_eq": 1712901173054517111, "num_range": 8000, "upper_bound": "1978-02-24"}, {"distinct_range": 0, "num_eq": 0, "num_range": 8743261509246629620, "upper_bound": "1982-05-24"}, {"distinct_range": 5070723929520482000, "num_eq": 900, "num_range": 5070723929520482413, "upper_bound": "1983-05-19"}, {"distinct_range": 1014020.8388609985, "num_eq": 0, "num_range": 10000000, "upper_bound": "1987-04-02"}, {"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": "1987-04-16"}, {"distinct_range": 0, "num_eq": 7052909795999327378, "num_range": 760310434395694157, "upper_bound": "1991-06-26"}, {"distinct_range": 3460047100.86752, "num_eq": 0, "num_range": 80000000000, "upper_bound": "1996-02-03"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 6938749918561325818}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 801401415141472159, "histo_col_type": "", "name": "__auto__", "null_count": 630109117462436145, "row_count": 6938749918561325818}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3604904001032685658, "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 6938749918561325818}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1360393297004729702, "histo_col_type": "STRING", "name": "__auto__", "null_count": 6734540409966721742, "row_count": 6938749918561325818}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3245721748471300846, "histo_col_type": "", "name": "__auto__", "null_count": 5812373885901024568, "row_count": 6938749918561325818}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6822799099891524634, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6938749918561325818}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1699507341768656504, "histo_buckets": [{"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "2217374998"}, {"distinct_range": 0, "num_eq": 5912414886626636234, "num_range": 0, "upper_bound": "3261620702"}, {"distinct_range": 37483.1813579805, "num_eq": 2114360999530717833, "num_range": 400000, "upper_bound": "3838858838"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 6938749918561325818}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 563072466621808223, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000, "num_range": 0, "upper_bound": "869129310"}, {"distinct_range": 70, "num_eq": 702573335782774877, "num_range": 70, "upper_bound": "1883936363"}, {"distinct_range": 0, "num_eq": 2978479469742322975, "num_range": 4000000000, "upper_bound": "2382889006"}, {"distinct_range": 200000, "num_eq": 3538666742734452089, "num_range": 200000, "upper_bound": "2736428041"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 10000, "upper_bound": "3028068726"}, {"distinct_range": 7098610.709478596, "num_eq": 700000, "num_range": 8000000, "upper_bound": "4020986115"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 4915149910053127489}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2183195136336073077, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4915149910053127489}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 494857235195557216, "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 4915149910053127489}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2749735345064298377, "histo_buckets": [{"distinct_range": 0, "num_eq": 3766743020709272464, "num_range": 0, "upper_bound": "1994-09-30"}, {"distinct_range": 5327871530483262000, "num_eq": 8000000000, "num_range": 5327871530483262557, "upper_bound": "5874897-12-31"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 4915149910053127489}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1516018857056651981, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 3755179635290728400, "num_eq": 6757986591615284632, "num_range": 3755179635290728496, "upper_bound": "1285345731"}, {"distinct_range": 0.7095054762384649, "num_eq": 10000000000, "num_range": 20, "upper_bound": "1581028634"}, {"distinct_range": 0, "num_eq": 8053999514150478956, "num_range": 400000, "upper_bound": "2178426552"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 4915149910053127489}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1633235855146400643, "histo_col_type": "", "name": "__auto__", "null_count": 3975172143217869434, "row_count": 4915149910053127489}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2911844226462562741, "histo_col_type": "", "name": "__auto__", "null_count": 2232084870889299782, "row_count": 4616939072389134329}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2993422212660832786, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4616939072389134329}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2692836811463320323, "histo_col_type": "", "name": "__auto__", "null_count": 3308879826294666334, "row_count": 4616939072389134329}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1963085775233660943, "histo_col_type": "", "name": "__auto__", "null_count": 2255916432519713531, "row_count": 4616939072389134329}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4476532253122228195, "histo_col_type": "", "name": "__auto__", "null_count": 2426334328377036833, "row_count": 4616939072389134329}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 136864535363350964, "histo_col_type": "", "name": "__auto__", "null_count": 894940344501119040, "row_count": 4616939072389134329}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2451186964595277904, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4616939072389134329}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1974639793202456954, "histo_col_type": "", "name": "__auto__", "null_count": 1844826155124979884, "row_count": 4616939072389134329}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3794742131774766368, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4616939072389134329}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2589390419881610106, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4616939072389134329}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 613038822193890911, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000000, "num_range": 0, "upper_bound": "91743677"}, {"distinct_range": 0, "num_eq": 689846656535625890, "num_range": 6635823551969782334, "upper_bound": "1883843158"}, {"distinct_range": 0, "num_eq": 8379930776999849036, "num_range": 0, "upper_bound": "4172049660"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 4616939072389134329}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 697659484642351156, "histo_col_type": "", "name": "__auto__", "null_count": 1960479627791389709, "row_count": 4616939072389134329}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3300456871927417307, "histo_col_type": "", "name": "__auto__", "null_count": 3109443644023755290, "row_count": 4616939072389134329}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2409871898263787110, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4616939072389134329}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 89801, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 135694, "histo_col_type": "", "name": "__auto__", "null_count": 28297, "row_count": 900000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 302848, "histo_col_type": "", "name": "__auto__", "null_count": 95044, "row_count": 900000}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 89593, "histo_col_type": "", "name": "__auto__", "null_count": 850866, "row_count": 900000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 725251, "histo_col_type": "", "name": "__auto__", "null_count": 19188, "row_count": 900000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 899534, "histo_col_type": "", "name": "__auto__", "null_count": 839329, "row_count": 900000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 213715, "histo_col_type": "BIT(28)", "name": "__auto__", "null_count": 410135, "row_count": 900000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 609801, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": "\\u000c/K"}, {"distinct_range": 0, "num_eq": 6553553871818454208, "num_range": 4108874359119099632, "upper_bound": "\\u0017\\u0018$\\"\\u0018]19h"}, {"distinct_range": 6912174615927837000, "num_eq": 8277282654681805467, "num_range": 6912174615927836509, "upper_bound": "DN"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 258998, "row_count": 900000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 449006, "histo_buckets": [{"distinct_range": 0, "num_eq": 6105368595489059960, "num_range": 0, "upper_bound": "2180468897"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 594673, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 793239, "histo_col_type": "", "name": "__auto__", "null_count": 514399, "row_count": 900000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 285777, "histo_col_type": "", "name": "__auto__", "null_count": 123241, "row_count": 900000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 752084, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 414357, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 626142, "histo_buckets": [{"distinct_range": 0, "num_eq": 8328962313541619865, "num_range": 0, "upper_bound": "-1.884348252866331"}, {"distinct_range": 5319926843118856000, "num_eq": 8000000000, "num_range": 5319926843118855775, "upper_bound": "-0.8073718287004887"}, {"distinct_range": 0, "num_eq": 6888648111331349817, "num_range": 0, "upper_bound": "-0.7058420221980608"}, {"distinct_range": 3480767230701974000, "num_eq": 500000, "num_range": 3480767230701974097, "upper_bound": "0.0"}, {"distinct_range": 0, "num_eq": 7000000, "num_range": 0, "upper_bound": "0.34180346432832914"}, {"distinct_range": 5222971835184723000, "num_eq": 4000, "num_range": 5222971835184722709, "upper_bound": "0.3565907003227451"}, {"distinct_range": 49384559.93287563, "num_eq": 0, "num_range": 50000000, "upper_bound": "1.0906173483097243"}, {"distinct_range": 500, "num_eq": 2734801500411353054, "num_range": 500, "upper_bound": "+Inf"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col2_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 768031, "histo_col_type": "", "name": "__auto__", "null_count": 422884, "row_count": 900000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 521519, "histo_buckets": [{"distinct_range": 0, "num_eq": 7427350530265519751, "num_range": 0, "upper_bound": "0.08920821931310823377"}, {"distinct_range": 0, "num_eq": 8046651555369898817, "num_range": 200000000, "upper_bound": "82209808.91844787716"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 513633, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 243338, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col2_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24734, "histo_col_type": "", "name": "__auto__", "null_count": 215865, "row_count": 900000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 744361, "histo_buckets": [{"distinct_range": 0, "num_eq": 6689920101809005785, "num_range": 0, "upper_bound": "1992-12-23"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 783410, "histo_buckets": [{"distinct_range": 0, "num_eq": 7049625876869579343, "num_range": 0, "upper_bound": "23838588"}, {"distinct_range": 0, "num_eq": 0, "num_range": 500000, "upper_bound": "434414567"}, {"distinct_range": 0, "num_eq": 6279540507758820817, "num_range": 1790623448795176935, "upper_bound": "593910348"}, {"distinct_range": 225177536.77040485, "num_eq": 7000000000, "num_range": 300000000, "upper_bound": "1521277359"}, {"distinct_range": 5148465557295506000, "num_eq": 1000, "num_range": 5148465557295506572, "upper_bound": "2207434390"}, {"distinct_range": 2286966072834636000, "num_eq": 6146301038448006235, "num_range": 5969007582697496029, "upper_bound": "3558610242"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 730417, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4062176291, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "1970-01-01"}, {"distinct_range": 84356357209311400, "num_eq": 700, "num_range": 753588692122395101, "upper_bound": "1970-09-10"}, {"distinct_range": 1862804257.046753, "num_eq": 50, "num_range": 8000000000, "upper_bound": "1978-12-24"}, {"distinct_range": 10000000000, "num_eq": 1000, "num_range": 10000000000, "upper_bound": "1986-10-17"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 80000, "upper_bound": "1988-09-04"}, {"distinct_range": 3187450580228798000, "num_eq": 6000, "num_range": 3187450580228798204, "upper_bound": "1990-08-14"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 50, "upper_bound": "1994-01-26"}, {"distinct_range": 0, "num_eq": 4000000, "num_range": 8418141355430593186, "upper_bound": "5874897-12-31"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5620717498, "histo_buckets": [{"distinct_range": 0, "num_eq": 4230570029692264583, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 9892240.414656298, "num_eq": 607000241458006580, "num_range": 80000000, "upper_bound": "1172697525"}, {"distinct_range": 0, "num_eq": 3062647759475701256, "num_range": 4772698740021793911, "upper_bound": "1426548278"}, {"distinct_range": 2000, "num_eq": 954671101783546210, "num_range": 2000, "upper_bound": "1720486911"}, {"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "2217488128"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1902305529, "histo_col_type": "", "name": "__auto__", "null_count": 3332060791, "row_count": 7000000000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2065535467, "histo_buckets": [{"distinct_range": 0, "num_eq": 6110133160429068524, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 1879634479697669414, "num_range": 7000000000, "upper_bound": "96806874"}, {"distinct_range": 836705.1951247196, "num_eq": 3072291385010835471, "num_range": 1000000, "upper_bound": "96875077"}, {"distinct_range": 200000, "num_eq": 2000000000, "num_range": 200000, "upper_bound": "247568506"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 2020262383169270336, "upper_bound": "365924864"}, {"distinct_range": 1080824148877960700, "num_eq": 4750321596035762242, "num_range": 1080824148877960675, "upper_bound": "1376998021"}, {"distinct_range": 1240814916009849600, "num_eq": 100000, "num_range": 8636616734810711303, "upper_bound": "3124632489"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6017008394, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1290331948, "histo_buckets": [{"distinct_range": 0, "num_eq": 5901787091549153673, "num_range": 0, "upper_bound": "\\u000e)g"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "\'bDpU)Sc\\u0008"}, {"distinct_range": 61.01019929763109, "num_eq": 500000, "num_range": 80, "upper_bound": ".Ox\\"t\\u0011n\\n0"}, {"distinct_range": 2321317193242812400, "num_eq": 3133626183078059593, "num_range": 3309321081305619762, "upper_bound": "/\\u0007"}, {"distinct_range": 28347405829655800, "num_eq": 3000000, "num_range": 68416005765342191, "upper_bound": "9J~p\\u0005<n "}, {"distinct_range": 0, "num_eq": 400000, "num_range": 900000, "upper_bound": "cv\\u0003;}\' "}, {"distinct_range": 4667656819258533000, "num_eq": 3276460376844046537, "num_range": 8286756899980500362, "upper_bound": "r\\u0015q"}, {"distinct_range": 24.801804223065442, "num_eq": 2002597856571350382, "num_range": 40, "upper_bound": "{"}, {"distinct_range": 1627860027573153300, "num_eq": 10000, "num_range": 1627860027573153401, "upper_bound": "}a\\u0002"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 7000000000}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "infinity" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        postgres_1    | CREATE EXTENSION
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2525486&tab=buildLog) on [master@eda9189cecbbc279f1857f6e6b992bdfd363397e](https://github.com/cockroachdb/cockroach/commits/eda9189cecbbc279f1857f6e6b992bdfd363397e):

```
        postgres_1    |     pg_ctl -D /var/lib/postgresql/data -l logfile start
        postgres_1    | 
        postgres_1    | waiting for server to start....2020-12-17 06:15:15.629 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2020-12-17 06:15:15.644 UTC [49] LOG:  database system was shut down at 2020-12-17 06:15:15 UTC
        postgres_1    | 2020-12-17 06:15:15.648 UTC [48] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2020-12-17 06:15:18.538 UTC [48] LOG:  received fast shutdown request
        postgres_1    | .2020-12-17 06:15:18.541 UTC [48] LOG:  aborting any active transactions
        postgres_1    | 2020-12-17 06:15:18.542 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2020-12-17 06:15:18.549 UTC [50] LOG:  shutting down
        postgres_1    | 2020-12-17 06:15:18.668 UTC [48] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2020-12-17 06:15:18.750 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2020-12-17 06:15:18.750 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2020-12-17 06:15:18.755 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2020-12-17 06:15:18.769 UTC [85] LOG:  database system was shut down at 2020-12-17 06:15:18 UTC
        postgres_1    | 2020-12-17 06:15:18.774 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (601.21s)
        test_1        |     --- FAIL: TestCompare/postgres (0.60s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2528190&tab=buildLog) on [master@46919380225dba7122130c338744b561d7eb6c56](https://github.com/cockroachdb/cockroach/commits/46919380225dba7122130c338744b561d7eb6c56):

```
        test_1        |             CREATE INDEX ON table4 (col1_0 ASC, col1_10, col1_6 ASC, col1_2 DESC, col1_5 DESC, col1_3 DESC, col1_8, col1_9, col4_11, col1_4, col4_10 ASC) INCLUDE (col4_9);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col1_3 ASC, col1_0 ASC, col4_9 ASC, col1_6, col4_10 ASC, col4_11, col1_2, col1_5, col1_8 DESC, col1_9, col1_10);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col1_6 ASC, col1_8, col4_10) INCLUDE (col4_11);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col1_4 DESC, col4_10 DESC, col1_5 DESC, col1_6 ASC, col4_11 DESC, col1_0 DESC, col1_2 DESC, col1_10 DESC);
        test_1        |             CREATE INDEX ON table4 (col4_9, col1_2 ASC, col1_3 DESC, col1_8 ASC, col1_6, col4_11, col1_10 DESC, col4_10 DESC, col1_4 DESC, col1_0 DESC);
        test_1        |             CREATE INDEX ON table4 (col1_5, col4_10 ASC, col1_0, col1_3 ASC, col4_11 DESC, col4_9, col1_2, col1_6 DESC, col1_10, col1_9 DESC, col1_8 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col1_9);
        test_1        |             CREATE TABLE table5 (col5_0 TIMETZ NULL, col5_1 BOOL NOT NULL, col5_2 INTERVAL NOT NULL, col5_3 INTERVAL NOT NULL, col5_4 FLOAT8 NULL, col5_5 TIME, col5_6 TIMESTAMP NULL, col5_7 INTERVAL NULL, col5_8 BOX2D NULL, col5_9 REGTYPE NULL, col5_10 "char" NOT NULL, col5_11 FLOAT8 NOT NULL, PRIMARY KEY (col5_2, col5_11));
        test_1        |             CREATE INDEX ON table5 (col5_11 ASC, col5_2) INCLUDE (col5_0, col5_4, col5_8, col5_9);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_6 ASC, col5_8 ASC, col5_0 DESC, col5_7 DESC, col5_3, col5_2 ASC, col5_5 ASC, col5_1 DESC, col5_4 ASC, col5_11);
        test_1        |             CREATE INDEX ON table5 (col5_9, col5_3 ASC, col5_0, col5_1 DESC, col5_4 DESC, col5_2, col5_5, col5_11 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_10);
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_3) REFERENCES table5 (col5_10);
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_10) REFERENCES table2 (col1_10) ON DELETE RESTRICT ON UPDATE CASCADE;
        test_1        |             ALTER TABLE table5 ADD FOREIGN KEY (col5_7) REFERENCES table4 (col1_9) ON DELETE CASCADE ON UPDATE CASCADE;
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_5) REFERENCES table2 (col1_5) ON DELETE RESTRICT;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('skwpsp', 'z', 'fy', 'zbacf', 'a', 'fk');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('pxk', 'hemx', 'xkup', 'rblex', 'fhze');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('r', 'jf', 'wyav');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('ar', 'jsgvoy', 'izap', 'xddwwb', 'a', 'eef');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('xihp', 'rnwd', 'wu', 're', 'tckcz');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (0.86s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 DECIMAL NOT NULL, col1_1 DECIMAL NOT NULL, col1_2 CHAR NOT NULL, col1_3 BIT(4) NOT NULL, col1_4 REGCLASS NOT NULL, col1_5 BOOL NOT NULL, col1_6 TIME NOT NULL, col1_7 INT8 NOT NULL, col1_8 TIMESTAMPTZ NOT NULL, col1_9 OID, col1_10 INT8 NOT NULL, col1_11 INT8 NOT NULL, col1_12 TIMESTAMPTZ NOT NULL, col1_13 DECIMAL NOT NULL, col1_14 INTERVAL NOT NULL, col1_15 FLOAT8 NOT NULL, col1_16 VARCHAR NOT NULL, col1_17 REGPROC NOT NULL, col1_18 DATE NOT NULL, PRIMARY KEY (col1_16 DESC, col1_0 DESC, col1_14 DESC, col1_18, col1_15 ASC, col1_3 DESC, col1_1 DESC, col1_5, col1_11 ASC, col1_6 DESC, col1_12 ASC, col1_8, col1_7 DESC, col1_4 DESC, col1_2 DESC, col1_17, col1_13 ASC), UNIQUE (col1_16, col1_0 ASC, col1_9, col1_17, col1_3 ASC, col1_14, col1_10 DESC, col1_5 ASC, col1_13 ASC) WHERE (((((((((((((table1.col1_6 != '24:00:00') OR (table1.col1_1 < Infinity)) OR (table1.col1_8 != '-4713-11-24 00:00:00+00:00')) AND (table1.col1_13 = NaN)) AND (table1.col1_15 < -Inf)) OR table1.col1_5) AND (table1.col1_16 = e'\U00002603')) OR (table1.col1_18 <= '1970-01-01')) OR (table1.col1_12 <= '-4713-11-24 00:00:00+00:00')) AND (table1.col1_7 < 2147483647)) OR (table1.col1_0 <= -1)) AND (table1.col1_2 = '"')) OR (table1.col1_10 != -32768)) AND (table1.col1_11 != -1), UNIQUE (col1_4 DESC, col1_15 DESC, col1_2 ASC, col1_13 DESC, col1_0 ASC, col1_10 ASC, col1_18, col1_8 ASC, col1_6 ASC, col1_11, col1_16, col1_7 DESC, col1_1 ASC, col1_12 DESC, col1_5, col1_3 ASC, col1_9) WHERE table1.col1_15 < 5e-324:::FLOAT8, INDEX (col1_2 ASC, col1_1 DESC) STORING (col1_10), UNIQUE (col1_0 DESC, col1_4 ASC, col1_2 DESC, col1_18 DESC, col1_7 DESC, col1_17 DESC, col1_3 DESC, col1_8, col1_6 ASC, col1_12 DESC, col1_11 DESC, col1_13, col1_14, col1_15 DESC, col1_16) STORING (col1_9), UNIQUE (col1_18 ASC, col1_6 DESC, col1_1 ASC, col1_8 ASC, col1_13 ASC, col1_4 ASC, col1_9 ASC, col1_12, col1_0 DESC, col1_16, col1_5 ASC, col1_15 ASC, col1_11 DESC, col1_3 DESC, col1_2, col1_17 ASC, col1_14, col1_7 ASC) STORING (col1_10) WHERE (table1.col1_7 > (-128):::INT8) AND (table1.col1_10 != (-1):::INT8), UNIQUE (col1_0 DESC, col1_4, col1_14, col1_8 ASC, col1_1, col1_7 ASC, col1_18, col1_5 ASC, col1_17 DESC, col1_9, col1_16, col1_3 DESC, col1_13, col1_10 DESC, col1_2 ASC, col1_11 DESC, col1_12 DESC, col1_6, col1_15 ASC), FAMILY (col1_0), FAMILY (col1_13, col1_12), FAMILY (col1_16, col1_2), FAMILY (col1_11), FAMILY (col1_5), FAMILY (col1_9), FAMILY (col1_10), FAMILY (col1_14), FAMILY (col1_6, col1_8), FAMILY (col1_4, col1_3, col1_18), FAMILY (col1_17, col1_7, col1_1), FAMILY (col1_15));
        test_1        |             CREATE TABLE table2 (col1_16 VARCHAR NOT NULL, col1_0 DECIMAL NOT NULL, col1_14 INTERVAL NOT NULL, col1_18 DATE NOT NULL, col1_15 FLOAT8 NOT NULL, col1_3 BIT(4) NOT NULL, col1_1 DECIMAL NOT NULL, col1_5 BOOL NOT NULL, col1_11 INT8 NOT NULL, col1_6 TIME NOT NULL, col1_12 TIMESTAMPTZ NOT NULL, col1_8 TIMESTAMPTZ NOT NULL, col1_7 INT8 NOT NULL, col1_4 REGCLASS NOT NULL, col1_2 CHAR NOT NULL, col1_17 REGPROC NOT NULL, col1_13 DECIMAL NOT NULL, col2_17 TIMESTAMPTZ, col2_18 BYTES NOT NULL, col2_19 OID, col2_20 INTERVAL NULL, col2_21 STRING NULL, col2_22 TIMESTAMP NULL, col2_23 INT2 NOT NULL, col2_24 VARCHAR, col2_25 REGPROC NULL, col2_26 BIT(43) NOT NULL, col2_27 VARCHAR NOT NULL, col2_28 INET, col2_29 FLOAT4, col2_30 TIME NOT NULL, col2_31 INT4 NULL, col2_32 UUID, col2_33 DECIMAL NOT NULL, PRIMARY KEY (col1_16 DESC, col1_0 DESC, col1_14 DESC, col1_18, col1_15 ASC, col1_3 DESC, col1_1 DESC, col1_5, col1_11 ASC, col1_6 DESC, col1_12 ASC, col1_8, col1_7 DESC, col1_4 DESC, col1_2 DESC, col1_17, col1_13 ASC), UNIQUE (col1_4, col2_28, col1_13 DESC, col1_6 DESC, col1_11 DESC, col1_2, col2_33 ASC, col1_7, col2_17 DESC, col2_24 ASC, col2_31 DESC, col1_5 ASC, col1_3, col1_18 DESC, col2_29 ASC, col2_30 DESC, col1_15 ASC, col1_1 DESC, col2_18 DESC) STORING (col2_21, col2_22, col2_26, col2_27, col2_32) WHERE (((((((((((((((((((((((table2.col1_0 = 'Infinity':::DECIMAL) OR (table2.col1_7 < 32767:::INT8)) OR (table2.col1_15 != 3.4028234663852886e+38:::FLOAT8)) OR (table2.col2_29 >= 1.0:::FLOAT8)) AND (table2.col1_11 > (-9223372036854775807):::INT8)) OR (table2.col2_33 <= 1:::DECIMAL)) AND (table2.col1_16 < e'\'':::STRING)) OR (table2.col2_30 < '24:00:00':::TIME)) OR (NOT table2.col1_5)) OR (table2.col1_12 != '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (table2.col2_24 <= 'X':::STRING)) AND (table2.col1_1 >= 'NaN':::DECIMAL)) OR (table2.col2_17 <= '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) OR (table2.col2_27 <= e'\x00':::STRING)) OR (table2.col2_31 <= 2147483647:::INT8)) AND (table2.col1_6 = '00:00:00':::TIME)) OR (table2.col1_2 <= e'\'':::STRING)) AND (table2.col2_21 <= 'X':::STRING)) AND (table2.col2_22 > '-2000-01-01 00:00:00':::TIMESTAMP)) AND (table2.col1_13 = (-1):::DECIMAL)) OR (table2.col1_18 >= '1970-01-01':::DATE)) OR (table2.col1_8 < '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table2.col2_23 != (-1):::INT8)) OR (table2.col2_18 < e'\\x00':::BYTES), INDEX (col1_6 ASC, col1_1 DESC, col2_23 ASC, col2_30, col2_21 ASC, col1_11 DESC, col2_17 ASC, col1_5 DESC, col1_2 DESC, col2_22 ASC, col2_31 DESC, col2_20 DESC, col2_25, col2_28, col1_3 DESC, col2_29 ASC, col2_19 DESC, col2_18 ASC, col2_27 DESC, col2_33 ASC, col1_17 ASC) WHERE ((((((((((((((table2.col1_15 > 0.0:::FLOAT8) AND (table2.col1_8 = '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table2.col1_0 >= 'Infinity':::DECIMAL)) AND (table2.col2_33 != (-1):::DECIMAL)) OR (table2.col1_2 > e'\x00':::STRING)) OR (table2.col2_23 > 127:::INT8)) AND (table2.col1_12 != '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table2.col1_18 > '5874897-12-31':::DATE)) OR (table2.col2_27 != '':::STRING)) AND (table2.col1_7 <= 9223372036854775807:::INT8)) OR (table2.col2_29 >= 0.0:::FLOAT8)) OR (table2.col2_21 >= '':::STRING)) OR (table2.col2_22 = '294276-12-31 23:59:59.999999':::TIMESTAMP)) AND (table2.col1_11 != 127:::INT8)) AND (table2.col1_13 <= (-1.234E+401):::DECIMAL), INDEX (col2_27 ASC, col1_8 DESC, col2_19, col2_33, col1_1 ASC, col2_25 DESC, col1_5, col2_22, col2_23 ASC, col1_11, col2_28 DESC, col2_24 DESC, col1_16 ASC, col2_31 DESC, col1_3 DESC, col2_32 DESC, col2_17 ASC, col1_13 DESC, col1_12, col1_6 ASC, col1_7 DESC, col2_30 ASC, col1_15, col2_18 ASC, col2_26, col2_29 DESC) WHERE (((((((((((((((((((((((table2.col1_0 < (-1):::DECIMAL) OR (table2.col1_11 >= 32767:::INT8)) OR (table2.col1_12 != '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table2.col1_2 != e'\x00':::STRING)) AND (table2.col2_23 > 1:::INT8)) OR (table2.col2_30 <= '24:00:00':::TIME)) OR (table2.col2_24 < e'\'':::STRING)) OR (table2.col2_21 != e'\x00':::STRING)) AND (table2.col1_18 >= '5874897-12-31':::DATE)) OR (table2.col1_6 = '00:00:00':::TIME)) AND (table2.col1_1 <= 1:::DECIMAL)) OR (table2.col2_29 > '+Inf':::FLOAT8)) AND (table2.col1_16 != e'\U00002603':::STRING)) AND (table2.col2_18 >= e'\\x22':::BYTES)) OR (table2.col2_17 != '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table2.col2_22 >= '294276-12-31 23:59:59.999999':::TIMESTAMP)) AND (table2.col2_33 <= 'NaN':::DECIMAL)) OR (NOT table2.col1_5)) OR (table2.col1_15 = 3.4028234663852886e+38:::FLOAT8)) OR (table2.col2_31 = 127:::INT8)) AND (table2.col1_7 = 0:::INT8)) OR (table2.col1_8 = '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table2.col2_27 < 'X':::STRING)) AND (table2.col1_13 <= 'NaN':::DECIMAL), UNIQUE (col2_32 DESC, col1_1 ASC, col2_19 DESC, col2_17 DESC, col1_16, col2_21) STORING (col2_22, col2_24, col2_25, col2_26, col2_29), UNIQUE (col1_6 DESC, col1_12, col1_8 DESC, col1_0 DESC, col1_5, col1_2, col2_23, col2_20 DESC, col2_17 DESC, col2_29 DESC, col1_14, col2_28 ASC, col1_16 ASC, col2_21 ASC, col2_33 ASC, col1_3, col1_11, col2_24 ASC, col1_13 DESC, col2_22 ASC, col2_26 DESC, col1_4, col2_32 DESC, col2_31) STORING (col2_25), INDEX (col2_25 ASC, col1_17, col1_11 DESC, col2_19 ASC, col2_18 ASC, col2_29, col2_33, col1_7, col1_5 ASC, col2_24 ASC, col1_4 ASC, col2_20 ASC, col1_8 ASC, col2_22 ASC, col2_28, col1_2, col1_6 ASC, col2_23 DESC, col1_0 DESC, col2_26, col1_1 ASC, col2_27, col2_17 ASC, col1_12 ASC, col1_18 DESC, col2_21 DESC) STORING (col2_30) WHERE (((((((((((((((((((((table2.col1_15 = -Inf) AND (table2.col1_7 < -128)) AND (table2.col2_27 <= e'\U00002603')) OR (table2.col1_0 <= -Infinity)) OR (table2.col1_6 > '24:00:00')) OR (table2.col2_17 >= '0001-01-01 00:00:00+00:00')) AND (table2.col1_18 <= '4714-11-24 BC')) OR (table2.col1_16 > e'\U00002603')) OR (table2.col2_33 = Infinity)) AND (table2.col2_21 != e'\U00002603')) AND (table2.col1_8 > '0001-01-01 00:00:00+00:00')) AND (table2.col2_30 != '24:00:00')) AND (table2.col2_18 > '\x58')) OR (table2.col2_29 >= -Inf)) AND table2.col1_5) OR (table2.col1_13 < -Infinity)) OR (table2.col1_1 != -1.234E+401)) AND (table2.col1_11 > 127)) OR (table2.col1_2 != e'\U00002603')) OR (table2.col2_31 < -128)) OR (table2.col2_23 < -1)) AND (table2.col2_24 <= e'\U00002603'), INDEX (col1_2 ASC, col1_18, col1_0, col1_14 DESC, col1_5 ASC, col2_24, col2_23 ASC, col1_1 ASC, col2_22, col1_11 DESC, col2_33 ASC, col1_12 DESC, col1_6, col1_8 DESC, col1_4, col2_26, col2_29, col1_7, col2_31 ASC, col2_28 DESC, col2_27 ASC, col1_13, col2_20 ASC, col2_18 ASC, col2_30 DESC, col1_17 DESC, col2_19, col1_3 DESC, col2_32, col2_21, col1_15 DESC, col2_25 ASC, col1_16 DESC) STORING (col2_17) WHERE (table2.col1_11 > -9223372036854775807) OR (table2.col1_6 != '24:00:00'), FAMILY (col2_26, col2_32), FAMILY (col1_6, col1_0, col1_2, col2_18), FAMILY (col1_4), FAMILY (col1_13, col2_20), FAMILY (col1_18), FAMILY (col1_14), FAMILY (col2_31), FAMILY (col1_1, col2_23, col2_21), FAMILY (col1_16, col1_5, col1_12, col1_17), FAMILY (col2_24), FAMILY (col2_22, col2_19, col1_8), FAMILY (col2_28), FAMILY (col2_33), FAMILY (col2_17, col2_30, col2_29), FAMILY (col1_15), FAMILY (col2_27, col1_3), FAMILY (col1_11), FAMILY (col1_7, col2_25), UNIQUE (col1_0) STORING (col2_19, col2_23, col2_25, col2_27, col2_31, col2_32) WHERE (((((((((((((((((((((((table2.col2_17 <= '3000-01-01 00:00:00+00:00') AND (table2.col2_23 < 0)) OR (NOT table2.col1_5)) OR (table2.col1_18 = 'infinity')) OR (table2.col2_22 = '-2000-01-01 00:00:00')) OR (table2.col1_16 > 'X')) AND (table2.col2_18 > '\x27')) OR (table2.col2_21 != 'X')) AND (table2.col1_11 != 9223372036854775807)) OR (table2.col1_0 <= -Infinity)) OR (table2.col1_6 < '24:00:00')) OR (table2.col1_1 != NaN)) OR (table2.col2_24 > e'\x00')) OR (table2.col1_12 >= '0001-01-01 00:00:00+00:00')) AND (table2.col1_15 >= -1.0)) OR (table2.col2_31 = 1)) OR (table2.col1_13 != -1.234E+401)) AND (table2.col2_27 != e'\U00002603')) OR (table2.col2_30 > '24:00:00')) OR (table2.col1_2 = '"')) AND (table2.col1_8 > '294276-12-31 23:59:59.999999+00:00')) AND (table2.col2_33 <= NaN)) AND (table2.col2_29 <= 0.0)) OR (table2.col1_7 < 1)) INTERLEAVE IN PARENT table1 (col1_16, col1_0, col1_14, col1_18, col1_15, col1_3, col1_1, col1_5, col1_11, col1_6, col1_12, col1_8, col1_7, col1_4, col1_2, col1_17, col1_13);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 702444914830330683, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 758170271210376062, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 287364661010044566, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1398335542415790603, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 325861432473223992, "histo_buckets": [{"distinct_range": 0, "num_eq": 6322725647939632754, "num_range": 0, "upper_bound": "\\u001a"}, {"distinct_range": 0, "num_eq": 8977511524412696794, "num_range": 7890824492680071770, "upper_bound": "\\""}, {"distinct_range": 8477838522522982000, "num_eq": 8642369701702890643, "num_range": 8477838522522982201, "upper_bound": "%"}, {"distinct_range": 0, "num_eq": 4597953061742221276, "num_range": 4010573842738899623, "upper_bound": "("}, {"distinct_range": 0, "num_eq": 0, "num_range": 100000000, "upper_bound": "8"}, {"distinct_range": 0, "num_eq": 50, "num_range": 3997343644448352422, "upper_bound": "D"}, {"distinct_range": 2218132.5386155234, "num_eq": 10, "num_range": 6000000, "upper_bound": "\x7f"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 229513362361680896, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 853534352711987304, "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12821150135806819, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 292752553887024622, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 784451608586551440, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 190750284284473044, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000, "num_range": 0, "upper_bound": "1983-01-09"}, {"distinct_range": 1873.3961318535546, "num_eq": 100, "num_range": 2000, "upper_bound": "infinity"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 853662389702077618, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000, "num_range": 0, "upper_bound": "3.160317326407955154E+19"}, {"distinct_range": 1506630544827895800, "num_eq": 4544572618018688011, "num_range": 7282000765664674063, "upper_bound": "7.241427583397916806E+35"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 909940915633761457, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 276917633557747676, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1178495043989363865, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1338479486936681274, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 484401208303193953, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": ""}, {"distinct_range": 3197914279423130000, "num_eq": 0, "num_range": 3519214432980620658, "upper_bound": "!\\u0012P"}, {"distinct_range": 2452407860.817692, "num_eq": 10, "num_range": 8000000000, "upper_bound": "B1"}, {"distinct_range": 3372584306963503600, "num_eq": 1415120576666408327, "num_range": 3372584306963503822, "upper_bound": "X"}, {"distinct_range": 30000000, "num_eq": 9173210856801547750, "num_range": 30000000, "upper_bound": "^3\\u0015YQ"}, {"distinct_range": 968493872042932200, "num_eq": 6010342345885048583, "num_range": 1721237665656014284, "upper_bound": "p\\u000f*\\t"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 675446224373473202, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1410634875304922900}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1344475061966411169, "histo_col_type": "", "name": "__auto__", "null_count": 991774297373694341, "row_count": 1410634875304922900}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5120063969853967022, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3787504222136955967, "histo_col_type": "", "name": "__auto__", "null_count": 7784823582277100456, "row_count": 8557693454150495240}, {"columns": ["col2_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5300423304646380504, "histo_col_type": "", "name": "__auto__", "null_count": 1553436942079866320, "row_count": 8557693454150495240}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7039306569479356047, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2040672333089470181, "histo_col_type": "", "name": "__auto__", "null_count": 4401767783864136857, "row_count": 8557693454150495240}, {"columns": ["col2_25"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 94653123729067675, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 1931918895192832000, "num_eq": 20, "num_range": 1931918895192831959, "upper_bound": "575418812"}, {"distinct_range": 8814323460417737000, "num_eq": 8082147252050929088, "num_range": 8814323460417736561, "upper_bound": "1240084474"}, {"distinct_range": 164371501855104450, "num_eq": 500000, "num_range": 164371501855104460, "upper_bound": "2325340772"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 5085238651067477630, "row_count": 8557693454150495240}, {"columns": ["col2_27"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 974943664955366434, "histo_buckets": [{"distinct_range": 0, "num_eq": 7779061085237239738, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 90, "num_range": 0, "upper_bound": "T`h\\u0001T$"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col2_30"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 629123556314294775, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 215474168684403354, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3857417653897552041, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "00:03:07.223856"}, {"distinct_range": 6310406932818114000, "num_eq": 5638354747580911400, "num_range": 7281627508110632862, "upper_bound": "03:51:01.702837"}, {"distinct_range": 0, "num_eq": 10, "num_range": 5798650135807650525, "upper_bound": "04:54:36.16542"}, {"distinct_range": 1000000, "num_eq": 900, "num_range": 1000000, "upper_bound": "06:40:32.01709"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col2_29"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 370410602543540685, "histo_col_type": "", "name": "__auto__", "null_count": 2249067356962706908, "row_count": 8557693454150495240}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3814430310958506835, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6861997775520072669, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1207920017917593292, "histo_col_type": "", "name": "__auto__", "null_count": 568059010540640769, "row_count": 8557693454150495240}, {"columns": ["col2_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5984827246175684008, "histo_col_type": "", "name": "__auto__", "null_count": 4024892302384556544, "row_count": 8557693454150495240}, {"columns": ["col2_28"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5947915159932981644, "histo_col_type": "", "name": "__auto__", "null_count": 2645408419131653232, "row_count": 8557693454150495240}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 168445063492812842, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4456667825554802630, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col2_32"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5102387501581665996, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000, "num_range": 0, "upper_bound": "003c6032-c0bb-4a6d-9fc7-f9c0ee74b4f4"}, {"distinct_range": 700000000, "num_eq": 80, "num_range": 700000000, "upper_bound": "321936f9-3cf7-4eb0-b437-ac3310a9f9b9"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 5000000000, "upper_bound": "3e9e7702-6779-4119-b792-4dc3d900330f"}, {"distinct_range": 8000, "num_eq": 200000, "num_range": 8000, "upper_bound": "8e37206d-f7f0-4c60-95ea-b4692d0759f0"}, {"distinct_range": 0, "num_eq": 30000000, "num_range": 2000000, "upper_bound": "d366009b-9131-4ec9-92e2-3e2a9f8b709f"}, {"distinct_range": 0, "num_eq": 5000, "num_range": 1587438226067508345, "upper_bound": "d5bf204c-c99a-421d-945d-522764331532"}, {"distinct_range": 65045356860.6902, "num_eq": 1076030875702382660, "num_range": 80000000000, "upper_bound": "ffffffff-ffff-ffff-ffff-ffffffffffff"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 6376497972753748550, "row_count": 8557693454150495240}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2518148547929587880, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6882552357487932441, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col1_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3366005000318478057, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6979288867975548697, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 211913686016064380, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 269288877303095364, "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col2_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 752963282122758452, "histo_col_type": "", "name": "__auto__", "null_count": 7106940240052105108, "row_count": 8557693454150495240}, {"columns": ["col2_31"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1020106890060989747, "histo_col_type": "", "name": "__auto__", "null_count": 5160192995357949753, "row_count": 8557693454150495240}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5922817297938493420, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 1506917379280774354, "num_range": 10000, "upper_bound": "\\nS\\u0019?J"}, {"distinct_range": 7633038275027558000, "num_eq": 932830771457283812, "num_range": 7633038275027558610, "upper_bound": "\\u0016mn:X,M\\u0019\\u0013"}, {"distinct_range": 0, "num_eq": 718323477477080782, "num_range": 0, "upper_bound": "\\u0018V7B"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3015348519700900290, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col2_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4255444744670698181, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col2_33"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6410899335607697683, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6409557820719862715, "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col2_26"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1842602622797666731, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4623841728279990697, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8557693454150495240}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('isjsm', 'ykkeyv', 'cr', 'k');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('z', 'sxk', 'epj');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6278255, "histo_buckets": [{"distinct_range": 0, "num_eq": 7309550721551250978, "num_range": 0, "upper_bound": "1971-02-23"}, {"distinct_range": 0, "num_eq": 439509722138561592, "num_range": 485604544492571324, "upper_bound": "1973-09-10"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 6417914331546280005, "upper_bound": "1976-02-22"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 900000, "upper_bound": "1976-03-30"}, {"distinct_range": 16985.04406677651, "num_eq": 0, "num_range": 20000, "upper_bound": "1981-09-15"}, {"distinct_range": 0, "num_eq": 7779948937364448517, "num_range": 400000, "upper_bound": "1981-11-11"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 1839432122814616988, "upper_bound": "1989-11-26"}, {"distinct_range": 5000000000, "num_eq": 4000000000, "num_range": 5000000000, "upper_bound": "1991-11-24"}, {"distinct_range": 0, "num_eq": 8166035422865798475, "num_range": 800000, "upper_bound": "1996-08-24"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1568636, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "0.4226683868864723198"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9320727, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8691965, "histo_buckets": [{"distinct_range": 0, "num_eq": 2582304500325457036, "num_range": 0, "upper_bound": ""}, {"distinct_range": 299567.6586302544, "num_eq": 3790433873860203851, "num_range": 700000, "upper_bound": "\\""}, {"distinct_range": 29531347.983675294, "num_eq": 8135056833188744031, "num_range": 600000000, "upper_bound": "(\\u0005\\u0001\\u00113h"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "BH\\u0003"}, {"distinct_range": 30000000000, "num_eq": 3095530120738320175, "num_range": 30000000000, "upper_bound": "C^1\\u001c2\\u0018?"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5297274, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6674027, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5343139, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9631657, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1634871, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6906552, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3182124, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5490826, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2729244, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2399807, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3744904, "histo_col_type": "", "name": "__auto__", "null_count": 8820675, "row_count": 10000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8137802, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5415229, "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6709587, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000, "num_range": 0, "upper_bound": "114706094"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 8574812446105374504, "upper_bound": "1912104560"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "2940780366"}, {"distinct_range": 70000, "num_eq": 1712888989555962450, "num_range": 70000, "upper_bound": "3277799300"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4829830, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30097, "histo_buckets": [{"distinct_range": 0, "num_eq": 606229205941463210, "num_range": 0, "upper_bound": "\\u001b%m;$\\u001c\\u001a,"}, {"distinct_range": 349168.8451309408, "num_eq": 20000000, "num_range": 600000, "upper_bound": "\\u001eT{\\u0013b"}, {"distinct_range": 10000, "num_eq": 3576756799732200980, "num_range": 10000, "upper_bound": "\\""}, {"distinct_range": 563433.6332875746, "num_eq": 10000, "num_range": 1000000, "upper_bound": "Hg\\u0016q\x7f\\u0018"}, {"distinct_range": 1575432.5778700418, "num_eq": 500000, "num_range": 80000000, "upper_bound": "P2L\\u0002a^X"}, {"distinct_range": 9092.792659144654, "num_eq": 100000, "num_range": 10000, "upper_bound": "zlJ!\\u000fNbu"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5376, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11672, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "21:08:19.462655"}, {"distinct_range": 126486609317407980, "num_eq": 9000, "num_range": 3303416117100329190, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26006, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13297, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16331, "histo_col_type": "", "name": "__auto__", "null_count": 26181, "row_count": 40000}, {"columns": ["col2_28"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39683, "histo_col_type": "", "name": "__auto__", "null_count": 11206, "row_count": 40000}, {"columns": ["col2_30"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2296, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17008, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9825, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32550, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_32"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2173, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "4695f679-30a1-4fe9-a84b-bf1ee484c73f"}, {"distinct_range": 117165516565893310, "num_eq": 7000, "num_range": 497423494850349636, "upper_bound": "8f3ecf30-865e-47a3-912f-229e421d9d71"}, {"distinct_range": 646.2415090368062, "num_eq": 5815650800953868475, "num_range": 700, "upper_bound": "b3c01191-2200-4509-93a7-9e1bc564fcda"}, {"distinct_range": 7620459377322453000, "num_eq": 6301482638628574909, "num_range": 7620459377322453137, "upper_bound": "b93c6bec-aee1-4485-9981-d08daec9f4cc"}, {"distinct_range": 130355084.08173138, "num_eq": 70, "num_range": 1000000000, "upper_bound": "c11ee00a-7026-43a0-a5fb-88d6aa867729"}, {"distinct_range": 0, "num_eq": 8217085790092281036, "num_range": 5573187448210817581, "upper_bound": "e24a44aa-ee10-425c-86fd-c931235d8a82"}, {"distinct_range": 0, "num_eq": 3442125163852144429, "num_range": 100, "upper_bound": "f2753d62-c6d2-4c08-9f31-99b155c517c0"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 36993, "row_count": 40000}, {"columns": ["col2_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15477, "histo_col_type": "", "name": "__auto__", "null_count": 24674, "row_count": 40000}, {"columns": ["col2_31"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12232, "histo_col_type": "", "name": "__auto__", "null_count": 94, "row_count": 40000}, {"columns": ["col2_33"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6554, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33422, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21081, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15786, "histo_col_type": "", "name": "__auto__", "null_count": 39696, "row_count": 40000}, {"columns": ["col2_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22288, "histo_col_type": "", "name": "__auto__", "null_count": 6929, "row_count": 40000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9157, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17049, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36134, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36620, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25430, "histo_buckets": [{"distinct_range": 0, "num_eq": 30, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 77.07708891615187, "num_eq": 3159058680723754269, "num_range": 80, "upper_bound": "1402464377"}, {"distinct_range": 2303562766989857500, "num_eq": 6107590973202122769, "num_range": 4610506159217123680, "upper_bound": "2001812800"}, {"distinct_range": 0, "num_eq": 400000, "num_range": 4000000, "upper_bound": "2462119438"}, {"distinct_range": 16.294083372054203, "num_eq": 377179093375817588, "num_range": 30, "upper_bound": "2739833413"}, {"distinct_range": 2551174086442808000, "num_eq": 6027119729054338816, "num_range": 7582185108918335206, "upper_bound": "3138631895"}, {"distinct_range": 0, "num_eq": 5483377738872680032, "num_range": 1684566416271761670, "upper_bound": "3334676580"}, {"distinct_range": 0, "num_eq": 1642424721472909479, "num_range": 0, "upper_bound": "3520104633"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26083, "histo_buckets": [{"distinct_range": 0, "num_eq": 6997100176110712171, "num_range": 0, "upper_bound": "\\u0004"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9751, "histo_col_type": "", "name": "__auto__", "null_count": 8459, "row_count": 40000}, {"columns": ["col2_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35116, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_27"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9953, "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_26"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10962, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_29"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35766, "histo_col_type": "", "name": "__auto__", "null_count": 23961, "row_count": 40000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34773, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22060, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23796, "histo_col_type": "", "name": "__auto__", "null_count": 1448, "row_count": 40000}, {"columns": ["col2_25"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2352, "histo_buckets": [{"distinct_range": 0, "num_eq": 1163464636355977005, "num_range": 0, "upper_bound": "1025744217"}, {"distinct_range": 0, "num_eq": 3566610533305192403, "num_range": 1000000000, "upper_bound": "1538508247"}, {"distinct_range": 0, "num_eq": 500000, "num_range": 4076416080952869640, "upper_bound": "1892661330"}, {"distinct_range": 184621.47000851217, "num_eq": 3155092669137156617, "num_range": 800000, "upper_bound": "3305277709"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 13154, "row_count": 40000}]';
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_0) REFERENCES table2 (col1_0);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 847797046854597309, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 419799820373644944, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1046187919933469023, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 625565486201975530, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 381801677416028259, "histo_buckets": [{"distinct_range": 0, "num_eq": 1368249240798717790, "num_range": 0, "upper_bound": "(\\u0013%"}, {"distinct_range": 0, "num_eq": 100, "num_range": 301986075604995412, "upper_bound": "A`\\u0001"}, {"distinct_range": 8000000, "num_eq": 80000000000, "num_range": 8000000, "upper_bound": "Eto:\\u0006\\r\\u001c\\u0004F"}, {"distinct_range": 0, "num_eq": 558670466437262078, "num_range": 9196241097681090380, "upper_bound": "O1"}, {"distinct_range": 0, "num_eq": 6677928942565178464, "num_range": 3258364846400283030, "upper_bound": "ac\\u0011"}, {"distinct_range": 3992.8133795683266, "num_eq": 3000000000, "num_range": 6000, "upper_bound": "o&(3,|>Ah"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 610592701143974360, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}, {"columns": ["col1_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 391175658513301403, "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 514989542999584660, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1263493592864776830, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 497111256412110559, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 370006150195076676, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1086747957404113749, "histo_col_type": "", "name": "__auto__", "null_count": 545423886412017901, "row_count": 1359415856665346064}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 119673952437196619, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1275353071836326718, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1026942373619245511, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000000, "num_range": 0, "upper_bound": "62724748.25290967281"}, {"distinct_range": 10000, "num_eq": 4096668783530337217, "num_range": 10000, "upper_bound": "13994326665424065.2"}, {"distinct_range": 23640968.472219724, "num_eq": 9137693948959326302, "num_range": 50000000, "upper_bound": "7.053616424888476776E+21"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 782193823603691184, "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 304572890377036615, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000000, "num_range": 0, "upper_bound": "2854823908"}, {"distinct_range": 6175638275398962000, "num_eq": 90000000000, "num_range": 6175638275398962508, "upper_bound": "2878320060"}, {"distinct_range": 590169130590633100, "num_eq": 3243677251315338831, "num_range": 1703097425203783054, "upper_bound": "3199874533"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1312348160085811387, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45312215478009644, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1359415856665346064}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1036428440451358997, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 601653277671300092, "histo_buckets": [{"distinct_range": 0, "num_eq": 800, "num_range": 0, "upper_bound": "\\u0011"}, {"distinct_range": 4453799604853912600, "num_eq": 80000, "num_range": 4453799604853912759, "upper_bound": "9"}, {"distinct_range": 2094.415097767602, "num_eq": 7640508868868238819, "num_range": 10000, "upper_bound": "H"}, {"distinct_range": 0, "num_eq": 1549604454669731897, "num_range": 2173089218801803727, "upper_bound": "["}, {"distinct_range": 0, "num_eq": 4000000000, "num_range": 4679395334707025271, "upper_bound": "g"}, {"distinct_range": 7000, "num_eq": 4843436291712412210, "num_range": 7000, "upper_bound": "i"}, {"distinct_range": 8502184922094273000, "num_eq": 418056780346151691, "num_range": 8502184922094272353, "upper_bound": "k"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 580916699389167484, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col2_33"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 737528837441713911, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col2_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 411973952828340818, "histo_col_type": "", "name": "__auto__", "null_count": 96349359407230104, "row_count": 1699004422312517022}, {"columns": ["col2_29"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1314579334354486605, "histo_col_type": "", "name": "__auto__", "null_count": 1436628492643450415, "row_count": 1699004422312517022}, {"columns": ["col2_31"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 876923560347437189, "histo_col_type": "", "name": "__auto__", "null_count": 619607394522955201, "row_count": 1699004422312517022}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1435590807811565748, "histo_buckets": [{"distinct_range": 0, "num_eq": 3162285177647800821, "num_range": 0, "upper_bound": "NaN"}, {"distinct_range": 756031461727602800, "num_eq": 500000, "num_range": 1662911532058558837, "upper_bound": "-2.968904412872605429E+26"}, {"distinct_range": 0, "num_eq": 5000, "num_range": 5058011785994511800, "upper_bound": "-8675564.281269692821"}, {"distinct_range": 3090041737654743600, "num_eq": 7036746118881928038, "num_range": 3839168923970516724, "upper_bound": "5026777529.300062553"}, {"distinct_range": 3745.41415782947, "num_eq": 20, "num_range": 4000, "upper_bound": "8.653595453762360218E+21"}, {"distinct_range": 0, "num_eq": 7000000, "num_range": 90, "upper_bound": "5.969265873542694383E+22"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 215992320604029803, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 758893672239474771, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1440920211036238576, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col2_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 913457197099494270, "histo_col_type": "", "name": "__auto__", "null_count": 1200015888866625405, "row_count": 1699004422312517022}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 273057947965829493, "histo_buckets": [{"distinct_range": 0, "num_eq": 7103149561755319882, "num_range": 0, "upper_bound": "\\u0003 \\u0006Vk0\\u001ck"}, {"distinct_range": 7000000, "num_eq": 7535044786004314117, "num_range": 7000000, "upper_bound": "\\u000cFwJ"}, {"distinct_range": 261080575.72426268, "num_eq": 5118636323127727926, "num_range": 2000000000, "upper_bound": "\\r"}, {"distinct_range": 2855.170973725459, "num_eq": 1827069591004818174, "num_range": 5000, "upper_bound": "\\u001a\\u0011 \\u0017Wv/0O"}, {"distinct_range": 10, "num_eq": 1266846952258743925, "num_range": 10, "upper_bound": "<>wiO2\\u0001s\\""}, {"distinct_range": 0, "num_eq": 8872954274993553917, "num_range": 80, "upper_bound": "G8\\u001f3,=\\t"}, {"distinct_range": 23566347523.578156, "num_eq": 1000000, "num_range": 100000000000, "upper_bound": "K]"}, {"distinct_range": 8.211713619114274, "num_eq": 1798158692751753254, "num_range": 10, "upper_bound": "Q\\t\\u001bf\'"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 3000000, "upper_bound": "al{8\\u0017/>4"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col1_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1085434541451252787, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 964740895424842045, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 286578065631210290, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1448306327320476520, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col2_30"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1031588548536258448, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col2_27"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1104042897939585301, "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 686556739598784933, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 538873029880683965, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 441254239271117752, "histo_col_type": "", "name": "__auto__", "null_count": 1604421829027414193, "row_count": 1699004422312517022}, {"columns": ["col2_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 302869535565013243, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col2_26"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 435939655645384290, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 782421888921491049, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1699004422312517022}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 500755292937320387, "histo_col_type": "", "name": "__auto__", "null_count": 319743131321853984, "row_count": 1699004422312517022}, {"columns": ["col2_25"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 833063151272682030, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "144886475"}, {"distinct_range": 0, "num_eq": 2998608767134217736, "num_range": 0, "upper_bound": "148631929"}, {"distinct_range": 426.73176003650866, "num_eq": 0, "num_range": 600, "upper_bound": "229809363"}, {"distinct_range": 900000000, "num_eq": 7911929280424735059, "num_range": 900000000, "upper_bound": "395878530"}, {"distinct_range": 100000000000, "num_eq": 7000000, "num_range": 100000000000, "upper_bound": "2841974139"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "2958220307"}, {"distinct_range": 50000, "num_eq": 600000000, "num_range": 50000, "upper_bound": "2999252726"}, {"distinct_range": 8784968250379513000, "num_eq": 600000, "num_range": 8784968250379512795, "upper_bound": "3030994410"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 1000000, "upper_bound": "4083440838"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 198367456121167357, "row_count": 1699004422312517022}, {"columns": ["col2_28"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 178826422993714622, "histo_col_type": "", "name": "__auto__", "null_count": 878403721514365807, "row_count": 1699004422312517022}, {"columns": ["co

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2530483&tab=buildLog) on [master@a782ee8d93a23fc53eedada3c51893c19f7bb41e](https://github.com/cockroachdb/cockroach/commits/a782ee8d93a23fc53eedada3c51893c19f7bb41e):

```
        postgres_1    | 2020-12-19 06:14:19.996 UTC [50] LOG:  shutting down
        test_1        | --- FAIL: TestCompare (2.36s)
        test_1        |     --- FAIL: TestCompare/mutators (0.70s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 REGNAMESPACE NOT NULL, col1_1 REGCLASS NOT NULL, col1_2 FLOAT8 NOT NULL, col1_3 FLOAT4 NOT NULL, col1_4 UUID NOT NULL, col1_5 BIT(18) NOT NULL, col1_6 VARCHAR NOT NULL, col1_7 "char" NOT NULL, col1_8 CHAR[] NULL, col1_9 REGTYPE, PRIMARY KEY (col1_5 DESC, col1_7 ASC, col1_6 DESC, col1_1 ASC, col1_2, col1_0 DESC, col1_4 ASC, col1_3), INDEX (col1_5, col1_7) STORING (col1_8, col1_9) WHERE (((table1.col1_7 = '') AND (table1.col1_2 > 1.0)) AND (table1.col1_6 > 'X')) AND (table1.col1_3 < 0.0), INDEX (col1_4 DESC, col1_9) STORING (col1_8) WHERE (table1.col1_2 < 1.0:::FLOAT8) OR (table1.col1_6 >= '':::STRING), FAMILY (col1_6, col1_7), FAMILY (col1_5, col1_3, col1_1, col1_9, col1_4), FAMILY (col1_2), FAMILY (col1_8), FAMILY (col1_0), UNIQUE (col1_0) WHERE (((table1.col1_7 = '"') AND (table1.col1_2 != 3.4028234663852886e+38)) OR (table1.col1_3 < 0.0)) OR (table1.col1_6 != e'\''));
        test_1        |             CREATE TABLE table2 (col2_0 REGPROC NOT NULL, col2_1 BOOL NOT NULL, col2_2 INT4 NOT NULL, col2_3 INTERVAL NOT NULL, col2_4 REGTYPE NOT NULL, col2_5 BIT(21) NOT NULL, col2_6 REGPROCEDURE NOT NULL, col2_7 TIME NOT NULL, col2_8 BIT(49) NOT NULL, col2_9 REGTYPE NOT NULL, col2_10 STRING NOT NULL, col2_11 CHAR NOT NULL, col2_12 VARCHAR NOT NULL, col2_13 BOX2D NOT NULL, col2_14 FLOAT4 NOT NULL, PRIMARY KEY (col2_14 DESC, col2_3, col2_7 ASC, col2_12 DESC, col2_13, col2_9, col2_11, col2_1 ASC, col2_6 DESC, col2_4 DESC, col2_10 ASC, col2_2, col2_5, col2_8 ASC, col2_0), FAMILY (col2_8, col2_2), FAMILY (col2_1), FAMILY (col2_11), FAMILY (col2_9), FAMILY (col2_13, col2_12, col2_6), FAMILY (col2_10), FAMILY (col2_14, col2_5, col2_7), FAMILY (col2_4), FAMILY (col2_0, col2_3));
        test_1        |             CREATE TABLE table3 (col1_5 BIT(18) NOT NULL, col1_7 "char" NOT NULL, col1_6 VARCHAR NOT NULL, col1_1 REGCLASS NOT NULL, col1_2 FLOAT8 NOT NULL, col1_0 REGNAMESPACE NOT NULL, col1_4 UUID NOT NULL, col1_3 FLOAT4 NOT NULL, col3_8 INT8 NULL, col3_9 FLOAT8 NULL, col3_10 NAME NOT NULL, col3_11 BIT(4) NULL, col3_12 DECIMAL NOT NULL, col3_13 REGPROC, col3_14 INT4, col3_15 BOX2D, col3_16 FLOAT8 NOT NULL, PRIMARY KEY (col1_5 DESC, col1_7 ASC, col1_6 DESC, col1_1 ASC, col1_2, col1_0 DESC, col1_4 ASC, col1_3, col3_16 DESC, col3_10 DESC, col3_14 ASC, col3_13, col3_8 DESC, col3_12, col3_15 ASC), INDEX (col1_1 ASC, col1_3 DESC, col3_12, col1_2 DESC, col3_10 ASC, col3_9 ASC, col3_8 DESC, col1_6 DESC, col3_14 ASC, col3_11 ASC, col3_15 ASC, col1_7 DESC, col3_13 ASC, col1_4 ASC, col3_16 ASC) WHERE (((((((((table3.col3_8 >= -128) AND (table3.col3_10 >= e'\'')) AND (table3.col1_7 >= 'X')) OR (table3.col1_3 >= +Inf)) OR (table3.col3_12 < -Infinity)) OR (table3.col3_14 != 32767)) OR (table3.col3_16 < 0.0)) AND (table3.col1_2 < 0.0)) OR (table3.col1_6 >= 'X')) AND (table3.col3_9 > -Inf), INDEX (col1_1) STORING (col3_9, col3_11) WHERE (((((((((table3.col1_6 > '"':::STRING) AND (table3.col3_12 <= 'Infinity':::DECIMAL)) AND (table3.col3_14 <= 1:::INT8)) OR (table3.col1_2 = 0.0:::FLOAT8)) OR (table3.col3_16 <= 5e-324:::FLOAT8)) AND (table3.col1_7 = 'X':::STRING)) OR (table3.col3_8 <= 1:::INT8)) OR (table3.col1_3 = '+Inf':::FLOAT8)) AND (table3.col3_10 >= e'\x00':::STRING)) OR (table3.col3_9 < 0.0:::FLOAT8), INDEX (col1_6 ASC, col3_16, col1_1 DESC, col1_0 DESC, col1_4, col1_3, col3_12 DESC, col3_15 ASC, col3_11, col1_7 ASC, col3_8 ASC) STORING (col3_9), INDEX (col3_14, col3_16 DESC, col1_7, col3_15 ASC, col3_11, col3_10 DESC, col1_4 ASC, col3_13 DESC, col1_0 ASC, col1_5, col3_8 ASC, col3_12 DESC, col1_1, col1_3, col1_2 DESC) WHERE ((table3.col3_16 = NaN) OR (table3.col3_12 <= -Infinity)) OR (table3.col1_6 != e'\x00'), UNIQUE (col1_7, col1_2 DESC, col3_16, col1_4 ASC, col1_6 ASC, col3_9 DESC, col3_15, col1_3 ASC, col3_8, col1_1 ASC) STORING (col3_11) WHERE ((((table3.col1_6 < 'X') AND (table3.col3_8 = 0)) AND (table3.col3_12 != NaN)) OR (table3.col1_7 > 'X')) OR (table3.col1_2 < 5e-324), UNIQUE (col1_2) WHERE (((((((((table3.col3_10 < e'\U00002603') AND (table3.col1_7 != e'\U00002603')) AND (table3.col1_2 != -1.0)) AND (table3.col1_6 = '"')) OR (table3.col3_8 != 127)) AND (table3.col3_12 != NaN)) OR (table3.col1_3 = NaN)) OR (table3.col3_16 >= 0.0)) AND (table3.col3_9 != NaN)) AND (table3.col3_14 = -1), FAMILY (col1_2), FAMILY (col3_15, col1_7), FAMILY (col3_12), FAMILY (col1_6), FAMILY (col3_9), FAMILY (col1_3, col1_4), FAMILY (col3_8), FAMILY (col1_0, col3_16, col1_1, col3_10, col3_14), FAMILY (col3_13), FAMILY (col3_11, col1_5)) INTERLEAVE IN PARENT table1 (col1_5, col1_7, col1_6, col1_1, col1_2, col1_0, col1_4, col1_3);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2831349568, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2840587275, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 148436770, "histo_col_type": "", "name": "__auto__", "null_count": 1335209772, "row_count": 3000000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 786579717, "histo_col_type": "", "name": "__auto__", "null_count": 1984832809, "row_count": 3000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1126925576, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 936313551, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2427919294, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 976716115, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000, "num_range": 0, "upper_bound": "22f5526c-d67d-487b-b627-092d83c7e163"}, {"distinct_range": 8000000000, "num_eq": 3012227477266740504, "num_range": 8000000000, "upper_bound": "27b04ab0-7e6f-4bd5-9a89-ec6806336c65"}, {"distinct_range": 1587885559111252200, "num_eq": 8059451917664863888, "num_range": 3951006199731922671, "upper_bound": "27d4b858-49fd-4143-919d-df471ddb4764"}, {"distinct_range": 200000, "num_eq": 40000000000, "num_range": 200000, "upper_bound": "3a1dcf86-36a8-4066-a7f3-40852bdfd388"}, {"distinct_range": 5870258050988684000, "num_eq": 2836398927938257711, "num_range": 5870258050988684075, "upper_bound": "4312fa9f-38eb-428c-a5cc-71393cac8e84"}, {"distinct_range": 2068041550290910200, "num_eq": 30000000, "num_range": 3418330304539582679, "upper_bound": "89b3d029-82cf-4d69-a970-359a5f3d4193"}, {"distinct_range": 0, "num_eq": 4010942422893547618, "num_range": 7754676719111619403, "upper_bound": "8a647c10-b427-442c-aba4-60e41219998d"}, {"distinct_range": 0, "num_eq": 1151552338097658323, "num_range": 1036817689305193479, "upper_bound": "a2a0257a-1ab5-47ab-937b-045cf046a782"}, {"distinct_range": 0, "num_eq": 40, "num_range": 2703981645944015414, "upper_bound": "bec29267-8acf-44e0-a473-efd250ae6fe3"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 3000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1673761844, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "000011001000110100"}, {"distinct_range": 519.4216498803867, "num_eq": 4000, "num_range": 600, "upper_bound": "001100101011000110"}, {"distinct_range": 473442189392939260, "num_eq": 1164970103837777385, "num_range": 2603194905304655287, "upper_bound": "010010111001110001"}], "histo_col_type": "BIT(18)", "name": "__auto__", "null_count": 0, "row_count": 3000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2499411652, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3005104231806175838, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4526795842020472629}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3160560883856917010, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4526795842020472629}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 477325329388442165, "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 4526795842020472629}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2945509706048386185, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4526795842020472629}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 61644902378156749, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4526795842020472629}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4238471826934439034, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4526795842020472629}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2239165263545803338, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4526795842020472629}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1921701387493904991, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4526795842020472629}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3713564044119092042, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4526795842020472629}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3348912437730097892, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4526795842020472629}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1926797806261598594, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4526795842020472629}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3048656715959910494, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4526795842020472629}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2176606728718218783, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4526795842020472629}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1284159196407717313, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4526795842020472629}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1144836872554452133, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4526795842020472629}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1943522245282950248, "histo_col_type": "", "name": "__auto__", "null_count": 1862498166943428348, "row_count": 3228633294713477081}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 508612972843060274, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3228633294713477081}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 810338667680317139, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "\\r"}, {"distinct_range": 556843614988184700, "num_eq": 3652449323362013512, "num_range": 972524322233615214, "upper_bound": "\\u001d"}, {"distinct_range": 4352646555649185300, "num_eq": 2000000000, "num_range": 7566494518211505134, "upper_bound": "\\u001e"}, {"distinct_range": 3057002998249189000, "num_eq": 7992270558322071417, "num_range": 5383897026530676597, "upper_bound": "\'"}, {"distinct_range": 80000000, "num_eq": 80, "num_range": 80000000, "upper_bound": "t"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 3228633294713477081}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1068185288853606638, "histo_buckets": [{"distinct_range": 0, "num_eq": 60, "num_range": 0, "upper_bound": "\\""}, {"distinct_range": 70000000, "num_eq": 30000000000, "num_range": 70000000, "upper_bound": "7.{\\u0015\\u0018Aji"}, {"distinct_range": 100000, "num_eq": 3000000, "num_range": 100000, "upper_bound": "dH"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 3228633294713477081}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20782533713940230, "histo_col_type": "", "name": "__auto__", "null_count": 3063872121535511738, "row_count": 3228633294713477081}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2658207156326976547, "histo_buckets": [{"distinct_range": 0, "num_eq": 1164377505486081860, "num_range": 0, "upper_bound": "-1925010752"}, {"distinct_range": 9162990938485398000, "num_eq": 4888112768314379263, "num_range": 9162990938485397829, "upper_bound": "-1580724864"}, {"distinct_range": 0, "num_eq": 5909763983193520846, "num_range": 0, "upper_bound": "-201526357"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 1943145291539767516, "upper_bound": "-135017653"}, {"distinct_range": 477550383381790700, "num_eq": 60000000, "num_range": 6147938100017621827, "upper_bound": "1834512381"}, {"distinct_range": 0, "num_eq": 74261629261426502, "num_range": 2053903616177299320, "upper_bound": "2064069895"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 1683524949239430017, "row_count": 3228633294713477081}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1238029496033545006, "histo_buckets": [{"distinct_range": 0, "num_eq": 5157583623817150956, "num_range": 0, "upper_bound": "000000000000000000"}, {"distinct_range": 3857151349404901400, "num_eq": 30000, "num_range": 4654898834299186971, "upper_bound": "001001000100110111"}, {"distinct_range": 0, "num_eq": 8355585468388622173, "num_range": 4982655395702486634, "upper_bound": "010001100100111100"}, {"distinct_range": 0, "num_eq": 8290423583830161826, "num_range": 7883924123917116989, "upper_bound": "010010101111101011"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 600000, "upper_bound": "011000010101011011"}, {"distinct_range": 808621199085732700, "num_eq": 60000000000, "num_range": 1566110656925717520, "upper_bound": "011000011011011100"}, {"distinct_range": 86738319037506530, "num_eq": 100000000, "num_range": 140925963874122053, "upper_bound": "011001101111100100"}, {"distinct_range": 57587.899464127106, "num_eq": 2525356606439943777, "num_range": 90000, "upper_bound": "100011101100100101"}, {"distinct_range": 41423630433741830, "num_eq": 8006088260402236660, "num_range": 3919188223112900355, "upper_bound": "110101011110011010"}], "histo_col_type": "BIT(18)", "name": "__auto__", "null_count": 0, "row_count": 3228633294713477081}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2632882232818910322, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3228633294713477081}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1580704976733997509, "histo_col_type": "", "name": "__auto__", "null_count": 2110747899922660009, "row_count": 3228633294713477081}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3119800131854182126, "histo_col_type": "", "name": "__auto__", "null_count": 84228668397438516, "row_count": 3228633294713477081}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2621934461307104207, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3228633294713477081}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 213234277111746649, "histo_buckets": [{"distinct_range": 0, "num_eq": 1334789471589660250, "num_range": 0, "upper_bound": "1779322915"}, {"distinct_range": 6770451151858733000, "num_eq": 1385058010574426388, "num_range": 6770451151858733108, "upper_bound": "1891402839"}, {"distinct_range": 428.83461124862623, "num_eq": 5347900788893345001, "num_range": 1000, "upper_bound": "2059598607"}, {"distinct_range": 372.7339375952997, "num_eq": 8034120335442472138, "num_range": 800, "upper_bound": "3334959581"}, {"distinct_range": 149144550.29659536, "num_eq": 4193005641994510578, "num_range": 900000000, "upper_bound": "3971563893"}, {"distinct_range": 60000000, "num_eq": 3295116106598750911, "num_range": 60000000, "upper_bound": "4288355224"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 3228633294713477081}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2236455041444547922, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3228633294713477081}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2795296385776128925, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3228633294713477081}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2862637645978124298, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3228633294713477081}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2636474366815459694, "histo_col_type": "", "name": "__auto__", "null_count": 1003046958960324131, "row_count": 3228633294713477081}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1554623997384253771, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3228633294713477081}]':::JSONB;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_9) REFERENCES table1 (col1_0);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('vor', 'kj', 'nxq', 'vdbajs', 'npc');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('mjzl');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('n', 's', 'hvcm', 'lobedo', 'gkxnnv', 'q');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('jrg', 'fodqw');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('hxs', 'zixnch', 'nhglg', 'rthg', 'mwkfbj');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 81831327, "histo_col_type": "", "name": "__auto__", "null_count": 177729432, "row_count": 500000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 272570104, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "1229149818"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 3432693040622933713, "upper_bound": "2450906078"}, {"distinct_range": 0, "num_eq": 20000000000, "num_range": 1822881117425665559, "upper_bound": "2470408940"}, {"distinct_range": 1518868410964220700, "num_eq": 8776842946877796084, "num_range": 1518868410964220590, "upper_bound": "4037350318"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 448422693, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 117125611, "histo_buckets": [{"distinct_range": 0, "num_eq": 4554753289162409010, "num_range": 0, "upper_bound": "84629777-7d49-43bc-9541-d590a6d6df1f"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 90000, "upper_bound": "ce9d219f-db30-460a-82f4-4d36b39d0889"}, {"distinct_range": 795302685.1826774, "num_eq": 745252006252340804, "num_range": 2000000000, "upper_bound": "e570a1c0-2313-4911-8963-b0df0ba78d74"}, {"distinct_range": 858338272284909200, "num_eq": 30000000, "num_range": 1267486335850293905, "upper_bound": "ea3bbf3d-9b3c-4bf3-b5f4-90982999ab87"}, {"distinct_range": 6740970503809577000, "num_eq": 3963260539141761317, "num_range": 6998188697302149629, "upper_bound": "ffc666a7-01d5-4985-a90e-31fbf48b603b"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 354820024, "histo_buckets": [{"distinct_range": 0, "num_eq": 3350640145592489371, "num_range": 0, "upper_bound": "000110011110001000"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 9000000, "upper_bound": "010110100111111110"}], "histo_col_type": "BIT(18)", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 343499747, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 300524090, "histo_col_type": "", "name": "__auto__", "null_count": 263028792, "row_count": 500000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 440662639, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 387002479, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 292857928, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3087121276511150225, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6710168101253483617}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5928754903529267086, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6710168101253483617}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4541469994962193751, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6710168101253483617}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4997366383092119185, "histo_buckets": [{"distinct_range": 0, "num_eq": 9153341018566438446, "num_range": 0, "upper_bound": "-0.1649187058210373"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 6710168101253483617}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3140078095356479706, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6710168101253483617}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3831999702964280461, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6710168101253483617}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2290974598651788941, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6710168101253483617}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 502711473243279101, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6710168101253483617}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6000168531190032851, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6710168101253483617}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 321330271642458639, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6710168101253483617}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1179657002067774455, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6710168101253483617}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1012711681333526278, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6710168101253483617}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1250959781329489769, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6710168101253483617}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1918506430585294498, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6710168101253483617}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3050677087724303352, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6710168101253483617}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2393472344184888370, "histo_col_type": "", "name": "__auto__", "null_count": 4226151295315541010, "row_count": 4651229773705826100}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1823585981712757650, "histo_col_type": "", "name": "__auto__", "null_count": 1777701676655290567, "row_count": 4651229773705826100}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2916622159925182388, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4651229773705826100}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3645356316617342375, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "010101100111111001"}, {"distinct_range": 193368280.1589536, "num_eq": 100000000000, "num_range": 200000000, "upper_bound": "101110110100100110"}], "histo_col_type": "BIT(18)", "name": "__auto__", "null_count": 0, "row_count": 4651229773705826100}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4461033044636181971, "histo_buckets": [{"distinct_range": 0, "num_eq": 3670373078536491163, "num_range": 0, "upper_bound": "382824479"}, {"distinct_range": 0, "num_eq": 0, "num_range": 6932853346276364138, "upper_bound": "1882032739"}, {"distinct_range": 350186443036660540, "num_eq": 2306897722777250178, "num_range": 1259763983734119115, "upper_bound": "2614416241"}, {"distinct_range": 0, "num_eq": 0, "num_range": 60000000, "upper_bound": "3634026146"}, {"distinct_range": 1858564278851627000, "num_eq": 0, "num_range": 1858564278851627096, "upper_bound": "4160683756"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 4651229773705826100}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1723755157443654848, "histo_col_type": "", "name": "__auto__", "null_count": 1937917051684344462, "row_count": 4651229773705826100}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 794546343967113854, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000000, "num_range": 0, "upper_bound": "!"}, {"distinct_range": 60, "num_eq": 50000000000, "num_range": 60, "upper_bound": "\\""}, {"distinct_range": 5617613742446519000, "num_eq": 800, "num_range": 5617613742446519138, "upper_bound": "J"}, {"distinct_range": 0, "num_eq": 4983621679851520713, "num_range": 9011320226306753155, "upper_bound": "k"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 4651229773705826100}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4412478888023297157, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4651229773705826100}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3055538145858179074, "histo_col_type": "", "name": "__auto__", "null_count": 4385843296996297144, "row_count": 4651229773705826100}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1932202533877304237, "histo_col_type": "", "name": "__auto__", "null_count": 1434257846807585888, "row_count": 4651229773705826100}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3247626745763853875, "histo_buckets": [{"distinct_range": 0, "num_eq": 7839714802701603947, "num_range": 0, "upper_bound": "-1950013682"}, {"distinct_range": 3000, "num_eq": 2414793583599203527, "num_range": 3000, "upper_bound": "-645855984"}, {"distinct_range": 287358208601425820, "num_eq": 100000000, "num_range": 1119496875330534465, "upper_bound": "-303032504"}, {"distinct_range": 0, "num_eq": 2232264317220769135, "num_range": 0, "upper_bound": "-1"}, {"distinct_range": 263133214627947940, "num_eq": 300000000, "num_range": 5651981737578389596, "upper_bound": "668664117"}, {"distinct_range": 6135415077060186000, "num_eq": 80000, "num_range": 7949099580219336265, "upper_bound": "1131437285"}, {"distinct_range": 8037002422472322000, "num_eq": 2525377494092331884, "num_range": 8037002422472322029, "upper_bound": "1238246623"}, {"distinct_range": 8000000, "num_eq": 7470561933917558887, "num_range": 8000000, "upper_bound": "2105798720"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 3353403893951817370, "row_count": 4651229773705826100}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1097975616638568822, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 8000, "num_range": 7000, "upper_bound": "\\u0000"}, {"distinct_range": 0, "num_eq": 4933933242416737715, "num_range": 0, "upper_bound": "\\u0018;W\\r.\'"}, {"distinct_range": 0, "num_eq": 1647879852279905861, "num_range": 600000000, "upper_bound": "M0"}, {"distinct_range": 278608056.6057704, "num_eq": 30000000, "num_range": 800000000, "upper_bound": "j\\u0002{2"}, {"distinct_range": 0, "num_eq": 6150400249394141097, "num_range": 50, "upper_bound": "j3cC\\u001bP*q"}, {"distinct_range": 6774210665412720000, "num_eq": 800000, "num_range": 6774210665412719324, "upper_bound": "x*\\u001c(a)<j\\u0019"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 1265832201051863509, "upper_bound": "|\\u001fz~\\\\"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 4651229773705826100}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4084922474323813226, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4651229773705826100}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1590984604799542560, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4651229773705826100}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3745988522398598737, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4651229773705826100}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4364914863217469965, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4651229773705826100}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1144442455965935421, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4651229773705826100}]';
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_2) REFERENCES table3 (col1_2);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 220091052165627971, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7150353056539507707}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1374148763345986012, "histo_buckets": [{"distinct_range": 0, "num_eq": 1161354689643089674, "num_range": 0, "upper_bound": "797169916"}, {"distinct_range": 0, "num_eq": 0, "num_range": 6241635071836215102, "upper_bound": "1550384581"}, {"distinct_range": 70, "num_eq": 40000, "num_range": 70, "upper_bound": "1780944596"}, {"distinct_range": 674736464384741100, "num_eq": 2000000000, "num_range": 674736464384741082, "upper_bound": "2276053850"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 4761983509529835181, "upper_bound": "2375206074"}, {"distinct_range": 0, "num_eq": 2388129723136445156, "num_range": 4668005511323061680, "upper_bound": "2990341189"}, {"distinct_range": 0, "num_eq": 200000, "num_range": 100, "upper_bound": "3783320093"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "3825448640"}, {"distinct_range": 1025640246385986000, "num_eq": 8083553896457839679, "num_range": 7320882031945542267, "upper_bound": "4294040908"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 7150353056539507707}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 629977747493592225, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7150353056539507707}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5522992751150114391, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7150353056539507707}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2298822321937118842, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7150353056539507707}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4141446207164418221, "histo_col_type": "", "name": "__auto__", "null_count": 2236074890984431856, "row_count": 7150353056539507707}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2082386434475960968, "histo_col_type": "", "name": "__auto__", "null_count": 3603517916912247046, "row_count": 7150353056539507707}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 100588277184055921, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7150353056539507707}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2042567456877377596, "histo_buckets": [{"distinct_range": 0, "num_eq": 72735300774910145, "num_range": 0, "upper_bound": "00e58569-e096-495b-a5c6-6a0536e81cbc"}, {"distinct_range": 0, "num_eq": 442424047828013381, "num_range": 2554371782104803933, "upper_bound": "36c4abc9-4712-4bd2-92e6-d3dccacd05a4"}, {"distinct_range": 8613681168285152000, "num_eq": 1247816680600771120, "num_range": 8613681168285152583, "upper_bound": "3a41ee66-bfa4-48f6-9570-d640dbd0aef5"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 1000000000, "upper_bound": "73143d40-5cc3-4194-8821-00d00b880d2d"}, {"distinct_range": 8402708.187174303, "num_eq": 20, "num_range": 100000000, "upper_bound": "aac75f0f-7011-4d0e-969e-7d2fcf571873"}, {"distinct_range": 3000, "num_eq": 3958720752679350233, "num_range": 3000, "upper_bound": "b780b02c-78a4-433d-9d02-b1df626a352b"}, {"distinct_range": 80, "num_eq": 4000000000, "num_range": 80, "upper_bound": "d7b6edf3-a7dc-4b3d-8984-e976650bdc8f"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 7150353056539507707}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 666793710931640460, "histo_buckets": [{"distinct_range": 0, "num_eq": 8334066495847969428, "num_range": 0, "upper_bound": "000110010001100111"}, {"distinct_range": 0, "num_eq": 4921209919201548152, "num_range": 10000000, "upper_bound": "001010100101111111"}, {"distinct_range": 7898606596123645000, "num_eq": 30, "num_range": 9139060447865987038, "upper_bound": "001010101110101010"}, {"distinct_range": 0, "num_eq": 70000000, "num_range": 6981224235618210810, "upper_bound": "001100110011000101"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 10000000000, "upper_bound": "011011011010010111"}, {"distinct_range": 0, "num_eq": 5480422248956935331, "num_range": 0, "upper_bound": "100001100100100110"}, {"distinct_range": 1797328857352274200, "num_eq": 844671400130778649, "num_range": 2073285375026057331, "upper_bound": "100100111100001111"}, {"distinct_range": 29670651907191060, "num_eq": 8000000, "num_range": 3064366696953488996, "upper_bound": "100111001001111101"}, {"distinct_range": 8669002799995330000, "num_eq": 8646975037891522837, "num_range": 8669002799995329273, "upper_bound": "110010100111010110"}], "histo_col_type": "BIT(18)", "name": "__auto__", "null_count": 0, "row_count": 7150353056539507707}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2748264, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 399926, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2062720, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 986485, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2666883, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2385070, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 144377, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2805026, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1839670, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000000, "num_range": 0, "upper_bound": "-0.8075383305549622"}, {"distinct_range": 8067119306969571000, "num_eq": 3000000000, "num_range": 8067119306969571101, "upper_bound": "-0.5787934064865112"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 1372319599022426062, "upper_bound": "-0.38319268822669983"}, {"distinct_range": 20000, "num_eq": 5088648074259197444, "num_range": 20000, "upper_bound": "-0.2784382998943329"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 7298705903406020824, "upper_bound": "0.6117160320281982"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2643200, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1758191, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1369626, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 836149, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 512514, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 101983, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53421, "histo_col_type": "", "name": "__auto__", "null_count": 7324, "row_count": 200000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 191902, "histo_col_type": "", "name": "__auto__", "null_count": 67592, "row_count": 200000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35136, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 178848, "histo_col_type": "", "name": "__auto__", "null_count": 151710, "row_count": 200000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 139540, "histo_col_type": "", "name": "__auto__", "null_count": 154795, "row_count": 200000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 155445, "histo_col_type": "", "name": "__auto__", "null_count": 15647, "row_count": 200000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24706, "histo_buckets": [{"distinct_range": 0, "num_eq": 2866465145855265958, "num_range": 0, "upper_bound": "-1156526248"}, {"distinct_range": 37485071820199050, "num_eq": 1000000, "num_range": 825557272486144387, "upper_bound": "-756952829"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 30000, "upper_bound": "125764945"}, {"distinct_range": 0, "num_eq": 0, "num_range": 4765561043931862307, "upper_bound": "182975550"}, {"distinct_range": 0, "num_eq": 5000000, "num_range": 6000, "upper_bound": "1801396610"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 72778, "row_count": 200000}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 146188, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 97378, "histo_buckets": [{"distinct_range": 0, "num_eq": 8301988685955887261, "num_range": 0, "upper_bound": "000000101101111111"}, {"distinct_range": 3747296544648232400, "num_eq": 100, "num_range": 3861263343408871632, "upper_bound": "000001110101101011"}, {"distinct_range": 64434195471.886566, "num_eq": 200000, "num_range": 70000000000, "upper_bound": "000010101111010110"}, {"distinct_range": 108694936589965780, "num_eq": 4000, "num_range": 1945199578625807557, "upper_bound": "001001010010000101"}, {"distinct_range": 3115486905350396400, "num_eq": 400000, "num_range": 8932830153586832484, "upper_bound": "011101111000110100"}, {"distinct_range": 3410740573298449400, "num_eq": 600, "num_range": 8838940957244455580, "upper_bound": "110100101101010101"}, {"distinct_range": 600000, "num_eq": 9000000, "num_range": 600000, "upper_bound": "110110110000000011"}], "histo_col_type": "BIT(18)", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45970, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "54249246"}, {"distinct_range": 2725419353050892300, "num_eq": 2523388091309740725, "num_range": 3559462257158543596, "upper_bound": "86559916"}, {"distinct_range": 49783753.62821239, "num_eq": 2122238362035916651, "num_range": 200000000, "upper_bound": "860771174"}, {"distinct_range": 1723965642678207200, "num_eq": 200000000, "num_range": 1723965642678207278, "upper_bound": "1148689364"}, {"distinct_range": 0, "num_eq": 963593303587026073, "num_range": 0, "upper_bound": "1205692470"}, {"distinct_range": 20000000, "num_eq": 361123111236285296, "num_range": 20000000, "upper_bound": "4175316848"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17500, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 120894, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 92965, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 152388, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000000000, "num_range": 0, "upper_bound": "\\u000f"}, {"distinct_range": 207692.81044822792, "num_eq": 8499812325822946508, "num_range": 2000000, "upper_bound": "("}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32286, "histo_buckets": [{"distinct_range": 0, "num_eq": 3819960502321705919, "num_range": 0, "upper_bound": "\\u0018w"}, {"distinct_range": 30284.784066264296, "num_eq": 1632049022488541235, "num_range": 60000, "upper_bound": ")&"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 128033, "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 180377, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "inf" does not exist (SQLSTATE 42703)
        test_1        |     --- FAIL: TestCompare/postgres (1.66s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.2:5432: connect: connection refused
        test_1        | FAIL
        postgres_1    | 2020-12-19 06:14:20.106 UTC [48] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2020-12-19 06:14:20.213 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2020-12-19 06:14:20.213 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2020-12-19 06:14:20.219 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2020-12-19 06:14:20.233 UTC [85] LOG:  database system was shut down at 2020-12-19 06:14:20 UTC
        postgres_1    | 2020-12-19 06:14:20.237 UTC [1] LOG:  database system is ready to accept connections
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2531672&tab=buildLog) on [master@75efa1fe6f2096adc9db474026a2b7235e53c388](https://github.com/cockroachdb/cockroach/commits/75efa1fe6f2096adc9db474026a2b7235e53c388):

```
        cockroach1_1  | build:               CCL v21.1.0-alpha.1-489-g75efa1fe6f @ 2020/12/20 06:12:18 (go1.15.6)
        cockroach1_1  | webui:               http://cockroach1:8080
        cockroach1_1  | sql:                 postgresql://root@cockroach1:26257?sslmode=disable
        cockroach1_1  | RPC client flags:    /cockroach/cockroach <client cmd> --host=cockroach1:26257 --insecure
        cockroach1_1  | logs:                /cockroach-data/logs
        cockroach1_1  | temp dir:            /cockroach-data/cockroach-temp749711566
        cockroach1_1  | external I/O path:   /cockroach-data/extern
        cockroach1_1  | store[0]:            path=/cockroach-data
        cockroach1_1  | storage engine:      pebble
        cockroach1_1  | status:              initialized new cluster
        cockroach1_1  | clusterID:           dff325ec-cb41-46a9-9422-becaf568b958
        cockroach1_1  | nodeID:              1
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        test_1        | --- FAIL: TestCompare (1.28s)
        test_1        |     --- FAIL: TestCompare/postgres (0.64s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.64s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 REGCLASS NULL, col1_1 FLOAT8 NOT NULL, col1_2 DECIMAL NOT NULL, PRIMARY KEY (col1_2 DESC, col1_1 DESC), INDEX (col1_0 DESC, col1_2, col1_1), FAMILY (col1_0, col1_1, col1_2));
        test_1        |             CREATE TABLE table2 (col1_2 DECIMAL NOT NULL, col1_1 FLOAT8 NOT NULL, col2_2 INT4, col2_3 DECIMAL NOT NULL, col2_4 NAME NOT NULL, col2_5 DECIMAL, col2_6 VARCHAR NOT NULL, col2_7 REGTYPE NULL, col2_8 REGTYPE, col2_9 TIMESTAMP NULL, col2_10 REGCLASS NOT NULL, col2_11 OID NOT NULL, col2_12 TIMETZ NULL, col2_13 DECIMAL NOT NULL, PRIMARY KEY (col1_2 DESC, col1_1 DESC, col2_12 DESC, col2_10 DESC, col2_2 DESC, col2_13, col2_4), INDEX (col1_1, col2_7 DESC, col2_10, col2_2 ASC, col2_5 ASC, col2_6 DESC, col2_12 ASC, col2_8 DESC, col2_3 DESC, col1_2 ASC, col2_9 DESC, col2_13 ASC), INDEX (col2_7 ASC, col1_1, col2_12 DESC, col2_10, col2_2, col2_9 ASC, col1_2, col2_5, col2_3 ASC, col2_13 ASC, col2_8 DESC, col2_11 DESC, col2_6 DESC), UNIQUE (col2_2, col2_6 ASC, col2_10 DESC, col1_2 DESC, col2_11, col2_8 DESC, col2_4 DESC, col2_12 DESC, col2_3, col2_5 DESC) STORING (col2_7, col2_9) WHERE (((table2.col1_1 > 1.0:::FLOAT8) AND (table2.col2_6 >= '':::STRING)) AND (table2.col2_4 >= '"':::STRING)) AND (table2.col2_9 != '294276-12-31 23:59:59.999999':::TIMESTAMP), INDEX (col2_10, col2_11 DESC, col2_2 DESC, col2_3 ASC) STORING (col2_7, col2_9) WHERE (((((((((table2.col2_2 > -1) AND (table2.col2_13 > -Infinity)) AND (table2.col1_2 <= Infinity)) AND (table2.col2_9 <= '-4713-11-24 00:00:00')) AND (table2.col2_12 > '00:00:00+15:59:00')) AND (table2.col2_4 > '')) OR (table2.col2_3 = 1)) OR (table2.col2_6 < '')) OR (table2.col1_1 = 5e-324)) AND (table2.col2_5 < -1), UNIQUE (col2_11 DESC, col2_2 DESC, col2_10 ASC, col2_5, col2_4 DESC, col1_1 ASC, col2_3 ASC, col1_2 DESC, col2_9 ASC, col2_12 ASC, col2_6, col2_7) STORING (col2_8) WHERE (((((((((table2.col2_6 < '':::STRING) AND (table2.col2_5 >= 'NaN':::DECIMAL)) OR (table2.col2_3 > '-Infinity':::DECIMAL)) OR (table2.col1_2 = 'Infinity':::DECIMAL)) OR (table2.col2_12 >= '00:00:00+15:59:00':::TIMETZ)) OR (table2.col2_4 >= e'\'':::STRING)) OR (table2.col1_1 = (-1.0):::FLOAT8)) OR (table2.col2_2 < 2147483647:::INT8)) OR (table2.col2_13 = (-1):::DECIMAL)) AND (table2.col2_9 < '0001-01-01 00:00:00':::TIMESTAMP), UNIQUE (col2_11 ASC, col2_12 DESC, col2_9 ASC, col2_3, col2_7 ASC, col2_10 ASC, col2_6, col2_8, col1_1 ASC) STORING (col2_5) WHERE (((table2.col1_1 != -1.0) OR (table2.col2_13 < NaN)) OR (table2.col2_6 <= '')) OR (table2.col1_2 < -1), UNIQUE (col2_9 DESC, col2_2, col2_3 DESC, col2_8, col1_2, col2_5, col2_4, col2_11 DESC, col2_7) STORING (col2_6), UNIQUE (col2_12, col1_1, col2_6 DESC, col2_7 DESC, col1_2 ASC, col2_4) STORING (col2_3, col2_5, col2_8), UNIQUE (col2_2 DESC, col2_10 DESC, col2_3, col2_11 ASC, col2_4 ASC, col1_1 DESC, col2_5 ASC, col2_6, col2_8 DESC, col2_9 ASC, col2_13 DESC, col1_2 DESC, col2_12 ASC, col2_7), FAMILY (col2_4, col2_10), FAMILY (col2_12), FAMILY (col2_3, col2_8), FAMILY (col1_1), FAMILY (col2_5, col2_11, col1_2, col2_7), FAMILY (col2_9, col2_13, col2_6), FAMILY (col2_2)) INTERLEAVE IN PARENT table1 (col1_2, col1_1);
        test_1        |             CREATE TABLE table3 (col1_2 DECIMAL NOT NULL, col1_1 FLOAT8 NOT NULL, col2_12 TIMETZ NULL, col2_10 REGCLASS NOT NULL, col2_2 INT4, col2_13 DECIMAL NOT NULL, col2_4 NAME NOT NULL, col3_7 BIT(5) NOT NULL, col3_8 TIMESTAMPTZ NOT NULL, col3_9 REGNAMESPACE NOT NULL, col3_10 TIMESTAMPTZ NOT NULL, col3_11 "char" NULL, col3_12 CHAR, col3_13 CHAR NOT NULL, col3_14 FLOAT4 NOT NULL, col3_15 INET NULL, col3_16 CHAR NOT NULL, col3_17 REGPROC NULL, col3_18 "char", col3_19 REGNAMESPACE NULL, col3_20 BIT(24) NOT NULL, col3_21 FLOAT8 NOT NULL, col3_22 INTERVAL NOT NULL, PRIMARY KEY (col1_2 DESC, col1_1 DESC, col2_12 DESC, col2_10 DESC, col2_2 DESC, col2_13, col2_4, col3_21 ASC, col3_14, col3_11, col3_10 ASC, col3_12 ASC, col3_19 ASC, col3_15, col3_7 ASC, col3_20 ASC), INDEX (col3_19 DESC, col3_20 ASC), UNIQUE (col1_2 DESC, col3_10) WHERE (((((((((((((table3.col3_13 = e'\'':::STRING) AND (table3.col2_13 != 'NaN':::DECIMAL)) OR (table3.col3_14 < 1.0:::FLOAT8)) AND (table3.col1_1 <= 0.0:::FLOAT8)) AND (table3.col3_11 = e'\x00':::STRING)) AND (table3.col3_21 < 0.0:::FLOAT8)) AND (table3.col3_12 = e'\U00002603':::STRING)) AND (table3.col1_2 > 'Infinity':::DECIMAL)) OR (table3.col3_16 < e'\'':::STRING)) AND (table3.col3_18 >= e'\'':::STRING)) OR (table3.col2_4 != e'\'':::STRING)) OR (table3.col3_10 > '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table3.col2_12 >= '24:00:00-15:59:00':::TIMETZ)) AND (table3.col3_8 != '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ), UNIQUE (col2_10 ASC, col3_22 DESC, col3_17, col2_2, col3_15, col2_4, col3_20, col3_18 ASC, col3_14, col3_7 DESC, col1_1, col2_12 ASC, col2_13, col3_9 ASC, col3_10, col3_13 DESC, col3_11 ASC, col3_16 DESC, col1_2) WHERE ((((((((((((table3.col3_10 <= '294276-12-31 23:59:59.999999+00:00') AND (table3.col3_12 <= 'X')) AND (table3.col2_4 != e'\'')) AND (table3.col3_14 != 1.401298464324817e-45)) AND (table3.col2_13 = -Infinity)) OR (table3.col3_11 = '')) AND (table3.col3_8 >= '0001-01-01 00:00:00+00:00')) OR (table3.col3_18 != e'\'')) OR (table3.col3_13 <= e'\x00')) AND (table3.col3_21 >= NaN)) OR (table3.col2_2 <= 1)) OR (table3.col1_1 <= 5e-324)) OR (table3.col3_16 < e'\U00002603'), FAMILY (col3_21, col2_4, col2_13), FAMILY (col3_9, col3_20), FAMILY (col3_16, col3_11), FAMILY (col3_19, col1_1), FAMILY (col3_15, col3_12), FAMILY (col2_2, col2_10, col3_13), FAMILY (col3_8, col2_12), FAMILY (col3_7, col3_10), FAMILY (col3_14, col3_17, col3_18), FAMILY (col1_2, col3_22), UNIQUE (col2_4)) INTERLEAVE IN PARENT table2 (col1_2, col1_1, col2_12, col2_10, col2_2, col2_13, col2_4);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": "275120133"}, {"distinct_range": 0, "num_eq": 200000000, "num_range": 10000000000, "upper_bound": "758292847"}, {"distinct_range": 1696586810676704000, "num_eq": 1070219771374411498, "num_range": 5066623538407836398, "upper_bound": "2565373636"}, {"distinct_range": 2034912188864700200, "num_eq": 4000, "num_range": 5646222521737197307, "upper_bound": "3030874888"}, {"distinct_range": 1884921845170732000, "num_eq": 300, "num_range": 1884921845170731918, "upper_bound": "3480446782"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 4836872999451659296, "num_range": 0, "upper_bound": "7.896725056224842259"}, {"distinct_range": 1899739989150290200, "num_eq": 10000000, "num_range": 1899739989150290249, "upper_bound": "3.769747156375427253E+21"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 728553004142835500, "histo_buckets": [{"distinct_range": 0, "num_eq": 5107387713687568630, "num_range": 0, "upper_bound": "-5.963019359239218832E+37"}, {"distinct_range": 20000000, "num_eq": 30000, "num_range": 20000000, "upper_bound": "-108344131547.0352276"}, {"distinct_range": 9.319131139566363, "num_eq": 500000, "num_range": 40, "upper_bound": "7.374080418708688696E+30"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 7091472164550442120}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3215393918448161832, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7091472164550442120}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2254669477162743886, "histo_col_type": "", "name": "__auto__", "null_count": 3600532457304644456, "row_count": 7091472164550442120}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4069787904344359595, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 700, "num_eq": 600, "num_range": 700, "upper_bound": "531162412"}, {"distinct_range": 1487296128813515800, "num_eq": 9000, "num_range": 8701254430492554691, "upper_bound": "1250062708"}, {"distinct_range": 40000000000, "num_eq": 100000, "num_range": 40000000000, "upper_bound": "1893747905"}, {"distinct_range": 3214992177873667000, "num_eq": 400, "num_range": 6891212995777454287, "upper_bound": "2592734857"}, {"distinct_range": 5000000, "num_eq": 3043837286817236756, "num_range": 5000000, "upper_bound": "2969932109"}, {"distinct_range": 6090865987620036000, "num_eq": 100000000, "num_range": 7068325926354940682, "upper_bound": "3453823939"}, {"distinct_range": 6000000, "num_eq": 4768205817637858955, "num_range": 6000000, "upper_bound": "4227110134"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 7091472164550442120}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 517665188647548129, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7091472164550442120}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5673601763543740162, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7091472164550442120}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6394337352774943829, "histo_buckets": [{"distinct_range": 0, "num_eq": 4422448112983074094, "num_range": 0, "upper_bound": "84197672"}, {"distinct_range": 0, "num_eq": 0, "num_range": 4069301147160539779, "upper_bound": "1202183527"}, {"distinct_range": 500000000, "num_eq": 1314074759387718983, "num_range": 500000000, "upper_bound": "1700263532"}, {"distinct_range": 0, "num_eq": 8391345754849795344, "num_range": 0, "upper_bound": "2107928874"}, {"distinct_range": 22033.934780054366, "num_eq": 10000000, "num_range": 30000, "upper_bound": "2395767308"}, {"distinct_range": 0, "num_eq": 1870967910540885290, "num_range": 4175762368743936952, "upper_bound": "2653657459"}, {"distinct_range": 0, "num_eq": 90000, "num_range": 2586362391405526014, "upper_bound": "3474501193"}, {"distinct_range": 0, "num_eq": 800000000, "num_range": 300000000, "upper_bound": "3723430235"}, {"distinct_range": 1142968488453371800, "num_eq": 8731593937393423714, "num_range": 2824333607827857634, "upper_bound": "3954015706"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 1516314759636052843, "row_count": 7091472164550442120}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3160067054702173160, "histo_buckets": [{"distinct_range": 0, "num_eq": 6665469481491487019, "num_range": 0, "upper_bound": "566431761"}, {"distinct_range": 96808.00433869165, "num_eq": 100000, "num_range": 500000, "upper_bound": "2276733952"}, {"distinct_range": 6522846500738100000, "num_eq": 400000, "num_range": 6522846500738100539, "upper_bound": "2434057510"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 7091472164550442120}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2409934456934972538, "histo_buckets": [{"distinct_range": 0, "num_eq": 229032158328890764, "num_range": 0, "upper_bound": "-1.4597626138274236"}, {"distinct_range": 0, "num_eq": 5833554854046319420, "num_range": 3445391907567865563, "upper_bound": "-1.1343121982634221"}, {"distinct_range": 80000, "num_eq": 8335009588347153958, "num_range": 80000, "upper_bound": "-0.809677329325801"}, {"distinct_range": 3000000000, "num_eq": 1000000000, "num_range": 3000000000, "upper_bound": "-0.19594362409142752"}, {"distinct_range": 0, "num_eq": 4795915639353455226, "num_range": 90000000, "upper_bound": "1.81873968814672"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 7091472164550442120}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3907188915544907128, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000000, "num_range": 0, "upper_bound": "09:35:08.207007+08:03:00"}, {"distinct_range": 0, "num_eq": 700, "num_range": 200, "upper_bound": "17:51:41.587713+13:58:00"}, {"distinct_range": 3176379088434759000, "num_eq": 4966920014974486051, "num_range": 3176379088434759342, "upper_bound": "14:32:57.989452+10:15:00"}, {"distinct_range": 23.03188782830031, "num_eq": 7747202929935560479, "num_range": 30, "upper_bound": "01:37:03.898253-02:51:00"}, {"distinct_range": 60000000000, "num_eq": 1204336216717792216, "num_range": 60000000000, "upper_bound": "03:37:54.193081-01:58:00"}, {"distinct_range": 0, "num_eq": 2000000, "num_range": 3298463347000964752, "upper_bound": "18:14:22.666903+09:54:00"}, {"distinct_range": 1141707190796676700, "num_eq": 60, "num_range": 1761471924090227612, "upper_bound": "14:54:31.678653-02:16:00"}, {"distinct_range": 900000, "num_eq": 677689396473491507, "num_range": 900000, "upper_bound": "14:30:44.795197-03:38:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 5118484628670659156, "row_count": 7091472164550442120}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 885052632271821448, "histo_buckets": [{"distinct_range": 0, "num_eq": 4239521224812367263, "num_range": 0, "upper_bound": "-2146810588"}, {"distinct_range": 411040.3338978069, "num_eq": 7297262366678064322, "num_range": 700000, "upper_bound": "-1562216106"}, {"distinct_range": 600000000, "num_eq": 681237453630230852, "num_range": 600000000, "upper_bound": "-32768"}, {"distinct_range": 0, "num_eq": 4884830984107214865, "num_range": 0, "upper_bound": "226558654"}, {"distinct_range": 70000000, "num_eq": 4279570709469479158, "num_range": 70000000, "upper_bound": "663481741"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 7085410886850013200, "row_count": 7091472164550442120}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1991921732741804525, "histo_col_type": "", "name": "__auto__", "null_count": 1828961633187848959, "row_count": 7091472164550442120}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3552773620410399979, "histo_buckets": [{"distinct_range": 0, "num_eq": 6617279817368415716, "num_range": 0, "upper_bound": "1970-01-06 21:55:04.000299"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 10, "upper_bound": "1977-07-06 10:26:52.000299"}, {"distinct_range": 0, "num_eq": 40000000000, "num_range": 976731867436200162, "upper_bound": "1981-05-13 08:32:57.000489"}, {"distinct_range": 0, "num_eq": 800000, "num_range": 0, "upper_bound": "1990-04-13 13:09:22.000752"}, {"distinct_range": 1596473919838079200, "num_eq": 5000000000, "num_range": 2007187205261640623, "upper_bound": "2009-06-21 04:18:50.000321"}, {"distinct_range": 400, "num_eq": 983516920942066936, "num_range": 400, "upper_bound": "2016-07-27 10:36:50.000688"}, {"distinct_range": 0, "num_eq": 300000, "num_range": 1000000000, "upper_bound": "2025-11-13 21:37:24.0001"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 2180882372834119547, "row_count": 7091472164550442120}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6730480742071499563, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7091472164550442120}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 274, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 669, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "-Infinity"}, {"distinct_range": 0, "num_eq": 206030188005067787, "num_range": 100, "upper_bound": "-9.098011440675178401E+32"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 3425422778115131636, "upper_bound": "-3.096451762225344135E+27"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 300, "upper_bound": "-5.659209621940537604E+24"}, {"distinct_range": 1360338895766391600, "num_eq": 2000000, "num_range": 2691574326910882382, "upper_bound": "-671280434411213.1721"}, {"distinct_range": 0, "num_eq": 20000000000, "num_range": 3634079927957603950, "upper_bound": "-55633003.54198729588"}, {"distinct_range": 9000, "num_eq": 0, "num_range": 9000, "upper_bound": "74848.31950768915553"}, {"distinct_range": 582763052528089900, "num_eq": 600000000, "num_range": 582763052528089909, "upper_bound": "363744722704854.4718"}, {"distinct_range": 4550107858113415700, "num_eq": 6000000, "num_range": 4585300250649476020, "upper_bound": "511288808537045.1245"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "265313366"}, {"distinct_range": 9315.177525996416, "num_eq": 0, "num_range": 40000, "upper_bound": "1004023910"}, {"distinct_range": 700000000, "num_eq": 1008789502702863840, "num_range": 700000000, "upper_bound": "1293054368"}, {"distinct_range": 2000000, "num_eq": 3245677020512330063, "num_range": 2000000, "upper_bound": "2656996147"}, {"distinct_range": 60000, "num_eq": 7000, "num_range": 60000, "upper_bound": "3038232344"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 419, "histo_col_type": "", "name": "__auto__", "null_count": 160, "row_count": 900}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 531, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 726, "histo_col_type": "", "name": "__auto__", "null_count": 314, "row_count": 900}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 115, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 346, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 620, "histo_col_type": "", "name": "__auto__", "null_count": 374, "row_count": 900}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 589, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 288, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 396, "histo_col_type": "", "name": "__auto__", "null_count": 477, "row_count": 900}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 882, "histo_col_type": "", "name": "__auto__", "null_count": 79, "row_count": 900}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 280, "histo_col_type": "", "name": "__auto__", "null_count": 282, "row_count": 900}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 609, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 250, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 534, "histo_col_type": "", "name": "__auto__", "null_count": 291, "row_count": 900}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 264, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 261, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 610, "histo_buckets": [{"distinct_range": 0, "num_eq": 5527643056483152135, "num_range": 0, "upper_bound": "761589846"}, {"distinct_range": 8986673053401678000, "num_eq": 3000000000, "num_range": 8986673053401677490, "upper_bound": "1216279164"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 9000000000, "upper_bound": "1538585341"}, {"distinct_range": 5192365394504739000, "num_eq": 8553940600442642288, "num_range": 5192365394504738319, "upper_bound": "2927407606"}, {"distinct_range": 1000, "num_eq": 2942774805819523676, "num_range": 1000, "upper_bound": "3059005549"}, {"distinct_range": 0, "num_eq": 5000000000, "num_range": 400000, "upper_bound": "3292629945"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 0, "upper_bound": "3900071777"}, {"distinct_range": 7683780193112883000, "num_eq": 3107849094428670559, "num_range": 7683780193112882978, "upper_bound": "4188205038"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 185, "row_count": 900}, {"columns": ["col3_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 741, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 508, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('fiqipx', 'cno', 'fcsew');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 337814808, "histo_buckets": [{"distinct_range": 0, "num_eq": 2188275556015390828, "num_range": 0, "upper_bound": "1105099900"}, {"distinct_range": 3688419419135085000, "num_eq": 2155136498687961877, "num_range": 5317592813604134342, "upper_bound": "1654789999"}, {"distinct_range": 0, "num_eq": 400, "num_range": 4995609865148530327, "upper_bound": "2003419820"}, {"distinct_range": 0, "num_eq": 200000, "num_range": 8768754180168340205, "upper_bound": "2592430511"}, {"distinct_range": 0, "num_eq": 2986095671202052604, "num_range": 70000000000, "upper_bound": "4247020098"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 485941476, "row_count": 1000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 226163548, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 126473832, "histo_buckets": [{"distinct_range": 0, "num_eq": 445892149753045256, "num_range": 0, "upper_bound": "-260324835130475302.8"}, {"distinct_range": 3066889472659546000, "num_eq": 3658112456609155632, "num_range": 6259006775709999467, "upper_bound": "-5440682141061163.199"}, {"distinct_range": 900, "num_eq": 7798371110115343300, "num_range": 900, "upper_bound": "-450559112474.067106"}, {"distinct_range": 284175106066614530, "num_eq": 6000000, "num_range": 3258667469414601134, "upper_bound": "-1"}, {"distinct_range": 0, "num_eq": 6758366987507208727, "num_range": 100000, "upper_bound": "-0.01315746422418329655"}, {"distinct_range": 1485.8800168838438, "num_eq": 90, "num_range": 8000, "upper_bound": "2.691173217194411729E+30"}, {"distinct_range": 0, "num_eq": 2339641683061284187, "num_range": 60000, "upper_bound": "6.4219206908973782E+37"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 1000000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 909865, "histo_buckets": [{"distinct_range": 0, "num_eq": 7156293812581747757, "num_range": 0, "upper_bound": "1976-07-05 22:37:06.000964"}, {"distinct_range": 0, "num_eq": 20000000, "num_range": 7000, "upper_bound": "1978-07-31 14:19:29.000083"}, {"distinct_range": 90000, "num_eq": 6369600734554884112, "num_range": 90000, "upper_bound": "1980-06-18 22:40:18.000673"}, {"distinct_range": 0, "num_eq": 30, "num_range": 0, "upper_bound": "1988-11-07 18:21:05.000657"}, {"distinct_range": 3564075356075999000, "num_eq": 10000, "num_range": 3564075356075999181, "upper_bound": "1997-03-11 08:40:40.000579"}, {"distinct_range": 1000000000, "num_eq": 400000, "num_range": 1000000000, "upper_bound": "2009-01-25 20:20:11.000845"}, {"distinct_range": 705355170.2565422, "num_eq": 2000000, "num_range": 1000000000, "upper_bound": "2024-10-16 21:17:20.000439"}, {"distinct_range": 0, "num_eq": 8000, "num_range": 6505387862456028540, "upper_bound": "2026-07-13 11:09:01.000417"}, {"distinct_range": 7117989198681147000, "num_eq": 10000, "num_range": 7117989198681147716, "upper_bound": "3000-01-01 00:00:00"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 1701633, "row_count": 7000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4498919, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1276177, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6364275, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 7192422893302212000, "num_eq": 6000000000, "num_range": 8096170291439590545, "upper_bound": "00:58:02.161939+09:44:00"}, {"distinct_range": 0, "num_eq": 4407995084654887045, "num_range": 7100216567240935728, "upper_bound": "11:35:27.729243+11:20:00"}, {"distinct_range": 2756.6470078405673, "num_eq": 6724523639584773154, "num_range": 6000, "upper_bound": "14:05:19.349251+09:34:00"}, {"distinct_range": 4732488355486990000, "num_eq": 487464426924850607, "num_range": 4732488355486990050, "upper_bound": "22:52:38.593379+11:10:00"}, {"distinct_range": 0, "num_eq": 3156977389071370121, "num_range": 20000, "upper_bound": "14:03:21.152096+01:57:00"}, {"distinct_range": 19.66090032668426, "num_eq": 200000000, "num_range": 20, "upper_bound": "12:29:17.89717-02:36:00"}, {"distinct_range": 50000000000, "num_eq": 2992455172550162115, "num_range": 50000000000, "upper_bound": "18:08:58.282217+00:28:00"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 2061743566414107440, "upper_bound": "18:01:04.32089+00:16:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 2494123, "row_count": 7000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6797058, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4072137, "histo_col_type": "", "name": "__auto__", "null_count": 6408289, "row_count": 7000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6502404, "histo_buckets": [{"distinct_range": 0, "num_eq": 3398025657105755535, "num_range": 0, "upper_bound": "1094018002"}, {"distinct_range": 5739434828708465000, "num_eq": 5625213509704917058, "num_range": 5739434828708464487, "upper_bound": "1390236067"}, {"distinct_range": 0, "num_eq": 600, "num_range": 80000000, "upper_bound": "1549145122"}, {"distinct_range": 10000, "num_eq": 0, "num_range": 10000, "upper_bound": "1774342753"}, {"distinct_range": 284746921.3016558, "num_eq": 7000000, "num_range": 700000000, "upper_bound": "2456602808"}, {"distinct_range": 0, "num_eq": 1610710890182756712, "num_range": 700000, "upper_bound": "3320258127"}, {"distinct_range": 510746377518013400, "num_eq": 40000, "num_range": 741338474371053946, "upper_bound": "3491931063"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 6631658, "row_count": 7000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2359024, "histo_col_type": "", "name": "__auto__", "null_count": 3098829, "row_count": 7000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2781282, "histo_buckets": [{"distinct_range": 0, "num_eq": 300000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 50000000, "num_range": 11093212483505786, "upper_bound": "1433275905"}, {"distinct_range": 4471565564027347000, "num_eq": 5000000000, "num_range": 4471565564027346879, "upper_bound": "3138675376"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3056092, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "570661402"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 2121648, "row_count": 7000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2725913, "histo_buckets": [{"distinct_range": 0, "num_eq": 2428501418237373569, "num_range": 0, "upper_bound": "-0.9382029119581188"}, {"distinct_range": 0, "num_eq": 200, "num_range": 6843905513922229462, "upper_bound": "-0.37020937659042213"}, {"distinct_range": 0, "num_eq": 5090997329056598298, "num_range": 80000, "upper_bound": "-0.3279359737020803"}, {"distinct_range": 4665805376521957000, "num_eq": 40, "num_range": 4665805376521956873, "upper_bound": "1.3604402340672872"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6732942, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4490411, "histo_buckets": [{"distinct_range": 0, "num_eq": 4385031501200069314, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 1757625665130427789, "upper_bound": "1265175536"}, {"distinct_range": 20000, "num_eq": 7223836355070666882, "num_range": 20000, "upper_bound": "1752290164"}, {"distinct_range": 1983986734857173000, "num_eq": 200000, "num_range": 1983986734857172892, "upper_bound": "3816900802"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 421770, "histo_buckets": [{"distinct_range": 0, "num_eq": 281139717334062893, "num_range": 0, "upper_bound": "3.287424277527988877E+37"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 7000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 585693295432064423, "histo_col_type": "", "name": "__auto__", "null_count": 5397333194479511533, "row_count": 7380811950069908608}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1691960145394960206, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7380811950069908608}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4051431938183298169, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7380811950069908608}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 270941999149353729, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7380811950069908608}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5470725618852363922, "histo_col_type": "", "name": "__auto__", "null_count": 5409994462196543759, "row_count": 7380811950069908608}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2392406839335210441, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "188544470"}, {"distinct_range": 1000000000, "num_eq": 3214115867948503017, "num_range": 1000000000, "upper_bound": "453190978"}, {"distinct_range": 400277.3248045431, "num_eq": 60000000000, "num_range": 1000000, "upper_bound": "1719923074"}, {"distinct_range": 3479842870778980000, "num_eq": 8198296076631217823, "num_range": 6434325010349283479, "upper_bound": "1787065108"}, {"distinct_range": 2291850088341553400, "num_eq": 50000, "num_range": 3979890506070164595, "upper_bound": "2249264271"}, {"distinct_range": 4941028930877378000, "num_eq": 500000, "num_range": 4941028930877377725, "upper_bound": "3373407948"}, {"distinct_range": 30000000000, "num_eq": 1054786066538950187, "num_range": 30000000000, "upper_bound": "3380057331"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 7380811950069908608}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5482830485641394501, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7380811950069908608}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7373427477674008314, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7380811950069908608}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4046189479496611989, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7380811950069908608}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1615222038390617811, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7380811950069908608}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5618683932849919098, "histo_col_type": "", "name": "__auto__", "null_count": 3278279461971391091, "row_count": 7380811950069908608}, {"columns": ["col3_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3753408336873733350, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "514964805"}, {"distinct_range": 0, "num_eq": 8689693889448034405, "num_range": 100000000000, "upper_bound": "937236157"}, {"distinct_range": 1928258399579726800, "num_eq": 5181690888020199253, "num_range": 1928258399579726896, "upper_bound": "2332516551"}, {"distinct_range": 2512117009954127000, "num_eq": 30000000, "num_range": 5590418965155589218, "upper_bound": "4143657533"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 606286983718657373, "row_count": 7380811950069908608}, {"columns": ["col3_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5950154873094226058, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7380811950069908608}, {"columns": ["col3_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4593424410260200844, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7380811950069908608}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3876726365597988676, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7380811950069908608}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2484501928553356032, "histo_col_type": "", "name": "__auto__", "null_count": 845373036287049624, "row_count": 7380811950069908608}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4360453858931239875, "histo_col_type": "", "name": "__auto__", "null_count": 4413228543746081417, "row_count": 7380811950069908608}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6143143752323461591, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "-7.219482212464312672E+33"}, {"distinct_range": 2096415671061184500, "num_eq": 8504187674280293163, "num_range": 2096415671061184433, "upper_bound": "-4.08309496826826318E+33"}, {"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": "-7.225319229008950551"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 7380811950069908608}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6363966903266278533, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7380811950069908608}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1350455475599046805, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7380811950069908608}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3886016551822769598, "histo_col_type": "", "name": "__auto__", "null_count": 1251458519815153444, "row_count": 7380811950069908608}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6484701605547703188, "histo_col_type": "", "name": "__auto__", "null_count": 197774158544171398, "row_count": 7380811950069908608}, {"columns": ["col3_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2902113484497722407, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7380811950069908608}]';
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_4) REFERENCES table3 (col2_4) ON UPDATE RESTRICT;
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4850568957351732053, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "915411664"}, {"distinct_range": 1019448094743279600, "num_eq": 4271408310443283805, "num_range": 6763571876899755175, "upper_bound": "1376405401"}, {"distinct_range": 1000000000, "num_eq": 6536565550220852821, "num_range": 1000000000, "upper_bound": "3131001697"}, {"distinct_range": 23804.923460707294, "num_eq": 3877090303903704959, "num_range": 100000, "upper_bound": "3245780612"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 6691163039699595767, "row_count": 7912619593874510007}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 650537993426875044, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7912619593874510007}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13213227475510239, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000000, "num_range": 0, "upper_bound": "-3.441668321158104047E+29"}, {"distinct_range": 5053413630.664155, "num_eq": 70000000, "num_range": 20000000000, "upper_bound": "-7.09954606358458911E+28"}, {"distinct_range": 4996780726414429000, "num_eq": 4665165996490802657, "num_range": 4996780726414429325, "upper_bound": "-1412117836002938.856"}, {"distinct_range": 2000, "num_eq": 7646228935364312497, "num_range": 2000, "upper_bound": "-3934600.549254006112"}, {"distinct_range": 4159074719000795000, "num_eq": 60000000000, "num_range": 5661575157635344274, "upper_bound": "-6.183313128519269959"}, {"distinct_range": 703855583745052700, "num_eq": 8084856670737338662, "num_range": 4737256634956472024, "upper_bound": "1"}, {"distinct_range": 0, "num_eq": 2314577478725200261, "num_range": 500, "upper_bound": "43.41285755301679941"}, {"distinct_range": 1191668870148376800, "num_eq": 3000, "num_range": 2068207154413073093, "upper_bound": "1682.004522728940657"}, {"distinct_range": 8040532634941038000, "num_eq": 0, "num_range": 8040532634941037914, "upper_bound": "9.171929298117498936E+30"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 7912619593874510007}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6445480388638375232, "histo_buckets": [{"distinct_range": 0, "num_eq": 8497780480226076134, "num_range": 0, "upper_bound": "-0.6997914460527684"}, {"distinct_range": 8534990721128057000, "num_eq": 80, "num_range": 8534990721128056865, "upper_bound": "-0.31847256290842085"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 8137967471502228899}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3764944160287653865, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8137967471502228899}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6730584719957364594, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "01:36:12.31574+14:26:00"}, {"distinct_range": 3154978668570867700, "num_eq": 5000000, "num_range": 3154978668570867794, "upper_bound": "04:50:05.544965+13:25:00"}, {"distinct_range": 0, "num_eq": 7710137813487787027, "num_range": 90000000, "upper_bound": "19:32:25.257493+07:19:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 6521473778581428278, "row_count": 8137967471502228899}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2483014617488996097, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "-1919852032"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 2841632716675201886, "row_count": 8137967471502228899}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3468147946945587670, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8137967471502228899}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5952464283215422103, "histo_col_type": "", "name": "__auto__", "null_count": 881923272540214518, "row_count": 8137967471502228899}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6628590681132304924, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8137967471502228899}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1108350959855094346, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "-692040729089.1992594"}, {"distinct_range": 866403870837283300, "num_eq": 3000000, "num_range": 7205602333191330412, "upper_bound": "-5370170196.969133225"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 70000000000, "upper_bound": "-3457.975917807232318"}, {"distinct_range": 1332848007816438800, "num_eq": 3217858339388251786, "num_range": 1332848007816438842, "upper_bound": "369.5780472848170329"}, {"distinct_range": 0, "num_eq": 4000000000, "num_range": 1000, "upper_bound": "5881451498391.847825"}, {"distinct_range": 478517376075826370, "num_eq": 6137690558604935091, "num_range": 478517376075826389, "upper_bound": "33400915377262476.79"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 1927511815798103159, "upper_bound": "2.107508883544690219E+36"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 8137967471502228899}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1602041398449405430, "histo_col_type": "", "name": "__auto__", "null_count": 6387093035288813170, "row_count": 8137967471502228899}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 558340866225661887, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "0001-01-01 00:00:00"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 4680264551339856068, "upper_bound": "1980-05-30 18:17:58.000795"}, {"distinct_range": 485760434.70650333, "num_eq": 4303595255372038051, "num_range": 1000000000, "upper_bound": "1989-11-24 07:40:13.000068"}, {"distinct_range": 1111284570252459000, "num_eq": 500000000, "num_range": 1111284570252458988, "upper_bound": "2021-11-02 12:36:46.000647"}, {"distinct_range": 4483103895162269700, "num_eq": 100, "num_range": 4483103895162269447, "upper_bound": "2022-03-07 15:57:45.000653"}, {"distinct_range": 0, "num_eq": 300000, "num_range": 10000000, "upper_bound": "294276-12-31 23:59:59.999999"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 4936412846373736892, "row_count": 8137967471502228899}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3046725023085754024, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8137967471502228899}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5707685654216790731, "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 2194925155685424039, "row_count": 8137967471502228899}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5300734050040586393, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 6335287999136757692, "num_range": 4000000000, "upper_bound": "1157924401"}, {"distinct_range": 1449347186424287500, "num_eq": 4837484665553651551, "num_range": 1449347186424287572, "upper_bound": "1272271015"}, {"distinct_range": 542088.1387589177, "num_eq": 100000000, "num_range": 700000, "upper_bound": "2204226305"}, {"distinct_range": 0, "num_eq": 0, "num_range": 9002470727154072762, "upper_bound": "2561844179"}, {"distinct_range": 7.046648805482256, "num_eq": 8236701876147752235, "num_range": 10, "upper_bound": "2616508663"}, {"distinct_range": 0, "num_eq": 3829219951298735011, "num_range": 7643525271196691170, "upper_bound": "3301571484"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 8137967471502228899}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7751699265929027800, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000, "num_range": 0, "upper_bound": "3892872388"}, {"distinct_range": 4561562314515583000, "num_eq": 70000000000, "num_range": 4561562314515583139, "upper_bound": "4055430638"}, {"distinct_range": 0, "num_eq": 50000, "num_range": 3041553482853086485, "upper_bound": "4221852017"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 8137967471502228899}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 153216913706237503, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000, "num_range": 0, "upper_bound": "-1.52063471473614108E+36"}, {"distinct_range": 339.1521679984309, "num_eq": 50, "num_range": 400, "upper_bound": "0"}, {"distinct_range": 51080751.31161626, "num_eq": 200000000, "num_range": 90000000, "upper_bound": "4.79207245972194217E+24"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 2488908805399056423}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 218781909341983991, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2488908805399056423}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1775043787479055754, "histo_col_type": "", "name": "__auto__", "null_count": 632867720721758588, "row_count": 2488908805399056423}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2238916124951778000, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 281404319109518940, "num_eq": 900, "num_range": 7510380966420686528, "upper_bound": "\\u00135"}, {"distinct_range": 0, "num_eq": 5547749912353727425, "num_range": 30, "upper_bound": "(\\u0016\\u0003\\u000e\\u0006\\u0019Tc"}, {"distinct_range": 600000000, "num_eq": 1000, "num_range": 600000000, "upper_bound": "+$\\u0007m-?\\u000eI"}, {"distinct_range": 0, "num_eq": 2260169343158247076, "num_range": 90000, "upper_bound": "X)c\\u000b\\u0013o:"}, {"distinct_range": 600000000, "num_eq": 0, "num_range": 600000000, "upper_bound": "h1Ceu\\u000fa2z"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 2488908805399056423}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 663102017849541962, "histo_col_type": "", "name": "__auto__", "null_count": 234035924194152686, "row_count": 2488908805399056423}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2482018531683859249, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2488908805399056423}, {"columns": ["col3_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 482413310157036066, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2488908805399056423}, {"columns": ["col3_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1405762006808043488, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2488908805399056423}, {"columns": ["col3_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 323004109857145252, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2488908805399056423}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 904489716882889922, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2488908805399056423}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 242970106982191422, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2488908805399056423}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1421059914981959369, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2488908805399056423}, {"columns": ["col3_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 654956188201473014, "histo_buckets": [{"distinct_range": 0, "num_eq": 1933221510130332134, "num_range": 0, "upper_bound": "824952560"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 10000000, "upper_bound": "1745256659"}, {"distinct_range": 0, "num_eq": 50000000, "num_range": 50000, "upper_bound": "1887306960"}, {"distinct_range": 838753373297956400, "num_eq": 10000000000, "num_range": 3486195931514511313, "upper_bound": "2580865052"}, {"distinct_range": 39362591894.965904, "num_eq": 7569440706464890049, "num_range": 100000000000, "upper_bound": "2725471014"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 873444533952173415, "row_count": 2488908805399056423}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 909675154736404565, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2488908805399056423}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1515799968285683021, "histo_col_type": "", "name": "__auto__", "null_count": 2321883532106943255, "row_count": 2488908805399056423}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1268106129355427824, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 30000, "num_eq": 5759116144215037405, "num_range": 30000, "upper_bound": "2759620615"}, {"distinct_range": 0, "num_eq": 1580362191864297697, "num_range": 0, "upper_bound": "3130993985"}, {"distinct_range": 7519587103.578717, "num_eq": 3000, "num_range": 8000000000, "upper_bound": "3655256811"}, {"distinct_range": 966620436202519300, "num_eq": 60000, "num_range": 3742248490856448404, "upper_bound": "4145711692"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 2488908805399056423}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 475683334230022170, "histo_col_type": "", "name": "__auto__", "null_count": 1705119256878044704, "row_count": 2488908805399056423}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1757886670321893002, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2488908805399056423}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1123357231878950612, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2488908805399056423}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 308581855827446931, "histo_col_type": "", "name": "__auto__", "null_count": 332510830652283646, "row_count": 2488908805399056423}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1252126905817027836, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2488908805399056423}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 707337127135485671, "histo_col_type": "", "name": "__auto__", "null_count": 23362827916335644, "row_count": 2488908805399056423}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2258022704124153860, "histo_col_type": "", "name": "__auto__", "null_count": 86294832618299037, "row_count": 2488908805399056423}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "infinity" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2535072&tab=buildLog) on [master@1a6ee6ebd9625f428568203205d0ae44e2ca1963](https://github.com/cockroachdb/cockroach/commits/1a6ee6ebd9625f428568203205d0ae44e2ca1963):

```
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_7, col2_0 DESC, col4_8) INCLUDE (col4_15);
        test_1        |             ALTER TABLE table4 ADD FOREIGN KEY (col2_0, col4_15) REFERENCES table1 (col1_0, col1_1) ON DELETE RESTRICT ON UPDATE CASCADE;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_0, col2_1, col2_3, col2_2) REFERENCES table1 (col1_0, col1_2, col1_1, col1_7) ON UPDATE CASCADE;
        test_1        |             ALTER TABLE table4 ADD FOREIGN KEY (col2_1, col2_2) REFERENCES table1 (col1_2, col1_7);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('pr');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('rzuoy', 'xxcuel', 'za');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('jxbyn', 'xe', 'mfcs', 'mrijm', 'jj', 'lmx');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('o', 'cnn', 'yu', 'ihmx', 'kbio', 'bqzjm');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: foreign key constraint "table4_col2_0_fkey" cannot be implemented (SQLSTATE 42804)
        test_1        |     --- FAIL: TestCompare/mutators (1.01s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 INTERVAL NOT NULL, col1_1 TIMETZ NOT NULL, col1_2 TIMESTAMPTZ NOT NULL, col1_3 BIT(47) NOT NULL, col1_4 REGPROCEDURE NOT NULL, col1_5 "char" NOT NULL, col1_6 BIT(20) NOT NULL, col1_7 UUID NOT NULL, PRIMARY KEY (col1_4, col1_7 DESC, col1_1 DESC, col1_3, col1_2 DESC, col1_0 DESC, col1_6 ASC, col1_5), INDEX (col1_1 DESC, col1_4 ASC, col1_0), UNIQUE (col1_1, col1_7 DESC, col1_5 ASC, col1_4, col1_6 ASC, col1_2 DESC, col1_3, col1_0 ASC), UNIQUE (col1_2, col1_5, col1_0, col1_1, col1_6 ASC, col1_4 DESC, col1_7 DESC) WHERE ((table1.col1_1 = '00:00:00+15:59:00':::TIMETZ) OR (table1.col1_2 != '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table1.col1_5 > e'\U00002603':::STRING), FAMILY (col1_4, col1_5, col1_7, col1_3), FAMILY (col1_1, col1_6), FAMILY (col1_2, col1_0));
        test_1        |             CREATE TABLE table2 (col2_0 REGPROC, col2_1 BOOL NOT NULL, col2_2 INTERVAL NOT NULL, col2_3 STRING NOT NULL, col2_4 TIMESTAMP NOT NULL, col2_5 INT8 NOT NULL, col2_6 VARCHAR, col2_7 BOX2D, col2_8 TIMETZ NULL, col2_9 TIMETZ, col2_10 TIMESTAMP NOT NULL, col2_11 CHAR NULL, col2_12 BIT(27) NULL, col2_13 NAME, col2_14 TIMETZ, col2_15 INT4[], col2_16 REGNAMESPACE, col2_17 REGPROC, col2_18 BYTES NULL, PRIMARY KEY (col2_4 DESC, col2_2 ASC), UNIQUE (col2_8 ASC, col2_7 DESC) STORING (col2_0, col2_1, col2_3, col2_9, col2_12, col2_13, col2_14, col2_15, col2_16), INDEX (col2_14 DESC, col2_5 ASC, col2_17 DESC, col2_2, col2_7, col2_1, col2_13 DESC, col2_3 ASC, col2_4 DESC, col2_6 DESC, col2_10 DESC, col2_12 DESC, col2_8, col2_0 DESC, col2_18 ASC) STORING (col2_11, col2_15, col2_16), UNIQUE (col2_16 ASC, col2_9 DESC, col2_3, col2_17 DESC, col2_0, col2_1, col2_2 ASC, col2_11 ASC, col2_13 DESC, col2_4 ASC, col2_18 ASC, col2_14 ASC, col2_6, col2_5 ASC, col2_8, col2_12 ASC) STORING (col2_10, col2_15) WHERE ((((((NOT table2.col2_1) OR (table2.col2_9 != '00:00:00+15:59:00':::TIMETZ)) OR (table2.col2_5 > 2147483647:::INT8)) AND (table2.col2_3 > '"':::STRING)) OR (table2.col2_18 >= e'\\xff':::BYTES)) OR (table2.col2_13 >= '"':::STRING)) OR (table2.col2_14 < '24:00:00-15:59:00':::TIMETZ), INDEX (col2_6 ASC, col2_0, col2_7 DESC, col2_10 DESC, col2_16 ASC, col2_5 DESC, col2_14 DESC, col2_3 ASC, col2_1, col2_12 ASC, col2_9, col2_8 ASC, col2_11 DESC, col2_4 DESC, col2_18, col2_2 DESC) STORING (col2_15) WHERE (((((((((((table2.col2_8 != '00:00:00+15:59:00') OR (table2.col2_9 = '24:00:00-15:59:00')) AND (table2.col2_6 >= '"')) AND (table2.col2_3 != e'\x00')) AND (table2.col2_5 > 1)) OR (table2.col2_11 != '"')) AND (table2.col2_14 < '24:00:00-15:59:00')) AND (table2.col2_4 <= '-4713-11-24 00:00:00')) AND (NOT table2.col2_1)) AND (table2.col2_18 > '\x58')) AND (table2.col2_10 != '294276-12-31 23:59:59.999999')) AND (table2.col2_13 <= 'X'), UNIQUE (col2_6 ASC, col2_4 ASC, col2_8 ASC, col2_3, col2_0 ASC, col2_17, col2_18 ASC, col2_2 DESC, col2_14 DESC, col2_16 DESC, col2_11 ASC, col2_12 DESC) STORING (col2_13, col2_15) WHERE table2.col2_9 != '24:00:00-15:59:00':::TIMETZ, INDEX (col2_8 DESC, col2_13 DESC, col2_7 ASC, col2_17 DESC, col2_10, col2_6 DESC, col2_16, col2_18 DESC, col2_1 DESC, col2_9 ASC, col2_14 ASC, col2_11, col2_2 ASC, col2_3, col2_12, col2_0, col2_5 DESC) WHERE (((((table2.col2_3 > e'\x00':::STRING) AND (table2.col2_14 > '00:00:00+15:59:00':::TIMETZ)) AND (table2.col2_6 > 'X':::STRING)) AND (table2.col2_9 >= '00:00:00+15:59:00':::TIMETZ)) AND (table2.col2_10 >= '-2000-01-01 00:00:00':::TIMESTAMP)) OR (NOT table2.col2_1), INDEX (col2_11, col2_18 DESC, col2_10, col2_6 ASC, col2_8 DESC) STORING (col2_0, col2_3, col2_5, col2_7, col2_12, col2_14, col2_15, col2_16) WHERE (((((((((((table2.col2_11 != e'\'':::STRING) OR (table2.col2_6 > '':::STRING)) OR (table2.col2_18 != e'\\x00':::BYTES)) OR (table2.col2_8 < '00:00:00+15:59:00':::TIMETZ)) OR (table2.col2_14 >= '24:00:00-15:59:00':::TIMETZ)) AND table2.col2_1) AND (table2.col2_5 >= (-1):::INT8)) AND (table2.col2_13 <= '"':::STRING)) OR (table2.col2_4 < '294276-12-31 23:59:59.999999':::TIMESTAMP)) OR (table2.col2_3 = e'\U00002603':::STRING)) OR (table2.col2_9 < '24:00:00-15:59:00':::TIMETZ)) AND (table2.col2_10 > '3000-01-01 00:00:00':::TIMESTAMP), INDEX (col2_12 DESC, col2_14 DESC, col2_13 ASC, col2_17 ASC, col2_5 DESC, col2_0, col2_8, col2_11 ASC, col2_2 DESC, col2_4 DESC, col2_18 DESC, col2_9) STORING (col2_3, col2_7, col2_16) WHERE (((((((((((table2.col2_4 < '-2000-01-01 00:00:00':::TIMESTAMP) AND (table2.col2_8 = '24:00:00-15:59:00':::TIMETZ)) OR (table2.col2_9 >= '00:00:00+15:59:00':::TIMETZ)) OR (table2.col2_6 > e'\'':::STRING)) OR (table2.col2_14 >= '00:00:00+15:59:00':::TIMETZ)) OR (table2.col2_10 = '-2000-01-01 00:00:00':::TIMESTAMP)) AND table2.col2_1) AND (table2.col2_11 = '':::STRING)) OR (table2.col2_3 >= '"':::STRING)) OR (table2.col2_5 >= 2147483647:::INT8)) OR (table2.col2_13 <= e'\U00002603':::STRING)) OR (table2.col2_18 != e'\\xe29883':::BYTES), INDEX (col2_9 DESC, col2_5 DESC, col2_1 DESC, col2_10 ASC, col2_2, col2_3 DESC, col2_13, col2_7, col2_18 DESC, col2_6 ASC, col2_12) STORING (col2_0, col2_11, col2_14, col2_15, col2_17), FAMILY (col2_16), FAMILY (col2_11, col2_9), FAMILY (col2_8, col2_12), FAMILY (col2_4), FAMILY (col2_14, col2_3, col2_17), FAMILY (col2_15), FAMILY (col2_7), FAMILY (col2_0), FAMILY (col2_6), FAMILY (col2_13), FAMILY (col2_5), FAMILY (col2_2), FAMILY (col2_10), FAMILY (col2_18), FAMILY (col2_1));
        test_1        |             CREATE TABLE table3 (col3_0 INT8 NOT NULL, col3_1 BOX2D NOT NULL, col3_2 BYTES, col3_3 VARCHAR, col3_4 OID NOT NULL, PRIMARY KEY (col3_1, col3_4 DESC, col3_0), UNIQUE (col3_1 DESC, col3_2 DESC) WHERE (table3.col3_2 > e'\\xff':::BYTES) OR (table3.col3_3 < '"':::STRING), UNIQUE (col3_3, col3_0 DESC, col3_1 DESC, col3_2 DESC) WHERE ((table3.col3_0 != 2147483647:::INT8) AND (table3.col3_3 != '':::STRING)) AND (table3.col3_2 <= e'\\x27':::BYTES), INDEX (col3_0 DESC, col3_2, col3_1 DESC, col3_4 DESC, col3_3 ASC) WHERE ((table3.col3_2 > e'\\x00':::BYTES) OR (table3.col3_0 >= 2147483647:::INT8)) AND (table3.col3_3 != e'\'':::STRING), UNIQUE (col3_3 DESC, col3_4 DESC) WHERE ((table3.col3_3 < '"':::STRING) AND (table3.col3_0 = (-1):::INT8)) OR (table3.col3_2 > e'\\xff':::BYTES), INDEX (col3_4 DESC, col3_0, col3_1 ASC) STORING (col3_3) WHERE (table3.col3_2 != e'\\x58':::BYTES) OR (table3.col3_0 = (-128):::INT8), INDEX (col3_3 DESC, col3_2 DESC) WHERE table3.col3_3 != 'X':::STRING, UNIQUE (col3_2, col3_4) STORING (col3_3) WHERE (table3.col3_3 > e'\'':::STRING) OR (table3.col3_0 < 0:::INT8), FAMILY (col3_2, col3_4, col3_3, col3_1), FAMILY (col3_0));
        test_1        |             CREATE TABLE table4 (col4_0 INTERVAL, col4_1 TIMETZ NOT NULL, col4_2 VARCHAR NOT NULL, col4_3 CHAR, col4_4 FLOAT8 NOT NULL, col4_5 "char" NULL, col4_6 UUID, col4_7 BOX2D, col4_8 "char" NOT NULL, col4_9 CHAR, col4_10 VARCHAR NOT NULL, col4_11 REGCLASS NULL, col4_12 BIT(31) NULL, col4_13 INET NULL, col4_14 TIMESTAMPTZ NOT NULL, col4_15 TIMETZ NOT NULL, col4_16 INET, PRIMARY KEY (col4_4 DESC, col4_10 ASC), UNIQUE (col4_10) STORING (col4_0, col4_1, col4_2, col4_5, col4_8, col4_9, col4_12) WHERE ((((((table4.col4_1 = '24:00:00-15:59:00':::TIMETZ) AND (table4.col4_14 < '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table4.col4_2 = e'\x00':::STRING)) AND (table4.col4_8 != '"':::STRING)) AND (table4.col4_15 > '00:00:00+15:59:00':::TIMETZ)) AND (table4.col4_9 >= e'\U00002603':::STRING)) AND (table4.col4_10 <= e'\'':::STRING), INDEX (col4_9 ASC, col4_11 ASC, col4_16 ASC, col4_0 DESC, col4_13 ASC, col4_6 DESC, col4_7 DESC, col4_8 ASC, col4_2 DESC, col4_15) STORING (col4_3) WHERE ((((((table4.col4_1 < '00:00:00+15:59:00':::TIMETZ) AND (table4.col4_2 >= e'\x00':::STRING)) AND (table4.col4_9 != e'\U00002603':::STRING)) AND (table4.col4_15 = '00:00:00+15:59:00':::TIMETZ)) AND (table4.col4_8 != e'\x00':::STRING)) AND (table4.col4_5 < e'\'':::STRING)) OR (table4.col4_10 != e'\'':::STRING), UNIQUE (col4_5, col4_3 ASC, col4_1, col4_13 DESC, col4_9 ASC, col4_2, col4_6 DESC, col4_0 ASC, col4_15 ASC, col4_12 DESC, col4_11 ASC, col4_14 ASC, col4_7 ASC) STORING (col4_8, col4_16) WHERE ((((((table4.col4_4 >= 3.4028234663852886e+38:::FLOAT8) AND (table4.col4_5 != '"':::STRING)) OR (table4.col4_2 > e'\'':::STRING)) OR (table4.col4_15 != '00:00:00+15:59:00':::TIMETZ)) OR (table4.col4_14 <= '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) AND (table4.col4_8 < 'X':::STRING)) AND (table4.col4_1 != '00:00:00+15:59:00':::TIMETZ), FAMILY (col4_9, col4_8), FAMILY (col4_3), FAMILY (col4_4), FAMILY (col4_1), FAMILY (col4_5), FAMILY (col4_6), FAMILY (col4_7), FAMILY (col4_14, col4_11, col4_0, col4_10, col4_15, col4_2, col4_12), FAMILY (col4_13), FAMILY (col4_16), UNIQUE (col4_0, col4_12) STORING (col4_7, col4_8, col4_9, col4_13, col4_14, col4_15));
        test_1        |             CREATE TABLE table5 (col4_4 FLOAT8 NOT NULL, col4_10 VARCHAR NOT NULL, col5_2 INT2 NOT NULL, col5_3 DATE NULL, col5_4 BIT(20) NOT NULL, col5_5 BIT(44), PRIMARY KEY (col4_4 DESC, col4_10 ASC, col5_2 ASC, col5_4 ASC), UNIQUE (col4_10) STORING (col5_3, col5_5) WHERE (((table5.col4_4 = 0.0) AND (table5.col5_2 >= -1)) OR (table5.col5_3 < '-infinity')) AND (table5.col4_10 != '"'), INDEX (col4_10 DESC, col5_5 DESC, col5_3 DESC, col5_2 ASC), UNIQUE (col4_4, col5_3 ASC, col4_10, col5_2) STORING (col5_5) WHERE (((table5.col4_4 > -Inf) AND (table5.col4_10 = '')) OR (table5.col5_3 >= 'infinity')) OR (table5.col5_2 = 0), INDEX (col4_10, col5_3 DESC, col5_4 DESC) WHERE (table5.col4_4 > 3.4028234663852886e+38:::FLOAT8) OR (table5.col5_3 >= '-infinity':::DATE), UNIQUE (col4_10, col4_4 ASC, col5_2, col5_3 DESC, col5_4) STORING (col5_5) WHERE table5.col5_3 != '1970-01-01':::DATE, INDEX (col5_2, col5_3 ASC, col4_4 ASC, col5_5 ASC, col4_10, col5_4), UNIQUE (col5_4, col5_3 DESC) WHERE (((table5.col4_4 > -1.0) AND (table5.col5_3 != '4714-11-24 BC')) AND (table5.col5_2 >= -1)) OR (table5.col4_10 != '"'), FAMILY (col5_5), FAMILY (col5_4, col5_2), FAMILY (col4_4), FAMILY (col5_3, col4_10)) INTERLEAVE IN PARENT table4 (col4_4, col4_10);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 889028983, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 683138722, "histo_buckets": [{"distinct_range": 0, "num_eq": 65912705595846087, "num_range": 0, "upper_bound": "07:51:03.263658-00:11:00"}, {"distinct_range": 589863.5294504735, "num_eq": 200000, "num_range": 10000000, "upper_bound": "17:31:19.401711+04:35:00"}, {"distinct_range": 3707607643.1105103, "num_eq": 0, "num_range": 7000000000, "upper_bound": "17:39:49.081091+04:36:00"}, {"distinct_range": 54913979762.19062, "num_eq": 6683980845087275578, "num_range": 80000000000, "upper_bound": "23:29:19.809973+08:24:00"}, {"distinct_range": 5974247664.931226, "num_eq": 10000, "num_range": 6000000000, "upper_bound": "18:58:02.285688-07:09:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 900000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 731079968, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000, "num_range": 0, "upper_bound": "1970-09-06 12:09:19.000794+00:00"}, {"distinct_range": 2642047170601220600, "num_eq": 6924563438020243903, "num_range": 5143195726961284016, "upper_bound": "1992-01-26 06:36:14.000777+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 900000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14316071, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 551765289, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 209887523725483000, "num_eq": 6355311191602679636, "num_range": 632534295861958109, "upper_bound": "1063695270"}, {"distinct_range": 0, "num_eq": 7071533074030455388, "num_range": 6058079423979272503, "upper_bound": "1332164244"}, {"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1914582088"}, {"distinct_range": 1409292938600512000, "num_eq": 2508874164567760837, "num_range": 6772726878665103196, "upper_bound": "2351369022"}, {"distinct_range": 393.098452472075, "num_eq": 1000000000, "num_range": 4000, "upper_bound": "3779083319"}, {"distinct_range": 0, "num_eq": 5000, "num_range": 7000000000, "upper_bound": "4064008312"}, {"distinct_range": 0, "num_eq": 0, "num_range": 5384793219973261942, "upper_bound": "4189639902"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 900000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 549828713, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 422993038, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 790258306, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8404624, "histo_col_type": "", "name": "__auto__", "null_count": 134818463, "row_count": 700000000}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 435501817, "histo_col_type": "", "name": "__auto__", "null_count": 566797697, "row_count": 700000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 661967336, "histo_col_type": "", "name": "__auto__", "null_count": 125043867, "row_count": 700000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22168399, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 118368999, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 192106774, "histo_col_type": "CHAR", "name": "__auto__", "null_count": 55828905, "row_count": 700000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 443951259, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 296378106, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 505907348, "histo_buckets": [{"distinct_range": 0, "num_eq": 31519053164919032, "num_range": 0, "upper_bound": "00:58:16.027726-00:14:00"}, {"distinct_range": 105895976.07370463, "num_eq": 6862955535443496189, "num_range": 300000000, "upper_bound": "01:10:32.893718-08:18:00"}, {"distinct_range": 7000, "num_eq": 2622378811253747514, "num_range": 7000, "upper_bound": "19:37:19.688255+00:22:00"}, {"distinct_range": 100000000000, "num_eq": 7034472524169931808, "num_range": 100000000000, "upper_bound": "18:30:49.616888-00:55:00"}, {"distinct_range": 500000, "num_eq": 5000000, "num_range": 500000, "upper_bound": "17:08:45.50218-03:56:00"}, {"distinct_range": 0, "num_eq": 1254842560501874241, "num_range": 6962048995051905561, "upper_bound": "12:23:19.589589-10:51:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 155123171, "row_count": 700000000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 588127533, "histo_buckets": [{"distinct_range": 0, "num_eq": 8548980929160643505, "num_range": 0, "upper_bound": "04:34:15.905321+13:54:00"}, {"distinct_range": 0, "num_eq": 6689964633820097855, "num_range": 5394193210992603780, "upper_bound": "03:30:38.292805+06:13:00"}, {"distinct_range": 73916371269203580, "num_eq": 10000, "num_range": 73916371269203583, "upper_bound": "06:47:46.922575+08:27:00"}, {"distinct_range": 500000, "num_eq": 10000000000, "num_range": 500000, "upper_bound": "17:42:43.546936+11:03:00"}, {"distinct_range": 400000000, "num_eq": 60, "num_range": 400000000, "upper_bound": "00:26:10.79477-08:38:00"}, {"distinct_range": 0, "num_eq": 5793553410213676185, "num_range": 5000000000, "upper_bound": "17:05:38.24187+02:30:00"}, {"distinct_range": 2331166.304127041, "num_eq": 6027094773455083426, "num_range": 3000000, "upper_bound": "23:20:58.9417+06:26:00"}, {"distinct_range": 1000, "num_eq": 8000000, "num_range": 1000, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 268902527, "row_count": 700000000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 518461413, "histo_col_type": "", "name": "__auto__", "null_count": 387127241, "row_count": 700000000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 204504529, "histo_buckets": [{"distinct_range": 0, "num_eq": 7590082405414248046, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 6000000000, "num_eq": 7975369306354361639, "num_range": 6000000000, "upper_bound": "133378025"}, {"distinct_range": 100000000, "num_eq": 5652009563680253985, "num_range": 100000000, "upper_bound": "498174962"}, {"distinct_range": 7835681987911016000, "num_eq": 2631179775356898314, "num_range": 7835681987911016725, "upper_bound": "959533598"}, {"distinct_range": 0, "num_eq": 4965234591332019163, "num_range": 20, "upper_bound": "2724446695"}, {"distinct_range": 0, "num_eq": 600, "num_range": 0, "upper_bound": "3119932173"}, {"distinct_range": 5555949241349394000, "num_eq": 40000000000, "num_range": 6117915724558420068, "upper_bound": "3959140222"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 526286605, "row_count": 700000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 219202179, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1970-10-29 18:01:17.000289"}, {"distinct_range": 5949181936388273000, "num_eq": 60000, "num_range": 5949181936388273480, "upper_bound": "1992-06-15 07:42:25.000735"}, {"distinct_range": 106512.07003269184, "num_eq": 0, "num_range": 500000, "upper_bound": "1995-09-26 19:48:09.000926"}, {"distinct_range": 50, "num_eq": 800000000, "num_range": 50, "upper_bound": "2001-04-08 07:15:16.000367"}, {"distinct_range": 300000, "num_eq": 400000000, "num_range": 300000, "upper_bound": "2009-03-01 22:45:09.00008"}, {"distinct_range": 4984178552463153000, "num_eq": 90, "num_range": 6739063215288357417, "upper_bound": "2021-01-25 10:44:37.000995"}, {"distinct_range": 10, "num_eq": 6080359347320367870, "num_range": 10, "upper_bound": "2032-09-12 20:29:34.000042"}, {"distinct_range": 0, "num_eq": 5282453243461845444, "num_range": 7914221599154149026, "upper_bound": "294276-12-31 23:59:59.999999"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 338327572, "histo_col_type": "", "name": "__auto__", "null_count": 146359358, "row_count": 700000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 363119625, "histo_col_type": "", "name": "__auto__", "null_count": 457311714, "row_count": 700000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 219117586, "histo_buckets": [{"distinct_range": 0, "num_eq": 4684855339414382560, "num_range": 0, "upper_bound": "000000000000000000000000000"}, {"distinct_range": 3445519866.908468, "num_eq": 800000000, "num_range": 7000000000, "upper_bound": "010001101010111001101100000"}, {"distinct_range": 907882951653531500, "num_eq": 90000, "num_range": 907882951653531564, "upper_bound": "100111011110001000001100010"}, {"distinct_range": 100000000, "num_eq": 10000000000, "num_range": 100000000, "upper_bound": "111000110000100101100101001"}], "histo_col_type": "BIT(27)", "name": "__auto__", "null_count": 448112550, "row_count": 700000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 374454495, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 323015276, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": ""}, {"distinct_range": 2191322440632215300, "num_eq": 8574880219350824560, "num_range": 8877387466021107733, "upper_bound": "hp\\u0008*]z\\u001a\\u0007"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 438241897, "row_count": 700000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 219299943, "histo_buckets": [{"distinct_range": 0, "num_eq": 1058564021610157558, "num_range": 0, "upper_bound": "20:58:43.613434-01:13:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 382207024, "row_count": 700000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 797, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": ">\\t\\nDr3c&+"}, {"distinct_range": 0, "num_eq": 10, "num_range": 60000000000, "upper_bound": "mt\\u000c\\u001e"}, {"distinct_range": 1610278734686953200, "num_eq": 22677180007847434, "num_range": 1610278734686953173, "upper_bound": "os"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 2051, "row_count": 4000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3826, "histo_buckets": [{"distinct_range": 0, "num_eq": 6444738770252168541, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 37060165071884290, "num_eq": 5000000, "num_range": 1793514635348290132, "upper_bound": "1279418525"}, {"distinct_range": 1497713980378468400, "num_eq": 70000, "num_range": 4149686910136666380, "upper_bound": "2065145924"}, {"distinct_range": 1234706564058386400, "num_eq": 40000, "num_range": 1234706564058386450, "upper_bound": "2442552272"}, {"distinct_range": 2577314751382113300, "num_eq": 900000000, "num_range": 5466968999842614789, "upper_bound": "4164533283"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2445, "histo_buckets": [{"distinct_range": 0, "num_eq": 4343004716725203067, "num_range": 0, "upper_bound": "-9223372036854775807"}, {"distinct_range": 0, "num_eq": 500000000, "num_range": 9000, "upper_bound": "-8767538603606677780"}, {"distinct_range": 50000, "num_eq": 2000, "num_range": 50000, "upper_bound": "-5382180215250275633"}, {"distinct_range": 10000, "num_eq": 0, "num_range": 10000, "upper_bound": "-4868036854108020467"}, {"distinct_range": 3481262524265230.5, "num_eq": 100000, "num_range": 2057150021146499223, "upper_bound": "-1912929840073514995"}, {"distinct_range": 90000, "num_eq": 30000, "num_range": 90000, "upper_bound": "-1844552277291694549"}, {"distinct_range": 300000, "num_eq": 4717424308828448075, "num_range": 300000, "upper_bound": "-2147483648"}, {"distinct_range": 1200215145661014000, "num_eq": 7228215878505287208, "num_range": 4451664999153561389, "upper_bound": "127"}, {"distinct_range": 80000000000, "num_eq": 2710698690899859786, "num_range": 80000000000, "upper_bound": "8515296479416747855"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3635, "histo_buckets": [{"distinct_range": 0, "num_eq": 4991309533453693106, "num_range": 0, "upper_bound": "BOX(0.24418669489488876 -1.8836894814762677,1.5040389111415813 1.1847688333168018)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3413, "histo_buckets": [{"distinct_range": 0, "num_eq": 6534125553381923409, "num_range": 0, "upper_bound": "\\\\x33aff0"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 123520520637150586, "upper_bound": "\\\\x58"}, {"distinct_range": 0, "num_eq": 0, "num_range": 4729869975666747934, "upper_bound": "\\\\xf9cee6ddd3"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 1072, "row_count": 4000}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2315703528778121369, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4551245838993861363}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 155717225893827908, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4551245838993861363}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 500935064722311188, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4551245838993861363}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1622191481063911620, "histo_col_type": "", "name": "__auto__", "null_count": 1735210949943842488, "row_count": 4551245838993861363}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3001690665722402425, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": "\\t)y0$n\\u0014O|"}, {"distinct_range": 3856480816409488400, "num_eq": 20000000000, "num_range": 3856480816409488534, "upper_bound": "X"}, {"distinct_range": 0, "num_eq": 8975538625128723474, "num_range": 0, "upper_bound": "s>L"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 4551245838993861363}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3555711002628912643, "histo_col_type": "", "name": "__auto__", "null_count": 756067232678910759, "row_count": 4551245838993861363}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4152086251702116533, "histo_col_type": "", "name": "__auto__", "null_count": 3579064085932322379, "row_count": 4551245838993861363}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3406174269458477428, "histo_col_type": "", "name": "__auto__", "null_count": 1368123147748301417, "row_count": 4551245838993861363}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2523185784357004606, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000000, "num_range": 0, "upper_bound": "-0.6732613461615613"}, {"distinct_range": 2055040387811321600, "num_eq": 7543971257904112316, "num_range": 2289955442086684453, "upper_bound": "0.057738231362934433"}, {"distinct_range": 2678.1361356220714, "num_eq": 3634002823689218883, "num_range": 5000, "upper_bound": "0.837231788227804"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 4551245838993861363}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2483415675453667475, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 7815942991855048934, "num_range": 30, "upper_bound": "B"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 2074723447621755243, "row_count": 4551245838993861363}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4056368122366783895, "histo_col_type": "", "name": "__auto__", "null_count": 21670456781129499, "row_count": 4551245838993861363}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4321126867044937367, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "\\u0017"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 10000000, "upper_bound": "4"}, {"distinct_range": 0, "num_eq": 0, "num_range": 9000000000, "upper_bound": "8"}, {"distinct_range": 1512811669837900000, "num_eq": 600000000, "num_range": 1638574552640187696, "upper_bound": "N"}, {"distinct_range": 0, "num_eq": 8447845374598602399, "num_range": 8577622517625027022, "upper_bound": "V"}, {"distinct_range": 135845.29872509817, "num_eq": 1223746570267506649, "num_range": 400000, "upper_bound": "l"}, {"distinct_range": 20000000000, "num_eq": 1835587173679702238, "num_range": 20000000000, "upper_bound": "t"}, {"distinct_range": 300000, "num_eq": 0, "num_range": 300000, "upper_bound": "w"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 2623417344253704345, "row_count": 4551245838993861363}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 414818236408997005, "histo_col_type": "", "name": "__auto__", "null_count": 2548779551471545164, "row_count": 4551245838993861363}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2393171167529342244, "histo_col_type": "", "name": "__auto__", "null_count": 3966883514811441578, "row_count": 4551245838993861363}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3808273069341873612, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4551245838993861363}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1278733241314568152, "histo_col_type": "", "name": "__auto__", "null_count": 434961068260880416, "row_count": 4551245838993861363}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3540823598485707388, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4551245838993861363}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS e'[{"columns": ["col5_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 656117277793815759, "histo_col_type": "", "name": "__auto__", "null_count": 5987782618074631782, "row_count": 6072397671791229763}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4871074841967961023, "histo_buckets": [{"distinct_range": 0, "num_eq": 7713139942675837088, "num_range": 0, "upper_bound": "-1.2931009709959902"}, {"distinct_range": 561579734343248000, "num_eq": 9140251549955054273, "num_range": 7438535487753382224, "upper_bound": "-0.08412014165455195"}, {"distinct_range": 1000000, "num_eq": 6458346174236018210, "num_range": 1000000, "upper_bound": "5e-324"}, {"distinct_range": 8.289352851030957, "num_eq": 7529406984601404564, "num_range": 20, "upper_bound": "0.23823931844662888"}, {"distinct_range": 5469603794222614000, "num_eq": 7059042751925732585, "num_range": 5469603794222613318, "upper_bound": "0.6587388870662372"}, {"distinct_range": 3367836.553675416, "num_eq": 3000000, "num_range": 9000000, "upper_bound": "1.468296443317177"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 6072397671791229763}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3328322142085184216, "histo_buckets": [{"distinct_range": 0, "num_eq": 9015984130062496349, "num_range": 0, "upper_bound": "\\u001a"}, {"distinct_range": 0, "num_eq": 3593134756665865139, "num_range": 20, "upper_bound": "\\u001ePS"}, {"distinct_range": 872945037750902000, "num_eq": 40000000000, "num_range": 872945037750902064, "upper_bound": "8"}, {"distinct_range": 0, "num_eq": 9000000000, "num_range": 5000000, "upper_bound": "DG1\\u0017V"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 6072397671791229763}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1678064555571388920, "histo_buckets": [{"distinct_range": 0, "num_eq": 8315844412108791680, "num_range": 0, "upper_bound": "-21209"}, {"distinct_range": 2572402844188648400, "num_eq": 5005783397924520997, "num_range": 2572402844188648251, "upper_bound": "-3497"}, {"distinct_range": 10.84909188464352, "num_eq": 0, "num_range": 30, "upper_bound": "19433"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 5605689692644412979, "upper_bound": "21450"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 1988724707864933692, "upper_bound": "25463"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 6072397671791229763}, {"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4084713364350992213, "histo_col_type": "", "name": "__auto__", "null_count": 176748651300443102, "row_count": 6072397671791229763}, {"columns": ["col5_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3654754985132772381, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000, "num_range": 0, "upper_bound": "00001100011001111000"}, {"distinct_range": 0, "num_eq": 40, "num_range": 2982493153781759530, "upper_bound": "10011010101011111001"}, {"distinct_range": 3639399372694045700, "num_eq": 700000000, "num_range": 3639399372694045706, "upper_bound": "10011111010010101011"}, {"distinct_range": 8707313746159117000, "num_eq": 60000, "num_range": 8707313746159117273, "upper_bound": "11011100110110111110"}], "histo_col_type": "BIT(20)", "name": "__auto__", "null_count": 0, "row_count": 6072397671791229763}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('obfok', 'dah', 'exv');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('pan', 'dvbquh', 'lxexe', 'lxtlt', 'hkyue', 'zgdjqj');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('e');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 7018334304164415242, "num_range": 0, "upper_bound": "0"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 413243280544337548, "num_range": 0, "upper_bound": "09:33:19.50654+09:35:00"}, {"distinct_range": 11454.560258271093, "num_eq": 400, "num_range": 60000, "upper_bound": "12:03:05.358338+05:54:00"}, {"distinct_range": 1000000000, "num_eq": 9025319835179703863, "num_range": 1000000000, "upper_bound": "19:53:52.772274+00:48:00"}, {"distinct_range": 0, "num_eq": 60000000000, "num_range": 300000, "upper_bound": "22:36:22.164343+01:33:00"}, {"distinct_range": 844721841.2370539, "num_eq": 2256849627817301640, "num_range": 7000000000, "upper_bound": "13:27:16.051926-08:32:00"}, {"distinct_range": 60466.263947109546, "num_eq": 200000000, "num_range": 80000, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000, "num_range": 0, "upper_bound": "1970-07-17 02:22:56.000041+00:00"}, {"distinct_range": 6000000000, "num_eq": 7182019358182066340, "num_range": 6000000000, "upper_bound": "1990-12-05 19:47:51.000312+00:00"}, {"distinct_range": 300, "num_eq": 503399694630889670, "num_range": 300, "upper_bound": "2004-10-22 08:17:56.000797+00:00"}, {"distinct_range": 1460624229981411800, "num_eq": 1777490631089352604, "num_range": 5843643501593987867, "upper_bound": "2006-12-07 10:57:24.000583+00:00"}, {"distinct_range": 82980411995.16826, "num_eq": 700, "num_range": 100000000000, "upper_bound": "2015-07-28 04:36:31.000077+00:00"}, {"distinct_range": 0, "num_eq": 30000000000, "num_range": 9146186035626761340, "upper_bound": "2033-02-12 06:59:24.000933+00:00"}, {"distinct_range": 3572217240273919000, "num_eq": 5950223825749164756, "num_range": 3729067230543566235, "upper_bound": "294276-12-31 23:59:59.999999+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3313, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11217, "histo_buckets": [{"distinct_range": 0, "num_eq": 2958884343116356262, "num_range": 0, "upper_bound": "1984-10-04 11:43:51.000339"}, {"distinct_range": 83237.1867780229, "num_eq": 8742091462578877898, "num_range": 100000, "upper_bound": "1989-02-14 15:42:58.000467"}, {"distinct_range": 0, "num_eq": 7320480204759103756, "num_range": 100000, "upper_bound": "1989-03-05 16:59:57.000073"}, {"distinct_range": 0, "num_eq": 4254711908601577502, "num_range": 30000, "upper_bound": "1989-07-25 04:57:57.000858"}, {"distinct_range": 0, "num_eq": 3453886781276829647, "num_range": 5844243998478682930, "upper_bound": "1989-07-30 13:16:27.000994"}, {"distinct_range": 248.17285144188895, "num_eq": 3694963334870183279, "num_range": 400, "upper_bound": "1998-06-01 00:19:42.000628"}, {"distinct_range": 6624580274158664000, "num_eq": 4702979233281783955, "num_range": 6624580274158663190, "upper_bound": "2025-12-02 06:03:03.0008"}, {"distinct_range": 2655819065247722500, "num_eq": 30, "num_range": 2655819065247722654, "upper_bound": "2030-05-02 02:16:23.000282"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7899, "histo_buckets": [{"distinct_range": 0, "num_eq": 8349049165900662445, "num_range": 0, "upper_bound": "\\u001b"}, {"distinct_range": 0, "num_eq": 70, "num_range": 3986119662661370564, "upper_bound": "."}, {"distinct_range": 3646878967756667400, "num_eq": 0, "num_range": 5547058909398677271, "upper_bound": "7"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 18797, "row_count": 20000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16911, "histo_col_type": "", "name": "__auto__", "null_count": 9152, "row_count": 20000}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7891, "histo_col_type": "", "name": "__auto__", "null_count": 14679, "row_count": 20000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5019, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15233, "histo_buckets": [{"distinct_range": 0, "num_eq": 5898481172433091121, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 800000, "num_eq": 0, "num_range": 800000, "upper_bound": "10:11:13.992721+09:58:00"}, {"distinct_range": 9000, "num_eq": 10, "num_range": 9000, "upper_bound": "08:27:24.775098+01:37:00"}, {"distinct_range": 0.49583314029989245, "num_eq": 2122558139448550595, "num_range": 20, "upper_bound": "22:53:34.922066+12:43:00"}, {"distinct_range": 1000000000, "num_eq": 2133449792410694372, "num_range": 1000000000, "upper_bound": "01:25:55.440335-09:04:00"}, {"distinct_range": 7000, "num_eq": 1000000000, "num_range": 7000, "upper_bound": "02:18:47.640668-13:08:00"}, {"distinct_range": 0, "num_eq": 2000000, "num_range": 1000, "upper_bound": "12:49:49.217457-05:10:00"}, {"distinct_range": 2136460.749665964, "num_eq": 5161467565219413146, "num_range": 5000000, "upper_bound": "18:13:09.096548-05:26:00"}, {"distinct_range": 2322562199536302600, "num_eq": 600000, "num_range": 5700339060952212628, "upper_bound": "14:32:04.36113-12:45:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 18871, "row_count": 20000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 942, "histo_col_type": "", "name": "__auto__", "null_count": 1966, "row_count": 20000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4419, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16936, "histo_buckets": [{"distinct_range": 0, "num_eq": 5129392285440976479, "num_range": 0, "upper_bound": "\\u0006"}, {"distinct_range": 0, "num_eq": 4000000, "num_range": 6318897492058454210, "upper_bound": "<"}, {"distinct_range": 8542574404898594000, "num_eq": 6000000, "num_range": 8542574404898593485, "upper_bound": "L3W\\u0008\\u0006Y@M1"}, {"distinct_range": 0, "num_eq": 0, "num_range": 800000000, "upper_bound": "V6}{LM"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "`rS. /"}, {"distinct_range": 900000, "num_eq": 10000000, "num_range": 900000, "upper_bound": "c"}, {"distinct_range": 6000, "num_eq": 4461881966941600590, "num_range": 6000, "upper_bound": "k\\"B"}, {"distinct_range": 1895797523982571800, "num_eq": 400000000, "num_range": 7739866755411888192, "upper_bound": "v=[p"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 4920, "row_count": 20000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17831, "histo_buckets": [{"distinct_range": 0, "num_eq": 4026789324371366473, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 7412055197107964000, "num_eq": 1000000, "num_range": 7412055197107963398, "upper_bound": "00:25:31.570883-00:03:00"}, {"distinct_range": 0, "num_eq": 100, "num_range": 50000000000, "upper_bound": "06:07:08.876744+05:35:00"}, {"distinct_range": 19805660.637261048, "num_eq": 60, "num_range": 20000000, "upper_bound": "21:30:27.485313+12:25:00"}, {"distinct_range": 6685927525705637000, "num_eq": 8936448446786768996, "num_range": 6685927525705636964, "upper_bound": "12:04:45.983595+01:15:00"}, {"distinct_range": 408861633.7748601, "num_eq": 5000, "num_range": 900000000, "upper_bound": "23:39:56.805208+05:00:00"}, {"distinct_range": 315139726663185860, "num_eq": 8782635482486519079, "num_range": 481266922076766908, "upper_bound": "13:35:55.052505-13:10:00"}, {"distinct_range": 0, "num_eq": 7413325962000534306, "num_range": 70, "upper_bound": "20:35:22.595488-13:46:00"}, {"distinct_range": 9000, "num_eq": 217048167661068955, "num_range": 9000, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 5126, "row_count": 20000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18659, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "09:52:24.624072+09:53:00"}, {"distinct_range": 6180012716079783000, "num_eq": 0, "num_range": 6180012716079782562, "upper_bound": "08:53:36.167386+07:01:00"}, {"distinct_range": 8000000, "num_eq": 8509849584727494034, "num_range": 8000000, "upper_bound": "08:57:02.702446+06:41:00"}, {"distinct_range": 656294995036387500, "num_eq": 10000000000, "num_range": 2568370199287523782, "upper_bound": "15:42:44.906325+05:08:00"}, {"distinct_range": 60, "num_eq": 0, "num_range": 60, "upper_bound": "06:51:07.372987-08:30:00"}, {"distinct_range": 0, "num_eq": 291948808564260045, "num_range": 1000, "upper_bound": "22:52:19.123049+05:45:00"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 50000000, "upper_bound": "12:50:23.891488-12:50:00"}, {"distinct_range": 5665137118.398429, "num_eq": 7344573627715496921, "num_range": 100000000000, "upper_bound": "16:28:06.02121-12:05:00"}, {"distinct_range": 0, "num_eq": 800, "num_range": 0, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 2471, "row_count": 20000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6620, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8808, "histo_col_type": "", "name": "__auto__", "null_count": 14738, "row_count": 20000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8496, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3274, "histo_buckets": [{"distinct_range": 0, "num_eq": 80, "num_range": 0, "upper_bound": "011011110010011111100001011"}, {"distinct_range": 8199743134226082000, "num_eq": 1836358856793443342, "num_range": 8199743134226081839, "upper_bound": "011101000010101000101000111"}, {"distinct_range": 400000, "num_eq": 40000000000, "num_range": 400000, "upper_bound": "011111111111111111111111111"}, {"distinct_range": 10000000000, "num_eq": 0, "num_range": 10000000000, "upper_bound": "100110001110010011011110010"}, {"distinct_range": 3258212123954914300, "num_eq": 0, "num_range": 4458499719639472623, "upper_bound": "101110011011100111111100000"}, {"distinct_range": 1431272567598659800, "num_eq": 100000000, "num_range": 5648963958527831753, "upper_bound": "110111000001101010111100111"}, {"distinct_range": 0, "num_eq": 400, "num_range": 118150838791777576, "upper_bound": "111111111000001100010110111"}], "histo_col_type": "BIT(27)", "name": "__auto__", "null_count": 3533, "row_count": 20000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12167, "histo_col_type": "", "name": "__auto__", "null_count": 9024, "row_count": 20000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19313, "histo_buckets": [{"distinct_range": 0, "num_eq": 6549843180764936192, "num_range": 0, "upper_bound": "1657399331"}, {"distinct_range": 99.8632160651205, "num_eq": 1000, "num_range": 700, "upper_bound": "2318884442"}, {"distinct_range": 83824327234647540, "num_eq": 0, "num_range": 1020345573654886639, "upper_bound": "2673781549"}, {"distinct_range": 2940971199584719400, "num_eq": 6043713267650440546, "num_range": 6377092227859835612, "upper_bound": "2676283440"}, {"distinct_range": 0, "num_eq": 9000000, "num_range": 7936609202289322835, "upper_bound": "3106911129"}, {"distinct_range": 3662099178240034300, "num_eq": 800000000, "num_range": 3662099178240034346, "upper_bound": "3227147597"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "3413047258"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 6496, "row_count": 20000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8917, "histo_col_type": "", "name": "__auto__", "null_count": 5230, "row_count": 20000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3853943826, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 665715259228012800, "num_eq": 2201652605527484156, "num_range": 4469260721273130370, "upper_bound": "634354507"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 500000, "upper_bound": "1686801463"}, {"distinct_range": 0, "num_eq": 3057901594919369366, "num_range": 0, "upper_bound": "2522276598"}, {"distinct_range": 0, "num_eq": 8392324630296428038, "num_range": 5052775815984347197, "upper_bound": "2739985914"}, {"distinct_range": 0, "num_eq": 3099826617663714747, "num_range": 0, "upper_bound": "3016223126"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 40000000000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23653318725, "histo_buckets": [{"distinct_range": 0, "num_eq": 70, "num_range": 0, "upper_bound": "7946843573961759751"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 40000000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4774491615, "histo_buckets": [{"distinct_range": 0, "num_eq": 1706071803566448284, "num_range": 0, "upper_bound": "BOX(-0.5549184240538955 -1.275097674736228,0.4875822333593409 -0.4634691529619239)"}, {"distinct_range": 3702195687.2046647, "num_eq": 80000000, "num_range": 10000000000, "upper_bound": "BOX(-0.44296790943611697 -0.5222929545051535,2.2030730794110345 0.775705489466272)"}, {"distinct_range": 0, "num_eq": 600, "num_range": 5688622526661104155, "upper_bound": "BOX(-0.268292460558921 0.10028293629364021,1.0288518148037151 1.1108388023637064)"}, {"distinct_range": 2160874497457341000, "num_eq": 7328440458320343170, "num_range": 7336102135541496285, "upper_bound": "BOX(0.1745205547333481 -0.8014958360468132,0.40298290620799737 0.6479844568797487)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 40000000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 31577972153, "histo_buckets": [{"distinct_range": 0, "num_eq": 3372021548036043426, "num_range": 0, "upper_bound": "\\\\x58"}, {"distinct_range": 2008323441216224000, "num_eq": 5772137903096533045, "num_range": 4559341707474465363, "upper_bound": "\\\\x63"}, {"distinct_range": 9000000000, "num_eq": 8640790431765798399, "num_range": 9000000000, "upper_bound": "\\\\x671c0bfd8b7f"}, {"distinct_range": 30000, "num_eq": 100000000000, "num_range": 30000, "upper_bound": "\\\\x87b8edacf750b40c25"}, {"distinct_range": 8657541986437385000, "num_eq": 3621709634780042905, "num_range": 8657541986437385227, "upper_bound": "\\\\x98bd3b7d"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 3842742262940633836, "upper_bound": "\\\\xbab046f0"}, {"distinct_range": 7829452006196617000, "num_eq": 30000, "num_range": 7829452006196617431, "upper_bound": "\\\\xc660fdf1"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 39395441086, "row_count": 40000000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4230133700, "histo_buckets": [{"distinct_range": 0, "num_eq": 735246478158946175, "num_range": 0, "upper_bound": ""}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 29492235615, "row_count": 40000000000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8442, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1256, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2888, "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9063, "histo_col_type": "", "name": "__auto__", "null_count": 5585, "row_count": 10000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3775, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "-0.35748841083812777"}, {"distinct_range": 0, "num_eq": 7503792350918748837, "num_range": 4805166597903155399, "upper_bound": "0.0751209935815309"}, {"distinct_range": 0, "num_eq": 4727922562344631190, "num_range": 1000000, "upper_bound": "0.4521771083894274"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6221, "histo_col_type": "", "name": "__auto__", "null_count": 72, "row_count": 10000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7806, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "\\u001f"}, {"distinct_range": 4735151450169613000, "num_eq": 1000000000, "num_range": 4735151450169613625, "upper_bound": "."}, {"distinct_range": 817378729.9885266, "num_eq": 1000, "num_range": 1000000000, "upper_bound": "P"}, {"distinct_range": 61303694.09944453, "num_eq": 7867052995803558639, "num_range": 3000000000, "upper_bound": "X"}, {"distinct_range": 0, "num_eq": 7487904091386824331, "num_range": 1082164772181830373, "upper_bound": "h"}, {"distinct_range": 4434410063233061400, "num_eq": 3000000, "num_range": 6008992270376553652, "upper_bound": "u"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 2710, "row_count": 10000}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1168, "histo_col_type": "", "name": "__auto__", "null_count": 7058, "row_count": 10000}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1170, "histo_col_type": "", "name": "__auto__", "null_count": 7061, "row_count": 10000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6011, "histo_buckets": [{"distinct_range": 0, "num_eq": 5915966110149776122, "num_range": 0, "upper_bound": "\\u0006"}, {"distinct_range": 18509041.44790972, "num_eq": 10000, "num_range": 50000000, "upper_bound": "."}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 825, "row_count": 10000}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2268, "histo_col_type": "", "name": "__auto__", "null_count": 1222, "row_count": 10000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5503, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4190, "histo_col_type": "", "name": "__auto__", "null_count": 222, "row_count": 10000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4007, "histo_col_type": "", "name": "__auto__", "null_count": 6524, "row_count": 10000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2179, "histo_col_type": "", "name": "__auto__", "null_count": 7989, "row_count": 10000}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4494, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5562, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}]';
        test_1        |             ALTER TABLE table5 INJECT STATISTICS e'[{"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21073374065, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000000, "num_range": 0, "upper_bound": "-1.3425849927577636"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 40000000000, "upper_bound": "0.4088208237858707"}, {"distinct_range": 80000, "num_eq": 2618544457279113098, "num_range": 80000, "upper_bound": "1.5082703301408822"}, {"distinct_range": 3570277927867656000, "num_eq": 700000, "num_range": 3867427191359304574, "upper_bound": "1.5227288541632888"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9833288612, "histo_buckets": [{"distinct_range": 0, "num_eq": 8037591708918445670, "num_range": 0, "upper_bound": ""}, {"distinct_range": 5832598038590321000, "num_eq": 40000, "num_range": 5832598038590320713, "upper_bound": "\\u0003"}, {"distinct_range": 5446015997426295000, "num_eq": 800000, "num_range": 5446015997426294345, "upper_bound": "\\n5\\u0007"}, {"distinct_range": 743087024009125500, "num_eq": 30000000, "num_range": 2840179737713936815, "upper_bound": "\\u0019\\u0006\\u0001ZY"}, {"distinct_range": 0, "num_eq": 90000, "num_range": 0, "upper_bound": "d(U0\\u0018\\u0006"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6666986874, "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8085284295, "histo_col_type": "", "name": "__auto__", "null_count": 15498495527, "row_count": 30000000000}, {"columns": ["col5_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23698297805, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000000, "num_range": 0, "upper_bound": "00101010111010110111"}, {"distinct_range": 4130620.7934601884, "num_eq": 2346381489517420134, "num_range": 5000000, "upper_bound": "01000101110111010000"}, {"distinct_range": 100, "num_eq": 7048421327020950843, "num_range": 100, "upper_bound": "01000111011010110111"}, {"distinct_range": 1917010208440371000, "num_eq": 9088709655023550821, "num_range": 2402964985582764638, "upper_bound": "10001001111001011001"}, {"distinct_range": 28.189907633

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2538661&tab=buildLog) on [master@cd62a02833eeb02d14944668d09ee54806927a2b](https://github.com/cockroachdb/cockroach/commits/cd62a02833eeb02d14944668d09ee54806927a2b):

```
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        test_1        | --- FAIL: TestCompare (2.50s)
        test_1        |     --- FAIL: TestCompare/postgres (1.76s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.73s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 NAME NOT NULL, col1_1 REGPROC NOT NULL, col1_2 CHAR, col1_3 DATE NOT NULL, col1_4 STRING NULL, col1_5 BIT(20), col1_6 TIMESTAMP NOT NULL, col1_7 DECIMAL NULL, UNIQUE (col1_7 DESC, col1_3 ASC) STORING (col1_0, col1_1, col1_2) WHERE (table1.col1_4 >= 'X') AND (table1.col1_2 <= e'\U00002603'), INDEX (col1_3, col1_4 DESC, col1_0 ASC, col1_7, col1_6 ASC) STORING (col1_2, col1_5) WHERE (table1.col1_3 < 'infinity') OR (table1.col1_7 != NaN), UNIQUE (col1_0 ASC, col1_7, col1_1 ASC, col1_5 DESC, col1_2 DESC, col1_6 DESC, col1_3 ASC) STORING (col1_4) WHERE (((((table1.col1_6 < '-2000-01-01 00:00:00':::TIMESTAMP) OR (table1.col1_4 != '':::STRING)) AND (table1.col1_3 < '4714-11-24 BC':::DATE)) OR (table1.col1_2 <= '"':::STRING)) OR (table1.col1_0 = '"':::STRING)) AND (table1.col1_7 = 'NaN':::DECIMAL), INDEX (col1_1, col1_5 DESC, col1_6, col1_3 DESC, col1_4 ASC, col1_0) WHERE (((((table1.col1_0 = '':::STRING) OR (table1.col1_2 >= e'\x00':::STRING)) AND (table1.col1_3 >= '-infinity':::DATE)) OR (table1.col1_6 <= '294276-12-31 23:59:59.999999':::TIMESTAMP)) AND (table1.col1_7 > 0:::DECIMAL)) AND (table1.col1_4 >= 'X':::STRING), FAMILY (col1_2, col1_1), FAMILY (col1_3, col1_4), FAMILY (col1_6, col1_7, col1_5, col1_0));
        test_1        |             CREATE TABLE table2 (col2_0 INT2 NOT NULL, col2_1 INT8, col2_2 REGNAMESPACE NOT NULL, col2_3 TIMESTAMPTZ, col2_4 REGNAMESPACE NOT NULL, col2_5 TIME NOT NULL, PRIMARY KEY (col2_2 DESC), UNIQUE (col2_5 DESC, col2_1, col2_4 ASC, col2_2, col2_0, col2_3) WHERE (((table2.col2_5 >= '24:00:00') OR (table2.col2_1 <= 9223372036854775807)) OR (table2.col2_0 != 1)) AND (table2.col2_3 >= '0001-01-01 00:00:00+00:00'), FAMILY (col2_0, col2_3, col2_5, col2_4, col2_1), FAMILY (col2_2));
        test_1        |             CREATE TABLE table3 (col3_0 REGTYPE NOT NULL, col3_1 UUID, col3_2 VARCHAR[], col3_3 STRING, col3_4 TIMESTAMPTZ, col3_5 DECIMAL NULL, col3_6 OID NULL, col3_7 BOOL, col3_8 DATE NOT NULL, col3_9 BYTES NOT NULL, PRIMARY KEY (col3_8, col3_0), UNIQUE (col3_4 ASC, col3_5 DESC) STORING (col3_1, col3_2, col3_3) WHERE (((((table3.col3_4 <= '-2000-01-01 00:00:00+00:00') AND (table3.col3_3 <= 'X')) OR (NOT table3.col3_7)) AND (table3.col3_9 > '\xe29883')) AND (table3.col3_5 <= 1)) OR (table3.col3_8 < '-infinity'), INDEX (col3_6, col3_5 DESC, col3_9, col3_1 DESC) STORING (col3_4, col3_7), FAMILY (col3_9), FAMILY (col3_4), FAMILY (col3_3), FAMILY (col3_6, col3_2), FAMILY (col3_8), FAMILY (col3_7), FAMILY (col3_0), FAMILY (col3_1, col3_5));
        test_1        |             CREATE TABLE table4 (col4_0 TIMESTAMP NOT NULL, col4_1 "char" NOT NULL, col4_2 DECIMAL, col4_3 VARCHAR, col4_4 VARCHAR, col4_5 FLOAT8, col4_6 REGNAMESPACE NOT NULL, col4_7 REGPROC NULL, col4_8 REGPROCEDURE NOT NULL, col4_9 INT4 NOT NULL, col4_10 REGPROC NULL, col4_11 REGNAMESPACE NULL, col4_12 BIT(39), col4_13 TIME, PRIMARY KEY (col4_6 ASC, col4_9 DESC, col4_0 DESC), FAMILY (col4_1), FAMILY (col4_10, col4_8, col4_5, col4_13, col4_6), FAMILY (col4_7, col4_4, col4_9), FAMILY (col4_12), FAMILY (col4_0, col4_3, col4_11), FAMILY (col4_2));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1899306822842186130, "histo_col_type": "", "name": "__auto__", "null_count": 2398468863337797055, "row_count": 5324615128060268650}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4045508735285532954, "histo_buckets": [{"distinct_range": 0, "num_eq": 8660368165219175240, "num_range": 0, "upper_bound": "1996-08-11"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 5324615128060268650}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1152102235841874677, "histo_col_type": "", "name": "__auto__", "null_count": 5058568409945995191, "row_count": 5324615128060268650}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 120567884988100028, "histo_col_type": "", "name": "__auto__", "null_count": 2110853366707166977, "row_count": 5324615128060268650}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1416663623344406480, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5324615128060268650}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2772588149183570874, "histo_buckets": [{"distinct_range": 0, "num_eq": 30, "num_range": 0, "upper_bound": "-8.508164918192031524E+28"}, {"distinct_range": 0, "num_eq": 200000000, "num_range": 0, "upper_bound": "-4.141010663950998442E+23"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 1546513124582050705, "upper_bound": "1"}, {"distinct_range": 38.857682553310845, "num_eq": 1844268376660379863, "num_range": 100, "upper_bound": "366667060893765316.1"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 1743780356951850531, "row_count": 5324615128060268650}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1146526787730901922, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000, "num_range": 0, "upper_bound": "\\u0004YsSiP"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 4573021991502524495, "upper_bound": "0$_D\\nBqv"}, {"distinct_range": 1919310968185031700, "num_eq": 583588702708614046, "num_range": 1919310968185031745, "upper_bound": "R\\u0016B\\u0014GhS"}, {"distinct_range": 800000, "num_eq": 100000000, "num_range": 800000, "upper_bound": "Y(!\'\\u0005"}, {"distinct_range": 2000, "num_eq": 1214325576302336006, "num_range": 2000, "upper_bound": "f=5 :\x7f;Q\\u0019"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 5324615128060268650}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3705369718546988550, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1810613116"}, {"distinct_range": 0, "num_eq": 0, "num_range": 10000000, "upper_bound": "2306861849"}, {"distinct_range": 441655350578933700, "num_eq": 4000, "num_range": 2709208394198124440, "upper_bound": "2452400684"}, {"distinct_range": 6226164533701665000, "num_eq": 7502510374445096801, "num_range": 6226164533701664500, "upper_bound": "3813696645"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 5324615128060268650}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 827728014803801906, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1995843512826109361}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1949442579910230061, "histo_col_type": "", "name": "__auto__", "null_count": 78410729608384920, "row_count": 1995843512826109361}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1486305511095867962, "histo_buckets": [{"distinct_range": 0, "num_eq": 8542227369702756147, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 900, "num_eq": 0, "num_range": 900, "upper_bound": "87525989"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 900000, "upper_bound": "530630826"}, {"distinct_range": 27041324293.996967, "num_eq": 600, "num_range": 60000000000, "upper_bound": "1021545060"}, {"distinct_range": 55894862.20500932, "num_eq": 3462091938798514599, "num_range": 60000000, "upper_bound": "1802908943"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 4223296375126267297, "upper_bound": "4291527305"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 1995843512826109361}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1359449293060109178, "histo_col_type": "", "name": "__auto__", "null_count": 1319134733464780416, "row_count": 1995843512826109361}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1150917365226813310, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1995843512826109361}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1083260072581415575, "histo_buckets": [{"distinct_range": 0, "num_eq": 5332275408999269156, "num_range": 0, "upper_bound": "01:52:25.568784"}, {"distinct_range": 6596378829447561000, "num_eq": 800000, "num_range": 6596378829447561445, "upper_bound": "06:57:58.696832"}, {"distinct_range": 5380153918338669000, "num_eq": 9000000000, "num_range": 7839518997293298074, "upper_bound": "15:15:07.149173"}, {"distinct_range": 7437081807723964000, "num_eq": 7338299330875093375, "num_range": 7437081807723964494, "upper_bound": "20:58:23.178252"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 1995843512826109361}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3351856, "histo_col_type": "", "name": "__auto__", "null_count": 49157196, "row_count": 90000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8659370, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 89966584, "histo_col_type": "", "name": "__auto__", "null_count": 74559941, "row_count": 90000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17616518, "histo_col_type": "", "name": "__auto__", "null_count": 82711562, "row_count": 90000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28642037, "histo_buckets": [{"distinct_range": 0, "num_eq": 5193367090837039689, "num_range": 0, "upper_bound": "0001-01-01 00:00:00+00:00"}, {"distinct_range": 2522329648523201000, "num_eq": 4000, "num_range": 3208389628854534061, "upper_bound": "1973-07-06 11:00:45.000139+00:00"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 317583601285606375, "upper_bound": "1981-04-05 14:52:06.000002+00:00"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 7076453252746399572, "upper_bound": "1991-12-03 23:01:17.000139+00:00"}, {"distinct_range": 1122845893099215500, "num_eq": 300000, "num_range": 1122845893099215488, "upper_bound": "1999-01-15 15:29:56.000724+00:00"}, {"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "2000-10-14 22:56:44.000135+00:00"}, {"distinct_range": 5000, "num_eq": 900000000, "num_range": 5000, "upper_bound": "2003-08-11 03:09:17.000243+00:00"}, {"distinct_range": 0, "num_eq": 80, "num_range": 0, "upper_bound": "2020-01-19 22:28:26.000993+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 19448958, "row_count": 90000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 962366, "histo_col_type": "", "name": "__auto__", "null_count": 68917374, "row_count": 90000000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33545496, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45465875, "histo_col_type": "", "name": "__auto__", "null_count": 87416120, "row_count": 90000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 56312936, "histo_buckets": [{"distinct_range": 0, "num_eq": 3998551876208159827, "num_range": 0, "upper_bound": "298444780"}, {"distinct_range": 1000000000, "num_eq": 6696633838789117942, "num_range": 1000000000, "upper_bound": "1273989919"}, {"distinct_range": 0, "num_eq": 2959000649560199780, "num_range": 500, "upper_bound": "1772736727"}, {"distinct_range": 1482785984200011000, "num_eq": 7378666467125799596, "num_range": 1482785984200010971, "upper_bound": "1816387662"}, {"distinct_range": 2911260871682598400, "num_eq": 8038078618209435367, "num_range": 5692868199836821881, "upper_bound": "3070853914"}, {"distinct_range": 1177671827823603000, "num_eq": 900, "num_range": 1177671827823602919, "upper_bound": "3370214136"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 90, "upper_bound": "4151395110"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 55274896, "row_count": 90000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 57809602, "histo_buckets": [{"distinct_range": 0, "num_eq": 6202118916553355498, "num_range": 0, "upper_bound": "1985-04-27"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 1362098637662084210, "upper_bound": "1997-01-07"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 90000000}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 61957, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 73139, "histo_col_type": "", "name": "__auto__", "null_count": 114439, "row_count": 800000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 400240, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 525892, "histo_col_type": "", "name": "__auto__", "null_count": 128721, "row_count": 800000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 563221, "histo_col_type": "", "name": "__auto__", "null_count": 92487, "row_count": 800000}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 484610, "histo_col_type": "", "name": "__auto__", "null_count": 436582, "row_count": 800000}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 458654, "histo_col_type": "", "name": "__auto__", "null_count": 450744, "row_count": 800000}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 665064, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 163278, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 4036602554432622325, "num_range": 60000000, "upper_bound": "472168690"}, {"distinct_range": 41135915907089590, "num_eq": 4169071776187515629, "num_range": 317784111398012494, "upper_bound": "1126042588"}, {"distinct_range": 4000, "num_eq": 0, "num_range": 4000, "upper_bound": "1507930298"}, {"distinct_range": 3497006104586029000, "num_eq": 30000, "num_range": 4295920640624548703, "upper_bound": "2094914114"}, {"distinct_range": 19455679565225130, "num_eq": 250014913906625003, "num_range": 4122726120872825604, "upper_bound": "2254217410"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 9046493490090142149, "upper_bound": "3083429595"}, {"distinct_range": 19253363905.34179, "num_eq": 0, "num_range": 30000000000, "upper_bound": "4066663688"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11489, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 81244, "histo_col_type": "", "name": "__auto__", "null_count": 797194, "row_count": 800000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 573977, "histo_col_type": "", "name": "__auto__", "null_count": 677145, "row_count": 800000}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 170306, "histo_col_type": "", "name": "__auto__", "null_count": 609405, "row_count": 800000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 71769, "histo_col_type": "", "name": "__auto__", "null_count": 431109, "row_count": 800000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('hf');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('xff', 'dunewj');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('jpsrsg', 'jfw', 'bpeiaa', 'shfrm', 'o');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('zimi', 'tgiwc', 'vcbkj', 'puuvrp');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17104357, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000000, "num_range": 0, "upper_bound": "-5.838089573123668508E+28"}, {"distinct_range": 8434442977806070000, "num_eq": 0, "num_range": 8434442977806069617, "upper_bound": "-7.512702820979802096E+20"}, {"distinct_range": 54.13086576523683, "num_eq": 4554129010661680842, "num_range": 100, "upper_bound": "-675197669072551.931"}, {"distinct_range": 0, "num_eq": 900000000, "num_range": 400, "upper_bound": "-43123495833933.64774"}, {"distinct_range": 2803.9931213052237, "num_eq": 7258352309804309073, "num_range": 9000, "upper_bound": "1.73928305706289326"}, {"distinct_range": 2130304460015651800, "num_eq": 3261855171639023256, "num_range": 2427470059893638934, "upper_bound": "139.5791692925790201"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 3445500368691961296, "upper_bound": "3.15255261381738131E+27"}, {"distinct_range": 0, "num_eq": 20, "num_range": 0, "upper_bound": "5.890329897212122563E+28"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 7005658, "row_count": 40000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24386312, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 8992700267551017748, "num_range": 3765049739427537500, "upper_bound": "\\u0005wATn}S"}, {"distinct_range": 122874.5377965173, "num_eq": 5000000, "num_range": 200000, "upper_bound": "\\n."}, {"distinct_range": 7178628261477372000, "num_eq": 8751782758101960472, "num_range": 7178628261477371694, "upper_bound": "!"}, {"distinct_range": 733.8714674989735, "num_eq": 4195543443784189611, "num_range": 4000, "upper_bound": "\\""}, {"distinct_range": 80000000, "num_eq": 7233099302132173080, "num_range": 80000000, "upper_bound": "7"}, {"distinct_range": 0, "num_eq": 500, "num_range": 0, "upper_bound": "<w`0%"}, {"distinct_range": 49984.82939029717, "num_eq": 4785662045258890114, "num_range": 400000, "upper_bound": "W@FJ"}, {"distinct_range": 0, "num_eq": 0, "num_range": 7425284291303391631, "upper_bound": "t_6\\u0006\\u0004t\\u0013"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 40000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32143054, "histo_buckets": [{"distinct_range": 0, "num_eq": 8439143377103293689, "num_range": 0, "upper_bound": "625111093"}, {"distinct_range": 565515212719177400, "num_eq": 6679235147525266760, "num_range": 831490800750581975, "upper_bound": "710067373"}, {"distinct_range": 21888.74359172332, "num_eq": 286357887921314963, "num_range": 90000, "upper_bound": "2368393459"}, {"distinct_range": 1000000000, "num_eq": 7339965769186253058, "num_range": 1000000000, "upper_bound": "2458563194"}, {"distinct_range": 700000, "num_eq": 1552431668498719476, "num_range": 700000, "upper_bound": "2712255877"}, {"distinct_range": 408827059703637700, "num_eq": 1200491980408223698, "num_range": 1594019399551822740, "upper_bound": "3028492472"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 40000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22525444, "histo_col_type": "", "name": "__auto__", "null_count": 3305438, "row_count": 40000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39213050, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000000, "num_range": 0, "upper_bound": "1971-05-07"}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 40, "upper_bound": "1974-03-21"}, {"distinct_range": 0, "num_eq": 6895961318110722152, "num_range": 800000000, "upper_bound": "1977-04-22"}, {"distinct_range": 6609554963046659000, "num_eq": 50000000, "num_range": 8690844002426400412, "upper_bound": "1977-07-24"}, {"distinct_range": 0, "num_eq": 644642430868483500, "num_range": 8000, "upper_bound": "1986-09-09"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 2762087093630289489, "upper_bound": "1987-12-28"}, {"distinct_range": 0, "num_eq": 3451022725221815011, "num_range": 3990069436950949356, "upper_bound": "1991-12-19"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 40000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19024898, "histo_col_type": "", "name": "__auto__", "null_count": 500545, "row_count": 40000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19853933, "histo_col_type": "", "name": "__auto__", "null_count": 27623805, "row_count": 40000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30091975, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2842835965006470522, "histo_col_type": "", "name": "__auto__", "null_count": 2140367277405304098, "row_count": 5954506955287165392}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3990161423182429781, "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 5954506955287165392}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1348428546608831771, "histo_col_type": "", "name": "__auto__", "null_count": 1151655760827017565, "row_count": 5954506955287165392}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1053560258168986872, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5954506955287165392}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3099039463071672571, "histo_buckets": [{"distinct_range": 0, "num_eq": 861833552861524021, "num_range": 0, "upper_bound": "00:31:26.160545"}, {"distinct_range": 7277003504842924000, "num_eq": 7979947288618112038, "num_range": 7875981934046370061, "upper_bound": "02:07:38.848894"}, {"distinct_range": 11120446889.335493, "num_eq": 1534984599109767340, "num_range": 60000000000, "upper_bound": "03:01:08.356264"}, {"distinct_range": 4120515835474939000, "num_eq": 10000000, "num_range": 7031408654256842417, "upper_bound": "15:41:35.430023"}, {"distinct_range": 0, "num_eq": 3635882490458375987, "num_range": 6382338208052523537, "upper_bound": "16:35:55.365273"}, {"distinct_range": 0, "num_eq": 60, "num_range": 400, "upper_bound": "17:55:40.461665"}, {"distinct_range": 48913220.321808174, "num_eq": 2269709685656341383, "num_range": 50000000, "upper_bound": "18:34:23.367689"}, {"distinct_range": 100000, "num_eq": 1000000, "num_range": 100000, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 5954506955287165392}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4379215517781894958, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5954506955287165392}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 93829778, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 88481831, "histo_col_type": "", "name": "__auto__", "null_count": 51591920, "row_count": 200000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 136949743, "histo_col_type": "", "name": "__auto__", "null_count": 31831388, "row_count": 200000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 151143521, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1970-01-01"}, {"distinct_range": 400000000, "num_eq": 9131152306397658889, "num_range": 400000000, "upper_bound": "1973-05-19"}, {"distinct_range": 0, "num_eq": 7365391308852888240, "num_range": 1000000000, "upper_bound": "1973-11-28"}, {"distinct_range": 0, "num_eq": 7076946100167940068, "num_range": 0, "upper_bound": "1976-03-02"}, {"distinct_range": 1653191513095160000, "num_eq": 500000, "num_range": 7406074750889515058, "upper_bound": "1985-01-15"}, {"distinct_range": 40000, "num_eq": 4000000, "num_range": 40000, "upper_bound": "1986-10-28"}, {"distinct_range": 400000, "num_eq": 900, "num_range": 400000, "upper_bound": "1988-04-30"}, {"distinct_range": 5254968824899798000, "num_eq": 1000000, "num_range": 5254968824899797939, "upper_bound": "1991-03-31"}, {"distinct_range": 80, "num_eq": 600000000, "num_range": 80, "upper_bound": "1997-05-12"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 200000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1453759, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 65279703, "histo_col_type": "", "name": "__auto__", "null_count": 72625466, "row_count": 200000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 73672135, "histo_buckets": [{"distinct_range": 0, "num_eq": 7930391165953302006, "num_range": 0, "upper_bound": "1970-05-03 00:41:23.000087+00:00"}, {"distinct_range": 868081430.3423802, "num_eq": 4598072961688604100, "num_range": 1000000000, "upper_bound": "1992-05-10 03:48:14.000337+00:00"}, {"distinct_range": 2177451667877730600, "num_eq": 3764094835046229478, "num_range": 2177451667877730650, "upper_bound": "2003-11-01 07:04:33.000396+00:00"}, {"distinct_range": 27.982594644906243, "num_eq": 1000, "num_range": 60, "upper_bound": "2020-05-14 19:36:13.000926+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 11359165, "row_count": 200000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 62774023, "histo_col_type": "", "name": "__auto__", "null_count": 171545858, "row_count": 200000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39980029, "histo_col_type": "OID", "name": "__auto__", "null_count": 162906112, "row_count": 200000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 50516455, "histo_col_type": "", "name": "__auto__", "null_count": 189602603, "row_count": 200000000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4444736861, "histo_col_type": "", "name": "__auto__", "null_count": 6678582081, "row_count": 8000000000}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3891109801, "histo_col_type": "", "name": "__auto__", "null_count": 6295879026, "row_count": 8000000000}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5388666364, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1421849867, "histo_col_type": "", "name": "__auto__", "null_count": 2654634695, "row_count": 8000000000}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6366698744, "histo_col_type": "", "name": "__auto__", "null_count": 4326826271, "row_count": 8000000000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6783879570, "histo_col_type": "", "name": "__auto__", "null_count": 1156905797, "row_count": 8000000000}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3795312922, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7569768437, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2747825065, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5554105035, "histo_col_type": "", "name": "__auto__", "null_count": 3544957138, "row_count": 8000000000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 199729383, "histo_col_type": "", "name": "__auto__", "null_count": 5119899038, "row_count": 8000000000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2664917454, "histo_col_type": "", "name": "__auto__", "null_count": 5313935064, "row_count": 8000000000}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1216634010, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000000, "num_range": 0, "upper_bound": "396992025"}, {"distinct_range": 3305754179.3506246, "num_eq": 700000000, "num_range": 10000000000, "upper_bound": "554722880"}, {"distinct_range": 1000, "num_eq": 80000000000, "num_range": 1000, "upper_bound": "720174604"}, {"distinct_range": 2073541545416639500, "num_eq": 20000, "num_range": 3754024754346575769, "upper_bound": "1110059659"}, {"distinct_range": 53676748948403530, "num_eq": 4282924475698575818, "num_range": 251715083600081590, "upper_bound": "1864603936"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 289483361120822407, "upper_bound": "2780651783"}, {"distinct_range": 4000000000, "num_eq": 10, "num_range": 4000000000, "upper_bound": "2948605155"}, {"distinct_range": 700, "num_eq": 800, "num_range": 700, "upper_bound": "3547125927"}, {"distinct_range": 1094692542528866400, "num_eq": 2743527797511705421, "num_range": 1094692542528866390, "upper_bound": "4275084567"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5732974420, "histo_col_type": "", "name": "__auto__", "null_count": 3338778438, "row_count": 8000000000}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 119403, "histo_buckets": [{"distinct_range": 0, "num_eq": 70, "num_range": 0, "upper_bound": "917158078"}, {"distinct_range": 65677.89084958287, "num_eq": 6000000000, "num_range": 90000, "upper_bound": "2027498034"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 63614, "histo_col_type": "", "name": "__auto__", "null_count": 56801, "row_count": 300000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 228347, "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 139917, "histo_col_type": "", "name": "__auto__", "null_count": 116077, "row_count": 300000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 55951, "histo_col_type": "", "name": "__auto__", "null_count": 61447, "row_count": 300000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14069, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 52542, "histo_buckets": [{"distinct_range": 0, "num_eq": 8258612257761476328, "num_range": 0, "upper_bound": "-9.152671058252100446E+36"}, {"distinct_range": 13218644.216389675, "num_eq": 2763282627998974232, "num_range": 40000000, "upper_bound": "-5.645388048152352261E+21"}, {"distinct_range": 50000, "num_eq": 6184506043977512331, "num_range": 50000, "upper_bound": "-0.1530026595837125973"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 4000, "upper_bound": "5039627069.57726696"}, {"distinct_range": 0, "num_eq": 193368704237589486, "num_range": 0, "upper_bound": "17802414124.87172472"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 244998, "row_count": 300000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44505, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 70, "num_range": 50000000000, "upper_bound": "("}, {"distinct_range": 24872271825.08039, "num_eq": 900, "num_range": 40000000000, "upper_bound": "CXT\\u0004\\u000c\x7fX"}, {"distinct_range": 0, "num_eq": 210492783098351311, "num_range": 4360765371787560530, "upper_bound": "S,m\\u000b"}, {"distinct_range": 70000000000, "num_eq": 424664180413350094, "num_range": 70000000000, "upper_bound": "`"}, {"distinct_range": 937321766983317900, "num_eq": 5525016162971795891, "num_range": 949114495669992690, "upper_bound": "lq\\u0013u\\u0011R-"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 300000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 449122641556087639, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 462770008330785396}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 83203802571138354, "histo_buckets": [{"distinct_range": 0, "num_eq": 1813537278394716014, "num_range": 0, "upper_bound": "00:00:00"}, {"distinct_range": 30000000, "num_eq": 60000, "num_range": 30000000, "upper_bound": "03:51:47.577053"}, {"distinct_range": 0, "num_eq": 4977796487465284143, "num_range": 2000, "upper_bound": "10:14:36.843357"}, {"distinct_range": 800000000, "num_eq": 6807561409927417608, "num_range": 800000000, "upper_bound": "14:47:20.47454"}, {"distinct_range": 331593961811314800, "num_eq": 400000, "num_range": 4908358939457574681, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 462770008330785396}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 162615026091971259, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 462770008330785396}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 82425046041974977, "histo_col_type": "", "name": "__auto__", "null_count": 450181828593806045, "row_count": 462770008330785396}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 158941389739222004, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000, "num_range": 0, "upper_bound": "3484073348"}, {"distinct_range": 0, "num_eq": 8569958266614082429, "num_range": 100, "upper_bound": "4148190876"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 462770008330785396}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9856753723900965, "histo_col_type": "", "name": "__auto__", "null_count": 327771610639332230, "row_count": 462770008330785396}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 242897, "histo_col_type": "", "name": "__auto__", "null_count": 169439, "row_count": 300000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 251217, "histo_col_type": "", "name": "__auto__", "null_count": 147365, "row_count": 300000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 179899, "histo_col_type": "", "name": "__auto__", "null_count": 40508, "row_count": 300000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5490, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000000, "num_range": 0, "upper_bound": "584404729"}, {"distinct_range": 1730166292274910200, "num_eq": 1031337172592609134, "num_range": 4354776249894425499, "upper_bound": "824761980"}, {"distinct_range": 9018629574320791000, "num_eq": 300000000, "num_range": 9018629574320790878, "upper_bound": "1278450679"}, {"distinct_range": 10000, "num_eq": 60000000, "num_range": 10000, "upper_bound": "2979445725"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 148153, "row_count": 300000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 158492, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 150563, "histo_col_type": "", "name": "__auto__", "null_count": 40441, "row_count": 300000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26234, "histo_buckets": [{"distinct_range": 0, "num_eq": 1360100801466565618, "num_range": 0, "upper_bound": "1975-07-06"}, {"distinct_range": 0, "num_eq": 800000000, "num_range": 4844366979795836567, "upper_bound": "1978-12-04"}, {"distinct_range": 380347039.75290567, "num_eq": 7609961558689815284, "num_range": 400000000, "upper_bound": "1980-09-02"}, {"distinct_range": 50, "num_eq": 100000000, "num_range": 50, "upper_bound": "1982-10-24"}, {"distinct_range": 0, "num_eq": 2679545429295983631, "num_range": 2958578893693731249, "upper_bound": "1985-06-15"}, {"distinct_range": 3638824374207046700, "num_eq": 900000000, "num_range": 3638824374207046547, "upper_bound": "1986-03-15"}, {"distinct_range": 0, "num_eq": 60000000000, "num_range": 3370875482829214062, "upper_bound": "1986-04-22"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15360, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 111858, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000, "num_range": 0, "upper_bound": "0001-01-01 00:00:00+00:00"}, {"distinct_range": 3708367621008153000, "num_eq": 9024205150334258604, "num_range": 3708367621008153149, "upper_bound": "1976-03-01 06:25:30.000142+00:00"}, {"distinct_range": 5850603118015508000, "num_eq": 3053326702797154500, "num_range": 5850603118015508591, "upper_bound": "1988-11-20 17:18:49.000876+00:00"}, {"distinct_range": 105842.95002116448, "num_eq": 2931897173314190037, "num_range": 3000000, "upper_bound": "1995-02-27 13:59:52.000175+00:00"}, {"distinct_range": 0, "num_eq": 5563126096857765058, "num_range": 5851104251890558587, "upper_bound": "2002-03-22 02:33:05.000411+00:00"}, {"distinct_range": 0, "num_eq": 9000000000, "num_range": 3000000000, "upper_bound": "2018-02-18 17:01:18.00064+00:00"}, {"distinct_range": 0, "num_eq": 428393116466779536, "num_range": 5000000, "upper_bound": "2027-05-14 20:38:14.000655+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 122749, "row_count": 300000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 265797, "histo_col_type": "", "name": "__auto__", "null_count": 295264, "row_count": 300000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1046889937445630614, "histo_col_type": "", "name": "__auto__", "null_count": 305085904526437521, "row_count": 1944821513191049399}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 657547566689503490, "histo_col_type": "", "name": "__auto__", "null_count": 1522778193213399134, "row_count": 1944821513191049399}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 664873097003196914, "histo_col_type": "", "name": "__auto__", "null_count": 1407289471039755061, "row_count": 1944821513191049399}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 948164876823476781, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1944821513191049399}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1312011761261421717, "histo_col_type": "", "name": "__auto__", "null_count": 643661276824396155, "row_count": 1944821513191049399}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1942042551391785016, "histo_col_type": "", "name": "__auto__", "null_count": 1662757544528294400, "row_count": 1944821513191049399}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1686343736999926790, "histo_buckets": [{"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 700, "num_eq": 2000000000, "num_range": 700, "upper_bound": "3318792974"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 1944821513191049399}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 320119027421699831, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1944821513191049399}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 502200626621173692, "histo_col_type": "", "name": "__auto__", "null_count": 1345479216901876218, "row_count": 1944821513191049399}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1836078775466483796, "histo_col_type": "", "name": "__auto__", "null_count": 3039140033794236, "row_count": 1944821513191049399}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 331633541919297511, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1944821513191049399}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 986698873117031723, "histo_col_type": "", "name": "__auto__", "null_count": 1577306356275606624, "row_count": 1944821513191049399}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1099802325364627572, "histo_col_type": "", "name": "__auto__", "null_count": 795673297777268820, "row_count": 1944821513191049399}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1869862453163102437, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1944821513191049399}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "nan" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2541956&tab=buildLog) on [master@735da601263e1d0731cefd6ef3e6709d0ab479a1](https://github.com/cockroachdb/cockroach/commits/735da601263e1d0731cefd6ef3e6709d0ab479a1):

```
        cockroach2_1  | webui:               http://cockroach2:8080
        cockroach2_1  | sql:                 postgresql://root@cockroach2:26257?sslmode=disable
        cockroach2_1  | RPC client flags:    /cockroach/cockroach <client cmd> --host=cockroach2:26257 --insecure
        cockroach2_1  | logs:                /cockroach-data/logs
        cockroach2_1  | temp dir:            /cockroach-data/cockroach-temp975933684
        cockroach2_1  | external I/O path:   /cockroach-data/extern
        cockroach2_1  | store[0]:            path=/cockroach-data
        cockroach2_1  | storage engine:      pebble
        cockroach2_1  | status:              initialized new cluster
        cockroach2_1  | clusterID:           62e32a1f-0c9b-4889-aa24-5c8cac46ee90
        cockroach2_1  | nodeID:              1
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2020-12-24 06:12:48.711 UTC [49] LOG:  received fast shutdown request
        postgres_1    | .2020-12-24 06:12:48.713 UTC [49] LOG:  aborting any active transactions
        postgres_1    | 2020-12-24 06:12:48.714 UTC [49] LOG:  background worker "logical replication launcher" (PID 56) exited with exit code 1
        postgres_1    | 2020-12-24 06:12:48.721 UTC [51] LOG:  shutting down
        postgres_1    | 2020-12-24 06:12:48.840 UTC [49] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2020-12-24 06:12:48.921 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2020-12-24 06:12:48.921 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2020-12-24 06:12:48.925 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2020-12-24 06:12:48.936 UTC [86] LOG:  database system was shut down at 2020-12-24 06:12:48 UTC
        postgres_1    | 2020-12-24 06:12:48.942 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (602.48s)
        test_1        |     --- FAIL: TestCompare/postgres (1.55s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2543442&tab=buildLog) on [master@152c3b30ae1dcb7a7cefc6aa7fa86c2dd264bd2e](https://github.com/cockroachdb/cockroach/commits/152c3b30ae1dcb7a7cefc6aa7fa86c2dd264bd2e):

```
        cockroach2_1  | status:              initialized new cluster
        cockroach2_1  | clusterID:           8507a841-5554-4150-9424-2affa9a46714
        cockroach2_1  | nodeID:              1
        postgres_1    | 2020-12-25 06:13:17.533 UTC [50] LOG:  database system was shut down at 2020-12-25 06:13:17 UTC
        postgres_1    | 2020-12-25 06:13:17.538 UTC [49] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2020-12-25 06:13:20.432 UTC [49] LOG:  received fast shutdown request
        postgres_1    | .2020-12-25 06:13:20.435 UTC [49] LOG:  aborting any active transactions
        postgres_1    | 2020-12-25 06:13:20.436 UTC [49] LOG:  background worker "logical replication launcher" (PID 56) exited with exit code 1
        postgres_1    | 2020-12-25 06:13:20.442 UTC [51] LOG:  shutting down
        postgres_1    | 2020-12-25 06:13:20.573 UTC [49] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2020-12-25 06:13:20.644 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2020-12-25 06:13:20.645 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2020-12-25 06:13:20.651 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2020-12-25 06:13:20.665 UTC [86] LOG:  database system was shut down at 2020-12-25 06:13:20 UTC
        postgres_1    | 2020-12-25 06:13:20.670 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (602.02s)
        test_1        |     --- FAIL: TestCompare/postgres (1.28s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.2:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2544833&tab=buildLog) on [master@4e3a9ccdb9cac89ae32d6248bb9b119b5bc250f7](https://github.com/cockroachdb/cockroach/commits/4e3a9ccdb9cac89ae32d6248bb9b119b5bc250f7):

```
        test_1        |             CREATE INDEX ON table4 (col2_0, col4_5 ASC, col2_2 ASC, col2_4, col2_1 DESC);
        test_1        |             CREATE INDEX ON table4 (col2_5, col4_5 DESC, col2_0 ASC, col2_2 ASC);
        test_1        |             CREATE INDEX ON table4 (col2_2, col2_0 ASC, col4_5, col2_5 DESC, col2_1 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_5 ASC, col2_5, col2_4 ASC, col2_2);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col2_4 ASC, col2_5 ASC, col4_5 DESC, col2_1 ASC, col2_0);
        test_1        |             CREATE INDEX ON table4 (col2_0, col2_2, col2_5, col2_4 ASC);
        test_1        |             CREATE INDEX ON table4 (col4_5, col2_1 ASC, col2_5);
        test_1        |             CREATE TABLE table5 (col2_1 INT8 NOT NULL, col2_5 INET NOT NULL, col2_2 INT8 NOT NULL, col2_0 INET NOT NULL, col2_4 BIT(17) NOT NULL, col5_5 REGNAMESPACE, col5_6 INET NOT NULL, col5_7 NAME, col5_8 REGCLASS NOT NULL, col5_9 REGPROC NOT NULL, col5_10 BOX2D NULL, col5_11 REGNAMESPACE NULL, col5_12 FLOAT8, col5_13 TIME NOT NULL, col5_14 UUID, col5_15 BOOL, PRIMARY KEY (col2_1, col2_5, col2_2, col2_0, col2_4, col5_10, col5_12));
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('kty', 'n', 'zp', 'qm', 'tgl');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (0.89s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 REGCLASS NOT NULL, col1_1 BYTES NOT NULL, col1_2 CHAR NOT NULL, col1_3 OID NOT NULL, col1_4 REGTYPE NULL, col1_5 TIMESTAMP, col1_6 BYTES NOT NULL, PRIMARY KEY (col1_6 ASC, col1_2 ASC, col1_3 ASC), INDEX (col1_5 ASC, col1_4, col1_0 ASC, col1_6, col1_1 ASC) WHERE ((table1.col1_1 > e'\\x':::BYTES) AND (table1.col1_2 < e'\'':::STRING)) AND (table1.col1_5 != '0001-01-01 00:00:00':::TIMESTAMP), INDEX (col1_6 ASC, col1_3 ASC, col1_1 DESC, col1_0 DESC, col1_2, col1_4 DESC, col1_5 ASC) WHERE (((table1.col1_2 = 'X') AND (table1.col1_1 < '\x')) OR (table1.col1_6 = '\x')) OR (table1.col1_5 > '3000-01-01 00:00:00'), INDEX (col1_5, col1_3 ASC) STORING (col1_0) WHERE (((table1.col1_1 > e'\\x22':::BYTES) OR (table1.col1_2 < e'\x00':::STRING)) OR (table1.col1_6 != e'\\x':::BYTES)) OR (table1.col1_5 > '-2000-01-01 00:00:00':::TIMESTAMP), INDEX (col1_0 ASC, col1_1) WHERE (((table1.col1_2 = e'\U00002603') OR (table1.col1_5 >= '0001-01-01 00:00:00')) AND (table1.col1_6 >= '\x00')) OR (table1.col1_1 != '\x00'), UNIQUE (col1_0 ASC, col1_3 ASC, col1_4 DESC) WHERE (((table1.col1_5 > '3000-01-01 00:00:00':::TIMESTAMP) OR (table1.col1_6 > e'\\x00':::BYTES)) AND (table1.col1_2 < 'X':::STRING)) AND (table1.col1_1 >= e'\\xe29883':::BYTES), UNIQUE (col1_4 ASC, col1_6 DESC), INDEX (col1_0, col1_1 ASC, col1_2, col1_5 DESC, col1_6, col1_3, col1_4 ASC) WHERE (((table1.col1_5 < '0001-01-01 00:00:00':::TIMESTAMP) AND (table1.col1_6 < e'\\x27':::BYTES)) OR (table1.col1_1 >= e'\\xe29883':::BYTES)) OR (table1.col1_2 > '"':::STRING), INDEX (col1_1 ASC) STORING (col1_5), UNIQUE (col1_5, col1_1 ASC, col1_4, col1_2 ASC, col1_0 ASC, col1_6), FAMILY (col1_6, col1_3, col1_1), FAMILY (col1_5), FAMILY (col1_4, col1_0, col1_2));
        test_1        |             CREATE TABLE table2 (col1_6 BYTES NOT NULL, col1_2 CHAR NOT NULL, col1_3 OID NOT NULL, col2_3 BIT(34) NULL, col2_4 NAME NULL, col2_5 REGTYPE NULL, col2_6 INT8, PRIMARY KEY (col1_6 ASC, col1_2 ASC, col1_3 ASC, col2_6 DESC, col2_3, col2_4 ASC), UNIQUE (col2_3 DESC, col2_6 ASC, col1_6 DESC, col2_4, col2_5 ASC, col1_3 DESC) WHERE (((table2.col1_2 < '':::STRING) AND (table2.col1_6 != e'\\xe29883':::BYTES)) OR (table2.col2_6 >= 32767:::INT8)) AND (table2.col2_4 > e'\U00002603':::STRING), UNIQUE (col1_6 ASC) STORING (col2_5), UNIQUE (col2_4 DESC, col2_6 DESC, col2_5 DESC, col2_3, col1_2 ASC, col1_3), UNIQUE (col2_3 DESC) STORING (col2_5) WHERE (((table2.col1_6 > e'\\x22':::BYTES) OR (table2.col2_6 < (-1):::INT8)) AND (table2.col2_4 = e'\x00':::STRING)) AND (table2.col1_2 = e'\'':::STRING), INDEX (col2_5 DESC), INDEX (col1_6 ASC, col2_6 ASC, col1_2 ASC, col2_5 DESC, col2_3, col1_3 ASC) WHERE (table2.col1_2 > e'\'':::STRING) AND (table2.col1_6 <= e'\\x58':::BYTES), UNIQUE (col2_4 ASC, col2_6, col2_5 DESC, col2_3, col1_2 DESC), INDEX (col2_3 DESC, col2_5 DESC, col2_4 DESC) WHERE (((table2.col2_4 >= e'\'') AND (table2.col1_2 != e'\x00')) OR (table2.col2_6 < -2147483648)) AND (table2.col1_6 != '\x58'), INDEX (col2_3 DESC, col2_5 DESC, col1_6 ASC) WHERE (table2.col2_4 != '"':::STRING) OR (table2.col1_6 > e'\\x':::BYTES), UNIQUE (col1_6) WHERE (((table2.col2_4 < '') AND (table2.col1_2 < '"')) AND (table2.col1_6 >= '\x27')) AND (table2.col2_6 > -9223372036854775807), FAMILY (col2_3, col1_2), FAMILY (col1_6), FAMILY (col2_5, col1_3, col2_4), FAMILY (col2_6)) INTERLEAVE IN PARENT table1 (col1_6, col1_2, col1_3);
        test_1        |             CREATE TABLE table3 (col3_0 REGTYPE NOT NULL, col3_1 REGTYPE NOT NULL, col3_2 INT4, col3_3 NAME NOT NULL, col3_4 REGTYPE NOT NULL, col3_5 TIMESTAMP NOT NULL, col3_6 REGTYPE NOT NULL, col3_7 REGCLASS NOT NULL, col3_8 BOX2D NOT NULL, PRIMARY KEY (col3_8 DESC, col3_7 ASC, col3_6 DESC, col3_1 DESC, col3_4 DESC, col3_5, col3_0 DESC), UNIQUE (col3_8 DESC, col3_1 ASC) WHERE ((table3.col3_3 > '"') AND (table3.col3_2 = -128)) OR (table3.col3_5 > '0001-01-01 00:00:00'), INDEX (col3_2 ASC, col3_3 ASC, col3_4 DESC) WHERE ((table3.col3_2 != 127:::INT8) AND (table3.col3_3 >= e'\U00002603':::STRING)) OR (table3.col3_5 <= '294276-12-31 23:59:59.999999':::TIMESTAMP), INDEX (col3_6 DESC, col3_7 DESC, col3_5, col3_1 DESC, col3_3) WHERE ((table3.col3_5 != '-2000-01-01 00:00:00':::TIMESTAMP) AND (table3.col3_3 > e'\'':::STRING)) OR (table3.col3_2 < 1:::INT8), INDEX (col3_7 DESC, col3_8 DESC, col3_6 DESC, col3_2 DESC, col3_3 DESC, col3_0 DESC, col3_4 ASC, col3_5 DESC, col3_1) WHERE ((table3.col3_5 > '-2000-01-01 00:00:00':::TIMESTAMP) OR (table3.col3_2 != 127:::INT8)) AND (table3.col3_3 != '"':::STRING), INDEX (col3_5 DESC, col3_7 ASC, col3_0 ASC, col3_6 DESC, col3_2, col3_3 DESC), INDEX (col3_6 ASC, col3_1 ASC, col3_3 DESC) WHERE ((table3.col3_3 >= '"') OR (table3.col3_5 > '0001-01-01 00:00:00')) OR (table3.col3_2 <= 2147483647), FAMILY (col3_3, col3_7, col3_0, col3_6), FAMILY (col3_2), FAMILY (col3_4, col3_5, col3_8), FAMILY (col3_1));
        test_1        |             CREATE TABLE table4 (col4_0 REGNAMESPACE NULL, col4_1 INT4 NOT NULL, col4_2 TIME, col4_3 BYTES NULL, col4_4 REGPROCEDURE NOT NULL, col4_5 BOX2D NOT NULL, col4_6 INTERVAL, col4_7 TIME NULL, col4_8 OID NULL, col4_9 BYTES NOT NULL, col4_10 CHAR NULL, col4_11 INTERVAL NOT NULL, col4_12 TIMESTAMPTZ NOT NULL, col4_13 FLOAT8 NOT NULL, PRIMARY KEY (col4_11, col4_1 ASC, col4_12 ASC, col4_4, col4_9), UNIQUE (col4_9 DESC, col4_5 DESC) STORING (col4_0, col4_3, col4_6), INDEX (col4_7, col4_13, col4_9 ASC, col4_2, col4_0 DESC, col4_5 ASC, col4_4 ASC, col4_12 DESC, col4_11 ASC, col4_8 ASC) STORING (col4_3) WHERE (((((((table4.col4_9 != e'\\x58':::BYTES) AND (table4.col4_1 = 1:::INT8)) AND (table4.col4_7 = '24:00:00':::TIME)) AND (table4.col4_10 > '"':::STRING)) AND (table4.col4_2 != '24:00:00':::TIME)) OR (table4.col4_13 > 1.401298464324817e-45:::FLOAT8)) OR (table4.col4_3 != e'\\xff':::BYTES)) OR (table4.col4_12 = '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ), FAMILY (col4_2), FAMILY (col4_3, col4_0, col4_5, col4_1), FAMILY (col4_11), FAMILY (col4_6, col4_4, col4_10), FAMILY (col4_12, col4_9), FAMILY (col4_8, col4_13), FAMILY (col4_7));
        test_1        |             CREATE TABLE table5 (col5_0 NAME NOT NULL, col5_1 "char" NOT NULL, col5_2 FLOAT4 NOT NULL, PRIMARY KEY (col5_0 DESC, col5_1 DESC), INDEX (col5_1 ASC, col5_0 ASC, col5_2 DESC), INDEX (col5_0 DESC, col5_2 ASC, col5_1 DESC) WHERE table5.col5_2 != +Inf, INDEX (col5_2 DESC, col5_0 ASC) WHERE ((table5.col5_0 = e'\U00002603') OR (table5.col5_1 <= '"')) OR (table5.col5_2 <= +Inf), INDEX (col5_1 DESC) WHERE table5.col5_0 != e'\x00':::STRING, INDEX (col5_1 DESC, col5_0, col5_2) WHERE (table5.col5_1 < 'X':::STRING) AND (table5.col5_0 >= e'\'':::STRING), UNIQUE (col5_1 DESC) WHERE ((table5.col5_2 < 'NaN':::FLOAT8) AND (table5.col5_0 < '"':::STRING)) OR (table5.col5_1 <= e'\x00':::STRING), FAMILY (col5_2), FAMILY (col5_0, col5_1));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5767558058218396610, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000000000, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 29245429743.32771, "num_eq": 9000000, "num_range": 30000000000, "upper_bound": "\\\\x07cef2bb720bf096"}, {"distinct_range": 300000, "num_eq": 0, "num_range": 300000, "upper_bound": "\\\\x22f586"}, {"distinct_range": 3082054070163141600, "num_eq": 60000, "num_range": 3082054070163141584, "upper_bound": "\\\\x30ab"}, {"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "\\\\x57d5b4dd82813d"}, {"distinct_range": 2024572055196014600, "num_eq": 1581921978941674080, "num_range": 2024572055196014581, "upper_bound": "\\\\x658a7e0468bee6bcb7"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 0, "upper_bound": "\\\\xb82f"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 8374392233439492314}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7933937419902267771, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8374392233439492314}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6638700718061635690, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8374392233439492314}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1499905639500376597, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "609020802"}, {"distinct_range": 0, "num_eq": 3364203722144442543, "num_range": 30000000, "upper_bound": "1993587937"}, {"distinct_range": 3619380915949476000, "num_eq": 500000000, "num_range": 3619380915949475961, "upper_bound": "2148604930"}, {"distinct_range": 6661222111722896000, "num_eq": 5054703133758307394, "num_range": 6661222111722896139, "upper_bound": "2207809828"}, {"distinct_range": 737505952037559900, "num_eq": 1278674411035459414, "num_range": 1492797126008553200, "upper_bound": "2448452316"}, {"distinct_range": 4313185614891551000, "num_eq": 200000, "num_range": 4313185614891551364, "upper_bound": "3454463904"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 8030331089605316390, "row_count": 8374392233439492314}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1801259997885052899, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "1993-11-06 06:42:33.000163"}, {"distinct_range": 3012540189027409400, "num_eq": 965327674926413895, "num_range": 3564578765133376672, "upper_bound": "2014-04-11 07:08:12.000714"}, {"distinct_range": 0, "num_eq": 6183245623086547582, "num_range": 0, "upper_bound": "2021-09-04 01:05:09.000258"}, {"distinct_range": 90274912.21862683, "num_eq": 10000000000, "num_range": 200000000, "upper_bound": "2026-10-29 12:52:05.000049"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 5056551449741417415, "row_count": 8374392233439492314}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6080789446820579890, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "\\\\x4108e51867e41e51"}, {"distinct_range": 0, "num_eq": 4718255833458899579, "num_range": 4439566598534860952, "upper_bound": "\\\\x9fa3d6aede20"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 8374392233439492314}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6965990998995867872, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 5521904185922649000, "num_eq": 8731469124166083466, "num_range": 5521904185922649534, "upper_bound": "489539013"}, {"distinct_range": 6517725569300675000, "num_eq": 8654922696239463273, "num_range": 6517725569300674159, "upper_bound": "928541316"}, {"distinct_range": 0, "num_eq": 7462501519668206629, "num_range": 8000000000, "upper_bound": "1136815653"}, {"distinct_range": 36541566.75941897, "num_eq": 500000, "num_range": 60000000, "upper_bound": "1688526548"}, {"distinct_range": 1724.8708734611382, "num_eq": 400000, "num_range": 5000, "upper_bound": "2485821918"}, {"distinct_range": 842925446119299800, "num_eq": 5678722003898597555, "num_range": 1446590951291517382, "upper_bound": "2679884503"}, {"distinct_range": 0, "num_eq": 2363496844570717114, "num_range": 5533242843461040730, "upper_bound": "2710494675"}, {"distinct_range": 1529887000615050500, "num_eq": 5124507272524786934, "num_range": 1541883461857952726, "upper_bound": "4204036249"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 8374392233439492314}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67, "histo_buckets": [{"distinct_range": 0, "num_eq": 3390598891069803684, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 800000000, "upper_bound": "\\\\x4c"}, {"distinct_range": 602606.7764576554, "num_eq": 50, "num_range": 800000, "upper_bound": "\\\\x58"}, {"distinct_range": 0, "num_eq": 300, "num_range": 5116704950553925001, "upper_bound": "\\\\x830775"}, {"distinct_range": 196623124136928000, "num_eq": 7163173012537621433, "num_range": 479221304499131137, "upper_bound": "\\\\xe29883"}, {"distinct_range": 2685789114087699500, "num_eq": 4273697586293818142, "num_range": 5355403712894757629, "upper_bound": "\\\\xe8"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 70}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 43, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": "0000000000000000000000000000000000"}, {"distinct_range": 0, "num_eq": 200000, "num_range": 20000000000, "upper_bound": "0000011111110101101000000111000110"}, {"distinct_range": 0, "num_eq": 90000, "num_range": 0, "upper_bound": "0100011100010111001010000000011100"}, {"distinct_range": 949240611990664800, "num_eq": 1000000, "num_range": 8501948269997158559, "upper_bound": "0110011001001010100010010101100100"}, {"distinct_range": 10.998237142922934, "num_eq": 60000000000, "num_range": 40, "upper_bound": "0110011001101000111100001010100111"}, {"distinct_range": 324210.1682478182, "num_eq": 1429290919347397374, "num_range": 1000000, "upper_bound": "1011010011010111101110011000011010"}, {"distinct_range": 3390435436371562500, "num_eq": 10, "num_range": 3390435436371562667, "upper_bound": "1110011101010010000010000001010111"}], "histo_col_type": "BIT(34)", "name": "__auto__", "null_count": 40, "row_count": 70}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13, "histo_buckets": [{"distinct_range": 0, "num_eq": 2367099650523791586, "num_range": 0, "upper_bound": ""}, {"distinct_range": 489924885176246500, "num_eq": 7681319754039571710, "num_range": 1853568472202386108, "upper_bound": "3\\u000b>\\u0002w\\u0011"}, {"distinct_range": 4004546244926863000, "num_eq": 6471376214343670090, "num_range": 4004546244926862795, "upper_bound": "4>"}, {"distinct_range": 319.6331131769742, "num_eq": 4000000000, "num_range": 500, "upper_bound": ";c[\\u0010\\u0018D\\u0005V"}, {"distinct_range": 1000000, "num_eq": 943955108203419472, "num_range": 1000000, "upper_bound": "M_FJrM\\u0012"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 100000, "upper_bound": "O"}, {"distinct_range": 1806288091100235800, "num_eq": 50000, "num_range": 1806288091100235648, "upper_bound": "`\\t="}, {"distinct_range": 50, "num_eq": 10000, "num_range": 50, "upper_bound": "wt~"}, {"distinct_range": 0, "num_eq": 7470048856882464324, "num_range": 1000, "upper_bound": "{("}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 15, "row_count": 70}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 60, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "493471837"}, {"distinct_range": 0, "num_eq": 500, "num_range": 8498254875007262383, "upper_bound": "517184278"}, {"distinct_range": 0, "num_eq": 551474582582260756, "num_range": 40000000, "upper_bound": "3769632567"}, {"distinct_range": 8798761095891886000, "num_eq": 9000000, "num_range": 8798761095891885585, "upper_bound": "4125875653"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 52, "row_count": 70}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33, "histo_col_type": "", "name": "__auto__", "null_count": 6, "row_count": 70}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 466283267981139829, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2802328152315298194}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2740278833056416462, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2802328152315298194}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 594230559020464818, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000, "num_range": 0, "upper_bound": "-1765225232"}, {"distinct_range": 10, "num_eq": 30000000000, "num_range": 10, "upper_bound": "-1"}, {"distinct_range": 0, "num_eq": 2000000000, "num_range": 6565859447358115540, "upper_bound": "32767"}, {"distinct_range": 0, "num_eq": 3837695890312686485, "num_range": 80000, "upper_bound": "393509619"}, {"distinct_range": 0, "num_eq": 5339940785974865044, "num_range": 900000, "upper_bound": "678084923"}, {"distinct_range": 0, "num_eq": 600, "num_range": 578255862583271047, "upper_bound": "872162839"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 6552932309335514513, "upper_bound": "1064791239"}, {"distinct_range": 8113800869510389000, "num_eq": 2000000000, "num_range": 8113800869510388631, "upper_bound": "1401148502"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 2453374008080571385, "row_count": 2802328152315298194}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 240419971653061084, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2802328152315298194}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1257454344763864450, "histo_buckets": [{"distinct_range": 0, "num_eq": 5645859753122006819, "num_range": 0, "upper_bound": "385774356"}, {"distinct_range": 10, "num_eq": 5456083273302051241, "num_range": 10, "upper_bound": "1179500324"}, {"distinct_range": 0, "num_eq": 5656370442277397124, "num_range": 5000, "upper_bound": "1571801757"}, {"distinct_range": 10000, "num_eq": 2253734623172421967, "num_range": 10000, "upper_bound": "3012256557"}, {"distinct_range": 3672415324162783000, "num_eq": 50000, "num_range": 6172679270196077209, "upper_bound": "3968098848"}, {"distinct_range": 2398652683126536700, "num_eq": 100000000000, "num_range": 3337154154176801463, "upper_bound": "4051872886"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 2802328152315298194}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2396944865872396884, "histo_buckets": [{"distinct_range": 0, "num_eq": 1554220102250287202, "num_range": 0, "upper_bound": "BOX(-1.5928672306263842 0.88101569282295,-0.6027429456985882 1.1866040257654298)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 2802328152315298194}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2258266758850031104, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2802328152315298194}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13896816051592695, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": "1991-02-21 06:22:16.000906"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 5000, "upper_bound": "2017-04-16 18:24:40.000292"}, {"distinct_range": 2671165042539657700, "num_eq": 0, "num_range": 2671165042539657686, "upper_bound": "2022-05-18 12:52:22.000949"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 2802328152315298194}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2015154941831009131, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000000, "num_range": 0, "upper_bound": "550740407"}, {"distinct_range": 7000000, "num_eq": 80000000, "num_range": 7000000, "upper_bound": "1752163016"}, {"distinct_range": 4782310993.532006, "num_eq": 10000000000, "num_range": 10000000000, "upper_bound": "1993227277"}, {"distinct_range": 3841168922830981600, "num_eq": 50000000000, "num_range": 3841168922830981681, "upper_bound": "2470718284"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 2802328152315298194}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1412393881418189164, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1499900420920186287}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 831533966678323384, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000000, "num_range": 0, "upper_bound": "41 years 7 mons 364 days 07:03:48.64513"}, {"distinct_range": 71798.98702164402, "num_eq": 2105694684832547958, "num_range": 80000, "upper_bound": "50 years 11 mons 68 days 16:54:01.088455"}, {"distinct_range": 400, "num_eq": 50000000, "num_range": 400, "upper_bound": "75 years 11 mons 919 days 21:56:41.458495"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 1499900420920186287}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1400036815719513328, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1499900420920186287}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46913642058329815, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1499900420920186287}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1140795006614378490, "histo_col_type": "", "name": "__auto__", "null_count": 525307780119127588, "row_count": 1499900420920186287}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1069566654738134062, "histo_col_type": "", "name": "__auto__", "null_count": 553361692080036445, "row_count": 1499900420920186287}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 182287711647610935, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000, "num_range": 0, "upper_bound": "06:47:20.446228"}, {"distinct_range": 80000000, "num_eq": 20000000, "num_range": 80000000, "upper_bound": "06:58:10.83706"}, {"distinct_range": 2674970669.3037114, "num_eq": 2000, "num_range": 10000000000, "upper_bound": "11:19:57.067434"}, {"distinct_range": 7194351826207613000, "num_eq": 30000000, "num_range": 7194351826207612576, "upper_bound": "14:42:27.536399"}, {"distinct_range": 30000000, "num_eq": 1062610258102334908, "num_range": 30000000, "upper_bound": "15:44:39.933299"}, {"distinct_range": 433853265911801500, "num_eq": 5598537114769946508, "num_range": 2244676661732392914, "upper_bound": "19:26:20.841519"}, {"distinct_range": 3016480457354627600, "num_eq": 3000000000, "num_range": 5324432721690747831, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 1182563042986986110, "row_count": 1499900420920186287}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 697517974335619193, "histo_col_type": "", "name": "__auto__", "null_count": 386320832631479456, "row_count": 1499900420920186287}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 547015363899855582, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1499900420920186287}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 253663278933744549, "histo_col_type": "", "name": "__auto__", "null_count": 1139943087628132559, "row_count": 1499900420920186287}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 777932321628526728, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 0, "num_eq": 2470127423336823092, "num_range": 100000000, "upper_bound": "\\\\x2d"}, {"distinct_range": 80000000, "num_eq": 800000, "num_range": 80000000, "upper_bound": "\\\\x3f2e65d92061"}, {"distinct_range": 2973.812416339503, "num_eq": 30, "num_range": 5000, "upper_bound": "\\\\xe656528b1c"}, {"distinct_range": 4748023578803654000, "num_eq": 8000000, "num_range": 4748023578803653949, "upper_bound": "\\\\xf5bb80f493e7"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 1499900420920186287}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 495445796086498051, "histo_col_type": "", "name": "__auto__", "null_count": 1067837478993661559, "row_count": 1499900420920186287}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1305349170732408148, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1499900420920186287}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1203218602923435984, "histo_col_type": "", "name": "__auto__", "null_count": 416206196548508198, "row_count": 1499900420920186287}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS e'[{"columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22, "histo_buckets": [{"distinct_range": 0, "num_eq": 2424369880162946555, "num_range": 0, "upper_bound": "\\u0010:I"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 40000000000, "upper_bound": "\\u001a\\u001f"}, {"distinct_range": 0, "num_eq": 1069642519600363508, "num_range": 5762044224631516193, "upper_bound": "&"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 600}, {"columns": ["col5_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3, "histo_buckets": [{"distinct_range": 0, "num_eq": 5561377449875566818, "num_range": 0, "upper_bound": ""}, {"distinct_range": 722427008.686935, "num_eq": 3328363043483775167, "num_range": 8000000000, "upper_bound": "\\u0019"}, {"distinct_range": 6000000, "num_eq": 4619776006469679274, "num_range": 6000000, "upper_bound": "5"}, {"distinct_range": 10000000, "num_eq": 6786837202787418846, "num_range": 10000000, "upper_bound": "R"}, {"distinct_range": 2000, "num_eq": 2903273240642390548, "num_range": 2000, "upper_bound": "X"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 600}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 166, "histo_buckets": [{"distinct_range": 0, "num_eq": 857071130905601725, "num_range": 0, "upper_bound": "-0.24887514114379883"}, {"distinct_range": 1154366464916854800, "num_eq": 3565669173791107339, "num_range": 1518956283137755498, "upper_bound": "0.4064579904079437"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 600}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('wwyruv', 'xtgtb', 'jun', 'jwzo');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('x', 'fnco', 'c', 'orz', 'tu', 'lcilx');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2351449945363747977, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7750864241727666052}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3555844620957282547, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7750864241727666052}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1151690628831558856, "histo_buckets": [{"distinct_range": 0, "num_eq": 599947449529408461, "num_range": 0, "upper_bound": "85594054"}, {"distinct_range": 3181593657167941000, "num_eq": 40000000000, "num_range": 3181593657167940927, "upper_bound": "1237113276"}, {"distinct_range": 70000000, "num_eq": 70000000000, "num_range": 70000000, "upper_bound": "2468920948"}, {"distinct_range": 3087085801907546000, "num_eq": 339506370431428192, "num_range": 7101160182207256748, "upper_bound": "2833902542"}, {"distinct_range": 1000000, "num_eq": 2594020262387137877, "num_range": 1000000, "upper_bound": "2888288314"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 4536867238755627815, "row_count": 7750864241727666052}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 229908323697576382, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "1973-05-26 09:55:14.000742"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 4000000000, "upper_bound": "1979-08-11 15:46:29.000047"}, {"distinct_range": 70.5833700342093, "num_eq": 5457687439675046314, "num_range": 100, "upper_bound": "1987-11-07 14:31:26.000122"}, {"distinct_range": 0, "num_eq": 900000000, "num_range": 3025253726320578633, "upper_bound": "1988-08-02 22:03:58.00049"}, {"distinct_range": 4436014373279223000, "num_eq": 505047076983300930, "num_range": 4436014373279222542, "upper_bound": "1994-02-11 01:07:20.000582"}, {"distinct_range": 0, "num_eq": 6480872634602735382, "num_range": 7168366020535411338, "upper_bound": "2002-09-07 11:58:59.000151"}, {"distinct_range": 0, "num_eq": 8949374179935050162, "num_range": 0, "upper_bound": "2020-04-29 21:44:10.000269"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 2772366832720734514, "row_count": 7750864241727666052}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4961858287051712514, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 7320352459739315000, "num_eq": 1243546449988152534, "num_range": 7320352459739315248, "upper_bound": "\\\\xa562fbc6"}, {"distinct_range": 0, "num_eq": 9056458112344088003, "num_range": 5800647844343490873, "upper_bound": "\\\\xca9087e8d67f"}, {"distinct_range": 0, "num_eq": 4000000, "num_range": 7000000, "upper_bound": "\\\\xd48e90217d84"}, {"distinct_range": 46173904.11920702, "num_eq": 4195295637732304539, "num_range": 60000000, "upper_bound": "\\\\xe03033b0"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 7750864241727666052}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 778635650357573098, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "182295324"}, {"distinct_range": 50000000, "num_eq": 6000000, "num_range": 50000000, "upper_bound": "1856795626"}, {"distinct_range": 700000000, "num_eq": 4784956438411642998, "num_range": 700000000, "upper_bound": "2923368611"}, {"distinct_range": 4978694290291445000, "num_eq": 5000000000, "num_range": 4978694290291444646, "upper_bound": "3581451739"}, {"distinct_range": 52210768455.677124, "num_eq": 7365992471661248470, "num_range": 100000000000, "upper_bound": "3752049637"}, {"distinct_range": 2000, "num_eq": 1000, "num_range": 2000, "upper_bound": "3850839323"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 7750864241727666052}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7050906377290601722, "histo_buckets": [{"distinct_range": 0, "num_eq": 3838647962126595188, "num_range": 0, "upper_bound": "\\\\x296d66697406"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 7750864241727666052}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1164, "histo_buckets": [{"distinct_range": 0, "num_eq": 5016015789624271941, "num_range": 0, "upper_bound": "0001000101010000101111111100001111"}, {"distinct_range": 9000000000, "num_eq": 9147955967708158717, "num_range": 9000000000, "upper_bound": "0001110011001000111001011011011101"}, {"distinct_range": 100000000000, "num_eq": 500000, "num_range": 100000000000, "upper_bound": "0011011110011011001000111111001000"}, {"distinct_range": 7237269588926117000, "num_eq": 0, "num_range": 7237269588926116412, "upper_bound": "0110001011011110011010001100100111"}, {"distinct_range": 2487593.7911017477, "num_eq": 2000, "num_range": 4000000, "upper_bound": "1001110101010100010101001101011101"}, {"distinct_range": 5916828122417631000, "num_eq": 50000000, "num_range": 5916828122417630841, "upper_bound": "1010100011100000000011101000011001"}, {"distinct_range": 3300227592194407000, "num_eq": 6000000000, "num_range": 7178022188341454085, "upper_bound": "1011010111110000010101001010110100"}], "histo_col_type": "BIT(34)", "name": "__auto__", "null_count": 1344, "row_count": 8000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 782, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000, "num_range": 0, "upper_bound": ""}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 4277, "row_count": 8000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1209, "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 967, "row_count": 8000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4363, "histo_col_type": "", "name": "__auto__", "null_count": 6887, "row_count": 8000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5232, "histo_buckets": [{"distinct_range": 0, "num_eq": 90, "num_range": 0, "upper_bound": "\\\\x63"}, {"distinct_range": 5117654117429577000, "num_eq": 9000000, "num_range": 5117654117429576669, "upper_bound": "\\\\x961c5eeacd80"}, {"distinct_range": 233468.39945449543, "num_eq": 7681967250697992259, "num_range": 400000, "upper_bound": "\\\\xe29883"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4192, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6489, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 56760233, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 20712644338389976, "num_eq": 100, "num_range": 213687986525564127, "upper_bound": "2385105764"}, {"distinct_range": 4382338051160047600, "num_eq": 20000000000, "num_range": 4382338051160047810, "upper_bound": "3348974681"}, {"distinct_range": 0, "num_eq": 30000000, "num_range": 0, "upper_bound": "3693941760"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 43555978, "histo_buckets": [{"distinct_range": 0, "num_eq": 8840147089223907722, "num_range": 0, "upper_bound": "240364103"}, {"distinct_range": 40000, "num_eq": 6000, "num_range": 40000, "upper_bound": "319960165"}, {"distinct_range": 690692.1525056146, "num_eq": 200, "num_range": 900000, "upper_bound": "1239355450"}, {"distinct_range": 1395969708841744600, "num_eq": 9000000, "num_range": 2268053421268772426, "upper_bound": "1292602419"}, {"distinct_range": 556095006487480400, "num_eq": 40000000, "num_range": 556095006487480409, "upper_bound": "1762840290"}, {"distinct_range": 1565.28806038236, "num_eq": 9097513829917655154, "num_range": 5000, "upper_bound": "2363246113"}, {"distinct_range": 1370523102841459200, "num_eq": 4470198521008183659, "num_range": 2692277517130040676, "upper_bound": "3346600063"}, {"distinct_range": 8000, "num_eq": 1000, "num_range": 8000, "upper_bound": "3735417302"}, {"distinct_range": 200, "num_eq": 20000, "num_range": 200, "upper_bound": "4262228768"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 40119982, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "BOX(-2.5407259946828926 -0.7395928068009939,-0.35720843412109016 1.5562018613321684)"}, {"distinct_range": 0, "num_eq": 2276355345432407458, "num_range": 8144129220030594195, "upper_bound": "BOX(-0.5103117488186076 -1.625707723051625,0.15768712022036294 -1.1328191963209862)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 57486781, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11493648, "histo_col_type": "INT4", "name": "__auto__", "null_count": 81698793, "row_count": 100000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9790516, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39263126, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 50664393, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25023580, "histo_buckets": [{"distinct_range": 0, "num_eq": 6007060618106174521, "num_range": 0, "upper_bound": "0001-01-01 00:00:00"}, {"distinct_range": 0, "num_eq": 5357218840041571045, "num_range": 7403921781785471342, "upper_bound": "1975-10-08 17:21:23.000338"}, {"distinct_range": 0, "num_eq": 8831191993617583981, "num_range": 5003526626812792335, "upper_bound": "2001-07-23 18:43:25.000287"}, {"distinct_range": 0, "num_eq": 3901529945280868612, "num_range": 300, "upper_bound": "2008-08-30 08:27:55.000702"}, {"distinct_range": 0, "num_eq": 1654216527820834478, "num_range": 2156577268498810403, "upper_bound": "2031-11-09 04:26:03.00016"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 100000000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4181629011209568743, "histo_col_type": "", "name": "__auto__", "null_count": 108712048800349169, "row_count": 7965508851241175357}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6015684059446542152, "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 7965508851241175357}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1869895127355980775, "histo_buckets": [{"distinct_range": 0, "num_eq": 2854195241825845886, "num_range": 0, "upper_bound": "-50 years -1 mons -217 days -00:15:07.030068"}, {"distinct_range": 1244054136416429600, "num_eq": 5729143226047329136, "num_range": 1244054136416429686, "upper_bound": "-13 years -10 mons -394 days -00:41:36.76606"}, {"distinct_range": 47322582630.3608, "num_eq": 5483705770985538011, "num_range": 80000000000, "upper_bound": "-10 years -6 mons -419 days -04:36:29.74956"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 7965508851241175357}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4428821591581074345, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7965508851241175357}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3326045125947093043, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7965508851241175357}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3186108050554505132, "histo_col_type": "", "name": "__auto__", "null_count": 1820189224286210457, "row_count": 7965508851241175357}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5293829687710845689, "histo_col_type": "", "name": "__auto__", "null_count": 4465733489035383357, "row_count": 7965508851241175357}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4212813530026719722, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7965508851241175357}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5726715574508727412, "histo_col_type": "", "name": "__auto__", "null_count": 5218609036500277202, "row_count": 7965508851241175357}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6722719018390172666, "histo_col_type": "", "name": "__auto__", "null_count": 491164620193368882, "row_count": 7965508851241175357}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 381197959807678786, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7965508851241175357}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 940324122364380937, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000000, "num_range": 0, "upper_bound": "00:14:59.175574"}, {"distinct_range": 904441972748751100, "num_eq": 8000000000, "num_range": 904441972748751126, "upper_bound": "04:42:40.096948"}, {"distinct_range": 0, "num_eq": 400, "num_range": 0, "upper_bound": "10:15:29.918838"}, {"distinct_range": 0, "num_eq": 20000000000, "num_range": 0, "upper_bound": "14:18:31.5707"}, {"distinct_range": 80000, "num_eq": 140108138468596690, "num_range": 80000, "upper_bound": "17:08:20.389168"}, {"distinct_range": 2956622964654857700, "num_eq": 10000000000, "num_range": 4143708461617552762, "upper_bound": "23:28:19.250556"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 5743919835211503707, "row_count": 7965508851241175357}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3633318577367383609, "histo_col_type": "", "name": "__auto__", "null_count": 2227574163539614117, "row_count": 7965508851241175357}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2214948033712214879, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7965508851241175357}]';
        test_1        |             ALTER TABLE table5 INJECT STATISTICS e'[{"columns": ["col5_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 152366, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000, "num_range": 0, "upper_bound": "\\r"}, {"distinct_range": 15745123156.760765, "num_eq": 2000000000, "num_range": 30000000000, "upper_bound": "J"}, {"distinct_range": 0, "num_eq": 4285558397801141791, "num_range": 4507855030854162431, "upper_bound": "Q"}, {"distinct_range": 0, "num_eq": 9211889823231251278, "num_range": 0, "upper_bound": "Y"}, {"distinct_range": 2695031867645575700, "num_eq": 800, "num_range": 2695031867645575748, "upper_bound": "{"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 78597, "histo_buckets": [{"distinct_range": 0, "num_eq": 532907915505074791, "num_range": 0, "upper_bound": "-1.4327157735824585"}, {"distinct_range": 0, "num_eq": 300, "num_range": 800, "upper_bound": "-1.2121526002883911"}, {"distinct_range": 3687171047568321000, "num_eq": 80000, "num_range": 3687171047568320842, "upper_bound": "-0.2567712068557739"}, {"distinct_range": 5825395257.613815, "num_eq": 10000000, "num_range": 20000000000, "upper_bound": "0.0"}, {"distinct_range": 7672523204118694000, "num_eq": 91041164771212324, "num_range": 7672523204118694292, "upper_bound": "0.3406580686569214"}, {"distinct_range": 80, "num_eq": 20000000, "num_range": 80, "upper_bound": "0.5824636816978455"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 1000, "upper_bound": "0.6532066464424133"}, {"distinct_range": 4122882864259637000, "num_eq": 0, "num_range": 4122882864259637129, "upper_bound": "0.8092713952064514"}, {"distinct_range": 600, "num_eq": 2783033488977553230, "num_range": 600, "upper_bound": "1.0"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 490480, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "\\u0008)"}, {"distinct_range": 5000000, "num_eq": 100, "num_range": 5000000, "upper_bound": "\\u001eo#\\u0001~-2"}, {"distinct_range": 0, "num_eq": 8979142647208414153, "num_range": 0, "upper_bound": "A"}, {"distinct_range": 40000000, "num_eq": 10, "num_range": 40000000, "upper_bound": "Y"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 100000000, "upper_bound": "_.cg2!I"}, {"distinct_range": 72804850.07300457, "num_eq": 329864117265454112, "num_range": 100000000, "upper_bound": "vY"}, {"distinct_range": 0, "num_eq": 5998963390316527645, "num_range": 0, "upper_bound": "\U00002603"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 500000}]';
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_1) REFERENCES table2 (col1_6);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7605075183, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1988-04-29 07:12:13.000943"}, {"distinct_range": 0, "num_eq": 2000000000, "num_range": 4000000000, "upper_bound": "3000-01-01 00:00:00"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 27018789962, "row_count": 30000000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14155897693, "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6086204067, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "222682998"}, {"distinct_range": 2686.230747119365, "num_eq": 1736783842136662108, "num_range": 40000, "upper_bound": "549439179"}, {"distinct_range": 3873759.854559747, "num_eq": 5000000, "num_range": 9000000, "upper_bound": "2702117015"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5351020096, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "\\\\x146dcf23ac129d6b"}, {"distinct_range": 2675366849044879400, "num_eq": 80000, "num_range": 7327625737767688346, "upper_bound": "\\\\x1b51aef981a44cf338"}, {"distinct_range": 2191059808181550600, "num_eq": 3242209611014254493, "num_range": 2602138940669550254, "upper_bound": "\\\\x1f1b32e21ab4d32bfb"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 665846986310167489, "upper_bound": "\\\\x4d5dda8e95"}, {"distinct_range": 0, "num_eq": 7234121314823971828, "num_range": 6668917923303682209, "upper_bound": "\\\\x93d2"}, {"distinct_range": 100, "num_eq": 20000000, "num_range": 100, "upper_bound": "\\\\x9d49"}, {"distinct_range": 0, "num_eq": 9009375609661831720, "num_range": 2145996074090762834, "upper_bound": "\\\\xa9d77d3e8c36"}, {"distinct_range": 4075129129667120000, "num_eq": 400000000, "num_range": 4075129129667120022, "upper_bound": "\\\\xf62c"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3779754735, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5398784491, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28453699419, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000, "num_range": 0, "upper_bound": "7040291"}, {"distinct_range": 0, "num_eq": 30000, "num_range": 0, "upper_bound": "665558261"}, {"distinct_range": 6392199.2740126345, "num_eq": 80, "num_range": 10000000, "upper_bound": "778937263"}, {"distinct_range": 200000, "num_eq": 6249222281382196116, "num_range": 200000, "upper_bound": "3281941605"}, {"distinct_range": 5334527645.448419, "num_eq": 5201008148599890427, "num_range": 9000000000, "upper_bound": "3413220607"}, {"distinct_range": 0, "num_eq": 1621670276817489435, "num_range": 2000000000, "upper_bound": "4241121546"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 13409895300, "row_count": 30000000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2598465044023796992, "histo_buckets": [{"distinct_range": 0, "num_eq": 800, "num_range": 0, "upper_bound": "\\\\x1a84"}, {"distinct_range": 37.02355401167709, "num_eq": 4804517971451992041, "num_range": 70, "upper_bound": "\\\\x27"}, {"distinct_range": 5175116002266576000, "num_eq": 30000000000, "num_range": 5175116002266576079, "upper_bound": "\\\\x430b"}, {"distinct_range": 0, "num_eq": 3704714047924276427, "num_range": 8296998564408279059, "upper_bound": "\\\\x560a3f30a847367db5"}, {"distinct_range": 50000000, "num_eq": 7372088010533477870, "num_range": 50000000, "upper_bound": "\\\\x571841e3fb1d84c5"}, {"distinct_range": 2209.818496019986, "num_eq": 80000000000, "num_range": 3000, "upper_bound": "\\\\xcd57b6a6"}, {"distinct_range": 0, "num_eq": 500000, "num_range": 8000, "upper_bound": "\\\\xdda1"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 8111945407442582234}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4919695200895377276, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8111945407442582234}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2586279570284681932, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8111945407442582234}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7812925687618810351, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000, "num_range": 0, "upper_bound": "1000101011000101010101101001010111"}, {"distinct_range": 0, "num_eq": 3681914202669789460, "num_range": 0, "upper_bound": "1010011011100000100001110111100001"}, {"distinct_range": 0, "num_eq": 819841412201910101, "num_range": 6448272243618752506, "upper_bound": "1010100101011100000101000110110100"}], "histo_col_type": "BIT(34)", "name": "__auto__", "null_count": 4138654152876823178, "row_count": 8111945407442582234}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7659620631676822292, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 9202558770478359028, "num_range": 2457432531938562313, "upper_bound": "!U\\u0003__;\\u0003>"}, {"distinct_range": 807781958855061200, "num_eq": 400, "num_range": 5105422470885785008, "upper_bound": ">;\\nF1"}, {"distinct_range": 0, "num_eq": 3908773233604019558, "num_range": 7512085408714108106, "upper_bound": "DHuaj8v\\u0015"}, {"distinct_range": 3272513422318978600, "num_eq": 700000, "num_range": 3272513422318978738, "upper_bound": "S0S$/\\\\~"}, {"distinct_range": 0, "num_eq": 2609577383795645713, "num_range": 30000000000, "upper_bound": "q+:s\\u00184"}, {"distinct_range": 20000, "num_eq": 3349749317181517505, "num_range": 20000, "upper_bound": "x"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 4163967780415112340, "row_count": 8111945407442582234}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1407043329514103702, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000000, "num_range": 0, "upper_bound": "181756502"}, {"distinct_range": 20000, "num_eq": 100000, "num_range": 20000, "upper_bound": "274531967"}, {"distinct_range": 185896.5768575723, "num_eq": 10000, "num_range": 200000, "upper_bound": "1345956016"}, {"distinct_range": 300, "num_eq": 6452968650953294496, "num_range": 300, "upper_bound": "1467431462"}, {"distinct_range": 815022154.9294838, "num_eq": 300000, "num_range": 10000000000, "upper_bound": "2983088390"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 5013271665880615615, "row_count": 8111945407442582234}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 446110416658442691, "histo_col_type": "", "name": "__auto__", "null_count": 6184852576508509952, "row_count": 8111945407442582234}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3673, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3567, "histo_buckets": [{"distinct_range": 0, "num_eq": 700, "num_range": 0, "upper_bound": "-4713-11-24 00:00:00"}, {"distinct_range": 8000000000, "num_eq": 2941510730267934413, "num_range": 8000000000, "upper_bound": "1984-07-09 19:53:19.000934"}, {"distinct_range": 1000, "num_eq": 100000000, "num_range": 1000, "upper_bound": "1989-01-03 15:42:14.000201"}, {"distinct_range": 0, "num_eq": 4603282165730614773, "num_range": 75300684851072920, "upper_bound": "1991-10-24 13:29:30.000557"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 600000000, "upper_bound": "1995-08-24 12:26:22.000902"}, {"distinct_range": 30000000000, "num_eq": 10, "num_range": 30000000000, "upper_bound": "2003-06-23 23:27:26.000622"}, {"distinct_range": 29395312615.971817, "num_eq": 3000000000, "num_range": 40000000000, "upper_bound": "2005-04-03 16:41:35.000651"}, {"distinct_range": 0, "num_eq": 2005192970086666731, "num_range": 0, "upper_bound": "2021-09-14 00:03:42.000542"}, {"distinct_range": 208263215421264740, "num_eq": 30000000000, "num_range": 8874810626561859408, "upper_bound": "2030-05-16 02:39:33.000338"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2155, "histo_buckets": [{"distinct_range": 0, "num_eq": 3427417519074360185, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 30000000000, "num_range": 2753715166059071630, "upper_bound": "347507334"}, {"distinct_range": 3602112484543599000, "num_eq": 9000, "num_range": 4218078826814728968, "upper_bound": "402396595"}, {"distinct_range": 17.051475312389016, "num_eq": 549200846603790845, "num_range": 40, "upper_bound": "2831651795"}, {"distinct_range": 0, "num_eq": 2934981130826433623, "num_range": 100000, "upper_bound": "4140387811"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 20000, "upper_bound": "4198707704"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4174, "histo_buckets": [{"distinct_range": 0, "num_eq": 2042905430872938551, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 0, "num_eq": 30000, "num_range": 100000, "upper_bound": "BOX(-1.1193410644420565 0.4570624407676416,-0.2637158303998386 1.5771645412576518)"}, {"distinct_range": 2216218221221140500, "num_eq": 6000000, "num_range": 2216218221221140370, "upper_bound": "BOX(-0.7647323743407695 -0.6646472524073088,-0.01631844618690259 0.8496057904264187)"}, {"distinct_range": 0, "num_eq": 500, "num_range": 2028454861660287189, "upper_bound": "BOX(-0.29444727795029574 -1.3556611787717325,2.1655626367714897 0.4353644327661854)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3389, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8522, "histo_buckets": [{"distinct_range": 0, "num_eq": 8684156343839065438, "num_range": 0, "upper_bound": "-1302780951"}, {"distinct_range": 0, "num_eq": 4973278912483548770, "num_range": 3091877876574525777, "upper_bound": "-976587274"}, {"distinct_range": 2000000, "num_eq": 7275167590070800246, "num_range": 2000000, "upper_bound": "-531267102"}, {"distinct_range": 0, "num_eq": 8216629763838698716, "num_range": 3388959733547786482, "upper_bound": "-128"}, {"distinct_range": 7468350565471388000, "num_eq": 7000, "num_range": 7468350565471387577, "upper_bound": "32767"}, {"distinct_range": 0, "num_eq": 7545107710918046887, "num_range": 2424480291940425544, "upper_bound": "156649220"}, {"distinct_range": 0, "num_eq": 0, "num_range": 300000000, "upper_bound": "824393146"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 2622, "row_count": 10000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2655, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 600000000, "num_eq": 1766938877500921909, "num_range": 600000000, "upper_bound": "971266385"}, {"distinct_range

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2546132&tab=buildLog) on [master@08c89a597a06520c30faf01965f9c74fe9b9854f](https://github.com/cockroachdb/cockroach/commits/08c89a597a06520c30faf01965f9c74fe9b9854f):

```
        test_1        |             CREATE INDEX ON table3 (col3_11, col3_16 ASC, col3_9 DESC, col3_4 ASC, col3_13 DESC, col3_3, col3_7 DESC, col3_1, col3_5) INCLUDE (col3_0, col3_2, col3_8, col3_14);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_14 DESC, col3_5 ASC, col3_10, col3_4 DESC, col3_13 ASC, col3_16 ASC, col3_15 ASC, col3_7 DESC, col3_9 ASC, col3_11 DESC, col3_1 DESC, col3_12 DESC) INCLUDE (col3_0, col3_8);
        test_1        |             CREATE TABLE table4 (col4_0 REGNAMESPACE NOT NULL, col4_1 TIMESTAMPTZ NOT NULL, col4_2 UUID NOT NULL, col4_3 NAME NOT NULL, col4_4 INT8 NOT NULL, col4_5 TEXT NOT NULL, col4_6 INT8[] NULL, col4_7 REGPROCEDURE NOT NULL, col4_8 BOOL, col4_9 TIMESTAMP NOT NULL, col4_10 INT8 NOT NULL, col4_11 TIMESTAMP NOT NULL, col4_12 VARCHAR NOT NULL, col4_13 REGPROCEDURE, col4_14 REGPROC NOT NULL, col4_15 INT8 NULL, col4_16 INTERVAL NOT NULL, PRIMARY KEY (col4_16, col4_4, col4_11, col4_5, col4_0, col4_1, col4_10, col4_12, col4_3, col4_9, col4_2));
        test_1        |             CREATE INDEX ON table4 (col4_11 ASC, col4_2 DESC, col4_0, col4_3, col4_4 ASC, col4_16 ASC, col4_14 ASC, col4_7, col4_1 ASC) INCLUDE (col4_6, col4_13, col4_15);
        test_1        |             CREATE INDEX ON table4 (col4_1 ASC, col4_16 DESC, col4_15, col4_5 ASC, col4_11 ASC, col4_13 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_4 DESC, col4_14, col4_7 DESC, col4_0 DESC, col4_16, col4_13 ASC, col4_11 ASC, col4_8 ASC, col4_3 DESC, col4_9, col4_5 ASC, col4_10 ASC);
        test_1        |             CREATE TABLE table5 (col5_0 INT8 NOT NULL, col5_1 INTERVAL NOT NULL, col5_2 REGPROC NOT NULL, col5_3 REGTYPE NOT NULL, col5_4 TIMESTAMPTZ NOT NULL, col5_5 DECIMAL NOT NULL, col5_6 BOX2D[] NOT NULL, col5_7 NAME NOT NULL, col5_8 FLOAT8 NOT NULL, col5_9 REGTYPE NOT NULL, col5_10 DECIMAL NOT NULL, col5_11 REGTYPE NOT NULL, col5_12 INT8 NOT NULL, col5_13 DATE NOT NULL, col5_14 TEXT NOT NULL, col5_15 INT8, col5_16 BIT(19), col5_17 CHAR NOT NULL, col5_18 INT8 NOT NULL, PRIMARY KEY (col5_2, col5_14, col5_5, col5_3, col5_17, col5_9, col5_18, col5_7, col5_11, col5_12, col5_4, col5_0, col5_10, col5_1, col5_8));
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_18 DESC, col5_7 ASC, col5_10 ASC, col5_5 ASC, col5_15 DESC, col5_9, col5_0 DESC, col5_4, col5_1 DESC, col5_14 DESC, col5_12) INCLUDE (col5_16);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_2, col5_11, col5_8 DESC, col5_0 DESC, col5_14 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_11 ASC, col5_13, col5_15, col5_10 DESC, col5_16 DESC, col5_14 DESC, col5_1 DESC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('ynhpw', 'sff', 'pnzsfb', 'jqpgo', 'kxyu', 'ca');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (1.10s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 TIMESTAMPTZ NULL, col1_1 NAME NULL, col1_2 VARCHAR NOT NULL, col1_3 DATE NOT NULL, PRIMARY KEY (col1_2 DESC, col1_3), INDEX (col1_0) STORING (col1_1), FAMILY (col1_3, col1_1), FAMILY (col1_0, col1_2), UNIQUE (col1_1) WHERE table1.col1_0 >= '294276-12-31 23:59:59.999999+00:00');
        test_1        |             CREATE TABLE table2 (col1_2 VARCHAR NOT NULL, col1_3 DATE NOT NULL, col2_2 TIMETZ, col2_3 BOX2D NULL, col2_4 DATE NULL, col2_5 BOX2D NULL, PRIMARY KEY (col1_2 DESC, col1_3), FAMILY (col2_5, col1_3), FAMILY (col2_3, col1_2, col2_2), FAMILY (col2_4)) INTERLEAVE IN PARENT table1 (col1_2, col1_3);
        test_1        |             CREATE TABLE table3 (col3_0 REGCLASS NOT NULL, col3_1 UUID NULL, col3_2 TIMESTAMP NOT NULL, col3_3 BIT(7) NOT NULL, col3_4 REGPROC NULL, col3_5 BYTES NOT NULL, col3_6 UUID, col3_7 UUID NOT NULL, col3_8 VARCHAR NOT NULL, col3_9 INET NOT NULL, col3_10 BOX2D NOT NULL, col3_11 INT2 NOT NULL, col3_12 BOOL NOT NULL, col3_13 REGCLASS NULL, col3_14 VARCHAR NOT NULL, col3_15 BIT(28) NOT NULL, PRIMARY KEY (col3_11 DESC, col3_14, col3_7 ASC, col3_3 ASC, col3_15 ASC, col3_9 ASC), UNIQUE (col3_13, col3_7, col3_5, col3_2 DESC, col3_10 DESC, col3_4 DESC, col3_6 DESC, col3_9, col3_8 DESC, col3_12 ASC, col3_11 DESC, col3_3, col3_0 DESC, col3_15) STORING (col3_1) WHERE (((((table3.col3_2 > '294276-12-31 23:59:59.999999') OR (NOT table3.col3_12)) AND (table3.col3_5 != '\x')) AND (table3.col3_11 > 0)) OR (table3.col3_8 > '"')) AND (table3.col3_14 < '"'), UNIQUE (col3_5, col3_3 DESC) STORING (col3_4, col3_6, col3_12) WHERE ((((table3.col3_11 > 1:::INT8) AND (table3.col3_2 = '3000-01-01 00:00:00':::TIMESTAMP)) OR (table3.col3_5 != e'\\x22':::BYTES)) AND table3.col3_12) OR (table3.col3_8 < 'X':::STRING), INDEX (col3_4, col3_1 ASC, col3_15 DESC, col3_0 ASC, col3_9, col3_11, col3_5 ASC, col3_7 ASC, col3_10, col3_6, col3_2) WHERE NOT table3.col3_12, INDEX (col3_2 DESC, col3_10 DESC, col3_11, col3_6 DESC, col3_7 ASC, col3_12, col3_5 DESC, col3_3, col3_14 ASC, col3_8 DESC, col3_4 ASC, col3_13 ASC, col3_9 ASC, col3_1) WHERE (((((table3.col3_2 >= '-4713-11-24 00:00:00':::TIMESTAMP) OR (table3.col3_5 < e'\\x00':::BYTES)) AND (table3.col3_11 = (-1):::INT8)) OR table3.col3_12) AND (table3.col3_8 != '':::STRING)) AND (table3.col3_14 != '':::STRING), INDEX (col3_13 ASC, col3_15 ASC, col3_12 ASC, col3_14 DESC, col3_0 DESC, col3_9 ASC, col3_7 DESC, col3_3 DESC, col3_5, col3_1, col3_2, col3_6 DESC, col3_4 DESC, col3_11 DESC, col3_8 ASC, col3_10) WHERE ((((table3.col3_5 >= e'\\x58':::BYTES) OR (table3.col3_14 != 'X':::STRING)) AND (table3.col3_8 != e'\x00':::STRING)) OR (table3.col3_2 != '-2000-01-01 00:00:00':::TIMESTAMP)) AND (table3.col3_11 <= 0:::INT8), FAMILY (col3_7, col3_12), FAMILY (col3_1), FAMILY (col3_8), FAMILY (col3_5), FAMILY (col3_14), FAMILY (col3_9), FAMILY (col3_0, col3_4, col3_13), FAMILY (col3_11), FAMILY (col3_15, col3_6), FAMILY (col3_10), FAMILY (col3_3, col3_2));
        test_1        |             CREATE TABLE table4 (col4_0 CHAR NOT NULL, col4_1 NAME NOT NULL, col4_2 BOX2D NOT NULL, PRIMARY KEY (col4_0 ASC, col4_2 DESC, col4_1), INDEX (col4_0 DESC, col4_2, col4_1 DESC) WHERE table4.col4_1 < 'X':::STRING, UNIQUE (col4_2 ASC, col4_1) WHERE (table4.col4_1 <= e'\U00002603') AND (table4.col4_0 != 'X'), INDEX (col4_0 ASC, col4_2 DESC), UNIQUE (col4_2, col4_1, col4_0) WHERE (table4.col4_0 != e'\U00002603') OR (table4.col4_1 < 'X'), INDEX (col4_2) WHERE table4.col4_1 = e'\U00002603', FAMILY (col4_2), FAMILY (col4_1, col4_0));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 56362480, "histo_buckets": [{"distinct_range": 0, "num_eq": 3787234027679095311, "num_range": 0, "upper_bound": "1981-12-04 17:33:54.000673+00:00"}, {"distinct_range": 0, "num_eq": 360583523037452128, "num_range": 4032346505050189947, "upper_bound": "1989-04-27 05:44:49.000724+00:00"}, {"distinct_range": 343325312.5877593, "num_eq": 90000000, "num_range": 30000000000, "upper_bound": "2007-06-19 04:56:05.000985+00:00"}, {"distinct_range": 900000, "num_eq": 5289388067830836685, "num_range": 900000, "upper_bound": "2009-06-08 13:29:22.000807+00:00"}, {"distinct_range": 10000000000, "num_eq": 0, "num_range": 10000000000, "upper_bound": "2016-07-07 15:28:23.000203+00:00"}, {"distinct_range": 0, "num_eq": 0, "num_range": 100000000000, "upper_bound": "2021-10-12 11:53:03.000088+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 33743218, "row_count": 70000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15806914, "histo_col_type": "", "name": "__auto__", "null_count": 24973991, "row_count": 70000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53076678, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000, "num_range": 0, "upper_bound": ""}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 70000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46345188, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70000000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3876, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9416, "histo_col_type": "", "name": "__auto__", "null_count": 9370, "row_count": 10000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9003, "histo_col_type": "", "name": "__auto__", "null_count": 7988, "row_count": 10000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9567, "histo_col_type": "", "name": "__auto__", "null_count": 3158, "row_count": 10000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 974, "histo_col_type": "", "name": "__auto__", "null_count": 5421, "row_count": 10000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1722, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 800, "num_eq": 4000, "num_range": 800, "upper_bound": "\\u0013;\\u0019{~k\\u0018"}, {"distinct_range": 0, "num_eq": 30000000000, "num_range": 8277117422130795925, "upper_bound": "\\u0015\\u0016^"}, {"distinct_range": 1164453005074366700, "num_eq": 709954692911667223, "num_range": 1164453005074366785, "upper_bound": "\\u0016"}, {"distinct_range": 0, "num_eq": 1756785178983562556, "num_range": 2911729597763407956, "upper_bound": "\\u0017M"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 900000, "upper_bound": "!!_lm\'"}, {"distinct_range": 463078428849573500, "num_eq": 3355117575662903701, "num_range": 463078428849573534, "upper_bound": "&4?oplC8"}, {"distinct_range": 700000, "num_eq": 7870740365850166055, "num_range": 700000, "upper_bound": ",:\\\\"}, {"distinct_range": 0, "num_eq": 5636299693199240413, "num_range": 8381774804510585487, "upper_bound": "v"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 10000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5403759412570549842, "histo_buckets": [{"distinct_range": 0, "num_eq": 6070489149922439820, "num_range": 0, "upper_bound": "-2000-01-01 00:00:00"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 90000, "upper_bound": "1976-09-06 02:22:23.000714"}, {"distinct_range": 3000000000, "num_eq": 8000000000, "num_range": 3000000000, "upper_bound": "1977-05-13 23:46:00.000285"}, {"distinct_range": 0, "num_eq": 60000000000, "num_range": 1067979181959969054, "upper_bound": "1999-11-09 01:49:32.000789"}, {"distinct_range": 1423.371032525505, "num_eq": 2331445937108800555, "num_range": 7000, "upper_bound": "2030-09-23 05:52:07.000527"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 8185912126046875976}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5363448103891520291, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8185912126046875976}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5178716773735939352, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 7281294906876141723, "num_range": 3000000, "upper_bound": "110895580"}, {"distinct_range": 961062124.9108759, "num_eq": 1624813933026232695, "num_range": 3000000000, "upper_bound": "2265351905"}, {"distinct_range": 3062094818742316500, "num_eq": 7000, "num_range": 3062094818742316593, "upper_bound": "3468643837"}, {"distinct_range": 142384479352332900, "num_eq": 60000000, "num_range": 1095090809152453548, "upper_bound": "4121450714"}, {"distinct_range": 70000000000, "num_eq": 30000, "num_range": 70000000000, "upper_bound": "4155763322"}, {"distinct_range": 0, "num_eq": 30000, "num_range": 0, "upper_bound": "4165471856"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 2550687226218746521, "row_count": 8185912126046875976}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1482853407355939473, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8185912126046875976}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 48949091752648868, "histo_buckets": [{"distinct_range": 0, "num_eq": 8651532767575989212, "num_range": 0, "upper_bound": "1332448212"}, {"distinct_range": 277870173785700450, "num_eq": 1000000, "num_range": 1303797227982321035, "upper_bound": "1720612291"}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 50000000, "upper_bound": "1732522405"}, {"distinct_range": 800000, "num_eq": 700, "num_range": 800000, "upper_bound": "2745332067"}, {"distinct_range": 0, "num_eq": 3208073344142667549, "num_range": 3000000, "upper_bound": "3378051397"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 3838879244096487545, "row_count": 8185912126046875976}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3161391078051070129, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 2430583737795157500, "num_eq": 1000, "num_range": 7180251161282607944, "upper_bound": "\\\\x00"}, {"distinct_range": 827538527682380900, "num_eq": 400, "num_range": 2430665894798758746, "upper_bound": "\\\\x3f76ec6126ee"}, {"distinct_range": 0, "num_eq": 4956118944041321831, "num_range": 7282053441143513525, "upper_bound": "\\\\x55a86ef5c407640062"}, {"distinct_range": 150431607.31780472, "num_eq": 0, "num_range": 7000000000, "upper_bound": "\\\\x6c8a31069ec0300c94"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 8185912126046875976}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2204109620869867260, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8185912126046875976}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2978027989737229543, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8185912126046875976}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1972366791577886555, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8185912126046875976}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 658087121387309076, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8185912126046875976}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3420057892567001635, "histo_buckets": [{"distinct_range": 0, "num_eq": 5677792017066218921, "num_range": 0, "upper_bound": "-22345"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 8185912126046875976}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3666376043545893986, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8185912126046875976}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 996385209429409000, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8185912126046875976}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4197715270799020867, "histo_col_type": "", "name": "__auto__", "null_count": 4561303492053613323, "row_count": 8185912126046875976}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7697306411598547917, "histo_col_type": "", "name": "__auto__", "null_count": 266430964227974446, "row_count": 8185912126046875976}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8068772512163697911, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8185912126046875976}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6666030360110722133, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000, "num_range": 0, "upper_bound": "\\u0012"}, {"distinct_range": 5270545388130800, "num_eq": 6277637730078364535, "num_range": 11022337543390958, "upper_bound": "\\u0018"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 635745190091591572, "upper_bound": "\\u001b"}, {"distinct_range": 0, "num_eq": 7577226998715705700, "num_range": 0, "upper_bound": "\\u001c"}, {"distinct_range": 7000000000, "num_eq": 40000, "num_range": 7000000000, "upper_bound": "*"}, {"distinct_range": 0, "num_eq": 3367030221392140923, "num_range": 0, "upper_bound": "O"}, {"distinct_range": 841078707.4990809, "num_eq": 9000000, "num_range": 9000000000, "upper_bound": "n"}, {"distinct_range": 0, "num_eq": 3227954706944456582, "num_range": 2930522878948794747, "upper_bound": "{"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 8967340800939881652}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8273600828691554864, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8967340800939881652}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7151217838825880883, "histo_buckets": [{"distinct_range": 0, "num_eq": 7784401632142692627, "num_range": 0, "upper_bound": "BOX(-1.6389798824948663 -1.5868931043207657,-1.2162161089162495 1.3366248512509984)"}, {"distinct_range": 1572940486280283400, "num_eq": 600, "num_range": 6952075261694805530, "upper_bound": "BOX(-0.4586092295845856 0.426870721404614,0.18677602962080453 0.6861127075318691)"}, {"distinct_range": 27.79630480632025, "num_eq": 3481717343448449963, "num_range": 50, "upper_bound": "BOX(0.24674065226222552 0.29187029465442826,0.3630798148884784 1.82402260721534)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 8967340800939881652}]':::JSONB;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col1_2) REFERENCES table1 (col1_1) ON UPDATE RESTRICT;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('kqwjw');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('bxm', 'nxsh', 'wiozum');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('egera', 'p', 'ty', 'w', 'mdan', 'gdrw');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('onnuk', 'f', 'njea', 'rpdjg', 'vn', 'gclm');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2340385184, "histo_buckets": [{"distinct_range": 0, "num_eq": 6476323155912971534, "num_range": 0, "upper_bound": "1972-03-14 16:21:09.000807+00:00"}, {"distinct_range": 3000000000, "num_eq": 2237117131310377075, "num_range": 3000000000, "upper_bound": "1972-08-25 15:56:32.00092+00:00"}, {"distinct_range": 0, "num_eq": 20, "num_range": 0, "upper_bound": "1993-01-06 00:35:24.000321+00:00"}, {"distinct_range": 8776210062982055000, "num_eq": 3225870556612348002, "num_range": 8776210062982054543, "upper_bound": "2020-04-01 12:26:31.000522+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 384316360, "row_count": 7000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4868246450, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000000, "num_range": 0, "upper_bound": "*1\\u0003N"}, {"distinct_range": 0, "num_eq": 3476418066499851428, "num_range": 7474363608951279609, "upper_bound": "4/@o~"}, {"distinct_range": 1550350650800550400, "num_eq": 1336830435888603408, "num_range": 2717525069615202296, "upper_bound": "DFUO\\u0010?[Hk"}, {"distinct_range": 6755953207128361000, "num_eq": 2475873133244652756, "num_range": 6755953207128361021, "upper_bound": "Fa>X"}, {"distinct_range": 0, "num_eq": 200000000, "num_range": 2721713884510047282, "upper_bound": "d\\u0017I\\u000f&;"}, {"distinct_range": 35607834.68530311, "num_eq": 8161506986454565787, "num_range": 60000000, "upper_bound": "nuIV@<"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 2042980563, "row_count": 7000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2986055250, "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1367608098, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44062375180132239, "histo_buckets": [{"distinct_range": 0, "num_eq": 3835792178772584246, "num_range": 0, "upper_bound": "lbU"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 64342814308386086}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16817220032583920, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 64342814308386086}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25079282829786705, "histo_col_type": "", "name": "__auto__", "null_count": 33464716086851160, "row_count": 64342814308386086}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32320290992529488, "histo_col_type": "", "name": "__auto__", "null_count": 53445576778369430, "row_count": 64342814308386086}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22710393380657895, "histo_col_type": "", "name": "__auto__", "null_count": 53162334472852218, "row_count": 64342814308386086}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 43555197284321222, "histo_col_type": "", "name": "__auto__", "null_count": 39835592059953451, "row_count": 64342814308386086}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2157, "histo_col_type": "", "name": "__auto__", "null_count": 845, "row_count": 6000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3131, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3552, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000000, "num_range": 0, "upper_bound": "\\\\x27"}, {"distinct_range": 0, "num_eq": 6030742022728639000, "num_range": 20000000000, "upper_bound": "\\\\x2d5a"}, {"distinct_range": 0, "num_eq": 50, "num_range": 20, "upper_bound": "\\\\x421f483a119f2177cb"}, {"distinct_range": 0, "num_eq": 8484939184098961636, "num_range": 600000, "upper_bound": "\\\\x484806"}, {"distinct_range": 150571289451340220, "num_eq": 4000, "num_range": 7441001462931520777, "upper_bound": "\\\\x54707719da"}, {"distinct_range": 0, "num_eq": 30000000, "num_range": 4000000000, "upper_bound": "\\\\x589ad7adca9039"}, {"distinct_range": 1000, "num_eq": 40000000000, "num_range": 1000, "upper_bound": "\\\\x64adc66cc40c2380"}, {"distinct_range": 1046174438.0050492, "num_eq": 9060793281085794517, "num_range": 3000000000, "upper_bound": "\\\\x6b"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1261, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000, "num_range": 0, "upper_bound": "103178387"}, {"distinct_range": 4.444433522572167, "num_eq": 541532674513360027, "num_range": 10, "upper_bound": "653992611"}, {"distinct_range": 171.0855956931631, "num_eq": 671279228363074381, "num_range": 800, "upper_bound": "832789480"}, {"distinct_range": 5000000, "num_eq": 100, "num_range": 5000000, "upper_bound": "988183028"}, {"distinct_range": 1592017347966688000, "num_eq": 1087513214216395204, "num_range": 3493149555733303719, "upper_bound": "1864960194"}, {"distinct_range": 7.064051250282385, "num_eq": 600, "num_range": 20, "upper_bound": "3111923230"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 2346, "row_count": 6000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4750, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3786, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5125, "histo_buckets": [{"distinct_range": 0, "num_eq": 5552293028087890416, "num_range": 0, "upper_bound": "36989419"}, {"distinct_range": 8225825927093625000, "num_eq": 7585727660161029641, "num_range": 8225825927093624322, "upper_bound": "115364259"}, {"distinct_range": 6300738080189173000, "num_eq": 2135497677618131996, "num_range": 7139976752969250555, "upper_bound": "1221071324"}, {"distinct_range": 0, "num_eq": 50, "num_range": 0, "upper_bound": "1489914962"}, {"distinct_range": 5244317871680306000, "num_eq": 700000000, "num_range": 5657894671816966801, "upper_bound": "1633924465"}, {"distinct_range": 6769056432733082000, "num_eq": 6115234043553741777, "num_range": 8954145140896650708, "upper_bound": "2723212500"}, {"distinct_range": 1000, "num_eq": 700000000, "num_range": 1000, "upper_bound": "3049284152"}, {"distinct_range": 841082578.5561991, "num_eq": 10000000, "num_range": 1000000000, "upper_bound": "3128654272"}, {"distinct_range": 0, "num_eq": 50000000, "num_range": 500000000, "upper_bound": "4238453473"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 3391, "row_count": 6000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1387, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2020, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000000, "num_range": 0, "upper_bound": "-30384"}, {"distinct_range": 8909919590716071000, "num_eq": 2463838453891591863, "num_range": 8909919590716070767, "upper_bound": "-8043"}, {"distinct_range": 3328174319.932272, "num_eq": 100, "num_range": 70000000000, "upper_bound": "-3717"}, {"distinct_range": 15964.494672105799, "num_eq": 600000000, "num_range": 20000, "upper_bound": "13934"}, {"distinct_range": 4000000, "num_eq": 636913002199841960, "num_range": 4000000, "upper_bound": "22343"}, {"distinct_range": 5707426797294423000, "num_eq": 70000, "num_range": 5707426797294423165, "upper_bound": "25135"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 313, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2626, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 804, "histo_col_type": "", "name": "__auto__", "null_count": 2823, "row_count": 6000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2925, "histo_buckets": [{"distinct_range": 0, "num_eq": 6879239246958850805, "num_range": 0, "upper_bound": "-4713-11-24 00:00:00"}, {"distinct_range": 0, "num_eq": 300000, "num_range": 20000, "upper_bound": "1970-06-10 21:03:12.000015"}, {"distinct_range": 0, "num_eq": 5371376696190983505, "num_range": 832495369920939448, "upper_bound": "2012-03-19 01:22:39.000238"}, {"distinct_range": 500, "num_eq": 4593466108529537433, "num_range": 500, "upper_bound": "2026-06-12 03:17:50.000121"}, {"distinct_range": 0, "num_eq": 9000, "num_range": 3398827561562247553, "upper_bound": "294276-12-31 23:59:59.999999"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3563, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5542, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4382298649914469855, "histo_buckets": [{"distinct_range": 0, "num_eq": 6452043607782690083, "num_range": 0, "upper_bound": "\\t"}, {"distinct_range": 5.420789202165185, "num_eq": 1453471255307289984, "num_range": 90, "upper_bound": "i"}, {"distinct_range": 776.6054386899552, "num_eq": 90000000000, "num_range": 800, "upper_bound": "v"}, {"distinct_range": 5951592868415874000, "num_eq": 4262670979635678647, "num_range": 5951592868415873854, "upper_bound": "\x7f"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 8050487332770136955}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2516945257268250586, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8050487332770136955}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4869448550199312419, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 1533163015945919000, "num_eq": 1000, "num_range": 4395552961210286258, "upper_bound": "BOX(-1.6224429051053857 -0.565001007624843,1.9199084841811374 0.7507824397193358)"}, {"distinct_range": 10000, "num_eq": 5447744750667576623, "num_range": 10000, "upper_bound": "BOX(-1.5816918490446445 -0.9769986186682407,1.672843584276203 0.051712558861508384)"}, {"distinct_range": 0, "num_eq": 6379924405575634390, "num_range": 100000000, "upper_bound": "BOX(-0.6169281632782448 -1.2844219020913932,0.003589016761539887 0.8030662071779329)"}, {"distinct_range": 8000000000, "num_eq": 9000000, "num_range": 8000000000, "upper_bound": "BOX(-0.40675468335591947 0.3155688813289356,0.3074055867923969 2.2202409558307603)"}, {"distinct_range": 200000, "num_eq": 10000000, "num_range": 200000, "upper_bound": "BOX(-0.08874122072114721 -1.3387633494991258,2.0951503280680246 0.10841433798332933)"}, {"distinct_range": 3000, "num_eq": 1447346924497774077, "num_range": 3000, "upper_bound": "BOX(0.19415909035535223 0.16769992910012743,0.5857162038761204 1.0711004044112036)"}, {"distinct_range": 9215615.571048897, "num_eq": 6017487436664435261, "num_range": 50000000, "upper_bound": "BOX(0.26390962330976664 -2.069545441351666,0.7676910405241821 0.7879395419678659)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 8050487332770136955}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 221881766381041463, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000, "num_range": 0, "upper_bound": "0001-01-01 00:00:00+00:00"}, {"distinct_range": 304431473774134100, "num_eq": 60, "num_range": 304431473774134108, "upper_bound": "1980-06-21 02:50:35.000508+00:00"}, {"distinct_range": 0, "num_eq": 2711205410394754749, "num_range": 9000000000, "upper_bound": "1989-07-31 13:38:02.000078+00:00"}, {"distinct_range": 10586.061796640748, "num_eq": 0, "num_range": 70000, "upper_bound": "1990-05-22 03:53:21.000675+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 1728665788608191726, "row_count": 2193327727930345304}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 687597784932564834, "histo_col_type": "NAME", "name": "__auto__", "null_count": 1693532091657628724, "row_count": 2193327727930345304}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 775100141669217009, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000000, "num_range": 0, "upper_bound": "\\u0001\\r\\u001c\\u0004.~J"}, {"distinct_range": 41.839106579769265, "num_eq": 5110777018949742869, "num_range": 70, "upper_bound": "\\u0019\\u0018VccD"}, {"distinct_range": 0, "num_eq": 0, "num_range": 70, "upper_bound": "2r&:;"}, {"distinct_range": 7178370331935895000, "num_eq": 7751558514907481618, "num_range": 7178370331935894970, "upper_bound": "]"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 2193327727930345304}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1822568688070019318, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2193327727930345304}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2489394906287970136, "histo_col_type": "", "name": "__auto__", "null_count": 4456233719846455734, "row_count": 4604487452516447294}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4219383200925489839, "histo_col_type": "", "name": "__auto__", "null_count": 2800013857363006202, "row_count": 4604487452516447294}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2761519842150775384, "histo_col_type": "", "name": "__auto__", "null_count": 2699750757920660134, "row_count": 4604487452516447294}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 607737353042258578, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "\\t*\\nF"}, {"distinct_range": 831787237795791700, "num_eq": 778283716667590218, "num_range": 2334432039763679609, "upper_bound": "`n/c\\n8z"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 4604487452516447294}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2704576798103478323, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4604487452516447294}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 720666172753745799, "histo_col_type": "", "name": "__auto__", "null_count": 1253031528416877836, "row_count": 4604487452516447294}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1173584618667701792, "histo_buckets": [{"distinct_range": 0, "num_eq": 4551732935550616457, "num_range": 0, "upper_bound": "1981-05-18 07:09:56.000314"}, {"distinct_range": 177320.43228377917, "num_eq": 8769392904714541557, "num_range": 700000, "upper_bound": "1999-12-01 00:21:45.000132"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 5010416820870244419}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3530613312285086907, "histo_col_type": "", "name": "__auto__", "null_count": 3393703937147577613, "row_count": 5010416820870244419}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3805398479631198964, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5010416820870244419}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2550380708549160040, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5010416820870244419}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3582416752157924454, "histo_buckets": [{"distinct_range": 0, "num_eq": 4553658579602179744, "num_range": 0, "upper_bound": "598370995"}, {"distinct_range": 40, "num_eq": 1336678627766239419, "num_range": 40, "upper_bound": "815771976"}, {"distinct_range": 6746954623011812000, "num_eq": 1000000000, "num_range": 6746954623011812589, "upper_bound": "2073006667"}, {"distinct_range": 9919.599492489926, "num_eq": 6833334362314877604, "num_range": 20000, "upper_bound": "3309085979"}, {"distinct_range": 3289140516431409700, "num_eq": 70000, "num_range": 8400826602114384499, "upper_bound": "4148716395"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 2980703650909255411, "row_count": 5010416820870244419}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 318772157640079797, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5010416820870244419}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2489960328537856870, "histo_col_type": "", "name": "__auto__", "null_count": 2462345999193986263, "row_count": 5010416820870244419}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 808412433035082323, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5010416820870244419}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2689189623775852001, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5010416820870244419}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2176582642739408694, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5010416820870244419}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1963615123386131726, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5010416820870244419}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 212942345552883466, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5010416820870244419}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 395516335018894806, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "\\\\x69bd2bb307"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 5010416820870244419}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3724579663484878596, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5010416820870244419}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4902845726335124131, "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 5010416820870244419}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1188310102006361177, "histo_buckets": [{"distinct_range": 0, "num_eq": 7321787925325337145, "num_range": 0, "upper_bound": "1990228047"}, {"distinct_range": 0, "num_eq": 6280240120588516071, "num_range": 6857590856958085248, "upper_bound": "3298549370"}, {"distinct_range": 1707376537390360600, "num_eq": 900000000, "num_range": 1707376537390360547, "upper_bound": "3376438644"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 90000000000, "upper_bound": "3716886397"}, {"distinct_range": 30, "num_eq": 226283045259951736, "num_range": 30, "upper_bound": "3865714125"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 3745999324693992207, "row_count": 5010416820870244419}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24346342, "histo_buckets": [{"distinct_range": 0, "num_eq": 7602415554611949410, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 861174904781506488, "upper_bound": "\\u0013"}, {"distinct_range": 0, "num_eq": 8249794150010430736, "num_range": 0, "upper_bound": "I"}, {"distinct_range": 3856131.2333421074, "num_eq": 1015270999941271734, "num_range": 7000000, "upper_bound": "P"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 200000000}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 142161902, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000000}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 129908270, "histo_buckets": [{"distinct_range": 0, "num_eq": 6140831932372169181, "num_range": 0, "upper_bound": "BOX(-1.0057415934972582 -1.6210461719795481,0.7254398598003122 1.455309964214747)"}, {"distinct_range": 6835858994915717000, "num_eq": 9221149476580436521, "num_range": 8601170395977890178, "upper_bound": "BOX(-0.7481408104752507 0.3729904578500187,0.9322262315805311 1.874261382108638)"}, {"distinct_range": 4257820182582382600, "num_eq": 700000, "num_range": 5868580336627839884, "upper_bound": "BOX(0.002580258572330796 -0.10815801988783791,1.117320787536272 1.2387390381717784)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 200000000}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: there is no unique constraint matching given keys for referenced table table1 (SQLSTATE 23503)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2548188&tab=buildLog) on [master@365c5504b75c9a9260365a628a5110c48312178b](https://github.com/cockroachdb/cockroach/commits/365c5504b75c9a9260365a628a5110c48312178b):

```
        postgres_1    | 	CREATE INDEX ON table1 (col1_2, col1_6, col1_5, col1_3, col1_7 DESC);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('v', 'pwvivo');
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('fsuupq', 'lsccl');
        postgres_1    | 	CREATE TYPE rand_typ_2 AS ENUM ('wfvy', 'p', 'sfzrmv', 'hry', 'xx');
        postgres_1    | 	CREATE TYPE rand_typ_3 AS ENUM ('trma', 'wifs', 'dhgai', 'bhf', 'kfoaxn');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (1.52s)
        test_1        |     --- FAIL: TestCompare/mutators (0.69s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 TIMESTAMP NOT NULL, col1_1 INT8 NOT NULL, col1_2 BOX2D NOT NULL, col1_3 REGNAMESPACE NOT NULL, col1_4 FLOAT4 NOT NULL, col1_5 INT4 NOT NULL, col1_6 REGCLASS NULL, PRIMARY KEY (col1_5, col1_4 ASC, col1_2 ASC, col1_3 DESC), FAMILY (col1_0, col1_1), FAMILY (col1_3, col1_6), FAMILY (col1_5, col1_2), FAMILY (col1_4));
        test_1        |             CREATE TABLE table2 (col1_5 INT4 NOT NULL, col1_4 FLOAT4 NOT NULL, col1_2 BOX2D NOT NULL, col1_3 REGNAMESPACE NOT NULL, col2_4 TIMESTAMP NULL, col2_5 BIT(5) NOT NULL, col2_6 STRING NULL, col2_7 INT8 NULL, col2_8 BOX2D, col2_9 NAME NOT NULL, col2_10 BOX2D NOT NULL, col2_11 INT2 NULL, col2_12 REGNAMESPACE NULL, col2_13 OID NOT NULL, PRIMARY KEY (col1_5, col1_4 ASC, col1_2 ASC, col1_3 DESC, col2_5 DESC), INDEX (col2_13 ASC, col1_5, col1_4) WHERE ((table2.col2_4 < '3000-01-01 00:00:00':::TIMESTAMP) AND (table2.col1_4 > '+Inf':::FLOAT8)) OR (table2.col2_11 <= (-1):::INT8), INDEX (col2_4 DESC, col2_12 ASC, col1_2, col2_8 DESC, col2_9 ASC, col2_13, col2_11 DESC, col2_10 DESC) STORING (col2_6, col2_7), UNIQUE (col2_4, col2_8 ASC, col1_3, col2_10 DESC, col2_9 ASC, col1_4, col2_13, col1_2 DESC, col2_5, col2_11, col1_5 DESC, col2_12) STORING (col2_7) WHERE table2.col2_4 <= '0001-01-01 00:00:00', INDEX (col2_5 DESC, col2_6 DESC, col2_12 DESC, col1_5 ASC, col1_4, col1_2, col2_4 ASC, col2_9 DESC, col2_7 ASC, col2_13 DESC, col2_8 ASC, col2_10 DESC, col1_3 ASC, col2_11 DESC) WHERE ((((table2.col1_5 >= 2147483647:::INT8) AND (table2.col1_4 > 'NaN':::FLOAT8)) AND (table2.col2_6 < '"':::STRING)) OR (table2.col2_11 != (-1):::INT8)) OR (table2.col2_7 >= 2147483647:::INT8), INDEX (col2_11 ASC, col2_10 DESC, col2_8, col2_5, col1_4 DESC, col2_4, col2_7 DESC, col2_6 ASC, col1_2 ASC, col1_5 ASC, col2_12, col1_3, col2_9 DESC) STORING (col2_13) WHERE ((((((table2.col2_9 < '"') OR (table2.col2_11 > 127)) OR (table2.col1_5 > -1)) AND (table2.col2_7 < 9223372036854775807)) AND (table2.col2_4 > '-2000-01-01 00:00:00')) OR (table2.col2_6 >= e'\x00')) OR (table2.col1_4 != 1.0), UNIQUE (col2_11, col2_10 ASC, col2_12 DESC, col1_2 DESC, col2_7 DESC, col1_4 ASC, col2_6 DESC), INDEX (col1_5, col1_3 DESC, col2_8, col2_4 DESC, col2_5 DESC, col2_9 ASC, col2_11, col2_7, col2_6 ASC, col2_12 DESC, col2_10, col2_13 DESC), UNIQUE (col2_8 DESC, col2_9, col2_4 ASC, col2_6, col2_12 ASC, col2_11 ASC, col2_7 DESC, col2_10 DESC, col1_5, col1_4, col2_5 DESC) WHERE (((table2.col1_4 > 1.401298464324817e-45:::FLOAT8) AND (table2.col2_11 = (-1):::INT8)) AND (table2.col1_5 != 32767:::INT8)) OR (table2.col2_4 > '-4713-11-24 00:00:00':::TIMESTAMP), FAMILY (col1_5, col2_13, col1_2), FAMILY (col2_4, col1_3), FAMILY (col2_8, col1_4), FAMILY (col2_5), FAMILY (col2_7), FAMILY (col2_11, col2_9, col2_12), FAMILY (col2_10), FAMILY (col2_6), UNIQUE (col1_3) STORING (col2_4, col2_6, col2_9, col2_10, col2_12) WHERE table2.col2_9 != e'\'', UNIQUE (col1_5) STORING (col2_6, col2_7, col2_9, col2_12) WHERE ((((((table2.col2_7 < 0) AND (table2.col2_4 < '294276-12-31 23:59:59.999999')) OR (table2.col2_9 != '"')) AND (table2.col2_6 != '')) AND (table2.col2_11 < 0)) OR (table2.col1_4 = +Inf)) AND (table2.col1_5 > -1), UNIQUE (col2_5) WHERE ((((((table2.col2_7 < 127) OR (table2.col1_5 >= 1)) AND (table2.col1_4 = 1.401298464324817e-45)) OR (table2.col2_9 <= '')) OR (table2.col2_6 = e'\U00002603')) OR (table2.col2_4 >= '3000-01-01 00:00:00')) AND (table2.col2_11 != 0)) INTERLEAVE IN PARENT table1 (col1_5, col1_4, col1_2, col1_3);
        test_1        |             CREATE TABLE table3 (col1_5 INT4 NOT NULL, col1_4 FLOAT4 NOT NULL, col1_2 BOX2D NOT NULL, col1_3 REGNAMESPACE NOT NULL, col2_5 BIT(5) NOT NULL, col3_5 DECIMAL NOT NULL, col3_6 REGNAMESPACE NOT NULL, col3_7 INT8 NOT NULL, col3_8 TIMETZ, col3_9 FLOAT8 NOT NULL, col3_10 BIT(20) NULL, col3_11 BIT(12) NOT NULL, col3_12 CHAR NOT NULL, col3_13 INT2 NOT NULL, col3_14 REGTYPE, col3_15 INT8 NULL, PRIMARY KEY (col1_5, col1_4 ASC, col1_2 ASC, col1_3 DESC, col2_5 DESC, col3_7 DESC), UNIQUE (col1_2 ASC, col3_12 DESC, col3_10 ASC, col3_7 DESC) STORING (col3_5, col3_9, col3_11, col3_15), UNIQUE (col3_12 ASC, col1_3 DESC, col1_5, col3_6, col3_13, col1_2 ASC, col1_4 DESC, col3_10 ASC, col3_14) STORING (col3_8) WHERE ((((((((table3.col3_7 != (-1):::INT8) OR (table3.col3_13 > 1:::INT8)) AND (table3.col3_12 <= '':::STRING)) AND (table3.col1_4 != 1.0:::FLOAT8)) OR (table3.col3_9 >= 1.401298464324817e-45:::FLOAT8)) AND (table3.col1_5 < 1:::INT8)) OR (table3.col3_8 = '00:00:00+15:59:00':::TIMETZ)) AND (table3.col3_5 != (-1.234E+401):::DECIMAL)) OR (table3.col3_15 != 1:::INT8), INDEX (col3_8 DESC, col3_11 ASC, col3_13 ASC, col1_4, col1_5 DESC, col3_14, col3_7, col3_10 DESC, col2_5 ASC, col3_5, col3_15, col3_12 DESC, col1_3 DESC) STORING (col3_9) WHERE (((((table3.col3_9 > 0.0) OR (table3.col1_5 >= 127)) AND (table3.col3_7 = 1)) OR (table3.col3_13 = 1)) AND (table3.col3_12 != '')) AND (table3.col3_8 > '24:00:00-15:59:00'), UNIQUE (col1_3, col3_7 DESC) STORING (col3_5, col3_6, col3_10, col3_12, col3_14), INDEX (col2_5, col3_8 DESC, col1_3, col3_6 ASC, col3_14 DESC) STORING (col3_9, col3_12, col3_13) WHERE ((((((((table3.col1_5 = 32767:::INT8) AND (table3.col3_8 > '24:00:00-15:59:00':::TIMETZ)) AND (table3.col3_13 != (-1):::INT8)) OR (table3.col3_15 >= 32767:::INT8)) AND (table3.col3_9 > (-1.0):::FLOAT8)) OR (table3.col3_12 != 'X':::STRING)) OR (table3.col1_4 <= 0.0:::FLOAT8)) AND (table3.col3_7 != 32767:::INT8)) AND (table3.col3_5 <= (-1):::DECIMAL), FAMILY (col3_15), FAMILY (col3_6), FAMILY (col3_8), FAMILY (col2_5), FAMILY (col1_2), FAMILY (col3_9), FAMILY (col3_10, col3_11), FAMILY (col1_3, col3_14), FAMILY (col3_7), FAMILY (col1_5, col3_13, col1_4, col3_12), FAMILY (col3_5)) INTERLEAVE IN PARENT table2 (col1_5, col1_4, col1_2, col1_3, col2_5);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 220, "histo_col_type": "", "name": "__auto__", "null_count": 69644, "row_count": 90000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1133, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 70699, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32959, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35781, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30956, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 71633, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000000, "num_range": 0, "upper_bound": "-1808198110"}, {"distinct_range": 0, "num_eq": 40, "num_range": 900000, "upper_bound": "-931914102"}, {"distinct_range": 251984859367542340, "num_eq": 5349586750793854426, "num_range": 723054239258091477, "upper_bound": "-448312563"}, {"distinct_range": 2736388803675055000, "num_eq": 40, "num_range": 3816404368321570460, "upper_bound": "-66773312"}, {"distinct_range": 60, "num_eq": 400000000, "num_range": 60, "upper_bound": "481450884"}, {"distinct_range": 9000000000, "num_eq": 600, "num_range": 9000000000, "upper_bound": "763872257"}, {"distinct_range": 25164174586.098755, "num_eq": 23917443362263495, "num_range": 50000000000, "upper_bound": "869349306"}, {"distinct_range": 1054502661547206700, "num_eq": 848217978394943717, "num_range": 7299218048214070790, "upper_bound": "1040363366"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 90000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 70, "num_range": 0, "upper_bound": "2004-09-27 14:58:45.000283"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 5039559991207963157, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 40105.71648936601, "num_eq": 6740519931905404575, "num_range": 600000, "upper_bound": "1061737704"}, {"distinct_range": 40000, "num_eq": 3000, "num_range": 40000, "upper_bound": "1372699313"}, {"distinct_range": 528284163199836400, "num_eq": 3491034850209928815, "num_range": 633019343837869077, "upper_bound": "2326426259"}, {"distinct_range": 58469026.49127173, "num_eq": 7771608627429675381, "num_range": 10000000000, "upper_bound": "3038256578"}, {"distinct_range": 846805.5809772569, "num_eq": 7047250667900498387, "num_range": 50000000, "upper_bound": "3485011973"}, {"distinct_range": 407973.07981469313, "num_eq": 1000000000, "num_range": 1000000, "upper_bound": "3745431992"}, {"distinct_range": 7789539122466070000, "num_eq": 5199698572589881426, "num_range": 7789539122466069172, "upper_bound": "3960176877"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000, "num_range": 0, "upper_bound": "-1353973256"}, {"distinct_range": 9209550782260240000, "num_eq": 20000000, "num_range": 9209550782260240590, "upper_bound": "-1259464185"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 1000, "upper_bound": "-566013464"}, {"distinct_range": 0, "num_eq": 226701480017914695, "num_range": 80000000, "upper_bound": "-32768"}, {"distinct_range": 0, "num_eq": 4664243636715271533, "num_range": 0, "upper_bound": "401797000"}, {"distinct_range": 2475051934737554400, "num_eq": 3431373730270250686, "num_range": 2910949971457825099, "upper_bound": "889144808"}, {"distinct_range": 0, "num_eq": 6172043247880192569, "num_range": 6715185070974729651, "upper_bound": "1109420184"}, {"distinct_range": 8508856098287950000, "num_eq": 8000, "num_range": 8508856098287949791, "upper_bound": "1827616997"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 4942177483243218993, "num_range": 0, "upper_bound": "00000"}, {"distinct_range": 40000, "num_eq": 7000000000, "num_range": 40000, "upper_bound": "00100"}, {"distinct_range": 0, "num_eq": 0, "num_range": 2000000, "upper_bound": "01010"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 7523327594072536009, "upper_bound": "01100"}, {"distinct_range": 3580100109364989000, "num_eq": 6000000000, "num_range": 3580100109364988975, "upper_bound": "10011"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 0, "upper_bound": "10111"}, {"distinct_range": 3000, "num_eq": 700, "num_range": 3000, "upper_bound": "11010"}], "histo_col_type": "BIT(5)", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 7648425378361516139, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 7655466138607861000, "num_eq": 1008472834954559930, "num_range": 7655466138607860496, "upper_bound": "BOX(-1.6918645124087048 -1.5619461151642138,0.6309249060879069 -1.096820172558571)"}, {"distinct_range": 29157627464.226437, "num_eq": 10000, "num_range": 100000000000, "upper_bound": "BOX(-1.4039586356281442 0.3934625545087689,0.7865272897820264 1.219275318618366)"}, {"distinct_range": 1000000, "num_eq": 3818567320257487713, "num_range": 1000000, "upper_bound": "BOX(-1.3711369430302636 -1.5388869138909251,-1.2924917261626894 1.162880496844103)"}, {"distinct_range": 701822081993531900, "num_eq": 1000000, "num_range": 6429063028456399476, "upper_bound": "BOX(-0.4455375587483633 -1.0599809391783848,1.2305273100613467 -0.9650767205653626)"}, {"distinct_range": 3349631839177600500, "num_eq": 700, "num_range": 3349631839177600739, "upper_bound": "BOX(-0.03085031462524146 -0.23356449596315665,0.6830722985023486 0.0011968975654603337)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000000, "num_range": 0, "upper_bound": "-31824"}, {"distinct_range": 0, "num_eq": 8540360161440250332, "num_range": 9000000, "upper_bound": "-19570"}, {"distinct_range": 0, "num_eq": 2000000, "num_range": 6000000, "upper_bound": "-7934"}, {"distinct_range": 0, "num_eq": 5578555689501206277, "num_range": 9000000, "upper_bound": "-5120"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 149, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 532, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 783, "histo_col_type": "", "name": "__auto__", "null_count": 115, "row_count": 900}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 121, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 138, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 405, "histo_col_type": "", "name": "__auto__", "null_count": 359, "row_count": 900}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 94, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "BOX(0.3114669880234615 -0.27799995291088986,1.8793144717551746 0.1873225651795032)"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 80, "upper_bound": "BOX(0.3682355372267615 -0.17688281185780064,1.022001685173235 1.276944265530976)"}, {"distinct_range": 0, "num_eq": 3756805007258505861, "num_range": 0, "upper_bound": "BOX(0.5671172117822776 -0.8375246835347478,1.122728919147562 1.710767380226188)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 214, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000000, "num_range": 0, "upper_bound": "1057913541"}, {"distinct_range": 0, "num_eq": 5000, "num_range": 808804191080382226, "upper_bound": "1449670983"}, {"distinct_range": 46460258.2422913, "num_eq": 7434150958008024910, "num_range": 100000000, "upper_bound": "2919995724"}, {"distinct_range": 0, "num_eq": 300, "num_range": 1000000000, "upper_bound": "3864089796"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 857, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 640, "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 620, "histo_col_type": "", "name": "__auto__", "null_count": 823, "row_count": 900}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 459, "histo_buckets": [{"distinct_range": 0, "num_eq": 7754012638831776196, "num_range": 0, "upper_bound": "-1472000734"}, {"distinct_range": 90, "num_eq": 800000000, "num_range": 90, "upper_bound": "-1038385994"}, {"distinct_range": 198271.28272817621, "num_eq": 5502470781615393935, "num_range": 300000, "upper_bound": "1"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 760, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 569, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000000, "num_range": 0, "upper_bound": "00100"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 1000000, "upper_bound": "01000"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "01010"}, {"distinct_range": 289097715.92047477, "num_eq": 400000000, "num_range": 800000000, "upper_bound": "01111"}, {"distinct_range": 800, "num_eq": 500000000, "num_range": 800, "upper_bound": "10000"}, {"distinct_range": 0, "num_eq": 8451639468693178373, "num_range": 8842395169663601438, "upper_bound": "10111"}], "histo_col_type": "BIT(5)", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 473, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 771, "histo_buckets": [{"distinct_range": 0, "num_eq": 300000000, "num_range": 0, "upper_bound": "12:52:58.466978+14:22:00"}, {"distinct_range": 14569923.454125125, "num_eq": 100, "num_range": 20000000, "upper_bound": "06:23:56.999547+06:29:00"}, {"distinct_range": 0, "num_eq": 8911666614037934015, "num_range": 7083835978707549146, "upper_bound": "07:07:48.31988-11:47:00"}, {"distinct_range": 0, "num_eq": 100, "num_range": 10000000, "upper_bound": "20:34:12.49057-06:55:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 563, "row_count": 900}]':::JSONB;
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_6) REFERENCES table2 (col1_3) ON UPDATE RESTRICT;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('suvzix');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('zdqkyw', 'hibswa', 'ypkyvy', 'xjelzr');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3730718754967499436, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6308332592985235526}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4869685940797662174, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6308332592985235526}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 686065417720136673, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6308332592985235526}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 514282541716933982, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6308332592985235526}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 917049750294896127, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6308332592985235526}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2822300379528267608, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000000, "num_range": 0, "upper_bound": "-1733730998"}, {"distinct_range": 3629787956.625001, "num_eq": 700000000, "num_range": 9000000000, "upper_bound": "-1318053969"}, {"distinct_range": 0, "num_eq": 2058846717276829905, "num_range": 900, "upper_bound": "-1071401580"}, {"distinct_range": 0, "num_eq": 60000000000, "num_range": 200000000, "upper_bound": "-585790934"}, {"distinct_range": 531602717521105400, "num_eq": 8752297662574394374, "num_range": 6245028221163989594, "upper_bound": "-151318554"}, {"distinct_range": 5000000000, "num_eq": 5000000000, "num_range": 5000000000, "upper_bound": "-111699082"}, {"distinct_range": 3366601902230186000, "num_eq": 6305605378604441740, "num_range": 3366601902230185898, "upper_bound": "381564194"}, {"distinct_range": 0, "num_eq": 8313019138160761989, "num_range": 7000000000, "upper_bound": "430108337"}, {"distinct_range": 0, "num_eq": 1781275547878372744, "num_range": 0, "upper_bound": "537253840"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 6308332592985235526}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1473597392803128988, "histo_col_type": "", "name": "__auto__", "null_count": 6023034161753799825, "row_count": 6308332592985235526}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9405627120, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000, "num_range": 0, "upper_bound": "10000"}], "histo_col_type": "BIT(5)", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3087000857, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4594454704, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 901481521747153700, "num_eq": 100, "num_range": 901481521747153687, "upper_bound": "771393635"}, {"distinct_range": 0, "num_eq": 30000000000, "num_range": 2464508990303702408, "upper_bound": "1225688347"}, {"distinct_range": 1206118973486165200, "num_eq": 3621522025883744368, "num_range": 1263518625982020545, "upper_bound": "2310316270"}, {"distinct_range": 0, "num_eq": 8444085274720240918, "num_range": 5598936286639907099, "upper_bound": "2501381431"}, {"distinct_range": 3227630874609797, "num_eq": 6244099731380963735, "num_range": 3227630874609797, "upper_bound": "2907356454"}, {"distinct_range": 195.2232553598663, "num_eq": 2000000000, "num_range": 800, "upper_bound": "3005252019"}, {"distinct_range": 7653412437.542186, "num_eq": 7505252477700808501, "num_range": 100000000000, "upper_bound": "3967836186"}, {"distinct_range": 6437703670436299000, "num_eq": 2048094415016261270, "num_range": 8608213587803183966, "upper_bound": "4218781840"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 633629454, "histo_col_type": "", "name": "__auto__", "null_count": 6663292232, "row_count": 10000000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9942906209, "histo_col_type": "", "name": "__auto__", "null_count": 9032120679, "row_count": 10000000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3669393080, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1515435878, "histo_buckets": [{"distinct_range": 0, "num_eq": 8591728933038744662, "num_range": 0, "upper_bound": "-25597"}, {"distinct_range": 300000000, "num_eq": 30000, "num_range": 300000000, "upper_bound": "-22293"}, {"distinct_range": 411274490.0414428, "num_eq": 70000, "num_range": 6000000000, "upper_bound": "-1931"}, {"distinct_range": 6586554716542435000, "num_eq": 4240439098638055568, "num_range": 6586554716542435681, "upper_bound": "1"}, {"distinct_range": 246008.91052822376, "num_eq": 5009852098259767429, "num_range": 400000, "upper_bound": "600"}, {"distinct_range": 0, "num_eq": 50, "num_range": 5135671313967005154, "upper_bound": "24002"}, {"distinct_range": 0, "num_eq": 50, "num_range": 10000000000, "upper_bound": "29612"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 8993502126, "row_count": 10000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2657704680, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": "-199545694"}, {"distinct_range": 143008532667058880, "num_eq": 40, "num_range": 276606022296428636, "upper_bound": "29176890"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1775890294, "histo_buckets": [{"distinct_range": 0, "num_eq": 800, "num_range": 0, "upper_bound": "55506546"}, {"distinct_range": 3717375.539980607, "num_eq": 6690367325197623272, "num_range": 20000000, "upper_bound": "439350627"}, {"distinct_range": 6000, "num_eq": 5000000, "num_range": 6000, "upper_bound": "2255443740"}, {"distinct_range": 7726193691.700615, "num_eq": 5838981871089525456, "num_range": 8000000000, "upper_bound": "3729770638"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5752756750, "histo_buckets": [{"distinct_range": 0, "num_eq": 5090915256695842666, "num_range": 0, "upper_bound": "1988-08-04 20:52:00.000537"}, {"distinct_range": 3792009819420450300, "num_eq": 2794453137214764746, "num_range": 4784747427005836395, "upper_bound": "1993-06-07 12:33:12.000373"}, {"distinct_range": 3080675645691996700, "num_eq": 400000000, "num_range": 3080675645691996584, "upper_bound": "1994-09-28 02:00:30.000674"}, {"distinct_range": 73.60481210908158, "num_eq": 4303542415680246401, "num_range": 900, "upper_bound": "1997-02-04 01:44:53.000114"}, {"distinct_range": 0, "num_eq": 1112110447253346699, "num_range": 10000, "upper_bound": "1997-09-02 11:18:47.000982"}, {"distinct_range": 0, "num_eq": 0, "num_range": 9213724437835904735, "upper_bound": "2001-01-28 13:25:51.000041"}, {"distinct_range": 6249044605072357000, "num_eq": 7109759455526484022, "num_range": 6249044605072357440, "upper_bound": "3000-01-01 00:00:00"}, {"distinct_range": 0, "num_eq": 20000000, "num_range": 4000, "upper_bound": "294276-12-31 23:59:59.999999"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 3515641170, "row_count": 10000000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5069570296, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000, "num_range": 0, "upper_bound": "BOX(-0.338424931341862 -0.699343770338934,0.6256156240302946 -0.27145823883042286)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 4923506433, "row_count": 10000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8729687045, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4117586203, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1513870201, "histo_col_type": "", "name": "__auto__", "null_count": 2447401229, "row_count": 10000000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 961511156977976974, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4135654686507819954}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4012198273932562094, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4135654686507819954}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1064997589335379531, "histo_buckets": [{"distinct_range": 0, "num_eq": 4963386442603683222, "num_range": 0, "upper_bound": "Q"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 4135654686507819954}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2469802496306464191, "histo_buckets": [{"distinct_range": 0, "num_eq": 7710500780915742558, "num_range": 0, "upper_bound": "BOX(-2.08263549619153 -0.3397958407657258,0.4652836215665876 0.09805734747946687)"}, {"distinct_range": 5000000000, "num_eq": 400000000, "num_range": 5000000000, "upper_bound": "BOX(-1.750755405635433 -1.0310230870380495,-0.5206005669360861 -0.4555306451514981)"}, {"distinct_range": 0, "num_eq": 1542539215561578853, "num_range": 465409504120986680, "upper_bound": "BOX(0.13221693007951574 -1.1214802307908933,0.32944260747980986 0.5395293164493513)"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "BOX(0.3468177364338965 -0.2093375317568077,0.8477136087612038 0.15557876347676924)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 4135654686507819954}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2081619645699933605, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "01100"}], "histo_col_type": "BIT(5)", "name": "__auto__", "null_count": 0, "row_count": 4135654686507819954}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2264476724319456253, "histo_col_type": "", "name": "__auto__", "null_count": 33287948415262927, "row_count": 4135654686507819954}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 443108790937030623, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4135654686507819954}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1926529075078666259, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4135654686507819954}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3738309324917586909, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4135654686507819954}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3796826212227336094, "histo_col_type": "", "name": "__auto__", "null_count": 1304577239800381037, "row_count": 4135654686507819954}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1870377674060444413, "histo_col_type": "", "name": "__auto__", "null_count": 2611180013712138057, "row_count": 4135654686507819954}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1984382757479049069, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-1444798335"}, {"distinct_range": 0, "num_eq": 804023017725280351, "num_range": 1477670528696629248, "upper_bound": "-1259121884"}, {"distinct_range": 0, "num_eq": 1791967984677800321, "num_range": 0, "upper_bound": "1256234789"}, {"distinct_range": 2827883902364825600, "num_eq": 60000000, "num_range": 2827883902364825622, "upper_bound": "1725239442"}, {"distinct_range": 5564166536535341000, "num_eq": 6358894239109111323, "num_range": 5564166536535340898, "upper_bound": "1989999940"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 4135654686507819954}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1708155152579327541, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4135654686507819954}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1688871363508311316, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4135654686507819954}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21377031565945774, "histo_buckets": [{"distinct_range": 0, "num_eq": 7305438560917472109, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 4502649845378764300, "num_eq": 0, "num_range": 4502649845378764312, "upper_bound": "1158498105"}, {"distinct_range": 0, "num_eq": 2993105191649792920, "num_range": 100000000, "upper_bound": "1338211669"}, {"distinct_range": 20000000000, "num_eq": 10000000, "num_range": 20000000000, "upper_bound": "2478958177"}, {"distinct_range": 0, "num_eq": 50000, "num_range": 5000000000, "upper_bound": "2560487263"}, {"distinct_range": 700000000, "num_eq": 5863122559102036455, "num_range": 700000000, "upper_bound": "2608144000"}, {"distinct_range": 0, "num_eq": 724791343838462078, "num_range": 90000000, "upper_bound": "3707789527"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 4135654686507819954}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3027048086548315428, "histo_buckets": [{"distinct_range": 0, "num_eq": 2148859405288767229, "num_range": 0, "upper_bound": "04:46:44.803741+05:46:00"}, {"distinct_range": 30000000, "num_eq": 588391542562604736, "num_range": 30000000, "upper_bound": "14:57:31.745151+12:58:00"}, {"distinct_range": 3142268290282282000, "num_eq": 4411899637519122046, "num_range": 3618874327027156209, "upper_bound": "12:53:33.162514+08:58:00"}, {"distinct_range": 0, "num_eq": 700, "num_range": 90000, "upper_bound": "01:09:12.913676-02:52:00"}, {"distinct_range": 30000000000, "num_eq": 90000000, "num_range": 30000000000, "upper_bound": "00:03:31.111023-08:18:00"}, {"distinct_range": 1905329274059076400, "num_eq": 7406105347224849823, "num_range": 6346143693005639519, "upper_bound": "10:50:13.20358-01:55:00"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 0, "upper_bound": "06:02:05.402503-12:58:00"}, {"distinct_range": 669177.6426019312, "num_eq": 10, "num_range": 700000, "upper_bound": "13:36:23.844998-07:15:00"}, {"distinct_range": 0, "num_eq": 60, "num_range": 0, "upper_bound": "20:10:19.049271-03:35:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 2267743009143753947, "row_count": 4135654686507819954}]';
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_1) REFERENCES table2 (col1_5);
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col2_5) REFERENCES table2 (col2_5) ON UPDATE CASCADE;
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5299037737, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 352045156, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1875918099, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2273590461, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5196056708, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4724839635, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000, "num_range": 0, "upper_bound": "-1406852231"}, {"distinct_range": 1586159200083795000, "num_eq": 7123826990920253587, "num_range": 1586159200083795056, "upper_bound": "-311676627"}, {"distinct_range": 54676936.46851513, "num_eq": 9000000, "num_range": 100000000, "upper_bound": "465807787"}, {"distinct_range": 3280558626029026000, "num_eq": 1000000000, "num_range": 3280558626029025932, "upper_bound": "498655547"}, {"distinct_range": 130069.23197674475, "num_eq": 40000000000, "num_range": 600000, "upper_bound": "1312417064"}, {"distinct_range": 7.250953399775142, "num_eq": 5486219490316454680, "num_range": 10, "upper_bound": "1561708077"}, {"distinct_range": 5068712138485040000, "num_eq": 8446746208160457657, "num_range": 5068712138485040356, "upper_bound": "1677586968"}, {"distinct_range": 0, "num_eq": 7649691968963887478, "num_range": 3000, "upper_bound": "1987149104"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3587538316, "histo_col_type": "", "name": "__auto__", "null_count": 2342928187, "row_count": 10000000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 852, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2421, "histo_buckets": [{"distinct_range": 0, "num_eq": 700, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 1000000000, "num_eq": 100000000000, "num_range": 1000000000, "upper_bound": "376681891"}, {"distinct_range": 55870110539276620, "num_eq": 7987030712393810755, "num_range": 339213445141883979, "upper_bound": "1016479305"}, {"distinct_range": 3031048624313575400, "num_eq": 40000000000, "num_range": 3031048624313575556, "upper_bound": "1067197971"}, {"distinct_range": 0, "num_eq": 5712916645039138803, "num_range": 700000000, "upper_bound": "2039698788"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 1606269653412379590, "upper_bound": "2085167197"}, {"distinct_range": 60000, "num_eq": 8604047655934430974, "num_range": 60000, "upper_bound": "2122030893"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4664, "histo_col_type": "BIT(5)", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2528, "histo_col_type": "", "name": "__auto__", "null_count": 942, "row_count": 6000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4825, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 190, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4057, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000, "num_range": 0, "upper_bound": "-3374"}, {"distinct_range": 865130409052446000, "num_eq": 9167454596042908126, "num_range": 7324780259599797456, "upper_bound": "-1"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 5243, "row_count": 6000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1747, "histo_col_type": "", "name": "__auto__", "null_count": 4523, "row_count": 6000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4664, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000, "num_range": 0, "upper_bound": "BOX(-1.907023199804252 -2.2375068802890086,0.028213775413126252 0.49887079360482617)"}, {"distinct_range": 6264139628987125000, "num_eq": 6890459858061614503, "num_range": 7291797795828145010, "upper_bound": "BOX(-1.071374640361047 -0.82896652813177,-0.2924367797758325 -0.4392270210024738)"}, {"distinct_range": 194184029606797730, "num_eq": 7203432334390741193, "num_range": 6985409213787838369, "upper_bound": "BOX(-0.875765942703639 -1.2808345626475661,-0.8729211352284529 1.1661663782131362)"}, {"distinct_range": 60000000000, "num_eq": 1000000000, "num_range": 60000000000, "upper_bound": "BOX(-0.7637006955338426 -0.5707441235241921,-0.41734517293138623 -0.04181078642008046)"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "BOX(-0.3384239305394668 -0.004292331003492855,1.4470683323638545 1.2525013696585847)"}, {"distinct_range": 0, "num_eq": 1721047494394623914, "num_range": 6722499304416463551, "upper_bound": "BOX(-0.16741109690811906 -0.8836213877422355,0.30051413294855045 -0.5067169169932716)"}, {"distinct_range": 0, "num_eq": 135406290956191224, "num_range": 80000, "upper_bound": "BOX(1.240013322908795 -0.5214631889408734,1.3875594135164167 -0.10294855787818719)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 2293, "row_count": 6000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3839, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "163830084"}, {"distinct_range": 1000, "num_eq": 4524583609557114298, "num_range": 1000, "upper_bound": "728871857"}, {"distinct_range": 3719851750843143700, "num_eq": 6182015432189018952, "num_range": 7521778694309447118, "upper_bound": "1423838252"}, {"distinct_range": 0, "num_eq": 7000000, "num_range": 10000, "upper_bound": "1494297833"}, {"distinct_range": 0, "num_eq": 60000000000, "num_range": 200000000, "upper_bound": "1544647569"}, {"distinct_range": 200000000, "num_eq": 3000000000, "num_range": 200000000, "upper_bound": "1976298997"}, {"distinct_range": 315902.15716986475, "num_eq": 6012285630425752059, "num_range": 400000, "upper_bound": "2376418625"}, {"distinct_range": 100000, "num_eq": 20000000, "num_range": 100000, "upper_bound": "2567658619"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2567, "histo_buckets": [{"distinct_range": 0, "num_eq": 8243841581294500875, "num_range": 0, "upper_bound": "-1192409297"}, {"distinct_range": 7429913.122798621, "num_eq": 3432054240432615830, "num_range": 10000000, "upper_bound": "-128"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 812, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "1986-10-28 07:22:47.00084"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 433, "row_count": 6000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2887, "histo_col_type": "", "name": "__auto__", "null_count": 4474, "row_count": 6000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1512, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4652057628, "histo_col_type": "", "name": "__auto__", "null_count": 926321179, "row_count": 8000000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3541413231, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 1000000, "num_eq": 9027359404656437156, "num_range": 1000000, "upper_bound": "01:20:05.93302+10:04:00"}, {"distinct_range": 800000000, "num_eq": 5000000, "num_range": 800000000, "upper_bound": "07:10:23.895029+10:08:00"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 6058365023345245553, "upper_bound": "01:32:21.873378+00:43:00"}, {"distinct_range": 4418.846403050335, "num_eq": 8252366256249644464, "num_range": 5000, "upper_bound": "18:57:39.494325-07:46:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 164856387, "row_count": 8000000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 522384005, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3435547238, "histo_col_type": "", "name": "__auto__", "null_count": 7207853169, "row_count": 8000000000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2163953284, "histo_buckets": [{"distinct_range": 0, "num_eq": 315038724784124204, "num_range": 0, "upper_bound": "+"}, {"distinct_range": 0, "num_eq": 800, "num_range": 4301237557227144579, "upper_bound": "F"}, {"distinct_range": 80000000000, "num_eq": 3098688960436126116, "num_range": 80000000000, "upper_bound": "R"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5395773158, "histo_buckets": [{"distinct_range": 0, "num_eq": 5654932153132196129, "num_range": 0, "upper_bound": "10110"}, {"distinct_range": 409553978411145800, "num_eq": 10000, "num_range": 409553978411145772, "upper_bound": "10111"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 10, "upper_bound": "11010"}], "histo_col_type": "BIT(5)", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4574918865, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4166572529, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6224255967, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7719157249, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6568346312, "histo_buckets": [{"distinct_range": 0, "num_eq": 300000000, "num_range": 0, "upper_bound": "-2137656211"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 10000, "upper_bound": "-1787095954"}, {"distinct_range": 4291915448449201000, "num_eq": 1742223716788070601, "num_range": 4291915448449201014, "upper_bound": "-1226879142"}, {"distinct_range": 6501.7994278786855, "num_eq": 7370414320790424196, "num_range": 9000, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 400000000, "upper_bound": "146793561"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7916504323, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4067452828, "histo_buckets": [{"distinct_range": 0, "num_eq": 70, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 60000000, "upper_bound": "BOX(-3.85839634441013 -0.08985706378222791,1.013546496618512 1.8074303041793802)"}, {"distinct_range": 0, "num_eq": 649074719322215285, "num_range": 0, "upper_bound": "BOX(-0.9982775788383516 -1.8243276156504002,-0.15809146063138435 0.3722532790513581)"}, {"distinct_range": 80, "num_eq": 6416527989576226174, "num_range": 80, "upper_bound": "BOX(-0.8203610438164584 0.12878100309883628,-0.09263383084665833 1.0264555883749475)"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 7844807391817090449, "upper_bound": "BOX(0.41278119245267914 -1.2947945084768244,0.9355791566727231 0.9510557795502133)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5446506237, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000000, "num_range": 0, "upper_bound": "37141907"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 2433697118900126818, "upper_bound": "216810552"}, {"distinct_range": 3000000000, "num_eq": 4967920269294205072, "num_range": 3000000000, "upper_bound": "1190937555"}, {"distinct_range": 471245055448634430, "num_eq": 896226690487956922, "num_range": 5559051978743119437, "upper_bound": "1251749740"}, {"distinct_range": 1000000000, "num_eq": 500, "num_range": 1000000000, "upper_bound": "2103380697"}, {"distinct_range": 400000000, "num_eq": 9000000000, "num_range": 400000000, "upper_bound": "3684342695"}, {"distinct_range": 508472207.8189293, "num_eq": 2369129285870057550, "num_range": 1000000000, "upper_bound": "4072731857"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 805922067, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23953046, "histo_col_type": "", "name": "__auto__", "null_count": 3868334799, "row_count": 8000000000}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "inf" does not exist (SQLSTATE 42703)
        test_1        |     --- FAIL: TestCompare/postgres (0.83s)
        test_1        |         compare_test.go:129: CREATE TABLE table1 (col1_0 BIT(46) NULL, col1_1 BIT(23), col1_2 BOX2D NOT NULL, col1_3 TIMESTAMP NOT NULL, col1_4 REGPROC NOT NULL, col1_5 BYTEA, col1_6 DECIMAL NOT NULL, col1_7 FLOAT8 NOT NULL, col1_8 BIT(38) NOT NULL, PRIMARY KEY (col1_3, col1_6, col1_4));
        test_1        |             CREATE INDEX ON table1 (col1_1, col1_8 DESC) INCLUDE (col1_2, col1_5);
        test_1        |             CREATE INDEX ON table1 (col1_2, col1_6, col1_5, col1_3, col1_7 DESC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('v', 'pwvivo');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('fsuupq', 'lsccl');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('wfvy', 'p', 'sfzrmv', 'hry', 'xx');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('trma', 'wifs', 'dhgai', 'bhf', 'kfoaxn');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2550097&tab=buildLog) on [master@6d49a323b52966becfe8a2c38a1a8ccdf8ee58a1](https://github.com/cockroachdb/cockroach/commits/6d49a323b52966becfe8a2c38a1a8ccdf8ee58a1):

```
        cockroach1_1  | webui:               http://cockroach1:8080
        cockroach1_1  | sql:                 postgresql://root@cockroach1:26257?sslmode=disable
        cockroach1_1  | RPC client flags:    /cockroach/cockroach <client cmd> --host=cockroach1:26257 --insecure
        cockroach1_1  | logs:                /cockroach-data/logs
        cockroach1_1  | temp dir:            /cockroach-data/cockroach-temp110205061
        cockroach1_1  | external I/O path:   /cockroach-data/extern
        cockroach1_1  | store[0]:            path=/cockroach-data
        cockroach1_1  | storage engine:      pebble
        cockroach1_1  | status:              initialized new cluster
        cockroach1_1  | clusterID:           99ca955a-e45a-443d-90d9-88234339ba68
        cockroach1_1  | nodeID:              1
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2020-12-30 06:12:39.293 UTC [49] LOG:  received fast shutdown request
        postgres_1    | .2020-12-30 06:12:39.296 UTC [49] LOG:  aborting any active transactions
        postgres_1    | 2020-12-30 06:12:39.297 UTC [49] LOG:  background worker "logical replication launcher" (PID 56) exited with exit code 1
        postgres_1    | 2020-12-30 06:12:39.303 UTC [51] LOG:  shutting down
        postgres_1    | 2020-12-30 06:12:39.413 UTC [49] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2020-12-30 06:12:39.505 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2020-12-30 06:12:39.505 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2020-12-30 06:12:39.511 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2020-12-30 06:12:39.524 UTC [86] LOG:  database system was shut down at 2020-12-30 06:12:39 UTC
        postgres_1    | 2020-12-30 06:12:39.530 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (602.17s)
        test_1        |     --- FAIL: TestCompare/postgres (1.36s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2552342&tab=buildLog) on [master@b93fd531b93cb010729cb73fe679cdff9388cf27](https://github.com/cockroachdb/cockroach/commits/b93fd531b93cb010729cb73fe679cdff9388cf27):

```
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_1 ASC, col1_2 DESC, col2_12 DESC, col1_11, col2_9 ASC, col1_0 ASC, col2_11, col1_9, col2_14 ASC);
        test_1        |             CREATE INDEX ON table2 (col1_9 ASC, col2_14 DESC, col2_15 ASC, col1_11 DESC, col1_1 DESC, col1_5 DESC) INCLUDE (col2_9, col2_13);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_1 DESC, col1_3 ASC, col2_14, col2_15 ASC, col1_0 DESC, col1_2 DESC);
        test_1        |             CREATE INDEX ON table2 (col1_9 DESC, col1_10 DESC, col2_15 ASC, col1_1 DESC, col1_2, col2_13 ASC, col1_5 DESC, col2_8 DESC, col2_12 ASC, col2_10, col1_11 DESC, col1_3 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_14);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_3, col2_9 ASC, col2_11 ASC);
        test_1        |             CREATE TABLE table3 (col1_9 TIMETZ NOT NULL, col1_11 INT8 NOT NULL, col1_2 REGCLASS NOT NULL, col1_10 REGPROC NOT NULL, col1_1 BOX2D NOT NULL, col1_5 INTERVAL NOT NULL, col1_0 TIMESTAMPTZ NOT NULL, col1_3 CHAR NOT NULL, col3_8 REGPROCEDURE, col3_9 UUID NULL, col3_10 DECIMAL NULL, col3_11 CHAR NOT NULL, col3_12 TIMESTAMPTZ, col3_13 TIMETZ NOT NULL, col3_14 TEXT, col3_15 REGTYPE NOT NULL, col3_16 BYTEA NOT NULL, PRIMARY KEY (col1_9, col1_11, col1_2, col1_10, col1_1, col1_5, col1_0, col1_3, col3_13, col3_16, col3_10, col3_8, col3_15, col3_9, col3_14, col3_11));
        test_1        |             CREATE INDEX ON table3 (col3_13 DESC, col1_0 DESC, col1_11 DESC, col3_9 DESC, col3_15, col1_5 DESC, col1_10, col1_2 ASC, col3_10 DESC, col1_9 DESC, col3_11 DESC, col1_3 ASC, col1_1 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col1_0, col1_9 DESC, col1_3 DESC) INCLUDE (col3_12);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col1_1 DESC, col1_11, col3_16, col3_8, col3_15 ASC, col1_9, col1_0 ASC, col3_11 ASC, col1_5, col3_10 ASC, col1_10 DESC, col3_12, col3_9, col3_13 DESC, col1_3, col3_14, col1_2 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_10, col1_1, col3_8 DESC, col3_13 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col1_11);
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col3_8, col3_12) REFERENCES table1 (col1_2, col1_0) ON DELETE RESTRICT;
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col3_15) REFERENCES table1 (col1_2) ON DELETE RESTRICT;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col1_11) REFERENCES table3 (col1_11);
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_14) REFERENCES table1 (col1_3);
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col1_0) REFERENCES table1 (col1_0) ON DELETE RESTRICT ON UPDATE RESTRICT;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('bwqn', 'o', 'mi', 'bc', 'oiy');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('mxdoai', 'srsuuj', 'p', 'ti', 'swtolq');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (0.63s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 OID NOT NULL, col1_1 BYTES NOT NULL, col1_2 OID NULL, col1_3 TIMESTAMPTZ NULL, col1_4 FLOAT4, UNIQUE (col1_4, col1_3 DESC) STORING (col1_1, col1_2) WHERE table1.col1_1 <= '\x', INDEX (col1_0 ASC, col1_1 ASC, col1_2, col1_3 DESC) WHERE (table1.col1_1 < e'\\x22':::BYTES) AND (table1.col1_3 <= '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ), INDEX (col1_2, col1_4 ASC, col1_3, col1_1, col1_0) WHERE ((table1.col1_1 > e'\\x27':::BYTES) AND (table1.col1_3 <= '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) OR (table1.col1_4 <= 1.0:::FLOAT8), UNIQUE (col1_2, col1_1, col1_0 ASC, col1_3 ASC) STORING (col1_4) WHERE ((table1.col1_3 = '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ) AND (table1.col1_4 <= 0.0:::FLOAT8)) AND (table1.col1_1 > e'\\xff':::BYTES), INDEX (col1_3 DESC, col1_0 ASC, col1_1 DESC, col1_4 DESC) WHERE (table1.col1_4 <= NaN) OR (table1.col1_1 != '\x00'), INDEX (col1_4, col1_0 ASC, col1_1 ASC, col1_3) WHERE ((table1.col1_4 = (-1.0):::FLOAT8) AND (table1.col1_3 >= '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table1.col1_1 >= e'\\xff':::BYTES), INDEX (col1_2, col1_3 DESC) STORING (col1_4) WHERE table1.col1_4 < (-1.0):::FLOAT8, FAMILY (col1_4), FAMILY (col1_2, col1_3), FAMILY (col1_0, col1_1));
        test_1        |             CREATE TABLE table2 (col2_0 INT8, col2_1 INT4, col2_2 BOX2D NOT NULL, col2_3 INT2 NOT NULL, col2_4 REGTYPE NOT NULL, col2_5 INT8 NOT NULL, col2_6 REGCLASS NOT NULL, col2_7 REGTYPE, col2_8 BIT(37) NULL, col2_9 BIT(25), col2_10 REGCLASS NULL, col2_11 FLOAT4 NOT NULL, col2_12 TIMETZ NOT NULL, PRIMARY KEY (col2_4, col2_3), UNIQUE (col2_8 DESC, col2_10 ASC, col2_2) STORING (col2_0, col2_6, col2_11, col2_12), FAMILY (col2_7, col2_9, col2_0), FAMILY (col2_10, col2_2), FAMILY (col2_1), FAMILY (col2_8, col2_6, col2_11, col2_5), FAMILY (col2_4), FAMILY (col2_12, col2_3));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 437015319580585375, "histo_buckets": [{"distinct_range": 0, "num_eq": 9219671397871109507, "num_range": 0, "upper_bound": "68635732"}, {"distinct_range": 0, "num_eq": 4227773727828514313, "num_range": 5970879461463554094, "upper_bound": "1208198200"}, {"distinct_range": 0, "num_eq": 60000000000, "num_range": 40, "upper_bound": "1400341759"}, {"distinct_range": 32244416.729893334, "num_eq": 6599524934136198848, "num_range": 40000000, "upper_bound": "1802777464"}, {"distinct_range": 431613955837869630, "num_eq": 100000000, "num_range": 597985477211953832, "upper_bound": "1855163012"}, {"distinct_range": 930712357645962600, "num_eq": 4815157323851340252, "num_range": 1057191853792108402, "upper_bound": "3159013904"}, {"distinct_range": 648.6083641658599, "num_eq": 40, "num_range": 800, "upper_bound": "3836877814"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 2069649317596063226}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 661557140567140270, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2069649317596063226}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1404397173452025159, "histo_buckets": [{"distinct_range": 0, "num_eq": 40, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 2480005162089088500, "num_eq": 10000000, "num_range": 2638209402837027783, "upper_bound": "517834619"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 10000000000, "upper_bound": "664473869"}, {"distinct_range": 6409582133.555092, "num_eq": 4000, "num_range": 20000000000, "upper_bound": "1072129024"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 3418620858114229874, "upper_bound": "2430413309"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 208881039612321565, "row_count": 2069649317596063226}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 128268102083345176, "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 8995641091595529, "row_count": 2069649317596063226}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 380049060201313373, "histo_buckets": [{"distinct_range": 0, "num_eq": 70281563443131472, "num_range": 0, "upper_bound": "-1.1379811763763428"}, {"distinct_range": 90000000, "num_eq": 700, "num_range": 90000000, "upper_bound": "-0.2579410672187805"}, {"distinct_range": 0, "num_eq": 8187970900366795268, "num_range": 4721039128191550187, "upper_bound": "-0.08459585905075073"}, {"distinct_range": 8715091033815191000, "num_eq": 0, "num_range": 9176012169815085965, "upper_bound": "0.7306168675422668"}, {"distinct_range": 4212263126266576400, "num_eq": 2000, "num_range": 4212263126266576532, "upper_bound": "1.1554126739501953"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 1009190099583909171, "row_count": 2069649317596063226}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2558738, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2841084, "histo_col_type": "", "name": "__auto__", "null_count": 5740661, "row_count": 10000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4239891, "histo_col_type": "", "name": "__auto__", "null_count": 5025408, "row_count": 10000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9518058, "histo_col_type": "", "name": "__auto__", "null_count": 4908141, "row_count": 10000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9785583, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 899501, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5740179, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9128419, "histo_buckets": [{"distinct_range": 0, "num_eq": 1215231427081570580, "num_range": 0, "upper_bound": "1100101111110010111100010011000101000"}, {"distinct_range": 675847909500096400, "num_eq": 10000000, "num_range": 675847909500096334, "upper_bound": "1101011011000100010110111000110001110"}], "histo_col_type": "BIT(37)", "name": "__auto__", "null_count": 2002855, "row_count": 10000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2019163, "histo_col_type": "", "name": "__auto__", "null_count": 5871171, "row_count": 10000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3947452, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6354114, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1629373, "histo_col_type": "", "name": "__auto__", "null_count": 8533696, "row_count": 10000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9796536, "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 10000000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('dfpej', 'hya', 'qnfabo', 'fg', 'qx');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('ix', 'edwjpx');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('fanrz');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('s', 'jgt');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('ccoj', 'wrc');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36, "histo_buckets": [{"distinct_range": 0, "num_eq": 500, "num_range": 0, "upper_bound": "1978-05-15 07:49:03.000499+00:00"}, {"distinct_range": 2707218619225228300, "num_eq": 8000000000, "num_range": 5316485648284092019, "upper_bound": "2016-07-23 19:21:38.000749+00:00"}, {"distinct_range": 0, "num_eq": 807106509422787651, "num_range": 0, "upper_bound": "294276-12-31 23:59:59.999999+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 592, "row_count": 600}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 577, "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 581, "row_count": 600}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_buckets": [{"distinct_range": 0, "num_eq": 6560559716054299728, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "803971460"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 600}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 268, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 280, "histo_buckets": [{"distinct_range": 0, "num_eq": 50, "num_range": 0, "upper_bound": "541776389"}, {"distinct_range": 506171934071008300, "num_eq": 7486731995031018810, "num_range": 2797224764622926702, "upper_bound": "1907774318"}, {"distinct_range": 7459557046129474000, "num_eq": 3561692650052276285, "num_range": 7459557046129473824, "upper_bound": "2150359136"}, {"distinct_range": 0, "num_eq": 6000000, "num_range": 400000, "upper_bound": "3637312494"}, {"distinct_range": 349310377649996160, "num_eq": 10000000000, "num_range": 349310377649996168, "upper_bound": "3847441273"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 226, "row_count": 600}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33420620, "histo_col_type": "", "name": "__auto__", "null_count": 55369488, "row_count": 60000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24210361, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9164557, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6797012, "histo_col_type": "", "name": "__auto__", "null_count": 41843650, "row_count": 60000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45028822, "histo_col_type": "BIT(37)", "name": "__auto__", "null_count": 54931423, "row_count": 60000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 47080446, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 59422056, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 55127757, "histo_col_type": "", "name": "__auto__", "null_count": 50502772, "row_count": 60000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30757643, "histo_buckets": [{"distinct_range": 0, "num_eq": 3051707790209416974, "num_range": 0, "upper_bound": "2465905654"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22480332, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11541733, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10190042, "histo_col_type": "", "name": "__auto__", "null_count": 33689336, "row_count": 60000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 37971869, "histo_col_type": "", "name": "__auto__", "null_count": 57339634, "row_count": 60000000}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44929199, "histo_buckets": [{"distinct_range": 0, "num_eq": 6888059381828062788, "num_range": 0, "upper_bound": "80298551"}, {"distinct_range": 0, "num_eq": 995270573073411432, "num_range": 30000, "upper_bound": "1000594699"}, {"distinct_range": 0, "num_eq": 8537882852936820023, "num_range": 4455276190965054538, "upper_bound": "2171933382"}, {"distinct_range": 7182286184180575000, "num_eq": 5000000000, "num_range": 7182286184180574888, "upper_bound": "2463432565"}, {"distinct_range": 40, "num_eq": 5000, "num_range": 40, "upper_bound": "2748802622"}, {"distinct_range": 8154751517275888000, "num_eq": 4719396621117805745, "num_range": 8154751517275887796, "upper_bound": "3216739013"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 108475894, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 189270297, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000000, "num_range": 0, "upper_bound": "2314752199"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 80000, "upper_bound": "2355532184"}, {"distinct_range": 14993727489.092678, "num_eq": 0, "num_range": 20000000000, "upper_bound": "3929360069"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 125637693, "row_count": 400000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 291975931, "histo_buckets": [{"distinct_range": 0, "num_eq": 5455091140196985887, "num_range": 0, "upper_bound": "1977-04-30 20:17:09.000688+00:00"}, {"distinct_range": 3293572970.368952, "num_eq": 0, "num_range": 4000000000, "upper_bound": "1978-11-18 00:27:58.000804+00:00"}, {"distinct_range": 1778414879241695000, "num_eq": 200000, "num_range": 2402010151198162400, "upper_bound": "1987-05-19 21:52:49.000022+00:00"}, {"distinct_range": 0, "num_eq": 30, "num_range": 10000, "upper_bound": "2002-01-07 00:36:10.0009+00:00"}, {"distinct_range": 0, "num_eq": 30000000000, "num_range": 10000000000, "upper_bound": "2004-09-05 16:02:25.000307+00:00"}, {"distinct_range": 1000000000, "num_eq": 7807836663624511959, "num_range": 1000000000, "upper_bound": "2024-04-19 01:35:27.000049+00:00"}, {"distinct_range": 1178201766133119200, "num_eq": 1346571386894366650, "num_range": 5956569154650652795, "upper_bound": "2026-06-11 04:50:15.000524+00:00"}, {"distinct_range": 80000000, "num_eq": 900, "num_range": 80000000, "upper_bound": "3000-01-01 00:00:00+00:00"}, {"distinct_range": 3525524350312095000, "num_eq": 2000, "num_range": 6176258904553512969, "upper_bound": "294276-12-31 23:59:59.999999+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 352292651, "row_count": 400000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 103632602, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "1.401298464324817e-45"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 87063960, "row_count": 400000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38070, "histo_col_type": "", "name": "__auto__", "null_count": 28240, "row_count": 50000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27730, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 43604, "histo_col_type": "", "name": "__auto__", "null_count": 12730, "row_count": 50000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6811, "histo_col_type": "", "name": "__auto__", "null_count": 49625, "row_count": 50000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2912, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13753, "histo_buckets": [{"distinct_range": 0, "num_eq": 7602686053126087121, "num_range": 0, "upper_bound": "0100111001001111101001100110100011000"}, {"distinct_range": 56495.09393530201, "num_eq": 8710939530775196514, "num_range": 100000, "upper_bound": "0110000010011000010110010001010111001"}, {"distinct_range": 0, "num_eq": 4828656768830750964, "num_range": 500000000, "upper_bound": "0111000011011100100010011101011000110"}, {"distinct_range": 686.1757939394071, "num_eq": 300000000, "num_range": 2000, "upper_bound": "0111111111111111111111111111111111111"}, {"distinct_range": 1001250384702565400, "num_eq": 40000000, "num_range": 8906009711934423720, "upper_bound": "1000101000010100111110110000001111110"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 6930444822006950019, "upper_bound": "1011101000000110111011001001110001010"}, {"distinct_range": 3478452276.7954297, "num_eq": 8000, "num_range": 7000000000, "upper_bound": "1110000101101101110010110001101001111"}], "histo_col_type": "BIT(37)", "name": "__auto__", "null_count": 13300, "row_count": 50000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19123, "histo_col_type": "", "name": "__auto__", "null_count": 2394, "row_count": 50000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14084, "histo_col_type": "", "name": "__auto__", "null_count": 27968, "row_count": 50000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 572, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42029, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14371, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19402, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "425767189"}, {"distinct_range": 193353104815752320, "num_eq": 300000, "num_range": 579861557835395328, "upper_bound": "999929963"}, {"distinct_range": 1715377081440161000, "num_eq": 8068059367558332695, "num_range": 2025889806649109317, "upper_bound": "1173836884"}, {"distinct_range": 800000000, "num_eq": 8175904704139428307, "num_range": 800000000, "upper_bound": "1459910291"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5306, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "nan" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2553799&tab=buildLog) on [master@d7bbe0060531063b9bee29f69bc4d23d41b84e3d](https://github.com/cockroachdb/cockroach/commits/d7bbe0060531063b9bee29f69bc4d23d41b84e3d):

```
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down....2021-01-01 06:12:43.237 UTC [52] LOG:  received fast shutdown request
        postgres_1    | 2021-01-01 06:12:43.239 UTC [52] LOG:  aborting any active transactions
        postgres_1    | 2021-01-01 06:12:43.240 UTC [52] LOG:  background worker "logical replication launcher" (PID 59) exited with exit code 1
        postgres_1    | 2021-01-01 06:12:43.247 UTC [54] LOG:  shutting down
        postgres_1    | 2021-01-01 06:12:43.375 UTC [52] LOG:  database system is shut down
        test_1        | --- FAIL: TestCompare (3.17s)
        test_1        |     --- FAIL: TestCompare/postgres (2.29s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.3:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.88s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 BOX2D NOT NULL, col1_1 INT2 NOT NULL, col1_2 TIMESTAMP NULL, col1_3 INTERVAL NOT NULL, col1_4 UUID NULL, col1_5 "char" NULL, col1_6 DATE NOT NULL, col1_7 VARCHAR, col1_8 BYTES NOT NULL, col1_9 FLOAT4, col1_10 DECIMAL NOT NULL, col1_11 REGPROC NOT NULL, col1_12 INT4 NOT NULL, col1_13 TIMETZ NULL, col1_14 BOX2D, col1_15 DATE NULL, PRIMARY KEY (col1_12 DESC, col1_3, col1_6 DESC, col1_10 DESC, col1_8, col1_11, col1_0 ASC), UNIQUE (col1_10, col1_13 ASC, col1_14, col1_11 ASC, col1_4, col1_8, col1_3, col1_1, col1_0, col1_9, col1_7 ASC, col1_15 DESC) STORING (col1_2, col1_5), UNIQUE (col1_4 ASC, col1_8, col1_3 DESC) STORING (col1_1, col1_2, col1_9, col1_13) WHERE ((table1.col1_7 < e'\U00002603':::STRING) AND (table1.col1_1 = (-1):::INT8)) OR (table1.col1_15 < '1970-01-01':::DATE), INDEX (col1_2 DESC, col1_11, col1_13 ASC), UNIQUE (col1_14, col1_4, col1_6 DESC, col1_7, col1_9 ASC, col1_11, col1_10 DESC, col1_1 ASC, col1_5 DESC) WHERE (((((((((table1.col1_6 <= 'infinity':::DATE) AND (table1.col1_2 >= '294276-12-31 23:59:59.999999':::TIMESTAMP)) OR (table1.col1_8 > e'\\x00':::BYTES)) AND (table1.col1_5 <= '':::STRING)) OR (table1.col1_7 != '':::STRING)) OR (table1.col1_9 != 'NaN':::FLOAT8)) OR (table1.col1_15 > '5874897-12-31':::DATE)) OR (table1.col1_1 > 1:::INT8)) OR (table1.col1_10 >= 'NaN':::DECIMAL)) AND (table1.col1_12 > 127:::INT8), INDEX (col1_2 ASC, col1_10 DESC, col1_0, col1_11 DESC) STORING (col1_15) WHERE ((((table1.col1_1 > (-1):::INT8) OR (table1.col1_9 != '-Inf':::FLOAT8)) AND (table1.col1_7 >= e'\'':::STRING)) OR (table1.col1_12 < (-128):::INT8)) AND (table1.col1_2 < '-2000-01-01 00:00:00':::TIMESTAMP), INDEX (col1_3, col1_7 ASC), FAMILY (col1_13, col1_0, col1_2), FAMILY (col1_9, col1_4), FAMILY (col1_14, col1_10, col1_3), FAMILY (col1_1, col1_8), FAMILY (col1_7, col1_12), FAMILY (col1_5, col1_15), FAMILY (col1_11, col1_6), UNIQUE (col1_11) STORING (col1_1, col1_4, col1_5, col1_13, col1_15) WHERE (((((((table1.col1_7 = '') AND (table1.col1_6 < 'infinity')) AND (table1.col1_13 < '00:00:00+15:59:00')) OR (table1.col1_1 > -1)) OR (table1.col1_12 = -2147483648)) OR (table1.col1_9 = 0.0)) OR (table1.col1_8 >= '\x27')) OR (table1.col1_5 <= '"'));
        test_1        |             CREATE TABLE table2 (col2_0 "char" NOT NULL, col2_1 TIMESTAMPTZ NOT NULL, PRIMARY KEY (col2_1), INDEX (col2_1 DESC) STORING (col2_0), UNIQUE (col2_1, col2_0) WHERE table2.col2_0 >= e'\'':::STRING, FAMILY (col2_0, col2_1));
        test_1        |             CREATE TABLE table3 (col3_0 FLOAT4 NOT NULL, col3_1 OID NULL, col3_2 OID NULL, col3_3 BIT(44) NOT NULL, col3_4 TIME, col3_5 "char", col3_6 CHAR NOT NULL, col3_7 NAME NOT NULL, col3_8 DECIMAL NULL, col3_9 VARCHAR NOT NULL, PRIMARY KEY (col3_6), INDEX (col3_8 ASC, col3_0, col3_9, col3_3 ASC, col3_6 DESC) STORING (col3_1, col3_4, col3_5, col3_7) WHERE (table3.col3_7 <= '"':::STRING) OR (table3.col3_9 <= e'\'':::STRING), INDEX (col3_0, col3_5 ASC, col3_1 ASC) STORING (col3_4, col3_7, col3_8) WHERE ((((((table3.col3_6 <= e'\U00002603':::STRING) AND (table3.col3_8 <= 'NaN':::DECIMAL)) OR (table3.col3_7 < e'\x00':::STRING)) OR (table3.col3_0 > '+Inf':::FLOAT8)) OR (table3.col3_4 = '24:00:00':::TIME)) AND (table3.col3_5 > 'X':::STRING)) OR (table3.col3_9 >= '"':::STRING), INDEX (col3_8, col3_0, col3_4, col3_2 ASC, col3_7, col3_3 DESC, col3_9, col3_1 DESC) STORING (col3_5) WHERE ((((((table3.col3_0 = '+Inf':::FLOAT8) AND (table3.col3_4 < '24:00:00':::TIME)) OR (table3.col3_7 = e'\'':::STRING)) AND (table3.col3_5 >= e'\x00':::STRING)) OR (table3.col3_9 != '':::STRING)) AND (table3.col3_6 <= 'X':::STRING)) OR (table3.col3_8 >= (-1.234E+401):::DECIMAL), UNIQUE (col3_9 DESC, col3_5, col3_8 ASC, col3_6 DESC, col3_3, col3_4 ASC, col3_7 ASC) STORING (col3_2) WHERE (table3.col3_0 <= 1.0:::FLOAT8) AND (table3.col3_6 <= '':::STRING), UNIQUE (col3_5, col3_0, col3_1 DESC, col3_9 ASC, col3_2 DESC, col3_6 ASC, col3_7) WHERE ((((((table3.col3_4 <= '00:00:00':::TIME) AND (table3.col3_6 = 'X':::STRING)) AND (table3.col3_7 = e'\x00':::STRING)) OR (table3.col3_8 < (-1.234E+401):::DECIMAL)) OR (table3.col3_5 >= '':::STRING)) AND (table3.col3_9 > e'\'':::STRING)) AND (table3.col3_0 = '-Inf':::FLOAT8), UNIQUE (col3_8 DESC, col3_7 DESC, col3_5 DESC, col3_6 ASC, col3_4, col3_2) STORING (col3_0, col3_3), UNIQUE (col3_3 ASC, col3_7 ASC, col3_0, col3_9 DESC, col3_8, col3_5, col3_1, col3_2 ASC, col3_6) STORING (col3_4) WHERE ((((table3.col3_7 >= '') AND (table3.col3_4 != '24:00:00')) AND (table3.col3_8 >= -Infinity)) OR (table3.col3_9 > '"')) AND (table3.col3_0 != +Inf), FAMILY (col3_9, col3_4, col3_2), FAMILY (col3_6, col3_1), FAMILY (col3_8, col3_5, col3_3), FAMILY (col3_7), FAMILY (col3_0));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 390282435177233015, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2099646108608587254}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1974338354580937779, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-2000-01-01 00:00:00"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 1644490979627952815, "row_count": 2099646108608587254}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1617872229572468295, "histo_col_type": "", "name": "__auto__", "null_count": 153456306127020969, "row_count": 2099646108608587254}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 714406474082716211, "histo_col_type": "", "name": "__auto__", "null_count": 1288749615638172311, "row_count": 2099646108608587254}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1411495304281574134, "histo_buckets": [{"distinct_range": 0, "num_eq": 500000000, "num_range": 0, "upper_bound": "BOX(-2.6957190203962647 -0.6972405441725504,1.3887767740129409 0.9792130491197351)"}, {"distinct_range": 877488094390374100, "num_eq": 90000000000, "num_range": 5160618178686358971, "upper_bound": "BOX(-1.2774645550056343 -0.25825645518962304,-0.21567997885959 0.6457325054786489)"}, {"distinct_range": 3696179251182673400, "num_eq": 10, "num_range": 8620868062195690875, "upper_bound": "BOX(-1.1160316717908292 0.013521425863113556,-0.582986513524897 0.4444560217065886)"}, {"distinct_range": 71.11943139849282, "num_eq": 300, "num_range": 100, "upper_bound": "BOX(-0.40789407835945285 0.38479626306171566,-0.13834941924002842 0.5753375884293042)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 1528956689844060109, "row_count": 2099646108608587254}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 214688573261761590, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2099646108608587254}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1830259696514243998, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000000, "num_range": 0, "upper_bound": "-59 years -4 mons -893 days -15:43:24.711092"}, {"distinct_range": 0, "num_eq": 800000, "num_range": 2368126495440193654, "upper_bound": "-4 years -8 mons -591 days -12:11:08.04895"}, {"distinct_range": 862842347.2992562, "num_eq": 100, "num_range": 1000000000, "upper_bound": "00:00:00"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 2543045304486879819, "upper_bound": "1 mon 1 day"}, {"distinct_range": 0, "num_eq": 5893665276329439013, "num_range": 3375863646262759235, "upper_bound": "2 years 164 days 10:14:29.519235"}, {"distinct_range": 800, "num_eq": 30000, "num_range": 800, "upper_bound": "6 years 2 mons 586 days 07:43:56.246107"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "10 years 1 mon 166 days 13:49:49.018549"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 2099646108608587254}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 333744221167297671, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "NaN"}, {"distinct_range": 0, "num_eq": 2000000, "num_range": 0, "upper_bound": "-4.027130519727173822E+26"}, {"distinct_range": 109268029014598300, "num_eq": 5536948730157493293, "num_range": 109268029014598308, "upper_bound": "70806823.07412052987"}, {"distinct_range": 300000000, "num_eq": 7014390310267956468, "num_range": 300000000, "upper_bound": "5.537322820042858401E+34"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 2099646108608587254}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 594714618940738547, "histo_buckets": [{"distinct_range": 0, "num_eq": 1138521366057566409, "num_range": 0, "upper_bound": "-2050730199"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 8780405144823180678, "upper_bound": "-1999904344"}, {"distinct_range": 100, "num_eq": 7264178818050435198, "num_range": 100, "upper_bound": "-1854603590"}, {"distinct_range": 1352540685697332500, "num_eq": 300, "num_range": 7339289764800418039, "upper_bound": "-690112923"}, {"distinct_range": 0, "num_eq": 40, "num_range": 0, "upper_bound": "-590977286"}, {"distinct_range": 6981023110090094000, "num_eq": 985445412184689041, "num_range": 6981023110090093590, "upper_bound": "-403712674"}, {"distinct_range": 1865599.5109729196, "num_eq": 9000000, "num_range": 10000000, "upper_bound": "-1"}, {"distinct_range": 0, "num_eq": 5000, "num_range": 100, "upper_bound": "346604967"}, {"distinct_range": 7101109627396893000, "num_eq": 8879036323076133551, "num_range": 7101109627396892844, "upper_bound": "1342905165"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 2099646108608587254}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 601101097600949507, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2099646108608587254}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1766622083105199234, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2099646108608587254}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 526180715653648116, "histo_col_type": "", "name": "__auto__", "null_count": 778992216616568100, "row_count": 2099646108608587254}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 767162299673672043, "histo_buckets": [{"distinct_range": 0, "num_eq": 3421662245073030206, "num_range": 0, "upper_bound": "00000000-0000-0000-0000-000000000000"}, {"distinct_range": 307990044494849860, "num_eq": 700, "num_range": 1738958874895814127, "upper_bound": "1bb62a3f-0109-411d-8aeb-bacf51d5a5e0"}, {"distinct_range": 0, "num_eq": 30000, "num_range": 7127078816644816591, "upper_bound": "73c24462-b01c-412c-a759-43e747484a68"}, {"distinct_range": 4501952523150958000, "num_eq": 1635560327445053622, "num_range": 8256234636950192284, "upper_bound": "cf89fc26-b361-4557-b9a6-9f500a050fd4"}, {"distinct_range": 3175977378.1675973, "num_eq": 0, "num_range": 10000000000, "upper_bound": "e9f6e531-888b-4fff-a68f-91975e49259a"}, {"distinct_range": 2554041121087942000, "num_eq": 6000000, "num_range": 4021728415511011683, "upper_bound": "fc6a5da5-ebf9-4325-923d-5a9c8175ce51"}, {"distinct_range": 70000, "num_eq": 1488406309891473619, "num_range": 70000, "upper_bound": "ff820083-1058-4f9d-ad2f-478b1bb3c4f3"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 332087125737399625, "row_count": 2099646108608587254}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2035528014612909614, "histo_col_type": "", "name": "__auto__", "null_count": 95550292701138238, "row_count": 2099646108608587254}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 478345749036959479, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2099646108608587254}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1510453688944685043, "histo_col_type": "", "name": "__auto__", "null_count": 73481442843476652, "row_count": 2099646108608587254}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1438, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5327, "histo_buckets": [{"distinct_range": 0, "num_eq": 10, "num_range": 0, "upper_bound": "1982-03-29 17:51:19.000828+00:00"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 654457598181628651, "upper_bound": "2006-06-09 02:12:54.000903+00:00"}, {"distinct_range": 2562407299.803973, "num_eq": 8904817756677505120, "num_range": 4000000000, "upper_bound": "2006-12-06 05:15:17.000688+00:00"}, {"distinct_range": 30, "num_eq": 6000000000, "num_range": 30, "upper_bound": "2029-04-09 18:04:24.000769+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 6000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46463356, "histo_col_type": "", "name": "__auto__", "null_count": 88559933, "row_count": 100000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 50511435, "histo_col_type": "", "name": "__auto__", "null_count": 41561873, "row_count": 100000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67625357, "histo_buckets": [{"distinct_range": 0, "num_eq": 3609208582317773444, "num_range": 0, "upper_bound": "\\u000c"}, {"distinct_range": 500000000, "num_eq": 40, "num_range": 500000000, "upper_bound": "-"}, {"distinct_range": 0, "num_eq": 288288656359761481, "num_range": 10000000, "upper_bound": "k"}, {"distinct_range": 0, "num_eq": 7033134679148911228, "num_range": 2861840159020880954, "upper_bound": "y"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 88701118, "row_count": 100000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14336750, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 54953850, "histo_buckets": [{"distinct_range": 0, "num_eq": 732605761400404378, "num_range": 0, "upper_bound": "NaN"}, {"distinct_range": 400, "num_eq": 5041813374610030038, "num_range": 400, "upper_bound": "-1.068054972631606262E+29"}, {"distinct_range": 90, "num_eq": 40000000, "num_range": 90, "upper_bound": "-5419.485357772259279"}, {"distinct_range": 7050426979805246000, "num_eq": 100, "num_range": 7050426979805245997, "upper_bound": "1.844755846742693549E+21"}, {"distinct_range": 0, "num_eq": 900000, "num_range": 315548395788679021, "upper_bound": "Infinity"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 82907676, "row_count": 100000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 52577732, "histo_buckets": [{"distinct_range": 0, "num_eq": 7515896234116634520, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 0, "num_eq": 6867546375786123526, "num_range": 9000, "upper_bound": "\\u0005\\u0007%H=c\\u0008r"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 700, "upper_bound": "\\u0016\\u0002F,\\"P"}, {"distinct_range": 400, "num_eq": 60, "num_range": 400, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 200000000, "num_range": 3000000, "upper_bound": "O5"}, {"distinct_range": 15617444919.374924, "num_eq": 8580291521173682831, "num_range": 30000000000, "upper_bound": "|\\u0007GOf"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19127464, "histo_buckets": [{"distinct_range": 0, "num_eq": 30, "num_range": 0, "upper_bound": "-1.2566118240356445"}, {"distinct_range": 906368194882426900, "num_eq": 750129190981171097, "num_range": 906368194882426868, "upper_bound": "0.4483576714992523"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 0, "upper_bound": "1.5103198289871216"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 57453870, "histo_col_type": "", "name": "__auto__", "null_count": 13733010, "row_count": 100000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 68277039, "histo_buckets": [{"distinct_range": 0, "num_eq": 300000000, "num_range": 0, "upper_bound": "11110010111001010010010001001111111011000110"}], "histo_col_type": "BIT(44)", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 69417678, "histo_buckets": [{"distinct_range": 0, "num_eq": 5686790822488228978, "num_range": 0, "upper_bound": "\\u000b"}, {"distinct_range": 20000, "num_eq": 1847869254764218172, "num_range": 20000, "upper_bound": ">"}, {"distinct_range": 0, "num_eq": 5632558493861806430, "num_range": 800, "upper_bound": "s"}, {"distinct_range": 30000000, "num_eq": 1721539516472505611, "num_range": 30000000, "upper_bound": "t"}, {"distinct_range": 0, "num_eq": 6053078399614548885, "num_range": 8000, "upper_bound": "y"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 100000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col3_2) REFERENCES table1 (col1_11);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('gu');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4025961682208715259, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5229781282819799533}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2600868593815634179, "histo_col_type": "", "name": "__auto__", "null_count": 4871461376943255009, "row_count": 5229781282819799533}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3162840404588793197, "histo_buckets": [{"distinct_range": 0, "num_eq": 3445687061671477468, "num_range": 0, "upper_bound": "-2105309577"}, {"distinct_range": 5813974729235328000, "num_eq": 700000, "num_range": 7654501203912247603, "upper_bound": "-1906772243"}, {"distinct_range": 40000000, "num_eq": 277811230200187058, "num_range": 40000000, "upper_bound": "-1812896819"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 70000, "upper_bound": "-1438398844"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "-410281350"}, {"distinct_range": 154497380934700130, "num_eq": 100000000, "num_range": 1006330234349095320, "upper_bound": "-404399236"}, {"distinct_range": 49709.069608795835, "num_eq": 500000, "num_range": 800000, "upper_bound": "709861664"}, {"distinct_range": 3.5070966635930185, "num_eq": 90, "num_range": 50, "upper_bound": "980255622"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 5229781282819799533}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4915049642712192002, "histo_col_type": "", "name": "__auto__", "null_count": 4485890948505655283, "row_count": 5229781282819799533}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2854586150573379364, "histo_col_type": "", "name": "__auto__", "null_count": 158016579958556308, "row_count": 5229781282819799533}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 867647037062666748, "histo_buckets": [{"distinct_range": 0, "num_eq": 7634297729324878439, "num_range": 0, "upper_bound": "-11 years -3 mons -922 days -23:50:06.927162"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 5229781282819799533}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4966177416913781448, "histo_buckets": [{"distinct_range": 0, "num_eq": 2753891877855815050, "num_range": 0, "upper_bound": "2b81a6f6-3420-4533-b41e-4a5dcfee6764"}, {"distinct_range": 370458975.40732414, "num_eq": 20000, "num_range": 1000000000, "upper_bound": "34f603b3-dd0c-4360-abe1-6607ba41ce7f"}, {"distinct_range": 6748997339.727617, "num_eq": 6000000, "num_range": 7000000000, "upper_bound": "d3814690-8ee1-438a-96af-5717a0cb735f"}, {"distinct_range": 261099.79274953742, "num_eq": 10000, "num_range": 500000, "upper_bound": "ebaf210d-0bed-44fc-b786-d8a16dae7bf9"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 1662894933823397281, "row_count": 5229781282819799533}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3408556586199223733, "histo_col_type": "", "name": "__auto__", "null_count": 3214162751967770208, "row_count": 5229781282819799533}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3529284507032391281, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5229781282819799533}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3657802870082342561, "histo_buckets": [{"distinct_range": 0, "num_eq": 368654905447355856, "num_range": 0, "upper_bound": "-1.407362861747663199E+36"}, {"distinct_range": 0, "num_eq": 40, "num_range": 412042888283930564, "upper_bound": "-6.011259866213500333E+35"}, {"distinct_range": 3514.5038137828296, "num_eq": 900000000, "num_range": 4000, "upper_bound": "-3103556171294470.899"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 300, "upper_bound": "442.3602305497288885"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 1805367107952019914, "upper_bound": "2310130376615722.334"}, {"distinct_range": 20000000, "num_eq": 800, "num_range": 20000000, "upper_bound": "5.389046239150710212E+28"}, {"distinct_range": 0, "num_eq": 9000000, "num_range": 900000000, "upper_bound": "9.100388380338719547E+35"}, {"distinct_range": 5511018.901948054, "num_eq": 90, "num_range": 8000000, "upper_bound": "4.30203797944174834E+36"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 5229781282819799533}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 212988032719650521, "histo_buckets": [{"distinct_range": 0, "num_eq": 70, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 50000000000, "num_eq": 70, "num_range": 50000000000, "upper_bound": "1287916652"}, {"distinct_range": 0, "num_eq": 3281275350115988500, "num_range": 6000, "upper_bound": "1363178363"}, {"distinct_range": 707.7855984977195, "num_eq": 200000, "num_range": 2000, "upper_bound": "1532948984"}, {"distinct_range": 7000, "num_eq": 90000, "num_range": 7000, "upper_bound": "2786746556"}, {"distinct_range": 0, "num_eq": 4000000, "num_range": 600, "upper_bound": "3894290574"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 5229781282819799533}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 750309214163729992, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5229781282819799533}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3488009487376750090, "histo_buckets": [{"distinct_range": 0, "num_eq": 5198451301242094752, "num_range": 0, "upper_bound": "2000-02-06 05:41:26.000104"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 3260535953976111745, "row_count": 5229781282819799533}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2787138863818967172, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5229781282819799533}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 474040435616311199, "histo_col_type": "", "name": "__auto__", "null_count": 5054452734025764018, "row_count": 5229781282819799533}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3875875317629994544, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "BOX(-2.7775840685445665 -1.039105437648749,-1.5541084727852499 0.3177016020542631)"}, {"distinct_range": 0, "num_eq": 8000, "num_range": 5000000000, "upper_bound": "BOX(-1.1873596795717196 -0.3968554294008547,0.9279791990334374 0.6390756923512076)"}, {"distinct_range": 0, "num_eq": 0, "num_range": 7653748559289053706, "upper_bound": "BOX(-1.0863827687550418 -1.1415966912602562,-0.4604140161236243 -0.15166623095108545)"}, {"distinct_range": 5837845304438192000, "num_eq": 50000000000, "num_range": 5837845304438192331, "upper_bound": "BOX(-0.7646870538157422 -0.4354132711499947,-0.028205436684978102 1.434947316324399)"}, {"distinct_range": 0, "num_eq": 5241774632706922459, "num_range": 0, "upper_bound": "BOX(-0.7381227089684488 -1.8804121806024905,1.7386363279942785 0.6698228591556502)"}, {"distinct_range": 0.11481312914600125, "num_eq": 5566547369973772116, "num_range": 20, "upper_bound": "BOX(-0.6932708485837056 -0.512050367610434,0.2835227861971399 -0.42955862552576923)"}, {"distinct_range": 643887153039221800, "num_eq": 8299434983899260465, "num_range": 643887153039221770, "upper_bound": "BOX(0.026979815865606716 0.004698098303340115,0.8758727678581418 1.095974751785565)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 581698882455148613, "row_count": 5229781282819799533}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 427389895617329900, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7411659476046217042}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6687738342424101401, "histo_buckets": [{"distinct_range": 0, "num_eq": 915512919920224929, "num_range": 0, "upper_bound": "-4713-11-24 00:00:00+00:00"}, {"distinct_range": 830168.4357213209, "num_eq": 3519010201842085038, "num_range": 6000000, "upper_bound": "1996-10-14 11:56:24.000555+00:00"}, {"distinct_range": 6000, "num_eq": 2892074276191635400, "num_range": 6000, "upper_bound": "2007-03-19 23:55:56.000459+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 7411659476046217042}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 430, "histo_col_type": "", "name": "__auto__", "null_count": 30, "row_count": 600}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 138, "histo_buckets": [{"distinct_range": 0, "num_eq": 8220740315854230152, "num_range": 0, "upper_bound": "00010001001011111101101100011001101110101101"}, {"distinct_range": 0, "num_eq": 1691117139210215633, "num_range": 40000, "upper_bound": "00011101010001001001100001001000011000011001"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 0, "upper_bound": "01000101101111000110000110011011111010010110"}, {"distinct_range": 2097928903511067000, "num_eq": 819932856400604354, "num_range": 2097928903511066899, "upper_bound": "01001001010000010111010110111010110110001001"}, {"distinct_range": 30000000000, "num_eq": 0, "num_range": 30000000000, "upper_bound": "01001100001100000100101110111100100010010010"}, {"distinct_range": 0, "num_eq": 8817510787411550370, "num_range": 7788220041169627959, "upper_bound": "10110110001000110010011101001001101101100101"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "10111000011010110010011001101101110100101000"}, {"distinct_range": 335018869566885300, "num_eq": 0, "num_range": 2976477775421621716, "upper_bound": "11001001110000110101010100100101001111100100"}], "histo_col_type": "BIT(44)", "name": "__auto__", "null_count": 0, "row_count": 600}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 478, "histo_col_type": "", "name": "__auto__", "null_count": 219, "row_count": 600}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 156, "histo_buckets": [{"distinct_range": 0, "num_eq": 5662339976073304266, "num_range": 0, "upper_bound": "\\u0001"}, {"distinct_range": 700, "num_eq": 80000000, "num_range": 700, "upper_bound": "("}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 50000000000, "upper_bound": "K"}, {"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "x"}, {"distinct_range": 0, "num_eq": 400000, "num_range": 40000000, "upper_bound": "z"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 600}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 583, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "820611070627020129.9"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 530, "row_count": 600}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 432, "histo_buckets": [{"distinct_range": 0, "num_eq": 8332579562051611808, "num_range": 0, "upper_bound": "-1.470873236656189"}, {"distinct_range": 780.154499229245, "num_eq": 569418739830222749, "num_range": 800, "upper_bound": "0.32321223616600037"}, {"distinct_range": 0, "num_eq": 4784528543617654904, "num_range": 50000000, "upper_bound": "+Inf"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 600}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 584, "histo_col_type": "", "name": "__auto__", "null_count": 345, "row_count": 600}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45, "histo_buckets": [{"distinct_range": 0, "num_eq": 500000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 4728519120.492427, "num_eq": 5918020739062712491, "num_range": 8000000000, "upper_bound": "\\u001b$\\u0011\\u0013^\\u000cq +"}, {"distinct_range": 320832196147641860, "num_eq": 4335879179399764351, "num_range": 320832196147641826, "upper_bound": "\\""}, {"distinct_range": 9514448477.008022, "num_eq": 60000000, "num_range": 90000000000, "upper_bound": "%xJ\\twi)X\\u0015"}, {"distinct_range": 0, "num_eq": 30000000000, "num_range": 3816893719778023797, "upper_bound": ";Y4Z3K\\u0005\\u000bn"}, {"distinct_range": 3099638649991654400, "num_eq": 100000000, "num_range": 8449463638849903337, "upper_bound": "NePo\\u001eH"}, {"distinct_range": 572946064886198460, "num_eq": 30000000000, "num_range": 1602703214744253849, "upper_bound": "Pp{fTT]b\\u0004"}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 5433813099787351413, "upper_bound": "U\\u0019:*1"}, {"distinct_range": 4149837382212649500, "num_eq": 500000000, "num_range": 4149837382212649636, "upper_bound": "~b "}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 600}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 137, "histo_buckets": [{"distinct_range": 0, "num_eq": 5073497600630444003, "num_range": 0, "upper_bound": "q"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 473, "row_count": 600}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 504, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7985217, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6192456, "histo_col_type": "", "name": "__auto__", "null_count": 5655491, "row_count": 8000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 682930, "histo_buckets": [{"distinct_range": 0, "num_eq": 8197621209421523801, "num_range": 0, "upper_bound": "6a98e73b-5113-4b9e-aafa-287b5c75f6bf"}, {"distinct_range": 0, "num_eq": 800, "num_range": 8000000000, "upper_bound": "757a00b6-6fcf-4414-9021-92b5624cee84"}, {"distinct_range": 0, "num_eq": 90000000000, "num_range": 5988150022949933044, "upper_bound": "a7c6f9a6-8991-4199-a077-7b715c8a9895"}, {"distinct_range": 23.446959098676817, "num_eq": 8000000, "num_range": 60, "upper_bound": "bf8ee34b-a7bd-4497-bfcf-cde23cc1f4cb"}, {"distinct_range": 3411998778168084000, "num_eq": 508461039439886603, "num_range": 3411998778168084172, "upper_bound": "c5683e88-8a60-4010-a180-187e7cee427c"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 9120743649247727697, "upper_bound": "ceca1581-f9be-48f7-abe3-a82a259b1552"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 70000, "upper_bound": "dae94a24-2cca-4c09-8b57-c1f0d0da594f"}, {"distinct_range": 72877.69246076011, "num_eq": 40000000000, "num_range": 90000, "upper_bound": "e6e45a7a-b124-41cf-8c6c-ed35c0ae8ae8"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 3744775, "row_count": 8000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7105797, "histo_col_type": "", "name": "__auto__", "null_count": 6484750, "row_count": 8000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5145284, "histo_col_type": "", "name": "__auto__", "null_count": 1860533, "row_count": 8000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7809279, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5745476, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3866993, "histo_col_type": "", "name": "__auto__", "null_count": 6458126, "row_count": 8000000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1303605, "histo_buckets": [{"distinct_range": 0, "num_eq": 300, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 0, "num_eq": 9122015133089594838, "num_range": 1259725113324995624, "upper_bound": "BOX(-2.205038152215444 -0.9754622962502664,0.4362393568181494 1.4268505718661642)"}, {"distinct_range": 3040639041913147400, "num_eq": 6696017123663581151, "num_range": 5843659627514410132, "upper_bound": "BOX(-1.2355319024987579 -1.4422503837346246,-0.4705023790789997 2.560020072265587)"}, {"distinct_range": 20000000, "num_eq": 3732878074104822682, "num_range": 20000000, "upper_bound": "BOX(-1.0880298908291315 0.021339080370413877,-0.25048702077910734 0.22321319995641442)"}, {"distinct_range": 0, "num_eq": 2844426653817622758, "num_range": 3607515649849350967, "upper_bound": "BOX(-0.9140882457536653 -0.5124756578810036,0.6676335153580126 -0.4714481696439577)"}, {"distinct_range": 151664511604457660, "num_eq": 300000000, "num_range": 151664511604457661, "upper_bound": "BOX(-0.7751648663663092 0.4366615972401645,1.2199011636715662 1.4782629251876438)"}, {"distinct_range": 0, "num_eq": 4890067267432506404, "num_range": 1662507564791544465, "upper_bound": "BOX(-0.09798319957518994 -0.1235862115908758,-0.024176045806924562 0.6112022072755857)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 2679450, "row_count": 8000000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3717, "histo_col_type": "", "name": "__auto__", "null_count": 7072312, "row_count": 8000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3764616, "histo_buckets": [{"distinct_range": 0, "num_eq": 3154611193005498723, "num_range": 0, "upper_bound": "-4713-11-24 00:00:00"}, {"distinct_range": 589.1977972081551, "num_eq": 8865687809300362433, "num_range": 200000, "upper_bound": "2000-08-22 18:00:59.000488"}, {"distinct_range": 1742275045297676800, "num_eq": 600, "num_range": 1742275045297676913, "upper_bound": "2003-06-02 21:25:30.000437"}, {"distinct_range": 6787544.9354684185, "num_eq": 0, "num_range": 10000000, "upper_bound": "2030-04-28 21:39:05.000602"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 4671946, "row_count": 8000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5338689, "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3521105, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000, "num_range": 0, "upper_bound": "315917117"}, {"distinct_range": 19.308454003309645, "num_eq": 50000000, "num_range": 20, "upper_bound": "1462925984"}, {"distinct_range": 0, "num_eq": 4363085051876749020, "num_range": 1956784203786338952, "upper_bound": "1468007443"}, {"distinct_range": 715.3501612542268, "num_eq": 99890289909492931, "num_range": 1000, "upper_bound": "1635241739"}, {"distinct_range": 235542871636060060, "num_eq": 0, "num_range": 1068891190728490379, "upper_bound": "4014382488"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7697189, "histo_buckets": [{"distinct_range": 0, "num_eq": 500000, "num_range": 0, "upper_bound": "1888053370"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7901118, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5237580, "histo_buckets": [{"distinct_range": 0, "num_eq": 30, "num_range": 0, "upper_bound": "-274007843701.7383798"}, {"distinct_range": 2704239465758141000, "num_eq": 6908075013622862896, "num_range": 3601815664987901201, "upper_bound": "-0.3468738474723014202"}, {"distinct_range": 0, "num_eq": 2960416214903685482, "num_range": 1000, "upper_bound": "42246309873638834.26"}, {"distinct_range": 5335462931125936000, "num_eq": 60000, "num_range": 5335462931125936465, "upper_bound": "Infinity"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 8000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000, "num_range": 0, "upper_bound": "1971-12-23 14:38:49.000838+00:00"}, {"distinct_range": 114728.27007006819, "num_eq": 90000000, "num_range": 5000000, "upper_bound": "1984-02-06 17:22:39.000171+00:00"}, {"distinct_range": 0, "num_eq": 90000000000, "num_range": 0, "upper_bound": "2002-04-21 05:29:58.000178+00:00"}, {"distinct_range": 90, "num_eq": 2723956932622979699, "num_range": 90, "upper_bound": "2012-05-27 08:55:28.000178+00:00"}, {"distinct_range": 0, "num_eq": 20, "num_range": 8570296875365310335, "upper_bound": "2020-05-07 22:56:17.000056+00:00"}, {"distinct_range": 2979941227712805400, "num_eq": 4000, "num_range": 4118975463448312311, "upper_bound": "2025-10-29 08:10:51.000848+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 97, "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5, "histo_buckets": [{"distinct_range": 0, "num_eq": 4343817506283546104, "num_range": 0, "upper_bound": "00001111001101110110001111010110101010001011"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 4386717439895961343, "upper_bound": "00010100110001010000001111111100000101101100"}, {"distinct_range": 3935211439410012700, "num_eq": 70, "num_range": 4730186487821003646, "upper_bound": "00010111111110100101100010001101111000110011"}, {"distinct_range": 205745808227075360, "num_eq": 10000, "num_range": 373556786085425299, "upper_bound": "00111011001100100101111110010010111010110000"}, {"distinct_range": 733376913015236100, "num_eq": 1000000, "num_range": 733376913015236033, "upper_bound": "01110100010110011010001011000110111011011001"}, {"distinct_range": 600, "num_eq": 7411332979364589366, "num_range": 600, "upper_bound": "01110101000111010001111110010110000111101101"}, {"distinct_range": 1315310236580966000, "num_eq": 3605908861748135517, "num_range": 2833699020470645145, "upper_bound": "01111000101111011110101010111000011000110111"}, {"distinct_range": 9000000, "num_eq": 100, "num_range": 9000000, "upper_bound": "11000111110010101011111001111001101111001101"}, {"distinct_range": 856907989407379200, "num_eq": 5948101500010856387, "num_range": 6946680098172344109, "upper_bound": "11001110110011100001110001111110100111111111"}], "histo_col_type": "BIT(44)", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35, "histo_col_type": "", "name": "__auto__", "null_count": 92, "row_count": 100}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 62, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 74, "histo_buckets": [{"distinct_range": 0, "num_eq": 300, "num_range": 0, "upper_bound": "-3965418.291811115619"}, {"distinct_range": 2684.1795070686526, "num_eq": 200, "num_range": 7000, "upper_bound": "8813.07439070099392"}, {"distinct_range": 500000, "num_eq": 50000, "num_range": 500000, "upper_bound": "2454806226038319544"}, {"distinct_range": 54742.9039393506, "num_eq": 10000000000, "num_range": 90000, "upper_bound": "6.585140320539900826E+27"}, {"distinct_range": 4000, "num_eq": 9000000000, "num_range": 4000, "upper_bound": "2.654159291235047592E+28"}, {"distinct_range": 1539823853303928800, "num_eq": 4378714123846973853, "num_range": 4105335197389571564, "upper_bound": "3.09653301414452037E+29"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 71, "row_count": 100}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 2610296845590003700, "num_eq": 8000000, "num_range": 4986879291099497189, "upper_bound": "\\u0017"}, {"distinct_range": 0, "num_eq": 5242375188503128165, "num_range": 6329874964454078930, "upper_bound": "5P\\u00124AYtL"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22, "histo_col_type": "", "name": "__auto__", "null_count": 75, "row_count": 100}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 79, "histo_col_type": "", "name": "__auto__", "null_count": 5, "row_count": 100}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 68, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "\\""}, {"distinct_range": 791085469863689000, "num_eq": 900000, "num_range": 7388155905842788149, "upper_bound": "2"}, {"distinct_range": 4000000000, "num_eq": 6000000, "num_range": 4000000000, "upper_bound": "J"}, {"distinct_range": 0, "num_eq": 3000000, "num_range": 900000, "upper_bound": "U"}, {"distinct_range": 57979.400841588606, "num_eq": 5438423419706998433, "num_range": 1000000, "upper_bound": "b"}, {"distinct_range": 5173125328333693000, "num_eq": 30000000, "num_range": 7590331053529684722, "upper_bound": "m"}, {"distinct_range": 0, "num_eq": 2000000000, "num_range": 500, "upper_bound": "\U00002603"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 50, "row_count": 100}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 58, "histo_buckets": [{"distinct_range": 0, "num_eq": 1738243539962177469, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 1612044915.691612, "num_eq": 1000000, "num_range": 50000000000, "upper_bound": "3"}, {"distinct_range": 0, "num_eq": 0, "num_range": 3110728734185741339, "upper_bound": "<"}, {"distinct_range": 800000000, "num_eq": 60000, "num_range": 800000000, "upper_bound": "["}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 100}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "infinity" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-01-01 06:12:43.459 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-01-01 06:12:43.459 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-01-01 06:12:43.465 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-01-01 06:12:43.485 UTC [89] LOG:  database system was shut down at 2021-01-01 06:12:43 UTC
        postgres_1    | 2021-01-01 06:12:43.490 UTC [1] LOG:  database system is ready to accept connections
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2561361&tab=buildLog) on [master@cee475331ca3629b503cd2e7c7919b72c98a5ca5](https://github.com/cockroachdb/cockroach/commits/cee475331ca3629b503cd2e7c7919b72c98a5ca5):

```
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        test_1        | --- FAIL: TestCompare (2.83s)
        test_1        |     --- FAIL: TestCompare/postgres (1.84s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.98s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 DECIMAL, col1_1 REGTYPE NULL, col1_2 FLOAT4 NOT NULL, col1_3 VARCHAR NOT NULL, col1_4 INT4 NOT NULL, col1_5 OID NOT NULL, col1_6 REGTYPE NOT NULL, col1_7 NAME NULL, col1_8 INTERVAL, col1_9 BOOL NULL, PRIMARY KEY (col1_4, col1_5 DESC, col1_3, col1_2 DESC), INDEX (col1_5 ASC, col1_2, col1_6 ASC, col1_1 DESC, col1_7 ASC, col1_8 ASC, col1_4) WHERE (table1.col1_2 = '+Inf':::FLOAT8) OR (NOT table1.col1_9), UNIQUE (col1_0 ASC, col1_9, col1_1 DESC, col1_4, col1_6 ASC, col1_8, col1_7, col1_3 DESC, col1_5 ASC) WHERE (((((table1.col1_7 >= e'\'') AND (table1.col1_2 >= 0.0)) AND (NOT table1.col1_9)) OR (table1.col1_4 = 1)) AND (table1.col1_3 > '')) AND (table1.col1_0 > Infinity), INDEX (col1_6 ASC, col1_3, col1_0 DESC, col1_1 DESC, col1_5 ASC, col1_9 DESC, col1_7 ASC, col1_8 ASC) WHERE ((((table1.col1_9 AND (table1.col1_7 = '"')) OR (table1.col1_2 != -Inf)) OR (table1.col1_0 <= Infinity)) AND (table1.col1_3 > e'\x00')) OR (table1.col1_4 = -128), UNIQUE (col1_4 ASC, col1_5 ASC, col1_0 DESC, col1_7 ASC, col1_8, col1_2, col1_1, col1_6, col1_3 ASC) STORING (col1_9) WHERE (((((NOT table1.col1_9) OR (table1.col1_4 <= (-128):::INT8)) AND (table1.col1_0 > (-1.234E+401):::DECIMAL)) OR (table1.col1_7 != 'X':::STRING)) AND (table1.col1_3 = 'X':::STRING)) AND (table1.col1_2 > 'NaN':::FLOAT8), INDEX (col1_7, col1_0 DESC, col1_4 DESC, col1_5) WHERE (table1.col1_2 = 1.401298464324817e-45) OR (table1.col1_3 > e'\x00'), FAMILY (col1_3, col1_7, col1_6, col1_4), FAMILY (col1_9, col1_1), FAMILY (col1_2), FAMILY (col1_0), FAMILY (col1_8, col1_5), UNIQUE (col1_2) STORING (col1_0, col1_1, col1_6, col1_7, col1_8, col1_9), UNIQUE (col1_4) STORING (col1_8, col1_9) WHERE (((((NOT table1.col1_9) OR (table1.col1_7 <= e'\x00')) OR (table1.col1_2 != NaN)) AND (table1.col1_3 > e'\x00')) AND (table1.col1_0 <= Infinity)) OR (table1.col1_4 >= 32767));
        test_1        |             CREATE TABLE table2 (col1_4 INT4 NOT NULL, col1_5 OID NOT NULL, col1_3 VARCHAR NOT NULL, col1_2 FLOAT4 NOT NULL, col2_4 BIT(49) NOT NULL, PRIMARY KEY (col1_4, col1_5 DESC, col1_3, col1_2 DESC), INDEX (col1_3 DESC) STORING (col2_4) WHERE ((table2.col1_2 <= 'NaN':::FLOAT8) OR (table2.col1_3 <= e'\'':::STRING)) AND (table2.col1_4 <= (-1):::INT8), UNIQUE (col1_5 DESC, col1_3 ASC) STORING (col2_4), INDEX (col1_3 ASC, col1_2) STORING (col2_4) WHERE ((table2.col1_2 = 3.4028234663852886e+38:::FLOAT8) AND (table2.col1_4 >= (-32768):::INT8)) OR (table2.col1_3 <= '"':::STRING), INDEX (col1_2 ASC, col1_5 DESC, col1_3, col1_4, col2_4 DESC) WHERE (table2.col1_3 >= 'X':::STRING) OR (table2.col1_4 <= (-32768):::INT8), FAMILY (col2_4), FAMILY (col1_2), FAMILY (col1_3), FAMILY (col1_4), FAMILY (col1_5)) INTERLEAVE IN PARENT table1 (col1_4, col1_5, col1_3, col1_2);
        test_1        |             CREATE TABLE table3 (col1_4 INT4 NOT NULL, col1_5 OID NOT NULL, col1_3 VARCHAR NOT NULL, col1_2 FLOAT4 NOT NULL, col3_4 INT2, col3_5 INT2, col3_6 REGTYPE NOT NULL, col3_7 CHAR, col3_8 "char", col3_9 TIMESTAMP NULL, col3_10 BIT(2) NOT NULL, col3_11 TIMETZ, col3_12 UUID NOT NULL, col3_13 "char" NOT NULL, col3_14 REGPROC NULL, col3_15 TIMESTAMPTZ NOT NULL, col3_16 REGPROCEDURE NULL, PRIMARY KEY (col1_4, col1_5 DESC, col1_3, col1_2 DESC, col3_6 DESC, col3_10 ASC, col3_14 DESC, col3_7, col3_16 ASC, col3_15 DESC), INDEX (col3_13, col1_2, col1_5 ASC) STORING (col3_5, col3_8, col3_9), INDEX (col1_2 DESC, col3_4 ASC, col3_5 ASC, col3_9 ASC, col3_10, col3_14 DESC, col1_4 ASC, col1_3 DESC), INDEX (col3_10 ASC, col3_15 DESC, col3_8, col1_5) STORING (col3_4, col3_5, col3_9, col3_11, col3_13) WHERE ((((table3.col3_4 >= (-1):::INT8) OR (table3.col1_4 >= 1:::INT8)) OR (table3.col3_9 >= '294276-12-31 23:59:59.999999':::TIMESTAMP)) OR (table3.col1_2 >= 3.4028234663852886e+38:::FLOAT8)) AND (table3.col3_11 >= '00:00:00+15:59:00':::TIMETZ), UNIQUE (col3_13 DESC, col1_2, col3_9 ASC, col3_7, col3_14, col3_11, col3_10 ASC, col1_5 ASC, col1_4 DESC, col1_3 ASC, col3_6 ASC, col3_16 ASC, col3_4, col3_15, col3_12 ASC, col3_5 DESC) WHERE (((((table3.col3_8 >= '':::STRING) OR (table3.col1_4 = 2147483647:::INT8)) AND (table3.col3_15 != '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) OR (table3.col1_3 >= e'\'':::STRING)) AND (table3.col3_11 < '00:00:00+15:59:00':::TIMETZ)) OR (table3.col3_5 <= (-128):::INT8), FAMILY (col3_13, col3_11), FAMILY (col3_12), FAMILY (col3_7, col3_15), FAMILY (col3_6, col1_2), FAMILY (col3_5, col1_5, col1_4), FAMILY (col3_16, col3_14), FAMILY (col3_8, col3_9, col1_3, col3_10), FAMILY (col3_4)) INTERLEAVE IN PARENT table1 (col1_4, col1_5, col1_3, col1_2);
        test_1        |             CREATE TABLE table4 (col4_0 TIMETZ NULL, col4_1 BYTES NOT NULL, col4_2 INET NOT NULL, col4_3 INT2 NOT NULL, col4_4 STRING, col4_5 INET NOT NULL, col4_6 TIMESTAMPTZ NOT NULL, col4_7 INT8 NOT NULL, col4_8 TIMESTAMP, col4_9 INT8 NULL, col4_10 INTERVAL NULL, col4_11 VARCHAR NOT NULL, PRIMARY KEY (col4_7 ASC, col4_1, col4_3, col4_5 ASC), INDEX (col4_8, col4_2 DESC, col4_9 DESC, col4_7 ASC, col4_4, col4_11 DESC, col4_5, col4_6 ASC, col4_0, col4_1 DESC, col4_3 DESC, col4_10) WHERE (((table4.col4_7 < 2147483647:::INT8) OR (table4.col4_4 < e'\'':::STRING)) AND (table4.col4_3 >= 0:::INT8)) AND (table4.col4_0 != '24:00:00-15:59:00':::TIMETZ), UNIQUE (col4_3 DESC, col4_7, col4_0, col4_8 DESC, col4_5 DESC, col4_6 DESC, col4_10 ASC, col4_4 DESC, col4_2) WHERE ((((((((table4.col4_11 > '':::STRING) AND (table4.col4_0 < '00:00:00+15:59:00':::TIMETZ)) AND (table4.col4_4 <= e'\'':::STRING)) AND (table4.col4_3 >= (-1):::INT8)) AND (table4.col4_7 <= 2147483647:::INT8)) AND (table4.col4_1 < e'\\x22':::BYTES)) OR (table4.col4_9 = 1:::INT8)) OR (table4.col4_8 <= '294276-12-31 23:59:59.999999':::TIMESTAMP)) OR (table4.col4_6 <= '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ), FAMILY (col4_11, col4_5), FAMILY (col4_8), FAMILY (col4_3), FAMILY (col4_0, col4_6, col4_9, col4_2, col4_7, col4_4, col4_10), FAMILY (col4_1));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22, "histo_col_type": "", "name": "__auto__", "null_count": 8, "row_count": 60}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9, "histo_col_type": "", "name": "__auto__", "null_count": 38, "row_count": 60}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13, "histo_buckets": [{"distinct_range": 0, "num_eq": 500, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 260231523482693702, "num_range": 0, "upper_bound": "\\u000eb\\u0005t.H\\u0015(u"}, {"distinct_range": 0, "num_eq": 900000, "num_range": 8150172223026895489, "upper_bound": "8pN"}, {"distinct_range": 6837521.847012548, "num_eq": 10000000, "num_range": 10000000, "upper_bound": "d"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 50, "row_count": 60}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1, "histo_col_type": "", "name": "__auto__", "null_count": 17, "row_count": 60}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 31, "histo_buckets": [{"distinct_range": 0, "num_eq": 400, "num_range": 0, "upper_bound": "1090574971"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 10000000, "upper_bound": "3556415578"}, {"distinct_range": 5461681.039955009, "num_eq": 500, "num_range": 9000000, "upper_bound": "4116186027"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3, "histo_buckets": [{"distinct_range": 0, "num_eq": 3513985781812760939, "num_range": 0, "upper_bound": "-6.497533433894887608E+36"}, {"distinct_range": 239164.8445955267, "num_eq": 0, "num_range": 800000, "upper_bound": "-4.796691960455296108E+33"}, {"distinct_range": 4000, "num_eq": 70000, "num_range": 4000, "upper_bound": "-2.696285764840032805E+22"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 29, "row_count": 60}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000000, "num_range": 0, "upper_bound": "-128"}, {"distinct_range": 500, "num_eq": 7132086166278353007, "num_range": 500, "upper_bound": "70380096"}, {"distinct_range": 979182.1965024053, "num_eq": 3560667668933396693, "num_range": 5000000, "upper_bound": "446230920"}, {"distinct_range": 0, "num_eq": 900000000, "num_range": 20000, "upper_bound": "1414825839"}, {"distinct_range": 0, "num_eq": 7264191035495957112, "num_range": 2755209988009347530, "upper_bound": "1855530397"}, {"distinct_range": 0, "num_eq": 400000, "num_range": 0, "upper_bound": "2051999366"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 596914631996346272, "upper_bound": "748117348"}, {"distinct_range": 400000, "num_eq": 10000000000, "num_range": 400000, "upper_bound": "1549861623"}, {"distinct_range": 2208406439677304800, "num_eq": 60000000, "num_range": 2852185480779693411, "upper_bound": "2407519282"}, {"distinct_range": 0, "num_eq": 8166743381479621084, "num_range": 8000000000, "upper_bound": "3626351052"}, {"distinct_range": 13367501604645402, "num_eq": 1912923770369657271, "num_range": 8400936568389700243, "upper_bound": "3758116350"}, {"distinct_range": 0, "num_eq": 6843870744155820312, "num_range": 10000000, "upper_bound": "3896960752"}, {"distinct_range": 897247348649288400, "num_eq": 10000, "num_range": 1470715059252406486, "upper_bound": "4173835066"}, {"distinct_range": 6361.277832133132, "num_eq": 400000000, "num_range": 8000, "upper_bound": "4223009307"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 60}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2049746, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 2468807552593864000, "num_eq": 80, "num_range": 2468807552593864447, "upper_bound": "191825706"}, {"distinct_range": 0, "num_eq": 1420571033630293844, "num_range": 5314619441347374271, "upper_bound": "2847070531"}, {"distinct_range": 0, "num_eq": 1693763752002814348, "num_range": 10000, "upper_bound": "4291634146"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1008223, "histo_buckets": [{"distinct_range": 0, "num_eq": 5410594541597707233, "num_range": 0, "upper_bound": "\\u0007c(59t\\u0004"}, {"distinct_range": 3507807746750929000, "num_eq": 7903032310115012177, "num_range": 3524198069782596457, "upper_bound": "\\u00113\\u0016 7"}, {"distinct_range": 50000000, "num_eq": 1000000, "num_range": 50000000, "upper_bound": "\\""}, {"distinct_range": 90000000000, "num_eq": 30, "num_range": 90000000000, "upper_bound": "%E\\u0004["}, {"distinct_range": 0, "num_eq": 7977512877857599714, "num_range": 1102664540791938983, "upper_bound": "X"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2043155, "histo_buckets": [{"distinct_range": 0, "num_eq": 7164698949095052413, "num_range": 0, "upper_bound": "NaN"}, {"distinct_range": 9000, "num_eq": 9051133330180476375, "num_range": 9000, "upper_bound": "-0.324604868888855"}, {"distinct_range": 1167185251.9164793, "num_eq": 7829298381558846113, "num_range": 3000000000, "upper_bound": "-0.0905175432562828"}, {"distinct_range": 2154780137311451400, "num_eq": 8353110701032362180, "num_range": 2154780137311451279, "upper_bound": "0.2709297835826874"}, {"distinct_range": 0, "num_eq": 6096278359954720693, "num_range": 3534151730385613923, "upper_bound": "0.8245680332183838"}, {"distinct_range": 1488728876520661000, "num_eq": 7362827663431490716, "num_range": 5271072902027161219, "upper_bound": "0.8949154615402222"}, {"distinct_range": 0, "num_eq": 9006120798925988908, "num_range": 8021427635729178777, "upper_bound": "1.5364630222320557"}, {"distinct_range": 4748403605013096000, "num_eq": 4020585461427618919, "num_range": 4748403605013096941, "upper_bound": "3.4028234663852886e+38"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 654286, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1721135, "histo_buckets": [{"distinct_range": 0, "num_eq": 7656500864878293949, "num_range": 0, "upper_bound": "-1520544564"}, {"distinct_range": 0, "num_eq": 5736048754112017224, "num_range": 3387539023977837533, "upper_bound": "-1104914184"}, {"distinct_range": 2498771704430672400, "num_eq": 6643706932669675774, "num_range": 4196360968758047779, "upper_bound": "-897729021"}, {"distinct_range": 4766796128590973000, "num_eq": 200000, "num_range": 4766796128590972499, "upper_bound": "-295654173"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 8000, "upper_bound": "-32768"}, {"distinct_range": 0, "num_eq": 30000000000, "num_range": 0, "upper_bound": "166411396"}, {"distinct_range": 0, "num_eq": 300, "num_range": 0, "upper_bound": "1150163368"}, {"distinct_range": 0, "num_eq": 248065747152583345, "num_range": 70000, "upper_bound": "1363504049"}, {"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": "1878066840"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 3000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42, "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000, "num_range": 0, "upper_bound": "00"}, {"distinct_range": 564642900609447740, "num_eq": 1658154602869132813, "num_range": 7955486395587327174, "upper_bound": "01"}, {"distinct_range": 3417167102145701400, "num_eq": 5074219130222430082, "num_range": 3425844507560140419, "upper_bound": "10"}], "histo_col_type": "BIT(2)", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000345723999563910, "num_range": 0, "upper_bound": ""}, {"distinct_range": 1761872600487523600, "num_eq": 700000000, "num_range": 1761872600487523544, "upper_bound": "\\u0019"}, {"distinct_range": 0, "num_eq": 3172654989047018467, "num_range": 10000000000, "upper_bound": "7"}, {"distinct_range": 7000000000, "num_eq": 60000000000, "num_range": 7000000000, "upper_bound": "F"}, {"distinct_range": 9.406427548842798, "num_eq": 2976026366203184314, "num_range": 100, "upper_bound": "I"}, {"distinct_range": 0, "num_eq": 0, "num_range": 5000000, "upper_bound": "X"}, {"distinct_range": 0, "num_eq": 60000, "num_range": 50000000, "upper_bound": "h"}, {"distinct_range": 262534652.25930932, "num_eq": 500000, "num_range": 500000000, "upper_bound": "~"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45, "histo_col_type": "", "name": "__auto__", "null_count": 42, "row_count": 60}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3, "histo_buckets": [{"distinct_range": 0, "num_eq": 7861151249978603750, "num_range": 0, "upper_bound": "-2035978318"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 697038332527511751, "upper_bound": "-2035275270"}, {"distinct_range": 80000, "num_eq": 30000000000, "num_range": 80000, "upper_bound": "-1774989091"}, {"distinct_range": 114499115350167000, "num_eq": 200, "num_range": 406718949649889347, "upper_bound": "-777016972"}, {"distinct_range": 2030464418740587000, "num_eq": 10000, "num_range": 2030464418740586933, "upper_bound": "-629897452"}, {"distinct_range": 30000000000, "num_eq": 40000, "num_range": 30000000000, "upper_bound": "-142535758"}, {"distinct_range": 0, "num_eq": 5000000000, "num_range": 30000000000, "upper_bound": "-32768"}, {"distinct_range": 0, "num_eq": 1239741826259043460, "num_range": 5000000000, "upper_bound": "852635096"}, {"distinct_range": 5000, "num_eq": 6343079292898456005, "num_range": 5000, "upper_bound": "1839054225"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 42, "row_count": 60}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_col_type": "", "name": "__auto__", "null_count": 30, "row_count": 60}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 48, "histo_col_type": "", "name": "__auto__", "null_count": 59, "row_count": 60}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7, "histo_col_type": "", "name": "__auto__", "null_count": 48, "row_count": 60}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 48, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 47, "histo_col_type": "", "name": "__auto__", "null_count": 50, "row_count": 60}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21, "histo_col_type": "", "name": "__auto__", "null_count": 14, "row_count": 60}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 52, "row_count": 60}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 234, "histo_col_type": "", "name": "__auto__", "null_count": 121, "row_count": 300}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 104, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 79, "histo_col_type": "", "name": "__auto__", "null_count": 221, "row_count": 300}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 200, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 55, "histo_buckets": [{"distinct_range": 0, "num_eq": 60, "num_range": 0, "upper_bound": "-6764115523292564193"}, {"distinct_range": 0, "num_eq": 5714435269480478629, "num_range": 7372635348841286977, "upper_bound": "-5170702864218204737"}, {"distinct_range": 0, "num_eq": 4842387378902778245, "num_range": 1000000000, "upper_bound": "350005466223585266"}, {"distinct_range": 6000000, "num_eq": 10000, "num_range": 6000000, "upper_bound": "4354152763991559012"}, {"distinct_range": 476441121523850000, "num_eq": 7000000000, "num_range": 2068202372972486766, "upper_bound": "6405648458889518551"}, {"distinct_range": 0, "num_eq": 10, "num_range": 316636783209369332, "upper_bound": "7890689855885473081"}, {"distinct_range": 700000, "num_eq": 4475103960731568842, "num_range": 700000, "upper_bound": "8663837832294829143"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 300}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 174, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000, "num_range": 0, "upper_bound": "1982-04-22 09:44:31.000504"}, {"distinct_range": 9.44174192426904, "num_eq": 5000000000, "num_range": 10, "upper_bound": "1989-12-18 05:34:17.000445"}, {"distinct_range": 2518103767817710600, "num_eq": 70000000000, "num_range": 2518103767817710499, "upper_bound": "1992-04-06 22:24:26.000537"}, {"distinct_range": 0, "num_eq": 8461180631033854427, "num_range": 20000000000, "upper_bound": "1992-12-05 07:46:26.000012"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 2000, "upper_bound": "2019-04-30 10:37:38.000061"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 9000, "upper_bound": "2022-11-14 09:03:42.000962"}, {"distinct_range": 1051085734421234700, "num_eq": 400000, "num_range": 1824178136335996183, "upper_bound": "2030-12-03 16:15:08.00089"}, {"distinct_range": 690445426369546400, "num_eq": 4523390208899146596, "num_range": 4435751994337728954, "upper_bound": "294276-12-31 23:59:59.999999"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 82, "row_count": 300}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 298, "histo_col_type": "", "name": "__auto__", "null_count": 216, "row_count": 300}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 75, "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 300}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 204, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 90, "histo_col_type": "", "name": "__auto__", "null_count": 96, "row_count": 300}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 237, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300}]':::JSONB;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col1_2) REFERENCES table1 (col1_2) ON DELETE RESTRICT ON UPDATE RESTRICT;
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col3_5) REFERENCES table1 (col1_4);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('vrmky', 'ntuys');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('l', 'iojta');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 857861407068740418, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2760147578427512848}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2571870531085124643, "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 2760147578427512848}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1631948328706606781, "histo_buckets": [{"distinct_range": 0, "num_eq": 6368525650468197031, "num_range": 0, "upper_bound": ""}, {"distinct_range": 13214.50912152254, "num_eq": 5166372375381927354, "num_range": 20000, "upper_bound": "\\u0000"}, {"distinct_range": 7742220828438111000, "num_eq": 600, "num_range": 7742220828438111594, "upper_bound": "\\u0010\\u000f\\u00173m<l\\u0014\\u0006"}, {"distinct_range": 2747649.7918173163, "num_eq": 80000, "num_range": 10000000, "upper_bound": "5&fDLDG5P"}, {"distinct_range": 3919797377890860500, "num_eq": 8000000, "num_range": 6658909045175507364, "upper_bound": "FxS\\u000c\\u0017\\t\\u0002K"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 1589637148931134286, "row_count": 2760147578427512848}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2658271476107511442, "histo_col_type": "", "name": "__auto__", "null_count": 1201435575067301215, "row_count": 2760147578427512848}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 941517575146877462, "histo_col_type": "", "name": "__auto__", "null_count": 1159242051892572331, "row_count": 2760147578427512848}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2515726449365370159, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000, "num_range": 0, "upper_bound": "0.7883323431015015"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 2760147578427512848}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1741795687207228010, "histo_buckets": [{"distinct_range": 0, "num_eq": 4678134846694518928, "num_range": 0, "upper_bound": "2091059183"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 2760147578427512848}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2447916278582430313, "histo_col_type": "", "name": "__auto__", "null_count": 1332812029008491195, "row_count": 2760147578427512848}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 115248835022046158, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000000, "num_range": 0, "upper_bound": "-25377547515961.21704"}, {"distinct_range": 0, "num_eq": 500000000, "num_range": 70000, "upper_bound": "-3958.876829291762002"}, {"distinct_range": 700, "num_eq": 9000000, "num_range": 700, "upper_bound": "5.748511688437738786E+31"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 465524592252628469, "row_count": 2760147578427512848}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1472380648464627110, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000000, "num_range": 0, "upper_bound": "-324954491"}, {"distinct_range": 0, "num_eq": 2020430635232255876, "num_range": 20000, "upper_bound": "-214523163"}, {"distinct_range": 822943277695615100, "num_eq": 5918769478951542308, "num_range": 825428093479342906, "upper_bound": "-2804291"}, {"distinct_range": 42869790015084510, "num_eq": 3104863352756561209, "num_range": 44124244492418481, "upper_bound": "56959492"}, {"distinct_range": 0, "num_eq": 7343863234168089079, "num_range": 8119020752332866282, "upper_bound": "395537344"}, {"distinct_range": 7302483816125658000, "num_eq": 7000000000, "num_range": 7402713406818567482, "upper_bound": "484407061"}, {"distinct_range": 0, "num_eq": 5941179245671290417, "num_range": 0, "upper_bound": "958784872"}, {"distinct_range": 50000000, "num_eq": 20, "num_range": 50000000, "upper_bound": "1602707700"}, {"distinct_range": 95903.71280988955, "num_eq": 6750257668983770845, "num_range": 100000, "upper_bound": "1660196105"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 2760147578427512848}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 435593, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000, "num_range": 0, "upper_bound": "-1918952427"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 6995401932559729009, "upper_bound": "-1067249818"}, {"distinct_range": 1528391490235612000, "num_eq": 2122541868967752752, "num_range": 1730931493623769956, "upper_bound": "-1"}, {"distinct_range": 3737815936.5956006, "num_eq": 0, "num_range": 5000000000, "upper_bound": "337434487"}, {"distinct_range": 14650.326660223103, "num_eq": 271237423911944647, "num_range": 90000, "upper_bound": "976342434"}, {"distinct_range": 5815360260260380000, "num_eq": 200, "num_range": 5815360260260379337, "upper_bound": "1999026081"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6966885, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000000, "num_range": 0, "upper_bound": "24970877"}, {"distinct_range": 6102434188226375000, "num_eq": 3843412915888586395, "num_range": 6102434188226374392, "upper_bound": "276348132"}, {"distinct_range": 38663.84281518099, "num_eq": 216373982996469768, "num_range": 100000, "upper_bound": "1135477641"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 2810482617364425089, "upper_bound": "1466795948"}, {"distinct_range": 0, "num_eq": 600, "num_range": 8208935048142801579, "upper_bound": "2351099365"}, {"distinct_range": 1649741605.2880607, "num_eq": 5285328620257118910, "num_range": 3000000000, "upper_bound": "3041868853"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8784416, "histo_buckets": [{"distinct_range": 0, "num_eq": 7419769833221838257, "num_range": 0, "upper_bound": "\\u0004"}, {"distinct_range": 0, "num_eq": 5996426350371274630, "num_range": 0, "upper_bound": "\\u000e\'aLsi"}, {"distinct_range": 2496964968366927400, "num_eq": 40000, "num_range": 8331869480654897726, "upper_bound": "C"}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 40000, "upper_bound": "T\\u0018"}, {"distinct_range": 2154956778640196000, "num_eq": 0, "num_range": 2154956778640195991, "upper_bound": "V/\\u001bc6\\u000b]W"}, {"distinct_range": 0, "num_eq": 700, "num_range": 1336850653378086019, "upper_bound": "r\\u0003\\u0011"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8733448, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000, "num_range": 0, "upper_bound": "-0.6581136584281921"}, {"distinct_range": 0, "num_eq": 4172080596811782445, "num_range": 0, "upper_bound": "0.09361860901117325"}, {"distinct_range": 50000, "num_eq": 100000000, "num_range": 50000, "upper_bound": "0.4956322908401489"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 1997228703915752084, "upper_bound": "1.0890671014785767"}, {"distinct_range": 0, "num_eq": 9083634957563470130, "num_range": 3000000, "upper_bound": "1.114502191543579"}, {"distinct_range": 0, "num_eq": 50, "num_range": 2013613938202194747, "upper_bound": "1.2888150215148926"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 4019181960577725958, "upper_bound": "1.3339167833328247"}, {"distinct_range": 2427281159596836400, "num_eq": 70000, "num_range": 3735845328046155103, "upper_bound": "1.4328579902648926"}, {"distinct_range": 3963168700332697600, "num_eq": 1136699262893788925, "num_range": 3963168700332697739, "upper_bound": "+Inf"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7601330, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 41101, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16307, "histo_col_type": "", "name": "__auto__", "null_count": 10714, "row_count": 80000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 79764, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 60532, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 73143, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": "-2.3359427452087402"}, {"distinct_range": 8400997.47074936, "num_eq": 50000, "num_range": 30000000, "upper_bound": "-1.0924553871154785"}, {"distinct_range": 10000000, "num_eq": 10000000000, "num_range": 10000000, "upper_bound": "0.35418030619621277"}, {"distinct_range": 2.4623196486486747, "num_eq": 80000, "num_range": 70, "upper_bound": "1.1085187196731567"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 71359, "histo_buckets": [{"distinct_range": 0, "num_eq": 4512265102233473484, "num_range": 0, "upper_bound": "00"}, {"distinct_range": 0, "num_eq": 1325460038872907748, "num_range": 2127649892593235334, "upper_bound": "01"}, {"distinct_range": 0, "num_eq": 6191576772679777024, "num_range": 3975921428297070788, "upper_bound": "11"}], "histo_col_type": "BIT(2)", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 66183, "histo_col_type": "", "name": "__auto__", "null_count": 9245, "row_count": 80000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17803, "histo_buckets": [{"distinct_range": 0, "num_eq": 7951254021129515050, "num_range": 0, "upper_bound": "2077373718"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67594, "histo_col_type": "", "name": "__auto__", "null_count": 68121, "row_count": 80000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 66703, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000000, "num_range": 0, "upper_bound": "\\u001c"}, {"distinct_range": 100, "num_eq": 8928675155751605441, "num_range": 100, "upper_bound": ";"}, {"distinct_range": 0, "num_eq": 4211500028852200763, "num_range": 4000, "upper_bound": "D"}, {"distinct_range": 30000000, "num_eq": 2766942046802048687, "num_range": 30000000, "upper_bound": "M"}, {"distinct_range": 0, "num_eq": 0, "num_range": 2000000, "upper_bound": "`"}, {"distinct_range": 2873281128482172000, "num_eq": 4899660166838397119, "num_range": 2873281128482172028, "upper_bound": "\x7f"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2200, "histo_col_type": "", "name": "__auto__", "null_count": 43032, "row_count": 80000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24783, "histo_col_type": "", "name": "__auto__", "null_count": 79364, "row_count": 80000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29653, "histo_col_type": "", "name": "__auto__", "null_count": 57300, "row_count": 80000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 56792, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 49023, "histo_col_type": "", "name": "__auto__", "null_count": 59695, "row_count": 80000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 41833, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 68142, "histo_col_type": "", "name": "__auto__", "null_count": 46010, "row_count": 80000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1305658164292464691, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8603974705377003998}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1365498952869427494, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8603974705377003998}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2513356194009311202, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000, "num_range": 0, "upper_bound": "-13491"}, {"distinct_range": 6393866122193536000, "num_eq": 7657732296094908148, "num_range": 8571124357499801462, "upper_bound": "-6939"}, {"distinct_range": 0, "num_eq": 500000, "num_range": 5000, "upper_bound": "127"}, {"distinct_range": 0, "num_eq": 5792716353145864604, "num_range": 399893397642650992, "upper_bound": "1869"}, {"distinct_range": 0, "num_eq": 5205986720247230821, "num_range": 990453829348308019, "upper_bound": "9938"}, {"distinct_range": 3000000000, "num_eq": 60, "num_range": 3000000000, "upper_bound": "18006"}, {"distinct_range": 0, "num_eq": 9016092093385188412, "num_range": 1000000, "upper_bound": "20851"}, {"distinct_range": 10, "num_eq": 1000000000, "num_range": 10, "upper_bound": "28103"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 8603974705377003998}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3787068113193824239, "histo_col_type": "", "name": "__auto__", "null_count": 3625093666628489127, "row_count": 8603974705377003998}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5591065101683923242, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000, "num_range": 0, "upper_bound": "1979-07-09 05:28:15.000643"}, {"distinct_range": 94.50645882174155, "num_eq": 2239471076825931366, "num_range": 100, "upper_bound": "1984-08-24 20:16:05.000083"}, {"distinct_range": 27234380.198811244, "num_eq": 4272017871803508036, "num_range": 40000000, "upper_bound": "1989-05-10 06:13:39.000467"}, {"distinct_range": 5009699827405186000, "num_eq": 7468126211358126915, "num_range": 5009699827405186378, "upper_bound": "1999-07-01 05:08:55.000668"}, {"distinct_range": 10000000, "num_eq": 600000, "num_range": 10000000, "upper_bound": "2005-09-24 07:31:11.00068"}, {"distinct_range": 0, "num_eq": 834235773987041981, "num_range": 3563512627034975622, "upper_bound": "2009-07-14 21:48:55.000848"}, {"distinct_range": 2000, "num_eq": 4071289334120422774, "num_range": 2000, "upper_bound": "2013-11-11 19:21:23.000859"}, {"distinct_range": 1268285.0685664015, "num_eq": 64783611094452855, "num_range": 6000000, "upper_bound": "2021-09-30 04:53:49.000359"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 7330803630814945763, "row_count": 8603974705377003998}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4892011723373875057, "histo_col_type": "", "name": "__auto__", "null_count": 8515540515342283113, "row_count": 8603974705377003998}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7983633159465166546, "histo_col_type": "", "name": "__auto__", "null_count": 6673893055256455794, "row_count": 8603974705377003998}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4615420922230028613, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8603974705377003998}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5770106905678095074, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8603974705377003998}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8006774877990759485, "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 8603974705377003998}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5902739726387963408, "histo_col_type": "", "name": "__auto__", "null_count": 7112880967619585134, "row_count": 8603974705377003998}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2885763791452378325, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8603974705377003998}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1466611, "histo_buckets": [{"distinct_range": 0, "num_eq": 5084069443536490095, "num_range": 0, "upper_bound": "-0.8957257866859436"}, {"distinct_range": 3484787608774534000, "num_eq": 0, "num_range": 6586132397386085774, "upper_bound": "-0.3779515027999878"}, {"distinct_range": 2830942360429893000, "num_eq": 5765837988300067373, "num_range": 3428459612810623092, "upper_bound": "-0.27892354130744934"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 1788632409561381797, "upper_bound": "1.6779624223709106"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 335558, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000, "num_range": 0, "upper_bound": "-1918496340"}, {"distinct_range": 0, "num_eq": 366356173474193893, "num_range": 0, "upper_bound": "490668809"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 80, "upper_bound": "2035689904"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1030426, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 200, "num_eq": 3621067350984504942, "num_range": 200, "upper_bound": "663206219"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "1097631507"}, {"distinct_range": 10000000000, "num_eq": 500, "num_range": 10000000000, "upper_bound": "1414971363"}, {"distinct_range": 80000000, "num_eq": 100000000000, "num_range": 80000000, "upper_bound": "1770817463"}, {"distinct_range": 0, "num_eq": 6342324682529906916, "num_range": 0, "upper_bound": "2066185703"}, {"distinct_range": 0, "num_eq": 0, "num_range": 3000000, "upper_bound": "2472310290"}, {"distinct_range": 0, "num_eq": 4631951148365125963, "num_range": 90, "upper_bound": "2528765278"}, {"distinct_range": 700000, "num_eq": 1517297953346383745, "num_range": 700000, "upper_bound": "3811545915"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39777, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": "-444140.2566922874873"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 90000000000, "upper_bound": "-7093.89258574939642"}, {"distinct_range": 2062395774518901200, "num_eq": 1498079715938767254, "num_range": 3068962259534650866, "upper_bound": "7784049336442991.745"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 1684684, "row_count": 4000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 77260, "histo_col_type": "", "name": "__auto__", "null_count": 1482424, "row_count": 4000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 972399, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1305529, "histo_buckets": [{"distinct_range": 0, "num_eq": 6683644710752567931, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 10000000000, "num_eq": 70000000, "num_range": 10000000000, "upper_bound": "1440032886"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3156762, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 5166597892.754112, "num_eq": 300, "num_range": 7000000000, "upper_bound": "\\u0005\\u000f\\u000b"}, {"distinct_range": 0, "num_eq": 3667979398834503763, "num_range": 400000, "upper_bound": "\'"}, {"distinct_range": 4311230195466981400, "num_eq": 80, "num_range": 4311230195466981625, "upper_bound": "c\\u0018rn\x7fX;KN"}, {"distinct_range": 0, "num_eq": 40, "num_range": 2926954760385781764, "upper_bound": "ldM1Y-V\\u0002\\u000f"}, {"distinct_range": 0, "num_eq": 90000, "num_range": 40000000000, "upper_bound": "}V/bo\x7fCR6"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 1460582, "row_count": 4000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1160689, "histo_col_type": "", "name": "__auto__", "null_count": 348130, "row_count": 4000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2034572, "histo_col_type": "", "name": "__auto__", "null_count": 3397253, "row_count": 4000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9564982996, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4301351509, "histo_buckets": [{"distinct_range": 0, "num_eq": 400, "num_range": 0, "upper_bound": "-1629290488"}, {"distinct_range": 80, "num_eq": 1716416587726014869, "num_range": 80, "upper_bound": "-1083372210"}, {"distinct_range": 1743539568765711000, "num_eq": 40, "num_range": 1743539568765711178, "upper_bound": "-385493138"}, {"distinct_range": 2349488553.9176526, "num_eq": 1403205293436275435, "num_range": 9000000000, "upper_bound": "-347075280"}, {"distinct_range": 5637548980897109000, "num_eq": 5000000, "num_range": 5637548980897108499, "upper_bound": "-32768"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 0, "upper_bound": "1887504877"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27396989179, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "771889213"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 4327823011455302113, "upper_bound": "2353925870"}, {"distinct_range": 4671685447143871000, "num_eq": 7320833841033216745, "num_range": 8381733016832953518, "upper_bound": "2452730886"}, {"distinct_range": 0, "num_eq": 3681834465948946932, "num_range": 5018585551659857535, "upper_bound": "3392321893"}, {"distinct_range": 0, "num_eq": 8003132908318966583, "num_range": 2000, "upper_bound": "3917460410"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5611641203, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": "$\\u000f"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9296298397, "histo_buckets": [{"distinct_range": 0, "num_eq": 4696896572440777511, "num_range": 0, "upper_bound": "-0.9446153044700623"}, {"distinct_range": 0, "num_eq": 900, "num_range": 801098166613778509, "upper_bound": "-0.9215059876441956"}, {"distinct_range": 4652704.243408435, "num_eq": 713786626340254354, "num_range": 7000000, "upper_bound": "-0.9185828566551208"}, {"distinct_range": 3000000000, "num_eq": 1000000000, "num_range": 3000000000, "upper_bound": "-0.5696486830711365"}, {"distinct_range": 993.3540545591818, "num_eq": 900, "num_range": 1000, "upper_bound": "2.545715808868408"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 30000000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 473349, "histo_buckets": [{"distinct_range": 0, "num_eq": 997587624295316947, "num_range": 0, "upper_bound": "-1.3981027603149414"}, {"distinct_range": 9020579069785050000, "num_eq": 30000, "num_range": 9020579069785050281, "upper_bound": "-0.8221862316131592"}, {"distinct_range": 716826948725688000, "num_eq": 2072981823409506308, "num_range": 6664170606892267901, "upper_bound": "+Inf"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1892359, "histo_col_type": "", "name": "__auto__", "null_count": 614404, "row_count": 2000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1717172, "histo_col_type": "", "name": "__auto__", "null_count": 253007, "row_count": 2000000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 740148, "histo_col_type": "", "name": "__auto__", "null_count": 1080773, "row_count": 2000000}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 798726, "histo_col_type": "", "name": "__auto__", "null_count": 92770, "row_count": 2000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1440753, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 636028, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1493343, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1411726, "histo_col_type": "", "name": "__auto__", "null_count": 1410174, "row_count": 2000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1701190, "histo_col_type": "", "name": "__auto__", "null_count": 666367, "row_count": 2000000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1398106, "histo_buckets": [{"distinct_range": 0, "num_eq": 300000, "num_range": 0, "upper_bound": "7"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 221953, "histo_col_type": "", "name": "__auto__", "null_count": 179759, "row_count": 2000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1118878, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "-902065306"}, {"distinct_range": 8166030432440609000, "num_eq": 2977552660607527340, "num_range": 8166030432440609214, "upper_bound": "-645173329"}, {"distinct_range": 50000000, "num_eq": 10000000000, "num_range": 50000000, "upper_bound": "-382671179"}, {"distinct_range": 0, "num_eq": 2657533976332233166, "num_range": 60, "upper_bound": "-269218697"}, {"distinct_range": 6714941709721436000, "num_eq": 1000000, "num_range": 7792095273372183397, "upper_bound": "1241875911"}, {"distinct_range": 0, "num_eq": 5557338961551858495, "num_range": 1095916330417300078, "upper_bound": "1294055534"}, {"distinct_range": 0, "num_eq": 5000000, "num_range": 8885037855294329018, "upper_bound": "1352911357"}, {"distinct_range": 500000000, "num_eq": 7947973556009549163, "num_range": 500000000, "upper_bound": "2006436246"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 7719276649488415952, "upper_bound": "2147483647"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1277734, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1909038, "histo_col_type": "", "name": "__auto__", "null_count": 853567, "row_count": 2000000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1764672, "histo_buckets": [{"distinct_range": 0, "num_eq": 5465901745191572220, "num_range": 0, "upper_bound": "00"}, {"distinct_range": 4219288892208050700, "num_eq": 5364038394190099227, "num_range": 5094958826206554825, "upper_bound": "01"}, {"distinct_range": 0, "num_eq": 0, "num_range": 2032670832446812543, "upper_bound": "11"}], "histo_col_type": "BIT(2)", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 266869, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2585855540318299977, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000000, "num_range": 0, "upper_bound": "0001-01-01 00:00:00"}, {"distinct_range": 3856054993069302300, "num_eq": 100, "num_range": 8033446301650151105, "upper_bound": "1972-05-24 16:43:52.000889"}, {"distinct_range": 60000, "num_eq": 3000000000, "num_range": 60000, "upper_bound": "1974-12-22 03:12:45.00048"}, {"distinct_range": 40000000, "num_eq": 3000000000, "num_range": 40000000, "upper_bound": "1978-12-13 07:34:41.000576"}, {"distinct_range": 0, "num_eq": 500000, "num_range": 0, "upper_bound": "2006-05-16 21:28:46.000471"}, {"distinct_range": 0, "num_eq": 5122085553994097691, "num_range": 6000, "upper_bound": "294276-12-31 23:59:59.999999"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 3718514083219609305, "row_count": 9188929641994862252}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8378549422144367791, "histo_col_type": "", "name": "__auto__", "null_count": 3235272924110234000, "row_count": 9188929641994862252}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4210807534820809076, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9188929641994862252}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2057874699256786296, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9188929641994862252}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2311965159195354876, "histo_col_type": "", "name": "__auto__", "null_count": 6536152366190471335, "row_count": 9188929641994862252}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2060234507217166906, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9188929641994862252}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1343227612834555674, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9188929641994862252}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5712352934799917017, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "-6286043100234455818"}, {"distinct_range": 123584353003416240, "num_eq": 6000000000, "num_range": 123584353003416242, "upper_bound": "7659775364956713188"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 9188929641994862252}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4699392573656620522, "histo_col_type": "", "name": "__auto__", "null_count": 8530265365824187609, "row_count": 9188929641994862252}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4057067242307498808, "histo_col_type": "", "name": "__auto__", "null_count": 2600574521777521879, "row_count": 9188929641994862252}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5180963877395238670, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9188929641994862252}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8609369084897817012, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000000, "num_range": 0, "upper_bound": "32067"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 9188929641994862252}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "infinity" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2564048&tab=buildLog) on [master@15765c0fa9118885dda0bd2ad1384b8801c412c3](https://github.com/cockroachdb/cockroach/commits/15765c0fa9118885dda0bd2ad1384b8801c412c3):

```
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        test_1        | --- FAIL: TestCompare (1.57s)
        test_1        |     --- FAIL: TestCompare/postgres (0.69s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.3:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.88s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 CHAR NOT NULL, col1_1 VARCHAR NOT NULL, col1_2 REGCLASS NOT NULL, col1_3 TIMESTAMPTZ NOT NULL, col1_4 TIMESTAMPTZ NOT NULL, PRIMARY KEY (col1_1, col1_3, col1_2, col1_0 DESC), INDEX (col1_2 ASC, col1_4, col1_3 ASC) WHERE ((table1.col1_0 > e'\'':::STRING) AND (table1.col1_1 != e'\x00':::STRING)) OR (table1.col1_4 = '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ), UNIQUE (col1_4, col1_0 DESC, col1_3, col1_2, col1_1 ASC) WHERE ((table1.col1_4 != '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ) AND (table1.col1_0 < '':::STRING)) OR (table1.col1_3 >= '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ), UNIQUE (col1_3, col1_0, col1_4 ASC) WHERE (((table1.col1_4 != '3000-01-01 00:00:00+00:00') OR (table1.col1_3 >= '3000-01-01 00:00:00+00:00')) AND (table1.col1_1 > e'\'')) AND (table1.col1_0 >= ''), INDEX (col1_2, col1_1 ASC, col1_0 ASC, col1_3, col1_4 ASC) WHERE (((table1.col1_4 < '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ) AND (table1.col1_3 >= '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (table1.col1_1 >= '"':::STRING)) OR (table1.col1_0 >= 'X':::STRING), INDEX (col1_3 DESC, col1_2 DESC, col1_0 DESC, col1_1, col1_4 ASC), INDEX (col1_4 DESC, col1_3 DESC, col1_0 DESC, col1_2 DESC, col1_1 DESC), FAMILY (col1_4, col1_0, col1_3, col1_1, col1_2));
        test_1        |             CREATE TABLE table2 (col1_1 VARCHAR NOT NULL, col1_3 TIMESTAMPTZ NOT NULL, col1_2 REGCLASS NOT NULL, col1_0 CHAR NOT NULL, col2_4 TIME NOT NULL, col2_5 REGTYPE, col2_6 INT2 NOT NULL, col2_7 TIMETZ, col2_8 VARCHAR NULL, col2_9 FLOAT4 NULL, col2_10 TIMESTAMPTZ, col2_11 INT8 NOT NULL, col2_12 FLOAT8, col2_13 INT2 NULL, PRIMARY KEY (col1_1, col1_3, col1_2, col1_0 DESC, col2_4, col2_12 DESC, col2_7 DESC, col2_9 ASC), UNIQUE (col2_7, col2_6, col1_3, col2_10, col2_11 DESC, col2_8, col2_12, col1_0 ASC, col2_4) WHERE (((((((((((table2.col2_6 > (-1):::INT8) OR (table2.col1_3 > '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table2.col2_8 <= e'\x00':::STRING)) OR (table2.col2_4 < '24:00:00':::TIME)) AND (table2.col2_9 < '+Inf':::FLOAT8)) AND (table2.col2_13 = 0:::INT8)) OR (table2.col2_11 >= (-128):::INT8)) AND (table2.col1_1 > e'\x00':::STRING)) AND (table2.col1_0 > e'\U00002603':::STRING)) AND (table2.col2_7 < '24:00:00-15:59:00':::TIMETZ)) OR (table2.col2_10 > '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (table2.col2_12 <= 0.0:::FLOAT8), INDEX (col2_12 DESC, col1_0 ASC, col2_4, col2_11 ASC, col2_10 ASC, col2_8 DESC, col2_9 DESC, col2_5 DESC, col1_1, col1_2 ASC, col1_3 ASC) STORING (col2_6, col2_13), UNIQUE (col2_12 DESC, col1_3 DESC) STORING (col2_5, col2_8, col2_13) WHERE (((((((((((table2.col2_7 >= '24:00:00-15:59:00':::TIMETZ) OR (table2.col1_0 < '':::STRING)) AND (table2.col2_12 <= 1.0:::FLOAT8)) OR (table2.col2_4 = '24:00:00':::TIME)) AND (table2.col2_10 < '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table2.col2_9 <= (-1.0):::FLOAT8)) AND (table2.col1_3 <= '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table2.col2_11 <= 2147483647:::INT8)) AND (table2.col2_8 != e'\U00002603':::STRING)) AND (table2.col1_1 >= e'\U00002603':::STRING)) OR (table2.col2_6 >= 127:::INT8)) AND (table2.col2_13 < 0:::INT8), UNIQUE (col2_11 DESC, col1_1 ASC, col1_2 ASC, col2_12, col2_8 ASC) STORING (col2_6) WHERE ((((table2.col2_9 < '+Inf':::FLOAT8) OR (table2.col2_10 != '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table2.col2_11 != (-2147483648):::INT8)) AND (table2.col1_0 = e'\'':::STRING)) AND (table2.col2_6 > (-128):::INT8), UNIQUE (col2_5 ASC, col1_3, col2_9 ASC, col2_8, col2_11, col1_2, col1_0, col2_7 ASC, col2_4 DESC, col2_13, col2_10) STORING (col2_6) WHERE (((table2.col2_10 > '-4713-11-24 00:00:00+00:00') OR (table2.col2_12 <= 0.0)) OR (table2.col2_7 < '00:00:00+15:59:00')) AND (table2.col1_3 = '3000-01-01 00:00:00+00:00'), INDEX (col2_6 ASC, col1_3, col2_4, col2_9 ASC, col2_8, col2_12 ASC, col1_0 ASC, col2_13 ASC, col2_7 ASC, col1_1 ASC, col1_2 ASC, col2_10 ASC, col2_11 DESC, col2_5 DESC), UNIQUE (col2_11 DESC, col1_0 DESC, col2_9 DESC, col2_6 ASC) STORING (col2_5, col2_13) WHERE table2.col2_9 < 1.401298464324817e-45:::FLOAT8, UNIQUE (col1_1, col1_2, col2_7) STORING (col2_5, col2_6, col2_10, col2_11), UNIQUE (col2_9) STORING (col2_5), FAMILY (col2_12), FAMILY (col2_7, col2_5, col1_0), FAMILY (col2_6), FAMILY (col2_11), FAMILY (col1_2), FAMILY (col1_1, col2_10, col2_13, col1_3, col2_4, col2_8, col2_9)) INTERLEAVE IN PARENT table1 (col1_1, col1_3, col1_2, col1_0);
        test_1        |             CREATE TABLE table3 (col1_1 VARCHAR NOT NULL, col1_3 TIMESTAMPTZ NOT NULL, col1_2 REGCLASS NOT NULL, col1_0 CHAR NOT NULL, col2_4 TIME NOT NULL, col2_12 FLOAT8, col2_7 TIMETZ, col2_9 FLOAT4 NULL, col3_8 TIME NOT NULL, col3_9 OID NOT NULL, col3_10 INTERVAL NOT NULL, col3_11 BYTES NULL, col3_12 OID, col3_13 BIT(18) NULL, col3_14 BYTES NULL, col3_15 TIME NULL, col3_16 CHAR, col3_17 FLOAT4 NULL, col3_18 "char", col3_19 FLOAT4 NULL, col3_20 BIT(49) NULL, col3_21 INT2, PRIMARY KEY (col1_1, col1_3, col1_2, col1_0 DESC, col2_4, col2_12 DESC, col2_7 DESC, col2_9 ASC, col3_20, col3_19, col3_21 DESC, col3_8 DESC), UNIQUE (col3_15, col3_17 ASC, col3_16 DESC, col1_2, col2_7 ASC, col3_11, col1_0 DESC, col3_21 ASC, col3_19 DESC, col3_20, col2_12 DESC) STORING (col3_10, col3_18) WHERE (((((((((((((((table3.col3_19 = +Inf) OR (table3.col3_11 < '\x27')) AND (table3.col3_14 != '\xe29883')) AND (table3.col1_1 != '"')) AND (table3.col1_0 > e'\U00002603')) OR (table3.col3_17 <= -1.0)) OR (table3.col2_9 >= +Inf)) AND (table3.col2_4 <= '24:00:00')) OR (table3.col3_21 > 32767)) OR (table3.col3_8 < '00:00:00')) OR (table3.col3_18 <= e'\'')) AND (table3.col3_16 > 'X')) AND (table3.col2_12 >= 5e-324)) AND (table3.col3_15 < '24:00:00')) OR (table3.col2_7 >= '00:00:00+15:59:00')) AND (table3.col1_3 <= '-2000-01-01 00:00:00+00:00'), INDEX (col3_10 ASC, col3_13 ASC, col2_4 ASC, col3_19 ASC, col3_11, col3_14 ASC, col2_12, col3_9 ASC, col3_12 DESC, col1_0 DESC, col2_7 DESC, col1_3, col3_21) STORING (col3_15, col3_16), INDEX (col3_11 ASC, col3_21, col3_12, col3_16 ASC, col3_18 DESC, col3_14, col2_12 ASC, col1_2, col3_10 DESC, col1_3 DESC, col2_4 DESC, col3_20 ASC, col2_9 ASC, col3_17 DESC) STORING (col3_9, col3_13, col3_15) WHERE (((table3.col2_4 != '24:00:00':::TIME) OR (table3.col1_0 <= '':::STRING)) OR (table3.col3_11 <= e'\\x27':::BYTES)) AND (table3.col3_8 < '24:00:00':::TIME), UNIQUE (col3_11 DESC, col2_4 DESC, col3_9 ASC, col1_0, col3_13 ASC, col1_1 DESC, col3_20, col3_19, col2_7, col3_15 DESC, col3_21 ASC, col1_3 DESC, col3_18 ASC, col3_8 DESC, col2_12 DESC) STORING (col3_16, col3_17), UNIQUE (col3_15 ASC, col3_12 ASC, col3_9 DESC, col2_12, col3_21, col3_19, col3_13 DESC, col3_17 DESC, col1_3 ASC) STORING (col3_10, col3_16, col3_18) WHERE ((((((((((((((table3.col3_17 <= '-Inf':::FLOAT8) AND (table3.col1_3 != '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table3.col2_7 != '24:00:00-15:59:00':::TIMETZ)) OR (table3.col2_12 >= 1.401298464324817e-45:::FLOAT8)) OR (table3.col3_15 <= '24:00:00':::TIME)) AND (table3.col3_21 < 1:::INT8)) OR (table3.col2_4 > '24:00:00':::TIME)) AND (table3.col1_0 > '"':::STRING)) OR (table3.col1_1 != e'\U00002603':::STRING)) AND (table3.col3_19 != 3.4028234663852886e+38:::FLOAT8)) OR (table3.col3_11 = e'\\x':::BYTES)) AND (table3.col2_9 != 'NaN':::FLOAT8)) OR (table3.col3_14 != e'\\x27':::BYTES)) OR (table3.col3_8 < '00:00:00':::TIME)) AND (table3.col3_18 <= 'X':::STRING), FAMILY (col3_13, col1_3), FAMILY (col2_9), FAMILY (col3_20), FAMILY (col3_19, col1_0), FAMILY (col2_7, col3_16), FAMILY (col3_10, col3_12), FAMILY (col2_4, col3_17), FAMILY (col1_1), FAMILY (col2_12), FAMILY (col3_21, col3_8), FAMILY (col3_15, col3_11), FAMILY (col3_18), FAMILY (col3_9, col3_14), FAMILY (col1_2)) INTERLEAVE IN PARENT table2 (col1_1, col1_3, col1_2, col1_0, col2_4, col2_12, col2_7, col2_9);
        test_1        |             CREATE TABLE table4 (col1_1 VARCHAR NOT NULL, col1_3 TIMESTAMPTZ NOT NULL, col1_2 REGCLASS NOT NULL, col1_0 CHAR NOT NULL, col2_4 TIME NOT NULL, col2_12 FLOAT8, col2_7 TIMETZ, col2_9 FLOAT4 NULL, col3_20 BIT(49) NULL, col3_19 FLOAT4 NULL, col3_21 INT2, col3_8 TIME NOT NULL, col4_12 "char" NULL, col4_13 DATE, col4_14 TIMETZ, col4_15 DATE, col4_16 REGPROC NULL, col4_17 FLOAT4 NULL, PRIMARY KEY (col1_1, col1_3, col1_2, col1_0 DESC, col2_4, col2_12 DESC, col2_7 DESC, col2_9 ASC, col3_20, col3_19, col3_21 DESC, col3_8 DESC), INDEX (col4_15, col2_4, col1_1 ASC, col2_12, col1_3, col1_2, col1_0 DESC, col3_8, col4_12 DESC, col3_19 ASC, col3_20 DESC, col4_16 ASC, col2_7 DESC, col2_9 ASC, col4_14) STORING (col4_13), INDEX (col4_17, col2_7 DESC, col1_1 DESC, col3_20, col1_3, col2_4 ASC, col3_19 DESC) STORING (col4_12, col4_13, col4_14, col4_15) WHERE (((((((((((table4.col2_4 <= '00:00:00':::TIME) OR (table4.col3_21 != (-128):::INT8)) AND (table4.col3_19 <= '+Inf':::FLOAT8)) AND (table4.col2_7 != '00:00:00+15:59:00':::TIMETZ)) OR (table4.col4_15 > '4714-11-24 BC':::DATE)) OR (table4.col4_13 <= '4714-11-24 BC':::DATE)) AND (table4.col4_17 <= 1.0:::FLOAT8)) AND (table4.col3_8 = '24:00:00':::TIME)) OR (table4.col2_9 < 3.4028234663852886e+38:::FLOAT8)) AND (table4.col4_12 > '':::STRING)) OR (table4.col2_12 < 5e-324:::FLOAT8)) AND (table4.col1_1 > e'\'':::STRING), INDEX (col1_2 ASC, col1_1 ASC, col1_3, col3_21 DESC, col4_17 ASC) STORING (col4_12, col4_13, col4_14, col4_15), UNIQUE (col1_2) WHERE ((((table4.col4_17 > 1.0) OR (table4.col4_13 = '-infinity')) AND (table4.col2_4 >= '24:00:00')) OR (table4.col2_12 >= 1.0)) AND (table4.col3_21 = 1), UNIQUE (col1_1), FAMILY (col3_20), FAMILY (col1_3), FAMILY (col4_16), FAMILY (col4_12), FAMILY (col1_0), FAMILY (col3_21), FAMILY (col2_7), FAMILY (col4_13), FAMILY (col3_8, col4_17), FAMILY (col4_15), FAMILY (col3_19), FAMILY (col1_1, col4_14), FAMILY (col2_4), FAMILY (col1_2), FAMILY (col2_9), FAMILY (col2_12)) INTERLEAVE IN PARENT table3 (col1_1, col1_3, col1_2, col1_0, col2_4, col2_12, col2_7, col2_9, col3_20, col3_19, col3_21, col3_8);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 747766236804434873, "histo_buckets": [{"distinct_range": 0, "num_eq": 5503005573347409650, "num_range": 0, "upper_bound": "1972-08-13 07:39:08.000634+00:00"}, {"distinct_range": 8000000000, "num_eq": 1637709797941383415, "num_range": 8000000000, "upper_bound": "1981-06-22 03:06:46.000856+00:00"}, {"distinct_range": 0, "num_eq": 1431044420717654580, "num_range": 800000000, "upper_bound": "3000-01-01 00:00:00+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 1898606476475380031}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 101070082036016595, "histo_buckets": [{"distinct_range": 0, "num_eq": 504947619287163031, "num_range": 0, "upper_bound": "-2000-01-01 00:00:00+00:00"}, {"distinct_range": 0, "num_eq": 7894896940253971156, "num_range": 300000, "upper_bound": "0001-01-01 00:00:00+00:00"}, {"distinct_range": 8000, "num_eq": 675316027693776705, "num_range": 8000, "upper_bound": "2018-01-03 22:41:52.000827+00:00"}, {"distinct_range": 450985.4502241177, "num_eq": 1000000, "num_range": 800000, "upper_bound": "2026-09-06 21:33:19.000905+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 1898606476475380031}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 385862749834101142, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1898606476475380031}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 515591678098074431, "histo_buckets": [{"distinct_range": 0, "num_eq": 10, "num_range": 0, "upper_bound": "/^"}, {"distinct_range": 925883904205900700, "num_eq": 3000000000, "num_range": 1294965315633300588, "upper_bound": "@W"}, {"distinct_range": 2000000, "num_eq": 8670404223198789485, "num_range": 2000000, "upper_bound": "c\\tgB\\tI\\u001a"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 1898606476475380031}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1203024004934055556, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 200, "upper_bound": "14874209"}, {"distinct_range": 13432041.867316278, "num_eq": 90000, "num_range": 100000000, "upper_bound": "43650965"}, {"distinct_range": 194676894.46140367, "num_eq": 3348621082438412392, "num_range": 300000000, "upper_bound": "1672286811"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 1898606476475380031}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 589548657, "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 580559723, "histo_col_type": "", "name": "__auto__", "null_count": 237409943, "row_count": 700000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 609808298, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000000, "num_range": 0, "upper_bound": "-5881913720699190502"}, {"distinct_range": 185945.23767414957, "num_eq": 30000000000, "num_range": 600000, "upper_bound": "578777849147571691"}, {"distinct_range": 4170884512550890500, "num_eq": 0, "num_range": 8342497792852590736, "upper_bound": "1387384685827724386"}, {"distinct_range": 300, "num_eq": 7063066205114517535, "num_range": 300, "upper_bound": "2187583657126926982"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 538125454, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000000, "num_range": 0, "upper_bound": "660097281"}, {"distinct_range": 2404221.0575547745, "num_eq": 80000000000, "num_range": 10000000, "upper_bound": "1262781491"}, {"distinct_range": 5000000000, "num_eq": 50000, "num_range": 5000000000, "upper_bound": "1886108318"}, {"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "2328468139"}, {"distinct_range": 0, "num_eq": 5244887291019607781, "num_range": 0, "upper_bound": "2345964237"}, {"distinct_range": 2496663080303777300, "num_eq": 3119036110363012506, "num_range": 2875207592971124458, "upper_bound": "2759175856"}, {"distinct_range": 2137324931867248400, "num_eq": 331899298862676605, "num_range": 2979493400856234261, "upper_bound": "3131026297"}, {"distinct_range": 1921366591795788500, "num_eq": 4000000000, "num_range": 3287545810567245334, "upper_bound": "4062492625"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 411829212, "row_count": 700000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 244239874, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000, "num_range": 0, "upper_bound": "-23534"}, {"distinct_range": 0, "num_eq": 3289255615138616104, "num_range": 100000000000, "upper_bound": "-20968"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 499131332, "histo_col_type": "", "name": "__auto__", "null_count": 107403349, "row_count": 700000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 161783490, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 328156568, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 454600345, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 363057483, "histo_col_type": "", "name": "__auto__", "null_count": 554732242, "row_count": 700000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 263012234, "histo_col_type": "", "name": "__auto__", "null_count": 274202708, "row_count": 700000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 274997030, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 227852825, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000, "num_range": 0, "upper_bound": "17:49:52.029303+12:16:00"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 2000000, "upper_bound": "18:10:22.615721-05:03:00"}, {"distinct_range": 4076557162204730400, "num_eq": 0, "num_range": 4076557162204730490, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 563511488, "row_count": 700000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 285639049, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-0.723248399425649"}, {"distinct_range": 2560018771003944000, "num_eq": 5274897867491854101, "num_range": 2560018771003943761, "upper_bound": "-0.10077150855081718"}, {"distinct_range": 0, "num_eq": 8488431106130577159, "num_range": 4535357202833526926, "upper_bound": "-0.0219235806248681"}, {"distinct_range": 7000, "num_eq": 600000, "num_range": 7000, "upper_bound": "0.03513544056286122"}, {"distinct_range": 4085533597605293000, "num_eq": 100000000, "num_range": 4085533597605292874, "upper_bound": "0.06509614643511796"}, {"distinct_range": 1888326441514226400, "num_eq": 2205056350298550656, "num_range": 2635906094745110124, "upper_bound": "1.0"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 511709516, "row_count": 700000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4642409, "histo_col_type": "TIME", "name": "__auto__", "null_count": 1968921, "row_count": 6000000}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 541432, "histo_col_type": "", "name": "__auto__", "null_count": 718988, "row_count": 6000000}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3637540, "histo_col_type": "", "name": "__auto__", "null_count": 4584886, "row_count": 6000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2422928, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000, "num_range": 0, "upper_bound": "\\u0013ADs}\\u000b"}, {"distinct_range": 0, "num_eq": 938797830979906313, "num_range": 5000000, "upper_bound": "+"}, {"distinct_range": 1653799700580445000, "num_eq": 4000, "num_range": 1653799700580444989, "upper_bound": ".vuLU\'"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 6000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4539290, "histo_col_type": "", "name": "__auto__", "null_count": 4073314, "row_count": 6000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4966103, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 453967, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5618142, "histo_col_type": "", "name": "__auto__", "null_count": 3618842, "row_count": 6000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1660751, "histo_col_type": "", "name": "__auto__", "null_count": 2802034, "row_count": 6000000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 629309, "histo_col_type": "", "name": "__auto__", "null_count": 4333284, "row_count": 6000000}, {"columns": ["col3_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3495129, "histo_col_type": "", "name": "__auto__", "null_count": 5106817, "row_count": 6000000}, {"columns": ["col3_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5583116, "histo_col_type": "", "name": "__auto__", "null_count": 1767516, "row_count": 6000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4345627, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5988644, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000, "num_range": 0, "upper_bound": "\\\\x2c237e9402bc"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 4558051, "row_count": 6000000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4886122, "histo_col_type": "", "name": "__auto__", "null_count": 5336946, "row_count": 6000000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1869812, "histo_col_type": "", "name": "__auto__", "null_count": 2173293, "row_count": 6000000}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3125221, "histo_col_type": "", "name": "__auto__", "null_count": 5669528, "row_count": 6000000}, {"columns": ["col3_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2633257, "histo_col_type": "", "name": "__auto__", "null_count": 5843320, "row_count": 6000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2016962, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1950198, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2773412, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53660, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000000, "num_range": 0, "upper_bound": "-53 years -1 mons -788 days -05:54:30.337661"}, {"distinct_range": 160810452084874020, "num_eq": 6673195834443539681, "num_range": 1967622492214279105, "upper_bound": "-50 years -8 mons -435 days -08:07:55.084011"}, {"distinct_range": 34885132432509924, "num_eq": 0, "num_range": 109344043977209696, "upper_bound": "18 years 4 mons 169 days 05:31:42.691254"}, {"distinct_range": 1556175409759759600, "num_eq": 1575375712244974468, "num_range": 5275019093028461187, "upper_bound": "36 years 4 mons 710 days 06:58:39.316402"}, {"distinct_range": 48487980239.64204, "num_eq": 800, "num_range": 60000000000, "upper_bound": "45 years 8 mons 924 days 22:09:45.300347"}, {"distinct_range": 1725721.5181763363, "num_eq": 50000000000, "num_range": 3000000, "upper_bound": "57 years 9 mons 209 days 11:37:12.426002"}, {"distinct_range": 60, "num_eq": 6000, "num_range": 60, "upper_bound": "75 years 824 days 06:32:35.976804"}, {"distinct_range": 0, "num_eq": 3168578831982909576, "num_range": 0, "upper_bound": "82 years 2 mons 746 days 05:08:33.163028"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 6000000}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col3_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 224309874185410875, "histo_col_type": "", "name": "__auto__", "null_count": 8688135274496023, "row_count": 6135917884333315331}, {"columns": ["col3_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5316929343719785332, "histo_col_type": "", "name": "__auto__", "null_count": 3908298713663536956, "row_count": 6135917884333315331}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3675568940420688093, "histo_col_type": "", "name": "__auto__", "null_count": 2015151768993260844, "row_count": 6135917884333315331}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3682050008752802641, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6135917884333315331}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5452282311451999890, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6135917884333315331}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6049172187341955918, "histo_col_type": "", "name": "__auto__", "null_count": 5966527350146445224, "row_count": 6135917884333315331}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 908878647172697839, "histo_col_type": "", "name": "__auto__", "null_count": 5800400020295559264, "row_count": 6135917884333315331}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 84080336108317336, "histo_col_type": "", "name": "__auto__", "null_count": 1501898771881368641, "row_count": 6135917884333315331}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5473360581773386800, "histo_buckets": [{"distinct_range": 0, "num_eq": 8710345429986143737, "num_range": 0, "upper_bound": "-infinity"}, {"distinct_range": 0, "num_eq": 90000000000, "num_range": 8093425314735903888, "upper_bound": "1974-12-02"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 100000, "upper_bound": "1977-09-02"}, {"distinct_range": 0, "num_eq": 7897225633227577455, "num_range": 2904266461303395310, "upper_bound": "1989-06-03"}, {"distinct_range": 77.57636419190557, "num_eq": 806059938964955889, "num_range": 100, "upper_bound": "1991-06-06"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 6023559573866522774, "row_count": 6135917884333315331}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1735142334525070409, "histo_col_type": "", "name": "__auto__", "null_count": 4063918212510821299, "row_count": 6135917884333315331}, {"columns": ["col3_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5746257601875261151, "histo_col_type": "", "name": "__auto__", "null_count": 2035106754932852454, "row_count": 6135917884333315331}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1924683790901149810, "histo_col_type": "", "name": "__auto__", "null_count": 2134517375884933818, "row_count": 6135917884333315331}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4176806040018196425, "histo_col_type": "", "name": "__auto__", "null_count": 1546575222420091124, "row_count": 6135917884333315331}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2793749498975329553, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "-0.4825224280357361"}, {"distinct_range": 30000, "num_eq": 2115737247938763227, "num_range": 30000, "upper_bound": "-0.09085128456354141"}, {"distinct_range": 1303380346425553000, "num_eq": 2635720296697958711, "num_range": 1557335554910108867, "upper_bound": "0.7263261675834656"}, {"distinct_range": 862721589648592300, "num_eq": 50000000, "num_range": 1152942427268256658, "upper_bound": "1.3261789083480835"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 5013049070621281001, "row_count": 6135917884333315331}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4425878240489935080, "histo_buckets": [{"distinct_range": 0, "num_eq": 353917671031327261, "num_range": 0, "upper_bound": "\\u0012K\\t1\\u00016M"}, {"distinct_range": 10, "num_eq": 0, "num_range": 10, "upper_bound": "3P\\u00084N@-t"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 6135917884333315331}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4590730572188960106, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000000, "num_range": 0, "upper_bound": "2069700537"}, {"distinct_range": 388.02697225211216, "num_eq": 40, "num_range": 400, "upper_bound": "2742477071"}, {"distinct_range": 89584304.91373585, "num_eq": 1068142939253834563, "num_range": 90000000, "upper_bound": "2901763606"}, {"distinct_range": 30, "num_eq": 9000000000, "num_range": 30, "upper_bound": "3658569867"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 6135917884333315331}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4636422910414134052, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6135917884333315331}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1721895896464477520, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6135917884333315331}]':::JSONB;
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_2) REFERENCES table4 (col1_2) ON UPDATE RESTRICT;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('ttibj', 'u', 'fluvah', 'oadx', 'old');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('t', 'bsh', 'cipbts', 'jckxyf');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('hjmsk', 'pdvpa', 'mzix', 'pcqczc');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('smbr', 'oxzuqa', 'jty', 'otvgeu');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1025, "histo_buckets": [{"distinct_range": 0, "num_eq": 5721519543144063036, "num_range": 0, "upper_bound": "1053415602"}, {"distinct_range": 5000000000, "num_eq": 300000, "num_range": 5000000000, "upper_bound": "1826619067"}, {"distinct_range": 6000, "num_eq": 70, "num_range": 6000, "upper_bound": "2820414025"}, {"distinct_range": 294751.0694798383, "num_eq": 623034224061025224, "num_range": 500000, "upper_bound": "2865853449"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 3000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1970, "histo_buckets": [{"distinct_range": 0, "num_eq": 50, "num_range": 0, "upper_bound": "1989-09-13 23:02:08.000602+00:00"}, {"distinct_range": 0, "num_eq": 5355944632330806060, "num_range": 0, "upper_bound": "1992-12-12 05:56:18.000469+00:00"}, {"distinct_range": 1241755951375252500, "num_eq": 3073140947243821899, "num_range": 2001909076510247749, "upper_bound": "2006-03-06 03:04:40.000717+00:00"}, {"distinct_range": 143657.41195221612, "num_eq": 1000000000, "num_range": 1000000, "upper_bound": "2020-07-12 08:13:58.000708+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 3000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 822, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000, "num_range": 0, "upper_bound": "1991-06-22 16:19:52.000181+00:00"}, {"distinct_range": 276672836.131176, "num_eq": 90000, "num_range": 600000000, "upper_bound": "1997-06-07 23:11:46.000341+00:00"}, {"distinct_range": 8746873813410672000, "num_eq": 1559291072895014654, "num_range": 8746873813410671529, "upper_bound": "2010-09-15 07:14:44.000364+00:00"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 50000, "upper_bound": "2022-07-09 16:28:43.000936+00:00"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 4481104552593788659, "upper_bound": "2026-07-10 20:07:48.000198+00:00"}, {"distinct_range": 0, "num_eq": 4243126955770660908, "num_range": 6419894545220450143, "upper_bound": "2028-06-09 22:19:32.000295+00:00"}, {"distinct_range": 8198797838238671000, "num_eq": 50000000, "num_range": 8198797838238671289, "upper_bound": "2031-12-14 16:18:34.000118+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 3000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1374, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2193, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 0, "num_eq": 30, "num_range": 6363628004688729712, "upper_bound": "-"}, {"distinct_range": 95947.91464525524, "num_eq": 4076666989605286925, "num_range": 400000, "upper_bound": "6\\u0011"}, {"distinct_range": 80000, "num_eq": 6360567591636708709, "num_range": 80000, "upper_bound": "8\'Q"}, {"distinct_range": 0, "num_eq": 4185994865370106630, "num_range": 2000000000, "upper_bound": "H}\\u0016b\\u0010b\\u0004\x7fk"}, {"distinct_range": 0, "num_eq": 400, "num_range": 2407714257109975932, "upper_bound": "n]\\u001cT\\u001daXl4"}, {"distinct_range": 40987.2823136098, "num_eq": 1825287374713342949, "num_range": 100000, "upper_bound": "z9W*cCU"}, {"distinct_range": 4676275023291088000, "num_eq": 30, "num_range": 4676275023291087887, "upper_bound": "\U00002603"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 3000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21251, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "\\u0012FK"}, {"distinct_range": 8548161352320464000, "num_eq": 100, "num_range": 8548161352320464293, "upper_bound": "*\\u000eaPs\\u0013 <V"}, {"distinct_range": 7000000000, "num_eq": 5253445290169450166, "num_range": 7000000000, "upper_bound": "1<9Mq!h"}, {"distinct_range": 2037508286807069700, "num_eq": 753911665840096265, "num_range": 6103237044064602153, "upper_bound": "1VS)iq;,"}, {"distinct_range": 1000, "num_eq": 3333918598102006548, "num_range": 1000, "upper_bound": "="}, {"distinct_range": 2822467545592425000, "num_eq": 40, "num_range": 2822467545592424783, "upper_bound": "T)i"}, {"distinct_range": 8285336106405109000, "num_eq": 7000000000, "num_range": 8285336106405108714, "upper_bound": "X"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18342, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000000, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 0, "num_eq": 3970917476091049075, "num_range": 20000000, "upper_bound": "11:00:47.82533+11:12:00"}, {"distinct_range": 2455748791484272600, "num_eq": 6000, "num_range": 2455748791484272658, "upper_bound": "15:58:04.051285+13:03:00"}, {"distinct_range": 0, "num_eq": 3862878303967134438, "num_range": 2990192315374139583, "upper_bound": "05:03:41.746291-01:00:00"}, {"distinct_range": 824879739905943600, "num_eq": 3546557475756975604, "num_range": 2249946856593971349, "upper_bound": "09:13:22.576623+02:12:00"}, {"distinct_range": 0, "num_eq": 30000, "num_range": 3760926512970544011, "upper_bound": "13:28:04.216039+00:51:00"}, {"distinct_range": 20, "num_eq": 20000000, "num_range": 20, "upper_bound": "12:48:53.053391-03:36:00"}, {"distinct_range": 400000, "num_eq": 664221290183072964, "num_range": 400000, "upper_bound": "13:39:16.049622-09:15:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 11914, "row_count": 30000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19118, "histo_buckets": [{"distinct_range": 0, "num_eq": 940852707564070893, "num_range": 0, "upper_bound": "-0.7316868712205432"}, {"distinct_range": 51.26223813741347, "num_eq": 4131700832476931175, "num_range": 70, "upper_bound": "0.1405019932205701"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 1299, "row_count": 30000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8246, "histo_col_type": "", "name": "__auto__", "null_count": 13722, "row_count": 30000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1972, "histo_buckets": [{"distinct_range": 0, "num_eq": 90, "num_range": 0, "upper_bound": "-5062070120351989847"}, {"distinct_range": 30000000000, "num_eq": 8000000, "num_range": 30000000000, "upper_bound": "5862774706777437974"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25688, "histo_col_type": "", "name": "__auto__", "null_count": 2371, "row_count": 30000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9543, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4130, "histo_col_type": "", "name": "__auto__", "null_count": 19938, "row_count": 30000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11186, "histo_col_type": "", "name": "__auto__", "null_count": 19153, "row_count": 30000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17720, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000000, "num_range": 0, "upper_bound": "5211"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 300000, "upper_bound": "19232"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10977, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5180, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7590, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23833, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000000, "num_range": 0, "upper_bound": "30757871"}, {"distinct_range": 0, "num_eq": 1549208957310946210, "num_range": 2159355004014333027, "upper_bound": "337214071"}, {"distinct_range": 0, "num_eq": 5580365812514312922, "num_range": 8517779891362785542, "upper_bound": "664169349"}, {"distinct_range": 60, "num_eq": 0, "num_range": 60, "upper_bound": "1483111311"}, {"distinct_range": 1967272198840187600, "num_eq": 5000000, "num_range": 2022064569861939409, "upper_bound": "2169229372"}, {"distinct_range": 900000, "num_eq": 0, "num_range": 900000, "upper_bound": "2180860413"}, {"distinct_range": 800000, "num_eq": 300, "num_range": 800000, "upper_bound": "3406897395"}, {"distinct_range": 5782743382347905000, "num_eq": 7727080421587902393, "num_range": 5782743382347905320, "upper_bound": "3746810167"}, {"distinct_range": 0, "num_eq": 2406426902188717511, "num_range": 9000, "upper_bound": "4182994113"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 16476, "row_count": 30000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1206249154, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1336023933, "histo_buckets": [{"distinct_range": 0, "num_eq": 871575716781023839, "num_range": 0, "upper_bound": "\\\\x5ce3630e918343e7"}, {"distinct_range": 4095516430566196000, "num_eq": 6000000, "num_range": 4095516430566196072, "upper_bound": "\\\\x85d3"}, {"distinct_range": 30, "num_eq": 80000, "num_range": 30, "upper_bound": "\\\\xc6"}, {"distinct_range": 656.4519809667854, "num_eq": 300000000, "num_range": 6000, "upper_bound": "\\\\xf7e478612201117164"}, {"distinct_range": 0, "num_eq": 1674577111032565015, "num_range": 200, "upper_bound": "\\\\xfece998fcf79ea4062"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 1590868815, "row_count": 2000000000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1297539162, "histo_col_type": "", "name": "__auto__", "null_count": 748955025, "row_count": 2000000000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 893181252, "histo_col_type": "", "name": "__auto__", "null_count": 877832534, "row_count": 2000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1477277778, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1285800262, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 883386822, "histo_col_type": "", "name": "__auto__", "null_count": 286543588, "row_count": 2000000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 663011936, "histo_col_type": "", "name": "__auto__", "null_count": 390883069, "row_count": 2000000000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23391871, "histo_buckets": [{"distinct_range": 0, "num_eq": 9208984121710285840, "num_range": 0, "upper_bound": "-22 years -9 mons -40 days -13:35:19.959018"}, {"distinct_range": 44709.48725329946, "num_eq": 800000, "num_range": 60000, "upper_bound": "-13 years -1 mons -135 days -03:36:09.317699"}, {"distinct_range": 2.1691249962037755, "num_eq": 3000000, "num_range": 50, "upper_bound": "17 years 11 mons 150 days 06:28:47.60557"}, {"distinct_range": 6694698310542047000, "num_eq": 5734225680119533895, "num_range": 8096927077874928902, "upper_bound": "46 years 11 mons 660 days 09:09:56.810954"}, {"distinct_range": 0, "num_eq": 300000, "num_range": 0, "upper_bound": "48 years 8 mons 373 days 17:18:10.408364"}, {"distinct_range": 0, "num_eq": 2901776762341850185, "num_range": 9069956876249019819, "upper_bound": "61 years 5 mons 738 days 00:34:24.554846"}, {"distinct_range": 0, "num_eq": 300000, "num_range": 731884252524750110, "upper_bound": "64 years 7 mons 286 days 04:24:04.731369"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 2000000000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 744054513, "histo_col_type": "TIME", "name": "__auto__", "null_count": 1144064596, "row_count": 2000000000}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 244680790, "histo_col_type": "", "name": "__auto__", "null_count": 193718937, "row_count": 2000000000}, {"columns": ["col3_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 845547194, "histo_col_type": "", "name": "__auto__", "null_count": 974154074, "row_count": 2000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1587659391, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000000}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1532838235, "histo_col_type": "", "name": "__auto__", "null_count": 824381302, "row_count": 2000000000}, {"columns": ["col3_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1195784653, "histo_col_type": "", "name": "__auto__", "null_count": 1537844979, "row_count": 2000000000}, {"columns": ["col3_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 543848002, "histo_col_type": "", "name": "__auto__", "null_count": 1262132289, "row_count": 2000000000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 221677671, "histo_col_type": "", "name": "__auto__", "null_count": 1536008249, "row_count": 2000000000}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 869223908, "histo_col_type": "", "name": "__auto__", "null_count": 1162570815, "row_count": 2000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1039167562, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 1131.232481050637, "num_eq": 4937941757346713581, "num_range": 6000, "upper_bound": "HK"}, {"distinct_range": 7714468111304334000, "num_eq": 183098386001731807, "num_range": 7714468111304334656, "upper_bound": "V\\u001f"}, {"distinct_range": 12024112.638253013, "num_eq": 6970615539782407289, "num_range": 50000000, "upper_bound": "]Qh=\\u0006&\\u0003`\\r"}, {"distinct_range": 20000, "num_eq": 1314914602670553129, "num_range": 20000, "upper_bound": "^dp\\u000cXd"}, {"distinct_range": 0, "num_eq": 3994389281219643503, "num_range": 8429579067221832761, "upper_bound": "`3\\u001dU\\n\\\\"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 2000000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 222724464, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1896441079, "histo_col_type": "", "name": "__auto__", "null_count": 1881049077, "row_count": 2000000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 791217850, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1610641035127294713, "histo_col_type": "", "name": "__auto__", "null_count": 5521797945034609686, "row_count": 6532273077833447112}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2025222934259887832, "histo_col_type": "", "name": "__auto__", "null_count": 4292541196219353464, "row_count": 6532273077833447112}, {"columns": ["col3_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1974153927297921086, "histo_col_type": "", "name": "__auto__", "null_count": 1406263657731671945, "row_count": 6532273077833447112}, {"columns": ["col3_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 299288024829984050, "histo_col_type": "", "name": "__auto__", "null_count": 6136008001012804835, "row_count": 6532273077833447112}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 726122874837404391, "histo_col_type": "", "name": "__auto__", "null_count": 4351873875494926448, "row_count": 6532273077833447112}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1389924541332361714, "histo_col_type": "", "name": "__auto__", "null_count": 1821524791946346857, "row_count": 6532273077833447112}, {"columns": ["col3_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4839767865390016364, "histo_col_type": "", "name": "__auto__", "null_count": 5221717345179898881, "row_count": 6532273077833447112}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4184789158132340791, "histo_buckets": [{"distinct_range": 0, "num_eq": 8284971274115615053, "num_range": 0, "upper_bound": "1970-09-09"}, {"distinct_range": 2707198190479519000, "num_eq": 1707096764083012591, "num_range": 4237603701962025735, "upper_bound": "1975-10-09"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 2471869526168319962, "row_count": 6532273077833447112}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6152610317226959854, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6532273077833447112}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1653762000984245311, "histo_col_type": "", "name": "__auto__", "null_count": 2550052723415222835, "row_count": 6532273077833447112}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4055143568161850182, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6532273077833447112}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4331315822116253019, "histo_col_type": "", "name": "__auto__", "null_count": 2214837196141370719, "row_count": 6532273077833447112}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6132348027693218007, "histo_buckets": [{"distinct_range": 0, "num_eq": 8658156406923877424, "num_range": 0, "upper_bound": ""}, {"distinct_range": 1033840292937878900, "num_eq": 7316791986763358806, "num_range": 2802059273984734354, "upper_bound": "\\t\\u001e"}, {"distinct_range": 6823117285806457000, "num_eq": 60, "num_range": 7639574356950035303, "upper_bound": "6F\\u001dcm*<"}, {"distinct_range": 7625359171013195000, "num_eq": 80000000000, "num_range": 7625359171013195179, "upper_bound": "8;\\u0001\\u0013RAM"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "JHJ22i\\r("}, {"distinct_range": 69283091744.50928, "num_eq": 2493413667354713088, "num_range": 100000000000, "upper_bound": "P@\\u0017W"}, {"distinct_range": 0, "num_eq": 3697468248105988243, "num_range": 50000000000, "upper_bound": "X\\u001dr\\u001du."}, {"distinct_range": 0, "num_eq": 6679208213141794012, "num_range": 70000000, "upper_bound": "w"}, {"distinct_range": 0, "num_eq": 4695329995299471524, "num_range": 8289108698247978631, "upper_bound": "\U00002603"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 6532273077833447112}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 971434816405998572, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6532273077833447112}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3273627443173290119, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6532273077833447112}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3243120053171007539, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000, "num_range": 0, "upper_bound": "741400758"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 8000, "upper_bound": "2538837342"}, {"distinct_range": 0, "num_eq": 3167977788744510573, "num_range": 0, "upper_bound": "3612534750"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 6532273077833447112}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2089387809926618842, "histo_col_type": "", "name": "__auto__", "null_count": 3353068156714889889, "row_count": 6532273077833447112}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4050225235549427342, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000, "num_range": 0, "upper_bound": "-0.6690963506698608"}, {"distinct_range": 4871753648608125000, "num_eq": 50000, "num_range": 4871753648608124562, "upper_bound": "-0.17608989775180817"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 325785399249433912, "row_count": 6532273077833447112}]';
        test_1        |             ALTER TABLE table4 ADD FOREIGN KEY (col1_1, col1_2, col4_14) REFERENCES table2 (col1_1, col1_2, col2_7);
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col1_1) REFERENCES table4 (col1_1);
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col2_9) REFERENCES table2 (col2_9);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1153067385204932814, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000000, "num_range": 0, "upper_bound": "1978-09-23 17:20:55.00081+00:00"}, {"distinct_range": 43446.3895646748, "num_eq": 50, "num_range": 60000, "upper_bound": "1986-10-30 17:06:43.000175+00:00"}, {"distinct_range": 0, "num_eq": 7519223040093508982, "num_range": 7128993374793886098, "upper_bound": "2001-11-28 21:31:33.000447+00:00"}, {"distinct_range": 170822916700019330, "num_eq": 8479938037580204745, "num_range": 6140197187130628880, "upper_bound": "2010-03-13 09:11:58.00014+00:00"}, {"distinct_range": 34.58779185485453, "num_eq": 7259831595753465379, "num_range": 60, "upper_bound": "2010-04-30 02:16:01.000042+00:00"}, {"distinct_range": 3000000000, "num_eq": 200000, "num_range": 3000000000, "upper_bound": "2012-08-21 23:09:10.000507+00:00"}, {"distinct_range": 0, "num_eq": 4956344178982735184, "num_range": 770575886405014625, "upper_bound": "2029-08-10 22:17:08.000882+00:00"}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 6590527348122192239, "upper_bound": "3000-01-01 00:00:00+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 1892189245967080688}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1799924725096887069, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1892189245967080688}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1031695281418557440, "histo_buckets": [{"distinct_range": 0, "num_eq": 8447409326053564112, "num_range": 0, "upper_bound": ""}, {"distinct_range": 157095072215383900, "num_eq": 3586093561262602099, "num_range": 207656102120245551, "upper_bound": "\\u0004q"}, {"distinct_range": 2445001666166502400, "num_eq": 5522776959589497474, "num_range": 2445001666166502409, "upper_bound": "IJY{{\\u0005p"}, {"distinct_range": 1590345192881767400, "num_eq": 201356331008692574, "num_range": 3995922524457092878, "upper_bound": "N1\\t;"}, {"distinct_range": 1783163.3365616081, "num_eq": 800000, "num_range": 3000000, "upper_bound": "V\\u001a\\u0019%\\u000c@6"}, {"distinct_range": 50, "num_eq": 4437251430947921819, "num_range": 50, "upper_bound": "^"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 1892189245967080688}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 765560765529713031, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 30000000, "num_eq": 148201841637449966, "num_range": 30000000, "upper_bound": "1275513653"}, {"distinct_range": 7378287705013462000, "num_eq": 2591938317919799354, "num_range": 7378287705013461739, "upper_bound": "1962078658"}, {"distinct_range": 0, "num_eq": 82781113062387432, "num_range": 0, "upper_bound": "2384876715"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 2309622518321851890, "upper_bound": "4285538890"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 1892189245967080688}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 200906941563374494, "histo_buckets": [{"distinct_range": 0, "num_eq": 7013644748206381400, "num_range": 0, "upper_bound": "1978-06-03 05:54:51.000998+00:00"}, {"distinct_range": 0, "num_eq": 1413827133215755248, "num_range": 0, "upper_bound": "1992-04-19 23:36:33.000053+00:00"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 1124327833346336119, "upper_bound": "1996-03-09 08:25:49.000235+00:00"}, {"distinct_range": 535986920735978400, "num_eq": 5488128442054903406, "num_range": 535986920735978338, "upper_bound": "1997-10-23 21:52:12.000528+00:00"}, {"distinct_range": 4532694179736748500, "num_eq": 6000000, "num_range": 4532694179736748556, "upper_bound": "1998-03-12 23:49:08.00096+00:00"}, {"distinct_range": 238873734835238400, "num_eq": 7181990190902419191, "num_range": 8530092395082230480, "upper_bound": "2001-02-23 11:35:48.000452+00:00"}, {"distinct_range": 2141913578590343000, "num_eq": 0, "num_range": 2141913578590342798, "upper_bound": "2015-06-04 05:22:13.00082+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 1892189245967080688}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6263733157, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5168526154, "histo_col_type": "", "name": "__auto__", "null_count": 3666772309, "row_count": 8000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1056476036, "histo_buckets": [{"distinct_range": 0, "num_eq": 5507438914474713095, "num_range": 0, "upper_bound": ""}, {"distinct_range": 116917.48156874446, "num_eq": 4146442082808227421, "num_range": 300000, "upper_bound": "\\u0003u\\u0006U\\u0017Q\\u001c"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 3843797044821637905, "upper_bound": "lN)Yt"}, {"distinct_range": 1000000000, "num_eq": 4888957348609599348, "num_range": 1000000000, "upper_bound": "yr1C\\"\\u0019"}, {"distinct_range": 1419420993902583000, "num_eq": 0, "num_range": 1419420993902583041, "upper_bound": "|n\\u0015\\u000c./L\\u0014"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1529235180, "histo_buckets": [{"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "09:25:43.344981+09:07:00"}, {"distinct_range": 2745639000232832500, "num_eq": 3000000000, "num_range": 2745639000232832331, "upper_bound": "09:51:54.544755+03:11:00"}, {"distinct_range": 41876322.915328436, "num_eq": 30, "num_range": 80000000, "upper_bound": "05:57:12.26207-02:26:00"}, {"distinct_range": 0, "num_eq": 5278948728981582519, "num_range": 4099144706217832665, "upper_bound": "11:28:56.425774+03:02:00"}, {"distinct_range": 1000, "num_eq": 50, "num_range": 1000, "upper_bound": "11:08:52.981588+01:14:00"}, {"distinct_range": 319.7125904008106, "num_eq": 4960203929796232902, "num_range": 700, "upper_bound": "03:41:08.013008-06:19:00"}, {"distinct_range": 0, "num_eq": 831922084190278712, "num_range": 6483560230138224425, "upper_b

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2566789&tab=buildLog) on [master@339275585b7d30b9ee2d49b0c696b9ddb8d51ad4](https://github.com/cockroachdb/cockroach/commits/339275585b7d30b9ee2d49b0c696b9ddb8d51ad4):

```
        postgres_1    | 		col_96522 NULLS FIRST
        test_1        | --- FAIL: TestCompare (55.41s)
        test_1        |     --- FAIL: TestCompare/postgres (54.76s)
        test_1        |         compare_test.go:155: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("2000-01-01 03:00:00D"),
        test_1        |             + 	string("2000-01-01 03:00:00+00D"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('2000-01-01 03:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ || 'D'::TEXT::TEXT)::TEXT AS col_96536)
        test_1        |             		AS tab_49432
        test_1        |             ORDER BY
        test_1        |             	col_96536 NULLS FIRST;
        test_1        |             
        test_1        |             postgres: [same as previous]
        test_1        |             
        test_1        |     --- FAIL: TestCompare/mutators (0.65s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 BOOL NOT NULL, col1_1 REGPROCEDURE NOT NULL, col1_2 BYTES NULL, col1_3 REGNAMESPACE NOT NULL, col1_4 REGPROC, col1_5 INT8 NULL, col1_6 INTERVAL NOT NULL, col1_7 BYTES NULL, col1_8 TIME NOT NULL, col1_9 REGPROCEDURE NULL, col1_10 DECIMAL NULL, PRIMARY KEY (col1_6, col1_1), UNIQUE (col1_6, col1_2 ASC, col1_1 DESC, col1_5 ASC, col1_9, col1_8 ASC) STORING (col1_3, col1_4, col1_7, col1_10), INDEX (col1_8 DESC, col1_4 ASC, col1_5, col1_1 ASC, col1_6 DESC, col1_9, col1_2, col1_7 DESC, col1_3, col1_0 DESC) WHERE ((((table1.col1_0 OR (table1.col1_8 > '00:00:00':::TIME)) OR (table1.col1_10 = 'NaN':::DECIMAL)) OR (table1.col1_7 != e'\\x':::BYTES)) OR (table1.col1_5 > 9223372036854775807:::INT8)) AND (table1.col1_2 < e'\\x58':::BYTES), UNIQUE (col1_3 ASC, col1_9, col1_4, col1_10, col1_8 DESC, col1_1 DESC), UNIQUE (col1_8 ASC, col1_3 ASC, col1_2 DESC, col1_0, col1_10 ASC, col1_4 DESC) STORING (col1_5) WHERE (((((table1.col1_8 <= '24:00:00':::TIME) OR table1.col1_0) AND (table1.col1_10 <= 'NaN':::DECIMAL)) AND (table1.col1_2 >= e'\\x':::BYTES)) OR (table1.col1_7 != e'\\x':::BYTES)) AND (table1.col1_5 <= 2147483647:::INT8), INDEX (col1_1 ASC, col1_10, col1_4 DESC, col1_8, col1_0 DESC, col1_5 ASC, col1_6 ASC, col1_3 ASC, col1_9 DESC, col1_7 ASC) STORING (col1_2) WHERE ((((table1.col1_0 OR (table1.col1_8 <= '00:00:00')) OR (table1.col1_5 != 9223372036854775807)) AND (table1.col1_2 < '\x27')) OR (table1.col1_10 < -1)) OR (table1.col1_7 > '\x00'), UNIQUE (col1_9 ASC, col1_5 ASC, col1_2 DESC, col1_1, col1_6, col1_0 ASC, col1_7 DESC, col1_4 DESC) STORING (col1_3, col1_8) WHERE (((table1.col1_0 OR (table1.col1_2 != e'\\x00':::BYTES)) OR (table1.col1_8 <= '00:00:00':::TIME)) AND (table1.col1_10 = (-1.234E+401):::DECIMAL)) AND (table1.col1_5 != 32767:::INT8), UNIQUE (col1_3 ASC, col1_1, col1_4 ASC, col1_7, col1_2 DESC, col1_5 DESC, col1_6, col1_0 ASC, col1_10 ASC, col1_8, col1_9) WHERE table1.col1_0 AND (table1.col1_10 = -Infinity), UNIQUE (col1_1) STORING (col1_0, col1_2, col1_4, col1_7, col1_8, col1_9), FAMILY (col1_6), FAMILY (col1_3), FAMILY (col1_8), FAMILY (col1_10), FAMILY (col1_7), FAMILY (col1_4), FAMILY (col1_2), FAMILY (col1_9, col1_1), FAMILY (col1_5), FAMILY (col1_0));
        test_1        |             CREATE TABLE table2 (col2_0 REGPROCEDURE NOT NULL, PRIMARY KEY (col2_0), INDEX (col2_0), UNIQUE (col2_0 DESC), INDEX (col2_0 ASC), INDEX (col2_0 DESC), UNIQUE (col2_0 ASC), INDEX (col2_0));
        test_1        |             CREATE TABLE table3 (col3_0 TIME NOT NULL, col3_1 NAME[] NOT NULL, col3_2 "char" NOT NULL, col3_3 TIMESTAMPTZ NOT NULL, col3_4 VARCHAR NOT NULL, col3_5 VARCHAR NOT NULL, col3_6 STRING NOT NULL, col3_7 OID NOT NULL, col3_8 DECIMAL NOT NULL, col3_9 INTERVAL NOT NULL, PRIMARY KEY (col3_8 DESC, col3_7, col3_4 DESC, col3_9 ASC, col3_2, col3_0 ASC, col3_6, col3_3 ASC), UNIQUE (col3_3, col3_7 ASC, col3_5 DESC, col3_6, col3_4 DESC, col3_8 ASC, col3_2 ASC) WHERE (table3.col3_4 > '"':::STRING) OR (table3.col3_5 <= e'\'':::STRING), UNIQUE (col3_6 ASC, col3_0 ASC, col3_2 ASC, col3_8 DESC, col3_7 ASC, col3_4 DESC, col3_9) WHERE ((((((table3.col3_5 < '"':::STRING) OR (table3.col3_0 < '24:00:00':::TIME)) AND (table3.col3_2 < '':::STRING)) OR (table3.col3_4 != e'\x00':::STRING)) AND (table3.col3_8 <= '-Infinity':::DECIMAL)) AND (table3.col3_6 != e'\'':::STRING)) OR (table3.col3_3 < '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ), INDEX (col3_6 ASC, col3_9 DESC, col3_5 DESC, col3_2 ASC, col3_7 ASC, col3_8 ASC, col3_3 DESC), UNIQUE (col3_7 ASC, col3_8, col3_9 ASC, col3_0, col3_6 DESC) STORING (col3_1, col3_5), UNIQUE (col3_4, col3_5 ASC, col3_7 DESC), UNIQUE (col3_7 ASC, col3_6, col3_2 DESC, col3_5 ASC, col3_8, col3_9 DESC, col3_4 DESC, col3_3 DESC, col3_0 DESC) STORING (col3_1) WHERE (table3.col3_3 <= '294276-12-31 23:59:59.999999+00:00') OR (table3.col3_8 != NaN), FAMILY (col3_1, col3_7), FAMILY (col3_2), FAMILY (col3_8), FAMILY (col3_9, col3_4, col3_3, col3_5, col3_0, col3_6));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6717677160, "histo_buckets": [{"distinct_range": 0, "num_eq": 9174087366438870603, "num_range": 0, "upper_bound": "281702248"}, {"distinct_range": 53963336611.33906, "num_eq": 7674082731366355580, "num_range": 60000000000, "upper_bound": "1614696592"}, {"distinct_range": 0, "num_eq": 1682239506977913446, "num_range": 1792178071546332401, "upper_bound": "1804823089"}, {"distinct_range": 78658334661631660, "num_eq": 2000, "num_range": 240215756463301174, "upper_bound": "1939606154"}, {"distinct_range": 661534842513227400, "num_eq": 400000000, "num_range": 4761565894620027629, "upper_bound": "4068572409"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3047378184, "histo_col_type": "", "name": "__auto__", "null_count": 1377654178, "row_count": 7000000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1971193944, "histo_buckets": [{"distinct_range": 0, "num_eq": 500, "num_range": 0, "upper_bound": "-74 years -8 mons -968 days -05:06:53.914772"}, {"distinct_range": 6415154.736592987, "num_eq": 40000000000, "num_range": 7000000, "upper_bound": "-59 years -2 mons -199 days -13:39:59.629155"}, {"distinct_range": 0, "num_eq": 6194930883657248656, "num_range": 3362107183441837146, "upper_bound": "-27 years -10 mons -23 days -17:31:32.076959"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 6413422750712189274, "upper_bound": "-15 years -8 mons -663 days -19:50:46.048073"}, {"distinct_range": 0, "num_eq": 300000, "num_range": 62792741573183708, "upper_bound": "-3 mons -479 days -24:44:07.477337"}, {"distinct_range": 0, "num_eq": 5306517223783747302, "num_range": 3885608066446947388, "upper_bound": "82 years 11 mons 356 days 18:18:16.520862"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9700005, "histo_col_type": "", "name": "__auto__", "null_count": 6365489532, "row_count": 7000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 123821092, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3142875859, "histo_buckets": [{"distinct_range": 0, "num_eq": 666797764157126445, "num_range": 0, "upper_bound": "2131160133"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 264132608, "histo_col_type": "", "name": "__auto__", "null_count": 5403492558, "row_count": 7000000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5627942059, "histo_buckets": [{"distinct_range": 0, "num_eq": 92091346558024215, "num_range": 0, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 614347975, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 4137522954410333000, "num_eq": 500000, "num_range": 4137522954410333060, "upper_bound": "861780767"}, {"distinct_range": 0, "num_eq": 50000, "num_range": 1000000000, "upper_bound": "1373806825"}, {"distinct_range": 8302165927076079000, "num_eq": 6554499255009277344, "num_range": 8302165927076079055, "upper_bound": "1900842162"}, {"distinct_range": 0, "num_eq": 20, "num_range": 70000000000, "upper_bound": "2302406821"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 0, "upper_bound": "2756753054"}, {"distinct_range": 1000000, "num_eq": 80000, "num_range": 1000000, "upper_bound": "3606010370"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 1807945724, "row_count": 7000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 847981170, "histo_col_type": "", "name": "__auto__", "null_count": 370122458, "row_count": 7000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5830442254, "histo_col_type": "", "name": "__auto__", "null_count": 2605466300, "row_count": 7000000000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20, "histo_buckets": [{"distinct_range": 0, "num_eq": 5239228999772356217, "num_range": 0, "upper_bound": "2190825036"}, {"distinct_range": 0, "num_eq": 50000, "num_range": 7396146314503372085, "upper_bound": "2198891976"}, {"distinct_range": 3125790730451351600, "num_eq": 50000000, "num_range": 7844048673659156597, "upper_bound": "3937160654"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 60}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30874674339099455, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9157289593161530477}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4522612876700448200, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000000, "num_range": 0, "upper_bound": "kY\\u0001"}, {"distinct_range": 0, "num_eq": 295181285288937647, "num_range": 9000, "upper_bound": "\U00002603"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 9157289593161530477}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7594908513156818910, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000, "num_range": 0, "upper_bound": "116028561"}, {"distinct_range": 8903.475280933842, "num_eq": 900000000, "num_range": 60000, "upper_bound": "129566179"}, {"distinct_range": 3748354411165911600, "num_eq": 10000000, "num_range": 8447997503197001106, "upper_bound": "480689247"}, {"distinct_range": 6647819542717853000, "num_eq": 7671580742047123639, "num_range": 6647819542717852791, "upper_bound": "2033598352"}, {"distinct_range": 0, "num_eq": 8025310824845398556, "num_range": 6019859577750513837, "upper_bound": "3177296770"}, {"distinct_range": 486838743675834400, "num_eq": 4000000000, "num_range": 486838743675834348, "upper_bound": "4213234579"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 9157289593161530477}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 767152462063560527, "histo_buckets": [{"distinct_range": 0, "num_eq": 2292323002616578396, "num_range": 0, "upper_bound": "-6.761443259001637432E+30"}, {"distinct_range": 40585.44427969509, "num_eq": 400000000, "num_range": 70000, "upper_bound": "-32321975561829998.08"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 9157289593161530477}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1027444851063028717, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9157289593161530477}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 380823113036251995, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9157289593161530477}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6583419017837249783, "histo_buckets": [{"distinct_range": 0, "num_eq": 30, "num_range": 0, "upper_bound": "\\u0008\\u0003\\\\\\u0005\\u0004~%"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 9157289593161530477}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4156667029175355236, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9157289593161530477}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7479918471348537624, "histo_buckets": [{"distinct_range": 0, "num_eq": 9086685695739428349, "num_range": 0, "upper_bound": "-2000-01-01 00:00:00+00:00"}, {"distinct_range": 1947065347826255000, "num_eq": 900, "num_range": 1947065347826254993, "upper_bound": "0001-01-01 00:00:00+00:00"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 5566299825563409127, "upper_bound": "1973-01-13 15:55:50.000145+00:00"}, {"distinct_range": 10000000000, "num_eq": 60000000, "num_range": 10000000000, "upper_bound": "1983-07-26 09:23:10.000355+00:00"}, {"distinct_range": 1818720397290671000, "num_eq": 5542711852113814381, "num_range": 6686133671251963417, "upper_bound": "1983-12-09 05:53:44.000386+00:00"}, {"distinct_range": 700000000, "num_eq": 0, "num_range": 700000000, "upper_bound": "2006-05-19 15:11:32.000327+00:00"}, {"distinct_range": 876472064.8434719, "num_eq": 60000, "num_range": 5000000000, "upper_bound": "2024-04-23 18:23:37.000754+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 9157289593161530477}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1584721974165229869, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9157289593161530477}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('ucwc', 'db', 'cmebq', 'plle', 'tnlkf', 'kmcgkv');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5393290359541080247, "histo_col_type": "", "name": "__auto__", "null_count": 5093578376820362435, "row_count": 9131334601598097495}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6204862030138931968, "histo_col_type": "", "name": "__auto__", "null_count": 273590393399087642, "row_count": 9131334601598097495}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1786083321672949108, "histo_col_type": "", "name": "__auto__", "null_count": 1523218422742816180, "row_count": 9131334601598097495}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 766510654396793157, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9131334601598097495}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3374289128824067838, "histo_col_type": "", "name": "__auto__", "null_count": 722506143567360748, "row_count": 9131334601598097495}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2776023392741949479, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 5112587993178310218, "upper_bound": "2651897198"}, {"distinct_range": 0, "num_eq": 800000000, "num_range": 6000000000, "upper_bound": "2804741537"}, {"distinct_range": 8716968496.141745, "num_eq": 10000, "num_range": 40000000000, "upper_bound": "3064604677"}, {"distinct_range": 0, "num_eq": 5000000, "num_range": 4219443889442588215, "upper_bound": "3453002272"}, {"distinct_range": 200, "num_eq": 4404353111310959879, "num_range": 200, "upper_bound": "3644599444"}, {"distinct_range": 842688.3998953539, "num_eq": 800000, "num_range": 5000000, "upper_bound": "3708618202"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 9131334601598097495}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6931685089493869268, "histo_buckets": [{"distinct_range": 0, "num_eq": 80, "num_range": 0, "upper_bound": "00:00:00"}, {"distinct_range": 0, "num_eq": 300000000, "num_range": 40, "upper_bound": "01:31:32.772533"}, {"distinct_range": 500, "num_eq": 3139838338483926130, "num_range": 500, "upper_bound": "10:56:03.278779"}, {"distinct_range": 0, "num_eq": 3174630202379105784, "num_range": 0, "upper_bound": "13:02:38.322951"}, {"distinct_range": 0, "num_eq": 8109799311357076817, "num_range": 30, "upper_bound": "17:49:27.771684"}, {"distinct_range": 4738.469262290869, "num_eq": 3164363672728101287, "num_range": 6000, "upper_bound": "19:01:13.31294"}, {"distinct_range": 7603440561237676000, "num_eq": 4539281254271907522, "num_range": 8584173944768755521, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 9131334601598097495}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7883360551731127552, "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 2122290827759652560, "row_count": 9131334601598097495}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5441332118214797762, "histo_buckets": [{"distinct_range": 0, "num_eq": 4128464292899810690, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 151331579.04069462, "num_eq": 700000, "num_range": 300000000, "upper_bound": "303434377"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 5087257542444471939, "upper_bound": "1401551417"}, {"distinct_range": 90000000, "num_eq": 800000, "num_range": 90000000, "upper_bound": "1667977166"}, {"distinct_range": 1149524.8311551684, "num_eq": 8207208029179138752, "num_range": 5000000, "upper_bound": "3201408187"}, {"distinct_range": 0, "num_eq": 792466996724490558, "num_range": 10, "upper_bound": "3434509349"}, {"distinct_range": 0, "num_eq": 3246121745636951639, "num_range": 1668179645122366214, "upper_bound": "3912532488"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 9131334601598097495}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4897552673758323320, "histo_buckets": [{"distinct_range": 0, "num_eq": 1152970324927004067, "num_range": 0, "upper_bound": "-69 years -8 mons -605 days -13:41:02.709567"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 514563925543394720, "upper_bound": "-59 years -4 mons -157 days -13:25:59.833429"}, {"distinct_range": 67.48527732379758, "num_eq": 8858030868918399245, "num_range": 90, "upper_bound": "-1 years -3 mons -914 days -04:15:14.1407"}, {"distinct_range": 800, "num_eq": 10000, "num_range": 800, "upper_bound": "21 years 3 mons 242 days 24:53:28.61684"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 80000000, "upper_bound": "31 years 7 mons 176 days 24:50:57.174157"}, {"distinct_range": 87822927782542700, "num_eq": 700, "num_range": 6188821448539245639, "upper_bound": "55 years 3 mons 663 days 24:13:54.104672"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 9131334601598097495}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4975098532305857214, "histo_col_type": "", "name": "__auto__", "null_count": 335641999458095556, "row_count": 9131334601598097495}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 328148228098753315, "histo_buckets": [{"distinct_range": 0, "num_eq": 6686634637340719757, "num_range": 0, "upper_bound": "271344179"}, {"distinct_range": 402147140.9105232, "num_eq": 70, "num_range": 1000000000, "upper_bound": "372977615"}, {"distinct_range": 304071.03516569803, "num_eq": 8000000, "num_range": 2000000, "upper_bound": "841814154"}, {"distinct_range": 1950627842719656200, "num_eq": 2449581470414289270, "num_range": 6000831231811872330, "upper_bound": "1088695120"}, {"distinct_range": 3124731929582659600, "num_eq": 8000000, "num_range": 3124731929582659575, "upper_bound": "2488150771"}, {"distinct_range": 0, "num_eq": 8000, "num_range": 1000000, "upper_bound": "3273408016"}, {"distinct_range": 7729160957472097000, "num_eq": 1677913919427592758, "num_range": 7729160957472097357, "upper_bound": "3690968542"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 6291542067886190224}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1085215, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7348873, "histo_buckets": [{"distinct_range": 0, "num_eq": 60, "num_range": 0, "upper_bound": "731019802"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3288812, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 550152, "histo_buckets": [{"distinct_range": 0, "num_eq": 7472438963757549621, "num_range": 0, "upper_bound": "2017-07-20 08:12:10.000761+00:00"}, {"distinct_range": 31210755358.991837, "num_eq": 6222925796637790897, "num_range": 70000000000, "upper_bound": "2027-08-18 08:41:00.000404+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8160306, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 7000, "num_range": 1263288534368257082, "upper_bound": "\\u0005\\u000b\\u0019\\u000cH^j"}, {"distinct_range": 0, "num_eq": 2884212452601024955, "num_range": 0, "upper_bound": "\\u001e(O"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1927179, "histo_buckets": [{"distinct_range": 0, "num_eq": 2526353851679915337, "num_range": 0, "upper_bound": "\\u0014yJBe\\u0015;g"}, {"distinct_range": 10000000, "num_eq": 7483692180686740253, "num_range": 10000000, "upper_bound": "\\u001bJ~C\\tx"}, {"distinct_range": 0, "num_eq": 300000000, "num_range": 10, "upper_bound": "VXgU8<"}, {"distinct_range": 5140025908.529859, "num_eq": 50000000, "num_range": 6000000000, "upper_bound": "xQK"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2968142, "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 101591, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8413162, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3362092, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}]';
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_0) REFERENCES table1 (col1_1) ON DELETE CASCADE;
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2426812, "histo_col_type": "", "name": "__auto__", "null_count": 2475139, "row_count": 4000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1051894, "histo_col_type": "", "name": "__auto__", "null_count": 1429336, "row_count": 4000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2755828, "histo_col_type": "", "name": "__auto__", "null_count": 3869759, "row_count": 4000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3642231, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 95393, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 400, "num_eq": 900000000, "num_range": 400, "upper_bound": "201928668"}, {"distinct_range": 0, "num_eq": 1552024198064900625, "num_range": 5559793932057505093, "upper_bound": "2960427303"}, {"distinct_range": 19802.522112260547, "num_eq": 2000000, "num_range": 30000, "upper_bound": "3933034154"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3815471, "histo_buckets": [{"distinct_range": 0, "num_eq": 2403121338132558735, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 2178864571246687618, "upper_bound": "591294284"}, {"distinct_range": 10000, "num_eq": 8453203596596474507, "num_range": 10000, "upper_bound": "829238205"}, {"distinct_range": 8000000000, "num_eq": 4088712881187379967, "num_range": 8000000000, "upper_bound": "841608976"}, {"distinct_range": 723.4125583741169, "num_eq": 8495700097516514886, "num_range": 2000, "upper_bound": "930064671"}, {"distinct_range": 819807962.517655, "num_eq": 0, "num_range": 6000000000, "upper_bound": "933694515"}, {"distinct_range": 2241930445647848400, "num_eq": 60000, "num_range": 2241930445647848323, "upper_bound": "1557429929"}, {"distinct_range": 6043740020150207000, "num_eq": 5074454320867030732, "num_range": 6043740020150207737, "upper_bound": "2712433577"}, {"distinct_range": 42.07270008000133, "num_eq": 2982701476847099437, "num_range": 60, "upper_bound": "3146836255"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 505300, "histo_col_type": "", "name": "__auto__", "null_count": 707895, "row_count": 4000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3500514, "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3688484, "histo_col_type": "", "name": "__auto__", "null_count": 3966457, "row_count": 4000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3362371, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": "00:00:00"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 8743402429868277475, "upper_bound": "04:19:58.334984"}, {"distinct_range": 8806010278449409000, "num_eq": 10000000000, "num_range": 8806010278449408705, "upper_bound": "04:42:30.896079"}, {"distinct_range": 6346194661965476000, "num_eq": 40000000000, "num_range": 6346194661965476329, "upper_bound": "08:16:53.543261"}, {"distinct_range": 2557822014648244700, "num_eq": 200, "num_range": 3903779971224350482, "upper_bound": "09:43:33.036888"}, {"distinct_range": 0, "num_eq": 3986052016562887538, "num_range": 2000000, "upper_bound": "14:57:04.505551"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "20:40:42.588504"}, {"distinct_range": 1718266091678125800, "num_eq": 50000000000, "num_range": 1718266091678125783, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 351089, "histo_buckets": [{"distinct_range": 0, "num_eq": 4236934889560602371, "num_range": 0, "upper_bound": "219922883"}, {"distinct_range": 2026280064896302000, "num_eq": 5256288344187018752, "num_range": 4378324547886459462, "upper_bound": "219959591"}, {"distinct_range": 7992374407790602000, "num_eq": 10, "num_range": 7992374407790602701, "upper_bound": "1951733303"}, {"distinct_range": 0, "num_eq": 80, "num_range": 300000, "upper_bound": "3360158964"}, {"distinct_range": 4000, "num_eq": 4693656395173003546, "num_range": 4000, "upper_bound": "3590319075"}, {"distinct_range": 100000000, "num_eq": 3214502363036062806, "num_range": 100000000, "upper_bound": "3707697442"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 1692013, "row_count": 4000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17488265164, "histo_buckets": [{"distinct_range": 0, "num_eq": 8228786973596182203, "num_range": 0, "upper_bound": "3621542981"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 40000000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 648, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000000, "num_range": 0, "upper_bound": "\\u000cR\\u0017\\u001bIR"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 3000000, "upper_bound": "\\u0017 2"}, {"distinct_range": 0, "num_eq": 8685404035467591050, "num_range": 400000000, "upper_bound": "V5;x\\u0004tgx"}, {"distinct_range": 0, "num_eq": 6508849202221486075, "num_range": 400, "upper_bound": "]7!0M\\u000c"}, {"distinct_range": 0, "num_eq": 1916506792056809564, "num_range": 300000000, "upper_bound": "^\\u0006\\tw"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 773326555112457989, "upper_bound": "^:HX"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 628, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 0, "num_eq": 8406788776594063823, "num_range": 0, "upper_bound": "KwaEO\\\\m"}, {"distinct_range": 3768705132182736000, "num_eq": 400000, "num_range": 5283660494848277756, "upper_bound": "M@i\\u0005"}, {"distinct_range": 8000000000, "num_eq": 3775158106320120394, "num_range": 8000000000, "upper_bound": "_"}, {"distinct_range": 4000, "num_eq": 7755495147802382506, "num_range": 4000, "upper_bound": "s\\u0015Y\\u0004"}, {"distinct_range": 7000, "num_eq": 100, "num_range": 7000, "upper_bound": "yS#"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 631, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 315, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 538, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 382, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 90, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000, "num_range": 0, "upper_bound": "1975-12-25 22:50:16.000803+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 653, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 10, "num_range": 0, "upper_bound": "2757916806"}, {"distinct_range": 2066389275494269400, "num_eq": 3936396340754959, "num_range": 5349072779063427612, "upper_bound": "3822050266"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 456, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000, "num_range": 0, "upper_bound": "-276278940130000525.8"}, {"distinct_range": 6830838152466916000, "num_eq": 80000000, "num_range": 6830838152466916529, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 200, "num_range": 100, "upper_bound": "9030570638953695293"}, {"distinct_range": 0, "num_eq": 6380000552618873683, "num_range": 70, "upper_bound": "1.710114620129178299E+35"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 700}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "infinity" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2569741&tab=buildLog) on [master@dbc7245c5d8c9f009072353fec261419e573032c](https://github.com/cockroachdb/cockroach/commits/dbc7245c5d8c9f009072353fec261419e573032c):

```
        postgres_1    |     pg_ctl -D /var/lib/postgresql/data -l logfile start
        postgres_1    | 
        postgres_1    | waiting for server to start....2021-01-09 06:12:40.620 UTC [47] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-01-09 06:12:40.636 UTC [48] LOG:  database system was shut down at 2021-01-09 06:12:40 UTC
        postgres_1    | 2021-01-09 06:12:40.641 UTC [47] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2021-01-09 06:12:43.424 UTC [47] LOG:  received fast shutdown request
        postgres_1    | .2021-01-09 06:12:43.426 UTC [47] LOG:  aborting any active transactions
        postgres_1    | 2021-01-09 06:12:43.428 UTC [47] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
        postgres_1    | 2021-01-09 06:12:43.434 UTC [49] LOG:  shutting down
        postgres_1    | 2021-01-09 06:12:43.550 UTC [47] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-01-09 06:12:43.636 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-01-09 06:12:43.636 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-01-09 06:12:43.641 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-01-09 06:12:43.657 UTC [84] LOG:  database system was shut down at 2021-01-09 06:12:43 UTC
        postgres_1    | 2021-01-09 06:12:43.663 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (603.41s)
        test_1        |     --- FAIL: TestCompare/postgres (1.74s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.2:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2571222&tab=buildLog) on [master@aa68427e69113a6b9d0aa1dac8469919c6f95e16](https://github.com/cockroachdb/cockroach/commits/aa68427e69113a6b9d0aa1dac8469919c6f95e16):

```
        postgres_1    | You can change this by editing pg_hba.conf or using the option -A, or
        postgres_1    | --auth-local and --auth-host, the next time you run initdb.
        postgres_1    | waiting for server to start....2021-01-11 06:12:40.706 UTC [47] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-01-11 06:12:40.723 UTC [48] LOG:  database system was shut down at 2021-01-11 06:12:40 UTC
        postgres_1    | 2021-01-11 06:12:40.727 UTC [47] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | 2021-01-11 06:12:43.544 UTC [47] LOG:  received fast shutdown request
        postgres_1    | waiting for server to shut down....2021-01-11 06:12:43.545 UTC [47] LOG:  aborting any active transactions
        postgres_1    | 2021-01-11 06:12:43.547 UTC [47] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
        postgres_1    | 2021-01-11 06:12:43.553 UTC [49] LOG:  shutting down
        postgres_1    | 2021-01-11 06:12:43.646 UTC [47] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-01-11 06:12:43.756 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-01-11 06:12:43.756 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-01-11 06:12:43.759 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-01-11 06:12:43.775 UTC [84] LOG:  database system was shut down at 2021-01-11 06:12:43 UTC
        postgres_1    | 2021-01-11 06:12:43.781 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (602.03s)
        test_1        |     --- FAIL: TestCompare/postgres (1.27s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2574823&tab=buildLog) on [master@63fba7f1ab321088619d04aa9a44e61d018016f6](https://github.com/cockroachdb/cockroach/commits/63fba7f1ab321088619d04aa9a44e61d018016f6):

```
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        test_1        | --- FAIL: TestCompare (3.06s)
        test_1        |     --- FAIL: TestCompare/postgres (2.12s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.93s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 TIME NOT NULL, col1_1 STRING, col1_2 INT2 NOT NULL, col1_3 OID NOT NULL, col1_4 DECIMAL NOT NULL, col1_5 DATE NOT NULL, col1_6 "char" NOT NULL, col1_7 TIMETZ NOT NULL, col1_8 INTERVAL NULL, col1_9 REGPROCEDURE, col1_10 TIMETZ, col1_11 DECIMAL NOT NULL, col1_12 NAME, col1_13 CHAR[] NULL, col1_14 DECIMAL, col1_15 TIMESTAMPTZ NULL, col1_16 DATE, PRIMARY KEY (col1_7 DESC, col1_0), INDEX (col1_15, col1_5, col1_9, col1_11 ASC, col1_14 ASC, col1_7 DESC, col1_8 ASC, col1_1, col1_3, col1_16 ASC, col1_4 DESC, col1_10, col1_6 ASC, col1_12, col1_0 ASC) STORING (col1_2) WHERE ((table1.col1_14 < 1) AND (table1.col1_11 > -1)) OR (table1.col1_16 > '-infinity'), INDEX (col1_2 ASC, col1_16 DESC, col1_11 ASC, col1_7 ASC, col1_4 DESC, col1_8, col1_10, col1_15 ASC, col1_0 DESC, col1_3, col1_14 ASC, col1_5 DESC, col1_12, col1_9 DESC) STORING (col1_6, col1_13) WHERE ((((((((((table1.col1_11 >= 0:::DECIMAL) OR (table1.col1_16 > '1970-01-01':::DATE)) OR (table1.col1_5 > '-infinity':::DATE)) AND (table1.col1_1 < e'\'':::STRING)) AND (table1.col1_0 <= '00:00:00':::TIME)) AND (table1.col1_12 != e'\x00':::STRING)) AND (table1.col1_2 = (-1):::INT8)) AND (table1.col1_10 <= '24:00:00-15:59:00':::TIMETZ)) AND (table1.col1_15 <= '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table1.col1_7 < '00:00:00+15:59:00':::TIMETZ)) OR (table1.col1_4 > 'NaN':::DECIMAL), UNIQUE (col1_16, col1_4 DESC, col1_8, col1_2 DESC, col1_6 ASC, col1_0 ASC, col1_15, col1_3, col1_11 DESC, col1_10 ASC, col1_1 ASC, col1_9, col1_14 ASC, col1_7, col1_5, col1_12 ASC) WHERE ((((((((((((table1.col1_1 > '':::STRING) AND (table1.col1_14 <= '-Infinity':::DECIMAL)) AND (table1.col1_0 > '00:00:00':::TIME)) AND (table1.col1_16 < '4714-11-24 BC':::DATE)) AND (table1.col1_15 <= '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) OR (table1.col1_7 < '00:00:00+15:59:00':::TIMETZ)) OR (table1.col1_12 = e'\x00':::STRING)) OR (table1.col1_5 = '4714-11-24 BC':::DATE)) AND (table1.col1_11 >= (-1):::DECIMAL)) OR (table1.col1_4 <= (-1.234E+401):::DECIMAL)) AND (table1.col1_6 <= e'\U00002603':::STRING)) AND (table1.col1_2 < (-1):::INT8)) AND (table1.col1_10 <= '24:00:00-15:59:00':::TIMETZ), INDEX (col1_15 DESC, col1_16 DESC, col1_14, col1_10 DESC, col1_4 DESC, col1_11 DESC, col1_5 DESC) STORING (col1_1, col1_3, col1_6, col1_9, col1_12) WHERE ((((((((((((table1.col1_14 > 1:::DECIMAL) OR (table1.col1_10 != '00:00:00+15:59:00':::TIMETZ)) OR (table1.col1_16 >= '-infinity':::DATE)) AND (table1.col1_7 >= '00:00:00+15:59:00':::TIMETZ)) OR (table1.col1_12 < e'\'':::STRING)) OR (table1.col1_5 < '-infinity':::DATE)) AND (table1.col1_11 >= 'NaN':::DECIMAL)) AND (table1.col1_0 < '00:00:00':::TIME)) OR (table1.col1_4 <= 1:::DECIMAL)) OR (table1.col1_15 > '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table1.col1_6 > e'\x00':::STRING)) OR (table1.col1_1 != '"':::STRING)) OR (table1.col1_2 >= (-32768):::INT8), INDEX (col1_8 ASC, col1_14 ASC, col1_9 DESC, col1_2 ASC, col1_15, col1_6, col1_4, col1_10, col1_0 ASC, col1_16 ASC, col1_3, col1_11, col1_5, col1_1 ASC) STORING (col1_13) WHERE ((((((((((((table1.col1_6 != '':::STRING) OR (table1.col1_12 <= e'\U00002603':::STRING)) AND (table1.col1_4 < 'NaN':::DECIMAL)) AND (table1.col1_1 = 'X':::STRING)) OR (table1.col1_15 = '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table1.col1_14 <= '-Infinity':::DECIMAL)) AND (table1.col1_7 > '00:00:00+15:59:00':::TIMETZ)) OR (table1.col1_11 > '-Infinity':::DECIMAL)) AND (table1.col1_10 = '00:00:00+15:59:00':::TIMETZ)) AND (table1.col1_2 >= (-32768):::INT8)) AND (table1.col1_5 <= '-infinity':::DATE)) AND (table1.col1_0 = '24:00:00':::TIME)) OR (table1.col1_16 > '5874897-12-31':::DATE), FAMILY (col1_0, col1_16, col1_11), FAMILY (col1_9), FAMILY (col1_3), FAMILY (col1_10, col1_5), FAMILY (col1_12, col1_4), FAMILY (col1_14), FAMILY (col1_2, col1_15), FAMILY (col1_1), FAMILY (col1_7, col1_6, col1_13), FAMILY (col1_8));
        test_1        |             CREATE TABLE table2 (col1_7 TIMETZ NOT NULL, col1_0 TIME NOT NULL, col2_2 FLOAT4, col2_3 REGTYPE, PRIMARY KEY (col1_7 DESC, col1_0, col2_2), INDEX (col2_3 ASC, col1_7 DESC), INDEX (col1_0) WHERE ((table2.col1_0 < '00:00:00':::TIME) AND (table2.col1_7 >= '00:00:00+15:59:00':::TIMETZ)) OR (table2.col2_2 > 1.0:::FLOAT8), INDEX (col2_3 DESC, col1_0 ASC, col1_7 DESC, col2_2 ASC) WHERE table2.col2_2 >= 0.0, INDEX (col2_2) WHERE (table2.col2_2 != 0.0:::FLOAT8) AND (table2.col1_0 < '00:00:00':::TIME), UNIQUE (col2_2) STORING (col2_3) WHERE ((table2.col1_0 != '24:00:00':::TIME) OR (table2.col1_7 != '00:00:00+15:59:00':::TIMETZ)) AND (table2.col2_2 >= 0.0:::FLOAT8), UNIQUE (col2_3 DESC, col1_0, col2_2) WHERE (table2.col2_2 < 1.0:::FLOAT8) AND (table2.col1_7 != '00:00:00+15:59:00':::TIMETZ), FAMILY (col1_0, col1_7), FAMILY (col2_3, col2_2)) INTERLEAVE IN PARENT table1 (col1_7, col1_0);
        test_1        |             CREATE TABLE table3 (col1_7 TIMETZ NOT NULL, col1_0 TIME NOT NULL, col2_2 FLOAT4, col3_3 OID NOT NULL, col3_4 TIME NULL, col3_5 TIME, col3_6 BOOL NULL, col3_7 DECIMAL, col3_8 REGTYPE, col3_9 TIME NOT NULL, col3_10 TIMESTAMP, col3_11 UUID NULL, PRIMARY KEY (col1_7 DESC, col1_0, col2_2, col3_11 DESC, col3_3, col3_9 ASC, col3_8 ASC, col3_4, col3_10 ASC), UNIQUE (col3_3 DESC, col2_2 DESC, col3_6 ASC) STORING (col3_5, col3_7) WHERE ((((((table3.col3_10 < '-4713-11-24 00:00:00':::TIMESTAMP) AND (table3.col2_2 > (-1.0):::FLOAT8)) AND (table3.col1_0 = '24:00:00':::TIME)) AND (table3.col1_7 <= '00:00:00+15:59:00':::TIMETZ)) AND table3.col3_6) AND (table3.col3_4 > '24:00:00':::TIME)) AND (table3.col3_9 >= '24:00:00':::TIME), INDEX (col3_6 ASC, col3_5 ASC) WHERE ((((((((table3.col3_9 = '24:00:00':::TIME) AND (table3.col3_5 < '24:00:00':::TIME)) OR (table3.col3_10 <= '-4713-11-24 00:00:00':::TIMESTAMP)) AND (table3.col1_7 > '00:00:00+15:59:00':::TIMETZ)) AND (NOT table3.col3_6)) OR (table3.col3_4 >= '24:00:00':::TIME)) OR (table3.col3_7 > 'Infinity':::DECIMAL)) OR (table3.col2_2 = 1.0:::FLOAT8)) AND (table3.col1_0 <= '24:00:00':::TIME), UNIQUE (col3_6 ASC, col3_7, col3_11 DESC) WHERE ((((((((table3.col3_10 >= '0001-01-01 00:00:00') OR (table3.col3_7 <= NaN)) AND (NOT table3.col3_6)) AND (table3.col3_9 >= '24:00:00')) OR (table3.col1_0 != '24:00:00')) OR (table3.col2_2 != +Inf)) AND (table3.col3_4 = '24:00:00')) AND (table3.col3_5 <= '00:00:00')) AND (table3.col1_7 > '00:00:00+15:59:00'), FAMILY (col1_7, col3_5), FAMILY (col3_6, col3_8), FAMILY (col3_9), FAMILY (col3_11), FAMILY (col3_10), FAMILY (col3_3), FAMILY (col3_4), FAMILY (col2_2), FAMILY (col1_0, col3_7)) INTERLEAVE IN PARENT table2 (col1_7, col1_0, col2_2);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 201103508081009020, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4664173230809089014}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 405129649305764926, "histo_col_type": "", "name": "__auto__", "null_count": 1194142388554810648, "row_count": 4664173230809089014}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4063917424216510235, "histo_col_type": "", "name": "__auto__", "null_count": 1099042602161445179, "row_count": 4664173230809089014}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3828509956349574729, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4664173230809089014}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 791387789723024369, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000000, "num_range": 0, "upper_bound": "1 day"}, {"distinct_range": 2888964201027140000, "num_eq": 5000000000, "num_range": 2888964201027140144, "upper_bound": "34 years 2 mons 868 days 15:46:48.546243"}, {"distinct_range": 3932247140945095700, "num_eq": 800, "num_range": 5137452405773896134, "upper_bound": "52 years 2 mons 890 days 18:44:08.034394"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 3172879442196006904, "row_count": 4664173230809089014}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 829020038010110937, "histo_col_type": "", "name": "__auto__", "null_count": 2874676527373931087, "row_count": 4664173230809089014}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2545480908086584037, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4664173230809089014}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2249166046939386287, "histo_col_type": "", "name": "__auto__", "null_count": 470007378752286054, "row_count": 4664173230809089014}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2343531279983854855, "histo_col_type": "", "name": "__auto__", "null_count": 1207412720702505178, "row_count": 4664173230809089014}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2991145520898998868, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4664173230809089014}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1270230601655469124, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-29215"}, {"distinct_range": 10694439816.025412, "num_eq": 1195780212956186954, "num_range": 100000000000, "upper_bound": "0"}, {"distinct_range": 3000000, "num_eq": 170218191455203153, "num_range": 3000000, "upper_bound": "25801"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 4664173230809089014}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3478210565943740566, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4664173230809089014}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 664633672093077620, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "10:49:06.96382-04:00:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 4664173230809089014}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1613318337441581904, "histo_col_type": "", "name": "__auto__", "null_count": 1201856843860364993, "row_count": 4664173230809089014}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1100163924222199302, "histo_buckets": [{"distinct_range": 0, "num_eq": 1606061131426180497, "num_range": 0, "upper_bound": "1975-06-06"}, {"distinct_range": 400000000, "num_eq": 300, "num_range": 400000000, "upper_bound": "1976-06-05"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 1362872628123239514, "upper_bound": "1978-04-22"}, {"distinct_range": 1377699798334861000, "num_eq": 20, "num_range": 7046531649238499462, "upper_bound": "1985-11-18"}, {"distinct_range": 0, "num_eq": 900, "num_range": 2048893649226207615, "upper_bound": "1989-03-13"}, {"distinct_range": 8520611390989055000, "num_eq": 10000000, "num_range": 8520611390989055102, "upper_bound": "1990-07-21"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 3994617868795562911, "row_count": 4664173230809089014}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3644251128787855257, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4664173230809089014}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2814232593933640574, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000000, "num_range": 0, "upper_bound": "1989-10-21 08:47:26.00092+00:00"}, {"distinct_range": 0, "num_eq": 6337511721572596365, "num_range": 7178176258610973068, "upper_bound": "1998-08-13 04:25:15.000998+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 986128343710014284, "row_count": 4664173230809089014}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 479, "histo_buckets": [{"distinct_range": 0, "num_eq": 1354996531984855673, "num_range": 0, "upper_bound": "-1.2744309902191162"}, {"distinct_range": 0, "num_eq": 0, "num_range": 600000, "upper_bound": "-1.229729175567627"}, {"distinct_range": 8000000, "num_eq": 80, "num_range": 8000000, "upper_bound": "-0.7315266132354736"}, {"distinct_range": 8021895591644289000, "num_eq": 201690823123500966, "num_range": 8021895591644289436, "upper_bound": "0.1977483332157135"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 174, "row_count": 600}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 105, "histo_buckets": [{"distinct_range": 0, "num_eq": 2620901456251829040, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 8506581401703615802, "num_range": 3267316423564285823, "upper_bound": "300968753"}, {"distinct_range": 0, "num_eq": 6027103615521003226, "num_range": 0, "upper_bound": "469069571"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 8456420530633615614, "upper_bound": "1914799880"}, {"distinct_range": 0, "num_eq": 4334600079160101628, "num_range": 0, "upper_bound": "2412201668"}, {"distinct_range": 7216457454692568000, "num_eq": 0, "num_range": 8381534313996355383, "upper_bound": "2594773233"}, {"distinct_range": 1101171628400755200, "num_eq": 3000, "num_range": 1408293583567231728, "upper_bound": "3129306661"}, {"distinct_range": 0, "num_eq": 4524219584260311540, "num_range": 900000, "upper_bound": "3375342440"}, {"distinct_range": 2450807284493068300, "num_eq": 7838554862757676391, "num_range": 7057980523424416696, "upper_bound": "3612963675"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 116, "row_count": 600}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 581, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "18:11:52.423492+13:05:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 600}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 322, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "00:00:00"}, {"distinct_range": 0, "num_eq": 4482941676375056716, "num_range": 800, "upper_bound": "22:29:27.472967"}, {"distinct_range": 0, "num_eq": 2526867330635545974, "num_range": 3000000, "upper_bound": "22:32:07.730508"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 600}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5113716805044401970, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5265119060373647482}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5017540802123643880, "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 5265119060373647482}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1924572003176920312, "histo_col_type": "", "name": "__auto__", "null_count": 3092709171231546199, "row_count": 5265119060373647482}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2118994458614448640, "histo_buckets": [{"distinct_range": 0, "num_eq": 5187473559271925165, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 713198943.5941014, "num_eq": 1000000000, "num_range": 800000000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 4824117020540438214, "row_count": 5265119060373647482}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4715841741190773453, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5265119060373647482}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 472744631416777426, "histo_col_type": "", "name": "__auto__", "null_count": 2097592412485768515, "row_count": 5265119060373647482}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2466865724483222658, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 700000000, "num_eq": 8172536224636495668, "num_range": 700000000, "upper_bound": "00:27:33.905484+08:45:00"}, {"distinct_range": 2013493745598516000, "num_eq": 9000, "num_range": 2013493745598515870, "upper_bound": "05:45:14.669173+12:04:00"}, {"distinct_range": 6951109.25884012, "num_eq": 100000000, "num_range": 7000000, "upper_bound": "04:50:26.830205-04:23:00"}, {"distinct_range": 812347959879610900, "num_eq": 300000, "num_range": 1594641345883800311, "upper_bound": "09:21:23.9174-12:37:00"}, {"distinct_range": 685921133064939400, "num_eq": 30000000, "num_range": 3788493756078726899, "upper_bound": "19:04:45.081045-03:39:00"}, {"distinct_range": 19542261427.485344, "num_eq": 1409943294123133107, "num_range": 60000000000, "upper_bound": "13:00:00.831373-12:43:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 5265119060373647482}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2137049567552966565, "histo_col_type": "", "name": "__auto__", "null_count": 1870077988185498883, "row_count": 5265119060373647482}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2854850848602679517, "histo_col_type": "", "name": "__auto__", "null_count": 3499895386241519429, "row_count": 5265119060373647482}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 130695101927208840, "histo_col_type": "", "name": "__auto__", "null_count": 3739589571363541317, "row_count": 5265119060373647482}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4950492514103003113, "histo_col_type": "", "name": "__auto__", "null_count": 4206182701035574909, "row_count": 5265119060373647482}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1939944058544570312, "histo_col_type": "", "name": "__auto__", "null_count": 4910095463755536664, "row_count": 5265119060373647482}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('fopj', 'gri', 'ajf', 'elcc', 'awwzc', 'hobiio');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('fdktg', 'sir', 'xssy', 'cauv', 'yp', 'sbov');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1590654099485202947, "histo_buckets": [{"distinct_range": 0, "num_eq": 7148311543257480935, "num_range": 0, "upper_bound": "-22867"}, {"distinct_range": 238838407.5464848, "num_eq": 3336981547123948497, "num_range": 400000000, "upper_bound": "4785"}, {"distinct_range": 300000, "num_eq": 8981994726570456995, "num_range": 300000, "upper_bound": "17844"}, {"distinct_range": 8000000, "num_eq": 8289273457900547391, "num_range": 8000000, "upper_bound": "27133"}, {"distinct_range": 0, "num_eq": 2049283361297567012, "num_range": 0, "upper_bound": "28743"}, {"distinct_range": 1771488163212577300, "num_eq": 5732383009166867220, "num_range": 7903333243972447431, "upper_bound": "29097"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 2630098056306889430}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2010299851648213618, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2630098056306889430}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 790118793780275452, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2630098056306889430}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 914923657547805689, "histo_col_type": "", "name": "__auto__", "null_count": 882093106103193222, "row_count": 2630098056306889430}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2068955385713047414, "histo_buckets": [{"distinct_range": 0, "num_eq": 8221250579518144468, "num_range": 0, "upper_bound": "1974-09-16"}, {"distinct_range": 0, "num_eq": 2000000, "num_range": 50000000000, "upper_bound": "1976-01-15"}, {"distinct_range": 8000000, "num_eq": 0, "num_range": 8000000, "upper_bound": "1984-01-03"}, {"distinct_range": 13072039.680535741, "num_eq": 1974159404313944668, "num_range": 20000000, "upper_bound": "1989-11-07"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 232181796128908497, "row_count": 2630098056306889430}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1904536267183823376, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2630098056306889430}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2512479565128020758, "histo_col_type": "", "name": "__auto__", "null_count": 2294798999062197323, "row_count": 2630098056306889430}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1826625072509669899, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": "2007-02-08 13:16:48.000883+00:00"}, {"distinct_range": 1040877249703771100, "num_eq": 500000000, "num_range": 5093597994112602813, "upper_bound": "2008-12-10 18:18:41.00044+00:00"}, {"distinct_range": 300000, "num_eq": 9000, "num_range": 300000, "upper_bound": "2014-01-05 22:44:01.000855+00:00"}, {"distinct_range": 0, "num_eq": 0, "num_range": 6000, "upper_bound": "2027-01-27 21:31:28.000115+00:00"}, {"distinct_range": 0, "num_eq": 900000000, "num_range": 80000000000, "upper_bound": "2028-09-06 13:53:51.000763+00:00"}, {"distinct_range": 0, "num_eq": 2000000, "num_range": 70000000, "upper_bound": "294276-12-31 23:59:59.999999+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 1137592747397742619, "row_count": 2630098056306889430}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 994245006196383670, "histo_col_type": "", "name": "__auto__", "null_count": 1623110460516651967, "row_count": 2630098056306889430}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 320308813812832074, "histo_col_type": "", "name": "__auto__", "null_count": 781489621679240684, "row_count": 2630098056306889430}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1283661198298211591, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2630098056306889430}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2616999987036954871, "histo_buckets": [{"distinct_range": 0, "num_eq": 6753851539174052450, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 0, "num_eq": 91962153265166662, "num_range": 800000000, "upper_bound": "17:44:24.315447+14:59:00"}, {"distinct_range": 716097902086511200, "num_eq": 8000000, "num_range": 2678708358360457235, "upper_bound": "10:56:27.539356+01:08:00"}, {"distinct_range": 8943543576375246000, "num_eq": 3735330171920449657, "num_range": 8943543576375245644, "upper_bound": "22:11:10.462708+10:01:00"}, {"distinct_range": 0, "num_eq": 7814825745575621522, "num_range": 3299856676888975319, "upper_bound": "22:55:49.274451+03:46:00"}, {"distinct_range": 5196099392981117000, "num_eq": 900000, "num_range": 6243916845760035007, "upper_bound": "16:10:56.291327-05:29:00"}, {"distinct_range": 4138227451067885600, "num_eq": 477005490649935707, "num_range": 8925611803653091841, "upper_bound": "22:54:59.95318-01:57:00"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 4266474605576276867, "upper_bound": "12:21:12.064649-13:41:00"}, {"distinct_range": 0, "num_eq": 3820734132506297952, "num_range": 700000000, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 2630098056306889430}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 751702882204569083, "histo_buckets": [{"distinct_range": 0, "num_eq": 900, "num_range": 0, "upper_bound": "-60 years -6 mons -839 days -21:57:01.943429"}, {"distinct_range": 5829748.599800146, "num_eq": 100000000000, "num_range": 8000000, "upper_bound": "-57 years -5 mons -95 days -19:16:59.355376"}, {"distinct_range": 0, "num_eq": 4345861498141557091, "num_range": 3147260473486907346, "upper_bound": "-16 years -159 days -24:51:19.384637"}, {"distinct_range": 0, "num_eq": 2598191850225683317, "num_range": 8505044714625437409, "upper_bound": "8 years 1 mon 934 days 14:46:44.736875"}, {"distinct_range": 0, "num_eq": 600000, "num_range": 100, "upper_bound": "46 years 8 mons 451 days 18:52:37.431471"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 8481879431526966564, "upper_bound": "59 years 4 mons 142 days 19:16:32.36316"}, {"distinct_range": 933379574973769700, "num_eq": 60, "num_range": 933379574973769786, "upper_bound": "290 years"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 2240734730732766281, "row_count": 2630098056306889430}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2439187983609342216, "histo_col_type": "", "name": "__auto__", "null_count": 1226289936099290256, "row_count": 2630098056306889430}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1285504111404154026, "histo_col_type": "", "name": "__auto__", "null_count": 2062639475289632892, "row_count": 2630098056306889430}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1620598282782292208, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2630098056306889430}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 149686979500370943, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2630098056306889430}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2021962575593847994, "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 8854191240611897353}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4830319647885213781, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000000, "num_range": 0, "upper_bound": "03:06:35.429775"}, {"distinct_range": 0, "num_eq": 200000000, "num_range": 90000, "upper_bound": "04:47:55.253729"}, {"distinct_range": 0, "num_eq": 5037334566861856631, "num_range": 500000000, "upper_bound": "06:29:48.964017"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 90000000, "upper_bound": "11:01:26.667594"}, {"distinct_range": 60000000000, "num_eq": 8257414813691163092, "num_range": 60000000000, "upper_bound": "20:48:03.99954"}, {"distinct_range": 0, "num_eq": 50, "num_range": 3637428551355778746, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 8854191240611897353}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1548180268969877044, "histo_buckets": [{"distinct_range": 0, "num_eq": 3776913465128694990, "num_range": 0, "upper_bound": "-0.9924279451370239"}, {"distinct_range": 10000, "num_eq": 9000000, "num_range": 10000, "upper_bound": "-0.9918012619018555"}, {"distinct_range": 0, "num_eq": 30000, "num_range": 3480072927110478997, "upper_bound": "-0.6857157945632935"}, {"distinct_range": 447424091.0218925, "num_eq": 6128094855233244387, "num_range": 800000000, "upper_bound": "-0.22159311175346375"}, {"distinct_range": 4129918082175357000, "num_eq": 700, "num_range": 4129918082175356746, "upper_bound": "1.401298464324817e-45"}, {"distinct_range": 0, "num_eq": 8362505602277804409, "num_range": 5000000, "upper_bound": "0.21727372705936432"}, {"distinct_range": 0, "num_eq": 1250982562488550101, "num_range": 0, "upper_bound": "0.3230416476726532"}, {"distinct_range": 0, "num_eq": 100, "num_range": 4199632834814947588, "upper_bound": "1.0003554821014404"}, {"distinct_range": 708056511426812700, "num_eq": 3000000000, "num_range": 1706884390083524092, "upper_bound": "2.524155378341675"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 8285933558451361180, "row_count": 8854191240611897353}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8084450714819687862, "histo_buckets": [{"distinct_range": 0, "num_eq": 6891526098125237935, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 3104330828596068000, "num_eq": 200, "num_range": 3104330828596067653, "upper_bound": "134585764"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 5658484575839278232, "upper_bound": "826272850"}, {"distinct_range": 0, "num_eq": 800000000, "num_range": 80000000000, "upper_bound": "1404329032"}, {"distinct_range": 0, "num_eq": 238014475335531181, "num_range": 0, "upper_bound": "1567418289"}, {"distinct_range": 422204310018933400, "num_eq": 100000000, "num_range": 1175613905871111089, "upper_bound": "2387209337"}, {"distinct_range": 1442982550912720100, "num_eq": 6981751713183995881, "num_range": 1442982550912720114, "upper_bound": "3028449310"}, {"distinct_range": 0, "num_eq": 5460976224976411067, "num_range": 1206419986684544768, "upper_bound": "3554514658"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 2749954163189307043, "row_count": 8854191240611897353}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 94, "histo_col_type": "", "name": "__auto__", "null_count": 46, "row_count": 100}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23, "histo_col_type": "", "name": "__auto__", "null_count": 74, "row_count": 100}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39, "histo_col_type": "", "name": "__auto__", "null_count": 43, "row_count": 100}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 43, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": "09:54:48.663411+07:25:00"}, {"distinct_range": 4.927433345235181, "num_eq": 1503313249156566301, "num_range": 100, "upper_bound": "10:05:53.275481+03:32:00"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 1000000, "upper_bound": "18:36:37.783487+11:09:00"}, {"distinct_range": 1000000000, "num_eq": 900, "num_range": 1000000000, "upper_bound": "05:32:53.044585-09:02:00"}, {"distinct_range": 6655582272388045000, "num_eq": 8476032944701421055, "num_range": 6655582272388044815, "upper_bound": "11:08:06.561456-11:20:00"}, {"distinct_range": 0, "num_eq": 2401603795099583705, "num_range": 8495070678500805629, "upper_bound": "15:55:38.953361-10:05:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 85, "histo_col_type": "", "name": "__auto__", "null_count": 61, "row_count": 100}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45, "histo_buckets": [{"distinct_range": 0, "num_eq": 1031817747019007412, "num_range": 0, "upper_bound": "false"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 43, "row_count": 100}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 66, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 75, "histo_col_type": "", "name": "__auto__", "null_count": 19, "row_count": 100}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 75, "histo_col_type": "", "name": "__auto__", "null_count": 1, "row_count": 100}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 7745804574382133891, "num_range": 1044567986072310017, "upper_bound": "658815382"}, {"distinct_range": 0, "num_eq": 5000, "num_range": 7330503032077012644, "upper_bound": "672436736"}, {"distinct_range": 4247794981344404000, "num_eq": 3260096109808620808, "num_range": 4247794981344403994, "upper_bound": "1175990858"}, {"distinct_range": 864785801.449844, "num_eq": 7770525904560207552, "num_range": 1000000000, "upper_bound": "1573541160"}, {"distinct_range": 0, "num_eq": 600, "num_range": 400000000, "upper_bound": "2232580584"}, {"distinct_range": 2932438622612423000, "num_eq": 7872173304924957888, "num_range": 6580507167034277320, "upper_bound": "3426384096"}, {"distinct_range": 0, "num_eq": 2499177876822910449, "num_range": 300000, "upper_bound": "3537829150"}, {"distinct_range": 7420757953094714000, "num_eq": 10000000000, "num_range": 7420757953094713909, "upper_bound": "3769720928"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46, "histo_col_type": "", "name": "__auto__", "null_count": 84, "row_count": 100}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15323590939, "histo_col_type": "", "name": "__auto__", "null_count": 9579761368, "row_count": 30000000000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17831419069, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": "-2000-01-01 00:00:00+00:00"}, {"distinct_range": 0, "num_eq": 745051825148499698, "num_range": 8325317855975468381, "upper_bound": "1972-03-30 01:14:12.000204+00:00"}, {"distinct_range": 29267324.846650023, "num_eq": 2057246199995534162, "num_range": 30000000, "upper_bound": "1979-10-21 21:51:12.000552+00:00"}, {"distinct_range": 90000000, "num_eq": 3471023872311039448, "num_range": 90000000, "upper_bound": "1998-10-18 07:55:01.00017+00:00"}, {"distinct_range": 0, "num_eq": 4000000000, "num_range": 900, "upper_bound": "2000-11-08 05:48:46.000773+00:00"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 400000000, "upper_bound": "2010-04-03 14:43:30.000848+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 26882506396, "row_count": 30000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22248515658, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25779872125, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "-55 years -8 mons -130 days -19:03:44.599369"}, {"distinct_range": 0, "num_eq": 2000000000, "num_range": 8000, "upper_bound": "-28 years -629 days -22:54:54.585466"}, {"distinct_range": 0, "num_eq": 8794447752515993521, "num_range": 20, "upper_bound": "23 years 6 mons 628 days 21:24:09.080449"}, {"distinct_range": 90000, "num_eq": 497019307668876407, "num_range": 90000, "upper_bound": "30 years 7 mons 140 days 07:19:34.927377"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 15651875952, "row_count": 30000000000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16165748360, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2804735460, "histo_buckets": [{"distinct_range": 0, "num_eq": 801118829210638100, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 0, "num_eq": 113737845495547638, "num_range": 1757770522230223241, "upper_bound": "06:24:07.948907-10:17:00"}, {"distinct_range": 3327904546075069400, "num_eq": 20, "num_range": 3327904546075069249, "upper_bound": "22:14:25.081497+04:32:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2453620200, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1975-11-03"}, {"distinct_range": 80000000, "num_eq": 3379304795727122566, "num_range": 80000000, "upper_bound": "1995-06-29"}, {"distinct_range": 1362790259738223900, "num_eq": 100000000, "num_range": 5360971559010533726, "upper_bound": "1996-09-22"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 19528473587, "row_count": 30000000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1415436018, "histo_col_type": "", "name": "__auto__", "null_count": 26121368635, "row_count": 30000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3004572367, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24838284439, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12371909398, "histo_col_type": "", "name": "__auto__", "null_count": 8765801737, "row_count": 30000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5135533310, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23641957985, "histo_col_type": "", "name": "__auto__", "null_count": 4092002351, "row_count": 30000000000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8449050226, "histo_col_type": "", "name": "__auto__", "null_count": 23594710099, "row_count": 30000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15117562987, "histo_col_type": "", "name": "__auto__", "null_count": 12359636490, "row_count": 30000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13031673668, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000, "num_range": 0, "upper_bound": "-23088"}, {"distinct_range": 24916881.169673663, "num_eq": 9000000, "num_range": 60000000, "upper_bound": "-19623"}, {"distinct_range": 6322854102320395000, "num_eq": 795968295188641332, "num_range": 6322854102320395060, "upper_bound": "19417"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5770702258, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4704801755731752195, "histo_buckets": [{"distinct_range": 0, "num_eq": 700, "num_range": 0, "upper_bound": "20271483"}, {"distinct_range": 0, "num_eq": 874171707426135818, "num_range": 50000000, "upper_bound": "726263489"}, {"distinct_range": 26918624.875801384, "num_eq": 3000000, "num_range": 50000000, "upper_bound": "1488898643"}, {"distinct_range": 21641870700364224, "num_eq": 70, "num_range": 75051183672927671, "upper_bound": "1595205711"}, {"distinct_range": 0, "num_eq": 8570346317830428221, "num_range": 3254989003109767829, "upper_bound": "3242102921"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 0, "upper_bound": "4138090571"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 4045569844857362323, "row_count": 9141890602742112028}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 811558061686212375, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000000000, "num_range": 0, "upper_bound": "07:07:56.502394+13:23:00"}, {"distinct_range": 200000, "num_eq": 4025837592071875363, "num_range": 200000, "upper_bound": "03:44:35.919269-00:45:00"}, {"distinct_range": 3702189214.852362, "num_eq": 8001412291001281472, "num_range": 6000000000, "upper_bound": "01:34:25.207496-07:46:00"}, {"distinct_range": 0, "num_eq": 8982043668876223366, "num_range": 6000, "upper_bound": "13:38:52.853728+03:10:00"}, {"distinct_range": 0, "num_eq": 6390565261233573690, "num_range": 0, "upper_bound": "19:19:06.72295+05:54:00"}, {"distinct_range": 400, "num_eq": 80, "num_range": 400, "upper_bound": "12:50:18.196471-07:03:00"}, {"distinct_range": 9999779287.708519, "num_eq": 967221637305024159, "num_range": 10000000000, "upper_bound": "13:27:05.831612-10:38:00"}, {"distinct_range": 461943711.0095141, "num_eq": 8534562400893371284, "num_range": 1000000000, "upper_bound": "21:48:52.462922-09:35:00"}, {"distinct_range": 8288265142916371000, "num_eq": 20000000, "num_range": 8288265142916371440, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 9141890602742112028}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2627210904146576090, "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 9141890602742112028}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5212639841009845148, "histo_buckets": [{"distinct_range": 0, "num_eq": 10, "num_range": 0, "upper_bound": "-1.1152143478393555"}, {"distinct_range": 0, "num_eq": 20000000000, "num_range": 861679349934597378, "upper_bound": "-0.8964664340019226"}, {"distinct_range": 0, "num_eq": 0, "num_range": 400, "upper_bound": "-0.3675788938999176"}, {"distinct_range": 0, "num_eq": 5823596332760131889, "num_range": 50000, "upper_bound": "-0.044819217175245285"}, {"distinct_range": 10, "num_eq": 3325161833350450159, "num_range": 10, "upper_bound": "0.0"}, {"distinct_range": 10000000, "num_eq": 2338923459449359832, "num_range": 10000000, "upper_bound": "0.24778614938259125"}, {"distinct_range": 0, "num_eq": 800000000, "num_range": 8560267567402374751, "upper_bound": "0.5638614892959595"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 1643097927466207137, "row_count": 9141890602742112028}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 15, "row_count": 20}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15, "histo_col_type": "", "name": "__auto__", "null_count": 17, "row_count": 20}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10, "histo_col_type": "", "name": "__auto__", "null_count": 4, "row_count": 20}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000000, "num_range": 0, "upper_bound": "09:25:57.504992+09:18:00"}, {"distinct_range": 0, "num_eq": 30000000, "num_range": 9069176392814506108, "upper_bound": "01:32:11.160594+01:10:00"}, {"distinct_range": 300000000, "num_eq": 100000, "num_range": 300000000, "upper_bound": "17:56:18.077972+11:52:00"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 7616602615327446111, "upper_bound": "05:09:03.250497-04:08:00"}, {"distinct_range": 435792513298203000, "num_eq": 8715309236568901235, "num_range": 5120356801034042799, "upper_bound": "20:54:27.626419+04:53:00"}, {"distinct_range": 4920661657040205000, "num_eq": 7161148393157139413, "num_range": 6136967075167905475, "upper_bound": "16:44:48.461573-11:37:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 20}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17, "histo_col_type": "", "name": "__auto__", "null_count": 15, "row_count": 20}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1, "histo_col_type": "", "name": "__auto__", "null_count": 8, "row_count": 20}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16, "histo_col_type": "", "name": "__auto__", "null_count": 4, "row_count": 20}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6, "histo_buckets": [{"distinct_range": 0, "num_eq": 300000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 3069765582756940300, "num_eq": 1357887795270459780, "num_range": 3069765582756940319, "upper_bound": "1195692979"}, {"distinct_range": 5324964541104935000, "num_eq": 100, "num_range": 5324964541104934525, "upper_bound": "1209852885"}, {"distinct_range": 4958464215022289000, "num_eq": 700000, "num_range": 4958464215022288453, "upper_bound": "1948158527"}, {"distinct_range": 3983554642.8372498, "num_eq": 2000, "num_range": 5000000000, "upper_bound": "2364828949"}, {"distinct_range": 23911237.39404549, "num_eq": 2035160105441816460, "num_range": 10000000000, "upper_bound": "2405137334"}, {"distinct_range": 0, "num_eq": 0, "num_range": 50, "upper_bound": "2912373023"}, {"distinct_range": 6422114407658395000, "num_eq": 8636112430814910941, "num_range": 6422114407658395112, "upper_bound": "3043958823"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 20}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1, "histo_col_type": "", "name": "__auto__", "null_count": 10, "row_count": 20}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1, "histo_col_type": "BOOL", "name": "__auto__", "null_count": 14, "row_count": 20}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "nan" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down....2021-01-12 06:15:22.070 UTC [48] LOG:  received fast shutdown request
        postgres_1    | 2021-01-12 06:15:22.073 UTC [48] LOG:  aborting any active transactions
        postgres_1    | 2021-01-12 06:15:22.075 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2021-01-12 06:15:22.081 UTC [50] LOG:  shutting down
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2578436&tab=buildLog) on [master@03797b17417ae34451537f8f76d66ac69dba2d07](https://github.com/cockroachdb/cockroach/commits/03797b17417ae34451537f8f76d66ac69dba2d07):

```
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_8240.col1_3, tab_8240.col1_3 ASC, tab_8240.col1_11 ASC, tab_8240.col1_3 DESC
        postgres_1    | 		)
        postgres_1    | 			AS tab_8241
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_14232 NULLS FIRST,
        postgres_1    | 		col_14233 NULLS FIRST,
        postgres_1    | 		col_14234 NULLS FIRST,
        postgres_1    | 		col_14235 NULLS FIRST,
        postgres_1    | 		col_14236 NULLS FIRST,
        postgres_1    | 		col_14237 NULLS FIRST,
        postgres_1    | 		col_14238 NULLS FIRST,
        postgres_1    | 		col_14239 NULLS FIRST,
        postgres_1    | 		col_14240 NULLS FIRST,
        postgres_1    | 		col_14241 NULLS FIRST,
        postgres_1    | 		col_14242 NULLS FIRST,
        postgres_1    | 		col_14243 NULLS FIRST,
        postgres_1    | 		col_14244 NULLS FIRST,
        postgres_1    | 		col_14245 NULLS FIRST,
        postgres_1    | 		col_14246 NULLS FIRST,
        postgres_1    | 		col_14247 NULLS FIRST
        test_1        | --- FAIL: TestCompare (612.98s)
        test_1        |     --- FAIL: TestCompare/postgres (12.08s)
        test_1        |         compare_test.go:155: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("falseJ"),
        test_1        |             + 	string("fJ"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT (false::BOOL || 'J'::TEXT::TEXT)::TEXT AS col_14248) AS tab_8242
        test_1        |             ORDER BY
        test_1        |             	col_14248 NULLS FIRST;
        test_1        |             
        test_1        |             postgres: [same as previous]
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2582296&tab=buildLog) on [master@f9b44f59d0451dc9efbb771baac3e88540485629](https://github.com/cockroachdb/cockroach/commits/f9b44f59d0451dc9efbb771baac3e88540485629):

```
        test_1        |             CREATE INDEX ON table3 (col3_3 ASC, col3_1 DESC, col3_8 DESC, col1_0 ASC, col3_2 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_8);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_3 ASC, col3_6, col3_7 ASC, col3_4, col3_8, col3_5 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_3 ASC, col3_7, col1_0) INCLUDE (col3_4, col3_5);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col1_0);
        test_1        |             CREATE TABLE table4 (col1_0 INT8 NOT NULL, col4_1 BIT(40) NOT NULL, col4_2 INT8 NULL, col4_3 TIMESTAMP, col4_4 INT8 NULL, col4_5 REGPROCEDURE NOT NULL, col4_6 TIMESTAMP NULL, col4_7 BIT(16), PRIMARY KEY (col1_0, col4_5, col4_7, col4_4, col4_6));
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_7, col4_2 DESC, col4_5 ASC, col4_6 ASC, col4_1);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_5);
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col3_7) REFERENCES table4 (col4_5);
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_0) REFERENCES table3 (col1_0);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('domr', 'e', 'xf', 'eocpw');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('dfjq', 'pcue');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('huiu', 'ott', 'bxbmjq', 'az', 'kkn', 'otuo');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('vzcne', 'lfafco', 'vod', 'bxerg', 'k', 'tehdgt');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('t', 'cupa', 'ss', 'tofs', 'thmr', 'w');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (1.13s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 BOOL, col1_1 REGPROC NOT NULL, col1_2 BOX2D NULL, col1_3 INT8, col1_4 REGNAMESPACE NULL, col1_5 INT4, col1_6 INT8 NULL, col1_7 FLOAT8 NULL, col1_8 REGCLASS NULL, col1_9 UUID, col1_10 CHAR NULL, col1_11 REGTYPE NULL, col1_12 TIMESTAMPTZ, col1_13 REGNAMESPACE NULL, col1_14 TIMESTAMP NULL, PRIMARY KEY (col1_1), UNIQUE (col1_1, col1_3 ASC, col1_0 DESC, col1_5 ASC, col1_9 ASC) STORING (col1_4, col1_6, col1_7, col1_10, col1_11, col1_12, col1_13, col1_14) WHERE (((((((table1.col1_12 >= '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ) OR (table1.col1_3 <= 0:::INT8)) AND (table1.col1_5 >= 32767:::INT8)) AND (table1.col1_6 = (-9223372036854775807):::INT8)) OR (table1.col1_7 <= (-1.0):::FLOAT8)) AND table1.col1_0) AND (table1.col1_14 < '294276-12-31 23:59:59.999999':::TIMESTAMP)) AND (table1.col1_10 = '"':::STRING), INDEX (col1_9 ASC, col1_11 DESC, col1_1, col1_14, col1_7 ASC, col1_2, col1_4 DESC, col1_5 DESC, col1_13) WHERE (((((((table1.col1_6 <= (-9223372036854775807):::INT8) OR (table1.col1_3 < 127:::INT8)) OR (table1.col1_5 = (-128):::INT8)) AND (table1.col1_10 <= e'\U00002603':::STRING)) AND (table1.col1_12 = '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table1.col1_7 != 1.7976931348623157e+308:::FLOAT8)) OR (NOT table1.col1_0)) OR (table1.col1_14 > '0001-01-01 00:00:00':::TIMESTAMP), UNIQUE (col1_13 DESC, col1_0 ASC, col1_9, col1_12, col1_6 ASC, col1_10 ASC, col1_14 ASC, col1_8, col1_11, col1_3 DESC, col1_2 DESC, col1_1 ASC, col1_4 ASC, col1_5 ASC) STORING (col1_7) WHERE (((((((table1.col1_12 = '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ) OR (NOT table1.col1_0)) OR (table1.col1_10 >= e'\'':::STRING)) OR (table1.col1_3 >= (-32768):::INT8)) AND (table1.col1_5 < (-1):::INT8)) OR (table1.col1_7 >= 'NaN':::FLOAT8)) AND (table1.col1_6 != (-1):::INT8)) OR (table1.col1_14 = '3000-01-01 00:00:00':::TIMESTAMP), INDEX (col1_1 DESC, col1_0 ASC, col1_10, col1_9 ASC, col1_12, col1_14 DESC, col1_5, col1_2 ASC, col1_7 ASC, col1_6) STORING (col1_3, col1_4) WHERE ((((((table1.col1_0 OR (table1.col1_12 > '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table1.col1_6 >= 1:::INT8)) OR (table1.col1_3 < (-1):::INT8)) OR (table1.col1_10 != e'\x00':::STRING)) OR (table1.col1_14 > '0001-01-01 00:00:00':::TIMESTAMP)) AND (table1.col1_7 <= 1.7976931348623157e+308:::FLOAT8)) OR (table1.col1_5 = 2147483647:::INT8), INDEX (col1_4 DESC, col1_8 DESC, col1_10, col1_7 DESC, col1_2 ASC) STORING (col1_0, col1_3, col1_6, col1_11) WHERE (((((((table1.col1_7 = 1.0:::FLOAT8) AND (table1.col1_14 = '-4713-11-24 00:00:00':::TIMESTAMP)) OR table1.col1_0) AND (table1.col1_10 < 'X':::STRING)) OR (table1.col1_6 <= 2147483647:::INT8)) AND (table1.col1_5 <= (-1):::INT8)) OR (table1.col1_12 >= '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) OR (table1.col1_3 != 127:::INT8), INDEX (col1_12 ASC, col1_0 DESC, col1_2) STORING (col1_4, col1_6, col1_7, col1_10, col1_13) WHERE (((((((table1.col1_5 > (-32768):::INT8) OR (table1.col1_12 <= '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (NOT table1.col1_0)) AND (table1.col1_10 < '':::STRING)) AND (table1.col1_6 != (-32768):::INT8)) OR (table1.col1_7 > '+Inf':::FLOAT8)) AND (table1.col1_14 < '-2000-01-01 00:00:00':::TIMESTAMP)) OR (table1.col1_3 >= 2147483647:::INT8), INDEX (col1_8 ASC, col1_2 DESC, col1_5 ASC, col1_3 ASC, col1_0) STORING (col1_6, col1_9, col1_11, col1_12, col1_13, col1_14), UNIQUE (col1_3) WHERE ((table1.col1_14 = '-4713-11-24 00:00:00') AND (table1.col1_7 >= 1.7976931348623157e+308)) OR (NOT table1.col1_0), FAMILY (col1_1), FAMILY (col1_14), FAMILY (col1_13), FAMILY (col1_0), FAMILY (col1_9, col1_7), FAMILY (col1_2, col1_5), FAMILY (col1_6, col1_11), FAMILY (col1_8), FAMILY (col1_4), FAMILY (col1_12), FAMILY (col1_3), FAMILY (col1_10));
        test_1        |             CREATE TABLE table2 (col1_1 REGPROC NOT NULL, col2_1 UUID, col2_2 FLOAT8, col2_3 REGTYPE NOT NULL, col2_4 INTERVAL NULL, col2_5 TIMETZ NULL, col2_6 REGPROCEDURE NOT NULL, col2_7 INT8 NOT NULL, col2_8 TIMESTAMPTZ NOT NULL, col2_9 INT8 NULL, col2_10 REGPROCEDURE NOT NULL, col2_11 INTERVAL NULL, col2_12 FLOAT4 NULL, col2_13 INT4 NULL, col2_14 INT4 NULL, col2_15 INTERVAL, col2_16 BYTES NOT NULL, col2_17 TIMESTAMP NULL, PRIMARY KEY (col1_1, col2_3, col2_8 DESC, col2_16 ASC), UNIQUE (col2_4, col2_12 ASC, col2_8 DESC) STORING (col2_6, col2_7, col2_9, col2_10, col2_13, col2_14) WHERE (((((((((table2.col2_7 >= (-2147483648):::INT8) AND (table2.col2_13 = 1:::INT8)) OR (table2.col2_14 >= (-128):::INT8)) OR (table2.col2_2 = 5e-324:::FLOAT8)) AND (table2.col2_16 < e'\\xe29883':::BYTES)) OR (table2.col2_12 != 0.0:::FLOAT8)) AND (table2.col2_5 = '24:00:00-15:59:00':::TIMETZ)) AND (table2.col2_17 != '-2000-01-01 00:00:00':::TIMESTAMP)) AND (table2.col2_9 >= (-1):::INT8)) AND (table2.col2_8 < '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ), INDEX (col2_15, col2_3 ASC) STORING (col2_2, col2_4, col2_5, col2_7, col2_13, col2_14) WHERE (((((((((table2.col2_12 >= '+Inf':::FLOAT8) OR (table2.col2_13 < 1:::INT8)) OR (table2.col2_8 < '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) AND (table2.col2_7 > (-2147483648):::INT8)) OR (table2.col2_14 != (-1):::INT8)) AND (table2.col2_16 = e'\\x27':::BYTES)) AND (table2.col2_5 <= '00:00:00+15:59:00':::TIMETZ)) OR (table2.col2_2 >= 3.4028234663852886e+38:::FLOAT8)) OR (table2.col2_17 < '294276-12-31 23:59:59.999999':::TIMESTAMP)) OR (table2.col2_9 >= 9223372036854775807:::INT8), UNIQUE (col2_3 ASC), UNIQUE (col2_8 DESC, col2_9, col2_16 ASC, col2_17, col2_11, col2_3 DESC, col2_6 ASC, col2_14 DESC) WHERE ((((((((table2.col2_2 < 0.0:::FLOAT8) AND (table2.col2_12 <= '-Inf':::FLOAT8)) AND (table2.col2_8 != '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) OR (table2.col2_7 != 2147483647:::INT8)) OR (table2.col2_5 = '24:00:00-15:59:00':::TIMETZ)) AND (table2.col2_9 != 9223372036854775807:::INT8)) AND (table2.col2_17 < '-4713-11-24 00:00:00':::TIMESTAMP)) OR (table2.col2_14 <= 0:::INT8)) OR (table2.col2_16 <= e'\\x27':::BYTES), UNIQUE (col2_9, col2_13 DESC, col2_16 DESC, col2_5, col2_7, col2_8 DESC, col1_1, col2_14 DESC, col2_6, col2_3 DESC, col2_11), INDEX (col2_15 ASC) STORING (col2_4, col2_7, col2_9, col2_10, col2_11, col2_12, col2_14) WHERE (((((((((table2.col2_14 < 0:::INT8) OR (table2.col2_8 < '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table2.col2_16 < e'\\x27':::BYTES)) AND (table2.col2_17 > '294276-12-31 23:59:59.999999':::TIMESTAMP)) AND (table2.col2_9 >= 0:::INT8)) AND (table2.col2_12 = 1.401298464324817e-45:::FLOAT8)) AND (table2.col2_13 > 1:::INT8)) OR (table2.col2_2 > 1.0:::FLOAT8)) AND (table2.col2_7 != 9223372036854775807:::INT8)) AND (table2.col2_5 != '24:00:00-15:59:00':::TIMETZ), INDEX (col2_9 DESC, col2_2 DESC, col2_6 ASC), FAMILY (col2_5, col2_12), FAMILY (col2_8), FAMILY (col2_14, col2_7), FAMILY (col2_13), FAMILY (col2_16), FAMILY (col2_17), FAMILY (col1_1, col2_1, col2_15, col2_2), FAMILY (col2_10), FAMILY (col2_4), FAMILY (col2_3, col2_11), FAMILY (col2_6, col2_9)) INTERLEAVE IN PARENT table1 (col1_1);
        test_1        |             CREATE TABLE table3 (col3_0 REGPROCEDURE NOT NULL, col3_1 BOOL NOT NULL, col3_2 OID NOT NULL, col3_3 DECIMAL NULL, col3_4 BIT(10) NOT NULL, col3_5 BOOL NOT NULL, col3_6 REGTYPE NULL, col3_7 BIT(4) NULL, col3_8 NAME NOT NULL, PRIMARY KEY (col3_0 DESC, col3_5, col3_1 ASC, col3_4 DESC, col3_2 DESC, col3_8 DESC), UNIQUE (col3_4, col3_1 ASC, col3_3, col3_0 DESC) WHERE (table3.col3_8 = e'\'':::STRING) OR (NOT table3.col3_1), INDEX (col3_4 DESC, col3_5, col3_7, col3_2 DESC, col3_3 DESC, col3_6 DESC), FAMILY (col3_5, col3_1), FAMILY (col3_3), FAMILY (col3_7, col3_4, col3_0), FAMILY (col3_8), FAMILY (col3_6, col3_2));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1706746279876295385, "histo_col_type": "", "name": "__auto__", "null_count": 3104402058874694259, "row_count": 4793845584741106158}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 688905222879762891, "histo_buckets": [{"distinct_range": 0, "num_eq": 3212455360372697606, "num_range": 0, "upper_bound": "0001-01-01 00:00:00+00:00"}, {"distinct_range": 1706286947465270000, "num_eq": 8000, "num_range": 2839700293468094674, "upper_bound": "1973-04-13 08:06:04.000625+00:00"}, {"distinct_range": 4673642.069277271, "num_eq": 50000000, "num_range": 5000000, "upper_bound": "1988-10-29 06:17:31.000768+00:00"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 9000000000, "upper_bound": "1993-07-09 00:56:02.000483+00:00"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 0, "upper_bound": "2012-11-21 05:57:01.000623+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 1676854355024978507, "row_count": 4793845584741106158}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1750393600549551724, "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 3690180292362714337, "row_count": 4793845584741106158}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3075870418903781262, "histo_col_type": "", "name": "__auto__", "null_count": 2181866192833129589, "row_count": 4793845584741106158}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2287949459698161909, "histo_col_type": "", "name": "__auto__", "null_count": 4501549272436195830, "row_count": 4793845584741106158}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1368855662151461827, "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 4363509051141813052, "row_count": 4793845584741106158}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3392719730524387745, "histo_buckets": [{"distinct_range": 0, "num_eq": 468927956128831675, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 7003783509262539472, "num_range": 7276709681430888744, "upper_bound": "3867079"}, {"distinct_range": 8711136548.360607, "num_eq": 8449888271038966219, "num_range": 10000000000, "upper_bound": "994477672"}, {"distinct_range": 30, "num_eq": 0, "num_range": 30, "upper_bound": "2773417288"}, {"distinct_range": 7.849130166925663, "num_eq": 49310623758339279, "num_range": 60, "upper_bound": "3441647663"}, {"distinct_range": 0, "num_eq": 595141672163178523, "num_range": 1161500105272540456, "upper_bound": "3513290340"}, {"distinct_range": 58964754.37796497, "num_eq": 600, "num_range": 60000000, "upper_bound": "3625667965"}, {"distinct_range": 0, "num_eq": 700, "num_range": 2350715922219332580, "upper_bound": "3833184022"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 484388277351749914, "row_count": 4793845584741106158}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4384868284201509869, "histo_col_type": "", "name": "__auto__", "null_count": 1589002371713257154, "row_count": 4793845584741106158}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3161623488136916554, "histo_col_type": "", "name": "__auto__", "null_count": 4143872026719417319, "row_count": 4793845584741106158}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 472843836580818983, "histo_buckets": [{"distinct_range": 0, "num_eq": 8864176587706062022, "num_range": 0, "upper_bound": "2031136801"}, {"distinct_range": 6042113768.651792, "num_eq": 6633700188165234744, "num_range": 30000000000, "upper_bound": "2917051535"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 4793845584741106158}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3042669560188631382, "histo_col_type": "", "name": "__auto__", "null_count": 484015155288143450, "row_count": 4793845584741106158}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2872984537319903498, "histo_col_type": "", "name": "__auto__", "null_count": 43280221181041705, "row_count": 4793845584741106158}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3087009871070377400, "histo_col_type": "", "name": "__auto__", "null_count": 1020647890450137079, "row_count": 4793845584741106158}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 963823083628672391, "histo_buckets": [{"distinct_range": 0, "num_eq": 3441054650610908686, "num_range": 0, "upper_bound": "00000000-0000-0000-0000-000000000000"}, {"distinct_range": 200000, "num_eq": 2000000, "num_range": 200000, "upper_bound": "dde41185-93cb-4b9b-af2b-101ee9b07477"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 2342016450270419649, "row_count": 4793845584741106158}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1425769647462820398, "histo_col_type": "", "name": "__auto__", "null_count": 2882952161143859909, "row_count": 4793845584741106158}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15826, "histo_col_type": "", "name": "__auto__", "null_count": 24133, "row_count": 50000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 37722, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 43951, "histo_col_type": "", "name": "__auto__", "null_count": 1684, "row_count": 50000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45424, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000000, "num_range": 0, "upper_bound": "-64 years -1 mons -794 days -09:28:58.438192"}, {"distinct_range": 84931679.51232302, "num_eq": 900000, "num_range": 100000000, "upper_bound": "-62 years -8 mons -938 days -12:15:56.568864"}, {"distinct_range": 0, "num_eq": 30000, "num_range": 4086937877251513208, "upper_bound": "-23 years -1 mons -94 days -12:19:54.807888"}, {"distinct_range": 8000000, "num_eq": 600, "num_range": 8000000, "upper_bound": "61 years 2 mons 443 days 18:48:31.37581"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 41126, "row_count": 50000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23011, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2234, "histo_col_type": "", "name": "__auto__", "null_count": 41084, "row_count": 50000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 40742, "histo_buckets": [{"distinct_range": 0, "num_eq": 4789097264464446890, "num_range": 0, "upper_bound": "1943923908"}, {"distinct_range": 8170060054.291329, "num_eq": 1743143956926862473, "num_range": 10000000000, "upper_bound": "2887990551"}, {"distinct_range": 3245510347357399600, "num_eq": 941149013441510504, "num_range": 3705904312755790713, "upper_bound": "3777777508"}, {"distinct_range": 2886419542205274000, "num_eq": 10, "num_range": 2886419542205274056, "upper_bound": "4012587075"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8903, "histo_col_type": "", "name": "__auto__", "null_count": 3709, "row_count": 50000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17655, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3598, "histo_col_type": "", "name": "__auto__", "null_count": 904, "row_count": 50000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 480, "histo_buckets": [{"distinct_range": 0, "num_eq": 4556114512743220977, "num_range": 0, "upper_bound": "-79 years -11 mons -219 days -14:24:06.900961"}, {"distinct_range": 800000000, "num_eq": 2824192409976663003, "num_range": 800000000, "upper_bound": "-48 years -1 mons -656 days -02:49:11.454713"}, {"distinct_range": 0, "num_eq": 900, "num_range": 80000000, "upper_bound": "-9 years -8 mons -443 days -08:09:34.324222"}, {"distinct_range": 0, "num_eq": 1815732306432461311, "num_range": 500, "upper_bound": "5 years 9 mons 815 days 18:42:37.970755"}, {"distinct_range": 3083496668852772000, "num_eq": 80000000, "num_range": 4091988486360298039, "upper_bound": "48 years 8 mons 112 days 00:59:45.085058"}, {"distinct_range": 4113776482849283600, "num_eq": 30, "num_range": 4219054332672252593, "upper_bound": "68 years 5 mons 729 days 08:56:07.781331"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 300, "upper_bound": "290 years"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 33076, "row_count": 50000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10134, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46146, "histo_col_type": "", "name": "__auto__", "null_count": 11521, "row_count": 50000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24769, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000, "num_range": 0, "upper_bound": "-1938988006824574183"}, {"distinct_range": 124772099.9339106, "num_eq": 800, "num_range": 500000000, "upper_bound": "4887349102379679038"}, {"distinct_range": 1401724604087905000, "num_eq": 2442107472200706658, "num_range": 5445469681895994725, "upper_bound": "5224143390425846415"}, {"distinct_range": 6036079003052997000, "num_eq": 1813624317961164820, "num_range": 6036079003052996154, "upper_bound": "5470035051814271947"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 35608, "row_count": 50000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21040, "histo_col_type": "", "name": "__auto__", "null_count": 22410, "row_count": 50000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30479, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "1080702380"}, {"distinct_range": 0, "num_eq": 8997024111924802142, "num_range": 8686631935996915934, "upper_bound": "2549841996"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 50000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44751, "histo_col_type": "", "name": "__auto__", "null_count": 43772, "row_count": 50000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7399, "histo_buckets": [{"distinct_range": 0, "num_eq": 4405081944445786790, "num_range": 0, "upper_bound": "1974-04-21 12:15:01.00061+00:00"}, {"distinct_range": 2000, "num_eq": 239956512869746459, "num_range": 2000, "upper_bound": "2010-08-23 01:38:30.000192+00:00"}, {"distinct_range": 0, "num_eq": 300, "num_range": 10000000, "upper_bound": "2016-05-18 03:43:02.000752+00:00"}, {"distinct_range": 100, "num_eq": 90, "num_range": 100, "upper_bound": "2028-03-28 20:14:22.000924+00:00"}, {"distinct_range": 0, "num_eq": 6228419787201945276, "num_range": 0, "upper_bound": "2032-08-23 01:24:56.000341+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 50000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4724, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 185, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1666, "histo_col_type": "BIT(10)", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5775, "histo_col_type": "", "name": "__auto__", "null_count": 6451, "row_count": 8000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7518, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2178, "histo_buckets": [{"distinct_range": 0, "num_eq": 3664809756447812737, "num_range": 0, "upper_bound": "241079221"}, {"distinct_range": 0, "num_eq": 5325079595275147497, "num_range": 40000000, "upper_bound": "277568667"}, {"distinct_range": 90, "num_eq": 7347480152275815705, "num_range": 90, "upper_bound": "607923146"}, {"distinct_range": 73.3702237164009, "num_eq": 100000000, "num_range": 500, "upper_bound": "1203892859"}, {"distinct_range": 4643475947928494000, "num_eq": 900, "num_range": 7607770409574697749, "upper_bound": "1243159599"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 1660699362729997050, "upper_bound": "2277962675"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6505, "histo_col_type": "", "name": "__auto__", "null_count": 3339, "row_count": 8000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6265, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1101, "histo_col_type": "", "name": "__auto__", "null_count": 6522, "row_count": 8000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('ccyrfl', 'fyqje', 'ls');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('zoqm', 'qsvzx', 'klnvmd', 'hrrn', 'ebeup', 'hnxvq');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('ixni', 'fwhflj', 'renx', 'r', 'd', 'v');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('bcst');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2870583234, "histo_col_type": "", "name": "__auto__", "null_count": 2721045943, "row_count": 3000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2479232439, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "84707560"}, {"distinct_range": 0, "num_eq": 6563509539813885667, "num_range": 2110925421820668309, "upper_bound": "309333096"}, {"distinct_range": 154467588179763600, "num_eq": 7551582692243914473, "num_range": 177810688710881409, "upper_bound": "796885981"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 0, "upper_bound": "1447347650"}, {"distinct_range": 0, "num_eq": 1299089609131237414, "num_range": 2221397210618162339, "upper_bound": "1742751778"}, {"distinct_range": 2502253315062486500, "num_eq": 0, "num_range": 4671349915781407013, "upper_bound": "2237034877"}, {"distinct_range": 0, "num_eq": 0, "num_range": 7266643098311759195, "upper_bound": "2794173228"}, {"distinct_range": 8000, "num_eq": 90000, "num_range": 8000, "upper_bound": "3774865876"}, {"distinct_range": 270.4576615044841, "num_eq": 4423742401741253819, "num_range": 300, "upper_bound": "3995098690"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 464050230, "row_count": 3000000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1263978418, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000, "num_range": 0, "upper_bound": "1753784785"}, {"distinct_range": 108314816.25805235, "num_eq": 5877038439182579146, "num_range": 200000000, "upper_bound": "2118640218"}, {"distinct_range": 900000, "num_eq": 80000, "num_range": 900000, "upper_bound": "2913334120"}, {"distinct_range": 2776273586628367400, "num_eq": 4298728391536345627, "num_range": 7418945990704136449, "upper_bound": "3290579982"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 84734622, "row_count": 3000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 599256962, "histo_col_type": "", "name": "__auto__", "null_count": 2376329399, "row_count": 3000000000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 549794925, "histo_col_type": "", "name": "__auto__", "null_count": 294855413, "row_count": 3000000000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 902298456, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000, "num_range": 0, "upper_bound": "1972-09-08 18:53:39.000009+00:00"}, {"distinct_range": 0, "num_eq": 8690187004869696648, "num_range": 4497178643476505889, "upper_bound": "1974-07-06 13:16:37.00005+00:00"}, {"distinct_range": 3060782485649680000, "num_eq": 2400005007401556628, "num_range": 3111977859151594258, "upper_bound": "1981-03-10 02:19:23.000331+00:00"}, {"distinct_range": 0, "num_eq": 3545567817822610999, "num_range": 6809075122133737380, "upper_bound": "1996-12-23 17:44:40.000531+00:00"}, {"distinct_range": 118558.73926915598, "num_eq": 1828842992622607227, "num_range": 1000000, "upper_bound": "1999-10-29 16:25:31.000995+00:00"}, {"distinct_range": 6233431629261501000, "num_eq": 70000, "num_range": 7544995513349641261, "upper_bound": "2007-09-24 03:45:13.000911+00:00"}, {"distinct_range": 8000000000, "num_eq": 2912783101963641230, "num_range": 8000000000, "upper_bound": "2012-09-12 00:27:33.000324+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 831495431, "row_count": 3000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 785611023, "histo_col_type": "", "name": "__auto__", "null_count": 2193429732, "row_count": 3000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 484255081, "histo_col_type": "", "name": "__auto__", "null_count": 2054117560, "row_count": 3000000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1364362828, "histo_col_type": "", "name": "__auto__", "null_count": 2356742556, "row_count": 3000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2714714165, "histo_col_type": "", "name": "__auto__", "null_count": 86598983, "row_count": 3000000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2940825768, "histo_col_type": "", "name": "__auto__", "null_count": 2791472965, "row_count": 3000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 81819109, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000, "num_range": 0, "upper_bound": "1199372898"}, {"distinct_range": 2841663681262386000, "num_eq": 8355723744057206590, "num_range": 2841663681262386417, "upper_bound": "1502301354"}, {"distinct_range": 3179673816256027000, "num_eq": 2077774324721041373, "num_range": 3179673816256026895, "upper_bound": "2039931475"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 40000000, "upper_bound": "2205949612"}, {"distinct_range": 634834589687604700, "num_eq": 70000000, "num_range": 7344861937880687472, "upper_bound": "2569166308"}, {"distinct_range": 900, "num_eq": 4886643333483671035, "num_range": 900, "upper_bound": "2911485852"}, {"distinct_range": 0, "num_eq": 8279209501099434611, "num_range": 0, "upper_bound": "4241302357"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 3000000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 783794489, "histo_col_type": "UUID", "name": "__auto__", "null_count": 1323402855, "row_count": 3000000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1456207503, "histo_buckets": [{"distinct_range": 0, "num_eq": 8708643965853028207, "num_range": 0, "upper_bound": "258142119"}, {"distinct_range": 0, "num_eq": 100, "num_range": 800, "upper_bound": "1299029210"}, {"distinct_range": 7336935842035870000, "num_eq": 7098505968409849406, "num_range": 7336935842035869219, "upper_bound": "1656064435"}, {"distinct_range": 100000000000, "num_eq": 1000000000, "num_range": 100000000000, "upper_bound": "1950563546"}, {"distinct_range": 0, "num_eq": 0, "num_range": 271670891884118153, "upper_bound": "2744418305"}, {"distinct_range": 2445004918157283300, "num_eq": 4168158677579844980, "num_range": 3204734380071055552, "upper_bound": "2758680239"}, {"distinct_range": 0, "num_eq": 90000, "num_range": 800, "upper_bound": "3406419060"}, {"distinct_range": 400, "num_eq": 100000000000, "num_range": 400, "upper_bound": "3418172434"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 2748140157, "row_count": 3000000000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 946625979, "histo_col_type": "", "name": "__auto__", "null_count": 425665180, "row_count": 3000000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 8763764962204741368, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 5979764227883264990, "num_range": 0, "upper_bound": "138928815"}, {"distinct_range": 10000, "num_eq": 70000000, "num_range": 10000, "upper_bound": "229338361"}, {"distinct_range": 7634.774913221521, "num_eq": 700000000, "num_range": 10000, "upper_bound": "685142083"}, {"distinct_range": 8124368522808119000, "num_eq": 1954468810476140539, "num_range": 8124368522808119491, "upper_bound": "1429229139"}, {"distinct_range": 905443680882793900, "num_eq": 8021635850614144180, "num_range": 5664077579952259578, "upper_bound": "1511282327"}, {"distinct_range": 116655.42006960178, "num_eq": 100000000, "num_range": 500000, "upper_bound": "2615572105"}, {"distinct_range": 607.0722429613852, "num_eq": 10000, "num_range": 800, "upper_bound": "2903320870"}, {"distinct_range": 7000000000, "num_eq": 500000, "num_range": 7000000000, "upper_bound": "3224291643"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 4148133434818868231, "num_range": 0, "upper_bound": "-62 years -6 mons -853 days -24:30:37.737726"}, {"distinct_range": 2000000, "num_eq": 0, "num_range": 2000000, "upper_bound": "-61 years -701 days -19:22:35.314086"}, {"distinct_range": 6987025387324117000, "num_eq": 200, "num_range": 6987025387324116795, "upper_bound": "-39 years -8 mons -438 days -23:35:37.441302"}, {"distinct_range": 5833197778836058000, "num_eq": 4845430740138661731, "num_range": 5833197778836057693, "upper_bound": "47 years 7 mons 353 days 10:38:03.020184"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "1974-10-15 17:12:24.000616+00:00"}, {"distinct_range": 0, "num_eq": 400000, "num_range": 2091009097875524666, "upper_bound": "2008-09-02 14:57:36.000945+00:00"}, {"distinct_range": 50000, "num_eq": 1000, "num_range": 50000, "upper_bound": "2011-03-16 01:19:28.000271+00:00"}, {"distinct_range": 61047300560.341064, "num_eq": 40, "num_range": 70000000000, "upper_bound": "2014-02-19 14:20:27.000761+00:00"}, {"distinct_range": 50000000000, "num_eq": 0, "num_range": 50000000000, "upper_bound": "2014-04-21 18:32:09.000637+00:00"}, {"distinct_range": 212665848198070850, "num_eq": 6000000000, "num_range": 6240589362815593550, "upper_bound": "2018-03-04 18:11:42.000793+00:00"}, {"distinct_range": 53470162.444123365, "num_eq": 7971690174906856596, "num_range": 60000000, "upper_bound": "2024-03-24 17:30:37.00049+00:00"}, {"distinct_range": 5000000000, "num_eq": 4686459041274615570, "num_range": 5000000000, "upper_bound": "2026-03-17 20:49:59.000386+00:00"}, {"distinct_range": 1477482915084463400, "num_eq": 0, "num_range": 1947861806867822244, "upper_bound": "2031-07-26 03:28:18.000921+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 40, "num_range": 0, "upper_bound": "48 years 2 mons 429 days 00:03:15.106386"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 3041221728276132189, "num_range": 0, "upper_bound": "401527061"}, {"distinct_range": 0, "num_eq": 210240734033311054, "num_range": 5530461408220613039, "upper_bound": "821941486"}, {"distinct_range": 623343.2653603671, "num_eq": 7626257649648627267, "num_range": 900000, "upper_bound": "994327827"}, {"distinct_range": 2580376769242496000, "num_eq": 381136872420922593, "num_range": 3312819331439741572, "upper_bound": "1199094075"}, {"distinct_range": 1666961734898037000, "num_eq": 20000, "num_range": 2860546498943278337, "upper_bound": "1418149813"}, {"distinct_range": 8453428945549787000, "num_eq": 70000, "num_range": 8453428945549787257, "upper_bound": "1985798851"}, {"distinct_range": 335596167165903400, "num_eq": 2799981605044928454, "num_range": 3461781828250973524, "upper_bound": "2091871089"}, {"distinct_range": 6074822443783910000, "num_eq": 0, "num_range": 6074822443783910119, "upper_bound": "2660523232"}, {"distinct_range": 90, "num_eq": 90000000000, "num_range": 90, "upper_bound": "4133377433"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 282866017671716137, "num_range": 0, "upper_bound": "-8362152497806198517"}, {"distinct_range": 0, "num_eq": 8096291542659250529, "num_range": 6468651333225749580, "upper_bound": "-7432483525186467208"}, {"distinct_range": 8.363025703797128, "num_eq": 3904185566407237110, "num_range": 100, "upper_bound": "-1195532099751709285"}, {"distinct_range": 2000, "num_eq": 0, "num_range": 2000, "upper_bound": "127"}, {"distinct_range": 0, "num_eq": 8565696615367927802, "num_range": 7000000000, "upper_bound": "1247138808152335394"}, {"distinct_range": 0, "num_eq": 8072045186184165711, "num_range": 2696564347529123871, "upper_bound": "1838179889609272754"}, {"distinct_range": 6751.006719686943, "num_eq": 10000, "num_range": 200000, "upper_bound": "3883628252681102756"}, {"distinct_range": 370.93681693218184, "num_eq": 7000000, "num_range": 400, "upper_bound": "8319436379608698336"}, {"distinct_range": 7552488952265955000, "num_eq": 6331855628190109294, "num_range": 7552488952265955805, "upper_bound": "9063299131022392746"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2630, "histo_col_type": "", "name": "__auto__", "null_count": 2068, "row_count": 3000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2300, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 279, "histo_buckets": [{"distinct_range": 0, "num_eq": 8380358899540453726, "num_range": 0, "upper_bound": "0000110111"}, {"distinct_range": 0, "num_eq": 100, "num_range": 80000000000, "upper_bound": "0010000010"}, {"distinct_range": 60619145.609239176, "num_eq": 5000000, "num_range": 100000000, "upper_bound": "0011101101"}, {"distinct_range": 5940367.232281824, "num_eq": 9000000, "num_range": 9000000, "upper_bound": "0101100010"}, {"distinct_range": 0, "num_eq": 4554078212722546626, "num_range": 70000, "upper_bound": "0110000011"}, {"distinct_range": 490860734004484860, "num_eq": 4000000000, "num_range": 490860734004484840, "upper_bound": "0110011000"}, {"distinct_range": 0, "num_eq": 40000000000, "num_range": 600, "upper_bound": "1001011111"}, {"distinct_range": 0, "num_eq": 2021847540812914408, "num_range": 8000, "upper_bound": "1100110010"}], "histo_col_type": "BIT(10)", "name": "__auto__", "null_count": 0, "row_count": 3000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1687, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1672, "histo_col_type": "", "name": "__auto__", "null_count": 1055, "row_count": 3000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2208, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "173000407"}, {"distinct_range": 100000000, "num_eq": 8000, "num_range": 100000000, "upper_bound": "2351468560"}, {"distinct_range": 0, "num_eq": 2799605632374421080, "num_range": 2854029939525296936, "upper_bound": "2587729861"}, {"distinct_range": 352831655.70876974, "num_eq": 5093732932342672955, "num_range": 400000000, "upper_bound": "2654563042"}, {"distinct_range": 726275997421940400, "num_eq": 4415478922168032718, "num_range": 3306627085746459628, "upper_bound": "3312015076"}, {"distinct_range": 0, "num_eq": 400, "num_range": 9000000000, "upper_bound": "3898228094"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 3000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2251, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 934, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 549, "histo_col_type": "", "name": "__auto__", "null_count": 402, "row_count": 3000}]';
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_9) REFERENCES table1 (col1_3);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7251072227094567755, "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 429779473937415106, "row_count": 7547729433429684419}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3048503656724063051, "histo_buckets": [{"distinct_range": 0, "num_eq": 104147854098571342, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 2088423937988312000, "num_eq": 90, "num_range": 2088423937988312182, "upper_bound": "847022749"}, {"distinct_range": 5754950269563420000, "num_eq": 4627172448457908229, "num_range": 5754950269563419805, "upper_bound": "998191425"}, {"distinct_range": 2418326732021282300, "num_eq": 5923937386106751406, "num_range": 3216413525540417322, "upper_bound": "1566524307"}, {"distinct_range": 6196334252399729000, "num_eq": 3838306121139087788, "num_range": 6196334252399728471, "upper_bound": "1641559178"}, {"distinct_range": 6251673415623131000, "num_eq": 1575772350078489675, "num_range": 6890035507826471204, "upper_bound": "2691611596"}, {"distinct_range": 0, "num_eq": 2881329294302599817, "num_range": 1197175305138595128, "upper_bound": "3989876682"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 4848477026364148791, "row_count": 7547729433429684419}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1635576858491781785, "histo_col_type": "", "name": "__auto__", "null_count": 4616315536683299903, "row_count": 7547729433429684419}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4324407776265553898, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 8818253403995766000, "num_eq": 800000000, "num_range": 8818253403995765403, "upper_bound": "153035462"}, {"distinct_range": 700293456521473700, "num_eq": 5683883618103312866, "num_range": 2841282682967224175, "upper_bound": "1037825907"}, {"distinct_range": 109079216197035500, "num_eq": 8000, "num_range": 218240788229834828, "upper_bound": "1222275675"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 5027684090541800803, "upper_bound": "2916737853"}, {"distinct_range": 0, "num_eq": 500, "num_range": 5562203303529749946, "upper_bound": "2974496184"}, {"distinct_range": 5000000, "num_eq": 500000, "num_range": 5000000, "upper_bound": "3907925350"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 2687198821429200928, "row_count": 7547729433429684419}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2430610825465967270, "histo_buckets": [{"distinct_range": 0, "num_eq": 552583060033394555, "num_range": 0, "upper_bound": "00000000-0000-0000-0000-000000000000"}, {"distinct_range": 1000, "num_eq": 6679897662774148772, "num_range": 1000, "upper_bound": "141f491c-39fc-4023-884f-9b125cd0aae9"}, {"distinct_range": 1300330838768945400, "num_eq": 9178222612131838497, "num_range": 6982420457171155885, "upper_bound": "456a2a8a-3512-4387-942d-5eba227f0e36"}, {"distinct_range": 0, "num_eq": 9000000, "num_range": 5919922288167697636, "upper_bound": "a977f1e7-5b38-45e7-8269-baa28d927d9a"}, {"distinct_range": 200000, "num_eq": 100000000, "num_range": 200000, "upper_bound": "e390ffb0-a690-446e-815c-a7f183e2345f"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 5905057844431028269, "row_count": 7547729433429684419}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5899764344087545120, "histo_col_type": "", "name": "__auto__", "null_count": 6389386102945969388, "row_count": 7547729433429684419}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3902223719986425137, "histo_col_type": "", "name": "__auto__", "null_count": 5863558263269362906, "row_count": 7547729433429684419}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 43808319337846818, "histo_col_type": "", "name": "__auto__", "null_count": 2813198754675385379, "row_count": 7547729433429684419}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1505846470375998366, "histo_buckets": [{"distinct_range": 0, "num_eq": 9146073437473254226, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 6508013619119232108, "num_range": 6891694485222564579, "upper_bound": "1243079420"}, {"distinct_range": 0, "num_eq": 84416970773630308, "num_range": 4533620514067486321, "upper_bound": "2043877276"}, {"distinct_range": 3000, "num_eq": 100000000000, "num_range": 3000, "upper_bound": "3083466571"}, {"distinct_range": 2881224021645805600, "num_eq": 6000000000, "num_range": 2881224021645805391, "upper_bound": "3358618154"}, {"distinct_range": 0, "num_eq": 3905129362935870246, "num_range": 30, "upper_bound": "3971429042"}, {"distinct_range": 478885062.5943345, "num_eq": 300000, "num_range": 900000000, "upper_bound": "4142505875"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 7547729433429684419}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4396685947451755238, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-6996771442242772669"}, {"distinct_range": 2071453139402953500, "num_eq": 1430608382090128533, "num_range": 3358662897736850759, "upper_bound": "32767"}, {"distinct_range": 0, "num_eq": 6399812561828953774, "num_range": 10000000000, "upper_bound": "2077183867346503567"}, {"distinct_range": 0, "num_eq": 5000000000, "num_range": 126675416971567537, "upper_bound": "6264876062311766904"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 5745358040345651661, "row_count": 7547729433429684419}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5496923308433302991, "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 6184971249474651548, "row_count": 7547729433429684419}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2327466958518067938, "histo_col_type": "", "name": "__auto__", "null_count": 3454876172379575877, "row_count": 7547729433429684419}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7235837865790422563, "histo_col_type": "", "name": "__auto__", "null_count": 7088792184779692571, "row_count": 7547729433429684419}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6751117202082266621, "histo_col_type": "", "name": "__auto__", "null_count": 6638329627125330913, "row_count": 7547729433429684419}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5298493437831519308, "histo_col_type": "", "name": "__auto__", "null_count": 47882421045730281, "row_count": 7547729433429684419}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3984069118691908168, "histo_col_type": "", "name": "__auto__", "null_count": 1330182368576714408, "row_count": 9130561196678898071}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5461255750004485712, "histo_buckets": [{"distinct_range": 0, "num_eq": 8657740905192949683, "num_range": 0, "upper_bound": "-65 years -5 mons -607 days -03:02:48.643034"}, {"distinct_range": 6644859.769350977, "num_eq": 8000000, "num_range": 7000000, "upper_bound": "-55 years -8 mons -275 days -17:09:38.039907"}, {"distinct_range": 2696100830236330000, "num_eq": 3092250957536936442, "num_range": 2696100830236329895, "upper_bound": "-45 years -10 mons -846 days -04:06:07.265299"}, {"distinct_range": 0, "num_eq": 300, "num_range": 9000000, "upper_bound": "-27 years -5 mons -660 days -02:55:14.824028"}, {"distinct_range": 40736756.73672595, "num_eq": 4692140531123977397, "num_range": 70000000, "upper_bound": "18 years 4 mons 378 days 20:58:03.880664"}, {"distinct_range": 3000000, "num_eq": 8274878102522101190, "num_range": 3000000, "upper_bound": "65 years 9 mons 771 days 15:35:30.777101"}, {"distinct_range": 0, "num_eq": 7591576120596541164, "num_range": 10000000000, "upper_bound": "77 years 2 mons 785 days 23:44:01.684865"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 4396657804657629083, "row_count": 9130561196678898071}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2341931456237450552, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9130561196678898071}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2692245162442125655, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9130561196678898071}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2367320096094933967, "histo_buckets": [{"distinct_range": 0, "num_eq": 2750378051521363377, "num_range": 0, "upper_bound": "1975-01-10 12:45:12.000825+00:00"}, {"distinct_range": 70000, "num_eq": 1363966692870118880, "num_range": 70000, "upper_bound": "1992-03-16 08:57:18.000265+00:00"}, {"distinct_range": 0, "num_eq": 5593811955819011779, "num_range": 835438218072705036, "upper_bound": "1999-01-26 16:26:44.000041+00:00"}, {"distinct_range": 6680713562828663000, "num_eq": 3256840570894764926, "num_range": 6680713562828662332, "upper_bound": "2003-03-27 02:50:15.000507+00:00"}, {"distinct_range": 100000, "num_eq": 400000000, "num_range": 100000, "upper_bound": "2011-07-27 03:18:12.000894+00:00"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 1000000000, "upper_bound": "2013-12-24 03:09:22.00078+00:00"}, {"distinct_range": 3640508581765267500, "num_eq": 0, "num_range": 5771083565023378968, "upper_bound": "2019-12-30 14:48:13.000403+00:00"}, {"distinct_range": 0, "num_eq": 9000000000, "num_range": 3847705944907841779, "upper_bound": "2023-07-27 22:56:01.000496+00:00"}, {"distinct_range": 4000, "num_eq": 70000000, "num_range": 4000, "upper_bound": "2032-02-27 16:18:18.000647+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 9130561196678898071}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5215263156093337162, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000000, "num_range": 0, "upper_bound": "-9063637013103932982"}, {"distinct_range": 40000, "num_eq": 20000000000, "num_range": 40000, "upper_bound": "-6790602739177342008"}, {"distinct_range": 0, "num_eq": 7576906181088661152, "num_range": 6700206728088964271, "upper_bound": "7016806132296078153"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 6648383901503352484, "row_count": 9130561196678898071}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3298980028711802385, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9130561196678898071}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6106142140882031645, "histo_col_type": "", "name": "__auto__", "null_count": 1864815617133231407, "row_count": 9130561196678898071}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1286315773512263478, "histo_buckets": [{"distinct_range": 0, "num_eq": 5735965937169690154, "num_range": 0, "upper_bound": "-76 years -1 mons -160 days -21:50:47.045221"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 3688777115641492015, "upper_bound": "-22 years -3 mons -631 days -10:42:16.40437"}, {"distinct_range": 15.008233460054702, "num_eq": 8319811035945891865, "num_range": 20, "upper_bound": "-17 years -837 days -07:07:45.388392"}, {"distinct_range": 7978347369847449000, "num_eq": 0, "num_range": 7978347369847448301, "upper_bound": "48 years 5 mons 53 days 05:17:53.577374"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 3450176500068259796, "row_count": 9130561196678898071}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9109863197163486972, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9130561196678898071}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1117629364289162544, "histo_buckets": [{"distinct_range": 0, "num_eq": 3565864034011127721, "num_range": 0, "upper_bound": "261575758"}, {"distinct_range": 0, "num_eq": 500000, "num_range": 7955829002467048508, "upper_bound": "1001758941"}, {"distinct_range": 0, "num_eq": 60, "num_range": 2288626165919413416, "upper_bound": "2130493298"}, {"distinct_range": 379317135407628350, "num_eq": 6842436454962769697, "num_range": 379317135407628333, "upper_bound": "2420768325"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 9130561196678898071}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3261931977895100740, "histo_col_type": "", "name": "__auto__", "null_count": 1904067067241708665, "row_count": 9130561196678898071}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4436425991074538505, "histo_col_type": "", "name": "__auto__", "null_count": 1567454772894911006, "row_count": 9130561196678898071}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3294078186048044961, "histo_col_type": "", "name": "__auto__", "null_count": 4416936062589420086, "row_count": 9130561196678898071}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 935931488232035444, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "604303962"}, {"distinct_range": 4112861257115177500, "num_eq": 5385690296777842234, "num_range": 4112861257115177546, "upper_bound": "2630056695"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 9130561196678898071}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5279746383726522927, "histo_col_type": "", "name": "__auto__", "null_count": 2819605737944694021, "row_count": 9130561196678898071}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7794633023198703903, "histo_col_type": "", "name": "__auto__", "null_count": 8640255865767109922, "row_count": 9130561196678898071}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7314515754833917958, "histo_col_type": "", "name": "__auto__", "null_count": 6781667453708086301, "row_count": 9130561196678898071}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 866585838902789512, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2341168986027940109}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2302264695555232615, "histo_col_type": "", "name": "__auto__", "null_count": 1453121988162735998, "row_count": 2341168986027940109}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1912537616561996249, "histo_buckets": [{"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "0000100100"}, {"distinct_range": 2554937795047010000, "num_eq": 4201804688428363908, "num_range": 2554937795047009880, "upper_bound": "0000111001"}, {"distinct_range": 0, "num_eq": 4081627539359081782, "num_range": 1789527613680544643, "upper_bound": "1010000100"}, {"distinct_range": 80000, "num_eq": 0, "num_range": 80000, "upper_bound": "1010011000"}, {"distinct_range": 6000000000, "num_eq": 6138814742719621873, "num_range": 6000000000, "upper_bound": "1110000100"}], "histo_col_type": "BIT(10)", "name": "__auto__", "null_count": 0, "row_count": 2341168986027940109}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1329566707779878300, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2341168986027940109}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1555572613233652527, "histo_col_type": "", "name": "__auto__", "null_count": 2316682885497083289, "row_count": 2341168986027940109}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1509351089649997307, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2341168986027940109}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distin

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2586233&tab=buildLog) on [master@a786c51627fe66e47b4a4445c67b2a9077ae2a93](https://github.com/cockroachdb/cockroach/commits/a786c51627fe66e47b4a4445c67b2a9077ae2a93):

```
        postgres_1    | creating configuration files ... ok
        postgres_1    | running bootstrap script ... ok
        postgres_1    | performing post-bootstrap initialization ... ok
        postgres_1    | syncing data to disk ... ok
        postgres_1    | 
        postgres_1    | Success. You can now start the database server using:
        postgres_1    | 
        postgres_1    |     pg_ctl -D /var/lib/postgresql/data -l logfile start
        postgres_1    | 
        postgres_1    | 
        postgres_1    | WARNING: enabling "trust" authentication for local connections
        postgres_1    | You can change this by editing pg_hba.conf or using the option -A, or
        postgres_1    | --auth-local and --auth-host, the next time you run initdb.
        postgres_1    | waiting for server to start....2021-01-15 06:14:12.734 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-01-15 06:14:12.752 UTC [49] LOG:  database system was shut down at 2021-01-15 06:14:12 UTC
        postgres_1    | 2021-01-15 06:14:12.757 UTC [48] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        test_1        | --- FAIL: TestCompare (0.89s)
        test_1        |     --- FAIL: TestCompare/postgres (0.37s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.3:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.52s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 REGNAMESPACE NULL, col1_1 INET NOT NULL, col1_2 BOOL NOT NULL, col1_3 BOX2D, col1_4 TIMESTAMP NOT NULL, col1_5 OID NOT NULL, col1_6 REGCLASS, col1_7 INT8, col1_8 REGPROCEDURE, col1_9 REGNAMESPACE NOT NULL, col1_10 REGPROCEDURE, col1_11 "char", col1_12 OID, col1_13 TIMESTAMPTZ NOT NULL, col1_14 DECIMAL NOT NULL, col1_15 VARCHAR NOT NULL, col1_16 CHAR NOT NULL, col1_17 INT4 NOT NULL, PRIMARY KEY (col1_15, col1_17, col1_14 DESC, col1_5 ASC, col1_9 ASC, col1_16 ASC, col1_2 DESC, col1_13), INDEX (col1_8, col1_2, col1_6 ASC, col1_9, col1_17 DESC, col1_12 ASC, col1_0 ASC, col1_4 DESC, col1_13 DESC, col1_15, col1_11 ASC, col1_16 ASC, col1_7 DESC, col1_3, col1_5, col1_10 DESC, col1_1 DESC, col1_14 DESC) WHERE ((((((((table1.col1_16 < '"':::STRING) OR (table1.col1_7 > 2147483647:::INT8)) OR (table1.col1_4 < '-4713-11-24 00:00:00':::TIMESTAMP)) AND (table1.col1_11 != e'\x00':::STRING)) OR (table1.col1_13 <= '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) AND (NOT table1.col1_2)) AND (table1.col1_15 != '"':::STRING)) AND (table1.col1_14 <= 0:::DECIMAL)) OR (table1.col1_17 = 1:::INT8), UNIQUE (col1_12 ASC, col1_15, col1_3 DESC, col1_14 ASC, col1_16, col1_6 ASC, col1_13 ASC, col1_4) WHERE ((((((((table1.col1_14 <= 1:::DECIMAL) OR table1.col1_2) OR (table1.col1_13 > '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table1.col1_17 < (-1):::INT8)) OR (table1.col1_16 >= e'\U00002603':::STRING)) OR (table1.col1_15 <= e'\'':::STRING)) OR (table1.col1_7 <= 32767:::INT8)) AND (table1.col1_11 <= e'\x00':::STRING)) AND (table1.col1_4 >= '-4713-11-24 00:00:00':::TIMESTAMP), INDEX (col1_1, col1_2 DESC, col1_6 DESC, col1_14 ASC, col1_12 DESC, col1_15 ASC, col1_7 DESC, col1_16 DESC, col1_5, col1_10 DESC, col1_13 DESC, col1_8 DESC, col1_11 DESC) WHERE ((((((((table1.col1_7 >= 32767) AND (table1.col1_4 > '-4713-11-24 00:00:00')) OR table1.col1_2) OR (table1.col1_16 != 'X')) OR (table1.col1_15 > 'X')) AND (table1.col1_11 <= e'\U00002603')) AND (table1.col1_17 = -2147483648)) AND (table1.col1_14 != -Infinity)) AND (table1.col1_13 >= '0001-01-01 00:00:00+00:00'), INDEX (col1_3, col1_17 ASC) WHERE ((((((table1.col1_2 OR (table1.col1_13 < '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) AND (table1.col1_16 <= e'\x00':::STRING)) AND (table1.col1_14 = 'NaN':::DECIMAL)) AND (table1.col1_15 != '"':::STRING)) AND (table1.col1_7 != 2147483647:::INT8)) AND (table1.col1_17 = (-1):::INT8)) OR (table1.col1_11 < e'\'':::STRING), INDEX (col1_2 DESC, col1_8 ASC, col1_0 ASC, col1_15 ASC) STORING (col1_1, col1_4, col1_6, col1_10) WHERE ((((((((table1.col1_13 <= '-2000-01-01 00:00:00+00:00') OR (NOT table1.col1_2)) OR (table1.col1_17 < -32768)) AND (table1.col1_11 < e'\x00')) AND (table1.col1_15 != 'X')) OR (table1.col1_7 = -32768)) AND (table1.col1_4 != '-2000-01-01 00:00:00')) AND (table1.col1_16 >= e'\U00002603')) OR (table1.col1_14 >= 1), INDEX (col1_7 ASC, col1_6 DESC, col1_16 ASC, col1_2, col1_5, col1_15 ASC, col1_11 ASC) WHERE (((NOT table1.col1_2) AND (table1.col1_16 <= e'\x00':::STRING)) AND (table1.col1_17 != (-1):::INT8)) AND (table1.col1_13 < '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ), FAMILY (col1_2), FAMILY (col1_11), FAMILY (col1_1, col1_10, col1_15), FAMILY (col1_8), FAMILY (col1_16, col1_13, col1_3), FAMILY (col1_4, col1_9, col1_12, col1_5), FAMILY (col1_17), FAMILY (col1_6, col1_14), FAMILY (col1_7, col1_0));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13295, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18008, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 96, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 90000000000, "num_eq": 7598393395479745731, "num_range": 90000000000, "upper_bound": "1923797577"}, {"distinct_range": 305352934.8242142, "num_eq": 7828694652484579056, "num_range": 400000000, "upper_bound": "2539446842"}, {"distinct_range": 0, "num_eq": 1908182838249042485, "num_range": 90000000, "upper_bound": "2625580490"}, {"distinct_range": 4708146626643475000, "num_eq": 2702999361405121839, "num_range": 4708146626643475432, "upper_bound": "2860380064"}, {"distinct_range": 0, "num_eq": 7004008966128126766, "num_range": 100000, "upper_bound": "2948140094"}, {"distinct_range": 0, "num_eq": 7523944667848968152, "num_range": 8892534361729496143, "upper_bound": "3513475933"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 17535, "row_count": 30000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20503, "histo_col_type": "", "name": "__auto__", "null_count": 28599, "row_count": 30000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10435, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23468, "histo_buckets": [{"distinct_range": 0, "num_eq": 1183768304723106422, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 500, "upper_bound": "BOX(-2.016900674990105 -0.009096963651901713,-0.6025473987509937 0.38784683989474544)"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 10000000000, "upper_bound": "BOX(-1.8422311198600023 0.5803546492379063,0.6266206069544014 0.6205893066985482)"}, {"distinct_range": 666358758.5464189, "num_eq": 10000000, "num_range": 700000000, "upper_bound": "BOX(-1.2907413213628411 -0.4013002273597698,-1.0329777218739176 2.0934021930682003)"}, {"distinct_range": 3000000, "num_eq": 8000, "num_range": 3000000, "upper_bound": "BOX(-1.0526915362286948 -1.2471329503398731,0.9522475859374921 0.8106773744566038)"}, {"distinct_range": 0, "num_eq": 0, "num_range": 100, "upper_bound": "BOX(-1.0291701901416728 -0.939940164252479,-0.03556810191354717 1.089725043668243)"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 6856357178113722152, "upper_bound": "BOX(-0.7123971424862992 0.08587916095042725,0.21375424244130262 0.6132356174979474)"}, {"distinct_range": 4254488179724757500, "num_eq": 3000, "num_range": 5601913123612258290, "upper_bound": "BOX(1.9176380282388537 -0.893320357363563,1.9346634192963106 0.1393508661149779)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 29872, "row_count": 30000}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7107, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7285, "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5784, "histo_col_type": "", "name": "__auto__", "null_count": 20339, "row_count": 30000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16252, "histo_buckets": [{"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 1332172617247648078, "num_range": 7996921390671529279, "upper_bound": "74094455"}, {"distinct_range": 10000000000, "num_eq": 1325589090672007147, "num_range": 10000000000, "upper_bound": "1470369577"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 0, "upper_bound": "1615451883"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 9000, "upper_bound": "3801970502"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 27341, "row_count": 30000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3797, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9197, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 794, "histo_col_type": "", "name": "__auto__", "null_count": 20117, "row_count": 30000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14252, "histo_col_type": "", "name": "__auto__", "null_count": 18558, "row_count": 30000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3003, "histo_col_type": "INT8", "name": "__auto__", "null_count": 15961, "row_count": 30000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18055, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9299, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "\\tw0_"}, {"distinct_range": 40000000000, "num_eq": 3000000000, "num_range": 40000000000, "upper_bound": "\\tx-\\u001fUJ"}, {"distinct_range": 107666498935594450, "num_eq": 6000000000, "num_range": 107666498935594441, "upper_bound": "#\\u000cyVFFc"}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 700, "upper_bound": "&F\\u0016Y\x7f^B"}, {"distinct_range": 1540034103547114800, "num_eq": 200000, "num_range": 1540034103547114751, "upper_bound": "[lf{++e1I"}, {"distinct_range": 140835091.837886, "num_eq": 2869703005348456792, "num_range": 8000000000, "upper_bound": "j\\u00162c9\\u0018t\\u000cV"}, {"distinct_range": 4533448207748157000, "num_eq": 200, "num_range": 7510701136021583383, "upper_bound": "y="}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 50000000000, "upper_bound": "\U00002603"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17353, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 1035313917617290800, "num_eq": 5000000000, "num_range": 1035313917617290806, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 30000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('nyq');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('qlemxs', 'xgjh', 'clayv', 'kugc', 'zsqq');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('yo', 'a');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('sqvimy', 'ex', 'gf');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('jny', 'cqh');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 128109095872528635, "histo_col_type": "", "name": "__auto__", "null_count": 1047056177070534083, "row_count": 1586657355582241261}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1206532121784452766, "histo_buckets": [{"distinct_range": 0, "num_eq": 1151711885460678425, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 8859832911741105000, "num_eq": 1052921141405839194, "num_range": 8859832911741105249, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 1586657355582241261}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 617453946285243751, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1586657355582241261}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1528050056138942057, "histo_col_type": "", "name": "__auto__", "null_count": 582814497167755480, "row_count": 1586657355582241261}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 194048571978103035, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1586657355582241261}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 495955461120466803, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "BOX(-0.8899971714148431 -0.19586192602567137,1.3889844452315396 0.48428184147660525)"}, {"distinct_range": 800000000, "num_eq": 2500528609452131948, "num_range": 800000000, "upper_bound": "BOX(-0.8590597496399104 -0.8727187951197186,0.696597245317365 2.124476203493578)"}, {"distinct_range": 0, "num_eq": 5821484184309127413, "num_range": 1000, "upper_bound": "BOX(0.23240450409487745 0.38503058025327497,0.49308243882447944 1.065006553511049)"}, {"distinct_range": 0, "num_eq": 1106168490410728392, "num_range": 9000000, "upper_bound": "BOX(1.31226750258697 -1.4985937953353365,1.8810781150849412 0.5743642993832898)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 1446424799261192282, "row_count": 1586657355582241261}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 719653025047430904, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1586657355582241261}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1445842740045583709, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1586657355582241261}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 805516910780516708, "histo_col_type": "", "name": "__auto__", "null_count": 748031929707236699, "row_count": 1586657355582241261}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 428309012431908541, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000, "num_range": 0, "upper_bound": "3666495076"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 344640876249916594, "row_count": 1586657355582241261}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1177063082678118784, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1586657355582241261}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1363314401532916984, "histo_buckets": [{"distinct_range": 0, "num_eq": 1904414042180590828, "num_range": 0, "upper_bound": "107.188.63.88/6"}, {"distinct_range": 0, "num_eq": 122565448220252501, "num_range": 8232940614341127780, "upper_bound": "8.69.147.134/15"}, {"distinct_range": 2000000, "num_eq": 4487042495887109971, "num_range": 2000000, "upper_bound": "92.153.223.43/28"}, {"distinct_range": 0, "num_eq": 10, "num_range": 60000, "upper_bound": "b6f9:4e24:4cf4:c53c:df79:de3a:f004:fd76/95"}, {"distinct_range": 200000000, "num_eq": 4717362502295054043, "num_range": 200000000, "upper_bound": "d847:5e65:c950:8f22:2f75:82a:c6e7:fd1c/96"}, {"distinct_range": 3027458105811704300, "num_eq": 400000, "num_range": 3027458105811704380, "upper_bound": "d546:fca4:a200:a12f:cdf1:13e4:777:8488/103"}, {"distinct_range": 624034339.0457952, "num_eq": 10000000000, "num_range": 900000000, "upper_bound": "c04:3b34:466a:ff7c:5dab:3c50:6b6c:e2b4/105"}, {"distinct_range": 0, "num_eq": 7966893090565532080, "num_range": 0, "upper_bound": "d8c3:df97:266f:dea4:2ff9:ba3:e45f:8add/110"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 1586657355582241261}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 296402493142229439, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1586657355582241261}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1047634196929925661, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000, "num_range": 0, "upper_bound": "-8810016109309321139"}, {"distinct_range": 7737113449010845000, "num_eq": 5000000, "num_range": 7737113449010844618, "upper_bound": "-6487900236746007213"}, {"distinct_range": 7722588276304210000, "num_eq": 500000, "num_range": 7722588276304209584, "upper_bound": "-3741753099422109159"}, {"distinct_range": 100000000000, "num_eq": 8445278140993891251, "num_range": 100000000000, "upper_bound": "-2147483648"}, {"distinct_range": 0, "num_eq": 0, "num_range": 400000, "upper_bound": "-128"}, {"distinct_range": 0, "num_eq": 411670091982409128, "num_range": 80000000, "upper_bound": "1578721095138571438"}, {"distinct_range": 0, "num_eq": 1816329662116183282, "num_range": 5000000, "upper_bound": "5839775212093163806"}, {"distinct_range": 0, "num_eq": 7607422657531173416, "num_range": 2000000, "upper_bound": "9223372036854775807"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 887686704851355386, "row_count": 1586657355582241261}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1233729661626758669, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1586657355582241261}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1242754211721635218, "histo_col_type": "", "name": "__auto__", "null_count": 216934958911715562, "row_count": 1586657355582241261}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1001539559975110109, "histo_buckets": [{"distinct_range": 0, "num_eq": 358762059990486906, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 2103199184769081300, "num_eq": 10000000, "num_range": 2103199184769081373, "upper_bound": "4176902818"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 974553511799155855, "row_count": 1586657355582241261}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1223042187572129677, "histo_buckets": [{"distinct_range": 0, "num_eq": 8081677943857584328, "num_range": 0, "upper_bound": "\\u0006F{EW8\\u0007"}, {"distinct_range": 533973225786080450, "num_eq": 7800411381112281485, "num_range": 660842946054159185, "upper_bound": "<7\\u0016F"}, {"distinct_range": 0, "num_eq": 50000000, "num_range": 800000000, "upper_bound": "C\\u001aG&o"}, {"distinct_range": 8000000000, "num_eq": 6000, "num_range": 8000000000, "upper_bound": "w"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 1586657355582241261}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6602, "histo_col_type": "", "name": "__auto__", "null_count": 9758, "row_count": 10000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3852, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2359, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6401, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3767, "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7432, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 6328884637023376798, "upper_bound": "BOX(-0.5075834729050025 0.7599283084840369,0.7144085197556138 1.2605392346462132)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 4207, "row_count": 10000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6275, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000000, "num_range": 0, "upper_bound": "-7543289967917781156"}, {"distinct_range": 900000, "num_eq": 1224050388546961815, "num_range": 900000, "upper_bound": "2081496561392173082"}, {"distinct_range": 1567156.7473790187, "num_eq": 6240294354409749816, "num_range": 3000000, "upper_bound": "2242460377319687797"}, {"distinct_range": 500000, "num_eq": 3918398578435672595, "num_range": 500000, "upper_bound": "4831095162265917133"}, {"distinct_range": 8146199569067864000, "num_eq": 507410730577258020, "num_range": 8945133925422300761, "upper_bound": "7781687443958226873"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 6387, "row_count": 10000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7549, "histo_buckets": [{"distinct_range": 0, "num_eq": 1843426024290742074, "num_range": 0, "upper_bound": "524320120"}, {"distinct_range": 1873456373899379200, "num_eq": 5510746450354655412, "num_range": 2830009745108225149, "upper_bound": "665620456"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 6080889755839064014, "upper_bound": "1857790396"}, {"distinct_range": 2148102553607384600, "num_eq": 50000000000, "num_range": 2204059880229356157, "upper_bound": "4163735711"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 5684, "row_count": 10000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6528, "histo_col_type": "", "name": "__auto__", "null_count": 4107, "row_count": 10000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9807, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "51817076"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 1000, "upper_bound": "397265764"}, {"distinct_range": 0, "num_eq": 6767551105248901368, "num_range": 8000000000, "upper_bound": "3151990344"}, {"distinct_range": 24594007.086641524, "num_eq": 850684674024600391, "num_range": 200000000, "upper_bound": "3522446018"}, {"distinct_range": 0, "num_eq": 423379191090886636, "num_range": 3000000, "upper_bound": "3656357304"}, {"distinct_range": 3426866499147359700, "num_eq": 4100907495311370377, "num_range": 3426866499147359572, "upper_bound": "3703420621"}, {"distinct_range": 1151765594535552400, "num_eq": 700000, "num_range": 7360198749178618020, "upper_bound": "3796887781"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 7348, "row_count": 10000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8054, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9069, "histo_col_type": "", "name": "__auto__", "null_count": 6902, "row_count": 10000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2971, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4749, "histo_buckets": [{"distinct_range": 0, "num_eq": 5058533929086589344, "num_range": 0, "upper_bound": "29.44.16.189"}, {"distinct_range": 0, "num_eq": 3708032098495435056, "num_range": 500, "upper_bound": "1196:ed78:3337:906:d155:9a:f9af:8d23/21"}, {"distinct_range": 0, "num_eq": 3469731232796084994, "num_range": 60000000, "upper_bound": "e9ff:83ef:3523:ba86:6d11:e667:dcfa:e4fa/44"}, {"distinct_range": 80, "num_eq": 0, "num_range": 80, "upper_bound": "955f:b0dc:9570:4a33:707b:d528:1d8:eb7d/60"}, {"distinct_range": 70, "num_eq": 70000000000, "num_range": 70, "upper_bound": "7128:ed3:e219:b38d:f0d:d5a7:e556:b627/73"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 6552883946994559748, "upper_bound": "2d26:7eba:b1f8:61f1:641f:6669:341e:3af5/91"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2351, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6809, "histo_col_type": "", "name": "__auto__", "null_count": 5455, "row_count": 10000}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9257, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3426, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "infinity" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2589064&tab=buildLog) on [master@422560bfa1d125b50ca820c82b313c340cd51bec](https://github.com/cockroachdb/cockroach/commits/422560bfa1d125b50ca820c82b313c340cd51bec):

```
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-01-16 06:13:59.559 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-01-16 06:13:59.559 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-01-16 06:13:59.563 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-01-16 06:13:59.578 UTC [86] LOG:  database system was shut down at 2021-01-16 06:13:59 UTC
        test_1        | --- FAIL: TestCompare (2.40s)
        test_1        |     --- FAIL: TestCompare/postgres (1.53s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.2:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.87s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 DATE NOT NULL, col1_1 BYTES NOT NULL, col1_2 REGCLASS NOT NULL, col1_3 NAME NOT NULL, col1_4 FLOAT4 NOT NULL, col1_5 STRING NOT NULL, col1_6 "char" NOT NULL, col1_7 TIMESTAMPTZ NOT NULL, col1_8 TIMESTAMP[] NULL, col1_9 INET NOT NULL, col1_10 FLOAT4 NOT NULL, col1_11 BIT(10) NOT NULL, col1_12 BIT(17) NOT NULL, col1_13 BIT(3) NOT NULL, col1_14 TIMESTAMPTZ NOT NULL, col1_15 INT4 NOT NULL, PRIMARY KEY (col1_3 ASC, col1_14 ASC, col1_15 DESC, col1_9 DESC, col1_6, col1_4, col1_5, col1_11 ASC, col1_13, col1_0, col1_10 DESC, col1_2 ASC, col1_12), FAMILY (col1_2, col1_15, col1_10, col1_13, col1_8, col1_3), FAMILY (col1_9, col1_4, col1_6), FAMILY (col1_5), FAMILY (col1_14, col1_11, col1_1), FAMILY (col1_7, col1_0), FAMILY (col1_12), UNIQUE (col1_3, col1_2) STORING (col1_7, col1_8) WHERE (((((((((table1.col1_0 >= '4714-11-24 BC') AND (table1.col1_15 <= -1)) OR (table1.col1_7 < '-2000-01-01 00:00:00+00:00')) AND (table1.col1_3 <= e'\U00002603')) OR (table1.col1_4 != 1.401298464324817e-45)) AND (table1.col1_5 >= e'\'')) OR (table1.col1_1 = '\xff')) OR (table1.col1_6 = '')) AND (table1.col1_14 = '0001-01-01 00:00:00+00:00')) AND (table1.col1_10 < -Inf));
        test_1        |             CREATE TABLE table2 (col1_3 NAME NOT NULL, col1_14 TIMESTAMPTZ NOT NULL, col1_15 INT4 NOT NULL, col1_9 INET NOT NULL, col1_6 "char" NOT NULL, col1_4 FLOAT4 NOT NULL, col1_5 STRING NOT NULL, col1_11 BIT(10) NOT NULL, col1_13 BIT(3) NOT NULL, col1_0 DATE NOT NULL, col1_10 FLOAT4 NOT NULL, col1_2 REGCLASS NOT NULL, col1_12 BIT(17) NOT NULL, col2_13 UUID NULL, col2_14 STRING NOT NULL, col2_15 BIT(30) NOT NULL, col2_16 BIT(20), col2_17 INET NULL, col2_18 INT2, col2_19 CHAR NULL, col2_20 REGPROCEDURE NULL, col2_21 CHAR NOT NULL, col2_22 VARCHAR NULL, col2_23 INT8 NOT NULL, col2_24 OID, col2_25 UUID NOT NULL, col2_26 BIT(9) NULL, col2_27 OID NULL, col2_28 REGNAMESPACE, col2_29 OID, PRIMARY KEY (col1_3 ASC, col1_14 ASC, col1_15 DESC, col1_9 DESC, col1_6, col1_4, col1_5, col1_11 ASC, col1_13, col1_0, col1_10 DESC, col1_2 ASC, col1_12, col2_18 ASC, col2_17, col2_13 DESC, col2_14 DESC, col2_21, col2_15 DESC, col2_22, col2_16 DESC, col2_27 DESC, col2_20 DESC, col2_25 DESC, col2_26 DESC, col2_23, col2_29), INDEX (col2_24 ASC, col1_9, col2_14 DESC, col1_14 DESC, col2_19 ASC, col1_3 ASC, col2_23 DESC, col2_17, col1_12 ASC, col1_11 DESC, col2_13 DESC, col1_0 ASC, col1_13 ASC, col2_27 DESC, col2_16 DESC, col1_10 ASC, col2_20 DESC, col2_29 ASC, col1_15, col2_26, col2_15 DESC, col2_22 ASC, col1_5 ASC, col2_28 DESC, col1_4 ASC, col2_25, col2_18 DESC, col1_2 ASC, col2_21 ASC), INDEX (col1_6 DESC, col1_13 ASC, col1_3, col1_2, col1_4, col2_13, col2_25 ASC, col2_23, col1_11 ASC, col2_18, col1_12 DESC, col2_28 DESC, col2_16) STORING (col2_24), UNIQUE (col2_24, col1_12 ASC, col2_23 DESC, col2_18 ASC, col1_15 ASC, col2_20 DESC, col2_14, col1_13 DESC, col1_0 DESC, col2_25, col2_21, col2_29, col2_28, col2_17, col2_27 DESC, col1_2 ASC, col2_15 DESC, col2_22 DESC, col2_13 DESC, col1_4 DESC, col2_16 ASC, col2_19 DESC, col1_11 DESC, col1_3 ASC, col1_5), INDEX (col2_20 DESC, col2_27, col1_4 DESC, col2_18, col2_24 ASC, col1_3 DESC, col2_14, col1_6, col1_0, col1_12 ASC, col2_26 ASC, col2_16 DESC, col2_25 DESC, col2_13, col1_13, col2_17, col1_10 DESC, col1_9 DESC, col2_21 ASC, col2_15 DESC, col1_15 DESC, col1_11 ASC, col2_22 ASC, col2_29 ASC, col1_2 ASC, col1_14, col2_23 ASC, col2_28 ASC, col1_5 ASC) WHERE (((((table2.col1_3 >= '':::STRING) OR (table2.col2_18 <= (-1):::INT8)) AND (table2.col1_5 = '':::STRING)) OR (table2.col2_22 != '"':::STRING)) OR (table2.col1_0 >= '4714-11-24 BC':::DATE)) AND (table2.col1_4 <= 1.0:::FLOAT8), INDEX (col2_16 DESC, col2_27, col1_12 ASC, col2_21 DESC, col2_29 ASC, col1_13 DESC, col2_17 ASC, col2_23 ASC, col2_13 DESC, col2_15 ASC, col1_14 DESC, col1_9, col1_0 ASC, col1_3, col1_5 ASC, col1_10 ASC, col2_14, col1_4, col2_26 ASC, col2_22 DESC, col2_28, col2_19 ASC, col2_18, col2_24 ASC, col1_2 DESC, col1_15), FAMILY (col2_25), FAMILY (col1_15, col1_11), FAMILY (col2_28), FAMILY (col1_9, col2_13, col2_24, col1_14), FAMILY (col2_20, col2_15), FAMILY (col2_17), FAMILY (col2_27), FAMILY (col2_14, col2_23, col1_5, col1_12, col2_19), FAMILY (col1_0, col1_10), FAMILY (col1_4, col2_21), FAMILY (col1_3, col2_16), FAMILY (col2_22), FAMILY (col2_18), FAMILY (col2_26, col1_13, col1_2, col1_6), FAMILY (col2_29), UNIQUE (col1_14)) INTERLEAVE IN PARENT table1 (col1_3, col1_14, col1_15, col1_9, col1_6, col1_4, col1_5, col1_11, col1_13, col1_0, col1_10, col1_2, col1_12);
        test_1        |             CREATE TABLE table3 (col3_0 REGPROC NULL, col3_1 "char" NULL, col3_2 BIT(40) NULL, col3_3 STRING, col3_4 TIMESTAMP[] NOT NULL, col3_5 VARCHAR[], col3_6 BOOL NOT NULL, col3_7 BIT(34) NULL, col3_8 VARCHAR, UNIQUE (col3_8 ASC, col3_3 DESC) STORING (col3_0, col3_1, col3_2, col3_6, col3_7) WHERE (((table3.col3_1 > '"':::STRING) OR (table3.col3_3 <= e'\x00':::STRING)) AND (table3.col3_8 > e'\U00002603':::STRING)) AND (NOT table3.col3_6), UNIQUE (col3_7, col3_6 ASC, col3_8) STORING (col3_0, col3_1, col3_2, col3_4), INDEX (col3_1 DESC, col3_2 ASC, col3_6 DESC, col3_3) STORING (col3_0, col3_5, col3_8) WHERE ((table3.col3_1 >= '':::STRING) AND (NOT table3.col3_6)) OR (table3.col3_8 >= e'\'':::STRING), UNIQUE (col3_8 DESC, col3_3 ASC) STORING (col3_1, col3_4, col3_7), INDEX (col3_8, col3_1, col3_0, col3_7 DESC, col3_3 DESC) STORING (col3_2, col3_4, col3_5, col3_6) WHERE (((table3.col3_3 > 'X':::STRING) AND (table3.col3_8 = e'\U00002603':::STRING)) AND (NOT table3.col3_6)) OR (table3.col3_1 != e'\x00':::STRING), INDEX (col3_2 DESC, col3_1 ASC) STORING (col3_3, col3_5, col3_7, col3_8), FAMILY (col3_2), FAMILY (col3_8), FAMILY (col3_1, col3_4, col3_5), FAMILY (col3_3), FAMILY (col3_0, col3_6, col3_7));
        test_1        |             CREATE TABLE table4 (col4_0 REGTYPE, col4_1 FLOAT4 NOT NULL, col4_2 BOOL NULL, col4_3 TIMETZ NULL, col4_4 INT4, col4_5 DATE NULL, col4_6 INT2, col4_7 BOX2D, col4_8 VARCHAR NULL, col4_9 "char", col4_10 INTERVAL NOT NULL, col4_11 NAME, col4_12 REGCLASS NULL, col4_13 INT4 NOT NULL, col4_14 TIMETZ, col4_15 REGCLASS NOT NULL, col4_16 FLOAT8, col4_17 STRING NOT NULL, PRIMARY KEY (col4_17 DESC, col4_15), UNIQUE (col4_6 ASC, col4_9 DESC, col4_7, col4_5, col4_17, col4_11, col4_1 DESC, col4_0 DESC, col4_12 ASC, col4_10 ASC, col4_13 ASC) STORING (col4_3, col4_8, col4_14, col4_16), UNIQUE (col4_14 DESC, col4_9, col4_6 ASC, col4_11 DESC, col4_13, col4_8 DESC, col4_3 ASC, col4_2 ASC, col4_1 DESC, col4_5 DESC, col4_7 DESC, col4_15 DESC, col4_10 ASC, col4_16, col4_12 DESC) WHERE ((((((((((((table4.col4_8 < e'\'':::STRING) AND (table4.col4_1 > 1.0:::FLOAT8)) AND (table4.col4_16 >= 3.4028234663852886e+38:::FLOAT8)) AND (table4.col4_11 < '"':::STRING)) AND (table4.col4_4 <= (-128):::INT8)) AND (table4.col4_17 = e'\'':::STRING)) OR (table4.col4_3 >= '00:00:00+15:59:00':::TIMETZ)) AND (table4.col4_6 <= 0:::INT8)) AND table4.col4_2) AND (table4.col4_9 < '':::STRING)) AND (table4.col4_13 >= (-2147483648):::INT8)) AND (table4.col4_14 != '00:00:00+15:59:00':::TIMETZ)) OR (table4.col4_5 = 'infinity':::DATE), INDEX (col4_1 DESC, col4_12 ASC, col4_15, col4_9 ASC) STORING (col4_0, col4_4, col4_7, col4_8, col4_10, col4_14, col4_16) WHERE (((((((table4.col4_9 >= 'X':::STRING) OR (table4.col4_6 = 1:::INT8)) AND (table4.col4_13 = 1:::INT8)) OR (table4.col4_3 < '00:00:00+15:59:00':::TIMETZ)) OR (table4.col4_1 > 'NaN':::FLOAT8)) AND (table4.col4_17 <= e'\U00002603':::STRING)) AND (table4.col4_16 > 1.0:::FLOAT8)) AND (table4.col4_5 = '5874897-12-31':::DATE), UNIQUE (col4_10, col4_6, col4_15, col4_16 DESC, col4_7 ASC, col4_9 DESC, col4_17 DESC, col4_5 ASC, col4_14, col4_4, col4_0 ASC) STORING (col4_8, col4_12, col4_13), INDEX (col4_9 DESC, col4_11, col4_14 ASC, col4_10 DESC, col4_1 ASC, col4_7 DESC, col4_6 DESC, col4_12 DESC, col4_15 DESC, col4_17 DESC, col4_4, col4_5 DESC, col4_8 ASC, col4_2, col4_3 DESC), INDEX (col4_17 DESC, col4_7 ASC, col4_11, col4_12 DESC, col4_16, col4_9, col4_0 DESC, col4_1, col4_10 ASC, col4_3 DESC, col4_6 DESC, col4_13 ASC) STORING (col4_4, col4_14) WHERE table4.col4_8 > 'X', INDEX (col4_10 DESC, col4_0, col4_3, col4_5 ASC, col4_6 DESC, col4_11, col4_1 ASC) STORING (col4_7, col4_8, col4_12, col4_13) WHERE ((((((NOT table4.col4_2) OR (table4.col4_14 <= '00:00:00+15:59:00')) OR (table4.col4_4 >= 1)) AND (table4.col4_13 < -2147483648)) OR (table4.col4_3 >= '00:00:00+15:59:00')) AND (table4.col4_9 > e'\U00002603')) AND (table4.col4_16 > +Inf), UNIQUE (col4_1 ASC, col4_4, col4_3 DESC, col4_0 DESC, col4_12 ASC, col4_15, col4_9, col4_11, col4_2, col4_7 DESC, col4_5 ASC, col4_17) WHERE ((((((((((((table4.col4_14 < '00:00:00+15:59:00':::TIMETZ) AND (table4.col4_13 > 1:::INT8)) OR (table4.col4_9 != e'\'':::STRING)) OR (table4.col4_17 > e'\U00002603':::STRING)) AND table4.col4_2) AND (table4.col4_1 != 'NaN':::FLOAT8)) OR (table4.col4_16 < '-Inf':::FLOAT8)) OR (table4.col4_5 > '1970-01-01':::DATE)) OR (table4.col4_3 != '24:00:00-15:59:00':::TIMETZ)) AND (table4.col4_11 = '"':::STRING)) OR (table4.col4_8 = e'\x00':::STRING)) OR (table4.col4_4 = (-128):::INT8)) AND (table4.col4_6 < 32767:::INT8), UNIQUE (col4_2, col4_6 DESC, col4_10 DESC, col4_15 ASC, col4_4 DESC, col4_17 ASC, col4_11, col4_13) STORING (col4_3, col4_8, col4_12, col4_14) WHERE (((((((((((table4.col4_9 <= e'\x00':::STRING) AND (table4.col4_13 > (-2147483648):::INT8)) OR (table4.col4_5 >= '-infinity':::DATE)) OR (table4.col4_6 = 0:::INT8)) AND (table4.col4_16 != '-Inf':::FLOAT8)) OR table4.col4_2) AND (table4.col4_3 < '00:00:00+15:59:00':::TIMETZ)) OR (table4.col4_4 <= (-2147483648):::INT8)) AND (table4.col4_17 > e'\'':::STRING)) AND (table4.col4_14 <= '24:00:00-15:59:00':::TIMETZ)) OR (table4.col4_11 < e'\'':::STRING)) AND (table4.col4_8 <= 'X':::STRING), FAMILY (col4_7), FAMILY (col4_9, col4_1), FAMILY (col4_13), FAMILY (col4_16, col4_6, col4_3), FAMILY (col4_10), FAMILY (col4_12), FAMILY (col4_17, col4_8, col4_14, col4_4), FAMILY (col4_5, col4_0, col4_2), FAMILY (col4_11, col4_15));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 165789646120859888, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4518424148999533267}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2685918057876230825, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4518424148999533267}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3084283862046118580, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4518424148999533267}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3597888320904306839, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4518424148999533267}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4129843744108228191, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4518424148999533267}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3069876388256281425, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4518424148999533267}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1700970633267392905, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4518424148999533267}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1738686331995108345, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4518424148999533267}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4372400454787032287, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4518424148999533267}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2473005545013479774, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4518424148999533267}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1598749245522852771, "histo_col_type": "", "name": "__auto__", "null_count": 3300134006498667954, "row_count": 4518424148999533267}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1287789431946952505, "histo_buckets": [{"distinct_range": 0, "num_eq": 8039109476941714453, "num_range": 0, "upper_bound": "\\u0013j\\u001d\\u0016$\\u0007]\\u001b&"}, {"distinct_range": 13948015.282837875, "num_eq": 2000000, "num_range": 20000000, "upper_bound": "\\u001a*\\u0019c"}, {"distinct_range": 0, "num_eq": 30000000000, "num_range": 9064492187465082908, "upper_bound": "\\""}, {"distinct_range": 42375354728.152306, "num_eq": 0, "num_range": 100000000000, "upper_bound": ")z)"}, {"distinct_range": 0, "num_eq": 200000000, "num_range": 100000, "upper_bound": "I\\u0004l\\n<4M"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 5664411944665850469, "upper_bound": "aK\\u0002"}, {"distinct_range": 400000000, "num_eq": 7000000000, "num_range": 400000000, "upper_bound": "ac\\u0014\\u001d\\\\y"}, {"distinct_range": 0, "num_eq": 5626570758365283408, "num_range": 80000, "upper_bound": "p;"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 4518424148999533267}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 888977622904915728, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4518424148999533267}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2988794518677765429, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4518424148999533267}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2745758581415025260, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4518424148999533267}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3060896472223202727, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4518424148999533267}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 164756, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 147900, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 226287, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 134402, "histo_col_type": "", "name": "__auto__", "null_count": 224410, "row_count": 300000}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8876, "histo_col_type": "", "name": "__auto__", "null_count": 287209, "row_count": 300000}, {"columns": ["col2_27"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 225534, "histo_col_type": "", "name": "__auto__", "null_count": 251142, "row_count": 300000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 270416, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 47794, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 293340, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53218, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col2_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 203827, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 114057, "histo_buckets": [{"distinct_range": 0, "num_eq": 374700380745146929, "num_range": 0, "upper_bound": "\\u0013\\u001b\\u0015"}, {"distinct_range": 5.045727320567305, "num_eq": 5699527669237608520, "num_range": 40, "upper_bound": "\'"}, {"distinct_range": 36939.04248117149, "num_eq": 300000, "num_range": 80000, "upper_bound": "4!Bf@"}, {"distinct_range": 0, "num_eq": 6618583488621495305, "num_range": 2160661688651112109, "upper_bound": "7-\\u0010H\\\\/1\\u001e"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 3883515977893644723, "upper_bound": "JIj$,"}, {"distinct_range": 5703992161616165000, "num_eq": 7205591786753380716, "num_range": 5703992161616164736, "upper_bound": "b!\\u0006!y\\u001f\\t"}, {"distinct_range": 1048212933644634900, "num_eq": 30000000, "num_range": 1048212933644634886, "upper_bound": "n"}, {"distinct_range": 1468488099053148700, "num_eq": 1155313851963545498, "num_range": 1590013993365594913, "upper_bound": "p?Ui9e"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col2_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 150321, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col2_26"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 88414, "histo_col_type": "", "name": "__auto__", "null_count": 202738, "row_count": 300000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 294368, "histo_buckets": [{"distinct_range": 0, "num_eq": 937140540604877772, "num_range": 0, "upper_bound": "\\u0011"}, {"distinct_range": 0, "num_eq": 2495688958807631890, "num_range": 900000000, "upper_bound": "\\u001c"}, {"distinct_range": 0, "num_eq": 2612742108201072673, "num_range": 600000, "upper_bound": "&"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 100000, "upper_bound": "5"}, {"distinct_range": 5000, "num_eq": 3399904332428507807, "num_range": 5000, "upper_bound": "M"}, {"distinct_range": 0, "num_eq": 1206001194443363880, "num_range": 5562133009811792929, "upper_bound": "`"}, {"distinct_range": 0, "num_eq": 9000000000, "num_range": 60000000, "upper_bound": "l"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 106329, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 75017, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 177693, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 0, "num_range": 6187905101058923325, "upper_bound": "1076849470"}, {"distinct_range": 200, "num_eq": 2843665709632181331, "num_range": 200, "upper_bound": "2653742282"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 100000000, "upper_bound": "3091930915"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 79663, "row_count": 300000}, {"columns": ["col2_29"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 256129, "histo_col_type": "", "name": "__auto__", "null_count": 277925, "row_count": 300000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 285560, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 227685, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 198344, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 224468, "histo_col_type": "", "name": "__auto__", "null_count": 203883, "row_count": 300000}, {"columns": ["col2_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 250953, "histo_col_type": "OID", "name": "__auto__", "null_count": 25550, "row_count": 300000}, {"columns": ["col2_25"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 239380, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col2_28"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 145278, "histo_col_type": "", "name": "__auto__", "null_count": 72173, "row_count": 300000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 241447, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 68264, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000, "num_range": 0, "upper_bound": "00010000011001100001"}, {"distinct_range": 1919.9030169232592, "num_eq": 70000, "num_range": 3000, "upper_bound": "00011101111101000011"}, {"distinct_range": 274583197851755680, "num_eq": 2537694492782097747, "num_range": 1950134577321722873, "upper_bound": "01010010001010000100"}, {"distinct_range": 0, "num_eq": 1180208329788888374, "num_range": 0, "upper_bound": "10000011010110100001"}, {"distinct_range": 0, "num_eq": 4720933159536173806, "num_range": 9000000000, "upper_bound": "10111111000100011010"}], "histo_col_type": "BIT(20)", "name": "__auto__", "null_count": 21919, "row_count": 300000}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 50364, "histo_col_type": "", "name": "__auto__", "null_count": 199893, "row_count": 300000}, {"columns": ["col2_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 126153, "histo_col_type": "", "name": "__auto__", "null_count": 46868, "row_count": 300000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "."}, {"distinct_range": 20000000000, "num_eq": 10000000, "num_range": 20000000000, "upper_bound": "4"}, {"distinct_range": 1943591646456984300, "num_eq": 1834966377864661542, "num_range": 1943591646456984321, "upper_bound": "6"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 1606455133962881621, "upper_bound": "L"}, {"distinct_range": 531942.8575283359, "num_eq": 3766135615592216244, "num_range": 700000, "upper_bound": "d"}, {"distinct_range": 8973456647737912000, "num_eq": 5000000000, "num_range": 8973456647737912034, "upper_bound": "\U00002603"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 8211363710327096693, "num_range": 0, "upper_bound": "0001101110101010100000110101100110"}, {"distinct_range": 2226690996958794800, "num_eq": 1359532869024845270, "num_range": 2226690996958794877, "upper_bound": "0001111111000101000010000111111111"}, {"distinct_range": 7492720929168604000, "num_eq": 7953884806523863283, "num_range": 7492720929168604389, "upper_bound": "0101101011000111100101100010111011"}, {"distinct_range": 205.9697182668953, "num_eq": 6000000000, "num_range": 400, "upper_bound": "1001010010010010011011101110001010"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 6885737464297430983, "upper_bound": "1011010100101000000010001001000111"}, {"distinct_range": 4702560.846138577, "num_eq": 6835871413958607661, "num_range": 50000000, "upper_bound": "1100111001011100001110100111100110"}, {"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": "1110000100010111001100010000100010"}, {"distinct_range": 237.6344584475076, "num_eq": 1135146368722740920, "num_range": 900, "upper_bound": "1110100011101010110010101111110000"}, {"distinct_range": 1540978371798732800, "num_eq": 5016822225859614108, "num_range": 2179970385678995652, "upper_bound": "1111100111111001110000100100110110"}], "histo_col_type": "BIT(34)", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 300000, "num_range": 0, "upper_bound": "0111111111111111111111111111111111111111"}], "histo_col_type": "BIT(40)", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 7390487801176638073, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "\\u0007"}, {"distinct_range": 42044.14923936516, "num_eq": 800000, "num_range": 90000, "upper_bound": "\\nE"}, {"distinct_range": 0, "num_eq": 40, "num_range": 10000000, "upper_bound": "\'"}, {"distinct_range": 0, "num_eq": 2629637747660779988, "num_range": 5000, "upper_bound": "D\\u0016\\n\\u0006\\u0008P"}, {"distinct_range": 7978116357646340000, "num_eq": 400000000, "num_range": 7978116357646340575, "upper_bound": "K<\\u001afaW\\u0001*"}, {"distinct_range": 3657596.319215924, "num_eq": 1015636059696824602, "num_range": 4000000, "upper_bound": "w\\u001f5\\u0004"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 746, "histo_col_type": "", "name": "__auto__", "null_count": 355, "row_count": 900}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 730, "histo_buckets": [{"distinct_range": 0, "num_eq": 4127746410988250084, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 3263131227808896000, "num_eq": 700, "num_range": 3263131227808896209, "upper_bound": "01:14:47.151132+13:27:00"}, {"distinct_range": 29.081862029467082, "num_eq": 1539761169038756708, "num_range": 100, "upper_bound": "15:38:52.539584-09:33:00"}, {"distinct_range": 191210313.84186476, "num_eq": 100000000000, "num_range": 600000000, "upper_bound": "15:21:07.391699-13:02:00"}, {"distinct_range": 6543644575704882000, "num_eq": 3197185750403139127, "num_range": 6543644575704882528, "upper_bound": "18:53:14.966656-12:01:00"}, {"distinct_range": 6482426782294282000, "num_eq": 300000, "num_range": 6482426782294282744, "upper_bound": "18:49:45.375608-12:10:00"}, {"distinct_range": 0, "num_eq": 0, "num_range": 1520900061679449652, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 357, "row_count": 900}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 291, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 2620294482024267745, "num_range": 3170238419189502022, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 12, "row_count": 900}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 798, "histo_col_type": "INT2", "name": "__auto__", "null_count": 274, "row_count": 900}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 266, "histo_col_type": "", "name": "__auto__", "null_count": 7, "row_count": 900}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 786, "histo_col_type": "", "name": "__auto__", "null_count": 92, "row_count": 900}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 533, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000000, "num_range": 0, "upper_bound": "%"}, {"distinct_range": 100000, "num_eq": 0, "num_range": 100000, "upper_bound": "6"}, {"distinct_range": 2934969959514415600, "num_eq": 90000, "num_range": 2934969959514415731, "upper_bound": "r"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 276, "row_count": 900}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 890, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 607, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 578, "histo_col_type": "", "name": "__auto__", "null_count": 89, "row_count": 900}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 103, "histo_buckets": [{"distinct_range": 0, "num_eq": 500, "num_range": 0, "upper_bound": ""}, {"distinct_range": 2729980901705647600, "num_eq": 800000, "num_range": 2729980901705647744, "upper_bound": "\\u0000"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 3131052097073983661, "upper_bound": "+"}, {"distinct_range": 9884658.614902265, "num_eq": 70000, "num_range": 10000000, "upper_bound": "<ia\\u001eSsGM"}, {"distinct_range": 28972399.740428694, "num_eq": 90000, "num_range": 40000000, "upper_bound": "[SsHd "}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 8000000, "upper_bound": "j\\u000f_ "}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 166, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": "-0.008807048201560974"}, {"distinct_range": 2.8127019591324407, "num_eq": 3899193360053689383, "num_range": 30, "upper_bound": "0.039569973945617676"}, {"distinct_range": 1737345311517275000, "num_eq": 20000, "num_range": 3971083042767529390, "upper_bound": "0.32344433665275574"}, {"distinct_range": 5131612166056552000, "num_eq": 2000000, "num_range": 5131612166056552578, "upper_bound": "1.666284441947937"}, {"distinct_range": 100, "num_eq": 6916028015518049817, "num_range": 100, "upper_bound": "2.101722240447998"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 215, "histo_col_type": "", "name": "__auto__", "null_count": 30, "row_count": 900}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 280, "histo_col_type": "", "name": "__auto__", "null_count": 53, "row_count": 900}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 407, "histo_col_type": "", "name": "__auto__", "null_count": 874, "row_count": 900}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 287, "histo_buckets": [{"distinct_range": 0, "num_eq": 8743025789767407045, "num_range": 0, "upper_bound": "-8 years -3 mons -173 days -11:34:42.252923"}, {"distinct_range": 1308.8098388931323, "num_eq": 5988167791553266473, "num_range": 2000, "upper_bound": "-1 years -3 mons -467 days -22:19:36.848945"}, {"distinct_range": 6634126597844151000, "num_eq": 400000000, "num_range": 6634126597844151646, "upper_bound": "42 years 5 mons 980 days 13:58:03.325457"}, {"distinct_range": 9312391.004600905, "num_eq": 9138648373864819194, "num_range": 10000000, "upper_bound": "65 years 323 days 13:48:25.972374"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 501, "histo_col_type": "", "name": "__auto__", "null_count": 235, "row_count": 900}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 872, "histo_col_type": "", "name": "__auto__", "null_count": 83, "row_count": 900}]':::JSONB;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_22, col2_24) REFERENCES table1 (col1_3, col1_2);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('yrke', 'gv', 'wuonvq', 'a', 'q', 'i');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('uxrbhg', 'mqtn', 'eqfyds', 'fdbiz', 'loq', 'yyc');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('ypoc');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('tkp', 'ibhsu');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5514618781826168116, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7968504722793522435}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2482256694789608408, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7968504722793522435}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2841489515992033202, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7968504722793522435}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 874911657583048162, "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 7968504722793522435}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4882389706357324303, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7968504722793522435}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4322189730033489045, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7968504722793522435}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3471616798912765093, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7968504722793522435}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4568754190853703544, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7968504722793522435}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4586223970367496006, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7968504722793522435}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6816354595665128071, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7968504722793522435}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3633592640724981264, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7968504722793522435}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6570134597838934458, "histo_col_type": "", "name": "__auto__", "null_count": 3020377444732870245, "row_count": 7968504722793522435}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 719922441078118269, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7968504722793522435}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1647107586388161054, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7968504722793522435}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3005704593748098872, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7968504722793522435}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4251434184031407916, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7968504722793522435}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6248756209103152230, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1600910460167090637, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3779670394716372244, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3760831243476283905, "histo_buckets": [{"distinct_range": 0, "num_eq": 8380526251066247878, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 663970571050822700, "num_eq": 8000000000, "num_range": 1262864741917310083, "upper_bound": "1727207244"}, {"distinct_range": 74.49312775832288, "num_eq": 100000000000, "num_range": 100, "upper_bound": "1973403896"}, {"distinct_range": 549523.907058668, "num_eq": 8000, "num_range": 2000000, "upper_bound": "2186270375"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 6496930366262872084, "row_count": 6790683112721254132}, {"columns": ["col2_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4352242629700243048, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col2_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3855350614887239879, "histo_buckets": [{"distinct_range": 0, "num_eq": 8414270802995555439, "num_range": 0, "upper_bound": "3179012490"}, {"distinct_range": 1603873093210648000, "num_eq": 0, "num_range": 4061251642149568439, "upper_bound": "3359526341"}, {"distinct_range": 5324.422751895326, "num_eq": 4000000, "num_range": 8000, "upper_bound": "4159557755"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 2082296251820916041, "row_count": 6790683112721254132}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 697866241351086681, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5710776445767054384, "histo_col_type": "", "name": "__auto__", "null_count": 1432998427894815986, "row_count": 6790683112721254132}, {"columns": ["col2_28"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5249813432224368706, "histo_col_type": "", "name": "__auto__", "null_count": 6604503068789046596, "row_count": 6790683112721254132}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2638370416802346906, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5374663262708618381, "histo_buckets": [{"distinct_range": 0, "num_eq": 2392862194603258274, "num_range": 0, "upper_bound": "\\u0010"}, {"distinct_range": 0, "num_eq": 1157777190798733640, "num_range": 4484092844433642398, "upper_bound": "\\u001a"}, {"distinct_range": 1504067460127584300, "num_eq": 70000, "num_range": 3313036078909817190, "upper_bound": "B"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 8384760515705017492, "upper_bound": "v"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4733961978472121451, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 736054487161895842, "histo_col_type": "", "name": "__auto__", "null_count": 5351774053025277808, "row_count": 6790683112721254132}, {"columns": ["col2_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 295896493131901975, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col2_25"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5944184697845546285, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col2_27"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2728386594498852637, "histo_col_type": "", "name": "__auto__", "null_count": 5550526493725047750, "row_count": 6790683112721254132}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2352493265274592460, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6724997956339594837, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col2_29"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 774136815199216965, "histo_col_type": "", "name": "__auto__", "null_count": 1139342521983213056, "row_count": 6790683112721254132}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1075276354390127335, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6744957163793248162, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1945722336230468711, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "00000000000000000000"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 70, "upper_bound": "00111010000110110011"}, {"distinct_range": 35552294.01691393, "num_eq": 9114629907967447414, "num_range": 50000000, "upper_bound": "01011100111011111011"}, {"distinct_range": 15.714573151849947, "num_eq": 5526248426990921138, "num_range": 80, "upper_bound": "10101100101110010111"}], "histo_col_type": "BIT(20)", "name": "__auto__", "null_count": 5599445949907909430, "row_count": 6790683112721254132}, {"columns": ["col2_26"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4954640047566724634, "histo_col_type": "", "name": "__auto__", "null_count": 1764258431178175045, "row_count": 6790683112721254132}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3544161634665830058, "histo_buckets": [{"distinct_range": 0, "num_eq": 2332856445888892961, "num_range": 0, "upper_bound": ""}, {"distinct_range": 2879618798617612000, "num_eq": 10000000000, "num_range": 2879618798617611998, "upper_bound": "\\u00166TC\\u001b(owS"}, {"distinct_range": 4483905118692047400, "num_eq": 1000, "num_range": 5374303927137633306, "upper_bound": "+"}, {"distinct_range": 0, "num_eq": 5464675398549611605, "num_range": 6232810542609275099, "upper_bound": "CG?R\\u0002"}, {"distinct_range": 600, "num_eq": 2927373930931001711, "num_range": 600, "upper_bound": "LWD"}, {"distinct_range": 3000, "num_eq": 500000, "num_range": 3000, "upper_bound": "l"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 2109744046842575741, "upper_bound": "pDGz"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2559900595052459961, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 869292363881395196, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4569606601908917684, "histo_col_type": "", "name": "__auto__", "null_count": 997647086976911651, "row_count": 6790683112721254132}, {"columns": ["col2_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3852652872805236124, "histo_col_type": "", "name": "__auto__", "null_count": 6003198619072184074, "row_count": 6790683112721254132}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5552228725045299272, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6790683112721254132}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 710731381119779363, "histo_col_type": "", "name": "__auto__", "null_count": 4920380554330957564, "row_count": 6790683112721254132}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1, "histo_buckets": [{"distinct_range": 0, "num_eq": 5600284941876055581, "num_range": 0, "upper_bound": "\\u0001"}, {"distinct_range": 7750786705733612000, "num_eq": 8858439320171619199, "num_range": 7750786705733611185, "upper_bound": "\\u0010"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 100000000, "upper_bound": "\'"}, {"distinct_range": 4100904.104569697, "num_eq": 8358358152090945368, "num_range": 50000000, "upper_bound": "3"}, {"distinct_range": 3163745739459554300, "num_eq": 80000, "num_range": 3163745739459554419, "upper_bound": "A"}, {"distinct_range": 600, "num_eq": 100000, "num_range": 600, "upper_bound": "R"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 3, "row_count": 10}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5, "histo_buckets": [{"distinct_range": 0, "num_eq": 6799494867581555477, "num_range": 0, "upper_bound": "1111000011001001010110111111100011001010"}], "histo_col_type": "BIT(40)", "name": "__auto__", "null_count": 1, "row_count": 10}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5, "histo_col_type": "", "name": "__auto__", "null_count": 5, "row_count": 10}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7, "histo_col_type": "", "name": "__auto__", "null_count": 3, "row_count": 10}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6, "histo_buckets": [{"distinct_range": 0, "num_eq": 3557892187893024786, "num_range": 0, "upper_bound": "0000000000000000000000000000000000"}], "histo_col_type": "BIT(34)", "name": "__auto__", "null_count": 2, "row_count": 10}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 1019115068550221300, "num_eq": 2000000, "num_range": 5176531335393645314, "upper_bound": "\\u0000"}, {"distinct_range": 0, "num_eq": 8987605972969873370, "num_range": 1000000000, "upper_bound": "\\u0001\\u0018U"}, {"distinct_range": 7119920961932846000, "num_eq": 4000000000, "num_range": 7119920961932846409, "upper_bound": "\\u0016V|}"}, {"distinct_range": 4000000000, "num_eq": 7126023217252084812, "num_range": 4000000000, "upper_bound": "L\\u0007G\\u001d&"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 1, "row_count": 10}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 208443040, "histo_buckets": [{"distinct_range": 0, "num_eq": 629706364401573691, "num_range": 0, "upper_bound": "\\u0005"}, {"distinct_range": 0, "num_eq": 4000000, "num_range": 700, "upper_bound": ")"}, {"distinct_range": 42358.03270986398, "num_eq": 3458614019750735780, "num_range": 50000, "upper_bound": "2"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 90, "upper_bound": "H"}, {"distinct_range": 64985647400.88878, "num_eq": 100000000, "num_range": 90000000000, "upper_bound": "P"}, {"distinct_range": 8000000000, "num_eq": 60, "num_range": 8000000000, "upper_bound": "["}, {"distinct_range": 100000, "num_eq": 3452888459051637326, "num_range": 100000, "upper_bound": "n"}, {"distinct_range": 4000000000, "num_eq": 500000000, "num_range": 4000000000, "upper_bound": "\x7f"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 182109960, "row_count": 400000000}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 239842157, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000, "num_range": 0, "upper_bound": "-55 years -1 mons -237 days -09:13:36.086217"}, {"distinct_range": 2612990502254552000, "num_eq": 80000, "num_range": 2612990502254552137, "upper_bound": "-33 years -10 mons -956 days -21:15:59.765582"}, {"distinct_range": 56682705784.996895, "num_eq": 8000000000, "num_range": 60000000000, "upper_bound": "-33 years -733 days -14:08:08.88057"}, {"distinct_range": 0, "num_eq": 4954510799690555350, "num_range": 6360522371036268640, "upper_bound": "-24 years -3 mons -760 days -04:21:30.608816"}, {"distinct_range": 0, "num_eq": 6762624812281161651, "num_range": 3863481346208844237, "upper_bound": "-17 years -9 mons -912 days -07:51:25.817858"}, {"distinct_range": 4000000000, "num_eq": 1972284840299262375, "num_range": 4000000000, "upper_bound": "-18 years -4 mons -240 days -13:28:48.401923"}, {"distinct_range": 600000, "num_eq": 900000000, "num_range": 600000, "upper_bound": "21 years 7 mons 158 days 09:08:35.2841"}, {"distinct_range": 800000000, "num_eq": 6979726095671386606, "num_range": 800000000, "upper_bound": "24 years 366 days 02:09:50.58997"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6476312, "histo_col_type": "", "name": "__auto__", "null_count": 74631660, "row_count": 400000000}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25799686, "histo_col_type": "", "name": "__auto__", "null_count": 219695471, "row_count": 400000000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 143856097, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 111944569, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 178003642, "histo_buckets": [{"distinct_range": 0, "num_eq": 9185423194300531933, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 1166448961266964700, "num_eq": 5000, "num_range": 2489072744312483543, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 143422462, "row_count": 400000000}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 159773587, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "-12043"}, {"distinct_range": 151.9456265216097, "num_eq": 553606787936338812, "num_range": 200, "upper_bound": "-4860"}, {"distinct_range": 0, "num_eq": 8208691937587947806, "num_range": 4330401870009751869, "upper_bound": "-128"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 295996316, "row_count": 400000000}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 114521792, "histo_col_type": "", "name": "__auto__", "null_count": 2056930, "row_count": 400000000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 243028170, "histo_col_type": "", "name": "__auto__", "null_count": 190683232, "row_count": 400000000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 161623890, "histo_col_type": "", "name": "__auto__", "null_count": 75429983, "row_count": 400000000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 364728527, "histo_col_type": "", "name": "__auto__", "null_count": 139527515, "row_count": 400000000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 147495436, "histo_col_type": "", "name": "__auto__", "null_count": 286169810, "row_count": 400000000}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 291707493, "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 394209848, "histo_col_type": "", "name": "__auto__", "null_count": 244294386, "row_count": 400000000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 368852023, "histo_col_type": "", "name": "__auto__", "null_count": 34052942, "row_count": 400000000}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 230395139, "histo_buckets": [{"distinct_range": 0, "num_eq": 6458783496022736283, "num_range": 0, "upper_bound": "NaN"}, {"distinct_range": 0, "num_eq": 500000, "num_range": 0, "upper_bound": "0.07233698666095734"}, {"distinct_range": 4458.415908669609, "num_eq": 4607860815862991708, "num_range": 10000, "upper_bound": "0.34142014384269714"}, {"distinct_range": 400, "num_eq": 100000, "num_range": 400, "upper_bound": "0.3738296926021576"}, {"distinct_range": 3643479831054584300, "num_eq": 7000, "num_range": 3643479831054584235, "upper_bound": "0.9345732927322388"}, {"distinct_range": 3853764110584848000, "num_eq": 5733599330464953876, "num_range": 7005559652329021974, "upper_bound": "1.037770390510559"}, {"distinct_range": 2855566583373389300, "num_eq": 3118394102199266864, "num_range": 2855566583373389441, "upper_bound": "3.4028234663852886e+38"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 368253285, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000, "num_range": 0, "upper_bound": "03:08:14.484942+11:55:00"}, {"distinct_range": 7155410431998847000, "num_eq": 400000, "num_range": 7155410431998847077, "upper_bound": "18:06:50.156141+13:42:00"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 4328416166875243182, "upper_bound": "17:34:00.118171+10:49:00"}, {"distinct_range": 4439669400954799600, "num_eq": 800000, "num_range": 8615792269009610251, "upper_bound": "04:39:47.626736-03:57:00"}, {"distinct_range": 10, "num_eq": 60000000, "num_range": 10, "upper_bound": "08:54:06.888499-00:21:00"}, {"distinct_range": 5000000000, "num_eq": 6000000000, "num_range": 5000000000, "upper_bound": "23:15:25.679845-12:46:00"}, {"distinct_range": 10000000000, "num_eq": 600000, "num_range": 10000000000, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 86386738, "row_count": 400000000}]';
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_7) REFERENCES table2 (col1_14) ON UPDATE RESTRICT;
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3497697547035257073, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3992650356484698946}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2504702973912306994, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3992650356484698946}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 468238904085668971, "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 3992650356484698946}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 838985873250464256, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3992650356484698946}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3435334019242142299, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3992650356484698946}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3130728735778945515, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3992650356484698946}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3150608999392593521, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3992650356484698946}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3881685743994288081, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3992650356484698946}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3622720996187946463, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3992650356484698946}, {"columns": ["col1_6"], "created_at": "2000

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2591878&tab=buildLog) on [master@a51f2e212a71ee356a01f14510238baac65e76e7](https://github.com/cockroachdb/cockroach/commits/a51f2e212a71ee356a01f14510238baac65e76e7):

```
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_14, col2_13 DESC, col2_15, col2_18, col2_20, col2_11, col2_10, col2_19 DESC, col2_16 DESC, col2_8, col1_5 ASC, col1_0 DESC, col1_6 DESC, col2_9 ASC, col1_4 ASC, col2_12, col2_17, col2_6);
        test_1        |             CREATE TABLE table3 (col1_0 BOOL NOT NULL, col1_4 FLOAT8 NOT NULL, col1_6 BIT(48) NOT NULL, col1_5 BOX2D NOT NULL, col1_2 INT8 NOT NULL, col1_1 TIMESTAMPTZ NOT NULL, col3_6 INT8, col3_7 REGNAMESPACE NOT NULL, col3_8 REGTYPE, col3_9 BIT(16) NULL, col3_10 DATE NOT NULL, col3_11 BOOL NULL, col3_12 TIMESTAMPTZ NULL, col3_13 BOX2D NULL, col3_14 TEXT, col3_15 REGCLASS NOT NULL, col3_16 INTERVAL NULL, col3_17 OID NOT NULL, col3_18 REGNAMESPACE NULL, col3_19 INT8 NULL, PRIMARY KEY (col1_0, col1_4, col1_6, col1_5, col1_2, col1_1, col3_8, col3_18, col3_10, col3_14));
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_12, col3_7, col1_6 DESC, col3_19 ASC, col3_18 ASC);
        test_1        |             CREATE INDEX ON table3 (col3_6 DESC, col3_7 DESC, col3_14 DESC, col3_15 ASC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('pzfmby', 'u');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('wwa');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('jnq', 'p', 'eqb', 'ydrlfp', 'ixd', 'ifvg');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('quhz', 'iwhp');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('kt', 'r', 'rnybi', 'vt', 'ufx');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (0.90s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 FLOAT8 NOT NULL, col1_1 BIT(6) NOT NULL, col1_2 REGPROC, col1_3 NAME, col1_4 TIMETZ NOT NULL, col1_5 BOOL NOT NULL, col1_6 FLOAT8 NOT NULL, col1_7 FLOAT4 NOT NULL, col1_8 DATE NULL, PRIMARY KEY (col1_0 DESC, col1_7 DESC, col1_5, col1_4 ASC), INDEX (col1_8, col1_3 DESC, col1_7) STORING (col1_1) WHERE (((((NOT table1.col1_5) OR (table1.col1_8 != '1970-01-01':::DATE)) AND (table1.col1_3 > e'\x00':::STRING)) AND (table1.col1_6 != 1.0:::FLOAT8)) AND (table1.col1_7 != '+Inf':::FLOAT8)) OR (table1.col1_0 <= '+Inf':::FLOAT8), FAMILY (col1_4, col1_5, col1_3, col1_2), FAMILY (col1_6, col1_0), FAMILY (col1_8, col1_1, col1_7));
        test_1        |             CREATE TABLE table2 (col2_0 REGTYPE NOT NULL, col2_1 BOX2D NOT NULL, PRIMARY KEY (col2_1, col2_0 ASC), INDEX (col2_0 ASC, col2_1 ASC), UNIQUE (col2_1 ASC, col2_0), FAMILY (col2_0), FAMILY (col2_1));
        test_1        |             CREATE TABLE table3 (col2_1 BOX2D NOT NULL, col2_0 REGTYPE NOT NULL, col3_2 NAME NULL, col3_3 REGPROC NULL, PRIMARY KEY (col2_1, col2_0 ASC), FAMILY (col2_0, col3_3), FAMILY (col2_1), FAMILY (col3_2)) INTERLEAVE IN PARENT table2 (col2_1, col2_0);
        test_1        |             CREATE TABLE table4 (col2_1 BOX2D NOT NULL, col2_0 REGTYPE NOT NULL, col4_2 "char" NOT NULL, col4_3 UUID NOT NULL, col4_4 UUID, col4_5 UUID NULL, col4_6 NAME NOT NULL, col4_7 NAME, col4_8 REGPROC NULL, col4_9 UUID NULL, col4_10 FLOAT8, col4_11 INT8 NOT NULL, col4_12 BIT(49) NULL, col4_13 TIMESTAMP NULL, col4_14 FLOAT4 NULL, col4_15 INT2, col4_16 DATE NULL, col4_17 BIT NOT NULL, col4_18 INET NOT NULL, PRIMARY KEY (col2_1, col2_0 ASC, col4_17 ASC, col4_9 ASC, col4_4 ASC, col4_3 DESC, col4_8, col4_14, col4_18), INDEX (col4_5, col4_18 DESC, col4_8 ASC, col4_6 DESC, col2_0, col2_1 ASC, col4_16 ASC, col4_10, col4_17 DESC, col4_11 ASC, col4_2) STORING (col4_7, col4_12, col4_13) WHERE ((((((((table4.col4_10 < 'NaN':::FLOAT8) OR (table4.col4_6 >= e'\U00002603':::STRING)) AND (table4.col4_16 > '5874897-12-31':::DATE)) AND (table4.col4_15 < (-1):::INT8)) AND (table4.col4_7 = '"':::STRING)) AND (table4.col4_14 = 0.0:::FLOAT8)) AND (table4.col4_13 >= '0001-01-01 00:00:00':::TIMESTAMP)) AND (table4.col4_11 = (-32768):::INT8)) OR (table4.col4_2 < 'X':::STRING), UNIQUE (col4_10 ASC, col4_17 DESC, col4_5, col4_14, col4_8, col4_3, col4_9, col4_18 DESC, col4_4, col4_13 DESC, col4_2 ASC, col4_15) STORING (col4_7, col4_12) WHERE ((((((((table4.col4_11 = (-2147483648):::INT8) AND (table4.col4_14 >= 0.0:::FLOAT8)) OR (table4.col4_7 != 'X':::STRING)) OR (table4.col4_13 != '0001-01-01 00:00:00':::TIMESTAMP)) OR (table4.col4_16 >= 'infinity':::DATE)) OR (table4.col4_6 < '':::STRING)) AND (table4.col4_10 > 0.0:::FLOAT8)) OR (table4.col4_2 != e'\x00':::STRING)) OR (table4.col4_15 <= 0:::INT8), UNIQUE (col4_2 ASC, col4_18 DESC, col4_7 ASC, col4_14, col4_4 ASC) STORING (col4_5, col4_11, col4_13), INDEX (col4_4 ASC, col4_14, col4_18 DESC, col4_13 ASC, col4_7 ASC, col4_15 DESC, col4_6 DESC, col4_3 DESC, col4_8 ASC, col4_11 ASC, col2_1 DESC, col4_9 ASC, col2_0 DESC, col4_17, col4_10 DESC, col4_2 ASC, col4_5 DESC, col4_16 ASC) STORING (col4_12) WHERE ((((((((table4.col4_14 != -Inf) OR (table4.col4_13 >= '3000-01-01 00:00:00')) AND (table4.col4_16 > 'infinity')) OR (table4.col4_11 >= -128)) AND (table4.col4_6 >= e'\'')) AND (table4.col4_2 > '')) AND (table4.col4_10 > 1.7976931348623157e+308)) OR (table4.col4_7 >= e'\U00002603')) AND (table4.col4_15 < 127), UNIQUE (col4_10, col4_7, col4_15 DESC, col4_11 DESC) STORING (col4_2, col4_12, col4_13, col4_16) WHERE ((table4.col4_7 <= e'\U00002603':::STRING) OR (table4.col4_13 < '294276-12-31 23:59:59.999999':::TIMESTAMP)) AND (table4.col4_16 < '1970-01-01':::DATE), UNIQUE (col4_12, col2_0 DESC, col4_9 DESC, col4_11 ASC, col4_7 ASC, col4_4 DESC, col4_14, col4_3), INDEX (col4_14 DESC, col2_1, col4_13 ASC, col4_16 ASC, col4_4 ASC, col4_5 ASC, col4_7 ASC, col4_9 DESC, col2_0, col4_10 ASC, col4_18 ASC, col4_12, col4_3 DESC, col4_2, col4_6 ASC, col4_11 DESC, col4_17, col4_8 ASC, col4_15 ASC), UNIQUE (col4_18 ASC, col4_11 ASC, col4_10, col4_3 ASC, col4_13 ASC, col2_0 DESC, col2_1, col4_7, col4_14 DESC, col4_6 DESC, col4_12, col4_4, col4_9 ASC) STORING (col4_2, col4_15, col4_16) WHERE (((((table4.col4_6 > e'\U00002603':::STRING) AND (table4.col4_7 <= '"':::STRING)) AND (table4.col4_13 >= '294276-12-31 23:59:59.999999':::TIMESTAMP)) OR (table4.col4_15 > 0:::INT8)) OR (table4.col4_2 > e'\'':::STRING)) AND (table4.col4_11 <= (-32768):::INT8), UNIQUE (col4_12) STORING (col4_5, col4_6, col4_7, col4_13, col4_15) WHERE (((((((table4.col4_2 <= e'\'') AND (table4.col4_13 != '3000-01-01 00:00:00')) AND (table4.col4_14 < 0.0)) AND (table4.col4_7 <= '')) AND (table4.col4_11 > 2147483647)) OR (table4.col4_15 = 127)) AND (table4.col4_16 = 'infinity')) OR (table4.col4_6 < e'\''), FAMILY (col4_12), FAMILY (col4_14, col4_16, col4_5), FAMILY (col4_18, col4_8), FAMILY (col4_3, col4_2), FAMILY (col4_9, col2_1), FAMILY (col2_0, col4_17), FAMILY (col4_4, col4_11), FAMILY (col4_15), FAMILY (col4_6), FAMILY (col4_10, col4_13, col4_7)) INTERLEAVE IN PARENT table2 (col2_1, col2_0);
        test_1        |             CREATE TABLE table5 (col2_1 BOX2D NOT NULL, col2_0 REGTYPE NOT NULL, col5_2 FLOAT4 NOT NULL, col5_3 BYTES, PRIMARY KEY (col2_1, col2_0 ASC, col5_2 ASC), INDEX (col5_2 DESC, col2_0), INDEX (col2_0 DESC, col2_1 DESC, col5_2) STORING (col5_3) WHERE (table5.col5_3 >= e'\\xe29883':::BYTES) AND (table5.col5_2 != 'NaN':::FLOAT8), UNIQUE (col2_0 DESC) WHERE (table5.col5_2 = 0.0) AND (table5.col5_3 != '\x00'), INDEX (col5_2 ASC, col2_1, col5_3 ASC, col2_0) WHERE (table5.col5_2 != -1.0) AND (table5.col5_3 <= '\x58'), INDEX (col2_0 DESC, col2_1 DESC, col5_2) WHERE table5.col5_3 <= e'\\xe29883':::BYTES, UNIQUE (col5_2, col2_0 ASC, col2_1, col5_3 ASC), UNIQUE (col2_1 ASC, col5_3 ASC, col5_2 ASC, col2_0 DESC) WHERE (table5.col5_3 != e'\\xe29883':::BYTES) AND (table5.col5_2 != 1.401298464324817e-45:::FLOAT8), INDEX (col2_1, col5_2 DESC) STORING (col5_3) WHERE (table5.col5_3 < '\x27') AND (table5.col5_2 < 0.0), FAMILY (col2_0, col5_3), FAMILY (col2_1), FAMILY (col5_2)) INTERLEAVE IN PARENT table3 (col2_1, col2_0);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4175, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1501, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2718, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3830, "histo_col_type": "", "name": "__auto__", "null_count": 2412, "row_count": 5000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2756, "histo_col_type": "", "name": "__auto__", "null_count": 2428, "row_count": 5000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4299, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4657, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4460, "histo_buckets": [{"distinct_range": 0, "num_eq": 8821976589689568661, "num_range": 0, "upper_bound": "4714-11-24 BC"}, {"distinct_range": 3407332018834007000, "num_eq": 100000, "num_range": 3407332018834007030, "upper_bound": "1972-10-16"}, {"distinct_range": 0, "num_eq": 7157830291699776676, "num_range": 0, "upper_bound": "1987-09-01"}, {"distinct_range": 0, "num_eq": 40, "num_range": 60000, "upper_bound": "1990-11-26"}, {"distinct_range": 90, "num_eq": 5987203557305460606, "num_range": 90, "upper_bound": "5874897-12-31"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 4297, "row_count": 5000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1509, "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 5000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 133, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 68.15369509205163, "num_eq": 200, "num_range": 5000, "upper_bound": "108773864"}, {"distinct_range": 7317906304036677000, "num_eq": 3000000, "num_range": 7317906304036676721, "upper_bound": "256383065"}, {"distinct_range": 3833787822167838700, "num_eq": 2456994094187856417, "num_range": 3833787822167838466, "upper_bound": "1355778960"}, {"distinct_range": 4453415710942587400, "num_eq": 0, "num_range": 4453415710942587171, "upper_bound": "1448769711"}, {"distinct_range": 0, "num_eq": 40, "num_range": 30, "upper_bound": "1473741524"}, {"distinct_range": 20, "num_eq": 0, "num_range": 20, "upper_bound": "1802569211"}, {"distinct_range": 2172906268856067600, "num_eq": 7042717355338530302, "num_range": 3028786773545748247, "upper_bound": "2180118459"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 300}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 270, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "BOX(-1.0008566580037686 -1.3754334337963272,1.0337192078825779 0.4697950983885777)"}, {"distinct_range": 0, "num_eq": 261137660642319439, "num_range": 60, "upper_bound": "BOX(-0.41020265799955935 0.3856541877972174,0.34463558912712067 1.086591168690044)"}, {"distinct_range": 0, "num_eq": 5488527300528113013, "num_range": 8799126008626844145, "upper_bound": "BOX(-0.04156736892492774 0.036808685026088916,0.5396416617750682 0.36500813946741584)"}, {"distinct_range": 19580455131.52203, "num_eq": 9111480981807544974, "num_range": 20000000000, "upper_bound": "BOX(0.16331098433225222 -1.4479726344575727,1.6109838572900337 -0.801019210816527)"}, {"distinct_range": 2802073006657854000, "num_eq": 8045136854640261079, "num_range": 3532728307247609949, "upper_bound": "BOX(0.42999570021133415 0.1044004444242066,0.8221419036502821 0.4479580182806826)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 300}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1645725899163169434, "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 6221132675810272472}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5798554175697363764, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6221132675810272472}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4490661737348972055, "histo_col_type": "", "name": "__auto__", "null_count": 5281047253333435293, "row_count": 6221132675810272472}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 633516673161728863, "histo_col_type": "", "name": "__auto__", "null_count": 4537137590367349000, "row_count": 6221132675810272472}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "22bb4bf2-13f1-4c93-9be5-ab89712dbeef"}, {"distinct_range": 552555943.7068787, "num_eq": 40, "num_range": 600000000, "upper_bound": "4d35119e-e741-4030-b090-5724bf458b63"}, {"distinct_range": 0, "num_eq": 2000000, "num_range": 8749988251892537754, "upper_bound": "6a63118f-a7f4-41d2-9f67-cc524b788801"}, {"distinct_range": 4206017609821407000, "num_eq": 736279325500049879, "num_range": 4206017609821407029, "upper_bound": "aa6533b9-7435-407c-8a9b-1e60f748a42f"}, {"distinct_range": 6303816268.063104, "num_eq": 50000000000, "num_range": 80000000000, "upper_bound": "b6e089d8-e73c-4850-8057-06463e607def"}, {"distinct_range": 0, "num_eq": 984414634779791191, "num_range": 30000000000, "upper_bound": "c203e0b7-ed2c-4750-9dd9-bdc7777e0c97"}, {"distinct_range": 0, "num_eq": 6301371713145393912, "num_range": 0, "upper_bound": "da356076-bc9b-4ecd-a695-d87c90ce9ae7"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 6080626733076919159, "num_range": 0, "upper_bound": "12.90.168.215/4"}, {"distinct_range": 4666394539.9120035, "num_eq": 40000000000, "num_range": 9000000000, "upper_bound": "36.19.233.245/5"}, {"distinct_range": 169597813338257900, "num_eq": 3078701301019045501, "num_range": 8820903868583149527, "upper_bound": "158.133.168.131/8"}, {"distinct_range": 2455816.998867056, "num_eq": 5475785864355124394, "num_range": 4000000, "upper_bound": "101.175.88.226/19"}, {"distinct_range": 7.516390549033355, "num_eq": 7379455487227155410, "num_range": 20, "upper_bound": "169.122.38.144/20"}, {"distinct_range": 4620835500615733000, "num_eq": 500000, "num_range": 4620835500615733649, "upper_bound": "31.50.152.233/23"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 2569952451346245126, "num_range": 0, "upper_bound": "0000101010100100111011100111001011011100010000101"}, {"distinct_range": 1419.0941285775555, "num_eq": 8467038187788089051, "num_range": 3000, "upper_bound": "0110010100001010110100111100110000101111111010001"}, {"distinct_range": 0, "num_eq": 2078342831263209552, "num_range": 4145468361622567980, "upper_bound": "0110100111000011000110001110100010000001011111000"}, {"distinct_range": 5000000, "num_eq": 7903346014250120604, "num_range": 5000000, "upper_bound": "0110110101101101111110110011110101011100000010000"}, {"distinct_range": 6163790189.819227, "num_eq": 10000000, "num_range": 7000000000, "upper_bound": "0111111111111111111111111111111111111111111111111"}, {"distinct_range": 0, "num_eq": 120476210994566121, "num_range": 7000000, "upper_bound": "1100100110000111101010000111001100001010010110111"}], "histo_col_type": "BIT(49)", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000000, "num_range": 0, "upper_bound": "NaN"}, {"distinct_range": 0, "num_eq": 648050386324459534, "num_range": 8000, "upper_bound": "1.8051471710205078"}, {"distinct_range": 1000000000, "num_eq": 20000, "num_range": 1000000000, "upper_bound": "1.9368258714675903"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 1000000, "num_eq": 80000000, "num_range": 1000000, "upper_bound": "\\u0007"}, {"distinct_range": 0, "num_eq": 9000000, "num_range": 9000000000, "upper_bound": "6"}, {"distinct_range": 0, "num_eq": 8831311286499250847, "num_range": 5130895053758967875, "upper_bound": "B"}, {"distinct_range": 0, "num_eq": 1703616793172438568, "num_range": 100, "upper_bound": "N"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 3387183572979844595, "num_range": 0, "upper_bound": "1f318de8-2ea2-492d-9426-64dda0c1824c"}, {"distinct_range": 100, "num_eq": 4916869225752101954, "num_range": 100, "upper_bound": "20c6e73e-6dae-49ae-b156-5dad901f1e05"}, {"distinct_range": 327.0612875798721, "num_eq": 40, "num_range": 800, "upper_bound": "2116b7c8-6429-42da-ab5e-0113da2a4b0c"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000000, "num_range": 0, "upper_bound": "BOX(-0.9734670646782002 -2.3076368300646095,-0.06208752232064896 0.08838101653698055)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000, "num_range": 0, "upper_bound": "-1.1280698299836658"}, {"distinct_range": 536198931465465150, "num_eq": 7713295718537724702, "num_range": 930250612616917402, "upper_bound": "-1.023351289566171"}, {"distinct_range": 40000, "num_eq": 10000, "num_range": 40000, "upper_bound": "-0.8428273958941535"}, {"distinct_range": 4000000000, "num_eq": 8000, "num_range": 4000000000, "upper_bound": "-0.7727745037127112"}, {"distinct_range": 191018.37578931777, "num_eq": 0, "num_range": 500000, "upper_bound": "-0.05244823334068105"}, {"distinct_range": 8872776.530752227, "num_eq": 242426554315233212, "num_range": 9000000, "upper_bound": "0.060688644521669555"}, {"distinct_range": 1511261017008234800, "num_eq": 8693103029628704260, "num_range": 1511261017008234860, "upper_bound": "0.45595570453389456"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4427241, "histo_buckets": [{"distinct_range": 0, "num_eq": 6996404230895338627, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 10.77845264697006, "num_eq": 7555739875405273384, "num_range": 20, "upper_bound": "BOX(-2.604273462344936 -0.1775115316486392,-0.4335642693131836 0.42630886962563486)"}, {"distinct_range": 1000000, "num_eq": 90000000000, "num_range": 1000000, "upper_bound": "BOX(-1.8799467415338142 -1.3496228746313976,-0.31822469998540304 -0.15374795845761657)"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 6897958809441266146, "upper_bound": "BOX(0.22451100047021527 -0.5908304993305056,0.46292424095964424 -0.01629424370745175)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 271453, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "43357417"}, {"distinct_range": 400000000, "num_eq": 6369254769423115253, "num_range": 400000000, "upper_bound": "1615250261"}, {"distinct_range": 7279750705540210000, "num_eq": 40, "num_range": 7279750705540209896, "upper_bound": "1836254938"}, {"distinct_range": 1701868435966125000, "num_eq": 5000000000, "num_range": 1701868435966124947, "upper_bound": "2287632511"}, {"distinct_range": 12000880.778449705, "num_eq": 100000000000, "num_range": 60000000, "upper_bound": "2405784154"}, {"distinct_range": 0, "num_eq": 5341267809711283067, "num_range": 8000000, "upper_bound": "2819032093"}, {"distinct_range": 8039373572307110000, "num_eq": 8999114099940695312, "num_range": 8039373572307110306, "upper_bound": "3309532544"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3619804, "histo_buckets": [{"distinct_range": 0, "num_eq": 1167279909721084709, "num_range": 0, "upper_bound": "-0.4927206337451935"}, {"distinct_range": 2844.454044152961, "num_eq": 700, "num_range": 3000, "upper_bound": "-0.4901171624660492"}, {"distinct_range": 0, "num_eq": 1163412537165091275, "num_range": 70000000, "upper_bound": "-0.46848931908607483"}, {"distinct_range": 3841079394131800600, "num_eq": 2000000, "num_range": 8473948723869140717, "upper_bound": "-0.20924879610538483"}, {"distinct_range": 0, "num_eq": 0, "num_range": 5081530395217220920, "upper_bound": "-0.018112333491444588"}, {"distinct_range": 2206928380.585315, "num_eq": 30, "num_range": 30000000000, "upper_bound": "0.6284602284431458"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 8593154970806755956, "upper_bound": "1.0"}, {"distinct_range": 80, "num_eq": 5000000, "num_range": 80, "upper_bound": "1.2306197881698608"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6757724, "histo_col_type": "", "name": "__auto__", "null_count": 5567487, "row_count": 7000000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('r', 'pdqcja');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('coar', 'qn', 'lxdlt');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 407231277357841176, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2639323189364161076}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 360224621265675001, "histo_col_type": "", "name": "__auto__", "null_count": 2103132648411947629, "row_count": 2639323189364161076}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1617852378336933946, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2639323189364161076}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1931696373348642053, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2639323189364161076}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1944280330041262959, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2639323189364161076}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2563207267127201467, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2639323189364161076}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2398330149928526987, "histo_buckets": [{"distinct_range": 0, "num_eq": 2445908345321369208, "num_range": 0, "upper_bound": "-1.3930869858953603"}, {"distinct_range": 3604215141680022500, "num_eq": 2508515324969128990, "num_range": 3604215141680022643, "upper_bound": "-0.7311270760030488"}, {"distinct_range": 5702499279808614000, "num_eq": 1000, "num_range": 5702499279808614667, "upper_bound": "-0.4775004372342555"}, {"distinct_range": 20000000000, "num_eq": 0, "num_range": 20000000000, "upper_bound": "-0.16550849510237686"}, {"distinct_range": 1789952633198720300, "num_eq": 3872487982035571056, "num_range": 4039359570476941811, "upper_bound": "0.0"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 0, "upper_bound": "0.5185471687944125"}, {"distinct_range": 0, "num_eq": 7963594961941083173, "num_range": 3597173015875833027, "upper_bound": "0.7226020720466939"}, {"distinct_range": 0, "num_eq": 700, "num_range": 4936672491270768201, "upper_bound": "1.1801418210450787"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 2639323189364161076}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1056444305127631224, "histo_col_type": "", "name": "__auto__", "null_count": 1472295098460839967, "row_count": 2639323189364161076}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46888882451835293, "histo_col_type": "DATE", "name": "__auto__", "null_count": 2221036911797969245, "row_count": 2639323189364161076}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 124882522410966251, "histo_buckets": [{"distinct_range": 0, "num_eq": 700, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 7737690150660950000, "num_eq": 4000000000, "num_range": 7737690150660949717, "upper_bound": "BOX(-1.7281023744313437 0.6586349701016379,0.03209386622209487 2.4050964546945615)"}, {"distinct_range": 3728650345990616000, "num_eq": 8262908633716668637, "num_range": 3728650345990616102, "upper_bound": "BOX(-1.582450774504525 0.07077589078284774,-0.05488330435448016 0.8974945514448024)"}, {"distinct_range": 90000, "num_eq": 1405523588899524023, "num_range": 90000, "upper_bound": "BOX(-1.1715950011589944 -0.11083079637840554,0.662235227886167 0.6460694246951775)"}, {"distinct_range": 806511388613750500, "num_eq": 1000000, "num_range": 806511388613750514, "upper_bound": "BOX(-1.1414971283371023 -0.7345062608021034,-0.8648787085949547 -0.14686398245711607)"}, {"distinct_range": 0, "num_eq": 5662973463242405604, "num_range": 1000000, "upper_bound": "BOX(-0.6980184412804594 0.6333807932850225,1.1416794434330186 1.2291685284829905)"}, {"distinct_range": 2709185173.5022626, "num_eq": 8000, "num_range": 20000000000, "upper_bound": "BOX(-0.6831970598303715 -0.19941682492590235,0.527404935747095 1.528794010253828)"}, {"distinct_range": 1000, "num_eq": 60000, "num_range": 1000, "upper_bound": "BOX(-0.14458007669611614 -0.5814100873243863,1.182908828616717 0.016534034858808888)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 3057875987972949097}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1241952740558241428, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000, "num_range": 0, "upper_bound": "1267678649"}, {"distinct_range": 433468042799584830, "num_eq": 400, "num_range": 1341938441845589263, "upper_bound": "1662695359"}, {"distinct_range": 0, "num_eq": 8369907246236474677, "num_range": 5427579088110724440, "upper_bound": "1881701385"}, {"distinct_range": 385848.641524001, "num_eq": 5000000000, "num_range": 6000000, "upper_bound": "1953121492"}, {"distinct_range": 5845244705929194000, "num_eq": 800000000, "num_range": 5902493807744820598, "upper_bound": "2211228911"}, {"distinct_range": 60000000000, "num_eq": 3244809097521558747, "num_range": 60000000000, "upper_bound": "2406733421"}, {"distinct_range": 1915.9258988142465, "num_eq": 7581540493555939180, "num_range": 3000, "upper_bound": "3151867945"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 3057875987972949097}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1363415505596762561, "histo_buckets": [{"distinct_range": 0, "num_eq": 3723898301275924724, "num_range": 0, "upper_bound": "BOX(-0.9693946169560512 -1.9940202053632863,0.1286161697083914 -1.5983307606397046)"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 30000000, "upper_bound": "BOX(-0.8316218237594118 0.47902456891035616,-0.30122016123791395 1.1783275576515546)"}, {"distinct_range": 6000000, "num_eq": 0, "num_range": 6000000, "upper_bound": "BOX(-0.7049242106050313 1.5378461897234594,-0.660157027002612 1.587824275254159)"}, {"distinct_range": 2145291974910881000, "num_eq": 70, "num_range": 2145291974910881139, "upper_bound": "BOX(-0.5223988929774508 -1.0824923544296876,-0.0297133456395271 -0.3758874134800253)"}, {"distinct_range": 4265611306680319500, "num_eq": 1645818838704266300, "num_range": 4265611306680319741, "upper_bound": "BOX(0.2828122151574017 0.3748167643658548,1.448393404645203 0.40083539677763097)"}, {"distinct_range": 400, "num_eq": 90000, "num_range": 400, "upper_bound": "BOX(1.363761862550693 1.29318343992814,2.1211037014507066 1.487102125407696)"}, {"distinct_range": 1952074818582368800, "num_eq": 6567611703356489281, "num_range": 1952074818582368764, "upper_bound": "BOX(1.66736113014686 0.21033882144876936,1.9891089542968206 0.5382117990240589)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 2614935963296664914}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2203466110179001371, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2614935963296664914}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 271624332442494015, "histo_col_type": "", "name": "__auto__", "null_count": 1315579976912113375, "row_count": 2614935963296664914}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1607532508978251220, "histo_col_type": "", "name": "__auto__", "null_count": 1360716518242415618, "row_count": 2614935963296664914}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 535456880499700915, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2039974029532820363}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1811613610276685877, "histo_col_type": "", "name": "__auto__", "null_count": 1626049754218777110, "row_count": 2039974029532820363}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 768538100262967855, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2039974029532820363}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 548206754268596734, "histo_buckets": [{"distinct_range": 0, "num_eq": 6239072449086716409, "num_range": 0, "upper_bound": "-2.1569268703460693"}, {"distinct_range": 2489024.2941448777, "num_eq": 9000000, "num_range": 3000000, "upper_bound": "-0.35410454869270325"}, {"distinct_range": 1000000, "num_eq": 4457132681460639469, "num_range": 1000000, "upper_bound": "-0.009522602893412113"}, {"distinct_range": 0, "num_eq": 6622557931683394728, "num_range": 7000, "upper_bound": "0.35860326886177063"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 900000, "upper_bound": "0.6903412342071533"}, {"distinct_range": 0, "num_eq": 70, "num_range": 1816647775167575987, "upper_bound": "1.1246575117111206"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 808691781529150207, "row_count": 2039974029532820363}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1179453220924841473, "histo_col_type": "", "name": "__auto__", "null_count": 128905616354962167, "row_count": 2039974029532820363}, {"columns": ["col4_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 96199038260205835, "histo_buckets": [{"distinct_range": 0, "num_eq": 2131736785973464852, "num_range": 0, "upper_bound": "158.126.163.61/5"}, {"distinct_range": 2517926887711014000, "num_eq": 4779513866443132528, "num_range": 3220002663888028832, "upper_bound": "124.78.225.137/19"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 40000000, "upper_bound": "93.60.191.59/21"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 197988326572875674, "upper_bound": "115.244.251.1/30"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 10000, "upper_bound": "e8f9:2b32:b715:728c:5ff0:eca9:3233:1957/59"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 2039974029532820363}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1095412556864065718, "histo_col_type": "", "name": "__auto__", "null_count": 159903883021010403, "row_count": 2039974029532820363}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 413500276141214977, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "-0.9278207812727923"}, {"distinct_range": 100000, "num_eq": 6174503933891952441, "num_range": 100000, "upper_bound": "-0.5860959172334199"}, {"distinct_range": 7856681946623927000, "num_eq": 400, "num_range": 7856681946623927559, "upper_bound": "1.6157860982865528"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 1931082847004662331, "row_count": 2039974029532820363}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 206087923839013143, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000000, "num_range": 0, "upper_bound": "1000111000001011100001011110000001001111011101111"}], "histo_col_type": "BIT(49)", "name": "__auto__", "null_count": 750063298730425285, "row_count": 2039974029532820363}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1457855396837490339, "histo_buckets": [{"distinct_range": 0, "num_eq": 8878871434039994020, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 154263.74041726536, "num_eq": 1355916861690120371, "num_range": 200000, "upper_bound": "BOX(-3.162267058703351 -2.23505706782393,-0.22345788322848126 -1.006066120617431)"}, {"distinct_range": 3231472209860029400, "num_eq": 10, "num_range": 3231472209860029362, "upper_bound": "BOX(-3.0490562917866804 0.8578977436275581,-1.4254284852499381 2.07938893952935)"}, {"distinct_range": 4404277507027957000, "num_eq": 0, "num_range": 7273075997576834248, "upper_bound": "BOX(-1.5866729877606711 -0.09631486012159213,0.8965999125396777 0.9467821812158022)"}, {"distinct_range": 6510430302826357000, "num_eq": 7168749968939522253, "num_range": 6972580348561335162, "upper_bound": "BOX(-1.3971997002971648 -0.26918071829173784,0.8748521596518184 0.29424801031243975)"}, {"distinct_range": 0, "num_eq": 523567956916659482, "num_range": 70, "upper_bound": "BOX(-0.5784211503805498 -0.7077292397744963,0.5532349106699069 -0.3593147864842767)"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 3000000, "upper_bound": "BOX(0.7255142355593681 -0.5554837192295876,0.8511300967932962 0.1314765193864758)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 2039974029532820363}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1948586190181080915, "histo_buckets": [{"distinct_range": 0, "num_eq": 4721177216114126809, "num_range": 0, "upper_bound": "\\u001d"}, {"distinct_range": 0, "num_eq": 2623106802600350818, "num_range": 8661517387163629192, "upper_bound": "-"}, {"distinct_range": 600000, "num_eq": 9000000000, "num_range": 600000, "upper_bound": "4"}, {"distinct_range": 90000000, "num_eq": 1219771449991222340, "num_range": 90000000, "upper_bound": "7"}, {"distinct_range": 22443036777349100, "num_eq": 487958709607907462, "num_range": 258184263752794816, "upper_bound": ";"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 2858536982246882058, "upper_bound": "P"}, {"distinct_range": 2554941431819697000, "num_eq": 1000000, "num_range": 4104474562704658766, "upper_bound": "t"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 2039974029532820363}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 444342644207416408, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2039974029532820363}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 120684133297651080, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000000, "num_range": 0, "upper_bound": "00000000-0000-0000-0000-000000000000"}, {"distinct_range": 0, "num_eq": 7023227735578820424, "num_range": 600000, "upper_bound": "6eb82de6-eb95-40de-b030-4e08e44f8ba6"}, {"distinct_range": 0, "num_eq": 4000000000, "num_range": 20000, "upper_bound": "741e6810-5405-4acd-92e3-e98cd763ed60"}, {"distinct_range": 3959063350034360000, "num_eq": 0, "num_range": 3959063350034359879, "upper_bound": "94857a93-f7a4-4d9a-891b-ae558095bd27"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 700, "upper_bound": "a33d65bd-5023-4729-8214-7100b260d7fb"}, {"distinct_range": 50969854.791209154, "num_eq": 5332137134982719190, "num_range": 70000000, "upper_bound": "fc3cee34-0fe4-4135-b871-2b6fc522c70b"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "ffffffff-ffff-ffff-ffff-ffffffffffff"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 600546505095245651, "row_count": 2039974029532820363}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1667204381232768937, "histo_col_type": "", "name": "__auto__", "null_count": 47397086324236074, "row_count": 2039974029532820363}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1719343525806013721, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2039974029532820363}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1711304637745547149, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000, "num_range": 0, "upper_bound": "00000000-0000-0000-0000-000000000000"}, {"distinct_range": 1320452141644714500, "num_eq": 500, "num_range": 1519658420262026923, "upper_bound": "0aaef4d0-c1b6-44ba-9f5e-dbfac294478b"}, {"distinct_range": 6000, "num_eq": 9119399579962267569, "num_range": 6000, "upper_bound": "2d47abe8-39c1-4435-9ca5-9c864a5a8d80"}, {"distinct_range": 6811727985560813000, "num_eq": 3437848636442549758, "num_range": 6811727985560812141, "upper_bound": "9cf4ac6a-81fe-4943-96b0-6c8275cc8af7"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 1499242922216854133, "row_count": 2039974029532820363}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1179785923921258856, "histo_col_type": "", "name": "__auto__", "null_count": 562161321281817228, "row_count": 2039974029532820363}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 146988883039491929, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2039974029532820363}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1644789229087682964, "histo_col_type": "", "name": "__auto__", "null_count": 1188613903826224363, "row_count": 2039974029532820363}]';
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 75204, "histo_buckets": [{"distinct_range": 0, "num_eq": 1558304648119176687, "num_range": 0, "upper_bound": "BOX(-1.3589677514204812 1.2615864904618976,0.519768446716075 1.6794144628308778)"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 100000, "upper_bound": "BOX(-1.1754944008134598 -0.2534147722065013,0.6642416387184691 -0.24252790854272047)"}, {"distinct_range": 7485008526335794000, "num_eq": 929089701715868237, "num_range": 7485008526335793683, "upper_bound": "BOX(-1.1055854886085568 -1.9271750174594047,1.0741005882933126 1.986264526119536)"}, {"distinct_range": 7434720944727165000, "num_eq": 10000000000, "num_range": 7434720944727164605, "upper_bound": "BOX(-1.0704368755987606 -1.1326583423692282,0.7253839226773573 -0.287523705219735)"}, {"distinct_range": 5000, "num_eq": 2802913875216526474, "num_range": 5000, "upper_bound": "BOX(-0.7083334321647032 0.2897314317765771,0.5977379468045572 0.9484250426386935)"}, {"distinct_range": 2000000000, "num_eq": 1434677780471355527, "num_range": 2000000000, "upper_bound": "BOX(-0.6025877862251374 -1.505364680674453,0.30584956872474944 -0.13444280334819925)"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 50000, "upper_bound": "BOX(-0.38817726051349144 -0.5968457628564056,0.3503633737163734 -0.47877099916733723)"}, {"distinct_range": 1176059988019740000, "num_eq": 2089596873002089151, "num_range": 6418238139256023364, "upper_bound": "BOX(0.18313440610766518 -0.4557604620526778,0.9984854351630228 0.591552774065246)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 49714, "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 79858, "histo_buckets": [{"distinct_range": 0, "num_eq": 1069132424981421059, "num_range": 0, "upper_bound": "-1.1854578256607056"}, {"distinct_range": 2952187908157694500, "num_eq": 7789125913728948701, "num_range": 8915067826653560121, "upper_bound": "-0.5888204574584961"}, {"distinct_range": 123706182.1117255, "num_eq": 200000000, "num_range": 500000000, "upper_bound": "-0.2262318730354309"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 400000, "upper_bound": "0.48372673988342285"}, {"distinct_range": 524446665852727300, "num_eq": 9000, "num_range": 3101203815320501760, "upper_bound": "1.0"}, {"distinct_range": 32668175488.921886, "num_eq": 600, "num_range": 90000000000, "upper_bound": "1.045576572418213"}, {"distinct_range": 0, "num_eq": 1536975236715459813, "num_range": 20000000000, "upper_bound": "1.9573956727981567"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 55077, "histo_col_type": "", "name": "__auto__", "null_count": 52531, "row_count": 80000}]';
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_1) REFERENCES table4 (col4_12) ON UPDATE CASCADE;
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 88, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42, "histo_col_type": "", "name": "__auto__", "null_count": 64, "row_count": 100}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45, "histo_col_type": "", "name": "__auto__", "null_count": 21, "row_count": 100}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 51, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 51, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "-0.6775101451899801"}, {"distinct_range": 5489000737487515000, "num_eq": 3053330491387922245, "num_range": 5489000737487514408, "upper_bound": "0.7449655309191243"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 62, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29, "histo_buckets": [{"distinct_range": 0, "num_eq": 581888794842272541, "num_range": 0, "upper_bound": "1970-01-01"}, {"distinct_range": 0, "num_eq": 500000, "num_range": 6000, "upper_bound": "1971-04-22"}, {"distinct_range": 0, "num_eq": 1384211227234086251, "num_range": 1000000000, "upper_bound": "1973-11-05"}, {"distinct_range": 0, "num_eq": 1029314545487245023, "num_range": 8683590769131320413, "upper_bound": "1977-11-24"}, {"distinct_range": 5000000, "num_eq": 9039684771403801967, "num_range": 5000000, "upper_bound": "1980-12-27"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 7000, "upper_bound": "1985-02-23"}, {"distinct_range": 2000, "num_eq": 5000000000, "num_range": 2000, "upper_bound": "1991-05-12"}, {"distinct_range": 0, "num_eq": 200, "num_range": 334685950200348396, "upper_bound": "1992-05-03"}, {"distinct_range": 8015487091914758000, "num_eq": 300, "num_range": 8015487091914758521, "upper_bound": "1994-12-05"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 96, "row_count": 100}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 63335, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 0, "upper_bound": "267709129"}, {"distinct_range": 6930.711322527098, "num_eq": 100000000, "num_range": 70000, "upper_bound": "434211408"}, {"distinct_range": 100000000, "num_eq": 80, "num_range": 100000000, "upper_bound": "641537244"}, {"distinct_range": 4694975805205048000, "num_eq": 4207208929198032593, "num_range": 4694975805205048795, "upper_bound": "2017498072"}, {"distinct_range": 4000000, "num_eq": 100000000, "num_range": 4000000, "upper_bound": "2376326038"}, {"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "3631706051"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 100000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 62722, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "BOX(-1.045317943056701 0.24596960787392608,-0.13368971874435664 0.759157878720577)"}, {"distinct_range": 1720134567016366000, "num_eq": 4000000000, "num_range": 1720134567016366048, "upper_bound": "BOX(-0.9638995448778261 0.6112056045352698,1.0685460610916504 1.40597695824849)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 100000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3551229, "histo_buckets": [{"distinct_range": 0, "num_eq": 8883871189934888714, "num_range": 0, "upper_bound": "BOX(-1.3249900479987347 0.03795585291354053,-0.1985677154675522 0.5679834045596271)"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "BOX(-1.148496021461498 0.4735368338523267,-0.37430258742405476 3.0438324934793286)"}, {"distinct_range": 3869409653.242207, "num_eq": 20000, "num_range": 8000000000, "upper_bound": "BOX(-1.1285971126545893 0.23143411723588592,2.2700003839702414 0.6216444304759372)"}, {"distinct_range": 0, "num_eq": 4000000000, "num_range": 1000000000, "upper_bound": "BOX(-0.2366896347131231 -0.0019577527424168295,-0.10213191842129038 0.0219680507012876)"}, {"distinct_range": 618790470570216300, "num_eq": 700000, "num_range": 618790470570216272, "upper_bound": "BOX(0.9429013735637691 -0.2705309038150132,1.0716077151404253 -0.20188687823165452)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8347228, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 152411, "histo_col_type": "", "name": "__auto__", "null_count": 798163, "row_count": 9000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 934883, "histo_col_type": "", "name": "__auto__", "null_count": 1947914, "row_count": 9000000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2690185734621891622, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2824211185813726531}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 847839847871282220, "histo_buckets": [{"distinct_range": 0, "num_eq": 900, "num_range": 0, "upper_bound": "f08330c3-1389-4982-8210-928b53b95b3a"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 486990231256027083, "row_count": 2824211185813726531}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2350993247344269815, "histo_col_type": "", "name": "__auto__", "null_count": 2584873041200048941, "row_count": 2824211185813726531}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2270161585308377615, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2824211185813726531}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2289578871540728455, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "0111110100111000101001111000101000001110001011010"}, {"distinct_range": 0, "num_eq": 50, "num_range": 500000000, "upper_bound": "1001100111101110101111011000110100110111001111110"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 70000000, "upper_bound": "1010111010111110100001000101011011110001011110111"}, {"distinct_range": 0, "num_eq": 6000000, "num_range": 10000, "upper_bound": "1100100010100010010010110000100001101111011001011"}], "histo_col_type": "BIT(49)", "name": "__auto__", "null_count": 2356306045035277507, "row_count": 2824211185813726531}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2448083869186231058, "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 1731609189962232831, "row_count": 2824211185813726531}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1666810165891865237, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2824211185813726531}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 77934226246022222, "histo_buckets": [{"distinct_range": 0, "num_eq": 28524783303881908, "num_range": 0, "upper_bound": "\\u0010"}, {"distinct_range": 0, "num_eq": 600000, "num_range": 7000000, "upper_bound": "!"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "N"}, {"distinct_range": 72655803267.27419, "num_eq": 20, "num_range": 80000000000, "upper_bound": "U"}, {"distinct_range": 3894316921898880500, "num_eq": 800000000, "num_range": 3894316921898880443, "upper_bound": "Y"}, {"distinct_range": 4596118304000703000, "num_eq": 2691830717953719264, "num_range": 4596118304000703025, "upper_bound": "]"}, {"distinct_range": 9000, "num_eq": 400000000, "num_range": 9000, "upper_bound": "t"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 2824211185813726531}, {"columns": ["col4_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1447386592325050812, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000000, "num_range": 0, "upper_bound": "149.79.71.21/1"}, {"distinct_range": 1000000, "num_eq": 1490711959183690368, "num_range": 1000000, "upper_bound": "98.229.253.237/20"}, {"distinct_range": 110757053525286600, "num_eq": 0, "num_range": 110757053525286590, "upper_bound": "ebae:eba1:c26c:1235:9a85:2d60:9c55:1c69/3"}, {"distinct_range": 0, "num_eq": 6686344159712984735, "num_range": 20000000000, "upper_bound": "e74:21ad:3c64:15b0:a6e:22b5:e4af:d203/112"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 2824211185813726531}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 558571145316588783, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2824211185813726531}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 427625989808436953, "histo_col_type": "", "name": "__auto__", "null_count": 1705018491434362513, "row_count": 2824211185813726531}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1878032202195157301, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000, "num_range": 0, "upper_bound": "0.28007408143402257"}, {"distinct_range": 5493980140829211000, "num_eq": 5000, "num_range": 5493980140829210421, "upper_bound": "0.3158716317029159"}, {"distinct_range": 0, "num_eq": 3346137396724303541, "num_range": 1000, "upper_bound": "0.5560043145894161"}, {"distinct_range": 98.90127422301546, "num_eq": 4864954260526974533, "num_range": 100, "upper_bound": "0.7689918261135276"}, {"distinct_range": 3071704523258953700, "num_eq": 6768965057091198889, "num_range": 7632806831565682264, "upper_bound": "1.1568556531167387"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 304621732446610226, "row_count": 2824211185813726531}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1174268064981842725, "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 2824211185813726531}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2244399312802949712, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2824211185813726531}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 712983900036426331, "histo_col_type": "", "name": "__auto__", "null_count": 98897752126814885, "row_count": 2824211185813726531}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 971833311831488472, "histo_col_type": "", "name": "__auto__", "null_count": 1379128878324883388, "row_count": 2824211185813726531}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2386933323045449432, "histo_col_type": "", "name": "__auto__", "null_count": 1002737638513010995, "row_count": 2824211185813726531}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2693298648507869389, "histo_col_type": "", "name": "__auto__", "null_count": 65145382841197646, "row_count": 2824211185813726531}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2072130876436299674, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000, "num_range": 0, "upper_bound": "452f8997-016d-4791-8d71-0dd68b712066"}, {"distinct_range": 20.351485705755124, "num_eq": 10000, "num_range": 40, "upper_bound": "8f8f289a-baf4-42d0-bcfd-df321777240c"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 2454023941185523282, "row_count": 2824211185813726531}]';
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1201335, "histo_buckets": [{"distinct_range": 0, "num_eq": 3910334589605962702, "num_range": 0, "upper_bound": "BOX(-0.4123926663233469 -0.7511076971234827,0.9404921000649246 0.43782877479705284)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4166139, "histo_buckets": [{"distinct_range": 0, "num_eq": 4546154079044551755, "num_range": 0, "upper_bound": "1303474032"}, {"distinct_range": 0, "num_eq": 600000, "num_range": 6309396139003812242, "upper_bound": "2834220709"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4891705, "histo_buckets": [{"distinct_range": 0, "num_eq": 2623389180391083334, "num_range": 0, "upper_bound": "-0.9137680530548096"}, {"distinct_range": 80000, "num_eq": 900000000, "num_range": 80000, "upper_bound": "0.26181653141975403"}, {"distinct_range": 203271811.92839974, "num_eq": 214981236468240812, "num_range": 300000000, "upper_bound": "0.7940626144409

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2593846&tab=buildLog) on [master@19addd018c462875515f94c5d77a3848f2995443](https://github.com/cockroachdb/cockroach/commits/19addd018c462875515f94c5d77a3848f2995443):

```
        test_1        |         compare_test.go:155: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("F6"),
        test_1        |             + 	string("F6.0"),
        test_1        |               }))
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('F'::TEXT::TEXT || 6.0::FLOAT8::FLOAT8)::TEXT AS col_76794) AS tab_40733
        test_1        |             ORDER BY
        test_1        |             	col_76794 NULLS FIRST;
        test_1        |             
        test_1        |             cockroach1: [same as previous]
        test_1        |             
        test_1        |     --- FAIL: TestCompare/mutators (0.93s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 REGPROCEDURE NOT NULL, col1_1 TIMESTAMPTZ NOT NULL, col1_2 TIMETZ NOT NULL, col1_3 CHAR NOT NULL, col1_4 INT8 NOT NULL, col1_5 INTERVAL NOT NULL, col1_6 REGPROC NOT NULL, col1_7 INT4 NOT NULL, col1_8 FLOAT8 NOT NULL, col1_9 INET NOT NULL, col1_10 TIME NOT NULL, col1_11 DECIMAL NOT NULL, col1_12 REGTYPE NOT NULL, col1_13 FLOAT4 NOT NULL, col1_14 TIMESTAMPTZ NOT NULL, col1_15 BOX2D NOT NULL, col1_16 TIMETZ NOT NULL, col1_17 FLOAT8 NOT NULL, PRIMARY KEY (col1_7 ASC, col1_6 ASC, col1_10 ASC, col1_1, col1_4 ASC, col1_9, col1_3 ASC, col1_14, col1_0 DESC, col1_13 ASC, col1_11 DESC, col1_8, col1_5 DESC, col1_17, col1_15, col1_12 DESC, col1_2, col1_16 ASC), UNIQUE (col1_6 ASC, col1_0 ASC, col1_8, col1_7 ASC, col1_12 DESC, col1_14 DESC, col1_16, col1_3 ASC, col1_10, col1_13, col1_4 DESC, col1_9 ASC, col1_1 DESC), UNIQUE (col1_4) WHERE table1.col1_10 <= '00:00:00', FAMILY (col1_6, col1_4), FAMILY (col1_11), FAMILY (col1_0), FAMILY (col1_7), FAMILY (col1_14, col1_8), FAMILY (col1_1, col1_9), FAMILY (col1_2, col1_5), FAMILY (col1_15, col1_16, col1_17), FAMILY (col1_10), FAMILY (col1_3, col1_13), FAMILY (col1_12));
        test_1        |             CREATE TABLE table2 (col2_0 BYTES NOT NULL, col2_1 INT8 NOT NULL, col2_2 BOX2D NULL, col2_3 CHAR NULL, col2_4 BYTES NOT NULL, col2_5 STRING, col2_6 TIME, col2_7 BYTES NOT NULL, col2_8 REGPROCEDURE NULL, INDEX (col2_3 DESC, col2_5, col2_7 ASC, col2_1 ASC, col2_0, col2_2 ASC, col2_8 DESC) WHERE ((((((table2.col2_4 >= '\x27') AND (table2.col2_5 > 'X')) OR (table2.col2_0 = '\x')) AND (table2.col2_6 <= '24:00:00')) OR (table2.col2_3 >= e'\x00')) OR (table2.col2_7 < '\x22')) OR (table2.col2_1 = 2147483647), INDEX (col2_1 DESC, col2_0 ASC, col2_7, col2_3 DESC, col2_5 DESC, col2_6 ASC, col2_4 ASC, col2_2 DESC) WHERE (((((table2.col2_4 < '\xff') OR (table2.col2_6 = '24:00:00')) AND (table2.col2_0 = '\xe29883')) AND (table2.col2_3 = 'X')) AND (table2.col2_1 != -9223372036854775807)) AND (table2.col2_7 != '\xff'), FAMILY (col2_6, col2_8, col2_1), FAMILY (col2_2, col2_4, col2_0), FAMILY (col2_7, col2_5), FAMILY (col2_3));
        test_1        |             CREATE TABLE table3 (col3_0 NAME NOT NULL, col3_1 OID NOT NULL, col3_2 FLOAT8 NOT NULL, col3_3 BOOL, col3_4 TIMESTAMP NULL, col3_5 INT2 NULL, col3_6 UUID[], col3_7 BIT(28) NOT NULL, col3_8 REGPROC, col3_9 DATE NULL, col3_10 BYTES NOT NULL, PRIMARY KEY (col3_10 ASC, col3_1 DESC), UNIQUE (col3_10 DESC, col3_1 ASC, col3_5, col3_8, col3_2, col3_4 ASC, col3_0 ASC, col3_7 ASC, col3_3 ASC) STORING (col3_6, col3_9) WHERE ((((((table3.col3_4 <= '-4713-11-24 00:00:00':::TIMESTAMP) OR table3.col3_3) OR (table3.col3_2 = 1.7976931348623157e+308:::FLOAT8)) OR (table3.col3_5 >= 1:::INT8)) OR (table3.col3_9 > '-infinity':::DATE)) AND (table3.col3_0 >= 'X':::STRING)) OR (table3.col3_10 >= e'\\xe29883':::BYTES), INDEX (col3_10) STORING (col3_0, col3_2, col3_4, col3_5, col3_7), INDEX (col3_8 DESC, col3_0 DESC, col3_7 DESC, col3_4 ASC, col3_2 ASC, col3_5) STORING (col3_3, col3_6, col3_9) WHERE ((table3.col3_10 < e'\\x22':::BYTES) AND table3.col3_3) OR (table3.col3_4 = '294276-12-31 23:59:59.999999':::TIMESTAMP), UNIQUE (col3_4, col3_5, col3_2 ASC) STORING (col3_0, col3_3, col3_6, col3_7, col3_9) WHERE (((((table3.col3_3 AND (table3.col3_2 < 1.0:::FLOAT8)) AND (table3.col3_5 >= (-32768):::INT8)) AND (table3.col3_9 = 'infinity':::DATE)) OR (table3.col3_10 = e'\\xff':::BYTES)) OR (table3.col3_0 != '"':::STRING)) AND (table3.col3_4 <= '-2000-01-01 00:00:00':::TIMESTAMP), INDEX (col3_3 DESC, col3_2 ASC, col3_8 ASC, col3_7 DESC, col3_0 ASC, col3_1 DESC, col3_4 DESC, col3_10 ASC, col3_9 ASC) STORING (col3_5) WHERE ((table3.col3_5 <= (-1):::INT8) AND (table3.col3_0 != e'\x00':::STRING)) AND (table3.col3_10 = e'\\x58':::BYTES), UNIQUE (col3_5 ASC, col3_0, col3_7 ASC, col3_2 ASC, col3_9, col3_4 DESC, col3_3 DESC, col3_1 DESC) STORING (col3_8) WHERE ((table3.col3_9 >= '5874897-12-31') OR (table3.col3_4 != '294276-12-31 23:59:59.999999')) AND table3.col3_3, INDEX (col3_0 ASC, col3_9, col3_8, col3_10 ASC, col3_2 DESC, col3_3, col3_1 DESC) WHERE (table3.col3_0 = 'X':::STRING) AND (table3.col3_10 < e'\\xff':::BYTES), UNIQUE (col3_7, col3_4 DESC, col3_2 DESC, col3_0 ASC, col3_8 ASC, col3_5 DESC) STORING (col3_3, col3_6, col3_9) WHERE table3.col3_5 = -128, UNIQUE (col3_1 ASC, col3_3, col3_10) STORING (col3_4, col3_9), FAMILY (col3_0), FAMILY (col3_1), FAMILY (col3_9), FAMILY (col3_2), FAMILY (col3_8), FAMILY (col3_10, col3_7), FAMILY (col3_6), FAMILY (col3_4, col3_5, col3_3));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2795713506506341133, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3631507901853077760, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3278078219364544343, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2743469103789528535, "histo_buckets": [{"distinct_range": 0, "num_eq": 6352767344518862668, "num_range": 0, "upper_bound": "-2111069979"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 0, "upper_bound": "-1760576351"}, {"distinct_range": 1751208721264125400, "num_eq": 5267000346565214947, "num_range": 2256051821891703857, "upper_bound": "-1340296125"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 200, "upper_bound": "-194824440"}, {"distinct_range": 91458.65722231775, "num_eq": 7003061541674705824, "num_range": 200000, "upper_bound": "-68272869"}, {"distinct_range": 2425278193848358400, "num_eq": 200, "num_range": 5889902726508350890, "upper_bound": "530803293"}, {"distinct_range": 248409.51437661788, "num_eq": 2643555654299545668, "num_range": 400000, "upper_bound": "1098074848"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2641906270362425480, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1743328931034874232, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2006175046283087977, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 256409607822523119, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3406140546691710346, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 311377659024251536, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2031769521472233708, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3577504888071801006, "histo_buckets": [{"distinct_range": 0, "num_eq": 900, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 9011851870890882953, "num_range": 300000000, "upper_bound": "689274464"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "2842386004"}, {"distinct_range": 0, "num_eq": 975390790449681244, "num_range": 500000, "upper_bound": "3327869788"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2866446050428068439, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3018138364317659137, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3769478109624155672, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2052587366949205783, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 491009260210896871, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1256152216156569602, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3960599353059477488}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 75, "histo_buckets": [{"distinct_range": 0, "num_eq": 30, "num_range": 0, "upper_bound": "-5863659080523255590"}, {"distinct_range": 0, "num_eq": 6076455453909612799, "num_range": 7262402426608415404, "upper_bound": "-5142564466026920992"}, {"distinct_range": 67.4771582033165, "num_eq": 8587523864985557566, "num_range": 70, "upper_bound": "-4182683876605610253"}, {"distinct_range": 2888738335997903400, "num_eq": 600, "num_range": 4020923217407090488, "upper_bound": "608107545589644282"}, {"distinct_range": 0, "num_eq": 1652735685143875730, "num_range": 0, "upper_bound": "4127563512786655305"}, {"distinct_range": 4113783472242794500, "num_eq": 3762666621469472581, "num_range": 4113783472242794687, "upper_bound": "7972148455776368970"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10, "histo_col_type": "", "name": "__auto__", "null_count": 37, "row_count": 90}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22, "histo_col_type": "", "name": "__auto__", "null_count": 67, "row_count": 90}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 48, "histo_col_type": "", "name": "__auto__", "null_count": 59, "row_count": 90}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25, "histo_buckets": [{"distinct_range": 0, "num_eq": 5205328016536582247, "num_range": 0, "upper_bound": "\\u0004"}, {"distinct_range": 681870442354435700, "num_eq": 1000000, "num_range": 1306588518399069249, "upper_bound": "\\u000c"}, {"distinct_range": 5872341128478166000, "num_eq": 7630957821833541371, "num_range": 5872341128478165686, "upper_bound": "\'"}, {"distinct_range": 95204589766382500, "num_eq": 40000000, "num_range": 5118954178444658585, "upper_bound": "("}, {"distinct_range": 80000, "num_eq": 6949319706967385470, "num_range": 80000, "upper_bound": "L"}, {"distinct_range": 0, "num_eq": 0, "num_range": 10000000000, "upper_bound": "P"}, {"distinct_range": 0, "num_eq": 1107402157717204062, "num_range": 20000000, "upper_bound": "|"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 76, "row_count": 90}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 77, "histo_col_type": "", "name": "__auto__", "null_count": 30, "row_count": 90}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 93, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 2240749061.785747, "num_eq": 5467767110355479751, "num_range": 10000000000, "upper_bound": "\\"X-\\u0010@\\u0015\\u000e\\u001a"}, {"distinct_range": 100000, "num_eq": 2145516377046830662, "num_range": 100000, "upper_bound": "/Tz*gZ\\u001c"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 60000000000, "upper_bound": "DW=w\\u0011"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17, "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 66, "histo_col_type": "INT2", "name": "__auto__", "null_count": 4, "row_count": 100}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 96, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000, "num_range": 0, "upper_bound": "0100010011111100101100101000"}, {"distinct_range": 0, "num_eq": 7431481632106219148, "num_range": 500000000, "upper_bound": "1000111111111000011000111110"}, {"distinct_range": 40, "num_eq": 8303606661982815013, "num_range": 40, "upper_bound": "1001010010111101010111110111"}, {"distinct_range": 1188370418327704600, "num_eq": 6064798431239114079, "num_range": 1188370418327704480, "upper_bound": "1101111001001110010110010110"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 8392812235626363691, "upper_bound": "1111000110011111011010101100"}], "histo_col_type": "BIT(28)", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 58, "histo_col_type": "", "name": "__auto__", "null_count": 62, "row_count": 100}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 72, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "\\\\x27"}, {"distinct_range": 5237950489689894000, "num_eq": 2574042540360328387, "num_range": 6080700228624513642, "upper_bound": "\\\\x6b3a243e6265"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 74, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3, "histo_buckets": [{"distinct_range": 0, "num_eq": 1897881997633040721, "num_range": 0, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 75, "row_count": 100}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 62, "histo_buckets": [{"distinct_range": 0, "num_eq": 8795307872499444175, "num_range": 0, "upper_bound": "1983-12-24 10:19:53.000678"}, {"distinct_range": 94374429134420770, "num_eq": 8397248571778845617, "num_range": 1154079195854366062, "upper_bound": "1988-04-23 05:02:22.000126"}, {"distinct_range": 0, "num_eq": 235940949194480061, "num_range": 40000, "upper_bound": "1995-06-04 01:58:54.000365"}, {"distinct_range": 8811406439519458000, "num_eq": 2598652400632609567, "num_range": 8811406439519458096, "upper_bound": "2011-07-16 17:20:48.000657"}, {"distinct_range": 2500832578507871700, "num_eq": 70000000000, "num_range": 5286653915689887277, "upper_bound": "2028-11-04 09:45:23.00087"}, {"distinct_range": 0, "num_eq": 900, "num_range": 60000000, "upper_bound": "294276-12-31 23:59:59.999999"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 60, "row_count": 100}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36, "histo_col_type": "", "name": "__auto__", "null_count": 34, "row_count": 100}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20, "histo_buckets": [{"distinct_range": 0, "num_eq": 600, "num_range": 0, "upper_bound": "142669579"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 3000000000, "upper_bound": "856890562"}, {"distinct_range": 0, "num_eq": 300, "num_range": 20000000000, "upper_bound": "2133239683"}, {"distinct_range": 0, "num_eq": 3744717771951646820, "num_range": 0, "upper_bound": "2193417541"}, {"distinct_range": 37345305.68330024, "num_eq": 1617324112078695943, "num_range": 80000000, "upper_bound": "3109745903"}, {"distinct_range": 1726542539.6945443, "num_eq": 6990639797597229008, "num_range": 3000000000, "upper_bound": "3596275250"}, {"distinct_range": 0, "num_eq": 6117874522182530256, "num_range": 600000000, "upper_bound": "4000720937"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 41, "row_count": 100}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('uyv', 'sd', 'jh', 'pda', 'decrea');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('gcnfe', 'hsbjj');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('egahu', 'ldrv');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('obibcu', 'oiz');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('r', 'xa', 'b', 'wwbxz', 'xgu', 'yaxdm');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8632124401489579693, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1579501424241536796, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1687469192996139061, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 7511626181121112474, "num_range": 6285348839605557214, "upper_bound": "452212976"}, {"distinct_range": 56477786.939328, "num_eq": 1000, "num_range": 80000000, "upper_bound": "3028729387"}, {"distinct_range": 700000000, "num_eq": 3265418679928193436, "num_range": 700000000, "upper_bound": "3350163711"}, {"distinct_range": 563503081566560900, "num_eq": 5822559562956324343, "num_range": 2901947732124770910, "upper_bound": "3695019812"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7542702584552779941, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7574759554313898544, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1370427277578853899, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4520732540148382758, "histo_buckets": [{"distinct_range": 0, "num_eq": 2945288329998789192, "num_range": 0, "upper_bound": "-1508225889"}, {"distinct_range": 90, "num_eq": 100000000, "num_range": 90, "upper_bound": "-836927763"}, {"distinct_range": 273096144343725700, "num_eq": 1000, "num_range": 746804784815137624, "upper_bound": "-38014643"}, {"distinct_range": 1000000000, "num_eq": 9112417567272700675, "num_range": 1000000000, "upper_bound": "687658350"}, {"distinct_range": 4183405008406341600, "num_eq": 6920466691925990111, "num_range": 8814629009901481357, "upper_bound": "959091564"}, {"distinct_range": 0, "num_eq": 500, "num_range": 0, "upper_bound": "1392008049"}, {"distinct_range": 1122419601402432900, "num_eq": 440851528424198375, "num_range": 6853306078495502487, "upper_bound": "1839592259"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5840182119493934785, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4704726887260414635, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2031940594564753268, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4828297301635477336, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5018807459188454583, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4256744196691601901, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7085642918954645846, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3252492089307130017, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6942532145767753963, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7924204401258867163, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2959787616029702745, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8904351987787495481}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 73, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23, "histo_col_type": "", "name": "__auto__", "null_count": 41, "row_count": 80}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38, "histo_col_type": "", "name": "__auto__", "null_count": 59, "row_count": 80}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": "-6426644313018432537"}, {"distinct_range": 59014736412.92949, "num_eq": 70000000000, "num_range": 60000000000, "upper_bound": "2147483647"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7, "histo_col_type": "", "name": "__auto__", "null_count": 38, "row_count": 80}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2, "histo_buckets": [{"distinct_range": 0, "num_eq": 4362526352511839658, "num_range": 0, "upper_bound": "O"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 59, "row_count": 80}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15, "histo_col_type": "", "name": "__auto__", "null_count": 6, "row_count": 80}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 66, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2800000, "histo_buckets": [{"distinct_range": 0, "num_eq": 1744739976016940433, "num_range": 0, "upper_bound": "\\u0013\\u0016OmsD\\u0017_\\u0015"}, {"distinct_range": 3239223397039250000, "num_eq": 6739268177967230813, "num_range": 3239223397039249811, "upper_bound": "\\u0015f@\\u0004"}, {"distinct_range": 800571.2956408939, "num_eq": 3653128318513317431, "num_range": 1000000, "upper_bound": "9\\rT"}, {"distinct_range": 842449275776948900, "num_eq": 4864788554390857579, "num_range": 842449275776948855, "upper_bound": ">&\\u000fR"}, {"distinct_range": 977679327185656000, "num_eq": 200000, "num_range": 977679327185656002, "upper_bound": "m\\u0011@"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 30000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13765506, "histo_buckets": [{"distinct_range": 0, "num_eq": 379287204222325886, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 70000000000, "upper_bound": "1671497896"}, {"distinct_range": 0, "num_eq": 8754508093062505630, "num_range": 4000000, "upper_bound": "1726547811"}, {"distinct_range": 0, "num_eq": 800000000, "num_range": 90000000000, "upper_bound": "1854974157"}, {"distinct_range": 2200130254745472000, "num_eq": 8021230992067354727, "num_range": 2200130254745472000, "upper_bound": "1881103166"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 6328940427013562429, "upper_bound": "2550131457"}, {"distinct_range": 0, "num_eq": 4041000516782668680, "num_range": 5000, "upper_bound": "3130244212"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 8000, "upper_bound": "3841202937"}, {"distinct_range": 0, "num_eq": 5750077225665997254, "num_range": 300000, "upper_bound": "4021018561"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 30000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21298340, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4749887, "histo_buckets": [{"distinct_range": 0, "num_eq": 9210383941045425506, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 54468208149846240, "num_eq": 8000000, "num_range": 5847394271699705662, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 24905552, "row_count": 30000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2160226, "histo_buckets": [{"distinct_range": 0, "num_eq": 1259583951586761704, "num_range": 0, "upper_bound": "-4713-11-24 00:00:00"}, {"distinct_range": 2363.400300432634, "num_eq": 0, "num_range": 3000, "upper_bound": "-2000-01-01 00:00:00"}, {"distinct_range": 0, "num_eq": 90000, "num_range": 1473528275179636776, "upper_bound": "1981-08-24 23:47:20.000233"}, {"distinct_range": 0, "num_eq": 6854436786050818749, "num_range": 700, "upper_bound": "2002-10-18 07:40:36.000076"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 225336, "row_count": 30000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3278392, "histo_buckets": [{"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "-13161"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 27486643, "row_count": 30000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5540864, "histo_col_type": "", "name": "__auto__", "null_count": 20592524, "row_count": 30000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10643478, "histo_buckets": [{"distinct_range": 0, "num_eq": 40, "num_range": 0, "upper_bound": "0000000000110011111101000000"}, {"distinct_range": 10000000, "num_eq": 60000000000, "num_range": 10000000, "upper_bound": "0010100101010100111001110001"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 4788282481791192411, "upper_bound": "1010100101100001111001100000"}], "histo_col_type": "BIT(28)", "name": "__auto__", "null_count": 0, "row_count": 30000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28693852, "histo_buckets": [{"distinct_range": 0, "num_eq": 7570483111121960555, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 9000000, "num_eq": 20000000, "num_range": 9000000, "upper_bound": "482236892"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 13773758, "row_count": 30000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7643661, "histo_col_type": "", "name": "__auto__", "null_count": 23236773, "row_count": 30000000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27233477, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000000, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 17309825.24890468, "num_eq": 9163902444304847492, "num_range": 20000000, "upper_bound": "\\\\x237a3a8336"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 6583020592871451985, "upper_bound": "\\\\x75f0ac"}, {"distinct_range": 0, "num_eq": 2458697300043061708, "num_range": 4526739027712377692, "upper_bound": "\\\\x85"}, {"distinct_range": 4000000000, "num_eq": 300, "num_range": 4000000000, "upper_bound": "\\\\x8a6f894f"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 8296385367607875040, "upper_bound": "\\\\xac60273e11c6ce2585"}, {"distinct_range": 2802572026331146000, "num_eq": 40000000000, "num_range": 6013802585822057347, "upper_bound": "\\\\xf9837ec0c908971d"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 30000000}]';
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_1) REFERENCES table1 (col1_4);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3350210668809340206, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2313707396632490341, "histo_buckets": [{"distinct_range": 0, "num_eq": 40, "num_range": 0, "upper_bound": "-1376282136"}, {"distinct_range": 0, "num_eq": 0, "num_range": 5422499501663530878, "upper_bound": "-1092286340"}, {"distinct_range": 52338718469.771385, "num_eq": 1374721010403052454, "num_range": 60000000000, "upper_bound": "-177653260"}, {"distinct_range": 3679316761.2269373, "num_eq": 10000, "num_range": 5000000000, "upper_bound": "633347814"}, {"distinct_range": 0, "num_eq": 7993205199873483668, "num_range": 5000000, "upper_bound": "828132362"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 999895801388129887, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 832720070257459345, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2332895565492335364, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5981707496162165612, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 328637534046815514, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2594845146469292778, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3221524860517753192, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1368569476222306448, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4179352997762773820, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2151148021693895787, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 711544944026728371, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4432708561385555695, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5983824730924452751, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-1603014573695892366"}, {"distinct_range": 0, "num_eq": 73031234442322596, "num_range": 6000000000, "upper_bound": "0"}, {"distinct_range": 629725745197664400, "num_eq": 20000, "num_range": 629725745197664358, "upper_bound": "1263837507571586764"}, {"distinct_range": 0, "num_eq": 8856898889836001538, "num_range": 30000000000, "upper_bound": "6076125686147678195"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1794391349972924543, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3623027496662239721, "histo_buckets": [{"distinct_range": 0, "num_eq": 3133196645364963936, "num_range": 0, "upper_bound": "2225418000"}, {"distinct_range": 0, "num_eq": 2918467958512051882, "num_range": 200000, "upper_bound": "2322893376"}, {"distinct_range": 0, "num_eq": 1105763395146593975, "num_range": 900000000, "upper_bound": "3495993587"}, {"distinct_range": 4666781186044528000, "num_eq": 3628284295807886735, "num_range": 4666781186044527771, "upper_bound": "3946372543"}, {"distinct_range": 0, "num_eq": 6523956583693248948, "num_range": 3574922937956497081, "upper_bound": "3971767515"}, {"distinct_range": 4759507735057674000, "num_eq": 70000000, "num_range": 6141729434872419963, "upper_bound": "4138193985"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6360578151904689162, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6554066410766373003}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32, "histo_col_type": "", "name": "__auto__", "null_count": 3, "row_count": 60}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 43, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 48, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": "-5688955101050574531"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 976523530629703436, "upper_bound": "5147943334930916574"}, {"distinct_range": 710615446326660600, "num_eq": 8028480799431590925, "num_range": 1449996118993337793, "upper_bound": "6530973518479737773"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53, "histo_col_type": "CHAR", "name": "__auto__", "null_count": 50, "row_count": 60}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 58, "histo_col_type": "", "name": "__auto__", "null_count": 44, "row_count": 60}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 37, "histo_col_type": "", "name": "__auto__", "null_count": 16, "row_count": 60}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10, "histo_col_type": "", "name": "__auto__", "null_count": 37, "row_count": 60}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 60, "histo_buckets": [{"distinct_range": 0, "num_eq": 3337634030657405806, "num_range": 0, "upper_bound": "\\u0006\\u001fD\\u0005"}, {"distinct_range": 1286389875435306500, "num_eq": 6577204623455288719, "num_range": 5144750467892507024, "upper_bound": "\\u000f+t#V"}, {"distinct_range": 1223941.172737636, "num_eq": 20000000000, "num_range": 3000000, "upper_bound": "\\u00133,33"}, {"distinct_range": 8263084034364196000, "num_eq": 7827039539948657257, "num_range": 8263084034364195529, "upper_bound": "20<8(h\\\\"}, {"distinct_range": 0, "num_eq": 7178185067546454276, "num_range": 0, "upper_bound": "@"}, {"distinct_range": 0, "num_eq": 1099956628794375429, "num_range": 0, "upper_bound": "S9g\x7fuD"}, {"distinct_range": 0, "num_eq": 9000000, "num_range": 10, "upper_bound": "e@\\u000b)Q+"}, {"distinct_range": 0, "num_eq": 3000000, "num_range": 10000, "upper_bound": "u\\u00045\\u0002+2g="}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000, "num_range": 0, "upper_bound": "-23321"}, {"distinct_range": 100000, "num_eq": 2943009082240920047, "num_range": 100000, "upper_bound": "-23223"}, {"distinct_range": 1535294557692157200, "num_eq": 5659587261783116701, "num_range": 1535294557692157068, "upper_bound": "-17510"}, {"distinct_range": 800, "num_eq": 600000, "num_range": 800, "upper_bound": "-17125"}, {"distinct_range": 60000, "num_eq": 6313993096943737517, "num_range": 60000, "upper_bound": "-11891"}, {"distinct_range": 0, "num_eq": 80, "num_range": 0, "upper_bound": "16756"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 30, "row_count": 90}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 54, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000000, "num_range": 0, "upper_bound": "0000001011110100011100001001"}, {"distinct_range": 700000, "num_eq": 6000000, "num_range": 700000, "upper_bound": "0001011110011101011101111111"}, {"distinct_range": 1500550982974378800, "num_eq": 100, "num_range": 6487725143052454841, "upper_bound": "1001000100111101111100001000"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "1011001101001010100011101000"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 2000000000, "upper_bound": "1011010010000000011100010001"}, {"distinct_range": 0, "num_eq": 5000000000, "num_range": 3611783854441516350, "upper_bound": "1101100100110001111111010110"}], "histo_col_type": "BIT(28)", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 52, "histo_col_type": "", "name": "__auto__", "null_count": 19, "row_count": 90}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9, "histo_col_type": "", "name": "__auto__", "null_count": 84, "row_count": 90}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38, "histo_buckets": [{"distinct_range": 0, "num_eq": 6034646046078747864, "num_range": 0, "upper_bound": "387208497"}, {"distinct_range": 6415424581618762000, "num_eq": 80000, "num_range": 6415424581618761786, "upper_bound": "567213545"}, {"distinct_range": 10000, "num_eq": 4704290541753844001, "num_range": 10000, "upper_bound": "2517088287"}, {"distinct_range": 6000000, "num_eq": 8700747180873306278, "num_range": 6000000, "upper_bound": "3476894368"}, {"distinct_range": 1252138656660608500, "num_eq": 8146084334420025189, "num_range": 2049881641394067967, "upper_bound": "3874335815"}, {"distinct_range": 131.11115331200625, "num_eq": 3136902164370618786, "num_range": 1000, "upper_bound": "4146336383"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 25, "row_count": 90}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12, "histo_buckets": [{"distinct_range": 0, "num_eq": 6067624532023575825, "num_range": 0, "upper_bound": "\\\\x18e7a91d3a019a"}, {"distinct_range": 250644.4113358364, "num_eq": 500000000, "num_range": 10000000, "upper_bound": "\\\\x3d692c"}, {"distinct_range": 473769614215850500, "num_eq": 10056387167600139, "num_range": 1103630323855731727, "upper_bound": "\\\\xd3dd0ba0ee201d"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 69, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000000, "num_range": 0, "upper_bound": "661657392"}, {"distinct_range": 6111837074407038000, "num_eq": 6897519295012872916, "num_range": 6126144597817520871, "upper_bound": "2292159353"}, {"distinct_range": 362195936473388400, "num_eq": 300, "num_range": 465510188709054459, "upper_bound": "2448799277"}, {"distinct_range": 8018508863343022000, "num_eq": 3504232745440232832, "num_range": 8018508863343021700, "upper_bound": "2514065404"}, {"distinct_range": 0, "num_eq": 3018611045155609720, "num_range": 700, "upper_bound": "2583197046"}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 0, "upper_bound": "3032357864"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 59, "histo_buckets": [{"distinct_range": 0, "num_eq": 5237212611288793080, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 7000000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 20, "row_count": 90}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35, "histo_buckets": [{"distinct_range": 0, "num_eq": 5961601956264716829, "num_range": 0, "upper_bound": "-2000-01-01 00:00:00"}, {"distinct_range": 90000000, "num_eq": 1000000000, "num_range": 90000000, "upper_bound": "1972-04-05 03:06:49.000678"}, {"distinct_range": 1504221160963796700, "num_eq": 2000000000, "num_range": 1509770376234621322, "upper_bound": "1991-09-02 19:43:58.000111"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 1742167090872276635, "upper_bound": "2000-03-10 09:45:23.000474"}, {"distinct_range": 33309136634565884, "num_eq": 90000, "num_range": 4313505464050844255, "upper_bound": "2010-04-18 18:31:46.000024"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 67, "row_count": 90}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: there is no unique constraint matching given keys for referenced table table1 (SQLSTATE 23503)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2597710&tab=buildLog) on [master@3d01fda046c609ee5947e2f41c099dae35b206c1](https://github.com/cockroachdb/cockroach/commits/3d01fda046c609ee5947e2f41c099dae35b206c1):

```
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('kxvvdb', 'uzp', 'isfm', 'fpgn');
        postgres_1    | 	CREATE TYPE rand_typ_2 AS ENUM ('lsyk', 't', 'xdjff');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (603.93s)
        test_1        |     --- FAIL: TestCompare/postgres (3.02s)
        test_1        |         compare_test.go:129: CREATE TABLE table1 (col1_0 UUID NOT NULL, col1_1 TEXT NOT NULL, col1_2 REGPROCEDURE NULL, col1_3 BOX2D NULL, col1_4 CHAR, col1_5 REGNAMESPACE NULL, col1_6 BIT, col1_7 INTERVAL, col1_8 REGPROC NULL, col1_9 TIMESTAMP NULL, col1_10 BOOL NULL, col1_11 INTERVAL NOT NULL, col1_12 FLOAT8 NULL, col1_13 INT8, col1_14 BOOL NOT NULL, col1_15 TIMESTAMPTZ);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_5 ASC, col1_9 DESC, col1_10 ASC, col1_2 DESC, col1_14 DESC, col1_8 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_15 ASC, col1_12 DESC, col1_13 DESC, col1_0, col1_7 ASC, col1_2, col1_8 DESC, col1_6 ASC) INCLUDE (col1_5, col1_10, col1_14);
        test_1        |             CREATE INDEX ON table1 (col1_8 ASC, col1_15 DESC, col1_10 DESC, col1_13 ASC, col1_14 DESC, col1_5 ASC, col1_12, col1_0 DESC, col1_2 ASC, col1_3, col1_1 ASC, col1_6 ASC, col1_4 DESC, col1_11 ASC);
        test_1        |             CREATE INDEX ON table1 (col1_11 ASC);
        test_1        |             CREATE TABLE table2 (col2_0 TIMESTAMPTZ NOT NULL, col2_1 INT8 NOT NULL, PRIMARY KEY (col2_0));
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_1 ASC);
        test_1        |             CREATE TABLE table3 (col3_0 BOOL NULL, col3_1 REGCLASS NOT NULL, col3_2 REGPROCEDURE NULL, col3_3 INT8, col3_4 VARCHAR, col3_5 REGNAMESPACE NOT NULL, col3_6 BOX2D NULL, col3_7 "char" NOT NULL, col3_8 INTERVAL NOT NULL, col3_9 BIT(27) NOT NULL, col3_10 INET[] NOT NULL, col3_11 TIMESTAMPTZ NOT NULL, col3_12 "char" NOT NULL, col3_13 TIME NOT NULL, col3_14 INTERVAL NOT NULL, col3_15 TIMESTAMP NOT NULL, col3_16 TIMESTAMP NOT NULL, col3_17 NAME, col3_18 TIMESTAMP, PRIMARY KEY (col3_7, col3_1, col3_5, col3_16, col3_13, col3_8));
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 DESC, col3_15, col3_14 DESC, col3_18 DESC, col3_17, col3_9 DESC, col3_6 DESC) INCLUDE (col3_2, col3_3, col3_4, col3_10);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_18 DESC, col3_15 ASC, col3_17, col3_3 ASC, col3_7 DESC);
        test_1        |             CREATE INDEX ON table3 (col3_4 DESC, col3_3, col3_5 DESC, col3_15, col3_9, col3_11 DESC, col3_17 DESC, col3_0, col3_18, col3_16, col3_1 DESC, col3_8 DESC, col3_6, col3_13, col3_2 DESC, col3_14 DESC) INCLUDE (col3_10);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_5 DESC, col3_13, col3_7 DESC, col3_0 DESC, col3_14, col3_6 DESC, col3_11 ASC, col3_3 DESC, col3_18 DESC, col3_12 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_8 DESC, col3_0 ASC, col3_6 DESC, col3_12 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_16, col3_11 ASC, col3_13, col3_17 ASC, col3_2 ASC, col3_1 ASC, col3_12 DESC, col3_15 DESC, col3_9, col3_18, col3_8, col3_7 DESC, col3_3 ASC, col3_5 ASC, col3_4 ASC, col3_0 ASC, col3_6 ASC, col3_14);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_7 DESC, col3_8 DESC, col3_17, col3_6 ASC, col3_14 DESC, col3_13, col3_9 ASC, col3_12 DESC, col3_18 ASC, col3_4 DESC, col3_2 DESC, col3_15 ASC, col3_16 ASC) INCLUDE (col3_11);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_1 ASC, col3_12 DESC, col3_16 ASC, col3_9 ASC, col3_13 DESC, col3_7 DESC, col3_17 DESC, col3_11, col3_3 DESC, col3_18 ASC, col3_4, col3_5 ASC, col3_15 DESC, col3_14 DESC, col3_0 ASC, col3_6 DESC, col3_8 ASC, col3_2);
        test_1        |             CREATE TABLE table4 (col4_0 FLOAT8 NOT NULL, col4_1 INT8 NOT NULL, col4_2 DECIMAL NULL, col4_3 TIMESTAMPTZ NOT NULL, col4_4 DECIMAL NOT NULL, col4_5 BIT(8) NOT NULL, col4_6 FLOAT8 NOT NULL, col4_7 TIMESTAMPTZ NOT NULL, col4_8 FLOAT8 NOT NULL, col4_9 REGPROCEDURE NOT NULL, col4_10 TIMETZ NOT NULL, col4_11 INT8 NOT NULL, col4_12 REGTYPE NOT NULL, col4_13 REGTYPE NOT NULL, PRIMARY KEY (col4_13, col4_4, col4_0, col4_10, col4_5, col4_6, col4_12, col4_7, col4_11, col4_9, col4_1, col4_8, col4_3));
        test_1        |             CREATE INDEX ON table4 (col4_0, col4_3 ASC, col4_4 DESC, col4_13 ASC, col4_5, col4_6 ASC, col4_1 DESC, col4_12 ASC, col4_10 ASC, col4_8 DESC, col4_9 ASC, col4_7 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_6 ASC, col4_7, col4_4, col4_5, col4_1, col4_9 ASC, col4_8 DESC, col4_10);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_2 DESC, col4_4 ASC, col4_12, col4_9 ASC, col4_8, col4_0, col4_10 ASC);
        test_1        |             CREATE INDEX ON table4 (col4_0 DESC, col4_6 DESC, col4_12, col4_5 DESC, col4_4, col4_10 ASC, col4_9, col4_1 DESC, col4_8 DESC, col4_13 ASC, col4_7 ASC, col4_11 ASC, col4_3 DESC, col4_2 ASC);
        test_1        |             CREATE INDEX ON table4 (col4_4 DESC, col4_10 ASC, col4_0 ASC, col4_12, col4_13 ASC, col4_3 DESC, col4_11 DESC, col4_6 DESC);
        test_1        |             CREATE TABLE table5 (col2_0 TIMESTAMPTZ NOT NULL, col5_1 FLOAT8 NULL, col5_2 "char", col5_3 REGPROCEDURE NOT NULL, PRIMARY KEY (col2_0));
        test_1        |             CREATE INDEX ON table5 (col2_0 DESC);
        test_1        |             CREATE INDEX ON table5 (col5_3);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_1 ASC, col5_2 ASC, col2_0 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_1, col5_3, col5_2 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_2 DESC, col5_1 DESC) INCLUDE (col5_3);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_1 ASC) INCLUDE (col5_2);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('pkhuwm', 'd', 'lo', 'xtiwsw', 'n', 'ekehf');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('kxvvdb', 'uzp', 'isfm', 'fpgn');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('lsyk', 't', 'xdjff');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2601073&tab=buildLog) on [master@fbf596c3e17fbb9ec0935b732f4b84469a5399e8](https://github.com/cockroachdb/cockroach/commits/fbf596c3e17fbb9ec0935b732f4b84469a5399e8):

```
        postgres_1    | 
        postgres_1    | 2021-01-21 06:20:35.748 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-01-21 06:20:35.748 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-01-21 06:20:35.751 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-01-21 06:20:35.766 UTC [86] LOG:  database system was shut down at 2021-01-21 06:20:35 UTC
        postgres_1    | 2021-01-21 06:20:35.771 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (4.36s)
        test_1        |     --- FAIL: TestCompare/postgres (3.36s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.3:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (1.00s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 REGPROC NOT NULL, col1_1 DECIMAL NOT NULL, col1_2 REGCLASS NOT NULL, col1_3 BOX2D NOT NULL, col1_4 REGPROCEDURE NOT NULL, col1_5 DATE NOT NULL, col1_6 BOOL NOT NULL, col1_7 VARCHAR NOT NULL, col1_8 VARCHAR NOT NULL, col1_9 INT4 NOT NULL, col1_10 REGPROC NOT NULL, col1_11 "char" NOT NULL, PRIMARY KEY (col1_7, col1_1 ASC, col1_8, col1_6, col1_4 DESC, col1_9, col1_11 ASC, col1_5, col1_2 ASC, col1_3 DESC, col1_0 DESC), UNIQUE (col1_10, col1_0 DESC), INDEX (col1_6, col1_2 ASC, col1_11 DESC, col1_4, col1_8 DESC), FAMILY (col1_3), FAMILY (col1_10, col1_4), FAMILY (col1_9), FAMILY (col1_11), FAMILY (col1_5, col1_0, col1_7, col1_6), FAMILY (col1_8), FAMILY (col1_1), FAMILY (col1_2));
        test_1        |             CREATE TABLE table2 (col2_0 BIT(31), col2_1 BIT(7) NOT NULL, col2_2 INT2, INDEX (col2_2, col2_1 DESC), INDEX (col2_0 DESC, col2_2, col2_1) WHERE table2.col2_2 < 32767, INDEX (col2_1, col2_2 DESC) STORING (col2_0), INDEX (col2_1 DESC, col2_2) WHERE table2.col2_2 >= (-1):::INT8, INDEX (col2_0) STORING (col2_1, col2_2) WHERE table2.col2_2 > -128, INDEX (col2_0 DESC) STORING (col2_1, col2_2) WHERE table2.col2_2 > (-32768):::INT8, UNIQUE (col2_2, col2_0 DESC, col2_1) WHERE table2.col2_2 > 127:::INT8, UNIQUE (col2_1 DESC, col2_0 DESC) STORING (col2_2), UNIQUE (col2_1 DESC, col2_0 DESC) WHERE table2.col2_2 >= (-1):::INT8, FAMILY (col2_0), FAMILY (col2_1, col2_2));
        test_1        |             CREATE TABLE table3 (col3_0 REGNAMESPACE NOT NULL, col3_1 BIT(7), col3_2 BOOL NOT NULL, PRIMARY KEY (col3_0, col3_2), INDEX (col3_1, col3_0 DESC, col3_2 DESC) WHERE table3.col3_2, INDEX (col3_2 ASC, col3_1, col3_0 DESC) WHERE NOT table3.col3_2, UNIQUE (col3_1 DESC, col3_2 ASC) WHERE NOT table3.col3_2, UNIQUE (col3_1 ASC, col3_2 DESC), UNIQUE (col3_1 ASC, col3_2 DESC) WHERE table3.col3_2, UNIQUE (col3_2 DESC, col3_1 ASC), INDEX (col3_2 DESC, col3_1 ASC) WHERE NOT table3.col3_2, FAMILY (col3_2, col3_0, col3_1));
        test_1        |             CREATE TABLE table4 (col4_0 REGTYPE NULL, col4_1 REGPROC NOT NULL, col4_2 REGPROCEDURE NOT NULL, col4_3 UUID NOT NULL, col4_4 DECIMAL NULL, col4_5 BOOL NOT NULL, col4_6 FLOAT8, col4_7 TIMETZ NOT NULL, col4_8 REGCLASS NULL, col4_9 DATE, PRIMARY KEY (col4_2 DESC, col4_7 DESC, col4_1 DESC, col4_3 ASC), INDEX (col4_7 ASC, col4_4, col4_0, col4_9, col4_5 ASC) WHERE NOT table4.col4_5, INDEX (col4_6, col4_3 ASC) STORING (col4_4) WHERE ((((table4.col4_9 <= 'infinity') OR (table4.col4_6 != 5e-324)) OR (table4.col4_4 <= 1)) AND table4.col4_5) AND (table4.col4_7 >= '00:00:00+15:59:00'), INDEX (col4_5, col4_7, col4_3 ASC, col4_8 ASC, col4_4, col4_1 ASC) STORING (col4_0, col4_6) WHERE ((((table4.col4_4 < 'Infinity':::DECIMAL) OR (NOT table4.col4_5)) AND (table4.col4_7 = '24:00:00-15:59:00':::TIMETZ)) OR (table4.col4_9 < '5874897-12-31':::DATE)) OR (table4.col4_6 = 0.0:::FLOAT8), INDEX (col4_4 DESC, col4_7 DESC, col4_8, col4_5, col4_9, col4_2 DESC, col4_6, col4_0 DESC) WHERE ((((table4.col4_4 >= NaN) OR (table4.col4_9 = 'infinity')) OR (table4.col4_7 = '00:00:00+15:59:00')) AND (table4.col4_6 < -1.0)) AND table4.col4_5, UNIQUE (col4_8 DESC, col4_0, col4_1, col4_7 ASC, col4_5, col4_9, col4_4) STORING (col4_6) WHERE ((((table4.col4_9 = '-infinity':::DATE) OR (table4.col4_7 >= '24:00:00-15:59:00':::TIMETZ)) AND (table4.col4_6 <= 3.4028234663852886e+38:::FLOAT8)) AND (NOT table4.col4_5)) AND (table4.col4_4 >= 0:::DECIMAL), INDEX (col4_1 DESC, col4_5, col4_9, col4_2 DESC, col4_7 ASC, col4_8) STORING (col4_4, col4_6) WHERE ((((table4.col4_9 < 'infinity':::DATE) AND (table4.col4_4 != 1:::DECIMAL)) AND (NOT table4.col4_5)) OR (table4.col4_7 <= '00:00:00+15:59:00':::TIMETZ)) AND (table4.col4_6 < 0.0:::FLOAT8), UNIQUE (col4_0, col4_4), FAMILY (col4_4), FAMILY (col4_7), FAMILY (col4_8), FAMILY (col4_0), FAMILY (col4_2, col4_3), FAMILY (col4_5, col4_9, col4_6), FAMILY (col4_1));
        test_1        |             CREATE TABLE table5 (col5_0 BIT(25) NOT NULL, col5_1 FLOAT8, col5_2 REGPROC NOT NULL, col5_3 TIME NOT NULL, PRIMARY KEY (col5_0 ASC, col5_2, col5_3 DESC), INDEX (col5_3 ASC, col5_0 DESC, col5_1), UNIQUE (col5_0 ASC, col5_2, col5_3 ASC, col5_1 DESC) WHERE table5.col5_1 != 0.0, INDEX (col5_0 DESC, col5_1, col5_3 DESC, col5_2 DESC), INDEX (col5_2 DESC, col5_3 ASC, col5_0) WHERE (table5.col5_1 >= 1.401298464324817e-45:::FLOAT8) AND (table5.col5_3 != '24:00:00':::TIME), UNIQUE (col5_1) WHERE (table5.col5_3 != '24:00:00':::TIME) OR (table5.col5_1 < '+Inf':::FLOAT8), FAMILY (col5_1), FAMILY (col5_3, col5_0), FAMILY (col5_2), UNIQUE (col5_1) WHERE table5.col5_3 <= '00:00:00');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1000, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1308, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 179, "histo_buckets": [{"distinct_range": 0, "num_eq": 8718872296110477427, "num_range": 0, "upper_bound": "\\tq}>3A<O\\u0006"}, {"distinct_range": 148119091347290370, "num_eq": 4656689514613682427, "num_range": 495205820961249724, "upper_bound": "):i^+"}, {"distinct_range": 7918498313021161000, "num_eq": 900, "num_range": 7918498313021161156, "upper_bound": "tE\\"B/"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1278, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 294, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1825, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": "163549384"}, {"distinct_range": 0, "num_eq": 7461398037953894865, "num_range": 2388849752838022160, "upper_bound": "827996696"}, {"distinct_range": 44859.31322847276, "num_eq": 30, "num_range": 80000, "upper_bound": "2905923639"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "3304547857"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 746, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 156, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1898, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 948, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1864, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 283, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 4000000, "num_range": 10000000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 2000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1644420754510368201, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000, "num_range": 0, "upper_bound": "0010100010111111110000111001110"}, {"distinct_range": 0, "num_eq": 5533047769757513055, "num_range": 700000, "upper_bound": "0011000101010101001110010101111"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 8045572287662641420, "upper_bound": "0111111111111111111111111111111"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 20, "upper_bound": "1000001000111111001101111100000"}, {"distinct_range": 10000000, "num_eq": 7179998625195616074, "num_range": 10000000, "upper_bound": "1011001001111100100010000111101"}], "histo_col_type": "BIT(31)", "name": "__auto__", "null_count": 1513137244328274265, "row_count": 7072419464497300853}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4551237409208325361, "histo_buckets": [{"distinct_range": 0, "num_eq": 8516450718169793764, "num_range": 0, "upper_bound": "0010001"}, {"distinct_range": 0, "num_eq": 4674409387393189097, "num_range": 6858092394800156750, "upper_bound": "1000100"}], "histo_col_type": "BIT(7)", "name": "__auto__", "null_count": 0, "row_count": 7072419464497300853}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3927342927322558025, "histo_buckets": [{"distinct_range": 0, "num_eq": 3136517278933107575, "num_range": 0, "upper_bound": "-25546"}, {"distinct_range": 0, "num_eq": 2694618145476716070, "num_range": 3283447126333326030, "upper_bound": "-25073"}, {"distinct_range": 6676724069948432000, "num_eq": 5000000000, "num_range": 8213322147171936479, "upper_bound": "-1"}, {"distinct_range": 125.37171344449857, "num_eq": 20000000000, "num_range": 500, "upper_bound": "5247"}, {"distinct_range": 0, "num_eq": 70, "num_range": 100000, "upper_bound": "5304"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 6856730837841916163, "upper_bound": "13653"}, {"distinct_range": 100000, "num_eq": 4082514045853448011, "num_range": 100000, "upper_bound": "24409"}, {"distinct_range": 1295980.6404468808, "num_eq": 6368974726238610054, "num_range": 6000000, "upper_bound": "27354"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 124489549355842861, "row_count": 7072419464497300853}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 367588632931696280, "histo_buckets": [{"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 2018659622579952400, "num_eq": 0, "num_range": 2018659622579952295, "upper_bound": "1314407250"}, {"distinct_range": 3373649583332040700, "num_eq": 1544578105034738280, "num_range": 3552224151451132059, "upper_bound": "1507786940"}, {"distinct_range": 643000035.8980676, "num_eq": 2896701389560628942, "num_range": 5000000000, "upper_bound": "2321598078"}, {"distinct_range": 33495769058.15424, "num_eq": 90, "num_range": 90000000000, "upper_bound": "3178238155"}, {"distinct_range": 500, "num_eq": 3202757428388789741, "num_range": 500, "upper_bound": "3294373020"}, {"distinct_range": 0, "num_eq": 9000, "num_range": 0, "upper_bound": "3866334592"}, {"distinct_range": 0, "num_eq": 1918344369033070348, "num_range": 10000000, "upper_bound": "4006862892"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 2539378467702650715}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 895923895242085132, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000, "num_range": 0, "upper_bound": "0000000"}, {"distinct_range": 0, "num_eq": 400000, "num_range": 9013369235591614150, "upper_bound": "0000001"}, {"distinct_range": 2069625048794358000, "num_eq": 9000000, "num_range": 2718913199745497779, "upper_bound": "0011000"}, {"distinct_range": 400000, "num_eq": 80000, "num_range": 400000, "upper_bound": "0111101"}, {"distinct_range": 7888888795416144000, "num_eq": 10000000, "num_range": 7888888795416144375, "upper_bound": "1001111"}], "histo_col_type": "BIT(7)", "name": "__auto__", "null_count": 1067585788019066798, "row_count": 2539378467702650715}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1482276179290852567, "histo_buckets": [{"distinct_range": 0, "num_eq": 446646193095396673, "num_range": 0, "upper_bound": "false"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 2539378467702650715}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2623, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 1330386790376885198, "num_range": 50000000000, "upper_bound": "177366926"}, {"distinct_range": 60000000000, "num_eq": 10000000, "num_range": 60000000000, "upper_bound": "1103881301"}, {"distinct_range": 2402147584219889000, "num_eq": 7193959799379762915, "num_range": 2402147584219889296, "upper_bound": "3168311255"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2689, "histo_buckets": [{"distinct_range": 0, "num_eq": 7330422990051937524, "num_range": 0, "upper_bound": "-18907496.01819838714"}, {"distinct_range": 5365677538202906000, "num_eq": 3988491079047698237, "num_range": 5365677538202905594, "upper_bound": "3984.234077389069934"}, {"distinct_range": 90000, "num_eq": 800, "num_range": 90000, "upper_bound": "369914.479851222965"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 3556, "row_count": 4000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2641, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000000, "num_range": 0, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3199, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-2.121562536046462"}, {"distinct_range": 4797134209203376000, "num_eq": 70000, "num_range": 5444028053582033500, "upper_bound": "-1.6589461106799672"}, {"distinct_range": 1718971602922336800, "num_eq": 4000, "num_range": 1718971602922336699, "upper_bound": "-0.6018487898843559"}, {"distinct_range": 0, "num_eq": 600, "num_range": 60690177968515768, "upper_bound": "-0.36878312597507"}, {"distinct_range": 20000000000, "num_eq": 7517083364977852405, "num_range": 20000000000, "upper_bound": "-0.08469448662172041"}, {"distinct_range": 0, "num_eq": 5495945236397329800, "num_range": 6475345048987354314, "upper_bound": "-0.0724584472650136"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 336559654460320167, "upper_bound": "1.0386514704903174"}, {"distinct_range": 0, "num_eq": 10, "num_range": 50000000000, "upper_bound": "1.6269963801037886"}, {"distinct_range": 2307403162807891500, "num_eq": 8970662075734062813, "num_range": 2918838125108358017, "upper_bound": "2.0198470354496862"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 1064, "row_count": 4000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 977, "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3363, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "171482596"}, {"distinct_range": 20000, "num_eq": 6019663426028902000, "num_range": 20000, "upper_bound": "183979789"}, {"distinct_range": 7998222566703671000, "num_eq": 2464975621596773035, "num_range": 9161539505473817768, "upper_bound": "387167736"}, {"distinct_range": 581717.2033718565, "num_eq": 0, "num_range": 1000000, "upper_bound": "696650560"}, {"distinct_range": 70, "num_eq": 80000000, "num_range": 70, "upper_bound": "845640524"}, {"distinct_range": 6412337074628484000, "num_eq": 4000000000, "num_range": 6412337074628484264, "upper_bound": "1759275638"}, {"distinct_range": 2946969417.059811, "num_eq": 30000000000, "num_range": 6000000000, "upper_bound": "2238702158"}, {"distinct_range": 272416454.7396217, "num_eq": 10000000, "num_range": 700000000, "upper_bound": "2404720478"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 1454, "row_count": 4000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1799, "histo_col_type": "", "name": "__auto__", "null_count": 216, "row_count": 4000}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3747, "histo_col_type": "", "name": "__auto__", "null_count": 2284, "row_count": 4000}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 546, "histo_buckets": [{"distinct_range": 0, "num_eq": 460306972546451248, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 1000, "num_eq": 600000000, "num_range": 1000, "upper_bound": "94165833"}, {"distinct_range": 0, "num_eq": 4864414393661861724, "num_range": 4226398291648203804, "upper_bound": "1289478033"}, {"distinct_range": 0, "num_eq": 3058904146587858586, "num_range": 1133351689510108886, "upper_bound": "2189387097"}, {"distinct_range": 10, "num_eq": 1817834878597079557, "num_range": 10, "upper_bound": "2429284700"}, {"distinct_range": 6652715468185480000, "num_eq": 60000, "num_range": 6652715468185479849, "upper_bound": "3040019541"}, {"distinct_range": 2303110458625326300, "num_eq": 60, "num_range": 3882536654108233728, "upper_bound": "3427767953"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1657, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 451207508008754250, "histo_buckets": [{"distinct_range": 0, "num_eq": 3698915680893861786, "num_range": 0, "upper_bound": "12:53:54.897069"}, {"distinct_range": 0, "num_eq": 2309775976110220013, "num_range": 141379146961134865, "upper_bound": "17:04:53.923704"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 7774949610566230535}, {"columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5099103312520068731, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": "0000101101111111001101110"}, {"distinct_range": 100000000, "num_eq": 100000000, "num_range": 100000000, "upper_bound": "0000101111110110010010100"}, {"distinct_range": 6990750843241937000, "num_eq": 8791528684068487199, "num_range": 7236373829120939945, "upper_bound": "0100111001000001001010010"}, {"distinct_range": 772.8110263160183, "num_eq": 3756351112885594257, "num_range": 1000, "upper_bound": "0100111011010000000110100"}, {"distinct_range": 400000000, "num_eq": 2000000000, "num_range": 400000000, "upper_bound": "0100111100100100010011110"}, {"distinct_range": 0, "num_eq": 30000, "num_range": 4000000, "upper_bound": "0101011011001111110010001"}, {"distinct_range": 582474402854067000, "num_eq": 1000, "num_range": 6534680294682362206, "upper_bound": "1011001011101100101110100"}, {"distinct_range": 2434325686936625000, "num_eq": 80000, "num_range": 3454012773991772150, "upper_bound": "1011110101100111101011111"}, {"distinct_range": 60000, "num_eq": 7413358749721380663, "num_range": 60000, "upper_bound": "1110111100001111111111011"}], "histo_col_type": "BIT(25)", "name": "__auto__", "null_count": 0, "row_count": 7774949610566230535}, {"columns": ["col5_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7275469438754121843, "histo_buckets": [{"distinct_range": 0, "num_eq": 8082064744262123069, "num_range": 0, "upper_bound": "-1.3454491247844436"}, {"distinct_range": 1342218488.1156573, "num_eq": 6000000000, "num_range": 30000000000, "upper_bound": "-0.8238004088887743"}, {"distinct_range": 4000000000, "num_eq": 8000000000, "num_range": 4000000000, "upper_bound": "-0.3257758644790628"}, {"distinct_range": 12.341218878886226, "num_eq": 200000, "num_range": 50, "upper_bound": "-0.028879468774996114"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 7000000000, "upper_bound": "0.6401679413806521"}, {"distinct_range": 0, "num_eq": 8833635475254852737, "num_range": 0, "upper_bound": "1.713633226086611"}, {"distinct_range": 1843.4544504477753, "num_eq": 50000000000, "num_range": 5000, "upper_bound": "1.7976931348623157e+308"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 3779449531264862285, "row_count": 7774949610566230535}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 148542034083766429, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 0, "num_range": 4283789541956834218, "upper_bound": "984857051"}, {"distinct_range": 2599871882798009000, "num_eq": 100000000, "num_range": 2599871882798009010, "upper_bound": "1632299236"}, {"distinct_range": 6253580720357852000, "num_eq": 40, "num_range": 6253580720357852368, "upper_bound": "2194429272"}, {"distinct_range": 100000, "num_eq": 9000, "num_range": 100000, "upper_bound": "2752937514"}, {"distinct_range": 3152433054954321000, "num_eq": 1667887124762448731, "num_range": 3858144985446638906, "upper_bound": "3248239675"}, {"distinct_range": 0, "num_eq": 3837735995852692864, "num_range": 1099453097968859720, "upper_bound": "3552279158"}, {"distinct_range": 6125325302295630000, "num_eq": 1687245893167783482, "num_range": 6732409082404318931, "upper_bound": "3976242010"}, {"distinct_range": 0, "num_eq": 4228608433632315686, "num_range": 7228899199596156695, "upper_bound": "4000488434"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 7774949610566230535}]':::JSONB;
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_2, col1_1) REFERENCES table4 (col4_0, col4_4);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('uazszw', 'g', 'kyj', 'gz', 'zpowng', 'r');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('yxs', 'pu', 'iotcx', 'b');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('shw', 'vl', 'i');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36101, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 75380, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 52011, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13608, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 40199, "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44708, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "J\\u00013{q,"}, {"distinct_range": 7773734978301530000, "num_eq": 80000, "num_range": 7773734978301529836, "upper_bound": "l,O\\u0014<"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 73004, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28739, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 10, "num_eq": 1000000, "num_range": 10, "upper_bound": "86938616"}, {"distinct_range": 0, "num_eq": 900000000, "num_range": 0, "upper_bound": "518529307"}, {"distinct_range": 2889887144016567300, "num_eq": 6678540435668667657, "num_range": 3032017410159316521, "upper_bound": "546513703"}, {"distinct_range": 60000000000, "num_eq": 0, "num_range": 60000000000, "upper_bound": "566032696"}, {"distinct_range": 0, "num_eq": 4699372182447202490, "num_range": 0, "upper_bound": "864397306"}, {"distinct_range": 0, "num_eq": 80, "num_range": 50000, "upper_bound": "2422842260"}, {"distinct_range": 0, "num_eq": 486433597222501309, "num_range": 0, "upper_bound": "3674523162"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6307, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21974, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20923, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35078, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 222, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "0111001001101101000000111010001"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 90, "upper_bound": "1100000110111111101011010000111"}, {"distinct_range": 300000000, "num_eq": 7053409190738827641, "num_range": 300000000, "upper_bound": "1110110111000101000100000000100"}], "histo_col_type": "BIT(31)", "name": "__auto__", "null_count": 2322, "row_count": 4000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3146, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "0111111"}, {"distinct_range": 0, "num_eq": 200000, "num_range": 200, "upper_bound": "1110001"}, {"distinct_range": 0, "num_eq": 300000, "num_range": 2795761541840799716, "upper_bound": "1110110"}], "histo_col_type": "BIT(7)", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3477, "histo_buckets": [{"distinct_range": 0, "num_eq": 2137767909348029360, "num_range": 0, "upper_bound": "-32768"}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 2000000000, "upper_bound": "-32758"}, {"distinct_range": 50000000000, "num_eq": 100000000, "num_range": 50000000000, "upper_bound": "-17589"}, {"distinct_range": 287965511925192450, "num_eq": 1754992609066980419, "num_range": 7560316343642998502, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 6930950095079522825, "num_range": 0, "upper_bound": "1"}, {"distinct_range": 3525262842783732700, "num_eq": 100000, "num_range": 3525262842783732657, "upper_bound": "23481"}, {"distinct_range": 800000000, "num_eq": 90000, "num_range": 800000000, "upper_bound": "25345"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 791, "row_count": 4000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2820, "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4620, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000, "num_range": 0, "upper_bound": "0001110"}, {"distinct_range": 8000, "num_eq": 5019171259545617110, "num_range": 8000, "upper_bound": "0010010"}, {"distinct_range": 0, "num_eq": 10, "num_range": 100000000000, "upper_bound": "0110010"}, {"distinct_range": 3375528961873622500, "num_eq": 10000000, "num_range": 3375528961873622765, "upper_bound": "0110100"}, {"distinct_range": 0, "num_eq": 8688994833572762124, "num_range": 1104564353333447212, "upper_bound": "1100011"}, {"distinct_range": 0, "num_eq": 2338932968507067951, "num_range": 8344311676100795255, "upper_bound": "1110110"}], "histo_col_type": "BIT(7)", "name": "__auto__", "null_count": 2799, "row_count": 10000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2448, "histo_buckets": [{"distinct_range": 0, "num_eq": 5313113746985790826, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 1295.7835509958468, "num_eq": 6000000000, "num_range": 9000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 10000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 940, "histo_buckets": [{"distinct_range": 0, "num_eq": 9001656992947188686, "num_range": 0, "upper_bound": "-1.4771690660934589"}, {"distinct_range": 0, "num_eq": 400, "num_range": 10000000000, "upper_bound": "-0.5452584199716018"}, {"distinct_range": 633375.049685163, "num_eq": 100000, "num_range": 10000000, "upper_bound": "-0.40532899865509153"}, {"distinct_range": 0, "num_eq": 30000000, "num_range": 4757369450913871845, "upper_bound": "-0.35999292730788834"}, {"distinct_range": 0, "num_eq": 4434805192044687627, "num_range": 20000, "upper_bound": "-0.0766057537018685"}, {"distinct_range": 500, "num_eq": 60000, "num_range": 500, "upper_bound": "0.6418332324517975"}, {"distinct_range": 4706149553669813000, "num_eq": 900000, "num_range": 4706149553669813429, "upper_bound": "0.7808421462106957"}, {"distinct_range": 3797.7828790144495, "num_eq": 30000, "num_range": 4000, "upper_bound": "1.1638229292746014"}, {"distinct_range": 0, "num_eq": 2880309131157346351, "num_range": 300, "upper_bound": "+Inf"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 1478, "row_count": 6000}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1253, "histo_buckets": [{"distinct_range": 0, "num_eq": 660817036909051105, "num_range": 0, "upper_bound": "2737515983"}, {"distinct_range": 0, "num_eq": 3608314693586238026, "num_range": 70000000000, "upper_bound": "3186864253"}, {"distinct_range": 4963058696329515000, "num_eq": 60000, "num_range": 7583953631999558023, "upper_bound": "3453389614"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 1702, "row_count": 6000}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4534, "histo_buckets": [{"distinct_range": 0, "num_eq": 5019823956091057396, "num_range": 0, "upper_bound": "0"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2711, "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 4817, "row_count": 6000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4671, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 3755236508851213000, "num_eq": 8385153211774503482, "num_range": 3755236508851212801, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2674, "histo_col_type": "", "name": "__auto__", "null_count": 2379, "row_count": 6000}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1329, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 66150013550914930, "num_eq": 80000, "num_range": 236104568948652591, "upper_bound": "671935838"}, {"distinct_range": 487.88633535332735, "num_eq": 8295695097341204054, "num_range": 800, "upper_bound": "3626435729"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1941, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4701, "histo_buckets": [{"distinct_range": 0, "num_eq": 7182443631228753911, "num_range": 0, "upper_bound": "10:39:30.491425+02:09:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 446, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000, "num_range": 0, "upper_bound": "1363390730"}, {"distinct_range": 0, "num_eq": 5514558250728949861, "num_range": 1627611852564637787, "upper_bound": "2368944005"}, {"distinct_range": 0, "num_eq": 2618691863178395664, "num_range": 8362864972457894888, "upper_bound": "3085027024"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 1601, "row_count": 6000}]';
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 79984, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000000, "num_range": 0, "upper_bound": "0000000000000000000000000"}, {"distinct_range": 441299353256740350, "num_eq": 276736531707851080, "num_range": 3535614430811445495, "upper_bound": "0001101000000000110000100"}, {"distinct_range": 0, "num_eq": 50000, "num_range": 4497054329500454353, "upper_bound": "0011100001111010100100110"}, {"distinct_range": 0, "num_eq": 4301526908891793237, "num_range": 3208742309355720482, "upper_bound": "0111111111111111111111111"}, {"distinct_range": 871536.0573354227, "num_eq": 7822093107090393033, "num_range": 3000000, "upper_bound": "1010011111100100011100011"}, {"distinct_range": 600000000, "num_eq": 6541285773756293860, "num_range": 600000000, "upper_bound": "1101100101000001000011111"}, {"distinct_range": 7681624486366618000, "num_eq": 402536323911234444, "num_range": 7681624486366617945, "upper_bound": "1110111001000101001001001"}], "histo_col_type": "BIT(25)", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col5_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 511649, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "-1.1096870257848914"}, {"distinct_range": 265261653.93943846, "num_eq": 4823434009439571604, "num_range": 700000000, "upper_bound": "-0.9536631752487941"}, {"distinct_range": 9171055730438913000, "num_eq": 5115954259655833357, "num_range": 9171055730438912551, "upper_bound": "-0.8866280339787922"}, {"distinct_range": 30, "num_eq": 3012875334747911792, "num_range": 30, "upper_bound": "-0.056064639100950164"}, {"distinct_range": 6087927429060388000, "num_eq": 20000000, "num_range": 6087927429060388134, "upper_bound": "0.1533541459734833"}, {"distinct_range": 5000000, "num_eq": 50000, "num_range": 5000000, "upper_bound": "1.0099810579134143"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 2000000000, "upper_bound": "1.1112309122338553"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 562839, "row_count": 700000}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 556905, "histo_buckets": [{"distinct_range": 0, "num_eq": 885057102853188740, "num_range": 0, "upper_bound": "652760580"}, {"distinct_range": 4123239682631217700, "num_eq": 1469820865190370809, "num_range": 8124220961508224258, "upper_bound": "658466430"}, {"distinct_range": 0, "num_eq": 2942536144125506710, "num_range": 10000000000, "upper_bound": "720591652"}, {"distinct_range": 300000000, "num_eq": 400000, "num_range": 300000000, "upper_bound": "1260576833"}, {"distinct_range": 0, "num_eq": 50000000, "num_range": 2448763081557986239, "upper_bound": "1442296659"}, {"distinct_range": 7833484025445553000, "num_eq": 90000, "num_range": 7833484025445553423, "upper_bound": "3092722661"}, {"distinct_range": 2903836974787915000, "num_eq": 10000000, "num_range": 2903836974787914572, "upper_bound": "3590638186"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 673892, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000000, "num_range": 0, "upper_bound": "12:11:33.571276"}, {"distinct_range": 60, "num_eq": 40000000000, "num_range": 60, "upper_bound": "13:28:44.87902"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 8833526057414708633, "upper_bound": "19:28:19.8169"}, {"distinct_range": 8000000000, "num_eq": 9000000, "num_range": 8000000000, "upper_bound": "20:23:23.438819"}, {"distinct_range": 675288297410651500, "num_eq": 10000000, "num_range": 675288297410651574, "upper_bound": "21:29:01.104492"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 700000}]';
        test_1        |             ALTER TABLE table4 ADD FOREIGN KEY (col4_6) REFERENCES table5 (col5_1) ON UPDATE CASCADE;
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9156406885, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 413360253, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1403393078, "histo_buckets": [{"distinct_range": 0, "num_eq": 2682071548101312344, "num_range": 0, "upper_bound": "\\u0007\\u0019i[X"}, {"distinct_range": 400000000, "num_eq": 600000, "num_range": 400000000, "upper_bound": "\\u000c}\\u001b(x\x7fN"}, {"distinct_range": 25.186936282864487, "num_eq": 70000000000, "num_range": 300, "upper_bound": "\'"}, {"distinct_range": 41.50271779308392, "num_eq": 8000000, "num_range": 100, "upper_bound": "4e#\\u0007GyX^q"}, {"distinct_range": 0, "num_eq": 3573525144876021655, "num_range": 20000000, "upper_bound": "7\\u001cP\\u001e]%"}, {"distinct_range": 5290897148592187000, "num_eq": 6625581623842803012, "num_range": 5916695844958452962, "upper_bound": "@\\u0006k\\u00075C"}, {"distinct_range": 1092187.0440078843, "num_eq": 60000, "num_range": 10000000, "upper_bound": "GC>@6"}, {"distinct_range": 3375780252346551000, "num_eq": 1000000000, "num_range": 3375780252346550770, "upper_bound": "Gq\\"\\u001dt\\""}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6869449838, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6532863967, "histo_buckets": [{"distinct_range": 0, "num_eq": 8090375998426544410, "num_range": 0, "upper_bound": "724708998"}, {"distinct_range": 2337509757367242000, "num_eq": 2419206205228314415, "num_range": 2337509757367242202, "upper_bound": "800875695"}, {"distinct_range": 0, "num_eq": 8749073033064443686, "num_range": 5152709494061414724, "upper_bound": "1240097541"}, {"distinct_range": 250.768084672165, "num_eq": 200000000, "num_range": 1000, "upper_bound": "2308728706"}, {"distinct_range": 37.4214006126748, "num_eq": 3704119375619766671, "num_range": 100, "upper_bound": "3976667977"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9798579427, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8507585389, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30887723, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2447940162, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9943349082, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7267577716, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6219335796, "histo_buckets": [{"distinct_range": 0, "num_eq": 4196117690341770835, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 900, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 10000000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 912574029391484793, "histo_buckets": [{"distinct_range": 0, "num_eq": 9010320922957802288, "num_range": 0, "upper_bound": "0001011101111100100010111100011"}, {"distinct_range": 0, "num_eq": 4314052601709444683, "num_range": 700, "upper_bound": "0010010100101111101101000001100"}, {"distinct_range": 90866435790505440, "num_eq": 5595727516792739265, "num_range": 655015750264922320, "upper_bound": "0011101010111100010000011110010"}, {"distinct_range": 160.45034824692306, "num_eq": 1185730965112865959, "num_range": 200, "upper_bound": "0011110011011100100000110010110"}, {"distinct_range": 7116941146861866000, "num_eq": 3423061348556108331, "num_range": 7116941146861866124, "upper_bound": "0100010101101010101011010111101"}, {"distinct_range": 2373826534790987300, "num_eq": 8099675564368678762, "num_range": 2373826534790987362, "upper_bound": "1001010011100011111010011010010"}, {"distinct_range": 0, "num_eq": 4757628938794171984, "num_range": 800, "upper_bound": "1001111000110011101001011111100"}, {"distinct_range": 0, "num_eq": 3417621951055099364, "num_range": 70000000000, "upper_bound": "1011010001011001011101011111010"}], "histo_col_type": "BIT(31)", "name": "__auto__", "null_count": 522184041449267068, "row_count": 5174149671657777323}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2570980787352589016, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000000, "num_range": 0, "upper_bound": "0010100"}, {"distinct_range": 0, "num_eq": 80, "num_range": 1000000, "upper_bound": "0011110"}, {"distinct_range": 3485922236150311400, "num_eq": 900000000, "num_range": 6262858040106281129, "upper_bound": "0101010"}, {"distinct_range": 4670286064655728, "num_eq": 4850891388358303371, "num_range": 8233740130019738, "upper_bound": "0110110"}, {"distinct_range": 10000000, "num_eq": 10000, "num_range": 10000000, "upper_bound": "1011000"}, {"distinct_range": 0, "num_eq": 400, "num_range": 800000, "upper_bound": "1101101"}, {"distinct_range": 0, "num_eq": 4330896616394691796, "num_range": 7833775194818155629, "upper_bound": "1110011"}, {"distinct_range": 171143.57421939904, "num_eq": 3000000, "num_range": 300000, "upper_bound": "1110110"}, {"distinct_range": 4513016138909488000, "num_eq": 5708235333887620219, "num_range": 8347830112734686730, "upper_bound": "1111100"}], "histo_col_type": "BIT(7)", "name": "__auto__", "null_count": 0, "row_count": 5174149671657777323}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4354095196725504707, "histo_buckets": [{"distinct_range": 0, "num_eq": 3186722951035049448, "num_range": 0, "upper_bound": "-12496"}, {"distinct_range": 0, "num_eq": 3337513778307379281, "num_range": 0, "upper_bound": "-4512"}, {"distinct_range": 0, "num_eq": 740172422362889817, "num_range": 7422248113913440706, "upper_bound": "13606"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 2231944954962103338, "row_count": 5174149671657777323}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 626217722, "histo_buckets": [{"distinct_range": 0, "num_eq": 2298088452769095200, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 50, "num_eq": 0, "num_range": 50, "upper_bound": "327911472"}, {"distinct_range": 226611767721716800, "num_eq": 8887022346961740436, "num_range": 649253511967339149, "upper_bound": "826037691"}, {"distinct_range": 9213356677165158000, "num_eq": 1000000000, "num_range": 9213356677165158074, "upper_bound": "927484925"}, {"distinct_range": 2594598569.2854843, "num_eq": 1988302145556156844, "num_range": 4000000000, "upper_bound": "2285434997"}, {"distinct_range": 0, "num_eq": 7593216956135100046, "num_range": 0, "upper_bound": "3881959945"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 700000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 632768118, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000, "num_range": 0, "upper_bound": "0011110"}, {"distinct_range": 3048238756605496300, "num_eq": 0, "num_range": 3153970749417644129, "upper_bound": "0100000"}, {"distinct_range": 28031365803.652317, "num_eq": 6000000000, "num_range": 70000000000, "upper_bound": "0111111"}, {"distinct_range": 398302860.96843565, "num_eq": 700000000, "num_range": 800000000, "upper_bound": "1100111"}, {"distinct_range": 100000000, "num_eq": 60000000000, "num_range": 100000000, "upper_bound": "1111110"}], "histo_col_type": "BIT(7)", "name": "__auto__", "null_count": 402313278, "row_count": 700000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 358731464, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 700000000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6362705949780370456, "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 475000011850423894, "row_count": 8460627191250791731}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2749329307358138028, "histo_buckets": [{"distinct_range": 0, "num_eq": 1916840979816610944, "num_range": 0, "upper_bound": "75080258"}, {"distinct_range": 0, "num_eq": 2000000000, "num_range": 30000000, "upper_bound": "157694266"}, {"distinct_range": 2336160838262347300, "num_eq": 4416769744595638414, "num_range": 2336160838262347106, "upper_bound": "1241744830"}, {"distinct_range": 216.72042351774937, "num_eq": 967357462627050695, "num_range": 500, "upper_bound": "2131234458"}, {"distinct_range": 6867305549045017000, "num_eq": 3457453080589313880, "num_range": 8177640117263597516, "upper_bound": "2399733963"}, {"distinct_range": 211.32091585819984, "num_eq": 50, "num_range": 300, "upper_bound": "3810489559"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 8460627191250791731}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2237572956949488859, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000, "num_range": 0, "upper_bound": "397893445"}, {"distinct_range": 2882044080057278000, "num_eq": 520471216765487320, "num_range": 2882044080057278005, "upper_bound": "1605086340"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 8460627191250791731}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3106309457598605000, "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 8460627191250791731}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8215506047241136148, "histo_col_type": "", "name": "__auto__", "null_count": 5541327403441811919, "row_count": 8460627191250791731}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2877381732737702549, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8460627191250791731}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4021789878365395504, "histo_buckets": [{"distinct_range": 0, "num_eq": 8373128649991951585, "num_range": 0, "upper_bound": "-6.263642463215615397E+24"}, {"distinct_range": 8068457412798413000, "num_eq": 193280127765916636, "num_range": 8068457412798412786, "upper_bound": "-3.3427023385771118E+24"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 900000, "upper_bound": "-3.014782690988898688E+23"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "-460937450128.1509918"}, {"distinct_range": 40.33679077326177, "num_eq": 200000, "num_range": 20000, "upper_bound": "-8535697.697170051547"}, {"distinct_range": 0, "num_eq": 1247985288438210127, "num_range": 6424491175521959184, "upper_bound": "-12918.79515825341166"}, {"distinct_range": 3247721888624453000, "num_eq": 3000000000, "num_range": 8905782865615701116, "upper_bound": "1.883903429035456039E+24"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 1173644109811297917, "row_count": 8460627191250791731}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5314681667158050497, "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 607801482805907409, "row_count": 8460627191250791731}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5591213861874048937, "histo_buckets": [{"distinct_range": 0, "num_eq": 2638443400126486383, "num_range": 0, "upper_bound": "21:25:54.054432+03:00:00"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 1469611918600613484, "upper_bound": "14:39:40.679698-09:00:00"}, {"distinct_range": 80000000, "num_eq": 500000000, "num_range": 80000000, "upper_bound": "20:07:56.713531-04:13:00"}, {"distinct_range": 6213470699544179000, "num_eq": 40000, "num_range": 6213470699544178854, "upper_bound": "19:52:56.669624-06:35:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 8460627191250791731}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 63905859296725274, "histo_buckets": [{"distinct_range": 0, "num_eq": 2016488001161908358, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 1925512502587587600, "num_eq": 6067600881415905909, "num_range": 3668808422123128800, "upper_bound": "411796073"}, {"distinct_range": 900000, "num_eq": 6772775544757923768, "num_range": 900000, "upper_bound": "2053046205"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 1942906771686946569, "row_count": 8460627191250791731}]';
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5416119795756879559, "histo_buckets": [{"distinct_range": 0, "num_eq": 6724367292301856507, "num_range": 0, "upper_bound": "894651783"}, {"distinct_range": 2675910503826844000, "num_eq": 1000, "num_range": 9049812559009479002, "upper_bound": "3402514237"}, {"distinct_range": 70000000000, "num_eq": 900, "num_range": 70000000000, "upper_bound": "3859401580"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 6262414236677276009}, {"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5940356400157328080, "histo_buckets": [{"distinct_range": 0, "num_eq": 5373390499808848696, "num_range": 0, "upper_bound": "00:00:00"}, {"distinct_range": 3193649.4619734157, "num_eq": 8963316453450970387, "num_range": 7000000, "upper_bound": "03:47:59.286937"}, {"distinct_range": 0, "num_eq": 0, "num_range": 763007533837654397, "upper_bound": "08:05:53.674239"}, {"distinct_range": 70000, "num_eq": 2000000000, "num_range": 70000, "upper_bound": "16:39:57.479485"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 7482357797519886724, "upper_bound": "17:07:13.172483"}, {"distinct_range": 4708.395157465412, "num_eq": 800000000, "num_range": 70000, "upper_bound": "17:55:46.318523"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 6262414236677276009}, {"columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 595269540305067246, "histo_buckets": [{"distinct_range": 0, "num_eq": 80, "num_range": 0, "upper_bound": "0001101001000111101001100"}, {"distinct_range": 0, "num_eq": 90, "num_range": 3190782928734457079, "upper_bound": "0100000000101011101010000"}, {"distinct_range": 0, "num_eq": 631773512438622663, "num_range": 0, "upper_bound": "0111001111101000001110001"}, {"distinct_range": 848994287886353500, "num_eq": 398506457849573970, "num_range": 3420880161121836297, "upper_bound": "0111111111111111111111111"}, {"distinct_range": 0, "num_eq": 200000000, "num_range": 0, "upper_bound": "1000010001101011110011110"}, {"distinct_range": 3109755500905273000, "num_eq": 6000000000, "num_range": 4991034101689701600, "upper_bound": "1000110110011000100110000"}, {"distinct_range": 0, "num_eq": 40000000000, "num_range": 2000, "upper_bound": "1001010111001011000000101"}, {"distinct_range": 2583.8303080203345, "num_eq": 800, "num_range": 6000, "upper_bound": "1101011011111100000011110"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 3459792439390294712, "upper_bound": "1111101000010011010000000"}], "histo_col_type": "BIT(25)", "name": "__auto__", "null_count": 0, "row_count": 6262414236677276009}, {"columns": ["col5_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3633103614597509643, "histo_buckets": [{"distinct_range": 0, "num_eq": 2539875277081328838, "num_range": 0, "upper_bound": "-2.2367815438768024"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 9000000000, "upper_bound": "-1.5563446939584122"}, {"distinct_range": 0, "num_eq": 9000000000, "num_range": 6000000000, "upper_bound": "-1.3715035246871146"}, {"distinct_range": 3689864616849510400, "num_eq": 8931392367078463667, "num_range": 3689864616849510638, "upper_bound": "-0.06806953400131421"}, {"distinct_range": 5268.228610442103, "num_eq": 2000560357390821682, "num_range": 20000, "upper_bound": "0.031046871282851884"}, {"distinct_range": 6069348421894270000, "num_eq": 0, "num_range": 6069348421894269849, "upper_bound": "0.04879499230695916"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 6683699872757327301, "upper_bound": "0.7586359145036692"}, {"distinct_range": 100000000, "num_eq": 8374254394444679905, "num_range": 100000000, "upper_bound": "0.9691739372469023"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 1550065718482988205, "row_count": 6262414236677276009}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "nan" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2604752&tab=buildLog) on [master@baf34b241c9ee27530f0a3a988149bfd7cca3f6d](https://github.com/cockroachdb/cockroach/commits/baf34b241c9ee27530f0a3a988149bfd7cca3f6d):

```
        postgres_1    | You can change this by editing pg_hba.conf or using the option -A, or
        postgres_1    | --auth-local and --auth-host, the next time you run initdb.
        postgres_1    | waiting for server to start....2021-01-22 06:21:32.243 UTC [49] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-01-22 06:21:32.260 UTC [50] LOG:  database system was shut down at 2021-01-22 06:21:32 UTC
        postgres_1    | 2021-01-22 06:21:32.265 UTC [49] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down....2021-01-22 06:21:35.195 UTC [49] LOG:  received fast shutdown request
        postgres_1    | 2021-01-22 06:21:35.197 UTC [49] LOG:  aborting any active transactions
        postgres_1    | 2021-01-22 06:21:35.199 UTC [49] LOG:  background worker "logical replication launcher" (PID 56) exited with exit code 1
        postgres_1    | 2021-01-22 06:21:35.204 UTC [51] LOG:  shutting down
        postgres_1    | 2021-01-22 06:21:35.316 UTC [49] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-01-22 06:21:35.406 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-01-22 06:21:35.406 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-01-22 06:21:35.410 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-01-22 06:21:35.425 UTC [86] LOG:  database system was shut down at 2021-01-22 06:21:35 UTC
        postgres_1    | 2021-01-22 06:21:35.431 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (603.09s)
        test_1        |     --- FAIL: TestCompare/postgres (2.35s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2607611&tab=buildLog) on [master@7a5d3d0f9465c4d6a6f204ac6a672a027cf585a6](https://github.com/cockroachdb/cockroach/commits/7a5d3d0f9465c4d6a6f204ac6a672a027cf585a6):

```
        postgres_1    | 2021-01-23 06:20:55.631 UTC [51] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-01-23 06:20:55.735 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-01-23 06:20:55.735 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-01-23 06:20:55.739 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-01-23 06:20:55.753 UTC [88] LOG:  database system was shut down at 2021-01-23 06:20:55 UTC
        postgres_1    | 2021-01-23 06:20:55.759 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (601.87s)
        test_1        |     --- FAIL: TestCompare/postgres (1.29s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 VARCHAR NOT NULL, col1_1 REGCLASS NOT NULL, col1_2 VARCHAR NOT NULL, col1_3 NAME NULL, col1_4 REGCLASS NOT NULL, col1_5 REGPROCEDURE NOT NULL, col1_6 REGPROCEDURE NOT NULL, col1_7 VARCHAR NOT NULL, col1_8 FLOAT4 NOT NULL, col1_9 REGNAMESPACE NOT NULL, col1_10 REGNAMESPACE NOT NULL, col1_11 REGPROC NOT NULL, col1_12 BOOL NOT NULL, col1_13 VARCHAR NOT NULL, col1_14 INT2, col1_15 BYTES NULL, col1_16 VARCHAR NOT NULL, col1_17 OID, PRIMARY KEY (col1_1, col1_4, col1_16, col1_6, col1_7, col1_13, col1_10, col1_5, col1_2), FAMILY (col1_9), FAMILY (col1_15), FAMILY (col1_11), FAMILY (col1_17), FAMILY (col1_16), FAMILY (col1_12, col1_14, col1_2), FAMILY (col1_3, col1_13, col1_8), FAMILY (col1_1), FAMILY (col1_5, col1_7, col1_0), FAMILY (col1_4), FAMILY (col1_10), FAMILY (col1_6));
        test_1        |             CREATE INDEX ON table1 (col1_8 ASC, col1_7 ASC, col1_17 ASC, col1_11 ASC, col1_0 ASC, col1_4 ASC, col1_1 ASC, col1_12 ASC, col1_5, col1_9 DESC, col1_16 DESC, col1_13, col1_6 ASC, col1_14, col1_3 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_0);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_9 DESC, col1_6 ASC);
        test_1        |             CREATE INDEX ON table1 (col1_4 DESC, col1_6 DESC, col1_15, col1_16 ASC, col1_8 ASC, col1_11, col1_2 ASC, col1_12 ASC, col1_17);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_15, col1_0 DESC, col1_12 ASC, col1_7 ASC, col1_1, col1_11, col1_6, col1_14 DESC, col1_8 ASC, col1_5 ASC, col1_16 DESC, col1_10);
        test_1        |             CREATE INDEX ON table1 (col1_10, col1_11 ASC, col1_16 ASC, col1_6 DESC, col1_14, col1_7 DESC, col1_3 ASC, col1_0, col1_8 ASC, col1_15, col1_13 DESC, col1_12 ASC) STORING (col1_9);
        test_1        |             CREATE INDEX ON table1 (col1_4, col1_15 ASC, col1_11 ASC, col1_6, col1_13, col1_17 DESC, col1_1 ASC, col1_7 ASC, col1_14 ASC, col1_12, col1_10 DESC, col1_9) STORING (col1_0, col1_3);
        test_1        |             CREATE INDEX ON table1 (col1_1 DESC, col1_17);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_13 ASC, col1_9 ASC, col1_1 ASC, col1_17, col1_11 ASC, col1_7 DESC, col1_14 DESC, col1_3 DESC, col1_6, col1_12 DESC);
        test_1        |             CREATE TABLE table2 (col1_1 REGCLASS NOT NULL, col1_4 REGCLASS NOT NULL, col1_16 VARCHAR NOT NULL, col1_6 REGPROCEDURE NOT NULL, col1_7 VARCHAR NOT NULL, col1_13 VARCHAR NOT NULL, col1_10 REGNAMESPACE NOT NULL, col1_5 REGPROCEDURE NOT NULL, col1_2 VARCHAR NOT NULL, col2_9 UUID NOT NULL, col2_10 INT8, col2_11 REGPROC NOT NULL, col2_12 UUID NULL, col2_13 BOOL, col2_14 TIMESTAMPTZ NULL, col2_15 INT4 NULL, PRIMARY KEY (col1_1, col1_4, col1_16, col1_6, col1_7, col1_13, col1_10, col1_5, col1_2, col2_11, col2_14), FAMILY (col1_16, col2_14, col1_6, col2_10, col2_11, col1_13), FAMILY (col1_5), FAMILY (col1_7), FAMILY (col1_4), FAMILY (col1_1), FAMILY (col1_10), FAMILY (col2_12, col2_9), FAMILY (col1_2), FAMILY (col2_13), FAMILY (col2_15)) INTERLEAVE IN PARENT table1 (col1_1, col1_4, col1_16, col1_6, col1_7, col1_13, col1_10, col1_5, col1_2);
        test_1        |             CREATE INDEX ON table2 (col1_2 DESC, col2_9, col2_14 DESC, col2_13 ASC, col1_4 DESC, col2_15 DESC, col2_11 DESC, col1_5 ASC, col1_16 ASC, col2_10, col1_7 ASC, col1_1 ASC, col2_12 ASC, col1_13 ASC);
        test_1        |             CREATE INDEX ON table2 (col1_10 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_9 DESC, col2_10 DESC, col1_6 DESC, col1_2, col1_16 ASC, col2_15 DESC, col2_14) STORING (col2_13);
        test_1        |             CREATE INDEX ON table2 (col2_11, col1_1 DESC, col1_5, col2_12 DESC, col1_4, col2_10 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_6 ASC, col2_14 ASC, col2_13, col2_11, col1_10 DESC, col1_7 DESC, col2_9 DESC, col1_4, col1_1 DESC, col1_16) STORING (col2_10, col2_12);
        test_1        |             CREATE INDEX ON table2 (col1_7, col2_13 ASC) STORING (col2_10, col2_12);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1607452211132409856, "histo_buckets": [{"distinct_range": 0, "num_eq": 4480105964240133784, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 0, "num_eq": 1445094456964676281, "num_range": 1906352368407741439, "upper_bound": "\\u0016"}, {"distinct_range": 0, "num_eq": 2236720162483652003, "num_range": 100, "upper_bound": "1,"}, {"distinct_range": 0, "num_eq": 4687672811241673323, "num_range": 600, "upper_bound": "f\\u0017s"}, {"distinct_range": 2000000000, "num_eq": 100000, "num_range": 2000000000, "upper_bound": "l&+Y!\\n>"}, {"distinct_range": 0, "num_eq": 3000000, "num_range": 8928850630761178286, "upper_bound": "n^\\u001d\\u001c_"}, {"distinct_range": 2000, "num_eq": 200, "num_range": 2000, "upper_bound": "\x7fe\\u000bV=gI"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 5599681562144882268}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2963587566654167526, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000000, "num_range": 0, "upper_bound": "-0.6637487411499023"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 5599681562144882268}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1516950032321172218, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5599681562144882268}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67568694994420972, "histo_buckets": [{"distinct_range": 0, "num_eq": 4203829694757635311, "num_range": 0, "upper_bound": "1371497519"}, {"distinct_range": 1000000000, "num_eq": 3778660584898975524, "num_range": 1000000000, "upper_bound": "1964768277"}, {"distinct_range": 0, "num_eq": 1792050023128381168, "num_range": 6000000, "upper_bound": "2620105120"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 800000000, "upper_bound": "3833059460"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 5599681562144882268}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 819039329236526902, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5599681562144882268}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5108712448398804087, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5599681562144882268}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2983342911473921981, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000000, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 3082688354932742827, "upper_bound": "\\\\x3d0d"}, {"distinct_range": 3926453294031220000, "num_eq": 4770598122071865406, "num_range": 3926453294031219993, "upper_bound": "\\\\x446e6f3ea8b0c9"}, {"distinct_range": 102564640313313120, "num_eq": 8000, "num_range": 102564640313313116, "upper_bound": "\\\\x4e7e6cc974c401cfba"}, {"distinct_range": 3231927081472785400, "num_eq": 3000, "num_range": 5226351328043760222, "upper_bound": "\\\\x793611926127a582"}, {"distinct_range": 0, "num_eq": 2842145782143603760, "num_range": 5207520791977522391, "upper_bound": "\\\\xae458f60bfb194a100"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 3374873906251883166, "row_count": 5599681562144882268}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3675405873106778748, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5599681562144882268}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5052039565832178068, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5599681562144882268}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2142382300275637492, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5599681562144882268}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 149149322693741572, "histo_buckets": [{"distinct_range": 0, "num_eq": 2114535679566886416, "num_range": 0, "upper_bound": "115329819"}, {"distinct_range": 10000, "num_eq": 3478689638297045509, "num_range": 10000, "upper_bound": "264357296"}, {"distinct_range": 7695656392624543000, "num_eq": 6314159536036418779, "num_range": 7695656392624542814, "upper_bound": "608951795"}, {"distinct_range": 16541.255062570977, "num_eq": 600000000, "num_range": 40000, "upper_bound": "1174912258"}, {"distinct_range": 0, "num_eq": 0, "num_range": 895809808892481097, "upper_bound": "1983189396"}, {"distinct_range": 2073241905691962600, "num_eq": 2634739699682758016, "num_range": 8684270579498478421, "upper_bound": "4196054237"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 5599681562144882268}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3687751345013050221, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5599681562144882268}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4474407310640100371, "histo_col_type": "", "name": "__auto__", "null_count": 4673944037850221462, "row_count": 5599681562144882268}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3518086695223638099, "histo_col_type": "", "name": "__auto__", "null_count": 4441471269777649104, "row_count": 5599681562144882268}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3835771820560452721, "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 5599681562144882268}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4787199445565276074, "histo_buckets": [{"distinct_range": 0, "num_eq": 7061834757414165874, "num_range": 0, "upper_bound": "525228469"}, {"distinct_range": 22425297.66843869, "num_eq": 6477250128330409602, "num_range": 60000000, "upper_bound": "3653372342"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 5599681562144882268}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2303148273782725389, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 1000, "num_eq": 4739737217471712219, "num_range": 1000, "upper_bound": "\\u0002["}, {"distinct_range": 60, "num_eq": 6953495612731270624, "num_range": 60, "upper_bound": "\\u0012R"}, {"distinct_range": 56810633949.999855, "num_eq": 7000, "num_range": 80000000000, "upper_bound": "6I\\u0008.\\u001bz"}, {"distinct_range": 146100.6335473097, "num_eq": 6113162946120737633, "num_range": 800000, "upper_bound": "8fQ(M\\u0019"}, {"distinct_range": 0, "num_eq": 7050127791818151928, "num_range": 7103195861704428453, "upper_bound": "B<\\u0014X\\u000f"}, {"distinct_range": 2901495605052126000, "num_eq": 9126998858044098941, "num_range": 2901495605052126381, "upper_bound": "X"}, {"distinct_range": 8472102369128263000, "num_eq": 404859599742116621, "num_range": 8472102369128262435, "upper_bound": "q\\u001f\'\\u000b"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 5599681562144882268}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2044041534039123103, "histo_col_type": "", "name": "__auto__", "null_count": 1144264060050717708, "row_count": 5599681562144882268}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24936011, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22418425, "histo_buckets": [{"distinct_range": 0, "num_eq": 2755728523330078954, "num_range": 0, "upper_bound": "\\u000e(:&N"}, {"distinct_range": 90000, "num_eq": 6511812699023512404, "num_range": 90000, "upper_bound": "$\\t=[J:\\u0003\x7f%"}, {"distinct_range": 0, "num_eq": 9000000, "num_range": 600, "upper_bound": "/"}, {"distinct_range": 8000000000, "num_eq": 300000000, "num_range": 8000000000, "upper_bound": "/t"}, {"distinct_range": 762511601668068000, "num_eq": 800000, "num_range": 1377450853349859213, "upper_bound": "3"}, {"distinct_range": 30000000, "num_eq": 7675034371787854518, "num_range": 30000000, "upper_bound": "E\\u0010-B)\\u0002\x7f"}, {"distinct_range": 8000000000, "num_eq": 7394541088987234932, "num_range": 8000000000, "upper_bound": "U["}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 30000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25615529, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1908426, "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 30000000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9794605, "histo_col_type": "", "name": "__auto__", "null_count": 28965137, "row_count": 30000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15751950, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "998899403"}, {"distinct_range": 957478496428912400, "num_eq": 5000000000, "num_range": 957478496428912337, "upper_bound": "1401661893"}, {"distinct_range": 17.486330993271572, "num_eq": 60000000000, "num_range": 30, "upper_bound": "3101681918"}, {"distinct_range": 0, "num_eq": 0, "num_range": 5677295412119404573, "upper_bound": "3325484450"}, {"distinct_range": 4905249711894275000, "num_eq": 1000, "num_range": 4905249711894275222, "upper_bound": "4043211969"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 30000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22918552, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23640777, "histo_col_type": "", "name": "__auto__", "null_count": 172692, "row_count": 30000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15121823, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26196942, "histo_col_type": "", "name": "__auto__", "null_count": 3939816, "row_count": 30000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1147547, "histo_buckets": [{"distinct_range": 0, "num_eq": 820417644926038653, "num_range": 0, "upper_bound": "355564279"}, {"distinct_range": 2962739209124415500, "num_eq": 4023099830590084884, "num_range": 2962739209124415259, "upper_bound": "1151715676"}, {"distinct_range": 0, "num_eq": 1578504383329957008, "num_range": 991928546778886793, "upper_bound": "3040816254"}, {"distinct_range": 2962676148455022000, "num_eq": 60000000, "num_range": 2962676148455022321, "upper_bound": "3653477599"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 30000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 566310, "histo_buckets": [{"distinct_range": 0, "num_eq": 3394818436273765311, "num_range": 0, "upper_bound": "c0675a78-41ac-4ff3-948d-2e569be2239c"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 30000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6610023, "histo_col_type": "", "name": "__auto__", "null_count": 28187960, "row_count": 30000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3492159, "histo_col_type": "", "name": "__auto__", "null_count": 23250525, "row_count": 30000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2154464, "histo_buckets": [{"distinct_range": 0, "num_eq": 1478292299776924569, "num_range": 0, "upper_bound": "1213759361"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 30000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23257879, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "W$"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 30000000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('twgjup', 'ofgoqp');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('qpeem', 'nxf', 'jxcmty', 'vf', 'dk');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('bbsguc');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2609828&tab=buildLog) on [master@0455055fafcef7c028b4af8e4efe242b89e7f8a3](https://github.com/cockroachdb/cockroach/commits/0455055fafcef7c028b4af8e4efe242b89e7f8a3):

```
        cockroach2_1  | webui:               http://cockroach2:8080
        cockroach2_1  | sql:                 postgresql://root@cockroach2:26257?sslmode=disable
        cockroach2_1  | RPC client flags:    /cockroach/cockroach <client cmd> --host=cockroach2:26257 --insecure
        cockroach2_1  | logs:                /cockroach-data/logs
        cockroach2_1  | temp dir:            /cockroach-data/cockroach-temp625882361
        cockroach2_1  | external I/O path:   /cockroach-data/extern
        cockroach2_1  | store[0]:            path=/cockroach-data
        cockroach2_1  | storage engine:      pebble
        cockroach2_1  | status:              initialized new cluster
        cockroach2_1  | clusterID:           8dc98723-1c4d-4233-920a-1417a3379209
        cockroach2_1  | nodeID:              1
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down....2021-01-24 06:19:39.966 UTC [49] LOG:  received fast shutdown request
        postgres_1    | 2021-01-24 06:19:39.968 UTC [49] LOG:  aborting any active transactions
        postgres_1    | 2021-01-24 06:19:39.970 UTC [49] LOG:  background worker "logical replication launcher" (PID 56) exited with exit code 1
        postgres_1    | 2021-01-24 06:19:39.976 UTC [51] LOG:  shutting down
        postgres_1    | 2021-01-24 06:19:40.073 UTC [49] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-01-24 06:19:40.178 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-01-24 06:19:40.179 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-01-24 06:19:40.182 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-01-24 06:19:40.199 UTC [86] LOG:  database system was shut down at 2021-01-24 06:19:40 UTC
        postgres_1    | 2021-01-24 06:19:40.204 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (601.03s)
        test_1        |     --- FAIL: TestCompare/postgres (0.41s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.3:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2611756&tab=buildLog) on [master@48fbbe7313fc70f889fa66c5749914f57ddd5a91](https://github.com/cockroachdb/cockroach/commits/48fbbe7313fc70f889fa66c5749914f57ddd5a91):

```
        postgres_1    | You can change this by editing pg_hba.conf or using the option -A, or
        postgres_1    | --auth-local and --auth-host, the next time you run initdb.
        postgres_1    | waiting for server to start....2021-01-25 06:21:01.854 UTC [47] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-01-25 06:21:01.871 UTC [48] LOG:  database system was shut down at 2021-01-25 06:21:01 UTC
        postgres_1    | 2021-01-25 06:21:01.876 UTC [47] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2021-01-25 06:21:04.829 UTC [47] LOG:  received fast shutdown request
        postgres_1    | .2021-01-25 06:21:04.831 UTC [47] LOG:  aborting any active transactions
        postgres_1    | 2021-01-25 06:21:04.832 UTC [47] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
        postgres_1    | 2021-01-25 06:21:04.838 UTC [49] LOG:  shutting down
        postgres_1    | 2021-01-25 06:21:04.935 UTC [47] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-01-25 06:21:05.042 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-01-25 06:21:05.042 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-01-25 06:21:05.045 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-01-25 06:21:05.062 UTC [84] LOG:  database system was shut down at 2021-01-25 06:21:04 UTC
        postgres_1    | 2021-01-25 06:21:05.068 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (602.08s)
        test_1        |     --- FAIL: TestCompare/postgres (1.27s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.3:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2615397&tab=buildLog) on [master@d86781c07065421f4a4d8bf5d988900ab07fdce5](https://github.com/cockroachdb/cockroach/commits/d86781c07065421f4a4d8bf5d988900ab07fdce5):

```
        postgres_1    |     pg_ctl -D /var/lib/postgresql/data -l logfile start
        postgres_1    | 
        postgres_1    | 
        postgres_1    | WARNING: enabling "trust" authentication for local connections
        postgres_1    | You can change this by editing pg_hba.conf or using the option -A, or
        postgres_1    | --auth-local and --auth-host, the next time you run initdb.
        postgres_1    | waiting for server to start....2021-01-26 06:12:10.733 UTC [50] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-01-26 06:12:10.749 UTC [51] LOG:  database system was shut down at 2021-01-26 06:12:10 UTC
        postgres_1    | 2021-01-26 06:12:10.755 UTC [50] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        test_1        | --- FAIL: TestCompare (1.58s)
        test_1        |     --- FAIL: TestCompare/postgres (1.13s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.45s)
        test_1        |         compare_test.go:129: 
        test_1        |             		SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;
        test_1        |             		SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;
        test_1        |             	CREATE TABLE table1 (col1_0 TIMETZ, col1_1 STRING NOT NULL, col1_2 REGPROCEDURE NOT NULL, col1_3 DATE NOT NULL, col1_4 OID, col1_5 BYTES NOT NULL, col1_6 TIMESTAMP NOT NULL, PRIMARY KEY (col1_6 DESC, col1_1 ASC, col1_5 ASC), UNIQUE (col1_1, col1_6 DESC, col1_4 ASC, col1_0 ASC, col1_3, col1_5 DESC) WHERE ((table1.col1_5 < '\xe29883':::BYTES) OR (table1.col1_6 < '-4713-11-24 00:00:00':::TIMESTAMP)) AND (table1.col1_1 <= e'\x00':::STRING), UNIQUE (col1_2 DESC, col1_1 DESC) WHERE ((((table1.col1_0 <= '24:00:00-15:59:00':::TIMETZ) OR (table1.col1_3 < '5874897-12-31':::DATE)) OR (table1.col1_5 = '\x27':::BYTES)) AND (table1.col1_1 >= '':::STRING)) OR (table1.col1_6 > '-2000-01-01 00:00:00':::TIMESTAMP), UNIQUE (col1_6 DESC) STORING (col1_2), INDEX (col1_5 ASC, col1_0, col1_1 ASC, col1_4 DESC, col1_3 ASC), INDEX (col1_2 DESC) STORING (col1_0, col1_4), UNIQUE (col1_2 ASC, col1_3 DESC, col1_4 DESC, col1_6, col1_0 ASC, col1_1 ASC), UNIQUE (col1_1));
        test_1        |             CREATE TABLE table2 (col2_0 TIMETZ NOT NULL, col2_1 VARCHAR NOT NULL, col2_2 OID NOT NULL, col2_3 BIT(41) NOT NULL, col2_4 TIMESTAMPTZ NOT NULL, col2_5 REGCLASS NOT NULL, col2_6 BIT(46) NOT NULL, col2_7 INET NOT NULL, col2_8 REGTYPE NOT NULL, col2_9 REGCLASS NOT NULL, col2_10 NAME NOT NULL, col2_11 REGTYPE NOT NULL, col2_12 FLOAT8 NOT NULL, col2_13 DATE NULL, PRIMARY KEY (col2_8 ASC, col2_10 ASC, col2_7 ASC, col2_4, col2_5 DESC, col2_11 DESC, col2_12 DESC, col2_1 DESC, col2_6 ASC, col2_0, col2_3 ASC, col2_9 ASC, col2_2 DESC), UNIQUE (col2_7 DESC, col2_11 ASC, col2_8, col2_12, col2_3 ASC, col2_9 ASC, col2_13 ASC, col2_4, col2_6 DESC, col2_5 ASC, col2_10), INDEX (col2_2, col2_12 DESC, col2_5 DESC, col2_3, col2_7, col2_13 DESC, col2_9 ASC, col2_0 DESC, col2_4, col2_10, col2_11, col2_6 DESC, col2_1) WHERE (((((table2.col2_12 <= 5e-324:::FLOAT8) OR (table2.col2_1 > 'X':::STRING)) AND (table2.col2_4 <= '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) OR (table2.col2_13 >= 'infinity':::DATE)) AND (table2.col2_10 >= e'\x00':::STRING)) AND (table2.col2_0 <= '24:00:00-15:59:00':::TIMETZ), UNIQUE (col2_8 DESC, col2_10, col2_2, col2_12 DESC, col2_5), UNIQUE (col2_7, col2_9 DESC, col2_10, col2_4, col2_8, col2_1 ASC, col2_5 ASC, col2_11 DESC, col2_3, col2_2 ASC, col2_12 DESC, col2_0 ASC, col2_13, col2_6 DESC), UNIQUE (col2_11 DESC) STORING (col2_13), INDEX (col2_10 ASC, col2_2, col2_4 ASC, col2_11 DESC, col2_8 DESC, col2_5, col2_6, col2_13 ASC, col2_0 DESC, col2_3 DESC, col2_12 ASC, col2_9 ASC, col2_7), FAMILY (col2_11, col2_3), FAMILY (col2_9), FAMILY (col2_12, col2_2, col2_5), FAMILY (col2_6), FAMILY (col2_8), FAMILY (col2_13), FAMILY (col2_10), FAMILY (col2_7), FAMILY (col2_4, col2_1), FAMILY (col2_0), UNIQUE (col2_2, col2_13));
        test_1        |             CREATE TABLE table3 (col1_6 TIMESTAMP NOT NULL, col1_1 STRING NOT NULL, col1_5 BYTES NOT NULL, col3_3 REGPROC NOT NULL, col3_4 REGNAMESPACE NOT NULL, col3_5 INET, col3_6 CHAR NOT NULL, col3_7 UUID, col3_8 DATE NOT NULL, PRIMARY KEY (col1_6 DESC, col1_1 ASC, col1_5 ASC, col3_4 ASC, col3_6 ASC, col3_7 ASC), UNIQUE (col3_3 DESC), UNIQUE (col3_3 DESC, col3_8 ASC, col3_7 ASC, col3_6 ASC, col1_5 ASC, col3_5 ASC, col3_4 ASC, col1_6 ASC), INDEX (col3_5, col3_3, col3_7 DESC, col3_6 ASC, col3_4, col1_1 ASC) WHERE ((((table3.col3_6 < e'\x00':::STRING) AND (table3.col1_1 <= e'\x00':::STRING)) AND (table3.col3_8 < '4714-11-24 BC':::DATE)) OR (table3.col1_5 = '\x':::BYTES)) OR (table3.col1_6 = '3000-01-01 00:00:00':::TIMESTAMP), INDEX (col3_4 ASC, col1_1 ASC, col3_5 ASC, col3_8 ASC, col3_6 ASC, col1_5, col3_3 ASC), UNIQUE (col3_5, col3_4 DESC, col3_6 DESC, col1_5, col1_1 ASC, col3_7 DESC, col3_8), INDEX (col1_5) WHERE ((((table3.col1_6 > '-2000-01-01 00:00:00':::TIMESTAMP) OR (table3.col3_8 <= '4714-11-24 BC':::DATE)) OR (table3.col1_1 = '':::STRING)) OR (table3.col3_6 <= 'X':::STRING)) AND (table3.col1_5 != '\x00':::BYTES), INDEX (col3_5 ASC, col3_4 ASC, col3_6, col3_3, col1_1, col1_5 DESC, col1_6) WHERE (((table3.col1_1 >= e'\'':::STRING) AND (table3.col3_8 = '1970-01-01':::DATE)) AND (table3.col1_5 != '\x':::BYTES)) OR (table3.col3_6 != e'\U00002603':::STRING), INDEX (col3_3 DESC, col3_6, col3_8, col3_4 DESC, col3_7 ASC, col1_6, col3_5 ASC) WHERE (table3.col1_5 > '\x58':::BYTES) OR (table3.col1_6 >= '-2000-01-01 00:00:00':::TIMESTAMP), UNIQUE (col1_1 ASC, col3_4, col3_8, col3_6, col1_5, col1_6 DESC) STORING (col3_3), FAMILY (col3_7, col3_8), FAMILY (col1_1), FAMILY (col3_6, col3_5, col3_3, col1_5), FAMILY (col1_6), FAMILY (col3_4)) INTERLEAVE IN PARENT table1 (col1_6, col1_1, col1_5);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 660620024503054077, "histo_col_type": "", "name": "__auto__", "null_count": 2107413109272469358, "row_count": 6526349001673019030}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4885933002617636548, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 4001483359140410353, "num_range": 30000000, "upper_bound": "\\u0008\\\\-1oTk"}, {"distinct_range": 1505469869348760800, "num_eq": 2000000, "num_range": 1559497998074714435, "upper_bound": "\\u001dtXeK/5\\u001c"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 6436787154174794377, "upper_bound": ";\\u000b/"}, {"distinct_range": 0, "num_eq": 8163584014065870139, "num_range": 3000000, "upper_bound": "F\\tdmk\\u0002\\u0015\\r;"}, {"distinct_range": 94595.1224396511, "num_eq": 50, "num_range": 200000, "upper_bound": "Hd?v\\u0008]!&6"}, {"distinct_range": 50, "num_eq": 2512131477151385471, "num_range": 50, "upper_bound": "Q~"}, {"distinct_range": 10000000000, "num_eq": 7320276246812840593, "num_range": 10000000000, "upper_bound": "p;7,+1"}, {"distinct_range": 0, "num_eq": 4304401181506074644, "num_range": 3000, "upper_bound": "x"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 6526349001673019030}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4950957270746300171, "histo_buckets": [{"distinct_range": 0, "num_eq": 5094913544064601007, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 276126.5943927332, "num_eq": 5351867751457223153, "num_range": 300000, "upper_bound": "729239074"}, {"distinct_range": 45067.869131974345, "num_eq": 2231073231509535272, "num_range": 70000, "upper_bound": "4222811654"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 6526349001673019030}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2172097969067689174, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6526349001673019030}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6338184646043656776, "histo_col_type": "", "name": "__auto__", "null_count": 2785819860972809109, "row_count": 6526349001673019030}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5665930352913428785, "histo_buckets": [{"distinct_range": 0, "num_eq": 8742423821679310754, "num_range": 0, "upper_bound": "\\\\x00"}, {"distinct_range": 2073696013163969000, "num_eq": 20000000, "num_range": 4521263393964730045, "upper_bound": "\\\\x01470898"}, {"distinct_range": 19.607195185280904, "num_eq": 600000000, "num_range": 70, "upper_bound": "\\\\x088b"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 4313157979818822608, "upper_bound": "\\\\x92ce008f3db5"}, {"distinct_range": 5744483298115402000, "num_eq": 7858362660177835495, "num_range": 5744483298115401660, "upper_bound": "\\\\xf0c74059808203"}, {"distinct_range": 100, "num_eq": 5802827435134702666, "num_range": 100, "upper_bound": "\\\\xf57e7b65f54b7e"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 6526349001673019030}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1557583351197165069, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "1992-02-08 00:40:10.000914"}, {"distinct_range": 0, "num_eq": 1379458633609480356, "num_range": 80000000, "upper_bound": "2001-05-11 23:09:27.000131"}, {"distinct_range": 8000000, "num_eq": 2000000, "num_range": 8000000, "upper_bound": "2001-09-30 06:06:25.000432"}, {"distinct_range": 748526987623119700, "num_eq": 8040406905908753951, "num_range": 748526987623119792, "upper_bound": "2008-10-10 09:24:21.000527"}, {"distinct_range": 53429.59367966852, "num_eq": 5057597273460943647, "num_range": 100000, "upper_bound": "2009-08-16 01:47:28.000732"}, {"distinct_range": 0, "num_eq": 800000, "num_range": 3000000000, "upper_bound": "2011-08-31 19:22:37.000786"}, {"distinct_range": 34932022024482932, "num_eq": 6462613979761976612, "num_range": 1978003461473898290, "upper_bound": "2024-01-18 07:31:21.000252"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 6526349001673019030}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 328818743485083942, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3875389114524645980}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1891174549723453091, "histo_buckets": [{"distinct_range": 0, "num_eq": 157730856335515890, "num_range": 0, "upper_bound": "639057664"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 3875389114524645980}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2769408824910575737, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3875389114524645980}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 185068975426793777, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3875389114524645980}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2187878612678377165, "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 3875389114524645980}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 767319554857071062, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 31239004.17580906, "num_eq": 5700606397972891161, "num_range": 300000000, "upper_bound": "700793697"}, {"distinct_range": 1182423842773959700, "num_eq": 900, "num_range": 9189205156885296259, "upper_bound": "1319685693"}, {"distinct_range": 0, "num_eq": 8468264022145682595, "num_range": 80000000000, "upper_bound": "2401853700"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 3875389114524645980}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3844842015485753471, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3875389114524645980}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1559859433666621417, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3875389114524645980}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 454330792775611418, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3875389114524645980}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1990861405492344310, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3875389114524645980}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2678031730955113416, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "\\u0003"}, {"distinct_range": 58111626777.83009, "num_eq": 195044633309143969, "num_range": 70000000000, "upper_bound": "\\r\\u0011%P\\n"}, {"distinct_range": 0, "num_eq": 50, "num_range": 0, "upper_bound": " \\u000etIpA\\u0008"}, {"distinct_range": 425659415268740700, "num_eq": 4868110971912662102, "num_range": 4389087081157027817, "upper_bound": "\'"}, {"distinct_range": 0, "num_eq": 3558620103580411976, "num_range": 2543732960619361403, "upper_bound": "\'6w"}, {"distinct_range": 70000000, "num_eq": 1000000000, "num_range": 70000000, "upper_bound": "VRu\\t"}, {"distinct_range": 416261.19179659156, "num_eq": 4000, "num_range": 500000, "upper_bound": "\U00002603"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 3875389114524645980}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1948011570097201553, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3875389114524645980}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3018625706343751915, "histo_buckets": [{"distinct_range": 0, "num_eq": 5521718420035623781, "num_range": 0, "upper_bound": "220.149.182.65/21"}, {"distinct_range": 0, "num_eq": 9210439279914799513, "num_range": 7015674516291360556, "upper_bound": "5b6c:5304:54c8:ca99:5e62:4c43:8906:f5cc/14"}, {"distinct_range": 0, "num_eq": 6306910371270315021, "num_range": 50, "upper_bound": "6663:91da:bf50:969a:afd6:2d27:b182:5452/43"}, {"distinct_range": 960287541.7006007, "num_eq": 500, "num_range": 10000000000, "upper_bound": "3a35:8663:d06a:4d74:b8d7:44c5:a4b5:47b9/53"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 7021010450265503019, "upper_bound": "7234:dc14:a2c3:1e1:31e9:814a:467d:637d/77"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 3875389114524645980}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2572155936501792319, "histo_col_type": "", "name": "__auto__", "null_count": 2473294085194812185, "row_count": 3875389114524645980}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 153, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000, "num_range": 0, "upper_bound": "2321252087"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 385, "histo_buckets": [{"distinct_range": 0, "num_eq": 7439548827811770834, "num_range": 0, "upper_bound": "2016-07-28 07:51:35.000396"}, {"distinct_range": 4098616652809122000, "num_eq": 30, "num_range": 4900684302850116989, "upper_bound": "2028-01-26 19:25:08.000516"}, {"distinct_range": 4041124741002938400, "num_eq": 0, "num_range": 4041124741002938557, "upper_bound": "2029-07-16 07:06:16.000698"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 674, "histo_buckets": [{"distinct_range": 0, "num_eq": 3107763939642370888, "num_range": 0, "upper_bound": "\\u0001|-\\u0017G&"}, {"distinct_range": 0, "num_eq": 5207195424995622653, "num_range": 0, "upper_bound": "\\u00037+G"}, {"distinct_range": 0, "num_eq": 9104318770493098949, "num_range": 0, "upper_bound": "G"}, {"distinct_range": 10, "num_eq": 6345609185772306173, "num_range": 10, "upper_bound": "PiOP$Xu"}, {"distinct_range": 180456552638840030, "num_eq": 221092036906001677, "num_range": 180456552638840017, "upper_bound": "hf1!\\u001d"}, {"distinct_range": 5825064392900351000, "num_eq": 30000000, "num_range": 5825064392900350750, "upper_bound": "i#X\\u0002\\u0001c2"}, {"distinct_range": 0, "num_eq": 7729556368904287574, "num_range": 1000000, "upper_bound": "t"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 340, "histo_buckets": [{"distinct_range": 0, "num_eq": 3258147325661359721, "num_range": 0, "upper_bound": "\\\\x46770062338c"}, {"distinct_range": 0, "num_eq": 7014447308991984756, "num_range": 0, "upper_bound": "\\\\x5867db619b45943d41"}, {"distinct_range": 339061194787615500, "num_eq": 5000000, "num_range": 4875590455390453198, "upper_bound": "\\\\x92e1cc"}, {"distinct_range": 0, "num_eq": 5000000000, "num_range": 7000, "upper_bound": "\\\\x9763c5663c"}, {"distinct_range": 0, "num_eq": 6000000, "num_range": 300000, "upper_bound": "\\\\xbe717d8594"}, {"distinct_range": 3690272017532776000, "num_eq": 600000000, "num_range": 7130516378625069778, "upper_bound": "\\\\xe272dae9f08a0841"}, {"distinct_range": 4000, "num_eq": 100000000, "num_range": 4000, "upper_bound": "\\\\xe29883"}, {"distinct_range": 434.35278412014935, "num_eq": 80000000, "num_range": 1000, "upper_bound": "\\\\xee5e046a87f30f"}, {"distinct_range": 60000, "num_eq": 4000000, "num_range": 60000, "upper_bound": "\\\\xf16e0efab9733183"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 602, "histo_buckets": [{"distinct_range": 0, "num_eq": 400, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 60000, "num_range": 90000000, "upper_bound": "101291055"}, {"distinct_range": 0, "num_eq": 1408364047641961940, "num_range": 3510321236762010236, "upper_bound": "355097284"}, {"distinct_range": 0, "num_eq": 9071532317952504280, "num_range": 6066533548324714453, "upper_bound": "1081298762"}, {"distinct_range": 50, "num_eq": 10, "num_range": 50, "upper_bound": "1643964519"}, {"distinct_range": 3379416716112419300, "num_eq": 700000000, "num_range": 3379416716112419427, "upper_bound": "1760214383"}, {"distinct_range": 2399137157584894500, "num_eq": 800, "num_range": 2399137157584894302, "upper_bound": "2342325402"}, {"distinct_range": 0, "num_eq": 3314494915784161521, "num_range": 1161474606636486521, "upper_bound": "3372410176"}, {"distinct_range": 2220669845626388000, "num_eq": 4110490408787639437, "num_range": 2220669845626388075, "upper_bound": "4236736058"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 272, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "21db:bc02:ee6f:7bad:8550:82e5:1220:1e18/28"}, {"distinct_range": 0, "num_eq": 8673597198832261408, "num_range": 9000000, "upper_bound": "e3d2:51d6:dbdd:a79:6ce7:691a:d1fe:7a0a/54"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 411, "row_count": 700}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 694, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 62, "histo_col_type": "", "name": "__auto__", "null_count": 176, "row_count": 700}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 530, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700}]':::JSONB;
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_2, col1_3) REFERENCES table2 (col2_2, col2_13);
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col1_1) REFERENCES table1 (col1_1) ON UPDATE CASCADE;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('vaia');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('qj', 'b', 'kd');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('juj', 'ljext');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('crcwgm');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2618884&tab=buildLog) on [master@f7c5898f3d552f7ab0751cdd9ffa95cdfd6b8a76](https://github.com/cockroachdb/cockroach/commits/f7c5898f3d552f7ab0751cdd9ffa95cdfd6b8a76):

```
        postgres_1    | creating configuration files ... ok
        postgres_1    | running bootstrap script ... ok
        postgres_1    | performing post-bootstrap initialization ... ok
        postgres_1    | syncing data to disk ... ok
        postgres_1    | 
        postgres_1    | Success. You can now start the database server using:
        postgres_1    | 
        postgres_1    |     pg_ctl -D /var/lib/postgresql/data -l logfile start
        postgres_1    | 
        postgres_1    | 
        postgres_1    | WARNING: enabling "trust" authentication for local connections
        postgres_1    | You can change this by editing pg_hba.conf or using the option -A, or
        postgres_1    | --auth-local and --auth-host, the next time you run initdb.
        postgres_1    | waiting for server to start....2021-01-27 06:12:09.608 UTC [49] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-01-27 06:12:09.625 UTC [50] LOG:  database system was shut down at 2021-01-27 06:12:09 UTC
        postgres_1    | 2021-01-27 06:12:09.631 UTC [49] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        test_1        | --- FAIL: TestCompare (1.20s)
        test_1        |     --- FAIL: TestCompare/postgres (0.97s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.23s)
        test_1        |         compare_test.go:129: 
        test_1        |             		SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;
        test_1        |             		SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;
        test_1        |             	CREATE TABLE table1 (col1_0 TIMESTAMP NOT NULL, col1_1 INTERVAL[], col1_2 DATE NULL, PRIMARY KEY (col1_0 DESC), INDEX (col1_2, col1_0 ASC), UNIQUE (col1_0 DESC), INDEX (col1_2 ASC), UNIQUE (col1_2 DESC) WHERE (table1.col1_2 != '1970-01-01':::DATE) AND (table1.col1_0 = '0001-01-01 00:00:00':::TIMESTAMP), UNIQUE (col1_2 ASC) WHERE table1.col1_2 >= '-infinity':::DATE, INDEX (col1_0 DESC, col1_2) WHERE (table1.col1_0 >= '3000-01-01 00:00:00':::TIMESTAMP) AND (table1.col1_2 > '4714-11-24 BC':::DATE), INDEX (col1_2 DESC), FAMILY (col1_1), FAMILY (col1_0, col1_2));
        test_1        |             CREATE TABLE table2 (col1_0 TIMESTAMP NOT NULL, col2_1 BIT(36), col2_2 TIME NULL, col2_3 INT2, col2_4 DATE, col2_5 STRING NULL, col2_6 UUID, col2_7 REGPROC NOT NULL, col2_8 TIMETZ NOT NULL, col2_9 INT8 NOT NULL, PRIMARY KEY (col1_0 DESC, col2_1, col2_7, col2_4, col2_6), FAMILY (col2_3), FAMILY (col2_7), FAMILY (col2_5, col2_2), FAMILY (col2_9), FAMILY (col2_6, col2_4), FAMILY (col2_8, col2_1, col1_0)) INTERLEAVE IN PARENT table1 (col1_0);
        test_1        |             CREATE TABLE table3 (col3_0 FLOAT8 NOT NULL, col3_1 DATE NOT NULL, PRIMARY KEY (col3_0 ASC, col3_1 DESC), INDEX (col3_1 ASC, col3_0), UNIQUE (col3_0 ASC, col3_1 ASC), INDEX (col3_1, col3_0) WHERE (table3.col3_1 <= '5874897-12-31':::DATE) OR (table3.col3_0 = 'NaN':::FLOAT8), FAMILY (col3_0), FAMILY (col3_1));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 220947, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "0001-01-01 00:00:00"}, {"distinct_range": 0, "num_eq": 7066641138013287786, "num_range": 2000000, "upper_bound": "1986-07-21 10:27:58.000648"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 6579317176377996618, "upper_bound": "1992-12-09 04:35:41.000844"}, {"distinct_range": 3878041940129912300, "num_eq": 80000000, "num_range": 3878041940129912534, "upper_bound": "1995-04-18 22:47:15.000623"}, {"distinct_range": 578775693698169300, "num_eq": 4515627522559271913, "num_range": 578775693698169289, "upper_bound": "1999-11-19 23:23:25.000255"}, {"distinct_range": 0, "num_eq": 1624887186107227085, "num_range": 2000, "upper_bound": "2004-05-13 17:29:52.000858"}, {"distinct_range": 0, "num_eq": 6000000, "num_range": 100, "upper_bound": "2013-08-31 00:49:56.000872"}, {"distinct_range": 0, "num_eq": 200000, "num_range": 5000, "upper_bound": "2014-07-01 15:10:47.000034"}, {"distinct_range": 0, "num_eq": 2000000000, "num_range": 40000000000, "upper_bound": "2014-11-14 00:03:13.000299"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 261034, "histo_col_type": "", "name": "__auto__", "null_count": 451569, "row_count": 900000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 278585, "histo_buckets": [{"distinct_range": 0, "num_eq": 9109168674260443539, "num_range": 0, "upper_bound": "1970-11-05"}, {"distinct_range": 0, "num_eq": 50000, "num_range": 40000000000, "upper_bound": "1973-07-31"}, {"distinct_range": 379236164236195800, "num_eq": 900000000, "num_range": 379236164236195788, "upper_bound": "1980-10-16"}, {"distinct_range": 4299908798118678000, "num_eq": 7034830142336106763, "num_range": 4299908798118678003, "upper_bound": "1992-01-08"}, {"distinct_range": 6784746921897624000, "num_eq": 6000000000, "num_range": 6784746921897623465, "upper_bound": "1994-08-10"}, {"distinct_range": 72936129570248400, "num_eq": 3000, "num_range": 5471203853478179705, "upper_bound": "1995-02-24"}, {"distinct_range": 0, "num_eq": 800000, "num_range": 1000000, "upper_bound": "1995-04-08"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 466848, "row_count": 900000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5514005259912136470, "histo_col_type": "", "name": "__auto__", "null_count": 3513670669553971231, "row_count": 5551060532795605985}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3614267803578208753, "histo_col_type": "", "name": "__auto__", "null_count": 3971930416662979043, "row_count": 5551060532795605985}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3821684295302576505, "histo_col_type": "", "name": "__auto__", "null_count": 3160890643886088892, "row_count": 5551060532795605985}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 798292205676063354, "histo_col_type": "", "name": "__auto__", "null_count": 4064180246514846287, "row_count": 5551060532795605985}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 773116702892824264, "histo_col_type": "", "name": "__auto__", "null_count": 5371651321001147713, "row_count": 5551060532795605985}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1101724920069245405, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5551060532795605985}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5099419984925997107, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "1976-05-27 23:03:05.000561"}, {"distinct_range": 0, "num_eq": 20, "num_range": 0, "upper_bound": "1986-08-19 20:39:41.00019"}, {"distinct_range": 384001477251387100, "num_eq": 100000000000, "num_range": 831722408263231903, "upper_bound": "2005-04-22 02:34:04.000047"}, {"distinct_range": 4496898017830321000, "num_eq": 3000000, "num_range": 7275795002951355852, "upper_bound": "2008-06-22 06:19:30.000945"}, {"distinct_range": 0, "num_eq": 400000, "num_range": 8634108416406850061, "upper_bound": "2024-12-26 16:47:20.000157"}, {"distinct_range": 0, "num_eq": 7649505940327512552, "num_range": 1635430488983503576, "upper_bound": "2027-05-20 21:19:42.000496"}, {"distinct_range": 9000, "num_eq": 3745051148430959378, "num_range": 9000, "upper_bound": "2029-03-27 22:39:24.000606"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 5551060532795605985}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 215512413856811224, "histo_col_type": "", "name": "__auto__", "null_count": 5174980634471412987, "row_count": 5551060532795605985}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1319068820301023348, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5551060532795605985}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2749768319491698742, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5551060532795605985}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 56546204422, "histo_buckets": [{"distinct_range": 0, "num_eq": 6371606787819074294, "num_range": 0, "upper_bound": "-1.5239283725560333"}, {"distinct_range": 4771544893990515000, "num_eq": 6428129345711829964, "num_range": 4771544893990514374, "upper_bound": "-0.885766223305848"}, {"distinct_range": 0, "num_eq": 7249464388337821611, "num_range": 0, "upper_bound": "-0.2435615672476965"}, {"distinct_range": 1027012349335927300, "num_eq": 3716648425894166569, "num_range": 1027012349335927327, "upper_bound": "0.3562833828766718"}, {"distinct_range": 0, "num_eq": 100, "num_range": 5000, "upper_bound": "0.725307493364357"}, {"distinct_range": 1377764171462264000, "num_eq": 80000000, "num_range": 1377764171462264186, "upper_bound": "0.8573087843920272"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 1502304538381747871, "upper_bound": "1.057693691901272"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 7000, "upper_bound": "2.3375560323072264"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 60000000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25142565333, "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 60000000000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('lsocs', 'mfbmyk', 'vjijjq', 'kser');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        | FAIL
        postgres_1    | CREATE EXTENSION
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2622648&tab=buildLog) on [master@1a459a81dba35b6a091f0a2954aa33d50f1e5d24](https://github.com/cockroachdb/cockroach/commits/1a459a81dba35b6a091f0a2954aa33d50f1e5d24):

```
        postgres_1    | 
        postgres_1    | 
        postgres_1    | WARNING: enabling "trust" authentication for local connections
        postgres_1    | You can change this by editing pg_hba.conf or using the option -A, or
        postgres_1    | --auth-local and --auth-host, the next time you run initdb.
        postgres_1    | waiting for server to start....2021-01-28 06:12:06.813 UTC [52] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-01-28 06:12:06.831 UTC [53] LOG:  database system was shut down at 2021-01-28 06:12:06 UTC
        postgres_1    | 2021-01-28 06:12:06.836 UTC [52] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        test_1        | --- FAIL: TestCompare (3.04s)
        test_1        |     --- FAIL: TestCompare/postgres (2.56s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.3:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.48s)
        test_1        |         compare_test.go:129: 
        test_1        |             		SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;
        test_1        |             		SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;
        test_1        |             	CREATE TABLE table1 (col1_0 REGTYPE NOT NULL, col1_1 BOOL, col1_2 OID NOT NULL, col1_3 CHAR, col1_4 INT8 NOT NULL, col1_5 REGPROCEDURE, col1_6 REGTYPE NOT NULL, col1_7 BOX2D NOT NULL, col1_8 VARCHAR NOT NULL, col1_9 "char" NOT NULL, col1_10 TIME NOT NULL, col1_11 DECIMAL[] NOT NULL, col1_12 BIT(42) NULL, col1_13 REGNAMESPACE NOT NULL, col1_14 CHAR NOT NULL, col1_15 INT2, PRIMARY KEY (col1_8 ASC, col1_2 DESC, col1_13 ASC, col1_10 DESC, col1_7 DESC, col1_9, col1_4 ASC, col1_14 ASC), UNIQUE (col1_0, col1_3 ASC, col1_4 ASC, col1_2 ASC, col1_7 ASC, col1_9 ASC) WHERE (NOT table1.col1_1) OR (table1.col1_14 <= 'X':::STRING), INDEX (col1_8 DESC, col1_0 DESC, col1_7 DESC, col1_4 DESC) WHERE (((((((NOT table1.col1_1) OR (table1.col1_10 != '24:00:00':::TIME)) AND (table1.col1_3 > '"':::STRING)) AND (table1.col1_9 < e'\U00002603':::STRING)) AND (table1.col1_8 > '"':::STRING)) OR (table1.col1_14 < e'\x00':::STRING)) OR (table1.col1_15 > (-32768):::INT8)) AND (table1.col1_4 < (-2147483648):::INT8), UNIQUE (col1_14 ASC, col1_10 DESC, col1_13 ASC, col1_2 DESC, col1_12 DESC, col1_6 ASC, col1_0, col1_8 DESC) WHERE (((((table1.col1_14 = e'\x00':::STRING) OR (table1.col1_9 != e'\'':::STRING)) OR (table1.col1_4 >= 127:::INT8)) AND (NOT table1.col1_1)) OR (table1.col1_10 < '00:00:00':::TIME)) OR (table1.col1_8 < 'X':::STRING), UNIQUE (col1_1 DESC, col1_13, col1_9 DESC, col1_3 ASC, col1_6 ASC, col1_2 ASC) WHERE (((((((table1.col1_14 > '':::STRING) AND (NOT table1.col1_1)) AND (table1.col1_15 >= 127:::INT8)) AND (table1.col1_9 != '"':::STRING)) AND (table1.col1_4 > 9223372036854775807:::INT8)) OR (table1.col1_3 != e'\U00002603':::STRING)) AND (table1.col1_10 > '00:00:00':::TIME)) AND (table1.col1_8 >= '"':::STRING), UNIQUE (col1_13 DESC, col1_5 DESC, col1_12 ASC, col1_2, col1_0, col1_10 ASC, col1_8 DESC), UNIQUE (col1_0 DESC, col1_14 ASC, col1_3 ASC, col1_6 ASC, col1_2 ASC, col1_12 ASC, col1_1 DESC, col1_13 DESC, col1_8, col1_15 ASC, col1_7 ASC), INDEX (col1_14, col1_12, col1_6 ASC, col1_8, col1_3 DESC, col1_15 DESC, col1_7 ASC, col1_4 ASC, col1_0, col1_10 ASC, col1_2 ASC, col1_9 ASC, col1_13, col1_1, col1_5 DESC), UNIQUE (col1_8 DESC, col1_0 DESC, col1_9, col1_2 DESC, col1_4 DESC, col1_10, col1_6 DESC, col1_12, col1_1, col1_14 ASC, col1_15 ASC) STORING (col1_5) WHERE (((((((table1.col1_4 = 9223372036854775807:::INT8) AND (table1.col1_10 >= '24:00:00':::TIME)) AND (table1.col1_14 != 'X':::STRING)) AND (table1.col1_9 <= e'\U00002603':::STRING)) OR (table1.col1_15 >= (-1):::INT8)) AND table1.col1_1) OR (table1.col1_3 <= e'\x00':::STRING)) OR (table1.col1_8 >= 'X':::STRING), UNIQUE (col1_12 ASC, col1_2 DESC, col1_14 ASC, col1_0 ASC, col1_7, col1_5, col1_6 DESC, col1_9, col1_8 ASC, col1_1 DESC) WHERE (((((((table1.col1_3 <= e'\x00':::STRING) OR (table1.col1_14 <= 'X':::STRING)) OR (table1.col1_4 >= 2147483647:::INT8)) OR (table1.col1_9 <= e'\'':::STRING)) OR (table1.col1_15 >= (-1):::INT8)) AND (NOT table1.col1_1)) OR (table1.col1_10 >= '00:00:00':::TIME)) OR (table1.col1_8 <= '"':::STRING));
        test_1        |             CREATE TABLE table2 (col1_8 VARCHAR NOT NULL, col1_2 OID NOT NULL, col1_13 REGNAMESPACE NOT NULL, col1_10 TIME NOT NULL, col1_7 BOX2D NOT NULL, col1_9 "char" NOT NULL, col1_4 INT8 NOT NULL, col1_14 CHAR NOT NULL, col2_8 BIT(46), col2_9 "char" NULL, col2_10 REGCLASS, col2_11 DECIMAL, col2_12 FLOAT8 NOT NULL, col2_13 TIME NOT NULL, col2_14 TIMESTAMPTZ NULL, col2_15 BIT(35) NULL, col2_16 INT2, col2_17 TIMETZ, col2_18 DATE NOT NULL, col2_19 OID NULL, col2_20 FLOAT8 NULL, col2_21 NAME NOT NULL, col2_22 VARCHAR, col2_23 REGPROCEDURE NOT NULL, col2_24 BOOL NOT NULL, col2_25 CHAR NOT NULL, col2_26 VARCHAR, PRIMARY KEY (col1_8 ASC, col1_2 DESC, col1_13 ASC, col1_10 DESC, col1_7 DESC, col1_9, col1_4 ASC, col1_14 ASC, col2_24 ASC, col2_17 DESC), INDEX (col2_24 ASC, col2_25, col1_4 ASC, col2_23, col2_8, col2_15, col2_21, col2_22 ASC, col2_18, col2_19 DESC, col2_17, col2_14, col2_12 DESC) STORING (col2_10, col2_11, col2_16, col2_20) WHERE ((((((table2.col2_21 >= 'X':::STRING) OR table2.col2_24) AND (table2.col2_22 = '':::STRING)) OR (table2.col2_18 != '1970-01-01':::DATE)) AND (table2.col1_10 < '00:00:00':::TIME)) OR (table2.col2_13 < '24:00:00':::TIME)) OR (table2.col1_8 = e'\x00':::STRING), UNIQUE (col1_13 ASC, col2_10, col1_14 ASC, col2_13) STORING (col2_8, col2_9, col2_14, col2_16, col2_18, col2_19, col2_20, col2_22) WHERE (((((((table2.col2_12 = 0.0:::FLOAT8) OR (table2.col1_4 = 2147483647:::INT8)) AND (table2.col2_25 > e'\U00002603':::STRING)) OR (table2.col1_14 <= 'X':::STRING)) AND (table2.col2_14 > '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table2.col1_8 < 'X':::STRING)) AND (table2.col2_20 > 1.0:::FLOAT8)) AND (table2.col2_18 < '1970-01-01':::DATE), INDEX (col2_10, col2_19 ASC, col1_13 DESC, col2_21, col1_8 ASC, col1_10, col1_7 ASC, col1_4 DESC, col1_14 ASC, col2_24) STORING (col2_8, col2_9, col2_12, col2_13, col2_16, col2_20, col2_23, col2_26) WHERE ((((((((((((((((((table2.col1_4 <= 127:::INT8) AND (table2.col1_8 > e'\U00002603':::STRING)) AND (table2.col1_14 < '"':::STRING)) AND table2.col2_24) OR (table2.col2_11 < (-1):::DECIMAL)) OR (table2.col2_14 >= '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table2.col2_16 >= (-32768):::INT8)) AND (table2.col1_9 > '"':::STRING)) OR (table2.col2_9 = '"':::STRING)) AND (table2.col2_21 < e'\U00002603':::STRING)) OR (table2.col2_13 != '24:00:00':::TIME)) OR (table2.col2_17 > '00:00:00+15:59:00':::TIMETZ)) OR (table2.col1_10 = '24:00:00':::TIME)) OR (table2.col2_20 = 1.401298464324817e-45:::FLOAT8)) AND (table2.col2_25 != '"':::STRING)) AND (table2.col2_22 <= e'\x00':::STRING)) AND (table2.col2_26 <= '':::STRING)) AND (table2.col2_18 < '1970-01-01':::DATE)) AND (table2.col2_12 >= 'NaN':::FLOAT8), INDEX (col2_9, col2_10, col2_18 ASC, col2_14 ASC, col2_24, col2_22 DESC, col2_21, col2_20, col1_14 DESC, col2_23 ASC, col2_8, col1_10 ASC, col2_13, col2_25 ASC, col2_26 ASC, col2_15 DESC, col2_17 ASC) WHERE ((((((((((((((((((table2.col2_26 != '"':::STRING) OR (table2.col2_20 <= '-Inf':::FLOAT8)) AND (table2.col1_8 <= '"':::STRING)) OR (table2.col2_9 >= '"':::STRING)) OR (table2.col2_22 < e'\'':::STRING)) AND (table2.col1_4 = 2147483647:::INT8)) OR (table2.col1_9 = '':::STRING)) AND (table2.col2_25 = '':::STRING)) OR (table2.col1_10 > '24:00:00':::TIME)) AND (table2.col2_12 <= '-Inf':::FLOAT8)) OR (table2.col2_17 = '00:00:00+15:59:00':::TIMETZ)) AND (table2.col2_13 = '24:00:00':::TIME)) OR (table2.col2_11 >= 1:::DECIMAL)) OR (table2.col2_18 != 'infinity':::DATE)) OR (table2.col1_14 > '':::STRING)) AND (NOT table2.col2_24)) AND (table2.col2_16 > 127:::INT8)) OR (table2.col2_21 < e'\x00':::STRING)) OR (table2.col2_14 >= '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ), INDEX (col1_4 DESC, col1_8 ASC, col2_10 ASC, col2_9 DESC, col2_25) WHERE ((((((((((((table2.col2_9 <= '"':::STRING) AND (table2.col2_26 <= e'\'':::STRING)) AND (table2.col1_10 != '24:00:00':::TIME)) OR (table2.col2_25 <= 'X':::STRING)) OR (NOT table2.col2_24)) OR (table2.col2_18 >= 'infinity':::DATE)) OR (table2.col2_20 < 5e-324:::FLOAT8)) OR (table2.col2_16 >= (-1):::INT8)) AND (table2.col2_14 = '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) OR (table2.col2_12 < 'NaN':::FLOAT8)) OR (table2.col2_21 > e'\x00':::STRING)) AND (table2.col1_14 > '':::STRING)) AND (table2.col1_9 <= e'\U00002603':::STRING), UNIQUE (col2_16 DESC, col1_4 ASC, col2_23, col2_18, col1_9, col2_25 ASC, col2_11 DESC, col2_24 ASC, col2_21 ASC), UNIQUE (col1_8 DESC, col1_10, col1_4, col2_18 ASC, col2_12 DESC, col2_19, col2_9 DESC, col2_11, col2_14 DESC, col1_9) STORING (col2_10, col2_13, col2_16, col2_20, col2_23, col2_25), INDEX (col2_12, col2_20 ASC, col2_14 ASC, col2_10 ASC, col1_13 ASC, col1_14, col2_19, col2_26 DESC, col2_22, col1_9 DESC, col2_18, col1_4 ASC, col2_24 ASC, col1_2, col2_9, col2_15 DESC, col2_11, col2_21 DESC, col2_13 DESC, col1_7, col2_8 DESC, col2_25 DESC, col1_10) STORING (col2_16) WHERE ((((table2.col2_11 < (-1):::DECIMAL) AND (table2.col1_8 < e'\x00':::STRING)) AND (table2.col1_9 = e'\U00002603':::STRING)) AND (table2.col2_12 < 1.401298464324817e-45:::FLOAT8)) OR (table2.col2_18 <= 'infinity':::DATE), FAMILY (col2_25), FAMILY (col2_11, col2_13, col1_4, col2_12), FAMILY (col2_15), FAMILY (col1_14), FAMILY (col2_14, col1_8), FAMILY (col2_18), FAMILY (col1_13), FAMILY (col2_17), FAMILY (col1_10, col2_9, col2_21, col2_24, col1_9, col2_22, col2_23, col2_8, col1_2, col1_7, col2_19, col2_10), FAMILY (col2_16, col2_20), FAMILY (col2_26)) INTERLEAVE IN PARENT table1 (col1_8, col1_2, col1_13, col1_10, col1_7, col1_9, col1_4, col1_14);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6597429832094807538, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7062742531618169949}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 789944878986428431, "histo_col_type": "", "name": "__auto__", "null_count": 293036896897836644, "row_count": 7062742531618169949}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7002309520914528379, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7062742531618169949}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1052157125580528189, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7062742531618169949}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2344457706777977820, "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 7062742531618169949}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5272782151198189076, "histo_buckets": [{"distinct_range": 0, "num_eq": 400, "num_range": 0, "upper_bound": "8"}, {"distinct_range": 239706804263866660, "num_eq": 154436672310410552, "num_range": 4025289398780887889, "upper_bound": "\U00002603"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 7062742531618169949}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4902343138504612511, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7062742531618169949}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4548747069445495661, "histo_col_type": "", "name": "__auto__", "null_count": 5583775085373135217, "row_count": 7062742531618169949}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3318418012299985436, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7062742531618169949}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 465415942207124502, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7062742531618169949}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4185920430091993348, "histo_col_type": "", "name": "__auto__", "null_count": 598917382856417464, "row_count": 7062742531618169949}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6715273483639575540, "histo_buckets": [{"distinct_range": 0, "num_eq": 300000000, "num_range": 0, "upper_bound": "2782409175"}, {"distinct_range": 4261043663464872400, "num_eq": 1000000000, "num_range": 5702207475420234433, "upper_bound": "3617031234"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 7062742531618169949}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6146589622840969720, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7062742531618169949}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 664917288137287015, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "000000000000000000000000000000000000000000"}, {"distinct_range": 0, "num_eq": 1134470534451434929, "num_range": 40000000000, "upper_bound": "000011000010000001000001100111010000001010"}, {"distinct_range": 754277591831494300, "num_eq": 10000000, "num_range": 851674581628059862, "upper_bound": "001001001111010011000110010011101000001010"}, {"distinct_range": 0, "num_eq": 2961623943074871482, "num_range": 445591514538874864, "upper_bound": "001101000010001001110011011000101100010011"}, {"distinct_range": 6000, "num_eq": 715004145571652361, "num_range": 6000, "upper_bound": "011111111111111111111111111111111111111111"}, {"distinct_range": 20717.975030917216, "num_eq": 20000, "num_range": 70000, "upper_bound": "110101000101110111100100010011111110111100"}], "histo_col_type": "BIT(42)", "name": "__auto__", "null_count": 3694018827960072784, "row_count": 7062742531618169949}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1246571018263555054, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "false"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 2232993388837009611, "row_count": 7062742531618169949}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1211745203732528616, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "8"}, {"distinct_range": 2000, "num_eq": 2193680072028231649, "num_range": 2000, "upper_bound": "LNQXK"}, {"distinct_range": 0, "num_eq": 5301201436874195831, "num_range": 40000000000, "upper_bound": "Suo"}, {"distinct_range": 2722.3318268910803, "num_eq": 100000, "num_range": 100000, "upper_bound": "k\\u001c`\\u0019v\\u0016"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 7062742531618169949}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 457714453, "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 729241290, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 432055138, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "0.5269103139469545"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 660212815, "histo_col_type": "", "name": "__auto__", "null_count": 68023164, "row_count": 800000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 361579913, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 417793187, "histo_col_type": "", "name": "__auto__", "null_count": 592078935, "row_count": 800000000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 663755746, "histo_col_type": "", "name": "__auto__", "null_count": 142306831, "row_count": 800000000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 591948621, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-21963"}, {"distinct_range": 5000000000, "num_eq": 600000, "num_range": 5000000000, "upper_bound": "-10038"}, {"distinct_range": 70, "num_eq": 400000000, "num_range": 70, "upper_bound": "8613"}, {"distinct_range": 12924.544879793084, "num_eq": 4000, "num_range": 100000, "upper_bound": "19525"}, {"distinct_range": 10000000000, "num_eq": 100, "num_range": 10000000000, "upper_bound": "20478"}, {"distinct_range": 8736.350742628509, "num_eq": 8272419422998877887, "num_range": 9000, "upper_bound": "31105"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 626460852, "row_count": 800000000}, {"columns": ["col2_26"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 366765913, "histo_col_type": "", "name": "__auto__", "null_count": 94884827, "row_count": 800000000}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 497664337, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 340404627, "histo_col_type": "", "name": "__auto__", "null_count": 219686871, "row_count": 800000000}, {"columns": ["col2_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 686365234, "histo_col_type": "", "name": "__auto__", "null_count": 132414188, "row_count": 800000000}, {"columns": ["col2_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 145622200, "histo_buckets": [{"distinct_range": 0, "num_eq": 7153847984296206341, "num_range": 0, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col2_25"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 637305819, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 722581778, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 739589475, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-812928299025394793"}, {"distinct_range": 6808414915861721000, "num_eq": 1000, "num_range": 6808414915861721211, "upper_bound": "-719568446944778446"}, {"distinct_range": 0, "num_eq": 5561711878946352956, "num_range": 700, "upper_bound": "1414520419206247157"}, {"distinct_range": 171477703.35742012, "num_eq": 3670525296511578428, "num_range": 500000000, "upper_bound": "3670476410558474319"}, {"distinct_range": 1027238834331330700, "num_eq": 8277513459955343268, "num_range": 6230811689478431860, "upper_bound": "4003772249605467044"}, {"distinct_range": 8200926161947546000, "num_eq": 6766183449404304756, "num_range": 8200926161947545760, "upper_bound": "4532833765581849821"}, {"distinct_range": 0, "num_eq": 6749625150219787711, "num_range": 100000000000, "upper_bound": "6847548359860525666"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col2_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 736684950, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col2_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 363461924, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 209214970, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 658529803, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 263324980, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "423809431"}, {"distinct_range": 1184334965625681200, "num_eq": 300, "num_range": 5965441322919863809, "upper_bound": "4053952258"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 374064028, "row_count": 800000000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 280902461, "histo_col_type": "", "name": "__auto__", "null_count": 540320689, "row_count": 800000000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 663832462, "histo_col_type": "", "name": "__auto__", "null_count": 313072692, "row_count": 800000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 352353193, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "613800581"}, {"distinct_range": 660736989512554100, "num_eq": 500000, "num_range": 660736989512554169, "upper_bound": "1512218461"}, {"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "3714999689"}, {"distinct_range": 70000000000, "num_eq": 70000, "num_range": 70000000000, "upper_bound": "3826071985"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 597926230, "histo_buckets": [{"distinct_range": 0, "num_eq": 6779278023504095805, "num_range": 0, "upper_bound": "\\u0016"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 10000000000, "upper_bound": "\\""}, {"distinct_range": 116774.22801785394, "num_eq": 3172687712407893151, "num_range": 300000, "upper_bound": "+"}, {"distinct_range": 8456965.26112901, "num_eq": 4764911288244914398, "num_range": 10000000, "upper_bound": "1"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 743225617, "row_count": 800000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 380999357, "histo_col_type": "", "name": "__auto__", "null_count": 200898040, "row_count": 800000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34132986, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('hfupd', 'riaxle', 'c', 'pgs', 'ofrh');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('xmbtbw', 'soix', 'lxrie', 'mhj', 'cq', 'aw');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('zhpt', 'se', 'jhky', 'bqelu');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('o');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2626248&tab=buildLog) on [master@86c619f768a4d6f4cc80efb5c9479e5fc0fd72c1](https://github.com/cockroachdb/cockroach/commits/86c619f768a4d6f4cc80efb5c9479e5fc0fd72c1):

```
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0);
        test_1        |             CREATE INDEX ON table3 (col3_2 DESC);
        test_1        |             CREATE INDEX ON table3 (col3_2 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 ASC, col3_2);
        test_1        |             CREATE INDEX ON table3 (col3_2 DESC, col3_0);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_2 ASC, col3_1 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_1 ASC, col3_0 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_2 DESC, col3_1 ASC) INCLUDE (col3_0);
        test_1        |             CREATE TABLE table4 (col4_0 BOX2D NOT NULL, PRIMARY KEY (col4_0));
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_0);
        test_1        |             CREATE INDEX ON table4 (col4_0 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_0 DESC);
        test_1        |             CREATE INDEX ON table4 (col4_0);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_0);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_0 ASC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('whjt');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (0.47s)
        test_1        |         compare_test.go:129: 
        test_1        |             		SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;
        test_1        |             		SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;
        test_1        |             	CREATE TABLE table1 (col1_0 INT4 NOT NULL, col1_1 BYTES NOT NULL, col1_2 TIMETZ NOT NULL, col1_3 NAME[] NOT NULL, col1_4 TIMESTAMPTZ NULL, PRIMARY KEY (col1_1, col1_2 ASC), UNIQUE (col1_4 DESC, col1_0, col1_2 ASC, col1_1) STORING (col1_3), UNIQUE (col1_2 DESC, col1_0 ASC, col1_4 DESC) STORING (col1_3) WHERE (table1.col1_2 < '00:00:00+15:59:00':::TIMETZ) OR (table1.col1_0 > 1:::INT8), UNIQUE (col1_4 ASC) STORING (col1_0));
        test_1        |             CREATE TABLE table2 (col1_1 BYTES NOT NULL, col1_2 TIMETZ NOT NULL, col2_2 BIT(26), col2_3 REGCLASS NOT NULL, col2_4 OID, col2_5 TIME NOT NULL, col2_6 TIMESTAMPTZ NOT NULL, col2_7 INET NOT NULL, col2_8 TIME, PRIMARY KEY (col1_1, col1_2 ASC, col2_5, col2_4 ASC, col2_3 ASC, col2_2 DESC, col2_7 ASC), INDEX (col1_2 DESC, col1_1, col2_6 DESC, col2_3 ASC, col2_5, col2_8 DESC, col2_7 DESC, col2_4 ASC), UNIQUE (col2_7 DESC, col2_4 DESC, col2_3 ASC, col2_5 DESC), UNIQUE (col2_4 DESC, col2_2, col1_2, col2_7 DESC, col1_1 ASC, col2_8), INDEX (col2_8, col1_1, col2_3 ASC, col2_7 ASC, col2_4 ASC, col2_5 DESC) STORING (col2_6), UNIQUE (col1_1) WHERE ((((table2.col2_6 >= '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ) AND (table2.col1_2 < '00:00:00+15:59:00':::TIMETZ)) OR (table2.col2_8 <= '00:00:00':::TIME)) AND (table2.col1_1 <= '\x00':::BYTES)) AND (table2.col2_5 < '24:00:00':::TIME), UNIQUE (col2_2 DESC, col2_5 ASC, col2_8 ASC, col1_2) STORING (col2_6) WHERE table2.col2_6 > '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ, UNIQUE (col2_5, col2_7 DESC, col2_2 DESC) STORING (col2_6), INDEX (col1_2 DESC, col2_6 DESC, col2_2 DESC, col2_7 ASC, col2_4 ASC, col2_5 ASC, col2_3 ASC, col1_1 DESC, col2_8 DESC) WHERE ((((table2.col1_2 <= '24:00:00-15:59:00':::TIMETZ) OR (table2.col2_8 = '24:00:00':::TIME)) AND (table2.col1_1 >= '\x22':::BYTES)) OR (table2.col2_5 <= '24:00:00':::TIME)) OR (table2.col2_6 > '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ), FAMILY (col2_7), FAMILY (col2_8, col2_6), FAMILY (col2_4), FAMILY (col2_3), FAMILY (col1_1), FAMILY (col2_5), FAMILY (col2_2), FAMILY (col1_2)) INTERLEAVE IN PARENT table1 (col1_1, col1_2);
        test_1        |             CREATE TABLE table3 (col3_0 REGPROC NOT NULL, col3_1 VARCHAR NOT NULL, col3_2 FLOAT4 NOT NULL, col3_3 BIT(3) NOT NULL, col3_4 TIMETZ NOT NULL, col3_5 INT4 NOT NULL, col3_6 INT8 NOT NULL, col3_7 OID NOT NULL, col3_8 INT4, col3_9 CHAR NOT NULL, col3_10 VARCHAR, col3_11 REGCLASS NOT NULL, col3_12 INT4 NULL, col3_13 UUID NOT NULL, col3_14 "char"[] NULL, col3_15 INT4 NOT NULL, PRIMARY KEY (col3_7, col3_0 DESC, col3_15 ASC, col3_2 DESC, col3_13 DESC, col3_6 ASC, col3_4 DESC, col3_11 DESC, col3_3 DESC, col3_5 ASC), INDEX (col3_6 ASC, col3_4 ASC, col3_11 ASC, col3_1, col3_12, col3_9 DESC) WHERE (((((((((table3.col3_1 = '"':::STRING) OR (table3.col3_15 != 0:::INT8)) OR (table3.col3_5 < 2147483647:::INT8)) AND (table3.col3_10 < e'\'':::STRING)) OR (table3.col3_4 < '00:00:00+15:59:00':::TIMETZ)) AND (table3.col3_6 <= (-128):::INT8)) AND (table3.col3_2 < 0.0:::FLOAT8)) AND (table3.col3_8 <= 1:::INT8)) OR (table3.col3_9 != '':::STRING)) OR (table3.col3_12 = (-1):::INT8), INDEX (col3_6, col3_4 ASC, col3_5 ASC, col3_3 ASC), UNIQUE (col3_15 ASC, col3_9, col3_6 ASC, col3_4 ASC, col3_5, col3_10, col3_11, col3_2, col3_0 DESC, col3_7, col3_1, col3_3 DESC, col3_13 DESC) STORING (col3_8, col3_14) WHERE (((((((((table3.col3_4 <= '24:00:00-15:59:00':::TIMETZ) AND (table3.col3_5 <= (-1):::INT8)) AND (table3.col3_1 >= 'X':::STRING)) OR (table3.col3_6 <= (-32768):::INT8)) AND (table3.col3_2 <= '+Inf':::FLOAT8)) OR (table3.col3_12 != (-32768):::INT8)) OR (table3.col3_8 < (-2147483648):::INT8)) AND (table3.col3_15 = (-32768):::INT8)) OR (table3.col3_10 <= '':::STRING)) OR (table3.col3_9 != 'X':::STRING), UNIQUE (col3_7, col3_4 ASC, col3_15, col3_0, col3_12), UNIQUE (col3_8 DESC, col3_15, col3_9 ASC, col3_2 DESC, col3_10 ASC, col3_6, col3_1 DESC, col3_12, col3_11 ASC, col3_7 ASC, col3_0 ASC, col3_3 ASC, col3_5, col3_13 DESC), INDEX (col3_0 DESC, col3_10 ASC, col3_3 DESC, col3_13 ASC, col3_12, col3_4 DESC, col3_11 ASC, col3_2 DESC, col3_9 DESC) STORING (col3_1));
        test_1        |             CREATE TABLE table4 (col1_1 BYTES NOT NULL, col1_2 TIMETZ NOT NULL, col2_5 TIME NOT NULL, col2_4 OID, col2_3 REGCLASS NOT NULL, col2_2 BIT(26), col2_7 INET NOT NULL, col4_7 INT8, col4_8 VARCHAR NULL, col4_9 BIT(12) NULL, col4_10 BIT(6) NOT NULL, col4_11 VARCHAR NULL, col4_12 CHAR NULL, col4_13 FLOAT8, col4_14 TIMESTAMP, col4_15 REGPROC, col4_16 REGPROC, col4_17 OID, col4_18 TIME NOT NULL, col4_19 INT8 NOT NULL, col4_20 INTERVAL, col4_21 TIME NULL, PRIMARY KEY (col1_1, col1_2 ASC, col2_5, col2_4 ASC, col2_3 ASC, col2_2 DESC, col2_7 ASC), UNIQUE (col4_7 ASC, col1_2 DESC) WHERE (table4.col2_5 != '24:00:00':::TIME) OR (table4.col4_7 = 127:::INT8), INDEX (col2_4 ASC, col4_7 ASC, col4_9 DESC, col4_18 DESC) STORING (col4_8, col4_12, col4_13, col4_17, col4_20, col4_21) WHERE (((((((((table4.col1_1 < '\x':::BYTES) AND (table4.col4_12 = '':::STRING)) OR (table4.col4_13 > 5e-324:::FLOAT8)) OR (table4.col4_8 != e'\x00':::STRING)) AND (table4.col4_11 = e'\x00':::STRING)) AND (table4.col2_5 >= '00:00:00':::TIME)) AND (table4.col4_14 <= '-4713-11-24 00:00:00':::TIMESTAMP)) AND (table4.col4_21 < '24:00:00':::TIME)) OR (table4.col4_19 > (-9223372036854775807):::INT8)) OR (table4.col1_2 > '00:00:00+15:59:00':::TIMETZ), INDEX (col4_11, col4_13 DESC, col2_3 ASC, col4_19, col4_18 ASC, col4_16 ASC, col4_17 DESC, col4_7 ASC, col4_15 DESC, col1_1, col2_2, col1_2, col4_21 DESC, col2_7 DESC, col4_14 DESC, col4_8 DESC, col2_5 DESC, col2_4, col4_12 DESC, col4_10, col4_20 DESC, col4_9 ASC), INDEX (col2_7 ASC, col4_7), INDEX (col4_7, col2_4 ASC, col4_11, col4_21 ASC, col2_7 DESC, col2_3, col4_10 ASC, col4_13, col4_20 ASC, col4_14, col4_16 DESC, col4_15, col4_17 DESC, col4_9 DESC, col2_2), UNIQUE (col1_2) STORING (col4_7, col4_8, col4_13, col4_14, col4_16, col4_17, col4_18), INDEX (col4_17 DESC, col4_8) WHERE (((((((((((table4.col4_13 < (-1.0):::FLOAT8) OR (table4.col2_5 = '00:00:00':::TIME)) AND (table4.col1_2 < '24:00:00-15:59:00':::TIMETZ)) OR (table4.col4_21 > '00:00:00':::TIME)) OR (table4.col4_19 >= (-9223372036854775807):::INT8)) OR (table4.col4_7 = 32767:::INT8)) OR (table4.col4_12 > 'X':::STRING)) OR (table4.col4_18 > '24:00:00':::TIME)) AND (table4.col4_11 >= e'\'':::STRING)) AND (table4.col4_14 < '294276-12-31 23:59:59.999999':::TIMESTAMP)) AND (table4.col1_1 <= '\x':::BYTES)) AND (table4.col4_8 >= 'X':::STRING), INDEX (col4_7 ASC) WHERE (((((((((((table4.col4_14 != '294276-12-31 23:59:59.999999':::TIMESTAMP) OR (table4.col4_21 != '00:00:00':::TIME)) OR (table4.col4_13 > '-Inf':::FLOAT8)) AND (table4.col4_11 > '"':::STRING)) OR (table4.col4_12 > '"':::STRING)) OR (table4.col1_2 != '00:00:00+15:59:00':::TIMETZ)) OR (table4.col4_7 > 127:::INT8)) OR (table4.col2_5 >= '24:00:00':::TIME)) OR (table4.col4_18 != '00:00:00':::TIME)) OR (table4.col4_19 != 0:::INT8)) AND (table4.col4_8 >= e'\'':::STRING)) AND (table4.col1_1 = '\xe29883':::BYTES)) INTERLEAVE IN PARENT table2 (col1_1, col1_2, col2_5, col2_4, col2_3, col2_2, col2_7);
        test_1        |             CREATE TABLE table5 (col3_7 OID NOT NULL, col3_0 REGPROC NOT NULL, col3_15 INT4 NOT NULL, col3_2 FLOAT4 NOT NULL, col3_13 UUID NOT NULL, col3_6 INT8 NOT NULL, col3_4 TIMETZ NOT NULL, col3_11 REGCLASS NOT NULL, col3_3 BIT(3) NOT NULL, col3_5 INT4 NOT NULL, col5_10 TIMESTAMP NOT NULL, col5_11 VARCHAR, col5_12 OID NULL, col5_13 TIMESTAMPTZ NULL, col5_14 REGCLASS NULL, col5_15 "char" NOT NULL, col5_16 BIT(35) NULL, col5_17 "char" NOT NULL, PRIMARY KEY (col3_7, col3_0 DESC, col3_15 ASC, col3_2 DESC, col3_13 DESC, col3_6 ASC, col3_4 DESC, col3_11 DESC, col3_3 DESC, col3_5 ASC, col5_11 ASC, col5_12 DESC, col5_15), INDEX (col3_0, col5_16 DESC, col3_7 ASC, col3_3 ASC, col3_13 DESC, col3_15 ASC, col3_11 ASC, col3_4, col5_11 DESC, col5_13 DESC, col5_14 ASC, col5_17 DESC, col5_12 ASC) WHERE ((((((table5.col3_5 > 0:::INT8) AND (table5.col5_11 = 'X':::STRING)) AND (table5.col3_2 < 1.0:::FLOAT8)) OR (table5.col5_15 > '"':::STRING)) OR (table5.col3_6 > (-2147483648):::INT8)) AND (table5.col5_10 <= '-2000-01-01 00:00:00':::TIMESTAMP)) AND (table5.col3_15 < (-1):::INT8), UNIQUE (col3_3, col5_15 DESC, col5_10, col3_4, col3_11), UNIQUE (col3_2 ASC, col5_14, col5_15 DESC) STORING (col5_13, col5_17) WHERE (((((((((table5.col3_4 != '00:00:00+15:59:00':::TIMETZ) AND (table5.col5_15 > e'\x00':::STRING)) AND (table5.col5_17 = 'X':::STRING)) OR (table5.col3_6 < 0:::INT8)) OR (table5.col3_2 <= '+Inf':::FLOAT8)) AND (table5.col5_13 < '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) OR (table5.col5_10 > '0001-01-01 00:00:00':::TIMESTAMP)) AND (table5.col5_11 > e'\x00':::STRING)) AND (table5.col3_5 > 1:::INT8)) OR (table5.col3_15 != (-32768):::INT8), UNIQUE (col5_11, col3_0 DESC, col3_6, col3_15 ASC, col5_15 ASC, col5_16 ASC, col3_13 ASC, col3_5 DESC) WHERE (((((((((table5.col3_15 >= 127:::INT8) OR (table5.col5_13 != '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) OR (table5.col3_4 = '24:00:00-15:59:00':::TIMETZ)) AND (table5.col3_6 = (-9223372036854775807):::INT8)) AND (table5.col5_11 < '"':::STRING)) AND (table5.col3_5 = (-1):::INT8)) OR (table5.col5_15 < '"':::STRING)) OR (table5.col5_10 != '0001-01-01 00:00:00':::TIMESTAMP)) OR (table5.col5_17 = 'X':::STRING)) OR (table5.col3_2 <= '+Inf':::FLOAT8)) INTERLEAVE IN PARENT table3 (col3_7, col3_0, col3_15, col3_2, col3_13, col3_6, col3_4, col3_11, col3_3, col3_5);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3751983326216152430, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 0, "num_eq": 70, "num_range": 8008264612080581212, "upper_bound": "\\\\x00"}, {"distinct_range": 549522818879391200, "num_eq": 1000000000, "num_range": 7849378343682551615, "upper_bound": "\\\\x4c0093f791"}, {"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": "\\\\x62ef6f4d72"}, {"distinct_range": 13682.79417675973, "num_eq": 10000000, "num_range": 30000, "upper_bound": "\\\\x7063e3505976de2727"}, {"distinct_range": 0, "num_eq": 8000, "num_range": 3001853570962421463, "upper_bound": "\\\\x9947d68a15a0877d1d"}, {"distinct_range": 2645763924482577000, "num_eq": 10000000, "num_range": 2645763924482576915, "upper_bound": "\\\\xff"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 6380666876073768689}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 84144953228539021, "histo_buckets": [{"distinct_range": 0, "num_eq": 3605465946655292999, "num_range": 0, "upper_bound": "16:33:41.837379+14:45:00"}, {"distinct_range": 0, "num_eq": 10, "num_range": 90000, "upper_bound": "14:40:57.360032+09:34:00"}, {"distinct_range": 0, "num_eq": 90000000000, "num_range": 564536492478979622, "upper_bound": "14:22:20.983218+07:17:00"}, {"distinct_range": 0, "num_eq": 6963990425491958928, "num_range": 50, "upper_bound": "00:49:54.126939-10:11:00"}, {"distinct_range": 0, "num_eq": 90000000000, "num_range": 200000, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 6380666876073768689}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2074177303173251277, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6380666876073768689}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 495996818282270296, "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 5245833829730397604, "row_count": 6380666876073768689}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5721260977952842395, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6380666876073768689}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6237046, "histo_buckets": [{"distinct_range": 0, "num_eq": 511917136819379245, "num_range": 0, "upper_bound": "12:40:36.990043+14:56:00"}, {"distinct_range": 0, "num_eq": 3353991555115801851, "num_range": 8298569018069783780, "upper_bound": "20:21:28.777378+07:57:00"}, {"distinct_range": 1761201881180679700, "num_eq": 1894814615970834760, "num_range": 3192122129393669693, "upper_bound": "11:41:51.776837-02:09:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4159133, "histo_buckets": [{"distinct_range": 0, "num_eq": 7433783675914566157, "num_range": 0, "upper_bound": "00000101110010001000011011"}, {"distinct_range": 700000000, "num_eq": 50000000, "num_range": 700000000, "upper_bound": "10001111111010111011010101"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "10011100101100101111110100"}, {"distinct_range": 10000000000, "num_eq": 30000, "num_range": 10000000000, "upper_bound": "10011110100010111100101001"}, {"distinct_range": 119491336847102850, "num_eq": 1729962721857661394, "num_range": 5232640804838311443, "upper_bound": "10110111010000000110010100"}, {"distinct_range": 1980858596380438000, "num_eq": 1000000000, "num_range": 9069681285393149681, "upper_bound": "11000110000100111111010111"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 5702845007062399395, "upper_bound": "11010101111010001000110100"}, {"distinct_range": 0, "num_eq": 30000, "num_range": 7826211415149225167, "upper_bound": "11111010111010101001010001"}], "histo_col_type": "BIT(26)", "name": "__auto__", "null_count": 4460852, "row_count": 8000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2633017, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 1572715256018634200, "num_eq": 6000, "num_range": 5270631154688405502, "upper_bound": "486631956"}, {"distinct_range": 323915853451884160, "num_eq": 7722207693467526780, "num_range": 1782225580856248880, "upper_bound": "1206793587"}, {"distinct_range": 1960902.9741659986, "num_eq": 3000000000, "num_range": 4000000, "upper_bound": "1410356675"}, {"distinct_range": 0, "num_eq": 2770466338820413926, "num_range": 30000, "upper_bound": "1563502109"}, {"distinct_range": 2434931899167208000, "num_eq": 3514014130823084689, "num_range": 2434931899167207937, "upper_bound": "2677654478"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 7993673, "row_count": 8000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1051090, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 795150, "histo_buckets": [{"distinct_range": 0, "num_eq": 1907435426974661536, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 4.963732788514685, "num_eq": 40, "num_range": 10, "upper_bound": "\\\\x23c7a70d02757e5e2a"}, {"distinct_range": 4592330953.151863, "num_eq": 4077659608459977662, "num_range": 6000000000, "upper_bound": "\\\\x2ae33d363c5aeea339"}, {"distinct_range": 4630912241201754000, "num_eq": 1959279620623524400, "num_range": 7885220347292964307, "upper_bound": "\\\\x3504de0360f8511c5f"}, {"distinct_range": 2961563489881778000, "num_eq": 4294844064087091092, "num_range": 8453142080327741390, "upper_bound": "\\\\x59"}, {"distinct_range": 0, "num_eq": 20, "num_range": 7605809607319398005, "upper_bound": "\\\\xce1f72ece50292f8ca"}, {"distinct_range": 2985906813986661000, "num_eq": 200000, "num_range": 7518582821432217347, "upper_bound": "\\\\xe67d4255b4ba2b"}, {"distinct_range": 20.605624010618964, "num_eq": 9000000000, "num_range": 30, "upper_bound": "\\\\xff"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 636026, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7916117, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000, "num_range": 0, "upper_bound": "03:57:45.92207"}, {"distinct_range": 1113764031332100500, "num_eq": 0, "num_range": 4831101323887575592, "upper_bound": "09:54:17.47868"}, {"distinct_range": 400, "num_eq": 10000000, "num_range": 400, "upper_bound": "16:10:02.829743"}, {"distinct_range": 2121343885280585000, "num_eq": 40000000000, "num_range": 2121343885280584973, "upper_bound": "21:35:46.278981"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7699103, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000000, "num_range": 0, "upper_bound": "40.84.218.70/14"}, {"distinct_range": 1000000000, "num_eq": 200000, "num_range": 1000000000, "upper_bound": "116.53.150.177/22"}, {"distinct_range": 10000, "num_eq": 100000000000, "num_range": 10000, "upper_bound": "10.252.236.114/23"}, {"distinct_range": 3599939359779078700, "num_eq": 7000000000, "num_range": 3896404330326909995, "upper_bound": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4295474, "histo_buckets": [{"distinct_range": 0, "num_eq": 800, "num_range": 0, "upper_bound": "05:55:28.012523"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 8200187682275103138, "upper_bound": "15:31:53.66669"}, {"distinct_range": 3948890.653021645, "num_eq": 7138154510607800013, "num_range": 6000000, "upper_bound": "16:38:12.127467"}, {"distinct_range": 313493714565330500, "num_eq": 40000000000, "num_range": 963677222410949672, "upper_bound": "16:40:38.183268"}, {"distinct_range": 50, "num_eq": 0, "num_range": 50, "upper_bound": "18:24:52.267021"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "19:31:20.571968"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 224231, "row_count": 8000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36385900, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 259182069, "histo_buckets": [{"distinct_range": 0, "num_eq": 1275317417247997669, "num_range": 0, "upper_bound": "-6141881556691273908"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 56731306, "histo_col_type": "", "name": "__auto__", "null_count": 84574820, "row_count": 300000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 48438937, "histo_buckets": [{"distinct_range": 0, "num_eq": 2965501660766517260, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 1662713308447673900, "num_eq": 4051395664418831307, "num_range": 5965297247212096245, "upper_bound": "1164323658"}, {"distinct_range": 7655590586225584000, "num_eq": 500000000, "num_range": 7655590586225583700, "upper_bound": "2003656004"}, {"distinct_range": 0, "num_eq": 200, "num_range": 4473595360746839436, "upper_bound": "2353138564"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 2000, "upper_bound": "3562385443"}, {"distinct_range": 62465020.475084156, "num_eq": 4660807131549764605, "num_range": 90000000, "upper_bound": "3920497338"}, {"distinct_range": 0, "num_eq": 7728786234037398941, "num_range": 3451550301663680290, "upper_bound": "3923180868"}, {"distinct_range": 0, "num_eq": 600000, "num_range": 90000, "upper_bound": "4085239118"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 115391034, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 37157791, "histo_buckets": [{"distinct_range": 0, "num_eq": 9091169229319375209, "num_range": 0, "upper_bound": "52004894"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 60254415, "histo_col_type": "", "name": "__auto__", "null_count": 254387975, "row_count": 300000000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 260366508, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000000, "num_range": 0, "upper_bound": "1100605127"}, {"distinct_range": 1554995912224956000, "num_eq": 70, "num_range": 3468797498829590409, "upper_bound": "1250374248"}, {"distinct_range": 4036077833511293400, "num_eq": 70000000, "num_range": 4036077833511293605, "upper_bound": "2175044584"}, {"distinct_range": 0, "num_eq": 392395578097951795, "num_range": 6608404874203998052, "upper_bound": "2181967748"}, {"distinct_range": 0, "num_eq": 4903653129322446567, "num_range": 6484269698181507522, "upper_bound": "3164683519"}, {"distinct_range": 0, "num_eq": 8510401765301935917, "num_range": 0, "upper_bound": "3252143384"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 80000000, "upper_bound": "3851256826"}, {"distinct_range": 0, "num_eq": 494419793652621943, "num_range": 70000000, "upper_bound": "4267925673"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2344985, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 263833952, "histo_buckets": [{"distinct_range": 0, "num_eq": 6285713797079237105, "num_range": 0, "upper_bound": "-1556947598"}, {"distinct_range": 31385164055.098545, "num_eq": 40000, "num_range": 50000000000, "upper_bound": "298768277"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "906755287"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 26383684, "row_count": 300000000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 75494358, "histo_col_type": "", "name": "__auto__", "null_count": 111020930, "row_count": 300000000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 157727056, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 40896066, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7519330, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34394761, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 102385748, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000000}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9877, "histo_buckets": [{"distinct_range": 0, "num_eq": 6438209925775351225, "num_range": 0, "upper_bound": "\\\\x06c3c923"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8163, "histo_buckets": [{"distinct_range": 0, "num_eq": 5456065050058366883, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 96927042997.75699, "num_eq": 3464542953658761964, "num_range": 100000000000, "upper_bound": "497807840"}, {"distinct_range": 0, "num_eq": 6015486833071169368, "num_range": 7141572867535749230, "upper_bound": "1308783624"}, {"distinct_range": 100000, "num_eq": 1000, "num_range": 100000, "upper_bound": "2571992096"}, {"distinct_range": 2862563708480360000, "num_eq": 2564147781530734424, "num_range": 3416173659094717325, "upper_bound": "3485799237"}, {"distinct_range": 4694931331904953000, "num_eq": 80000, "num_range": 4694931331904953015, "upper_bound": "3786874581"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 2850, "row_count": 10000}, {"columns": ["col4_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8935, "histo_col_type": "", "name": "__auto__", "null_count": 3439, "row_count": 10000}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9835, "histo_col_type": "", "name": "__auto__", "null_count": 5853, "row_count": 10000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2978, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000, "num_range": 0, "upper_bound": "02:03:27.978742+05:42:00"}, {"distinct_range": 0, "num_eq": 800000, "num_range": 70000000000, "upper_bound": "04:34:48.833327+04:01:00"}, {"distinct_range": 10000, "num_eq": 5000000000, "num_range": 10000, "upper_bound": "04:32:43.393726-00:10:00"}, {"distinct_range": 400, "num_eq": 4713749562738377684, "num_range": 400, "upper_bound": "23:14:50.462969+07:54:00"}, {"distinct_range": 10654.951848425564, "num_eq": 8547806302529451621, "num_range": 800000, "upper_bound": "10:16:11.448917-06:32:00"}, {"distinct_range": 0, "num_eq": 1700683529367185846, "num_range": 1000000000, "upper_bound": "23:14:32.993232-09:11:00"}, {"distinct_range": 1523098379982995700, "num_eq": 2625753458753062928, "num_range": 7043388833004180926, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2498, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6810, "histo_col_type": "", "name": "__auto__", "null_count": 2302, "row_count": 10000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8698, "histo_col_type": "INT8", "name": "__auto__", "null_count": 8099, "row_count": 10000}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8571, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2460, "histo_buckets": [{"distinct_range": 0, "num_eq": 1892048977020567330, "num_range": 0, "upper_bound": "(T\\"x\\u0014\\\\\\r"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 7569644104516861107, "upper_bound": "VZ`\\u0016"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 8698, "row_count": 10000}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1261, "histo_buckets": [{"distinct_range": 0, "num_eq": 9121698806694304730, "num_range": 0, "upper_bound": "3559608434"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 6712, "row_count": 10000}, {"columns": ["col4_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5446, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5908, "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9906, "histo_col_type": "", "name": "__auto__", "null_count": 3522, "row_count": 10000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7730, "histo_col_type": "", "name": "__auto__", "null_count": 2097, "row_count": 10000}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3972, "histo_col_type": "", "name": "__auto__", "null_count": 6633, "row_count": 10000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5336, "histo_col_type": "", "name": "__auto__", "null_count": 9656, "row_count": 10000}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8580, "histo_col_type": "", "name": "__auto__", "null_count": 7558, "row_count": 10000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 997, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 824, "histo_col_type": "", "name": "__auto__", "null_count": 5174, "row_count": 10000}, {"columns": ["col4_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1444, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col4_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 309, "histo_col_type": "", "name": "__auto__", "null_count": 9551, "row_count": 10000}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS e'[{"columns": ["col5_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22, "histo_col_type": "", "name": "__auto__", "null_count": 31, "row_count": 40}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11, "histo_buckets": [{"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 3629334182713422000, "num_eq": 100000, "num_range": 5041049304069800584, "upper_bound": "198937813"}, {"distinct_range": 212522856410356600, "num_eq": 90000000, "num_range": 1367051133962240405, "upper_bound": "526495422"}, {"distinct_range": 38394242198.02326, "num_eq": 800000, "num_range": 100000000000, "upper_bound": "1376306427"}, {"distinct_range": 2090839720992778800, "num_eq": 8000, "num_range": 7159639291093863565, "upper_bound": "1528399166"}, {"distinct_range": 1741807.4408971916, "num_eq": 800000, "num_range": 7000000, "upper_bound": "2296114066"}, {"distinct_range": 1275892677688660000, "num_eq": 600000, "num_range": 5266922274050704584, "upper_bound": "4215907606"}, {"distinct_range": 336.6428119408814, "num_eq": 500, "num_range": 700, "upper_bound": "4231093401"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 40}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000000, "num_range": 0, "upper_bound": "-0.5378897190093994"}, {"distinct_range": 7000, "num_eq": 100000000, "num_range": 7000, "upper_bound": "3.4028234663852886e+38"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 40}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40}, {"columns": ["col5_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 235950220.5051463, "num_eq": 100000000, "num_range": 300000000, "upper_bound": "\\u0005\\u0015}:ct\\u0001$$"}, {"distinct_range": 2902791574135617500, "num_eq": 4198038825103391437, "num_range": 2902791574135617453, "upper_bound": "d-ah\\u001fd;"}, {"distinct_range": 1786527827410251300, "num_eq": 9000000, "num_range": 3948438698151283788, "upper_bound": "sO\\u0011+"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 15, "row_count": 40}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29, "histo_col_type": "BIT(3)", "name": "__auto__", "null_count": 0, "row_count": 40}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40}, {"columns": ["col5_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23, "histo_col_type": "", "name": "__auto__", "null_count": 11, "row_count": 40}, {"columns": ["col5_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40}, {"columns": ["col5_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35, "histo_col_type": "", "name": "__auto__", "null_count": 13, "row_count": 40}, {"columns": ["col5_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15, "histo_buckets": [{"distinct_range": 0, "num_eq": 7983451113218472368, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 80000000000, "num_eq": 60000000, "num_range": 80000000000, "upper_bound": "63876119"}, {"distinct_range": 4818245826345657000, "num_eq": 6623912096480060, "num_range": 7736331837564580993, "upper_bound": "838927084"}, {"distinct_range": 0, "num_eq": 2403966113839946575, "num_range": 1211588790837968904, "upper_bound": "955365979"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 4361763553390115753, "upper_bound": "2529255090"}, {"distinct_range": 3180712287143708700, "num_eq": 3000, "num_range": 7148605548514819334, "upper_bound": "2862516461"}, {"distinct_range": 10000000000, "num_eq": 90000000, "num_range": 10000000000, "upper_bound": "3104593059"}, {"distinct_range": 90000000000, "num_eq": 8066187896546726603, "num_range": 90000000000, "upper_bound": "3447904506"}, {"distinct_range": 0, "num_eq": 8146146730618199580, "num_range": 8000, "upper_bound": "4289655208"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 40}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40}, {"columns": ["col5_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40}, {"columns": ["col5_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30, "histo_col_type": "", "name": "__auto__", "null_count": 28, "row_count": 40}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('mzhpej', 'utymxp', 'j', 'hsoej', 'c');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('q');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('cb', 'ujb', 'nkwo', 'pvytv', 'qxli', 'b');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('czzxjf', 'nubl', 'pdqbkn', 'ae');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2629402&tab=buildLog) on [master@171cc86d6d84c110197485b5d886d77cddc12399](https://github.com/cockroachdb/cockroach/commits/171cc86d6d84c110197485b5d886d77cddc12399):

```
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_10, col3_15 DESC, col3_6 DESC, col3_9, col3_14, col3_1 DESC, col3_2, col3_8 DESC, col3_11 ASC, col3_16, col3_12);
        test_1        |             CREATE INDEX ON table3 (col3_8 DESC, col3_18) STORING (col3_0, col3_3, col3_5, col3_7, col3_11, col3_14, col3_15, col3_16);
        test_1        |             CREATE INDEX ON table3 (col3_12 DESC, col3_17 DESC, col3_0 ASC, col3_2, col3_3, col3_9, col3_16, col3_1, col3_18 DESC, col3_15, col3_7);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_2, col3_18);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_9, col3_0);
        test_1        |             CREATE TABLE table4 (col2_8 DATE NOT NULL, col2_15 BYTES NOT NULL, col2_10 VARCHAR NOT NULL, col2_6 BOX2D NOT NULL, col2_5 BIT(42) NOT NULL, col2_9 DECIMAL NOT NULL, col2_0 INT2 NOT NULL, col4_7 INET NOT NULL, col4_8 REGTYPE NOT NULL, col4_9 REGCLASS NOT NULL, col4_10 BIT(7), col4_11 UUID NOT NULL, col4_12 INTERVAL, col4_13 REGPROC, col4_14 BIT(46) NOT NULL, col4_15 REGNAMESPACE, col4_16 FLOAT8 NOT NULL, PRIMARY KEY (col2_8, col2_15, col2_10, col2_6, col2_5, col2_9, col2_0), FAMILY (col2_10, col2_8), FAMILY (col4_13, col4_7, col4_16), FAMILY (col4_8), FAMILY (col2_6, col2_9), FAMILY (col4_9, col2_0, col2_15, col4_10), FAMILY (col4_11, col4_15), FAMILY (col4_12), FAMILY (col2_5, col4_14)) INTERLEAVE IN PARENT table2 (col2_8, col2_15, col2_10, col2_6, col2_5, col2_9, col2_0);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_13 DESC, col4_12, col2_9, col2_10, col4_8) STORING (col4_10, col4_14, col4_15);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col2_0, col2_15);
        test_1        |             CREATE TABLE table5 (col3_1 INT2 NOT NULL, col3_12 INT8 NOT NULL, col5_2 INT4, col5_3 "char" NULL, col5_4 BIT(3) NOT NULL, col5_5 TIMETZ NULL, PRIMARY KEY (col3_1, col3_12)) INTERLEAVE IN PARENT table3 (col3_1, col3_12);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col3_1, col5_3 ASC, col5_5 ASC) STORING (col5_4);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_4, col5_3);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10240055, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34440402, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 48992218, "histo_col_type": "", "name": "__auto__", "null_count": 23915630, "row_count": 90000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24851238, "histo_buckets": [{"distinct_range": 0, "num_eq": 500000000, "num_range": 0, "upper_bound": "00000"}, {"distinct_range": 37634772.59940575, "num_eq": 1335121734128841300, "num_range": 70000000, "upper_bound": "00101"}, {"distinct_range": 300000, "num_eq": 0, "num_range": 300000, "upper_bound": "01001"}, {"distinct_range": 15.696178899211512, "num_eq": 3612020187236495673, "num_range": 80, "upper_bound": "10011"}, {"distinct_range": 0, "num_eq": 3811809731039713686, "num_range": 0, "upper_bound": "10100"}], "histo_col_type": "BIT(5)", "name": "__auto__", "null_count": 0, "row_count": 90000000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9537746, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4642645, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 77835133, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000, "num_range": 0, "upper_bound": "00001111111001011100000011101010100111"}, {"distinct_range": 10000, "num_eq": 9000000, "num_range": 10000, "upper_bound": "00010010101011011000100001110001111111"}, {"distinct_range": 0, "num_eq": 402204310280954764, "num_range": 4000, "upper_bound": "11001111011000100100111110111001110100"}, {"distinct_range": 0, "num_eq": 5000000000, "num_range": 7217252842669945737, "upper_bound": "11111011000011110111011001010110111010"}], "histo_col_type": "BIT(38)", "name": "__auto__", "null_count": 55754588, "row_count": 90000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 72897130, "histo_buckets": [{"distinct_range": 0, "num_eq": 4768050219092762171, "num_range": 0, "upper_bound": "245309158"}, {"distinct_range": 869212576161567700, "num_eq": 30000000, "num_range": 2531313630557336815, "upper_bound": "1470170953"}, {"distinct_range": 43737120.758041285, "num_eq": 700, "num_range": 60000000, "upper_bound": "1743316679"}, {"distinct_range": 128540499360351380, "num_eq": 300000, "num_range": 128540499360351383, "upper_bound": "1915640491"}, {"distinct_range": 0, "num_eq": 4576923000583779974, "num_range": 5694590169063295018, "upper_bound": "2447879734"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 9000000000, "upper_bound": "2969657535"}, {"distinct_range": 2463443552344169000, "num_eq": 3705113638490902700, "num_range": 8275459510393887840, "upper_bound": "4063327616"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 90000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15195077, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35107477, "histo_buckets": [{"distinct_range": 0, "num_eq": 500, "num_range": 0, "upper_bound": "00:00:00"}, {"distinct_range": 9000, "num_eq": 80000, "num_range": 9000, "upper_bound": "01:03:10.016533"}, {"distinct_range": 1369279510848751400, "num_eq": 3832198176756905192, "num_range": 1369279510848751425, "upper_bound": "08:41:51.596525"}, {"distinct_range": 700, "num_eq": 700000000, "num_range": 700, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 90000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 74284059, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000, "num_range": 0, "upper_bound": "-6217549494031114178"}, {"distinct_range": 5837423295688259000, "num_eq": 4559458192091542280, "num_range": 6170252140149443345, "upper_bound": "-3843108637172624832"}, {"distinct_range": 0, "num_eq": 4360526430504187431, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 179341163655291300, "num_eq": 4407038547172915189, "num_range": 342900360949621536, "upper_bound": "1617568668452871292"}, {"distinct_range": 741454975054645200, "num_eq": 4425156066904347882, "num_range": 741454975054645184, "upper_bound": "2136401891732964155"}, {"distinct_range": 0, "num_eq": 300000000, "num_range": 200, "upper_bound": "2865439536334572404"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 90000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 60236428, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 59155175, "histo_col_type": "", "name": "__auto__", "null_count": 63077930, "row_count": 90000000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9095246, "histo_col_type": "", "name": "__auto__", "null_count": 65032079, "row_count": 90000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46879540, "histo_col_type": "", "name": "__auto__", "null_count": 5847545, "row_count": 90000000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12659846, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 78386887, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39254, "histo_col_type": "", "name": "__auto__", "null_count": 9096, "row_count": 40000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21949, "histo_col_type": "", "name": "__auto__", "null_count": 9008, "row_count": 40000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8630, "histo_col_type": "", "name": "__auto__", "null_count": 22089, "row_count": 40000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25341, "histo_col_type": "", "name": "__auto__", "null_count": 6771, "row_count": 40000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20245, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 37310, "histo_buckets": [{"distinct_range": 0, "num_eq": 1636176414325756842, "num_range": 0, "upper_bound": "-8172282126826413460"}, {"distinct_range": 4321219212508866600, "num_eq": 4235359691166647046, "num_range": 5906392172457280806, "upper_bound": "-3182196098671459735"}, {"distinct_range": 20000, "num_eq": 7000000000, "num_range": 20000, "upper_bound": "-1142475696313967233"}, {"distinct_range": 1215834271412575700, "num_eq": 1000000000, "num_range": 1215834271412575812, "upper_bound": "6769692668252907537"}, {"distinct_range": 0, "num_eq": 500000, "num_range": 3000000, "upper_bound": "9223372036854775807"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14622, "histo_col_type": "", "name": "__auto__", "null_count": 9316, "row_count": 40000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29993, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20729, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3883, "histo_buckets": [{"distinct_range": 0, "num_eq": 50816633897968959, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 2759610.086030734, "num_eq": 1303825054769951865, "num_range": 4000000, "upper_bound": "BOX(-1.7404151099270169 -0.7880080329304815,-0.4316332137165906 -0.08676797021029858)"}, {"distinct_range": 0, "num_eq": 30000000, "num_range": 2000000, "upper_bound": "BOX(-0.47041605869184105 0.39933954658919935,0.6945772209330814 0.46778245389304796)"}, {"distinct_range": 0, "num_eq": 7935886584753725654, "num_range": 600, "upper_bound": "BOX(-0.29340851067708673 -0.7156041909823969,0.3479338098019521 -0.15156649413708756)"}, {"distinct_range": 0, "num_eq": 2603761174938858768, "num_range": 300000000, "upper_bound": "BOX(-0.2884904750164059 -2.4298355662027076,0.8768292157454606 -1.064733377996185)"}, {"distinct_range": 2540664299916822000, "num_eq": 200000000, "num_range": 7776432626304596209, "upper_bound": "BOX(-0.1774632238100391 0.14004206004893938,1.610005577691984 0.6661962350189934)"}, {"distinct_range": 0, "num_eq": 3931996294021606462, "num_range": 0, "upper_bound": "BOX(0.7237477790756248 -0.17583742089550103,1.5468865093884723 2.229236315678022)"}, {"distinct_range": 8000, "num_eq": 900, "num_range": 8000, "upper_bound": "BOX(0.813478980831603 -0.5907611918856762,1.1609556513606512 0.6932337054504865)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9688, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "1970-12-14"}, {"distinct_range": 7708199099489375000, "num_eq": 3579794959108928902, "num_range": 7708199099489374920, "upper_bound": "1973-05-17"}, {"distinct_range": 6821511870715142000, "num_eq": 1434466792961108011, "num_range": 8896366221995384766, "upper_bound": "1974-09-26"}, {"distinct_range": 0, "num_eq": 4326665142013970370, "num_range": 0, "upper_bound": "1974-11-20"}, {"distinct_range": 379930050687746900, "num_eq": 0, "num_range": 411201568328454810, "upper_bound": "1976-04-29"}, {"distinct_range": 76806047765.63028, "num_eq": 700000000, "num_range": 90000000000, "upper_bound": "1977-08-27"}, {"distinct_range": 5162101964967865000, "num_eq": 7199513886382831635, "num_range": 6801138061961515888, "upper_bound": "1980-11-22"}, {"distinct_range": 0, "num_eq": 3115503982162560968, "num_range": 7428990144448661266, "upper_bound": "1988-09-02"}, {"distinct_range": 3475.6757597670166, "num_eq": 419905201139296022, "num_range": 7000, "upper_bound": "1995-08-07"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24478, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10518, "histo_col_type": "", "name": "__auto__", "null_count": 8987, "row_count": 40000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34420, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14939, "histo_col_type": "", "name": "__auto__", "null_count": 22823, "row_count": 40000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38051, "histo_col_type": "", "name": "__auto__", "null_count": 19031, "row_count": 40000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 489552167159870570, "histo_col_type": "", "name": "__auto__", "null_count": 5523865462760872065, "row_count": 7633443861450572658}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7492612153851928478, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7633443861450572658}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2431340813879209892, "histo_col_type": "", "name": "__auto__", "null_count": 5870226570626701322, "row_count": 7633443861450572658}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5663955830951632255, "histo_col_type": "", "name": "__auto__", "null_count": 1369566047946232070, "row_count": 7633443861450572658}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4920676903977357700, "histo_col_type": "", "name": "__auto__", "null_count": 910005578661436777, "row_count": 7633443861450572658}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6277022141663604500, "histo_col_type": "", "name": "__auto__", "null_count": 972600843254432784, "row_count": 7633443861450572658}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7255925306881281861, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7633443861450572658}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7587426756619402339, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7633443861450572658}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5624886398268009468, "histo_buckets": [{"distinct_range": 0, "num_eq": 6169767544050813774, "num_range": 0, "upper_bound": "-24126"}, {"distinct_range": 1495480521128140300, "num_eq": 1000, "num_range": 1495480521128140378, "upper_bound": "-16373"}, {"distinct_range": 0, "num_eq": 8900431105096434311, "num_range": 2412987480805538164, "upper_bound": "-9123"}, {"distinct_range": 0, "num_eq": 200000000, "num_range": 4630510473308396993, "upper_bound": "-7922"}, {"distinct_range": 6745662.526921711, "num_eq": 395687126404427645, "num_range": 8000000, "upper_bound": "-1"}, {"distinct_range": 10.874524045463746, "num_eq": 6000000, "num_range": 30, "upper_bound": "18819"}, {"distinct_range": 1244333154.9845748, "num_eq": 2626513717963990269, "num_range": 6000000000, "upper_bound": "29033"}, {"distinct_range": 78260508.02459612, "num_eq": 300, "num_range": 100000000, "upper_bound": "32767"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 7633443861450572658}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6772336459514639949, "histo_col_type": "", "name": "__auto__", "null_count": 3534944061535755465, "row_count": 7633443861450572658}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1533496956146031580, "histo_col_type": "", "name": "__auto__", "null_count": 7449418074941534904, "row_count": 7633443861450572658}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7079083894528609285, "histo_col_type": "", "name": "__auto__", "null_count": 4168481135638337397, "row_count": 7633443861450572658}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3150776006132774761, "histo_col_type": "", "name": "__auto__", "null_count": 5023285286847613392, "row_count": 7633443861450572658}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6510355464626063693, "histo_col_type": "", "name": "__auto__", "null_count": 3363364552244451711, "row_count": 7633443861450572658}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7106357797353950062, "histo_col_type": "", "name": "__auto__", "null_count": 6263066604051120459, "row_count": 7633443861450572658}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7434016792574494003, "histo_col_type": "", "name": "__auto__", "null_count": 4431227034005108717, "row_count": 7633443861450572658}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1347929586000157746, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 610885792173903100, "num_eq": 6904880067669967246, "num_range": 610885792173903060, "upper_bound": "1445373504"}, {"distinct_range": 6799667240797107000, "num_eq": 1833256475695928599, "num_range": 6799667240797107299, "upper_bound": "1624231258"}, {"distinct_range": 600000, "num_eq": 70000000, "num_range": 600000, "upper_bound": "3275989241"}, {"distinct_range": 0, "num_eq": 6066050872021693888, "num_range": 7000000, "upper_bound": "3643144774"}, {"distinct_range": 1476186984912219100, "num_eq": 5509377103251842123, "num_range": 1476186984912219180, "upper_bound": "3682675510"}, {"distinct_range": 900000, "num_eq": 200, "num_range": 900000, "upper_bound": "4174859257"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 2772861633615916329, "row_count": 7633443861450572658}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5211559250432504208, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000, "num_range": 0, "upper_bound": "2519748530"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 3831851528951116839, "row_count": 7633443861450572658}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7216644678381338235, "histo_buckets": [{"distinct_range": 0, "num_eq": 1850013659931931754, "num_range": 0, "upper_bound": "-6020787474756037528"}, {"distinct_range": 0, "num_eq": 5800335823654677616, "num_range": 4975140557397307004, "upper_bound": "-5439421726441937948"}, {"distinct_range": 323510.2917450744, "num_eq": 3527672492130722780, "num_range": 1000000, "upper_bound": "-32768"}, {"distinct_range": 1000000, "num_eq": 8748327223607164413, "num_range": 1000000, "upper_bound": "2106244349477714489"}, {"distinct_range": 731356385470152200, "num_eq": 1152619370464927349, "num_range": 2291500604896633500, "upper_bound": "2999255190057716232"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 7633443861450572658}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1095095520451598506, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "4714-11-24 BC"}, {"distinct_range": 5181942994527562000, "num_eq": 100000000000, "num_range": 5539495016264613294, "upper_bound": "1970-03-19"}, {"distinct_range": 100000000, "num_eq": 400000000, "num_range": 100000000, "upper_bound": "1972-10-09"}, {"distinct_range": 30, "num_eq": 6621894827698365623, "num_range": 30, "upper_bound": "1979-08-07"}, {"distinct_range": 442124956.8770482, "num_eq": 2703509403258847337, "num_range": 500000000, "upper_bound": "1985-01-24"}, {"distinct_range": 0, "num_eq": 1723488995877924079, "num_range": 0, "upper_bound": "1991-11-05"}, {"distinct_range": 10000000000, "num_eq": 10000000, "num_range": 10000000000, "upper_bound": "5874897-12-31"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 5865216759902529091}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1597017785769883660, "histo_col_type": "", "name": "__auto__", "null_count": 2641584194209895669, "row_count": 5865216759902529091}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3266469696835169602, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5865216759902529091}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 481218695515488865, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5865216759902529091}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3578373120106888236, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5865216759902529091}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4491724431301951492, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5865216759902529091}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2405130000798669652, "histo_col_type": "", "name": "__auto__", "null_count": 1007087788982880617, "row_count": 5865216759902529091}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 688863021726163295, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5865216759902529091}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1383214121251828773, "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 3717230778920201173, "row_count": 5865216759902529091}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4461525581985023339, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5865216759902529091}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3030188980858330716, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5865216759902529091}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2362468172172177198, "histo_col_type": "", "name": "__auto__", "null_count": 3729887976298925216, "row_count": 5865216759902529091}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3441319209218773592, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5865216759902529091}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1186996815574270553, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5865216759902529091}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5567920770244717034, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5865216759902529091}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 857734434677927195, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5865216759902529091}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3026094795135487774, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5865216759902529091}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 482, "histo_col_type": "", "name": "__auto__", "null_count": 190, "row_count": 1000}, {"columns": ["col5_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 349, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col5_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 558, "histo_col_type": "", "name": "__auto__", "null_count": 703, "row_count": 1000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 786, "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 625, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 662, "histo_col_type": "", "name": "__auto__", "null_count": 160, "row_count": 1000}]':::JSONB;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_13, col2_9) REFERENCES table3 (col3_2, col3_18);
        test_1        |             ALTER TABLE table4 ADD FOREIGN KEY (col4_14, col2_10) REFERENCES table5 (col5_4, col5_3) ON DELETE CASCADE;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_8, col2_5) REFERENCES table3 (col3_9, col3_0) ON DELETE CASCADE;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_7, col2_15) REFERENCES table4 (col2_0, col2_15) ON UPDATE RESTRICT;
        test_1        |             ALTER TABLE table5 ADD FOREIGN KEY (col5_4) REFERENCES table1 (col1_3) ON DELETE CASCADE;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('nstgnz', 'sqax', 'srtti', 'hbw', 'zbmdly');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('puk');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        |     --- FAIL: TestCompare/mutators (0.31s)
        test_1        |         compare_test.go:129: 
        test_1        |             		SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;
        test_1        |             		SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;
        test_1        |             	CREATE TABLE table1 (col1_0 REGTYPE NOT NULL, col1_1 INET NOT NULL, col1_2 BIT(20) NOT NULL, col1_3 INT8 NOT NULL, col1_4 TIME NOT NULL, col1_5 INT8 NOT NULL, col1_6 REGNAMESPACE, col1_7 TIMESTAMPTZ NOT NULL, col1_8 INT8[] NULL, PRIMARY KEY (col1_3 DESC, col1_2, col1_4, col1_7 ASC, col1_1 DESC), INDEX (col1_2, col1_1, col1_6 ASC, col1_7 DESC, col1_3 DESC) WHERE (((table1.col1_3 > (-2147483648):::INT8) OR (table1.col1_5 > 32767:::INT8)) OR (table1.col1_4 <= '00:00:00':::TIME)) AND (table1.col1_7 < '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ), INDEX (col1_7 ASC, col1_0 ASC, col1_2 ASC) WHERE (((table1.col1_5 > 0:::INT8) OR (table1.col1_4 <= '24:00:00':::TIME)) AND (table1.col1_7 != '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) OR (table1.col1_3 < 1:::INT8));
        test_1        |             CREATE TABLE table2 (col1_3 INT8 NOT NULL, col1_2 BIT(20) NOT NULL, col1_4 TIME NOT NULL, col1_7 TIMESTAMPTZ NOT NULL, col1_1 INET NOT NULL, col2_5 BIT(23) NOT NULL, col2_6 "char" NULL, col2_7 STRING NULL, col2_8 TIMESTAMPTZ, col2_9 TIMETZ NULL, col2_10 BIT(41), col2_11 FLOAT4 NULL, col2_12 CHAR NOT NULL, col2_13 REGPROCEDURE, col2_14 BOX2D NOT NULL, col2_15 TIMETZ NOT NULL, col2_16 BIT(15), col2_17 TIME, col2_18 INTERVAL NULL, col2_19 REGCLASS NULL, col2_20 TIMESTAMPTZ NOT NULL, PRIMARY KEY (col1_3 DESC, col1_2, col1_4, col1_7 ASC, col1_1 DESC, col2_17 DESC, col2_16 DESC, col2_19, col2_18 ASC, col2_14 DESC, col2_5 ASC, col2_9, col2_11, col2_20), INDEX (col2_11, col1_3, col2_20 DESC, col2_17 DESC, col2_9, col2_13 ASC, col2_8, col2_18 ASC, col1_2, col2_7 ASC, col2_19 DESC, col2_6 DESC, col2_10, col2_12 DESC), UNIQUE (col2_17 DESC, col2_10 ASC, col1_2, col2_9 DESC, col2_8 DESC, col2_16 ASC, col1_3, col1_7 DESC, col2_19 ASC, col2_15, col2_6 ASC, col2_13) WHERE (((((((((((table2.col1_3 < (-2147483648):::INT8) OR (table2.col2_15 >= '00:00:00+15:59:00':::TIMETZ)) OR (table2.col2_17 != '24:00:00':::TIME)) AND (table2.col1_7 = '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table2.col2_20 >= '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (table2.col2_6 > e'\'':::STRING)) OR (table2.col2_8 = '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table2.col2_12 >= 'X':::STRING)) AND (table2.col2_9 < '24:00:00-15:59:00':::TIMETZ)) OR (table2.col1_4 >= '24:00:00':::TIME)) AND (table2.col2_11 <= 1.401298464324817e-45:::FLOAT8)) OR (table2.col2_7 > '"':::STRING), INDEX (col1_3 DESC, col2_19 DESC, col2_13 ASC, col1_1, col2_6, col1_7), INDEX (col2_13 DESC, col1_3 ASC), UNIQUE (col2_13, col2_9 ASC, col2_7, col2_14 DESC, col1_1, col1_7, col1_2, col1_4, col2_6 DESC, col2_11, col2_5 DESC, col2_8 ASC, col2_19 DESC, col2_18 ASC, col2_10 DESC, col2_15, col2_17, col1_3 ASC, col2_16) WHERE (((((((table2.col1_7 < '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ) OR (table2.col1_4 >= '24:00:00':::TIME)) AND (table2.col2_6 <= e'\'':::STRING)) OR (table2.col2_17 < '24:00:00':::TIME)) OR (table2.col2_7 = e'\'':::STRING)) OR (table2.col2_8 <= '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table2.col2_12 >= e'\'':::STRING)) OR (table2.col2_11 != '+Inf':::FLOAT8), UNIQUE (col2_8, col1_1 ASC, col1_2 ASC, col2_9 ASC, col2_10 ASC, col2_5 ASC, col2_15, col1_7 DESC, col2_13), UNIQUE (col2_12, col1_7 ASC, col2_10 DESC, col2_9, col2_5 ASC, col2_18 ASC, col2_14 DESC, col2_11 ASC, col1_2 DESC, col2_8, col2_19 DESC, col1_3 DESC, col2_20, col2_15, col2_17 ASC, col2_6 DESC, col1_4 ASC, col2_16) STORING (col2_13) WHERE (((((((((((table2.col2_15 = '24:00:00-15:59:00':::TIMETZ) AND (table2.col2_7 < '"':::STRING)) AND (table2.col1_4 > '00:00:00':::TIME)) OR (table2.col2_9 != '00:00:00+15:59:00':::TIMETZ)) AND (table2.col2_17 >= '24:00:00':::TIME)) OR (table2.col2_12 >= '"':::STRING)) AND (table2.col2_6 >= '':::STRING)) AND (table2.col2_8 != '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) AND (table2.col1_3 = 0:::INT8)) AND (table2.col2_20 <= '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table2.col1_7 >= '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table2.col2_11 >= 0.0:::FLOAT8), FAMILY (col2_14, col1_3, col2_15, col2_9, col2_19, col2_7, col2_5), FAMILY (col1_7, col2_10, col2_20), FAMILY (col2_16), FAMILY (col2_13), FAMILY (col2_12, col1_2), FAMILY (col2_6, col2_8), FAMILY (col2_17, col2_11, col1_1, col2_18, col1_4)) INTERLEAVE IN PARENT table1 (col1_3, col1_2, col1_4, col1_7, col1_1);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36915292, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 76242264, "histo_col_type": "BIT(20)", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 49985517, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "5885796290765046630"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 57610075, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 440665, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 97649438, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35846250, "histo_col_type": "", "name": "__auto__", "null_count": 87086326, "row_count": 100000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 41682962, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "1973-01-26 03:45:41.000007+00:00"}, {"distinct_range": 7418322325956553000, "num_eq": 50000000000, "num_range": 7563150348344325700, "upper_bound": "2018-10-29 06:25:34.000874+00:00"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 2000000, "upper_bound": "2025-12-17 22:36:13.000303+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23455486, "histo_col_type": "", "name": "__auto__", "null_count": 99768280, "row_count": 100000000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22, "histo_col_type": "", "name": "__auto__", "null_count": 20, "row_count": 50}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35, "histo_col_type": "", "name": "__auto__", "null_count": 21, "row_count": 50}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000000, "num_range": 0, "upper_bound": "-0.6533978581428528"}, {"distinct_range": 2363966271114472000, "num_eq": 6150357105358052702, "num_range": 2363966271114471801, "upper_bound": "0.2824713885784149"}, {"distinct_range": 2190276087523926300, "num_eq": 900000000, "num_range": 2382185530995945099, "upper_bound": "0.4052334725856781"}, {"distinct_range": 605.3340986519585, "num_eq": 9040536723467206213, "num_range": 800, "upper_bound": "0.7207545042037964"}, {"distinct_range": 0, "num_eq": 8684337915033248650, "num_range": 4017380963863467599, "upper_bound": "2.4354634284973145"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 24, "row_count": 50}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 2282069720338208500, "num_eq": 500000000, "num_range": 4114893628687279188, "upper_bound": "375242582"}, {"distinct_range": 465854.0172181225, "num_eq": 600000, "num_range": 600000, "upper_bound": "510941867"}, {"distinct_range": 30000000000, "num_eq": 800000000, "num_range": 30000000000, "upper_bound": "1888820621"}, {"distinct_range": 69228.26328026825, "num_eq": 8382555148019501888, "num_range": 70000, "upper_bound": "2025328831"}, {"distinct_range": 0, "num_eq": 7714201785609925068, "num_range": 3000000, "upper_bound": "2672864067"}, {"distinct_range": 10000000, "num_eq": 2000000, "num_range": 10000000, "upper_bound": "3384621017"}, {"distinct_range": 1657248243980282600, "num_eq": 709363738646810175, "num_range": 1657248243980282683, "upper_bound": "3511356303"}, {"distinct_range": 3519201388714861000, "num_eq": 60, "num_range": 3519201388714861007, "upper_bound": "4186474473"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 37, "row_count": 50}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32, "histo_col_type": "", "name": "__auto__", "null_count": 24, "row_count": 50}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32, "histo_col_type": "", "name": "__auto__", "null_count": 3, "row_count": 50}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 37, "histo_col_type": "", "name": "__auto__", "null_count": 37, "row_count": 50}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000, "num_range": 0, "upper_bound": "-8434457321605435423"}, {"distinct_range": 90000, "num_eq": 700000, "num_range": 90000, "upper_bound": "-8148730810667577496"}, {"distinct_range": 300, "num_eq": 900000000, "num_range": 300, "upper_bound": "-6252517886905005470"}, {"distinct_range": 2683.574095775568, "num_eq": 700000, "num_range": 5000, "upper_bound": "-3491952588578324580"}, {"distinct_range": 0, "num_eq": 4481831018631038057, "num_range": 4545980585514185421, "upper_bound": "1439577717452740428"}, {"distinct_range": 0, "num_eq": 6188761155608109340, "num_range": 5794670223497582665, "upper_bound": "1985754306127952991"}, {"distinct_range": 511.9523502568017, "num_eq": 6000000000, "num_range": 700, "upper_bound": "3293091412672910518"}, {"distinct_range": 8000000000, "num_eq": 8702904968126672197, "num_range": 8000000000, "upper_bound": "5863788805988216713"}, {"distinct_range": 2967918286.3234224, "num_eq": 3690577102525192913, "num_range": 7000000000, "upper_bound": "7057356215277412556"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5, "histo_col_type": "", "name": "__auto__", "null_count": 17, "row_count": 50}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000000, "num_range": 0, "upper_bound": "-4713-11-24 00:00:00+00:00"}, {"distinct_range": 100000000, "num_eq": 7435063552757529807, "num_range": 100000000, "upper_bound": "1975-01-26 00:44:01.000278+00:00"}, {"distinct_range": 22274725.624649305, "num_eq": 1509784018330036146, "num_range": 30000000, "upper_bound": "1975-03-15 13:37:23.000282+00:00"}, {"distinct_range": 20000000, "num_eq": 5752293966310633274, "num_range": 20000000, "upper_bound": "1993-02-23 04:27:34.000908+00:00"}, {"distinct_range": 0, "num_eq": 3978694573137570701, "num_range": 7162212415028834284, "upper_bound": "1997-07-19 10:29:51.000128+00:00"}, {"distinct_range": 0, "num_eq": 80, "num_range": 7804491468185640450, "upper_bound": "2012-09-29 04:16:39.00044+00:00"}, {"distinct_range": 1000, "num_eq": 8208239996491126607, "num_range": 1000, "upper_bound": "2028-06-17 05:56:57.000578+00:00"}, {"distinct_range": 1632550308564861400, "num_eq": 50000000000, "num_range": 3966562582653363361, "upper_bound": "2031-02-25 15:50:17.000083+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 30, "row_count": 50}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29, "histo_col_type": "", "name": "__auto__", "null_count": 38, "row_count": 50}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000000, "num_range": 0, "upper_bound": "#"}, {"distinct_range": 46.16911013540334, "num_eq": 3957752156760549216, "num_range": 50, "upper_bound": "*"}, {"distinct_range": 6304.125765218535, "num_eq": 400000000, "num_range": 9000, "upper_bound": "1"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 30000, "upper_bound": "K"}, {"distinct_range": 1616503204823593200, "num_eq": 600000000, "num_range": 4154204727132667475, "upper_bound": "["}, {"distinct_range": 9000, "num_eq": 100, "num_range": 9000, "upper_bound": "z"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9, "histo_buckets": [{"distinct_range": 0, "num_eq": 4148787249036987228, "num_range": 0, "upper_bound": "03:08:44.996116"}, {"distinct_range": 0, "num_eq": 400000, "num_range": 60000, "upper_bound": "12:31:40.498857"}, {"distinct_range": 3290986298577380000, "num_eq": 6820097329477176026, "num_range": 4090096610205055494, "upper_bound": "14:25:13.412257"}, {"distinct_range": 0, "num_eq": 8282778272618470189, "num_range": 100000000000, "upper_bound": "16:32:54.883986"}, {"distinct_range": 50000000, "num_eq": 3908212673386215110, "num_range": 50000000, "upper_bound": "19:02:53.763209"}, {"distinct_range": 0, "num_eq": 788496879485233409, "num_range": 8000, "upper_bound": "23:24:14.670888"}, {"distinct_range": 0, "num_eq": 5559519096501764628, "num_range": 7582487234275546847, "upper_bound": "23:45:53.856284"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 7968831220424530924, "upper_bound": "23:50:37.360668"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 22, "row_count": 50}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('ykt');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('m', 'riv', 'zhrbgg', 'dcoxw');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('etedv', 'wzjy', 'mcm', 'gozvz');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('re');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2631442&tab=buildLog) on [master@e4ac407b14ed315662050f09670707cdf8ad9bfe](https://github.com/cockroachdb/cockroach/commits/e4ac407b14ed315662050f09670707cdf8ad9bfe):

```
        test_1        |             CREATE TABLE table2 (col1_9 UUID NOT NULL, col1_7 REGTYPE NOT NULL, col1_14 INT4 NOT NULL, col1_3 REGNAMESPACE NOT NULL, col2_4 OID NOT NULL, col2_5 DECIMAL NULL, col2_6 TIMESTAMP NULL, col2_7 BIT(17) NOT NULL, col2_8 TIME NULL, PRIMARY KEY (col1_9, col1_7, col1_14, col1_3), FAMILY (col2_5, col1_3, col1_7), FAMILY (col1_14), FAMILY (col2_4), FAMILY (col2_6, col2_8), FAMILY (col2_7), FAMILY (col1_9)) INTERLEAVE IN PARENT table1 (col1_9, col1_7, col1_14, col1_3);
        test_1        |             CREATE INDEX ON table2 (col1_7 ASC, col2_5 DESC, col1_9, col1_3, col1_14 DESC, col2_7 DESC, col2_4 ASC, col2_8 DESC, col2_6);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_14 ASC, col1_7) STORING (col2_6, col2_7);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_14, col1_3 DESC, col1_9, col1_7 DESC, col2_7 DESC, col2_6, col2_8, col2_4 ASC);
        test_1        |             CREATE INDEX ON table2 (col2_8 ASC, col1_9 ASC, col2_5 DESC) STORING (col2_7);
        test_1        |             CREATE TABLE table3 (col3_0 TIMESTAMPTZ[] NOT NULL, col3_1 VARCHAR NOT NULL, col3_2 UUID NOT NULL, col3_3 REGPROC NOT NULL, col3_4 REGCLASS NOT NULL, col3_5 INT2 NULL, col3_6 REGNAMESPACE NOT NULL, col3_7 REGNAMESPACE NOT NULL, col3_8 INT8 NOT NULL, col3_9 FLOAT4 NOT NULL, col3_10 DECIMAL NOT NULL, PRIMARY KEY (col3_7, col3_8, col3_1, col3_6, col3_10, col3_3, col3_4));
        test_1        |             CREATE INDEX ON table3 (col3_3 DESC, col3_9);
        test_1        |             CREATE INDEX ON table3 (col3_2, col3_1, col3_4 DESC, col3_6, col3_3 DESC, col3_10 DESC, col3_9);
        test_1        |             CREATE INDEX ON table3 (col3_1, col3_4 ASC, col3_7, col3_3 ASC, col3_2 DESC, col3_8 DESC, col3_5 ASC, col3_10 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_9, col3_4, col3_7 DESC, col3_3, col3_10 ASC, col3_5 ASC, col3_8 DESC, col3_6 DESC, col3_2);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 85699, "histo_col_type": "", "name": "__auto__", "null_count": 185851, "row_count": 300000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 290438, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 41986, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 226372, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 158579, "histo_col_type": "", "name": "__auto__", "null_count": 274645, "row_count": 300000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 259770, "histo_col_type": "", "name": "__auto__", "null_count": 258389, "row_count": 300000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 144557, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33373, "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 275491, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 231627, "histo_buckets": [{"distinct_range": 0, "num_eq": 2135215354197977517, "num_range": 0, "upper_bound": "167.24.44.79/6"}, {"distinct_range": 2403966383919942000, "num_eq": 70000, "num_range": 5308386631621773208, "upper_bound": "191f:4f94:b417:85cf:8a9c:32b2:93fb:971e/3"}, {"distinct_range": 0, "num_eq": 2584684224596550663, "num_range": 20000000000, "upper_bound": "9ba3:d838:3abc:1956:3894:3b43:72b:38b/82"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 258141, "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 236883, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 264379, "histo_col_type": "", "name": "__auto__", "null_count": 147904, "row_count": 300000}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 142955, "histo_col_type": "", "name": "__auto__", "null_count": 161337, "row_count": 300000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 184295, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 51967, "histo_buckets": [{"distinct_range": 0, "num_eq": 4896701242722090936, "num_range": 0, "upper_bound": "-0.4036230038546822"}, {"distinct_range": 767342357404366100, "num_eq": 3524495112430264519, "num_range": 767342357404366054, "upper_bound": "0.3800753515473081"}, {"distinct_range": 3940.3423206646353, "num_eq": 100, "num_range": 6000, "upper_bound": "0.9916169353388836"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 25297, "row_count": 300000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 84378, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 47502, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 563659, "histo_buckets": [{"distinct_range": 0, "num_eq": 500000, "num_range": 0, "upper_bound": "01:58:57.98286"}, {"distinct_range": 4947067175515129000, "num_eq": 8425899467263485962, "num_range": 4947067175515128515, "upper_bound": "09:32:56.029626"}, {"distinct_range": 10000000, "num_eq": 30000, "num_range": 10000000, "upper_bound": "11:13:57.773127"}, {"distinct_range": 100000000, "num_eq": 600, "num_range": 100000000, "upper_bound": "19:51:24.481331"}, {"distinct_range": 3025860739181861400, "num_eq": 3305630457026440211, "num_range": 4098707570488771889, "upper_bound": "22:12:57.836128"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 701244, "row_count": 2000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1438140, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "1630415527"}, {"distinct_range": 257191338634968060, "num_eq": 1000, "num_range": 581881467175743523, "upper_bound": "2080974831"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 402991, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1920415, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 255642, "histo_col_type": "", "name": "__auto__", "null_count": 1428539, "row_count": 2000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 968877, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000000, "num_range": 0, "upper_bound": "00000000-0000-0000-0000-000000000000"}, {"distinct_range": 6941.149597760384, "num_eq": 1692972308603006763, "num_range": 7000, "upper_bound": "121cdb99-c66a-45c5-b39d-bd05b5d25c34"}, {"distinct_range": 4649492077.3333, "num_eq": 8332897951681366999, "num_range": 40000000000, "upper_bound": "4977ba13-7c5d-459d-9bf3-11fd037e031e"}, {"distinct_range": 410.4460169129369, "num_eq": 5000, "num_range": 700, "upper_bound": "52dcbd67-f678-4fe6-96cf-3b2ea9ed3fdc"}, {"distinct_range": 300000, "num_eq": 6191974670011478538, "num_range": 300000, "upper_bound": "86ddaf3c-2718-46b1-91b6-98bcf1533a51"}, {"distinct_range": 0, "num_eq": 8115904348440234124, "num_range": 9133913789199457996, "upper_bound": "8cfa7873-403a-4409-9e92-2abb1e53a89e"}, {"distinct_range": 378935159377080100, "num_eq": 80000000000, "num_range": 444771139163462791, "upper_bound": "fd1878c4-a418-4bef-9b77-ed1855c7107c"}, {"distinct_range": 0, "num_eq": 5681772706163118083, "num_range": 7799502274816624249, "upper_bound": "ffffffff-ffff-ffff-ffff-ffffffffffff"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1630430, "histo_buckets": [{"distinct_range": 0, "num_eq": 2650768856868230970, "num_range": 0, "upper_bound": "1744484133"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 2000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 719231, "histo_col_type": "", "name": "__auto__", "null_count": 134816, "row_count": 2000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 80206962068, "histo_buckets": [{"distinct_range": 0, "num_eq": 6393396224954901814, "num_range": 0, "upper_bound": "a2e60155-4a77-4799-b679-6e9573584293"}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 100, "upper_bound": "baf74e81-85fc-4c4c-b1b2-fdc32a3d2f6b"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 82543923691, "histo_buckets": [{"distinct_range": 0, "num_eq": 4396171675703396886, "num_range": 0, "upper_bound": "90956774"}, {"distinct_range": 63220.44989500625, "num_eq": 3223082297623607537, "num_range": 900000, "upper_bound": "1177624739"}, {"distinct_range": 0, "num_eq": 800, "num_range": 1000, "upper_bound": "1232669196"}, {"distinct_range": 300, "num_eq": 100000, "num_range": 300, "upper_bound": "1411493208"}, {"distinct_range": 7.076776965494441, "num_eq": 4968472968360677823, "num_range": 20, "upper_bound": "1631056444"}, {"distinct_range": 90000, "num_eq": 30000, "num_range": 90000, "upper_bound": "2240720239"}, {"distinct_range": 30000000000, "num_eq": 5000075019584812430, "num_range": 30000000000, "upper_bound": "2522618975"}, {"distinct_range": 3806737665.938144, "num_eq": 20000, "num_range": 4000000000, "upper_bound": "2770326412"}, {"distinct_range": 0, "num_eq": 5000, "num_range": 30000000, "upper_bound": "4125401254"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2646770482, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 1923345458699759888, "num_range": 4000000000, "upper_bound": "325585014"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 47363732685, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1383850115, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 87036135273, "histo_buckets": [{"distinct_range": 0, "num_eq": 700, "num_range": 0, "upper_bound": "\\u000e\\u0019"}, {"distinct_range": 900000, "num_eq": 80000000000, "num_range": 900000, "upper_bound": ",L*\'\\u001f6"}, {"distinct_range": 4928179.272231471, "num_eq": 4757397028909801164, "num_range": 40000000, "upper_bound": "_5("}, {"distinct_range": 1879223375872256800, "num_eq": 7834164212070645665, "num_range": 1879223375872256889, "upper_bound": "egOr39U"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 69802193758, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 76934307506, "histo_col_type": "", "name": "__auto__", "null_count": 26266396326, "row_count": 90000000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5094455213, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 62743567602, "histo_buckets": [{"distinct_range": 0, "num_eq": 3862237481563833882, "num_range": 0, "upper_bound": "0.6759721636772156"}, {"distinct_range": 0, "num_eq": 8829498483175769894, "num_range": 200000000, "upper_bound": "1.3847509622573853"}, {"distinct_range": 68.31539565797459, "num_eq": 7184733386584043153, "num_range": 100, "upper_bound": "3.4028234663852886e+38"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33471482672, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('votd', 'um', 'rb', 'hyd');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('uddbsg', 'wsf', 'r', 'ryome', 'cwcjaa', 'hrvlo');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('cass', 'ccpjq', 'dnhh', 'zogg');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('zw', 'xwfbk', 'bvqzv');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('hsdkn', 'zp', 'll', 'etwiky', 'ub', 'jdg');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        |     --- FAIL: TestCompare/mutators (0.18s)
        test_1        |         compare_test.go:129: 
        test_1        |             		SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;
        test_1        |             		SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;
        test_1        |             	CREATE TABLE table1 (col1_0 TIMESTAMP NOT NULL, col1_1 BIT(30) NOT NULL, col1_2 REGCLASS NOT NULL, col1_3 TIME NULL, col1_4 UUID NOT NULL, col1_5 BIT(23) NOT NULL, col1_6 REGTYPE NOT NULL, col1_7 TIMESTAMPTZ NOT NULL, col1_8 TIMETZ NOT NULL, col1_9 UUID NOT NULL, PRIMARY KEY (col1_9, col1_5 DESC, col1_2 DESC, col1_7 DESC, col1_6 DESC, col1_0, col1_8 DESC, col1_1 DESC), UNIQUE (col1_9 ASC, col1_2 DESC, col1_3 DESC, col1_8, col1_0, col1_1 ASC, col1_4, col1_7, col1_6) WHERE (table1.col1_7 < '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ) AND (table1.col1_8 != '24:00:00-15:59:00':::TIMETZ), INDEX (col1_6 DESC, col1_1 ASC) WHERE ((table1.col1_3 <= '24:00:00':::TIME) AND (table1.col1_7 > '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table1.col1_8 <= '24:00:00-15:59:00':::TIMETZ), UNIQUE (col1_5 DESC, col1_1, col1_8) WHERE (((table1.col1_0 = '-4713-11-24 00:00:00':::TIMESTAMP) OR (table1.col1_3 = '00:00:00':::TIME)) AND (table1.col1_7 < '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table1.col1_8 > '24:00:00-15:59:00':::TIMETZ), INDEX (col1_1 DESC, col1_5 ASC, col1_2 DESC, col1_0 ASC) STORING (col1_3) WHERE (table1.col1_7 <= '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ) AND (table1.col1_0 >= '-2000-01-01 00:00:00':::TIMESTAMP), UNIQUE (col1_6 DESC, col1_2 DESC, col1_9 DESC, col1_3, col1_5) STORING (col1_4) WHERE (((table1.col1_0 >= '294276-12-31 23:59:59.999999':::TIMESTAMP) AND (table1.col1_7 >= '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table1.col1_3 != '24:00:00':::TIME)) OR (table1.col1_8 < '24:00:00-15:59:00':::TIMETZ), INDEX (col1_0 DESC, col1_4, col1_1 DESC, col1_6 ASC, col1_3, col1_9, col1_8 DESC) WHERE (((table1.col1_3 <= '00:00:00':::TIME) OR (table1.col1_8 < '00:00:00+15:59:00':::TIMETZ)) AND (table1.col1_0 <= '0001-01-01 00:00:00':::TIMESTAMP)) AND (table1.col1_7 < '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ), UNIQUE (col1_5, col1_3) WHERE ((table1.col1_7 > '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ) OR (table1.col1_8 >= '00:00:00+15:59:00':::TIMETZ)) OR (table1.col1_3 = '00:00:00':::TIME), INDEX (col1_7, col1_0, col1_4 DESC, col1_9 ASC, col1_5, col1_2) STORING (col1_3) WHERE (((table1.col1_3 <= '24:00:00':::TIME) AND (table1.col1_0 >= '-2000-01-01 00:00:00':::TIMESTAMP)) AND (table1.col1_7 <= '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) OR (table1.col1_8 < '00:00:00+15:59:00':::TIMETZ), UNIQUE (col1_0 DESC, col1_6, col1_8 ASC, col1_5 DESC, col1_1, col1_9, col1_3));
        test_1        |             CREATE TABLE table2 (col2_0 REGPROC NOT NULL, col2_1 FLOAT8 NULL, col2_2 DATE NOT NULL, col2_3 REGNAMESPACE NOT NULL, col2_4 REGNAMESPACE NOT NULL, col2_5 TIMESTAMPTZ NULL, col2_6 FLOAT4 NOT NULL, col2_7 "char" NOT NULL, col2_8 CHAR NOT NULL, col2_9 VARCHAR NOT NULL, col2_10 TIMETZ NOT NULL, col2_11 FLOAT4, col2_12 DECIMAL NOT NULL, col2_13 CHAR NOT NULL, col2_14 FLOAT8 NOT NULL, col2_15 VARCHAR, col2_16 INT8 NOT NULL, col2_17 TIME NOT NULL, col2_18 VARCHAR NOT NULL, PRIMARY KEY (col2_3 ASC, col2_6 ASC, col2_4 ASC, col2_13, col2_0, col2_18, col2_16 DESC, col2_10 ASC, col2_8 DESC, col2_7 DESC, col2_2 DESC, col2_14 ASC, col2_12, col2_9 DESC), FAMILY (col2_16, col2_7), FAMILY (col2_17, col2_13, col2_8, col2_3), FAMILY (col2_15), FAMILY (col2_12), FAMILY (col2_1, col2_14, col2_5), FAMILY (col2_11), FAMILY (col2_2), FAMILY (col2_0), FAMILY (col2_4, col2_6), FAMILY (col2_18, col2_9, col2_10));
        test_1        |             CREATE TABLE table3 (col2_3 REGNAMESPACE NOT NULL, col2_6 FLOAT4 NOT NULL, col2_4 REGNAMESPACE NOT NULL, col2_13 CHAR NOT NULL, col2_0 REGPROC NOT NULL, col2_18 VARCHAR NOT NULL, col2_16 INT8 NOT NULL, col2_10 TIMETZ NOT NULL, col2_8 CHAR NOT NULL, col2_7 "char" NOT NULL, col2_2 DATE NOT NULL, col2_14 FLOAT8 NOT NULL, col2_12 DECIMAL NOT NULL, col2_9 VARCHAR NOT NULL, col3_14 INT2, col3_15 REGNAMESPACE NOT NULL, col3_16 DECIMAL NOT NULL, PRIMARY KEY (col2_3 ASC, col2_6 ASC, col2_4 ASC, col2_13, col2_0, col2_18, col2_16 DESC, col2_10 ASC, col2_8 DESC, col2_7 DESC, col2_2 DESC, col2_14 ASC, col2_12, col2_9 DESC, col3_16, col3_14 ASC), UNIQUE (col2_7 ASC, col2_10) STORING (col3_15) WHERE ((((((table3.col2_7 > e'\'':::STRING) AND (table3.col2_6 >= 1.0:::FLOAT8)) OR (table3.col2_14 < 3.4028234663852886e+38:::FLOAT8)) OR (table3.col2_9 < '"':::STRING)) AND (table3.col3_14 = (-1):::INT8)) OR (table3.col2_2 < '-infinity':::DATE)) AND (table3.col2_13 = '"':::STRING), INDEX (col2_3, col2_7 ASC, col2_6, col2_10 DESC, col2_14 ASC, col3_15, col2_18 ASC, col2_8 ASC) WHERE ((((((((((((table3.col2_6 != 1.0:::FLOAT8) OR (table3.col3_16 < 'Infinity':::DECIMAL)) OR (table3.col2_8 = e'\'':::STRING)) AND (table3.col2_12 >= (-1):::DECIMAL)) OR (table3.col2_2 < '-infinity':::DATE)) OR (table3.col3_14 != (-1):::INT8)) AND (table3.col2_13 > e'\x00':::STRING)) AND (table3.col2_9 = '':::STRING)) AND (table3.col2_16 = (-1):::INT8)) OR (table3.col2_18 > '"':::STRING)) AND (table3.col2_14 != 1.7976931348623157e+308:::FLOAT8)) AND (table3.col2_10 != '00:00:00+15:59:00':::TIMETZ)) OR (table3.col2_7 = e'\'':::STRING), UNIQUE (col2_18 DESC, col2_0 DESC, col2_13, col2_2 ASC, col2_3, col2_9 DESC, col2_12, col2_10 DESC, col3_15, col3_16, col2_8 DESC, col2_14 ASC, col2_7 ASC, col2_4 ASC, col2_16 DESC, col2_6 ASC, col3_14) WHERE ((((((((((table3.col2_10 = '24:00:00-15:59:00':::TIMETZ) OR (table3.col2_13 <= '"':::STRING)) OR (table3.col2_16 >= (-1):::INT8)) OR (table3.col2_7 = '':::STRING)) AND (table3.col3_16 = 'Infinity':::DECIMAL)) AND (table3.col2_14 <= 3.4028234663852886e+38:::FLOAT8)) OR (table3.col2_9 != 'X':::STRING)) OR (table3.col2_2 = '5874897-12-31':::DATE)) AND (table3.col2_12 = 0:::DECIMAL)) AND (table3.col3_14 <= (-1):::INT8)) OR (table3.col2_18 < '':::STRING), FAMILY (col2_9), FAMILY (col2_18), FAMILY (col2_16, col2_10, col2_3, col2_4, col2_0), FAMILY (col2_2), FAMILY (col3_15, col2_14), FAMILY (col2_7, col3_14, col2_13, col2_8, col2_12, col3_16, col2_6)) INTERLEAVE IN PARENT table2 (col2_3, col2_6, col2_4, col2_13, col2_0, col2_18, col2_16, col2_10, col2_8, col2_7, col2_2, col2_14, col2_12, col2_9);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19824123, "histo_buckets": [{"distinct_range": 0, "num_eq": 3412347818958548599, "num_range": 0, "upper_bound": "010001001001000111011110000110"}, {"distinct_range": 16636804204.704023, "num_eq": 5689563680731235749, "num_range": 40000000000, "upper_bound": "110001101100110111101011010011"}, {"distinct_range": 8345627488963273000, "num_eq": 400000000, "num_range": 8345627488963272910, "upper_bound": "111010000101010011001000111000"}], "histo_col_type": "BIT(30)", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 72293922, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21577976, "histo_col_type": "", "name": "__auto__", "null_count": 69227735, "row_count": 100000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34139603, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8002249, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "00000000000000000000000"}, {"distinct_range": 1114771167233462000, "num_eq": 6845885287913250574, "num_range": 5715327080127929794, "upper_bound": "01111111111111111111111"}, {"distinct_range": 2734090531585431000, "num_eq": 9000000000, "num_range": 8238969232298040676, "upper_bound": "11000011101001110101100"}, {"distinct_range": 2743603.6484721075, "num_eq": 100000000000, "num_range": 3000000, "upper_bound": "11011100100010100110100"}], "histo_col_type": "BIT(23)", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 87941456, "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9736513, "histo_buckets": [{"distinct_range": 0, "num_eq": 2881074013663038558, "num_range": 0, "upper_bound": "-2000-01-01 00:00:00"}, {"distinct_range": 0, "num_eq": 4673369406398900737, "num_range": 8000000, "upper_bound": "0001-01-01 00:00:00"}, {"distinct_range": 10000000000, "num_eq": 4954609209561507714, "num_range": 10000000000, "upper_bound": "1970-08-25 15:24:35.000681"}, {"distinct_range": 5565780837210426000, "num_eq": 8000, "num_range": 5565780837210426799, "upper_bound": "1988-12-19 19:02:30.000973"}, {"distinct_range": 7949622461689668000, "num_eq": 2000, "num_range": 8128011798779979740, "upper_bound": "2013-12-04 00:52:11.000563"}, {"distinct_range": 4399501481408548400, "num_eq": 7430019000054368300, "num_range": 4399501481408548557, "upper_bound": "2013-12-19 12:00:02.0003"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 293136999538016529, "upper_bound": "2017-10-01 17:30:13.000417"}, {"distinct_range": 3099785.567810703, "num_eq": 2995769576176836883, "num_range": 4000000, "upper_bound": "2021-06-14 20:08:27.000115"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19562904, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 1725977663075645700, "num_eq": 117263818339361513, "num_range": 1725977663075645581, "upper_bound": "367599757"}, {"distinct_range": 0, "num_eq": 6305238023466894175, "num_range": 1894592614513694501, "upper_bound": "865918134"}, {"distinct_range": 47691.85714083942, "num_eq": 670170653368703432, "num_range": 100000, "upper_bound": "1177279431"}, {"distinct_range": 1243131742059812400, "num_eq": 0, "num_range": 2402880897083660054, "upper_bound": "1763886721"}, {"distinct_range": 0, "num_eq": 8866281570904561478, "num_range": 2000000000, "upper_bound": "2112056429"}, {"distinct_range": 20000, "num_eq": 6789036911090074525, "num_range": 20000, "upper_bound": "2658431339"}, {"distinct_range": 1673986907412727300, "num_eq": 2000000, "num_range": 6296892951770673902, "upper_bound": "4239262980"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 70565516, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4839064, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "00000000-0000-0000-0000-000000000000"}, {"distinct_range": 0, "num_eq": 2359923274179788140, "num_range": 8000000000, "upper_bound": "50d4d3b4-103c-4115-a6e8-b43a8b870b54"}, {"distinct_range": 6000, "num_eq": 900000000, "num_range": 6000, "upper_bound": "9f51cee1-8512-4ee9-bc17-89c80cd2307a"}, {"distinct_range": 4052608872245673000, "num_eq": 10, "num_range": 5415489989845580066, "upper_bound": "b9ed5650-b07f-4647-8ef3-2417cf7c6911"}, {"distinct_range": 87542.52300771294, "num_eq": 6000000, "num_range": 100000, "upper_bound": "c116bc39-5a53-4498-88fa-c8a6f7b23d22"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 100000000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1182743925637295195, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8473136832933893760}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4087270194017277040, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8473136832933893760}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6162538157066995609, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8473136832933893760}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2719501510930675810, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8473136832933893760}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1673135543915770298, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8473136832933893760}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3610216913372461893, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8473136832933893760}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6962589880978750711, "histo_col_type": "", "name": "__auto__", "null_count": 2303176892092045530, "row_count": 8473136832933893760}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1678033747303397438, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8473136832933893760}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5816420615045313647, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8473136832933893760}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 907984968693510893, "histo_col_type": "", "name": "__auto__", "null_count": 5601453185810229211, "row_count": 8473136832933893760}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8254022923853771403, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8473136832933893760}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5489760229084953254, "histo_col_type": "", "name": "__auto__", "null_count": 2135460554888781750, "row_count": 8473136832933893760}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4886860691106562110, "histo_buckets": [{"distinct_range": 0, "num_eq": 5717899436426659749, "num_range": 0, "upper_bound": "1588505656"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 3000000, "upper_bound": "1967930060"}, {"distinct_range": 0, "num_eq": 7326601836652020025, "num_range": 0, "upper_bound": "3575540362"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 100000, "upper_bound": "3697246005"}, {"distinct_range": 0, "num_eq": 6525997617261704245, "num_range": 0, "upper_bound": "4196370951"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 8473136832933893760}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2459519407716863767, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8473136832933893760}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 481522395958103943, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8473136832933893760}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7752631953452948397, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8473136832933893760}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5204720889571120849, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8473136832933893760}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6180821263729515083, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8473136832933893760}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1795200266960532329, "histo_col_type": "", "name": "__auto__", "null_count": 2692368473927457509, "row_count": 8473136832933893760}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1368572608400561119, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2676532944595763427}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1723902936494188372, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2676532944595763427}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1475410151434249415, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2676532944595763427}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1866445916778847986, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2676532944595763427}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 294407056214010813, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2676532944595763427}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1468048883091374760, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2676532944595763427}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1749036378800124247, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2676532944595763427}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1258334166109528935, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2676532944595763427}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 69584039672964596, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2676532944595763427}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1263793045538652753, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000, "num_range": 0, "upper_bound": "1211229123"}, {"distinct_range": 2181879753497008600, "num_eq": 100000000000, "num_range": 2725802868387428656, "upper_bound": "2813102961"}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 600000000, "upper_bound": "3333552675"}, {"distinct_range": 3909675795.87574, "num_eq": 8942919659541080328, "num_range": 6000000000, "upper_bound": "4212118198"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 2676532944595763427}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2368996805246580798, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 7787099859956995120, "num_range": 50000000, "upper_bound": "\\u0013:2\x7fy"}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 5764781375131834590, "upper_bound": "\\""}, {"distinct_range": 0, "num_eq": 421015628548562216, "num_range": 9029660006152700405, "upper_bound": "D\\u0014\\u0012\\u000c"}, {"distinct_range": 4.236988908819964, "num_eq": 2136278443030878283, "num_range": 80, "upper_bound": "P@!\\u000cL^E!"}, {"distinct_range": 2989728841554398700, "num_eq": 7518763806102834226, "num_range": 2989728841554398564, "upper_bound": "r_y"}, {"distinct_range": 172.6771224851679, "num_eq": 700000, "num_range": 200, "upper_bound": "v&)5?"}, {"distinct_range": 20000000, "num_eq": 1535733630284488418, "num_range": 20000000, "upper_bound": "x\\u001f\\tYGk\\u0010"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 2676532944595763427}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 794204952132977017, "histo_buckets": [{"distinct_range": 0, "num_eq": 518892566584755210, "num_range": 0, "upper_bound": "\\u001c"}, {"distinct_range": 70000000000, "num_eq": 500000000, "num_range": 70000000000, "upper_bound": "3"}, {"distinct_range": 8883483751537667000, "num_eq": 60, "num_range": 8883483751537666656, "upper_bound": "4"}, {"distinct_range": 7976324450167516000, "num_eq": 400000000, "num_range": 7976324450167516074, "upper_bound": "M"}, {"distinct_range": 0, "num_eq": 8036461527549072041, "num_range": 4075887275025967903, "upper_bound": "\\\\"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 80, "upper_bound": "d"}, {"distinct_range": 5221548800172676000, "num_eq": 4000000000, "num_range": 5221548800172675836, "upper_bound": "z"}, {"distinct_range": 20000, "num_eq": 4644039675824493637, "num_range": 20000, "upper_bound": "\U00002603"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 2676532944595763427}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1190323974397509339, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2676532944595763427}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 795127374351390125, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2676532944595763427}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 524275731601645281, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2676532944595763427}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1020463406221313745, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2676532944595763427}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 814653196062713908, "histo_col_type": "", "name": "__auto__", "null_count": 725448625436794748, "row_count": 2676532944595763427}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('rckdlf', 'ocyi', 'oj');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('u', 'aikoh', 'ywek', 'nxzbu', 'c', 'gp');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('v');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('k', 'wsabqb', 'ubtl', 'gnkkb', 'lssq', 'dbxdp');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('mvoth', 'f', 'fuhbm', 'nc');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        | FAIL
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2635612&tab=buildLog) on [master@0e6727832d58faf0f900601cd6fa6807e0a2ba75](https://github.com/cockroachdb/cockroach/commits/0e6727832d58faf0f900601cd6fa6807e0a2ba75):

```
        test_1        |             CREATE TABLE table5 (col5_0 DECIMAL, col5_1 TIMETZ NULL, col5_2 INTERVAL NOT NULL, col5_3 FLOAT8 NOT NULL, col5_4 BOX2D);
        test_1        |             CREATE INDEX ON table5 (col5_2, col5_1 DESC);
        test_1        |             CREATE INDEX ON table5 (col5_1 DESC, col5_0 DESC, col5_4 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_0 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_0 DESC, col5_2 ASC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('fpcc', 'yo');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('fjzujg', 'cs', 'pqv', 'hqfy', 'ivc');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('wbbd');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (0.80s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 TIMESTAMPTZ NOT NULL, col1_1 BOOL NOT NULL, col1_2 BIT(45) NULL, col1_3 REGPROCEDURE NOT NULL, col1_4 BOOL[] NULL, col1_5 BOX2D NULL, col1_6 REGCLASS NULL, col1_7 BOX2D NOT NULL, col1_8 CHAR, col1_9 TIME NULL, col1_10 CHAR, UNIQUE (col1_10, col1_5, col1_9, col1_6, col1_7, col1_1, col1_2 DESC, col1_3 DESC, col1_0 DESC) STORING (col1_4, col1_8) WHERE ((table1.col1_0 < '3000-01-01 00:00:00+00:00') OR (table1.col1_8 <= e'\'')) OR table1.col1_1, INDEX (col1_5 DESC, col1_8 ASC) STORING (col1_0, col1_1, col1_3, col1_4, col1_6, col1_7, col1_10) WHERE table1.col1_9 >= '00:00:00', UNIQUE (col1_6 ASC, col1_7 ASC, col1_8 DESC, col1_1 ASC, col1_0 ASC, col1_5 ASC, col1_2 ASC) STORING (col1_3, col1_4) WHERE ((((table1.col1_8 >= e'\U00002603':::STRING) OR (table1.col1_9 <= '24:00:00':::TIME)) OR (NOT table1.col1_1)) OR (table1.col1_0 > '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table1.col1_10 = '"':::STRING), INDEX (col1_0 ASC, col1_1 ASC, col1_2, col1_6 DESC, col1_8 DESC, col1_10 DESC, col1_3, col1_5) WHERE ((((table1.col1_8 >= '"':::STRING) AND (table1.col1_9 != '24:00:00':::TIME)) OR (table1.col1_10 > 'X':::STRING)) OR table1.col1_1) OR (table1.col1_0 < '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ), INDEX (col1_3 ASC, col1_1 ASC, col1_0 DESC, col1_5 DESC) STORING (col1_8, col1_10) WHERE table1.col1_8 >= e'\U00002603':::STRING, INDEX (col1_5 ASC, col1_8, col1_6, col1_7, col1_9 DESC, col1_1, col1_0 DESC, col1_10 DESC) WHERE ((((table1.col1_10 != e'\x00':::STRING) OR (table1.col1_0 < '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR table1.col1_1) OR (table1.col1_9 >= '24:00:00':::TIME)) AND (table1.col1_8 >= e'\'':::STRING), UNIQUE (col1_9 ASC, col1_0, col1_3 ASC, col1_6 ASC, col1_10 DESC, col1_7 DESC, col1_8 DESC, col1_5 ASC, col1_2 DESC, col1_1) STORING (col1_4) WHERE ((((table1.col1_9 < '00:00:00':::TIME) OR (table1.col1_0 > '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table1.col1_10 >= e'\U00002603':::STRING)) OR (NOT table1.col1_1)) OR (table1.col1_8 = e'\'':::STRING), UNIQUE (col1_3, col1_5 ASC, col1_9, col1_8 DESC, col1_6 ASC) STORING (col1_0, col1_1, col1_4, col1_7) WHERE ((table1.col1_9 >= '24:00:00':::TIME) OR (table1.col1_0 = '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) OR (NOT table1.col1_1), FAMILY (col1_6), FAMILY (col1_7), FAMILY (col1_5, col1_8), FAMILY (col1_3, col1_9), FAMILY (col1_4), FAMILY (col1_0, col1_1, col1_2), FAMILY (col1_10));
        test_1        |             CREATE TABLE table2 (col2_0 TIMESTAMPTZ NOT NULL, col2_1 VARCHAR NOT NULL, col2_2 REGCLASS NOT NULL, col2_3 REGPROCEDURE, col2_4 INET, col2_5 TIME NOT NULL, col2_6 DECIMAL NOT NULL, col2_7 STRING NOT NULL, col2_8 TIMETZ, PRIMARY KEY (col2_6, col2_1, col2_7 DESC, col2_5 ASC, col2_0 ASC, col2_2), UNIQUE (col2_1 ASC) WHERE (((((table2.col2_0 = '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ) OR (table2.col2_8 > '00:00:00+15:59:00':::TIMETZ)) OR (table2.col2_7 > 'X':::STRING)) AND (table2.col2_5 <= '00:00:00':::TIME)) AND (table2.col2_6 >= 'Infinity':::DECIMAL)) AND (table2.col2_1 >= e'\x00':::STRING), UNIQUE (col2_5, col2_1 DESC, col2_2 DESC, col2_6 ASC, col2_0, col2_7 ASC, col2_4, col2_8 ASC, col2_3 DESC) WHERE (((((table2.col2_7 = '"') AND (table2.col2_8 > '00:00:00+15:59:00')) OR (table2.col2_5 >= '24:00:00')) AND (table2.col2_6 >= -Infinity)) OR (table2.col2_1 >= '')) AND (table2.col2_0 <= '294276-12-31 23:59:59.999999+00:00'), INDEX (col2_3, col2_8) STORING (col2_4) WHERE (table2.col2_5 > '24:00:00') OR (table2.col2_6 >= Infinity), INDEX (col2_3 DESC) WHERE (((((table2.col2_1 <= e'\U00002603') AND (table2.col2_5 <= '24:00:00')) OR (table2.col2_6 < NaN)) OR (table2.col2_8 >= '24:00:00-15:59:00')) AND (table2.col2_0 > '-4713-11-24 00:00:00+00:00')) OR (table2.col2_7 > e'\x00'), INDEX (col2_3 DESC, col2_2 DESC, col2_7 DESC, col2_1 ASC, col2_4 DESC, col2_0) WHERE table2.col2_0 = '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ, UNIQUE (col2_1 ASC, col2_0, col2_5, col2_6 DESC, col2_7 DESC, col2_8, col2_2, col2_3) STORING (col2_4) WHERE table2.col2_5 != '24:00:00':::TIME, INDEX (col2_7 ASC, col2_4 ASC, col2_5 DESC, col2_8 ASC, col2_2 ASC, col2_3 DESC, col2_0 DESC, col2_6) WHERE ((((table2.col2_8 != '24:00:00-15:59:00':::TIMETZ) OR (table2.col2_7 < '':::STRING)) AND (table2.col2_6 != 'Infinity':::DECIMAL)) OR (table2.col2_1 >= e'\x00':::STRING)) OR (table2.col2_0 <= '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ), UNIQUE (col2_4, col2_7 DESC, col2_1 ASC, col2_2 DESC, col2_5) STORING (col2_3) WHERE table2.col2_1 <= '"', FAMILY (col2_4), FAMILY (col2_2), FAMILY (col2_5, col2_7), FAMILY (col2_1, col2_3), FAMILY (col2_6, col2_0, col2_8));
        test_1        |             CREATE TABLE table3 (col3_0 NAME, col3_1 REGCLASS, col3_2 FLOAT4 NULL, col3_3 TIMESTAMPTZ NULL, col3_4 STRING, col3_5 INT8 NOT NULL, col3_6 INT8 NOT NULL, col3_7 INTERVAL NULL, FAMILY (col3_7), FAMILY (col3_3, col3_1, col3_0, col3_4, col3_2), FAMILY (col3_6), FAMILY (col3_5));
        test_1        |             CREATE TABLE table4 (col4_0 TIMETZ NOT NULL, col4_1 REGTYPE, col4_2 FLOAT8 NOT NULL, col4_3 OID NOT NULL, col4_4 TIME, col4_5 REGNAMESPACE NOT NULL, col4_6 BYTES, col4_7 OID NOT NULL, col4_8 BOOL NULL, col4_9 REGPROCEDURE NOT NULL, PRIMARY KEY (col4_0, col4_9 DESC), INDEX (col4_7 ASC, col4_8 DESC) STORING (col4_2, col4_3, col4_4, col4_5, col4_6) WHERE ((((table4.col4_0 >= '24:00:00-15:59:00') OR table4.col4_8) AND (table4.col4_2 != 3.4028234663852886e+38)) OR (table4.col4_6 = '\x22')) AND (table4.col4_4 = '24:00:00'), UNIQUE (col4_5 ASC, col4_9 DESC, col4_1 DESC, col4_6 ASC) WHERE ((((table4.col4_4 != '24:00:00':::TIME) OR (NOT table4.col4_8)) OR (table4.col4_6 < e'\\x00':::BYTES)) AND (table4.col4_0 > '00:00:00+15:59:00':::TIMETZ)) AND (table4.col4_2 >= 'NaN':::FLOAT8), INDEX (col4_7, col4_2, col4_5, col4_4, col4_1, col4_6 DESC, col4_3, col4_0 ASC) STORING (col4_8) WHERE table4.col4_6 <= e'\\x27':::BYTES, UNIQUE (col4_2, col4_7, col4_5 DESC, col4_4 ASC, col4_3 DESC, col4_6 DESC) STORING (col4_8), INDEX (col4_4 DESC, col4_8) STORING (col4_2, col4_5, col4_6) WHERE ((table4.col4_0 > '00:00:00+15:59:00':::TIMETZ) AND (table4.col4_6 < e'\\x27':::BYTES)) AND (table4.col4_4 < '24:00:00':::TIME), INDEX (col4_0 DESC, col4_1, col4_5 ASC, col4_8 ASC, col4_7, col4_9, col4_2, col4_6 DESC, col4_4, col4_3 DESC) WHERE ((((table4.col4_0 < '24:00:00-15:59:00') AND (NOT table4.col4_8)) OR (table4.col4_6 < '\x27')) AND (table4.col4_2 >= 5e-324)) AND (table4.col4_4 >= '00:00:00'), INDEX (col4_0 DESC, col4_6) STORING (col4_1, col4_2, col4_3, col4_4, col4_5, col4_7), FAMILY (col4_5), FAMILY (col4_6), FAMILY (col4_3, col4_7, col4_4), FAMILY (col4_1, col4_8, col4_2, col4_0, col4_9));
        test_1        |             CREATE TABLE table5 (col5_0 REGTYPE NOT NULL, col5_1 REGPROC NOT NULL, col5_2 TIMESTAMP NULL, col5_3 INET NOT NULL, col5_4 BYTES NULL, col5_5 NAME NULL, col5_6 INET NOT NULL, col5_7 INTERVAL NULL, col5_8 BIT(17) NOT NULL, PRIMARY KEY (col5_1 ASC, col5_6, col5_8 DESC, col5_3), UNIQUE (col5_7 ASC, col5_2 DESC, col5_5 ASC, col5_8 DESC) STORING (col5_4) WHERE table5.col5_5 != '', INDEX (col5_6 DESC, col5_1 DESC) STORING (col5_2, col5_4, col5_7) WHERE (table5.col5_4 <= e'\\x58':::BYTES) OR (table5.col5_5 <= '':::STRING), INDEX (col5_4, col5_1 ASC, col5_0 ASC, col5_7, col5_3 ASC, col5_2 ASC), UNIQUE (col5_3 DESC, col5_6 ASC, col5_7, col5_4 DESC) WHERE ((table5.col5_5 != e'\'':::STRING) AND (table5.col5_2 >= '-4713-11-24 00:00:00':::TIMESTAMP)) AND (table5.col5_4 < e'\\x22':::BYTES), UNIQUE (col5_0 ASC, col5_8 DESC, col5_2 ASC, col5_6 ASC, col5_3 ASC, col5_4 ASC) WHERE (table5.col5_2 = '0001-01-01 00:00:00':::TIMESTAMP) AND (table5.col5_5 < e'\U00002603':::STRING), UNIQUE (col5_4 DESC) STORING (col5_2, col5_5, col5_7) WHERE ((table5.col5_4 >= e'\\x58':::BYTES) OR (table5.col5_2 > '-4713-11-24 00:00:00':::TIMESTAMP)) AND (table5.col5_5 != 'X':::STRING), INDEX (col5_6 ASC, col5_0 DESC, col5_1 DESC) WHERE ((table5.col5_5 = e'\x00':::STRING) AND (table5.col5_2 != '3000-01-01 00:00:00':::TIMESTAMP)) OR (table5.col5_4 <= e'\\x58':::BYTES), FAMILY (col5_3, col5_1), FAMILY (col5_8, col5_6), FAMILY (col5_5, col5_7, col5_0), FAMILY (col5_2, col5_4));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2264102730945280009, "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 68924864493171153, "row_count": 2655029972590966795}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1048116851975621861, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2655029972590966795}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1215775622996684609, "histo_col_type": "", "name": "__auto__", "null_count": 983925251647006066, "row_count": 2655029972590966795}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1205868512961611765, "histo_buckets": [{"distinct_range": 0, "num_eq": 2172281495400438776, "num_range": 0, "upper_bound": "$"}, {"distinct_range": 797591212240301400, "num_eq": 8432879888736356277, "num_range": 1635936423511832026, "upper_bound": "O"}, {"distinct_range": 4276246183.976829, "num_eq": 10000000000, "num_range": 8000000000, "upper_bound": "j"}, {"distinct_range": 6741205566430837000, "num_eq": 2910314842039008335, "num_range": 6741205566430836599, "upper_bound": "s"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 1968552766962409161, "row_count": 2655029972590966795}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 872573805131704609, "histo_col_type": "", "name": "__auto__", "null_count": 1077118731506946099, "row_count": 2655029972590966795}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2486235715873519042, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 0, "num_eq": 2338689805177929201, "num_range": 3100858432237599985, "upper_bound": "BOX(-2.0053570186265066 0.6059950962977958,-0.35006868032616373 2.6362361607185485)"}, {"distinct_range": 3453750099177750500, "num_eq": 3000000, "num_range": 5304876038518062165, "upper_bound": "BOX(-0.03357797972829424 -0.8768212002327594,0.02232223339082462 1.302219265075035)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 1465788724689335616, "row_count": 2655029972590966795}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1270888000269774357, "histo_buckets": [{"distinct_range": 0, "num_eq": 6198319306744011600, "num_range": 0, "upper_bound": "02:22:02.662866"}, {"distinct_range": 0, "num_eq": 8382860214619603885, "num_range": 90000000000, "upper_bound": "05:35:25.991059"}, {"distinct_range": 54.023359101780066, "num_eq": 5000000, "num_range": 800, "upper_bound": "08:42:01.761151"}, {"distinct_range": 784130.305347048, "num_eq": 8621980203830169008, "num_range": 800000, "upper_bound": "13:47:27.304027"}, {"distinct_range": 172.5293080086706, "num_eq": 300000, "num_range": 400, "upper_bound": "16:33:08.222951"}, {"distinct_range": 84326358.03479776, "num_eq": 80000000000, "num_range": 100000000, "upper_bound": "23:55:37.418091"}, {"distinct_range": 8561733211558497000, "num_eq": 8000, "num_range": 8561733211558497703, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 381619170854875344, "row_count": 2655029972590966795}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2187571566939749150, "histo_buckets": [{"distinct_range": 0, "num_eq": 7747476440152674207, "num_range": 0, "upper_bound": "1975-10-22 00:10:16.000504+00:00"}, {"distinct_range": 10000000000, "num_eq": 100, "num_range": 10000000000, "upper_bound": "1981-12-02 22:46:03.000567+00:00"}, {"distinct_range": 0, "num_eq": 1926974144896994326, "num_range": 200, "upper_bound": "1985-02-01 22:26:32.00073+00:00"}, {"distinct_range": 0, "num_eq": 20000000, "num_range": 6910909996675429461, "upper_bound": "3000-01-01 00:00:00+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 2655029972590966795}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2000215397799781386, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2655029972590966795}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 827417788513715341, "histo_col_type": "", "name": "__auto__", "null_count": 920283791323988209, "row_count": 2655029972590966795}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1754225486128095442, "histo_buckets": [{"distinct_range": 0, "num_eq": 8132044054403242333, "num_range": 0, "upper_bound": "1270057530"}, {"distinct_range": 0, "num_eq": 0, "num_range": 70, "upper_bound": "1709564373"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 2655029972590966795}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1631413824899560322, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2184478002217229657}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1182794570645299673, "histo_buckets": [{"distinct_range": 0, "num_eq": 4653902980691940494, "num_range": 0, "upper_bound": "00:30:58.7012"}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 2301535597892635007, "upper_bound": "11:14:20.209688"}, {"distinct_range": 0, "num_eq": 9117917228671533243, "num_range": 2000000, "upper_bound": "14:28:30.932752"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 2184478002217229657}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1791522389389590521, "histo_buckets": [{"distinct_range": 0, "num_eq": 6912142896459794307, "num_range": 0, "upper_bound": "496807.8752630326972"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 2184478002217229657}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2042822457259922513, "histo_col_type": "", "name": "__auto__", "null_count": 1339669963776832806, "row_count": 2184478002217229657}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2081126791620218131, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2184478002217229657}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1167858449230184905, "histo_buckets": [{"distinct_range": 0, "num_eq": 2045348669026929323, "num_range": 0, "upper_bound": "(A\\u0012.MR\\u001d"}, {"distinct_range": 6181427910085550000, "num_eq": 4405157705725028657, "num_range": 8362959337729455537, "upper_bound": "6%\\"0A"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 6735593671105116074, "upper_bound": "R\\u000b\\u0010#\\u001e"}, {"distinct_range": 8000, "num_eq": 2000, "num_range": 8000, "upper_bound": "bA-9\\u000b<~yN"}, {"distinct_range": 99622169861181570, "num_eq": 400000, "num_range": 122801767127269452, "upper_bound": "\U00002603"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 2184478002217229657}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1393399847520311625, "histo_buckets": [{"distinct_range": 0, "num_eq": 5922706549247612434, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 5961222262647604462, "num_range": 8380367832683917591, "upper_bound": "1661638434"}, {"distinct_range": 5000, "num_eq": 6623961219714604713, "num_range": 5000, "upper_bound": "1756821673"}, {"distinct_range": 4408308299862999000, "num_eq": 0, "num_range": 5203028531029069523, "upper_bound": "2241072991"}, {"distinct_range": 0, "num_eq": 2150259654307442676, "num_range": 6594099833661062642, "upper_bound": "2251473405"}, {"distinct_range": 0, "num_eq": 500000, "num_range": 40000, "upper_bound": "3867608922"}, {"distinct_range": 0, "num_eq": 5000000, "num_range": 10000, "upper_bound": "4208465732"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 1285493348762311701, "row_count": 2184478002217229657}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 414819379280804790, "histo_buckets": [{"distinct_range": 0, "num_eq": 5538632212431076368, "num_range": 0, "upper_bound": "4.135.50.15/4"}, {"distinct_range": 70000000000, "num_eq": 400, "num_range": 70000000000, "upper_bound": "48.150.140.143/7"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 600000000, "upper_bound": "172.62.25.45/16"}, {"distinct_range": 0, "num_eq": 1031912710876665405, "num_range": 100000, "upper_bound": "b33e:1ec0:2337:492c:d1cd:41a7:d724:78e3/15"}, {"distinct_range": 300000000, "num_eq": 50000, "num_range": 300000000, "upper_bound": "379c:58b6:ecfa:9749:f79a:a0a9:58c0:8ce2/20"}, {"distinct_range": 0, "num_eq": 0, "num_range": 70, "upper_bound": "a9ac:dc44:9e55:a7ed:db31:dfd7:7fa3:953b/64"}, {"distinct_range": 3000000000, "num_eq": 6000, "num_range": 3000000000, "upper_bound": "7c1c:c113:8e62:49bc:5485:147d:2c1f:827d/90"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 1334796086063652953, "row_count": 2184478002217229657}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1028074934881791230, "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 2184478002217229657}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 628432, "histo_buckets": [{"distinct_range": 0, "num_eq": 5449315655979698225, "num_range": 0, "upper_bound": "-0.5413851757075747"}, {"distinct_range": 5157346201881247000, "num_eq": 0, "num_range": 5157346201881246241, "upper_bound": "5e-324"}, {"distinct_range": 70000000000, "num_eq": 665192024756717382, "num_range": 70000000000, "upper_bound": "0.10222329829365734"}, {"distinct_range": 1126612852094126000, "num_eq": 50000000, "num_range": 1972270095911343876, "upper_bound": "1.697107180505587"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 653155, "histo_col_type": "TIME", "name": "__auto__", "null_count": 159924, "row_count": 700000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 442336, "histo_col_type": "", "name": "__auto__", "null_count": 256440, "row_count": 700000}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 366066, "histo_col_type": "", "name": "__auto__", "null_count": 534400, "row_count": 700000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 540643, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53199, "histo_buckets": [{"distinct_range": 0, "num_eq": 7051177317204354438, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 8330842886789296432, "num_range": 1713212556456966805, "upper_bound": "740352258"}, {"distinct_range": 80000000, "num_eq": 800000, "num_range": 80000000, "upper_bound": "2767997179"}, {"distinct_range": 0, "num_eq": 1269380456780637945, "num_range": 5000000000, "upper_bound": "2784882863"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 2451677848427029478, "upper_bound": "3057843709"}, {"distinct_range": 0, "num_eq": 3746555374103284210, "num_range": 900000000, "upper_bound": "3585916248"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 510309, "histo_col_type": "", "name": "__auto__", "null_count": 46527, "row_count": 700000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 591998, "histo_buckets": [{"distinct_range": 0, "num_eq": 7777037268254400470, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 2861658180265537822, "num_range": 0, "upper_bound": "394268894"}, {"distinct_range": 2999320194978614300, "num_eq": 10000000, "num_range": 2999320194978614100, "upper_bound": "1575093369"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 30000000000, "upper_bound": "2885313206"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 41171, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 590664, "histo_buckets": [{"distinct_range": 0, "num_eq": 8236210048204069762, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 2378057354092122600, "num_eq": 400, "num_range": 5776622057481384363, "upper_bound": "02:21:01.032387+08:59:00"}, {"distinct_range": 0, "num_eq": 6344660329721511345, "num_range": 4338868097194684673, "upper_bound": "07:35:24.577841+07:06:00"}, {"distinct_range": 2602566.0426454875, "num_eq": 5000000, "num_range": 4000000, "upper_bound": "13:40:06.946553-00:08:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 700000}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col5_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1076063965225293974, "histo_col_type": "BYTES", "name": "__auto__", "null_count": 2474553637295822972, "row_count": 5445407718753576799}, {"columns": ["col5_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2045821551134510933, "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 5445407718753576799}, {"columns": ["col5_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2395098508629423318, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5445407718753576799}, {"columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5263147972837079064, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000, "num_range": 0, "upper_bound": "3196703018"}, {"distinct_range": 3253293854737872000, "num_eq": 4176859190730168975, "num_range": 3962977420852961248, "upper_bound": "3262696322"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 5445407718753576799}, {"columns": ["col5_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2510047503280209126, "histo_buckets": [{"distinct_range": 0, "num_eq": 7949649330926767220, "num_range": 0, "upper_bound": "1429979028"}, {"distinct_range": 0, "num_eq": 900000000, "num_range": 3343704662455443457, "upper_bound": "2473173629"}, {"distinct_range": 0, "num_eq": 4146466594104757178, "num_range": 600000000, "upper_bound": "3060236231"}, {"distinct_range": 2530700746167833600, "num_eq": 3450080776989586092, "num_range": 5169567172078292155, "upper_bound": "3340737880"}, {"distinct_range": 0, "num_eq": 0, "num_range": 6670936339529024633, "upper_bound": "3753572103"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 5445407718753576799}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4236683593803015547, "histo_col_type": "", "name": "__auto__", "null_count": 818232358296908984, "row_count": 5445407718753576799}, {"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4441877871561830596, "histo_buckets": [{"distinct_range": 0, "num_eq": 4825134910360266893, "num_range": 0, "upper_bound": "159.7.195.18/2"}, {"distinct_range": 0, "num_eq": 2207848756148841771, "num_range": 1081625263573328969, "upper_bound": "aab9:c156:9e47:a652:ef31:a92d:f537:ef45/11"}, {"distinct_range": 0, "num_eq": 5000, "num_range": 9118744148600004113, "upper_bound": "1641:9a08:2e54:cf02:3b4e:f3de:6538:c115/21"}, {"distinct_range": 92092997396601660, "num_eq": 5000, "num_range": 153663267839038701, "upper_bound": "50c9:b10c:41f8:6c63:b50f:b226:2d9f:80b9/42"}, {"distinct_range": 15009964550120558, "num_eq": 183256049431099464, "num_range": 26558891059051230, "upper_bound": "c968:b071:e177:9cca:a7c5:577e:c8c7:51f4/49"}, {"distinct_range": 4381586811306000400, "num_eq": 5000, "num_range": 4381586811306000457, "upper_bound": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 5445407718753576799}, {"columns": ["col5_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5175226937222993712, "histo_col_type": "", "name": "__auto__", "null_count": 4634766295149990371, "row_count": 5445407718753576799}, {"columns": ["col5_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1617636352626191512, "histo_buckets": [{"distinct_range": 0, "num_eq": 1365179094341941952, "num_range": 0, "upper_bound": "15 years 11 mons 981 days 11:35:44.789934"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 1696114561475972099, "row_count": 5445407718753576799}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('atqv', 'qplbw', 'xfucd', 'vaaf');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('aozcb', 'a');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('kwp', 'nlrdbn', 'xhu', 'dih', 'dtkvg', 'ffyaxb');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('oa', 'q', 't', 'ibsfws', 'mby');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 486903826043521926, "histo_buckets": [{"distinct_range": 0, "num_eq": 300, "num_range": 0, "upper_bound": "\\u0015"}, {"distinct_range": 0, "num_eq": 7445021964439109161, "num_range": 1000000000, "upper_bound": "\'"}, {"distinct_range": 4049297.5642967243, "num_eq": 900000, "num_range": 10000000, "upper_bound": "g"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 5000000, "upper_bound": "\U00002603"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 496985486145430766, "row_count": 718401458327785582}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 459932537641798488, "histo_buckets": [{"distinct_range": 0, "num_eq": 5631055101588516473, "num_range": 0, "upper_bound": "1979-07-22 16:07:37.000926+00:00"}, {"distinct_range": 127794014.57830535, "num_eq": 10000000000, "num_range": 400000000, "upper_bound": "2001-03-05 04:40:26.000679+00:00"}, {"distinct_range": 217695830.21666938, "num_eq": 200000, "num_range": 600000000, "upper_bound": "2020-01-17 08:40:55.000076+00:00"}, {"distinct_range": 92493.73328068819, "num_eq": 200000000, "num_range": 100000, "upper_bound": "2029-04-24 19:51:58.000473+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 718401458327785582}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 505052401183129218, "histo_buckets": [{"distinct_range": 0, "num_eq": 6910428471771089721, "num_range": 0, "upper_bound": "2872670576"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 718401458327785582}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 150872260304734888, "histo_buckets": [{"distinct_range": 0, "num_eq": 5138707143371696406, "num_range": 0, "upper_bound": "2603269213"}, {"distinct_range": 877.4479591562773, "num_eq": 30000000000, "num_range": 8000, "upper_bound": "2605954138"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 273854959721166641, "row_count": 718401458327785582}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 336968670147944526, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 718401458327785582}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 247234700263239266, "histo_col_type": "", "name": "__auto__", "null_count": 572401123411312672, "row_count": 718401458327785582}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 698031847059763112, "histo_buckets": [{"distinct_range": 0, "num_eq": 8426708072486026811, "num_range": 0, "upper_bound": "00:00:00"}, {"distinct_range": 0, "num_eq": 90, "num_range": 5000000, "upper_bound": "00:43:45.780648"}, {"distinct_range": 0, "num_eq": 30000000000, "num_range": 400, "upper_bound": "16:24:31.526055"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 0, "upper_bound": "19:42:25.278001"}, {"distinct_range": 7020536700339586000, "num_eq": 3911400219004278419, "num_range": 7020536700339585753, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 695765232354691359, "row_count": 718401458327785582}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 518449081842758904, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 718401458327785582}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 150849840702274376, "histo_col_type": "", "name": "__auto__", "null_count": 348777564777505260, "row_count": 718401458327785582}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 526816933751113835, "histo_col_type": "", "name": "__auto__", "null_count": 368882378269378442, "row_count": 718401458327785582}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 659395695133523926, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 7239078371161546000, "num_eq": 100000, "num_range": 8768118785329673354, "upper_bound": "BOX(-1.0098519543771063 -0.7608057313730585,-0.5784136896984938 0.9696960117270929)"}, {"distinct_range": 200000000, "num_eq": 5000, "num_range": 200000000, "upper_bound": "BOX(-0.5085005684263971 -1.1705931445712818,0.6670310705246659 0.7770384496228739)"}, {"distinct_range": 0, "num_eq": 6364467721023107561, "num_range": 70000000000, "upper_bound": "BOX(-0.4776861129658254 -1.7173862734758139,1.0513958930804792 -0.4309075208255022)"}, {"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": "BOX(-0.17572569156315487 -0.5806229432311759,0.6037669357008693 0.7058623633051748)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 66209819794584668, "row_count": 718401458327785582}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 93150970839, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22317674211, "histo_buckets": [{"distinct_range": 0, "num_eq": 3981814721934058037, "num_range": 0, "upper_bound": "01:35:22.156669"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 100000000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28922465495, "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 100000000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16727556748, "histo_col_type": "", "name": "__auto__", "null_count": 97963896569, "row_count": 100000000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25151436113, "histo_buckets": [{"distinct_range": 0, "num_eq": 80, "num_range": 0, "upper_bound": "\\u0015/ q\x7f+3"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 100000000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 97394462921, "histo_buckets": [{"distinct_range": 0, "num_eq": 53676552543459906, "num_range": 0, "upper_bound": "19696283"}, {"distinct_range": 0, "num_eq": 9000, "num_range": 4902414196427209950, "upper_bound": "80472282"}, {"distinct_range": 1661796593077553000, "num_eq": 6390562352409546790, "num_range": 1661796593077552835, "upper_bound": "804234188"}, {"distinct_range": 12277793935.019852, "num_eq": 10000, "num_range": 100000000000, "upper_bound": "877260923"}, {"distinct_range": 355303185.2881732, "num_eq": 6000000, "num_range": 600000000, "upper_bound": "2216593539"}, {"distinct_range": 0, "num_eq": 20000000000, "num_range": 80000000, "upper_bound": "2776327700"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 33590351235, "row_count": 100000000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14998375713, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": "0.0.0.0/0"}, {"distinct_range": 0, "num_eq": 600000, "num_range": 2193191223972811756, "upper_bound": "18.15.86.93/20"}, {"distinct_range": 9020478115342477000, "num_eq": 3000000, "num_range": 9020478115342477388, "upper_bound": "c682:3564:7bf1:2b7e:d6df:a3:5a15:1655/22"}, {"distinct_range": 0, "num_eq": 7256953882465687477, "num_range": 5816998503787177797, "upper_bound": "e4b0:b9da:1c0f:358a:859e:47f:b949:8b86/43"}, {"distinct_range": 0, "num_eq": 6000000, "num_range": 6016054107466570141, "upper_bound": "5ffe:a426:780e:fdbd:a60f:5f44:b84f:2fc1/48"}, {"distinct_range": 0, "num_eq": 3549880848702132233, "num_range": 7029519442681569184, "upper_bound": "c3b5:7383:f754:367c:296a:b7a0:939d:63ae/49"}, {"distinct_range": 0, "num_eq": 800, "num_range": 1483263476110449372, "upper_bound": "d794:2978:a827:29c9:2b07:e155:93c:8e72/104"}, {"distinct_range": 2576153226603555300, "num_eq": 3000000000, "num_range": 2576153226603555515, "upper_bound": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 4562358119, "row_count": 100000000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 92448220132, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000, "num_range": 0, "upper_bound": "-6.496525646345333E+18"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 100000000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 92191951155, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1566429253, "histo_col_type": "", "name": "__auto__", "null_count": 22568116199, "row_count": 40000000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35470856402, "histo_col_type": "", "name": "__auto__", "null_count": 17896954358, "row_count": 40000000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5851618079, "histo_col_type": "", "name": "__auto__", "null_count": 7261156822, "row_count": 40000000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15328985800, "histo_col_type": "", "name": "__auto__", "null_count": 23533342856, "row_count": 40000000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24166662773, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19562421237, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17235703551, "histo_col_type": "", "name": "__auto__", "null_count": 16710299282, "row_count": 40000000000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24276306961, "histo_col_type": "", "name": "__auto__", "null_count": 27887394042, "row_count": 40000000000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2218605855594270017, "histo_buckets": [{"distinct_range": 0, "num_eq": 5319920772716672144, "num_range": 0, "upper_bound": "06:35:05.467433"}, {"distinct_range": 2408547017369829000, "num_eq": 3000, "num_range": 5184755495909576230, "upper_bound": "07:37:23.647162"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 8000, "upper_bound": "08:11:27.165496"}, {"distinct_range": 9017936715528493000, "num_eq": 8419784293026414664, "num_range": 9017936715528492952, "upper_bound": "09:09:24.152304"}, {"distinct_range": 8025950751469540000, "num_eq": 200000, "num_range": 8025950751469539847, "upper_bound": "20:04:05.910857"}, {"distinct_range": 1731269121798273300, "num_eq": 700000000, "num_range": 3005108415355626434, "upper_bound": "20:10:19.494754"}, {"distinct_range": 0, "num_eq": 4686455794653955073, "num_range": 5198793706968760034, "upper_bound": "21:33:45.098634"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 1431577506625754884, "row_count": 2956535953560722476}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 441648819058424563, "histo_buckets": [{"distinct_range": 0, "num_eq": 9222421402057452534, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 1084303305475614575, "num_range": 7000000000, "upper_bound": "650479937"}, {"distinct_range": 3732054078777149400, "num_eq": 6241978499142014945, "num_range": 4228255628661561934, "upper_bound": "760942653"}, {"distinct_range": 77.93241095746056, "num_eq": 0, "num_range": 90, "upper_bound": "804522948"}, {"distinct_range": 1077833048682435700, "num_eq": 5499427574907049368, "num_range": 3512627545240782508, "upper_bound": "1331096211"}, {"distinct_range": 1453.4176992287046, "num_eq": 7353994547475193212, "num_range": 7000, "upper_bound": "1612421155"}, {"distinct_range": 5259.674044683262, "num_eq": 3030541013387005785, "num_range": 8000, "upper_bound": "2518554739"}, {"distinct_range": 3569794972279908000, "num_eq": 4531168040924549009, "num_range": 3569794972279907898, "upper_bound": "3116291161"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 2956535953560722476}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 807595316045646817, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 3440656691700441000, "num_eq": 900000, "num_range": 6500887627444115808, "upper_bound": "09:56:06.209168+08:26:00"}, {"distinct_range": 1005.7983782298578, "num_eq": 9052314904106284139, "num_range": 3000, "upper_bound": "04:36:06.101123+02:43:00"}, {"distinct_range": 5000000, "num_eq": 0, "num_range": 5000000, "upper_bound": "19:46:29.08023+02:34:00"}, {"distinct_range": 70000000000, "num_eq": 8662181005918214997, "num_range": 70000000000, "upper_bound": "11:09:09.087901-12:56:00"}, {"distinct_range": 50000000000, "num_eq": 20000000000, "num_range": 50000000000, "upper_bound": "23:58:11.778627-08:47:00"}, {"distinct_range": 0, "num_eq": 1614574976628619869, "num_range": 0, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 2956535953560722476}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2605463534814742158, "histo_col_type": "", "name": "__auto__", "null_count": 1891829893980362224, "row_count": 2956535953560722476}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2216931745233227369, "histo_buckets": [{"distinct_range": 0, "num_eq": 3799735775018095385, "num_range": 0, "upper_bound": "0.30720417329793454"}, {"distinct_range": 0, "num_eq": 0, "num_range": 70000000000, "upper_bound": "0.6990411332434869"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 2956535953560722476}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 427256291030618935, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2956535953560722476}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 443040945325344172, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000, "num_range": 0, "upper_bound": "1180415745"}, {"distinct_range": 816220744219779000, "num_eq": 674849979037002951, "num_range": 4683180288790340885, "upper_bound": "1515737883"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 2956535953560722476}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 660481884642140640, "histo_col_type": "", "name": "__auto__", "null_count": 2895913909029543950, "row_count": 2956535953560722476}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1808249076742867685, "histo_col_type": "", "name": "__auto__", "null_count": 2808719493097221368, "row_count": 2956535953560722476}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 811082086432286960, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2956535953560722476}]';
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col5_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6861455935086947871, "histo_buckets": [{"distinct_range": 0, "num_eq": 5359065999343246878, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 2745667225760831000, "num_eq": 400000000, "num_range": 8801008815565000220, "upper_bound": "1074588819"}, {"distinct_range": 0, "num_eq": 4073274592724270061, "num_range": 7306221625649945453, "upper_bound": "1738989721"}, {"distinct_range": 3058880942404715500, "num_eq": 0, "num_range": 8693161924113009922, "upper_bound": "2122045092"}, {"distinct_range": 0, "num_eq": 8904008913393000176, "num_range": 7000000000, "upper_bound": "2624942138"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 8983451758877856359}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 507207011340530736, "histo_col_type": "", "name": "__auto__", "null_count": 2175661151585198633, "row_count": 8983451758877856359}, {"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1745765449830638463, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000000000, "num_range": 0, "upper_bound": "232.138.120.141/26"}, {"distinct_range": 40000000000, "num_eq": 631884860698360332, "num_range": 40000000000, "upper_bound": "705c:ab3a:8df:e1ec:36b6:9f73:f424:fa18/10"}, {"distinct_range": 450792.2046549807, "num_eq": 80000, "num_range": 1000000, "upper_bound": "9fe9:235f:6a6c:c022:245a:8507:7267:88c8/105"}, {"distinct_range": 0, "num_eq": 1467437737678260253, "num_range": 1439273599759622706, "upper_bound": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 8983451758877856359}, {"columns": ["col5_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7247674444481615111, "histo_col_type": "BYTES", "name": "__auto__", "null_count": 3371787678180437225, "row_count": 8983451758877856359}, {"columns": ["col5_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4604795303233843952, "histo_buckets": [{"distinct_range": 0, "num_eq": 30, "num_range": 0, "upper_bound": "41.140.208.149/24"}, {"distinct_range": 0, "num_eq": 1258029871773460987, "num_range": 950254330615341873, "upper_bound": "55.74.78.24"}, {"distinct_range": 2864719396333678000, "num_eq": 2391718908640950907, "num_range": 5751422528231583403, "upper_bound": "71b3:5ce2:f059:7541:19b2:7cbb:c0b9:3ee9/39"}, {"distinct_range": 0, "num_eq": 7987911214889866413, "num_range": 500000, "upper_bound": "6e5e:d39c:eb59:d40d:426:d9c9:4007:ae0e/51"}, {"distinct_range": 100000, "num_eq": 6000000, "num_range": 100000, "upper_bound": "8a9e:3ee1:20d0:601f:8721:346:2579:c371/64"}, {"distinct_range": 5514829.412192738, "num_eq": 70000000, "num_range": 6000000, "upper_bound": "6444:fcd7:df2d:f79:cfcd:a9c5:f135:2ee4/73"}, {"distinct_range": 3324409633512253000, "num_eq": 200000000, "num_range": 6914105886487284135, "upper_bound": "4aa0:e89c:2d6d:c788:4baf:e2bd:7dfe:102d/81"}, {"distinct_range": 3460913373078402000, "num_eq": 7004492857106811979, "num_range": 7558949204310971831, "upper_bound": "145b:a1ed:c403:e8db:3a67:9a2a:ef8b:e6a/86"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 8983451758877856359}, {"columns": ["col5_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2024746456864465963, "histo_buckets": [{"distinct_range": 0, "num_eq": 2987134268361635572, "num_range": 0, "upper_bound": "-59 years -3 mons -528 days -15:18:52.387692"}, {"distinct_range": 0, "num_eq": 1832036300597369634, "num_range": 3914268483185716799, "upper_bound": "-57 years -11 mons -314 days -23:11:15.396202"}, {"distinct_range": 4417599782052415500, "num_eq": 30000000000, "num_range": 4417599782052415270, "upper_bound": "00:00:00"}, {"distinct_range": 0, "num_eq": 5000, "num_range": 4437490321863464256, "upper_bound": "1 mon 1 day"}, {"distinct_range": 40000000000, "num_eq": 3853737113330643576, "num_range": 40000000000, "upper_bound": "2 years 2 mons 601 days 16:37:13.013479"}, {"distinct_range": 3252912668406858000, "num_eq": 900, "num_range": 3252912668406858399, "upper_bound": "27 years 10 mons 561 days 04:44:16.120404"}, {"distinct_range": 30000, "num_eq": 0, "num_range": 30000, "upper_bound": "40 years 10 mons 26 days 17:07:15.983275"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 1880387153798937213, "row_count": 8983451758877856359}, {"columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8517981090670506576, "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 8983451758877856359}, {"columns": ["col5_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1801809311747110626, "histo_col_type": "", "name": "__auto__", "null_count": 6707132095022468923, "row_count": 8983451758877856359}, {"columns": ["col5_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3436848304282998623, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8983451758877856359}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5149467414421673709, "histo_col_type": "", "name": "__auto__", "null_count": 4668883296209596311, "row_count": 8114663594395470733}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4359357771469875306, "histo_buckets": [{"distinct_range": 0, "num_eq": 2537163967616268831, "num_range": 0, "upper_bound": "4102778088"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 3202908974688145405, "row_count": 8114663594395470733}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2467878555501261667, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "00:00:00"}, {"distinct_range": 1781205283333638100, "num_eq": 500000, "num_range": 3490959378540847352, "upper_bound": "04:47:31.371414"}, {"distinct_range": 2470277907080031700, "num_eq": 200, "num_range": 2572718731446321821, "upper_bound": "05:02:19.230681"}, {"distinct_range": 60000, "num_eq": 5986753272881437958, "num_range": 60000, "upper_bound": "10:55:32.454138"}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 3309507519668823690, "upper_bound": "11:37:46.344777"}, {"distinct_range": 0, "num_eq": 9194366071905334802, "num_range": 8614168758658547402, "upper_bound": "11:46:17.292726"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 6000000, "upper_bound": "15:48:15.373184"}, {"distinct_range": 244222629978052900, "num_eq": 8544182879396926104, "num_range": 862160155210072018, "upper_bound": "21:54:00.981421"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 2423194804255373654, "row_count": 8114663594395470733}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10404732369687829, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000, "num_range": 0, "upper_bound": "1992-08-16 19:48:01.000359+00:00"}, {"distinct_range": 49903365205851650, "num_eq": 30, "num_range": 1666491288508374989, "upper_bound": "1997-01-19 13:51:59.000306+00:00"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 200000000, "upper_bound": "2000-07-14 19:30:34.000098+00:00"}, {"distinct_range": 0, "num_eq": 90000000000, "num_range": 9000000000, "upper_bound": "2015-06-13 09:57:26.000302+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 8114663594395470733}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2072573109390053048, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8114663594395470733}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 430648077310897676, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "BOX(-1.7303098801851582 -0.8197352299239984,-1.510203535989909 1.6205896205183752)"}, {"distinct_range": 0, "num_eq": 1239508435263474153, "num_range": 0, "upper_bound": "BOX(-0.4623145627299688 -1.1682199522490775,-0.10417569840753504 1.247797074991277)"}, {"distinct_range": 0, "num_eq": 2660545175523015137, "num_range": 60000000, "upper_bound": "BOX(0.9514795032636749 -0.4318447927773139,1.4543337370497469 0.01824804728990409)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 1091284622928428288, "row_count": 8114663594395470733}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2981170488751757458, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8114663594395470733}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3632073923663507513, "histo_col_type": "", "name": "__auto__", "null_count": 7838161274431118924, "row_count": 8114663594395470733}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4415059347973062207, "histo_buckets": [{"distinct_range": 0, "num_eq": 2672898234013971717, "num_range": 0, "upper_bound": "\\u0012"}, {"distinct_range": 0, "num_eq": 0, "num_range": 495267934774773364, "upper_bound": "\\""}, {"distinct_range": 20000000, "num_eq": 1438800475248793789, "num_range": 20000000, "upper_bound": "("}, {"distinct_range": 3640064.975013995, "num_eq": 5204124561019853151, "num_range": 6000000, "upper_bound": "*"}, {"distinct_range": 30000000, "num_eq": 0, "num_range": 30000000, "upper_bound": ","}, {"distinct_range": 6745149070496887000, "num_eq": 283242418522673181, "num_range": 6745149070496886671, "upper_bound": "F"}, {"distinct_range": 4111170345771469000, "num_eq": 446293656691829948, "num_range": 4111170345771468571, "upper_bound": "H"}, {"distinct_range": 9000000, "num_eq": 5401496249169473119, "num_range": 9000000, "upper_bound": "f"}, {"distinct_range": 0, "num_eq": 50, "num_range": 2210245913876704518, "upper_bound": "j"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 3763653777134314761, "row_count": 8114663594395470733}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6475886465717150997, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000000, "num_range": 0, "upper_bound": "371500363"}, {"distinct_range": 929179418836744200, "num_eq": 600000000, "num_range": 7296533842259008851, "upper_bound": "3209126764"}, {"distinct_range": 0, "num_eq": 100, "num_range": 8000000, "upper_bound": "3528740322"}, {"distinct_range": 700000, "num_eq": 5000000000, "num_range": 700000, "upper_bound": "3631410811"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 8114663594395470733}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 869482179744324814, "histo_col_type": "", "name": "__auto__", "null_count": 4747975394239309409, "row_count": 8114663594395470733}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 247755441953638622, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 614507555074977049}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 407913899367129489, "histo_buckets": [{"distinct_range": 0, "num_eq": 8540105886539917841, "num_range": 0, "upper_bound": "\\u0012\\u00189MF$\\u0001)\\""}, {"distinct_range": 0, "num_eq": 900000000, "num_range": 5346012893443834890, "upper_bound": "\'"}, {"distinct_range": 3414976157238744000, "num_eq": 9000, "num_range": 3414976157238744132, "upper_bound": "("}, {"distinct_range": 6000000, "num_eq": 1300756778925433848, "num_range": 6000000, "upper_bound": "+"}, {"distinct_range": 0, "num_eq": 230541484111477708, "num_range": 9000000000, "upper_bound": "Q0Uy"}, {"distinct_range": 0, "num_eq": 8646712888610148397, "num_range": 200000, "upper_bound": "[8+m"}, {"distinct_range": 0, "num_eq": 0, "num_range": 1000, "upper_bound": "\\\\l"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 614507555074977049}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 60836533623670186, "histo_buckets": [{"distinct_range": 0, "num_eq": 3616694426728522793, "num_range": 0, "upper_bound": "P"}, {"distinct_range": 0, "num_eq": 5000000, "num_range": 400000, "upper_bound": "UDfN#\\u0007\\u0018"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 614507555074977049}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 113877277517412002, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 614507555074977049}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 233622810727485489, "histo_buckets": [{"distinct_range": 0, "num_eq": 1162967570354227050, "num_range": 0, "upper_bound": "1028294194"}, {"distinct_range": 0, "num_eq": 8917539019779544103, "num_range": 400, "upper_bound": "1218517133"}, {"distinct_range": 0, "num_eq": 60, "num_range": 5000000, "upper_bound": "1506457814"}, {"distinct_range": 0, "num_eq": 0, "num_range": 300000000, "upper_bound": "3559467970"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 190622856610261714, "row_count": 614507555074977049}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 245484725666109529, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000, "num_range": 0, "upper_bound": "46.179.165.158/2"}, {"distinct_range": 4413324283751060000, "num_eq": 4053797063447003040, "num_range": 4413324283751059929, "upper_bound": "177.145.38.236/13"}, {"distinct_range": 0, "num_eq": 5533834451464461858, "num_range": 6863309543028535163, "upper_bound": "20.201.17.182/23"}, {"distinct_range": 2501569654.595691, "num_eq": 1719670359922343990, "num_range": 4000000000, "upper_bound": "190.168.252.6/24"}, {"distinct_range": 0.9112461380030066, "num_eq": 30, "num_range": 10, "upper_bound": "255.228.99.5/31"}, {"distinct_range": 0, "num_eq": 1091092325606738030, "num_range": 2350652855540431281, "upper_bound": "6b:82ae:b63f:67f0:3303:afc3:f9d6:656c/23"}, {"distinct_range": 1546742017732571400, "num_eq": 6000000000, "num_range": 7937556892007191201, "upper_bound": "de64:388e:e70b:6671:f5d4:fa48:13b7:a14c/98"}, {"distinct_range": 100000, "num_eq": 3000000000, "num_range": 100000, "upper_bound": "fdce:e3af:924a:acaf:8422:f9a9:3723:515c/119"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 246847033768406620, "row_count": 614507555074977049}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 372991234208364239, "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 614507555074977049}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 544847258009283916, "histo_buckets": [{"distinct_range": 0, "num_eq": 2837617702621051635, "num_range": 0, "upper_bound": "-Infinity"}, {"distinct_range": 3363388230820559000, "num_eq": 824532194633429235

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2639233&tab=buildLog) on [master@4736b8dd2c36b58fffbb0a665690e2c0e719d7c3](https://github.com/cockroachdb/cockroach/commits/4736b8dd2c36b58fffbb0a665690e2c0e719d7c3):

```
        test_1        |             CREATE INDEX ON table5 (col5_0 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_0);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_0 DESC);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 278, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000000000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 4940471614214057319, "num_range": 1000000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 691, "row_count": 800}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 353, "histo_buckets": [{"distinct_range": 0, "num_eq": 3645492383993722925, "num_range": 0, "upper_bound": "-2.013864097742168585E+35"}, {"distinct_range": 0, "num_eq": 2234432530522602887, "num_range": 8000000, "upper_bound": "-1.336879529698030384E+34"}, {"distinct_range": 7129654554666742000, "num_eq": 8276861759169769643, "num_range": 7129654554666741634, "upper_bound": "-2.037379964756336304E+21"}, {"distinct_range": 8000, "num_eq": 9000000, "num_range": 8000, "upper_bound": "-422079647660807363.1"}, {"distinct_range": 0, "num_eq": 30000000, "num_range": 0, "upper_bound": "-90.37093081139125006"}, {"distinct_range": 0, "num_eq": 5954171762882514838, "num_range": 568681960061113352, "upper_bound": "821673724284.852292"}, {"distinct_range": 5932674599635818000, "num_eq": 2000000000, "num_range": 5932674599635818136, "upper_bound": "5.363270431622302358E+30"}, {"distinct_range": 100000, "num_eq": 700000, "num_range": 100000, "upper_bound": "Infinity"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 588, "histo_buckets": [{"distinct_range": 0, "num_eq": 319096339997412840, "num_range": 0, "upper_bound": "\\\\x09ee5e89da29d1b802"}, {"distinct_range": 50000, "num_eq": 3735570504475252196, "num_range": 50000, "upper_bound": "\\\\x7299132ba0e6141f"}, {"distinct_range": 10000000, "num_eq": 7475003052232107289, "num_range": 10000000, "upper_bound": "\\\\x80aa92c48f"}, {"distinct_range": 17844392.799908202, "num_eq": 7000000000, "num_range": 100000000, "upper_bound": "\\\\x8fac961f61242fcb"}, {"distinct_range": 0, "num_eq": 7810224562253564991, "num_range": 5597370961166241050, "upper_bound": "\\\\xfc250175b15466d7"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 484, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000000, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 7000000, "num_eq": 6114846471992232323, "num_range": 7000000, "upper_bound": "^"}, {"distinct_range": 68.58940774617965, "num_eq": 1865780532050139376, "num_range": 80, "upper_bound": "e"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 232, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 891755442, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000, "num_range": 0, "upper_bound": "\'"}, {"distinct_range": 2050949921874527200, "num_eq": 70000, "num_range": 2050949921874527318, "upper_bound": "J"}, {"distinct_range": 975638525210682600, "num_eq": 300000, "num_range": 5097231119377995998, "upper_bound": "L"}, {"distinct_range": 2815369865053686000, "num_eq": 1000000, "num_range": 4260431341612538804, "upper_bound": "d"}, {"distinct_range": 0, "num_eq": 3555087170690215101, "num_range": 7755082642882721914, "upper_bound": "p"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 752248577, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "0.0.0.0/0"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 900, "upper_bound": "126.42.174.69/6"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 500, "upper_bound": "44.107.220.124/20"}, {"distinct_range": 5227299676249610000, "num_eq": 4182703571635544733, "num_range": 5227299676249609936, "upper_bound": "162.147.52.97/26"}, {"distinct_range": 0, "num_eq": 2303928575239141127, "num_range": 50, "upper_bound": "111.118.194.247/27"}, {"distinct_range": 400000000, "num_eq": 784087050098006761, "num_range": 400000000, "upper_bound": "bd64:5c1b:9355:ad1e:9ed:d462:4322:8f61/3"}, {"distinct_range": 100000, "num_eq": 3536620552541305098, "num_range": 100000, "upper_bound": "bf20:7f96:48a1:c9cf:ea95:335c:78d2:c71/52"}, {"distinct_range": 4221158800628996600, "num_eq": 7402939399172902399, "num_range": 4221158800628996766, "upper_bound": "825d:d187:593b:2b59:b3e7:a1f:8a68:fa14/93"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 853291607, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 857866264, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 589729197, "histo_col_type": "", "name": "__auto__", "null_count": 131843452, "row_count": 1000000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 477418651, "histo_buckets": [{"distinct_range": 0, "num_eq": 2355017146107839715, "num_range": 0, "upper_bound": "-Inf"}, {"distinct_range": 0, "num_eq": 900000000, "num_range": 6000000, "upper_bound": "-0.45378994941711426"}, {"distinct_range": 7647238552826033000, "num_eq": 1000000, "num_range": 7647238552826033132, "upper_bound": "-0.44646885991096497"}, {"distinct_range": 5042540.519894369, "num_eq": 4274755106060549364, "num_range": 6000000, "upper_bound": "-0.24441342055797577"}, {"distinct_range": 837596.1873451264, "num_eq": 5000000, "num_range": 1000000, "upper_bound": "0.15669812262058258"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 2000000, "upper_bound": "0.2868598401546478"}, {"distinct_range": 74424273802.61292, "num_eq": 4000000000, "num_range": 80000000000, "upper_bound": "0.6626138687133789"}, {"distinct_range": 1526703624101105200, "num_eq": 2982738433901892796, "num_range": 5109572149039909038, "upper_bound": "0.7538302540779114"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 496011438, "row_count": 1000000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 184576661, "histo_buckets": [{"distinct_range": 0, "num_eq": 3534864867684713735, "num_range": 0, "upper_bound": "0101110101100101000111111101100100100100000111"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 60, "upper_bound": "0111111111111111111111111111111111111111111111"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 200, "upper_bound": "1001000100000001011000011010100011000000000010"}, {"distinct_range": 8832830002396893000, "num_eq": 9109917613021194392, "num_range": 8832830002396893433, "upper_bound": "1010100100101010111100001100001100000011011100"}, {"distinct_range": 8000, "num_eq": 8537210349831887319, "num_range": 8000, "upper_bound": "1101000011000011111010001101000101100011000001"}, {"distinct_range": 7206788538171785000, "num_eq": 8444654467856594170, "num_range": 7206788538171785539, "upper_bound": "1111100111101001011111110111101111010111110010"}], "histo_col_type": "BIT(46)", "name": "__auto__", "null_count": 728490280, "row_count": 1000000000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 768144844, "histo_col_type": "", "name": "__auto__", "null_count": 556721469, "row_count": 1000000000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 910042181, "histo_col_type": "", "name": "__auto__", "null_count": 493301206, "row_count": 1000000000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 634191369, "histo_col_type": "", "name": "__auto__", "null_count": 771242273, "row_count": 1000000000}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 661871005, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "-2470249605571681856"}, {"distinct_range": 5072654082.161617, "num_eq": 70000, "num_range": 20000000000, "upper_bound": "318428666509413728"}, {"distinct_range": 90842176410600380, "num_eq": 400, "num_range": 273397055775234820, "upper_bound": "1180361447057336821"}, {"distinct_range": 2712382170399795700, "num_eq": 50000000000, "num_range": 2913342815203316868, "upper_bound": "1635285398494498604"}, {"distinct_range": 191382.72669898396, "num_eq": 12332586665109600, "num_range": 1000000, "upper_bound": "3010572445164229618"}, {"distinct_range": 480915.71192237776, "num_eq": 700000000, "num_range": 800000, "upper_bound": "5359851086716331928"}, {"distinct_range": 0, "num_eq": 9000000, "num_range": 5830697746215513077, "upper_bound": "6854042426575411223"}, {"distinct_range": 80000000, "num_eq": 80000000, "num_range": 80000000, "upper_bound": "7540928394318072388"}, {"distinct_range": 1962883060397139200, "num_eq": 2840556487529528883, "num_range": 5888115277859195984, "upper_bound": "8870506959394727632"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 15748942, "row_count": 1000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27633340, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 127140913, "histo_col_type": "", "name": "__auto__", "null_count": 860877713, "row_count": 1000000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 866090211, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 821014896, "histo_col_type": "", "name": "__auto__", "null_count": 579189485, "row_count": 1000000000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 312345445, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 81790886, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 577664981, "histo_col_type": "", "name": "__auto__", "null_count": 393239107, "row_count": 1000000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 966743700, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 251, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 961, "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 647, "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 436, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 964, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 468, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 495, "histo_col_type": "", "name": "__auto__", "null_count": 729, "row_count": 1000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 703, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 305, "histo_buckets": [{"distinct_range": 0, "num_eq": 395054460730272569, "num_range": 0, "upper_bound": "-5453319672425270286"}, {"distinct_range": 563.0324879408757, "num_eq": 10000000000, "num_range": 900, "upper_bound": "-1411031010630334109"}, {"distinct_range": 0, "num_eq": 3299051704671572430, "num_range": 10, "upper_bound": "-999165841295543806"}, {"distinct_range": 600, "num_eq": 6287857978893058094, "num_range": 600, "upper_bound": "4094740611928405570"}, {"distinct_range": 8000, "num_eq": 100000000000, "num_range": 8000, "upper_bound": "4265445806934562121"}, {"distinct_range": 139780.89404713342, "num_eq": 700000000, "num_range": 200000, "upper_bound": "4353413448406052345"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 604, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 995, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 641, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 158535070, "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 95937649, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 487342088, "histo_col_type": "", "name": "__auto__", "null_count": 329204838, "row_count": 500000000}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 402331396, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 10, "num_range": 6000, "upper_bound": "130153573"}, {"distinct_range": 0, "num_eq": 800, "num_range": 5461741360149568067, "upper_bound": "1301932024"}, {"distinct_range": 1307456328626766800, "num_eq": 70000000000, "num_range": 1882785311832593010, "upper_bound": "2188217066"}, {"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "2851807414"}, {"distinct_range": 0, "num_eq": 10, "num_range": 1000, "upper_bound": "3202076709"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 36456493, "row_count": 500000000}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 282973257, "histo_col_type": "", "name": "__auto__", "null_count": 405131979, "row_count": 500000000}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 369914332, "histo_buckets": [{"distinct_range": 0, "num_eq": 6594167806346223785, "num_range": 0, "upper_bound": "false"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 482416498, "histo_col_type": "", "name": "__auto__", "null_count": 215383701, "row_count": 500000000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 230922636, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col4_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 163932535, "histo_col_type": "", "name": "__auto__", "null_count": 105190650, "row_count": 500000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 161774535, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 255839267, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000, "num_range": 0, "upper_bound": "-7143025944879506799"}, {"distinct_range": 600000000, "num_eq": 3000, "num_range": 600000000, "upper_bound": "-6043631991960113931"}, {"distinct_range": 0, "num_eq": 60000, "num_range": 8000000000, "upper_bound": "-3438259774416239975"}, {"distinct_range": 800000, "num_eq": 127256376198109531, "num_range": 800000, "upper_bound": "-2093964876060673127"}, {"distinct_range": 3000, "num_eq": 2218120674389544054, "num_range": 3000, "upper_bound": "-911281289574179064"}, {"distinct_range": 0, "num_eq": 3609352123329341442, "num_range": 189078288948116567, "upper_bound": "46156198130496920"}, {"distinct_range": 0, "num_eq": 80, "num_range": 600000, "upper_bound": "6075906450426368673"}, {"distinct_range": 0, "num_eq": 80, "num_range": 4543583365388111865, "upper_bound": "6802763063223578691"}, {"distinct_range": 0, "num_eq": 4074555213576953098, "num_range": 300, "upper_bound": "8466804283778739738"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col4_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 140130311, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 52765887, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000, "num_range": 0, "upper_bound": "-1.639272689819336"}, {"distinct_range": 722702395558067500, "num_eq": 800000000, "num_range": 722702395558067495, "upper_bound": "0.16675198078155518"}, {"distinct_range": 6310496920.432929, "num_eq": 3865257833880811605, "num_range": 100000000000, "upper_bound": "0.2591170072555542"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 63120973, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 272332827, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 350145174, "histo_col_type": "", "name": "__auto__", "null_count": 284162740, "row_count": 500000000}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 384543210, "histo_col_type": "", "name": "__auto__", "null_count": 413770029, "row_count": 500000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 377789573, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 305749060, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 430610002, "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 500000000}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 85317, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 0, "num_eq": 500000, "num_range": 8809074546997383240, "upper_bound": "BOX(-2.168319895042056 -1.675049342666171,0.5168755168529708 0.9616963112632231)"}, {"distinct_range": 0, "num_eq": 7537561807843101856, "num_range": 10000000000, "upper_bound": "BOX(-1.073256403936169 -0.5327175336728942,-0.4716763318017886 -0.26678157504098216)"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 100000000000, "upper_bound": "BOX(-1.0141311479309438 -0.6967849200706437,1.1629150867650275 1.2849333480849627)"}, {"distinct_range": 731443.9680930328, "num_eq": 382175344308067545, "num_range": 1000000, "upper_bound": "BOX(-0.4224973122122356 -0.6380700681851212,-0.4055005460400668 0.06665387062381711)"}, {"distinct_range": 50000, "num_eq": 2000, "num_range": 50000, "upper_bound": "BOX(-0.37869956781596853 1.0043596845849287,0.8382862960513195 1.8238105519449799)"}, {"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "BOX(0.3050423349687642 -1.5542605946614403,0.7896476068379353 -0.12008446416314461)"}, {"distinct_range": 4692864.606527916, "num_eq": 2665256235416163974, "num_range": 7000000, "upper_bound": "BOX(0.3969137348811713 -0.9186059594642748,1.2972256922177883 1.5908515178760987)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 90000}]':::JSONB;
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_2) REFERENCES table2 (col1_2) ON DELETE RESTRICT;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('bvbe', 'et', 'yjugbo', 'ui', 'idrpc', 'ehm');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('xf', 'xuhcza', 'asiaik', 'u', 'nnmq');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('t');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        |     --- FAIL: TestCompare/mutators (0.19s)
        test_1        |         compare_test.go:129: 
        test_1        |             		SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;
        test_1        |             		SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;
        test_1        |             	CREATE TABLE table1 (col1_0 BOX2D NOT NULL, col1_1 TIMESTAMPTZ NOT NULL, col1_2 STRING, col1_3 BOOL NOT NULL, col1_4 REGNAMESPACE NOT NULL, col1_5 DATE NOT NULL, col1_6 "char" NOT NULL, col1_7 BIT(17) NOT NULL, col1_8 REGCLASS NOT NULL, col1_9 OID[] NOT NULL, col1_10 BOOL NOT NULL, col1_11 VARCHAR, col1_12 DECIMAL NOT NULL, col1_13 FLOAT4 NOT NULL, col1_14 STRING NOT NULL, col1_15 FLOAT4 NOT NULL, col1_16 BIT(6) NOT NULL, PRIMARY KEY (col1_15 ASC, col1_8 ASC, col1_13 ASC, col1_14 ASC, col1_3 ASC, col1_1, col1_5 DESC, col1_6, col1_16 DESC, col1_0 DESC, col1_4, col1_7), INDEX (col1_14 DESC, col1_0, col1_7 DESC, col1_2, col1_11 ASC, col1_12, col1_5, col1_1, col1_10 DESC, col1_6 DESC, col1_13), INDEX (col1_16 ASC, col1_3 ASC, col1_11 ASC, col1_0 DESC, col1_15, col1_8 ASC) STORING (col1_2, col1_9), INDEX (col1_6 ASC, col1_3, col1_10, col1_16 ASC, col1_0 ASC, col1_11, col1_15, col1_2 DESC, col1_7, col1_4, col1_14 DESC, col1_8 DESC, col1_1), INDEX (col1_7 ASC, col1_0 DESC, col1_8 ASC, col1_6 ASC, col1_14 ASC, col1_10 DESC, col1_15 ASC, col1_5 ASC, col1_11 DESC, col1_1 ASC, col1_2) WHERE (((((((table1.col1_12 = (-1.234E+401):::DECIMAL) AND (table1.col1_1 <= '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table1.col1_15 = 1.401298464324817e-45:::FLOAT8)) OR table1.col1_3) OR table1.col1_10) AND (table1.col1_14 = e'\'':::STRING)) OR (table1.col1_5 > '-infinity':::DATE)) AND (table1.col1_13 < 1.401298464324817e-45:::FLOAT8));
        test_1        |             CREATE TABLE table2 (col1_15 FLOAT4 NOT NULL, col1_8 REGCLASS NOT NULL, col1_13 FLOAT4 NOT NULL, col1_14 STRING NOT NULL, col1_3 BOOL NOT NULL, col1_1 TIMESTAMPTZ NOT NULL, col1_5 DATE NOT NULL, col1_6 "char" NOT NULL, col1_16 BIT(6) NOT NULL, col1_0 BOX2D NOT NULL, col1_4 REGNAMESPACE NOT NULL, col1_7 BIT(17) NOT NULL, col2_12 TIME, col2_13 BOOL NOT NULL, col2_14 REGCLASS, col2_15 STRING NOT NULL, col2_16 NAME NOT NULL, col2_17 BOOL, col2_18 BYTES NOT NULL, col2_19 REGCLASS NULL, col2_20 TIMESTAMPTZ, col2_21 NAME NOT NULL, col2_22 INT4 NOT NULL, col2_23 REGTYPE NULL, col2_24 REGPROCEDURE NOT NULL, col2_25 BOX2D, col2_26 BOOL, col2_27 "char" NOT NULL, col2_28 BYTES NULL, col2_29 NAME, col2_30 BYTES NOT NULL, PRIMARY KEY (col1_15 ASC, col1_8 ASC, col1_13 ASC, col1_14 ASC, col1_3 ASC, col1_1, col1_5 DESC, col1_6, col1_16 DESC, col1_0 DESC, col1_4, col1_7, col2_17 ASC, col2_15 ASC, col2_18 DESC, col2_20 ASC, col2_24 DESC, col2_25 ASC, col2_12 ASC, col2_22, col2_30, col2_23 ASC, col2_13, col2_16 DESC, col2_27, col2_14 ASC, col2_29 ASC, col2_28 DESC, col2_26 ASC), INDEX (col1_7, col2_21 DESC, col1_0 DESC, col2_25 ASC, col2_26 DESC, col2_24 ASC, col1_4 ASC, col1_3 DESC, col2_12, col1_14, col1_5 ASC, col2_14 DESC, col2_20, col2_13, col2_28 DESC, col1_16 ASC, col2_27, col1_15 ASC, col2_16 ASC, col2_23, col2_19 ASC, col2_17 ASC, col1_13 ASC, col2_15 ASC, col2_22 DESC, col2_29), INDEX (col2_25, col2_16 ASC, col2_23 ASC, col2_13 ASC, col2_15 ASC, col2_29 ASC, col1_6, col2_27 DESC, col1_5 ASC, col2_19, col1_7 DESC, col1_1 DESC, col1_8 ASC, col2_12 DESC, col2_22 DESC, col2_26, col2_30 DESC, col2_28 DESC, col2_14 ASC, col2_24, col1_15 ASC, col1_13, col1_3 ASC, col1_14 ASC, col1_4 DESC, col1_16 ASC, col2_21 DESC), UNIQUE (col2_21 ASC, col2_12 DESC, col1_6, col1_5 ASC, col1_7 DESC, col2_13, col2_18 DESC, col1_8, col2_27 ASC, col2_16, col1_3 ASC, col2_23 ASC, col2_26, col1_0 ASC, col2_22, col1_14, col2_17 ASC, col1_13 ASC, col2_14, col1_1, col2_15, col1_16, col2_29 ASC, col1_4 DESC, col1_15, col2_24 ASC, col2_19 DESC, col2_20 DESC, col2_28, col2_30 DESC, col2_25 ASC)) INTERLEAVE IN PARENT table1 (col1_15, col1_8, col1_13, col1_14, col1_3, col1_1, col1_5, col1_6, col1_16, col1_0, col1_4, col1_7);
        test_1        |             CREATE TABLE table3 (col3_0 INT8 NULL, col3_1 CHAR NULL, col3_2 REGCLASS NOT NULL, UNIQUE (col3_2 ASC, col3_0) WHERE (table3.col3_1 >= '"':::STRING) OR (table3.col3_0 <= 0:::INT8), UNIQUE (col3_2) STORING (col3_0) WHERE (table3.col3_0 < 1:::INT8) AND (table3.col3_1 > e'\'':::STRING), UNIQUE (col3_2 ASC) WHERE (table3.col3_0 < 2147483647:::INT8) OR (table3.col3_1 != 'X':::STRING), INDEX (col3_0, col3_1 ASC, col3_2 DESC) WHERE table3.col3_0 = 2147483647:::INT8, INDEX (col3_2, col3_1 ASC, col3_0), FAMILY (col3_1), FAMILY (col3_0), FAMILY (col3_2));
        test_1        |             CREATE TABLE table4 (col1_15 FLOAT4 NOT NULL, col1_8 REGCLASS NOT NULL, col1_13 FLOAT4 NOT NULL, col1_14 STRING NOT NULL, col1_3 BOOL NOT NULL, col1_1 TIMESTAMPTZ NOT NULL, col1_5 DATE NOT NULL, col1_6 "char" NOT NULL, col1_16 BIT(6) NOT NULL, col1_0 BOX2D NOT NULL, col1_4 REGNAMESPACE NOT NULL, col1_7 BIT(17) NOT NULL, col4_12 INT2, col4_13 DATE, col4_14 BOOL NULL, PRIMARY KEY (col1_15 ASC, col1_8 ASC, col1_13 ASC, col1_14 ASC, col1_3 ASC, col1_1, col1_5 DESC, col1_6, col1_16 DESC, col1_0 DESC, col1_4, col1_7, col4_14 ASC, col4_12 ASC), INDEX (col1_1 ASC, col1_15 ASC), FAMILY (col1_13), FAMILY (col1_7, col1_4), FAMILY (col1_1), FAMILY (col4_12, col4_14), FAMILY (col1_0, col1_6), FAMILY (col4_13, col1_15), FAMILY (col1_5), FAMILY (col1_8), FAMILY (col1_14), FAMILY (col1_16, col1_3)) INTERLEAVE IN PARENT table1 (col1_15, col1_8, col1_13, col1_14, col1_3, col1_1, col1_5, col1_6, col1_16, col1_0, col1_4, col1_7);
        test_1        |             CREATE TABLE table5 (col5_0 STRING NOT NULL, col5_1 INT2 NOT NULL, col5_2 INTERVAL NOT NULL, col5_3 TIMETZ NOT NULL, col5_4 TIMESTAMPTZ NULL, col5_5 INT8 NOT NULL, col5_6 INTERVAL NOT NULL, col5_7 NAME NOT NULL, col5_8 BOX2D NOT NULL, col5_9 INET NOT NULL, col5_10 STRING NOT NULL, col5_11 INTERVAL NOT NULL, col5_12 UUID NOT NULL, PRIMARY KEY (col5_6 ASC, col5_10, col5_8, col5_7 DESC, col5_2, col5_3 ASC, col5_9, col5_1 ASC, col5_5 ASC, col5_0, col5_12 DESC), FAMILY (col5_8), FAMILY (col5_7), FAMILY (col5_10), FAMILY (col5_5, col5_11), FAMILY (col5_1), FAMILY (col5_12, col5_2), FAMILY (col5_3), FAMILY (col5_6), FAMILY (col5_0), FAMILY (col5_4, col5_9));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4421220, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2177283, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1995827, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4089467, "histo_col_type": "", "name": "__auto__", "null_count": 5227323, "row_count": 9000000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 624051, "histo_buckets": [{"distinct_range": 0, "num_eq": 1519277934407476013, "num_range": 0, "upper_bound": "LVD|k6\\u0019"}, {"distinct_range": 0, "num_eq": 400, "num_range": 6914802052751402391, "upper_bound": "Wph\\u0007s\\\\\\u0002\\u001a"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8518896, "histo_buckets": [{"distinct_range": 0, "num_eq": 2135373546690087068, "num_range": 0, "upper_bound": "-0.6018646359443665"}, {"distinct_range": 8057471900646522000, "num_eq": 6000000000, "num_range": 8057471900646521963, "upper_bound": "-0.4760020971298218"}, {"distinct_range": 400, "num_eq": 4874776056933070185, "num_range": 400, "upper_bound": "0.05681411549448967"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 100000000000, "upper_bound": "0.3634089231491089"}, {"distinct_range": 2248486110707539000, "num_eq": 8000, "num_range": 2581518674125490630, "upper_bound": "0.4216398298740387"}, {"distinct_range": 16790112454526602, "num_eq": 730226615948399051, "num_range": 62681090901573098, "upper_bound": "0.491224080324173"}, {"distinct_range": 3675698547245577000, "num_eq": 0, "num_range": 4328000293764960405, "upper_bound": "1.367444634437561"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5910014, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7229706, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 555067, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4948572, "histo_buckets": [{"distinct_range": 0, "num_eq": 2247584304765955632, "num_range": 0, "upper_bound": "00101100010010010"}, {"distinct_range": 40000000, "num_eq": 70000000, "num_range": 40000000, "upper_bound": "01111111111111111"}, {"distinct_range": 4137903363047047000, "num_eq": 40000000000, "num_range": 4137903363047047017, "upper_bound": "10100110001001110"}], "histo_col_type": "BIT(17)", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4764073, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4769307, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "001110"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 4978909433702386016, "upper_bound": "010000"}, {"distinct_range": 0, "num_eq": 2644502364472099572, "num_range": 0, "upper_bound": "010010"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 6000000000, "upper_bound": "101110"}, {"distinct_range": 70000000000, "num_eq": 3313446826697535794, "num_range": 70000000000, "upper_bound": "110101"}], "histo_col_type": "BIT(6)", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6362499, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5170189, "histo_col_type": "", "name": "__auto__", "null_count": 7395459, "row_count": 9000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 278133, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3768894, "histo_buckets": [{"distinct_range": 0, "num_eq": 3227981486869955936, "num_range": 0, "upper_bound": "8"}, {"distinct_range": 43090136437.77325, "num_eq": 2000, "num_range": 80000000000, "upper_bound": ">"}, {"distinct_range": 1311427142459287300, "num_eq": 1000, "num_range": 2819935266531839148, "upper_bound": "?"}, {"distinct_range": 500, "num_eq": 100, "num_range": 500, "upper_bound": "f"}, {"distinct_range": 8527.60646974802, "num_eq": 8513960975551298263, "num_range": 30000, "upper_bound": "m"}, {"distinct_range": 5454406744622175000, "num_eq": 80000000, "num_range": 5898890158260434325, "upper_bound": "p"}, {"distinct_range": 1159758744917491200, "num_eq": 4000000000, "num_range": 1159758744917491097, "upper_bound": "v"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7075826, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 71116394777445312, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col2_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 231835469754587499, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 25.3978501575304, "num_eq": 3605246072760479996, "num_range": 100, "upper_bound": "\\u000cZ05qLm\\n"}, {"distinct_range": 584614.5465348907, "num_eq": 4000000000, "num_range": 5000000, "upper_bound": "\\u0019)X\\u0014."}, {"distinct_range": 6785078881134102000, "num_eq": 20000000000, "num_range": 6785078881134101066, "upper_bound": "3"}, {"distinct_range": 0, "num_eq": 40000000000, "num_range": 2012931588060412290, "upper_bound": "8o\\u0008\\u0016:l[%"}, {"distinct_range": 800, "num_eq": 70, "num_range": 800, "upper_bound": "a\\u0012#"}, {"distinct_range": 6494680844225282000, "num_eq": 5076931221837946088, "num_range": 7371035161585907660, "upper_bound": "nGpN?P"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 910455335739909111, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2490063190275895661, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2186158435613938646, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2544956048552392979, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 412497540585348899, "histo_col_type": "", "name": "__auto__", "null_count": 1642755371354144317, "row_count": 3002383212648800958}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 187392801179285144, "histo_col_type": "", "name": "__auto__", "null_count": 1253051094269325593, "row_count": 3002383212648800958}, {"columns": ["col2_30"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 322522186764788728, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2057641786171481742, "histo_col_type": "BIT(17)", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col2_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1189102824153829493, "histo_col_type": "", "name": "__auto__", "null_count": 700154941060263808, "row_count": 3002383212648800958}, {"columns": ["col2_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38917815044367447, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2674723239683710924, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col2_27"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2569403090261111857, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col2_29"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2492883833736907924, "histo_col_type": "", "name": "__auto__", "null_count": 1292644254524004968, "row_count": 3002383212648800958}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 651494830465092625, "histo_col_type": "", "name": "__auto__", "null_count": 1778498756452672888, "row_count": 3002383212648800958}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 509232875163074179, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1292452863164506660, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col2_26"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 208641828864603857, "histo_col_type": "", "name": "__auto__", "null_count": 2455788391629751991, "row_count": 3002383212648800958}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14153388287797341, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2115578940982385963, "histo_col_type": "", "name": "__auto__", "null_count": 88345854690788219, "row_count": 3002383212648800958}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1134783772533786432, "histo_col_type": "", "name": "__auto__", "null_count": 298773358038993775, "row_count": 3002383212648800958}, {"columns": ["col2_25"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2179296824640862289, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000000, "num_range": 0, "upper_bound": "BOX(-2.534896864662036 -0.8081323708550268,0.9622861677356547 -0.15752630931885792)"}, {"distinct_range": 3209757715728929000, "num_eq": 7430042495895921728, "num_range": 3209757715728928639, "upper_bound": "BOX(-1.0056307629340684 -1.0315332904203052,0.9116891121942066 -0.5906015156136053)"}, {"distinct_range": 0, "num_eq": 4220950631322216045, "num_range": 0, "upper_bound": "BOX(-0.08882399186595918 -0.7704574896468638,0.22471593816625046 1.0765866745969799)"}, {"distinct_range": 41.98354033468898, "num_eq": 9000000, "num_range": 60, "upper_bound": "BOX(0.044566445952931355 0.08662082747578126,0.3686823930570008 0.28412997007364016)"}, {"distinct_range": 249833066.3111215, "num_eq": 5887538171179125245, "num_range": 2000000000, "upper_bound": "BOX(0.4117179331208476 -0.588420003317087,2.038136031693118 0.07267728687279895)"}, {"distinct_range": 5411255473889487000, "num_eq": 900, "num_range": 5411255473889486642, "upper_bound": "BOX(0.9728408729954913 -0.10494836855554274,1.9855759659221488 1.1532141872704822)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 1538153136178702473, "row_count": 3002383212648800958}, {"columns": ["col2_28"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1523257648927035586, "histo_col_type": "", "name": "__auto__", "null_count": 2944945522837497446, "row_count": 3002383212648800958}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2893465247071087577, "histo_buckets": [{"distinct_range": 0, "num_eq": 7178962716911732978, "num_range": 0, "upper_bound": "NaN"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 0, "upper_bound": "-1.807637095451355"}, {"distinct_range": 293.2594584021164, "num_eq": 400000, "num_range": 10000, "upper_bound": "-0.3755286633968353"}, {"distinct_range": 1000000000, "num_eq": 0, "num_range": 1000000000, "upper_bound": "0.0956520214676857"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2713726665047185998, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2380119346334557679, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1681628956131935497, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 578062891182359284, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 372365199344353324, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}, {"columns": ["col2_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2729301751417801829, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3002383212648800958}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 908961272250670064, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "-2204180437432209214"}, {"distinct_range": 602.30732159708, "num_eq": 8000000, "num_range": 2000, "upper_bound": "6675425411502198334"}, {"distinct_range": 15.79309196936479, "num_eq": 2966558790239263878, "num_range": 60, "upper_bound": "6968703304057623858"}, {"distinct_range": 0, "num_eq": 6303332617753569408, "num_range": 0, "upper_bound": "7708729317067613198"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 642234699485305665, "row_count": 2458636077067528098}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 999094426478513100, "histo_col_type": "", "name": "__auto__", "null_count": 2264649350475679859, "row_count": 2458636077067528098}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1093554198720417629, "histo_buckets": [{"distinct_range": 0, "num_eq": 3121525454195156121, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 2029288561616442600, "num_eq": 8361701426754424012, "num_range": 2029288561616442747, "upper_bound": "1195613967"}, {"distinct_range": 5461652669369164000, "num_eq": 60, "num_range": 5461652669369164126, "upper_bound": "1323182742"}, {"distinct_range": 2038645869545020700, "num_eq": 7260680173000125521, "num_range": 7476889313523064896, "upper_bound": "1423730652"}, {"distinct_range": 428694433886037060, "num_eq": 10, "num_range": 4776026222974317631, "upper_bound": "2767216768"}, {"distinct_range": 20, "num_eq": 10000000000, "num_range": 20, "upper_bound": "2885519943"}, {"distinct_range": 2161297547579566600, "num_eq": 7600961811488401986, "num_range": 8027222608011232542, "upper_bound": "3283825136"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 2458636077067528098}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1995-10-30 08:14:16.000958+00:00"}, {"distinct_range": 90000000, "num_eq": 6554260297364088246, "num_range": 90000000, "upper_bound": "2017-02-22 07:36:52.000934+00:00"}, {"distinct_range": 37957530722.91739, "num_eq": 5657517060708863551, "num_range": 40000000000, "upper_bound": "2019-01-27 12:39:04.000111+00:00"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 1000000, "upper_bound": "3000-01-01 00:00:00+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 10, "num_range": 0, "upper_bound": "-1.5131326913833618"}, {"distinct_range": 297988.57800416107, "num_eq": 650172488263542721, "num_range": 600000, "upper_bound": "-0.09706350415945053"}, {"distinct_range": 1000000, "num_eq": 8498622318444199095, "num_range": 1000000, "upper_bound": "0.1813085973262787"}, {"distinct_range": 6559262340373366000, "num_eq": 100000, "num_range": 6559262340373366005, "upper_bound": "0.9488418102264404"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 453642448226854622, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8012141967984667125}, {"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 741853043862352072, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8012141967984667125}, {"columns": ["col5_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2048577277146467711, "histo_col_type": "", "name": "__auto__", "null_count": 1647601401086622158, "row_count": 8012141967984667125}, {"columns": ["col5_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5667205422939569754, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8012141967984667125}, {"columns": ["col5_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6151688454615873002, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8012141967984667125}, {"columns": ["col5_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4702091381507160313, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8012141967984667125}, {"columns": ["col5_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6027447227466035480, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8012141967984667125}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5227644742939992672, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8012141967984667125}, {"columns": ["col5_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 151144796480927873, "histo_buckets": [{"distinct_range": 0, "num_eq": 9007652360619271292, "num_range": 0, "upper_bound": "-59 years -1 mons -506 days -20:36:20.290008"}, {"distinct_range": 900, "num_eq": 8074618935752110039, "num_range": 900, "upper_bound": "-43 years -9 mons -864 days -04:03:36.837081"}, {"distinct_range": 0, "num_eq": 1629812727730746676, "num_range": 500, "upper_bound": "-39 years -2 mons -431 days -19:58:38.238621"}, {"distinct_range": 4573907.011740209, "num_eq": 7508542493745025918, "num_range": 10000000, "upper_bound": "-20 years -3 mons -898 days -01:24:29.977359"}, {"distinct_range": 7463712880569001000, "num_eq": 700, "num_range": 7463712880569000875, "upper_bound": "72 years 3 mons 809 days 22:53:25.987441"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 8012141967984667125}, {"columns": ["col5_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5550219483324370093, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8012141967984667125}, {"columns": ["col5_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4675051077102051465, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8012141967984667125}, {"columns": ["col5_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3775712566377621709, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8012141967984667125}, {"columns": ["col5_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3755570135388462336, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8012141967984667125}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('hrglht', 'qosm', 'om', 'lzzez', 'wslklb', 'hjl');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('pt', 'a', 'cj');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('qzz', 'zhfnvl', 'pewt');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        | FAIL
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down....2021-02-03 06:11:31.727 UTC [48] LOG:  received fast shutdown request
        postgres_1    | 2021-02-03 06:11:31.729 UTC [48] LOG:  aborting any active transactions
        postgres_1    | 2021-02-03 06:11:31.731 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2021-02-03 06:11:31.737 UTC [50] LOG:  shutting down
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2642938&tab=buildLog) on [master@100c09f4f6eb3f5b18a67ec4bbfdfe989e0d6ce2](https://github.com/cockroachdb/cockroach/commits/100c09f4f6eb3f5b18a67ec4bbfdfe989e0d6ce2):

```
        test_1        |             CREATE TABLE table2 (col1_13 FLOAT8 NOT NULL, col1_17 FLOAT4 NOT NULL, col1_8 STRING NOT NULL, col1_18 REGCLASS NOT NULL, col1_16 TIME NOT NULL, col1_2 REGPROC NOT NULL, col1_7 "char" NOT NULL, col2_7 CHAR NOT NULL, col2_8 INTERVAL NULL, col2_9 FLOAT8 NOT NULL, col2_10 TIMESTAMPTZ, col2_11 INTERVAL NOT NULL, col2_12 DECIMAL NOT NULL, col2_13 REGCLASS, col2_14 TIME, col2_15 INT2 NOT NULL, col2_16 BOOL NOT NULL, col2_17 REGPROC NULL, col2_18 REGCLASS NOT NULL, PRIMARY KEY (col1_13, col1_17, col1_8, col1_18, col1_16, col1_2, col1_7, col2_12)) INTERLEAVE IN PARENT table1 (col1_13, col1_17, col1_8, col1_18, col1_16, col1_2, col1_7);
        test_1        |             CREATE INDEX ON table2 (col1_17, col2_14 ASC, col1_2, col2_9 ASC, col1_7, col2_10 ASC, col2_13 DESC, col2_15 ASC, col2_7 ASC, col2_18 ASC, col2_17 ASC, col1_16, col2_8 ASC, col2_11 DESC, col1_18 ASC, col2_12, col1_13, col1_8, col2_16 DESC);
        test_1        |             CREATE INDEX ON table2 (col2_12, col2_8 ASC, col1_7, col1_16 DESC);
        test_1        |             CREATE INDEX ON table2 (col2_9 ASC, col1_17 DESC, col2_10 DESC, col2_11, col2_12, col1_13 ASC) STORING (col2_13, col2_15, col2_18);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_7, col2_17 ASC, col1_8, col2_9 DESC, col2_13, col2_15 ASC, col2_10 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_11 DESC, col1_17 ASC, col2_12 DESC, col1_18 ASC, col2_13 ASC, col1_2, col2_16, col2_14 DESC, col1_16 DESC, col2_8, col1_13 DESC, col2_9 ASC, col1_7 ASC, col2_10 ASC, col2_15 DESC, col1_8 ASC, col2_18 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_9, col1_7 ASC, col1_18 ASC, col2_18 ASC, col2_10 DESC, col2_7 ASC, col1_13, col1_16 ASC, col2_15 ASC, col1_2 ASC, col2_11, col1_8 DESC, col2_8);
        test_1        |             CREATE TABLE table3 (col3_0 INET NOT NULL, col3_1 BIT(10) NOT NULL);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 DESC) STORING (col3_1);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0, col3_1 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_1 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0, col3_1);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6711, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 202314.36651666023, "num_eq": 4056808411227326779, "num_range": 500000, "upper_bound": "260189578"}, {"distinct_range": 0, "num_eq": 4000000, "num_range": 0, "upper_bound": "532754786"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 9000000, "upper_bound": "1173006487"}, {"distinct_range": 4000000000, "num_eq": 5278653434002741198, "num_range": 4000000000, "upper_bound": "2448995839"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 4825, "row_count": 7000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 61, "histo_buckets": [{"distinct_range": 0, "num_eq": 947810232286893101, "num_range": 0, "upper_bound": "-18512"}, {"distinct_range": 2936273181031287300, "num_eq": 60000000, "num_range": 2936273181031287502, "upper_bound": "11198"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 1583, "row_count": 7000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4066, "histo_col_type": "", "name": "__auto__", "null_count": 5732, "row_count": 7000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3227, "histo_col_type": "", "name": "__auto__", "null_count": 6761, "row_count": 7000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6990, "histo_col_type": "", "name": "__auto__", "null_count": 5010, "row_count": 7000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4567, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4110, "histo_buckets": [{"distinct_range": 0, "num_eq": 5011943444633366447, "num_range": 0, "upper_bound": "01ff0775-4a2a-43a6-b87a-9b4c2011d8e8"}, {"distinct_range": 0, "num_eq": 6309089843782860467, "num_range": 20, "upper_bound": "24598af1-93a6-4605-b883-2042fc90245a"}, {"distinct_range": 0, "num_eq": 803790473583038637, "num_range": 0, "upper_bound": "33d375bc-f9d2-4bae-8a76-651049fa2318"}, {"distinct_range": 10000000000, "num_eq": 90000000, "num_range": 10000000000, "upper_bound": "61e469b6-058e-4e2c-abb9-a311b56bf228"}, {"distinct_range": 0, "num_eq": 8961612413151086885, "num_range": 400, "upper_bound": "658cf903-584a-4550-99f2-47d05a254231"}, {"distinct_range": 8671835252954615000, "num_eq": 5204072714132446476, "num_range": 8671835252954614998, "upper_bound": "6b10de55-72da-4943-ba53-e0a0847f5479"}, {"distinct_range": 4530337107842328600, "num_eq": 468069085355253090, "num_range": 4530337107842328416, "upper_bound": "7f7437f9-23e4-4c70-8ed4-54980e19d5af"}, {"distinct_range": 0, "num_eq": 2821187297598372568, "num_range": 500000, "upper_bound": "9d11886e-14e7-4e26-8ac1-29c850a68061"}, {"distinct_range": 9037307217114700000, "num_eq": 7797656371702788986, "num_range": 9174771149548518402, "upper_bound": "f71e3bc7-fd26-4da5-81fa-a862d5c0020c"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 1314, "row_count": 7000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2447, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2865, "histo_buckets": [{"distinct_range": 0, "num_eq": 2932226677210490323, "num_range": 0, "upper_bound": "-0.87062668800354"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 7000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4813, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000, "num_range": 0, "upper_bound": "BOX(-1.7620650160106641 -1.0514542394320046,0.3659031148867591 0.22328965420215452)"}, {"distinct_range": 941062.4924613684, "num_eq": 500000000, "num_range": 1000000, "upper_bound": "BOX(-1.3435998637613569 0.15115175130518513,0.7279537395754048 0.24143749052538813)"}, {"distinct_range": 935523090502551600, "num_eq": 400, "num_range": 935523090502551567, "upper_bound": "BOX(-0.9685842430996414 0.2281088749459727,1.547513553890365 0.32447946019133933)"}, {"distinct_range": 0, "num_eq": 300000000, "num_range": 0, "upper_bound": "BOX(-0.5609646177321297 0.09402650795165435,-0.5587030237012294 0.17898656365247484)"}, {"distinct_range": 844958181405609300, "num_eq": 0, "num_range": 2660207664939094268, "upper_bound": "BOX(-0.16187378141159403 -0.8209477000351686,0.9374343462410367 -0.4829659888402993)"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 7000000, "upper_bound": "BOX(0.05022977892192415 0.4653519804156838,0.09428899990986568 1.9530348936604867)"}, {"distinct_range": 0, "num_eq": 20000000000, "num_range": 20000, "upper_bound": "BOX(0.7906143196054566 -0.941706802548639,0.9117948026177295 -0.08481790085713925)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 7000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5237, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3680, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3398, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000, "num_range": 0, "upper_bound": "-0.48952052389128403"}, {"distinct_range": 0, "num_eq": 8666913637538051924, "num_range": 100, "upper_bound": "0.5089638827658299"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 7000}, {"columns": ["col1_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4784, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5884, "histo_col_type": "", "name": "__auto__", "null_count": 4217, "row_count": 7000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5435, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 82252453.33986959, "num_eq": 2901183319026274361, "num_range": 100000000, "upper_bound": "1310390210"}, {"distinct_range": 3430688850303366700, "num_eq": 6243957892456218760, "num_range": 3430688850303366771, "upper_bound": "2653707994"}, {"distinct_range": 0, "num_eq": 30, "num_range": 6677144007076293630, "upper_bound": "3386481206"}, {"distinct_range": 461787409.6275784, "num_eq": 5538192700826865381, "num_range": 500000000, "upper_bound": "3714726816"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 7000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2209, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2441, "histo_col_type": "", "name": "__auto__", "null_count": 6416, "row_count": 7000}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3810, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 8629759865753205177, "num_range": 0, "upper_bound": "-180480.5546326911897"}, {"distinct_range": 5000000, "num_eq": 100000000, "num_range": 5000000, "upper_bound": "1"}, {"distinct_range": 3212453855038770000, "num_eq": 2609481814604087093, "num_range": 8292972403786615329, "upper_bound": "372.505141069385335"}, {"distinct_range": 3396826907465456600, "num_eq": 200000000, "num_range": 3396826907465456584, "upper_bound": "70592.86986446542409"}, {"distinct_range": 0, "num_eq": 0, "num_range": 6000000, "upper_bound": "1.199907631288757074E+21"}, {"distinct_range": 0, "num_eq": 5791527741241827533, "num_range": 10, "upper_bound": "7.62345030772716231E+22"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 300, "num_range": 0, "upper_bound": "-1.5280100704859785"}, {"distinct_range": 0, "num_eq": 30000, "num_range": 1311878330253975774, "upper_bound": "-1.2106579489805873"}, {"distinct_range": 49180.78186000185, "num_eq": 300000, "num_range": 60000, "upper_bound": "-1.0892095390383192"}, {"distinct_range": 800000000, "num_eq": 4899949281619104348, "num_range": 800000000, "upper_bound": "-0.5913559879579366"}, {"distinct_range": 24262968.198152125, "num_eq": 0, "num_range": 800000000, "upper_bound": "-0.586625433807045"}, {"distinct_range": 0, "num_eq": 3709089977098808540, "num_range": 90000, "upper_bound": "2.081143495536817"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000, "num_range": 0, "upper_bound": "-1.3542861938476562"}, {"distinct_range": 50, "num_eq": 30000000000, "num_range": 50, "upper_bound": "-0.3067168593406677"}, {"distinct_range": 5000000000, "num_eq": 1011790446675574954, "num_range": 5000000000, "upper_bound": "1.5456395149230957"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 284280080, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000000, "num_range": 0, "upper_bound": "205.5.243.155/21"}, {"distinct_range": 30000, "num_eq": 1000000, "num_range": 30000, "upper_bound": "199.194.125.85/28"}, {"distinct_range": 2251559948788546000, "num_eq": 200000, "num_range": 3544774794208228493, "upper_bound": "4282:d493:976f:51c1:c313:e6af:f64f:4ccb/38"}, {"distinct_range": 1381775.896226545, "num_eq": 1040218958401582498, "num_range": 7000000, "upper_bound": "abcf:84c2:570f:97d1:dc8b:bdcd:cec1:2f0/79"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 223120222, "histo_col_type": "BIT(10)", "name": "__auto__", "null_count": 0, "row_count": 300000000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('x', 'guklah', 'pbez');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('qd', 'bhciub', 'u', 'mjw', 'jqviqd', 'op');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('twaws', 'ixvdl', 'cac', 'yqwpq');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        |     --- FAIL: TestCompare/mutators (0.20s)
        test_1        |         compare_test.go:129: 
        test_1        |             		SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;
        test_1        |             		SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;
        test_1        |             	CREATE TABLE table1 (col1_0 REGCLASS NOT NULL, col1_1 FLOAT8 NOT NULL, col1_2 DATE, PRIMARY KEY (col1_1 DESC, col1_0 ASC), INDEX (col1_0, col1_2, col1_1 DESC) WHERE (table1.col1_2 != '1970-01-01':::DATE) AND (table1.col1_1 != 'NaN':::FLOAT8), UNIQUE (col1_2 DESC, col1_1 ASC) WHERE (table1.col1_1 < 0.0:::FLOAT8) OR (table1.col1_2 <= '5874897-12-31':::DATE), UNIQUE (col1_1 DESC, col1_2 DESC, col1_0 DESC), UNIQUE (col1_1 ASC, col1_2 DESC) WHERE table1.col1_1 != 'NaN':::FLOAT8, UNIQUE (col1_1 DESC), UNIQUE (col1_0, col1_1 DESC) STORING (col1_2), UNIQUE (col1_0 ASC, col1_2 ASC, col1_1 ASC), UNIQUE (col1_0 ASC, col1_1 DESC) WHERE table1.col1_2 < 'infinity':::DATE, FAMILY (col1_1), FAMILY (col1_2, col1_0));
        test_1        |             CREATE TABLE table2 (col2_0 BOX2D NOT NULL, col2_1 BIT(20) NULL, col2_2 TIMESTAMP NOT NULL, col2_3 TIMESTAMPTZ NULL, col2_4 STRING NOT NULL, PRIMARY KEY (col2_2, col2_4), UNIQUE (col2_1 ASC, col2_4, col2_2 DESC, col2_3 ASC, col2_0), INDEX (col2_4 ASC, col2_0 DESC), INDEX (col2_1 ASC, col2_3 ASC, col2_2 DESC, col2_4, col2_0 DESC) WHERE (table2.col2_2 >= '-4713-11-24 00:00:00':::TIMESTAMP) AND (table2.col2_3 <= '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ), FAMILY (col2_4), FAMILY (col2_1, col2_0), FAMILY (col2_3), FAMILY (col2_2));
        test_1        |             CREATE TABLE table3 (col2_2 TIMESTAMP NOT NULL, col2_4 STRING NOT NULL, col3_2 INT4, col3_3 INT4, col3_4 INT2 NOT NULL, col3_5 OID, PRIMARY KEY (col2_2, col2_4, col3_5 ASC, col3_2), UNIQUE (col3_4 ASC, col3_5 ASC, col2_4 ASC, col3_3, col3_2 DESC, col2_2) WHERE (table3.col3_3 = (-1):::INT8) OR (table3.col3_2 != 2147483647:::INT8), INDEX (col2_2 ASC, col3_2, col2_4 DESC, col3_3 DESC) WHERE ((table3.col3_4 != (-128):::INT8) AND (table3.col2_4 <= 'X':::STRING)) AND (table3.col2_2 > '-2000-01-01 00:00:00':::TIMESTAMP), UNIQUE (col3_2, col3_5, col2_4, col2_2) WHERE ((((table3.col3_3 = 2147483647:::INT8) OR (table3.col3_4 > 127:::INT8)) OR (table3.col2_2 != '294276-12-31 23:59:59.999999':::TIMESTAMP)) AND (table3.col2_4 < 'X':::STRING)) OR (table3.col3_2 < (-2147483648):::INT8), INDEX (col3_5 DESC, col2_2 ASC, col3_3 ASC, col2_4, col3_4 DESC, col3_2 ASC), UNIQUE (col3_4, col3_3, col3_2 ASC, col3_5 ASC), UNIQUE (col3_5 ASC, col3_4 DESC), UNIQUE (col2_2 ASC) WHERE ((((table3.col2_2 > '294276-12-31 23:59:59.999999':::TIMESTAMP) OR (table3.col3_2 < 2147483647:::INT8)) OR (table3.col2_4 > 'X':::STRING)) AND (table3.col3_3 > (-1):::INT8)) OR (table3.col3_4 <= 1:::INT8), INDEX (col3_2) WHERE ((((table3.col2_4 <= e'\'':::STRING) OR (table3.col3_3 <= (-128):::INT8)) AND (table3.col3_2 <= (-1):::INT8)) AND (table3.col3_4 > 1:::INT8)) AND (table3.col2_2 >= '294276-12-31 23:59:59.999999':::TIMESTAMP)) INTERLEAVE IN PARENT table2 (col2_2, col2_4);
        test_1        |             CREATE TABLE table4 (col4_0 VARCHAR NOT NULL, col4_1 FLOAT8 NOT NULL, col4_2 REGTYPE NOT NULL, col4_3 REGNAMESPACE NOT NULL, col4_4 REGPROCEDURE NOT NULL, col4_5 CHAR, col4_6 STRING NOT NULL, col4_7 FLOAT4 NOT NULL, col4_8 TIME NOT NULL, col4_9 TIME NOT NULL, col4_10 STRING NOT NULL, PRIMARY KEY (col4_2 ASC, col4_10 ASC, col4_7 DESC, col4_8 DESC, col4_0 DESC, col4_9 DESC, col4_4 ASC), UNIQUE (col4_0) WHERE (((((((table4.col4_6 >= e'\U00002603':::STRING) AND (table4.col4_7 = '+Inf':::FLOAT8)) AND (table4.col4_1 = 1.7976931348623157e+308:::FLOAT8)) OR (table4.col4_5 >= '':::STRING)) AND (table4.col4_10 != e'\U00002603':::STRING)) OR (table4.col4_0 <= 'X':::STRING)) OR (table4.col4_9 < '24:00:00':::TIME)) OR (table4.col4_8 > '00:00:00':::TIME), FAMILY (col4_9), FAMILY (col4_2), FAMILY (col4_5), FAMILY (col4_1), FAMILY (col4_4), FAMILY (col4_3, col4_10, col4_6, col4_0), FAMILY (col4_8, col4_7));
        test_1        |             CREATE TABLE table5 (col4_2 REGTYPE NOT NULL, col4_10 STRING NOT NULL, col4_7 FLOAT4 NOT NULL, col4_8 TIME NOT NULL, col4_0 VARCHAR NOT NULL, col4_9 TIME NOT NULL, col4_4 REGPROCEDURE NOT NULL, col5_7 TIMETZ, col5_8 BYTES NULL, col5_9 INTERVAL NOT NULL, PRIMARY KEY (col4_2 ASC, col4_10 ASC, col4_7 DESC, col4_8 DESC, col4_0 DESC, col4_9 DESC, col4_4 ASC, col5_8 ASC), UNIQUE (col4_0, col5_7, col4_2) STORING (col5_9) WHERE table5.col4_9 >= '24:00:00':::TIME, INDEX (col4_7, col4_8) WHERE ((table5.col5_7 = '00:00:00+15:59:00':::TIMETZ) AND (table5.col4_8 > '24:00:00':::TIME)) OR (table5.col4_10 >= '"':::STRING), UNIQUE (col4_7 DESC, col5_7 DESC, col4_0, col4_2 ASC, col5_8 ASC, col4_10 ASC, col5_9 ASC, col4_4 DESC) WHERE ((((((table5.col4_9 >= '24:00:00':::TIME) AND (table5.col4_10 > e'\'':::STRING)) AND (table5.col4_7 < 'NaN':::FLOAT8)) OR (table5.col5_7 < '00:00:00+15:59:00':::TIMETZ)) AND (table5.col4_8 != '00:00:00':::TIME)) OR (table5.col4_0 <= 'X':::STRING)) OR (table5.col5_8 <= '\x22':::BYTES), UNIQUE (col4_4 DESC, col4_2 DESC, col4_0 ASC, col4_7 DESC, col5_8 DESC, col4_9), FAMILY (col4_10), FAMILY (col5_8, col4_7), FAMILY (col4_9, col4_2, col5_9, col4_8), FAMILY (col5_7, col4_4), FAMILY (col4_0)) INTERLEAVE IN PARENT table4 (col4_2, col4_10, col4_7, col4_8, col4_0, col4_9, col4_4);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 331977, "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 435274, "histo_buckets": [{"distinct_range": 0, "num_eq": 4537955954324984440, "num_range": 0, "upper_bound": "-1.4413412031269406"}, {"distinct_range": 0, "num_eq": 7401446920914269955, "num_range": 7231751586782091302, "upper_bound": "-1.071112477157019"}, {"distinct_range": 12633693867.250082, "num_eq": 1837574351692095862, "num_range": 30000000000, "upper_bound": "0.5962885237380313"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12405, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "1971-06-21"}, {"distinct_range": 0, "num_eq": 3607839594522957782, "num_range": 1368374546429163405, "upper_bound": "1976-02-13"}, {"distinct_range": 2024120321135443000, "num_eq": 7198317404946196255, "num_range": 8652259460126452100, "upper_bound": "1976-03-08"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 7887117141515155373, "upper_bound": "1979-01-14"}, {"distinct_range": 2355449247511316000, "num_eq": 0, "num_range": 2355449247511315901, "upper_bound": "1980-10-17"}, {"distinct_range": 3645896630944154600, "num_eq": 7112296363577132467, "num_range": 3645896630944154470, "upper_bound": "1985-06-17"}, {"distinct_range": 0, "num_eq": 9000000, "num_range": 0, "upper_bound": "1996-06-15"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 430430, "row_count": 700000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1838, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000000, "num_range": 0, "upper_bound": "1991-04-14 02:23:49.000913"}, {"distinct_range": 1243559903658697200, "num_eq": 143795378627485925, "num_range": 1376499370035661418, "upper_bound": "1995-08-23 05:48:39.000782"}, {"distinct_range": 0, "num_eq": 5624767984126791355, "num_range": 9171304595880818757, "upper_bound": "2028-01-30 12:56:34.000484"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1658, "histo_col_type": "", "name": "__auto__", "null_count": 506, "row_count": 6000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2956, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "!\\u0007"}, {"distinct_range": 0, "num_eq": 7177497022051162751, "num_range": 2000000000, "upper_bound": "\'"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 3780097148611687480, "upper_bound": "7\\u001cp"}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 50000000, "upper_bound": "9\\u0013#t\\u0014"}, {"distinct_range": 7883718985.48364, "num_eq": 8173157008981971355, "num_range": 10000000000, "upper_bound": "QX6\\u0002W\\u0015m?"}, {"distinct_range": 0, "num_eq": 3822872222570104483, "num_range": 10000, "upper_bound": "\\\\T5n\\u001c}uv"}, {"distinct_range": 0, "num_eq": 837312384002113728, "num_range": 1619826614660095597, "upper_bound": "i\\u000e"}, {"distinct_range": 8000000000, "num_eq": 5000000, "num_range": 8000000000, "upper_bound": "mrq&\\u0011Hgh>"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2222, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4169, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "00000000000000000000"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "01000010110010110000"}, {"distinct_range": 2932903275758189000, "num_eq": 600000, "num_range": 2932903275758189162, "upper_bound": "10101100110000100100"}, {"distinct_range": 5687799637080621000, "num_eq": 5353670765176017833, "num_range": 8802903096602625764, "upper_bound": "11000010110011000101"}], "histo_col_type": "BIT(20)", "name": "__auto__", "null_count": 1631, "row_count": 6000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2110939, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "-1632045046"}, {"distinct_range": 0, "num_eq": 4000000000, "num_range": 6805244133180601645, "upper_bound": "1401965217"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 3359625, "row_count": 4000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1400871, "histo_col_type": "", "name": "__auto__", "null_count": 368228, "row_count": 4000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1016861, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000, "num_range": 0, "upper_bound": "-1"}, {"distinct_range": 0, "num_eq": 6000000, "num_range": 7000, "upper_bound": "0"}, {"distinct_range": 1366848327.4343355, "num_eq": 0, "num_range": 3000000000, "upper_bound": "13155"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2692404, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000, "num_range": 0, "upper_bound": "173124832"}, {"distinct_range": 0, "num_eq": 354533496298338747, "num_range": 1020464920989371872, "upper_bound": "1024490668"}, {"distinct_range": 475.49070281004845, "num_eq": 700000, "num_range": 1000, "upper_bound": "1648981236"}, {"distinct_range": 600000000, "num_eq": 400000000, "num_range": 600000000, "upper_bound": "3322165604"}, {"distinct_range": 617941485403915900, "num_eq": 1871771269427391390, "num_range": 4686512025092376270, "upper_bound": "3403250053"}, {"distinct_range": 200000000, "num_eq": 60000, "num_range": 200000000, "upper_bound": "4017171402"}, {"distinct_range": 0, "num_eq": 200, "num_range": 100, "upper_bound": "4126982911"}, {"distinct_range": 8217785386540178000, "num_eq": 8873851196064354858, "num_range": 8963288179775904882, "upper_bound": "4153122769"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 637531, "row_count": 4000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2885243, "histo_buckets": [{"distinct_range": 0, "num_eq": 700, "num_range": 0, "upper_bound": "1974-01-14 22:21:17.000358"}, {"distinct_range": 6388040649585116000, "num_eq": 6194577118811031440, "num_range": 6388040649585115779, "upper_bound": "1979-11-26 08:15:08.000371"}, {"distinct_range": 3472325111048082000, "num_eq": 500, "num_range": 5155455486717835971, "upper_bound": "1988-01-05 15:22:49.000881"}, {"distinct_range": 400000000, "num_eq": 600000000, "num_range": 400000000, "upper_bound": "2018-07-25 13:39:30.000013"}, {"distinct_range": 5851115728.687025, "num_eq": 2341746910488965668, "num_range": 10000000000, "upper_bound": "2021-01-28 12:21:18.000562"}, {"distinct_range": 3512657807952036400, "num_eq": 2841332974745336372, "num_range": 3512657807952036375, "upper_bound": "2027-09-17 03:50:57.000187"}, {"distinct_range": 90000, "num_eq": 6000000, "num_range": 90000, "upper_bound": "294276-12-31 23:59:59.999999"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1376571, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 324760020574045151, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1297873891721643980}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 183371226073857956, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1297873891721643980}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 84412023519458316, "histo_buckets": [{"distinct_range": 0, "num_eq": 500, "num_range": 0, "upper_bound": "2774018917"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 1297873891721643980}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1091773247916781323, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1297873891721643980}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 358087837049710016, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1297873891721643980}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1124851698916693523, "histo_col_type": "", "name": "__auto__", "null_count": 1290029545343439694, "row_count": 1297873891721643980}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38420852487791933, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1297873891721643980}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 421582080311089272, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1297873891721643980}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 848278848609585183, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": " vG\\u0013xO"}, {"distinct_range": 0, "num_eq": 7708154289037892377, "num_range": 80000000000, "upper_bound": "0n{JdH\'"}, {"distinct_range": 0, "num_eq": 7235462198203891844, "num_range": 8408797474180982921, "upper_bound": "Sf)\\u0003\\\\%;- "}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 1669889650830555289, "upper_bound": "\\\\Ao]#A\\u000c\x7f("}, {"distinct_range": 0, "num_eq": 1414061991817874213, "num_range": 6000000, "upper_bound": "\U00002603"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 1297873891721643980}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 410419374565935234, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1297873891721643980}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1059446325664278622, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1297873891721643980}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS e'[{"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "1153885343"}, {"distinct_range": 2456.1598954967994, "num_eq": 800000000, "num_range": 9000, "upper_bound": "1535809731"}, {"distinct_range": 0, "num_eq": 500, "num_range": 0, "upper_bound": "1717881327"}, {"distinct_range": 4003020.2765410263, "num_eq": 5692204321185057689, "num_range": 6000000, "upper_bound": "1731104497"}, {"distinct_range": 6627021622506082000, "num_eq": 3073851808798030422, "num_range": 6627021622506081813, "upper_bound": "1842743439"}, {"distinct_range": 29000.117648286607, "num_eq": 800, "num_range": 100000, "upper_bound": "2044214654"}, {"distinct_range": 490992985.1561233, "num_eq": 3802305308746053681, "num_range": 700000000, "upper_bound": "2729966250"}, {"distinct_range": 0, "num_eq": 5148564042293420145, "num_range": 200000000, "upper_bound": "2826905195"}, {"distinct_range": 50668152.43996814, "num_eq": 2176374697487227165, "num_range": 70000000, "upper_bound": "2841592928"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 102909310868714923, "num_range": 0, "upper_bound": "$LHJ%zt"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "\'"}, {"distinct_range": 4987915849511460000, "num_eq": 2000000, "num_range": 6765122253521813809, "upper_bound": "4?\\u0008m1q:"}, {"distinct_range": 6.971868352482381, "num_eq": 10000000, "num_range": 20, "upper_bound": "5\\u0011\\u0005`e6"}, {"distinct_range": 90000000, "num_eq": 7971608696550953271, "num_range": 90000000, "upper_bound": "B+*0Kvz"}, {"distinct_range": 0, "num_eq": 0, "num_range": 1151046926747777852, "upper_bound": "~\\u0002"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col5_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 6115880090010826519, "num_range": 0, "upper_bound": "1600247429"}, {"distinct_range": 8819206618076823000, "num_eq": 100000, "num_range": 8819206618076823026, "upper_bound": "2658564858"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 1000, "upper_bound": "2693247184"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 1356395118975148660, "num_range": 0, "upper_bound": "-0.5640982389450073"}, {"distinct_range": 0, "num_eq": 90000000000, "num_range": 545583714395926569, "upper_bound": "-0.0863722711801529"}, {"distinct_range": 7439909923386613000, "num_eq": 1574549847952237585, "num_range": 7439909923386612951, "upper_bound": "0.39509275555610657"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 6156471194895457369, "upper_bound": "0.5852057337760925"}, {"distinct_range": 3443467751009678300, "num_eq": 0, "num_range": 3443467751009678545, "upper_bound": "1.056668758392334"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('fruk', 'zvhw', 'vb', 'ccuq', 'pip', 'm');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('fp', 'se', 'tjsui', 'ew');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2646694&tab=buildLog) on [master@c584f62067a45aa540c26fc9081a83e460bfe37a](https://github.com/cockroachdb/cockroach/commits/c584f62067a45aa540c26fc9081a83e460bfe37a):

```
        test_1        |             CREATE INDEX ON table2 (col1_1 ASC, col1_4, col2_3 DESC, col1_5 ASC);
        test_1        |             CREATE INDEX ON table2 (col1_4 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_5 DESC, col1_4, col2_3 ASC);
        test_1        |             CREATE INDEX ON table2 (col1_1, col2_3 ASC, col1_5);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_1, col1_4 ASC, col2_3);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_1 DESC);
        test_1        |             CREATE TABLE table3 (col3_0 BOX2D NOT NULL, col3_1 TIMETZ NOT NULL, col3_2 REGTYPE NOT NULL, col3_3 BOX2D NOT NULL, col3_4 CHAR NOT NULL, col3_5 REGPROCEDURE NOT NULL, col3_6 REGPROC NOT NULL, col3_7 INT2 NOT NULL, col3_8 UUID NOT NULL, col3_9 REGTYPE NOT NULL, PRIMARY KEY (col3_0, col3_2, col3_6, col3_7, col3_5, col3_8, col3_1, col3_4, col3_3, col3_9), FAMILY (col3_3), FAMILY (col3_4), FAMILY (col3_8), FAMILY (col3_7), FAMILY (col3_5), FAMILY (col3_0), FAMILY (col3_1, col3_2, col3_6, col3_9));
        test_1        |             CREATE INDEX ON table3 (col3_2, col3_1 DESC, col3_8 ASC, col3_9 DESC, col3_5 DESC, col3_6, col3_3, col3_0 DESC);
        test_1        |             CREATE INDEX ON table3 (col3_7, col3_5, col3_6, col3_3 ASC, col3_8);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_8 DESC, col3_5, col3_9 DESC, col3_3, col3_4, col3_6 DESC, col3_7 DESC, col3_0 DESC, col3_2 DESC, col3_1 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_1 DESC, col3_2 ASC, col3_8, col3_0 ASC, col3_3 ASC, col3_5 ASC);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3730110394873511268, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000, "num_range": 0, "upper_bound": "\\\\x85"}, {"distinct_range": 0, "num_eq": 4126269142763096341, "num_range": 7807089554476849198, "upper_bound": "\\\\x8792ef"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 215666590619684381, "row_count": 4130781723647565935}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 843186852339489908, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "82657865"}, {"distinct_range": 0, "num_eq": 30000000, "num_range": 20000000, "upper_bound": "389056860"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 90000000, "upper_bound": "744747726"}, {"distinct_range": 0, "num_eq": 2000000, "num_range": 500, "upper_bound": "2056951382"}, {"distinct_range": 0, "num_eq": 6437355435323075953, "num_range": 2979334854778349452, "upper_bound": "2201249733"}, {"distinct_range": 4000, "num_eq": 4000, "num_range": 4000, "upper_bound": "2554999665"}, {"distinct_range": 3772879014213444600, "num_eq": 4820705813336212724, "num_range": 3772879014213444444, "upper_bound": "2949395799"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "3406858280"}, {"distinct_range": 8567584328570662000, "num_eq": 10, "num_range": 8567584328570662144, "upper_bound": "3753337936"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 4130781723647565935}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2463171178185231701, "histo_buckets": [{"distinct_range": 0, "num_eq": 7323968502151020016, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 20000000, "num_range": 0, "upper_bound": "1024709850"}, {"distinct_range": 4394214046373999600, "num_eq": 100000, "num_range": 6484507403801921385, "upper_bound": "1028328511"}, {"distinct_range": 1485717102466005500, "num_eq": 8150774289057307521, "num_range": 3604914256880146910, "upper_bound": "1709539108"}, {"distinct_range": 6000, "num_eq": 1067710978208440472, "num_range": 6000, "upper_bound": "2311841413"}, {"distinct_range": 1166352898306084400, "num_eq": 400000000, "num_range": 1166352898306084360, "upper_bound": "3083059891"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 1263533081004155992, "row_count": 4130781723647565935}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2452389292187108070, "histo_buckets": [{"distinct_range": 0, "num_eq": 60, "num_range": 0, "upper_bound": "-227087685"}, {"distinct_range": 30, "num_eq": 100000, "num_range": 30, "upper_bound": "127"}, {"distinct_range": 1326847277.8636732, "num_eq": 6073282495516523052, "num_range": 2000000000, "upper_bound": "32767"}, {"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "183885633"}, {"distinct_range": 400, "num_eq": 800000000, "num_range": 400, "upper_bound": "393229543"}, {"distinct_range": 4293931346273146400, "num_eq": 90000000, "num_range": 5644102093025631619, "upper_bound": "2095725800"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 4130781723647565935}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3159427613232268823, "histo_buckets": [{"distinct_range": 0, "num_eq": 1786836265189778285, "num_range": 0, "upper_bound": " \\u0012\\rkr?"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 4130781723647565935}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1433622567548828727, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4130781723647565935}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19008756523, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 406500234113197900, "num_eq": 70000000, "num_range": 550178269929488428, "upper_bound": "64124483"}, {"distinct_range": 90000000000, "num_eq": 6432023480550667573, "num_range": 90000000000, "upper_bound": "2001319720"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8953276557, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "3026032044"}, {"distinct_range": 0, "num_eq": 7863365923708114521, "num_range": 9076299169497559954, "upper_bound": "3761048151"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20337755960, "histo_buckets": [{"distinct_range": 0, "num_eq": 7584554146253762722, "num_range": 0, "upper_bound": "\\u000bA\\tFx:"}, {"distinct_range": 7035016992371121000, "num_eq": 30000000, "num_range": 7035016992371121201, "upper_bound": "\\""}, {"distinct_range": 0, "num_eq": 4664318869247508739, "num_range": 2000000000, "upper_bound": "#\\\\\\u0014\\u000c\\r\\u00069&T"}, {"distinct_range": 1980540.1171237596, "num_eq": 4536566917350895209, "num_range": 2000000, "upper_bound": "_7\\u000bT"}, {"distinct_range": 30000000000, "num_eq": 90000, "num_range": 30000000000, "upper_bound": "y"}, {"distinct_range": 1076022430.1310918, "num_eq": 4058291564156084655, "num_range": 8000000000, "upper_bound": "\U00002603"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 26533726386, "row_count": 30000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18235173854, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 800, "num_range": 6678722544966713010, "upper_bound": "\\u0016\\u001f}^"}, {"distinct_range": 8501623331270861000, "num_eq": 4038281053688362023, "num_range": 8501623331270861055, "upper_bound": "\\u0016|"}, {"distinct_range": 1145.824544828848, "num_eq": 30000000, "num_range": 3000, "upper_bound": "\\u0019\\u001c#gxhp("}, {"distinct_range": 0, "num_eq": 70, "num_range": 50000, "upper_bound": "Q\\u0003_\\u0005\\u0003C\\n\'A"}, {"distinct_range": 64.28517905121689, "num_eq": 900, "num_range": 70, "upper_bound": "X_\\t=r<"}, {"distinct_range": 0, "num_eq": 2080283057732965589, "num_range": 5898663640391041721, "upper_bound": "o\\u001f\\u0014\\u0015\\" \\u000c"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 30000000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 778437118695547598, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-11370"}, {"distinct_range": 63394269404488216, "num_eq": 5932682871003749338, "num_range": 63394269404488215, "upper_bound": "-6628"}, {"distinct_range": 171751741482566530, "num_eq": 5584383784979870450, "num_range": 1973392626762915765, "upper_bound": "-1"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 5647674411901358934, "upper_bound": "1"}, {"distinct_range": 534168874.35054004, "num_eq": 90000000, "num_range": 900000000, "upper_bound": "4291"}, {"distinct_range": 900000000, "num_eq": 8630836974473911816, "num_range": 900000000, "upper_bound": "12275"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 867786781621361801}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 243268133035399629, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 867786781621361801}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 258361957912034941, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 867786781621361801}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 561620106054647632, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 584.8152625200346, "num_eq": 3000, "num_range": 1000, "upper_bound": "03:51:45.267995+07:37:00"}, {"distinct_range": 1430933577013006600, "num_eq": 50000, "num_range": 2887809350618124735, "upper_bound": "08:46:01.108011-03:58:00"}, {"distinct_range": 100000000000, "num_eq": 2149511834836944547, "num_range": 100000000000, "upper_bound": "02:10:05.206629-12:28:00"}, {"distinct_range": 1641248747600917000, "num_eq": 70, "num_range": 1641248747600916885, "upper_bound": "22:37:12.053868+02:42:00"}, {"distinct_range": 0, "num_eq": 60000000000, "num_range": 3699603334551172962, "upper_bound": "23:18:43.790095-13:27:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 867786781621361801}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 598161973384864466, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "651019429"}, {"distinct_range": 21786.07154988557, "num_eq": 8000, "num_range": 60000, "upper_bound": "2019578715"}, {"distinct_range": 4201208144050465300, "num_eq": 7049069660235988398, "num_range": 4201208144050465307, "upper_bound": "2459099203"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 0, "upper_bound": "3095368417"}, {"distinct_range": 0, "num_eq": 2351826992980841818, "num_range": 30000000, "upper_bound": "3705497009"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 867786781621361801}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 521821404815865393, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 867786781621361801}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 548669829027160605, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 867786781621361801}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 761561899452786343, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 867786781621361801}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 107716114702145046, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "43b6dde3-9376-4fb3-95ea-747a372466b4"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 867786781621361801}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 712287013787243393, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": "BOX(-1.9277264454435439 -1.5195461311025646,0.6178425365662714 0.7073132473763177)"}, {"distinct_range": 5000000, "num_eq": 7000000, "num_range": 5000000, "upper_bound": "BOX(-1.3897500297886891 -0.744351807008227,-0.07007182743009488 0.48161389151953643)"}, {"distinct_range": 0, "num_eq": 6151685857022073600, "num_range": 500000, "upper_bound": "BOX(-0.4827701131472313 -1.2169791854393441,0.26791534850722876 0.7068118209365963)"}, {"distinct_range": 0, "num_eq": 50000, "num_range": 1000000, "upper_bound": "BOX(0.11559344899333124 0.3455041819829543,1.429745965681189 1.7200546974540945)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 867786781621361801}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('mwuz', 'yizz', 'ip', 'nscog');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('ej', 'ikxxgd', 'eftkwe', 'gmhc', 'jzh');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('wwsyb', 'ee', 'xhjczz');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        |     --- FAIL: TestCompare/mutators (0.23s)
        test_1        |         compare_test.go:129: 
        test_1        |             		SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;
        test_1        |             		SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;
        test_1        |             	CREATE TABLE table1 (col1_0 INT8, col1_1 "char" NOT NULL, col1_2 INT2 NOT NULL, col1_3 FLOAT8 NOT NULL, col1_4 DECIMAL NOT NULL, PRIMARY KEY (col1_4 DESC, col1_1 ASC, col1_2, col1_3 ASC), INDEX (col1_0, col1_4 DESC) WHERE (((table1.col1_4 <= (-1.234E+401):::DECIMAL) OR (table1.col1_0 < 9223372036854775807:::INT8)) AND (table1.col1_3 = 5e-324:::FLOAT8)) OR (table1.col1_1 < '':::STRING), UNIQUE (col1_4 ASC, col1_0 DESC) WHERE table1.col1_1 != e'\'':::STRING);
        test_1        |             CREATE TABLE table2 (col1_4 DECIMAL NOT NULL, col1_1 "char" NOT NULL, col1_2 INT2 NOT NULL, col1_3 FLOAT8 NOT NULL, col2_4 REGTYPE NULL, col2_5 NAME NOT NULL, col2_6 STRING NOT NULL, col2_7 INT2 NOT NULL, col2_8 REGNAMESPACE, col2_9 TIMETZ NOT NULL, col2_10 REGPROCEDURE, col2_11 VARCHAR NOT NULL, col2_12 VARCHAR, col2_13 UUID NULL, col2_14 TIME, col2_15 BIT(39) NULL, col2_16 INT4 NULL, col2_17 INT2, col2_18 TIMETZ NULL, col2_19 REGPROC NOT NULL, col2_20 TIMESTAMP NOT NULL, col2_21 DECIMAL, col2_22 INET NOT NULL, PRIMARY KEY (col1_4 DESC, col1_1 ASC, col1_2, col1_3 ASC, col2_8, col2_13, col2_21 DESC, col2_4 ASC, col2_22 DESC, col2_9, col2_7, col2_18, col2_14, col2_16 ASC, col2_17, col2_5 DESC, col2_10 ASC), UNIQUE (col2_5 ASC, col2_14, col2_8, col2_10, col1_2 DESC, col2_12), UNIQUE (col1_1 ASC, col2_19, col2_12 DESC), INDEX (col2_19 ASC, col2_6, col2_12 DESC, col1_3 DESC, col2_9, col2_18 ASC, col2_4 ASC, col2_13, col2_14 ASC, col1_4 DESC, col2_5, col2_16, col1_2 ASC, col2_20 DESC, col1_1 ASC, col2_15 ASC, col2_8), INDEX (col1_4, col2_20 DESC, col2_16, col2_5 ASC, col2_4 ASC, col2_7 ASC, col2_9 DESC, col2_14)) INTERLEAVE IN PARENT table1 (col1_4, col1_1, col1_2, col1_3);
        test_1        |             CREATE TABLE table3 (col3_0 FLOAT8 NOT NULL, col3_1 FLOAT4 NOT NULL, col3_2 INT2, col3_3 DATE, col3_4 FLOAT4 NOT NULL, col3_5 BOX2D, col3_6 DECIMAL NULL, col3_7 TIMESTAMPTZ NOT NULL, col3_8 INT8 NOT NULL, col3_9 TIMESTAMPTZ NULL, col3_10 UUID NOT NULL, col3_11 REGPROC NULL, col3_12 REGNAMESPACE NOT NULL, col3_13 TIMESTAMPTZ, PRIMARY KEY (col3_10, col3_12 DESC, col3_4 DESC, col3_1), UNIQUE (col3_12, col3_13 DESC, col3_10, col3_7 ASC, col3_3 DESC, col3_2, col3_8 ASC, col3_1 ASC, col3_5, col3_9), INDEX (col3_11, col3_8, col3_12 ASC, col3_5 ASC, col3_4, col3_0 DESC, col3_9, col3_2 ASC, col3_10, col3_6 DESC, col3_7, col3_3 ASC) WHERE (table3.col3_2 <= (-1):::INT8) AND (table3.col3_9 < '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ), INDEX (col3_0, col3_7 DESC, col3_11 DESC) STORING (col3_3, col3_5, col3_6, col3_13) WHERE (table3.col3_0 = '-Inf':::FLOAT8) AND (table3.col3_9 >= '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ), INDEX (col3_5, col3_1, col3_12 ASC, col3_9 ASC, col3_3, col3_13 DESC, col3_8 DESC, col3_11, col3_2 DESC, col3_10, col3_0 DESC, col3_6, col3_7 ASC, col3_4 ASC), INDEX (col3_12, col3_0, col3_4, col3_8 DESC, col3_5 DESC, col3_9 DESC, col3_2 DESC, col3_13 DESC, col3_10, col3_1, col3_7) WHERE (((((((((table3.col3_3 <= '-infinity':::DATE) OR (table3.col3_0 >= '+Inf':::FLOAT8)) AND (table3.col3_8 = 9223372036854775807:::INT8)) OR (table3.col3_4 >= (-1.0):::FLOAT8)) AND (table3.col3_2 = (-32768):::INT8)) OR (table3.col3_6 <= '-Infinity':::DECIMAL)) AND (table3.col3_9 = '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table3.col3_1 = 1.401298464324817e-45:::FLOAT8)) AND (table3.col3_13 < '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table3.col3_7 <= '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ), UNIQUE (col3_4 ASC, col3_9 DESC, col3_11 ASC, col3_7) STORING (col3_5, col3_6, col3_8, col3_13) WHERE (((((((((table3.col3_3 < '1970-01-01':::DATE) AND (table3.col3_6 > 0:::DECIMAL)) OR (table3.col3_7 != '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table3.col3_1 > 1.0:::FLOAT8)) OR (table3.col3_13 < '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table3.col3_9 = '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) AND (table3.col3_4 < '+Inf':::FLOAT8)) AND (table3.col3_0 <= '-Inf':::FLOAT8)) OR (table3.col3_8 >= (-1):::INT8)) AND (table3.col3_2 = 1:::INT8), UNIQUE (col3_1, col3_5 DESC, col3_10 DESC, col3_3 DESC, col3_13, col3_0) WHERE ((((table3.col3_9 != '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ) AND (table3.col3_13 != '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table3.col3_1 > '+Inf':::FLOAT8)) OR (table3.col3_4 >= 0.0:::FLOAT8)) OR (table3.col3_2 != 1:::INT8), UNIQUE (col3_9 ASC, col3_11, col3_0, col3_3 ASC, col3_1 ASC, col3_6 DESC, col3_4 ASC, col3_5, col3_8 DESC, col3_10, col3_7, col3_2) WHERE (((table3.col3_4 >= '+Inf':::FLOAT8) OR (table3.col3_1 < '-Inf':::FLOAT8)) OR (table3.col3_8 = 2147483647:::INT8)) AND (table3.col3_0 > 0.0:::FLOAT8), INDEX (col3_11 DESC, col3_12, col3_0 ASC, col3_1 ASC, col3_10 DESC, col3_2 DESC), FAMILY (col3_9, col3_12), FAMILY (col3_4, col3_10, col3_0), FAMILY (col3_5, col3_13, col3_1), FAMILY (col3_3, col3_8), FAMILY (col3_6, col3_2), FAMILY (col3_11), FAMILY (col3_7), UNIQUE (col3_2));
        test_1        |             CREATE TABLE table4 (col1_4 DECIMAL NOT NULL, col1_1 "char" NOT NULL, col1_2 INT2 NOT NULL, col1_3 FLOAT8 NOT NULL, col2_8 REGNAMESPACE, col2_13 UUID NULL, col2_21 DECIMAL, col2_4 REGTYPE NULL, col2_22 INET NOT NULL, col2_9 TIMETZ NOT NULL, col2_7 INT2 NOT NULL, col2_18 TIMETZ NULL, col2_14 TIME, col2_16 INT4 NULL, col2_17 INT2, col2_5 NAME NOT NULL, col2_10 REGPROCEDURE, col4_17 REGPROC NOT NULL, col4_18 VARCHAR, col4_19 VARCHAR, col4_20 REGPROCEDURE, col4_21 DATE, col4_22 BYTES NOT NULL, col4_23 BYTES NULL, col4_24 UUID NULL, col4_25 INT8, col4_26 FLOAT4 NULL, col4_27 FLOAT8, col4_28 INTERVAL, col4_29 TIMETZ NOT NULL, col4_30 DATE NULL, col4_31 REGPROC, col4_32 REGTYPE, col4_33 "char" NULL, col4_34 BYTES NULL, col4_35 INT8, PRIMARY KEY (col1_4 DESC, col1_1 ASC, col1_2, col1_3 ASC, col2_8, col2_13, col2_21 DESC, col2_4 ASC, col2_22 DESC, col2_9, col2_7, col2_18, col2_14, col2_16 ASC, col2_17, col2_5 DESC, col2_10 ASC, col4_22 DESC, col4_28 DESC, col4_19, col4_33, col4_23 ASC, col4_17 DESC), INDEX (col2_22 ASC, col4_29, col4_26 ASC, col4_22, col2_10 ASC, col2_9 ASC, col2_14 ASC, col2_17 ASC, col2_4 ASC) STORING (col4_18, col4_20, col4_27, col4_30) WHERE ((((((((((table4.col2_16 < 32767:::INT8) AND (table4.col4_29 = '24:00:00-15:59:00':::TIMETZ)) AND (table4.col4_33 >= e'\'':::STRING)) OR (table4.col2_14 <= '24:00:00':::TIME)) AND (table4.col2_21 < 1:::DECIMAL)) OR (table4.col4_30 < '-infinity':::DATE)) AND (table4.col4_35 != (-128):::INT8)) OR (table4.col1_3 = (-1.0):::FLOAT8)) OR (table4.col4_26 < 'NaN':::FLOAT8)) AND (table4.col2_5 <= e'\U00002603':::STRING)) AND (table4.col1_4 != (-1):::DECIMAL), INDEX (col4_21 DESC) WHERE ((((((((((table4.col4_34 > '\x58':::BYTES) AND (table4.col2_9 != '00:00:00+15:59:00':::TIMETZ)) OR (table4.col1_1 < e'\x00':::STRING)) OR (table4.col2_21 > 1:::DECIMAL)) AND (table4.col2_18 < '00:00:00+15:59:00':::TIMETZ)) AND (table4.col4_26 <= 3.4028234663852886e+38:::FLOAT8)) AND (table4.col2_16 > 1:::INT8)) AND (table4.col4_25 = 0:::INT8)) AND (table4.col2_17 > (-128):::INT8)) OR (table4.col4_30 != 'infinity':::DATE)) OR (table4.col1_4 = 'Infinity':::DECIMAL), UNIQUE (col4_30, col1_3, col4_28 ASC, col4_25) WHERE ((((((((((((((((((table4.col4_22 >= '\x00':::BYTES) AND (table4.col2_9 = '24:00:00-15:59:00':::TIMETZ)) AND (table4.col4_30 < 'infinity':::DATE)) OR (table4.col4_25 >= 127:::INT8)) OR (table4.col4_21 = '4714-11-24 BC':::DATE)) AND (table4.col4_33 != e'\U00002603':::STRING)) AND (table4.col4_34 != '\x22':::BYTES)) AND (table4.col4_19 <= 'X':::STRING)) OR (table4.col4_27 >= '+Inf':::FLOAT8)) OR (table4.col1_1 <= e'\U00002603':::STRING)) OR (table4.col2_14 > '00:00:00':::TIME)) AND (table4.col4_35 > 2147483647:::INT8)) OR (table4.col2_18 < '00:00:00+15:59:00':::TIMETZ)) OR (table4.col2_5 <= e'\x00':::STRING)) OR (table4.col1_3 > 1.0:::FLOAT8)) AND (table4.col2_7 = (-1):::INT8)) OR (table4.col4_18 = e'\'':::STRING)) OR (table4.col4_29 < '24:00:00-15:59:00':::TIMETZ)) AND (table4.col2_16 > 0:::INT8), INDEX (col2_16 DESC, col2_10 ASC, col1_1, col4_25 DESC, col4_26, col2_18 DESC, col4_32, col1_3 DESC, col2_4, col4_28 DESC, col4_29, col4_23, col4_21 DESC, col4_19, col4_34, col1_2 ASC, col4_30 DESC, col2_13 ASC, col4_31 ASC, col2_22 ASC, col4_33 ASC, col2_9 DESC, col4_35 ASC, col2_21 ASC, col4_22 DESC, col2_14 ASC, col2_5 ASC, col4_18 DESC, col4_17 DESC) STORING (col4_20), UNIQUE (col4_35 ASC, col4_19 ASC) WHERE ((((((((((((((((((((((((table4.col4_23 = '\x':::BYTES) AND (table4.col2_5 >= e'\x00':::STRING)) OR (table4.col4_30 >= '-infinity':::DATE)) OR (table4.col2_9 <= '24:00:00-15:59:00':::TIMETZ)) OR (table4.col4_34 <= '\x22':::BYTES)) OR (table4.col2_17 > (-1):::INT8)) OR (table4.col1_1 <= e'\x00':::STRING)) OR (table4.col1_3 != 5e-324:::FLOAT8)) OR (table4.col4_33 < e'\x00':::STRING)) AND (table4.col4_18 < 'X':::STRING)) AND (table4.col2_14 > '24:00:00':::TIME)) AND (table4.col2_16 <= (-32768):::INT8)) OR (table4.col4_26 > 0.0:::FLOAT8)) OR (table4.col1_4 < 'Infinity':::DECIMAL)) OR (table4.col4_19 < e'\x00':::STRING)) AND (table4.col1_2 <= 1:::INT8)) AND (table4.col2_18 < '24:00:00-15:59:00':::TIMETZ)) AND (table4.col4_29 = '00:00:00+15:59:00':::TIMETZ)) AND (table4.col4_22 < '\xff':::BYTES)) OR (table4.col4_21 = 'infinity':::DATE)) AND (table4.col4_35 = (-1):::INT8)) OR (table4.col4_27 = 'NaN':::FLOAT8)) OR (table4.col2_21 < '-Infinity':::DECIMAL)) OR (table4.col4_25 >= 9223372036854775807:::INT8)) AND (table4.col2_7 <= 0:::INT8), INDEX (col1_3 DESC, col2_5, col4_24, col4_18 DESC, col2_21 ASC, col4_21 DESC, col4_27 ASC, col4_20 DESC, col2_9 ASC, col4_28, col2_8 DESC, col2_14, col2_7 DESC, col2_16 DESC, col1_2, col4_22, col4_30 ASC, col4_35, col2_10, col2_4 DESC), UNIQUE (col1_3 DESC, col4_19 DESC, col4_30 ASC, col2_7 DESC, col2_16, col2_5 ASC, col4_22 DESC, col4_27 ASC, col4_33 DESC, col1_2 ASC, col4_18 DESC, col4_31 ASC, col4_32 DESC, col2_10 ASC, col4_24, col1_1 DESC, col2_8 ASC, col4_17 DESC, col4_23 DESC, col2_4 DESC, col2_9 ASC, col2_18 DESC, col4_29 ASC, col2_22, col2_14 ASC, col2_17, col2_21, col1_4 DESC, col4_25, col4_28 DESC, col2_13 DESC, col4_21 DESC, col4_26 ASC, col4_34, col4_35), INDEX (col2_17, col2_13 DESC, col4_23 ASC, col2_4) WHERE ((((table4.col2_9 > '00:00:00+15:59:00':::TIMETZ) AND (table4.col4_23 != '\xff':::BYTES)) AND (table4.col2_18 != '00:00:00+15:59:00':::TIMETZ)) OR (table4.col2_5 < e'\U00002603':::STRING)) OR (table4.col1_2 > (-1):::INT8), UNIQUE (col1_4 DESC, col4_22, col4_26, col4_32 ASC, col4_21, col4_31 ASC, col2_16, col4_23 DESC, col4_20 ASC, col2_13 ASC), FAMILY (col4_28), FAMILY (col4_21), FAMILY (col2_4, col4_18, col1_1), FAMILY (col2_21), FAMILY (col2_5, col4_23), FAMILY (col4_32, col1_2, col4_19, col2_13, col4_34, col2_22), FAMILY (col4_33), FAMILY (col2_9), FAMILY (col1_4, col2_16), FAMILY (col4_26, col2_7, col2_17, col4_30, col4_31), FAMILY (col4_22, col4_25, col2_10, col4_35, col4_20), FAMILY (col1_3, col2_18, col2_14), FAMILY (col4_29), FAMILY (col4_17), FAMILY (col4_24, col2_8, col4_27)) INTERLEAVE IN PARENT table2 (col1_4, col1_1, col1_2, col1_3, col2_8, col2_13, col2_21, col2_4, col2_22, col2_9, col2_7, col2_18, col2_14, col2_16, col2_17, col2_5, col2_10);
        test_1        |             CREATE TABLE table5 (col5_0 NAME NULL, col5_1 TIMESTAMP NOT NULL, col5_2 TIMETZ NOT NULL, col5_3 FLOAT4 NULL, col5_4 TIMETZ NOT NULL, PRIMARY KEY (col5_2 ASC, col5_1), INDEX (col5_3 ASC) STORING (col5_4), INDEX (col5_1 DESC) WHERE (table5.col5_2 < '24:00:00-15:59:00':::TIMETZ) OR (table5.col5_4 >= '24:00:00-15:59:00':::TIMETZ), INDEX (col5_2 DESC, col5_0, col5_3 ASC), UNIQUE (col5_4 DESC, col5_1) STORING (col5_0), UNIQUE (col5_3 ASC, col5_2, col5_0 DESC, col5_4 ASC, col5_1), INDEX (col5_0 ASC, col5_1 DESC, col5_3, col5_2 DESC, col5_4 DESC));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 8016569331679328529, "num_range": 0, "upper_bound": "-3.08771650828286307E+31"}, {"distinct_range": 31.2002679619504, "num_eq": 900000, "num_range": 40, "upper_bound": "-6.992220898415646365E+30"}, {"distinct_range": 60000000, "num_eq": 2942919400495525137, "num_range": 60000000, "upper_bound": "-537433053191.8190101"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 4667991557078676014, "upper_bound": "-829827127.3373701265"}, {"distinct_range": 4867534559251828000, "num_eq": 4113536402701719443, "num_range": 4867534559251827555, "upper_bound": "-61956476.34973587593"}, {"distinct_range": 526949315.9039276, "num_eq": 8000000, "num_range": 3000000000, "upper_bound": "7.792526075861995057"}, {"distinct_range": 4966416638015491000, "num_eq": 3000, "num_range": 4966416638015490654, "upper_bound": "505912942829724726.8"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 1222050618001480131, "num_range": 0, "upper_bound": "-1693947214136299751"}, {"distinct_range": 600, "num_eq": 100, "num_range": 600, "upper_bound": "2147483647"}, {"distinct_range": 2716134060718289400, "num_eq": 7000, "num_range": 5416819312663039777, "upper_bound": "1954528962838943400"}, {"distinct_range": 1708054970.6263926, "num_eq": 800, "num_range": 2000000000, "upper_bound": "2331532772944168951"}, {"distinct_range": 600000000, "num_eq": 1000000000, "num_range": 600000000, "upper_bound": "8842957422290110941"}, {"distinct_range": 0, "num_eq": 304937838785665790, "num_range": 100000000, "upper_bound": "9022680501725431526"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 314462696, "histo_col_type": "", "name": "__auto__", "null_count": 525795023, "row_count": 600000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 515924847, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 495064176, "histo_col_type": "", "name": "__auto__", "null_count": 530331764, "row_count": 600000000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 352701907, "histo_col_type": "", "name": "__auto__", "null_count": 133222964, "row_count": 600000000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 68435942, "histo_col_type": "", "name": "__auto__", "null_count": 365969532, "row_count": 600000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 485498122, "histo_buckets": [{"distinct_range": 0, "num_eq": 6824444095359174822, "num_range": 0, "upper_bound": "-905327418948998015.5"}, {"distinct_range": 559.1955931833805, "num_eq": 2556280023577073054, "num_range": 8000, "upper_bound": "0.4067095172935149621"}, {"distinct_range": 0, "num_eq": 3199664515830140267, "num_range": 4843821745994437094, "upper_bound": "8.736688150439167897"}, {"distinct_range": 0, "num_eq": 60000000000, "num_range": 1000000000, "upper_bound": "88.8935317129474726"}, {"distinct_range": 700000000, "num_eq": 90000, "num_range": 700000000, "upper_bound": "6032.395991692574408"}, {"distinct_range": 300000000, "num_eq": 700, "num_range": 300000000, "upper_bound": "27827.73328662437696"}, {"distinct_range": 27.83020103714969, "num_eq": 60000, "num_range": 50, "upper_bound": "39506264649322.90654"}, {"distinct_range": 6664872152798676000, "num_eq": 4878992141557724694, "num_range": 6664872152798676312, "upper_bound": "5.802360106625897643E+26"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 600000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 80043605, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 183193677, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 140036731, "histo_col_type": "", "name": "__auto__", "null_count": 181285580, "row_count": 600000000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 466730134, "histo_col_type": "", "name": "__auto__", "null_count": 280093394, "row_count": 600000000}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 557241931, "histo_col_type": "", "name": "__auto__", "null_count": 257172923, "row_count": 600000000}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1084853, "histo_buckets": [{"distinct_range": 0, "num_eq": 1811113015119711067, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 3325579457341968795, "num_range": 700000, "upper_bound": "3995345025"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 600000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 542390808, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 469453025, "histo_col_type": "", "name": "__auto__", "null_count": 170600463, "row_count": 600000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 94488435, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 145924499, "histo_col_type": "", "name": "__auto__", "null_count": 331249962, "row_count": 600000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 184400877, "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 600000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 113201267, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000000}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 271474179, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000000}, {"columns": ["col2_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 565719164, "histo_col_type": "", "name": "__auto__", "null_count": 300589144, "row_count": 600000000}, {"columns": ["col2_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 115007282, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 114369080, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "f)\\u0010"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 600000000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 488039923, "histo_col_type": "", "name": "__auto__", "null_count": 272674518, "row_count": 600000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3759860708887808072, "histo_col_type": "", "name": "__auto__", "null_count": 2162620771088918337, "row_count": 8121879582202589545}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7211721461761842352, "histo_col_type": "", "name": "__auto__", "null_count": 913699131875008659, "row_count": 8121879582202589545}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5709037263305556336, "histo_buckets": [{"distinct_range": 0, "num_eq": 600, "num_range": 0, "upper_bound": "BOX(-1.4653146196548237 2.0930747749508347,-0.10353832251740921 2.2501999575869096)"}, {"distinct_range": 353651.0206571285, "num_eq": 3403917675433197906, "num_range": 1000000, "upper_bound": "BOX(-1.3396780210878614 -1.1103576433867854,2.2738559539504943 -0.9506593333131466)"}, {"distinct_range": 50, "num_eq": 200000000, "num_range": 50, "upper_bound": "BOX(-1.0564027819331274 -0.6246142960361264,1.847630893656717 0.75042380808848)"}, {"distinct_range": 4519038715069122600, "num_eq": 2698654458292303808, "num_range": 8914656339967410213, "upper_bound": "BOX(-0.9692590360712977 -0.3284894794722635,0.9456451012280733 0.6775168088792222)"}, {"distinct_range": 0, "num_eq": 4901665256517146266, "num_range": 10000, "upper_bound": "BOX(-0.6901729584299464 -1.512982938613684,1.0437770651461757 -0.7125595028937884)"}, {"distinct_range": 2947776292.5364647, "num_eq": 5000000, "num_range": 7000000000, "upper_bound": "BOX(-0.5626480330898862 -1.0934965766183307,0.6557159857674002 -0.11297769687254981)"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 6556313479818401686, "upper_bound": "BOX(-0.21624783456410773 -0.7773804600658807,0.0988825896909028 0.5100789867398059)"}, {"distinct_range": 3931392449438860000, "num_eq": 0, "num_range": 3931392449438859948, "upper_bound": "BOX(0.799028696078501 0.08712893062837135,1.7232558478899398 0.20749847732236515)"}, {"distinct_range": 2510765575660397000, "num_eq": 7304140058382770296, "num_range": 2510765575660397167, "upper_bound": "BOX(0.943478286199615 -0.3923298928524711,1.2721367961300585 0.09735124130590089)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 4521017217171495307, "row_count": 8121879582202589545}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5375205441037616326, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8121879582202589545}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7833834748769988705, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "9b33ee2b-5dd0-4715-905f-73204d3f8fec"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 9000, "upper_bound": "a94baf54-296c-4065-be15-ff3ddcff8b23"}, {"distinct_range": 3861104473036389000, "num_eq": 100000000, "num_range": 4936576235677882064, "upper_bound": "b3a21b47-568f-4818-a249-4d9a758dacbb"}, {"distinct_range": 42525092661668744, "num_eq": 70000000, "num_range": 1138530083869852391, "upper_bound": "d0e3d195-ab1d-479e-b2fb-b315b03d30b6"}, {"distinct_range": 70, "num_eq": 300000000, "num_range": 70, "upper_bound": "dc45559f-14f5-4a0b-ade5-d12567b7a8d0"}, {"distinct_range": 69258.12333698272, "num_eq": 1008172698512909090, "num_range": 90000, "upper_bound": "e27017bd-2033-4856-adb7-7254f8efadef"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 8121879582202589545}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2351580337095816420, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000, "num_range": 0, "upper_bound": "581007189"}, {"distinct_range": 243136812.48977214, "num_eq": 9000000, "num_range": 400000000, "upper_bound": "721342584"}, {"distinct_range": 3415167169117929500, "num_eq": 9000, "num_range": 3415167169117929300, "upper_bound": "740839553"}, {"distinct_range": 0, "num_eq": 1702201876006316470, "num_range": 100, "upper_bound": "891464854"}, {"distinct_range": 0, "num_eq": 400, "num_range": 1000000000, "upper_bound": "2083180584"}, {"distinct_range": 0, "num_eq": 0, "num_range": 20000000000, "upper_bound": "2579989936"}, {"distinct_range": 0, "num_eq": 200000000, "num_range": 0, "upper_bound": "4147422229"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 2701057413211739432, "row_count": 8121879582202589545}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4935227031555838868, "histo_col_type": "", "name": "__auto__", "null_count": 3323246709728515026, "row_count": 8121879582202589545}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1138370281610192601, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "-0.961844364254076"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 8121879582202589545}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5070860461683445171, "histo_buckets": [{"distinct_range": 0, "num_eq": 4477845267989426212, "num_range": 0, "upper_bound": "-1.966738224029541"}, {"distinct_range": 0, "num_eq": 8070901523624350417, "num_range": 2807628542564276385, "upper_bound": "-0.8351617455482483"}, {"distinct_range": 0, "num_eq": 1481025875618826573, "num_range": 6984444130278378749, "upper_bound": "-0.19852451980113983"}, {"distinct_range": 7000000000, "num_eq": 200, "num_range": 7000000000, "upper_bound": "0.3275810182094574"}, {"distinct_range": 9000, "num_eq": 994704225757439437, "num_range": 9000, "upper_bound": "0.9319469332695007"}, {"distinct_range": 138164127.2609842, "num_eq": 90000000, "num_range": 200000000, "upper_bound": "1.4346126317977905"}, {"distinct_range": 8996758033142967000, "num_eq": 7048041887111894293, "num_range": 8996758033142966885, "upper_bound": "1.7268130779266357"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 8121879582202589545}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1601536231238088507, "histo_col_type": "", "name": "__auto__", "null_count": 3587707892581346401, "row_count": 8121879582202589545}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5937479542559348312, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8121879582202589545}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6435665509445322114, "histo_buckets": [{"distinct_range": 0, "num_eq": 500, "num_range": 0, "upper_bound": "1970-09-17 22:29:21.000873+00:00"}, {"distinct_range": 50000000, "num_eq": 40000000, "num_range": 50000000, "upper_bound": "1971-07-06 05:32:45.000712+00:00"}, {"distinct_range": 3647132721772856300, "num_eq": 103268929816741896, "num_range": 3826197353584150926, "upper_bound": "1981-07-08 11:49:15.00035+00:00"}, {"distinct_range": 646656.9594764425, "num_eq": 600000, "num_range": 900000, "upper_bound": "1981-12-18 16:53:47.000634+00:00"}, {"distinct_range": 4197211743120378400, "num_eq": 500, "num_range": 8378107373783474590, "upper_bound": "1982-09-25 22:39:06.000732+00:00"}, {"distinct_range": 0, "num_eq": 100, "num_range": 5806570727218541912, "upper_bound": "2007-04-24 14:12:24.000536+00:00"}, {"distinct_range": 0, "num_eq": 9000000000, "num_range": 10, "upper_bound": "2026-12-15 01:55:00.000343+00:00"}, {"distinct_range": 2440341100253200400, "num_eq": 4037586372754377099, "num_range": 6540609836639026796, "upper_bound": "2027-07-03 14:24:44.000072+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 3800533339927861771, "row_count": 8121879582202589545}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 795934727696800941, "histo_buckets": [{"distinct_range": 0, "num_eq": 503720937522387875, "num_range": 0, "upper_bound": "304789200"}, {"distinct_range": 9000000000, "num_eq": 0, "num_range": 9000000000, "upper_bound": "340524382"}, {"distinct_range": 0, "num_eq": 802420754659001242, "num_range": 2271022087233718304, "upper_bound": "851108364"}, {"distinct_range": 100000000, "num_eq": 2073516976790464841, "num_range": 100000000, "upper_bound": "3186770864"}, {"distinct_range": 5000000000, "num_eq": 7000000000, "num_range": 5000000000, "upper_bound": "3561421410"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 8121879582202589545}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1954556786885269652, "histo_buckets": [{"distinct_range": 0, "num_eq": 6236187013403589534, "num_range": 0, "upper_bound": "-0.7675374746322632"}, {"distinct_range": 1000, "num_eq": 800000000, "num_range": 1000, "upper_bound": "1.8485679626464844"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 8121879582202589545}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4996892320, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col4_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 191863498, "histo_col_type": "", "name": "__auto__", "null_count": 2861036853, "row_count": 7000000000}, {"columns": ["col4_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5860004494, "histo_col_type": "", "name": "__auto__", "null_count": 4756325705, "row_count": 7000000000}, {"columns": ["col4_26"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5394840633, "histo_col_type": "", "name": "__auto__", "null_count": 3707493409, "row_count": 7000000000}, {"columns": ["col4_29"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4027467656, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col4_34"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4792776677, "histo_col_type": "", "name": "__auto__", "null_count": 3319675977, "row_count": 7000000000}, {"columns": ["col4_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6376817771, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4929118460, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6471641165, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3991647129, "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6375549967, "histo_col_type": "", "name": "__auto__", "null_count": 5606080623, "row_count": 7000000000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6206337627, "histo_col_type": "", "name": "__auto__", "null_count": 4655571176, "row_count": 7000000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4474395824, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3449626582, "histo_col_type": "", "name": "__auto__", "null_count": 4393718553, "row_count": 7000000000}, {"columns": ["col4_31"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1350900325, "histo_col_type": "", "name": "__auto__", "null_count": 4431234666, "row_count": 7000000000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2468773302, "histo_buckets": [{"distinct_range": 0, "num_eq": 5845429515474440525, "num_range": 0, "upper_bound": "-19947"}, {"distinct_range": 0, "num_eq": 6630746086584690816, "num_range": 3825397675077608068, "upper_bound": "-2523"}, {"distinct_range": 0, "num_eq": 200000, "num_range": 3787167047597985264, "upper_bound": "393"}, {"distinct_range": 200, "num_eq": 4100726429405382449, "num_range": 200, "upper_bound": "6438"}, {"distinct_range": 1458882023791981800, "num_eq": 1946297469964109939, "num_range": 8771615823973112748, "upper_bound": "10590"}, {"distinct_range": 70000, "num_eq": 700000, "num_range": 70000, "upper_bound": "17050"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 6438617415, "row_count": 7000000000}, {"columns": ["col4_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5275938540, "histo_col_type": "", "name": "__auto__", "null_count": 3882034768, "row_count": 7000000000}, {"columns": ["col4_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1291153007, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000000, "num_range": 0, "upper_bound": "1976-05-13"}, {"distinct_range": 165.90494592030547, "num_eq": 6606411489206945124, "num_range": 6000, "upper_bound": "1977-04-05"}, {"distinct_range": 900000, "num_eq": 5745173166609325413, "num_range": 900000, "upper_bound": "1978-08-12"}, {"distinct_range": 7239209345006042000, "num_eq": 5133244566672055961, "num_range": 7239209345006042532, "upper_bound": "1978-12-17"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 6307393864, "row_count": 7000000000}, {"columns": ["col4_28"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1163675503, "histo_col_type": "", "name": "__auto__", "null_count": 855333469, "row_count": 7000000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5982982286, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4794925930, "histo_col_type": "", "name": "__auto__", "null_count": 1052564485, "row_count": 7000000000}, {"columns": ["col4_27"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6875970044, "histo_col_type": "", "name": "__auto__", "null_count": 4249759755, "row_count": 7000000000}, {"columns": ["col4_30"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5431455759, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000, "num_range": 0, "upper_bound": "1970-01-01"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "1970-09-02"}, {"distinct_range": 5784602.874289215, "num_eq": 0, "num_range": 6000000, "upper_bound": "1974-11-07"}, {"distinct_range": 300000000, "num_eq": 90000000000, "num_range": 300000000, "upper_bound": "1979-02-03"}, {"distinct_range": 1113049333.084809, "num_eq": 8475467883701960882, "num_range": 5000000000, "upper_bound": "1979-10-16"}, {"distinct_range": 1355774447381595600, "num_eq": 8383614710984532909, "num_range": 1679239865177093869, "upper_bound": "1981-03-09"}, {"distinct_range": 2599222058718074400, "num_eq": 30, "num_range": 2599222058718074281, "upper_bound": "1995-04-08"}, {"distinct_range": 7701576333673710000, "num_eq": 80000000000, "num_range": 7701576333673709447, "upper_bound": "1996-04-15"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 5897727257, "row_count": 7000000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5363005258, "histo_col_type": "", "name": "__auto__", "null_count": 4175800643, "row_count": 7000000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1567029507, "histo_col_type": "", "name": "__auto__", "null_count": 258326252, "row_count": 7000000000}, {"columns": ["col2_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2622463572, "histo_buckets": [{"distinct_range": 0, "num_eq": 8428471732921716866, "num_range": 0, "upper_bound": "b838:ce32:b9ad:2694:64a7:3573:524b:d661/23"}, {"distinct_range": 9218216716678122000, "num_eq": 60000000000, "num_range": 9218216716678122907, "upper_bound": "b3d5:ab31:212e:c263:f39d:b2d3:a98e:81fc/69"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6818403591, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6038850296, "histo_buckets": [{"distinct_range": 0, "num_eq": 922710037366410433, "num_range": 0, "upper_bound": "-880989756"}, {"distinct_range": 0, "num_eq": 4732992249013883215, "num_range": 7873796535387765576, "upper_bound": "370243161"}, {"distinct_range": 2188.2051087027967, "num_eq": 8702295855184939212, "num_range": 3000, "upper_bound": "693543035"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 1925121475, "row_count": 7000000000}, {"columns": ["col4_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1698970174, "histo_col_type": "", "name": "__auto__", "null_count": 5797691968, "row_count": 7000000000}, {"columns": ["col4_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4685382560, "histo_col_type": "", "name": "__auto__", "null_count": 5721350161, "row_count": 7000000000}, {"columns": ["col4_25"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3011004777, "histo_col_type": "", "name": "__auto__", "null_count": 1987347884, "row_count": 7000000000}, {"columns": ["col4_35"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3833288361, "histo_buckets": [{"distinct_range": 0, "num_eq": 7788159989677216873, "num_range": 0, "upper_bound": "-8212456481990480864"}, {"distinct_range": 2841686945311891500, "num_eq": 1000, "num_range": 2841686945311891654, "upper_bound": "-5325872627972183619"}, {"distinct_range": 2943757510715029000, "num_eq": 7000000, "num_range": 2943757510715028906, "upper_bound": "-3212585390411409729"}, {"distinct_range": 0, "num_eq": 3585702725292764092, "num_range": 9049308028160493071, "upper_bound": "-1837973610787019090"}, {"distinct_range": 70, "num_eq": 1893053803963031714, "num_range": 70, "upper_bound": "-1725412389388203332"}, {"distinct_range": 563724523040836600, "num_eq": 2541580826567385529, "num_range": 2205766208164648948, "upper_bound": "752919231583139774"}, {"distinct_range": 0, "num_eq": 5231868838845688105, "num_range": 4167864643804401804, "upper_bound": "2857807223003948434"}, {"distinct_range": 2000000, "num_eq": 80000000000, "num_range": 2000000, "upper_bound": "4005635894786412441"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 952590936, "row_count": 7000000000}, {"columns": ["col2_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3713536497, "histo_col_type": "", "name": "__auto__", "null_count": 827260857, "row_count": 7000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6149648121, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000, "num_range": 0, "upper_bound": "-2.048735429887712502E+36"}, {"distinct_range": 88651653816126640, "num_eq": 800000, "num_range": 582243080726140377, "upper_bound": "3532589122373.94466"}, {"distinct_range": 7166047939649562000, "num_eq": 0, "num_range": 7166047939649562076, "upper_bound": "5.525553531802222425E+24"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 30, "upper_bound": "7.20542219408609801E+25"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 7000000000}, {"columns": ["col4_32"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6179076517, "histo_col_type": "", "name": "__auto__", "null_count": 5832211142, "row_count": 7000000000}, {"columns": ["col4_33"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6929669102, "histo_col_type": "", "name": "__auto__", "null_count": 1589245144, "row_count": 7000000000}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS e'[{"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 212557632177075517, "histo_buckets": [{"distinct_range": 0, "num_eq": 3399180840382023470, "num_range": 0, "upper_bound": "09:01:48.131284-05:51:00"}, {"distinct_range": 5130174950841035000, "num_eq": 1280117157107947435, "num_range": 8938325732095822266, "upper_bound": "12:33:54.454443-13:47:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 8277827564545904686}, {"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 442850481020018577, "histo_buckets": [{"distinct_range": 0, "num_eq": 5638876843588232645, "num_range": 0, "upper_bound": "-0.7121386528015137"}, {"distinct_range": 0, "num_eq": 300, "num_range": 3272523068902166918, "upper_bound": "-0.6773631572723389"}, {"distinct_range": 335486.652498713, "num_eq": 5649078911725709109, "num_range": 600000, "upper_bound": "0.0"}, {"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "0.05189306288957596"}, {"distinct_range": 8769.317166051394, "num_eq": 1000000, "num_range": 100000, "upper_bound": "0.17333751916885376"}, {"distinct_range": 2074677887089784800, "num_eq": 8394649033428314987, "num_range": 2074677887089784884, "upper_bound": "0.20643921196460724"}, {"distinct_range": 5205435061029106000, "num_eq": 0, "num_range": 6063766574285120485, "upper_bound": "0.4543229937553406"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 20, "upper_bound": "0.5168222784996033"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 6169107615026353576, "row_count": 8277827564545904686}, {"columns": ["col5_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2923238528523372083, "histo_buckets": [{"distinct_range": 0, "num_eq": 2992378593333882932, "num_range": 0, "upper_bound": "03:16:36.249479+08:00:00"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 400, "upper_bound": "03:11:04.73954+03:06:00"}, {"distinct_range": 7467937511994235000, "num_eq": 50000000, "num_range": 8231533719054921961, "upper_bound": "08:20:12.181365+04:05:00"}, {"distinct_range": 800000, "num_eq": 1018372647040028223, "num_range": 800000, "upper_bound": "19:26:40.852639+09:27:00"}, {"distinct_range": 0, "num_eq": 600000, "num_range": 1000, "upper_bound": "11:51:41.195737+01:25:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 8277827564545904686}, {"columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5210980292152731920, "histo_buckets": [{"distinct_range": 0, "num_eq": 822293405236260330, "num_range": 0, "upper_bound": ""}, {"distinct_range": 2289765.8366869586, "num_eq": 1000000000, "num_range": 6000000000, "upper_bound": "&m"}, {"distinct_range": 7259648959703318000, "num_eq": 20000, "num_range": 7259648959703317191, "upper_bound": ")[\\u001bM k\\u0017\\u0013*"}, {"distinct_range": 0, "num_eq": 0, "num_range": 5138771064386188432, "upper_bound": "/ +K"}, {"distinct_range": 6023785299254441000, "num_eq": 50, "num_range": 6023785299254441348, "upper_bound": "6*)\\u0014^:KWd"}, {"distinct_range": 1768046691372512500, "num_eq": 6899180932946546239, "num_range": 1768046691372512533, "upper_bound": "8^B1\\u0006\\u000e6"}, {"distinct_range": 0, "num_eq": 800, "num_range": 40000, "upper_bound": "}\\u000fa\\u0005o\\u00013\\u000c-"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 5305037117389724862, "row_count": 8277827564545904686}, {"columns": ["col5_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6455614239483465050, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000, "num_range": 0, "upper_bound": "1980-11-03 00:00:17.000085"}, {"distinct_range": 624178772558566900, "num_eq": 9000000, "num_range": 859146542920824705, "upper_bound": "1991-07-02 03:00:16.000639"}, {"distinct_range": 60, "num_eq": 8166693016139466938, "num_range": 60, "upper_bound": "1994-06-10 07:30:56.000621"}, {"distinct_range": 2551693.264105763, "num_eq": 1171225107700204908, "num_range": 100000000, "upper_bound": "2021-11-16 12:19:14.000652"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 8277827564545904686}]':::JSONB;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_7) REFERENCES table3 (col3_2);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('uyeuq');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('bwowt', 'mwttd', 'k', 'v', 'mnbu');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('ih', 'crkmp', 'me', 'bguiti', 'ig');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ...

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2650261&tab=buildLog) on [master@81a2c26a104fa8cc7e8b530b837ffb6ff85ddc5a](https://github.com/cockroachdb/cockroach/commits/81a2c26a104fa8cc7e8b530b837ffb6ff85ddc5a):

```
        test_1        |             CREATE INDEX ON table2 (col2_0 ASC, col2_3 ASC, col2_5 ASC) INCLUDE (col2_2, col2_7);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_4 DESC) INCLUDE (col2_1, col2_2, col2_3, col2_5, col2_7);
        test_1        |             CREATE INDEX ON table2 (col2_1);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_0 ASC, col2_3, col2_8 ASC, col2_1 ASC, col2_7 DESC, col2_5 DESC, col2_6, col2_2 ASC);
        test_1        |             CREATE INDEX ON table2 (col2_5 ASC, col2_6 ASC) INCLUDE (col2_1, col2_3, col2_4);
        test_1        |             CREATE TABLE table3 (col3_0 REGPROCEDURE, col3_1 TIME NOT NULL, col3_2 REGNAMESPACE NOT NULL, col3_3 REGCLASS NULL, col3_4 INT8, PRIMARY KEY (col3_2, col3_1));
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_1, col3_2 ASC, col3_4, col3_3 ASC);
        test_1        |             CREATE INDEX ON table3 (col3_2, col3_1, col3_4) INCLUDE (col3_0, col3_3);
        test_1        |             CREATE TABLE table4 (col4_0 BOX2D NOT NULL, col4_1 REGTYPE NOT NULL, col4_2 TEXT NOT NULL, col4_3 INT8, col4_4 CHAR NOT NULL, col4_5 TIMESTAMPTZ NOT NULL, col4_6 INTERVAL NOT NULL, col4_7 CHAR NOT NULL, col4_8 INET NOT NULL, col4_9 BIT(18), col4_10 DATE NOT NULL, col4_11 CHAR, col4_12 "char" NULL, col4_13 TIMETZ NULL, col4_14 REGPROC NOT NULL, col4_15 REGNAMESPACE NOT NULL, col4_16 REGTYPE NOT NULL, col4_17 TIMESTAMP NOT NULL, col4_18 INT8 NOT NULL, PRIMARY KEY (col4_7, col4_2, col4_1, col4_14, col4_8, col4_18, col4_5, col4_15, col4_0, col4_4, col4_10, col4_6, col4_16));
        test_1        |             CREATE INDEX ON table4 (col4_2 DESC, col4_11, col4_15 DESC, col4_1 DESC, col4_12, col4_5, col4_13) INCLUDE (col4_3);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_18 ASC, col4_0 DESC, col4_7, col4_6 ASC, col4_10 DESC, col4_11 DESC, col4_4 DESC, col4_9 ASC, col4_8, col4_12, col4_5 DESC, col4_16 ASC, col4_2 DESC, col4_15, col4_14 ASC, col4_3 DESC);
        test_1        |             CREATE INDEX ON table4 (col4_2 ASC, col4_0, col4_4, col4_18 DESC, col4_1 DESC, col4_9, col4_7 ASC, col4_6 ASC, col4_17 DESC, col4_12 DESC) INCLUDE (col4_3, col4_11, col4_13);
        test_1        |             CREATE INDEX ON table4 (col4_11 ASC, col4_4, col4_9 ASC, col4_8, col4_6, col4_2 ASC, col4_0, col4_13 ASC, col4_5 DESC, col4_15, col4_17, col4_14 ASC, col4_12, col4_16 ASC) INCLUDE (col4_3);
        test_1        |             CREATE INDEX ON table4 (col4_17, col4_4 DESC, col4_1, col4_5 ASC, col4_2 DESC, col4_0 ASC, col4_10 ASC, col4_8, col4_6, col4_13 ASC, col4_14 ASC, col4_3 ASC, col4_11 DESC, col4_12 ASC, col4_15, col4_16 DESC) INCLUDE (col4_9);
        test_1        |             CREATE INDEX ON table4 (col4_8 ASC, col4_6 DESC, col4_4 DESC, col4_14 ASC, col4_13 ASC, col4_17 ASC, col4_9 DESC, col4_2 ASC, col4_12, col4_3 ASC, col4_5, col4_11, col4_10, col4_16, col4_18 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_11, col4_10, col4_2 ASC, col4_4 ASC, col4_0, col4_9 DESC, col4_17, col4_13 ASC, col4_12 DESC, col4_6 ASC, col4_1 ASC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('au', 'lrnfo', 'oowjwo');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('dimsc', 'gphphj', 'f');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('wi', 'jujflt', 'fs');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('w', 'j', 'vvawd', 'xicklq', 'mxj');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (0.57s)
        test_1        |         compare_test.go:129: 
        test_1        |             		SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;
        test_1        |             		SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;
        test_1        |             	CREATE TABLE table1 (col1_0 TIME NOT NULL, col1_1 TIMESTAMPTZ NOT NULL, col1_2 BYTES NOT NULL, col1_3 NAME NOT NULL, col1_4 REGPROCEDURE, col1_5 TIME NOT NULL, col1_6 BIT(18) NOT NULL, col1_7 INTERVAL NOT NULL, PRIMARY KEY (col1_3, col1_7 ASC, col1_2, col1_0 DESC, col1_1 ASC, col1_6 DESC), INDEX (col1_0, col1_6, col1_7 DESC, col1_2 ASC) WHERE (table1.col1_2 >= '\x27':::BYTES) OR (table1.col1_0 = '24:00:00':::TIME), INDEX (col1_1 ASC, col1_7 DESC, col1_2, col1_6, col1_0, col1_5 DESC) WHERE ((table1.col1_3 < e'\'':::STRING) OR (table1.col1_0 = '24:00:00':::TIME)) AND (table1.col1_2 <= '\x58':::BYTES), UNIQUE (col1_0 ASC, col1_4 DESC) STORING (col1_5), INDEX (col1_3 ASC, col1_5 ASC, col1_4 DESC, col1_2 DESC, col1_1 ASC, col1_0 ASC, col1_6), UNIQUE (col1_7 DESC, col1_0 ASC, col1_4 ASC, col1_2 ASC, col1_1), FAMILY (col1_1, col1_4), FAMILY (col1_2, col1_7, col1_5, col1_0), FAMILY (col1_3), FAMILY (col1_6));
        test_1        |             CREATE TABLE table2 (col1_3 NAME NOT NULL, col1_7 INTERVAL NOT NULL, col1_2 BYTES NOT NULL, col1_0 TIME NOT NULL, col1_1 TIMESTAMPTZ NOT NULL, col1_6 BIT(18) NOT NULL, col2_6 REGNAMESPACE, col2_7 TIME NULL, col2_8 INT4 NOT NULL, col2_9 REGPROCEDURE NULL, col2_10 OID, col2_11 REGTYPE NULL, col2_12 TIME, col2_13 STRING, col2_14 REGNAMESPACE NULL, col2_15 REGNAMESPACE NOT NULL, col2_16 INT8, col2_17 TIMESTAMPTZ NOT NULL, col2_18 OID NOT NULL, col2_19 INT2, col2_20 BOX2D NULL, col2_21 REGCLASS, col2_22 REGCLASS, col2_23 FLOAT4, PRIMARY KEY (col1_3, col1_7 ASC, col1_2, col1_0 DESC, col1_1 ASC, col1_6 DESC, col2_11 ASC), INDEX (col2_13 ASC, col2_6, col1_3, col2_9 ASC, col2_22, col2_7 ASC, col2_23, col2_10 ASC, col1_7, col1_2 DESC, col2_14 DESC, col2_11 ASC, col2_8, col1_1 ASC, col2_17 DESC, col2_19, col2_16, col2_12, col2_15 DESC, col1_6, col2_20 ASC) WHERE (((((table2.col1_1 = '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ) AND (table2.col2_13 < '"':::STRING)) AND (table2.col2_8 != (-128):::INT8)) OR (table2.col2_16 >= (-1):::INT8)) AND (table2.col2_23 <= '+Inf':::FLOAT8)) AND (table2.col2_7 != '24:00:00':::TIME), INDEX (col2_6 DESC, col2_13, col2_14 DESC, col2_20 DESC, col2_22, col2_17 DESC, col2_16, col2_21 ASC, col2_9) STORING (col2_7, col2_10, col2_12, col2_15) WHERE table2.col1_0 < '24:00:00':::TIME, INDEX (col2_16 DESC, col2_6 ASC, col2_17, col2_13, col2_18 DESC, col1_0, col2_20 DESC, col1_1 DESC, col2_11) STORING (col2_9, col2_10, col2_12, col2_14, col2_15, col2_19, col2_21, col2_22, col2_23), INDEX (col1_1, col2_19 DESC, col2_9 ASC, col2_7 DESC, col1_0, col2_23 DESC, col2_12) STORING (col2_6, col2_10, col2_14, col2_15, col2_17, col2_20, col2_21, col2_22), UNIQUE (col2_19, col2_10 DESC, col2_11, col2_17 ASC, col2_18 DESC, col2_14 DESC, col2_21) STORING (col2_6, col2_7, col2_8, col2_9, col2_12, col2_13, col2_16, col2_20, col2_22, col2_23), UNIQUE (col2_10, col2_9, col2_18, col1_7, col1_3 DESC, col1_6, col2_8 ASC, col2_20 DESC, col1_1 DESC, col2_21, col2_17, col2_16 ASC, col2_15 ASC, col1_2, col2_6 DESC, col2_11 ASC, col1_0) STORING (col2_12, col2_14, col2_23), UNIQUE (col2_11 DESC, col2_22 DESC, col2_15, col2_12, col1_2 DESC, col2_8 DESC, col2_17 DESC, col2_21 ASC, col2_16, col2_9 ASC, col1_6 DESC, col1_0 DESC, col2_20 DESC, col1_1 ASC) STORING (col2_7, col2_14, col2_18, col2_19) WHERE (((((table2.col1_2 > '\x00':::BYTES) OR (table2.col1_0 <= '24:00:00':::TIME)) OR (table2.col2_8 != (-1):::INT8)) OR (table2.col2_19 >= 0:::INT8)) OR (table2.col1_3 < e'\'':::STRING)) AND (table2.col2_16 <= 9223372036854775807:::INT8), INDEX (col2_7) STORING (col2_8, col2_12, col2_13, col2_19, col2_23) WHERE ((((table2.col2_17 < '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ) AND (table2.col1_0 = '24:00:00':::TIME)) AND (table2.col1_2 < '\xe29883':::BYTES)) AND (table2.col2_19 = (-1):::INT8)) AND (table2.col2_16 > 127:::INT8), INDEX (col1_2 ASC, col2_9 ASC, col2_18 ASC, col2_10, col2_12, col2_21 DESC, col2_15, col1_0, col1_6 DESC, col2_14 ASC, col2_6 ASC, col2_22, col2_23 ASC) WHERE (((((((((((table2.col2_17 != '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ) OR (table2.col1_1 > '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table2.col1_2 < '\xe29883':::BYTES)) AND (table2.col2_23 > '+Inf':::FLOAT8)) OR (table2.col2_13 = 'X':::STRING)) OR (table2.col1_3 >= e'\U00002603':::STRING)) AND (table2.col2_19 < 1:::INT8)) AND (table2.col2_8 <= 1:::INT8)) OR (table2.col2_12 < '24:00:00':::TIME)) AND (table2.col2_16 < (-32768):::INT8)) OR (table2.col1_0 > '24:00:00':::TIME)) OR (table2.col2_7 > '00:00:00':::TIME)) INTERLEAVE IN PARENT table1 (col1_3, col1_7, col1_2, col1_0, col1_1, col1_6);
        test_1        |             CREATE TABLE table3 (col1_3 NAME NOT NULL, col1_7 INTERVAL NOT NULL, col1_2 BYTES NOT NULL, col1_0 TIME NOT NULL, col1_1 TIMESTAMPTZ NOT NULL, col1_6 BIT(18) NOT NULL, col3_6 REGPROCEDURE NULL, col3_7 TIMESTAMP NOT NULL, col3_8 TIMESTAMPTZ NOT NULL, col3_9 REGNAMESPACE NULL, col3_10 REGCLASS NOT NULL, col3_11 REGTYPE NULL, col3_12 INTERVAL, col3_13 "char" NOT NULL, col3_14 INT8 NOT NULL, col3_15 TIME NULL, col3_16 STRING NOT NULL, col3_17 REGNAMESPACE NOT NULL, col3_18 DATE, PRIMARY KEY (col1_3, col1_7 ASC, col1_2, col1_0 DESC, col1_1 ASC, col1_6 DESC, col3_7 DESC, col3_9 ASC), UNIQUE (col3_13 ASC, col1_7, col3_11 ASC, col3_12, col3_8 ASC, col1_6 ASC, col3_18, col3_14, col3_10, col3_6 ASC, col1_1 ASC, col3_7 DESC, col3_17, col1_3, col3_16 DESC, col1_0 DESC, col3_15, col3_9, col1_2 DESC) WHERE ((((((((((table3.col3_18 < '5874897-12-31':::DATE) AND (table3.col3_13 != e'\'':::STRING)) OR (table3.col3_14 != (-128):::INT8)) OR (table3.col3_8 > '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table3.col1_3 > e'\x00':::STRING)) OR (table3.col3_16 != 'X':::STRING)) OR (table3.col1_1 != '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table3.col1_0 >= '00:00:00':::TIME)) AND (table3.col3_15 < '24:00:00':::TIME)) AND (table3.col1_2 > '\x27':::BYTES)) OR (table3.col3_7 <= '3000-01-01 00:00:00':::TIMESTAMP), INDEX (col3_14 DESC, col1_3 DESC, col1_0 ASC, col3_10, col3_7 DESC, col3_16 DESC, col1_1 DESC, col3_9 ASC, col3_18 ASC, col3_11 ASC, col3_12 DESC, col1_6 ASC, col3_13 ASC, col3_6 DESC, col1_7 DESC, col1_2 DESC, col3_8 ASC, col3_15), INDEX (col1_7 DESC, col3_6 ASC) STORING (col3_8, col3_10, col3_13, col3_14, col3_15) WHERE table3.col3_14 > 1:::INT8, UNIQUE (col3_15 ASC, col3_11, col3_7 DESC, col1_7 DESC, col3_13 ASC, col3_16 DESC, col3_6 DESC, col1_3 ASC, col3_8 DESC, col3_10, col3_9, col3_17 ASC, col1_2 ASC, col3_18, col1_0 DESC, col1_6, col1_1 ASC), INDEX (col3_18 ASC, col3_12 ASC, col1_0, col1_3 ASC, col3_9 ASC, col3_13, col3_10 DESC, col3_15 DESC, col1_6, col3_17, col3_14 DESC, col1_1, col1_2, col3_11 DESC, col3_8 DESC) WHERE ((((((((((table3.col1_3 >= 'X':::STRING) AND (table3.col1_1 <= '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) OR (table3.col3_16 > '':::STRING)) OR (table3.col3_8 > '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (table3.col3_7 <= '3000-01-01 00:00:00':::TIMESTAMP)) OR (table3.col3_14 < (-9223372036854775807):::INT8)) OR (table3.col3_18 <= '1970-01-01':::DATE)) OR (table3.col1_0 < '00:00:00':::TIME)) OR (table3.col3_15 >= '24:00:00':::TIME)) AND (table3.col3_13 >= e'\U00002603':::STRING)) OR (table3.col1_2 < '\xff':::BYTES), INDEX (col1_0 DESC, col1_3 DESC) STORING (col3_6, col3_8, col3_12) WHERE ((((((((((table3.col3_16 >= e'\U00002603':::STRING) OR (table3.col3_14 != 127:::INT8)) AND (table3.col1_2 < '\x':::BYTES)) AND (table3.col3_18 > '4714-11-24 BC':::DATE)) OR (table3.col3_7 >= '3000-01-01 00:00:00':::TIMESTAMP)) OR (table3.col3_13 < 'X':::STRING)) OR (table3.col3_15 >= '24:00:00':::TIME)) OR (table3.col1_1 = '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table3.col1_0 = '24:00:00':::TIME)) OR (table3.col3_8 < '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table3.col1_3 = e'\'':::STRING), INDEX (col1_3 ASC, col3_7, col3_12 DESC, col3_18 DESC, col1_2 DESC) STORING (col3_8, col3_10, col3_11, col3_13, col3_16) WHERE ((((((((((table3.col3_16 = e'\'':::STRING) AND (table3.col3_13 <= e'\U00002603':::STRING)) AND (table3.col3_7 != '-2000-01-01 00:00:00':::TIMESTAMP)) OR (table3.col1_3 >= e'\x00':::STRING)) AND (table3.col1_0 != '24:00:00':::TIME)) AND (table3.col3_18 > 'infinity':::DATE)) AND (table3.col1_1 < '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) OR (table3.col3_15 >= '00:00:00':::TIME)) AND (table3.col1_2 < '\xe29883':::BYTES)) OR (table3.col3_8 >= '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (table3.col3_14 <= (-32768):::INT8), INDEX (col3_18 ASC, col3_11 DESC, col3_8 ASC, col3_17, col1_2, col1_6 DESC, col3_9 ASC, col1_0 DESC, col3_10 ASC, col3_6, col3_15 ASC, col3_12, col1_3 DESC) WHERE ((table3.col1_0 != '24:00:00':::TIME) AND (table3.col3_14 >= (-128):::INT8)) OR (table3.col3_8 < '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ), INDEX (col3_18, col1_2, col3_15 ASC, col1_6 ASC, col3_16, col3_13, col1_3 ASC, col1_0 ASC, col3_14 ASC, col3_7 DESC, col3_11 ASC, col3_8 DESC, col3_12) STORING (col3_10, col3_17) WHERE ((((((((((table3.col3_7 >= '-2000-01-01 00:00:00':::TIMESTAMP) AND (table3.col1_1 != '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (table3.col3_8 = '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table3.col3_18 <= '4714-11-24 BC':::DATE)) AND (table3.col1_3 != e'\U00002603':::STRING)) AND (table3.col3_14 >= (-9223372036854775807):::INT8)) OR (table3.col3_13 = 'X':::STRING)) OR (table3.col3_16 >= e'\x00':::STRING)) AND (table3.col1_2 >= '\x58':::BYTES)) AND (table3.col3_15 < '24:00:00':::TIME)) OR (table3.col1_0 != '24:00:00':::TIME), FAMILY (col1_7), FAMILY (col3_6, col3_12, col3_15, col1_3), FAMILY (col3_7, col3_18), FAMILY (col1_0), FAMILY (col3_10, col1_6), FAMILY (col3_13), FAMILY (col3_14), FAMILY (col1_2), FAMILY (col3_9), FAMILY (col3_17), FAMILY (col3_16), FAMILY (col3_11, col1_1), FAMILY (col3_8)) INTERLEAVE IN PARENT table1 (col1_3, col1_7, col1_2, col1_0, col1_1, col1_6);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 573331, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 560599, "histo_buckets": [{"distinct_range": 0, "num_eq": 5590483145246831566, "num_range": 0, "upper_bound": "-63 years -2 mons -737 days -02:44:40.42645"}, {"distinct_range": 1255675173263688700, "num_eq": 6000000000, "num_range": 1255675173263688689, "upper_bound": "4 years 10 mons 402 days 08:31:55.30698"}, {"distinct_range": 0, "num_eq": 4000000, "num_range": 5139727588341763775, "upper_bound": "44 years 10 mons 872 days 00:19:38.914257"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 157255, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "12:17:49.293298"}, {"distinct_range": 40000, "num_eq": 4564384687381271133, "num_range": 40000, "upper_bound": "17:40:57.831317"}, {"distinct_range": 0, "num_eq": 400000, "num_range": 70, "upper_bound": "19:42:20.216911"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 1014342163288421197, "upper_bound": "22:35:25.099421"}, {"distinct_range": 0, "num_eq": 10, "num_range": 10, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 235961, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000, "num_range": 0, "upper_bound": "0001-01-01 00:00:00+00:00"}, {"distinct_range": 9000000000, "num_eq": 10000, "num_range": 9000000000, "upper_bound": "1970-05-05 02:56:36.000331+00:00"}, {"distinct_range": 0, "num_eq": 200, "num_range": 100000000000, "upper_bound": "1977-03-05 13:06:53.000292+00:00"}, {"distinct_range": 300, "num_eq": 1373105974386741788, "num_range": 300, "upper_bound": "1978-09-07 21:36:54.00056+00:00"}, {"distinct_range": 0, "num_eq": 9000000, "num_range": 3000000, "upper_bound": "2000-06-19 13:49:19.000562+00:00"}, {"distinct_range": 0, "num_eq": 500000, "num_range": 0, "upper_bound": "2000-10-07 04:09:05.000156+00:00"}, {"distinct_range": 4000, "num_eq": 300000000, "num_range": 4000, "upper_bound": "2010-09-18 16:55:23.000562+00:00"}, {"distinct_range": 0, "num_eq": 3012495730663853660, "num_range": 100, "upper_bound": "2017-09-10 11:40:08.000276+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 372164, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 251049, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000, "num_range": 0, "upper_bound": "]\\u001d"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 271436, "histo_col_type": "", "name": "__auto__", "null_count": 92480, "row_count": 600000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 250277, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3807845473427356943, "histo_col_type": "", "name": "__auto__", "null_count": 82555695190049589, "row_count": 8571238728128130130}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3889124496062029410, "histo_buckets": [{"distinct_range": 0, "num_eq": 7395147135880983652, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 3522160519487620039, "num_range": 8360714390181658307, "upper_bound": "?\\u000b\\u0006\\u0015"}, {"distinct_range": 464187.60492968676, "num_eq": 6697255166151128410, "num_range": 1000000, "upper_bound": "C\\u0010P\\u000f"}, {"distinct_range": 0, "num_eq": 9000, "num_range": 100000000, "upper_bound": "I.\\\\*h"}, {"distinct_range": 1094680359130415900, "num_eq": 800000000, "num_range": 4899003010665235557, "upper_bound": "R\\u000cON1"}, {"distinct_range": 1957122751981858800, "num_eq": 8540723713721564893, "num_range": 2813655043700018373, "upper_bound": "h"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 8571238728128130130}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3298693351911429913, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8571238728128130130}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7850268582800907718, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 8000000, "num_eq": 400, "num_range": 8000000, "upper_bound": "153689497"}, {"distinct_range": 7031346024884151000, "num_eq": 1030102955298230895, "num_range": 7031346024884150839, "upper_bound": "485701558"}, {"distinct_range": 34063225629.90314, "num_eq": 3000000000, "num_range": 40000000000, "upper_bound": "2288090138"}, {"distinct_range": 1459816.7642494275, "num_eq": 7000, "num_range": 5000000, "upper_bound": "2314776485"}, {"distinct_range": 6344729226671277000, "num_eq": 9000000000, "num_range": 6344729226671276631, "upper_bound": "4211756774"}, {"distinct_range": 0, "num_eq": 60, "num_range": 0, "upper_bound": "4232572152"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 4045657862668590409, "row_count": 8571238728128130130}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5356426550636819219, "histo_col_type": "STRING", "name": "__auto__", "null_count": 104218423056095092, "row_count": 8571238728128130130}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1093711078715518877, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8571238728128130130}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7819028163351156708, "histo_col_type": "", "name": "__auto__", "null_count": 7118666754697533433, "row_count": 8571238728128130130}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5443373356614540166, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8571238728128130130}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5771186105237049954, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8571238728128130130}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5568158635856492511, "histo_col_type": "", "name": "__auto__", "null_count": 163347229206166338, "row_count": 8571238728128130130}, {"columns": ["col2_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 508184896056924622, "histo_col_type": "", "name": "__auto__", "null_count": 3862204316959503308, "row_count": 8571238728128130130}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5060473713728164921, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8571238728128130130}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7658533430641448754, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8571238728128130130}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3474832790475741437, "histo_col_type": "OID", "name": "__auto__", "null_count": 2887932249130065873, "row_count": 8571238728128130130}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1281307121348625476, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8571238728128130130}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6210264266287365556, "histo_col_type": "", "name": "__auto__", "null_count": 1191022700148947380, "row_count": 8571238728128130130}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5329656589209755977, "histo_col_type": "", "name": "__auto__", "null_count": 851016465951688639, "row_count": 8571238728128130130}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 537378925927628187, "histo_buckets": [{"distinct_range": 0, "num_eq": 4783917580899011827, "num_range": 0, "upper_bound": "33369709641724962"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 6602107449252920711, "row_count": 8571238728128130130}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4041251518238645456, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000, "num_range": 0, "upper_bound": "-17730"}, {"distinct_range": 8890518082.16548, "num_eq": 500, "num_range": 40000000000, "upper_bound": "-17208"}, {"distinct_range": 231165526866456350, "num_eq": 5664350197077551020, "num_range": 2328020224962464103, "upper_bound": "27122"}, {"distinct_range": 0, "num_eq": 7499292641025268122, "num_range": 2000000000, "upper_bound": "28054"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 5668741670616155598, "row_count": 8571238728128130130}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8552063230374131504, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "\\\\x08af"}, {"distinct_range": 448.71734383988706, "num_eq": 601679939587571461, "num_range": 500, "upper_bound": "\\\\x22"}, {"distinct_range": 0, "num_eq": 300, "num_range": 7763902361781288258, "upper_bound": "\\\\x765b364ce16f5716"}, {"distinct_range": 1000000000, "num_eq": 40000, "num_range": 1000000000, "upper_bound": "\\\\xd9f23291de"}, {"distinct_range": 80000, "num_eq": 700000000, "num_range": 80000, "upper_bound": "\\\\xdd61c5e8f69b830b"}, {"distinct_range": 0, "num_eq": 8000, "num_range": 300, "upper_bound": "\\\\xf3141107c3"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 8571238728128130130}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 515866615624591081, "histo_buckets": [{"distinct_range": 0, "num_eq": 4974627389210662526, "num_range": 0, "upper_bound": "-2000-01-01 00:00:00+00:00"}, {"distinct_range": 0, "num_eq": 5527226957845980731, "num_range": 8902335278842738355, "upper_bound": "0001-01-01 00:00:00+00:00"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 6099514758223278268, "upper_bound": "1973-07-22 11:20:02.00013+00:00"}, {"distinct_range": 836317310378850800, "num_eq": 2000, "num_range": 836317310378850781, "upper_bound": "1994-11-18 13:14:09.000606+00:00"}, {"distinct_range": 200, "num_eq": 8066832305146178331, "num_range": 200, "upper_bound": "1999-05-06 12:49:58.000329+00:00"}, {"distinct_range": 0, "num_eq": 7744948441284209285, "num_range": 1318588613286974274, "upper_bound": "2019-01-17 18:59:47.00082+00:00"}, {"distinct_range": 0, "num_eq": 4713398806956232170, "num_range": 2354261630641066059, "upper_bound": "2021-11-26 14:15:26.000733+00:00"}, {"distinct_range": 90, "num_eq": 4127016338176917238, "num_range": 90, "upper_bound": "2022-03-13 17:07:09.000571+00:00"}, {"distinct_range": 80000000, "num_eq": 1469026426679328746, "num_range": 80000000, "upper_bound": "2028-03-13 04:25:50.000484+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 8571238728128130130}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7294458193390853941, "histo_buckets": [{"distinct_range": 0, "num_eq": 40, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 10000, "num_eq": 1051793623441558159, "num_range": 10000, "upper_bound": "236782027"}, {"distinct_range": 0, "num_eq": 814432638427110164, "num_range": 8632532365998141693, "upper_bound": "305831096"}, {"distinct_range": 4064664646045413400, "num_eq": 543679654344633485, "num_range": 6907412297631862646, "upper_bound": "595282727"}, {"distinct_range": 7877835785371613000, "num_eq": 0, "num_range": 7877835785371612793, "upper_bound": "1735304500"}, {"distinct_range": 7111.59793552079, "num_eq": 2722622874867194659, "num_range": 8000, "upper_bound": "2242286618"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 172441437685785633, "row_count": 8571238728128130130}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5184774965386464579, "histo_buckets": [{"distinct_range": 0, "num_eq": 8509076346969619912, "num_range": 0, "upper_bound": "02:05:49.134719"}, {"distinct_range": 1453975067314194700, "num_eq": 10000, "num_range": 1453975067314194735, "upper_bound": "08:22:07.542533"}, {"distinct_range": 83636.3667753091, "num_eq": 288548224757226354, "num_range": 900000, "upper_bound": "09:14:30.712641"}, {"distinct_range": 82089224.81264375, "num_eq": 725559403245501188, "num_range": 90000000, "upper_bound": "12:17:28.221088"}, {"distinct_range": 641003335.467503, "num_eq": 90000000000, "num_range": 800000000, "upper_bound": "18:23:57.957551"}, {"distinct_range": 2729.185165809116, "num_eq": 200000000, "num_range": 3000, "upper_bound": "19:49:19.988695"}, {"distinct_range": 3344.334776776309, "num_eq": 2284260022259082920, "num_range": 30000, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 3939124625896793614, "row_count": 8571238728128130130}, {"columns": ["col2_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1042298219668195717, "histo_col_type": "", "name": "__auto__", "null_count": 2776999882451110577, "row_count": 8571238728128130130}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 334591486, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "-55 years -7 mons -619 days -21:35:22.243436"}, {"distinct_range": 91818082.30840869, "num_eq": 90000000, "num_range": 4000000000, "upper_bound": "-24 years -11 mons -906 days -17:55:37.741833"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 5001112629142757744, "upper_bound": "00:00:00"}, {"distinct_range": 0, "num_eq": 9110120242261776987, "num_range": 90000000000, "upper_bound": "1 year 3 mons 311 days 05:58:53.066649"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 650835480, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 779635071, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 794204829, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8124157, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 701203164, "histo_col_type": "", "name": "__auto__", "null_count": 203964862, "row_count": 800000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 413655410, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 107798691, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 653548775, "histo_buckets": [{"distinct_range": 0, "num_eq": 1681172233626105608, "num_range": 0, "upper_bound": "-976446786547120702"}, {"distinct_range": 1053075179644724200, "num_eq": 500000000, "num_range": 1053075179644724182, "upper_bound": "4399560499771435250"}, {"distinct_range": 0, "num_eq": 4000000, "num_range": 4652547575208462707, "upper_bound": "5005202463216062334"}, {"distinct_range": 0, "num_eq": 2441714151771160048, "num_range": 24503942029936038, "upper_bound": "5375904445338990817"}, {"distinct_range": 8146404893400601000, "num_eq": 853261653230929911, "num_range": 8146404893400600511, "upper_bound": "7618215854851149369"}, {"distinct_range": 0, "num_eq": 3121845656185056758, "num_range": 773210506130964542, "upper_bound": "8100687232056886931"}, {"distinct_range": 0, "num_eq": 4534741405058897407, "num_range": 764655398131123196, "upper_bound": "8649398297152461605"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 664996315, "histo_buckets": [{"distinct_range": 0, "num_eq": 6265095625676523537, "num_range": 0, "upper_bound": "\\u00165"}, {"distinct_range": 30000000, "num_eq": 4114039430350953499, "num_range": 30000000, "upper_bound": "\\""}, {"distinct_range": 14366434.432569439, "num_eq": 80000000000, "num_range": 40000000, "upper_bound": "#"}, {"distinct_range": 657697220598574500, "num_eq": 60000000000, "num_range": 657697220598574448, "upper_bound": ":\\u0015"}, {"distinct_range": 42008.326342473025, "num_eq": 200000000, "num_range": 100000, "upper_bound": "[\\u0013q+n"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 673360794, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 622909914, "histo_col_type": "", "name": "__auto__", "null_count": 670246238, "row_count": 800000000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 722239360, "histo_col_type": "", "name": "__auto__", "null_count": 30257094, "row_count": 800000000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 420820614, "histo_col_type": "", "name": "__auto__", "null_count": 450847317, "row_count": 800000000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 642380348, "histo_buckets": [{"distinct_range": 0, "num_eq": 1717048843047712652, "num_range": 0, "upper_bound": "03:40:57.522128"}, {"distinct_range": 600000000, "num_eq": 0, "num_range": 600000000, "upper_bound": "16:33:14.458223"}, {"distinct_range": 50000000000, "num_eq": 1596151006296167794, "num_range": 50000000000, "upper_bound": "18:48:06.845565"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 454070323, "row_count": 800000000}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 673594014, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 124483069, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000000, "num_range": 0, "upper_bound": "1970-01-01"}, {"distinct_range": 0, "num_eq": 4035433864267171667, "num_range": 191273162945289616, "upper_bound": "1972-05-09"}, {"distinct_range": 34115865.38982884, "num_eq": 6127448210907215281, "num_range": 60000000, "upper_bound": "1973-02-12"}, {"distinct_range": 834.0790847301184, "num_eq": 3063658292574748420, "num_range": 9000, "upper_bound": "1975-01-15"}, {"distinct_range": 8000, "num_eq": 0, "num_range": 8000, "upper_bound": "1980-11-11"}, {"distinct_range": 700000000, "num_eq": 8000000, "num_range": 700000000, "upper_bound": "1984-09-19"}, {"distinct_range": 2713.5736812373007, "num_eq": 6587930574751117756, "num_range": 6000, "upper_bound": "1989-11-23"}, {"distinct_range": 1565457552852541000, "num_eq": 100000, "num_range": 5556249481514712897, "upper_bound": "1993-03-25"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 251115110, "row_count": 800000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 141253805, "histo_buckets": [{"distinct_range": 0, "num_eq": 4444797191237839699, "num_range": 0, "upper_bound": "04:47:04.722424"}, {"distinct_range": 0, "num_eq": 20000000, "num_range": 8443285091116820690, "upper_bound": "10:36:11.648763"}, {"distinct_range": 6000, "num_eq": 90000000000, "num_range": 6000, "upper_bound": "11:57:16.739722"}, {"distinct_range": 100, "num_eq": 500, "num_range": 100, "upper_bound": "15:36:39.185995"}, {"distinct_range": 603702785482773200, "num_eq": 100000, "num_range": 603702785482773310, "upper_bound": "21:45:02.74425"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 69757742, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000, "num_range": 0, "upper_bound": "&"}, {"distinct_range": 5629751748014307000, "num_eq": 5652002853714197288, "num_range": 5629751748014306927, "upper_bound": "6"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 800000000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('g', 'gaovmw', 'fclxu');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('gfxl', 'pkge', 'gjfzs', 'i');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('v', 'oh');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('zr', 'jskub', 'uqhayq', 'udsy', 'xf');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('l');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2651622&tab=buildLog) on [master@0e98670c8fcec566937e899fdf77d2a68c702d62](https://github.com/cockroachdb/cockroach/commits/0e98670c8fcec566937e899fdf77d2a68c702d62):

```
        test_1        |             CREATE INDEX ON table2 (col2_6 ASC, col2_2 ASC, col2_4, col2_3 DESC) STORING (col2_1, col2_5, col2_11);
        test_1        |             CREATE INDEX ON table2 (col2_7 ASC, col2_5 ASC);
        test_1        |             CREATE INDEX ON table2 (col2_8 ASC, col2_6 DESC, col2_9 DESC) STORING (col2_1, col2_3, col2_5, col2_7, col2_11);
        test_1        |             CREATE INDEX ON table2 (col2_0, col2_1 ASC, col2_7 DESC, col2_9 DESC, col2_10 ASC, col2_3 ASC, col2_5 ASC, col2_11) STORING (col2_6);
        test_1        |             CREATE INDEX ON table2 (col2_8 ASC, col2_3) STORING (col2_4, col2_5, col2_6, col2_9);
        test_1        |             CREATE TABLE table3 (col1_4 TIMESTAMPTZ NOT NULL, col1_3 BOOL NOT NULL, col1_5 REGNAMESPACE NOT NULL, col1_0 NAME NOT NULL, col3_4 TIMESTAMPTZ NOT NULL, col3_5 REGCLASS NULL, col3_6 REGPROCEDURE NULL, col3_7 UUID NOT NULL, col3_8 REGPROCEDURE NOT NULL, col3_9 REGTYPE, col3_10 OID, col3_11 DATE NULL, col3_12 BYTES, PRIMARY KEY (col1_4, col1_3, col1_5, col1_0, col3_9, col3_7, col3_4, col3_11, col3_12), FAMILY (col3_9, col1_4), FAMILY (col3_10), FAMILY (col1_0, col1_5), FAMILY (col3_6), FAMILY (col3_4, col3_12), FAMILY (col3_5, col3_7, col1_3), FAMILY (col3_11), FAMILY (col3_8)) INTERLEAVE IN PARENT table1 (col1_4, col1_3, col1_5, col1_0);
        test_1        |             CREATE INDEX ON table3 (col3_6 ASC, col1_0 ASC, col1_5, col3_12, col3_5 DESC, col3_10 DESC, col1_3 ASC, col3_7 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_5 DESC, col3_4 DESC, col3_12, col1_0 ASC, col3_10, col1_4 ASC) STORING (col3_8);
        test_1        |             CREATE INDEX ON table3 (col3_10, col3_7 ASC, col1_0, col3_5 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_11) STORING (col3_5, col3_8, col3_10);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col1_5 DESC, col3_5, col3_12 ASC, col3_7 DESC, col3_9, col3_8 ASC, col3_11 ASC, col1_3 ASC, col3_10 DESC, col1_0, col3_6 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_6 DESC, col3_9 DESC, col3_11 DESC, col3_12 ASC, col1_0, col1_5) STORING (col3_5);
        test_1        |             CREATE INDEX ON table3 (col3_6 DESC, col3_5 ASC, col1_5 ASC, col1_0) STORING (col3_8, col3_10);
        test_1        |             CREATE INDEX ON table3 (col1_3 ASC, col1_4 ASC, col3_8 ASC, col3_4 DESC, col3_10 ASC, col1_0 DESC, col3_9 ASC, col3_6 DESC, col3_12 DESC, col3_11 DESC, col1_5 ASC, col3_5 ASC, col3_7 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_6 DESC, col3_8 DESC, col3_5 ASC, col1_3 DESC, col1_5 ASC);
        test_1        |             CREATE TABLE table4 (col4_0 VARCHAR NOT NULL, col4_1 UUID NOT NULL, col4_2 "char", col4_3 REGTYPE NOT NULL, col4_4 DECIMAL NULL, col4_5 STRING NOT NULL, col4_6 INT2, col4_7 OID NOT NULL, col4_8 FLOAT4 NOT NULL, col4_9 TIMETZ[], PRIMARY KEY (col4_8, col4_3, col4_1), FAMILY (col4_8), FAMILY (col4_2), FAMILY (col4_6), FAMILY (col4_1, col4_3, col4_4), FAMILY (col4_0), FAMILY (col4_7, col4_9), FAMILY (col4_5));
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_5 ASC, col4_6 ASC) STORING (col4_2, col4_4);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_2 ASC, col4_1 ASC, col4_8, col4_3 DESC, col4_4);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_7, col4_1 ASC, col4_5, col4_3 ASC) STORING (col4_4, col4_6);
        test_1        |             CREATE INDEX ON table4 (col4_0 DESC);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 448744095691528097, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 1000000, "num_eq": 8000, "num_range": 1000000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 1677291323235284787}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 432652683838440669, "histo_buckets": [{"distinct_range": 0, "num_eq": 30, "num_range": 0, "upper_bound": "1970-04-10 19:20:51.000114+00:00"}, {"distinct_range": 0, "num_eq": 5331459971024270761, "num_range": 80, "upper_bound": "1974-05-17 00:37:25.000483+00:00"}, {"distinct_range": 109496494138702080, "num_eq": 3184698255257811648, "num_range": 653418608142161864, "upper_bound": "1978-08-15 05:05:06.000386+00:00"}, {"distinct_range": 50000000000, "num_eq": 400000000, "num_range": 50000000000, "upper_bound": "1992-11-02 07:59:23.000861+00:00"}, {"distinct_range": 238229015468677440, "num_eq": 168227974146463458, "num_range": 238229015468677454, "upper_bound": "1997-02-17 03:17:34.000718+00:00"}, {"distinct_range": 8271151037083227000, "num_eq": 2751218444949473143, "num_range": 8271151037083227360, "upper_bound": "2010-03-24 02:28:38.000553+00:00"}, {"distinct_range": 100000, "num_eq": 10000, "num_range": 100000, "upper_bound": "2024-09-16 03:43:27.000218+00:00"}, {"distinct_range": 727614189731428000, "num_eq": 1000000000, "num_range": 2970379772073232710, "upper_bound": "2031-11-08 14:42:43.000794+00:00"}, {"distinct_range": 0, "num_eq": 8632268798653659742, "num_range": 10000000000, "upper_bound": "294276-12-31 23:59:59.999999+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 1677291323235284787}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1573723811052104218, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1677291323235284787}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 783318601278401382, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 1.4330858738340608, "num_eq": 700000000, "num_range": 30, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 552938327332120957, "row_count": 1677291323235284787}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 347844669053438896, "histo_buckets": [{"distinct_range": 0, "num_eq": 5326260151196241765, "num_range": 0, "upper_bound": ""}, {"distinct_range": 50455150.39328714, "num_eq": 7413664098121390886, "num_range": 300000000, "upper_bound": "\\tAwm\\u0008W"}, {"distinct_range": 0, "num_eq": 8137269167883553877, "num_range": 70, "upper_bound": "\\r>\\u0015\\u0006"}, {"distinct_range": 0, "num_eq": 900, "num_range": 2373713611499494569, "upper_bound": "\\u000f"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 40000, "upper_bound": ";dM~\\u0016\\u001bNJv"}, {"distinct_range": 3470915772.7585716, "num_eq": 900000, "num_range": 7000000000, "upper_bound": "FY"}, {"distinct_range": 0, "num_eq": 900000, "num_range": 6170969553234908027, "upper_bound": "Pg"}, {"distinct_range": 20000000000, "num_eq": 60, "num_range": 20000000000, "upper_bound": "VnF"}, {"distinct_range": 459850934147956030, "num_eq": 5629946996132852537, "num_range": 459850934147956038, "upper_bound": "d*\\u0014X\\u0005De3"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 1677291323235284787}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1122082382208407686, "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 1318990436405600566, "row_count": 1677291323235284787}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 553214615529243855, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000000, "num_range": 0, "upper_bound": "667927526"}, {"distinct_range": 0, "num_eq": 7602490992954609984, "num_range": 0, "upper_bound": "1217254310"}, {"distinct_range": 80000000000, "num_eq": 9127337364064369468, "num_range": 80000000000, "upper_bound": "1289456111"}, {"distinct_range": 0, "num_eq": 4839431800149165323, "num_range": 3000000, "upper_bound": "2745703517"}, {"distinct_range": 7288660140.857782, "num_eq": 1000, "num_range": 8000000000, "upper_bound": "3634838573"}, {"distinct_range": 295147947178656960, "num_eq": 70000000, "num_range": 2791776005005788516, "upper_bound": "3791584347"}, {"distinct_range": 0, "num_eq": 2448569804275313994, "num_range": 400000000, "upper_bound": "3848373533"}, {"distinct_range": 40000, "num_eq": 0, "num_range": 40000, "upper_bound": "3867640008"}, {"distinct_range": 8902050556456572000, "num_eq": 60000, "num_range": 8902050556456572132, "upper_bound": "4190134075"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 1406133407007283259, "row_count": 1677291323235284787}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "15:50:48.842978+05:37:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "-0.6853147149085999"}, {"distinct_range": 0, "num_eq": 5355780343751009636, "num_range": 3056515409355433130, "upper_bound": "0.1477014571428299"}, {"distinct_range": 4269274.432143122, "num_eq": 6017782001318048864, "num_range": 10000000, "upper_bound": "0.21425770223140717"}, {"distinct_range": 40000000, "num_eq": 2000000, "num_range": 40000000, "upper_bound": "0.5912001729011536"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "10:06:36.543958"}, {"distinct_range": 0, "num_eq": 4011567656962732115, "num_range": 6000000, "upper_bound": "10:11:54.906904"}, {"distinct_range": 385.01817724934244, "num_eq": 3739364871685609035, "num_range": 8000, "upper_bound": "13:49:20.375475"}, {"distinct_range": 157087711839489800, "num_eq": 5657971532167695570, "num_range": 1376737155666762588, "upper_bound": "18:47:47.500789"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 7952076540741735135, "num_range": 0, "upper_bound": "07:38:04.296316"}, {"distinct_range": 0, "num_eq": 0, "num_range": 4816173336007199359, "upper_bound": "08:34:42.34987"}, {"distinct_range": 35780.04986790805, "num_eq": 50000000, "num_range": 40000, "upper_bound": "08:57:17.494225"}, {"distinct_range": 719981504.2690027, "num_eq": 8012973505339272476, "num_range": 10000000000, "upper_bound": "13:41:43.378353"}, {"distinct_range": 1000, "num_eq": 50, "num_range": 1000, "upper_bound": "19:33:47.616882"}, {"distinct_range": 0, "num_eq": 90, "num_range": 1000000000, "upper_bound": "21:54:07.574872"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 30000000000, "upper_bound": "23:08:25.82282"}, {"distinct_range": 35687629.12796215, "num_eq": 5370639048564416145, "num_range": 80000000, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": "08:55:58.335695+06:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 89717821325543835, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 227814416554873368}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 204386332879137778, "histo_col_type": "", "name": "__auto__", "null_count": 58188988702476105, "row_count": 227814416554873368}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 106829726575190413, "histo_buckets": [{"distinct_range": 0, "num_eq": 6645462571717555767, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 400000000, "upper_bound": "2138349382"}, {"distinct_range": 2312473243572028400, "num_eq": 8809558006323837380, "num_range": 2312473243572028270, "upper_bound": "3581288758"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 196259022253600756, "row_count": 227814416554873368}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 216902740668628978, "histo_col_type": "", "name": "__auto__", "null_count": 205468753801029561, "row_count": 227814416554873368}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 163302439558842234, "histo_buckets": [{"distinct_range": 0, "num_eq": 4917291158390260127, "num_range": 0, "upper_bound": "1970-07-07 05:44:51.000065+00:00"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 1000000, "upper_bound": "1979-03-11 11:11:59.000845+00:00"}, {"distinct_range": 5402491227751381000, "num_eq": 5373062099771122313, "num_range": 5718330481738957689, "upper_bound": "1981-12-12 22:25:58.000036+00:00"}, {"distinct_range": 80000000, "num_eq": 4566525809169103936, "num_range": 80000000, "upper_bound": "1992-11-01 02:34:11.000641+00:00"}, {"distinct_range": 7409969108674474000, "num_eq": 30000, "num_range": 7409969108674473994, "upper_bound": "2027-08-11 23:42:24.000996+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 227814416554873368}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 101889335193752064, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 227814416554873368}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 164943016803233087, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 227814416554873368}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32185567625767124, "histo_buckets": [{"distinct_range": 0, "num_eq": 4606583803784158684, "num_range": 0, "upper_bound": "213821010"}, {"distinct_range": 3702774742212037000, "num_eq": 70, "num_range": 3702774742212037091, "upper_bound": "1707998280"}, {"distinct_range": 1179299722391554800, "num_eq": 300, "num_range": 1179299722391554740, "upper_bound": "3014436679"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 191063708570909535, "row_count": 227814416554873368}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 188343807887271437, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "1971-02-07"}, {"distinct_range": 114553613847144740, "num_eq": 70000000000, "num_range": 179702257060713933, "upper_bound": "1982-07-27"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 185213925674653368, "row_count": 227814416554873368}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19645755188976753, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "false"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 227814416554873368}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 85274230068867827, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "3080961268"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 227814416554873368}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 84991201471335752, "histo_buckets": [{"distinct_range": 0, "num_eq": 400, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 3412550392369931000, "num_eq": 0, "num_range": 3412550392369930603, "upper_bound": "374300959"}, {"distinct_range": 0, "num_eq": 500000000, "num_range": 80000000000, "upper_bound": "2134215444"}, {"distinct_range": 3321329710.033676, "num_eq": 20, "num_range": 10000000000, "upper_bound": "3323754261"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 100000000000, "upper_bound": "4219046473"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 158238472482112357, "row_count": 227814416554873368}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 74366815358230169, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 227814416554873368}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1566143, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000000, "num_range": 0, "upper_bound": "\\u0002"}, {"distinct_range": 2075811445732112000, "num_eq": 30000, "num_range": 3123751024652586447, "upper_bound": "\\""}, {"distinct_range": 40000, "num_eq": 8234432178064835684, "num_range": 40000, "upper_bound": "e"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 6000000}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3206415, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2152801, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "k"}, {"distinct_range": 2927119662907398700, "num_eq": 50000, "num_range": 4593635143722987486, "upper_bound": "s"}, {"distinct_range": 73531930.83423291, "num_eq": 8402122524478549038, "num_range": 300000000, "upper_bound": "x"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 4139448, "row_count": 6000000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1586017, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2395749, "histo_col_type": "", "name": "__auto__", "null_count": 1374042, "row_count": 6000000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5689151, "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 6000000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1124113, "histo_col_type": "", "name": "__auto__", "null_count": 4248711, "row_count": 6000000}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5475472, "histo_col_type": "", "name": "__auto__", "null_count": 3444618, "row_count": 6000000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1800836, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "1097727940"}, {"distinct_range": 0, "num_eq": 5000000000, "num_range": 6168985973089818789, "upper_bound": "1157509157"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 6000000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3798899, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": "-0.2995868921279907"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 6000000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('n', 'fo', 'jumj');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('ssyryr', 'jfphi', 'pbvs', 'xm', 'dyee');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        |     --- FAIL: TestCompare/mutators (0.27s)
        test_1        |         compare_test.go:129: 
        test_1        |             		SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;
        test_1        |             		SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;
        test_1        |             	CREATE TABLE table1 (col1_0 INET NOT NULL, col1_1 REGTYPE NOT NULL, col1_2 STRING NULL, col1_3 TIME NOT NULL, col1_4 OID NOT NULL, col1_5 BIT(13) NOT NULL, col1_6 BIT(2) NOT NULL, col1_7 NAME NOT NULL, col1_8 INT2 NOT NULL, col1_9 DECIMAL NOT NULL, col1_10 FLOAT4 NOT NULL, col1_11 TIMESTAMPTZ NOT NULL, col1_12 TIME NOT NULL, PRIMARY KEY (col1_0 DESC, col1_7, col1_8 ASC, col1_1 ASC, col1_3 DESC, col1_6, col1_12 ASC, col1_9, col1_11 DESC, col1_10 DESC, col1_5), INDEX (col1_9, col1_5) WHERE (((((((table1.col1_3 >= '00:00:00':::TIME) AND (table1.col1_8 > 0:::INT8)) AND (table1.col1_7 < e'\x00':::STRING)) OR (table1.col1_10 <= '+Inf':::FLOAT8)) AND (table1.col1_2 != '':::STRING)) AND (table1.col1_11 != '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (table1.col1_9 >= 1:::DECIMAL)) AND (table1.col1_12 > '00:00:00':::TIME), FAMILY (col1_7), FAMILY (col1_1), FAMILY (col1_5, col1_6), FAMILY (col1_8, col1_12), FAMILY (col1_4), FAMILY (col1_11), FAMILY (col1_9, col1_10, col1_0, col1_3), FAMILY (col1_2));
        test_1        |             CREATE TABLE table2 (col1_0 INET NOT NULL, col1_7 NAME NOT NULL, col1_8 INT2 NOT NULL, col1_1 REGTYPE NOT NULL, col1_3 TIME NOT NULL, col1_6 BIT(2) NOT NULL, col1_12 TIME NOT NULL, col1_9 DECIMAL NOT NULL, col1_11 TIMESTAMPTZ NOT NULL, col1_10 FLOAT4 NOT NULL, col1_5 BIT(13) NOT NULL, col2_11 DATE NOT NULL, col2_12 DECIMAL NULL, col2_13 REGPROCEDURE NULL, col2_14 INET NULL, col2_15 TIMESTAMP NOT NULL, col2_16 TIMETZ NULL, col2_17 TIMESTAMP, PRIMARY KEY (col1_0 DESC, col1_7, col1_8 ASC, col1_1 ASC, col1_3 DESC, col1_6, col1_12 ASC, col1_9, col1_11 DESC, col1_10 DESC, col1_5, col2_11, col2_16 ASC, col2_12 ASC, col2_15 ASC, col2_13 DESC), INDEX (col1_1 ASC, col1_12 DESC, col2_14, col1_10 DESC, col1_6 DESC, col2_12 DESC, col1_5 DESC, col2_17 ASC, col1_9, col1_3, col1_7, col1_0 ASC) WHERE (((((((((((table2.col2_11 > '4714-11-24 BC':::DATE) OR (table2.col2_17 <= '-4713-11-24 00:00:00':::TIMESTAMP)) AND (table2.col2_15 = '0001-01-01 00:00:00':::TIMESTAMP)) AND (table2.col1_12 = '00:00:00':::TIME)) AND (table2.col1_7 >= e'\'':::STRING)) OR (table2.col1_8 < 1:::INT8)) AND (table2.col1_3 >= '24:00:00':::TIME)) OR (table2.col1_11 = '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table2.col2_16 >= '24:00:00-15:59:00':::TIMETZ)) OR (table2.col1_10 = 'NaN':::FLOAT8)) OR (table2.col2_12 != 'Infinity':::DECIMAL)) AND (table2.col1_9 != 0:::DECIMAL)) INTERLEAVE IN PARENT table1 (col1_0, col1_7, col1_8, col1_1, col1_3, col1_6, col1_12, col1_9, col1_11, col1_10, col1_5);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1189, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6834, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000, "num_range": 0, "upper_bound": "231.82.166.37/2"}, {"distinct_range": 0, "num_eq": 3142756618207936644, "num_range": 800000, "upper_bound": "69.123.68.125/30"}, {"distinct_range": 20000, "num_eq": 5136009785066782114, "num_range": 20000, "upper_bound": "b87b:b47d:d693:68cd:5888:bc4f:2afe:ca08/10"}, {"distinct_range": 600000000, "num_eq": 6000000000, "num_range": 600000000, "upper_bound": "e32:3844:8a22:86d1:e:674c:7f06:6b3e/36"}, {"distinct_range": 2936720990974331000, "num_eq": 80000000000, "num_range": 6493211418778560894, "upper_bound": "994:d4fe:441a:2926:2b4e:8087:f4e3:507f/66"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6016, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4213, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2227, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5600, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5190, "histo_buckets": [{"distinct_range": 0, "num_eq": 8372208730220954313, "num_range": 0, "upper_bound": "-8.314250813919075488E+24"}, {"distinct_range": 323161231443483200, "num_eq": 100000, "num_range": 1275288054131931251, "upper_bound": "-2901747903353413.133"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5780, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8892, "histo_col_type": "", "name": "__auto__", "null_count": 1950, "row_count": 10000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8679, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3864, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4333, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1930, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3022, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1107, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1413, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3659, "histo_col_type": "", "name": "__auto__", "null_count": 4348, "row_count": 5000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4320, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2379, "histo_col_type": "", "name": "__auto__", "null_count": 3141, "row_count": 5000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 940, "histo_col_type": "", "name": "__auto__", "null_count": 4730, "row_count": 5000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1018, "histo_buckets": [{"distinct_range": 0, "num_eq": 8908442070575797984, "num_range": 0, "upper_bound": "117.254.114.158/3"}, {"distinct_range": 200000000, "num_eq": 90000, "num_range": 200000000, "upper_bound": "248.5.132.155/11"}, {"distinct_range": 0, "num_eq": 700, "num_range": 60000000000, "upper_bound": "145.189.131.51/22"}, {"distinct_range": 2690200503455499300, "num_eq": 8754220886226894067, "num_range": 3627871909430863631, "upper_bound": "38a:61c:f049:ce1c:efb8:14e5:f42c:ec49/89"}, {"distinct_range": 0, "num_eq": 3844768664672394992, "num_range": 0, "upper_bound": "b98c:a2b:e5c1:a01e:e678:3fe6:5dfc:71a0/109"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1281, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1248, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000000, "num_range": 0, "upper_bound": "1677332928"}, {"distinct_range": 0, "num_eq": 300, "num_range": 8435442437672948387, "upper_bound": "2379289033"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 1345094538371960054, "upper_bound": "2821687951"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2020, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4122, "histo_col_type": "", "name": "__auto__", "null_count": 2750, "row_count": 5000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2364, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4322, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4104, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4310, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4374, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1888, "histo_col_type": "", "name": "__auto__", "null_count": 1748, "row_count": 5000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('jm');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2653403&tab=buildLog) on [master@5b33e6dfc47000de831745a851e3bf9e2cf7fd95](https://github.com/cockroachdb/cockroach/commits/5b33e6dfc47000de831745a851e3bf9e2cf7fd95):

```
        test_1        |     --- FAIL: TestCompare/postgres (1.12s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             CREATE TABLE table1 (col1_0 CHAR NOT NULL, col1_1 CHAR[], col1_2 FLOAT8 NULL, col1_3 FLOAT4 NULL, col1_4 TIMESTAMPTZ NOT NULL, col1_5 REGPROCEDURE NOT NULL, col1_6 TIMESTAMPTZ NOT NULL, col1_7 TIMESTAMPTZ, col1_8 INET, col1_9 BYTES NOT NULL, col1_10 NAME NOT NULL, col1_11 VARCHAR NULL, col1_12 REGNAMESPACE, PRIMARY KEY (col1_10, col1_0, col1_9, col1_6, col1_4));
        test_1        |             CREATE INDEX ON table1 (col1_2, col1_3 DESC, col1_0) STORING (col1_5, col1_7);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_0 DESC, col1_10 DESC, col1_4 DESC, col1_11 DESC) STORING (col1_2, col1_3, col1_5, col1_7, col1_8);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_12) STORING (col1_1, col1_2, col1_7, col1_8);
        test_1        |             CREATE INDEX ON table1 (col1_10 ASC, col1_2 ASC, col1_12, col1_9 ASC, col1_6 ASC, col1_3 ASC, col1_8, col1_11 ASC, col1_5 DESC, col1_0 DESC, col1_4 DESC, col1_7 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_5, col1_10 ASC, col1_3 ASC, col1_8 ASC, col1_7 DESC, col1_0, col1_9 ASC, col1_11, col1_2 ASC, col1_12);
        test_1        |             CREATE INDEX ON table1 (col1_4, col1_7, col1_2 ASC) STORING (col1_1, col1_3, col1_8);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_6, col1_11 ASC, col1_5 ASC, col1_9 DESC, col1_8 ASC, col1_3 ASC, col1_0 DESC) STORING (col1_12);
        test_1        |             CREATE INDEX ON table1 (col1_3, col1_8 ASC);
        test_1        |             CREATE TABLE table2 (col1_10 NAME NOT NULL, col1_0 CHAR NOT NULL, col1_9 BYTES NOT NULL, col1_6 TIMESTAMPTZ NOT NULL, col1_4 TIMESTAMPTZ NOT NULL, col2_5 NAME NOT NULL, col2_6 REGNAMESPACE NULL, col2_7 INET NULL, col2_8 NAME NOT NULL, col2_9 INT8, col2_10 REGPROC NOT NULL, col2_11 DECIMAL NULL, PRIMARY KEY (col1_10, col1_0, col1_9, col1_6, col1_4, col2_9, col2_11, col2_8, col2_6, col2_7, col2_10), FAMILY (col1_6, col2_9), FAMILY (col2_10, col1_4), FAMILY (col2_11), FAMILY (col2_8), FAMILY (col2_6, col2_7), FAMILY (col2_5, col1_0), FAMILY (col1_9), FAMILY (col1_10)) INTERLEAVE IN PARENT table1 (col1_10, col1_0, col1_9, col1_6, col1_4);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_5 ASC, col2_11 ASC, col2_7 ASC, col2_6 ASC, col2_8 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_10 ASC, col2_6 ASC, col2_7 ASC, col2_9, col2_11 ASC) STORING (col2_5);
        test_1        |             CREATE TABLE table3 (col3_0 INET NOT NULL, col3_1 INT2, col3_2 BOX2D NOT NULL, col3_3 BOX2D, col3_4 REGCLASS NOT NULL, col3_5 BYTES, col3_6 DECIMAL NULL, col3_7 BOOL NOT NULL, col3_8 BOOL NOT NULL, col3_9 INT8 NOT NULL, col3_10 OID NOT NULL, PRIMARY KEY (col3_7, col3_9, col3_10, col3_8, col3_4));
        test_1        |             CREATE INDEX ON table3 (col3_5, col3_3);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_5 DESC, col3_6 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_3 DESC, col3_4 DESC, col3_6 DESC, col3_2 DESC, col3_8) STORING (col3_0, col3_1, col3_5);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 ASC, col3_7 DESC) STORING (col3_2, col3_3, col3_6);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_3, col3_5 ASC, col3_1 ASC, col3_6 ASC, col3_10, col3_8) STORING (col3_0, col3_2);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_2 ASC, col3_6 DESC) STORING (col3_0, col3_5);
        test_1        |             CREATE INDEX ON table3 (col3_4, col3_2, col3_7, col3_8, col3_3) STORING (col3_0, col3_1, col3_5, col3_6);
        test_1        |             CREATE TABLE table4 (col3_7 BOOL NOT NULL, col3_9 INT8 NOT NULL, col3_10 OID NOT NULL, col3_8 BOOL NOT NULL, col3_4 REGCLASS NOT NULL, col4_5 BYTES NOT NULL, col4_6 REGPROCEDURE NULL, col4_7 INT8 NOT NULL, col4_8 INT2 NOT NULL, col4_9 INET, col4_10 INT8 NOT NULL, col4_11 "char" NOT NULL, col4_12 BOX2D NULL, col4_13 INET NULL, col4_14 CHAR NOT NULL, PRIMARY KEY (col3_7, col3_9, col3_10, col3_8, col3_4, col4_12, col4_6, col4_13, col4_8, col4_11, col4_7, col4_9, col4_10), FAMILY (col3_7, col4_6), FAMILY (col3_10), FAMILY (col4_14, col4_11, col4_9), FAMILY (col4_7, col3_9, col3_8), FAMILY (col4_10), FAMILY (col3_4), FAMILY (col4_13), FAMILY (col4_12, col4_8), FAMILY (col4_5)) INTERLEAVE IN PARENT table3 (col3_7, col3_9, col3_10, col3_8, col3_4);
        test_1        |             CREATE INDEX ON table4 (col3_7, col3_4 DESC);
        test_1        |             CREATE INDEX ON table4 (col4_14 ASC, col4_6 DESC, col4_7 DESC, col3_4 ASC, col4_13 ASC, col4_10, col4_5, col4_12, col4_8, col3_7 DESC, col3_8, col3_10 DESC, col4_11 DESC, col3_9 DESC, col4_9 ASC);
        test_1        |             CREATE INDEX ON table4 (col4_10 DESC);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 214279015, "histo_col_type": "", "name": "__auto__", "null_count": 139255235, "row_count": 500000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53859005, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12906322, "histo_buckets": [{"distinct_range": 0, "num_eq": 90, "num_range": 0, "upper_bound": "\\t"}, {"distinct_range": 0, "num_eq": 0, "num_range": 2562520752440142772, "upper_bound": "\\u000c"}, {"distinct_range": 0, "num_eq": 3341050191655091164, "num_range": 6366760203832717253, "upper_bound": "$"}, {"distinct_range": 1920144175338712600, "num_eq": 80000, "num_range": 1920144175338712504, "upper_bound": "2"}, {"distinct_range": 900000000, "num_eq": 5000000, "num_range": 900000000, "upper_bound": "8"}, {"distinct_range": 4659376759712910000, "num_eq": 8814329177986219476, "num_range": 4659376759712910096, "upper_bound": "O"}, {"distinct_range": 300650790823536960, "num_eq": 1713338901415235136, "num_range": 300650790823536986, "upper_bound": "i"}, {"distinct_range": 0, "num_eq": 4156641004579449642, "num_range": 2744174733034350049, "upper_bound": "r"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 485814603, "histo_buckets": [{"distinct_range": 0, "num_eq": 479301801295261470, "num_range": 0, "upper_bound": "-0.10336146503686905"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 9106278, "row_count": 500000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 379749168, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000000, "num_range": 0, "upper_bound": "0"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8917750, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000000, "num_range": 0, "upper_bound": "1970-09-24 12:57:26.000831+00:00"}, {"distinct_range": 0, "num_eq": 0, "num_range": 10000000, "upper_bound": "1975-03-12 21:20:42.000666+00:00"}, {"distinct_range": 0, "num_eq": 500, "num_range": 6923448768189571152, "upper_bound": "1984-09-05 13:00:51.000574+00:00"}, {"distinct_range": 0, "num_eq": 20000000, "num_range": 70000000000, "upper_bound": "1990-03-08 23:25:48.000977+00:00"}, {"distinct_range": 0, "num_eq": 40000000000, "num_range": 0, "upper_bound": "2017-07-01 01:14:23.000624+00:00"}, {"distinct_range": 0, "num_eq": 600, "num_range": 2771863570664895291, "upper_bound": "2028-06-04 06:09:37.00052+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 142188523, "histo_col_type": "", "name": "__auto__", "null_count": 413394110, "row_count": 500000000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 200749124, "histo_buckets": [{"distinct_range": 0, "num_eq": 3447790843924106257, "num_range": 0, "upper_bound": "612341307"}, {"distinct_range": 0, "num_eq": 1782464160656286718, "num_range": 0, "upper_bound": "2233390730"}, {"distinct_range": 0, "num_eq": 100, "num_range": 7285892840411156773, "upper_bound": "2496828671"}, {"distinct_range": 898.809518492832, "num_eq": 0, "num_range": 1000, "upper_bound": "2544076175"}, {"distinct_range": 0, "num_eq": 26694498467824708, "num_range": 8182294006253767658, "upper_bound": "3887127864"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 81277590, "row_count": 500000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 200702549, "histo_col_type": "", "name": "__auto__", "null_count": 86208489, "row_count": 500000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 140513480, "histo_buckets": [{"distinct_range": 0, "num_eq": 5368822285675604009, "num_range": 0, "upper_bound": "-0.3019578823996033"}, {"distinct_range": 14.619495270485523, "num_eq": 200000, "num_range": 900, "upper_bound": "-0.20100067989321957"}, {"distinct_range": 0, "num_eq": 90000000000, "num_range": 0, "upper_bound": "0.20769570203783427"}, {"distinct_range": 0, "num_eq": 6945575215439522459, "num_range": 3000000000, "upper_bound": "0.46580402246778585"}, {"distinct_range": 0, "num_eq": 7237232774153289275, "num_range": 8958694721193437873, "upper_bound": "0.8220139281634192"}, {"distinct_range": 15781216.383510517, "num_eq": 10000000, "num_range": 60000000, "upper_bound": "1.0"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 700000000, "upper_bound": "1.1071665509411286"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 2000000000, "upper_bound": "1.3996889992970578"}, {"distinct_range": 2538648.6994608804, "num_eq": 20000, "num_range": 6000000, "upper_bound": "1.4363869079144642"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 54916179, "row_count": 500000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 157895439, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000, "num_range": 0, "upper_bound": "1980-09-22 14:18:57.000267+00:00"}, {"distinct_range": 0, "num_eq": 6500439839562171722, "num_range": 6000, "upper_bound": "1983-09-04 20:01:18.000382+00:00"}, {"distinct_range": 153320177.81498298, "num_eq": 8000000, "num_range": 600000000, "upper_bound": "2024-11-09 18:41:25.000178+00:00"}, {"distinct_range": 0, "num_eq": 3054448021629451664, "num_range": 0, "upper_bound": "2031-09-20 14:58:02.000507+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 207462868, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": "\\u0006"}, {"distinct_range": 3255667499116801000, "num_eq": 3569830122226462519, "num_range": 3255667499116800974, "upper_bound": "\\u000e"}, {"distinct_range": 0, "num_eq": 0, "num_range": 1000, "upper_bound": "-\\"lhCK:"}, {"distinct_range": 2569271.0513508706, "num_eq": 200, "num_range": 3000000, "upper_bound": "9"}, {"distinct_range": 4228703844981390300, "num_eq": 1303218033948183235, "num_range": 4228703844981390476, "upper_bound": "F\\u0003\\u00070"}, {"distinct_range": 2471790.2447195477, "num_eq": 1551238922952972792, "num_range": 9000000, "upper_bound": "[SFS3q"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 90144922, "histo_col_type": "", "name": "__auto__", "null_count": 424852821, "row_count": 500000000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 558556, "histo_col_type": "", "name": "__auto__", "null_count": 908850, "row_count": 1000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 616258, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34704, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 589035, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 806500, "histo_col_type": "", "name": "__auto__", "null_count": 982350, "row_count": 1000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 47257, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 249219, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1004305919"}, {"distinct_range": 895715454416407800, "num_eq": 1771060000627880677, "num_range": 895715454416407866, "upper_bound": "1935259677"}, {"distinct_range": 0, "num_eq": 40, "num_range": 30000000, "upper_bound": "2148649332"}, {"distinct_range": 0, "num_eq": 1415678149278849824, "num_range": 3000000000, "upper_bound": "2148984213"}, {"distinct_range": 80, "num_eq": 80, "num_range": 80, "upper_bound": "3077190078"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 1000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12129, "histo_buckets": [{"distinct_range": 0, "num_eq": 2671816447897157064, "num_range": 0, "upper_bound": "\\u00017"}, {"distinct_range": 1904312032527162400, "num_eq": 20, "num_range": 4883817358418211620, "upper_bound": "\\t.40\\u0008r|\\u0015"}, {"distinct_range": 533323963.10486907, "num_eq": 0, "num_range": 1000000000, "upper_bound": "\\u0018\\n\\u0003"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 6756423680167052920, "upper_bound": "%"}, {"distinct_range": 0, "num_eq": 200000, "num_range": 800000, "upper_bound": "\'"}, {"distinct_range": 37240.550389994816, "num_eq": 90000000, "num_range": 60000, "upper_bound": "F"}, {"distinct_range": 5908630165703887000, "num_eq": 3231800867433087775, "num_range": 5908630165703886748, "upper_bound": "P\\u001d"}, {"distinct_range": 0, "num_eq": 300000, "num_range": 2150747778560955331, "upper_bound": "Xiu\\nnN\\u0001 "}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 1000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27742, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 403939, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "\\u0006L"}, {"distinct_range": 2420647411863988000, "num_eq": 2948329072045778533, "num_range": 2708849528961737796, "upper_bound": "\\t.p\\u000e "}, {"distinct_range": 0, "num_eq": 694888111596663162, "num_range": 5000000, "upper_bound": "\\r..\'>`"}, {"distinct_range": 1847.887118034213, "num_eq": 5226533566632514089, "num_range": 8000, "upper_bound": "\\u001e-iw3\\u000eP"}, {"distinct_range": 4.945624117591082, "num_eq": 1372098228220247606, "num_range": 10, "upper_bound": "7h\\u001cL2z\\u001f#"}, {"distinct_range": 630740090051706200, "num_eq": 4874233394861014894, "num_range": 741027237375572895, "upper_bound": "Si\\u001c"}, {"distinct_range": 2241263907542999800, "num_eq": 600, "num_range": 8648687348801639550, "upper_bound": "l\\t."}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 1000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 357824, "histo_col_type": "", "name": "__auto__", "null_count": 469680, "row_count": 1000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 951248, "histo_col_type": "", "name": "__auto__", "null_count": 826384, "row_count": 1000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15, "histo_buckets": [{"distinct_range": 0, "num_eq": 4402294175155868065, "num_range": 0, "upper_bound": "142.152.216.143/7"}, {"distinct_range": 6024191614307893000, "num_eq": 9194158307250998479, "num_range": 6024191614307893421, "upper_bound": "88.172.222.234/23"}, {"distinct_range": 0, "num_eq": 6171624705750781483, "num_range": 100, "upper_bound": "32.117.240.178/24"}, {"distinct_range": 0, "num_eq": 7553118349113880933, "num_range": 80000000, "upper_bound": "9250:7f00:126b:3c4e:1e1d:51a:d242:309d/8"}, {"distinct_range": 4410880833058989000, "num_eq": 1461661451275499652, "num_range": 4410880833058989102, "upper_bound": "fece:4ffc:7571:260a:64ef:68f0:8b10:870b/41"}, {"distinct_range": 3704.3159803030367, "num_eq": 7954464286610945676, "num_range": 4000, "upper_bound": "8994:8aeb:fad6:f607:8da1:e702:1652:b488/57"}, {"distinct_range": 0, "num_eq": 8123444476308069953, "num_range": 5906896439694701879, "upper_bound": "ba98:bc78:7e10:8fe:fb64:bc9e:fca5:75d8/74"}, {"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 30}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "BOX(-2.128941416367994 -0.4956503057501799,1.0753884381917316 1.4154985317276094)"}, {"distinct_range": 0, "num_eq": 8558010264542877739, "num_range": 1467039814868626750, "upper_bound": "BOX(-0.08004818562222937 0.7358960734603501,0.5542339265455931 0.7984422659308742)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 30}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "\\\\x2992559d3c98b0"}, {"distinct_range": 1596949714978650400, "num_eq": 50000, "num_range": 1910115973202538125, "upper_bound": "\\\\x900c78"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 20, "row_count": 30}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17, "histo_col_type": "", "name": "__auto__", "null_count": 19, "row_count": 30}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000000, "num_range": 0, "upper_bound": "BOX(-1.4339078939254146 -1.1784108759271656,0.250856299879931 1.8109823765010298)"}, {"distinct_range": 846687799756003500, "num_eq": 50000000000, "num_range": 846687799756003493, "upper_bound": "BOX(-0.5952483059865132 0.14194680890308758,1.346881588582848 1.0295045891719499)"}, {"distinct_range": 19955.586343724975, "num_eq": 8000000000, "num_range": 70000, "upper_bound": "BOX(0.6150912643734858 -1.1188805297465056,0.7477387981150097 2.032667893982911)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 16, "row_count": 30}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 6000000, "num_eq": 0, "num_range": 6000000, "upper_bound": "2852855091"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 5000, "upper_bound": "3328568464"}, {"distinct_range": 7865127086261765000, "num_eq": 1000000000, "num_range": 7865127086261764830, "upper_bound": "3410679536"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 30}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23, "histo_col_type": "", "name": "__auto__", "null_count": 22, "row_count": 30}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11, "histo_buckets": [{"distinct_range": 0, "num_eq": 8919671709147078577, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 6542883624363810265, "num_range": 10000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 30}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3359162165503056953, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6946489844068915438}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6466419618468398266, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6946489844068915438}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5351388798391357617, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6946489844068915438}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1159395845874272062, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 2027682924401866465, "num_range": 100000000000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 6946489844068915438}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1094154036122018494, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6946489844068915438}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1974323802388877516, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6946489844068915438}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4001371510708948877, "histo_col_type": "", "name": "__auto__", "null_count": 5847447152334312084, "row_count": 6946489844068915438}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2860542548401816901, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6946489844068915438}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1324199167953579696, "histo_col_type": "", "name": "__auto__", "null_count": 5226545291413185113, "row_count": 6946489844068915438}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 622050336597963997, "histo_buckets": [{"distinct_range": 0, "num_eq": 5275864831852887270, "num_range": 0, "upper_bound": "-7922042345605643706"}, {"distinct_range": 997359108481324900, "num_eq": 4113145773045216659, "num_range": 2977785926969395751, "upper_bound": "-3820795409638204157"}, {"distinct_range": 0, "num_eq": 20000000000, "num_range": 8000000, "upper_bound": "8981180254688493159"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 6946489844068915438}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1374006920194888703, "histo_col_type": "", "name": "__auto__", "null_count": 6089493500841514488, "row_count": 6946489844068915438}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2140801111157868869, "histo_buckets": [{"distinct_range": 0, "num_eq": 4001909043164767047, "num_range": 0, "upper_bound": "\\u0001"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 0, "upper_bound": ")"}, {"distinct_range": 100, "num_eq": 400000, "num_range": 100, "upper_bound": "*"}, {"distinct_range": 800000, "num_eq": 10000000, "num_range": 800000, "upper_bound": "."}, {"distinct_range": 0, "num_eq": 8775585018725422292, "num_range": 40000000000, "upper_bound": "?"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 6946489844068915438}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1825598218571475723, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6946489844068915438}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 916567284581742168, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6946489844068915438}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1179305703364618576, "histo_col_type": "", "name": "__auto__", "null_count": 4392007868648412163, "row_count": 6946489844068915438}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('muxbsm', 'ssxala', 'jtgpa', 'fheelr', 'xpuse');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('isqdak', 'l', 'x');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('zpkzck', 'owfmft', 'i', 'lo', 'hn');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        | FAIL
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2657142&tab=buildLog) on [master@7853fd32de8b6dea869f2a2a92dcd7506f4a8998](https://github.com/cockroachdb/cockroach/commits/7853fd32de8b6dea869f2a2a92dcd7506f4a8998):

```
        test_1        |             CREATE INDEX ON table2 (col2_7 ASC, col2_0 DESC, col2_3 ASC, col2_4 DESC, col2_8 ASC, col2_1 DESC, col2_11 DESC, col2_9 ASC, col2_5 ASC, col2_2 DESC) STORING (col2_6);
        test_1        |             CREATE INDEX ON table2 (col2_5 ASC, col2_3, col2_7);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_9 DESC, col2_2, col2_6, col2_11, col2_8 ASC, col2_4 DESC, col2_3 DESC, col2_1 ASC, col2_5, col2_0 ASC, col2_10 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_7 DESC, col2_3 ASC, col2_5);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_3, col2_11 ASC, col2_10 ASC, col2_5 ASC, col2_1, col2_2, col2_7 DESC, col2_6 ASC, col2_9 DESC, col2_8 ASC, col2_0 ASC, col2_4 ASC);
        test_1        |             CREATE INDEX ON table2 (col2_3 DESC, col2_7, col2_1, col2_6 ASC, col2_10, col2_9);
        test_1        |             CREATE TABLE table3 (col1_14 "char" NOT NULL, col1_6 BOX2D NOT NULL, col1_12 REGNAMESPACE NOT NULL, col1_13 REGPROCEDURE NOT NULL, col1_1 TIMESTAMP NOT NULL, col1_7 REGPROCEDURE NOT NULL, col1_15 UUID NOT NULL, col3_7 INT8 NOT NULL, col3_8 VARCHAR NOT NULL, col3_9 BYTES NOT NULL, col3_10 BOOL, col3_11 REGPROCEDURE NULL, col3_12 VARCHAR, col3_13 UUID NULL, PRIMARY KEY (col1_14, col1_6, col1_12, col1_13, col1_1, col1_7, col1_15, col3_12, col3_9, col3_11, col3_7, col3_8, col3_13), FAMILY (col3_12), FAMILY (col3_10, col3_9, col1_13, col3_8, col1_12, col1_15, col3_11), FAMILY (col1_1, col3_13), FAMILY (col1_6), FAMILY (col1_7, col3_7, col1_14)) INTERLEAVE IN PARENT table1 (col1_14, col1_6, col1_12, col1_13, col1_1, col1_7, col1_15);
        test_1        |             CREATE INDEX ON table3 (col1_14 DESC, col1_12 DESC);
        test_1        |             CREATE INDEX ON table3 (col3_10, col1_15 DESC, col1_6 ASC, col3_7 DESC, col1_13 DESC, col1_7, col3_8 ASC, col3_12 ASC, col1_1 ASC, col1_14 DESC, col1_12, col3_11);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_13, col1_14 DESC, col1_13 ASC, col3_10 ASC, col3_12 ASC);
        test_1        |             CREATE INDEX ON table3 (col3_11 ASC, col1_13 ASC, col1_12 ASC, col1_14 DESC, col1_1, col1_7 DESC, col1_15, col1_6 ASC, col3_7 ASC, col3_13 ASC, col3_9, col3_12, col3_10 DESC);
        test_1        |             CREATE INDEX ON table3 (col3_12 ASC, col1_1 DESC, col1_12 DESC, col3_11, col3_8, col3_7 DESC, col1_14 DESC, col3_13 DESC, col1_13 DESC, col1_15 ASC, col1_6 DESC, col1_7, col3_9 DESC) STORING (col3_10);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col1_1, col3_11 DESC, col1_7, col1_13, col3_13);
        test_1        |             CREATE INDEX ON table3 (col3_10, col3_11 ASC, col1_15 ASC, col3_8, col3_12 DESC, col3_9, col1_12 ASC, col3_7 DESC);
        test_1        |             CREATE INDEX ON table3 (col1_7 ASC, col3_10 DESC, col1_14 DESC, col1_12, col3_8 ASC, col3_7, col3_12 ASC, col1_13 ASC);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000, "num_range": 0, "upper_bound": "000001010000011110001"}, {"distinct_range": 30000000, "num_eq": 8000000, "num_range": 30000000, "upper_bound": "000001101101001001100"}, {"distinct_range": 2189998267408620000, "num_eq": 4388163932047933695, "num_range": 2189998267408620087, "upper_bound": "000100111011011001000"}, {"distinct_range": 66.74136160224045, "num_eq": 4158284868270808567, "num_range": 900, "upper_bound": "010100010001001100100"}, {"distinct_range": 300, "num_eq": 4514158222465627722, "num_range": 300, "upper_bound": "110111111111001010111"}], "histo_col_type": "BIT(21)", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "BIT(8)", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 6481105046830160352, "num_range": 0, "upper_bound": "2"}, {"distinct_range": 2928074340566967000, "num_eq": 5494178332813911097, "num_range": 2928074340566966850, "upper_bound": "<"}, {"distinct_range": 0, "num_eq": 5279204435883890004, "num_range": 1000000000, "upper_bound": "M"}, {"distinct_range": 7000, "num_eq": 90000, "num_range": 7000, "upper_bound": "O"}, {"distinct_range": 0, "num_eq": 6485238037188779988, "num_range": 70000000000, "upper_bound": "U"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 3150373305662788346, "num_range": 0, "upper_bound": "-26610"}, {"distinct_range": 5640196837774938000, "num_eq": 8652437497804129102, "num_range": 7426993986122726600, "upper_bound": "20995"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 158750.3561530345, "num_eq": 320284888085105723, "num_range": 200000, "upper_bound": "1813728184"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25, "histo_buckets": [{"distinct_range": 0, "num_eq": 2817541098020288881, "num_range": 0, "upper_bound": "215526836"}, {"distinct_range": 1850222628082845000, "num_eq": 70000000, "num_range": 3038949437072991267, "upper_bound": "508151153"}, {"distinct_range": 0, "num_eq": 800, "num_range": 0, "upper_bound": "1237525547"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 77, "row_count": 100}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 66, "histo_buckets": [{"distinct_range": 0, "num_eq": 7814082031077855004, "num_range": 0, "upper_bound": "1980-09-26"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 49, "histo_buckets": [{"distinct_range": 0, "num_eq": 5096841032816223031, "num_range": 0, "upper_bound": "00:00:00"}, {"distinct_range": 60, "num_eq": 200, "num_range": 60, "upper_bound": "02:37:56.052211"}, {"distinct_range": 6102021646380653, "num_eq": 6181661908823111754, "num_range": 6893832489600972, "upper_bound": "15:46:30.785978"}, {"distinct_range": 0, "num_eq": 300, "num_range": 4162705313736932008, "upper_bound": "18:49:30.809379"}, {"distinct_range": 0, "num_eq": 384169019741754382, "num_range": 0, "upper_bound": "19:35:58.641417"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 84, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 82, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "00000010011101"}, {"distinct_range": 8921038510614585000, "num_eq": 40000000000, "num_range": 8930899496415837856, "upper_bound": "01011101000001"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 7314227735138747547, "upper_bound": "01101100110000"}, {"distinct_range": 0, "num_eq": 5038448636523645101, "num_range": 0, "upper_bound": "10100101000011"}, {"distinct_range": 58207.591246663236, "num_eq": 9000000, "num_range": 60000, "upper_bound": "11000110000001"}, {"distinct_range": 8643500114107373000, "num_eq": 7651729475817096626, "num_range": 8643500114107372620, "upper_bound": "11011010010111"}, {"distinct_range": 19362448.24030227, "num_eq": 300000000, "num_range": 200000000, "upper_bound": "11011011001110"}], "histo_col_type": "BIT(14)", "name": "__auto__", "null_count": 18, "row_count": 100}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 84, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12, "histo_col_type": "", "name": "__auto__", "null_count": 12, "row_count": 100}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15, "histo_buckets": [{"distinct_range": 0, "num_eq": 500, "num_range": 0, "upper_bound": "000000000000000000000000000000000000"}, {"distinct_range": 0, "num_eq": 200, "num_range": 600000000, "upper_bound": "010001011110100111110100101010101011"}, {"distinct_range": 2583950964049852400, "num_eq": 100, "num_range": 3344189708609063559, "upper_bound": "010101011001011010001111100101001101"}, {"distinct_range": 4930.267202947971, "num_eq": 80, "num_range": 9000, "upper_bound": "100110110011001111111101000000110101"}, {"distinct_range": 0, "num_eq": 3381903307255201165, "num_range": 7000, "upper_bound": "101001101111111110000100100110000101"}, {"distinct_range": 6790622969083372000, "num_eq": 200000, "num_range": 8764462664722673703, "upper_bound": "110001000110001010100110010100000000"}, {"distinct_range": 0, "num_eq": 7094616918071045062, "num_range": 7585914028719778084, "upper_bound": "110101100110111011011110001111101011"}], "histo_col_type": "BIT(36)", "name": "__auto__", "null_count": 43, "row_count": 100}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5020, "histo_buckets": [{"distinct_range": 0, "num_eq": 4066118526019577675, "num_range": 0, "upper_bound": "2009-02-25 07:45:26.000834"}, {"distinct_range": 7229663976274903000, "num_eq": 8000, "num_range": 8478026852715270040, "upper_bound": "2022-08-14 19:45:20.000548"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5448, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 798837670503644086, "upper_bound": "3905346505"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3947, "histo_buckets": [{"distinct_range": 0, "num_eq": 1780285635547626826, "num_range": 0, "upper_bound": "false"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 2674, "row_count": 8000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5683, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7086, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 154, "histo_buckets": [{"distinct_range": 0, "num_eq": 900, "num_range": 0, "upper_bound": "1024649805"}, {"distinct_range": 2363652.014339078, "num_eq": 8503469070633858904, "num_range": 3000000, "upper_bound": "1384397914"}, {"distinct_range": 5603963045135683000, "num_eq": 6232051924190345386, "num_range": 5603963045135682806, "upper_bound": "1733322413"}, {"distinct_range": 0, "num_eq": 50000000, "num_range": 9014583596589276632, "upper_bound": "2023092478"}, {"distinct_range": 0, "num_eq": 338823223457465789, "num_range": 800000000, "upper_bound": "2140892836"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 9000, "upper_bound": "2961123194"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 6000, "upper_bound": "3458092349"}, {"distinct_range": 7604366.449716715, "num_eq": 0, "num_range": 10000000, "upper_bound": "3610377155"}, {"distinct_range": 0, "num_eq": 300, "num_range": 7232016847870932276, "upper_bound": "3955397418"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 755, "row_count": 8000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6123, "histo_buckets": [{"distinct_range": 0, "num_eq": 3879539362456106532, "num_range": 0, "upper_bound": "(S1i\\u0011N\\u001c\'"}, {"distinct_range": 3935922947425760000, "num_eq": 8294193202862550718, "num_range": 3935922947425760183, "upper_bound": "7?`$"}, {"distinct_range": 780305892.9102318, "num_eq": 7264271835694453992, "num_range": 1000000000, "upper_bound": "?e\\u000c"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 2870, "row_count": 8000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3705, "histo_buckets": [{"distinct_range": 0, "num_eq": 5904436497454190397, "num_range": 0, "upper_bound": "03421afc-0930-4480-a318-afabe0816af3"}, {"distinct_range": 100000, "num_eq": 5094494444211076142, "num_range": 100000, "upper_bound": "29c7bb6a-9a38-4434-b1a5-b4a073dee4d6"}, {"distinct_range": 0, "num_eq": 757640558629725457, "num_range": 10000, "upper_bound": "4d44452b-3c33-4648-9e07-c7f25f684e06"}, {"distinct_range": 4364864022561665500, "num_eq": 70000000, "num_range": 4364864022561665486, "upper_bound": "7a97cfce-efdd-4971-b193-88504d860b19"}, {"distinct_range": 215092709335049400, "num_eq": 100000000, "num_range": 215092709335049398, "upper_bound": "d1919f37-f338-4496-806b-d0fc09ce4c2c"}, {"distinct_range": 14711472700.413948, "num_eq": 600000000, "num_range": 70000000000, "upper_bound": "d73b4fc8-664c-470b-9b70-bd878b449d36"}, {"distinct_range": 253241.72379917366, "num_eq": 5453839236061980965, "num_range": 300000, "upper_bound": "fa142999-7172-4572-b7eb-1d8ecbfe6181"}, {"distinct_range": 427254.11534943554, "num_eq": 1842818806715420881, "num_range": 500000, "upper_bound": "fe05453a-3c7a-4bb3-a331-8e0ba1a7d313"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 713, "row_count": 8000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2941, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2809, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6184, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2028, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2415, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 352, "histo_buckets": [{"distinct_range": 0, "num_eq": 3221296891854155391, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 59453868.67403484, "num_eq": 3881668314241680217, "num_range": 80000000, "upper_bound": "q"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 8000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('xv', 'aaobxy', 'wsogp', 'tdlet', 'l', 'hye');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('jy', 'umixs', 'mrly', 'kqmq', 'cp', 'vqdain');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        |     --- FAIL: TestCompare/mutators (0.28s)
        test_1        |         compare_test.go:129: 
        test_1        |             		SET CLUSTER SETTING sql.stats.automatic_collection.enabled = false;
        test_1        |             		SET CLUSTER SETTING sql.stats.histogram_collection.enabled = false;
        test_1        |             	CREATE TABLE table1 (col1_0 OID NOT NULL, col1_1 CHAR NOT NULL, col1_2 REGCLASS NOT NULL, col1_3 TIMETZ NOT NULL, col1_4 INT2 NOT NULL, col1_5 INTERVAL NOT NULL, col1_6 REGNAMESPACE NOT NULL, col1_7 "char" NOT NULL, col1_8 NAME NOT NULL, col1_9 BYTES NOT NULL, col1_10 VARCHAR NOT NULL, col1_11 BIT(12) NOT NULL, col1_12 DECIMAL NOT NULL, col1_13 REGCLASS NOT NULL, col1_14 DATE NOT NULL, col1_15 BIT(44) NOT NULL, col1_16 TIMESTAMPTZ NOT NULL, col1_17 INTERVAL NOT NULL, col1_18 INTERVAL NOT NULL, PRIMARY KEY (col1_13 DESC, col1_1 ASC, col1_5 ASC, col1_4 ASC, col1_2 DESC, col1_10, col1_3, col1_15 DESC, col1_16 DESC, col1_7, col1_8, col1_18, col1_11 ASC, col1_12 ASC, col1_6 ASC, col1_17, col1_9 DESC, col1_14 ASC, col1_0), INDEX (col1_4, col1_12 ASC, col1_10, col1_3, col1_2 DESC, col1_7 ASC, col1_1 DESC, col1_13 ASC, col1_14 ASC, col1_0, col1_18 ASC, col1_17, col1_11 DESC, col1_6 ASC, col1_16 ASC, col1_9 ASC, col1_5 ASC) WHERE ((((((table1.col1_9 > '\x00':::BYTES) AND (table1.col1_3 = '00:00:00+15:59:00':::TIMETZ)) OR (table1.col1_12 != 'Infinity':::DECIMAL)) OR (table1.col1_14 < '5874897-12-31':::DATE)) OR (table1.col1_7 = '"':::STRING)) OR (table1.col1_8 = e'\U00002603':::STRING)) OR (table1.col1_10 >= 'X':::STRING), INDEX (col1_6, col1_0 DESC, col1_9 ASC, col1_5 ASC, col1_11 ASC, col1_2, col1_7 ASC), FAMILY (col1_7), FAMILY (col1_15), FAMILY (col1_3, col1_5, col1_8), FAMILY (col1_1, col1_17), FAMILY (col1_18), FAMILY (col1_13), FAMILY (col1_2), FAMILY (col1_16, col1_11, col1_4), FAMILY (col1_9), FAMILY (col1_10, col1_12, col1_6), FAMILY (col1_0), FAMILY (col1_14));
        test_1        |             CREATE TABLE table2 (col2_0 VARCHAR NULL, col2_1 REGCLASS NULL, col2_2 VARCHAR NULL, col2_3 REGTYPE NOT NULL, col2_4 REGPROC NOT NULL, col2_5 VARCHAR NULL, col2_6 INT4 NOT NULL, col2_7 REGCLASS NULL, col2_8 INTERVAL NOT NULL, col2_9 TIMESTAMPTZ NULL, col2_10 FLOAT4 NOT NULL, col2_11 "char", col2_12 INT4 NOT NULL, col2_13 OID NULL, col2_14 FLOAT4 NOT NULL, col2_15 REGPROC NOT NULL, PRIMARY KEY (col2_6), UNIQUE (col2_13 DESC, col2_5 ASC, col2_0 ASC, col2_14 DESC, col2_8, col2_9, col2_3 DESC, col2_7 ASC) STORING (col2_4, col2_10, col2_12, col2_15) WHERE ((((((((table2.col2_5 > e'\x00':::STRING) AND (table2.col2_0 = '':::STRING)) OR (table2.col2_11 > e'\x00':::STRING)) OR (table2.col2_14 >= 3.4028234663852886e+38:::FLOAT8)) AND (table2.col2_10 < 'NaN':::FLOAT8)) AND (table2.col2_9 = '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table2.col2_12 >= (-32768):::INT8)) AND (table2.col2_2 <= 'X':::STRING)) AND (table2.col2_6 < 1:::INT8), INDEX (col2_11, col2_13 ASC, col2_0 ASC, col2_9 DESC, col2_3 DESC, col2_1 DESC, col2_14 ASC, col2_2 DESC, col2_12 DESC) WHERE ((((((((table2.col2_0 >= e'\U00002603':::STRING) AND (table2.col2_11 <= 'X':::STRING)) OR (table2.col2_14 < 3.4028234663852886e+38:::FLOAT8)) AND (table2.col2_5 != e'\x00':::STRING)) OR (table2.col2_10 >= '+Inf':::FLOAT8)) AND (table2.col2_2 < e'\U00002603':::STRING)) AND (table2.col2_12 < (-1):::INT8)) AND (table2.col2_9 > '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (table2.col2_6 >= 1:::INT8));
        test_1        |             CREATE TABLE table3 (col1_13 REGCLASS NOT NULL, col1_1 CHAR NOT NULL, col1_5 INTERVAL NOT NULL, col1_4 INT2 NOT NULL, col1_2 REGCLASS NOT NULL, col1_10 VARCHAR NOT NULL, col1_3 TIMETZ NOT NULL, col1_15 BIT(44) NOT NULL, col1_16 TIMESTAMPTZ NOT NULL, col1_7 "char" NOT NULL, col1_8 NAME NOT NULL, col1_18 INTERVAL NOT NULL, col1_11 BIT(12) NOT NULL, col1_12 DECIMAL NOT NULL, col1_6 REGNAMESPACE NOT NULL, col1_17 INTERVAL NOT NULL, col1_9 BYTES NOT NULL, col1_14 DATE NOT NULL, col1_0 OID NOT NULL, col3_19 INET, col3_20 FLOAT8 NULL, col3_21 REGNAMESPACE NULL, col3_22 "char", col3_23 CHAR NULL, col3_24 INT4 NULL, PRIMARY KEY (col1_13 DESC, col1_1 ASC, col1_5 ASC, col1_4 ASC, col1_2 DESC, col1_10, col1_3, col1_15 DESC, col1_16 DESC, col1_7, col1_8, col1_18, col1_11 ASC, col1_12 ASC, col1_6 ASC, col1_17, col1_9 DESC, col1_14 ASC, col1_0, col3_20, col3_23), FAMILY (col1_11), FAMILY (col1_12), FAMILY (col3_20, col1_9, col1_14), FAMILY (col1_2), FAMILY (col1_17, col1_7), FAMILY (col1_10, col1_15), FAMILY (col1_3, col1_18), FAMILY (col3_22), FAMILY (col3_23), FAMILY (col1_4, col3_21, col1_13, col3_19, col1_8, col3_24), FAMILY (col1_0), FAMILY (col1_6), FAMILY (col1_16), FAMILY (col1_5, col1_1)) INTERLEAVE IN PARENT table1 (col1_13, col1_1, col1_5, col1_4, col1_2, col1_10, col1_3, col1_15, col1_16, col1_7, col1_8, col1_18, col1_11, col1_12, col1_6, col1_17, col1_9, col1_14, col1_0);
        test_1        |             CREATE TABLE table4 (col4_0 INT8, col4_1 FLOAT8 NOT NULL, col4_2 TIMESTAMPTZ NOT NULL, PRIMARY KEY (col4_2), INDEX (col4_1 ASC, col4_0 DESC) WHERE (table4.col4_0 != 127:::INT8) AND (table4.col4_1 != 5e-324:::FLOAT8), INDEX (col4_2 DESC) STORING (col4_0) WHERE (table4.col4_1 >= 1.401298464324817e-45:::FLOAT8) OR (table4.col4_0 < (-128):::INT8), INDEX (col4_1), INDEX (col4_0 DESC) WHERE (table4.col4_1 != 1.401298464324817e-45:::FLOAT8) OR (table4.col4_0 = (-1):::INT8), UNIQUE (col4_2 DESC, col4_0 DESC) WHERE ((table4.col4_0 >= (-1):::INT8) OR (table4.col4_2 > '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table4.col4_1 != 0.0:::FLOAT8), INDEX (col4_1 ASC) WHERE table4.col4_2 != '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ, INDEX (col4_0 DESC, col4_1 ASC), INDEX (col4_1 DESC, col4_0 ASC, col4_2 ASC), INDEX (col4_1 DESC) WHERE ((table4.col4_2 != '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ) AND (table4.col4_0 >= (-2147483648):::INT8)) AND (table4.col4_1 <= 1.0:::FLOAT8));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2678515685664458262, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3046195429909876610, "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3231963881444918399, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 691813654510863351, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1289208624381118630, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1485078475088015807, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 934913409124647833, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3353545579565924136, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3094613503525451480, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2409428498770677306, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1912134784618035980, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1351265609979843257, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1298510235286047886, "histo_buckets": [{"distinct_range": 0, "num_eq": 6252250038405107146, "num_range": 0, "upper_bound": "-25963"}, {"distinct_range": 0, "num_eq": 191139407457524795, "num_range": 90000, "upper_bound": "2195"}, {"distinct_range": 42498666.00076179, "num_eq": 700, "num_range": 50000000, "upper_bound": "15958"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1148875505489787500, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2366361816312090559, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2554936017418439385, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1630382114787905226, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3434062089623689295, "histo_buckets": [{"distinct_range": 0, "num_eq": 90, "num_range": 0, "upper_bound": "50763241"}, {"distinct_range": 0, "num_eq": 8335418351024445609, "num_range": 4856824130899544099, "upper_bound": "903246018"}, {"distinct_range": 1848450071192845800, "num_eq": 80, "num_range": 7804301348552464273, "upper_bound": "964828051"}, {"distinct_range": 0, "num_eq": 6492828003093371711, "num_range": 800000, "upper_bound": "1105340408"}, {"distinct_range": 50000000, "num_eq": 0, "num_range": 50000000, "upper_bound": "1715863645"}, {"distinct_range": 70000, "num_eq": 9000, "num_range": 70000, "upper_bound": "2989838548"}, {"distinct_range": 27201970.859696418, "num_eq": 70000000, "num_range": 30000000, "upper_bound": "3612397567"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 1000, "upper_bound": "3630718435"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 195453459099099969, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3696846535959704087}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 641054, "histo_buckets": [{"distinct_range": 0, "num_eq": 4743689128076661526, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 5502597753556755967, "upper_bound": "\\u0001"}, {"distinct_range": 0, "num_eq": 7039610113298834697, "num_range": 2000000, "upper_bound": "\\u0015"}, {"distinct_range": 0, "num_eq": 5346654183733122221, "num_range": 2797548560671339135, "upper_bound": "!"}, {"distinct_range": 47.38023848496275, "num_eq": 1278452636605404527, "num_range": 400, "upper_bound": "+"}, {"distinct_range": 2581327533145872000, "num_eq": 2622487249936569580, "num_range": 2581327533145872088, "upper_bound": "_"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 50000000, "upper_bound": "m"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 649897, "row_count": 800000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 603206, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 197355, "histo_col_type": "", "name": "__auto__", "null_count": 155048, "row_count": 800000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 657916, "histo_col_type": "", "name": "__auto__", "null_count": 300440, "row_count": 800000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 741224, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-953145294"}, {"distinct_range": 845176268.203251, "num_eq": 1000, "num_range": 4000000000, "upper_bound": "135041788"}, {"distinct_range": 3759995.264066119, "num_eq": 0, "num_range": 4000000, "upper_bound": "182563075"}, {"distinct_range": 411895756778027650, "num_eq": 1018246830610299162, "num_range": 1290917060396470228, "upper_bound": "656441521"}, {"distinct_range": 829727222796734800, "num_eq": 50000000, "num_range": 1226464175381078205, "upper_bound": "1199348749"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 749234, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 368829, "histo_col_type": "", "name": "__auto__", "null_count": 155232, "row_count": 800000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 152945, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 254091, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 175986, "histo_col_type": "", "name": "__auto__", "null_count": 453963, "row_count": 800000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 685971, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 695287, "histo_col_type": "", "name": "__auto__", "null_count": 161403, "row_count": 800000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 781401, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "3399575760"}, {"distinct_range": 853782599640181400, "num_eq": 1000, "num_range": 5899877185998302693, "upper_bound": "4262464097"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 112936, "row_count": 800000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 703761, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 56465, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 489831, "histo_col_type": "", "name": "__auto__", "null_count": 28220, "row_count": 800000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 319343398677855684, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 509843730855992250, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 64802797854051991, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 265912410122978230, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col3_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 220696637696883427, "histo_col_type": "", "name": "__auto__", "null_count": 162875704337859203, "row_count": 561971739998679797}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 75138055255209258, "histo_buckets": [{"distinct_range": 0, "num_eq": 5659557082308523004, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 8913041039387272000, "num_eq": 0, "num_range": 8913041039387271873, "upper_bound": "674920350"}, {"distinct_range": 0, "num_eq": 6386718873493358731, "num_range": 4618589973380444954, "upper_bound": "1368927988"}, {"distinct_range": 0, "num_eq": 30000000, "num_range": 2197383183429023427, "upper_bound": "1888455700"}, {"distinct_range": 2351068951878439000, "num_eq": 9068161548915116438, "num_range": 8119927384480807444, "upper_bound": "4127946112"}, {"distinct_range": 13027.970699413961, "num_eq": 900000000, "num_range": 70000, "upper_bound": "4187152806"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 241699814772019544, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 552559611939545301, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 489246924144620715, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 264452673656567144, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col3_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 212425919810917521, "histo_col_type": "", "name": "__auto__", "null_count": 20206374017471396, "row_count": 561971739998679797}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 441657986380381123, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 202911066537468631, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 125643235958000401, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 216524070021890034, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 553282105109180186, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 252795555443657071, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 201123471857524563, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 257661781320309154, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col3_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 552440989405334606, "histo_col_type": "", "name": "__auto__", "null_count": 159743867357115211, "row_count": 561971739998679797}, {"columns": ["col3_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27959586761936337, "histo_col_type": "", "name": "__auto__", "null_count": 308492575077616826, "row_count": 561971739998679797}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 504148579571050461, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col1_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 254089726176742047, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 561971739998679797}, {"columns": ["col3_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 162587133268467964, "histo_col_type": "", "name": "__auto__", "null_count": 168392222387281468, "row_count": 561971739998679797}, {"columns": ["col3_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 249976052403467923, "histo_col_type": "", "name": "__auto__", "null_count": 414861019862406449, "row_count": 561971739998679797}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 541605060522236588, "histo_buckets": [{"distinct_range": 0, "num_eq": 1425572094560759069, "num_range": 0, "upper_bound": "-4701960017407491431"}, {"distinct_range": 6814332155206355000, "num_eq": 8248350145570707041, "num_range": 6814332155206355348, "upper_bound": "-4280181280762270247"}, {"distinct_range": 0, "num_eq": 2383378913597559077, "num_range": 7326307642260149575, "upper_bound": "2210502443702421579"}, {"distinct_range": 4780527079503846000, "num_eq": 800000, "num_range": 5388406872519510466, "upper_bound": "3468248109925426279"}, {"distinct_range": 400000000, "num_eq": 5836303161277215192, "num_range": 400000000, "upper_bound": "3908255175251636566"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 600000000, "upper_bound": "5690461816414129750"}, {"distinct_range": 0, "num_eq": 3510641185842179106, "num_range": 0, "upper_bound": "7835762651390279048"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 3262969888661615641, "row_count": 6676059651204348869}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1881380406576759855, "histo_buckets": [{"distinct_range": 0, "num_eq": 4849582894382161121, "num_range": 0, "upper_bound": "-1.4368113354179974"}, {"distinct_range": 9635.685037978463, "num_eq": 5576040803296882659, "num_range": 50000, "upper_bound": "-1.0566125823908477"}, {"distinct_range": 1839260998.0699952, "num_eq": 6100030401244294923, "num_range": 2000000000, "upper_bound": "1.2023845129554094"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 6676059651204348869}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1452221567602991812, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000, "num_range": 0, "upper_bound": "1993-06-23 23:35:45.0006+00:00"}, {"distinct_range": 0, "num_eq": 10, "num_range": 300, "upper_bound": "2000-11-10 00:20:04.000521+00:00"}, {"distinct_range": 742459.1989793727, "num_eq": 1318980867668022513, "num_range": 800000, "upper_bound": "2006-11-16 09:52:08.000295+00:00"}, {"distinct_range": 881407385675912200, "num_eq": 20000000, "num_range": 1037648742136230760, "upper_bound": "2010-06-17 23:34:29.00092+00:00"}, {"distinct_range": 0, "num_eq": 100, "num_range": 10000, "upper_bound": "2026-09-05 15:17:40.000655+00:00"}, {"distinct_range": 26.70591928055307, "num_eq": 5776916167894290312, "num_range": 100, "upper_bound": "2031-10-12 01:31:54.00026+00:00"}, {"distinct_range": 5780093157887261000, "num_eq": 5000, "num_range": 5780093157887260916, "upper_bound": "3000-01-01 00:00:00+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 6676059651204348869}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('ycz', 'wmbyyz');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('mbxii', 'd', 'nyui', 'n', 'mbj');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('hxcdaf', 'nhhn');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: interleaved tables and interleaved indexes are disabled due to the sql.defaults.interleaved_tables.enabled cluster setting. Note that interleaved tables and interleaved indexes will be removed in a future release. For details, see https://www.cockroachlabs.com/docs/releases/v20.2.0#deprecations (SQLSTATE 01P01)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2661544&tab=buildLog) on [master@9f1d67ae30f149fba017c67fb5b7d0cc56dfe21d](https://github.com/cockroachdb/cockroach/commits/9f1d67ae30f149fba017c67fb5b7d0cc56dfe21d):

```
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_3 DESC, col4_12 DESC, col4_13 ASC, col4_6 ASC, col4_7, col4_18, col4_1 DESC, col4_8 DESC, col4_4 ASC, col4_5, col4_17, col4_9, col4_0 ASC, col4_11 ASC, col4_2 DESC, col4_16 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_7 ASC, col4_2, col4_12 ASC, col4_9, col4_18 ASC, col4_13 ASC, col4_6 ASC, col4_16 DESC, col4_15, col4_1, col4_0, col4_5, col4_11 DESC, col4_8, col4_3 DESC, col4_4 ASC, col4_17 ASC) INCLUDE (col4_10);
        test_1        |             CREATE INDEX ON table4 (col4_8 ASC, col4_11 DESC, col4_9 DESC, col4_13);
        test_1        |             CREATE INDEX ON table4 (col4_11 ASC, col4_15) INCLUDE (col4_14);
        test_1        |             CREATE INDEX ON table4 (col4_17 ASC, col4_0 DESC, col4_13 ASC, col4_16 ASC, col4_6, col4_15, col4_11 DESC, col4_5, col4_8, col4_9 ASC, col4_7 ASC, col4_12, col4_4, col4_1) INCLUDE (col4_14);
        test_1        |             CREATE INDEX ON table4 (col4_5, col4_6 DESC, col4_8, col4_1 ASC, col4_13 ASC, col4_15, col4_7 ASC, col4_11 ASC, col4_4 ASC, col4_9 DESC, col4_16, col4_2 DESC, col4_17);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('s', 'q', 'gom', 'bvini');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('ikdgmq', 'fet', 'uhnkq');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (0.96s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 INT2, col1_1 CHAR NOT NULL, col1_2 OID, col1_3 INT2, col1_4 REGPROCEDURE NOT NULL, col1_5 CHAR NOT NULL, col1_6 FLOAT4, col1_7 REGPROCEDURE NOT NULL, col1_8 NAME, col1_9 INT8 NOT NULL, col1_10 BYTES NOT NULL, FAMILY (col1_0), FAMILY (col1_10, col1_8), FAMILY (col1_4), FAMILY (col1_5, col1_6, col1_2), FAMILY (col1_3), FAMILY (col1_1), FAMILY (col1_7, col1_9));
        test_1        |             CREATE TABLE table2 (col2_0 REGNAMESPACE, col2_1 BYTES NOT NULL, col2_2 INT4 NULL, col2_3 STRING, col2_4 NAME NOT NULL, col2_5 BIT(46) NULL, col2_6 REGPROC NOT NULL, col2_7 STRING NOT NULL, col2_8 DATE NOT NULL, col2_9 REGNAMESPACE, col2_10 FLOAT8 NOT NULL, col2_11 TIMETZ NOT NULL, col2_12 INTERVAL NOT NULL, PRIMARY KEY (col2_7 DESC, col2_11 DESC, col2_1 ASC, col2_4), INDEX (col2_8 DESC) STORING (col2_3, col2_6, col2_10, col2_12) WHERE ((((table2.col2_2 < -128) OR (table2.col2_10 = -1.0)) AND (table2.col2_8 <= '1970-01-01')) AND (table2.col2_1 > '\x27')) OR (table2.col2_4 > e'\U00002603'), FAMILY (col2_2, col2_7), FAMILY (col2_6), FAMILY (col2_1, col2_3), FAMILY (col2_12, col2_4, col2_11), FAMILY (col2_9), FAMILY (col2_8), FAMILY (col2_5, col2_10), FAMILY (col2_0));
        test_1        |             CREATE TABLE table3 (col3_0 BIT(29) NOT NULL, col3_1 REGTYPE NULL, col3_2 REGTYPE NOT NULL, col3_3 INT4 NOT NULL, col3_4 BIT(5) NOT NULL, col3_5 CHAR NULL, col3_6 NAME NOT NULL, col3_7 INT4 NOT NULL, col3_8 BOX2D NOT NULL, col3_9 REGTYPE NOT NULL, col3_10 UUID NOT NULL, col3_11 FLOAT8 NOT NULL, col3_12 "char" NOT NULL, PRIMARY KEY (col3_7 DESC, col3_8 DESC, col3_6 ASC, col3_2 ASC, col3_4 ASC, col3_9 DESC, col3_11, col3_10 ASC), FAMILY (col3_3, col3_0), FAMILY (col3_11, col3_2), FAMILY (col3_8), FAMILY (col3_7), FAMILY (col3_1, col3_10), FAMILY (col3_9), FAMILY (col3_12, col3_6, col3_4), FAMILY (col3_5));
        test_1        |             CREATE TABLE table4 (col3_7 INT4 NOT NULL, col3_8 BOX2D NOT NULL, col3_6 NAME NOT NULL, col3_2 REGTYPE NOT NULL, col3_4 BIT(5) NOT NULL, col3_9 REGTYPE NOT NULL, col3_11 FLOAT8 NOT NULL, col3_10 UUID NOT NULL, col4_8 TIMETZ NOT NULL, col4_9 VARCHAR, col4_10 INT2, col4_11 INT4 NULL, col4_12 BIT(11) NULL, col4_13 VARCHAR, col4_14 REGTYPE NULL, col4_15 NAME, col4_16 INT4 NULL, col4_17 TIMESTAMPTZ, col4_18 NAME, col4_19 BYTES NULL, col4_20 INT2, col4_21 BYTES NULL, col4_22 INTERVAL NOT NULL, col4_23 BIT(41) NULL, col4_24 INET NOT NULL, col4_25 REGPROCEDURE, col4_26 BYTES NULL, PRIMARY KEY (col3_7 DESC, col3_8 DESC, col3_6 ASC, col3_2 ASC, col3_4 ASC, col3_9 DESC, col3_11, col3_10 ASC, col4_19 ASC, col4_11 ASC, col4_26 DESC, col4_15 DESC, col4_18 DESC, col4_25 ASC, col4_10 DESC, col4_12 ASC, col4_23, col4_8 DESC, col4_9 ASC, col4_21, col4_16 ASC, col4_17, col4_20, col4_14 DESC), UNIQUE (col3_4, col4_21, col4_19, col3_11 DESC, col3_6 DESC, col4_24, col3_8, col4_16 DESC) STORING (col4_13), INDEX (col3_9 DESC, col3_6 DESC, col3_8 DESC, col4_25, col4_15 DESC, col4_12 DESC, col4_16, col4_18 DESC, col4_26 DESC, col4_11 ASC, col4_20 ASC, col4_8 ASC, col4_17, col3_11 ASC, col3_10 DESC, col4_21) STORING (col4_22) WHERE (((((((((((((((table4.col3_11 < '-Inf':::FLOAT8) OR (table4.col4_20 > 0:::INT8)) OR (table4.col4_15 != 'X':::STRING)) OR (table4.col4_13 != e'\x00':::STRING)) OR (table4.col3_7 < (-128):::INT8)) AND (table4.col4_19 <= e'\\xff':::BYTES)) OR (table4.col4_9 < '':::STRING)) OR (table4.col4_8 <= '24:00:00-15:59:00':::TIMETZ)) OR (table4.col3_6 < e'\U00002603':::STRING)) AND (table4.col4_16 < 1:::INT8)) AND (table4.col4_10 = (-32768):::INT8)) OR (table4.col4_17 >= '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table4.col4_18 != 'X':::STRING)) AND (table4.col4_11 > (-1):::INT8)) OR (table4.col4_21 <= e'\\x00':::BYTES)) AND (table4.col4_26 >= e'\\x58':::BYTES), UNIQUE (col4_18 ASC, col3_6, col3_10 ASC, col3_11 DESC, col4_11 ASC, col3_7, col4_8 ASC, col4_16 DESC, col4_9, col3_4 DESC, col4_15, col4_17 DESC, col4_21, col4_23, col4_14 ASC, col4_13) STORING (col4_22) WHERE (((table4.col4_9 < e'\x00':::STRING) AND (table4.col4_15 < '':::STRING)) OR (table4.col4_16 > (-2147483648):::INT8)) OR (table4.col4_21 < e'\\x22':::BYTES), UNIQUE (col4_22 DESC, col3_10, col4_11 ASC, col4_10, col4_15 ASC, col3_7 DESC, col4_9 DESC, col4_17 DESC, col4_19, col4_25 ASC, col3_8 DESC, col3_11 ASC, col4_23, col3_2 ASC, col4_14 ASC, col4_21 ASC, col4_13 DESC, col4_16 DESC, col4_26 ASC, col4_20 DESC, col4_24 DESC, col3_9 DESC) WHERE (((((((((((((((table4.col4_8 != '24:00:00-15:59:00') AND (table4.col4_19 = '\xff')) AND (table4.col4_18 = e'\'')) AND (table4.col4_11 != 1)) AND (table4.col3_11 = -Inf)) OR (table4.col4_13 >= '"')) OR (table4.col4_16 > -32768)) OR (table4.col3_7 <= 1)) OR (table4.col4_9 = '')) AND (table4.col4_21 > '\xe29883')) AND (table4.col4_15 < '"')) OR (table4.col4_26 < '\x58')) OR (table4.col4_20 = -1)) OR (table4.col3_6 != '')) OR (table4.col4_10 <= -1)) AND (table4.col4_17 <= '-4713-11-24 00:00:00+00:00'), UNIQUE (col4_21 ASC) WHERE (((((((((((((((table4.col3_7 < (-1):::INT8) AND (table4.col4_10 >= 0:::INT8)) OR (table4.col4_17 <= '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table4.col4_26 > e'\\xff':::BYTES)) OR (table4.col4_15 = '':::STRING)) AND (table4.col3_6 > '"':::STRING)) AND (table4.col4_20 > (-1):::INT8)) AND (table4.col4_16 >= (-128):::INT8)) AND (table4.col4_19 != e'\\x00':::BYTES)) OR (table4.col4_8 >= '24:00:00-15:59:00':::TIMETZ)) AND (table4.col4_13 >= '"':::STRING)) AND (table4.col4_11 < 1:::INT8)) OR (table4.col4_21 = e'\\x00':::BYTES)) OR (table4.col3_11 > 1.7976931348623157e+308:::FLOAT8)) OR (table4.col4_9 != e'\'':::STRING)) OR (table4.col4_18 > e'\'':::STRING), INDEX (col3_10 DESC, col4_9, col4_24 ASC, col4_10 ASC, col4_18 ASC, col4_12 ASC, col4_14 ASC, col4_21 ASC, col4_13, col3_9, col4_15 ASC, col3_7, col3_2 ASC, col3_4, col4_20 DESC, col4_8 DESC, col4_19, col3_6 ASC, col4_23 ASC, col3_8 DESC, col4_17 DESC, col4_26, col4_22 DESC, col4_16 ASC, col4_11 ASC, col4_25) WHERE (((((((((((table4.col4_21 <= e'\\x58':::BYTES) OR (table4.col3_11 != 1.7976931348623157e+308:::FLOAT8)) OR (table4.col4_15 > 'X':::STRING)) OR (table4.col4_20 != 1:::INT8)) OR (table4.col4_19 != e'\\x22':::BYTES)) AND (table4.col4_8 = '00:00:00+15:59:00':::TIMETZ)) AND (table4.col4_26 >= e'\\xe29883':::BYTES)) AND (table4.col4_17 = '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) OR (table4.col4_10 = 0:::INT8)) AND (table4.col4_16 >= (-1):::INT8)) AND (table4.col4_13 >= e'\x00':::STRING)) OR (table4.col4_18 >= 'X':::STRING), UNIQUE (col4_24, col4_10 DESC, col4_17) WHERE ((table4.col4_17 = '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ) OR (table4.col3_6 = e'\U00002603':::STRING)) AND (table4.col4_20 > (-128):::INT8), UNIQUE (col4_17, col4_24, col4_10, col4_20 DESC, col3_9 ASC, col3_10 DESC) STORING (col4_13, col4_22), FAMILY (col4_20, col3_7), FAMILY (col4_12), FAMILY (col4_16), FAMILY (col3_2, col4_23), FAMILY (col4_22, col4_14), FAMILY (col4_24, col3_6), FAMILY (col4_18, col4_21), FAMILY (col4_11), FAMILY (col4_19), FAMILY (col4_9), FAMILY (col4_10, col3_9, col4_26), FAMILY (col3_8, col4_8), FAMILY (col4_25), FAMILY (col4_15, col3_10, col4_17), FAMILY (col3_11, col4_13, col3_4)) INTERLEAVE IN PARENT table3 (col3_7, col3_8, col3_6, col3_2, col3_4, col3_9, col3_11, col3_10);
        test_1        |             CREATE TABLE table5 (col3_7 INT4 NOT NULL, col3_8 BOX2D NOT NULL, col3_6 NAME NOT NULL, col3_2 REGTYPE NOT NULL, col3_4 BIT(5) NOT NULL, col3_9 REGTYPE NOT NULL, col3_11 FLOAT8 NOT NULL, col3_10 UUID NOT NULL, col5_8 BIT(49) NULL, col5_9 STRING NOT NULL, col5_10 REGNAMESPACE, col5_11 TIMESTAMP NULL, col5_12 DATE, col5_13 INTERVAL NULL, col5_14 VARCHAR NULL, col5_15 INTERVAL NOT NULL, col5_16 REGCLASS NOT NULL, col5_17 UUID, col5_18 BIT(19), col5_19 REGPROCEDURE NOT NULL, col5_20 BIT(4) NULL, PRIMARY KEY (col3_7 DESC, col3_8 DESC, col3_6 ASC, col3_2 ASC, col3_4 ASC, col3_9 DESC, col3_11, col3_10 ASC, col5_13 DESC), INDEX (col5_13 DESC, col5_14 ASC, col5_8 ASC, col3_7 ASC, col3_9 ASC, col5_20, col5_12, col3_4 DESC, col3_2 ASC, col3_10 DESC, col3_11 ASC, col3_6, col5_9 DESC) STORING (col5_11, col5_15, col5_16, col5_17, col5_18) WHERE ((((((table5.col5_9 >= '':::STRING) AND (table5.col3_7 != 2147483647:::INT8)) OR (table5.col5_14 = 'X':::STRING)) OR (table5.col5_11 != '0001-01-01 00:00:00':::TIMESTAMP)) AND (table5.col3_11 < 5e-324:::FLOAT8)) AND (table5.col5_12 != '-infinity':::DATE)) OR (table5.col3_6 != '':::STRING), UNIQUE (col5_18 DESC, col5_11, col3_9, col5_19 DESC, col5_20, col5_14, col3_6) STORING (col5_9, col5_10, col5_12, col5_15, col5_16) WHERE (table5.col3_6 < '':::STRING) OR (table5.col3_11 < 1.401298464324817e-45:::FLOAT8), INDEX (col5_17 ASC, col3_2 DESC, col5_10 ASC, col5_9, col5_12, col5_20, col5_19 DESC, col3_6 DESC, col5_11 ASC, col5_18 DESC, col5_15 DESC, col3_4 DESC, col3_10 DESC, col5_16, col5_13 DESC, col3_9 ASC, col3_11 ASC, col5_14, col3_8 ASC, col5_8, col3_7 DESC) WHERE ((((((table5.col3_11 <= (-1.0):::FLOAT8) AND (table5.col3_6 <= e'\U00002603':::STRING)) AND (table5.col5_11 >= '0001-01-01 00:00:00':::TIMESTAMP)) OR (table5.col5_14 <= 'X':::STRING)) OR (table5.col5_12 != '4714-11-24 BC':::DATE)) AND (table5.col3_7 >= 2147483647:::INT8)) OR (table5.col5_9 > '"':::STRING), INDEX (col5_15, col5_10 DESC, col5_9, col5_18 DESC, col3_4 DESC, col5_12 DESC, col3_11, col5_11 DESC, col5_13 DESC, col3_9 DESC, col3_10 ASC) WHERE (table5.col5_12 >= '4714-11-24 BC':::DATE) AND (table5.col3_6 != '"':::STRING), UNIQUE (col5_18, col5_8, col3_6, col5_9, col3_2, col5_10, col3_8 DESC, col5_16 ASC, col3_7 DESC, col5_20 ASC, col3_4 DESC, col5_12, col5_13 ASC, col3_9, col3_11 ASC, col3_10, col5_17, col5_14 ASC, col5_11, col5_15 ASC), FAMILY (col5_15), FAMILY (col5_17, col3_2, col3_4), FAMILY (col5_10, col5_20), FAMILY (col5_8, col3_8), FAMILY (col5_13), FAMILY (col5_14, col3_10), FAMILY (col5_18), FAMILY (col3_7, col3_6), FAMILY (col5_19, col5_11, col3_9, col5_16), FAMILY (col5_12), FAMILY (col3_11, col5_9)) INTERLEAVE IN PARENT table3 (col3_7, col3_8, col3_6, col3_2, col3_4, col3_9, col3_11, col3_10);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1147674388, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16368612081, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28471958984, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24478113410, "histo_col_type": "", "name": "__auto__", "null_count": 28607724679, "row_count": 30000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20594514811, "histo_col_type": "", "name": "__auto__", "null_count": 149139925, "row_count": 30000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7767679140, "histo_col_type": "", "name": "__auto__", "null_count": 23454194000, "row_count": 30000000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6074834779, "histo_col_type": "", "name": "__auto__", "null_count": 13359217000, "row_count": 30000000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19146832870, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21632211833, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25886737111, "histo_col_type": "", "name": "__auto__", "null_count": 4954001320, "row_count": 30000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4139288483, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000000000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 813592897, "histo_col_type": "", "name": "__auto__", "null_count": 936730468, "row_count": 2000000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 549493692, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1610217936, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1778609847, "histo_buckets": [{"distinct_range": 0, "num_eq": 8648899508958174692, "num_range": 0, "upper_bound": "1981-03-26"}, {"distinct_range": 1018008.9014071604, "num_eq": 2823780379335883894, "num_range": 2000000, "upper_bound": "1985-12-02"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "infinity"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 2000000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 755502311, "histo_col_type": "", "name": "__auto__", "null_count": 1620991327, "row_count": 2000000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1751325169, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 565759025, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1414814104, "histo_col_type": "", "name": "__auto__", "null_count": 645762140, "row_count": 2000000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 989766194, "histo_col_type": "", "name": "__auto__", "null_count": 284688211, "row_count": 2000000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1560458498, "histo_buckets": [{"distinct_range": 0, "num_eq": 559438137478243716, "num_range": 0, "upper_bound": "\\u001a\\u0010TG>\\u0008Wg6"}, {"distinct_range": 2398199008348604000, "num_eq": 6000000000, "num_range": 2398199008348604158, "upper_bound": "<"}, {"distinct_range": 16.17847034943926, "num_eq": 4288168042881835338, "num_range": 40, "upper_bound": "=]0\\u000eP\\u0017gZ\\u0005"}, {"distinct_range": 0, "num_eq": 0, "num_range": 532627604074492462, "upper_bound": "Q@vCI[\\u001co"}, {"distinct_range": 10000000000, "num_eq": 800, "num_range": 10000000000, "upper_bound": "g"}, {"distinct_range": 1000000000, "num_eq": 60000, "num_range": 1000000000, "upper_bound": "s!`+"}, {"distinct_range": 4175249561917303000, "num_eq": 7000000, "num_range": 4175249561917302951, "upper_bound": "uJ]\\\\\\u0014\\u0008\\u0019\\t"}, {"distinct_range": 38.882738152387816, "num_eq": 8499963900256645932, "num_range": 200, "upper_bound": "~"}, {"distinct_range": 0, "num_eq": 5786931203211704217, "num_range": 800000, "upper_bound": "\U00002603"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 2000000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1875118489, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24233969, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 742669934, "histo_col_type": "", "name": "__auto__", "null_count": 1996000247, "row_count": 2000000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 7750137431182652002, "num_range": 0, "upper_bound": "-2147483648"}, {"distinct_range": 0, "num_eq": 4000000000, "num_range": 50000, "upper_bound": "-32768"}, {"distinct_range": 0, "num_eq": 400000, "num_range": 8000000000, "upper_bound": "-128"}, {"distinct_range": 7052072127.933181, "num_eq": 743767593360931736, "num_range": 30000000000, "upper_bound": "-1"}, {"distinct_range": 6000000000, "num_eq": 500, "num_range": 6000000000, "upper_bound": "972322278"}, {"distinct_range": 5807930209899678000, "num_eq": 4498460262857071086, "num_range": 8795099000638174262, "upper_bound": "1778275313"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 202293994887474107, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-1634954473"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 745810823327323660}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 646694309627127798, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000000, "num_range": 0, "upper_bound": "171ac7c3-661f-4251-88bb-30a3a10c6efc"}, {"distinct_range": 1932500025564450800, "num_eq": 600000000, "num_range": 1932500025564450880, "upper_bound": "4829a1e4-5db3-4a9a-883b-47e7dbefa2d3"}, {"distinct_range": 5000000000, "num_eq": 9091777330946140332, "num_range": 5000000000, "upper_bound": "6a18ff98-e7b9-4c62-8a7c-4d9dbc7e17da"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 8000, "upper_bound": "7779c7d1-da33-489d-b1a7-3b34ba4bc628"}, {"distinct_range": 577690652922848500, "num_eq": 2051186163747736588, "num_range": 8753416113397790197, "upper_bound": "b307638c-8972-4339-93a2-58658407afbc"}, {"distinct_range": 0, "num_eq": 2000000, "num_range": 0, "upper_bound": "e2a221a4-e4e8-4fb7-82c2-f692462a337e"}, {"distinct_range": 10000, "num_eq": 700000000, "num_range": 10000, "upper_bound": "e8bc7f62-688b-4add-a0ca-3080b557b5b7"}, {"distinct_range": 5870616528593958000, "num_eq": 30, "num_range": 6910615438163090162, "upper_bound": "f3aa9d5d-78e3-4ddc-834e-026cbee54951"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 745810823327323660}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 416147742215435499, "histo_col_type": "", "name": "__auto__", "null_count": 95820735460154689, "row_count": 745810823327323660}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 365101293828689825, "histo_col_type": "", "name": "__auto__", "null_count": 560800714319001227, "row_count": 745810823327323660}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 693516920160712008, "histo_buckets": [{"distinct_range": 0, "num_eq": 27815775849413323, "num_range": 0, "upper_bound": "-4713-11-24 00:00:00+00:00"}, {"distinct_range": 0, "num_eq": 7533704317531583536, "num_range": 7000000, "upper_bound": "1980-12-09 00:19:41.000367+00:00"}, {"distinct_range": 225072301838302180, "num_eq": 1392676344811095235, "num_range": 4937112822172470414, "upper_bound": "1988-07-17 13:24:09.000413+00:00"}, {"distinct_range": 200, "num_eq": 4468438631544115278, "num_range": 200, "upper_bound": "1990-11-11 00:34:48.000948+00:00"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 1000000, "upper_bound": "1992-07-25 11:34:33.000162+00:00"}, {"distinct_range": 0, "num_eq": 4868194382336613549, "num_range": 8847362967839599910, "upper_bound": "2028-11-27 19:16:52.000718+00:00"}, {"distinct_range": 8046729786634200000, "num_eq": 0, "num_range": 8046729786634200360, "upper_bound": "2029-05-11 20:43:33.000636+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 554901844997134543, "row_count": 745810823327323660}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 489493526149274099, "histo_col_type": "", "name": "__auto__", "null_count": 192196066402239348, "row_count": 745810823327323660}, {"columns": ["col4_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 198360099190166198, "histo_col_type": "", "name": "__auto__", "null_count": 316649548055632943, "row_count": 745810823327323660}, {"columns": ["col4_26"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 469923202623440823, "histo_col_type": "", "name": "__auto__", "null_count": 59798350754012832, "row_count": 745810823327323660}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 450678841277226416, "histo_col_type": "BIT(5)", "name": "__auto__", "null_count": 0, "row_count": 745810823327323660}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 348511112689746322, "histo_col_type": "", "name": "__auto__", "null_count": 423784573909706227, "row_count": 745810823327323660}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 108386142481393920, "histo_col_type": "", "name": "__auto__", "null_count": 577074473287730617, "row_count": 745810823327323660}, {"columns": ["col4_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 502381663111905360, "histo_buckets": [{"distinct_range": 0, "num_eq": 2667273582000928108, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 1146629691277794600, "num_eq": 1465659299927266610, "num_range": 3333161851866188502, "upper_bound": "\\\\x036419de88f7c419"}, {"distinct_range": 6187185.751667369, "num_eq": 80000, "num_range": 10000000, "upper_bound": "\\\\x8b"}, {"distinct_range": 44130.48625971286, "num_eq": 6909960821122635864, "num_range": 90000, "upper_bound": "\\\\xea02f82fc8ee0397b4"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 292022400169423360, "row_count": 745810823327323660}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 452041039359806335, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 745810823327323660}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 691065436830672984, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 745810823327323660}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 426654470048364909, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 745810823327323660}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 369402445721741903, "histo_col_type": "", "name": "__auto__", "null_count": 210324867629288726, "row_count": 745810823327323660}, {"columns": ["col4_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 258085454237824560, "histo_buckets": [{"distinct_range": 0, "num_eq": 8685364041358411779, "num_range": 0, "upper_bound": "-28 years -6 mons -526 days -23:45:22.104509"}, {"distinct_range": 0, "num_eq": 20000000000, "num_range": 300000000, "upper_bound": "-27 years -5 mons -744 days -01:03:08.163132"}, {"distinct_range": 0, "num_eq": 1974922855547306105, "num_range": 900, "upper_bound": "-12 years -3 mons -193 days -04:08:28.771941"}, {"distinct_range": 810627.1350406992, "num_eq": 5795236768225589273, "num_range": 1000000, "upper_bound": "-6 mons -233 days -10:58:14.570193"}, {"distinct_range": 20000, "num_eq": 0, "num_range": 20000, "upper_bound": "4 years 6 mons 277 days 12:13:33.159395"}, {"distinct_range": 2743317838210162700, "num_eq": 3235107601813553918, "num_range": 2743317838210162728, "upper_bound": "64 years 709 days 24:22:45.923887"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 745810823327323660}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 320517534244724079, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 745810823327323660}, {"columns": ["col4_25"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 263180144043729096, "histo_col_type": "", "name": "__auto__", "null_count": 704732866543208739, "row_count": 745810823327323660}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 577067637159746106, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 745810823327323660}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 326189893496296806, "histo_col_type": "", "name": "__auto__", "null_count": 80456652640509996, "row_count": 745810823327323660}, {"columns": ["col4_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 345924035764132039, "histo_col_type": "NAME", "name": "__auto__", "null_count": 119380365360734208, "row_count": 745810823327323660}, {"columns": ["col4_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 296760554953430113, "histo_col_type": "", "name": "__auto__", "null_count": 642294070032945899, "row_count": 745810823327323660}, {"columns": ["col4_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 154296123950748212, "histo_col_type": "", "name": "__auto__", "null_count": 444788675953826986, "row_count": 745810823327323660}, {"columns": ["col4_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 285505728481505558, "histo_buckets": [{"distinct_range": 0, "num_eq": 8877843100980782816, "num_range": 0, "upper_bound": "0.0.0.0/0"}, {"distinct_range": 417.66333949948495, "num_eq": 80000000000, "num_range": 500, "upper_bound": "158.114.104.239/20"}, {"distinct_range": 5726070846574624000, "num_eq": 1019349162776136927, "num_range": 5726070846574624139, "upper_bound": "230.150.92.228/29"}, {"distinct_range": 1665793841096222700, "num_eq": 6712437188447631175, "num_range": 1665793841096222845, "upper_bound": "73.14.61.221/31"}, {"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": "99.102.92.47"}, {"distinct_range": 7235705.639346855, "num_eq": 7910805458054792874, "num_range": 90000000, "upper_bound": "99ec:d648:a4dd:388f:5a3:b5a4:1bf2:a738/30"}, {"distinct_range": 3697616378711240000, "num_eq": 5000000000, "num_range": 4754694020634257875, "upper_bound": "9d3b:3ff:b9d3:3684:acbf:2254:a72a:5c9c/34"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 745810823327323660}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 190387356773438913, "histo_buckets": [{"distinct_range": 0, "num_eq": 80, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 5000000, "upper_bound": "4090687098"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 745810823327323660}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 541080517401657965, "histo_col_type": "", "name": "__auto__", "null_count": 483212419603796863, "row_count": 745810823327323660}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 394, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 301, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col5_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46, "histo_col_type": "", "name": "__auto__", "null_count": 282, "row_count": 400}, {"columns": ["col5_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 339, "histo_buckets": [{"distinct_range": 0, "num_eq": 2315074804113182374, "num_range": 0, "upper_bound": "-76 years -3 mons -876 days -22:34:57.574262"}, {"distinct_range": 5893583860874008000, "num_eq": 293205547391834167, "num_range": 5893583860874007755, "upper_bound": "68 years 6 mons 173 days 08:01:03.214614"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 329, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 235, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col5_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 337, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col5_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 226, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": "-43 years -5 mons -282 days -00:37:21.130529"}, {"distinct_range": 3773056676125686000, "num_eq": 7076376150583244266, "num_range": 6521352521146403256, "upper_bound": "-29 years -2 mons -750 days -07:05:28.752057"}, {"distinct_range": 400000000, "num_eq": 60000, "num_range": 400000000, "upper_bound": "-19 years -6 mons -503 days -08:07:19.397246"}, {"distinct_range": 6285508984052868000, "num_eq": 600, "num_range": 6285508984052867855, "upper_bound": "5 years 5 mons 291 days 11:11:54.312741"}, {"distinct_range": 500, "num_eq": 8000000000, "num_range": 500, "upper_bound": "17 years 6 mons 627 days 04:13:56.175333"}, {"distinct_range": 2009344759780204000, "num_eq": 21832026394579677, "num_range": 2379933527532748862, "upper_bound": "38 years 9 mons 966 days 22:36:26.450685"}, {"distinct_range": 951938.5973822661, "num_eq": 0, "num_range": 2000000, "upper_bound": "42 years 6 mons 312 days 00:42:34.119782"}, {"distinct_range": 424.7206808978613, "num_eq": 0, "num_range": 4000, "upper_bound": "56 years 5 mons 862 days 04:10:25.343045"}, {"distinct_range": 3070827749675945000, "num_eq": 6000, "num_range": 3070827749675945101, "upper_bound": "79 years 8 mons 534 days 09:19:13.238027"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 2, "row_count": 400}, {"columns": ["col5_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 226, "histo_col_type": "", "name": "__auto__", "null_count": 332, "row_count": 400}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 249, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col5_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 369, "histo_col_type": "", "name": "__auto__", "null_count": 161, "row_count": 400}, {"columns": ["col5_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 99, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col5_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 242, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000, "num_range": 0, "upper_bound": "0100111111011110000"}, {"distinct_range": 0, "num_eq": 1905503516009673957, "num_range": 60000000000, "upper_bound": "0110001111011110010"}, {"distinct_range": 0, "num_eq": 200000000, "num_range": 9161585051219070122, "upper_bound": "1010111001000011001"}, {"distinct_range": 4641083754047115000, "num_eq": 20000, "num_range": 6989535183146118961, "upper_bound": "1011110010010110100"}, {"distinct_range": 70000000000, "num_eq": 80, "num_range": 70000000000, "upper_bound": "1101001010111101000"}], "histo_col_type": "BIT(19)", "name": "__auto__", "null_count": 284, "row_count": 400}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 380, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 286, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col5_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 245, "histo_col_type": "", "name": "__auto__", "null_count": 227, "row_count": 400}, {"columns": ["col5_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 259, "histo_col_type": "", "name": "__auto__", "null_count": 334, "row_count": 400}, {"columns": ["col5_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 386, "histo_col_type": "", "name": "__auto__", "null_count": 42, "row_count": 400}, {"columns": ["col5_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 239, "histo_buckets": [{"distinct_range": 0, "num_eq": 7955451209755103797, "num_range": 0, "upper_bound": "00000000-0000-0000-0000-000000000000"}, {"distinct_range": 0, "num_eq": 3321818012636495638, "num_range": 80000000000, "upper_bound": "4f0b91ef-5823-48af-9381-e7296f8768ca"}, {"distinct_range": 331483032.8939263, "num_eq": 0, "num_range": 1000000000, "upper_bound": "7716b71e-91be-4bed-a527-3a424f8db33e"}, {"distinct_range": 0, "num_eq": 6405037187525878903, "num_range": 300000000, "upper_bound": "7cb93488-3ae5-4b76-8857-6141a8f894f5"}, {"distinct_range": 0, "num_eq": 50000000, "num_range": 6492928428116872262, "upper_bound": "8af25b5e-c7aa-4c08-9ff9-aeb2d18413ee"}, {"distinct_range": 1173248702953672000, "num_eq": 5557196100037328705, "num_range": 8372730318716403844, "upper_bound": "a14d7583-5f12-47de-a474-02406515f8cc"}, {"distinct_range": 0, "num_eq": 3100709667599986332, "num_range": 4737654359151249685, "upper_bound": "ca599223-3e55-44df-a971-a9f79767ed41"}, {"distinct_range": 2668163213559644000, "num_eq": 7489978817842755950, "num_range": 2668163213559643968, "upper_bound": "d7af22ce-5865-4114-8ef3-2aa863124a82"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 64, "row_count": 400}, {"columns": ["col5_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 154, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 169, "histo_buckets": [{"distinct_range": 0, "num_eq": 4810630315641726780, "num_range": 0, "upper_bound": "-1500346436"}, {"distinct_range": 900000000, "num_eq": 6484362871752771482, "num_range": 900000000, "upper_bound": "-1019782188"}, {"distinct_range": 0, "num_eq": 5000000000, "num_range": 1000000000, "upper_bound": "267008329"}, {"distinct_range": 0, "num_eq": 900000, "num_range": 8903834638066059986, "upper_bound": "308462677"}, {"distinct_range": 33072874623.95393, "num_eq": 6809785131126999205, "num_range": 60000000000, "upper_bound": "852021322"}, {"distinct_range": 0, "num_eq": 400, "num_range": 8657493003633139941, "upper_bound": "1094952213"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 400}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('wasyfx', 'ejmu', 'l', 'd', 'mafqo', 'eizj');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('bkxbzt');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('lup', 'oid');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5584557811, "histo_col_type": "", "name": "__auto__", "null_count": 1470831197, "row_count": 6000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1119008067, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1735357646, "histo_col_type": "", "name": "__auto__", "null_count": 5580192184, "row_count": 6000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2661733458, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4769861157, "histo_col_type": "", "name": "__auto__", "null_count": 254897903, "row_count": 6000000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4469890715, "histo_col_type": "", "name": "__auto__", "null_count": 55119213, "row_count": 6000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 180957882, "histo_col_type": "", "name": "__auto__", "null_count": 1684599989, "row_count": 6000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4331425210, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3099793051, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3762878617, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2333266302, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28710, "histo_col_type": "", "name": "__auto__", "null_count": 3411, "row_count": 40000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39504, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10632, "histo_col_type": "", "name": "__auto__", "null_count": 39377, "row_count": 40000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9348, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23147, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17964, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6218, "histo_col_type": "", "name": "__auto__", "null_count": 35460, "row_count": 40000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16777, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7947, "histo_col_type": "", "name": "__auto__", "null_count": 9172, "row_count": 40000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12750, "histo_buckets": [{"distinct_range": 0, "num_eq": 9205356000154030614, "num_range": 0, "upper_bound": "\\tfU\\u0007\\u000f\\u001bj\\u001e"}, {"distinct_range": 0, "num_eq": 20, "num_range": 5000000, "upper_bound": "2"}, {"distinct_range": 3504294007088061000, "num_eq": 3874644527402740734, "num_range": 4239997342332340408, "upper_bound": "=Rr[t"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5489, "histo_buckets": [{"distinct_range": 0, "num_eq": 5985131664479326478, "num_range": 0, "upper_bound": "1986-10-13"}, {"distinct_range": 42.38010541790827, "num_eq": 0, "num_range": 300, "upper_bound": "1992-01-09"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34111, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27328, "histo_col_type": "", "name": "__auto__", "null_count": 39342, "row_count": 40000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4996, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18171, "histo_col_type": "", "name": "__auto__", "null_count": 4573, "row_count": 30000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26120, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13066, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": "-900782587"}, {"distinct_range": 0, "num_eq": 20000000, "num_range": 3078285610146923133, "upper_bound": "-857048174"}, {"distinct_range": 849253257719491500, "num_eq": 3918515373118260872, "num_range": 1392002692701143777, "upper_bound": "-831598274"}, {"distinct_range": 0, "num_eq": 5690574291777394068, "num_range": 0, "upper_bound": "-638004715"}, {"distinct_range": 3254324046157084000, "num_eq": 6000000, "num_range": 7175469854698335333, "upper_bound": "232223641"}, {"distinct_range": 0, "num_eq": 4883607737355058273, "num_range": 30000000, "upper_bound": "682532140"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 5192014497253444788, "upper_bound": "1160024579"}, {"distinct_range": 1537140072786345500, "num_eq": 200, "num_range": 2848153294540962325, "upper_bound": "1337861726"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10299, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24828, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11684, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42, "histo_col_type": "", "name": "__auto__", "null_count": 24358, "row_count": 30000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10618, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13934, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4322, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13604, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 685, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1870885862852683123, "histo_col_type": "", "name": "__auto__", "null_count": 138841333289080995, "row_count": 3646500332026053377}, {"columns": ["col4_25"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 870744398385822948, "histo_col_type": "", "name": "__auto__", "null_count": 1304775553361077785, "row_count": 3646500332026053377}, {"columns": ["col4_26"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 132150980869858409, "histo_col_type": "", "name": "__auto__", "null_count": 1425762450682297628, "row_count": 3646500332026053377}, {"columns": ["col4_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3318631531558356095, "histo_col_type": "", "name": "__auto__", "null_count": 648751759293018201, "row_count": 3646500332026053377}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 338064624591805907, "histo_col_type": "", "name": "__auto__", "null_count": 1559155600822502700, "row_count": 3646500332026053377}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3633739376035969451, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3646500332026053377}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 140396680079384291, "histo_buckets": [{"distinct_range": 0, "num_eq": 300000000, "num_range": 0, "upper_bound": "134235887"}, {"distinct_range": 0, "num_eq": 4479604606859315109, "num_range": 3383171672869434149, "upper_bound": "1359352188"}, {"distinct_range": 5411261168.747589, "num_eq": 3663412862578452906, "num_range": 10000000000, "upper_bound": "2950415586"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 3646500332026053377}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3343481104506050183, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3646500332026053377}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2902984655026428808, "histo_col_type": "", "name": "__auto__", "null_count": 1263845097600119497, "row_count": 3646500332026053377}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2689202557659068677, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3646500332026053377}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 523638052293852900, "histo_col_type": "", "name": "__auto__", "null_count": 713516450467827802, "row_count": 3646500332026053377}, {"columns": ["col4_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1226119978791011069, "histo_buckets": [{"distinct_range": 0, "num_eq": 7334749280035503653, "num_range": 0, "upper_bound": "-50 years -10 mons -26 days -24:02:11.449938"}, {"distinct_range": 1285594448543304200, "num_eq": 5282269741744023768, "num_range": 2535239172073886664, "upper_bound": "-28 years -8 mons -840 days -03:22:45.581692"}, {"distinct_range": 0, "num_eq": 9057701281378209159, "num_range": 60000000000, "upper_bound": "1 day"}, {"distinct_range": 247713796063160740, "num_eq": 6301355471719069260, "num_range": 955060441590181722, "upper_bound": "6 years 3 mons 19 days 24:14:28.452586"}, {"distinct_range": 0, "num_eq": 3308559915011840438, "num_range": 7000, "upper_bound": "29 years 9 mons 389 days 20:31:14.736344"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 3646500332026053377}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1523570897714337510, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3646500332026053377}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2572155455564905389, "histo_col_type": "", "name": "__auto__", "null_count": 1177791723736863026, "row_count": 3646500332026053377}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3026373434901137729, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3646500332026053377}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3630041176523081081, "histo_buckets": [{"distinct_range": 0, "num_eq": 60, "num_range": 0, "upper_bound": "14e6aad8-d48c-4146-b1ab-9557c4a919d6"}, {"distinct_range": 800000000, "num_eq": 20, "num_range": 800000000, "upper_bound": "6f23b17e-ae5d-4b50-9f87-b05f80d9fc69"}, {"distinct_range": 1304162.303438846, "num_eq": 3629107529855407825, "num_range": 9000000, "upper_bound": "78dee2e9-b61a-4572-aac1-47a8fc643c20"}, {"distinct_range": 176992792478049440, "num_eq": 5606745247041565170, "num_range": 400481447762048065, "upper_bound": "7d10e082-9a0e-4c75-92bc-12e41dfb8622"}, {"distinct_range": 0, "num_eq": 400000, "num_range": 3187602909323518603, "upper_bound": "95f49f4a-ca0f-4460-a5c6-6a8047a15b5f"}, {"distinct_range": 0, "num_eq": 7619200075917875789, "num_range": 6000, "upper_bound": "a63dc168-d688-4942-b25a-0b1339bd3ab7"}, {"distinct_range": 1225276068101434000, "num_eq": 2374652347847544461, "num_range": 1225276068101434090, "upper_bound": "b4dcd304-d1f8-487c-83eb-e54b56fa0120"}, {"distinct_range": 1466059383914931000, "num_eq": 10000000000, "num_range": 5742485437266310214, "upper_bound": "b9666557-05c1-496f-8ef4-8ca2f48de8b5"}, {"distinct_range": 754.6096020454878, "num_eq": 700, "num_range": 800, "upper_bound": "d5151345-2180-4137-90fc-120a79d266ac"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 3646500332026053377}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3595869617577196007, "histo_col_type": "", "name": "__auto__", "null_count": 2036438746490737581, "row_count": 3646500332026053377}, {"columns": ["col4_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1918094757802348899, "histo_col_type": "", "name": "__auto__", "null_count": 1026425257851869547, "row_count": 3646500332026053377}, {"columns": ["col4_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2486825775442205318, "histo_buckets": [{"distinct_range": 0, "num_eq": 695798777915901664, "num_range": 0, "upper_bound": "128.10.123.198/29"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 3646500332026053377}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1317807289103347140, "histo_col_type": "", "name": "__auto__", "null_count": 3326903398439283439, "row_count": 3646500332026053377}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 500897699591726164, "histo_col_type": "", "name": "__auto__", "null_count": 2960107357499820433, "row_count": 3646500332026053377}, {"columns": ["col4_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1431017254201889110, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000000, "num_range": 0, "upper_bound": "\\u0012hi7{\\u000b\\u000c"}, {"distinct_range": 22900234059788804, "num_eq": 5000000000, "num_range": 22900234059788803, "upper_bound": "9*4!VX\\u0006"}, {"distinct_range": 1256078886279783400, "num_eq": 6327860789146494125, "num_range": 3613414551655384657, "upper_bound": "?/wY\' v{"}, {"distinct_range": 4000, "num_eq": 9000, "num_range": 4000, "upper_bound": "X"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 410188420927706607, "row_count": 3646500332026053377}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1370239536872622364, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "00000"}, {"distinct_range": 0, "num_eq": 500, "num_range": 1366644243673100020, "upper_bound": "01010"}, {"distinct_range": 0, "num_eq": 9000, "num_range": 0, "upper_bound": "01011"}, {"distinct_range": 8000, "num_eq": 229952255904225293, "num_range": 8000, "upper_bound": "10001"}, {"distinct_range": 16904291172214250, "num_eq": 3695951707935197943, "num_range": 48975443088514626, "upper_bound": "10110"}, {"distinct_range": 2036830198678655500, "num_eq": 8000000000, "num_range": 2077470339683198912, "upper_bound": "11011"}, {"distinct_range": 21068217.56091483, "num_eq": 8000, "num_range": 80000000, "upper_bound": "11111"}], "histo_col_type": "BIT(5)", "name": "__auto__", "null_count": 0, "row_count": 3646500332026053377}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 556216872444620388, "histo_col_type": "", "name": "__auto__", "null_count": 2353522115452633970, "row_count": 3646500332026053377}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3577763147703779745, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000, "num_range": 0, "upper_bound": "0001-01-01 00:00:00+00:00"}, {"distinct_range": 360214326.4375017, "num_eq": 6043606548787621233, "num_range": 900000000, "upper_bound": "1974-06-08 07:18:12.000069+00:00"}, {"distinct_range": 5553207053133563000, "num_eq": 100000, "num_range": 5553207053133562792, "upper_bound": "1976-11-21 08:18:50.000822+00:00"}, {"distinct_range": 0, "num_eq": 0, "num_range": 4689015727075279619, "upper_bound": "1986-06-25 16:57:13.000642+00:00"}, {"distinct_range": 90000000000, "num_eq": 10000000000, "num_range": 90000000000, "upper_bound": "2011-03-31 01:14:34.000575+00:00"}, {"distinct_range": 0, "num_eq": 0, "num_range": 100000, "upper_bound": "2018-12-15 03:34:32.000003+00:00"}, {"distinct_range": 0, "num_eq": 1413319653755263216, "num_range": 50000000000, "upper_bound": "2030-12-17 07:29:12.000998+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 334908831304472114, "row_count": 3646500332026053377}, {"columns": ["col4_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 158928346232060481, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000, "num_range": 0, "upper_bound": "\\\\x22"}, {"distinct_range": 5486082843611874000, "num_eq": 5982247680049284705, "num_range": 5486082843611874602, "upper_bound": "\\\\x27"}, {"distinct_range": 0, "num_eq": 5707448055103052172, "num_range": 0, "upper_bound": "\\\\x28f172"}, {"distinct_range": 0, "num_eq": 8154529463054512669, "num_range": 70000000, "upper_bound": "\\\\x44c4e094"}, {"distinct_range": 0, "num_eq": 10, "num_range": 4817805401641772173, "upper_bound": "\\\\x48"}, {"distinct_range": 0, "num_eq": 400, "num_range": 2198137688862516320, "upper_bound": "\\\\x83d76231a18d8084"}, {"distinct_range": 0, "num_eq": 300, "num_range": 30000, "upper_bound": "\\\\xf44d18ae63f0"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 3572601345454512087, "row_count": 3646500332026053377}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2356935773964158845, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000, "num_range": 0, "upper_bound": "-1946983378"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 6192873808312614784, "upper_bound": "-250563787"}, {"distinct_range": 37988.16234408571, "num_eq": 5745750057511358544, "num_range": 50000, "upper_bound": "0"}, {"distinct_range": 430.35225885507487, "num_eq": 600, "num_range": 800, "upper_bound": "485541818"}, {"distinct_range": 50000, "num_eq": 5676697612569418655, "num_range": 50000, "upper_bound": "1048358470"}, {"distinct_range": 0, "num_eq": 6239073644513494200, "num_range": 20000000, "upper_bound": "1309260179"}, {"distinct_range": 1677192.8680662017, "num_eq": 10000, "num_range": 2000000, "upper_bound": "1458866247"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 3646500332026053377}]';
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2804180953254611307, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7532791439393324645}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5865111483938613232, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7532791439393324645}, {"columns": ["col5_9"]

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2665937&tab=buildLog) on [master@57de93a997db31d97cd7e23fb866fe0e44f90de2](https://github.com/cockroachdb/cockroach/commits/57de93a997db31d97cd7e23fb866fe0e44f90de2):

```
        test_1        |             CREATE INDEX ON table2 (col2_15, col2_19, col2_11 DESC, col1_3 ASC, col2_14 DESC, col1_6 DESC, col2_13 DESC, col1_2, col2_12, col2_16 DESC, col1_0, col2_8, col1_5 DESC, col1_1 DESC, col2_18 DESC, col2_17 DESC, col2_10, col1_4 DESC, col2_9 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_16, col2_13 ASC, col1_0 DESC, col2_11 DESC, col1_1, col2_18 ASC, col1_6, col2_9 ASC, col1_7 DESC, col1_2 DESC, col2_19 DESC, col2_12, col2_10, col2_17 ASC, col1_4 ASC);
        test_1        |             CREATE INDEX ON table2 (col2_17, col2_12 DESC, col2_14 DESC, col2_8 DESC, col2_10 DESC, col2_13, col1_0 ASC, col1_1, col2_16 ASC, col2_19, col1_4, col2_9 ASC, col1_7 DESC, col2_18, col1_2 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_7, col1_0, col1_2);
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_7, col1_0, col1_2) REFERENCES table2 (col1_7, col1_0, col1_2) ON UPDATE CASCADE;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('fhrvav');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('hhicoc', 'edfhk', 'odcocc');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (0.75s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 CHAR NULL, col1_1 REGPROC NOT NULL, col1_2 BYTES NOT NULL, col1_3 NAME NULL, col1_4 OID NOT NULL, col1_5 FLOAT8 NULL, col1_6 TIME, col1_7 "char" NOT NULL, col1_8 BIT(36) NULL, col1_9 STRING NULL, PRIMARY KEY (col1_4 ASC, col1_2 DESC, col1_1), INDEX (col1_2 DESC, col1_1) STORING (col1_0, col1_5, col1_6, col1_8) WHERE ((((((table1.col1_7 <= '"') AND (table1.col1_2 != '\x22')) OR (table1.col1_5 < NaN)) OR (table1.col1_0 = e'\'')) OR (table1.col1_3 <= 'X')) AND (table1.col1_9 != '"')) OR (table1.col1_6 = '00:00:00'), UNIQUE (col1_9 ASC, col1_0 DESC, col1_6 DESC, col1_7 ASC, col1_2, col1_3 ASC, col1_8, col1_4, col1_5, col1_1) WHERE ((((((table1.col1_2 >= '\xe29883') OR (table1.col1_7 <= e'\U00002603')) AND (table1.col1_3 > 'X')) AND (table1.col1_5 <= -1.0)) AND (table1.col1_0 <= e'\x00')) OR (table1.col1_6 > '00:00:00')) AND (table1.col1_9 = e'\''), INDEX (col1_1 ASC) WHERE (((table1.col1_6 = '24:00:00':::TIME) AND (table1.col1_5 = (-1.0):::FLOAT8)) OR (table1.col1_0 <= '"':::STRING)) AND (table1.col1_7 <= 'X':::STRING), UNIQUE (col1_5 DESC, col1_1 ASC, col1_9, col1_6 DESC, col1_4 DESC, col1_7 DESC, col1_2 ASC, col1_8, col1_0, col1_3 DESC), INDEX (col1_5 DESC, col1_9 ASC, col1_4, col1_1, col1_8 ASC) STORING (col1_0, col1_3), UNIQUE (col1_2 ASC) STORING (col1_6) WHERE ((table1.col1_9 <= 'X':::STRING) AND (table1.col1_7 != e'\x00':::STRING)) OR (table1.col1_2 > e'\\xff':::BYTES), UNIQUE (col1_0, col1_2 DESC, col1_9, col1_4) STORING (col1_5, col1_6, col1_7, col1_8) WHERE ((table1.col1_7 <= e'\'') OR (table1.col1_0 < e'\x00')) OR (table1.col1_5 < -Inf), UNIQUE (col1_5 ASC, col1_9) STORING (col1_0, col1_7, col1_8), INDEX (col1_6 DESC) STORING (col1_3, col1_5, col1_7, col1_8) WHERE (((((table1.col1_6 = '24:00:00') AND (table1.col1_5 = -Inf)) OR (table1.col1_9 < '')) AND (table1.col1_2 < '\x27')) OR (table1.col1_3 >= e'\x00')) AND (table1.col1_0 = e'\U00002603'), FAMILY (col1_7), FAMILY (col1_3, col1_2), FAMILY (col1_4), FAMILY (col1_1, col1_8, col1_6, col1_5), FAMILY (col1_9, col1_0));
        test_1        |             CREATE TABLE table2 (col2_0 FLOAT8 NOT NULL, col2_1 DECIMAL NOT NULL, col2_2 REGNAMESPACE NOT NULL, PRIMARY KEY (col2_0 ASC, col2_2, col2_1), UNIQUE (col2_2 ASC) WHERE (table2.col2_0 > 5e-324) AND (table2.col2_1 != -1.234E+401), UNIQUE (col2_1 DESC, col2_2, col2_0), UNIQUE (col2_1 ASC, col2_2, col2_0 ASC) WHERE table2.col2_1 <= (-1.234E+401):::DECIMAL, INDEX (col2_1, col2_2 ASC) WHERE (table2.col2_1 = 1:::DECIMAL) OR (table2.col2_0 > 'NaN':::FLOAT8), INDEX (col2_0 ASC), UNIQUE (col2_2 DESC, col2_1 DESC) WHERE (table2.col2_0 != 5e-324:::FLOAT8) AND (table2.col2_1 > (-1.234E+401):::DECIMAL), INDEX (col2_0 DESC, col2_1 DESC, col2_2 DESC) WHERE (table2.col2_0 < (-1.0):::FLOAT8) AND (table2.col2_1 > '-Infinity':::DECIMAL), UNIQUE (col2_2) WHERE table2.col2_0 > '+Inf':::FLOAT8, UNIQUE (col2_1 ASC, col2_0 ASC) WHERE (table2.col2_1 <= (-1.234E+401):::DECIMAL) AND (table2.col2_0 < 3.4028234663852886e+38:::FLOAT8), FAMILY (col2_1), FAMILY (col2_2, col2_0));
        test_1        |             CREATE TABLE table3 (col1_4 OID NOT NULL, col1_2 BYTES NOT NULL, col1_1 REGPROC NOT NULL, col3_3 INTERVAL NULL, col3_4 BOX2D NOT NULL, col3_5 FLOAT4 NULL, col3_6 REGTYPE NULL, col3_7 TIMESTAMP NOT NULL, col3_8 INT8 NOT NULL, col3_9 INT8 NOT NULL, col3_10 INT8, col3_11 BOOL NULL, col3_12 UUID NULL, PRIMARY KEY (col1_4 ASC, col1_2 DESC, col1_1, col3_10 ASC, col3_12 DESC), INDEX (col3_11 ASC, col3_5 DESC, col3_3, col3_4 DESC, col3_6 DESC, col3_7, col3_10 ASC, col1_1 DESC, col3_12 ASC, col3_8, col3_9 ASC) WHERE (((((table3.col3_5 >= 3.4028234663852886e+38:::FLOAT8) AND (table3.col3_9 != (-2147483648):::INT8)) OR table3.col3_11) OR (table3.col3_8 > 127:::INT8)) OR (table3.col1_2 = e'\\x':::BYTES)) OR (table3.col3_10 >= 1:::INT8), INDEX (col3_11 DESC, col3_12 DESC, col3_7, col1_2 DESC, col1_1, col1_4 DESC, col3_4 DESC, col3_6, col3_8 ASC, col3_10 DESC) WHERE ((((table3.col3_9 != (-128):::INT8) AND table3.col3_11) OR (table3.col3_8 = (-128):::INT8)) AND (table3.col3_7 > '0001-01-01 00:00:00':::TIMESTAMP)) AND (table3.col3_10 != 32767:::INT8), INDEX (col1_2, col3_11, col1_1 DESC, col3_6, col3_4 ASC, col3_9, col1_4 DESC, col3_12 DESC) WHERE table3.col3_5 >= 0.0:::FLOAT8, UNIQUE (col3_11, col3_8 DESC, col1_4 DESC, col3_4, col3_7 ASC, col3_6 DESC, col3_10 ASC), UNIQUE (col1_4) STORING (col3_4, col3_6, col3_7, col3_9) WHERE ((((((table3.col1_2 < '\x') OR (table3.col3_9 >= -32768)) AND (table3.col3_7 < '-2000-01-01 00:00:00')) AND (table3.col3_8 > -1)) AND (table3.col3_10 <= -128)) AND (table3.col3_5 <= +Inf)) OR table3.col3_11, FAMILY (col3_3, col3_4), FAMILY (col3_9), FAMILY (col1_4), FAMILY (col3_5), FAMILY (col1_2, col3_12), FAMILY (col3_8), FAMILY (col3_11, col3_7, col1_1, col3_10), FAMILY (col3_6)) INTERLEAVE IN PARENT table1 (col1_4, col1_2, col1_1);
        test_1        |             CREATE TABLE table4 (col4_0 TIME NOT NULL, col4_1 INET NOT NULL, col4_2 BOX2D NOT NULL, col4_3 INT4 NOT NULL, col4_4 TIME NOT NULL, PRIMARY KEY (col4_3, col4_1 DESC, col4_4, col4_2 DESC), INDEX (col4_3) STORING (col4_0), INDEX (col4_1, col4_4 ASC, col4_3 DESC, col4_2, col4_0) WHERE ((table4.col4_4 = '24:00:00') OR (table4.col4_0 = '00:00:00')) OR (table4.col4_3 < 32767), UNIQUE (col4_4, col4_3 ASC, col4_2) STORING (col4_0) WHERE (table4.col4_4 = '24:00:00':::TIME) AND (table4.col4_0 < '24:00:00':::TIME), INDEX (col4_1 ASC), INDEX (col4_0 DESC, col4_1, col4_4 ASC, col4_2 DESC) WHERE ((table4.col4_3 <= 32767) AND (table4.col4_0 < '00:00:00')) AND (table4.col4_4 = '00:00:00'), FAMILY (col4_0), FAMILY (col4_3, col4_1, col4_4, col4_2));
        test_1        |             CREATE TABLE table5 (col5_0 REGNAMESPACE NOT NULL, col5_1 INT4 NOT NULL, col5_2 NAME, col5_3 DECIMAL NOT NULL, col5_4 VARCHAR NOT NULL, col5_5 INT2 NOT NULL, col5_6 UUID NOT NULL, col5_7 FLOAT8 NULL, col5_8 DECIMAL NOT NULL, PRIMARY KEY (col5_3 ASC, col5_6 ASC, col5_0 ASC, col5_4, col5_8 ASC, col5_1 DESC), UNIQUE (col5_4, col5_7, col5_0 DESC, col5_2, col5_3 DESC), INDEX (col5_6 DESC, col5_1 ASC, col5_0, col5_7 DESC, col5_8 DESC, col5_3 ASC, col5_5 DESC, col5_4) WHERE (table5.col5_1 != (-32768):::INT8) AND (table5.col5_3 < 'Infinity':::DECIMAL), INDEX (col5_6 DESC, col5_4) STORING (col5_5, col5_7) WHERE ((((table5.col5_2 >= e'\U00002603':::STRING) OR (table5.col5_3 >= (-1):::DECIMAL)) OR (table5.col5_8 < (-1.234E+401):::DECIMAL)) AND (table5.col5_5 >= 127:::INT8)) OR (table5.col5_4 > '"':::STRING), UNIQUE (col5_1 ASC, col5_2 DESC, col5_3 DESC, col5_0 ASC, col5_6) STORING (col5_5) WHERE ((((((table5.col5_2 < e'\x00':::STRING) AND (table5.col5_8 < (-1):::DECIMAL)) AND (table5.col5_1 < 1:::INT8)) OR (table5.col5_3 >= 'NaN':::DECIMAL)) AND (table5.col5_5 <= 0:::INT8)) AND (table5.col5_4 = 'X':::STRING)) OR (table5.col5_7 = 1.0:::FLOAT8), UNIQUE (col5_7) STORING (col5_2, col5_5), FAMILY (col5_4), FAMILY (col5_7), FAMILY (col5_2, col5_1), FAMILY (col5_3), FAMILY (col5_0, col5_6, col5_8), FAMILY (col5_5));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 48293584, "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 51068608, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "\\\\x68db194bd4"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34474131, "histo_col_type": "", "name": "__auto__", "null_count": 16326983, "row_count": 60000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24062555, "histo_buckets": [{"distinct_range": 0, "num_eq": 3784586614968399734, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 1739921.406113917, "num_eq": 500, "num_range": 30000000, "upper_bound": "785238421"}, {"distinct_range": 4926748001823779000, "num_eq": 1000000000, "num_range": 5179700991253474078, "upper_bound": "2206801638"}, {"distinct_range": 50, "num_eq": 100000, "num_range": 50, "upper_bound": "2763957242"}, {"distinct_range": 54568578.594363056, "num_eq": 345010327974837796, "num_range": 60000000, "upper_bound": "3331823030"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39759385, "histo_buckets": [{"distinct_range": 0, "num_eq": 3017338264515596744, "num_range": 0, "upper_bound": "01:26:50.808801"}, {"distinct_range": 0, "num_eq": 2309767441527128161, "num_range": 5878606462847227724, "upper_bound": "03:45:40.395943"}, {"distinct_range": 1000, "num_eq": 68740431129424967, "num_range": 1000, "upper_bound": "17:43:56.917351"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 54050712, "row_count": 60000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45639975, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 51877380, "histo_buckets": [{"distinct_range": 0, "num_eq": 90, "num_range": 0, "upper_bound": ""}, {"distinct_range": 7622601283.402488, "num_eq": 1488294547691859760, "num_range": 10000000000, "upper_bound": "\\u0017a\\u001a\\u0017\\u001b"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 0, "upper_bound": "\\u001d\\u0001\\u0016\\u0001j\\u0011;\\u00075"}, {"distinct_range": 0, "num_eq": 6308587815940695647, "num_range": 300, "upper_bound": "\\u001eH\\"\\"Yr"}, {"distinct_range": 0, "num_eq": 6948845202765365042, "num_range": 5523724753281843821, "upper_bound": "\\""}, {"distinct_range": 70000, "num_eq": 40000, "num_range": 70000, "upper_bound": "$K\\u0018\'p\\u000bq1"}, {"distinct_range": 1000000, "num_eq": 70000, "num_range": 1000000, "upper_bound": "&g"}, {"distinct_range": 1660195069893544400, "num_eq": 300000, "num_range": 1660195069893544373, "upper_bound": "riVmvw\\n"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 35348786, "row_count": 60000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46920374, "histo_buckets": [{"distinct_range": 0, "num_eq": 3553966875409573027, "num_range": 0, "upper_bound": "\\u0002"}, {"distinct_range": 350.75910959677844, "num_eq": 400, "num_range": 6000, "upper_bound": ","}, {"distinct_range": 1611500993287469000, "num_eq": 7537653705796516717, "num_range": 1611500993287469060, "upper_bound": "1"}, {"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "G"}, {"distinct_range": 0, "num_eq": 100, "num_range": 800000, "upper_bound": "L"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 58618334, "row_count": 60000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24448551, "histo_buckets": [{"distinct_range": 0, "num_eq": 8043384558079995613, "num_range": 0, "upper_bound": "-0.2983224932020795"}, {"distinct_range": 47.39767024736654, "num_eq": 1319724545098756903, "num_range": 80, "upper_bound": "-0.24808425761426117"}, {"distinct_range": 100000, "num_eq": 30000, "num_range": 100000, "upper_bound": "0.05624345637578426"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 400000000, "upper_bound": "0.060051203224489036"}, {"distinct_range": 30000000, "num_eq": 60, "num_range": 30000000, "upper_bound": "1.109544677948865"}, {"distinct_range": 0, "num_eq": 30, "num_range": 1501029389313847114, "upper_bound": "1.67771945794135"}, {"distinct_range": 0, "num_eq": 667158076250815400, "num_range": 100, "upper_bound": "1.7976931348623157e+308"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 39358963, "row_count": 60000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20746476, "histo_col_type": "", "name": "__auto__", "null_count": 39485012, "row_count": 60000000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 902257814845942557, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000000, "num_range": 0, "upper_bound": "-Inf"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 3808251309809347508, "upper_bound": "0.07956960062096968"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 3919664850896956514}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1254788864041485417, "histo_buckets": [{"distinct_range": 0, "num_eq": 7245242132353837076, "num_range": 0, "upper_bound": "-9639405021744.08933"}, {"distinct_range": 10000000, "num_eq": 3771746830639449301, "num_range": 10000000, "upper_bound": "148360.8371949596104"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 5477971645821730914, "upper_bound": "385648436376.0337556"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 3919664850896956514}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2149270040114897798, "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 3919664850896956514}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 912, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1366, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1484, "histo_col_type": "", "name": "__auto__", "null_count": 1012, "row_count": 2000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 182, "histo_buckets": [{"distinct_range": 0, "num_eq": 90, "num_range": 0, "upper_bound": "\\\\x26df8b84dd"}, {"distinct_range": 4434104509988408000, "num_eq": 80000, "num_range": 4867934474555478089, "upper_bound": "\\\\x39faf0759bd998"}, {"distinct_range": 445057460.67769367, "num_eq": 0, "num_range": 500000000, "upper_bound": "\\\\x5a08c1e4c45869c6"}, {"distinct_range": 0, "num_eq": 400, "num_range": 4484527799179682313, "upper_bound": "\\\\xb2a1ac"}, {"distinct_range": 0, "num_eq": 50000, "num_range": 1000000, "upper_bound": "\\\\xbf66c8292c75"}, {"distinct_range": 2736610667421951000, "num_eq": 0, "num_range": 6357798232645869025, "upper_bound": "\\\\xf7247c00"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 692, "histo_col_type": "", "name": "__auto__", "null_count": 857, "row_count": 2000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 887, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3, "histo_col_type": "", "name": "__auto__", "null_count": 1299, "row_count": 2000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1908, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1512, "histo_col_type": "BOOL", "name": "__auto__", "null_count": 1251, "row_count": 2000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1696, "histo_buckets": [{"distinct_range": 0, "num_eq": 5429273757678118588, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 32837647496.1073, "num_eq": 2060605838376101327, "num_range": 90000000000, "upper_bound": "301853891"}, {"distinct_range": 4719755710931635000, "num_eq": 8437058065949929644, "num_range": 4719755710931634732, "upper_bound": "655928384"}, {"distinct_range": 614822176.1512501, "num_eq": 4000000000, "num_range": 700000000, "upper_bound": "670270980"}, {"distinct_range": 1725288955271823600, "num_eq": 5826450681440659148, "num_range": 8005447348747573951, "upper_bound": "2347345942"}, {"distinct_range": 0, "num_eq": 110493788171282647, "num_range": 50000000000, "upper_bound": "2404372151"}, {"distinct_range": 5226.092221714258, "num_eq": 70, "num_range": 9000, "upper_bound": "2514553162"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 0, "upper_bound": "2991067114"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1660, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1595, "histo_col_type": "", "name": "__auto__", "null_count": 168, "row_count": 2000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1062, "histo_col_type": "", "name": "__auto__", "null_count": 1204, "row_count": 2000}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2342124894189111295, "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 2552164709173650844}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2455266008558203800, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "e3ee:17b4:495e:9469:3e31:4d5f:e0e6:ca95/71"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 2552164709173650844}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2047682363211278599, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2552164709173650844}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1368766903439439706, "histo_buckets": [{"distinct_range": 0, "num_eq": 7128675585592818068, "num_range": 0, "upper_bound": "-154966078"}, {"distinct_range": 48823053.121388555, "num_eq": 200000000, "num_range": 500000000, "upper_bound": "1241632734"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 388712870991547227, "upper_bound": "1887598996"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 2552164709173650844}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 974775611912559052, "histo_buckets": [{"distinct_range": 0, "num_eq": 6816195950963036811, "num_range": 0, "upper_bound": "15:50:26.581302"}, {"distinct_range": 2462460957964539400, "num_eq": 1118336134114856287, "num_range": 2462460957964539628, "upper_bound": "15:58:08.973133"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "18:12:04.789059"}, {"distinct_range": 2999137347864993300, "num_eq": 1829519072095845462, "num_range": 2999137347864993181, "upper_bound": "21:54:48.634793"}, {"distinct_range": 1050613389855602600, "num_eq": 8288006183377377011, "num_range": 4013107376073422638, "upper_bound": "23:01:41.846517"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 2552164709173650844}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS e'[{"columns": ["col5_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 513451039401751087, "histo_buckets": [{"distinct_range": 0, "num_eq": 428240511688942020, "num_range": 0, "upper_bound": "0191e677-bbbd-447c-9389-ae3ad27b1a0a"}, {"distinct_range": 2555011538536424000, "num_eq": 900000000, "num_range": 3951601381518274395, "upper_bound": "24c34a4f-8131-4cb5-9426-8c5e1b37349c"}, {"distinct_range": 4834238523214208000, "num_eq": 50, "num_range": 4834238523214208039, "upper_bound": "2c161f4b-60fb-4f0a-a05c-c19ba2d00914"}, {"distinct_range": 1310652751637461000, "num_eq": 6179728345444411327, "num_range": 2942111892022492618, "upper_bound": "3bd320e6-c351-46bc-bc59-fd38add3522a"}, {"distinct_range": 430192084180344600, "num_eq": 100000, "num_range": 2383670506545133175, "upper_bound": "b4f049ea-0ce3-463c-a4b7-04a2c4e6cc2b"}, {"distinct_range": 0, "num_eq": 2648752051031350466, "num_range": 5196314781885508141, "upper_bound": "beac2c04-aeb1-4bf6-a722-8032578edf84"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 933334413034165206}, {"columns": ["col5_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 740118240896932622, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 933334413034165206}, {"columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 157903752440985816, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 933334413034165206}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 567766468372783094, "histo_col_type": "", "name": "__auto__", "null_count": 856461997545780373, "row_count": 933334413034165206}, {"columns": ["col5_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 576592456069059732, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 933334413034165206}, {"columns": ["col5_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 191137047404422177, "histo_buckets": [{"distinct_range": 0, "num_eq": 8312143394083516102, "num_range": 0, "upper_bound": "0.026028423517778787"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 30, "upper_bound": "2.3752880392823714"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 855208315461367565, "row_count": 933334413034165206}, {"columns": ["col5_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 50061279192106672, "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 933334413034165206}, {"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 923640193486873598, "histo_buckets": [{"distinct_range": 0, "num_eq": 544188967108672213, "num_range": 0, "upper_bound": "NaN"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 700000, "upper_bound": "-7.912182138507187622E+35"}, {"distinct_range": 1793174960.7322927, "num_eq": 7648785888468886302, "num_range": 8000000000, "upper_bound": "-6.59078141337049535E+31"}, {"distinct_range": 0, "num_eq": 3978905362458220668, "num_range": 6559364450125997103, "upper_bound": "-8.072946629599308546E+25"}, {"distinct_range": 90, "num_eq": 800, "num_range": 90, "upper_bound": "-538604666547394.3447"}, {"distinct_range": 446361565448477900, "num_eq": 6000000000, "num_range": 6929912122903846793, "upper_bound": "0.301983031676058385"}, {"distinct_range": 0, "num_eq": 6202705126906345284, "num_range": 5436301167540580071, "upper_bound": "8.21593642198136366"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 933334413034165206}, {"columns": ["col5_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 418042375944691692, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 4481427927452103088, "num_range": 8000, "upper_bound": "Ad\\"s3X%^"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 10000000000, "upper_bound": "[\\u000biv"}, {"distinct_range": 700, "num_eq": 8014122774467313706, "num_range": 700, "upper_bound": "`^g@\\u0006fI"}, {"distinct_range": 304690508676449600, "num_eq": 8000000000, "num_range": 929929418346643222, "upper_bound": "g\\u0008/Q)u:\\u0013"}, {"distinct_range": 788084554.7277807, "num_eq": 50000, "num_range": 2000000000, "upper_bound": "n"}, {"distinct_range": 4595008548860664300, "num_eq": 8665768086491083558, "num_range": 4595008548860664084, "upper_bound": "u_yVY"}, {"distinct_range": 0, "num_eq": 5860363309908990661, "num_range": 900000, "upper_bound": "{~\\u0019&L?\\u000e0d"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 933334413034165206}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('lu', 'gsgz', 'z');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('jueh', 'in', 'la');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('ub', 'acljdv', 'hna');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10, "histo_buckets": [{"distinct_range": 0, "num_eq": 1919445021961967435, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 8228257447602953482, "upper_bound": "1353255522"}, {"distinct_range": 2185469798541373400, "num_eq": 2854780758141331000, "num_range": 3232871155855087235, "upper_bound": "1606849469"}, {"distinct_range": 0, "num_eq": 900, "num_range": 6000, "upper_bound": "2658558158"}, {"distinct_range": 0, "num_eq": 7237147019830233348, "num_range": 8298664654984790778, "upper_bound": "2774801127"}, {"distinct_range": 1560407435328793300, "num_eq": 400000, "num_range": 3791006694648272604, "upper_bound": "2827020779"}, {"distinct_range": 8011443383854509000, "num_eq": 9000, "num_range": 8011443383854509301, "upper_bound": "3716601923"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": "1528867709"}, {"distinct_range": 8054730817333117000, "num_eq": 6000000, "num_range": 8054730817333117354, "upper_bound": "1863940929"}, {"distinct_range": 0, "num_eq": 8807405486922431423, "num_range": 0, "upper_bound": "4007443747"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "-1.1654177123280745"}, {"distinct_range": 0, "num_eq": 3000000, "num_range": 8000, "upper_bound": "-0.8222586954027029"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 25, "row_count": 60}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23, "histo_col_type": "TIME", "name": "__auto__", "null_count": 22, "row_count": 60}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 58, "histo_buckets": [{"distinct_range": 0, "num_eq": 400, "num_range": 0, "upper_bound": ""}, {"distinct_range": 100000000, "num_eq": 373356770166489685, "num_range": 100000000, "upper_bound": "\\u0001#f?@K"}, {"distinct_range": 0, "num_eq": 2763530040755073432, "num_range": 400000, "upper_bound": "\\u0004)S"}, {"distinct_range": 0, "num_eq": 0, "num_range": 200000, "upper_bound": "\\u001e@\\""}, {"distinct_range": 1000000000, "num_eq": 2876790364605902492, "num_range": 1000000000, "upper_bound": "\\""}, {"distinct_range": 862774537663765900, "num_eq": 100000, "num_range": 2666396346337674285, "upper_bound": "b\\u0010@;"}, {"distinct_range": 4304533648953181700, "num_eq": 2473405988236051049, "num_range": 4304533648953181780, "upper_bound": "c\\u0008b:C\\u00024\\u0002\\u000b"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 31, "row_count": 60}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28, "histo_buckets": [{"distinct_range": 0, "num_eq": 9033620231899806894, "num_range": 0, "upper_bound": "\\u0007"}, {"distinct_range": 0, "num_eq": 5126360495380280957, "num_range": 2499257979655147483, "upper_bound": "\\r"}, {"distinct_range": 0, "num_eq": 100, "num_range": 7337167916822353768, "upper_bound": "\\u0013"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 4000000000, "upper_bound": "$"}, {"distinct_range": 94732487955169040, "num_eq": 8243240827183254232, "num_range": 155545904694184507, "upper_bound": "\'"}, {"distinct_range": 2000000, "num_eq": 6329126377813045283, "num_range": 2000000, "upper_bound": ")"}, {"distinct_range": 2110172324539039200, "num_eq": 5050210277526034481, "num_range": 2539396328574259345, "upper_bound": "G"}, {"distinct_range": 80.66957045110642, "num_eq": 700000000, "num_range": 90, "upper_bound": "Z"}, {"distinct_range": 500000000, "num_eq": 968203240274224592, "num_range": 500000000, "upper_bound": "\U00002603"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 15, "row_count": 60}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4, "histo_buckets": [{"distinct_range": 0, "num_eq": 3828357312110834625, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 0, "num_eq": 3241679409113629851, "num_range": 0, "upper_bound": "\\\\x588f445d8576bd7c"}, {"distinct_range": 0, "num_eq": 800000, "num_range": 400000000, "upper_bound": "\\\\xa89347f7d6"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33, "histo_col_type": "", "name": "__auto__", "null_count": 54, "row_count": 60}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42, "histo_col_type": "", "name": "__auto__", "null_count": 34, "row_count": 60}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1060, "histo_buckets": [{"distinct_range": 0, "num_eq": 70, "num_range": 0, "upper_bound": "-3.735750410986688127E+29"}, {"distinct_range": 30.05516384079056, "num_eq": 592813184291028333, "num_range": 5000, "upper_bound": "-7.88769763602986528E+26"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 5345073412058909123, "upper_bound": "7.016770023897100508"}, {"distinct_range": 6353638986468059000, "num_eq": 8000, "num_range": 6353638986468058807, "upper_bound": "63570.04396802960308"}, {"distinct_range": 3770275764014558000, "num_eq": 30000, "num_range": 3770275764014558462, "upper_bound": "8354570214.215888322"}, {"distinct_range": 11834546071.484129, "num_eq": 800, "num_range": 40000000000, "upper_bound": "6.387220507719085772E+32"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 3000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 503, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": "573514619"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 3000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2251, "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 3000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5074714, "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2504513, "histo_buckets": [{"distinct_range": 0, "num_eq": 3995325271835730127, "num_range": 0, "upper_bound": "\\\\x0326"}, {"distinct_range": 4432587904255770600, "num_eq": 6569877908428021921, "num_range": 7130851699479503337, "upper_bound": "\\\\x27"}, {"distinct_range": 5000000, "num_eq": 10000, "num_range": 5000000, "upper_bound": "\\\\x30b5"}, {"distinct_range": 255866.26896305988, "num_eq": 3851821221756411553, "num_range": 300000, "upper_bound": "\\\\x3c3a8d0befb35ea7"}, {"distinct_range": 2361879484814592000, "num_eq": 0, "num_range": 2361879484814592097, "upper_bound": "\\\\x7f62523b652170"}, {"distinct_range": 177789175732394340, "num_eq": 3710570630123508148, "num_range": 522773879858148160, "upper_bound": "\\\\x9e"}, {"distinct_range": 9036.21440804441, "num_eq": 9180253368896811662, "num_range": 10000, "upper_bound": "\\\\xa4eb5fb7"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 3948884287892214332, "upper_bound": "\\\\xe2895419f0"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13422372, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17587715, "histo_col_type": "", "name": "__auto__", "null_count": 12584588, "row_count": 20000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16370805, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17854220, "histo_col_type": "", "name": "__auto__", "null_count": 17423668, "row_count": 20000000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16623478, "histo_col_type": "", "name": "__auto__", "null_count": 16941818, "row_count": 20000000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13627055, "histo_buckets": [{"distinct_range": 0, "num_eq": 400, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 2467582999112651593, "num_range": 3021400966379735113, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 9653212, "row_count": 20000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17716999, "histo_col_type": "", "name": "__auto__", "null_count": 1611286, "row_count": 20000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10287095, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2928498, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18852052, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15598101, "histo_col_type": "", "name": "__auto__", "null_count": 3648021, "row_count": 20000000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "00:38:02.772723"}, {"distinct_range": 0, "num_eq": 41642979172175296, "num_range": 3443445002137921734, "upper_bound": "10:38:05.729219"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 10000000000, "upper_bound": "11:15:58.099903"}, {"distinct_range": 8548891171559399000, "num_eq": 70000, "num_range": 8548891171559399561, "upper_bound": "22:27:51.762629"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "129.165.94.241/6"}, {"distinct_range": 5641378441227689000, "num_eq": 6056205095898915850, "num_range": 5641378441227689174, "upper_bound": "3a6e:4ff0:442f:a89f:eff0:b991:ef8d:1de8/5"}, {"distinct_range": 0, "num_eq": 9131185701348271059, "num_range": 0, "upper_bound": "e88c:d438:7dea:377c:2ae8:ca7b:68a4:86da/51"}, {"distinct_range": 1000000, "num_eq": 400, "num_range": 1000000, "upper_bound": "2634:b6f2:9ee9:2201:41cc:9ade:9de8:287b/58"}, {"distinct_range": 3988760189849150000, "num_eq": 7394450120821136969, "num_range": 6018348720031733300, "upper_bound": "7669:82e3:b1db:a535:2941:2ad7:80a6:92f5/65"}, {"distinct_range": 1000000000, "num_eq": 600000, "num_range": 1000000000, "upper_bound": "832c:2b9d:fee8:8844:f498:7467:c2be:2831/97"}, {"distinct_range": 747528748.4057741, "num_eq": 7000000000, "num_range": 1000000000, "upper_bound": "9dc2:7158:cfc0:4dc9:23a2:96ab:fc8c:62e5/124"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "02:13:19.677526"}, {"distinct_range": 500000, "num_eq": 5000000000, "num_range": 500000, "upper_bound": "04:02:18.509372"}, {"distinct_range": 60000, "num_eq": 900000000, "num_range": 60000, "upper_bound": "04:50:07.695949"}, {"distinct_range": 6405252354.113585, "num_eq": 300000000, "num_range": 8000000000, "upper_bound": "06:37:33.742964"}, {"distinct_range": 1904349151080614700, "num_eq": 80, "num_range": 2990197787191408815, "upper_bound": "08:14:39.142688"}, {"distinct_range": 1000000000, "num_eq": 2132655381804708692, "num_range": 1000000000, "upper_bound": "08:36:56.224588"}, {"distinct_range": 50000, "num_eq": 500000, "num_range": 50000, "upper_bound": "11:56:09.152132"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 4159431622300601380, "upper_bound": "15:26:44.663216"}, {"distinct_range": 2645880781376813000, "num_eq": 80000000000, "num_range": 2854064775865584711, "upper_bound": "16:18:35.790899"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table5 INJECT STATISTICS e'[{"columns": ["col5_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2645, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 100000000, "num_eq": 1562908240365048317, "num_range": 100000000, "upper_bound": "\\u001flel>"}, {"distinct_range": 2557395015444109000, "num_eq": 0, "num_range": 5148122672868811843, "upper_bound": "+^Mz\\u0004.l\\u0018z"}, {"distinct_range": 20.765057138749093, "num_eq": 4672455847038214835, "num_range": 200, "upper_bound": "7X"}, {"distinct_range": 62.26523544716125, "num_eq": 184319247206867495, "num_range": 90, "upper_bound": "B\\u0019G_\\u0004"}, {"distinct_range": 589798103931838200, "num_eq": 1000000000, "num_range": 5333800628886159820, "upper_bound": "h"}, {"distinct_range": 0, "num_eq": 800000, "num_range": 7976214662377035675, "upper_bound": "sn(\\u0001q"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 40000, "upper_bound": "x"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 100000, "upper_bound": "\U00002603"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col5_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2515, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col5_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1146, "histo_buckets": [{"distinct_range": 0, "num_eq": 10, "num_range": 0, "upper_bound": "00000000-0000-0000-0000-000000000000"}, {"distinct_range": 0, "num_eq": 2178591404549604275, "num_range": 7000583761296108230, "upper_bound": "21658bfb-18d4-4ee0-b254-ec26b15573aa"}, {"distinct_range": 528185379202278500, "num_eq": 6702404341612769783, "num_range": 528185379202278510, "upper_bound": "81c73ee6-80d6-4248-9b5f-fc5917f643e8"}, {"distinct_range": 1000000000, "num_eq": 3752791747031919101, "num_range": 1000000000, "upper_bound": "8f5aed41-32f0-4272-8764-2df98e775b25"}, {"distinct_range": 0, "num_eq": 20000000, "num_range": 600, "upper_bound": "a3951098-e0ad-4049-a297-cb9f1939534b"}, {"distinct_range": 4770756323415049000, "num_eq": 50000000, "num_range": 6651169731581655951, "upper_bound": "c2992bf2-7cfa-4f13-99ab-97a4722d0274"}, {"distinct_range": 1634868129020758300, "num_eq": 2000000000, "num_range": 1634868129020758272, "upper_bound": "c29fed2f-4d0f-4401-ab2e-444503706346"}, {"distinct_range": 0, "num_eq": 9000000, "num_range": 0, "upper_bound": "ffffffff-ffff-ffff-ffff-ffffffffffff"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col5_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3334, "histo_buckets": [{"distinct_range": 0, "num_eq": 3280469325039344282, "num_range": 0, "upper_bound": "-0.8647175685856731"}, {"distinct_range": 3000, "num_eq": 9000000, "num_range": 3000, "upper_bound": "-0.6693520977723963"}, {"distinct_range": 0, "num_eq": 264915399868990335, "num_range": 7483630632324492517, "upper_bound": "-0.5746015692760237"}, {"distinct_range": 1769.9565655780066, "num_eq": 100, "num_range": 4000, "upper_bound": "-0.3993068520781587"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 1333, "row_count": 4000}, {"columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3852, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col5_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3964, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "-1765114229"}, {"distinct_range": 25976.12209270608, "num_eq": 50000000000, "num_range": 100000, "upper_bound": "-1706073463"}, {"distinct_range": 592702436471642600, "num_eq": 8630881629491508192, "num_range": 2557733160784869531, "upper_bound": "-1112030921"}, {"distinct_range": 0, "num_eq": 7159029684728574901, "num_range": 6000, "upper_bound": "-1093165468"}, {"distinct_range": 0, "num_eq": 6360448733462554582, "num_range": 4229556907899228005, "upper_bound": "-953957034"}, {"distinct_range": 600, "num_eq": 307252265437579769, "num_range": 600, "upper_bound": "-686674077"}, {"distinct_range": 0, "num_eq": 5000000000, "num_range": 8000, "upper_bound": "1530134999"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 100, "upper_bound": "1776914036"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 148, "histo_col_type": "", "name": "__auto__", "null_count": 2633, "row_count": 4000}, {"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2812, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000000, "num_range": 0, "upper_bound": "-0.02140939124748890175"}, {"distinct_range": 4478541845425275400, "num_eq": 6350139677277413730, "num_range": 4478541845425275483, "upper_bound": "-0.00463667805594865582"}, {"distinct_range": 669534800892655600, "num_eq": 40000, "num_range": 3116711845115558509, "upper_bound": "0"}, {"distinct_range": 2111440511612356400, "num_eq": 5678244721529866485, "num_range": 2996308782901718216, "upper_bound": "0.0902399717933827366"}, {"distinct_range": 4000000000, "num_eq": 8331446769045110944, "num_range": 4000000000, "upper_bound": "545537515096607431.2"}, {"distinct_range": 4527435191839892000, "num_eq": 6676732919919744986, "num_range": 4527435191839891928, "upper_bound": "4.618785501132298921E+20"}, {"distinct_range": 1289426975577508000, "num_eq": 70, "num_range": 1289426975577508065, "upper_bound": "1.620662346221221265E+26"}, {"distinct_range": 2094475876686669600, "num_eq": 3315967915641134240, "num_range": 2094475876686669552, "upper_bound": "3.173151667696547768E+33"}, {"distinct_range": 650566.067560372, "num_eq": 4182538505368163330, "num_range": 800000, "upper_bound": "8.18398416281072647E+35"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col5_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 909, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}]';
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_2) REFERENCES table3 (col1_4) ON UPDATE RESTRICT;
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2813086784184023269, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 800, "num_range": 4416623192464260611, "upper_bound": "0"}, {"distinct_range": 855702269.4919037, "num_eq": 50000, "num_range": 10000000000, "upper_bound": "N"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 5153215386150304255, "row_count": 7081165066906209596}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1685797535561567774, "histo_buckets": [{"distinct_range": 0, "num_eq": 4330586500292655083, "num_range": 0, "upper_bound": "-1.7852782336642534"}, {"distinct_range": 2628482757585093000, "num_eq": 5645649145931011847, "num_range": 2628482757585093102, "upper_bound": "-1.0431684745179193"}, {"distinct_range": 4000000, "num_eq": 1234967280127898429, "num_range": 4000000, "upper_bound": "-0.6701806383937174"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 70000000000, "upper_bound": "-0.618148936409907"}, {"distinct_range": 90, "num_eq": 3729125681110314106, "num_range": 90, "upper_bound": "0.13072328951865458"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 3024907807504023634, "row_count": 7081165066906209596}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3606175983252840505, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7081165066906209596}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 788069810869616550, "histo_col_type": "", "name": "__auto__", "null_count": 4596936686516626639, "row_count": 7081165066906209596}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 422680919886615063, "histo_col_type": "STRING", "name": "__auto__", "null_count": 1516703201078681342, "row_count": 7081165066906209596}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1375388920589470578, "histo_buckets": [{"distinct_range": 0, "num_eq": 5297830248485761773, "num_range": 0, "upper_bound": "1356570863"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 7081165066906209596}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5273485190428296051, "histo_buckets": [{"distinct_range": 0, "num_eq": 5413253918926269226, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 60000000, "num_eq": 10000, "num_range": 60000000, "upper_bound": "\\\\x2aa45e958f63f173"}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 9000, "upper_bound": "\\\\x407460"}, {"distinct_range": 8187070931580318000, "num_eq": 2977325252925979051, "num_range": 8187070931580317660, "upper_bound": "\\\\x48df62"}, {"distinct_range": 50000, "num_eq": 900, "num_range": 50000, "upper_bound": "\\\\xace304d86a80d3b7"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 7081165066906209596}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2352171695186466837, "histo_col_type": "", "name": "__auto__", "null_count": 4706684864391345183, "row_count": 7081165066906209596}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 447189704140868585, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": "135934359"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 700000000, "upper_bound": "958065904"}, {"distinct_range": 622.0344172448666, "num_eq": 7496076330777170272, "num_range": 10000, "upper_bound": "1875465848"}, {"distinct_range": 5000000000, "num_eq": 0, "num_range": 5000000000, "upper_bound": "2887434268"}, {"distinct_range": 7555574086937059000, "num_eq": 1000000000, "num_range": 7555574086937059670, "upper_bound": "3711741573"}, {"distinct_range": 5591547894478134000, "num_eq": 600, "num_range": 5591547894478134327, "upper_bound": "3965712096"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 7081165066906209596}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 169378118081696448, "histo_buckets": [{"distinct_range": 0, "num_eq": 1160638332461342597, "num_range": 0, "upper_bound": "02:57:30.639605"}, {"distinct_range": 59675.95403361119, "num_eq": 1000, "num_range": 90000, "upper_bound": "08:03:09.96542"}, {"distinct_range": 8391945587585000000, "num_eq": 1000000, "num_range": 8391945587585000809, "upper_bound": "09:05:41.017932"}, {"distinct_range": 700000, "num_eq": 171641943227687023, "num_range": 700000, "upper_bound": "09:52:53.767485"}, {"distinct_range": 9018712258673156000, "num_eq": 1262207558275015230, "num_range": 9018712258673156451, "upper_bound": "20:10:17.744806"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 2596181560059865178, "row_count": 7081165066906209596}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1631831, "histo_buckets": [{"distinct_range": 0, "num_eq": 500000000, "num_range": 0, "upper_bound": "-1.3384872054325803"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 100000, "upper_bound": "-1.1376765358727425"}, {"distinct_range": 10000000000, "num_eq": 2137485904325319502, "num_range": 10000000000, "upper_bound": "-0.4005829557935946"}, {"distinct_range": 4750241379539623000, "num_eq": 7896569617610773342, "num_range": 4750241379539622409, "upper_bound": "2.272994217179054"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1183118, "histo_buckets": [{"distinct_range": 0, "num_eq": 6688751670928106278, "num_range": 0, "upper_bound": "-6.747180343728508962E+36"}, {"distinct_range": 70000000, "num_eq": 8250943607739128150, "num_range": 70000000, "upper_bound": "-5.76076756297398689E+23"}, {"distinct_range": 7397891253696808000, "num_eq": 0, "num_range": 7722086933416972159, "upper_bound": "-3.268307617967356693E+22"}, {"distinct_range": 10000000000, "num_eq": 80, "num_range": 10000000000, "upper_bound": "-23610.92028799385337"}, {"distinct_range": 2000, "num_eq": 1000000, "num_range": 2000, "upper_bound": "887013840982809953.5"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 7000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1523271, "histo_buckets": [{"distinct_range": 0, "num_eq": 70, "num_range": 0, "upper_bound": "766661467"}, {"distinct_range": 0, "num_eq": 600, "num_range": 1000000, "upper_bound": "2380320139"}, {"distinct_range": 6642199060766096000, "num_eq": 8000000, "num_range": 6642199060766096610, "upper_bound": "3875957169"}, {"distinct_range": 100000000000, "num_eq": 1244445034090920591, "num_range": 100000000000, "upper_bound": "3901268037"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 7000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1980589896347742515, "histo_col_type": "", "name": "__auto__", "null_count": 2168980048420891318, "row_count": 4253482300855963091}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3667222181457467869, "histo_col_type": "", "name": "__auto__", "null_count": 3209824698281717456, "row_count": 4253482300855963091}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4156584294938500057, "histo_col_type": "", "name": "__auto__", "null_count": 1921878728246763331, "row_count": 4253482300855963091}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2259142840057104679, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 300000000, "num_eq": 50, "num_range": 300000000, "upper_bound": "395536499"}, {"distinct_range": 0, "num_eq": 2853413864550023690, "num_range": 3000000000, "upper_bound": "2134501556"}, {"distinct_range": 1095.5146280769122, "num_eq": 1323045570265097523, "num_range": 5000, "upper_bound": "3933520571"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 4253482300855963091}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2369648616830505522, "histo_buckets": [{"distinct_range": 0, "num_eq": 30, "num_range": 0, "upper_bound": "\\\\x1458"}, {"distinct_range": 551760293434242240, "num_eq": 6000, "num_range": 551760293434242218, "upper_bound": "\\\\x52abf88a7c72f4be0f"}, {"distinct_range": 406452675599866400, "num_eq": 4000000000, "num_range": 1497709240968777839, "upper_bound": "\\\\x5a31516a833f9b61"}, {"distinct_range": 6130984485407479000, "num_eq": 3000000000, "num_range": 6130984485407478408, "upper_bound": "\\\\x7184fb1f5d"}, {"distinct_range": 3000000, "num_eq": 3411631923695750274, "num_range": 3000000, "upper_bound": "\\\\xadbe40a1838f13570c"}, {"distinct_range": 8602887171730138000, "num_eq": 100000, "num_range": 8602887171730137927, "upper_bound": "\\\\xbdadcf"}, {"distinct_range": 0, "num_eq": 7482811599965455158, "num_range": 0, "upper_bound": "\\\\xe29883"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 4253482300855963091}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1576322580286348702, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4253482300855963091}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1120249553106137956, "histo_col_type": "", "name": "__auto__", "null_count": 2611574278778187013, "row_count": 4253482300855963091}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2897280233357763306, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4253482300855963091}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1063051463155790504, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000, "num_range": 0, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 3135286585830311670, "row_count": 4253482300855963091}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3777429497443124040, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4253482300855963091}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4155042107669579988, "histo_col_type": "", "name": "__auto__", "null_count": 832486568538086525, "row_count": 4253482300855963091}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1402790788913318500, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4253482300855963091}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3298670596297660210, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4253482300855963091}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2935988008830743903, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8866354461008642112}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1366919473885055196, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000000, "num_range": 0, "upper_bound": "-858586299"}, {"distinct_range": 413915200826668160, "num_eq": 2000000, "num_range": 413915200826668180, "upper_bound": "-546534110"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 5749855081888547099, "upper_bound": "1162383082"}, {"distinct_range": 0, "num_eq": 0, "num_range": 90000, "upper_bound": "1510382004"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 8866354461008642112}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 52728838580782330, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "05:49:59.429122"}, {"distinct_range": 100000000, "num_eq": 9162806505673182377, "num_range": 100000000, "upper_bound": "21:57:28.374555"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 8866354461008642112}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1979703706822131643, "histo_buckets": [{"distinct_range": 0, "num_eq": 6567750592963269740, "num_range": 0, "upper_bound": "01:17:07.603547"}, {"distinct_range": 50.25564289722668, "num_eq": 4866799604560016022, "num_range": 100, "upper_bound": "04:08:23.430533"}, {"distinct_range": 0, "num_eq": 9000000, "num_range": 1249472517557762261, "upper_bound": "07:01:36.007249"}, {"distinct_range": 3000000000, "num_eq": 70, "num_range": 3000000000, "upper_bound": "11:49:53.521126"}, {"distinct_range": 377240336.8319479, "num_eq": 50000000, "num_range": 500000000, "upper_bound": "13:03:11.718631"}, {

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2670115&tab=buildLog) on [master@e9e372122a2e3db7090b5705da07128f828e2441](https://github.com/cockroachdb/cockroach/commits/e9e372122a2e3db7090b5705da07128f828e2441):

```
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        test_1        | --- FAIL: TestCompare (1.35s)
        test_1        |     --- FAIL: TestCompare/postgres (0.75s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.60s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 BYTES NOT NULL, col1_1 TIMETZ NULL, col1_2 REGCLASS, col1_3 INET NOT NULL, col1_4 BOX2D NOT NULL, col1_5 REGNAMESPACE NOT NULL, col1_6 DATE NOT NULL, col1_7 DECIMAL, col1_8 OID, col1_9 INT2 NULL, col1_10 DECIMAL, col1_11 REGCLASS, PRIMARY KEY (col1_5 DESC, col1_6 ASC), INDEX (col1_5 ASC, col1_1 DESC, col1_3 DESC, col1_2 ASC) STORING (col1_7, col1_10), INDEX (col1_9, col1_5 ASC) STORING (col1_1, col1_2, col1_4, col1_7, col1_8), INDEX (col1_1, col1_3, col1_10, col1_2, col1_5, col1_4, col1_0 DESC, col1_11 DESC, col1_9 DESC) STORING (col1_7), INDEX (col1_2, col1_9 DESC), UNIQUE (col1_4 ASC, col1_9 ASC, col1_8 ASC, col1_1 DESC, col1_11, col1_3, col1_0 DESC, col1_6 ASC, col1_7 ASC, col1_2, col1_10 ASC, col1_5), UNIQUE (col1_2, col1_5) WHERE (((table1.col1_7 != 1) OR (table1.col1_10 < NaN)) AND (table1.col1_1 > '00:00:00+15:59:00')) OR (table1.col1_0 <= '\xff'), FAMILY (col1_5), FAMILY (col1_3, col1_0, col1_11, col1_1, col1_2, col1_10), FAMILY (col1_4), FAMILY (col1_9), FAMILY (col1_8), FAMILY (col1_6, col1_7));
        test_1        |             CREATE TABLE table2 (col1_5 REGNAMESPACE NOT NULL, col1_6 DATE NOT NULL, col2_2 TIMESTAMP NOT NULL, col2_3 BIT(3) NULL, col2_4 FLOAT8 NULL, col2_5 INT4 NOT NULL, col2_6 INT2 NULL, col2_7 REGPROC NULL, col2_8 FLOAT8 NULL, col2_9 BOOL NOT NULL, PRIMARY KEY (col1_5 DESC, col1_6 ASC, col2_6, col2_3 ASC, col2_5, col2_8 ASC, col2_4, col2_9 ASC, col2_2), INDEX (col2_2 ASC, col2_6, col1_5 ASC, col2_9, col2_7 DESC) WHERE ((((((table2.col2_6 >= (-128):::INT8) AND (table2.col2_8 >= (-1.0):::FLOAT8)) AND (table2.col1_6 > '5874897-12-31':::DATE)) OR (table2.col2_5 > (-32768):::INT8)) AND (table2.col2_4 >= '-Inf':::FLOAT8)) AND (table2.col2_2 < '3000-01-01 00:00:00':::TIMESTAMP)) AND table2.col2_9, UNIQUE (col2_6, col1_5 DESC, col2_9 ASC, col1_6 ASC, col2_4 DESC, col2_7 ASC) WHERE ((table2.col2_4 > '+Inf':::FLOAT8) OR (table2.col2_2 >= '0001-01-01 00:00:00':::TIMESTAMP)) AND (table2.col1_6 != '5874897-12-31':::DATE), UNIQUE (col2_7 DESC, col2_6 DESC, col1_5 ASC, col2_8 ASC, col2_2 DESC, col2_3 ASC) WHERE (((table2.col2_5 >= 127:::INT8) OR (table2.col2_8 > 1.7976931348623157e+308:::FLOAT8)) OR (table2.col2_6 = (-1):::INT8)) AND (table2.col1_6 >= '-infinity':::DATE), FAMILY (col1_5), FAMILY (col2_7, col2_9, col2_6), FAMILY (col2_4, col2_5, col1_6, col2_8), FAMILY (col2_2), FAMILY (col2_3), UNIQUE (col2_9, col1_5) WHERE (((table2.col2_8 <= 5e-324) OR (table2.col2_2 < '0001-01-01 00:00:00')) AND (table2.col2_4 <= -1.0)) AND (table2.col2_6 = 0)) INTERLEAVE IN PARENT table1 (col1_5, col1_6);
        test_1        |             CREATE TABLE table3 (col1_5 REGNAMESPACE NOT NULL, col1_6 DATE NOT NULL, col2_6 INT2 NULL, col2_3 BIT(3) NULL, col2_5 INT4 NOT NULL, col2_8 FLOAT8 NULL, col2_4 FLOAT8 NULL, col2_9 BOOL NOT NULL, col2_2 TIMESTAMP NOT NULL, col3_9 FLOAT8, col3_10 VARCHAR NULL, col3_11 REGCLASS, col3_12 INT8, col3_13 DECIMAL, col3_14 REGCLASS NULL, col3_15 TIMESTAMP, col3_16 UUID NULL, col3_17 REGTYPE, col3_18 OID NULL, col3_19 INT4 NULL, PRIMARY KEY (col1_5 DESC, col1_6 ASC, col2_6, col2_3 ASC, col2_5, col2_8 ASC, col2_4, col2_9 ASC, col2_2, col3_19 ASC, col3_17 ASC, col3_16, col3_12 DESC, col3_10 ASC, col3_15, col3_11), UNIQUE (col3_12 DESC, col3_11 ASC, col3_19 ASC, col3_14 ASC, col2_3 DESC, col3_15, col2_8, col3_16 DESC, col3_10 DESC) STORING (col3_18), UNIQUE (col2_9, col2_3 ASC, col3_19, col2_5 DESC, col3_12 DESC, col3_10, col3_9, col1_6 DESC, col3_18, col2_2, col3_13, col1_5 DESC, col2_8 ASC) WHERE ((((((((((((table3.col2_5 >= 1:::INT8) AND (table3.col3_15 > '-4713-11-24 00:00:00':::TIMESTAMP)) AND (table3.col2_8 >= 1.401298464324817e-45:::FLOAT8)) OR (table3.col3_13 > '-Infinity':::DECIMAL)) OR (table3.col2_4 < 'NaN':::FLOAT8)) OR (table3.col3_12 < 1:::INT8)) AND (table3.col2_6 <= 1:::INT8)) AND (table3.col2_2 >= '3000-01-01 00:00:00':::TIMESTAMP)) OR (table3.col3_10 = 'X':::STRING)) AND (NOT table3.col2_9)) AND (table3.col1_6 > '4714-11-24 BC':::DATE)) AND (table3.col3_19 >= 1:::INT8)) OR (table3.col3_9 >= 3.4028234663852886e+38:::FLOAT8), INDEX (col3_18, col3_13 ASC, col2_3 ASC, col3_19 ASC, col3_12 DESC, col1_6 DESC, col3_14 ASC) STORING (col3_9) WHERE ((((((((table3.col2_9 OR (table3.col3_13 > 'Infinity':::DECIMAL)) AND (table3.col1_6 != 'infinity':::DATE)) AND (table3.col3_9 >= 1.7976931348623157e+308:::FLOAT8)) AND (table3.col2_4 != (-1.0):::FLOAT8)) OR (table3.col2_5 < 32767:::INT8)) AND (table3.col2_6 != 32767:::INT8)) AND (table3.col3_12 > 0:::INT8)) AND (table3.col2_8 < 3.4028234663852886e+38:::FLOAT8)) AND (table3.col2_2 > '-2000-01-01 00:00:00':::TIMESTAMP), UNIQUE (col3_12, col2_4, col2_9 DESC, col2_8 DESC, col3_17 DESC, col3_13, col3_15 ASC) STORING (col3_9, col3_14) WHERE ((((((((((((table3.col3_10 <= e'\x00':::STRING) AND (table3.col2_5 <= 1:::INT8)) AND (table3.col2_2 = '-2000-01-01 00:00:00':::TIMESTAMP)) AND (table3.col2_4 >= (-1.0):::FLOAT8)) OR (table3.col1_6 >= 'infinity':::DATE)) OR (table3.col3_9 <= 1.401298464324817e-45:::FLOAT8)) OR (table3.col3_15 = '0001-01-01 00:00:00':::TIMESTAMP)) AND (table3.col2_6 != 0:::INT8)) AND (table3.col3_13 < 1:::DECIMAL)) AND (NOT table3.col2_9)) OR (table3.col2_8 >= '-Inf':::FLOAT8)) OR (table3.col3_19 > (-1):::INT8)) OR (table3.col3_12 <= 2147483647:::INT8), FAMILY (col2_6, col3_14), FAMILY (col3_16), FAMILY (col3_17), FAMILY (col3_18, col3_12, col3_11), FAMILY (col3_9), FAMILY (col2_9), FAMILY (col2_4, col3_19), FAMILY (col1_5, col2_8, col1_6, col2_5, col3_13, col3_15), FAMILY (col3_10), FAMILY (col2_3, col2_2)) INTERLEAVE IN PARENT table2 (col1_5, col1_6, col2_6, col2_3, col2_5, col2_8, col2_4, col2_9, col2_2);
        test_1        |             CREATE TABLE table4 (col4_0 OID, col4_1 STRING NOT NULL, col4_2 DECIMAL, col4_3 NAME NULL, col4_4 VARCHAR NOT NULL, col4_5 REGCLASS, col4_6 REGTYPE NOT NULL, col4_7 BIT(3) NULL, col4_8 TIME NOT NULL, col4_9 NAME, col4_10 BIT(18), col4_11 FLOAT8 NOT NULL, col4_12 FLOAT4 NULL, col4_13 TIME, col4_14 INT2 NOT NULL, col4_15 "char" NULL, col4_16 TIMESTAMP NOT NULL, PRIMARY KEY (col4_14 DESC), UNIQUE (col4_10 DESC, col4_2 ASC, col4_0 DESC, col4_3 DESC, col4_16 DESC), INDEX (col4_5, col4_2, col4_13 ASC, col4_12 ASC, col4_11 ASC, col4_0, col4_6, col4_15 DESC, col4_1 DESC, col4_9 DESC, col4_3, col4_14 ASC) STORING (col4_4, col4_10) WHERE (((table4.col4_11 <= +Inf) AND (table4.col4_15 != '')) OR (table4.col4_14 = -1)) OR (table4.col4_9 = 'X'), UNIQUE (col4_5 ASC, col4_1, col4_7 ASC, col4_16 ASC, col4_4 ASC, col4_13, col4_6 ASC, col4_9 ASC, col4_0, col4_15 DESC) WHERE (((table4.col4_4 < 'X':::STRING) AND (table4.col4_8 <= '24:00:00':::TIME)) AND (table4.col4_12 <= 3.4028234663852886e+38:::FLOAT8)) OR (table4.col4_13 >= '24:00:00':::TIME), UNIQUE (col4_6 ASC, col4_12 ASC, col4_15 ASC, col4_13 DESC, col4_9 ASC, col4_2 ASC, col4_8 DESC, col4_1, col4_10 ASC, col4_3, col4_16, col4_0, col4_5, col4_14 DESC, col4_4 ASC, col4_7) STORING (col4_11) WHERE (((((table4.col4_11 >= -Inf) OR (table4.col4_12 = 3.4028234663852886e+38)) OR (table4.col4_1 < e'\x00')) OR (table4.col4_8 > '24:00:00')) OR (table4.col4_14 >= -1)) AND (table4.col4_4 <= ''), FAMILY (col4_3, col4_9), FAMILY (col4_11), FAMILY (col4_1), FAMILY (col4_4), FAMILY (col4_12, col4_6, col4_0, col4_5, col4_16, col4_10), FAMILY (col4_15, col4_13), FAMILY (col4_8, col4_14), FAMILY (col4_2), FAMILY (col4_7));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "808472237"}, {"distinct_range": 0, "num_eq": 3830721199854395252, "num_range": 1000000000, "upper_bound": "1075982625"}, {"distinct_range": 80000000000, "num_eq": 800000000, "num_range": 80000000000, "upper_bound": "1205239969"}, {"distinct_range": 0, "num_eq": 400000, "num_range": 4967142901342706857, "upper_bound": "2662194439"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 1351295726185407314, "upper_bound": "2953034764"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 7128563381411827033, "upper_bound": "3603343609"}, {"distinct_range": 0, "num_eq": 6048433420434203921, "num_range": 0, "upper_bound": "4280805276"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 4394718025663571194, "num_range": 0, "upper_bound": "-2301"}, {"distinct_range": 200000, "num_eq": 7351883394508710818, "num_range": 200000, "upper_bound": "1"}, {"distinct_range": 3000000000, "num_eq": 5327922423853557846, "num_range": 3000000000, "upper_bound": "6177"}, {"distinct_range": 0, "num_eq": 7098182537625389033, "num_range": 3000000, "upper_bound": "10354"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 3000000000, "upper_bound": "11897"}, {"distinct_range": 1000000000, "num_eq": 6794006537810237645, "num_range": 1000000000, "upper_bound": "17868"}, {"distinct_range": 0, "num_eq": 50000, "num_range": 7616851658586522780, "upper_bound": "30515"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 800000, "upper_bound": "30728"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 5439974556065030872, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 61398.72390173821, "num_eq": 100000000, "num_range": 90000, "upper_bound": "08:13:40.92422+02:15:00"}, {"distinct_range": 8400575790844614000, "num_eq": 60000000000, "num_range": 8400575790844613178, "upper_bound": "03:14:15.727535-05:23:00"}, {"distinct_range": 400000000, "num_eq": 10000, "num_range": 400000000, "upper_bound": "09:25:25.712768-00:31:00"}, {"distinct_range": 121733176594900940, "num_eq": 20000000, "num_range": 3454239705495086129, "upper_bound": "03:25:03.235743-07:34:00"}, {"distinct_range": 5242154910831188000, "num_eq": 8857022345800302060, "num_range": 5242154910831187552, "upper_bound": "04:04:05.74654-07:25:00"}, {"distinct_range": 100000000, "num_eq": 8792640210764848656, "num_range": 100000000, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000, "num_range": 0, "upper_bound": "956266216"}, {"distinct_range": 7670309008742187000, "num_eq": 800, "num_range": 7670309008742187149, "upper_bound": "1461677835"}, {"distinct_range": 5364123330955929000, "num_eq": 60, "num_range": 5797324791659561460, "upper_bound": "1548419689"}, {"distinct_range": 213490672544799550, "num_eq": 2904985463653084701, "num_range": 6767687842708788429, "upper_bound": "3125438710"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 60000000, "upper_bound": "3253917974"}, {"distinct_range": 1000000, "num_eq": 1000000, "num_range": 1000000, "upper_bound": "3993370993"}, {"distinct_range": 547072180932765600, "num_eq": 2818711149593984610, "num_range": 8606885810410439511, "upper_bound": "4005669243"}, {"distinct_range": 368620858785233800, "num_eq": 8622799026839208124, "num_range": 368620858785233799, "upper_bound": "4180607096"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 7641298804575634270, "num_range": 0, "upper_bound": "BOX(-0.25491811433336853 -1.0603475706792416,0.4001866922099572 -0.30242575837335794)"}, {"distinct_range": 0, "num_eq": 4029788177551175147, "num_range": 8728432820630755555, "upper_bound": "BOX(0.03321589898568217 0.7558004316673639,1.6269021788683884 1.73188211765134)"}, {"distinct_range": 6000000, "num_eq": 4859295491473879773, "num_range": 6000000, "upper_bound": "BOX(0.6620385372047657 -0.5679831078476245,1.7777041880911233 -0.32547081771295105)"}, {"distinct_range": 2952596686086966000, "num_eq": 2000, "num_range": 7256858152975974483, "upper_bound": "BOX(0.9087296756336952 -1.0721303752926032,1.5900623275808083 -0.6224120566520751)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 41947534283, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22142662500, "histo_col_type": "", "name": "__auto__", "null_count": 22993410832, "row_count": 90000000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42918263409, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25924722385, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 353649657278277000, "num_eq": 100, "num_range": 3759852110547857346, "upper_bound": "302848750"}, {"distinct_range": 0, "num_eq": 3049563700782591779, "num_range": 4485960607891081652, "upper_bound": "882021908"}, {"distinct_range": 0, "num_eq": 20, "num_range": 2000, "upper_bound": "1043840868"}, {"distinct_range": 0, "num_eq": 3925790946887383775, "num_range": 5691833704371501577, "upper_bound": "2317550979"}, {"distinct_range": 6722809811261613000, "num_eq": 2000000, "num_range": 6722809811261612791, "upper_bound": "2516695130"}, {"distinct_range": 400145612421066400, "num_eq": 9000000, "num_range": 400145612421066364, "upper_bound": "3867574920"}, {"distinct_range": 10000000000, "num_eq": 700, "num_range": 10000000000, "upper_bound": "4222195967"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 25958255014, "row_count": 90000000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17935322097, "histo_col_type": "", "name": "__auto__", "null_count": 52291104857, "row_count": 90000000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15281521302, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 54446215076, "histo_buckets": [{"distinct_range": 0, "num_eq": 3426794410131161433, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 10000000, "num_eq": 0, "num_range": 10000000, "upper_bound": "2748109051"}, {"distinct_range": 6748757181265811000, "num_eq": 40, "num_range": 6748757181265811026, "upper_bound": "3592079628"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 63976881545, "histo_buckets": [{"distinct_range": 0, "num_eq": 300000, "num_range": 0, "upper_bound": "1981-06-02 10:27:53.000296"}, {"distinct_range": 0, "num_eq": 90000, "num_range": 253883181744551595, "upper_bound": "1985-02-11 09:02:10.000518"}, {"distinct_range": 5000000, "num_eq": 100000, "num_range": 5000000, "upper_bound": "1999-12-28 20:20:38.000311"}, {"distinct_range": 0, "num_eq": 0, "num_range": 3528276724934616571, "upper_bound": "2009-06-13 03:51:55.000358"}, {"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": "2025-07-31 01:21:32.000082"}, {"distinct_range": 50000000000, "num_eq": 30000000, "num_range": 50000000000, "upper_bound": "294276-12-31 23:59:59.999999"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 69082307468, "histo_col_type": "", "name": "__auto__", "null_count": 27343127038, "row_count": 90000000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 65094724689, "histo_buckets": [{"distinct_range": 0, "num_eq": 50, "num_range": 0, "upper_bound": "24320"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 61639159758, "row_count": 90000000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38, "histo_col_type": "", "name": "__auto__", "null_count": 50, "row_count": 60}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29, "histo_col_type": "INT8", "name": "__auto__", "null_count": 17, "row_count": 60}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 31, "histo_col_type": "", "name": "__auto__", "null_count": 52, "row_count": 60}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5, "histo_col_type": "", "name": "__auto__", "null_count": 58, "row_count": 60}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 43, "histo_col_type": "", "name": "__auto__", "null_count": 21, "row_count": 60}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7, "histo_buckets": [{"distinct_range": 0, "num_eq": 8427250320004884762, "num_range": 0, "upper_bound": "431723514"}, {"distinct_range": 603616108913562600, "num_eq": 9048447560532856567, "num_range": 603616108913562590, "upper_bound": "1520350704"}, {"distinct_range": 1300454348.5444262, "num_eq": 7000000000, "num_range": 3000000000, "upper_bound": "2971901576"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 56, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38, "histo_col_type": "", "name": "__auto__", "null_count": 1, "row_count": 60}, {"columns": ["col3_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2, "histo_col_type": "", "name": "__auto__", "null_count": 19, "row_count": 60}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_col_type": "", "name": "__auto__", "null_count": 31, "row_count": 60}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15, "histo_col_type": "", "name": "__auto__", "null_count": 26, "row_count": 60}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2, "histo_col_type": "", "name": "__auto__", "null_count": 47, "row_count": 60}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39, "histo_buckets": [{"distinct_range": 0, "num_eq": 2447404154397327719, "num_range": 0, "upper_bound": "false"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 31, "histo_col_type": "", "name": "__auto__", "null_count": 1, "row_count": 60}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26, "histo_col_type": "", "name": "__auto__", "null_count": 25, "row_count": 60}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 6671134283254789000, "num_eq": 5291064442440078846, "num_range": 6671134283254788831, "upper_bound": "467071922"}, {"distinct_range": 644998439265691500, "num_eq": 7760967589613667774, "num_range": 1789080837441974404, "upper_bound": "877877512"}, {"distinct_range": 1054410116397686500, "num_eq": 6891945033534979365, "num_range": 6632347496135116179, "upper_bound": "3225943195"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 54, "row_count": 60}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29, "histo_col_type": "", "name": "__auto__", "null_count": 52, "row_count": 60}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 430308368101427219, "histo_col_type": "", "name": "__auto__", "null_count": 5075082431427954990, "row_count": 6110373083440437850}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4124636151432704038, "histo_buckets": [{"distinct_range": 0, "num_eq": 7465565497717839192, "num_range": 0, "upper_bound": "-29015"}, {"distinct_range": 8061617634748549000, "num_eq": 2000000, "num_range": 8061617634748549560, "upper_bound": "-20763"}, {"distinct_range": 558233078822478200, "num_eq": 300000, "num_range": 653483561967022720, "upper_bound": "-16725"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 5477416176049982382, "upper_bound": "-7301"}, {"distinct_range": 20000000, "num_eq": 80000, "num_range": 20000000, "upper_bound": "-128"}, {"distinct_range": 4000000000, "num_eq": 3267922298922510127, "num_range": 4000000000, "upper_bound": "9930"}, {"distinct_range": 0, "num_eq": 300000, "num_range": 1000000000, "upper_bound": "12406"}, {"distinct_range": 6000000000, "num_eq": 2000000, "num_range": 6000000000, "upper_bound": "25096"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 6110373083440437850}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3071786496727964969, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6110373083440437850}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2136759721682817545, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6110373083440437850}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1612328484881631678, "histo_col_type": "", "name": "__auto__", "null_count": 3787329612527035997, "row_count": 6110373083440437850}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2050354000662282211, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6110373083440437850}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5568617095140253214, "histo_col_type": "", "name": "__auto__", "null_count": 5970154401262970985, "row_count": 6110373083440437850}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2895224943931458610, "histo_col_type": "", "name": "__auto__", "null_count": 2783892368947720201, "row_count": 6110373083440437850}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3204400621354147797, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 5645547706074442755, "num_range": 3731560963321444760, "upper_bound": "312151467"}, {"distinct_range": 1220739737.534284, "num_eq": 100000000, "num_range": 3000000000, "upper_bound": "644702328"}, {"distinct_range": 0, "num_eq": 100, "num_range": 8000000, "upper_bound": "898250796"}, {"distinct_range": 11275065174318318, "num_eq": 5771974330922954340, "num_range": 8399870976145323501, "upper_bound": "3409089650"}, {"distinct_range": 3539506278166041000, "num_eq": 60000, "num_range": 3539506278166040919, "upper_bound": "4113771514"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 6110373083440437850}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1771628935073080844, "histo_col_type": "", "name": "__auto__", "null_count": 805124268919153947, "row_count": 6110373083440437850}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4923416675078148908, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "211341092"}, {"distinct_range": 70000000000, "num_eq": 90, "num_range": 70000000000, "upper_bound": "286233480"}, {"distinct_range": 2609003991849403000, "num_eq": 751588015839435466, "num_range": 2609003991849402836, "upper_bound": "453932774"}, {"distinct_range": 0, "num_eq": 50000000, "num_range": 4735972407178024034, "upper_bound": "1361472565"}, {"distinct_range": 2918182307468541000, "num_eq": 584195113687694940, "num_range": 2918182307468540983, "upper_bound": "1378586676"}, {"distinct_range": 8366321337568561000, "num_eq": 1851947490456301633, "num_range": 8366321337568560801, "upper_bound": "1814881922"}, {"distinct_range": 0, "num_eq": 6388147176618957540, "num_range": 418457436177011174, "upper_bound": "2635527731"}, {"distinct_range": 1000000000, "num_eq": 6801340892024418444, "num_range": 1000000000, "upper_bound": "3115100175"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 1374894038142764340, "row_count": 6110373083440437850}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2010621369160297417, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6110373083440437850}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 609259932387434403, "histo_col_type": "", "name": "__auto__", "null_count": 500031636045360798, "row_count": 6110373083440437850}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 728877832716014958, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6110373083440437850}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4699957365767273110, "histo_col_type": "", "name": "__auto__", "null_count": 27730012891411144, "row_count": 6110373083440437850}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 432838230223237534, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "000001101101101011"}, {"distinct_range": 0, "num_eq": 2446963256158264534, "num_range": 2295759178886720432, "upper_bound": "011111111111111111"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 0, "upper_bound": "111110100101101110"}], "histo_col_type": "BIT(18)", "name": "__auto__", "null_count": 2023865460957545787, "row_count": 6110373083440437850}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2783929290925752893, "histo_col_type": "", "name": "__auto__", "null_count": 2838379158722022910, "row_count": 6110373083440437850}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('dm', 'ao', 'b', 'orig', 'fdjy', 'rpyaxl');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('oll', 'xie');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('mgvqce', 'v', 'libr', 'chghjp', 'sd');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2526555026381633180, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1074888930"}, {"distinct_range": 6709306381850634000, "num_eq": 600000, "num_range": 8642419351367659302, "upper_bound": "3704375365"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 5342498882816186007}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2647918052564258797, "histo_col_type": "", "name": "__auto__", "null_count": 5065783248476823215, "row_count": 5342498882816186007}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4233344338489089118, "histo_col_type": "", "name": "__auto__", "null_count": 2826917503836820428, "row_count": 5342498882816186007}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1281153546081065199, "histo_buckets": [{"distinct_range": 0, "num_eq": 6677280764472392806, "num_range": 0, "upper_bound": "-20218"}, {"distinct_range": 3088917116964407300, "num_eq": 5193776591923474282, "num_range": 3088917116964407128, "upper_bound": "-20194"}, {"distinct_range": 1187.1474920612022, "num_eq": 10000000000, "num_range": 200000, "upper_bound": "-9451"}, {"distinct_range": 0, "num_eq": 8170127493829975243, "num_range": 0, "upper_bound": "1"}, {"distinct_range": 0, "num_eq": 600, "num_range": 4268688028002661277, "upper_bound": "4371"}, {"distinct_range": 5436385548926779000, "num_eq": 3902179120905835046, "num_range": 6146773052198845453, "upper_bound": "16900"}, {"distinct_range": 4000, "num_eq": 8000000, "num_range": 4000, "upper_bound": "20911"}, {"distinct_range": 0, "num_eq": 600, "num_range": 20000, "upper_bound": "25927"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 10000000, "upper_bound": "28450"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 5134943473333115472, "row_count": 5342498882816186007}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3980450864030111478, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5342498882816186007}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 826343536070166288, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000, "num_range": 0, "upper_bound": "0"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 3383581000278978540, "row_count": 5342498882816186007}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3405123063295395283, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5342498882816186007}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 926201164456055917, "histo_buckets": [{"distinct_range": 0, "num_eq": 858182333899056788, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 586.5704056035851, "num_eq": 5032191688313326064, "num_range": 80000, "upper_bound": "BOX(-1.1807750514058286 -0.7754043007767661,-1.1049478285242629 0.3302771300707715)"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 6000000, "upper_bound": "BOX(-0.8319678263632475 -1.8314825991663675,0.5477701314244479 -1.0355132678848449)"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 1000, "upper_bound": "BOX(-0.3306658164872447 -0.7958832336074604,-0.03381557392523721 1.5112567483524586)"}, {"distinct_range": 4591231837522286000, "num_eq": 5308885163570345232, "num_range": 6541887689415936406, "upper_bound": "BOX(0.4941966680290569 0.8230097508699468,0.545170038150707 1.732855399238052)"}, {"distinct_range": 0, "num_eq": 5792959910650340793, "num_range": 755884388026716780, "upper_bound": "BOX(0.5359840817329582 -1.1506167904429687,0.7994232088789206 -0.978055554328265)"}, {"distinct_range": 90472.92410721321, "num_eq": 100000, "num_range": 300000, "upper_bound": "BOX(1.922961805213617 -0.7409694498593341,2.07917705013677 0.09821810998615371)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 5342498882816186007}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2332699835437518913, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5342498882816186007}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3144406490060594244, "histo_col_type": "", "name": "__auto__", "null_count": 596831921189073752, "row_count": 5342498882816186007}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1872116867922584348, "histo_col_type": "", "name": "__auto__", "null_count": 2357713769094460286, "row_count": 5342498882816186007}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 137672093200095396, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000000, "num_range": 0, "upper_bound": "01:33:14.450112+14:30:00"}, {"distinct_range": 0, "num_eq": 90000000000, "num_range": 900000000, "upper_bound": "08:40:38.49442+03:28:00"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 200, "upper_bound": "18:04:06.556179+06:09:00"}, {"distinct_range": 10000000, "num_eq": 40000000, "num_range": 10000000, "upper_bound": "12:39:06.538816-08:16:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 378775012844098550, "row_count": 5342498882816186007}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8286220757678567372, "histo_buckets": [{"distinct_range": 0, "num_eq": 8245147373965754144, "num_range": 0, "upper_bound": "1911273423"}, {"distinct_range": 0, "num_eq": 8914692460089469203, "num_range": 6462866024211230181, "upper_bound": "3525035347"}, {"distinct_range": 90000000000, "num_eq": 4491095988435551154, "num_range": 90000000000, "upper_bound": "3753153617"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 8643803094895515653}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3558771175742774977, "histo_col_type": "", "name": "__auto__", "null_count": 4218206621089617281, "row_count": 8643803094895515653}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1399331457928120221, "histo_col_type": "", "name": "__auto__", "null_count": 2257759433137210439, "row_count": 8643803094895515653}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5923857145723403636, "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 3959356346618696927, "row_count": 8643803094895515653}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6328134808799006684, "histo_col_type": "", "name": "__auto__", "null_count": 6883008631724348732, "row_count": 8643803094895515653}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1926823618577480639, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8643803094895515653}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3855066735384492276, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8643803094895515653}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5179196495317793853, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000000, "num_range": 0, "upper_bound": "1981-06-16 02:23:12.000559"}, {"distinct_range": 316479706662280000, "num_eq": 5289073274080183919, "num_range": 471628008222488244, "upper_bound": "1987-03-22 07:20:02.000154"}, {"distinct_range": 6274114496939897000, "num_eq": 8074302182903613227, "num_range": 7227487412388106658, "upper_bound": "2003-08-23 22:31:41.000132"}, {"distinct_range": 2274165980943903500, "num_eq": 1955438809256009964, "num_range": 2274165980943903384, "upper_bound": "2025-08-26 12:41:02.000486"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 8643803094895515653}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 889159358960649120, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8643803094895515653}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2967162395162738642, "histo_buckets": [{"distinct_range": 0, "num_eq": 8415350705426477095, "num_range": 0, "upper_bound": "-28687"}, {"distinct_range": 3652669215410046000, "num_eq": 9049351570616872241, "num_range": 4050977021589611898, "upper_bound": "-7462"}, {"distinct_range": 0, "num_eq": 3658424447107460145, "num_range": 30000, "upper_bound": "-3467"}, {"distinct_range": 0, "num_eq": 90000, "num_range": 100, "upper_bound": "-128"}, {"distinct_range": 1348000907725055000, "num_eq": 8774196851135009739, "num_range": 6883933544085403677, "upper_bound": "7511"}, {"distinct_range": 4048155162465959000, "num_eq": 4960030735457087406, "num_range": 5836138806013954908, "upper_bound": "18940"}, {"distinct_range": 5.121581184120711, "num_eq": 5000, "num_range": 10, "upper_bound": "21824"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 2114488186366156928, "row_count": 8643803094895515653}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1219, "histo_col_type": "", "name": "__auto__", "null_count": 1118, "row_count": 10000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9455, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 70000000, "num_eq": 3257264985234022484, "num_range": 70000000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1554, "histo_col_type": "", "name": "__auto__", "null_count": 4913, "row_count": 10000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 530, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2734, "histo_col_type": "", "name": "__auto__", "null_count": 6390, "row_count": 10000}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 530, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "384398059"}, {"distinct_range": 1402092563358658800, "num_eq": 3862989372790969169, "num_range": 4981065622733290882, "upper_bound": "3129942887"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 2372036001511858347, "upper_bound": "3210316745"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 2323, "row_count": 10000}, {"columns": ["col3_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7100, "histo_col_type": "", "name": "__auto__", "null_count": 8019, "row_count": 10000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1523, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7339, "histo_col_type": "", "name": "__auto__", "null_count": 5887, "row_count": 10000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1620, "histo_col_type": "", "name": "__auto__", "null_count": 6228, "row_count": 10000}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5018, "histo_col_type": "", "name": "__auto__", "null_count": 7974, "row_count": 10000}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4001, "histo_col_type": "", "name": "__auto__", "null_count": 8636, "row_count": 10000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2027, "histo_col_type": "", "name": "__auto__", "null_count": 7510, "row_count": 10000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7207, "histo_buckets": [{"distinct_range": 0, "num_eq": 5173271722062918815, "num_range": 0, "upper_bound": "-6811533962710777028"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 100000, "upper_bound": "-3295549059102592390"}, {"distinct_range": 0, "num_eq": 1470538824759786510, "num_range": 90, "upper_bound": "-2457391741330623031"}, {"distinct_range": 25.77131612396305, "num_eq": 30000000000, "num_range": 50, "upper_bound": "2042942708792098285"}, {"distinct_range": 19118887.82762225, "num_eq": 9051433438499153338, "num_range": 30000000, "upper_bound": "2046302705641701170"}, {"distinct_range": 0, "num_eq": 200000, "num_range": 4489498820214089031, "upper_bound": "4721802544458510468"}, {"distinct_range": 30000, "num_eq": 4414474195953676881, "num_range": 30000, "upper_bound": "6966194362777526481"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 9781, "row_count": 10000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2104, "histo_col_type": "", "name": "__auto__", "null_count": 1993, "row_count": 10000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3321, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 0, "num_range": 40000000000, "upper_bound": "300981567"}, {"distinct_range": 0, "num_eq": 6343812043987479027, "num_range": 90000000000, "upper_bound": "1443317473"}, {"distinct_range": 0, "num_eq": 6886611726412109049, "num_range": 4000000, "upper_bound": "1652348109"}, {"distinct_range": 456917268.09218466, "num_eq": 1988212769606881286, "num_range": 1000000000, "upper_bound": "2553184845"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4980, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7098, "histo_col_type": "", "name": "__auto__", "null_count": 5908, "row_count": 10000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4352, "histo_col_type": "", "name": "__auto__", "null_count": 437, "row_count": 10000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4473, "histo_col_type": "", "name": "__auto__", "null_count": 2190, "row_count": 10000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5109554, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2929262, "histo_col_type": "", "name": "__auto__", "null_count": 1345172, "row_count": 8000000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4770348, "histo_col_type": "", "name": "__auto__", "null_count": 2771262, "row_count": 8000000}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 610331, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "010100111001001101"}, {"distinct_range": 0, "num_eq": 4848787445506704243, "num_range": 6227745541622899597, "upper_bound": "011010101110111110"}, {"distinct_range": 0, "num_eq": 1870592522461933714, "num_range": 0, "upper_bound": "100000000100010100"}, {"distinct_range": 1870876117784083700, "num_eq": 80000, "num_range": 3243074072693218886, "upper_bound": "100010100101000111"}, {"distinct_range": 0, "num_eq": 900000000, "num_range": 7731956976385491183, "upper_bound": "100011000001100111"}, {"distinct_range": 10000000, "num_eq": 5689888503476280416, "num_range": 10000000, "upper_bound": "100101011000001001"}], "histo_col_type": "BIT(18)", "name": "__auto__", "null_count": 6972093, "row_count": 8000000}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5710198, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7146608, "histo_col_type": "", "name": "__auto__", "null_count": 1024546, "row_count": 8000000}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6379243, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4621623, "histo_col_type": "", "name": "__auto__", "null_count": 4709211, "row_count": 8000000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7845712, "histo_buckets": [{"distinct_range": 0, "num_eq": 20, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 526297335314554556, "num_range": 7408466884601521778, "upper_bound": "388180045"}, {"distinct_range": 28933.31949713854, "num_eq": 8607904961523265161, "num_range": 50000, "upper_bound": "611388808"}, {"distinct_range": 461725784143659840, "num_eq": 8000000, "num_range": 1455781295374412518, "upper_bound": "1952792368"}, {"distinct_range": 3000, "num_eq": 2000000000, "num_range": 3000, "upper_bound": "2062720340"}, {"distinct_range": 0, "num_eq": 7762617083743139738, "num_range": 900, "upper_bound": "2478967535"}, {"distinct_range": 3435276615566373000, "num_eq": 800, "num_range": 4157441880877800794, "upper_bound": "2704198051"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 20000000, "upper_bound": "3383382371"}, {"distinct_range": 7402.312125419993, "num_eq": 50000000000, "num_range": 9000, "upper_bound": "4294678286"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 656597, "row_count": 8000000}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2224499, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000, "num_range": 0, "upper_bound": "291127400"}, {"distinct_range": 0, "num_eq": 0, "num_range": 1756748171815857289, "upper_bound": "1596896286"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 485913, "histo_buckets": [{"distinct_range": 0, "num_eq": 4211450169297668115, "num_range": 0, "upper_bound": "-26955"}, {"distinct_range": 60000000000, "num_eq": 139748889109317386, "num_range": 60000000000, "upper_bound": "-11810"}, {"distinct_range": 1000000000, "num_eq": 700, "num_range": 1000000000, "upper_bound": "-11337"}, {"distinct_range": 38824.18512998976, "num_eq": 10000000, "num_range": 90000, "upper_bound": "1"}, {"distinct_range": 0, "num_eq": 200000, "num_range": 8000000000, "upper_bound": "9172"}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 70000, "upper_bound": "10198"}, {"distinct_range": 906.1292591197252, "num_eq": 800, "num_range": 4000, "upper_bound": "27382"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5422111, "histo_col_type": "", "name": "__auto__", "null_count": 2247268, "row_count": 8000000}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7430656, "histo_col_type": "", "name": "__auto__", "null_count": 5056953, "row_count": 8000000}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4184762, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3654662, "histo_col_type": "", "name": "__auto__", "null_count": 1107550, "row_count": 8000000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7139372, "histo_col_type": "", "name": "__auto__", "null_count": 6647274, "row_count": 8000000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7499236, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000}]';
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col2_9, col3_11) REFERENCES table2 (col2_9, col1_5) ON DELETE CASCADE;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col1_5, col2_7) REFERENCES table1 (col1_2, col1_5);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 176, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "7597"}, {"distinct_range": 50000000, "num_eq": 100000000, "num_range": 50000000, "upper_bound": "17797"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 263, "row_count": 400}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 100, "histo_col_type": "", "name": "__auto__", "null_count": 291, "row_count": 400}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 338, "histo_col_type": "", "name": "__auto__", "null_count": 25, "row_count": 400}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 114, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 292, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000, "num_range": 0, "upper_bound": "BOX(-1.4598871272543992 -0.20914099692939642,0.3059470945130899 0.27884663351114114)"}, {"distinct_range": 2555114296452390000, "num_eq": 3611340823853176808, "num_range": 2555114296452389704, "upper_bound": "BOX(-1.1385974026568184 0.8407620106462445,0.16633462690794487 1.4236042529631288)"}, {"distinct_range": 8023591229300097000, "num_eq": 2802579136861146361, "num_range": 8023591229300097129, "upper_bound": "BOX(-0.7394121126269573 0.5885066639071967,-0.3533594515241113 0.6048663582951439)"}, {"distinct_range": 1928803801062003000, "num_eq": 0, "num_range": 3874437593904964241, "upper_bound": "BOX(-0.2876452475512603 -0.4158744545523163,1.586585301138787 0.7192908363241979)"}, {"distinct_range": 5691558679616762000, "num_eq": 1783021970168220425, "num_range": 5691558679616761911, "upper_bound": "BOX(-0.2539471168138313 -0.9672827283573417,0.20131456938746128 -0.3122173541316617)"}, {"distinct_range": 174.55942848531777, "num_eq": 0, "num_range": 700, "upper_bound": "BOX(-0.1739649809642796 -2.515124999564232,1.2590124728420597 1.272591560263539)"}, {"distinct_range": 0, "num_eq": 80, "num_range": 2835220065808343307, "upper_bound": "BOX(0.05597043628350595 0.31695423313285215,2.0700321602484184 0.3853620351790524)"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 3104174583056111725, "upper_bound": "BOX(0.25604066268486136 0.04290986418098615,0.5316351118859863 0.09300968861854497)"}, {"distinct_range": 900, "num_eq": 2416367916741096321, "num_range": 900, "upper_bound": "BOX(0.5737586464807045 -1.0115811252379536,0.8026223154700979 -0.4319995810689134)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 318, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 64, "histo_col_type": "", "name": "__auto__", "null_count": 25, "row_count": 400}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 108, "histo_col_type": "", "name": "__auto__", "null_count": 196, "row_count": 400}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 227, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 191, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "00:56:23.682949-03:53:00"}, {"distinct_range": 6866547391589646000, "num_eq": 5447569824268328897, "num_range": 6866547391589646802, "upper_bound": "09:26:34.70687+02:27:00"}, {"distinct_range": 0, "num_eq": 6630712462771057877, "num_range": 400000, "upper_bound": "00:42:00.931392-10:24:00"}, {"distinct_range": 822204513089367000, "num_eq": 6040260355780639574, "num_range": 857280019363551711, "upper_bound": "08:49:27.282274-04:02:00"}, {"distinct_range": 0, "num_eq": 7122890893833999956, "num_range": 0, "upper_bound": "16:41:44.238012-09:04:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 193, "row_count": 400}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17, "histo_buckets": [{"distinct_range": 0, "num_eq": 6392537948018084422, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 4062349609544356000, "num_eq": 8000000000, "num_range": 7802091083175372630, "upper_bound": "2533067375"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 6000000000, "upper_bound": "2589621323"}, {"distinct_range": 3147564268840592000, "num_eq": 300000, "num_range": 5720381743902786591, "upper_bound": "2723465455"}, {"distinct_range": 3180378577309690400, "num_eq": 100, "num_range": 3180378577309690254, "upper_bound": "2766360743"}, {"distinct_range": 43.24555091762849, "num_eq": 5198653928997541477, "num_range": 100, "upper_bound": "2842831172"}, {"distinct_range": 8000, "num_eq": 3298204035938308989, "num_range": 8000, "upper_bound": "2919064612"}, {"distinct_range": 60000000000, "num_eq": 6114496702003665484, "num_range": 60000000000, "upper_bound": "3137432689"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 281, "row_count": 400}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 384, "histo_buckets": [{"distinct_range": 0, "num_eq": 6697905899279805146, "num_range": 0, "upper_bound": "4242038330"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 400}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 595, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 100, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "-27456"}, {"distinct_range": 382657398265856900, "num_eq": 50, "num_range": 1306661992955069207, "upper_bound": "-24489"}, {"distinct_range": 95908052.80056615, "num_eq": 1000, "num_range": 100000000, "upper_bound": "-23459"}, {"distinct_range": 200000000, "num_eq": 70000, "num_range": 200000000, "upper_bound": "-21741"}, {"distinct_range": 0, "num_eq": 1758746066150434755, "num_range": 0, "upper_bound": "127"}, {"distinct_range": 2708058321328720400, "num_eq": 6567338530112490535, "num_range": 6213968767631246095, "upper_bound": "4833"}, {"distinct_range": 3000000000, "num_eq": 30, "num_range": 3000000000, "upper_bound": "10283"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 1000000, "upper_bound": "25795"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 403, "row_count": 900}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 107, "histo_buckets": [{"distinct_range": 0, "num_eq": 7321194801648446954, "num_range": 0, "upper_bound": "3478528705"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 580, "row_count": 900}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 658, "row_count": 900}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 399, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 526, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "2019-02-07 09:00:22.000568"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 525, "histo_col_type": "", "name": "__auto__", "null_count": 682, "row_count": 900}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 851, "histo_col_type": "", "name": "__auto__", "null_count": 321, "row_count": 900}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 554, "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 708, "histo_buckets": [{"distinct_range": 0, "num_eq": 7499071990319809401, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 5625629905021228196, "num_range": 491805611040800925, "upper_bound": "1023985893"}, {"distinct_range": 0, "num_eq": 1476707331507649375, "num_range": 10000000, "upper_bound": "1300475286"}, {"distinct_range": 7455325005006647000, "num_eq": 400000000, "num_range": 7455325005006647088, "upper_bound": "2470034263"}, {"distinct_range": 90000, "num_eq": 5274411369045714040, "num_range": 90000, "upper_bound": "3397092656"}], "histo_col_ty

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2673412&tab=buildLog) on [master@ba1a1440714b8c514f8a994b0a82c426c3ee6e99](https://github.com/cockroachdb/cockroach/commits/ba1a1440714b8c514f8a994b0a82c426c3ee6e99):

```
        cockroach1_1  | webui:               http://cockroach1:8080
        cockroach1_1  | sql:                 postgresql://root@cockroach1:26257?sslmode=disable
        cockroach1_1  | RPC client flags:    /cockroach/cockroach <client cmd> --host=cockroach1:26257 --insecure
        cockroach1_1  | logs:                /cockroach-data/logs
        cockroach1_1  | temp dir:            /cockroach-data/cockroach-temp416683368
        cockroach1_1  | external I/O path:   /cockroach-data/extern
        cockroach1_1  | store[0]:            path=/cockroach-data
        cockroach1_1  | storage engine:      pebble
        cockroach1_1  | status:              initialized new cluster
        cockroach1_1  | clusterID:           e7dd8345-0b04-496f-baf5-1ff8a6bf68ca
        cockroach1_1  | nodeID:              1
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2021-02-13 06:12:22.070 UTC [48] LOG:  received fast shutdown request
        postgres_1    | .2021-02-13 06:12:22.072 UTC [48] LOG:  aborting any active transactions
        postgres_1    | 2021-02-13 06:12:22.075 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2021-02-13 06:12:22.079 UTC [50] LOG:  shutting down
        postgres_1    | 2021-02-13 06:12:22.186 UTC [48] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-02-13 06:12:22.281 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-02-13 06:12:22.281 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-02-13 06:12:22.284 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-02-13 06:12:22.298 UTC [85] LOG:  database system was shut down at 2021-02-13 06:12:22 UTC
        postgres_1    | 2021-02-13 06:12:22.303 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (602.56s)
        test_1        |     --- FAIL: TestCompare/postgres (1.43s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2674692&tab=buildLog) on [master@9969862335cf501c1aa67a4aaf018ad5dc8e85e8](https://github.com/cockroachdb/cockroach/commits/9969862335cf501c1aa67a4aaf018ad5dc8e85e8):

```
        test_1        |         compare_test.go:155: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("fC"),
        test_1        |             + 	string("falseC"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT (false::BOOL || 'C'::TEXT::TEXT)::TEXT AS col_33829) AS tab_20266
        test_1        |             ORDER BY
        test_1        |             	col_33829 NULLS FIRST;
        test_1        |             
        test_1        |             postgres: [same as previous]
        test_1        |             
        test_1        |     --- FAIL: TestCompare/mutators (0.98s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 INT4 NULL, col1_1 REGPROC NULL, col1_2 INT8 NOT NULL, col1_3 REGCLASS NOT NULL, col1_4 REGPROCEDURE NOT NULL, col1_5 BOX2D NULL, col1_6 REGTYPE NOT NULL, PRIMARY KEY (col1_6 ASC), INDEX (col1_1 ASC, col1_4 DESC) STORING (col1_0, col1_2, col1_3, col1_5) WHERE (table1.col1_2 <= 0:::INT8) AND (table1.col1_0 < 1:::INT8), UNIQUE (col1_6 ASC) STORING (col1_0, col1_1, col1_3), FAMILY (col1_2), FAMILY (col1_3), FAMILY (col1_4), FAMILY (col1_6, col1_0, col1_5), FAMILY (col1_1));
        test_1        |             CREATE TABLE table2 (col1_6 REGTYPE NOT NULL, col2_1 TIMESTAMP NULL, col2_2 BOOL, col2_3 BIT(31), col2_4 INT8, col2_5 FLOAT4 NOT NULL, col2_6 CHAR NOT NULL, PRIMARY KEY (col1_6 ASC, col2_2 DESC), FAMILY (col2_3), FAMILY (col2_2), FAMILY (col2_6), FAMILY (col1_6), FAMILY (col2_4), FAMILY (col2_1, col2_5), UNIQUE (col2_2) WHERE ((table2.col2_4 = 127) AND (table2.col2_1 >= '294276-12-31 23:59:59.999999')) OR (table2.col2_5 < -1.0)) INTERLEAVE IN PARENT table1 (col1_6);
        test_1        |             CREATE TABLE table3 (col1_6 REGTYPE NOT NULL, col2_2 BOOL, col3_2 REGCLASS, col3_3 UUID, col3_4 DATE NULL, col3_5 REGNAMESPACE NULL, col3_6 STRING, col3_7 BIT NOT NULL, col3_8 REGTYPE, col3_9 REGCLASS NULL, col3_10 TIME, col3_11 BIT(18), PRIMARY KEY (col1_6 ASC, col2_2 DESC, col3_10 DESC, col3_7, col3_3, col3_8), UNIQUE (col3_11 ASC, col3_4, col3_5 DESC) WHERE (((table3.col3_10 <= '24:00:00':::TIME) OR (NOT table3.col2_2)) OR (table3.col3_6 != '':::STRING)) OR (table3.col3_4 <= 'infinity':::DATE), INDEX (col3_3 ASC, col3_4 DESC, col3_10 ASC, col1_6 ASC, col3_11, col3_6 DESC, col3_8, col2_2, col3_7), FAMILY (col3_6), FAMILY (col3_11, col3_4, col3_7), FAMILY (col3_9), FAMILY (col2_2, col3_10), FAMILY (col3_8, col3_2), FAMILY (col3_5), FAMILY (col3_3), FAMILY (col1_6)) INTERLEAVE IN PARENT table2 (col1_6, col2_2);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9206, "histo_col_type": "", "name": "__auto__", "null_count": 5713, "row_count": 10000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8729, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "0"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 1066, "row_count": 10000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3728, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4659, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2676, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8397, "histo_col_type": "", "name": "__auto__", "null_count": 2803, "row_count": 10000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8819, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "1101281046"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 4791709429273422997, "upper_bound": "1244596352"}, {"distinct_range": 7751370413597031000, "num_eq": 3922408712879861370, "num_range": 7751370413597031894, "upper_bound": "1654037998"}, {"distinct_range": 232557.40789041264, "num_eq": 2559233314101574471, "num_range": 1000000, "upper_bound": "1833766908"}, {"distinct_range": 1333608217127042300, "num_eq": 700, "num_range": 2949207025054157062, "upper_bound": "2486222516"}, {"distinct_range": 5971068097316223000, "num_eq": 898620002534028268, "num_range": 5971068097316222552, "upper_bound": "2867564768"}, {"distinct_range": 0, "num_eq": 7390459483296935928, "num_range": 2000, "upper_bound": "3056579867"}, {"distinct_range": 0, "num_eq": 118348618541089883, "num_range": 7074210998685678641, "upper_bound": "3127178664"}, {"distinct_range": 80000000, "num_eq": 200000000, "num_range": 80000000, "upper_bound": "3250627629"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 10000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4690487283262214020, "histo_buckets": [{"distinct_range": 0, "num_eq": 3559443060576834420, "num_range": 0, "upper_bound": "0"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 5666229048271959108}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2648512664513535860, "histo_col_type": "", "name": "__auto__", "null_count": 4456461119011975055, "row_count": 5666229048271959108}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3499275810303907505, "histo_col_type": "", "name": "__auto__", "null_count": 1859804684206881991, "row_count": 5666229048271959108}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4034586777132549836, "histo_col_type": "", "name": "__auto__", "null_count": 4138891898522042866, "row_count": 5666229048271959108}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3553187726836154468, "histo_col_type": "", "name": "__auto__", "null_count": 5114519857219643156, "row_count": 5666229048271959108}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1311449394399013869, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5666229048271959108}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4754103061553568427, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5666229048271959108}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8786938, "histo_col_type": "", "name": "__auto__", "null_count": 9674361, "row_count": 40000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38015692, "histo_col_type": "", "name": "__auto__", "null_count": 17305760, "row_count": 40000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3359033, "histo_col_type": "", "name": "__auto__", "null_count": 26796817, "row_count": 40000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2896411, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14325663, "histo_col_type": "", "name": "__auto__", "null_count": 28334591, "row_count": 40000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35992863, "histo_col_type": "", "name": "__auto__", "null_count": 8273614, "row_count": 40000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17896387, "histo_col_type": "", "name": "__auto__", "null_count": 17702097, "row_count": 40000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1541972, "histo_col_type": "", "name": "__auto__", "null_count": 13531177, "row_count": 40000000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21521685, "histo_buckets": [{"distinct_range": 0, "num_eq": 5465923340763628967, "num_range": 0, "upper_bound": "000100100011100001"}, {"distinct_range": 7.948402306477992, "num_eq": 7807637851950251544, "num_range": 10, "upper_bound": "001001110101110100"}, {"distinct_range": 3866920590672980000, "num_eq": 3007731172808586576, "num_range": 5124659258988205747, "upper_bound": "001111101000000001"}, {"distinct_range": 0, "num_eq": 8024867313776349449, "num_range": 0, "upper_bound": "011111111111111111"}, {"distinct_range": 0, "num_eq": 3869583284577754680, "num_range": 600000000, "upper_bound": "111111000100011100"}], "histo_col_type": "BIT(18)", "name": "__auto__", "null_count": 7860220, "row_count": 40000000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33056625, "histo_col_type": "", "name": "__auto__", "null_count": 16201109, "row_count": 40000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4009265, "histo_buckets": [{"distinct_range": 0, "num_eq": 2848255430548590033, "num_range": 0, "upper_bound": "1152001272"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 70000, "upper_bound": "1339301232"}, {"distinct_range": 100, "num_eq": 50000000000, "num_range": 100, "upper_bound": "1688529796"}, {"distinct_range": 80000000, "num_eq": 0, "num_range": 80000000, "upper_bound": "2009000061"}, {"distinct_range": 3036761650.7361665, "num_eq": 1000000000, "num_range": 8000000000, "upper_bound": "3046316750"}, {"distinct_range": 62.597319241790515, "num_eq": 90000, "num_range": 1000, "upper_bound": "3643293041"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 40000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17077165, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000000, "num_range": 0, "upper_bound": "a1225bd9-4065-46b5-b412-9973dbb9c35a"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 13465782, "row_count": 40000000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('jbbn', 'yr');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('htrq', 'bnr', 'lw', 'lh');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('mbq', 'm', 'g', 'yzhs', 'te');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('ohlk', 'fpcm', 'stos');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4636579, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1097231, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3606550, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 503102, "histo_col_type": "", "name": "__auto__", "null_count": 4747544, "row_count": 9000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8356086, "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8737321, "histo_col_type": "", "name": "__auto__", "null_count": 1742073, "row_count": 9000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8421880, "histo_buckets": [{"distinct_range": 0, "num_eq": 2726921767008171036, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 6919387765583428000, "num_eq": 90000, "num_range": 6919387765583427344, "upper_bound": "454794134"}, {"distinct_range": 0, "num_eq": 7636227594057361143, "num_range": 1436712170707575606, "upper_bound": "2132453638"}, {"distinct_range": 743767864.4538817, "num_eq": 20000000, "num_range": 4000000000, "upper_bound": "2599163337"}, {"distinct_range": 0, "num_eq": 9000, "num_range": 7328011274917841889, "upper_bound": "2671964399"}, {"distinct_range": 3122574583117516000, "num_eq": 3633008070958976199, "num_range": 5561723520971390587, "upper_bound": "3551197993"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 2841580, "row_count": 9000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1077334467765637723, "histo_col_type": "", "name": "__auto__", "null_count": 1283415105778720925, "row_count": 1927492194863825850}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 108331954897343799, "histo_col_type": "", "name": "__auto__", "null_count": 1859829211899297343, "row_count": 1927492194863825850}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 947534719405846903, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1927492194863825850}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1383287778352055002, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1927492194863825850}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1008724608623422404, "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 1927492194863825850}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1555505382767371096, "histo_col_type": "", "name": "__auto__", "null_count": 1377329877683347314, "row_count": 1927492194863825850}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1554655202378061156, "histo_col_type": "", "name": "__auto__", "null_count": 162818689175147955, "row_count": 1927492194863825850}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1117, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "442179203"}, {"distinct_range": 0, "num_eq": 1770431318096079300, "num_range": 3288885944512805734, "upper_bound": "1114165263"}, {"distinct_range": 2685603482566153000, "num_eq": 90000000000, "num_range": 3488595159818425486, "upper_bound": "2573845144"}, {"distinct_range": 977781.0207514707, "num_eq": 50000000, "num_range": 1000000, "upper_bound": "2922588222"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 0, "upper_bound": "3330938172"}, {"distinct_range": 0, "num_eq": 4195617256446236146, "num_range": 400000, "upper_bound": "3618709560"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 3000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2081, "histo_col_type": "", "name": "__auto__", "null_count": 1974, "row_count": 3000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1290, "histo_col_type": "", "name": "__auto__", "null_count": 670, "row_count": 3000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2238, "histo_buckets": [{"distinct_range": 0, "num_eq": 2032115638535544609, "num_range": 0, "upper_bound": "3359e605-9738-43ff-9097-a744a7feaa0d"}, {"distinct_range": 166912422520372380, "num_eq": 7000000000, "num_range": 976351687659808041, "upper_bound": "5e95fb13-2b5d-4f9e-a0d2-8463066de370"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 1573, "row_count": 3000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 341, "histo_col_type": "", "name": "__auto__", "null_count": 1424, "row_count": 3000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 60, "histo_col_type": "", "name": "__auto__", "null_count": 323, "row_count": 3000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1451, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2495, "histo_col_type": "", "name": "__auto__", "null_count": 2900, "row_count": 3000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 789, "histo_col_type": "", "name": "__auto__", "null_count": 622, "row_count": 3000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1410, "histo_col_type": "", "name": "__auto__", "null_count": 73, "row_count": 3000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 618, "histo_col_type": "", "name": "__auto__", "null_count": 2803, "row_count": 3000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 348, "histo_buckets": [{"distinct_range": 0, "num_eq": 84807841857375212, "num_range": 0, "upper_bound": "001010100011000001"}, {"distinct_range": 3.8331518356891863, "num_eq": 300000, "num_range": 80, "upper_bound": "010010101100001011"}, {"distinct_range": 400, "num_eq": 8130638717681861886, "num_range": 400, "upper_bound": "011111111111111111"}, {"distinct_range": 0, "num_eq": 700, "num_range": 20000, "upper_bound": "100010111110111011"}, {"distinct_range": 90000, "num_eq": 40, "num_range": 90000, "upper_bound": "101100101011111000"}, {"distinct_range": 0, "num_eq": 5268234804054200100, "num_range": 9105275454166845760, "upper_bound": "101110101111010000"}, {"distinct_range": 145294739485876030, "num_eq": 70000000, "num_range": 908636063141089282, "upper_bound": "110110100110110100"}, {"distinct_range": 271567882029121540, "num_eq": 7062915969286192888, "num_range": 378890334829077231, "upper_bound": "111011101101110011"}], "histo_col_type": "BIT(18)", "name": "__auto__", "null_count": 2494, "row_count": 3000}]';
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col2_2) REFERENCES table2 (col2_2) ON DELETE CASCADE ON UPDATE RESTRICT;
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 31, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32, "histo_col_type": "", "name": "__auto__", "null_count": 13, "row_count": 40}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9, "histo_buckets": [{"distinct_range": 0, "num_eq": 3273775529110683668, "num_range": 0, "upper_bound": "2465287657"}, {"distinct_range": 0, "num_eq": 4180099832555120397, "num_range": 7000, "upper_bound": "3498953830"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 40}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20, "histo_col_type": "", "name": "__auto__", "null_count": 25, "row_count": 40}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28, "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 4, "row_count": 40}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2822260693038196111, "histo_buckets": [{"distinct_range": 0, "num_eq": 2945978239400410463, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 27573.30822086418, "num_eq": 10000, "num_range": 70000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 985941477492157298, "row_count": 5375434431521706896}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1204689956516862006, "histo_col_type": "", "name": "__auto__", "null_count": 2982948381884690497, "row_count": 5375434431521706896}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5118202057114323460, "histo_col_type": "", "name": "__auto__", "null_count": 1629147675827176980, "row_count": 5375434431521706896}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1657826964899220549, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5375434431521706896}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 221306966313631889, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5375434431521706896}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3301891385329904482, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "1050696298"}, {"distinct_range": 0, "num_eq": 5540606796838961386, "num_range": 0, "upper_bound": "1680653296"}, {"distinct_range": 2447348161446350300, "num_eq": 3987669084253085258, "num_range": 3284988977296878491, "upper_bound": "2061312148"}, {"distinct_range": 0, "num_eq": 4265094362275816638, "num_range": 200000, "upper_bound": "4275967381"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 5375434431521706896}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 177906370510973197, "histo_col_type": "", "name": "__auto__", "null_count": 4194271172852141723, "row_count": 5375434431521706896}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13391407166, "histo_buckets": [{"distinct_range": 0, "num_eq": 2143691163259110326, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 40, "num_eq": 8090848290730173509, "num_range": 40, "upper_bound": "60162585"}, {"distinct_range": 0, "num_eq": 300000000, "num_range": 8086079513998441341, "upper_bound": "488039372"}, {"distinct_range": 7000, "num_eq": 70000, "num_range": 7000, "upper_bound": "1005944975"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 5598882496494320962, "upper_bound": "1874298528"}, {"distinct_range": 0, "num_eq": 6887936734019435663, "num_range": 7055995581680176344, "upper_bound": "3510115046"}, {"distinct_range": 6882173879141192000, "num_eq": 7412094445287391443, "num_range": 7099588279073274978, "upper_bound": "3547738998"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 20000000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12104203051, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000000, "num_range": 0, "upper_bound": "00000000-0000-0000-0000-000000000000"}, {"distinct_range": 310943281039179300, "num_eq": 8000000, "num_range": 310943281039179358, "upper_bound": "2950c28d-383c-4e97-9e81-06f2768ba983"}, {"distinct_range": 4832033931998324000, "num_eq": 10000000, "num_range": 6726918537679769481, "upper_bound": "3b153476-d4bf-42e3-b5d1-a588e09d2538"}, {"distinct_range": 7186489721932492000, "num_eq": 200, "num_range": 7186489721932491812, "upper_bound": "5a926904-105b-42d3-89fc-f1e16f97eced"}, {"distinct_range": 0, "num_eq": 50000, "num_range": 0, "upper_bound": "c410c4a6-f8bd-4191-bc4f-3a95b8362953"}, {"distinct_range": 57523.946394347804, "num_eq": 4561699883774966398, "num_range": 100000, "upper_bound": "d8bed886-a43b-4bf7-b797-c14b290be028"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 16308762837, "row_count": 20000000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 693653703, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4744735076, "histo_col_type": "", "name": "__auto__", "null_count": 11210612643, "row_count": 20000000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8975973873, "histo_col_type": "", "name": "__auto__", "null_count": 2731085767, "row_count": 20000000000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5574981065, "histo_col_type": "", "name": "__auto__", "null_count": 8667067676, "row_count": 20000000000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5701141456, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": "001111000110000101"}, {"distinct_range": 80000000000, "num_eq": 30000, "num_range": 80000000000, "upper_bound": "011100010001110001"}, {"distinct_range": 3874728690392751000, "num_eq": 7069380994023862458, "num_range": 6499607793426209453, "upper_bound": "011111111111111111"}, {"distinct_range": 1966430266880827400, "num_eq": 6000000000, "num_range": 1966430266880827366, "upper_bound": "101010111101101001"}, {"distinct_range": 0, "num_eq": 300, "num_range": 6366390936243883278, "upper_bound": "111000001101110000"}, {"distinct_range": 0, "num_eq": 300, "num_range": 0, "upper_bound": "111101000000101100"}], "histo_col_type": "BIT(18)", "name": "__auto__", "null_count": 16414576151, "row_count": 20000000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 362115533, "histo_col_type": "", "name": "__auto__", "null_count": 11883345138, "row_count": 20000000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11386098825, "histo_col_type": "", "name": "__auto__", "null_count": 3142047800, "row_count": 20000000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9560874976, "histo_col_type": "", "name": "__auto__", "null_count": 9990845709, "row_count": 20000000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3589967209, "histo_col_type": "", "name": "__auto__", "null_count": 11925019429, "row_count": 20000000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18185698935, "histo_col_type": "", "name": "__auto__", "null_count": 10167867766, "row_count": 20000000000}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: there is no unique constraint matching given keys for referenced table table2 (SQLSTATE 23503)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2676657&tab=buildLog) on [master@5971ecb9dd1a25c81cd6012d6be1ff922802eae5](https://github.com/cockroachdb/cockroach/commits/5971ecb9dd1a25c81cd6012d6be1ff922802eae5):

```
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2021-02-15 06:12:29.490 UTC [48] LOG:  received fast shutdown request
        postgres_1    | .2021-02-15 06:12:29.492 UTC [48] LOG:  aborting any active transactions
        postgres_1    | 2021-02-15 06:12:29.494 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2021-02-15 06:12:29.499 UTC [50] LOG:  shutting down
        postgres_1    | 2021-02-15 06:12:29.585 UTC [48] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-02-15 06:12:29.600 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-02-15 06:12:29.600 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-02-15 06:12:29.604 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-02-15 06:12:29.620 UTC [85] LOG:  database system was shut down at 2021-02-15 06:12:29 UTC
        postgres_1    | 2021-02-15 06:12:29.625 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (1.11s)
        test_1        |     --- FAIL: TestCompare/postgres (0.63s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.2:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.49s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 INT2, col1_1 "char" NOT NULL, col1_2 REGCLASS, col1_3 INT8 NULL, col1_4 UUID NOT NULL, col1_5 TIMETZ, col1_6 "char", col1_7 DECIMAL NOT NULL, col1_8 UUID, col1_9 TIMESTAMPTZ NOT NULL, col1_10 REGTYPE NULL, col1_11 DECIMAL NOT NULL, col1_12 INT2, col1_13 TIME, col1_14 INT4 NOT NULL, col1_15 BIT(46) NOT NULL, UNIQUE (col1_9 ASC, col1_13 DESC, col1_15, col1_1 ASC, col1_7 DESC, col1_12, col1_8 DESC, col1_2, col1_3 DESC, col1_0 DESC) STORING (col1_5, col1_6, col1_10, col1_11) WHERE table1.col1_12 <= 0:::INT8, UNIQUE (col1_10, col1_2 DESC, col1_9 DESC, col1_11 ASC, col1_7, col1_1, col1_13 ASC) WHERE ((((((((((table1.col1_0 != 32767) AND (table1.col1_7 > 1)) OR (table1.col1_12 > -1)) AND (table1.col1_6 >= '"')) AND (table1.col1_13 >= '00:00:00')) OR (table1.col1_5 < '00:00:00+15:59:00')) AND (table1.col1_14 <= 32767)) AND (table1.col1_3 < -32768)) OR (table1.col1_1 < '"')) OR (table1.col1_11 = Infinity)) AND (table1.col1_9 > '294276-12-31 23:59:59.999999+00:00'), UNIQUE (col1_7) WHERE table1.col1_1 <= '', FAMILY (col1_6, col1_14, col1_0), FAMILY (col1_10, col1_2), FAMILY (col1_4), FAMILY (col1_8), FAMILY (col1_15, col1_1), FAMILY (col1_7), FAMILY (col1_12), FAMILY (col1_11), FAMILY (col1_3, col1_13, col1_9), FAMILY (col1_5));
        test_1        |             CREATE TABLE table2 (col2_0 REGCLASS NOT NULL, col2_1 INT4[] NULL, col2_2 REGCLASS NOT NULL, col2_3 NAME, col2_4 DECIMAL NOT NULL, col2_5 TIMESTAMPTZ NULL, col2_6 INT2 NOT NULL, col2_7 INT2 NOT NULL, col2_8 FLOAT4 NULL, col2_9 STRING NULL, col2_10 DECIMAL NOT NULL, PRIMARY KEY (col2_2, col2_0, col2_10), INDEX (col2_8, col2_7 ASC, col2_2, col2_4 ASC) STORING (col2_5, col2_6) WHERE ((table2.col2_8 <= '+Inf':::FLOAT8) OR (table2.col2_6 = 0:::INT8)) AND (table2.col2_3 >= e'\U00002603':::STRING), INDEX (col2_5, col2_9 DESC, col2_3 DESC, col2_7 ASC, col2_4, col2_8, col2_6, col2_0 ASC, col2_10 DESC) WHERE table2.col2_10 > 'NaN':::DECIMAL, FAMILY (col2_10, col2_7), FAMILY (col2_1), FAMILY (col2_0), FAMILY (col2_3), FAMILY (col2_6), FAMILY (col2_9, col2_4), FAMILY (col2_8), FAMILY (col2_2), FAMILY (col2_5));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3764, "histo_col_type": "", "name": "__auto__", "null_count": 2585, "row_count": 6000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4627, "histo_col_type": "", "name": "__auto__", "null_count": 5491, "row_count": 6000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3046, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 527, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 446, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 15806.11898310995, "num_eq": 90, "num_range": 30000, "upper_bound": "1065433934"}, {"distinct_range": 0, "num_eq": 6695210471498299309, "num_range": 5000000000, "upper_bound": "1686731554"}, {"distinct_range": 627022210131616400, "num_eq": 957160643925452035, "num_range": 1343005151800722647, "upper_bound": "2558431918"}, {"distinct_range": 0, "num_eq": 3036736666799592457, "num_range": 70000, "upper_bound": "3998771518"}, {"distinct_range": 0, "num_eq": 9084928663705709564, "num_range": 3027427102342350953, "upper_bound": "4082893468"}, {"distinct_range": 0, "num_eq": 1838980425891087406, "num_range": 1112177107177093868, "upper_bound": "4206064024"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 4871, "row_count": 6000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 838, "histo_col_type": "", "name": "__auto__", "null_count": 4638, "row_count": 6000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1401, "histo_col_type": "", "name": "__auto__", "null_count": 4006, "row_count": 6000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2686, "histo_col_type": "", "name": "__auto__", "null_count": 4156, "row_count": 6000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3615, "histo_col_type": "", "name": "__auto__", "null_count": 1536, "row_count": 6000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4822, "histo_col_type": "", "name": "__auto__", "null_count": 4034, "row_count": 6000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4606, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000000, "num_range": 0, "upper_bound": "1977-03-18 06:48:35.000507+00:00"}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 0, "upper_bound": "1987-07-02 21:48:02.000415+00:00"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 1868806498902405313, "upper_bound": "2000-11-15 05:46:22.000864+00:00"}, {"distinct_range": 0, "num_eq": 6030644609586893028, "num_range": 8996813698783017203, "upper_bound": "2005-04-20 05:07:22.000776+00:00"}, {"distinct_range": 0, "num_eq": 5626093048519976969, "num_range": 5908180001445574953, "upper_bound": "2018-11-28 22:11:55.000874+00:00"}, {"distinct_range": 4852955699351686000, "num_eq": 1578880028856949290, "num_range": 4852955699351685633, "upper_bound": "2019-05-04 09:49:59.000455+00:00"}, {"distinct_range": 277335580.91952413, "num_eq": 844420798913094242, "num_range": 400000000, "upper_bound": "2020-10-06 20:33:33.000401+00:00"}, {"distinct_range": 3694963032313938400, "num_eq": 90000000000, "num_range": 3694963032313938626, "upper_bound": "3000-01-01 00:00:00+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2919, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3077, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2723, "histo_col_type": "", "name": "__auto__", "null_count": 1770, "row_count": 6000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4168, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5164, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3245551310125498117, "histo_buckets": [{"distinct_range": 0, "num_eq": 7421847512153744470, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 749790.5012054121, "num_eq": 3278448631189691190, "num_range": 800000, "upper_bound": "195012493"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 6663142969446362964}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 443308350153802014, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6663142969446362964}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30469485212838547, "histo_col_type": "", "name": "__auto__", "null_count": 2191837531384383872, "row_count": 6663142969446362964}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5643774267389524484, "histo_buckets": [{"distinct_range": 0, "num_eq": 4310557108723511110, "num_range": 0, "upper_bound": "0001-01-01 00:00:00+00:00"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 6000000, "upper_bound": "1990-04-15 03:38:25.000062+00:00"}, {"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1992-06-27 07:20:32.000101+00:00"}, {"distinct_range": 0, "num_eq": 8793547099653964275, "num_range": 0, "upper_bound": "1997-04-26 05:38:21.000635+00:00"}, {"distinct_range": 3798761645740662000, "num_eq": 2320389010820675014, "num_range": 3798761645740661773, "upper_bound": "2001-08-24 19:16:57.000538+00:00"}, {"distinct_range": 0, "num_eq": 40, "num_range": 1000000, "upper_bound": "2005-08-02 13:54:18.000444+00:00"}, {"distinct_range": 127755753370450460, "num_eq": 30000000000, "num_range": 2969044325092714009, "upper_bound": "2006-07-04 13:20:26.000249+00:00"}, {"distinct_range": 5000, "num_eq": 7629334918946974755, "num_range": 5000, "upper_bound": "2016-12-15 01:33:20.000686+00:00"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 4047485265143985195, "upper_bound": "2030-10-29 15:26:15.000644+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 366112225599091923, "row_count": 6663142969446362964}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 74123961307593439, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6663142969446362964}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2982255938044925276, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6663142969446362964}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1779324930636591307, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": "-3.6246416568756104"}, {"distinct_range": 0, "num_eq": 5703981058417747764, "num_range": 829390889121408437, "upper_bound": "-1.1546635627746582"}, {"distinct_range": 50, "num_eq": 9000000000, "num_range": 50, "upper_bound": "-1.0"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 481928835031977957, "upper_bound": "-0.5599135160446167"}, {"distinct_range": 630.8754287947296, "num_eq": 40, "num_range": 20000, "upper_bound": "-0.4644906222820282"}, {"distinct_range": 1000, "num_eq": 700000, "num_range": 1000, "upper_bound": "-0.17014740407466888"}, {"distinct_range": 19.534917074350243, "num_eq": 8000, "num_range": 30, "upper_bound": "0.6290709972381592"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 3701055953799232274, "row_count": 6663142969446362964}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1149906786789592909, "histo_col_type": "", "name": "__auto__", "null_count": 414657416384721734, "row_count": 6663142969446362964}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5394310236746772493, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6663142969446362964}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 683661152271123725, "histo_col_type": "", "name": "__auto__", "null_count": 3255243311676397543, "row_count": 6663142969446362964}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5145300631349733191, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6663142969446362964}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('aam', 'enry', 'tkb', 'rw', 'des');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "230329153"}, {"distinct_range": 412389506085677800, "num_eq": 7568697711230505904, "num_range": 412389506085677822, "upper_bound": "395991800"}, {"distinct_range": 0, "num_eq": 300000, "num_range": 3783537000078200410, "upper_bound": "1229309930"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000, "num_range": 0, "upper_bound": "1979-03-26 07:52:35.000838+00:00"}, {"distinct_range": 1891096318266934500, "num_eq": 496454460806103483, "num_range": 4866434943460602384, "upper_bound": "2024-08-10 00:12:06.000991+00:00"}, {"distinct_range": 5000, "num_eq": 2794214512634870652, "num_range": 5000, "upper_bound": "2029-12-12 17:39:57.000401+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17848462420, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "111896214"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 20000000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1660354816, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12334613463, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10535552847, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5699355303, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 900061686, "histo_col_type": "", "name": "__auto__", "null_count": 3868838487, "row_count": 20000000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9201682245, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "1970-08-07 22:35:26.000224+00:00"}, {"distinct_range": 0, "num_eq": 5154009307280512692, "num_range": 20000000, "upper_bound": "1980-06-11 01:48:39.000886+00:00"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 10000000, "upper_bound": "1987-01-12 20:09:42.000386+00:00"}, {"distinct_range": 0, "num_eq": 8482292743864999781, "num_range": 3000000, "upper_bound": "1999-11-21 20:08:32.000587+00:00"}, {"distinct_range": 3796534066782892000, "num_eq": 3098252569360825598, "num_range": 4343692975687420645, "upper_bound": "2012-10-26 18:57:48.000404+00:00"}, {"distinct_range": 0, "num_eq": 8527006965276942903, "num_range": 8900378080833669315, "upper_bound": "2017-03-07 06:58:30.000156+00:00"}, {"distinct_range": 1261981332238394400, "num_eq": 1262230501074539487, "num_range": 1766235601878032117, "upper_bound": "2025-11-25 15:47:55.000191+00:00"}, {"distinct_range": 0, "num_eq": 3913868771544115024, "num_range": 8000000, "upper_bound": "2030-03-10 04:36:24.000738+00:00"}, {"distinct_range": 4425539317148428300, "num_eq": 20, "num_range": 8690069932241352438, "upper_bound": "2031-09-26 16:10:14.000335+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 3936959281, "row_count": 20000000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7011586952, "histo_buckets": [{"distinct_range": 0, "num_eq": 800, "num_range": 0, "upper_bound": "-1.5090677738189697"}, {"distinct_range": 118948949570856540, "num_eq": 8359039876739301476, "num_range": 124248858160724211, "upper_bound": "-0.8568615317344666"}, {"distinct_range": 0, "num_eq": 0, "num_range": 3264412200247029652, "upper_bound": "-0.5050467252731323"}, {"distinct_range": 0, "num_eq": 20, "num_range": 800, "upper_bound": "0.2635083794593811"}, {"distinct_range": 0, "num_eq": 4006596495437031509, "num_range": 8857876444165227418, "upper_bound": "0.45774519443511963"}, {"distinct_range": 9000000, "num_eq": 8172371256976811785, "num_range": 9000000, "upper_bound": "0.4793725907802582"}, {"distinct_range": 7651.915355390597, "num_eq": 9123913802198319366, "num_range": 400000, "upper_bound": "0.7451812624931335"}, {"distinct_range": 898.2202300328657, "num_eq": 100000000000, "num_range": 400000, "upper_bound": "1.9069044589996338"}, {"distinct_range": 1301612094873003000, "num_eq": 1000, "num_range": 3538479954059333783, "upper_bound": "1.9179121255874634"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 341034859, "row_count": 20000000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 911923116, "histo_col_type": "", "name": "__auto__", "null_count": 18065753822, "row_count": 20000000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 115603072, "histo_col_type": "", "name": "__auto__", "null_count": 13085916677, "row_count": 20000000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7776826799, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000000}]';
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_4) REFERENCES table1 (col1_7);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1428260874402937838, "histo_col_type": "", "name": "__auto__", "null_count": 1142227144148624314, "row_count": 1580694485024041528}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 296406734038139909, "histo_col_type": "", "name": "__auto__", "null_count": 1391473372413437189, "row_count": 1580694485024041528}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 444608906237732111, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1580694485024041528}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 992702758017488089, "histo_col_type": "", "name": "__auto__", "null_count": 1411697811945558213, "row_count": 1580694485024041528}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1117891091163359156, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1580694485024041528}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 285626254182765821, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1580694485024041528}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1005887574551437814, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1580694485024041528}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 91735512718969986, "histo_col_type": "", "name": "__auto__", "null_count": 1364953003987920336, "row_count": 1580694485024041528}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 640418297598464113, "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 1580694485024041528}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 767340560464600251, "histo_col_type": "", "name": "__auto__", "null_count": 1058087866338595900, "row_count": 1580694485024041528}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 353123553521597520, "histo_col_type": "", "name": "__auto__", "null_count": 1231072151006595101, "row_count": 1580694485024041528}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 135921977914068612, "histo_col_type": "", "name": "__auto__", "null_count": 1172636385474696575, "row_count": 1580694485024041528}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1542974125707201338, "histo_col_type": "", "name": "__auto__", "null_count": 1030462293304778022, "row_count": 1580694485024041528}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1485493583026468189, "histo_buckets": [{"distinct_range": 0, "num_eq": 5964369657294556388, "num_range": 0, "upper_bound": "2020-12-10 00:30:13.000587+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 1580694485024041528}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46171728347920986, "histo_buckets": [{"distinct_range": 0, "num_eq": 400, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 1124491.1398443251, "num_eq": 3014294087096222546, "num_range": 6000000, "upper_bound": "1602637787"}, {"distinct_range": 0, "num_eq": 5000000, "num_range": 330946675954427001, "upper_bound": "1863636530"}, {"distinct_range": 556.1573971030142, "num_eq": 80000, "num_range": 1000, "upper_bound": "2177722346"}, {"distinct_range": 55866.87559586845, "num_eq": 1000000000, "num_range": 100000, "upper_bound": "2178191077"}, {"distinct_range": 3033547043419467000, "num_eq": 2176011156508940946, "num_range": 3033547043419466828, "upper_bound": "2547862143"}, {"distinct_range": 50, "num_eq": 200, "num_range": 50, "upper_bound": "3833789990"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 1199205815884459993, "row_count": 1580694485024041528}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1449348634800038482, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1580694485024041528}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 577706856256741887, "histo_col_type": "", "name": "__auto__", "null_count": 707222861178473297, "row_count": 3197529909729375179}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 548811125549632430, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3197529909729375179}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2605683162790146829, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3197529909729375179}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1264535853285844856, "histo_buckets": [{"distinct_range": 0, "num_eq": 1696928259397660472, "num_range": 0, "upper_bound": "0.43036913871765137"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 1284245171340058464, "row_count": 3197529909729375179}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2463627921449451019, "histo_col_type": "", "name": "__auto__", "null_count": 383522389160292340, "row_count": 3197529909729375179}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 320216271834164396, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3197529909729375179}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 411111399349492577, "histo_col_type": "", "name": "__auto__", "null_count": 2426978984028793603, "row_count": 3197529909729375179}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 694912596401070105, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000, "num_range": 0, "upper_bound": "3320058513"}, {"distinct_range": 0, "num_eq": 4579314985181773590, "num_range": 795747188520684413, "upper_bound": "4256031972"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 3197529909729375179}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2773828620951273727, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "0001-01-01 00:00:00+00:00"}, {"distinct_range": 10000, "num_eq": 2892843913388803710, "num_range": 10000, "upper_bound": "2000-10-29 03:03:54.000032+00:00"}, {"distinct_range": 600, "num_eq": 8808566331231653903, "num_range": 600, "upper_bound": "2021-07-20 04:31:43.000612+00:00"}, {"distinct_range": 920146081018700700, "num_eq": 90000000, "num_range": 920146081018700653, "upper_bound": "2023-06-27 10:51:52.000096+00:00"}, {"distinct_range": 176.11282822834602, "num_eq": 8000000000, "num_range": 800, "upper_bound": "294276-12-31 23:59:59.999999+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 1794354961522419574, "row_count": 3197529909729375179}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 146069714224139949, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3197529909729375179}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2675348165658329143, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3197529909729375179}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "infinity" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2679182&tab=buildLog) on [master@fe919cccf0aa8f400dbe9cce3e0dd1b401fd6dc5](https://github.com/cockroachdb/cockroach/commits/fe919cccf0aa8f400dbe9cce3e0dd1b401fd6dc5):

```
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 DESC, col3_2 DESC, col3_1);
        test_1        |             CREATE INDEX ON table3 (col3_2 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_1 ASC, col3_2 ASC, col3_0 ASC);
        test_1        |             CREATE TABLE table4 (col4_0 TIMESTAMP NOT NULL, col4_1 BIT(47) NOT NULL, col4_2 BIT(3) NOT NULL, col4_3 CHAR NOT NULL, col4_4 INT8 NOT NULL, col4_5 BOX2D NOT NULL, col4_6 VARCHAR NOT NULL, col4_7 TEXT NOT NULL, col4_8 REGTYPE NOT NULL, col4_9 INT8 NOT NULL, col4_10 REGPROC NOT NULL, col4_11 INT8 NOT NULL, col4_12 DECIMAL NOT NULL, col4_13 BOOL NOT NULL, PRIMARY KEY (col4_3, col4_9, col4_12, col4_5, col4_11, col4_4, col4_6, col4_10, col4_0, col4_1, col4_8, col4_7, col4_13));
        test_1        |             CREATE INDEX ON table4 (col4_11 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_12, col4_5 DESC, col4_4 DESC, col4_10 DESC, col4_9 ASC, col4_1 DESC, col4_6 ASC, col4_3, col4_0, col4_7 DESC, col4_8 ASC, col4_13 DESC, col4_11 ASC, col4_2 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_13, col4_1, col4_12 DESC, col4_9, col4_4, col4_5 ASC, col4_10 DESC, col4_8 ASC, col4_2 DESC, col4_7);
        test_1        |             CREATE INDEX ON table4 (col4_1 ASC, col4_6, col4_11 ASC, col4_9 ASC, col4_12 ASC, col4_0 ASC, col4_2, col4_7, col4_13 ASC, col4_4 DESC, col4_5, col4_3 ASC, col4_10, col4_8);
        test_1        |             CREATE INDEX ON table4 (col4_4 ASC, col4_7 DESC, col4_10 ASC, col4_3 ASC, col4_6 DESC, col4_8 ASC, col4_1 ASC, col4_12 DESC, col4_9 DESC, col4_5);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_5, col4_0, col4_8);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_0 DESC, col4_9 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_8);
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_14) REFERENCES table4 (col4_8);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('d', 'dz');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('xx', 'jklueg', 'aqe', 'kkotq', 'xc', 'qlewp');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('v', 'ibd', 'j');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('uaaa', 'lzgb', 'qlrgp', 'dpgvkj', 'm');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (1.05s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 CHAR NOT NULL, col1_1 BOX2D NOT NULL, col1_2 REGPROC NOT NULL, col1_3 INET NOT NULL, PRIMARY KEY (col1_1 DESC, col1_3 ASC, col1_2 DESC, col1_0), INDEX (col1_2, col1_3, col1_0) WHERE table1.col1_0 != e'\'':::STRING, INDEX (col1_1 DESC, col1_2 DESC, col1_0 ASC, col1_3 DESC) WHERE table1.col1_0 <= e'\U00002603', INDEX (col1_2) WHERE table1.col1_0 > e'\'':::STRING, UNIQUE (col1_2, col1_3 DESC, col1_1 DESC, col1_0) WHERE table1.col1_0 >= e'\'', UNIQUE (col1_0 DESC, col1_3, col1_1 DESC, col1_2 ASC) WHERE table1.col1_0 != e'\x00':::STRING, FAMILY (col1_0), FAMILY (col1_1, col1_3, col1_2));
        test_1        |             CREATE TABLE table2 (col1_1 BOX2D NOT NULL, col1_3 INET NOT NULL, col1_2 REGPROC NOT NULL, col1_0 CHAR NOT NULL, col2_4 BOOL NULL, col2_5 BOOL NULL, col2_6 TIMESTAMPTZ NULL, col2_7 DATE NOT NULL, col2_8 OID NULL, col2_9 REGPROC, PRIMARY KEY (col1_1 DESC, col1_3 ASC, col1_2 DESC, col1_0, col2_5, col2_6 ASC, col2_9 ASC, col2_4), INDEX (col2_7, col1_0 DESC, col1_1, col2_5 ASC, col2_8 DESC) WHERE NOT table2.col2_4, UNIQUE (col2_5 DESC, col2_4 ASC, col1_3 DESC, col1_0, col2_9 ASC, col1_1 DESC, col2_6) STORING (col2_7) WHERE ((((NOT table2.col2_5) OR (table2.col1_0 = 'X':::STRING)) AND (table2.col2_6 >= '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table2.col2_7 = '5874897-12-31':::DATE)) OR table2.col2_4, INDEX (col2_8 ASC, col2_6, col1_0 DESC, col2_4, col2_5, col1_1, col1_3 ASC, col2_7, col2_9 DESC) WHERE ((((table2.col2_6 >= '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ) AND (table2.col1_0 > '':::STRING)) AND table2.col2_5) AND (table2.col2_7 != '-infinity':::DATE)) AND table2.col2_4, UNIQUE (col2_4 DESC, col1_1 ASC, col1_0 DESC, col2_9 ASC, col2_6, col1_2) STORING (col2_7) WHERE ((((table2.col2_7 != '1970-01-01':::DATE) OR (table2.col2_6 < '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (NOT table2.col2_4)) OR table2.col2_5) OR (table2.col1_0 != '"':::STRING), UNIQUE (col1_1 DESC, col2_8 ASC, col1_0 DESC, col2_6 ASC, col2_4, col2_5, col1_2 DESC, col1_3, col2_7, col2_9) WHERE (((table2.col2_4 AND (table2.col2_6 != '3000-01-01 00:00:00+00:00')) OR (table2.col1_0 <= '"')) OR (table2.col2_7 = '-infinity')) AND (NOT table2.col2_5), INDEX (col1_0 ASC, col2_4 ASC, col2_5, col2_9 DESC, col2_6, col1_1 DESC) WHERE (table2.col2_7 >= '1970-01-01':::DATE) AND (table2.col1_0 > e'\U00002603':::STRING), INDEX (col1_3 ASC) STORING (col2_7) WHERE (NOT table2.col2_5) OR (NOT table2.col2_4), UNIQUE (col2_8 DESC, col2_9 ASC) WHERE ((((table2.col2_6 < '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ) OR table2.col2_5) OR (table2.col1_0 = e'\x00':::STRING)) AND (NOT table2.col2_4)) AND (table2.col2_7 < 'infinity':::DATE), FAMILY (col2_6), FAMILY (col2_4), FAMILY (col2_7), FAMILY (col2_8), FAMILY (col1_3), FAMILY (col2_5), FAMILY (col1_2, col2_9), FAMILY (col1_1), FAMILY (col1_0), UNIQUE (col1_0) WHERE ((table2.col1_0 = e'\x00') OR (table2.col2_6 <= '-2000-01-01 00:00:00+00:00')) AND table2.col2_5) INTERLEAVE IN PARENT table1 (col1_1, col1_3, col1_2, col1_0);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2014096903609355272, "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 5190525611828533029}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5032055933763161206, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000000, "num_range": 0, "upper_bound": "BOX(-1.65792621303487 -0.7776280538413537,1.6243867260343618 0.41634354064754076)"}, {"distinct_range": 30000, "num_eq": 7499878013297369396, "num_range": 30000, "upper_bound": "BOX(-1.294684786557224 -2.2339982968018957,0.6031340976356662 -0.7528798715735308)"}, {"distinct_range": 1332240922389810000, "num_eq": 80, "num_range": 7734862567278826609, "upper_bound": "BOX(-0.9910984110742099 -1.411189864744484,0.2635041553281683 -0.29617435472155385)"}, {"distinct_range": 200, "num_eq": 2038549219361433492, "num_range": 200, "upper_bound": "BOX(0.0790823165607677 -0.6378081631804554,0.683015019968148 -0.27593708849837273)"}, {"distinct_range": 0, "num_eq": 100, "num_range": 10000000000, "upper_bound": "BOX(0.2916888862608188 -1.0296629156449388,1.293220407481352 -0.35226056615900225)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 5190525611828533029}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1586603602813203132, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000000, "num_range": 0, "upper_bound": "1041588855"}, {"distinct_range": 0, "num_eq": 0, "num_range": 20000000000, "upper_bound": "1142881247"}, {"distinct_range": 627067171200958100, "num_eq": 5000000, "num_range": 2847448171775744060, "upper_bound": "1268871072"}, {"distinct_range": 0, "num_eq": 300000000, "num_range": 100, "upper_bound": "1362932449"}, {"distinct_range": 2878.4812786381663, "num_eq": 5000000000, "num_range": 7000, "upper_bound": "2997480932"}, {"distinct_range": 0, "num_eq": 77250955280619466, "num_range": 80000, "upper_bound": "3920104385"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 5190525611828533029}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1953096581806666237, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5190525611828533029}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2804283080994023604, "histo_buckets": [{"distinct_range": 0, "num_eq": 4941994904460962668, "num_range": 0, "upper_bound": "121.71.219.58/11"}, {"distinct_range": 345577.20072717563, "num_eq": 8875912753470937719, "num_range": 700000, "upper_bound": "205.138.226.220/14"}, {"distinct_range": 827219777795421700, "num_eq": 4891418960115604531, "num_range": 1932064843996058112, "upper_bound": "98.44.107.78/27"}, {"distinct_range": 0, "num_eq": 500000000, "num_range": 0, "upper_bound": "4692:213a:9fdf:1972:46e6:2835:dc17:a65a/104"}, {"distinct_range": 200050818854357730, "num_eq": 300, "num_range": 315374029629863390, "upper_bound": "6bfa:ca74:722c:9c2a:2351:a685:643e:44a7/111"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 3397526637260916534}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2336055027570432853, "histo_buckets": [{"distinct_range": 0, "num_eq": 2283911763927715272, "num_range": 0, "upper_bound": ")"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 3397526637260916534}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2845488018882373118, "histo_buckets": [{"distinct_range": 0, "num_eq": 3500252322979958628, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 4000000000, "num_range": 30000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 1079191881937354687, "row_count": 3397526637260916534}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2335652377713052336, "histo_buckets": [{"distinct_range": 0, "num_eq": 960379605149112870, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 900, "num_range": 8231341639034573477, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 803496171077440591, "row_count": 3397526637260916534}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2676372857616301985, "histo_buckets": [{"distinct_range": 0, "num_eq": 1937059703594037684, "num_range": 0, "upper_bound": "4714-11-24 BC"}, {"distinct_range": 4184684286197654500, "num_eq": 9000000000, "num_range": 4184684286197654578, "upper_bound": "1970-04-28"}, {"distinct_range": 10000000000, "num_eq": 300000000, "num_range": 10000000000, "upper_bound": "1971-10-21"}, {"distinct_range": 2872383138.661876, "num_eq": 7906084746536901194, "num_range": 4000000000, "upper_bound": "1977-09-04"}, {"distinct_range": 6191684720662722000, "num_eq": 867439821517066448, "num_range": 6805354821469539890, "upper_bound": "1977-11-10"}, {"distinct_range": 3924714144582132700, "num_eq": 2000000, "num_range": 3924714144582132678, "upper_bound": "1978-11-22"}, {"distinct_range": 180127.02438048957, "num_eq": 3000000000, "num_range": 500000, "upper_bound": "1982-12-18"}, {"distinct_range": 2525018031671511600, "num_eq": 10000000000, "num_range": 3104113307415598762, "upper_bound": "1984-01-29"}, {"distinct_range": 6000000, "num_eq": 1719125307080714149, "num_range": 6000000, "upper_bound": "1985-01-11"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 3397526637260916534}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2175006683496932421, "histo_col_type": "", "name": "__auto__", "null_count": 1382946751437022292, "row_count": 3397526637260916534}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2263657193350740638, "histo_buckets": [{"distinct_range": 0, "num_eq": 5781592364028458217, "num_range": 0, "upper_bound": "BOX(-0.3935737958805601 0.4584825932287444,0.024718540357561908 0.5456842017751837)"}, {"distinct_range": 74.14191595147705, "num_eq": 4071254025808119898, "num_range": 100, "upper_bound": "BOX(-0.35907176993657164 -1.3504594777579197,0.659716302854032 1.1853558789678331)"}, {"distinct_range": 8203648893241875000, "num_eq": 5650563895405760914, "num_range": 8203648893241875002, "upper_bound": "BOX(-0.22175311379684884 -0.19970759250862127,0.06448441368166069 0.18130401927756123)"}, {"distinct_range": 1645318456436819200, "num_eq": 2759895107570879988, "num_range": 1645318456436819092, "upper_bound": "BOX(-0.20651808582274409 -0.39370626245645923,0.6900577592058914 0.3776259200745516)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 3397526637260916534}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2132447149659743272, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3397526637260916534}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2289588731482275227, "histo_col_type": "", "name": "__auto__", "null_count": 2506303585829432548, "row_count": 3397526637260916534}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3059661844221460666, "histo_buckets": [{"distinct_range": 0, "num_eq": 4342168613045441300, "num_range": 0, "upper_bound": "585749006"}, {"distinct_range": 18731643735.614643, "num_eq": 100000, "num_range": 80000000000, "upper_bound": "625982721"}, {"distinct_range": 564305029.5627662, "num_eq": 70, "num_range": 40000000000, "upper_bound": "1228677488"}, {"distinct_range": 6.108890495248724, "num_eq": 2000, "num_range": 80, "upper_bound": "1790532966"}, {"distinct_range": 100000000, "num_eq": 20, "num_range": 100000000, "upper_bound": "1857832302"}, {"distinct_range": 1916284880.0381181, "num_eq": 548007991482568132, "num_range": 50000000000, "upper_bound": "2158177628"}, {"distinct_range": 40, "num_eq": 10000, "num_range": 40, "upper_bound": "2439245538"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 857952795172144590, "row_count": 3397526637260916534}]':::JSONB;
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_0) REFERENCES table2 (col1_0) ON UPDATE RESTRICT;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('le', 'vf', 'ey', 'widim');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('bieqt', 'quko', 'bfd', 'pgjq', 'esfesk');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('lqsya', 'vkdvta', 'bol', 'd', 'umfw', 'oytavv');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('um');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('baq');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21, "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000, "num_range": 0, "upper_bound": "BOX(-1.658170508465302 0.09988072769598183,-0.585073548069762 0.5440623662993568)"}, {"distinct_range": 8000, "num_eq": 3214246778222455683, "num_range": 8000, "upper_bound": "BOX(-1.0696196225907844 0.14890640332142335,-0.6107320791189044 0.3453088368609018)"}, {"distinct_range": 2595074.96968132, "num_eq": 9000, "num_range": 7000000, "upper_bound": "BOX(-0.6821521459378291 0.6315977607824713,1.1385559629338138 0.6390716008958333)"}, {"distinct_range": 90000000, "num_eq": 6007790369544339369, "num_range": 90000000, "upper_bound": "BOX(-0.19544435993601592 -0.8886614986214241,1.003326006774133 0.4142851153972282)"}, {"distinct_range": 3994715363399886300, "num_eq": 30000, "num_range": 3994715363399886520, "upper_bound": "BOX(0.10349325038313817 -0.6395133741751116,0.5389188757681437 0.8752052297083199)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15, "histo_buckets": [{"distinct_range": 0, "num_eq": 400, "num_range": 0, "upper_bound": "2285652569"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1933, "histo_col_type": "", "name": "__auto__", "null_count": 1454, "row_count": 10000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7033, "histo_buckets": [{"distinct_range": 0, "num_eq": 8505538566930271223, "num_range": 0, "upper_bound": "11.80.115.25/2"}, {"distinct_range": 0, "num_eq": 8000, "num_range": 100000, "upper_bound": "22.187.99.237/6"}, {"distinct_range": 1000000, "num_eq": 500, "num_range": 1000000, "upper_bound": "a862:e5fc:8434:b833:e72c:dc41:ec4e:aaf1/26"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 2000, "upper_bound": "4e91:18ab:d5e4:7665:29cf:dbee:99d4:be1f/55"}, {"distinct_range": 5889593626790484000, "num_eq": 1121101765298337640, "num_range": 8255787599951774896, "upper_bound": "2c38:ceec:1d19:6f61:e845:8cb0:7496:73b3/110"}, {"distinct_range": 0, "num_eq": 500, "num_range": 500000000, "upper_bound": "3c4d:b69c:7d47:9b45:4389:ca9e:8d3d:cbd7/124"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6482, "histo_buckets": [{"distinct_range": 0, "num_eq": 3180919282293034547, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 20000000000, "num_eq": 1000000, "num_range": 20000000000, "upper_bound": "@"}, {"distinct_range": 1000, "num_eq": 871357230820955886, "num_range": 1000, "upper_bound": "h"}, {"distinct_range": 0, "num_eq": 3013400161504455227, "num_range": 7017918577907582898, "upper_bound": "m"}, {"distinct_range": 800, "num_eq": 40, "num_range": 800, "upper_bound": "}"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2346, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 447201063024816640, "num_eq": 300000, "num_range": 447201063024816633, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 7271, "row_count": 10000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3832, "histo_buckets": [{"distinct_range": 0, "num_eq": 6616304201792324985, "num_range": 0, "upper_bound": "1970-03-01"}, {"distinct_range": 703793257498345100, "num_eq": 2237530207329562941, "num_range": 1890749999953214348, "upper_bound": "1970-12-08"}, {"distinct_range": 0, "num_eq": 900, "num_range": 2162969413714103568, "upper_bound": "1973-09-01"}, {"distinct_range": 0, "num_eq": 4000000, "num_range": 60000000000, "upper_bound": "1976-09-10"}, {"distinct_range": 0, "num_eq": 9096114495101326028, "num_range": 100, "upper_bound": "1978-09-28"}, {"distinct_range": 0, "num_eq": 200000000, "num_range": 8000, "upper_bound": "1987-04-24"}, {"distinct_range": 0, "num_eq": 7178661575446208763, "num_range": 160014898267942160, "upper_bound": "1997-05-09"}, {"distinct_range": 3674027029366289000, "num_eq": 200000000, "num_range": 3674027029366288695, "upper_bound": "5874897-12-31"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5377, "histo_buckets": [{"distinct_range": 0, "num_eq": 988906438276036977, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 234074512.6870758, "num_eq": 90, "num_range": 400000000, "upper_bound": "31903883"}, {"distinct_range": 0, "num_eq": 200000000, "num_range": 8875491983272066746, "upper_bound": "830184877"}, {"distinct_range": 9714690877.415836, "num_eq": 0, "num_range": 10000000000, "upper_bound": "924824040"}, {"distinct_range": 16699250579.49454, "num_eq": 20000000000, "num_range": 30000000000, "upper_bound": "957963270"}, {"distinct_range": 203688376.5298659, "num_eq": 1923144213420431143, "num_range": 300000000, "upper_bound": "2020667821"}, {"distinct_range": 242855225442624700, "num_eq": 3816664202777529437, "num_range": 7018229869664383004, "upper_bound": "3041580593"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 3381, "row_count": 10000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5187, "histo_col_type": "", "name": "__auto__", "null_count": 8885, "row_count": 10000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 648, "histo_buckets": [{"distinct_range": 0, "num_eq": 6009576906155629326, "num_range": 0, "upper_bound": "BOX(-1.5043680050397603 0.08968847333132857,-0.13973737128830188 0.2541332074611543)"}, {"distinct_range": 0, "num_eq": 7719210388993332140, "num_range": 100000000, "upper_bound": "BOX(-0.28384242649100344 1.0841358346072405,-0.12321174344423513 1.4071165761087912)"}, {"distinct_range": 177443565.90371472, "num_eq": 8713008389381950783, "num_range": 1000000000, "upper_bound": "BOX(-0.07183840400987407 -0.5939596542192114,0.6128242024108711 0.9011993460812315)"}, {"distinct_range": 7785219629610851000, "num_eq": 200000000, "num_range": 7785219629610850956, "upper_bound": "BOX(-0.07049113154950318 -0.742340328283842,0.08852866790514918 0.44558672952049294)"}, {"distinct_range": 17616136494.287666, "num_eq": 700000, "num_range": 50000000000, "upper_bound": "BOX(0.29357943833287903 -0.4731006379390934,1.3000501614352908 2.0113922984855606)"}, {"distinct_range": 50000000000, "num_eq": 7831390480065975492, "num_range": 50000000000, "upper_bound": "BOX(0.575757012403284 -0.4964086891575066,1.7531894174358298 0.6816370238234635)"}, {"distinct_range": 7394073847895684000, "num_eq": 1166592065259767554, "num_range": 7394073847895683950, "upper_bound": "BOX(0.8269324008425779 -0.12143423432459577,1.0869645698329276 1.0919614129464215)"}, {"distinct_range": 45844.5100129491, "num_eq": 5657193173021102993, "num_range": 60000, "upper_bound": "BOX(0.9040283693499402 0.6463663437320515,1.493544817936721 1.0910982445823214)"}, {"distinct_range": 0, "num_eq": 5826604497237752981, "num_range": 80, "upper_bound": "BOX(1.1527898455499344 -0.6961123260071097,1.200404403750035 1.5839121251358668)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6752, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3837, "histo_col_type": "BOOL", "name": "__auto__", "null_count": 4065, "row_count": 10000}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7744939837558977378, "histo_buckets": [{"distinct_range": 0, "num_eq": 2951048501738269223, "num_range": 0, "upper_bound": "BOX(-2.0200706726919657 -1.6818717461940524,-1.3417180226584868 1.3055427250111875)"}, {"distinct_range": 7050138.576189045, "num_eq": 9205742453318272788, "num_range": 20000000, "upper_bound": "BOX(-0.7351921305936248 -1.6345260299280269,0.9767600950371073 -0.08710227058528752)"}, {"distinct_range": 0, "num_eq": 20000000000, "num_range": 6391789913855832884, "upper_bound": "BOX(-0.7164961051226912 0.6379858891479271,1.6775478306932814 1.1550024363878355)"}, {"distinct_range": 0, "num_eq": 8486500835633434459, "num_range": 9000000000, "upper_bound": "BOX(-0.30989182211805355 -0.37689827560495515,1.623875113203181 -0.10294805576395238)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 8761222895278688571}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 637390296151389601, "histo_buckets": [{"distinct_range": 0, "num_eq": 6345618826534674517, "num_range": 0, "upper_bound": "253827936"}, {"distinct_range": 5773877866511900000, "num_eq": 0, "num_range": 8363673042792308926, "upper_bound": "353515775"}, {"distinct_range": 5590938302215093000, "num_eq": 200000000, "num_range": 5638650317829616749, "upper_bound": "1331475946"}, {"distinct_range": 1461390655578006800, "num_eq": 8408371664674596067, "num_range": 6828397710879809159, "upper_bound": "1483782194"}, {"distinct_range": 0, "num_eq": 500, "num_range": 0, "upper_bound": "2172003689"}, {"distinct_range": 0, "num_eq": 3986584237691969740, "num_range": 700000000, "upper_bound": "2645193096"}, {"distinct_range": 325954684084473300, "num_eq": 90000, "num_range": 325954684084473309, "upper_bound": "3132969804"}, {"distinct_range": 50000, "num_eq": 80000000000, "num_range": 50000, "upper_bound": "3555675837"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 300, "upper_bound": "4258932994"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 8761222895278688571}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2931068674326947620, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8761222895278688571}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5132901828234966190, "histo_buckets": [{"distinct_range": 0, "num_eq": 9141719778196160928, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 0, "num_eq": 700, "num_range": 50000, "upper_bound": "\\u0008"}, {"distinct_range": 497691.9701318993, "num_eq": 1349050139397978518, "num_range": 1000000, "upper_bound": "\\u0019"}, {"distinct_range": 600000000, "num_eq": 40000000000, "num_range": 600000000, "upper_bound": "3"}, {"distinct_range": 10, "num_eq": 900000000, "num_range": 10, "upper_bound": ";"}, {"distinct_range": 348692224187767740, "num_eq": 3646881384580037962, "num_range": 1577848326443619363, "upper_bound": "M"}, {"distinct_range": 1877754247316264400, "num_eq": 10000, "num_range": 1877754247316264333, "upper_bound": "~"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 8761222895278688571}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5056741757425053240, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8582797576347861278}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 654958172515242551, "histo_buckets": [{"distinct_range": 0, "num_eq": 1325367323072655571, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 6563581741719932000, "num_eq": 500000, "num_range": 6563581741719932178, "upper_bound": "51144658"}, {"distinct_range": 0, "num_eq": 8877442339922870676, "num_range": 4250305426146927094, "upper_bound": "433508597"}, {"distinct_range": 6000, "num_eq": 770809243257629925, "num_range": 6000, "upper_bound": "1696309299"}, {"distinct_range": 4448744.393823541, "num_eq": 9137447057533803027, "num_range": 7000000, "upper_bound": "2650680926"}, {"distinct_range": 5690942283407374000, "num_eq": 7444963205362780923, "num_range": 6598308267542632822, "upper_bound": "2720659094"}, {"distinct_range": 5000, "num_eq": 1251170026938049021, "num_range": 5000, "upper_bound": "3091438447"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 1954068532852014100, "row_count": 8582797576347861278}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1428954777235575842, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000000, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 0, "num_eq": 0, "num_range": 10000000, "upper_bound": "BOX(-2.076093391390872 -0.33219995731159657,0.4116974552834991 0.8449064719034043)"}, {"distinct_range": 90000000000, "num_eq": 50000, "num_range": 90000000000, "upper_bound": "BOX(-1.6352862809821214 -1.2033174630119836,-0.4397910289636614 0.7682342993898783)"}, {"distinct_range": 52818901.27549476, "num_eq": 5248024724857639054, "num_range": 80000000, "upper_bound": "BOX(-1.5564310415922944 -0.35004280538316124,-1.3955111835457323 0.025375492095107754)"}, {"distinct_range": 0, "num_eq": 6528876665718513317, "num_range": 4000000000, "upper_bound": "BOX(-0.5729582329471704 -1.2652801992624183,-0.5067532661027485 0.9091883795404848)"}, {"distinct_range": 0, "num_eq": 8389816124148043337, "num_range": 2000000, "upper_bound": "BOX(-0.510065726933381 -0.07195917195550044,-0.05990017587978813 0.7008787270103598)"}, {"distinct_range": 0, "num_eq": 8115563341871979052, "num_range": 700, "upper_bound": "BOX(-0.07441582937900346 0.3661451989282919,0.023608279613894112 1.3940201781087576)"}, {"distinct_range": 7451726974641609000, "num_eq": 8017847573695272667, "num_range": 7451726974641608876, "upper_bound": "BOX(0.21303067893861555 -0.6054240455372333,0.33942070011176084 0.028001483784849734)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 8582797576347861278}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1932642692002398830, "histo_buckets": [{"distinct_range": 0, "num_eq": 5123211424122020265, "num_range": 0, "upper_bound": "0.0.0.0/0"}, {"distinct_range": 7242440365637249000, "num_eq": 0, "num_range": 7242440365637248655, "upper_bound": "8fb:29b1:57ec:b770:fe27:fd39:8204:3178/10"}, {"distinct_range": 6803862251999834000, "num_eq": 400000, "num_range": 6803862251999834505, "upper_bound": "e045:3cf7:7e7f:8f71:31a6:456f:8eb3:7ef1/90"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 8582797576347861278}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1673904228459340363, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "\\u0001"}, {"distinct_range": 0, "num_eq": 6702550609829621134, "num_range": 500000, "upper_bound": "\\u0004"}, {"distinct_range": 0, "num_eq": 5760241822308349374, "num_range": 6000000000, "upper_bound": "\\n"}, {"distinct_range": 0, "num_eq": 0, "num_range": 5706097607684488463, "upper_bound": "\\""}, {"distinct_range": 0, "num_eq": 1338120905527889538, "num_range": 1000000000, "upper_bound": "N"}, {"distinct_range": 900, "num_eq": 7026980666431043327, "num_range": 900, "upper_bound": "k"}, {"distinct_range": 3.2945412184354956, "num_eq": 4124344716010416388, "num_range": 10, "upper_bound": "o"}, {"distinct_range": 438348.49628068483, "num_eq": 5926430805375365034, "num_range": 40000000, "upper_bound": "\U00002603"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 8582797576347861278}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 694336444166471962, "histo_col_type": "BOOL", "name": "__auto__", "null_count": 7414553039591234975, "row_count": 8582797576347861278}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2655222358247362660, "histo_buckets": [{"distinct_range": 0, "num_eq": 30, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 246457356106328480, "num_eq": 7000000000, "num_range": 885253212542308543, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 4686059048729281834, "row_count": 8582797576347861278}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 792588541342959901, "histo_col_type": "", "name": "__auto__", "null_count": 3960769351674064769, "row_count": 8582797576347861278}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7841338794008566952, "histo_buckets": [{"distinct_range": 0, "num_eq": 4990577627311986995, "num_range": 0, "upper_bound": "-infinity"}, {"distinct_range": 0, "num_eq": 70, "num_range": 3694772707310026969, "upper_bound": "1981-09-22"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 0, "upper_bound": "1989-05-05"}, {"distinct_range": 0, "num_eq": 400, "num_range": 20, "upper_bound": "1990-11-22"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 100000000, "upper_bound": "infinity"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 8582797576347861278}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6474792097236823421, "histo_col_type": "", "name": "__auto__", "null_count": 2017636038829600271, "row_count": 8582797576347861278}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: there is no unique constraint matching given keys for referenced table table2 (SQLSTATE 23503)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2683687&tab=buildLog) on [master@fdfcf2335e9743ecec1d944f8924982d6f64ccd6](https://github.com/cockroachdb/cockroach/commits/fdfcf2335e9743ecec1d944f8924982d6f64ccd6):

```
        test_1        |             CREATE TABLE table4 (col1_15 TIMETZ NOT NULL, col1_9 BIT(17) NOT NULL, col1_14 UUID NOT NULL, col1_10 TIMESTAMP NOT NULL, col1_7 FLOAT8 NOT NULL, col1_6 TIMESTAMP NOT NULL, col1_1 CHAR NOT NULL, col1_16 CHAR NOT NULL, col3_20 VARCHAR NOT NULL, col3_19 FLOAT8 NOT NULL, col3_16 REGTYPE NOT NULL, col3_18 DATE, col3_14 TIMETZ NOT NULL, col3_12 TIME, col3_10 TIMETZ, col3_13 DECIMAL NOT NULL, col4_16 NAME NOT NULL, col4_17 REGCLASS, col4_18 TIMESTAMP, col4_19 DECIMAL NOT NULL, col4_20 DATE, col4_21 TIMESTAMPTZ, col4_22 TIMESTAMPTZ NULL, col4_23 REGPROCEDURE NULL, col4_24 TIMESTAMP NULL, col4_25 "char" NULL, col4_26 BYTEA NULL, PRIMARY KEY (col1_15, col1_9, col1_14, col1_10, col1_7, col1_6, col1_1, col1_16, col3_20, col3_19, col3_16, col3_18, col3_14, col3_12, col3_10, col3_13, col4_26, col4_23));
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_22 ASC, col4_20 DESC, col3_14, col1_9, col4_25, col4_21 DESC, col4_26 ASC, col3_20, col3_10, col1_10 DESC, col3_19 ASC, col1_7 DESC, col4_16, col1_16, col1_15, col1_14, col4_18 DESC, col3_12 ASC, col4_19 ASC, col3_18 DESC, col4_24 ASC, col1_1 DESC);
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_8) REFERENCES table3 (col3_9) ON DELETE CASCADE;
        test_1        |             ALTER TABLE table4 ADD FOREIGN KEY (col1_15) REFERENCES table1 (col1_12);
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_0, col2_1) REFERENCES table3 (col3_8, col1_1) ON DELETE SET DEFAULT ON UPDATE RESTRICT;
        test_1        |             ALTER TABLE table4 ADD FOREIGN KEY (col3_18, col3_19, col1_10) REFERENCES table3 (col3_18, col1_7, col1_10) ON UPDATE CASCADE;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('f', 'zczsfr', 'fj');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: foreign key constraint "table2_col2_0_fkey" cannot be implemented (SQLSTATE 42804)
        test_1        |     --- FAIL: TestCompare/mutators (1.24s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 TIMESTAMPTZ NOT NULL, col1_1 FLOAT8 NOT NULL, col1_2 VARCHAR NOT NULL, col1_3 STRING NULL, col1_4 FLOAT8, col1_5 BOOL, col1_6 "char" NOT NULL, col1_7 VARCHAR NOT NULL, col1_8 INET NOT NULL, col1_9 TIMESTAMPTZ[] NULL, col1_10 REGNAMESPACE NOT NULL, col1_11 REGNAMESPACE NOT NULL, col1_12 REGPROC NOT NULL, col1_13 INTERVAL NOT NULL, col1_14 FLOAT4 NOT NULL, col1_15 VARCHAR[] NOT NULL, col1_16 DATE NOT NULL, PRIMARY KEY (col1_14 ASC, col1_6 ASC, col1_12, col1_16, col1_1 DESC, col1_10 DESC, col1_8, col1_13 ASC, col1_0 ASC, col1_7, col1_2 ASC), UNIQUE (col1_2 ASC) STORING (col1_5, col1_9, col1_11) WHERE ((((((table1.col1_0 < '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ) OR (table1.col1_3 <= 'X':::STRING)) OR (table1.col1_4 != 5e-324:::FLOAT8)) AND (table1.col1_2 = e'\'':::STRING)) OR (table1.col1_1 > 5e-324:::FLOAT8)) OR (table1.col1_16 != '1970-01-01':::DATE)) AND (table1.col1_7 <= e'\'':::STRING), UNIQUE (col1_12 DESC, col1_13 ASC, col1_16) STORING (col1_4, col1_9) WHERE (((((((table1.col1_2 > '"') OR (table1.col1_6 < e'\U00002603')) OR (NOT table1.col1_5)) AND (table1.col1_7 < '')) AND (table1.col1_16 > '4714-11-24 BC')) AND (table1.col1_0 != '0001-01-01 00:00:00+00:00')) OR (table1.col1_14 < 0.0)) AND (table1.col1_1 > 5e-324), FAMILY (col1_12, col1_15), FAMILY (col1_10), FAMILY (col1_6, col1_2), FAMILY (col1_4), FAMILY (col1_8, col1_11), FAMILY (col1_5, col1_14, col1_0), FAMILY (col1_9), FAMILY (col1_16, col1_13), FAMILY (col1_7), FAMILY (col1_3, col1_1));
        test_1        |             CREATE TABLE table2 (col2_0 TIMESTAMP NULL, col2_1 INTERVAL NOT NULL, col2_2 REGCLASS NOT NULL, col2_3 TIME NOT NULL, PRIMARY KEY (col2_2 DESC, col2_1 ASC, col2_3 ASC), INDEX (col2_1 DESC, col2_0 ASC, col2_2 DESC) WHERE table2.col2_0 > '-4713-11-24 00:00:00', INDEX (col2_2 ASC) WHERE (table2.col2_3 > '24:00:00':::TIME) AND (table2.col2_0 != '3000-01-01 00:00:00':::TIMESTAMP), UNIQUE (col2_2), FAMILY (col2_0, col2_2), FAMILY (col2_1, col2_3));
        test_1        |             CREATE TABLE table3 (col3_0 BYTES NOT NULL, col3_1 BYTES[] NOT NULL, col3_2 STRING NOT NULL, col3_3 INT8 NOT NULL, col3_4 INT2, col3_5 REGTYPE NOT NULL, col3_6 DECIMAL NOT NULL, col3_7 FLOAT4 NOT NULL, col3_8 INET NOT NULL, col3_9 FLOAT8 NOT NULL, col3_10 FLOAT4 NOT NULL, col3_11 DATE[] NOT NULL, col3_12 INT8 NOT NULL, col3_13 TIMESTAMPTZ NOT NULL, col3_14 REGTYPE NOT NULL, col3_15 TIMESTAMPTZ, col3_16 TIMETZ NOT NULL, col3_17 REGCLASS NOT NULL, PRIMARY KEY (col3_10, col3_9 ASC, col3_3, col3_16, col3_14 DESC, col3_8, col3_0, col3_6 ASC, col3_7, col3_17, col3_13, col3_2 ASC, col3_5), INDEX (col3_9 ASC) STORING (col3_1, col3_15) WHERE (((((((((((table3.col3_13 != '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ) OR (table3.col3_6 <= (-1.234E+401):::DECIMAL)) AND (table3.col3_3 > (-128):::INT8)) OR (table3.col3_0 = e'\\xff':::BYTES)) OR (table3.col3_15 > '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table3.col3_16 >= '24:00:00-15:59:00':::TIMETZ)) OR (table3.col3_9 != '+Inf':::FLOAT8)) OR (table3.col3_7 < 1.401298464324817e-45:::FLOAT8)) OR (table3.col3_4 < 1:::INT8)) OR (table3.col3_2 <= '':::STRING)) AND (table3.col3_12 < 9223372036854775807:::INT8)) AND (table3.col3_10 >= 3.4028234663852886e+38:::FLOAT8), INDEX (col3_10, col3_7 ASC, col3_12 DESC, col3_14, col3_3 DESC, col3_2 ASC, col3_5 ASC, col3_17 ASC, col3_15 ASC, col3_13 ASC, col3_4 DESC) STORING (col3_1, col3_11) WHERE table3.col3_7 > -1.0, FAMILY (col3_11), FAMILY (col3_15), FAMILY (col3_9, col3_8), FAMILY (col3_0), FAMILY (col3_17), FAMILY (col3_16), FAMILY (col3_1, col3_13), FAMILY (col3_14), FAMILY (col3_5), FAMILY (col3_7), FAMILY (col3_3), FAMILY (col3_6, col3_2, col3_12, col3_10), FAMILY (col3_4));
        test_1        |             CREATE TABLE table4 (col1_14 FLOAT4 NOT NULL, col1_6 "char" NOT NULL, col1_12 REGPROC NOT NULL, col1_16 DATE NOT NULL, col1_1 FLOAT8 NOT NULL, col1_10 REGNAMESPACE NOT NULL, col1_8 INET NOT NULL, col1_13 INTERVAL NOT NULL, col1_0 TIMESTAMPTZ NOT NULL, col1_7 VARCHAR NOT NULL, col1_2 VARCHAR NOT NULL, col4_11 BOOL NOT NULL, col4_12 BIT(28) NULL, col4_13 TIMETZ NULL, col4_14 REGTYPE, col4_15 TIMETZ NOT NULL, col4_16 VARCHAR NULL, col4_17 FLOAT4 NOT NULL, col4_18 REGNAMESPACE NOT NULL, col4_19 FLOAT8 NULL, PRIMARY KEY (col1_14 ASC, col1_6 ASC, col1_12, col1_16, col1_1 DESC, col1_10 DESC, col1_8, col1_13 ASC, col1_0 ASC, col1_7, col1_2 ASC, col4_16 DESC, col4_14, col4_11 ASC), INDEX (col4_18 ASC, col1_8, col1_10, col1_0 ASC, col1_1 DESC, col1_14, col1_16 ASC, col4_19 DESC, col1_13, col4_17 DESC, col1_12 DESC, col4_13 DESC, col4_14 DESC, col1_6 ASC, col4_12 ASC, col1_7 ASC, col4_15, col4_16, col4_11), INDEX (col4_17 DESC, col1_16 DESC, col4_19 DESC, col4_13, col1_7, col1_2, col4_11 DESC) WHERE ((((((((((((table4.col4_13 > '00:00:00+15:59:00':::TIMETZ) AND (table4.col4_19 < 1.0:::FLOAT8)) AND (NOT table4.col4_11)) AND (table4.col1_2 < e'\'':::STRING)) AND (table4.col1_16 <= 'infinity':::DATE)) OR (table4.col4_16 = '"':::STRING)) OR (table4.col1_7 < '"':::STRING)) OR (table4.col1_14 < 0.0:::FLOAT8)) OR (table4.col4_17 < '-Inf':::FLOAT8)) OR (table4.col4_15 < '24:00:00-15:59:00':::TIMETZ)) AND (table4.col1_6 < e'\'':::STRING)) OR (table4.col1_0 = '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) OR (table4.col1_1 >= 0.0:::FLOAT8), UNIQUE (col1_7 DESC, col1_16, col4_14, col4_15 ASC, col4_11 ASC, col1_0, col4_13 DESC, col1_8 DESC, col4_18 DESC, col1_2, col4_16 ASC, col1_14, col1_13, col4_17) WHERE ((((((((table4.col4_16 <= e'\'':::STRING) OR (table4.col1_7 != e'\'':::STRING)) OR (table4.col4_13 > '00:00:00+15:59:00':::TIMETZ)) AND (table4.col1_1 = '-Inf':::FLOAT8)) OR table4.col4_11) OR (table4.col1_16 <= '1970-01-01':::DATE)) AND (table4.col1_14 != 0.0:::FLOAT8)) OR (table4.col4_17 < 0.0:::FLOAT8)) OR (table4.col1_0 = '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ), UNIQUE (col1_1 ASC, col1_12 DESC, col1_16 ASC, col4_13 DESC, col1_6 DESC) STORING (col4_15, col4_18, col4_19) WHERE (((table4.col4_11 AND (table4.col1_7 <= e'\'':::STRING)) AND (table4.col1_6 = 'X':::STRING)) AND (table4.col4_13 != '00:00:00+15:59:00':::TIMETZ)) OR (table4.col1_0 != '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ), UNIQUE (col4_14 DESC, col1_1 ASC, col1_6 ASC, col1_2, col1_12, col1_14 ASC, col1_16) STORING (col4_17) WHERE (((((((table4.col1_1 >= '-Inf':::FLOAT8) AND (table4.col4_16 < '':::STRING)) AND (table4.col1_14 = 0.0:::FLOAT8)) AND (table4.col4_19 = 1.7976931348623157e+308:::FLOAT8)) OR (table4.col1_0 > '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) AND (table4.col1_16 != '5874897-12-31':::DATE)) AND (table4.col1_2 > 'X':::STRING)) OR (table4.col1_6 = 'X':::STRING), FAMILY (col4_17, col4_19, col4_16), FAMILY (col1_14), FAMILY (col4_11), FAMILY (col1_0), FAMILY (col1_7), FAMILY (col4_18), FAMILY (col1_10), FAMILY (col1_8), FAMILY (col4_15, col1_12, col1_1, col1_16, col4_12), FAMILY (col1_13), FAMILY (col4_13), FAMILY (col1_6), FAMILY (col4_14, col1_2), UNIQUE (col1_6, col1_7) STORING (col4_12, col4_17, col4_18, col4_19) WHERE (((((((table4.col1_7 > e'\'') AND (table4.col4_16 < e'\x00')) AND (table4.col1_0 >= '0001-01-01 00:00:00+00:00')) OR (table4.col1_6 != 'X')) AND (table4.col4_13 <= '24:00:00-15:59:00')) OR (table4.col4_19 > +Inf)) OR (table4.col1_14 < 1.401298464324817e-45)) OR (table4.col1_2 >= '"')) INTERLEAVE IN PARENT table1 (col1_14, col1_6, col1_12, col1_16, col1_1, col1_10, col1_8, col1_13, col1_0, col1_7, col1_2);
        test_1        |             CREATE TABLE table5 (col1_14 FLOAT4 NOT NULL, col1_6 "char" NOT NULL, col1_12 REGPROC NOT NULL, col1_16 DATE NOT NULL, col1_1 FLOAT8 NOT NULL, col1_10 REGNAMESPACE NOT NULL, col1_8 INET NOT NULL, col1_13 INTERVAL NOT NULL, col1_0 TIMESTAMPTZ NOT NULL, col1_7 VARCHAR NOT NULL, col1_2 VARCHAR NOT NULL, col5_11 REGPROC NULL, col5_12 TIME NOT NULL, col5_13 BOX2D, col5_14 "char", col5_15 INET, col5_16 DECIMAL, col5_17 BIT(30) NOT NULL, col5_18 REGPROC NOT NULL, col5_19 REGNAMESPACE NULL, col5_20 INT2 NOT NULL, col5_21 INET NULL, col5_22 INT2 NOT NULL, col5_23 FLOAT4 NOT NULL, col5_24 FLOAT4 NULL, PRIMARY KEY (col1_14 ASC, col1_6 ASC, col1_12, col1_16, col1_1 DESC, col1_10 DESC, col1_8, col1_13 ASC, col1_0 ASC, col1_7, col1_2 ASC, col5_17 ASC, col5_12, col5_21 ASC, col5_14, col5_18 DESC, col5_13 DESC, col5_22, col5_16 ASC, col5_20, col5_11 DESC, col5_19 DESC, col5_15 ASC), FAMILY (col1_10), FAMILY (col5_20, col1_16, col5_15), FAMILY (col5_19), FAMILY (col1_14), FAMILY (col1_7, col1_6), FAMILY (col1_0), FAMILY (col1_13, col1_8), FAMILY (col5_14, col1_2), FAMILY (col5_21, col1_12, col5_22), FAMILY (col5_23), FAMILY (col5_11), FAMILY (col5_13, col5_12, col5_16), FAMILY (col5_18, col1_1, col5_24), FAMILY (col5_17)) INTERLEAVE IN PARENT table1 (col1_14, col1_6, col1_12, col1_16, col1_1, col1_10, col1_8, col1_13, col1_0, col1_7, col1_2);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2310575749555499727, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2684338147266972377}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 608597942129745904, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2684338147266972377}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 326033592992012484, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-Inf"}, {"distinct_range": 100000000000, "num_eq": 200000000, "num_range": 100000000000, "upper_bound": "0.344734787940979"}, {"distinct_range": 0, "num_eq": 6388929058083158013, "num_range": 200000000, "upper_bound": "1.156491756439209"}, {"distinct_range": 10, "num_eq": 50000000000, "num_range": 10, "upper_bound": "1.9676094055175781"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 2684338147266972377}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 589826521071249749, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2684338147266972377}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1543095233171422985, "histo_col_type": "", "name": "__auto__", "null_count": 195420185331612257, "row_count": 2684338147266972377}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1479177390596964274, "histo_col_type": "", "name": "__auto__", "null_count": 1287561538608469494, "row_count": 2684338147266972377}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1132419401087529872, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2684338147266972377}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1585855807431094868, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2684338147266972377}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1564793305578192440, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2684338147266972377}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 504917625577069370, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2684338147266972377}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 914151327107818764, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2684338147266972377}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1751872310582094750, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2684338147266972377}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 546503261507158121, "histo_buckets": [{"distinct_range": 0, "num_eq": 2270891403795252190, "num_range": 0, "upper_bound": "\\u000e\\u0012\\\\\\u000b3"}, {"distinct_range": 5410033163805823000, "num_eq": 30000000, "num_range": 5410033163805823483, "upper_bound": "\\u001cn{\\u0014\\u00084"}, {"distinct_range": 4865044800629188000, "num_eq": 120734529159394528, "num_range": 4865044800629187141, "upper_bound": "7]@\\u0008hs"}, {"distinct_range": 8133758488469304000, "num_eq": 90, "num_range": 8133758488469304491, "upper_bound": "A,"}, {"distinct_range": 3535374.1215885123, "num_eq": 8011496370979907865, "num_range": 4000000, "upper_bound": "S~dg\\u0007[u&"}, {"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "X"}, {"distinct_range": 0, "num_eq": 70, "num_range": 0, "upper_bound": "e5"}, {"distinct_range": 0, "num_eq": 2167872811226173884, "num_range": 6901041867995468130, "upper_bound": "kh-\\u0005"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 2684338147266972377}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1004095979976742150, "histo_col_type": "", "name": "__auto__", "null_count": 1948317242093492922, "row_count": 2684338147266972377}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2034238520165475841, "histo_col_type": "", "name": "__auto__", "null_count": 1682441990838426625, "row_count": 2684338147266972377}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1330078793045336010, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2684338147266972377}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2330694792958663717, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "1174525576"}, {"distinct_range": 1347661430276232400, "num_eq": 8618083241483847553, "num_range": 2901073228235623629, "upper_bound": "2231997373"}, {"distinct_range": 700000, "num_eq": 6020665365572162783, "num_range": 700000, "upper_bound": "2505172199"}, {"distinct_range": 0, "num_eq": 1310107804011200011, "num_range": 6220511940831948053, "upper_bound": "2560357213"}, {"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "3023940596"}, {"distinct_range": 5120.062541737859, "num_eq": 700000000, "num_range": 6000, "upper_bound": "3240120773"}, {"distinct_range": 500, "num_eq": 2058788708493102733, "num_range": 500, "upper_bound": "3522750949"}, {"distinct_range": 8937637.85376089, "num_eq": 3000000, "num_range": 20000000, "upper_bound": "3523665229"}, {"distinct_range": 20, "num_eq": 4000000000, "num_range": 20, "upper_bound": "3603722297"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 2684338147266972377}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2358647027622363581, "histo_col_type": "", "name": "__auto__", "null_count": 21170995730686760, "row_count": 6459207599754715710}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1122435364507456991, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "1 mon 1 day"}, {"distinct_range": 1002364.821205181, "num_eq": 2203496677113037051, "num_range": 9000000, "upper_bound": "51 years 5 mons 414 days 01:33:17.400441"}, {"distinct_range": 2118141549760549000, "num_eq": 1553286195315730408, "num_range": 2118141549760549152, "upper_bound": "76 years 4 mons 701 days 19:01:01.511055"}, {"distinct_range": 80000000, "num_eq": 3000000, "num_range": 80000000, "upper_bound": "79 years 4 mons 12 days 15:44:58.890576"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 6459207599754715710}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4036267722269463857, "histo_buckets": [{"distinct_range": 0, "num_eq": 70, "num_range": 0, "upper_bound": "1264499373"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 6459207599754715710}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5293748888049720160, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6459207599754715710}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 965133808474384429, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5616980124644572995}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4047480174318875585, "histo_buckets": [{"distinct_range": 0, "num_eq": 2641706106225589372, "num_range": 0, "upper_bound": "-0.624363124370575"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 283157257549335263, "upper_bound": "-0.2066689133644104"}, {"distinct_range": 794793852728309900, "num_eq": 100000000000, "num_range": 967132966970422065, "upper_bound": "-0.06183040142059326"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 3798794913946077890, "upper_bound": "-0.028696469962596893"}, {"distinct_range": 5453936.352261231, "num_eq": 6646901316580596926, "num_range": 90000000, "upper_bound": "0.20200365781784058"}, {"distinct_range": 0, "num_eq": 3028022806399800540, "num_range": 1923263742588828029, "upper_bound": "1.3445348739624023"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 5616980124644572995}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3754964895211120923, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5616980124644572995}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1082487041231265011, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5616980124644572995}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1033276439152589160, "histo_col_type": "", "name": "__auto__", "null_count": 5030707781494473528, "row_count": 5616980124644572995}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2517110433524491591, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5616980124644572995}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4359021462897911245, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5616980124644572995}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5564551683206389437, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "2.3777672028831525"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 5616980124644572995}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1093919510718211827, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5616980124644572995}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4560025863968087575, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5616980124644572995}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4227216235137514461, "histo_col_type": "", "name": "__auto__", "null_count": 1443086245236238748, "row_count": 5616980124644572995}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1108681265277154512, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5616980124644572995}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1174025202209778135, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5616980124644572995}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3540261480944453434, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5616980124644572995}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3497915133395496085, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5616980124644572995}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1309356613435236328, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5616980124644572995}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3621527897177738603, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5616980124644572995}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 890539939231019207, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5616980124644572995}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1064098695732743603, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5974473588768956450}, {"columns": ["col4_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2430389639249911950, "histo_col_type": "", "name": "__auto__", "null_count": 4666765370023416446, "row_count": 5974473588768956450}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 847406449138122304, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5974473588768956450}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5643319941921081347, "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 5974473588768956450}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 506645621728328486, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5974473588768956450}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3533106336942342463, "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 5974473588768956450}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 769391754766369982, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5974473588768956450}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5949496014655511367, "histo_col_type": "", "name": "__auto__", "null_count": 3341143389488920912, "row_count": 5974473588768956450}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3932650626115644222, "histo_col_type": "", "name": "__auto__", "null_count": 2414930414515546189, "row_count": 5974473588768956450}, {"columns": ["col4_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4173900221505067754, "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 5974473588768956450}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1764333070005594458, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5974473588768956450}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2746920231733626391, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5974473588768956450}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 965209354387868356, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5974473588768956450}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4594326492117872086, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5974473588768956450}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3353495275376552210, "histo_buckets": [{"distinct_range": 0, "num_eq": 7479789285262061573, "num_range": 0, "upper_bound": "-0.5959458947181702"}, {"distinct_range": 597908.9108372596, "num_eq": 1667468286312020207, "num_range": 7000000, "upper_bound": "-0.4039437770843506"}, {"distinct_range": 20, "num_eq": 1165050791308458508, "num_range": 20, "upper_bound": "-0.36417829990386963"}, {"distinct_range": 0, "num_eq": 2077349391304342970, "num_range": 4461651794261370255, "upper_bound": "-0.053577203303575516"}, {"distinct_range": 1072256994728970800, "num_eq": 800000, "num_range": 1072256994728970795, "upper_bound": "0.015880882740020752"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 3468148397595620839, "upper_bound": "+Inf"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 5974473588768956450}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5241079793797890094, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-0.9245904684066772"}, {"distinct_range": 0, "num_eq": 6170320620405243000, "num_range": 7562327905276212180, "upper_bound": "0.02493995614349842"}, {"distinct_range": 0, "num_eq": 393710786405927598, "num_range": 1541327460868791457, "upper_bound": "1.3095595836639404"}, {"distinct_range": 1040808421657982500, "num_eq": 3000000000, "num_range": 4910029256956964388, "upper_bound": "1.4120553731918335"}, {"distinct_range": 2265478885.497625, "num_eq": 20000000, "num_range": 30000000000, "upper_bound": "1.4886128902435303"}, {"distinct_range": 7000000000, "num_eq": 800, "num_range": 7000000000, "upper_bound": "1.648205041885376"}, {"distinct_range": 3932.1270828365996, "num_eq": 50, "num_range": 10000, "upper_bound": "1.869255542755127"}, {"distinct_range": 24279842.22388363, "num_eq": 60000000000, "num_range": 100000000, "upper_bound": "2.006828784942627"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 5974473588768956450}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2585485231791814026, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5974473588768956450}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5926800075596968326, "histo_buckets": [{"distinct_range": 0, "num_eq": 2879983223393060696, "num_range": 0, "upper_bound": "2055878562"}, {"distinct_range": 341495925.76652205, "num_eq": 0, "num_range": 500000000, "upper_bound": "3131445787"}, {"distinct_range": 0, "num_eq": 4518908702944129301, "num_range": 0, "upper_bound": "4065080522"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 2759227637232033995, "row_count": 5974473588768956450}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2260285842627552057, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5974473588768956450}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4359380551397356752, "histo_col_type": "", "name": "__auto__", "null_count": 900895932329098211, "row_count": 5974473588768956450}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col5_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3274759156395308921, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5425931237741300064}, {"columns": ["col5_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3606446561052015353, "histo_col_type": "", "name": "__auto__", "null_count": 5282510672959609187, "row_count": 5425931237741300064}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3301124027604486610, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5425931237741300064}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2291189774059681731, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5425931237741300064}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1334114000554184673, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5425931237741300064}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3791640195246427074, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5425931237741300064}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2818380611361039969, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5425931237741300064}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4517924749060711526, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5425931237741300064}, {"columns": ["col5_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1008647417432885766, "histo_col_type": "", "name": "__auto__", "null_count": 5039564841453445470, "row_count": 5425931237741300064}, {"columns": ["col5_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4286369652847957724, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5425931237741300064}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4991767230089277330, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5425931237741300064}, {"columns": ["col5_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2094870055352427560, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5425931237741300064}, {"columns": ["col5_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5301472814708573920, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5425931237741300064}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3374312674075364644, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5425931237741300064}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3055274245024226876, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5425931237741300064}, {"columns": ["col5_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2092885810467765243, "histo_col_type": "", "name": "__auto__", "null_count": 714859958370193211, "row_count": 5425931237741300064}, {"columns": ["col5_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5046229336250430578, "histo_col_type": "", "name": "__auto__", "null_count": 4295535448023476879, "row_count": 5425931237741300064}, {"columns": ["col5_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 456130709577354489, "histo_col_type": "", "name": "__auto__", "null_count": 156161571085040855, "row_count": 5425931237741300064}, {"columns": ["col5_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5421096866521931730, "histo_col_type": "", "name": "__auto__", "null_count": 2384849752580652268, "row_count": 5425931237741300064}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4616015281764439107, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "-1.2988814115524292"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 5425931237741300064}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1293465175093323254, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5425931237741300064}, {"columns": ["col5_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4479394141065266580, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5425931237741300064}, {"columns": ["col5_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2812717040501814252, "histo_col_type": "", "name": "__auto__", "null_count": 3319185221175136199, "row_count": 5425931237741300064}, {"columns": ["col5_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1525548466330077267, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5425931237741300064}, {"columns": ["col5_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1726153846688003691, "histo_col_type": "", "name": "__auto__", "null_count": 2246131669826253359, "row_count": 5425931237741300064}]':::JSONB;
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_2, col1_7) REFERENCES table4 (col1_6, col1_7);
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_11) REFERENCES table2 (col2_2) ON DELETE RESTRICT;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('wvtw');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('lakb', 'e', 'h', 'vbtt', 'huxtgu');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 2275438632260850207, "num_range": 0, "upper_bound": "#>\\u0013Q*\\u0002.Bc"}, {"distinct_range": 28357.631645018784, "num_eq": 4582337305097261689, "num_range": 100000, "upper_bound": ",0\\u0001\\u001f\\u001e\\\\9\\""}, {"distinct_range": 973397035105540200, "num_eq": 235244881072157160, "num_range": 7618004816910417610, "upper_bound": "7^OmC\\u001d\\u0010N>"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 800000000, "upper_bound": ";\\u001c"}, {"distinct_range": 4254246833772306400, "num_eq": 7425112431046431368, "num_range": 8111962035432255147, "upper_bound": "hsT\\u001a\\t^9b5"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "1301337260"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "0.6476607918739319"}, {"distinct_range": 0, "num_eq": 1482815444039665630, "num_range": 7582961552411390833, "upper_bound": "0.7155643701553345"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22, "histo_col_type": "", "name": "__auto__", "null_count": 10, "row_count": 50}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "-77 years -6 mons -998 days -11:45:22.523906"}, {"distinct_range": 800000000, "num_eq": 1658064114841846540, "num_range": 800000000, "upper_bound": "-70 years -7 mons -815 days -20:40:26.508139"}, {"distinct_range": 3720875570.318071, "num_eq": 50, "num_range": 4000000000, "upper_bound": "-60 years -11 mons -866 days -16:16:23.285601"}, {"distinct_range": 0, "num_eq": 6012075648616778424, "num_range": 0, "upper_bound": "-17 years -10 mons -792 days -08:07:44.733028"}, {"distinct_range": 400, "num_eq": 0, "num_range": 400, "upper_bound": "-14 years -475 days -08:22:44.455291"}, {"distinct_range": 9122072562285756000, "num_eq": 2340336767497104584, "num_range": 9122072562285755926, "upper_bound": "00:00:00"}, {"distinct_range": 455118418869211840, "num_eq": 156024468120108410, "num_range": 877487169222693050, "upper_bound": "3 years 4 mons 449 days 11:46:35.816149"}, {"distinct_range": 1113134747440875400, "num_eq": 3967231586732307190, "num_range": 1113134747440875377, "upper_bound": "4 years 8 mons 960 days 19:10:56.756558"}, {"distinct_range": 100000000000, "num_eq": 0, "num_range": 100000000000, "upper_bound": "77 years 8 mons 94 days 04:29:58.391404"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11, "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2979766, "histo_col_type": "", "name": "__auto__", "null_count": 516234, "row_count": 4000000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1301478, "histo_buckets": [{"distinct_range": 0, "num_eq": 8938627216587297331, "num_range": 0, "upper_bound": "-Inf"}, {"distinct_range": 1948065860565280500, "num_eq": 70, "num_range": 1948065860565280540, "upper_bound": "-1.6283267736434937"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 9000000000, "upper_bound": "-0.7607542276382446"}, {"distinct_range": 0, "num_eq": 3789037848052160205, "num_range": 2642481879261483877, "upper_bound": "-0.6446517705917358"}, {"distinct_range": 0, "num_eq": 7034746787742484279, "num_range": 700000000, "upper_bound": "-0.5448095798492432"}, {"distinct_range": 0, "num_eq": 3990851919794780194, "num_range": 0, "upper_bound": "-0.257279634475708"}, {"distinct_range": 530840652153993600, "num_eq": 4520862382343810051, "num_range": 6597058987681432575, "upper_bound": "3.4028234663852886e+38"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1994172, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 977555, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 589121, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3570600, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1709790, "histo_col_type": "", "name": "__auto__", "null_count": 1094956, "row_count": 4000000}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2304221, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1252792, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3253811, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 83077, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3333600, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1117875, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3510348, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "0.9805267199015952"}, {"distinct_range": 100000, "num_eq": 2603130575923630317, "num_range": 100000, "upper_bound": "+Inf"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3381955, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 630239, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1312125, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1676187, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29744825402, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10299454242, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20263398326, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "\\u000f~\\r"}, {"distinct_range": 6030359954121064000, "num_eq": 6434848238430982564, "num_range": 7602027182173856914, "upper_bound": ":dQ=9n"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30520975301, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34542950926, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11159190640, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "492601740"}, {"distinct_range": 0, "num_eq": 50, "num_range": 30000000, "upper_bound": "571648761"}, {"distinct_range": 0, "num_eq": 7010693005149035247, "num_range": 200, "upper_bound": "667984422"}, {"distinct_range": 0, "num_eq": 20000000, "num_range": 700000000, "upper_bound": "1132545624"}, {"distinct_range": 4318782914749694500, "num_eq": 40, "num_range": 6406936767909957818, "upper_bound": "1861606145"}, {"distinct_range": 3607973560381179400, "num_eq": 10000000, "num_range": 3607973560381179323, "upper_bound": "3088416373"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 2708569284027373907, "upper_bound": "3345323838"}, {"distinct_range": 56495.531262293516, "num_eq": 8000000000, "num_range": 60000, "upper_bound": "4242495201"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 30354103887, "row_count": 50000000000}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22337210936, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 49945753979, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col4_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24182271786, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 500000, "num_eq": 70000000, "num_range": 500000, "upper_bound": "1199221332"}, {"distinct_range": 6000, "num_eq": 0, "num_range": 6000, "upper_bound": "1659007242"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 8000000, "upper_bound": "2145070763"}, {"distinct_range": 304910869167923300, "num_eq": 1000000, "num_range": 321942687751745957, "upper_bound": "3635067452"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 40885766916, "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 43541370426, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11662349777, "histo_col_type": "", "name": "__auto__", "null_count": 15929871959, "row_count": 50000000000}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35174799711, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44958329794, "histo_buckets": [{"distinct_range": 0, "num_eq": 4101621442016477528, "num_range": 0, "upper_bound": "\\u0008"}, {"distinct_range": 65362673911161250, "num_eq": 4895846029566951177, "num_range": 2361790641439507724, "upper_bound": "\\u001b"}, {"distinct_range": 0, "num_eq": 6120010809234320377, "num_range": 0, "upper_bound": ","}, {"distinct_range": 80, "num_eq": 0, "num_range": 80, "upper_bound": "S"}, {"distinct_range": 50, "num_eq": 8000, "num_range": 50, "upper_bound": "X"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 3122372641755742233, "upper_bound": "["}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19336128539, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11116452788, "histo_col_type": "", "name": "__auto__", "null_count": 28350966601, "row_count": 50000000000}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16199752734, "histo_col_type": "", "name": "__auto__", "null_count": 17334385470, "row_count": 50000000000}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38178800501, "histo_buckets": [{"distinct_range": 0, "num_eq": 4845508726134444221, "num_range": 0, "upper_bound": "-2.253985643386841"}, {"distinct_range": 4238443685201346600, "num_eq": 3365918225706200385, "num_range": 4238443685201346314, "upper_bound": "-1.0566800832748413"}, {"distinct_range": 0, "num_eq": 900000, "num_range": 90000000000, "upper_bound": "-0.603903591632843"}, {"distinct_range": 1406600821745517800, "num_eq": 900, "num_range": 7287699173957648908, "upper_bound": "-0.33364933729171753"}, {"distinct_range": 40000, "num_eq": 6000000000, "num_range": 40000, "upper_bound": "0.0"}, {"distinct_range": 2940783912714742300, "num_eq": 100, "num_range": 2940783912714742124, "upper_bound": "0.3208156228065491"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col4_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14519926086, "histo_col_type": "", "name": "__auto__", "null_count": 30980757251, "row_count": 50000000000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9333170303, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000000, "num_range": 0, "upper_bound": "-0.5055456161499023"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 50000000000}]';
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col5_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11, "histo_col_type": "", "name": "__auto__", "null_count": 16, "row_count": 90}, {"columns": ["col5_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_col_type": "", "name": "__auto__", "null_count": 41, "row_count": 90}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 87, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 78, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 52, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col5_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 85, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col5_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27, "histo_col_type": "", "name": "__auto__", "null_count": 46, "row_count": 90}, {"columns": ["col5_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col5_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col5_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 63, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col5_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 59, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col5_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38, "histo_col_type": "", "name": "__auto__", "null_count": 62, "row_count": 90}, {"columns": ["col5_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 70, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 60, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 86, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 66, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 47, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col5_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 71, "histo_col_type": "", "name": "__auto__", "null_count": 89, "row_count": 90}, {"columns": ["col5_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3, "histo_col_type": "", "name": "__auto__", "null_count": 4, "row_count": 90}, {"columns": ["col5_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 60, "histo_col_type": "", "name": "__auto__", "null_count": 71, "row_count": 90}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22, "histo_buckets": [{"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "-1.2123686075210571"}, {"distinct_range": 0, "num_eq": 7372382868107395493, "num_range": 2000000000, "upper_bound": "-1.0556572675704956"}, {"distinct_range": 0, "num_eq": 3579595115980031517, "num_range": 700000, "upper_bound": "-0.527361273765564"}, {"distinct_range": 40883581641285790, "num_eq": 1000000, "num_range": 502805710838661334, "upper_bound": "-0.36311259865760803"}, {"distinct_range": 10000000000, "num_eq": 40000000000, "num_range": 10000000000, "upper_bound": "0.2792329788208008"}, {"distinct_range": 1944007654732068600, "num_eq": 3721523819626151933, "num_range": 1944007654732068537, "upper_bound": "0.7776670455932617"}, {"distinct_range": 4000, "num_eq": 2000000000, "num_range": 4000, "upper_bound": "1.1724185943603516"}, {"distinct_range": 6242861186654001000, "num_eq": 5331197748958070648, "num_range": 7896562083508964682, "upper_bound": "1.329123616218567"}, {"distinct_range": 2223039948554335700, "num_eq": 100000000, "num_range": 2223039948554335848, "upper_bound": "1.5785433053970337"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90}, {"columns": ["col5_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 73, "histo_col_type": "", "name": "__auto__", "null_count": 17, "row_count": 90}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 204397, "histo_buckets": [{"distinct_range": 0, "num_eq": 8314361627146340235, "num_range": 0, "upper_bound": ">\\u0006\'?yK"}, {"distinct_range": 0, "num_eq": 0, "num_range": 2332122136351187548, "upper_bound": "I.r)s"}, {"distinct_range": 8179425103456025000, "num_eq": 30000, "num_range": 8179425103456024995, "upper_bound": "X+\\u000egQc"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 60000, "upper_bound": "pwibWf"}, {"distinct_range": 4029865709622605300, "num_eq": 500000000, "num_range": 4029865709622605285, "upper_bound": "}psr2\\u0014"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6892656, "histo_col_type": "", "name": "__auto__", "null_count": 230426, "row_count": 8000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2577513, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5381456, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1443993, "histo_col_type": "", "name": "__auto__", "null_count": 3729015, "row_count": 8000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4275464, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1828233, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "d

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2688410&tab=buildLog) on [master@3c223f5f5162103110a790743b687ef2bf952489](https://github.com/cockroachdb/cockroach/commits/3c223f5f5162103110a790743b687ef2bf952489):

```
        cockroach2_1  | webui:               http://cockroach2:8080
        cockroach2_1  | sql:                 postgresql://root@cockroach2:26257?sslmode=disable
        cockroach2_1  | RPC client flags:    /cockroach/cockroach <client cmd> --host=cockroach2:26257 --insecure
        cockroach2_1  | logs:                /cockroach-data/logs
        cockroach2_1  | temp dir:            /cockroach-data/cockroach-temp147169251
        cockroach2_1  | external I/O path:   /cockroach-data/extern
        cockroach2_1  | store[0]:            path=/cockroach-data
        cockroach2_1  | storage engine:      pebble
        cockroach2_1  | status:              initialized new cluster
        cockroach2_1  | clusterID:           078ae173-1b72-4a88-94a8-5a0e1af408dd
        cockroach2_1  | nodeID:              1
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2021-02-18 06:14:16.984 UTC [48] LOG:  received fast shutdown request
        postgres_1    | .2021-02-18 06:14:16.987 UTC [48] LOG:  aborting any active transactions
        postgres_1    | 2021-02-18 06:14:16.988 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2021-02-18 06:14:16.997 UTC [50] LOG:  shutting down
        postgres_1    | 2021-02-18 06:14:17.107 UTC [48] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-02-18 06:14:17.197 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-02-18 06:14:17.197 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-02-18 06:14:17.202 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-02-18 06:14:17.217 UTC [85] LOG:  database system was shut down at 2021-02-18 06:14:17 UTC
        postgres_1    | 2021-02-18 06:14:17.223 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (602.14s)
        test_1        |     --- FAIL: TestCompare/postgres (1.46s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.2:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2693236&tab=buildLog) on [master@83e70ce84b740e27e721c3b73c38a4b8b515094a](https://github.com/cockroachdb/cockroach/commits/83e70ce84b740e27e721c3b73c38a4b8b515094a):

```
        test_1        | --- FAIL: TestCompare (3.19s)
        test_1        |     --- FAIL: TestCompare/postgres (2.06s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (1.13s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 NAME NOT NULL, col1_1 REGTYPE NOT NULL, col1_2 OID NOT NULL, col1_3 OID NOT NULL, col1_4 CHAR NULL, col1_5 INTERVAL NOT NULL, col1_6 NAME NOT NULL, col1_7 TIMETZ NOT NULL, col1_8 BIT(6)[] NULL, col1_9 NAME NOT NULL, col1_10 BIT(25) NULL, col1_11 CHAR NOT NULL, col1_12 FLOAT4 NOT NULL, col1_13 CHAR, PRIMARY KEY (col1_11 ASC, col1_12 DESC, col1_9 DESC, col1_0, col1_3 DESC, col1_1 ASC, col1_5 ASC, col1_7 ASC), INDEX (col1_11, col1_5 ASC, col1_4 DESC, col1_13 DESC, col1_3 DESC, col1_9, col1_6) STORING (col1_2) WHERE (table1.col1_0 <= e'\x00':::STRING) AND (table1.col1_6 != 'X':::STRING), INDEX (col1_3) WHERE (((table1.col1_11 = e'\U00002603':::STRING) AND (table1.col1_0 >= e'\'':::STRING)) AND (table1.col1_13 = e'\x00':::STRING)) AND (table1.col1_12 < 0.0:::FLOAT8), INDEX (col1_4 DESC, col1_9 DESC, col1_12 ASC, col1_3, col1_1 ASC, col1_2, col1_6 DESC, col1_10 ASC, col1_5 DESC) STORING (col1_8, col1_13) WHERE (((((((table1.col1_7 <= '00:00:00+15:59:00':::TIMETZ) OR (table1.col1_13 < e'\x00':::STRING)) AND (table1.col1_4 = 'X':::STRING)) OR (table1.col1_12 != 'NaN':::FLOAT8)) OR (table1.col1_0 = '':::STRING)) OR (table1.col1_11 < e'\x00':::STRING)) OR (table1.col1_9 != 'X':::STRING)) AND (table1.col1_6 <= e'\x00':::STRING), INDEX (col1_3 DESC, col1_4 ASC, col1_13 DESC, col1_9 DESC, col1_2 DESC) WHERE (((((((table1.col1_0 > e'\'':::STRING) OR (table1.col1_9 = '':::STRING)) OR (table1.col1_6 < e'\'':::STRING)) AND (table1.col1_7 != '24:00:00-15:59:00':::TIMETZ)) OR (table1.col1_13 <= '"':::STRING)) AND (table1.col1_4 > '"':::STRING)) OR (table1.col1_12 < '+Inf':::FLOAT8)) OR (table1.col1_11 = 'X':::STRING), UNIQUE (col1_0, col1_12, col1_7, col1_4 ASC) STORING (col1_2, col1_6, col1_8, col1_10, col1_13) WHERE (((((((table1.col1_11 != e'\'':::STRING) AND (table1.col1_12 = 0.0:::FLOAT8)) OR (table1.col1_7 >= '00:00:00+15:59:00':::TIMETZ)) OR (table1.col1_6 <= e'\x00':::STRING)) AND (table1.col1_4 != '':::STRING)) AND (table1.col1_9 >= e'\U00002603':::STRING)) AND (table1.col1_13 >= '':::STRING)) AND (table1.col1_0 != '"':::STRING), UNIQUE (col1_7, col1_0, col1_1 DESC, col1_3, col1_2 ASC, col1_4, col1_6, col1_9) STORING (col1_8), FAMILY (col1_5, col1_9), FAMILY (col1_13, col1_7, col1_11, col1_0, col1_3), FAMILY (col1_12), FAMILY (col1_1, col1_10), FAMILY (col1_4), FAMILY (col1_6, col1_2), FAMILY (col1_8), UNIQUE (col1_12));
        test_1        |             CREATE TABLE table2 (col2_0 NAME NOT NULL, col2_1 TIMESTAMPTZ, col2_2 BIT(25) NOT NULL, col2_3 INET NULL, col2_4 REGNAMESPACE, col2_5 FLOAT8, col2_6 "char" NOT NULL, FAMILY (col2_4, col2_2), FAMILY (col2_1), FAMILY (col2_0), FAMILY (col2_6), FAMILY (col2_3, col2_5));
        test_1        |             CREATE TABLE table3 (col3_0 TIMESTAMP[], col3_1 BYTES NOT NULL, col3_2 OID NOT NULL, col3_3 UUID NOT NULL, col3_4 TIME NOT NULL, col3_5 TIMESTAMP NOT NULL, col3_6 INT4 NOT NULL, col3_7 CHAR NOT NULL, col3_8 TIME NOT NULL, col3_9 TIMESTAMP NULL, col3_10 REGCLASS NOT NULL, col3_11 INET NOT NULL, col3_12 TIMESTAMP NULL, col3_13 REGNAMESPACE NULL, col3_14 TIMETZ, col3_15 "char" NOT NULL, col3_16 REGCLASS NOT NULL, PRIMARY KEY (col3_6, col3_8 DESC, col3_11, col3_3 ASC, col3_1 ASC, col3_15 DESC, col3_5 DESC, col3_7 ASC, col3_16, col3_4 DESC, col3_10 DESC, col3_2), INDEX (col3_13, col3_15 ASC, col3_3, col3_1 ASC, col3_14, col3_16 ASC) STORING (col3_0, col3_9) WHERE (((((((table3.col3_15 <= e'\'') OR (table3.col3_1 < '\xe29883')) OR (table3.col3_5 != '3000-01-01 00:00:00')) OR (table3.col3_7 = e'\'')) AND (table3.col3_9 > '-2000-01-01 00:00:00')) OR (table3.col3_14 != '24:00:00-15:59:00')) AND (table3.col3_6 >= 0)) OR (table3.col3_8 != '24:00:00'), INDEX (col3_12 DESC, col3_7, col3_14 ASC, col3_3 ASC, col3_13, col3_5 ASC) STORING (col3_9) WHERE ((((table3.col3_1 < '\x58') OR (table3.col3_8 > '00:00:00')) OR (table3.col3_5 < '-4713-11-24 00:00:00')) OR (table3.col3_15 <= '"')) OR (table3.col3_4 = '24:00:00'), FAMILY (col3_4), FAMILY (col3_9), FAMILY (col3_13), FAMILY (col3_16), FAMILY (col3_7, col3_2, col3_12, col3_1), FAMILY (col3_3, col3_5, col3_8), FAMILY (col3_15), FAMILY (col3_6, col3_10), FAMILY (col3_14), FAMILY (col3_0, col3_11));
        test_1        |             CREATE TABLE table4 (col4_0 UUID NOT NULL, col4_1 TIMESTAMP NOT NULL, col4_2 FLOAT8[] NULL, col4_3 FLOAT8, col4_4 STRING NULL, col4_5 "char", col4_6 INET NOT NULL, col4_7 INT4, col4_8 DATE NOT NULL, col4_9 REGPROC, col4_10 REGCLASS, col4_11 BIT(9) NULL, col4_12 BIT(2) NOT NULL, col4_13 TIME, col4_14 DECIMAL NOT NULL, col4_15 REGCLASS, col4_16 NAME NOT NULL, col4_17 BIT(7) NOT NULL, PRIMARY KEY (col4_14 ASC, col4_16 DESC, col4_1 DESC, col4_6 DESC), INDEX (col4_15 DESC, col4_6, col4_9 ASC, col4_8 ASC, col4_17, col4_7 ASC, col4_0, col4_14 DESC, col4_4 ASC, col4_10, col4_1 DESC, col4_5 DESC, col4_16, col4_12, col4_11 ASC), INDEX (col4_12 DESC, col4_0 ASC, col4_1 DESC, col4_6 DESC, col4_9, col4_7 ASC, col4_14 ASC, col4_8, col4_10 ASC, col4_15, col4_11 DESC, col4_13 DESC, col4_3 ASC, col4_5 DESC, col4_4 DESC, col4_17 ASC, col4_16) WHERE ((((((((table4.col4_8 != 'infinity':::DATE) AND (table4.col4_4 != '':::STRING)) AND (table4.col4_5 != 'X':::STRING)) OR (table4.col4_7 > 1:::INT8)) AND (table4.col4_3 < 1.7976931348623157e+308:::FLOAT8)) OR (table4.col4_13 >= '24:00:00':::TIME)) OR (table4.col4_14 >= 'NaN':::DECIMAL)) AND (table4.col4_16 = e'\'':::STRING)) OR (table4.col4_1 <= '3000-01-01 00:00:00':::TIMESTAMP), INDEX (col4_3 ASC, col4_12 DESC, col4_11, col4_6, col4_5, col4_13, col4_4 ASC, col4_8 ASC) STORING (col4_7, col4_10, col4_17) WHERE ((((((((table4.col4_1 >= '3000-01-01 00:00:00':::TIMESTAMP) OR (table4.col4_4 > '':::STRING)) OR (table4.col4_7 <= 0:::INT8)) OR (table4.col4_5 <= e'\U00002603':::STRING)) AND (table4.col4_3 = (-1.0):::FLOAT8)) OR (table4.col4_8 >= 'infinity':::DATE)) OR (table4.col4_16 > e'\'':::STRING)) AND (table4.col4_13 != '24:00:00':::TIME)) OR (table4.col4_14 = 1:::DECIMAL), FAMILY (col4_0), FAMILY (col4_12), FAMILY (col4_6), FAMILY (col4_2, col4_7), FAMILY (col4_15, col4_8, col4_9), FAMILY (col4_11, col4_17), FAMILY (col4_3), FAMILY (col4_4, col4_13, col4_16, col4_1, col4_10), FAMILY (col4_14, col4_5), UNIQUE (col4_1) WHERE (((table4.col4_13 >= '24:00:00') OR (table4.col4_7 <= -1)) AND (table4.col4_5 >= '')) OR (table4.col4_3 != NaN));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 244820844964041733, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4103234287248356822}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2106192745811243876, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4103234287248356822}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1474545878949090674, "histo_buckets": [{"distinct_range": 0, "num_eq": 2335597924732106675, "num_range": 0, "upper_bound": "267504691"}, {"distinct_range": 0, "num_eq": 400000, "num_range": 1000000, "upper_bound": "314776158"}, {"distinct_range": 18878439608.922695, "num_eq": 100, "num_range": 80000000000, "upper_bound": "3265719321"}, {"distinct_range": 0, "num_eq": 6294052514075510703, "num_range": 2000000, "upper_bound": "3640759669"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 4103234287248356822}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2891128293543965651, "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 4103234287248356822}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1071986272369546119, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4103234287248356822}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2115883378054475135, "histo_col_type": "", "name": "__auto__", "null_count": 2120898239429334606, "row_count": 4103234287248356822}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3674401034189443941, "histo_buckets": [{"distinct_range": 0, "num_eq": 60, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 327623834606137800, "num_eq": 70000, "num_range": 7397216998344606179, "upper_bound": "R"}, {"distinct_range": 0, "num_eq": 20000000000, "num_range": 0, "upper_bound": "_"}, {"distinct_range": 0, "num_eq": 900000000, "num_range": 70000000000, "upper_bound": "l"}, {"distinct_range": 1455622995024254000, "num_eq": 1000, "num_range": 1455622995024253989, "upper_bound": "}"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 1683584751526967203, "row_count": 4103234287248356822}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2171804121731671441, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4103234287248356822}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 100884029409219564, "histo_col_type": "", "name": "__auto__", "null_count": 2719485133189999221, "row_count": 4103234287248356822}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1270000227817961324, "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 4103234287248356822}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3277965482112507200, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 4614314441687883610, "upper_bound": "7~\\t4"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 4103234287248356822}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2219933875333106120, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4103234287248356822}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 939504841786144970, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4103234287248356822}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2171106850940891879, "histo_col_type": "", "name": "__auto__", "null_count": 3932747848135399738, "row_count": 4103234287248356822}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3000, "histo_col_type": "", "name": "__auto__", "null_count": 4712, "row_count": 9000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3939, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 358, "histo_col_type": "", "name": "__auto__", "null_count": 7914, "row_count": 9000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4324, "histo_col_type": "", "name": "__auto__", "null_count": 295, "row_count": 9000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8264, "histo_col_type": "", "name": "__auto__", "null_count": 7910, "row_count": 9000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6605, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4287, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2169096037340783661, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3628209371721266766}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2138738823230086134, "histo_col_type": "", "name": "__auto__", "null_count": 2797359231037108971, "row_count": 3628209371721266766}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1574540659213370747, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3628209371721266766}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 377758733861035677, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3628209371721266766}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3489580087662567414, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3628209371721266766}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1478198590720573530, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3628209371721266766}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 419434062354778140, "histo_buckets": [{"distinct_range": 0, "num_eq": 5059881942560286379, "num_range": 0, "upper_bound": "-2147483648"}, {"distinct_range": 8919791785.142067, "num_eq": 5417831849386513891, "num_range": 9000000000, "upper_bound": "-2143415901"}, {"distinct_range": 70000, "num_eq": 60000000, "num_range": 70000, "upper_bound": "-1437188167"}, {"distinct_range": 10000000000, "num_eq": 40, "num_range": 10000000000, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 5000000000, "num_range": 8696360119251869394, "upper_bound": "1"}, {"distinct_range": 600000000, "num_eq": 30000000, "num_range": 600000000, "upper_bound": "3313524"}, {"distinct_range": 1590644216478797800, "num_eq": 60000000000, "num_range": 1590644216478797795, "upper_bound": "1713985124"}, {"distinct_range": 542579593338689300, "num_eq": 1694596673602782350, "num_range": 899616291609955739, "upper_bound": "1946870828"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 3628209371721266766}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1529474465226811228, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3628209371721266766}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1080102986770372943, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3628209371721266766}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2324106889833163881, "histo_buckets": [{"distinct_range": 0, "num_eq": 839028569511631273, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 4359626596747936300, "num_eq": 4587495356341070597, "num_range": 7691741844281624228, "upper_bound": "522250899"}, {"distinct_range": 0, "num_eq": 1272394406501618976, "num_range": 2605829895380388381, "upper_bound": "553269593"}, {"distinct_range": 0, "num_eq": 2074114715115444838, "num_range": 10, "upper_bound": "783096452"}, {"distinct_range": 3951406.4220533594, "num_eq": 2198178532023374398, "num_range": 10000000, "upper_bound": "861884841"}, {"distinct_range": 0, "num_eq": 500, "num_range": 3878853407160020870, "upper_bound": "884716868"}, {"distinct_range": 30000, "num_eq": 4813964125030279921, "num_range": 30000, "upper_bound": "1617254568"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 3425236268648298105, "row_count": 3628209371721266766}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 935104876647595224, "histo_col_type": "", "name": "__auto__", "null_count": 329066763000203198, "row_count": 3628209371721266766}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1333661311282802359, "histo_col_type": "", "name": "__auto__", "null_count": 562284457373376536, "row_count": 3628209371721266766}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1260192954053864991, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3628209371721266766}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2558829679823689515, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3628209371721266766}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3297161675828646129, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3628209371721266766}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1047342859106960974, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000000, "num_range": 0, "upper_bound": "-4713-11-24 00:00:00"}, {"distinct_range": 0.3628581955816427, "num_eq": 20, "num_range": 80, "upper_bound": "1972-06-12 09:48:27.000876"}, {"distinct_range": 17412941373.07829, "num_eq": 600, "num_range": 20000000000, "upper_bound": "1980-11-14 03:53:33.000535"}, {"distinct_range": 3114.8968821248786, "num_eq": 40000000000, "num_range": 40000, "upper_bound": "2004-03-23 01:32:52.000829"}, {"distinct_range": 135436092.26355004, "num_eq": 2475170069251291915, "num_range": 10000000000, "upper_bound": "2024-12-08 04:24:16.000934"}, {"distinct_range": 0, "num_eq": 3305908347768323614, "num_range": 2000, "upper_bound": "2028-10-17 00:53:08.000372"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 1469748422086965982, "row_count": 3628209371721266766}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2463787341181185012, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3628209371721266766}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1927953580974886926, "histo_col_type": "", "name": "__auto__", "null_count": 5225879627813391746, "row_count": 7046248761138430257}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2257585649237251624, "histo_col_type": "", "name": "__auto__", "null_count": 3278134147586746375, "row_count": 7046248761138430257}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 231329847564692675, "histo_col_type": "", "name": "__auto__", "null_count": 7017792343164082500, "row_count": 7046248761138430257}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6575595461293751842, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7046248761138430257}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4574237015898522540, "histo_col_type": "", "name": "__auto__", "null_count": 3126011858630368063, "row_count": 7046248761138430257}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1513112398723386997, "histo_col_type": "", "name": "__auto__", "null_count": 2066133926983553862, "row_count": 7046248761138430257}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5088517325240667058, "histo_buckets": [{"distinct_range": 0, "num_eq": 412072884326472532, "num_range": 0, "upper_bound": "-6.78837343869418398E+33"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 3487806787118311637, "upper_bound": "-3.462508719005496788E+33"}, {"distinct_range": 0, "num_eq": 50000, "num_range": 40000000, "upper_bound": "-6.328640483215972347E+31"}, {"distinct_range": 1098099806365445000, "num_eq": 8059607527868990267, "num_range": 7126036878085538905, "upper_bound": "-2.237651910267403071E+30"}, {"distinct_range": 0, "num_eq": 3307407648039809602, "num_range": 900000000, "upper_bound": "-8.659334335691056249E+25"}, {"distinct_range": 0, "num_eq": 39498960360314086, "num_range": 400, "upper_bound": "-1174716455.637425802"}, {"distinct_range": 2560186846682844000, "num_eq": 6364077954286746256, "num_range": 6343333847672067594, "upper_bound": "1423517317.787934285"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 2941021505698874644, "upper_bound": "41037904839145228.04"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 7046248761138430257}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1231619770329355030, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000000, "num_range": 0, "upper_bound": "1274855618"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 1362528837889408876, "row_count": 7046248761138430257}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5474431241320497078, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7046248761138430257}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5768217771136392738, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7046248761138430257}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3382831978067787762, "histo_col_type": "", "name": "__auto__", "null_count": 232185003194857377, "row_count": 7046248761138430257}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5479936661897880455, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7046248761138430257}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 408124760985302749, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7046248761138430257}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5891751207388168337, "histo_col_type": "", "name": "__auto__", "null_count": 5794365915619604370, "row_count": 7046248761138430257}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 602398170466617405, "histo_buckets": [{"distinct_range": 0, "num_eq": 6905146125936842259, "num_range": 0, "upper_bound": "00"}, {"distinct_range": 5978342.1592098195, "num_eq": 1131032848838111029, "num_range": 10000000, "upper_bound": "01"}, {"distinct_range": 824258532975382300, "num_eq": 9043512747171779222, "num_range": 824258532975382250, "upper_bound": "10"}, {"distinct_range": 0, "num_eq": 5962558907903946042, "num_range": 600000, "upper_bound": "11"}], "histo_col_type": "BIT(2)", "name": "__auto__", "null_count": 0, "row_count": 7046248761138430257}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 330713491992226243, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7046248761138430257}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2543958788390589274, "histo_buckets": [{"distinct_range": 0, "num_eq": 4352176347686846079, "num_range": 0, "upper_bound": "-0.28446674712218945"}, {"distinct_range": 1873883874217034000, "num_eq": 10000000, "num_range": 1873883874217034085, "upper_bound": "0.057683068849643526"}, {"distinct_range": 0, "num_eq": 7248313309889039192, "num_range": 0, "upper_bound": "0.08791218263459527"}, {"distinct_range": 157426234.8491741, "num_eq": 900, "num_range": 200000000, "upper_bound": "1.3575478978821842"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 125353607947139900, "row_count": 7046248761138430257}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 48967920692249309, "histo_col_type": "", "name": "__auto__", "null_count": 5294130138464869923, "row_count": 7046248761138430257}]':::JSONB;
        test_1        |             ALTER TABLE table4 ADD FOREIGN KEY (col4_3) REFERENCES table1 (col1_12);
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col3_5) REFERENCES table4 (col4_1) ON DELETE CASCADE;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('wn', 'k');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('bkhwxa');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('txw', 'tfxaaa');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('r', 'fjadz');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('aowebp', 'hgqev', 'sj', 'ozdtty');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 176257739213613289, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 287958949837542482}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 62251126097789258, "histo_buckets": [{"distinct_range": 0, "num_eq": 7142772098585336784, "num_range": 0, "upper_bound": "00:11:19.099395-00:54:00"}, {"distinct_range": 0, "num_eq": 0, "num_range": 1000000000, "upper_bound": "06:27:55.509395-01:04:00"}, {"distinct_range": 9180482390227472000, "num_eq": 500, "num_range": 9180482390227472080, "upper_bound": "06:57:17.203587-00:46:00"}, {"distinct_range": 0, "num_eq": 5474535043193238345, "num_range": 2251560053587185037, "upper_bound": "07:52:01.116468-05:50:00"}, {"distinct_range": 7918952358775839000, "num_eq": 3506142167595754124, "num_range": 7918952358775838220, "upper_bound": "22:02:38.986811+05:51:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 287958949837542482}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 186894455067218225, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 287958949837542482}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 271115984087034114, "histo_col_type": "", "name": "__auto__", "null_count": 49243151110181478, "row_count": 287958949837542482}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 232197714115230208, "histo_col_type": "", "name": "__auto__", "null_count": 281753201532838688, "row_count": 287958949837542482}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 169775563857986534, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 287958949837542482}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 183890794336596494, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000, "num_range": 0, "upper_bound": "+"}, {"distinct_range": 0, "num_eq": 772420957781355261, "num_range": 0, "upper_bound": "I\\u001c\\u0015A2u\\u0004b"}, {"distinct_range": 0, "num_eq": 178252752515480730, "num_range": 6000000000, "upper_bound": "_"}, {"distinct_range": 802995986776714800, "num_eq": 400000000, "num_range": 802995986776714765, "upper_bound": "r]E\\u0008l"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 287958949837542482}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12395948402806969, "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 287958949837542482}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 244269076755910740, "histo_buckets": [{"distinct_range": 0, "num_eq": 4953369079325191227, "num_range": 0, "upper_bound": "\\u0015"}, {"distinct_range": 9129605774139751000, "num_eq": 50000, "num_range": 9129605774139751146, "upper_bound": "\\""}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 6765280119402266110, "upper_bound": "A"}, {"distinct_range": 0, "num_eq": 6822972918009977899, "num_range": 0, "upper_bound": "J"}, {"distinct_range": 6862807962.308184, "num_eq": 4453448183003374609, "num_range": 10000000000, "upper_bound": "f"}, {"distinct_range": 6339243405939099000, "num_eq": 3733495618004146402, "num_range": 6339243405939098470, "upper_bound": "t"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 174521685417095370, "row_count": 287958949837542482}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 73056439905844259, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "E"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 287958949837542482}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 107303276792489916, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "-0.904147744178772"}, {"distinct_range": 0, "num_eq": 400000, "num_range": 600000000, "upper_bound": "-0.5218731760978699"}, {"distinct_range": 8133336770288416000, "num_eq": 2000, "num_range": 8133336770288415926, "upper_bound": "0.14903616905212402"}, {"distinct_range": 0, "num_eq": 6418122122711540881, "num_range": 10000000, "upper_bound": "0.9228636026382446"}, {"distinct_range": 30000000000, "num_eq": 4019267457835471339, "num_range": 30000000000, "upper_bound": "1.0"}, {"distinct_range": 0, "num_eq": 551478364757171902, "num_range": 0, "upper_bound": "1.2082444429397583"}, {"distinct_range": 3703578865151021000, "num_eq": 1028678245203395901, "num_range": 6395634440926987776, "upper_bound": "1.4219474792480469"}, {"distinct_range": 0, "num_eq": 600000, "num_range": 1000000000, "upper_bound": "2.7807223796844482"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 287958949837542482}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 229163277928537266, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 287958949837542482}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 165418989345879116, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 287958949837542482}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 72387503298971237, "histo_col_type": "", "name": "__auto__", "null_count": 74270859247306095, "row_count": 287958949837542482}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 465423454833707286, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7787759013123961604}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3432169803837731016, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7787759013123961604}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4759365827798523583, "histo_col_type": "", "name": "__auto__", "null_count": 4374950983295637562, "row_count": 7787759013123961604}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1810767732297900326, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7787759013123961604}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5981565580734519708, "histo_col_type": "", "name": "__auto__", "null_count": 2437827036361080997, "row_count": 7787759013123961604}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2591787612576391795, "histo_col_type": "", "name": "__auto__", "null_count": 5827278342201285631, "row_count": 7787759013123961604}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3258456461178516856, "histo_col_type": "", "name": "__auto__", "null_count": 7561202735714682458, "row_count": 7787759013123961604}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 822117798, "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 981232043, "histo_col_type": "", "name": "__auto__", "null_count": 768329502, "row_count": 1000000000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 84805180, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 518091907, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 70, "num_eq": 0, "num_range": 70, "upper_bound": "367694104"}, {"distinct_range": 30, "num_eq": 8838343041025748164, "num_range": 30, "upper_bound": "438900957"}, {"distinct_range": 5000, "num_eq": 4269316567695533604, "num_range": 5000, "upper_bound": "588076564"}, {"distinct_range": 2249007787443466800, "num_eq": 3945269019187083215, "num_range": 2249007787443466865, "upper_bound": "687167757"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 0, "upper_bound": "891561411"}, {"distinct_range": 161070338922551300, "num_eq": 7116910221164778914, "num_range": 1447947280168437819, "upper_bound": "4008501710"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 519742959, "row_count": 1000000000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 184200754, "histo_col_type": "", "name": "__auto__", "null_count": 777793895, "row_count": 1000000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 265825259, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 431992189, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 696196495, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 232359042, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 670736082, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 846844276, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 769233584, "histo_buckets": [{"distinct_range": 0, "num_eq": 4577578676877421166, "num_range": 0, "upper_bound": "-4713-11-24 00:00:00"}, {"distinct_range": 0, "num_eq": 41178770710579264, "num_range": 0, "upper_bound": "1971-05-27 01:41:24.000947"}, {"distinct_range": 50.54524328680504, "num_eq": 7128616186975195029, "num_range": 100, "upper_bound": "1976-04-07 12:10:19.000879"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 0, "upper_bound": "1978-03-28 06:08:21.000156"}, {"distinct_range": 759225.6918745764, "num_eq": 4194495083196638370, "num_range": 1000000, "upper_bound": "1982-04-19 12:41:37.000896"}, {"distinct_range": 0, "num_eq": 3202501501766090210, "num_range": 100000000, "upper_bound": "1985-09-01 21:21:12.000851"}, {"distinct_range": 1394911.8838355609, "num_eq": 10000, "num_range": 3000000, "upper_bound": "1988-07-05 17:53:39.000805"}, {"distinct_range": 789.2128698789371, "num_eq": 5743530021636541432, "num_range": 1000, "upper_bound": "2000-03-09 00:04:25.000585"}, {"distinct_range": 166.02422062236226, "num_eq": 90000, "num_range": 1000, "upper_bound": "2001-07-30 06:10:36.00068"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 201308141, "row_count": 1000000000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 819926585, "histo_col_type": "", "name": "__auto__", "null_count": 79333784, "row_count": 1000000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 181537296, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 879841899, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 373892359, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 813009738, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000000000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 8331939218988342603, "num_range": 0, "upper_bound": "259211248"}, {"distinct_range": 6934643721511772000, "num_eq": 2750549808707082795, "num_range": 6934643721511772393, "upper_bound": "511709254"}, {"distinct_range": 1725212911790752800, "num_eq": 50000, "num_range": 1725212911790752724, "upper_bound": "773026536"}, {"distinct_range": 265200.97932123224, "num_eq": 1478810187915101968, "num_range": 4000000, "upper_bound": "1722642846"}, {"distinct_range": 5492.0773504709, "num_eq": 200000, "num_range": 9000, "upper_bound": "2460169377"}, {"distinct_range": 82520716876949900, "num_eq": 1000, "num_range": 227699249900070368, "upper_bound": "2501249447"}, {"distinct_range": 4044570852488192500, "num_eq": 8506609365703994113, "num_range": 7875658818046767930, "upper_bound": "3970427868"}, {"distinct_range": 80000, "num_eq": 5614465131296409207, "num_range": 80000, "upper_bound": "4170848622"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "-1.1040839167341083"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 1024852851612481569, "upper_bound": "0.7830377910337211"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 800, "num_range": 0, "upper_bound": "1979-03-08 20:36:06.000771"}, {"distinct_range": 0, "num_eq": 5182440140700907027, "num_range": 7421476688221833716, "upper_bound": "1993-05-29 19:31:01.000606"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 7348479376210985816, "upper_bound": "2002-12-01 02:18:56.00015"}, {"distinct_range": 0, "num_eq": 7801476680982472371, "num_range": 10000000, "upper_bound": "2003-10-28 05:37:26.000609"}, {"distinct_range": 0, "num_eq": 80, "num_range": 2278157773401192853, "upper_bound": "2017-07-26 11:04:53.000377"}, {"distinct_range": 0, "num_eq": 500000000, "num_range": 70, "upper_bound": "2019-03-15 13:34:22.000239"}, {"distinct_range": 5126346156190185000, "num_eq": 7000, "num_range": 5126346156190185521, "upper_bound": "2030-06-23 09:05:11.000903"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "00"}, {"distinct_range": 434492416.11106604, "num_eq": 7663649972704587670, "num_range": 500000000, "upper_bound": "01"}], "histo_col_type": "BIT(2)", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 2661237074656601222, "num_range": 0, "upper_bound": "-2.852894354977152058E+33"}, {"distinct_range": 0, "num_eq": 1305006816591312409, "num_range": 0, "upper_bound": "-5438774.716965319"}, {"distinct_range": 100, "num_eq": 5470901144606126617, "num_range": 100, "upper_bound": "-1"}, {"distinct_range": 0, "num_eq": 60000, "num_range": 4182686713384386099, "upper_bound": "-0.772562794250365191"}, {"distinct_range": 0, "num_eq": 600, "num_range": 7130504022619728465, "upper_bound": "6.781013739402671048"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 50000000, "upper_bound": "4.59538983053323033E+29"}, {"distinct_range": 10767913.412457181, "num_eq": 8414404856326559455, "num_range": 50000000, "upper_bound": "Infinity"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1837393304649846236, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4987804523967768522}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2086682548512436873, "histo_col_type": "", "name": "__auto__", "null_count": 69764455352141089, "row_count": 4987804523967768522}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2911324973485773330, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4987804523967768522}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3668343127637254514, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4987804523967768522}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2317675263750185946, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 0, "num_eq": 2000000000, "num_range": 1247740982910740003, "upper_bound": "04:33:19.647775+12:19:00"}, {"distinct_range": 2954746779291029000, "num_eq": 0, "num_range": 4992428876153750163, "upper_bound": "19:49:10.808887+11:57:00"}, {"distinct_range": 1000000, "num_eq": 80000, "num_range": 1000000, "upper_bound": "22:53:53.431734+13:24:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 4987804523967768522}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1501314234811907954, "histo_col_type": "", "name": "__auto__", "null_count": 1204126154817625957, "row_count": 4987804523967768522}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3428304473518185724, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "\\u0002"}, {"distinct_range": 554.9136834050788, "num_eq": 90000000000, "num_range": 700, "upper_bound": "\\u0016"}, {"distinct_range": 0, "num_eq": 40, "num_range": 1000000, "upper_bound": "B"}, {"distinct_range": 41056.60475139763, "num_eq": 700000, "num_range": 50000, "upper_bound": "\U00002603"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 4987804523967768522}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 88647773839453446, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000, "num_range": 0, "upper_bound": "-1.6271196603775024"}, {"distinct_range": 6608847346932025000, "num_eq": 30000000, "num_range": 6608847346932025572, "upper_bound": "-0.9125024676322937"}, {"distinct_range": 4348499338579409000, "num_eq": 4472853302115757763, "num_range": 4348499338579408867, "upper_bound": "-0.5698105692863464"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 10000000, "upper_bound": "-0.02997708134353161"}, {"distinct_range": 319484.5357145818, "num_eq": 4000, "num_range": 1000000, "upper_bound": "0.6353017091751099"}, {"distinct_range": 8603081514781134000, "num_eq": 0, "num_range": 8603081514781133794, "upper_bound": "0.8656134009361267"}, {"distinct_range": 24483402357.915783, "num_eq": 8401731334468079055, "num_range": 40000000000, "upper_bound": "1.6268389225006104"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 20, "upper_bound": "+Inf"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 4987804523967768522}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2665228925552808438, "histo_buckets": [{"distinct_range": 0, "num_eq": 4717202974876388071, "num_range": 0, "upper_bound": "\\u0010"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 40000000000, "upper_bound": "/"}, {"distinct_range": 59.26777168026035, "num_eq": 10000000000, "num_range": 60000, "upper_bound": "4"}, {"distinct_range": 900000000, "num_eq": 0, "num_range": 900000000, "upper_bound": ">"}, {"distinct_range": 4570607490691940400, "num_eq": 4143828234462311572, "num_range": 5092124621611536657, "upper_bound": "Z"}, {"distinct_range": 109314.69259335559, "num_eq": 100000000, "num_range": 500000, "upper_bound": "{"}, {"distinct_range": 800000000, "num_eq": 4737945471571783392, "num_range": 800000000, "upper_bound": "\x7f"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 4743503384367331323, "row_count": 4987804523967768522}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4949955470427114087, "histo_col_type": "", "name": "__auto__", "null_count": 2799442398491883119, "row_count": 4987804523967768522}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2948482344045134515, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 2836076238752151905, "num_range": 8921315938276167557, "upper_bound": "\\u00027;"}, {"distinct_range": 400000000, "num_eq": 4076372915378690609, "num_range": 400000000, "upper_bound": "\\nxJp"}, {"distinct_range": 0, "num_eq": 1239358392773773258, "num_range": 3000, "upper_bound": "\\u0016>l\\u0007RM"}, {"distinct_range": 6282298298635477000, "num_eq": 40000000, "num_range": 7465242282033123619, "upper_bound": "\\""}, {"distinct_range": 0, "num_eq": 5550854347083719083, "num_range": 6213426677845685915, "upper_bound": "&(8E\\ns\\u00016"}, {"distinct_range": 4000, "num_eq": 6563182366328555087, "num_range": 4000, "upper_bound": "D\\u0002"}, {"distinct_range": 1497313687.3363044, "num_eq": 8607908961098359757, "num_range": 3000000000, "upper_bound": "fZ#i#H|HT"}, {"distinct_range": 6411.784373138607, "num_eq": 600000000, "num_range": 30000, "upper_bound": "l=P\\u0002(\\u0006\\u00176t"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 0, "row_count": 4987804523967768522}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2708688131480503694, "histo_buckets": [{"distinct_range": 0, "num_eq": 4782891371052211544, "num_range": 0, "upper_bound": "152992502"}, {"distinct_range": 1612432655774454800, "num_eq": 571372442570587383, "num_range": 1612432655774454818, "upper_bound": "1997659381"}, {"distinct_range": 0, "num_eq": 2521503870543553418, "num_range": 8278707345585236004, "upper_bound": "2603015434"}, {"distinct_range": 5790942890334774000, "num_eq": 900, "num_range": 5790942890334774379, "upper_bound": "4015005372"}, {"distinct_range": 3431251150533771000, "num_eq": 70000000, "num_range": 3431251150533770836, "upper_bound": "4190870312"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 4987804523967768522}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3660894436028795324, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4987804523967768522}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 865530015939628375, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4987804523967768522}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2, "histo_col_type": "", "name": "__auto__", "null_count": 91, "row_count": 100}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 47, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42, "histo_col_type": "", "name": "__auto__", "null_count": 68, "row_count": 100}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 72, "histo_col_type": "", "name": "__auto__", "null_count": 37, "row_count": 100}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 91, "histo_col_type": "", "name": "__auto__", "null_count": 19, "row_count": 100}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 43, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2064163418250350787, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3659684344736147375}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1114039254545305570, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3659684344736147375}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1160058188135275007, "histo_col_type": "", "name": "__auto__", "null_count": 2505234979162147531, "row_count": 3659684344736147375}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1910825602788225693, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3659684344736147375}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1394349134025258340, "histo_buckets": [{"distinct_range": 0, "num_eq": 80, "num_range": 0, "upper_bound": "0001-01-01 00:00:00"}, {"distinct_range": 0, "num_eq": 8244306695840056659, "num_range": 6590096540064328733, "upper_bound": "1977-08-16 01:21:58.000541"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 30000000, "upper_bound": "1980-03-03 04:40:37.000011"}, {"distinct_range": 688396550697528700, "num_eq": 200, "num_range": 2423436190844835391, "upper_bound": "1980-08-05 17:22:04.000195"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 0, "upper_bound": "1999-11-02 06:21:18.00039"}, {"distinct_range": 0, "num_eq": 402227107144291956, "num_range": 208382048769812742, "upper_bound": "2000-02-16 18:12:04.000915"}, {"distinct_range": 3998057668228678000, "num_eq": 50, "num_range": 8251235792446362777, "upper_bound": "2003-10-11 04:01:22.000665"}, {"distinct_range": 59583.48231313812, "num_eq": 7082591489144325603, "num_range": 1000000, "upper_bound": "2010-09-01 09:04:43.000623"}, {"distinct_range": 100, "num_eq": 1000000000, "num_range": 100, "upper_bound": "2019-01-30 10:07:50.000007"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 1614170547777516181, "row_count": 3659684344736147375}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16630528831620186, "histo_col_type": "", "name": "__auto__", "null_count": 2979057967756954089, "row_count": 3659684344736147375}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 366021146501008261, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3659684344736147375}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2235209230828623519, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3659684344736147375}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3601707502424587540, "histo_col_type": "", "name": "__auto__", "null_count": 2778322095152478563, "row_count": 3659684344736147375}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1814809255359153560, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3659684344736147375}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1202932537051886306, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3659684344736147375}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2679450409885769697, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3659684344736147375}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 103420173830611129, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3659684344736147375}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 819761634405779201, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3659684344736147375}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 907012109025915839, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "-1619639508"}, {"distinct_range": 0, "num_eq": 2658723994365375634, "num_range": 100000000, "upper_bound": "-1470937890"}, {"distinct_range": 0, "num_eq": 4227096753058429307, "num_range": 0, "upper_bound": "-1207068668"}, {"distinct_range": 900000000, "num_eq": 3555755585570593435, "num_range": 900000000, "upper_bound": "-1"}, {"distinct_range": 0, "num_eq": 6962960264059688185, "num_range": 4037330260464692669, "upper_bound": "96642774"}, {"distinct_range": 300, "num_eq": 60000, "num_range": 300, "upper_bound": "334178149"}, {"distinct_range": 2683498926829080600, "num_eq": 5183866988397953935, "num_range": 3252321822871808092, "upper_bound": "1890061446"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 3659684344736147375}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1026042510300316266, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3659684344736147375}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3357176682457679530, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": "785105924"}, {"distinct_range": 0, "num_eq": 3377846471734109187, "num_range": 60000, "upper_bound": "1514001243"}, {"distinct_range": 83543.13785486092, "num_eq": 4075902365879035, "num_range": 100000, "upper_bound": "1832260259"}, {"distinct_range": 1541431572883327200, "num_eq": 0, "num_range": 1541431572883327196, "upper_bound": "1986718525"}, {"distinct_range": 0, "num_eq": 3751258923164999179, "num_range": 7152728482870204030, "upper_bound": "2275203520"}, {"distinct_range": 0, "num_eq": 500, "num_range": 70, "upper_bound": "3630698987"}, {"distinct_range": 15314234172.112936, "num_eq": 1453548165899614099, "num_range": 50000000000, "upper_bound": "3885171565"}, {"distinct_range": 0, "num_eq": 7034928299217872021, "num_range": 0, "upper_bound": "4061438836"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 547079628459727791, "row_count": 3659684344736147375}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 57920846788, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000, "num_range": 0, "upper_bound": "-7.495737550499790357E+25"}, {"distinct_range": 0, "num_eq": 4878971460236834505, "num_range": 200, "upper_bound": "-2.97817981223574355E+20"}, {"distinct_range": 7232046169070061000, "num_eq": 2073203840864437611, "num_range": 7232046169070060462, "upper_bound": "2.744823212836167863E+20"}, {"distinct_range": 40000000, "num_eq": 3000000000, "num_range": 40000000, "upper_bound": "2.686830291820749021E+36"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 80000000000}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 79228707839, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000000000}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4289851352, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000000000}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16885320688, "histo_buckets": [{"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "1992-07-27 00:26:16.000386"}, {"distinct_range": 202855097.67335662, "num_eq": 2000000000, "num_range": 300000000, "upper_bound": "2011-05-25 09:02:04.000559"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 80000000000}, {"colum

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2697362&tab=buildLog) on [master@8b6f3c84cc256debeeb4d4055c5f0d5c9a481213](https://github.com/cockroachdb/cockroach/commits/8b6f3c84cc256debeeb4d4055c5f0d5c9a481213):

```
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2021-02-20 06:12:47.524 UTC [47] LOG:  received fast shutdown request
        postgres_1    | .2021-02-20 06:12:47.526 UTC [47] LOG:  aborting any active transactions
        postgres_1    | 2021-02-20 06:12:47.528 UTC [47] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
        postgres_1    | 2021-02-20 06:12:47.534 UTC [49] LOG:  shutting down
        postgres_1    | 2021-02-20 06:12:47.636 UTC [47] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-02-20 06:12:47.735 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-02-20 06:12:47.735 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-02-20 06:12:47.739 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-02-20 06:12:47.753 UTC [84] LOG:  database system was shut down at 2021-02-20 06:12:47 UTC
        postgres_1    | 2021-02-20 06:12:47.759 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (3.44s)
        test_1        |     --- FAIL: TestCompare/postgres (2.68s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.2:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.75s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 INT8, col1_1 VARCHAR NOT NULL, PRIMARY KEY (col1_1 ASC), INDEX (col1_0, col1_1 DESC) WHERE (table1.col1_0 >= -1) OR (table1.col1_1 < 'X'), UNIQUE (col1_1 DESC, col1_0 ASC) WHERE table1.col1_0 >= (-9223372036854775807):::INT8, FAMILY (col1_1), FAMILY (col1_0));
        test_1        |             CREATE TABLE table2 (col1_1 VARCHAR NOT NULL, col2_1 FLOAT4, col2_2 BYTES NULL, col2_3 FLOAT8 NULL, col2_4 UUID, col2_5 FLOAT4, PRIMARY KEY (col1_1 ASC, col2_2, col2_4), INDEX (col2_5 ASC, col2_3 ASC, col2_1 ASC, col1_1 ASC, col2_4 ASC, col2_2 ASC) WHERE (((table2.col2_3 < 1.401298464324817e-45) AND (table2.col2_1 > -Inf)) AND (table2.col2_2 > '\x58')) AND (table2.col2_5 > NaN), UNIQUE (col2_3, col2_5 DESC, col2_2 DESC, col2_4, col2_1 ASC, col1_1) WHERE ((((table2.col2_2 > e'\\x00':::BYTES) OR (table2.col2_3 <= '+Inf':::FLOAT8)) AND (table2.col2_5 < '+Inf':::FLOAT8)) OR (table2.col1_1 > '':::STRING)) OR (table2.col2_1 > (-1.0):::FLOAT8), UNIQUE (col2_3 ASC, col2_4 DESC, col1_1, col2_2, col2_5 DESC) STORING (col2_1) WHERE ((((table2.col1_1 = e'\'') AND (table2.col2_2 > '\x00')) OR (table2.col2_1 > +Inf)) AND (table2.col2_3 <= -Inf)) AND (table2.col2_5 != +Inf), INDEX (col2_5 DESC, col2_1 ASC, col2_2 DESC, col1_1, col2_3 ASC) WHERE table2.col2_5 < 1.401298464324817e-45, INDEX (col2_4 ASC) WHERE table2.col2_5 < NaN, FAMILY (col1_1), FAMILY (col2_4), FAMILY (col2_2, col2_3), FAMILY (col2_5, col2_1)) INTERLEAVE IN PARENT table1 (col1_1);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2852033148228545936, "histo_buckets": [{"distinct_range": 0, "num_eq": 3412320438647225399, "num_range": 0, "upper_bound": "-7326885086501513535"}, {"distinct_range": 6402.520834121462, "num_eq": 252425585392388877, "num_range": 7000, "upper_bound": "-1565060115805193063"}, {"distinct_range": 0, "num_eq": 0, "num_range": 500000000, "upper_bound": "-813961715652879218"}, {"distinct_range": 500000, "num_eq": 109193535909293737, "num_range": 500000, "upper_bound": "-771876370366686348"}, {"distinct_range": 226535117.4943039, "num_eq": 6996012617516914109, "num_range": 600000000, "upper_bound": "-32768"}, {"distinct_range": 3000, "num_eq": 300, "num_range": 3000, "upper_bound": "2265191402832355260"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 2107566237059539750, "row_count": 5049900521215559176}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1655818281030063472, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 7323832793882249000, "num_eq": 4716088701628326724, "num_range": 7323832793882249022, "upper_bound": "GJ\\u0019Se>"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 5049900521215559176}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10666, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": "2508e2aa-d713-444c-b2ab-5b67a8105d28"}, {"distinct_range": 0, "num_eq": 500000000, "num_range": 4000000000, "upper_bound": "30df409e-0eb1-4e7c-abc7-9ca4fadfad6c"}, {"distinct_range": 0, "num_eq": 800, "num_range": 4445086852165913533, "upper_bound": "4f8614a6-5f75-487e-b0f1-750a06766741"}, {"distinct_range": 2974762317252962000, "num_eq": 80, "num_range": 3349174246988801598, "upper_bound": "8f1de23b-6a05-4719-a77a-3734faa560d3"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 0, "upper_bound": "a263a7b7-f9f8-4884-a6a7-d0386707fedd"}, {"distinct_range": 90000000, "num_eq": 763418515988003088, "num_range": 90000000, "upper_bound": "a7897875-bc19-464e-9012-84b7728489b0"}, {"distinct_range": 0, "num_eq": 40000000000, "num_range": 4000000000, "upper_bound": "ffffffff-ffff-ffff-ffff-ffffffffffff"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 25495, "row_count": 30000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 872, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "-Inf"}, {"distinct_range": 1538185544749383700, "num_eq": 5130318225150663028, "num_range": 1538185544749383664, "upper_bound": "-0.9628655314445496"}, {"distinct_range": 0, "num_eq": 3426699816844912153, "num_range": 7678724229328301026, "upper_bound": "0.47958770394325256"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 19223, "row_count": 30000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27211, "histo_buckets": [{"distinct_range": 0, "num_eq": 800, "num_range": 0, "upper_bound": "5I"}, {"distinct_range": 1862050.4066334364, "num_eq": 398937246733583488, "num_range": 3000000, "upper_bound": "8+"}, {"distinct_range": 0, "num_eq": 6174526877740295330, "num_range": 740410186898691409, "upper_bound": ":*@5zt$&x"}, {"distinct_range": 800, "num_eq": 1000000000, "num_range": 800, "upper_bound": "@f\\"N6\\u0018"}, {"distinct_range": 0, "num_eq": 800000, "num_range": 2800909847716499065, "upper_bound": "O2\\"\\u0018$/"}, {"distinct_range": 0, "num_eq": 4137720261261325861, "num_range": 4622714255486986215, "upper_bound": "s"}, {"distinct_range": 4494224847.554977, "num_eq": 1000000, "num_range": 60000000000, "upper_bound": "}S"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9112, "histo_col_type": "", "name": "__auto__", "null_count": 7894, "row_count": 30000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2640, "histo_col_type": "", "name": "__auto__", "null_count": 3932, "row_count": 30000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4113, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000000, "num_range": 0, "upper_bound": "-1.106263267391015"}, {"distinct_range": 430539080.69506484, "num_eq": 7000, "num_range": 700000000, "upper_bound": "-0.44381200890610784"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 25170, "row_count": 30000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('uockm');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('ciqe');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('advohs', 'lokqik', 'c');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3068, "histo_buckets": [{"distinct_range": 0, "num_eq": 3942867998740717974, "num_range": 0, "upper_bound": "166244555501884612"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 7130, "row_count": 10000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2398, "histo_buckets": [{"distinct_range": 0, "num_eq": 6823159828540726534, "num_range": 0, "upper_bound": "4"}, {"distinct_range": 0, "num_eq": 2135682391932864123, "num_range": 0, "upper_bound": "6\\u0017f:\\\\\'y\\u001e"}, {"distinct_range": 1000, "num_eq": 500, "num_range": 1000, "upper_bound": ":"}, {"distinct_range": 1352840365296573700, "num_eq": 4000000000, "num_range": 1352840365296573784, "upper_bound": "u\\u0017-Sc"}, {"distinct_range": 600000, "num_eq": 577645545596725571, "num_range": 600000, "upper_bound": "\U00002603"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 10000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 9008479923173419059, "num_range": 0, "upper_bound": "\\n2=5\\u001e5J"}, {"distinct_range": 6000000, "num_eq": 900000, "num_range": 6000000, "upper_bound": "\\u000c\\u0016%_T\\n[\\"t"}, {"distinct_range": 0, "num_eq": 2560963489089570321, "num_range": 40000, "upper_bound": "\\u0018p\\u0017Vk"}, {"distinct_range": 5981927160084693000, "num_eq": 3000, "num_range": 5981927160084693437, "upper_bound": "\'"}, {"distinct_range": 870400366918900000, "num_eq": 7390666344229437123, "num_range": 870400366918899904, "upper_bound": "*0\\u0017"}, {"distinct_range": 0, "num_eq": 4000000, "num_range": 70000, "upper_bound": "=E"}, {"distinct_range": 0, "num_eq": 8151586867644773388, "num_range": 3378683303308142864, "upper_bound": "C}\x7fs"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 2313968490619917562, "upper_bound": "y0eQ"}, {"distinct_range": 0, "num_eq": 0, "num_range": 900000, "upper_bound": "\U00002603"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 6199656683561370172, "num_range": 0, "upper_bound": "-1.0"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000000, "num_range": 0, "upper_bound": "8671ff35-18e5-45de-91be-f4e0ad9f2b19"}, {"distinct_range": 305.6533166287047, "num_eq": 7754903150248645392, "num_range": 400, "upper_bound": "bf34208f-4ce3-4fb8-81d5-061785c22f6d"}, {"distinct_range": 0, "num_eq": 800, "num_range": 80000, "upper_bound": "fbfa8230-3ce3-4e09-a475-8d9087a95920"}, {"distinct_range": 2885491119928013000, "num_eq": 1078108133241119059, "num_range": 2885491119928012858, "upper_bound": "ffffffff-ffff-ffff-ffff-ffffffffffff"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 40, "num_range": 0, "upper_bound": "-0.8305765390396118"}, {"distinct_range": 3048279732221506600, "num_eq": 5701829784951958975, "num_range": 3048279732221506304, "upper_bound": "-0.34229034185409546"}, {"distinct_range": 0, "num_eq": 900000, "num_range": 5323551159069995548, "upper_bound": "-0.23147276043891907"}, {"distinct_range": 0, "num_eq": 2628081109079205246, "num_range": 60000000000, "upper_bound": "-0.031078465282917023"}, {"distinct_range": 17.172132441894444, "num_eq": 7000000000, "num_range": 80, "upper_bound": "0.24004998803138733"}, {"distinct_range": 0, "num_eq": 5000000000, "num_range": 70000000000, "upper_bound": "1.029278039932251"}, {"distinct_range": 0, "num_eq": 6781871881440531360, "num_range": 1000000000, "upper_bound": "1.0655025243759155"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46569522, "histo_buckets": [{"distinct_range": 0, "num_eq": 2727535707340381378, "num_range": 0, "upper_bound": "-6629211217502044616"}, {"distinct_range": 237989843256409600, "num_eq": 4271175833348799847, "num_range": 1199830940602965246, "upper_bound": "-6504931560917800951"}, {"distinct_range": 0, "num_eq": 753395621412501577, "num_range": 7182530993495183092, "upper_bound": "-4409495258449405797"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 6000, "upper_bound": "-2178492429813992703"}, {"distinct_range": 9000000, "num_eq": 9162655931505780767, "num_range": 9000000, "upper_bound": "-182595874226307518"}, {"distinct_range": 28.3641820139437, "num_eq": 7000, "num_range": 80, "upper_bound": "165937800763392071"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 500, "upper_bound": "3612964251002986864"}, {"distinct_range": 3528319398424994300, "num_eq": 7872415049897022500, "num_range": 3528319398424994106, "upper_bound": "4535918281010049669"}, {"distinct_range": 0, "num_eq": 971437013070221146, "num_range": 2376154776350319128, "upper_bound": "4662637046050285044"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 17748327, "row_count": 70000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38845437, "histo_buckets": [{"distinct_range": 0, "num_eq": 5390620959160373041, "num_range": 0, "upper_bound": "*\\u0010t:("}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 7849578841328980576, "upper_bound": ",R+tH3;}"}, {"distinct_range": 993570400777989800, "num_eq": 9018587116070451555, "num_range": 993570400777989711, "upper_bound": "HoF:="}, {"distinct_range": 7713918269091290000, "num_eq": 1000, "num_range": 7713918269091290180, "upper_bound": "ODG=D\\u001d"}, {"distinct_range": 1406955587815846400, "num_eq": 1000000000, "num_range": 1406955587815846404, "upper_bound": "eX\\u000fji0"}, {"distinct_range": 0, "num_eq": 7992879385719428068, "num_range": 4461461585673857158, "upper_bound": "mq^k"}, {"distinct_range": 997612465210024000, "num_eq": 1000000, "num_range": 2049372542703668452, "upper_bound": "s\\u0015\\u0003q"}, {"distinct_range": 976368153323858600, "num_eq": 100000000, "num_range": 7267453587963561682, "upper_bound": "\U00002603"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 70000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 324371840, "histo_buckets": [{"distinct_range": 0, "num_eq": 3816710547306143557, "num_range": 0, "upper_bound": "-1.433028999924715"}, {"distinct_range": 100, "num_eq": 300, "num_range": 100, "upper_bound": "-1.0415472801097243"}, {"distinct_range": 0, "num_eq": 0, "num_range": 1930036635168369908, "upper_bound": "0.37479463735624774"}, {"distinct_range": 2649.7195323453516, "num_eq": 60000000, "num_range": 5000, "upper_bound": "1.0794364766742475"}, {"distinct_range": 0, "num_eq": 1517232993657280548, "num_range": 400, "upper_bound": "1.7976931348623157e+308"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 601025644, "row_count": 1000000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 402203037, "histo_buckets": [{"distinct_range": 0, "num_eq": 1912717757429409910, "num_range": 0, "upper_bound": "00000000-0000-0000-0000-000000000000"}, {"distinct_range": 0, "num_eq": 90000, "num_range": 300000, "upper_bound": "0e28d21a-3009-41ba-abf1-a556d92c2cb2"}, {"distinct_range": 3093961601612961000, "num_eq": 6222232534599866171, "num_range": 3093961601612960534, "upper_bound": "11637f75-04e3-4fb1-802c-5147207c07b7"}, {"distinct_range": 20, "num_eq": 60000, "num_range": 20, "upper_bound": "136d547e-a9ce-4385-a6e9-26350456e630"}, {"distinct_range": 400000, "num_eq": 60000000, "num_range": 400000, "upper_bound": "cc9882e3-3ae0-4ae4-9260-6337ed8956d3"}, {"distinct_range": 23598.869856309968, "num_eq": 600000000, "num_range": 30000, "upper_bound": "d9d533cf-08ee-4be1-91ad-3ceca86477db"}, {"distinct_range": 505578202203568960, "num_eq": 6311476213412680483, "num_range": 1416875959919770785, "upper_bound": "e6e6de94-fd47-492d-a1f6-fb1bd6c2d98a"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 850982821, "row_count": 1000000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 798753286, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000000, "num_range": 0, "upper_bound": "-1.3476471900939941"}, {"distinct_range": 6168145288263461000, "num_eq": 900, "num_range": 6168145288263461131, "upper_bound": "-1.102073311805725"}, {"distinct_range": 2002831621809455600, "num_eq": 8000000000, "num_range": 2415045422636195459, "upper_bound": "0.6620413064956665"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 90000000000, "upper_bound": "1.5499053001403809"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 841676525, "row_count": 1000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39556776, "histo_buckets": [{"distinct_range": 0, "num_eq": 2203928386878730259, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 461.55775370829883, "num_eq": 5114790958492963787, "num_range": 1000, "upper_bound": "\\u001d|S\\u0017\\u000e"}, {"distinct_range": 6318125512978872000, "num_eq": 4136243088912154501, "num_range": 6318125512978872612, "upper_bound": ",(\\u001eN|:3"}, {"distinct_range": 257913057765386620, "num_eq": 6300595439220713342, "num_range": 617483045923086154, "upper_bound": ".9]"}, {"distinct_range": 7642311178556022000, "num_eq": 1855431009743179057, "num_range": 7642311178556021800, "upper_bound": "6:B\\u00042Z,"}, {"distinct_range": 0, "num_eq": 5098385848259195208, "num_range": 3999611026582925262, "upper_bound": "EB"}, {"distinct_range": 3269356058736422400, "num_eq": 20000, "num_range": 4595820929600128656, "upper_bound": "^I"}, {"distinct_range": 30, "num_eq": 90, "num_range": 30, "upper_bound": "n)A\\u001a"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 1000000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 83026916, "histo_col_type": "", "name": "__auto__", "null_count": 283485664, "row_count": 1000000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 124388409, "histo_col_type": "", "name": "__auto__", "null_count": 612493573, "row_count": 1000000000}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "inf" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2699363&tab=buildLog) on [master@64c4aef909f4382523cd9248341ca9f4448d841a](https://github.com/cockroachdb/cockroach/commits/64c4aef909f4382523cd9248341ca9f4448d841a):

```
        cockroach2_1  | store[0]:            path=/cockroach-data
        cockroach2_1  | storage engine:      pebble
        cockroach2_1  | status:              initialized new cluster
        cockroach2_1  | clusterID:           71d40b98-6365-40e2-806c-ec73c166e07e
        cockroach2_1  | nodeID:              1
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        test_1        | --- FAIL: TestCompare (1.39s)
        test_1        |     --- FAIL: TestCompare/mutators (0.49s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 BIT(40) NOT NULL, PRIMARY KEY (col1_0), UNIQUE (col1_0 ASC), UNIQUE (col1_0 ASC), UNIQUE (col1_0 DESC), INDEX (col1_0), INDEX (col1_0 DESC), UNIQUE (col1_0 ASC), INDEX (col1_0 ASC), INDEX (col1_0), INDEX (col1_0));
        test_1        |             CREATE TABLE table2 (col1_0 BIT(40) NOT NULL, col2_1 BIT(27) NULL, col2_2 TIMESTAMPTZ, col2_3 INT4, col2_4 INT8, col2_5 DATE NOT NULL, col2_6 REGNAMESPACE NULL, col2_7 BYTES NULL, col2_8 DATE, col2_9 REGCLASS NULL, col2_10 DECIMAL, col2_11 TIMETZ NULL, PRIMARY KEY (col1_0), UNIQUE (col2_1 DESC, col2_4, col2_7 ASC, col1_0 DESC, col2_5, col2_9, col2_11 DESC, col2_3 ASC) STORING (col2_2, col2_8), INDEX (col2_11, col1_0, col2_5, col2_6 DESC, col2_2 DESC, col2_9, col2_8 DESC, col2_10 ASC, col2_1) WHERE (((((((table2.col2_2 <= '-4713-11-24 00:00:00+00:00') OR (table2.col2_4 = 2147483647)) AND (table2.col2_5 != '5874897-12-31')) OR (table2.col2_8 < '5874897-12-31')) AND (table2.col2_7 >= '\x58')) AND (table2.col2_3 <= 127)) AND (table2.col2_10 > Infinity)) AND (table2.col2_11 <= '24:00:00-15:59:00'), INDEX (col2_5 DESC, col2_1 DESC, col2_8 ASC, col1_0 DESC, col2_7 DESC, col2_9 ASC, col2_11 ASC, col2_2, col2_6 ASC, col2_4 ASC, col2_3 DESC, col2_10 ASC) WHERE (((((((table2.col2_5 < '5874897-12-31':::DATE) AND (table2.col2_7 < e'\\x22':::BYTES)) AND (table2.col2_4 >= (-2147483648):::INT8)) AND (table2.col2_8 < '-infinity':::DATE)) OR (table2.col2_10 >= 'Infinity':::DECIMAL)) AND (table2.col2_2 = '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table2.col2_3 <= 2147483647:::INT8)) AND (table2.col2_11 < '24:00:00-15:59:00':::TIMETZ), INDEX (col2_6 DESC, col2_5) STORING (col2_1, col2_2, col2_3, col2_4, col2_8, col2_9, col2_11) WHERE (((((((table2.col2_8 > '4714-11-24 BC':::DATE) AND (table2.col2_11 = '24:00:00-15:59:00':::TIMETZ)) OR (table2.col2_10 = 1:::DECIMAL)) AND (table2.col2_4 > (-1):::INT8)) OR (table2.col2_2 = '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table2.col2_3 = (-1):::INT8)) OR (table2.col2_5 >= '1970-01-01':::DATE)) OR (table2.col2_7 < e'\\x22':::BYTES), UNIQUE (col2_9 DESC) STORING (col2_1, col2_2, col2_3, col2_8) WHERE (((table2.col2_7 >= '\x22') AND (table2.col2_8 != '-infinity')) OR (table2.col2_5 = '5874897-12-31')) AND (table2.col2_11 > '00:00:00+15:59:00'), INDEX (col2_6 ASC, col2_10 DESC, col2_2 ASC, col2_1 DESC, col2_5 ASC) STORING (col2_3, col2_4, col2_7, col2_9) WHERE (((((((table2.col2_4 = 1) OR (table2.col2_11 = '00:00:00+15:59:00')) AND (table2.col2_5 > '5874897-12-31')) AND (table2.col2_3 <= 0)) OR (table2.col2_10 > 1)) OR (table2.col2_2 >= '-4713-11-24 00:00:00+00:00')) AND (table2.col2_8 >= '5874897-12-31')) AND (table2.col2_7 > '\x58'), INDEX (col2_1 DESC) STORING (col2_2, col2_5, col2_6, col2_9), INDEX (col2_5 ASC, col2_9 DESC, col2_8, col1_0, col2_10, col2_11, col2_6 DESC, col2_1 DESC, col2_2 ASC) STORING (col2_3, col2_7), INDEX (col2_6 ASC, col2_8 DESC, col2_7, col2_1 DESC, col2_2, col2_11 ASC, col2_10 ASC, col2_9 ASC, col2_4 ASC, col2_3 ASC) STORING (col2_5), FAMILY (col2_4, col2_11), FAMILY (col2_7), FAMILY (col2_3), FAMILY (col2_5), FAMILY (col2_6), FAMILY (col2_10), FAMILY (col2_9), FAMILY (col2_2, col2_1), FAMILY (col2_8), FAMILY (col1_0)) INTERLEAVE IN PARENT table1 (col1_0);
        test_1        |             CREATE TABLE table3 (col3_0 REGPROCEDURE NULL, col3_1 INT8 NOT NULL, col3_2 REGNAMESPACE NOT NULL, col3_3 INT8 NOT NULL, col3_4 BYTES NOT NULL, col3_5 FLOAT8 NULL, col3_6 OID NOT NULL, col3_7 REGTYPE NULL, col3_8 CHAR, col3_9 BIT(5) NULL, col3_10 BYTES NOT NULL, col3_11 BIT(16) NOT NULL, col3_12 FLOAT4 NOT NULL, col3_13 INT8 NOT NULL, col3_14 INT2, col3_15 NAME, col3_16 INET NOT NULL, col3_17 BIT(15) NOT NULL, PRIMARY KEY (col3_12, col3_4 ASC, col3_11 ASC, col3_3 DESC, col3_2 ASC, col3_16, col3_1), UNIQUE (col3_12 DESC, col3_7 ASC) STORING (col3_0, col3_5, col3_14) WHERE ((((table3.col3_10 != e'\\x58':::BYTES) OR (table3.col3_4 != e'\\xff':::BYTES)) OR (table3.col3_13 = (-32768):::INT8)) AND (table3.col3_5 < '+Inf':::FLOAT8)) OR (table3.col3_3 >= (-128):::INT8), INDEX (col3_11 DESC, col3_0, col3_15 ASC, col3_1, col3_13 DESC, col3_2 ASC, col3_14, col3_16 ASC, col3_17 ASC, col3_8, col3_5 DESC, col3_7 DESC) STORING (col3_9, col3_10) WHERE (((table3.col3_1 >= 1:::INT8) AND (table3.col3_15 = 'X':::STRING)) OR (table3.col3_14 <= (-1):::INT8)) OR (table3.col3_5 >= 1.401298464324817e-45:::FLOAT8), UNIQUE (col3_2 ASC, col3_10 DESC, col3_16, col3_4 ASC, col3_9 DESC, col3_7, col3_0 ASC, col3_12 DESC, col3_14 ASC, col3_5, col3_6 ASC, col3_17) WHERE (((((((table3.col3_3 >= -1) AND (table3.col3_12 >= 1.401298464324817e-45)) AND (table3.col3_15 >= 'X')) OR (table3.col3_13 <= -1)) AND (table3.col3_8 <= e'\x00')) OR (table3.col3_1 < -1)) OR (table3.col3_10 > '\xe29883')) AND (table3.col3_14 <= 127), INDEX (col3_0 ASC, col3_16 ASC, col3_2 DESC) STORING (col3_5, col3_6, col3_7, col3_9, col3_13) WHERE ((((((((table3.col3_15 >= '"':::STRING) OR (table3.col3_13 = 0:::INT8)) AND (table3.col3_1 >= 9223372036854775807:::INT8)) AND (table3.col3_3 > (-9223372036854775807):::INT8)) AND (table3.col3_12 >= 1.401298464324817e-45:::FLOAT8)) OR (table3.col3_14 > 32767:::INT8)) OR (table3.col3_10 > e'\\x':::BYTES)) AND (table3.col3_5 < 0.0:::FLOAT8)) AND (table3.col3_8 <= e'\'':::STRING), UNIQUE (col3_10 ASC, col3_11, col3_0 ASC, col3_3 ASC, col3_4 ASC, col3_17, col3_16 DESC, col3_1 ASC, col3_2 ASC, col3_13 ASC, col3_12 DESC, col3_8 DESC, col3_15, col3_7, col3_6 DESC, col3_9 DESC, col3_14 ASC, col3_5 ASC) WHERE ((((((((table3.col3_14 > -1) OR (table3.col3_1 < -32768)) OR (table3.col3_12 != 0.0)) OR (table3.col3_5 = 0.0)) AND (table3.col3_13 <= -9223372036854775807)) AND (table3.col3_8 = e'\x00')) AND (table3.col3_3 <= -2147483648)) AND (table3.col3_4 >= '\x22')) OR (table3.col3_15 < 'X'), INDEX (col3_7, col3_6 ASC, col3_1 DESC, col3_10 DESC, col3_2 ASC, col3_14 DESC, col3_15, col3_13 ASC, col3_12 DESC, col3_17) STORING (col3_0) WHERE (((((((((table3.col3_8 != 'X') AND (table3.col3_1 < 2147483647)) OR (table3.col3_14 = -1)) OR (table3.col3_5 != 1.0)) AND (table3.col3_12 = 1.401298464324817e-45)) AND (table3.col3_4 > '\xff')) OR (table3.col3_13 != 1)) OR (table3.col3_15 >= e'\x00')) AND (table3.col3_3 != 127)) AND (table3.col3_10 != '\x27'), INDEX (col3_2 ASC, col3_12 ASC) WHERE ((table3.col3_4 = '\xff') OR (table3.col3_10 != '\x22')) AND (table3.col3_15 != '"'), INDEX (col3_16 DESC, col3_12 ASC, col3_0 DESC, col3_13 ASC, col3_4, col3_17, col3_10, col3_9 ASC, col3_7 DESC, col3_14, col3_6 ASC) STORING (col3_8, col3_15) WHERE (table3.col3_13 <= 1:::INT8) AND (table3.col3_8 >= '"':::STRING), INDEX (col3_7, col3_1, col3_6 DESC, col3_0 DESC, col3_11) STORING (col3_5, col3_9, col3_13, col3_17) WHERE (((((((((table3.col3_5 > 1.7976931348623157e+308) OR (table3.col3_3 > 1)) OR (table3.col3_10 >= '\x')) OR (table3.col3_15 >= e'\x00')) AND (table3.col3_14 > 1)) AND (table3.col3_13 > -32768)) AND (table3.col3_12 >= 1.401298464324817e-45)) OR (table3.col3_1 = 32767)) AND (table3.col3_4 >= '\x58')) AND (table3.col3_8 < e'\x00'), FAMILY (col3_6, col3_11, col3_1), FAMILY (col3_14), FAMILY (col3_17, col3_3), FAMILY (col3_10), FAMILY (col3_12), FAMILY (col3_15), FAMILY (col3_8), FAMILY (col3_13, col3_16, col3_0, col3_7), FAMILY (col3_9, col3_2, col3_4), FAMILY (col3_5));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2451354438461721417, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": "0000000000000000000000000000000000000000"}, {"distinct_range": 66.4521265728753, "num_eq": 400, "num_range": 200, "upper_bound": "0111111010001011010111101110001110001101"}, {"distinct_range": 1038499.0073622307, "num_eq": 30, "num_range": 5000000, "upper_bound": "0111111111111111111111111111111111111111"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 50000, "upper_bound": "1010011010011110111110001000100110000011"}, {"distinct_range": 37.29986350260293, "num_eq": 4669367414708529185, "num_range": 60, "upper_bound": "1011011010100110000110001010110110111101"}], "histo_col_type": "BIT(40)", "name": "__auto__", "null_count": 0, "row_count": 2745103776649661884}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6461647, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000, "num_range": 0, "upper_bound": "0000001000110010010101111110011010011010"}, {"distinct_range": 10000, "num_eq": 20, "num_range": 10000, "upper_bound": "0001010001010111101111010011010101011110"}, {"distinct_range": 516061.7261750924, "num_eq": 0, "num_range": 4000000, "upper_bound": "0001111001010000111010001100111111110010"}, {"distinct_range": 3454840281342854700, "num_eq": 40000000, "num_range": 3454840281342854884, "upper_bound": "0110101000011100010010100011011100110000"}, {"distinct_range": 196579590.38230777, "num_eq": 8288253184508841176, "num_range": 3000000000, "upper_bound": "1010000001001010100111111111000111001100"}, {"distinct_range": 5000000000, "num_eq": 0, "num_range": 5000000000, "upper_bound": "1010010000101110000101010100101101001111"}, {"distinct_range": 6991415.024245448, "num_eq": 7000, "num_range": 10000000, "upper_bound": "1010101101010010100000110101101101001110"}, {"distinct_range": 3508793868325554700, "num_eq": 6190171229689129457, "num_range": 3508793868325554887, "upper_bound": "1111101101111011101010111000100000110111"}], "histo_col_type": "BIT(40)", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 945241, "histo_buckets": [{"distinct_range": 0, "num_eq": 2813807595825995670, "num_range": 0, "upper_bound": "011001100010001110001111001"}], "histo_col_type": "BIT(27)", "name": "__auto__", "null_count": 3485366, "row_count": 9000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5594048, "histo_col_type": "", "name": "__auto__", "null_count": 8444048, "row_count": 9000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1317410, "histo_col_type": "", "name": "__auto__", "null_count": 4443284, "row_count": 9000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 901163, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1970-12-07"}, {"distinct_range": 3339174167834980400, "num_eq": 30, "num_range": 3339174167834980536, "upper_bound": "1971-04-04"}, {"distinct_range": 700000, "num_eq": 0, "num_range": 700000, "upper_bound": "1972-03-06"}, {"distinct_range": 7000000, "num_eq": 4000000, "num_range": 7000000, "upper_bound": "1974-06-24"}, {"distinct_range": 6472651660.966355, "num_eq": 900000000, "num_range": 7000000000, "upper_bound": "1977-08-31"}, {"distinct_range": 800000000, "num_eq": 20000, "num_range": 800000000, "upper_bound": "1977-09-11"}, {"distinct_range": 11305449653.8328, "num_eq": 3555391283566866545, "num_range": 50000000000, "upper_bound": "1990-02-21"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 9000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8223948, "histo_col_type": "", "name": "__auto__", "null_count": 6570852, "row_count": 9000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7798881, "histo_buckets": [{"distinct_range": 0, "num_eq": 5211241567617254595, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 0, "upper_bound": "985371507"}, {"distinct_range": 5000000000, "num_eq": 0, "num_range": 5000000000, "upper_bound": "1287772806"}, {"distinct_range": 2024850217262073900, "num_eq": 8000000000, "num_range": 2024850217262073957, "upper_bound": "1346863760"}, {"distinct_range": 50000000, "num_eq": 7646459608932501672, "num_range": 50000000, "upper_bound": "1687569959"}, {"distinct_range": 7873613908930998000, "num_eq": 5000, "num_range": 7873613908930998536, "upper_bound": "3262388847"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 7605509, "row_count": 9000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3788142, "histo_col_type": "", "name": "__auto__", "null_count": 6876319, "row_count": 9000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6740073, "histo_col_type": "", "name": "__auto__", "null_count": 57610, "row_count": 9000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6357029, "histo_buckets": [{"distinct_range": 0, "num_eq": 4130924554241144074, "num_range": 0, "upper_bound": "512211587"}, {"distinct_range": 0, "num_eq": 900000, "num_range": 789728659388132732, "upper_bound": "1555294043"}, {"distinct_range": 0, "num_eq": 900000000, "num_range": 2000000000, "upper_bound": "2766717029"}, {"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "3305444112"}, {"distinct_range": 0, "num_eq": 6557074596017729141, "num_range": 50000000, "upper_bound": "4087736781"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 3054751, "row_count": 9000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3584206, "histo_col_type": "", "name": "__auto__", "null_count": 7335413, "row_count": 9000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2205302, "histo_buckets": [{"distinct_range": 0, "num_eq": 2358943009536959860, "num_range": 0, "upper_bound": "16:59:20.580476+11:21:00"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 418310095311177191, "upper_bound": "10:08:48.181288+03:01:00"}, {"distinct_range": 0, "num_eq": 2727252363644666078, "num_range": 5000000000, "upper_bound": "06:18:13.854016-09:53:00"}, {"distinct_range": 1274024477924156700, "num_eq": 3421612535438097517, "num_range": 4877499821088281393, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 2734058, "row_count": 9000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 292, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 834, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 280, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 600, "num_range": 70000, "upper_bound": "853734714"}, {"distinct_range": 6000000000, "num_eq": 20000000, "num_range": 6000000000, "upper_bound": "1601491496"}, {"distinct_range": 8000, "num_eq": 0, "num_range": 8000, "upper_bound": "2515122545"}, {"distinct_range": 1928921578512829000, "num_eq": 0, "num_range": 6774598209047971330, "upper_bound": "3045542289"}, {"distinct_range": 5281749672430330000, "num_eq": 40000, "num_range": 6123328968630111986, "upper_bound": "3578320028"}, {"distinct_range": 1000000000, "num_eq": 5000, "num_range": 1000000000, "upper_bound": "3743192872"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 593, "histo_buckets": [{"distinct_range": 0, "num_eq": 1141333355060259479, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 0, "num_eq": 90000, "num_range": 2261393497417447382, "upper_bound": "\\\\x275f8e45563241"}, {"distinct_range": 2108074152.648787, "num_eq": 1000000, "num_range": 10000000000, "upper_bound": "\\\\x4100966ca1"}, {"distinct_range": 2530572067013852700, "num_eq": 6000000, "num_range": 2530572067013852540, "upper_bound": "\\\\xd01fac63a5711701ac"}, {"distinct_range": 88.33975004695309, "num_eq": 700000, "num_range": 100, "upper_bound": "\\\\xe29883"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 200, "histo_col_type": "BIT(16)", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 204, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000, "num_range": 0, "upper_bound": "281616371"}, {"distinct_range": 0, "num_eq": 6625934935291632518, "num_range": 7000000, "upper_bound": "1847363000"}, {"distinct_range": 2139999128565954300, "num_eq": 3000000000, "num_range": 6650336729265306579, "upper_bound": "2524289661"}, {"distinct_range": 0, "num_eq": 7536984719997011138, "num_range": 6260070402717786982, "upper_bound": "3856249622"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 458, "row_count": 1000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3, "histo_buckets": [{"distinct_range": 0, "num_eq": 9184766816534833004, "num_range": 0, "upper_bound": "0"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 78, "row_count": 1000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 833, "histo_buckets": [{"distinct_range": 0, "num_eq": 576931786975905776, "num_range": 0, "upper_bound": "-1.6874405145645142"}, {"distinct_range": 10747505.376606466, "num_eq": 100000, "num_range": 30000000, "upper_bound": "-1.2268632650375366"}, {"distinct_range": 2503414874871258000, "num_eq": 70000, "num_range": 7809949010067844697, "upper_bound": "0.0"}, {"distinct_range": 5070876869805999000, "num_eq": 3344288819634429361, "num_range": 5070876869805999285, "upper_bound": "0.5254782438278198"}, {"distinct_range": 0, "num_eq": 2000000, "num_range": 7000, "upper_bound": "0.6326291561126709"}, {"distinct_range": 9.979281110477423, "num_eq": 7000, "num_range": 100, "upper_bound": "3.4028234663852886e+38"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 220, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 972, "histo_col_type": "", "name": "__auto__", "null_count": 286, "row_count": 1000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29, "histo_col_type": "", "name": "__auto__", "null_count": 708, "row_count": 1000}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 616, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 330, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 48, "histo_col_type": "", "name": "__auto__", "null_count": 230, "row_count": 1000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 64, "histo_col_type": "", "name": "__auto__", "null_count": 152, "row_count": 1000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 298, "histo_col_type": "", "name": "__auto__", "null_count": 627, "row_count": 1000}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 677, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": "212.124.136.138/4"}, {"distinct_range": 0, "num_eq": 2209886929602013049, "num_range": 3426480581689490486, "upper_bound": "11.159.179.71/20"}, {"distinct_range": 0, "num_eq": 5318135401760500625, "num_range": 70, "upper_bound": "177.146.146.75/26"}, {"distinct_range": 2984834764956753400, "num_eq": 200000000, "num_range": 8737255501046648991, "upper_bound": "86.11.158.3/29"}, {"distinct_range": 4807949702461923000, "num_eq": 10000000, "num_range": 5490124495545799304, "upper_bound": "5bcf:5073:96a9:689:807f:dcfc:e8df:2708/49"}, {"distinct_range": 20000000, "num_eq": 4089306306706488188, "num_range": 20000000, "upper_bound": "61d7:d190:f7a2:b3f2:5d39:af76:5f22:1bcb/50"}, {"distinct_range": 700000000, "num_eq": 6750577378684127665, "num_range": 700000000, "upper_bound": "ef1f:9f92:4b8f:cd1a:a97d:9af1:b1ee:1cc1/77"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 1000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('tl', 'tk', 'itrylu', 'ekht', 'qten', 'civevr');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('zlscan', 'iqlzrn', 'kv');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('uoi');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('jpom');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('z', 'jr', 'jm');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9482720, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": "0000011010100110001010011111101001011011"}, {"distinct_range": 0, "num_eq": 2273053873593731281, "num_range": 50000000000, "upper_bound": "0100111111100101010011000001111010011000"}, {"distinct_range": 254927.4121840417, "num_eq": 8000000, "num_range": 600000, "upper_bound": "0111010011001011101111011111111111001111"}, {"distinct_range": 0, "num_eq": 6703719793794054984, "num_range": 6000000, "upper_bound": "0111111111111111111111111111111111111111"}, {"distinct_range": 8362878681370619000, "num_eq": 3925868096558788816, "num_range": 8362878681370619016, "upper_bound": "1001100100000001001001001000010100001100"}, {"distinct_range": 1343026425159542300, "num_eq": 1660650238845496761, "num_range": 2188797122225161137, "upper_bound": "1111011111110010101000110111011001001111"}], "histo_col_type": "BIT(40)", "name": "__auto__", "null_count": 0, "row_count": 20000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12270920, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000000, "num_range": 0, "upper_bound": "0001001111101101110000010011000000010001"}], "histo_col_type": "BIT(40)", "name": "__auto__", "null_count": 0, "row_count": 70000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 61211559, "histo_buckets": [{"distinct_range": 0, "num_eq": 7496853567991267789, "num_range": 0, "upper_bound": "101000110011001011110111100"}], "histo_col_type": "BIT(27)", "name": "__auto__", "null_count": 43035598, "row_count": 70000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33996155, "histo_col_type": "", "name": "__auto__", "null_count": 21678331, "row_count": 70000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 48327628, "histo_col_type": "", "name": "__auto__", "null_count": 1620777, "row_count": 70000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14976764, "histo_col_type": "", "name": "__auto__", "null_count": 7858079, "row_count": 70000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27429670, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "1179931325"}, {"distinct_range": 30000000, "num_eq": 40000000, "num_range": 30000000, "upper_bound": "1201980199"}, {"distinct_range": 60000, "num_eq": 1000, "num_range": 60000, "upper_bound": "1538538976"}, {"distinct_range": 100, "num_eq": 30000000000, "num_range": 100, "upper_bound": "2395549656"}, {"distinct_range": 1112873222673886100, "num_eq": 70, "num_range": 1112873222673886069, "upper_bound": "2880774830"}, {"distinct_range": 499.23665212558996, "num_eq": 282029653518756963, "num_range": 900, "upper_bound": "2887443417"}, {"distinct_range": 0, "num_eq": 8062654600591637726, "num_range": 8488595157456624010, "upper_bound": "3259521811"}, {"distinct_range": 934435349739654700, "num_eq": 1415292185712259790, "num_range": 934435349739654621, "upper_bound": "3445864828"}, {"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": "3702421110"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 22208739, "row_count": 70000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42047298, "histo_col_type": "", "name": "__auto__", "null_count": 3065985, "row_count": 70000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 68434246, "histo_col_type": "", "name": "__auto__", "null_count": 69177983, "row_count": 70000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8916234, "histo_col_type": "", "name": "__auto__", "null_count": 14553540, "row_count": 70000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 63589612, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000, "num_range": 0, "upper_bound": "1970-01-01"}, {"distinct_range": 2802450755518386700, "num_eq": 7804434264099854432, "num_range": 2802450755518386523, "upper_bound": "1976-05-14"}, {"distinct_range": 5000000000, "num_eq": 719287504800164039, "num_range": 5000000000, "upper_bound": "1977-11-29"}, {"distinct_range": 6563579397846455000, "num_eq": 0, "num_range": 6563579397846454809, "upper_bound": "1984-05-12"}, {"distinct_range": 0, "num_eq": 6425439979445561737, "num_range": 9118433133441230936, "upper_bound": "1985-08-02"}, {"distinct_range": 60000, "num_eq": 10000000, "num_range": 60000, "upper_bound": "1991-12-11"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 70000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26412831, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "2238885145"}, {"distinct_range": 4692858824144700000, "num_eq": 9040816984289905591, "num_range": 5222693690345445476, "upper_bound": "2809170941"}, {"distinct_range": 73643.76864200873, "num_eq": 100000000, "num_range": 200000, "upper_bound": "3566127812"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 48724897, "row_count": 70000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 63873084, "histo_buckets": [{"distinct_range": 0, "num_eq": 5563897912909147482, "num_range": 0, "upper_bound": "06:45:51.029046+06:15:00"}, {"distinct_range": 10000000, "num_eq": 800, "num_range": 10000000, "upper_bound": "13:25:14.539491+12:06:00"}, {"distinct_range": 0, "num_eq": 7591330725661813935, "num_range": 743282734335130432, "upper_bound": "15:02:49.539916+13:32:00"}, {"distinct_range": 1321.0727087217274, "num_eq": 5175957506298532187, "num_range": 5000, "upper_bound": "02:32:33.288597-05:47:00"}, {"distinct_range": 0, "num_eq": 2158041767978613764, "num_range": 9000000000, "upper_bound": "22:22:19.777086+10:15:00"}, {"distinct_range": 900, "num_eq": 999300681703224758, "num_range": 900, "upper_bound": "22:46:33.41266+09:54:00"}, {"distinct_range": 495858677019979500, "num_eq": 100000, "num_range": 834053716473536363, "upper_bound": "17:51:11.28727-02:43:00"}, {"distinct_range": 5424035793519902000, "num_eq": 0, "num_range": 5424035793519901939, "upper_bound": "17:08:29.12291-05:41:00"}, {"distinct_range": 0, "num_eq": 30, "num_range": 0, "upper_bound": "21:43:31.138678-08:36:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 34458720, "row_count": 70000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16, "histo_buckets": [{"distinct_range": 0, "num_eq": 300000000, "num_range": 0, "upper_bound": "NaN"}, {"distinct_range": 6803287182474016000, "num_eq": 0, "num_range": 6803287182474015280, "upper_bound": "-0.6237409710884094"}, {"distinct_range": 60000, "num_eq": 9023079477914073957, "num_range": 60000, "upper_bound": "0.5799923539161682"}, {"distinct_range": 1785261017375714000, "num_eq": 40000, "num_range": 4349959464386063672, "upper_bound": "0.5949209928512573"}, {"distinct_range": 0, "num_eq": 3603570914369766337, "num_range": 4812905251669472959, "upper_bound": "1.0744788646697998"}, {"distinct_range": 0, "num_eq": 3976090786666050662, "num_range": 300000000, "upper_bound": "1.163066029548645"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 30}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19, "histo_col_type": "", "name": "__auto__", "null_count": 14, "row_count": 30}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11, "histo_buckets": [{"distinct_range": 0, "num_eq": 7347555338993052510, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 121461738.33995464, "num_eq": 3603076180491551356, "num_range": 300000000, "upper_bound": "1004408920"}, {"distinct_range": 0, "num_eq": 3391016158709050272, "num_range": 80, "upper_bound": "2841350028"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 7416235310576835349, "upper_bound": "3533661381"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 30}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14, "histo_col_type": "", "name": "__auto__", "null_count": 23, "row_count": 30}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16, "histo_col_type": "", "name": "__auto__", "null_count": 10, "row_count": 30}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_col_type": "", "name": "__auto__", "null_count": 7, "row_count": 30}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000, "num_range": 0, "upper_bound": "35216347"}, {"distinct_range": 0, "num_eq": 4984014336472173393, "num_range": 6103730791936281582, "upper_bound": "680527741"}, {"distinct_range": 2885883716794579000, "num_eq": 2363978592032088816, "num_range": 3031588968019262853, "upper_bound": "868538489"}, {"distinct_range": 4770953696062100000, "num_eq": 60000, "num_range": 4770953696062100008, "upper_bound": "943385857"}, {"distinct_range": 149140000566314880, "num_eq": 4000, "num_range": 2110431212550612472, "upper_bound": "1082232096"}, {"distinct_range": 0, "num_eq": 2001069275000286021, "num_range": 297455817216638070, "upper_bound": "1231327359"}, {"distinct_range": 3475714881350854700, "num_eq": 166521571729925301, "num_range": 3475714881350854724, "upper_bound": "2389618750"}, {"distinct_range": 0, "num_eq": 5000000, "num_range": 500000000, "upper_bound": "2716247628"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 6000000, "upper_bound": "4289634223"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 8, "row_count": 30}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20, "histo_col_type": "", "name": "__auto__", "null_count": 25, "row_count": 30}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15, "histo_buckets": [{"distinct_range": 0, "num_eq": 4277830328824940128, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 8199583199895546000, "num_eq": 90, "num_range": 8199583199895545778, "upper_bound": "\\\\x07e4"}, {"distinct_range": 0, "num_eq": 4050241111735629816, "num_range": 8064536756985267176, "upper_bound": "\\\\x093ec86ae2407e05"}, {"distinct_range": 0, "num_eq": 1933940749794805705, "num_range": 900000, "upper_bound": "\\\\x2b739071a2250dfb"}, {"distinct_range": 842785745517846100, "num_eq": 4000, "num_range": 842785745517846136, "upper_bound": "\\\\x35"}, {"distinct_range": 0, "num_eq": 3061587009968078066, "num_range": 643717788082833503, "upper_bound": "\\\\x42"}, {"distinct_range": 1355663518421044700, "num_eq": 1545884381628566140, "num_range": 1355663518421044701, "upper_bound": "\\\\xe75e6b3087"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 30}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4, "histo_buckets": [{"distinct_range": 0, "num_eq": 2900320798810465197, "num_range": 0, "upper_bound": "1101000100110010"}], "histo_col_type": "BIT(16)", "name": "__auto__", "null_count": 0, "row_count": 30}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17, "histo_buckets": [{"distinct_range": 0, "num_eq": 5461906163193278004, "num_range": 0, "upper_bound": "539452271"}, {"distinct_range": 26.960931507113216, "num_eq": 700, "num_range": 40, "upper_bound": "1497970638"}, {"distinct_range": 1.9826909379332611, "num_eq": 1000000, "num_range": 30, "upper_bound": "1580778621"}, {"distinct_range": 11281.747153202612, "num_eq": 10000000, "num_range": 100000, "upper_bound": "2041254899"}, {"distinct_range": 6893810695887195000, "num_eq": 90000000, "num_range": 6893810695887195147, "upper_bound": "2204666834"}, {"distinct_range": 7983072662433078000, "num_eq": 7691975695992313789, "num_range": 7983072662433077910, "upper_bound": "2234845709"}, {"distinct_range": 3777211737205911000, "num_eq": 0, "num_range": 3777211737205910971, "upper_bound": "2466236980"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 4207599648838173765, "upper_bound": "3089199417"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 29, "row_count": 30}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4, "histo_buckets": [{"distinct_range": 0, "num_eq": 300000, "num_range": 0, "upper_bound": "250.133.223.117/30"}, {"distinct_range": 2365970500088937500, "num_eq": 80000000000, "num_range": 2365970500088937462, "upper_bound": "a502:7db7:1af:6a69:1f11:38fc:602:2ddf/50"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 30}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4136473430033744209, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000, "num_range": 0, "upper_bound": "0010110000010101011100001110100110010100"}, {"distinct_range": 5000000000, "num_eq": 6274535289205319833, "num_range": 5000000000, "upper_bound": "0011101011101010110011000001100110111101"}], "histo_col_type": "BIT(40)", "name": "__auto__", "null_count": 0, "row_count": 8965359039781688400}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3148347212, "histo_buckets": [{"distinct_range": 0, "num_eq": 2120331802150431073, "num_range": 0, "upper_bound": "0001110110010010100001100110011001011110"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 0, "upper_bound": "0110010101000110010011010010011011000000"}, {"distinct_range": 878762364632905500, "num_eq": 0, "num_range": 1803991469020784508, "upper_bound": "1100001100100000110000000101010001110110"}, {"distinct_range": 60000000000, "num_eq": 8766668147681870411, "num_range": 60000000000, "upper_bound": "1110011100010100000110100100000100010011"}], "histo_col_type": "BIT(40)", "name": "__auto__", "null_count": 0, "row_count": 6000000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2751233771, "histo_buckets": [{"distinct_range": 0, "num_eq": 1833586045056462998, "num_range": 0, "upper_bound": "000000100001001101010011100"}, {"distinct_range": 17.64296379075499, "num_eq": 781169904763467450, "num_range": 40, "upper_bound": "001000101001100011101101011"}, {"distinct_range": 0, "num_eq": 5311300996671376373, "num_range": 4826821878361141962, "upper_bound": "011011100101110001111111010"}, {"distinct_range": 2348969087537827000, "num_eq": 80000, "num_range": 3268186419644223925, "upper_bound": "100001000110111010100010001"}, {"distinct_range": 6409300366117772000, "num_eq": 3177623612641138977, "num_range": 8138143899115733515, "upper_bound": "101001011010000000010101001"}, {"distinct_range": 1108079048440221300, "num_eq": 0, "num_range": 4176753838527327206, "upper_bound": "110000100101001000010001100"}, {"distinct_range": 40000000, "num_eq": 6158736583674279133, "num_range": 40000000, "upper_bound": "111100110011101101000101100"}], "histo_col_type": "BIT(27)", "name": "__auto__", "null_count": 3534274730, "row_count": 6000000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1767981950, "histo_col_type": "", "name": "__auto__", "null_count": 3699414648, "row_count": 6000000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 411193593, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "1970-04-25"}, {"distinct_range": 0, "num_eq": 6938485440959898834, "num_range": 5004703509903216787, "upper_bound": "1972-03-11"}, {"distinct_range": 0, "num_eq": 0, "num_range": 200, "upper_bound": "1979-05-31"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 6000000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3536977996, "histo_col_type": "", "name": "__auto__", "null_count": 5390730229, "row_count": 6000000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3577530366, "histo_col_type": "", "name": "__auto__", "null_count": 1271186043, "row_count": 6000000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3487059014, "histo_col_type": "", "name": "__auto__", "null_count": 5759594099, "row_count": 6000000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1164927213, "histo_buckets": [{"distinct_range": 0, "num_eq": 6946851036844727452, "num_range": 0, "upper_bound": "1203018995"}, {"distinct_range": 26.994621862485392, "num_eq": 50000000, "num_range": 50, "upper_bound": "1674162601"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 1747556436, "row_count": 6000000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4334649750, "histo_col_type": "", "name": "__auto__", "null_count": 1347791219, "row_count": 6000000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1009144221, "histo_col_type": "", "name": "__auto__", "null_count": 297431620, "row_count": 6000000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5586404407, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": "1086556529"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 3480190036, "row_count": 6000000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1212658600, "histo_buckets": [{"distinct_range": 0, "num_eq": 2787787636957296953, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 50000, "upper_bound": "16:30:17.8674+07:16:00"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "08:38:37.687024-03:58:00"}, {"distinct_range": 0, "num_eq": 5573484372105707971, "num_range": 7083479176253886479, "upper_bound": "12:57:56.486467-02:21:00"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 50, "upper_bound": "19:02:43.853362+00:22:00"}, {"distinct_range": 0, "num_eq": 4956640019171757557, "num_range": 700000000, "upper_bound": "22:44:31.301885+04:01:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 544148599, "row_count": 6000000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 69225, "histo_col_type": "", "name": "__auto__", "null_count": 83653, "row_count": 90000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 77309, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "982542693"}, {"distinct_range": 0, "num_eq": 0, "num_range": 7203964131855680552, "upper_bound": "1015696257"}, {"distinct_range": 0, "num_eq": 6550623297911608031, "num_range": 0, "upper_bound": "2255093996"}, {"distinct_range": 0, "num_eq": 3428440015642743854, "num_range": 50, "upper_bound": "2299655695"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 1944102752940969688, "upper_bound": "2860367260"}, {"distinct_range": 9000000, "num_eq": 70000000000, "num_range": 9000000, "upper_bound": "3797434845"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 0, "row_count": 90000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 82110, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 43241, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000, "num_range": 0, "upper_bound": "0"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 9870, "row_count": 90000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23582, "histo_col_type": "", "name": "__auto__", "null_count": 9549, "row_count": 90000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 63331, "histo_col_type": "", "name": "__auto__", "null_count": 76083, "row_count": 90000}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 52613, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 88368, "histo_buckets": [{"distinct_range": 0, "num_eq": 90, "num_range": 0, "upper_bound": "912251031"}, {"distinct_range": 0, "num_eq": 0, "num_range": 6000000, "upper_bound": "2220693790"}, {"distinct_range": 0, "num_eq": 3635381813914586077, "num_range": 100000, "upper_bound": "2294333332"}, {"distinct_range": 3210235403038671000, "num_eq": 70, "num_range": 9005989528399080169, "upper_bound": "2932079903"}, {"distinct_range": 8000000000, "num_eq": 2717693125581119107, "num_range": 8000000000, "upper_bound": "3251550270"}, {"distinct_range": 4000, "num_eq": 8005197482774352457, "num_range": 4000, "upper_bound": "3509029253"}, {"distinct_range": 94548.09941903017, "num_eq": 6364324082190794083, "num_range": 700000, "upper_bound": "4144032966"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 30495, "row_count": 90000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20360, "histo_col_type": "", "name": "__auto__", "null_count": 71397, "row_count": 90000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19558, "histo_col_type": "", "name": "__auto__", "null_count": 53090, "row_count": 90000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44342, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "-0.9903762340545654"}, {"distinct_range": 0, "num_eq": 100, "num_range": 20000000, "upper_bound": "1.401298464324817e-45"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 3424862889876786235, "upper_bound": "0.005768716800957918"}, {"distinct_range": 6226653535345496000, "num_eq": 7000, "num_range": 6226653535345495761, "upper_bound": "0.518553614616394"}, {"distinct_range": 4000, "num_eq": 70000000, "num_range": 4000, "upper_bound": "+Inf"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 90000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 82155, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20209, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14043, "histo_col_type": "BIT(16)", "name": "__auto__", "null_count": 0, "row_count": 90000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 50737, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42786, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "25.171.190.38/22"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 90000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36496, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30848, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 8290908207347807000, "num_eq": 30000000000, "num_range": 8290908207347807701, "upper_bound": "\\\\x0f42ed717012db"}, {"distinct_range": 369943305.5600943, "num_eq": 50, "num_range": 900000000, "upper_bound": "\\\\x148bd53849"}, {"distinct_range": 273024.56282669073, "num_eq": 2533004194820156426, "num_range": 4000000, "upper_bound": "\\\\x8c197328e94d9f"}, {"distinct_range": 6617884215807697000, "num_eq": 938409639466168374, "num_range": 6617884215807696769, "upper_bound": "\\\\xa1f9a9"}, {"distinct_range": 3879814586411021000, "num_eq": 8849462292670089212, "num_range": 7464979619247061832, "upper_bound": "\\\\xacd4c82407e32c"}, {"distinct_range": 0, "num_eq": 0, "num_range": 3000000, "upper_bound": "\\\\xca"}, {"distinct_range": 75374774.3149393, "num_eq": 30, "num_range": 100000000, "upper_bound": "\\\\xcc14ad7d5c"}, {"distinct_range": 7520741791615907000, "num_eq": 600000, "num_range": 7520741791615907003, "upper_bound": "\\\\xf405"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 90000}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "infinity" does not exist (SQLSTATE 42703)
        test_1        |     --- FAIL: TestCompare/postgres (0.90s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.2:5432: connect: connection refused
        test_1        | FAIL
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2702233&tab=buildLog) on [master@bf9744bad5a416a4b06907f0f3dd42896f7342f3](https://github.com/cockroachdb/cockroach/commits/bf9744bad5a416a4b06907f0f3dd42896f7342f3):

```
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        test_1        | --- FAIL: TestCompare (1.54s)
        test_1        |     --- FAIL: TestCompare/mutators (0.94s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 INTERVAL NULL, col1_1 REGPROCEDURE, col1_2 TIMETZ, col1_3 TIME NOT NULL, col1_4 INET NULL, col1_5 TIMESTAMPTZ, col1_6 TIMESTAMP, col1_7 BIT(6) NOT NULL, col1_8 DATE NOT NULL, col1_9 TIME[] NULL, PRIMARY KEY (col1_8), INDEX (col1_7 DESC, col1_6 DESC, col1_0, col1_8, col1_5 DESC, col1_2 DESC, col1_1, col1_3 ASC) WHERE table1.col1_5 != '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ, UNIQUE (col1_2 ASC, col1_8 ASC, col1_1, col1_6, col1_5 ASC, col1_3, col1_4, col1_7 ASC, col1_0 ASC) STORING (col1_9) WHERE ((((table1.col1_2 = '00:00:00+15:59:00':::TIMETZ) OR (table1.col1_5 = '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table1.col1_8 <= '4714-11-24 BC':::DATE)) AND (table1.col1_3 >= '24:00:00':::TIME)) OR (table1.col1_6 <= '0001-01-01 00:00:00':::TIMESTAMP), UNIQUE (col1_2 DESC, col1_1 DESC, col1_4 ASC, col1_6 DESC, col1_3, col1_8 DESC, col1_5 DESC, col1_0) WHERE (table1.col1_3 < '00:00:00':::TIME) OR (table1.col1_5 != '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ), FAMILY (col1_1, col1_8), FAMILY (col1_7, col1_4, col1_9, col1_6, col1_0, col1_3), FAMILY (col1_5, col1_2));
        test_1        |             CREATE TABLE table2 (col1_8 DATE NOT NULL, col2_1 REGPROCEDURE NOT NULL, col2_2 BIT(45) NOT NULL, col2_3 STRING NOT NULL, col2_4 REGPROC, col2_5 REGTYPE, col2_6 TIMETZ, col2_7 CHAR, col2_8 BOOL NOT NULL, col2_9 REGTYPE NOT NULL, col2_10 INET NOT NULL, col2_11 TIMETZ NOT NULL, col2_12 TIMESTAMP NOT NULL, PRIMARY KEY (col1_8, col2_4 ASC), INDEX (col2_12 ASC, col2_6, col2_5 DESC, col2_3 DESC, col2_8 DESC) STORING (col2_1, col2_2, col2_7, col2_11), UNIQUE (col2_9 DESC, col2_11 ASC, col2_6 DESC, col2_2 DESC, col2_12 ASC, col2_10 ASC, col2_7 ASC, col2_3, col2_4 ASC, col2_1, col2_5 DESC, col2_8 DESC) WHERE table2.col2_7 < 'X':::STRING, INDEX (col1_8 ASC, col2_7 ASC, col2_10, col2_3, col2_4 DESC, col2_12, col2_6 DESC, col2_2, col2_11 ASC, col2_8, col2_5, col2_9, col2_1 ASC) WHERE (table2.col2_11 < '24:00:00-15:59:00') AND (table2.col2_6 <= '24:00:00-15:59:00'), UNIQUE (col2_10 DESC, col2_5 DESC, col2_11, col2_3 DESC, col2_4 ASC, col2_7 DESC, col1_8, col2_6 ASC) WHERE ((((((table2.col2_6 != '00:00:00+15:59:00':::TIMETZ) AND (table2.col2_11 = '24:00:00-15:59:00':::TIMETZ)) OR (NOT table2.col2_8)) AND (table2.col1_8 >= 'infinity':::DATE)) AND (table2.col2_12 < '294276-12-31 23:59:59.999999':::TIMESTAMP)) OR (table2.col2_7 <= '':::STRING)) AND (table2.col2_3 != 'X':::STRING), UNIQUE (col2_10, col2_5 ASC, col2_12 ASC, col1_8 ASC, col2_1, col2_2, col2_4 DESC) STORING (col2_6, col2_8, col2_11) WHERE ((((((table2.col2_12 <= '-2000-01-01 00:00:00':::TIMESTAMP) OR (table2.col2_3 >= e'\'':::STRING)) AND (table2.col2_11 < '00:00:00+15:59:00':::TIMETZ)) OR (table2.col2_7 <= e'\U00002603':::STRING)) AND table2.col2_8) AND (table2.col2_6 < '00:00:00+15:59:00':::TIMETZ)) AND (table2.col1_8 >= 'infinity':::DATE), UNIQUE (col2_11, col2_6 DESC, col2_1, col2_8 ASC, col2_10 ASC, col2_2 DESC, col2_9, col2_7 DESC, col1_8, col2_4, col2_5 DESC), INDEX (col2_12, col2_1 DESC, col2_5, col2_7 DESC, col1_8, col2_4 DESC, col2_3 DESC, col2_2, col2_8 ASC, col2_9 DESC, col2_10), INDEX (col1_8, col2_6 ASC, col2_10 DESC, col2_3 ASC, col2_7 DESC, col2_9 DESC, col2_4 ASC, col2_8, col2_1) STORING (col2_2, col2_5, col2_11, col2_12), UNIQUE (col2_8, col1_8 ASC, col2_3 ASC, col2_10, col2_7 ASC, col2_11, col2_1, col2_12, col2_9 DESC, col2_6 ASC, col2_2 DESC, col2_5 ASC) WHERE ((((((table2.col2_3 >= 'X') AND (table2.col2_12 > '3000-01-01 00:00:00')) OR (table2.col2_7 < e'\x00')) AND (table2.col2_11 <= '24:00:00-15:59:00')) AND table2.col2_8) AND (table2.col1_8 >= 'infinity')) OR (table2.col2_6 <= '24:00:00-15:59:00'), UNIQUE (col1_8, col2_3) STORING (col2_2, col2_5, col2_6, col2_7, col2_9) WHERE (((table2.col2_8 OR (table2.col1_8 = 'infinity')) OR (table2.col2_6 > '24:00:00-15:59:00')) AND (table2.col2_12 != '294276-12-31 23:59:59.999999')) AND (table2.col2_7 <= ''), FAMILY (col2_10), FAMILY (col2_11), FAMILY (col2_3, col2_2), FAMILY (col2_1, col2_9, col1_8, col2_6), FAMILY (col2_12), FAMILY (col2_4, col2_8, col2_7), FAMILY (col2_5)) INTERLEAVE IN PARENT table1 (col1_8);
        test_1        |             CREATE TABLE table3 (col3_0 CHAR NOT NULL, col3_1 FLOAT8 NULL, col3_2 VARCHAR, col3_3 BIT(33) NOT NULL, col3_4 BOX2D NOT NULL, col3_5 BYTES NOT NULL, col3_6 REGPROC NULL, col3_7 DATE NOT NULL, col3_8 STRING, col3_9 FLOAT4 NULL, INDEX (col3_3, col3_5, col3_7 ASC) STORING (col3_2, col3_4, col3_6), UNIQUE (col3_9 DESC) STORING (col3_0, col3_2, col3_4, col3_5, col3_6) WHERE ((((((table3.col3_7 >= 'infinity':::DATE) OR (table3.col3_8 >= '"':::STRING)) OR (table3.col3_2 > e'\x00':::STRING)) OR (table3.col3_0 <= e'\'':::STRING)) OR (table3.col3_5 >= e'\\x':::BYTES)) OR (table3.col3_1 >= 'NaN':::FLOAT8)) OR (table3.col3_9 = 0.0:::FLOAT8), UNIQUE (col3_6) WHERE ((((((table3.col3_5 < '\x27') AND (table3.col3_7 >= '-infinity')) AND (table3.col3_0 <= e'\U00002603')) AND (table3.col3_2 = e'\x00')) OR (table3.col3_1 < 1.0)) AND (table3.col3_8 <= e'\U00002603')) OR (table3.col3_9 < -1.0), FAMILY (col3_2), FAMILY (col3_7, col3_1, col3_0), FAMILY (col3_9), FAMILY (col3_4), FAMILY (col3_6), FAMILY (col3_8), FAMILY (col3_5, col3_3));
        test_1        |             CREATE TABLE table4 (col1_8 DATE NOT NULL, col4_1 DATE NOT NULL, col4_2 REGTYPE NOT NULL, col4_3 TIMETZ NULL, col4_4 TIMESTAMPTZ NULL, col4_5 DATE NULL, col4_6 REGPROCEDURE NULL, col4_7 INT8, col4_8 INTERVAL NULL, col4_9 TIMESTAMPTZ, col4_10 INT2 NOT NULL, col4_11 NAME NOT NULL, col4_12 STRING, col4_13 REGPROC, col4_14 REGTYPE, col4_15 "char" NULL, PRIMARY KEY (col1_8, col4_1 ASC, col4_4 DESC, col4_8 ASC, col4_7 ASC, col4_12 ASC, col4_15 DESC, col4_11 ASC, col4_13 DESC, col4_6 DESC, col4_14 ASC), UNIQUE (col4_12 ASC, col4_15 ASC, col4_14 DESC, col1_8 ASC, col4_1, col4_7 ASC, col4_3, col4_11, col4_6 DESC, col4_9 ASC, col4_2 DESC, col4_13, col4_4, col4_5 ASC) WHERE (((((table4.col4_1 >= 'infinity') OR (table4.col4_5 > 'infinity')) AND (table4.col4_7 = 32767)) AND (table4.col4_12 > '"')) AND (table4.col1_8 != '1970-01-01')) AND (table4.col4_4 > '3000-01-01 00:00:00+00:00'), UNIQUE (col4_12 DESC, col4_5, col4_10 DESC, col4_3), INDEX (col4_7 ASC, col4_12 ASC, col4_13 DESC, col4_2 ASC, col1_8 DESC, col4_10, col4_5 DESC, col4_9, col4_15 ASC, col4_8) WHERE ((table4.col4_12 = 'X') AND (table4.col4_1 < '5874897-12-31')) OR (table4.col4_11 > e'\U00002603'), UNIQUE (col4_8, col4_11 ASC, col4_2 ASC, col4_12, col4_14, col4_1, col4_9 ASC, col4_10 DESC, col4_3, col4_6 DESC) WHERE (table4.col4_10 > (-1):::INT8) AND (table4.col4_7 > 127:::INT8), UNIQUE (col1_8 ASC, col4_13 DESC) WHERE (table4.col4_7 != 9223372036854775807:::INT8) OR (table4.col4_1 >= '1970-01-01':::DATE), FAMILY (col4_1), FAMILY (col1_8), FAMILY (col4_5), FAMILY (col4_7, col4_13, col4_12), FAMILY (col4_2, col4_3, col4_11), FAMILY (col4_9, col4_14, col4_6), FAMILY (col4_10, col4_15), FAMILY (col4_4, col4_8)) INTERLEAVE IN PARENT table1 (col1_8);
        test_1        |             CREATE TABLE table5 (col1_8 DATE NOT NULL, col2_4 REGPROC, col5_2 INTERVAL NOT NULL, col5_3 CHAR, col5_4 VARCHAR NOT NULL, col5_5 BIT(4) NULL, col5_6 INT4, col5_7 BOX2D, col5_8 BOX2D NOT NULL, col5_9 BIT(18), col5_10 "char" NULL, col5_11 VARCHAR, col5_12 INT8 NOT NULL, col5_13 "char", col5_14 TIME, col5_15 INT8 NOT NULL, PRIMARY KEY (col1_8, col2_4 ASC, col5_11 DESC, col5_5 DESC, col5_12 ASC, col5_3 ASC, col5_9, col5_15 ASC, col5_8 DESC, col5_14 DESC, col5_4 DESC, col5_13, col5_7, col5_10 ASC, col5_2 ASC), UNIQUE (col5_6 ASC, col5_7 DESC, col5_8 ASC, col5_15 ASC) WHERE (((((((((table5.col5_10 != e'\x00':::STRING) AND (table5.col5_12 != (-32768):::INT8)) OR (table5.col5_13 < e'\'':::STRING)) OR (table5.col5_15 = 0:::INT8)) AND (table5.col5_3 > e'\x00':::STRING)) AND (table5.col5_14 = '24:00:00':::TIME)) OR (table5.col5_11 <= e'\U00002603':::STRING)) OR (table5.col1_8 <= '-infinity':::DATE)) AND (table5.col5_4 > '':::STRING)) AND (table5.col5_6 = (-2147483648):::INT8), INDEX (col5_5 DESC, col5_4, col5_8 DESC) WHERE (((((((((table5.col5_3 > e'\x00') OR (table5.col5_6 >= -2147483648)) AND (table5.col5_13 <= '"')) AND (table5.col5_14 >= '24:00:00')) AND (table5.col1_8 = '-infinity')) OR (table5.col5_11 > 'X')) AND (table5.col5_12 <= 9223372036854775807)) OR (table5.col5_10 = e'\'')) AND (table5.col5_15 = 32767)) OR (table5.col5_4 >= e'\''), UNIQUE (col5_7 DESC, col5_10, col5_8, col1_8 ASC) STORING (col5_6) WHERE (((((((((table5.col5_14 >= '24:00:00':::TIME) AND (table5.col5_15 >= 2147483647:::INT8)) OR (table5.col5_12 = (-128):::INT8)) AND (table5.col5_4 > e'\'':::STRING)) OR (table5.col5_13 < '"':::STRING)) OR (table5.col5_10 >= 'X':::STRING)) OR (table5.col5_11 < '"':::STRING)) AND (table5.col5_6 <= 1:::INT8)) OR (table5.col1_8 < '4714-11-24 BC':::DATE)) AND (table5.col5_3 = '':::STRING), FAMILY (col5_15), FAMILY (col5_8), FAMILY (col5_4), FAMILY (col5_11, col1_8), FAMILY (col5_13), FAMILY (col5_6), FAMILY (col5_2, col5_14), FAMILY (col5_9, col5_7, col5_12), FAMILY (col5_3, col5_5, col5_10, col2_4)) INTERLEAVE IN PARENT table2 (col1_8, col2_4);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 783057799778229281, "histo_col_type": "", "name": "__auto__", "null_count": 253970858759872761, "row_count": 7283136478918214741}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5924177186257921488, "histo_col_type": "", "name": "__auto__", "null_count": 4284654549978522553, "row_count": 7283136478918214741}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5793917624117962590, "histo_col_type": "", "name": "__auto__", "null_count": 1352358182612314252, "row_count": 7283136478918214741}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1505921393598125617, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1985-09-01"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 7283136478918214741}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 398462452080319592, "histo_col_type": "", "name": "__auto__", "null_count": 6091193589827638052, "row_count": 7283136478918214741}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2222073163874818877, "histo_col_type": "", "name": "__auto__", "null_count": 5719691954549029931, "row_count": 7283136478918214741}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3693763835556532392, "histo_buckets": [{"distinct_range": 0, "num_eq": 1798347116088995239, "num_range": 0, "upper_bound": "000000"}, {"distinct_range": 786522888070156900, "num_eq": 0, "num_range": 5161849145977478934, "upper_bound": "010110"}, {"distinct_range": 0, "num_eq": 5764638105825175646, "num_range": 300, "upper_bound": "011110"}, {"distinct_range": 3951.573062790999, "num_eq": 90000000, "num_range": 7000, "upper_bound": "111100"}], "histo_col_type": "BIT(6)", "name": "__auto__", "null_count": 0, "row_count": 7283136478918214741}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6125344760640331136, "histo_col_type": "", "name": "__auto__", "null_count": 6921634247059315339, "row_count": 7283136478918214741}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5999106552485690545, "histo_buckets": [{"distinct_range": 0, "num_eq": 5627100668786237593, "num_range": 0, "upper_bound": "09:19:51.99486+10:07:00"}, {"distinct_range": 281011620481084350, "num_eq": 3000000, "num_range": 3259441225191291957, "upper_bound": "09:54:19.243479+07:42:00"}, {"distinct_range": 1002466822335910000, "num_eq": 7254107535370063781, "num_range": 6032822529362174631, "upper_bound": "14:31:07.571141-02:50:00"}, {"distinct_range": 127.17410261592872, "num_eq": 0, "num_range": 400, "upper_bound": "10:09:18.394543-08:05:00"}, {"distinct_range": 50000, "num_eq": 10000000, "num_range": 50000, "upper_bound": "17:29:05.7305-00:56:00"}, {"distinct_range": 0.23398208203339083, "num_eq": 1419879870466838587, "num_range": 10, "upper_bound": "07:53:33.585208-10:44:00"}, {"distinct_range": 700000000, "num_eq": 40, "num_range": 700000000, "upper_bound": "13:08:31.023968-12:57:00"}, {"distinct_range": 0, "num_eq": 7077650842444808348, "num_range": 0, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 6044432139474536183, "row_count": 7283136478918214741}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3544507410291156499, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7283136478918214741}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39324336, "histo_col_type": "", "name": "__auto__", "null_count": 27426552, "row_count": 50000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44209292, "histo_col_type": "", "name": "__auto__", "null_count": 22287713, "row_count": 50000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 47981553, "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 50000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14697425, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000, "num_range": 0, "upper_bound": "16:00:05.860537-07:42:00"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 20000000, "upper_bound": "21:21:07.582353-07:17:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 50000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 956020, "histo_buckets": [{"distinct_range": 0, "num_eq": 253041257432223740, "num_range": 0, "upper_bound": "1985-10-23 12:39:57.000366"}, {"distinct_range": 0, "num_eq": 6829326208474203426, "num_range": 0, "upper_bound": "1993-10-24 01:35:17.000759"}, {"distinct_range": 2979204337738035000, "num_eq": 9000000, "num_range": 2979204337738035194, "upper_bound": "2000-11-09 20:19:40.000157"}, {"distinct_range": 440779815.2674349, "num_eq": 3000, "num_range": 6000000000, "upper_bound": "2013-09-02 16:47:08.000234"}, {"distinct_range": 10476.930964562987, "num_eq": 400000000, "num_range": 20000, "upper_bound": "3000-01-01 00:00:00"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 50000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42959228, "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 50000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 43596053, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36612012, "histo_col_type": "", "name": "__auto__", "null_count": 15308477, "row_count": 50000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3752556, "histo_buckets": [{"distinct_range": 0, "num_eq": 7122831920350136374, "num_range": 0, "upper_bound": "291447114"}, {"distinct_range": 72852.54633500661, "num_eq": 2288094498513240809, "num_range": 700000, "upper_bound": "298496092"}, {"distinct_range": 0, "num_eq": 8299390915029826337, "num_range": 70000, "upper_bound": "2443633146"}, {"distinct_range": 1867234740.6359904, "num_eq": 2469261920597878899, "num_range": 2000000000, "upper_bound": "3442761970"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 50000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20477023, "histo_buckets": [{"distinct_range": 0, "num_eq": 70, "num_range": 0, "upper_bound": "84.70.247.26/14"}, {"distinct_range": 1197834728791880200, "num_eq": 0, "num_range": 2747176591146321487, "upper_bound": "159.80.151.2/27"}, {"distinct_range": 1154690727090632200, "num_eq": 1542049914250192552, "num_range": 1154690727090632126, "upper_bound": "65d8:79a3:5dbc:fbdd:c743:bc95:afa:bb66/24"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 50000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4535294, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 31308956, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45905322, "histo_col_type": "", "name": "__auto__", "null_count": 31452251, "row_count": 50000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2657, "histo_col_type": "", "name": "__auto__", "null_count": 1494, "row_count": 5000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 37, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000, "num_range": 0, "upper_bound": "000000000000000000000000000000000"}, {"distinct_range": 0, "num_eq": 400, "num_range": 0, "upper_bound": "000100101011110101000000000010100"}, {"distinct_range": 40, "num_eq": 80000000, "num_range": 40, "upper_bound": "001001010110011010001011001000001"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 4449316308024740680, "upper_bound": "001011100101111000111111001100011"}, {"distinct_range": 0, "num_eq": 60000000000, "num_range": 8161043393376986307, "upper_bound": "001101101000000001111101010011010"}, {"distinct_range": 1694010982439037700, "num_eq": 500000, "num_range": 2762589981367473768, "upper_bound": "011111111111111111111111111111111"}, {"distinct_range": 540993063734945150, "num_eq": 6675444141879853965, "num_range": 540993063734945122, "upper_bound": "111100100000100110110111001001011"}], "histo_col_type": "BIT(33)", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3264, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 567, "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 4261, "row_count": 5000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4715, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3353, "histo_col_type": "", "name": "__auto__", "null_count": 1547, "row_count": 5000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4980, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 663, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2904, "histo_col_type": "", "name": "__auto__", "null_count": 3431, "row_count": 5000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3465, "histo_col_type": "", "name": "__auto__", "null_count": 4091, "row_count": 5000}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 348795, "histo_col_type": "STRING", "name": "__auto__", "null_count": 193146, "row_count": 600000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 498398, "histo_col_type": "", "name": "__auto__", "null_count": 65940, "row_count": 600000}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 380022, "histo_col_type": "", "name": "__auto__", "null_count": 439700, "row_count": 600000}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 149493, "histo_col_type": "", "name": "__auto__", "null_count": 99425, "row_count": 600000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 359987, "histo_buckets": [{"distinct_range": 0, "num_eq": 81020585780969160, "num_range": 0, "upper_bound": "1997-02-16"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 127395, "histo_col_type": "", "name": "__auto__", "null_count": 211114, "row_count": 600000}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 241862, "histo_col_type": "", "name": "__auto__", "null_count": 281759, "row_count": 600000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 557099, "histo_col_type": "", "name": "__auto__", "null_count": 508047, "row_count": 600000}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 435315, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 258051, "histo_col_type": "", "name": "__auto__", "null_count": 58169, "row_count": 600000}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 417592, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 136094, "histo_col_type": "", "name": "__auto__", "null_count": 445769, "row_count": 600000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 169852, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000000, "num_range": 0, "upper_bound": "-4547228508782594699"}, {"distinct_range": 0, "num_eq": 6316061956959912723, "num_range": 4527946716019548171, "upper_bound": "1063074379428272182"}, {"distinct_range": 0, "num_eq": 0, "num_range": 1000, "upper_bound": "2567462672892947069"}, {"distinct_range": 30, "num_eq": 0, "num_range": 30, "upper_bound": "4590948769281314905"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 550434, "row_count": 600000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 318996, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "-72 years -9 mons -248 days -01:41:45.311774"}, {"distinct_range": 737030933426912100, "num_eq": 7256811857594871979, "num_range": 737030933426912067, "upper_bound": "-71 years -7 mons -621 days -13:49:40.412597"}, {"distinct_range": 1008355847026861700, "num_eq": 4000, "num_range": 1527096096269817184, "upper_bound": "-54 years -10 mons -345 days -04:18:36.506769"}, {"distinct_range": 4488136638395919400, "num_eq": 50000000, "num_range": 7627320034639157630, "upper_bound": "-5 years -5 mons -95 days -08:12:19.337664"}, {"distinct_range": 74355229252.84755, "num_eq": 0, "num_range": 80000000000, "upper_bound": "3 years 7 mons 153 days 10:39:01.49746"}, {"distinct_range": 0, "num_eq": 6666432654131614537, "num_range": 5315558635282513794, "upper_bound": "4 years 1 mon 647 days 15:55:38.012634"}, {"distinct_range": 282516232.6080988, "num_eq": 1000000, "num_range": 4000000000, "upper_bound": "6 years 8 mons 94 days 20:25:08.984823"}, {"distinct_range": 0, "num_eq": 80, "num_range": 30, "upper_bound": "14 years 4 mons 9 days 21:32:18.326102"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 569361, "row_count": 600000}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 330728, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 338018, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 591252601190645603, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "1975-10-07"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 985900451050871130}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 383665618677803329, "histo_col_type": "", "name": "__auto__", "null_count": 662115893789463477, "row_count": 985900451050871130}, {"columns": ["col5_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 379992540960886342, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1010"}], "histo_col_type": "BIT(4)", "name": "__auto__", "null_count": 345485945220610446, "row_count": 985900451050871130}, {"columns": ["col5_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 363285451520076959, "histo_buckets": [{"distinct_range": 0, "num_eq": 8600822990795292811, "num_range": 0, "upper_bound": "-1384242553"}, {"distinct_range": 767983913593023600, "num_eq": 4768782730092546859, "num_range": 1457696203861148391, "upper_bound": "-793343378"}, {"distinct_range": 846868396803754200, "num_eq": 1000000, "num_range": 9160551776029656316, "upper_bound": "-426775520"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 100000, "upper_bound": "366959616"}, {"distinct_range": 208608046.75029624, "num_eq": 7549091621861360320, "num_range": 900000000, "upper_bound": "615198672"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 216685778420137695, "row_count": 985900451050871130}, {"columns": ["col5_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 275451474636335618, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000, "num_range": 0, "upper_bound": "BOX(0.5255135385730576 -0.5940714954058687,1.1161764893169321 0.17606903927532258)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 276958206976864967, "row_count": 985900451050871130}, {"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 371662638386941897, "histo_col_type": "", "name": "__auto__", "null_count": 692119402326458319, "row_count": 985900451050871130}, {"columns": ["col5_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19287050705945692, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 985900451050871130}, {"columns": ["col5_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33252355453143084, "histo_col_type": "", "name": "__auto__", "null_count": 71938021996024083, "row_count": 985900451050871130}, {"columns": ["col5_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 967964470317874069, "histo_col_type": "", "name": "__auto__", "null_count": 968120113595270843, "row_count": 985900451050871130}, {"columns": ["col5_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 389780464219897835, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 985900451050871130}, {"columns": ["col5_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 293214859270870211, "histo_col_type": "", "name": "__auto__", "null_count": 401800470269442975, "row_count": 985900451050871130}, {"columns": ["col5_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 427736991100159670, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 985900451050871130}, {"columns": ["col5_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24167077965614034, "histo_col_type": "", "name": "__auto__", "null_count": 871578539044419747, "row_count": 985900451050871130}, {"columns": ["col5_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 670410811543682813, "histo_col_type": "", "name": "__auto__", "null_count": 48510310946104598, "row_count": 985900451050871130}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 960202009122443515, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 985900451050871130}, {"columns": ["col5_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 760481784854740805, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 985900451050871130}]':::JSONB;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_5) REFERENCES table3 (col3_6);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('wriwrp', 'o', 'fb', 'txnb', 'ck');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('erxtll', 'fsgpm', 'afe', 'cehv', 'axi');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8869146590074563, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000000, "num_range": 0, "upper_bound": "08:20:53.990011+07:07:00"}, {"distinct_range": 32073080095.081318, "num_eq": 60, "num_range": 80000000000, "upper_bound": "13:07:28.135379+07:53:00"}, {"distinct_range": 9000, "num_eq": 50000000, "num_range": 9000, "upper_bound": "09:45:19.331684-01:01:00"}, {"distinct_range": 727888164309409500, "num_eq": 6772068980203212251, "num_range": 727888164309409582, "upper_bound": "05:50:34.054263-12:17:00"}, {"distinct_range": 0, "num_eq": 2847087429813571606, "num_range": 4000000000, "upper_bound": "14:22:55.759972-04:17:00"}, {"distinct_range": 6224991147087715000, "num_eq": 1000000000, "num_range": 6224991147087714869, "upper_bound": "21:41:57.857514+01:28:00"}, {"distinct_range": 6094197619255868000, "num_eq": 30000000, "num_range": 6094197619255868401, "upper_bound": "13:09:35.641696-08:24:00"}, {"distinct_range": 145.29941533066844, "num_eq": 3289283715982662415, "num_range": 200, "upper_bound": "21:38:42.991754-02:12:00"}, {"distinct_range": 6674640264826236000, "num_eq": 641653419517871039, "num_range": 6674640264826236199, "upper_bound": "17:01:01.902869-13:37:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 3232658334566732552, "row_count": 8520558382889753323}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6972164979772997984, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8520558382889753323}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2270487423330048949, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000, "num_range": 0, "upper_bound": "1971-03-24"}, {"distinct_range": 400000, "num_eq": 60000000000, "num_range": 400000, "upper_bound": "1981-12-07"}, {"distinct_range": 2109.201945050159, "num_eq": 8000, "num_range": 4000, "upper_bound": "1987-11-18"}, {"distinct_range": 0, "num_eq": 3430538039295772503, "num_range": 1000, "upper_bound": "1988-11-23"}, {"distinct_range": 66.71202343307577, "num_eq": 40000000000, "num_range": 70, "upper_bound": "1989-12-18"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 500, "upper_bound": "1992-11-03"}, {"distinct_range": 700000, "num_eq": 3782322477985745257, "num_range": 700000, "upper_bound": "1994-06-07"}, {"distinct_range": 793518776746561700, "num_eq": 2942286744003663601, "num_range": 3812230092868376932, "upper_bound": "1995-07-12"}, {"distinct_range": 47358302266.18138, "num_eq": 1390329677324181037, "num_range": 50000000000, "upper_bound": "1996-03-23"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 8520558382889753323}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5364250332664031015, "histo_col_type": "", "name": "__auto__", "null_count": 7606426276136857870, "row_count": 8520558382889753323}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8058974601570508918, "histo_col_type": "", "name": "__auto__", "null_count": 6321397725366583708, "row_count": 8520558382889753323}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 726337921658998552, "histo_col_type": "", "name": "__auto__", "null_count": 437970104334820042, "row_count": 8520558382889753323}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1221814859350392762, "histo_col_type": "", "name": "__auto__", "null_count": 1201072295073886965, "row_count": 8520558382889753323}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 535994839264836948, "histo_buckets": [{"distinct_range": 0, "num_eq": 110098434678658232, "num_range": 0, "upper_bound": "000010"}, {"distinct_range": 1449850986478372600, "num_eq": 70, "num_range": 3963458457153607201, "upper_bound": "010010"}, {"distinct_range": 41462547.190545626, "num_eq": 1713351853475614938, "num_range": 90000000, "upper_bound": "110001"}, {"distinct_range": 7029700897543641000, "num_eq": 7953748183261165004, "num_range": 7029700897543641126, "upper_bound": "111010"}], "histo_col_type": "BIT(6)", "name": "__auto__", "null_count": 0, "row_count": 8520558382889753323}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1582100287605758179, "histo_col_type": "", "name": "__auto__", "null_count": 5748353447131991225, "row_count": 8520558382889753323}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6220134479078359713, "histo_col_type": "", "name": "__auto__", "null_count": 3265797635459952045, "row_count": 8520558382889753323}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11392053, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27195913, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 43651371, "histo_col_type": "", "name": "__auto__", "null_count": 885409, "row_count": 50000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20554829, "histo_col_type": "", "name": "__auto__", "null_count": 14760068, "row_count": 50000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35957293, "histo_col_type": "", "name": "__auto__", "null_count": 23121531, "row_count": 50000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38170389, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 6115280969573407899, "num_range": 70000000, "upper_bound": "1746897702"}, {"distinct_range": 0, "num_eq": 5400325640887654017, "num_range": 5924739960464932705, "upper_bound": "2175759182"}, {"distinct_range": 0, "num_eq": 4000000, "num_range": 0, "upper_bound": "2432449069"}, {"distinct_range": 60, "num_eq": 5000000000, "num_range": 60, "upper_bound": "3920357789"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 50000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4737997, "histo_buckets": [{"distinct_range": 0, "num_eq": 30, "num_range": 0, "upper_bound": "2015-06-24 19:04:27.000515"}, {"distinct_range": 110376383957100460, "num_eq": 90, "num_range": 122087092559184992, "upper_bound": "2029-07-25 15:21:39.000278"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 50000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10657912, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "1970-06-30"}, {"distinct_range": 1000, "num_eq": 7265892609614558529, "num_range": 1000, "upper_bound": "1971-02-27"}, {"distinct_range": 0, "num_eq": 4694230929131889759, "num_range": 3978562660413033724, "upper_bound": "1980-06-10"}, {"distinct_range": 0, "num_eq": 400, "num_range": 5255232482296085743, "upper_bound": "1980-10-13"}, {"distinct_range": 50, "num_eq": 700000000, "num_range": 50, "upper_bound": "1989-07-03"}, {"distinct_range": 3668706839124099600, "num_eq": 2412214790366527454, "num_range": 3668706839124099611, "upper_bound": "1993-03-18"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 50000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30650031, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 49445030, "histo_col_type": "", "name": "__auto__", "null_count": 40665540, "row_count": 50000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42208509, "histo_buckets": [{"distinct_range": 0, "num_eq": 8120835544767608731, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 50000000000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 50000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16980653, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000000, "num_range": 0, "upper_bound": "142.198.190.33/21"}, {"distinct_range": 4501760441386041, "num_eq": 4966458894116432391, "num_range": 192226403120026479, "upper_bound": "18.63.36.248/27"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 3900624730410952215, "upper_bound": "78.42.26.70/28"}, {"distinct_range": 9564573650176320, "num_eq": 3647922370926121044, "num_range": 281801804810849963, "upper_bound": "6010:f4dc:2023:3429:4fde:5bbb:d1b:5976/3"}, {"distinct_range": 0, "num_eq": 2126761449483018156, "num_range": 60000000, "upper_bound": "798d:82f:116e:c616:6362:550d:6404:65d9/89"}, {"distinct_range": 0, "num_eq": 7144349952612501147, "num_range": 800, "upper_bound": "3192:3afe:c766:5ecb:49f4:9c2c:ea5f:d477/125"}, {"distinct_range": 0, "num_eq": 60, "num_range": 90000000000, "upper_bound": "e8ea:80e9:c70c:c9c2:5e4a:ff64:7a0:9737/125"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 50000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7356571, "histo_buckets": [{"distinct_range": 0, "num_eq": 3366438950941284603, "num_range": 0, "upper_bound": "15:36:59.202308+02:25:00"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 0, "upper_bound": "05:25:29.085215-11:03:00"}, {"distinct_range": 500, "num_eq": 10, "num_range": 500, "upper_bound": "06:54:32.132533-12:48:00"}, {"distinct_range": 50444.258327806194, "num_eq": 8730954429241352344, "num_range": 6000000, "upper_bound": "11:53:22.967927-13:37:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 50000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 763, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 279, "histo_buckets": [{"distinct_range": 0, "num_eq": 1263761123889403935, "num_range": 0, "upper_bound": "6464682"}, {"distinct_range": 300000000, "num_eq": 7015057897524853837, "num_range": 300000000, "upper_bound": "1534548372"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 9205491195437980040, "upper_bound": "2254300069"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 255, "row_count": 900}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 515, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 523, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000000, "num_range": 0, "upper_bound": "1.1418839693069458"}, {"distinct_range": 6559661481020028000, "num_eq": 2100561764883791063, "num_range": 6559661481020027678, "upper_bound": "1.3372397422790527"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 202, "row_count": 900}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 570, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 711, "histo_col_type": "", "name": "__auto__", "null_count": 255, "row_count": 900}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38, "histo_col_type": "", "name": "__auto__", "null_count": 816, "row_count": 900}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 519, "histo_buckets": [{"distinct_range": 0, "num_eq": 5740934385341927002, "num_range": 0, "upper_bound": "000000000000000000000000000000000"}, {"distinct_range": 11066.789798790249, "num_eq": 5915335796928096567, "num_range": 70000, "upper_bound": "101010000110001100000001110110101"}, {"distinct_range": 1471.393923896621, "num_eq": 5000, "num_range": 7000, "upper_bound": "110101110001011001111110011010111"}, {"distinct_range": 3028251423620795000, "num_eq": 2263646810809125249, "num_range": 3028251423620795074, "upper_bound": "111001111110101011010010000100111"}], "histo_col_type": "BIT(33)", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 894, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 681, "histo_col_type": "", "name": "__auto__", "null_count": 596, "row_count": 900}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5, "histo_col_type": "", "name": "__auto__", "null_count": 39, "row_count": 100}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000000, "num_range": 0, "upper_bound": "1977-06-15"}, {"distinct_range": 0, "num_eq": 4193063174819479965, "num_range": 535786781383621760, "upper_bound": "1979-11-23"}, {"distinct_range": 0, "num_eq": 50000000, "num_range": 30000000, "upper_bound": "1980-02-18"}, {"distinct_range": 5918940660464320000, "num_eq": 400000000, "num_range": 5918940660464320543, "upper_bound": "1985-03-20"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 20000, "upper_bound": "1990-09-05"}, {"distinct_range": 30000, "num_eq": 50, "num_range": 30000, "upper_bound": "1991-01-21"}, {"distinct_range": 100000, "num_eq": 2761127916559183726, "num_range": 100000, "upper_bound": "1992-03-15"}, {"distinct_range": 1000, "num_eq": 90000000000, "num_range": 1000, "upper_bound": "1992-07-06"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 56, "histo_col_type": "", "name": "__auto__", "null_count": 44, "row_count": 100}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 66, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 92, "histo_buckets": [{"distinct_range": 0, "num_eq": 3009962091987829545, "num_range": 0, "upper_bound": "TzH"}, {"distinct_range": 686.6778880920422, "num_eq": 7586880236591032928, "num_range": 800, "upper_bound": "^jap\\u0004KJ"}, {"distinct_range": 314317963.4506967, "num_eq": 9211872347827933441, "num_range": 900000000, "upper_bound": "\U00002603"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 19, "row_count": 100}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 64, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12, "histo_col_type": "", "name": "__auto__", "null_count": 82, "row_count": 100}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22, "histo_col_type": "", "name": "__auto__", "null_count": 80, "row_count": 100}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 50, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35, "histo_col_type": "", "name": "__auto__", "null_count": 29, "row_count": 100}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 31, "histo_col_type": "", "name": "__auto__", "null_count": 7, "row_count": 100}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 57, "histo_col_type": "", "name": "__auto__", "null_count": 21, "row_count": 100}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 97, "histo_col_type": "", "name": "__auto__", "null_count": 42, "row_count": 100}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 99, "histo_buckets": [{"distinct_range": 0, "num_eq": 3569778568697599721, "num_range": 0, "upper_bound": "-5611833705461404403"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 9000000, "upper_bound": "-1159809781094842940"}, {"distinct_range": 243282249503821980, "num_eq": 9000000, "num_range": 293628625557233813, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 3589226260943279070, "upper_bound": "1554988885609982812"}, {"distinct_range": 0, "num_eq": 0, "num_range": 8089048756223166338, "upper_bound": "1972729884094874418"}, {"distinct_range": 60000000, "num_eq": 10000, "num_range": 60000000, "upper_bound": "2599752294073324640"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 15, "row_count": 100}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 79, "histo_buckets": [{"distinct_range": 0, "num_eq": 20, "num_range": 0, "upper_bound": "63 years 11 mons 368 days 12:42:27.85948"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 77, "row_count": 100}]';
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col5_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8841218, "histo_col_type": "", "name": "__auto__", "null_count": 5836586, "row_count": 20000000}, {"columns": ["col5_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 995772, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col5_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8512088, "histo_col_type": "", "name": "__auto__", "null_count": 16502355, "row_count": 20000000}, {"columns": ["col5_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 986879, "histo_col_type": "", "name": "__auto__", "null_count": 76165, "row_count": 20000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19162418, "histo_buckets": [{"distinct_range": 0, "num_eq": 8601190483526925964, "num_range": 0, "upper_bound": "1970-01-01"}, {"distinct_range": 0, "num_eq": 8819674913847396967, "num_range": 70, "upper_bound": "1970-02-03"}, {"distinct_range": 0, "num_eq": 0, "num_range": 40000000, "upper_bound": "1978-08-30"}, {"distinct_range": 600000, "num_eq": 1894604013829163576, "num_range": 600000, "upper_bound": "1986-08-31"}, {"distinct_range": 1523547106141728300, "num_eq": 1570577958399074885, "num_range": 3090701877063237378, "upper_bound": "1988-01-13"}, {"distinct_range": 4697456957084994000, "num_eq": 6902243502127421385, "num_range": 7598401023351486485, "upper_bound": "1988-05-27"}, {"distinct_range": 1896773.9171219708, "num_eq": 7400198752206235638, "num_range": 5000000, "upper_bound": "1996-01-16"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col5_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3144632, "histo_col_type": "", "name": "__auto__", "null_count": 3178080, "row_count": 20000000}, {"columns": ["col5_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5778935, "histo_col_type": "", "name": "__auto__", "null_count": 15058772, "row_count": 20000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11910173, "histo_col_type": "", "name": "__auto__", "null_count": 1001483, "row_count": 20000000}, {"columns": ["col5_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16655677, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col5_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1933055, "histo_buckets": [{"distinct_range": 0, "num_eq": 6159360541888224685, "num_range": 0, "upper_bound": "127"}, {"distinct_range": 0, "num_eq": 800000, "num_range": 239074216372108500, "upper_bound": "32767"}, {"distinct_range": 0, "num_eq": 0, "num_range": 6000000, "upper_bound": "1937567285"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 13228656, "row_count": 20000000}, {"columns": ["col5_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19247705, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col5_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15555953, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000, "num_range": 0, "upper_bound": "0000"}, {"distinct_range": 409504129639142800, "num_eq": 10000, "num_range": 409504129639142754, "upper_bound": "0101"}], "histo_col_type": "BIT(4)", "name": "__auto__", "null_count": 6725337, "row_count": 20000000}, {"columns": ["col5_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8216732, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000, "num_range": 0, "upper_bound": "BOX(-2.090583551991614 -1.2006330144340271,1.5172152498501927 0.9089565823564512)"}, {"distinct_range": 2953937542901084700, "num_eq": 100, "num_range": 2953937542901084836, "upper_bound": "BOX(-1.7979130049385639 -0.8495862400473315,-0.25037838747166974 -0.794316168706205)"}, {"distinct_range": 23285975.636270035, "num_eq": 9000000000, "num_range": 90000000, "upper_bound": "BOX(-1.0747882848755683 0.4136213260190916,0.10888542483182037 1.5712686073426418)"}, {"distinct_range": 7162345061399795000, "num_eq": 40000000000, "num_range": 7162345061399794933, "upper_bound": "BOX(-0.728783102343389 0.7003877922603923,0.018404462283885925 0.8271670406756755)"}, {"distinct_range": 0, "num_eq": 5877873570824562891, "num_range": 100000, "upper_bound": "BOX(-0.7274636151320552 -1.275091002981704,0.2722507319485859 0.3414844088866047)"}, {"distinct_range": 754089374629338500, "num_eq": 1072396097067602414, "num_range": 5285796096477205659, "upper_bound": "BOX(-0.2482914081432213 -0.45842294833268826,0.44334862830716276 -0.030789532349127846)"}, {"distinct_range": 505380171322744770, "num_eq": 0, "num_range": 538724845208655424, "upper_bound": "BOX(0.22015094532004897 -0.13278053852794613,0.371171363697931 0.8742845620015318)"}, {"distinct_range": 43337.75447054407, "num_eq": 600, "num_range": 50000, "upper_bound": "BOX(0.22551884235399133 -1.5376610898563419,0.7849603243749992 -0.7790102513283936)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 10603502, "row_count": 20000000}, {"columns": ["col5_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10495287, "histo_col_type": "", "name": "__auto__", "null_count": 18579016, "row_count": 20000000}, {"columns": ["col5_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3323875, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16051240, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000}]';
        test_1        |             ALTER TABLE table5 ADD FOREIGN KEY (col1_8, col5_11) REFERENCES table2 (col1_8, col2_3) ON DELETE CASCADE;
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2322212982447704265, "histo_col_type": "", "name": "__auto__", "null_count": 2829119397524619793, "row_count": 4401117408978292240}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 798486225921511488, "histo_col_type": "", "name": "__auto__", "null_count": 831284888676453599, "row_count": 4401117408978292240}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 320664334438171823, "histo_col_type": "", "name": "__auto__", "null_count": 3227497939014648226, "row_count": 4401117408978292240}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3479087994227589825, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "10:32:16.17681+06:25:00"}, {"distinct_range": 83765.24013751281, "num_eq": 8272586457043946934, "num_range": 90000, "upper_bound": "05:31:47.456282+00:54:00"}, {"distinct_range": 900000000, "num_eq": 2200147347776725819, "num_range": 900000000, "upper_bound": "05:20:17.51321-02:14:00"}, {"distinct_range": 10000, "num_eq": 2751430145915347620, "num_range": 10000, "upper_bound": "18:14:59.768759+01:59:00"}, {"distinct_range": 40000, "num_eq": 200000, "num_range": 40000, "upper_bound": "16:07:39.432794-05:57:00"}, {"distinct_range": 2410081427178363400, "num_eq": 0, "num_range": 5059121634257096412, "upper_bound": "21:34:13.522292-02:04:00"}, {"distinct_range": 3784.669440819999, "num_eq": 20000000000, "num_range": 6000, "upper_bound": "16:01:42.390866-09:27:00"}, {"distinct_range": 1871227.099052789, "num_eq": 8000000000, "num_range": 2000000, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 2385740867053648279, "row_count": 4401117408978292240}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2905799530407584165, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4401117408978292240}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1969735987313876531, "histo_col_type": "", "name": "__auto__", "null_count": 390660334067006210, "row_count": 4401117408978292240}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2069598460032077719, "histo_col_type": "", "name": "__auto__", "null_count": 1159266921339616564, "row_count": 4401117408978292240}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3549297427040545779, "histo_col_type": "", "name": "__auto__", "null_count": 2113899135912911686, "row_count": 4401117408978292240}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2982289127709577609, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "000100"}, {"distinct_range": 8670184393522476000, "num_eq": 2063174570813110879, "num_range": 8670184393522476164, "upper_bound": "000101"}, {"distinct_range": 6433693.165906773, "num_eq": 5851750640772747971, "num_range": 60000000, "upper_bound": "010000"}, {"distinct_range": 0, "num_eq": 7075796036521563372, "num_range": 6340570552055032929, "upper_bound": "010111"}, {"distinct_range": 0, "num_eq": 500000, "num_range": 100000, "upper_bound": "011001"}, {"distinct_range": 7621964929542934000, "num_eq": 2336941736712788969, "num_range": 7621964929542933503, "upper_bound": "100100"}, {"distinct_range": 7946187669766277000, "num_eq": 8798131415956621768, "num_range": 7946187669766276713, "upper_bound": "101001"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 3000000, "upper_bound": "101111"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 10000, "upper_bound": "111100"}], "histo_col_type": "BIT(6)", "name": "__auto__", "null_count": 0, "row_count": 4401117408978292240}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4206174818893724519, "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 4401117408978292240}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6733, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 17946576.884084858, "num_eq": 600, "num_range": 20000000, "upper_bound": "341381049"}, {"distinct_range": 5675040345545582000, "num_eq": 5000000, "num_range": 5675040345545581395, "upper_bound": "1279012588"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 3988019867309387949, "upper_bound": "1960095604"}, {"distinct_range": 0, "num_eq": 100, "num_range": 1509003004420906819, "upper_bound": "3470314749"}, {"distinct_range": 80, "num_eq": 6951842136158556636, "num_range": 80, "upper_bound": "4233333668"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 100000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 58502, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44357, "histo_col_type": "", "name": "__auto__", "null_count": 20361, "row_count": 100000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 94573, "histo_col_type": "", "name": "__auto__", "null_count": 87733, "row_count": 100000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 88877, "histo_col_type": "", "name": "__auto__", "null_count": 87687, "row_count": 100000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 96016, "histo_buckets": [{"distinct_range": 0, "num_eq": 60, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 482398522677552600, "num_eq": 561409258955627645, "n

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2707824&tab=buildLog) on [master@5cfd7e5553a3072a1490d392390dddf968844215](https://github.com/cockroachdb/cockroach/commits/5cfd7e5553a3072a1490d392390dddf968844215):

```
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | 2021-02-23 06:13:35.223 UTC [49] LOG:  received fast shutdown request
        postgres_1    | waiting for server to shut down....2021-02-23 06:13:35.224 UTC [49] LOG:  aborting any active transactions
        postgres_1    | 2021-02-23 06:13:35.226 UTC [49] LOG:  background worker "logical replication launcher" (PID 56) exited with exit code 1
        postgres_1    | 2021-02-23 06:13:35.232 UTC [51] LOG:  shutting down
        test_1        | --- FAIL: TestCompare (1.14s)
        test_1        |     --- FAIL: TestCompare/postgres (0.38s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.76s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 TIMESTAMP[], col1_1 "char", col1_2 FLOAT4, col1_3 CHAR NULL, col1_4 BOOL NOT NULL, col1_5 TIMESTAMPTZ NULL, col1_6 BIT(33) NOT NULL, col1_7 DECIMAL, col1_8 NAME, col1_9 REGCLASS NOT NULL, col1_10 BOX2D NULL, col1_11 DECIMAL NULL, col1_12 INET NULL, col1_13 STRING NULL, col1_14 TIMESTAMPTZ NOT NULL, col1_15 NAME NOT NULL, col1_16 REGPROC, PRIMARY KEY (col1_4 DESC, col1_6 ASC, col1_9 ASC), INDEX (col1_16 ASC, col1_2 ASC, col1_6 DESC, col1_3 DESC, col1_7, col1_1, col1_4 DESC, col1_15 ASC, col1_13 DESC, col1_8, col1_12 ASC, col1_10, col1_5, col1_9 ASC, col1_14 DESC, col1_11 DESC) STORING (col1_0) WHERE (((table1.col1_3 > 'X':::STRING) AND (table1.col1_8 >= e'\x00':::STRING)) OR (table1.col1_5 >= '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND table1.col1_4, UNIQUE (col1_5 ASC, col1_10, col1_9 DESC, col1_4 DESC, col1_6 ASC, col1_11, col1_2, col1_14 ASC) WHERE table1.col1_13 <= e'\x00':::STRING, UNIQUE (col1_12 DESC, col1_4, col1_10 DESC, col1_14 DESC, col1_1 ASC, col1_8 ASC, col1_3 DESC) STORING (col1_5, col1_7, col1_11, col1_15) WHERE ((((((((((NOT table1.col1_4) AND (table1.col1_11 != (-1):::DECIMAL)) OR (table1.col1_3 < e'\U00002603':::STRING)) OR (table1.col1_5 = '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table1.col1_1 < e'\U00002603':::STRING)) AND (table1.col1_7 = (-1):::DECIMAL)) OR (table1.col1_15 < 'X':::STRING)) AND (table1.col1_14 > '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table1.col1_8 >= e'\'':::STRING)) OR (table1.col1_2 > 1.401298464324817e-45:::FLOAT8)) OR (table1.col1_13 < '':::STRING), UNIQUE (col1_1 DESC, col1_16 DESC, col1_9 DESC, col1_6 ASC, col1_8, col1_5 DESC, col1_11 ASC, col1_14 DESC) STORING (col1_0, col1_2, col1_10, col1_13) WHERE ((table1.col1_4 OR (table1.col1_8 = e'\x00')) AND (table1.col1_5 != '-4713-11-24 00:00:00+00:00')) AND (table1.col1_13 = e'\U00002603'), UNIQUE (col1_16, col1_7 DESC, col1_15 ASC, col1_4, col1_11 DESC, col1_1) WHERE ((((((((((table1.col1_1 = e'\'':::STRING) OR (NOT table1.col1_4)) OR (table1.col1_15 <= e'\x00':::STRING)) OR (table1.col1_2 >= '+Inf':::FLOAT8)) OR (table1.col1_5 != '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) OR (table1.col1_14 < '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table1.col1_8 > e'\x00':::STRING)) AND (table1.col1_7 = 0:::DECIMAL)) OR (table1.col1_3 >= '"':::STRING)) AND (table1.col1_11 = 0:::DECIMAL)) OR (table1.col1_13 != e'\'':::STRING), UNIQUE (col1_11, col1_12, col1_6 DESC, col1_8, col1_16 DESC, col1_15 ASC) STORING (col1_0, col1_10, col1_13) WHERE ((((((((((table1.col1_8 > e'\U00002603':::STRING) AND (table1.col1_1 > e'\'':::STRING)) OR (table1.col1_2 = 1.0:::FLOAT8)) AND (table1.col1_7 < '-Infinity':::DECIMAL)) AND (table1.col1_13 >= e'\U00002603':::STRING)) AND (table1.col1_14 < '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (table1.col1_5 < '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table1.col1_15 < e'\'':::STRING)) OR table1.col1_4) OR (table1.col1_3 > e'\'':::STRING)) OR (table1.col1_11 >= (-1.234E+401):::DECIMAL), INDEX (col1_11, col1_8 DESC, col1_6, col1_7, col1_10, col1_13 DESC, col1_12, col1_3 DESC, col1_2, col1_1, col1_14, col1_15, col1_4 DESC, col1_9, col1_16, col1_5 ASC) STORING (col1_0) WHERE (((((((table1.col1_7 = -1.234E+401) OR (table1.col1_11 != Infinity)) AND (table1.col1_15 <= e'\x00')) AND (table1.col1_13 != '"')) OR (table1.col1_2 > 3.4028234663852886e+38)) OR (NOT table1.col1_4)) AND (table1.col1_1 > '')) OR (table1.col1_8 <= e'\U00002603'), UNIQUE (col1_6 ASC, col1_3 ASC, col1_4, col1_5, col1_9 DESC, col1_10 DESC, col1_1 DESC, col1_8 ASC, col1_2, col1_14 DESC, col1_15 ASC, col1_13, col1_12 DESC, col1_16, col1_7 DESC) WHERE ((((((table1.col1_15 > 'X':::STRING) OR (table1.col1_14 <= '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table1.col1_7 <= '-Infinity':::DECIMAL)) OR (table1.col1_13 = e'\'':::STRING)) OR table1.col1_4) AND (table1.col1_3 = e'\U00002603':::STRING)) OR (table1.col1_11 < (-1.234E+401):::DECIMAL), UNIQUE (col1_6) WHERE ((((((((((table1.col1_3 <= '') AND (table1.col1_1 <= '')) AND (table1.col1_13 <= '"')) OR (table1.col1_5 != '-2000-01-01 00:00:00+00:00')) AND (table1.col1_8 = e'\x00')) OR (table1.col1_7 <= -Infinity)) AND (table1.col1_15 > e'\U00002603')) OR (NOT table1.col1_4)) OR (table1.col1_2 = NaN)) OR (table1.col1_11 <= NaN)) AND (table1.col1_14 <= '-4713-11-24 00:00:00+00:00'), FAMILY (col1_2), FAMILY (col1_9, col1_5, col1_8), FAMILY (col1_12), FAMILY (col1_1), FAMILY (col1_11), FAMILY (col1_0), FAMILY (col1_10, col1_6), FAMILY (col1_16), FAMILY (col1_4, col1_3), FAMILY (col1_14, col1_13, col1_15), FAMILY (col1_7));
        test_1        |             CREATE TABLE table2 (col1_4 BOOL NOT NULL, col1_6 BIT(33) NOT NULL, col1_9 REGCLASS NOT NULL, col2_3 OID NULL, col2_4 BIT(20) NULL, col2_5 BOOL NULL, col2_6 VARCHAR NULL, PRIMARY KEY (col1_4 DESC, col1_6 ASC, col1_9 ASC, col2_6 ASC, col2_3 ASC, col2_4 DESC), INDEX (col2_4, col2_5) WHERE (table2.col2_5 AND (NOT table2.col1_4)) AND (table2.col2_6 != e'\x00'), UNIQUE (col1_6 DESC, col1_4 DESC), INDEX (col2_6 DESC, col2_5, col1_4 DESC, col2_4 ASC, col1_9, col2_3 ASC, col1_6 DESC) WHERE (table2.col2_5 AND (table2.col2_6 = e'\x00':::STRING)) OR (NOT table2.col1_4), FAMILY (col2_4), FAMILY (col2_6), FAMILY (col2_5), FAMILY (col1_6), FAMILY (col1_9, col2_3), FAMILY (col1_4)) INTERLEAVE IN PARENT table1 (col1_4, col1_6, col1_9);
        test_1        |             CREATE TABLE table3 (col3_0 BOOL NOT NULL, col3_1 STRING NOT NULL, col3_2 INTERVAL NULL, col3_3 UUID NOT NULL, col3_4 DECIMAL NOT NULL, col3_5 BIT(46) NOT NULL, col3_6 BYTES NOT NULL, col3_7 REGPROCEDURE NOT NULL, col3_8 FLOAT8 NOT NULL, col3_9 REGTYPE NOT NULL, col3_10 CHAR NOT NULL, PRIMARY KEY (col3_1 ASC, col3_5 ASC, col3_9 DESC, col3_10 ASC, col3_7, col3_6 DESC, col3_3, col3_0 ASC, col3_8, col3_4 DESC), UNIQUE (col3_9 ASC, col3_0 ASC) WHERE table3.col3_0 OR (table3.col3_6 != '\x58'), FAMILY (col3_9), FAMILY (col3_1), FAMILY (col3_2), FAMILY (col3_4), FAMILY (col3_7, col3_8, col3_5, col3_3, col3_10), FAMILY (col3_6), FAMILY (col3_0));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2013944479385497847, "histo_buckets": [{"distinct_range": 0, "num_eq": 6093382011881368642, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 2115776071554436029}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2073639129492099170, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2115776071554436029}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1857713348375596599, "histo_buckets": [{"distinct_range": 0, "num_eq": 32086330437877068, "num_range": 0, "upper_bound": "0.0.0.0/0"}, {"distinct_range": 0, "num_eq": 8102873293524797406, "num_range": 7060518835084155694, "upper_bound": "226.37.21.206/5"}, {"distinct_range": 4209934766403362000, "num_eq": 1000000000, "num_range": 4209934766403362016, "upper_bound": "214.127.76.41/10"}, {"distinct_range": 2836727462863761400, "num_eq": 40000, "num_range": 2836727462863761299, "upper_bound": "197.239.77.139/31"}, {"distinct_range": 0, "num_eq": 8334953258311335400, "num_range": 2533515651456419710, "upper_bound": "7db4:a380:b5fc:fbda:5053:f914:519b:84dd/17"}, {"distinct_range": 6347347443042877000, "num_eq": 300, "num_range": 7034988042234331748, "upper_bound": "df0d:897d:185f:976b:3640:bd2f:50d4:3ce0/24"}, {"distinct_range": 27951879.532459635, "num_eq": 600, "num_range": 50000000, "upper_bound": "5e44:500c:f427:1579:c9e:52e1:3e94:f5f4/126"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 1735479042771229877, "row_count": 2115776071554436029}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1729780400224836732, "histo_col_type": "", "name": "__auto__", "null_count": 1738321436611616310, "row_count": 2115776071554436029}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 815416686502068398, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2115776071554436029}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 97076488089540872, "histo_col_type": "", "name": "__auto__", "null_count": 930986909106338518, "row_count": 2115776071554436029}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1490695309811761037, "histo_col_type": "", "name": "__auto__", "null_count": 684315513998914862, "row_count": 2115776071554436029}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1008701190483899529, "histo_buckets": [{"distinct_range": 0, "num_eq": 6684617447677904340, "num_range": 0, "upper_bound": "-4.896246486541077434E+35"}, {"distinct_range": 0, "num_eq": 4000000, "num_range": 1772782076359510637, "upper_bound": "-1.592356261487604309E+26"}, {"distinct_range": 410962165501896640, "num_eq": 4663790097982856041, "num_range": 2187388708142665494, "upper_bound": "-7.187723976820917808E+22"}, {"distinct_range": 3387.7524215345074, "num_eq": 50000000000, "num_range": 4000, "upper_bound": "-6.04350342893798819E+19"}, {"distinct_range": 0, "num_eq": 20000000, "num_range": 4088163558931129295, "upper_bound": "-77769171174647820.26"}, {"distinct_range": 0, "num_eq": 235957166126194919, "num_range": 426885425491287518, "upper_bound": "-1"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 100000000000, "upper_bound": "9.136350670135912802E+21"}, {"distinct_range": 400000, "num_eq": 50000000, "num_range": 400000, "upper_bound": "1.3955397796308027E+31"}, {"distinct_range": 24344393712.816788, "num_eq": 8000, "num_range": 30000000000, "upper_bound": "9.208028366326669241E+34"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 2084509038746392922, "row_count": 2115776071554436029}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 154197389276726075, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2115776071554436029}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1758161313424882729, "histo_col_type": "", "name": "__auto__", "null_count": 780877116276348360, "row_count": 2115776071554436029}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1269610463163183388, "histo_col_type": "", "name": "__auto__", "null_count": 1134004104053201652, "row_count": 2115776071554436029}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2039017189124554195, "histo_buckets": [{"distinct_range": 0, "num_eq": 2553456300077285707, "num_range": 0, "upper_bound": "\\u000f"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 6000, "upper_bound": "v"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 1967338320682021029, "row_count": 2115776071554436029}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1387803240706876983, "histo_col_type": "", "name": "__auto__", "null_count": 1760299455560955729, "row_count": 2115776071554436029}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1567523365074271277, "histo_buckets": [{"distinct_range": 0, "num_eq": 700, "num_range": 0, "upper_bound": "1983-04-30 01:05:07.000179+00:00"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 7648289935427601046, "upper_bound": "2026-11-04 22:34:32.000361+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 1385312371492519531, "row_count": 2115776071554436029}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1013743389760793916, "histo_buckets": [{"distinct_range": 0, "num_eq": 3553561961988138791, "num_range": 0, "upper_bound": "010101110011100100111101100110011"}, {"distinct_range": 1000000000, "num_eq": 80000000, "num_range": 1000000000, "upper_bound": "101110100100100000001011000000110"}, {"distinct_range": 11059.77885908279, "num_eq": 20000000, "num_range": 20000, "upper_bound": "111100100100011101000111010110011"}], "histo_col_type": "BIT(33)", "name": "__auto__", "null_count": 0, "row_count": 2115776071554436029}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2047808475077858229, "histo_col_type": "", "name": "__auto__", "null_count": 853888375998583075, "row_count": 2115776071554436029}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 798265017837204135, "histo_buckets": [{"distinct_range": 0, "num_eq": 2477659492997651856, "num_range": 0, "upper_bound": "87854745"}, {"distinct_range": 0, "num_eq": 900, "num_range": 8296709188004793610, "upper_bound": "498976717"}, {"distinct_range": 5000, "num_eq": 4009287511612520640, "num_range": 5000, "upper_bound": "2921851584"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 1370488706988740182, "row_count": 2115776071554436029}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7831741991666241988, "histo_col_type": "", "name": "__auto__", "null_count": 6399423723064881659, "row_count": 7991300389440540706}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1925876540325453619, "histo_buckets": [{"distinct_range": 0, "num_eq": 6845556989717260109, "num_range": 0, "upper_bound": ""}, {"distinct_range": 3303521968244922000, "num_eq": 8000, "num_range": 3303521968244921982, "upper_bound": "\\u000b\\u001cxms:"}, {"distinct_range": 40000000, "num_eq": 300000, "num_range": 40000000, "upper_bound": "\\u0011H^\\u0008"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 60000000000, "upper_bound": "\\u0013\\u001d&v"}, {"distinct_range": 60000000000, "num_eq": 2514495699716234370, "num_range": 60000000000, "upper_bound": "\\u001b9\\u001d?"}, {"distinct_range": 4460178505275252000, "num_eq": 7000000, "num_range": 4460178505275252242, "upper_bound": "\\u001f"}, {"distinct_range": 2395533459421721600, "num_eq": 7374224477631052140, "num_range": 6437614671575611301, "upper_bound": "K^V^\\u0001\\t)O"}, {"distinct_range": 9166890.630964166, "num_eq": 4439613027187609590, "num_range": 10000000, "upper_bound": "R\\u001cxJA"}, {"distinct_range": 500, "num_eq": 355561290010843612, "num_range": 500, "upper_bound": "\U00002603"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 2018260474133893782, "row_count": 7991300389440540706}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4789775162338863908, "histo_buckets": [{"distinct_range": 0, "num_eq": 8864690923469999577, "num_range": 0, "upper_bound": "false"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 7991300389440540706}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 322932640113823100, "histo_buckets": [{"distinct_range": 0, "num_eq": 80, "num_range": 0, "upper_bound": "000000000000000000000000000000000"}, {"distinct_range": 8444030251345067000, "num_eq": 1338554675527816732, "num_range": 8444030251345067430, "upper_bound": "000000000010011011010001100001110"}, {"distinct_range": 6600897155443369000, "num_eq": 1000, "num_range": 7677940497941577315, "upper_bound": "100001010011010011101101000000110"}, {"distinct_range": 5087310322210275000, "num_eq": 2981731576035366254, "num_range": 5087310322210275052, "upper_bound": "100001100000101000110100000011011"}], "histo_col_type": "BIT(33)", "name": "__auto__", "null_count": 0, "row_count": 7991300389440540706}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3300262547868218396, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7991300389440540706}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7733364657831867314, "histo_col_type": "", "name": "__auto__", "null_count": 6081472628457559774, "row_count": 7991300389440540706}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 107031060774457406, "histo_buckets": [{"distinct_range": 0, "num_eq": 8902603159857852218, "num_range": 0, "upper_bound": "00000000000000000000"}, {"distinct_range": 0, "num_eq": 1351480024855987604, "num_range": 2095794156430432298, "upper_bound": "00010111100101010001"}, {"distinct_range": 0, "num_eq": 3011014241647668460, "num_range": 3076175744807960402, "upper_bound": "00111101011100100000"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 3000000, "upper_bound": "01010101110010110011"}, {"distinct_range": 3629198636015684000, "num_eq": 3692328683472973864, "num_range": 3629198636015684096, "upper_bound": "10000001010101001111"}, {"distinct_range": 5973410985536659000, "num_eq": 8000, "num_range": 5973410985536659083, "upper_bound": "10010100101011010011"}, {"distinct_range": 122450324.5825613, "num_eq": 696357314065829219, "num_range": 300000000, "upper_bound": "10011100110010010110"}, {"distinct_range": 6000000000, "num_eq": 1000, "num_range": 6000000000, "upper_bound": "10101111001100001100"}], "histo_col_type": "BIT(20)", "name": "__auto__", "null_count": 177044140009530031, "row_count": 7991300389440540706}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 185457365, "histo_col_type": "", "name": "__auto__", "null_count": 335123357, "row_count": 9000000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3338786199, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7226052410, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8992589761, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 741346914, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8677472188, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7613374132, "histo_buckets": [{"distinct_range": 0, "num_eq": 6533672092176076394, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 9035739454826990798, "num_range": 0, "upper_bound": "2[\\"\\u0017T6L2\\u0001"}, {"distinct_range": 2014030007912634000, "num_eq": 70, "num_range": 5414070968564500610, "upper_bound": "P\\u001d_q["}, {"distinct_range": 7000, "num_eq": 70000000000, "num_range": 7000, "upper_bound": "``"}, {"distinct_range": 0, "num_eq": 30000000000, "num_range": 7089886842677519073, "upper_bound": "ca"}, {"distinct_range": 0, "num_eq": 6575257480897780015, "num_range": 8111642486015262424, "upper_bound": "}B"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 9000000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6800775001, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1739286221, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2489207881, "histo_buckets": [{"distinct_range": 0, "num_eq": 7576984657912189830, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 1937528834417018628, "num_range": 8237687368308767676, "upper_bound": "599483708"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 90000000, "upper_bound": "958439590"}, {"distinct_range": 3755827380797821000, "num_eq": 4919205036376416146, "num_range": 9001158899742009915, "upper_bound": "2052922836"}, {"distinct_range": 0, "num_eq": 3220370227789434629, "num_range": 0, "upper_bound": "3766065030"}, {"distinct_range": 552198462026012800, "num_eq": 900000000, "num_range": 552198462026012822, "upper_bound": "4062788341"}, {"distinct_range": 573883268322945400, "num_eq": 10, "num_range": 5758067240085810115, "upper_bound": "4147683593"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 9000000000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4576960102, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9000000000}]':::JSONB;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_4) REFERENCES table1 (col1_6) ON DELETE RESTRICT;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('zwmew', 'rzfvw', 'wtbr');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1853384769425589150, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000000, "num_range": 0, "upper_bound": "010110110100111110000000111100110"}], "histo_col_type": "BIT(33)", "name": "__auto__", "null_count": 0, "row_count": 3834425765258027705}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2474267603216081430, "histo_col_type": "", "name": "__auto__", "null_count": 888205510324683527, "row_count": 3834425765258027705}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1334141060787540927, "histo_col_type": "", "name": "__auto__", "null_count": 3770427592424632422, "row_count": 3834425765258027705}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3181806285023493001, "histo_col_type": "", "name": "__auto__", "null_count": 1420377984715529972, "row_count": 3834425765258027705}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 589982481432617126, "histo_col_type": "", "name": "__auto__", "null_count": 769088968680333097, "row_count": 3834425765258027705}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3260885482840048011, "histo_buckets": [{"distinct_range": 0, "num_eq": 2311575484910998780, "num_range": 0, "upper_bound": "1.948866463732243573E+29"}, {"distinct_range": 3961629352006087000, "num_eq": 20000, "num_range": 3961629352006087011, "upper_bound": "2.174706221124933738E+32"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 882702884670352562, "row_count": 3834425765258027705}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2559053379660054236, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3834425765258027705}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 940488349124753363, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000, "num_range": 0, "upper_bound": "1347377791"}, {"distinct_range": 10169306.856862072, "num_eq": 200000000, "num_range": 30000000, "upper_bound": "2326600821"}, {"distinct_range": 0, "num_eq": 5000, "num_range": 6250216184939241308, "upper_bound": "2655585507"}, {"distinct_range": 14503990676.451004, "num_eq": 70, "num_range": 30000000000, "upper_bound": "3161973922"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 500000, "upper_bound": "3871477630"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 1199559890355873342, "row_count": 3834425765258027705}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 509511831973737665, "histo_col_type": "", "name": "__auto__", "null_count": 3204930716076289975, "row_count": 3834425765258027705}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1920939483181738559, "histo_buckets": [{"distinct_range": 0, "num_eq": 7925152553305079762, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 60659901710470980, "num_eq": 6077753424250913545, "num_range": 4891501802431904260, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 3834425765258027705}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1462860390407742992, "histo_col_type": "", "name": "__auto__", "null_count": 1709934506591067392, "row_count": 3834425765258027705}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 816250218620512987, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3834425765258027705}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2803152560441724407, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "\\u0005"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 70000, "upper_bound": "\\u0019"}, {"distinct_range": 0, "num_eq": 1894251408510720209, "num_range": 6745411275847354124, "upper_bound": "%"}, {"distinct_range": 0, "num_eq": 6978296508899499791, "num_range": 5429498038346101883, "upper_bound": "<"}, {"distinct_range": 1254669605048850000, "num_eq": 10000, "num_range": 1254669605048850030, "upper_bound": "T"}, {"distinct_range": 7468969825068780000, "num_eq": 700, "num_range": 7468969825068779922, "upper_bound": "W"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 2187837015061619323, "row_count": 3834425765258027705}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3621266465136989081, "histo_buckets": [{"distinct_range": 0, "num_eq": 116522583389543685, "num_range": 0, "upper_bound": "2029-08-11 12:12:52.00033+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 3448317859976998604, "row_count": 3834425765258027705}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3706749981122467849, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3834425765258027705}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3704404891318049125, "histo_col_type": "INET", "name": "__auto__", "null_count": 3437301021855446584, "row_count": 3834425765258027705}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3149846815913512866, "histo_col_type": "", "name": "__auto__", "null_count": 2248419068508657563, "row_count": 3834425765258027705}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 518537737410238846, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000, "num_range": 0, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 571916168562331763}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 372540366829879122, "histo_buckets": [{"distinct_range": 0, "num_eq": 981584654162214805, "num_range": 0, "upper_bound": "000101101110010001001101011100010"}, {"distinct_range": 7015691694764990000, "num_eq": 700000000, "num_range": 7015691694764990945, "upper_bound": "011001100011100110100100101011111"}, {"distinct_range": 557037972851197500, "num_eq": 30000, "num_range": 1453080820780494265, "upper_bound": "011001110110111110010011100110001"}, {"distinct_range": 4388688492297588700, "num_eq": 60, "num_range": 4388688492297588951, "upper_bound": "011101001110001111110100101010100"}, {"distinct_range": 464060939.4654494, "num_eq": 10000000, "num_range": 5000000000, "upper_bound": "101111110111001010111000100001110"}], "histo_col_type": "BIT(33)", "name": "__auto__", "null_count": 0, "row_count": 571916168562331763}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 456215264568729203, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 571916168562331763}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 275695126234111108, "histo_col_type": "", "name": "__auto__", "null_count": 7741067211940983, "row_count": 571916168562331763}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 241284612547132953, "histo_buckets": [{"distinct_range": 0, "num_eq": 6834443597514304405, "num_range": 0, "upper_bound": "01110110001000001111"}], "histo_col_type": "BIT(20)", "name": "__auto__", "null_count": 420946019802747338, "row_count": 571916168562331763}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 183927882209740570, "histo_col_type": "", "name": "__auto__", "null_count": 492525481338189437, "row_count": 571916168562331763}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 356797145303651595, "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 20194670175596398, "row_count": 571916168562331763}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 4426318607808536000, "num_eq": 1000000000, "num_range": 6606799072248577421, "upper_bound": "2495974582"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "\\u0018\\u0013U"}, {"distinct_range": 4322269224334001700, "num_eq": 1000, "num_range": 4322269224334001897, "upper_bound": "\\""}, {"distinct_range": 229277290.1231634, "num_eq": 2614514472720406089, "num_range": 700000000, "upper_bound": "\'VZ\\u0019oO\\u0014\\u0018W"}, {"distinct_range": 700, "num_eq": 7643971414475841875, "num_range": 700, "upper_bound": "-\\u001a|q\\r},q"}, {"distinct_range": 8000, "num_eq": 40, "num_range": 8000, "upper_bound": "6.RR%"}, {"distinct_range": 8000000000, "num_eq": 6723349779209869614, "num_range": 8000000000, "upper_bound": "M["}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6133983822586351182, "histo_buckets": [{"distinct_range": 0, "num_eq": 5458059676800990481, "num_range": 0, "upper_bound": "1970-11-16 11:41:59.000743+00:00"}, {"distinct_range": 234557985024839200, "num_eq": 700, "num_range": 8992076248605681077, "upper_bound": "1988-06-01 13:05:00.000173+00:00"}, {"distinct_range": 0, "num_eq": 3756822507733787245, "num_range": 600000000, "upper_bound": "1991-11-11 16:00:39.000379+00:00"}, {"distinct_range": 73855594.48172437, "num_eq": 3000000000, "num_range": 100000000, "upper_bound": "2003-04-21 00:39:22.000934+00:00"}, {"distinct_range": 113671133.52207483, "num_eq": 8113900912062668744, "num_range": 300000000, "upper_bound": "2017-12-14 14:46:13.000515+00:00"}, {"distinct_range": 400000, "num_eq": 1554626695069371282, "num_range": 400000, "upper_bound": "2031-05-16 18:25:40.000053+00:00"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 300000, "upper_bound": "294276-12-31 23:59:59.999999+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 5260666325260500703, "row_count": 7930595101102203914}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2518213602672186931, "histo_col_type": "", "name": "__auto__", "null_count": 5651326907573219197, "row_count": 7930595101102203914}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6952435380588301655, "histo_col_type": "", "name": "__auto__", "null_count": 6346496720744284010, "row_count": 7930595101102203914}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3036472591422106760, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 533904.1282912852, "num_eq": 3000000000, "num_range": 700000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 7930595101102203914}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6316519583699695702, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "0.0.0.0/0"}, {"distinct_range": 0, "num_eq": 9076791881414865792, "num_range": 100000, "upper_bound": "79.73.198.9/1"}, {"distinct_range": 10000000000, "num_eq": 60000, "num_range": 10000000000, "upper_bound": "27.248.25.251/2"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 319478410403296672, "upper_bound": "168.49.6.25/8"}, {"distinct_range": 0, "num_eq": 400, "num_range": 9000, "upper_bound": "129.87.34.196/29"}, {"distinct_range": 2776822637881530400, "num_eq": 1702285781320933411, "num_range": 3853749546830902277, "upper_bound": "c495:46df:4080:4f0d:e475:7b89:77c2:bf32/49"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 4144247444843064725, "upper_bound": "9ca:1162:98e0:9428:303d:590a:7a43:9a93/72"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 1804010959732290192, "row_count": 7930595101102203914}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7215432936679215992, "histo_col_type": "", "name": "__auto__", "null_count": 1180068976251512286, "row_count": 7930595101102203914}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3656205223516722447, "histo_col_type": "", "name": "__auto__", "null_count": 4427310252685432057, "row_count": 7930595101102203914}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4837692968585472913, "histo_col_type": "", "name": "__auto__", "null_count": 1534565722067827517, "row_count": 7930595101102203914}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3020353836351940478, "histo_col_type": "", "name": "__auto__", "null_count": 4725246635588758204, "row_count": 7930595101102203914}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7859402802956044715, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7930595101102203914}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2323694814524049475, "histo_buckets": [{"distinct_range": 0, "num_eq": 4793147016088268042, "num_range": 0, "upper_bound": "\\t"}, {"distinct_range": 1000, "num_eq": 1000000000, "num_range": 1000, "upper_bound": "\\u0015"}, {"distinct_range": 0, "num_eq": 8623952451011138306, "num_range": 5941718598085568748, "upper_bound": "\\u0017"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 400000000, "upper_bound": "\'"}, {"distinct_range": 5474827415082059000, "num_eq": 100000, "num_range": 8626167121107218259, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 90000000000, "num_range": 200000000, "upper_bound": "2"}, {"distinct_range": 494771778.1131209, "num_eq": 0, "num_range": 1000000000, "upper_bound": "N"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "f"}, {"distinct_range": 7332797349128066000, "num_eq": 3474894036283017000, "num_range": 7332797349128066209, "upper_bound": "\U00002603"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 7836381980370528620, "row_count": 7930595101102203914}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5765662551217144161, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "010011011111100010111110000010010"}, {"distinct_range": 7079067011013889000, "num_eq": 100000000, "num_range": 7079067011013888915, "upper_bound": "110111101001110111001100101001111"}], "histo_col_type": "BIT(33)", "name": "__auto__", "null_count": 0, "row_count": 7930595101102203914}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2915369522360718514, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7930595101102203914}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7045142453642489520, "histo_buckets": [{"distinct_range": 0, "num_eq": 8947072761283052496, "num_range": 0, "upper_bound": "-Infinity"}, {"distinct_range": 353824652.74824095, "num_eq": 6000000000, "num_range": 700000000, "upper_bound": "-6.043255464940583315E+24"}, {"distinct_range": 2173.3265471637665, "num_eq": 3411660570141128247, "num_range": 4000, "upper_bound": "316866347569033631.2"}, {"distinct_range": 208702477.60556895, "num_eq": 4000000000, "num_range": 500000000, "upper_bound": "770733093483769313.3"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 5845422163497241114, "row_count": 7930595101102203914}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3649751723003239388, "histo_col_type": "", "name": "__auto__", "null_count": 2433677047551815667, "row_count": 7930595101102203914}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2618594946989573436, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7930595101102203914}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5344054696888075150, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "593865606"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 6000000000, "upper_bound": "1093779255"}, {"distinct_range": 0, "num_eq": 1136302656738383008, "num_range": 347106537196169449, "upper_bound": "1392737044"}, {"distinct_range": 0, "num_eq": 60000, "num_range": 100000000, "upper_bound": "2273976508"}, {"distinct_range": 251803201.23155737, "num_eq": 7000000000, "num_range": 1000000000, "upper_bound": "2785711237"}, {"distinct_range": 400, "num_eq": 6000000000, "num_range": 400, "upper_bound": "3260054014"}, {"distinct_range": 3895242483282958000, "num_eq": 7110746871212349596, "num_range": 8909591394264034575, "upper_bound": "3299499527"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 7727426857763155607, "row_count": 7930595101102203914}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2251656804847097878, "histo_buckets": [{"distinct_range": 0, "num_eq": 750386800928151970, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 3448588211093225088, "num_range": 8062873635055039633, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 2497409405466575204}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 440876813583187961, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000, "num_range": 0, "upper_bound": "011100110010100010010101000010111"}, {"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "011101000111011101100001111010001"}, {"distinct_range": 3550581612222294500, "num_eq": 40000000000, "num_range": 5068392681990172999, "upper_bound": "011111111111111111111111111111111"}], "histo_col_type": "BIT(33)", "name": "__auto__", "null_count": 0, "row_count": 2497409405466575204}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1857079491637696670, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2497409405466575204}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 186427246864767037, "histo_col_type": "", "name": "__auto__", "null_count": 1897006304132928905, "row_count": 2497409405466575204}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1175519748895733235, "histo_col_type": "BIT(20)", "name": "__auto__", "null_count": 822098452080134733, "row_count": 2497409405466575204}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 561304055764106121, "histo_col_type": "", "name": "__auto__", "null_count": 1330328487739396573, "row_count": 2497409405466575204}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1368624707640641369, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000000, "num_range": 0, "upper_bound": "d^"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 443065665477336263, "row_count": 2497409405466575204}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 58184817, "histo_col_type": "", "name": "__auto__", "null_count": 56106303, "row_count": 60000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28674297, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 1841251252418490600, "num_eq": 2942034187892747884, "num_range": 5982900560094693560, "upper_bound": "435935369"}, {"distinct_range": 0, "num_eq": 4062072558932592611, "num_range": 3000000000, "upper_bound": "1260633681"}, {"distinct_range": 3721806607822549000, "num_eq": 7000, "num_range": 5998636913571559627, "upper_bound": "1659671189"}, {"distinct_range": 529.6155659084664, "num_eq": 3000, "num_range": 1000, "upper_bound": "2053267027"}, {"distinct_range": 2087674335696332500, "num_eq": 100000, "num_range": 2087674335696332510, "upper_bound": "2895181805"}, {"distinct_range": 5738082654576682000, "num_eq": 5892663239172446818, "num_range": 7323754691380591632, "upper_bound": "3006155855"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 6736210080871487334, "upper_bound": "3608847223"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38611790, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15074879, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36825178, "histo_buckets": [{"distinct_range": 0, "num_eq": 7997535328441236766, "num_range": 0, "upper_bound": "o1\\u001bT"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 40348351, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38145619, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38432311, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3330810, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29649646, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22733744, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "infinity" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        postgres_1    | 2021-02-23 06:13:35.327 UTC [49] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-02-23 06:13:35.434 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-02-23 06:13:35.434 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-02-23 06:13:35.438 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2712401&tab=buildLog) on [master@ec011620c7cf299fdbb898db692b36454defc4a2](https://github.com/cockroachdb/cockroach/commits/ec011620c7cf299fdbb898db692b36454defc4a2):

```
        cockroach2_1  | sql:                 postgresql://root@cockroach2:26257?sslmode=disable
        cockroach2_1  | RPC client flags:    /cockroach/cockroach <client cmd> --host=cockroach2:26257 --insecure
        cockroach2_1  | logs:                /cockroach-data/logs
        cockroach2_1  | temp dir:            /cockroach-data/cockroach-temp482438044
        cockroach2_1  | external I/O path:   /cockroach-data/extern
        cockroach2_1  | store[0]:            path=/cockroach-data
        cockroach2_1  | storage engine:      pebble
        cockroach2_1  | status:              initialized new cluster
        cockroach2_1  | clusterID:           c4be3113-dfa9-4c4e-9a95-8df79fbf3a59
        cockroach2_1  | nodeID:              1
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2021-02-24 06:13:05.363 UTC [48] LOG:  received fast shutdown request
        postgres_1    | .2021-02-24 06:13:05.365 UTC [48] LOG:  aborting any active transactions
        postgres_1    | 2021-02-24 06:13:05.367 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2021-02-24 06:13:05.374 UTC [50] LOG:  shutting down
        postgres_1    | 2021-02-24 06:13:05.474 UTC [48] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-02-24 06:13:05.575 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-02-24 06:13:05.575 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-02-24 06:13:05.579 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-02-24 06:13:05.594 UTC [85] LOG:  database system was shut down at 2021-02-24 06:13:05 UTC
        postgres_1    | 2021-02-24 06:13:05.596 UTC [87] FATAL:  the database system is starting up
        postgres_1    | 2021-02-24 06:13:05.600 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (603.09s)
        test_1        |     --- FAIL: TestCompare/postgres (2.34s)
        test_1        |         compare_test.go:119: pgx conn: FATAL: the database system is starting up (SQLSTATE 57P03)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2716824&tab=buildLog) on [master@c7e088826bc079620dfd3b5ae75d1c15cd9cd16d](https://github.com/cockroachdb/cockroach/commits/c7e088826bc079620dfd3b5ae75d1c15cd9cd16d):

```
        cockroach1_1  | clusterID:           465949ed-5346-4bd4-b5ba-ee3b93165fcc
        cockroach1_1  | nodeID:              1
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2021-02-25 06:12:57.554 UTC [47] LOG:  received fast shutdown request
        postgres_1    | .2021-02-25 06:12:57.555 UTC [47] LOG:  aborting any active transactions
        postgres_1    | 2021-02-25 06:12:57.557 UTC [47] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
        postgres_1    | 2021-02-25 06:12:57.565 UTC [49] LOG:  shutting down
        postgres_1    | 2021-02-25 06:12:57.650 UTC [47] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-02-25 06:12:57.664 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-02-25 06:12:57.664 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-02-25 06:12:57.667 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-02-25 06:12:57.681 UTC [84] LOG:  database system was shut down at 2021-02-25 06:12:57 UTC
        postgres_1    | 2021-02-25 06:12:57.683 UTC [86] FATAL:  the database system is starting up
        postgres_1    | 2021-02-25 06:12:57.686 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (1.62s)
        test_1        |     --- FAIL: TestCompare/postgres (1.14s)
        test_1        |         compare_test.go:119: pgx conn: FATAL: the database system is starting up (SQLSTATE 57P03)
        test_1        |     --- FAIL: TestCompare/mutators (0.48s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 OID NOT NULL, col1_1 REGPROC NOT NULL, col1_2 TIME NOT NULL, col1_3 BYTES NOT NULL, col1_4 INTERVAL NOT NULL, col1_5 INT8 NOT NULL, col1_6 BOX2D NOT NULL, col1_7 UUID NULL, col1_8 FLOAT4 NULL, col1_9 BYTES NOT NULL, col1_10 REGPROC NOT NULL, col1_11 FLOAT4 NULL, col1_12 REGTYPE NOT NULL, col1_13 INT2 NOT NULL, col1_14 FLOAT8, col1_15 BIT(48) NOT NULL, col1_16 REGNAMESPACE, col1_17 BYTES NOT NULL, col1_18 FLOAT8 NOT NULL, PRIMARY KEY (col1_6, col1_5, col1_9 DESC, col1_2 DESC, col1_1, col1_13, col1_12 DESC, col1_10 ASC, col1_0 DESC, col1_15 DESC, col1_4 ASC), INDEX (col1_8 DESC, col1_17) STORING (col1_3, col1_7, col1_14, col1_16) WHERE (((((((table1.col1_14 > 5e-324) OR (table1.col1_17 >= '\xe29883')) OR (table1.col1_18 >= 1.0)) AND (table1.col1_5 >= -2147483648)) OR (table1.col1_9 < '\x00')) OR (table1.col1_8 != 0.0)) OR (table1.col1_11 < 3.4028234663852886e+38)) AND (table1.col1_2 <= '00:00:00'), UNIQUE (col1_7 DESC, col1_2, col1_3 DESC, col1_17 DESC, col1_16 DESC, col1_12 ASC) WHERE (((((((table1.col1_11 != 0.0:::FLOAT8) OR (table1.col1_17 = e'\\x58':::BYTES)) AND (table1.col1_8 != '+Inf':::FLOAT8)) OR (table1.col1_5 > (-32768):::INT8)) AND (table1.col1_2 != '24:00:00':::TIME)) OR (table1.col1_13 <= (-1):::INT8)) AND (table1.col1_18 = '-Inf':::FLOAT8)) OR (table1.col1_14 = 1.401298464324817e-45:::FLOAT8), UNIQUE (col1_6, col1_0, col1_11, col1_1, col1_18 ASC, col1_13 ASC, col1_12 DESC, col1_4 DESC, col1_14 ASC, col1_16, col1_10 ASC, col1_2 ASC, col1_15 DESC, col1_17 DESC, col1_3 DESC) STORING (col1_7) WHERE (((((((((table1.col1_13 < -128) AND (table1.col1_18 < 5e-324)) AND (table1.col1_3 > '\x58')) OR (table1.col1_9 >= '\xff')) AND (table1.col1_11 = 0.0)) OR (table1.col1_2 >= '00:00:00')) OR (table1.col1_14 < 1.401298464324817e-45)) AND (table1.col1_8 = +Inf)) OR (table1.col1_5 != 9223372036854775807)) AND (table1.col1_17 != '\xe29883'), INDEX (col1_14 DESC, col1_7 DESC, col1_4 DESC, col1_12 DESC, col1_1 DESC, col1_17) STORING (col1_3, col1_8, col1_18) WHERE (((((((((table1.col1_2 < '00:00:00':::TIME) OR (table1.col1_5 >= (-128):::INT8)) AND (table1.col1_3 != e'\\xe29883':::BYTES)) AND (table1.col1_11 != (-1.0):::FLOAT8)) OR (table1.col1_8 = (-1.0):::FLOAT8)) AND (table1.col1_9 = e'\\x27':::BYTES)) OR (table1.col1_14 > '-Inf':::FLOAT8)) OR (table1.col1_17 >= e'\\x':::BYTES)) OR (table1.col1_13 >= (-32768):::INT8)) OR (table1.col1_18 != 3.4028234663852886e+38:::FLOAT8), INDEX (col1_9, col1_7 DESC, col1_0 DESC, col1_17, col1_18 DESC, col1_1 ASC, col1_15 DESC, col1_8, col1_6, col1_14, col1_13 DESC, col1_16 ASC, col1_3, col1_5 ASC, col1_2, col1_4 ASC, col1_11, col1_10 DESC, col1_12 DESC) WHERE ((((((((table1.col1_11 > 'NaN':::FLOAT8) OR (table1.col1_2 < '24:00:00':::TIME)) OR (table1.col1_5 >= 2147483647:::INT8)) AND (table1.col1_14 > '+Inf':::FLOAT8)) AND (table1.col1_9 != e'\\xff':::BYTES)) OR (table1.col1_3 > e'\\x00':::BYTES)) AND (table1.col1_17 != e'\\xe29883':::BYTES)) AND (table1.col1_18 = 'NaN':::FLOAT8)) OR (table1.col1_8 >= 0.0:::FLOAT8), INDEX (col1_3 ASC, col1_8 DESC, col1_6, col1_4 DESC, col1_15, col1_0 ASC, col1_13 ASC, col1_10 DESC, col1_7 ASC) STORING (col1_11, col1_17, col1_18) WHERE (((((((((table1.col1_18 >= 1.401298464324817e-45:::FLOAT8) OR (table1.col1_17 <= e'\\x27':::BYTES)) AND (table1.col1_14 > 1.7976931348623157e+308:::FLOAT8)) OR (table1.col1_8 != '-Inf':::FLOAT8)) AND (table1.col1_13 != (-1):::INT8)) OR (table1.col1_9 <= e'\\x58':::BYTES)) OR (table1.col1_11 < '+Inf':::FLOAT8)) AND (table1.col1_5 <= (-2147483648):::INT8)) OR (table1.col1_3 < e'\\x':::BYTES)) OR (table1.col1_2 < '00:00:00':::TIME), UNIQUE (col1_8 DESC) WHERE ((table1.col1_8 <= 1.401298464324817e-45:::FLOAT8) OR (table1.col1_13 >= 32767:::INT8)) AND (table1.col1_2 > '24:00:00':::TIME), INDEX (col1_11 DESC, col1_15), FAMILY (col1_15), FAMILY (col1_14, col1_16, col1_0, col1_8), FAMILY (col1_3, col1_4, col1_1, col1_17, col1_12), FAMILY (col1_13, col1_18), FAMILY (col1_10), FAMILY (col1_9, col1_2, col1_5, col1_6), FAMILY (col1_7, col1_11));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1359304278542407578, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1818337787518245239}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 241466889045375132, "histo_buckets": [{"distinct_range": 0, "num_eq": 3523293580200806867, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 0, "num_eq": 8997190742790342210, "num_range": 2771967329140294004, "upper_bound": "\\\\x4ebad1b9"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 5000000, "upper_bound": "\\\\x7b235178941c9c65"}, {"distinct_range": 8964123347870990000, "num_eq": 800000000, "num_range": 8964123347870989844, "upper_bound": "\\\\x9eba"}, {"distinct_range": 7190478275251573000, "num_eq": 0, "num_range": 8447371343296091935, "upper_bound": "\\\\xa9983da10f5731"}, {"distinct_range": 1796220776644644400, "num_eq": 600, "num_range": 1796220776644644462, "upper_bound": "\\\\xf1b4f9"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 1818337787518245239}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1266964292749365219, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000, "num_range": 0, "upper_bound": "00000000-0000-0000-0000-000000000000"}, {"distinct_range": 0, "num_eq": 2682161395314498567, "num_range": 900, "upper_bound": "113dbeae-77c4-4812-b867-6c81e1d83717"}, {"distinct_range": 0, "num_eq": 50, "num_range": 6000000, "upper_bound": "3fa560f5-a4d2-43f9-892d-cf9026eadb05"}, {"distinct_range": 497120822725616060, "num_eq": 3554079840353422989, "num_range": 891482460167086021, "upper_bound": "5a31dfa0-502e-4b43-ab68-203cb11a6b78"}, {"distinct_range": 0, "num_eq": 6796291762030517846, "num_range": 7357562303024782717, "upper_bound": "5c1cdc71-cc02-4891-b4c6-85878b2aa328"}, {"distinct_range": 125927900.71897304, "num_eq": 6501347314403848503, "num_range": 400000000, "upper_bound": "6391e0c2-5dff-4459-a0a0-f173fe7d9952"}, {"distinct_range": 7114758486886215000, "num_eq": 10000, "num_range": 9092300382176577917, "upper_bound": "8dc772d0-fd52-40f1-8493-3ec19493e057"}, {"distinct_range": 1677419619215456000, "num_eq": 100, "num_range": 1677419619215456003, "upper_bound": "a28289c4-f373-4dd3-91b0-56aec9772898"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 3650994437720509820, "upper_bound": "c981ffb9-121e-4e75-8f95-1df552b2fce6"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 1098974170690234200, "row_count": 1818337787518245239}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 272163727905363956, "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 1443081213451015268, "row_count": 1818337787518245239}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1539504569225509187, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1818337787518245239}, {"columns": ["col1_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9152250560941892, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1818337787518245239}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 132834996575358942, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1818337787518245239}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 392081212611405163, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1818337787518245239}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1061527249825857357, "histo_buckets": [{"distinct_range": 0, "num_eq": 800, "num_range": 0, "upper_bound": "-0.4469807744026184"}, {"distinct_range": 400000000, "num_eq": 2000000000, "num_range": 400000000, "upper_bound": "0.0"}, {"distinct_range": 4000, "num_eq": 5000000, "num_range": 4000, "upper_bound": "0.011357692070305347"}, {"distinct_range": 2325789612047445500, "num_eq": 6702187743567053173, "num_range": 2325789612047445554, "upper_bound": "0.7078315019607544"}, {"distinct_range": 1220239614186644000, "num_eq": 1000000, "num_range": 7009842926839771247, "upper_bound": "0.7187279462814331"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 1593497376266641451, "row_count": 1818337787518245239}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 495460523890091659, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1818337787518245239}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1300279202349479031, "histo_buckets": [{"distinct_range": 0, "num_eq": 8748853329632741438, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 4000000, "upper_bound": "\\\\x4287041b91463e"}, {"distinct_range": 111548.69090634598, "num_eq": 0, "num_range": 900000, "upper_bound": "\\\\x58"}, {"distinct_range": 11549.470620271964, "num_eq": 9071571149712368981, "num_range": 30000, "upper_bound": "\\\\x86426b26ff9bbd09"}, {"distinct_range": 0, "num_eq": 5929190084677093918, "num_range": 4384394509539046572, "upper_bound": "\\\\xb4bf1f289680cca843"}, {"distinct_range": 0, "num_eq": 2342268549378602648, "num_range": 807600576893227841, "upper_bound": "\\\\xdbe3b1d6a4"}, {"distinct_range": 0, "num_eq": 1108533719108098608, "num_range": 200000000, "upper_bound": "\\\\xdce64c9d"}, {"distinct_range": 0, "num_eq": 466182521370690693, "num_range": 0, "upper_bound": "\\\\xe89e7ecc7b3398"}, {"distinct_range": 2592296444448389600, "num_eq": 8701285373420886725, "num_range": 6057313581797298093, "upper_bound": "\\\\xedffa303f7"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 1818337787518245239}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1493197515744041680, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1818337787518245239}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 624203624870049235, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1818337787518245239}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1289291053058165365, "histo_col_type": "", "name": "__auto__", "null_count": 258482409869269663, "row_count": 1818337787518245239}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 773138674126875940, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1818337787518245239}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1489196023791948148, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1818337787518245239}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 296143515011070021, "histo_buckets": [{"distinct_range": 0, "num_eq": 2162972972315333633, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 4836571814595682000, "num_eq": 6780567618266593705, "num_range": 4836571814595682056, "upper_bound": "BOX(-1.0360563177944968 -0.6980560863897902,-0.015174505913304853 0.23920325450204621)"}, {"distinct_range": 7551100420135629000, "num_eq": 500000, "num_range": 7551100420135628419, "upper_bound": "BOX(-0.6414349163552275 -1.437071806068353,0.9286454199057698 0.02145507303053354)"}, {"distinct_range": 0, "num_eq": 1253039294465088705, "num_range": 8000000, "upper_bound": "BOX(-0.04249577719053754 0.14035296934600433,1.0556964240771949 0.7903438852010269)"}, {"distinct_range": 4707547973.314749, "num_eq": 2904892387514392300, "num_range": 10000000000, "upper_bound": "BOX(0.04424918601787145 -0.28954513556822103,1.1550172480082257 0.8182366598358914)"}, {"distinct_range": 6172549.155073714, "num_eq": 50, "num_range": 7000000, "upper_bound": "BOX(1.0398603406134121 -0.4433650688484344,1.3211771163789603 0.6972783318915675)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 1818337787518245239}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 894159094353875669, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1818337787518245239}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 403784249981461111, "histo_buckets": [{"distinct_range": 0, "num_eq": 5821175144195297878, "num_range": 0, "upper_bound": "-1.5423367028640775"}, {"distinct_range": 300, "num_eq": 70000, "num_range": 300, "upper_bound": "-0.9676600098539601"}, {"distinct_range": 35.14384997129074, "num_eq": 6471445795060615617, "num_range": 90, "upper_bound": "-0.09320195968936801"}, {"distinct_range": 8000000, "num_eq": 30000000000, "num_range": 8000000, "upper_bound": "0.03566730450982947"}, {"distinct_range": 100000000, "num_eq": 7081954543947834935, "num_range": 100000000, "upper_bound": "0.14797188514108414"}, {"distinct_range": 0, "num_eq": 30000000000, "num_range": 5125150172426329961, "upper_bound": "0.3999837174017451"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 1264915572223088950, "row_count": 1818337787518245239}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('beop', 'zialgi', 'ih', 'vc');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('dwyzmb', 'e');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('vpcsdw', 'dz', 'ppgwg');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000, "num_range": 0, "upper_bound": "11534272-272b-4a1c-88ea-f68e781fe1dc"}, {"distinct_range": 0, "num_eq": 0, "num_range": 4000000000, "upper_bound": "14541849-0872-4e95-b980-7fb0b382c7b8"}, {"distinct_range": 7918008979324730000, "num_eq": 200, "num_range": 8310574753165799293, "upper_bound": "202dcdef-e003-47aa-b5b9-cc8adf6e9b6a"}, {"distinct_range": 0, "num_eq": 4309909341041259496, "num_range": 7807654442044640902, "upper_bound": "40922a46-cfa6-4a40-85e6-d2c59336455a"}, {"distinct_range": 0, "num_eq": 6629913158499962656, "num_range": 200000000, "upper_bound": "5e121359-e832-46b2-b671-583863d136cd"}, {"distinct_range": 0, "num_eq": 100, "num_range": 1000000, "upper_bound": "73aeaa79-e393-4bbf-9084-421fe30de68b"}, {"distinct_range": 2793119935789476000, "num_eq": 5000000, "num_range": 4750356853691048076, "upper_bound": "7c64bf73-2f2f-455e-a1aa-3968d70d1c48"}, {"distinct_range": 19613.565187981207, "num_eq": 5476715074076513437, "num_range": 90000, "upper_bound": "d3b7fba2-d56b-4d45-b1bb-e09d95ae1db5"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 90000000, "upper_bound": "fd9b5510-83df-46c2-a9b3-8225a3440ae8"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 6483660330508669589, "num_range": 0, "upper_bound": "-1.4404234886169434"}, {"distinct_range": 3702635933050243000, "num_eq": 100000, "num_range": 3702635933050242944, "upper_bound": "-0.9841440320014954"}, {"distinct_range": 0, "num_eq": 300000, "num_range": 30000, "upper_bound": "-0.8009100556373596"}, {"distinct_range": 573944558362124500, "num_eq": 50000000000, "num_range": 6395578186367844130, "upper_bound": "-0.32142481207847595"}, {"distinct_range": 327.9871244330158, "num_eq": 2014897261015615935, "num_range": 500, "upper_bound": "-0.2662348449230194"}, {"distinct_range": 0, "num_eq": 600, "num_range": 3350306688970826966, "upper_bound": "0.35190093517303467"}, {"distinct_range": 0, "num_eq": 20, "num_range": 136730395131371123, "upper_bound": "0.5514706969261169"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 700, "num_range": 0, "upper_bound": "BOX(-0.9842052264448498 -0.5738055413119323,-0.1160530080053265 0.04678472817824919)"}, {"distinct_range": 0, "num_eq": 7506576791604940223, "num_range": 8634075491352123199, "upper_bound": "BOX(-0.9168066940143906 -0.2347582553040395,0.2234311473535191 0.5976395346890262)"}, {"distinct_range": 0, "num_eq": 30000, "num_range": 7044670672558350718, "upper_bound": "BOX(-0.6496605544498787 -1.348077321312939,0.39589308566611925 -0.4781404759264829)"}, {"distinct_range": 3150058079782395400, "num_eq": 60000000000, "num_range": 3150058079782395442, "upper_bound": "BOX(-0.5715755588320874 0.1686254262561519,1.153892193945499 1.4239533870725007)"}, {"distinct_range": 100000000, "num_eq": 2369411604312908823, "num_range": 100000000, "upper_bound": "BOX(-0.5058294335754628 0.029905093254409543,0.7607492498567237 0.28001605349833114)"}, {"distinct_range": 7412273763859044000, "num_eq": 20000000000, "num_range": 7412273763859044683, "upper_bound": "BOX(0.3627806305294642 -1.8116721546017651,0.5285557578163017 -0.340228465854409)"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 613030819473533890, "upper_bound": "BOX(0.4495201208568418 0.27420481336903735,2.148899383755138 0.32038658348654603)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 2854830111728947426, "num_range": 0, "upper_bound": "\\\\x5a6155df"}, {"distinct_range": 1356031.3750817575, "num_eq": 300000000, "num_range": 10000000, "upper_bound": "\\\\xe9e4e9f579a7"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 1932596452227307068, "num_range": 0, "upper_bound": "-1.8592458963394165"}, {"distinct_range": 500000, "num_eq": 8771072079690789441, "num_range": 500000, "upper_bound": "-0.6199957728385925"}, {"distinct_range": 0, "num_eq": 4801102966612855520, "num_range": 1000000000, "upper_bound": "-0.03365634009242058"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 2779389088868965927, "upper_bound": "1.401298464324817e-45"}, {"distinct_range": 9510805.196381075, "num_eq": 40, "num_range": 90000000, "upper_bound": "0.5514110922813416"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000, "num_range": 0, "upper_bound": "-0.8280269387817424"}, {"distinct_range": 0, "num_eq": 5853069339644336469, "num_range": 6000, "upper_bound": "-0.23742524367327222"}, {"distinct_range": 0, "num_eq": 40000000000, "num_range": 3000000000, "upper_bound": "-0.140876889466567"}, {"distinct_range": 8836567811181785000, "num_eq": 90000000000, "num_range": 8836567811181784950, "upper_bound": "-0.09586414718129876"}, {"distinct_range": 3000000, "num_eq": 9149985702928530600, "num_range": 3000000, "upper_bound": "0.10574241785461375"}, {"distinct_range": 1174075517795851800, "num_eq": 70000000, "num_range": 1652591104635379140, "upper_bound": "0.6383879416617994"}, {"distinct_range": 285649017215134200, "num_eq": 0, "num_range": 505478123851946064, "upper_bound": "1.7976931348623157e+308"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1002116287913261582, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4680519681040192442}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 177531545455124886, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "\\\\x0e02b0d49b97"}, {"distinct_range": 4077285102749260000, "num_eq": 8000000, "num_range": 4206227278785626378, "upper_bound": "\\\\x2f41d4e3"}, {"distinct_range": 4803024636381995000, "num_eq": 600000000, "num_range": 4803024636381995138, "upper_bound": "\\\\x8b0ccda4b4d9"}, {"distinct_range": 0, "num_eq": 6608918998609095935, "num_range": 2464725804450017510, "upper_bound": "\\\\xa8"}, {"distinct_range": 8228537741135200000, "num_eq": 8343697948193100544, "num_range": 8228537741135200647, "upper_bound": "\\\\xbf22f51b"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 5932349804429477407, "upper_bound": "\\\\xd0b21864"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 4680519681040192442}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3469097376400486874, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4680519681040192442}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1079703411188895643, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4680519681040192442}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3649705213970806132, "histo_col_type": "", "name": "__auto__", "null_count": 3326224706516450813, "row_count": 4680519681040192442}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2306732398812390206, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4680519681040192442}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2106580616580754565, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4680519681040192442}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3001866501197960258, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4680519681040192442}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1834476666836641749, "histo_buckets": [{"distinct_range": 0, "num_eq": 7168993609295757339, "num_range": 0, "upper_bound": "32fb6813-96ad-421d-9c28-2c77121d2f30"}, {"distinct_range": 6340353911719690000, "num_eq": 3916448425444256551, "num_range": 6340353911719690522, "upper_bound": "450dbdca-c513-4c1e-b67d-6fe267ba2eea"}, {"distinct_range": 258018482702632670, "num_eq": 5514591275548371081, "num_range": 4146901623074434387, "upper_bound": "d80e7c46-6e5f-440a-af98-a5289eb33999"}, {"distinct_range": 7.872096743811329, "num_eq": 7508323404848083937, "num_range": 20, "upper_bound": "dad659dd-fc95-4ea7-ba9f-970b4ea95dff"}, {"distinct_range": 6000, "num_eq": 60000, "num_range": 6000, "upper_bound": "fa29cced-1b2d-4979-a368-4caa169328f9"}, {"distinct_range": 5395899033505374000, "num_eq": 4383469939560822747, "num_range": 5395899033505373976, "upper_bound": "ffffffff-ffff-ffff-ffff-ffffffffffff"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 2166631184345676377, "row_count": 4680519681040192442}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2351802981876056908, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4680519681040192442}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1923501269599381550, "histo_buckets": [{"distinct_range": 0, "num_eq": 2819533814973338463, "num_range": 0, "upper_bound": "NaN"}, {"distinct_range": 352742633549519940, "num_eq": 6024059472727382520, "num_range": 1437777240000280184, "upper_bound": "-1.42410938201043"}, {"distinct_range": 724498.7231150186, "num_eq": 2000, "num_range": 1000000, "upper_bound": "-0.3330914607546087"}, {"distinct_range": 100000000, "num_eq": 4401192450083546464, "num_range": 100000000, "upper_bound": "-0.03997558113869737"}, {"distinct_range": 836665807294542600, "num_eq": 5000, "num_range": 836665807294542552, "upper_bound": "0.21014656242421392"}, {"distinct_range": 40000000, "num_eq": 10000, "num_range": 40000000, "upper_bound": "0.3802037089773549"}, {"distinct_range": 60000000000, "num_eq": 600, "num_range": 60000000000, "upper_bound": "0.7687574634798131"}, {"distinct_range": 0, "num_eq": 6287999944337310755, "num_range": 20000, "upper_bound": "0.8756907548020345"}, {"distinct_range": 23037308237.044315, "num_eq": 3967035001793362728, "num_range": 40000000000, "upper_bound": "1.0"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 2607048701945363898, "row_count": 4680519681040192442}, {"columns": ["col1_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 430788941650685213, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4680519681040192442}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1695994990568262279, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4680519681040192442}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3863093307236733025, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4680519681040192442}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3063308175534244784, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 10000000000, "upper_bound": "BOX(-1.5767457887503018 -1.6006507752732828,-0.3479157144244187 0.15968474673403676)"}, {"distinct_range": 8000, "num_eq": 80, "num_range": 8000, "upper_bound": "BOX(-1.2408669174154554 -1.1060804018725185,0.6878944814667558 -0.4472153559248857)"}, {"distinct_range": 6.277628218339605, "num_eq": 4109821880469283248, "num_range": 300, "upper_bound": "BOX(-1.0726953803032309 0.21402503637941772,-0.07707956082009193 1.0783639497896624)"}, {"distinct_range": 0, "num_eq": 60, "num_range": 200000, "upper_bound": "BOX(0.501915092809359 -1.3208015873790662,2.5317782330966736 1.8967113991213274)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 4680519681040192442}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1122460901127042729, "histo_buckets": [{"distinct_range": 0, "num_eq": 3031227732826067572, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 6000, "num_eq": 90000000, "num_range": 6000, "upper_bound": "\\\\x3720bb71a932e03b"}, {"distinct_range": 80, "num_eq": 10000000000, "num_range": 80, "upper_bound": "\\\\x54"}, {"distinct_range": 90481690.5880969, "num_eq": 30000000, "num_range": 700000000, "upper_bound": "\\\\xa52ba6"}, {"distinct_range": 600000, "num_eq": 70000000000, "num_range": 600000, "upper_bound": "\\\\xc5"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 4680519681040192442}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2715074032080250426, "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 4198225548572145612, "row_count": 4680519681040192442}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 166190010819423686, "histo_buckets": [{"distinct_range": 0, "num_eq": 5770959699295442956, "num_range": 0, "upper_bound": "-0.040090516209602356"}, {"distinct_range": 9171260.68748332, "num_eq": 90, "num_range": 10000000, "upper_bound": "2.0025041103363037"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 1352946536152164057, "row_count": 4680519681040192442}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 721257622465832404, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4680519681040192442}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "inf" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2721383&tab=buildLog) on [master@6601d827b814d4e85a1081b03bf2562d8ac2a4ab](https://github.com/cockroachdb/cockroach/commits/6601d827b814d4e85a1081b03bf2562d8ac2a4ab):

```
        postgres_1    | 		(
        postgres_1    | 			SELECT
        postgres_1    | 				tab_43483.crdb_internal_mvcc_timestamp AS col_65686
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table1 AS tab_43483
        postgres_1    | 		)
        postgres_1    | 			AS tab_43484
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_65686 NULLS FIRST
        test_1        | --- FAIL: TestCompare (48.10s)
        test_1        |     --- FAIL: TestCompare/postgres (47.42s)
        test_1        |         compare_test.go:155: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("E9.0"),
        test_1        |             + 	string("E9"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('E'::TEXT::TEXT || 9.0::FLOAT8::FLOAT8)::TEXT AS col_65687) AS tab_43485
        test_1        |             ORDER BY
        test_1        |             	col_65687 NULLS FIRST;
        test_1        |             
        test_1        |             postgres: [same as previous]
        test_1        |             
        test_1        |     --- FAIL: TestCompare/mutators (0.68s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 STRING NOT NULL, col1_1 "char" NOT NULL, col1_2 INT2, col1_3 BOOL NOT NULL, col1_4 TIMESTAMPTZ NOT NULL, col1_5 TIMESTAMPTZ NOT NULL, col1_6 INTERVAL NOT NULL, col1_7 REGPROCEDURE NOT NULL, col1_8 UUID, col1_9 OID NOT NULL, col1_10 TIMESTAMPTZ NOT NULL, col1_11 DECIMAL NOT NULL, col1_12 INT8 NOT NULL, col1_13 BOX2D NOT NULL, col1_14 DECIMAL NOT NULL, col1_15 BYTES NOT NULL, col1_16 BYTES NOT NULL, col1_17 BIT(29) NOT NULL, PRIMARY KEY (col1_10 ASC, col1_13 ASC, col1_11, col1_6, col1_17, col1_12 DESC, col1_16 ASC, col1_1, col1_5 ASC, col1_14 DESC, col1_7 ASC, col1_4, col1_0 ASC, col1_3 DESC), UNIQUE (col1_11 ASC, col1_13, col1_4, col1_17 DESC) STORING (col1_2, col1_15), UNIQUE (col1_7 DESC, col1_8 ASC, col1_5, col1_11 DESC, col1_15 DESC, col1_1, col1_6 ASC) STORING (col1_9) WHERE (((((((((table1.col1_15 >= '\x00') AND (table1.col1_1 >= '"')) AND (table1.col1_11 > NaN)) AND table1.col1_3) OR (table1.col1_4 != '-4713-11-24 00:00:00+00:00')) AND (table1.col1_12 > -1)) AND (table1.col1_0 < '')) OR (table1.col1_5 < '-4713-11-24 00:00:00+00:00')) AND (table1.col1_2 > -1)) OR (table1.col1_16 != '\x00'), FAMILY (col1_3), FAMILY (col1_9), FAMILY (col1_1), FAMILY (col1_14, col1_15), FAMILY (col1_5), FAMILY (col1_17, col1_6, col1_12, col1_16), FAMILY (col1_11, col1_2), FAMILY (col1_7, col1_13, col1_8, col1_4), FAMILY (col1_10, col1_0));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 71, "histo_buckets": [{"distinct_range": 0, "num_eq": 4538526123510633839, "num_range": 0, "upper_bound": "1241472477"}, {"distinct_range": 0, "num_eq": 0, "num_range": 8099524545836783029, "upper_bound": "4275966769"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 318, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 463, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000, "num_range": 0, "upper_bound": "-4713-11-24 00:00:00+00:00"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 700, "upper_bound": "1985-02-03 19:59:06.000537+00:00"}, {"distinct_range": 399046012072135700, "num_eq": 657935349089732121, "num_range": 1761879781539821232, "upper_bound": "1988-08-24 01:07:32.000802+00:00"}, {"distinct_range": 171888233127302600, "num_eq": 90000000, "num_range": 306158012696979871, "upper_bound": "2011-10-29 07:49:11.000999+00:00"}, {"distinct_range": 395256871732537600, "num_eq": 600000000, "num_range": 4206627675415502703, "upper_bound": "2019-03-20 18:40:48.000359+00:00"}, {"distinct_range": 0, "num_eq": 30, "num_range": 7573848981247402425, "upper_bound": "2022-08-08 16:45:01.000507+00:00"}, {"distinct_range": 16683184.21835554, "num_eq": 500000, "num_range": 700000000, "upper_bound": "3000-01-01 00:00:00+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 506, "histo_buckets": [{"distinct_range": 0, "num_eq": 190920219114958651, "num_range": 0, "upper_bound": "-6.029099997771041848E+33"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 2681787514077585420, "upper_bound": "-4.270535436438284181E+27"}, {"distinct_range": 80000, "num_eq": 20, "num_range": 80000, "upper_bound": "-7.546073242379167194E+26"}, {"distinct_range": 604920594474141300, "num_eq": 7000000, "num_range": 3092011865084370855, "upper_bound": "-8710832697.400094877"}, {"distinct_range": 70000, "num_eq": 9000000, "num_range": 70000, "upper_bound": "-5.215126571851816168"}, {"distinct_range": 4726045980831491000, "num_eq": 6515616731370539468, "num_range": 6129707028575607108, "upper_bound": "26201076856210.65516"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 744, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 751, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 368, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 401, "histo_col_type": "", "name": "__auto__", "null_count": 398, "row_count": 900}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 382, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 341, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 592, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 529, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 769, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 404, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 668, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 350, "histo_col_type": "", "name": "__auto__", "null_count": 678, "row_count": 900}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 504, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 769, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('yblhmc', 'nr', 'bmg');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('okf', 'uqrhf');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 403282497648900571, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1684347845239582443}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 672070417918323602, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1684347845239582443}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 922270646591127622, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1684347845239582443}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 219003330484938256, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1684347845239582443}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 333880260812335917, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1684347845239582443}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 378236447539983163, "histo_col_type": "", "name": "__auto__", "null_count": 1591418602053453560, "row_count": 1684347845239582443}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 908745717870973703, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1684347845239582443}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 289652455624200681, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1684347845239582443}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1150493319666307136, "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 1684347845239582443}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1234561042250465673, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1684347845239582443}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27539866167307138, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1684347845239582443}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 840247079540509811, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000, "num_range": 0, "upper_bound": "1464397396"}, {"distinct_range": 60000, "num_eq": 2946414600249735032, "num_range": 60000, "upper_bound": "2518688502"}, {"distinct_range": 297039869007372350, "num_eq": 0, "num_range": 336524117766442605, "upper_bound": "2584785473"}, {"distinct_range": 0, "num_eq": 4000000000, "num_range": 3926219344151405673, "upper_bound": "2921825652"}, {"distinct_range": 6463475292771683000, "num_eq": 8784128268044083330, "num_range": 6463475292771683409, "upper_bound": "3135461533"}, {"distinct_range": 5579484795258241000, "num_eq": 7465627836640744177, "num_range": 5579484795258240980, "upper_bound": "3825059487"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 1684347845239582443}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 798016718357311213, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1684347845239582443}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1011021446476675348, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1684347845239582443}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 550525214074328618, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": "1984-09-30 22:46:52.000946+00:00"}, {"distinct_range": 45230.3585021531, "num_eq": 30000000000, "num_range": 50000, "upper_bound": "1990-09-01 06:11:41.000482+00:00"}, {"distinct_range": 0, "num_eq": 5414464336047099667, "num_range": 200000, "upper_bound": "1991-08-01 04:29:41.000558+00:00"}, {"distinct_range": 20, "num_eq": 2000000, "num_range": 20, "upper_bound": "2008-11-14 23:31:59.000697+00:00"}, {"distinct_range": 3303557276720968000, "num_eq": 5525905958939034354, "num_range": 3303557276720968190, "upper_bound": "2010-11-16 10:00:08.000149+00:00"}, {"distinct_range": 70, "num_eq": 7737133965396112479, "num_range": 70, "upper_bound": "2012-10-20 14:11:48.000943+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 1684347845239582443}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 430854473560661718, "histo_col_type": "", "name": "__auto__", "null_count": 1116058286134860810, "row_count": 1684347845239582443}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 253558023589566816, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1684347845239582443}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 681599386127102814, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1684347845239582443}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 31055124198909920, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 744340609255334879}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 415799155907299670, "histo_col_type": "", "name": "__auto__", "null_count": 679554212169942107, "row_count": 744340609255334879}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 208912157566966582, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 744340609255334879}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 136682976871084476, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 744340609255334879}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 665030377933976484, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 744340609255334879}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 273317900272913268, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 744340609255334879}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 461880266558301429, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 744340609255334879}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 311291815180847190, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 744340609255334879}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 711979165299848160, "histo_col_type": "", "name": "__auto__", "null_count": 202266841159818650, "row_count": 744340609255334879}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 457023647969464303, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 744340609255334879}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 240733428124349175, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 744340609255334879}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 199114759427322346, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 744340609255334879}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 330664613306761391, "histo_buckets": [{"distinct_range": 0, "num_eq": 2989117949168476055, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 2195722477109884183, "num_range": 4368914915840046824, "upper_bound": "1842834865"}, {"distinct_range": 4856489691940454000, "num_eq": 80, "num_range": 9034476587818935911, "upper_bound": "2009894850"}, {"distinct_range": 0, "num_eq": 111719283716978992, "num_range": 20000, "upper_bound": "3421586840"}, {"distinct_range": 189.71705692426218, "num_eq": 8207413946995755748, "num_range": 400, "upper_bound": "3477954383"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 744340609255334879}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 708005361364810854, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 744340609255334879}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 378999896952244362, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 744340609255334879}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 132423027730674229, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "1978-11-20 11:42:32.000452+00:00"}, {"distinct_range": 44673374081611790, "num_eq": 0, "num_range": 356705509268303330, "upper_bound": "2009-03-05 07:05:38.00005+00:00"}, {"distinct_range": 0, "num_eq": 520672848106322611, "num_range": 0, "upper_bound": "2021-06-10 02:50:37.000207+00:00"}, {"distinct_range": 90, "num_eq": 6647750264171078966, "num_range": 90, "upper_bound": "2031-11-27 15:24:45.00065+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 744340609255334879}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 92495721598497365, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000, "num_range": 0, "upper_bound": "-8.593870520362308087E+36"}, {"distinct_range": 2000000, "num_eq": 100000000000, "num_range": 2000000, "upper_bound": "-5.000588442591369773E+28"}, {"distinct_range": 2574.5151657858887, "num_eq": 6000, "num_range": 5000, "upper_bound": "-27978555.48000704263"}, {"distinct_range": 2531828021634711600, "num_eq": 8545989345170069039, "num_range": 2531828021634711762, "upper_bound": "-0.08524523044893762851"}, {"distinct_range": 0, "num_eq": 5751201952672251144, "num_range": 1986375583622808047, "upper_bound": "1"}, {"distinct_range": 90, "num_eq": 500000000, "num_range": 90, "upper_bound": "7773.524582085546084"}, {"distinct_range": 0, "num_eq": 5527855151698918662, "num_range": 1258841372833916125, "upper_bound": "8.077038197104189376E+34"}, {"distinct_range": 423572296527432600, "num_eq": 60, "num_range": 1950359380122930706, "upper_bound": "4.922092579979770217E+36"}, {"distinct_range": 40000000000, "num_eq": 368869151446767161, "num_range": 40000000000, "upper_bound": "Infinity"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 744340609255334879}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 509417164711139116, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 744340609255334879}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "nan" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2725106&tab=buildLog) on [master@9595a158f0233e1c3d86786ec4462dd39c7beb20](https://github.com/cockroachdb/cockroach/commits/9595a158f0233e1c3d86786ec4462dd39c7beb20):

```
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45848121109, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "-6406005409281826253"}, {"distinct_range": 70000, "num_eq": 7794246587246867268, "num_range": 70000, "upper_bound": "-3388830254185697035"}, {"distinct_range": 8857906450222050000, "num_eq": 8776525070490543580, "num_range": 8857906450222050027, "upper_bound": "-1893857188000934411"}, {"distinct_range": 40000, "num_eq": 400000, "num_range": 40000, "upper_bound": "1599955995763222060"}, {"distinct_range": 4363996.195075395, "num_eq": 8403123333653655084, "num_range": 6000000, "upper_bound": "2176841002183626322"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 60000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 57532356850, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53179609311, "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 60000000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 56978977277, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28869021459, "histo_buckets": [{"distinct_range": 0, "num_eq": 1904472486327436765, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 70, "num_eq": 3213998331568243195, "num_range": 70, "upper_bound": "BOX(-0.47686138983102877 -0.9635549348557233,-0.003617948985761288 1.3561224045792963)"}, {"distinct_range": 153551938013219420, "num_eq": 5222564550256139821, "num_range": 3457639313619896382, "upper_bound": "BOX(-0.14042976491070502 0.17787829896231622,0.36186617867105464 0.17805918423854028)"}, {"distinct_range": 300000, "num_eq": 1899653818342475541, "num_range": 300000, "upper_bound": "BOX(0.9394880710626374 1.5489719338228984,1.8603288981860138 2.289915419009981)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 60000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19191326501, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5177834336, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6435194, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1608913857, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35205321450, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32425737287, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33393134082, "histo_buckets": [{"distinct_range": 0, "num_eq": 8991698628513231358, "num_range": 0, "upper_bound": "01:37:42.968947"}, {"distinct_range": 2641477973264868000, "num_eq": 600000000, "num_range": 8803331028322795348, "upper_bound": "03:46:21.794904"}, {"distinct_range": 60000, "num_eq": 7000, "num_range": 60000, "upper_bound": "04:03:10.820531"}, {"distinct_range": 0, "num_eq": 300000000, "num_range": 2637386620759601153, "upper_bound": "13:20:30.72694"}, {"distinct_range": 802455132357660300, "num_eq": 0, "num_range": 7728536836456000674, "upper_bound": "17:23:11.475071"}, {"distinct_range": 0, "num_eq": 2936398584022899469, "num_range": 3876399452542797959, "upper_bound": "20:37:39.241485"}, {"distinct_range": 0, "num_eq": 2425139386904554617, "num_range": 0, "upper_bound": "23:50:41.817626"}, {"distinct_range": 1805419342439927800, "num_eq": 1144311314491229773, "num_range": 1805419342439927840, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 60000000000}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14162197127, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 52571417033, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 55901058775, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39849868665, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000000000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53983621339, "histo_col_type": "", "name": "__auto__", "null_count": 30758728443, "row_count": 60000000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1870749635687823315, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "14:37:38.370399+14:13:00"}, {"distinct_range": 4265496841694601700, "num_eq": 8825584921065753275, "num_range": 4406103690764283066, "upper_bound": "05:58:07.391145+00:30:00"}, {"distinct_range": 2000000, "num_eq": 5000000, "num_range": 2000000, "upper_bound": "02:46:13.570805-06:31:00"}, {"distinct_range": 79.46126710631832, "num_eq": 3000, "num_range": 100, "upper_bound": "06:50:26.228483-07:42:00"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 3000, "upper_bound": "13:11:40.568886-02:41:00"}, {"distinct_range": 1353968616078949400, "num_eq": 40000, "num_range": 1353968616078949360, "upper_bound": "19:21:55.575607-12:57:00"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 4024205229077771504, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 292287831425886645, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1043252564571198285, "histo_col_type": "", "name": "__auto__", "null_count": 938702344910719080, "row_count": 2425193006743394140}, {"columns": ["col2_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 637123100913573868, "histo_buckets": [{"distinct_range": 0, "num_eq": 4360850866546861217, "num_range": 0, "upper_bound": "176.169.133.196/3"}, {"distinct_range": 90000, "num_eq": 90000000, "num_range": 90000, "upper_bound": "207.17.71.172/11"}, {"distinct_range": 3543575580083677000, "num_eq": 6196340182464025251, "num_range": 8360591221844726375, "upper_bound": "16.158.2.220/17"}, {"distinct_range": 50000000, "num_eq": 10000000, "num_range": 50000000, "upper_bound": "a492:a209:be55:fb5b:dc33:6600:a6d4:76e6/2"}, {"distinct_range": 800, "num_eq": 1000000000, "num_range": 800, "upper_bound": "d180:c997:102d:5f1:5d90:5f15:1f2f:aa77/49"}, {"distinct_range": 3110682345178509000, "num_eq": 10000000, "num_range": 3697615649119534861, "upper_bound": "8887:b33b:fc1e:25a0:e720:5acb:7c11:bb79/105"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 1543562214995203156, "row_count": 2425193006743394140}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1053372670004285811, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col2_28"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2158628153852721295, "histo_col_type": "", "name": "__auto__", "null_count": 755365988544130142, "row_count": 2425193006743394140}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2376114407055295980, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1229719994786025978, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col2_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2169811165693572284, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 339569745720457667, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1103775514343222246, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2119064931299320086, "histo_col_type": "", "name": "__auto__", "null_count": 1304367298441479746, "row_count": 2425193006743394140}, {"columns": ["col2_27"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1205094409405604389, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1535686156759320403, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2056779553644894670, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2051310079154686604, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 760937545876553558, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col2_25"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 489461456231885101, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000, "num_range": 0, "upper_bound": "00:31:25.643332"}, {"distinct_range": 6721450010315537000, "num_eq": 60, "num_range": 6721450010315537778, "upper_bound": "01:38:23.26804"}, {"distinct_range": 0, "num_eq": 5057517707196326613, "num_range": 9195435926629959407, "upper_bound": "02:22:28.945367"}, {"distinct_range": 50000, "num_eq": 4375990147855076515, "num_range": 50000, "upper_bound": "08:59:42.289828"}, {"distinct_range": 3000, "num_eq": 2000000, "num_range": 3000, "upper_bound": "09:51:16.875561"}, {"distinct_range": 6147148298256725000, "num_eq": 8687873743939463387, "num_range": 6147148298256725162, "upper_bound": "15:12:10.10314"}, {"distinct_range": 4204207434747452000, "num_eq": 100000000, "num_range": 5453679482168960402, "upper_bound": "16:30:48.313932"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 700000000, "upper_bound": "16:51:35.1072"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 2127840509646541034, "row_count": 2425193006743394140}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2082653200009443242, "histo_col_type": "", "name": "__auto__", "null_count": 1926209563059700226, "row_count": 2425193006743394140}, {"columns": ["col2_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1984514127067278773, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col2_26"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 334608822726726899, "histo_col_type": "", "name": "__auto__", "null_count": 1757240604892954198, "row_count": 2425193006743394140}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1565936938449580933, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1151608051364110957, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1331068509591227916, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col2_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1926351550927972521, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col2_29"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1814136177599772473, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 994906882075521573, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2082774956093512525, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1945196197587590850, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2425193006743394140}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 516354998718994341, "histo_col_type": "", "name": "__auto__", "null_count": 972346524261216834, "row_count": 2425193006743394140}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29062, "histo_col_type": "", "name": "__auto__", "null_count": 174289, "row_count": 200000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 187213, "histo_buckets": [{"distinct_range": 0, "num_eq": 7346357419630786227, "num_range": 0, "upper_bound": "false"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 152058, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 148252, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "BOX(-1.701564157714393 -1.5227445473521377,-0.13338206729398183 0.5274581911869336)"}, {"distinct_range": 8001234451507888000, "num_eq": 30000000, "num_range": 8839478891626341457, "upper_bound": "BOX(-1.36207629804716 1.0833219227370463,1.1049311955966288 1.409909443480628)"}, {"distinct_range": 1056945675732459900, "num_eq": 2000, "num_range": 4594004202482167570, "upper_bound": "BOX(-0.7121387985967884 0.04019716821223998,1.0421920565164617 0.05767667264880161)"}, {"distinct_range": 5000, "num_eq": 6189853687130542373, "num_range": 5000, "upper_bound": "BOX(-0.35720920538946344 -1.6373968366361726,0.06428890758181094 1.7845278488822873)"}, {"distinct_range": 1351599261495606000, "num_eq": 1071384545345519565, "num_range": 4427712063218529455, "upper_bound": "BOX(0.5258417656704282 -0.953960875216171,0.8463798818748296 0.0004874292670047353)"}, {"distinct_range": 5556770711336312000, "num_eq": 672855200263388763, "num_range": 6823439444285949970, "upper_bound": "BOX(0.6164085433596772 -0.7283697425208026,1.2547228294215915 -0.2086378342699379)"}, {"distinct_range": 2563433071018136600, "num_eq": 6056519779641649685, "num_range": 4132855518789233397, "upper_bound": "BOX(1.4100586543802953 -0.3178128969220373,1.732995438190927 0.2631412599651989)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 126409, "row_count": 200000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22910, "histo_col_type": "", "name": "__auto__", "null_count": 120179, "row_count": 200000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 136055, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 148650, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 192461, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000000, "num_range": 0, "upper_bound": "02:24:52.507305"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 3147009533522402179, "upper_bound": "06:19:43.659595"}, {"distinct_range": 0, "num_eq": 5693867485692988759, "num_range": 9146125271095651785, "upper_bound": "08:12:55.914582"}, {"distinct_range": 5000000000, "num_eq": 949814964852641413, "num_range": 5000000000, "upper_bound": "09:23:03.067405"}, {"distinct_range": 0, "num_eq": 900, "num_range": 100000000, "upper_bound": "13:38:59.821566"}, {"distinct_range": 1322457236376127700, "num_eq": 274173421811910954, "num_range": 1532888849597069831, "upper_bound": "17:18:17.066589"}, {"distinct_range": 415.9369997031642, "num_eq": 100000, "num_range": 900, "upper_bound": "20:12:46.289909"}, {"distinct_range": 7000000000, "num_eq": 5352758408775972953, "num_range": 7000000000, "upper_bound": "21:03:23.78035"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 177137, "histo_col_type": "", "name": "__auto__", "null_count": 83622, "row_count": 200000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17452, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 190972, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000, "num_range": 0, "upper_bound": "124.203.169.225/8"}, {"distinct_range": 2000, "num_eq": 867483004399147873, "num_range": 2000, "upper_bound": "230.93.64.68/16"}, {"distinct_range": 2978349302434345500, "num_eq": 300000000, "num_range": 4478642112410693848, "upper_bound": "106.29.159.181/19"}, {"distinct_range": 9000000000, "num_eq": 50000000000, "num_range": 9000000000, "upper_bound": "171.131.107.64/26"}, {"distinct_range": 1000, "num_eq": 200, "num_range": 1000, "upper_bound": "119.82.86.222/28"}, {"distinct_range": 1000000000, "num_eq": 4936504122433991113, "num_range": 1000000000, "upper_bound": "55.228.47.93/31"}, {"distinct_range": 5093873278.671905, "num_eq": 5177512486973994418, "num_range": 6000000000, "upper_bound": "18e2:223c:1744:b62a:d47:82b6:9cf3:84a/44"}, {"distinct_range": 0, "num_eq": 5742750307951221377, "num_range": 4034934550622511716, "upper_bound": "1121:5d97:cb6e:f2a:bb1:2b18:ed33:b999/99"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 10033, "row_count": 200000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67826, "histo_col_type": "", "name": "__auto__", "null_count": 36163, "row_count": 200000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 98897, "histo_buckets": [{"distinct_range": 0, "num_eq": 3631319272597082637, "num_range": 0, "upper_bound": "127.187.73.57/6"}, {"distinct_range": 291119156.5745224, "num_eq": 8381018043432660184, "num_range": 600000000, "upper_bound": "163.190.32.7/24"}, {"distinct_range": 0, "num_eq": 30000000, "num_range": 0, "upper_bound": "d9d7:d5e4:b753:b872:f33b:c460:21db:940f/25"}, {"distinct_range": 300, "num_eq": 500, "num_range": 300, "upper_bound": "c363:9519:a307:3fdf:70ac:6d6a:1e62:859e/37"}, {"distinct_range": 100, "num_eq": 1000000, "num_range": 100, "upper_bound": "b508:d424:b629:2450:3917:5450:4340:15b2/56"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 2700121145280236372, "upper_bound": "ba99:456d:a5d2:6684:bd5b:6faf:166e:480d/102"}, {"distinct_range": 500000, "num_eq": 100000, "num_range": 500000, "upper_bound": "83b3:9206:3169:6dc2:f2b8:7993:1b3f:0/103"}, {"distinct_range": 21.666621197912722, "num_eq": 6144113791993930102, "num_range": 60, "upper_bound": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23226, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "-1.6785328388214111"}, {"distinct_range": 2515278569124573000, "num_eq": 5000, "num_range": 3624180807914218869, "upper_bound": "-0.7053647041320801"}, {"distinct_range": 1639309388163519500, "num_eq": 1474822594572462530, "num_range": 4527822070183331088, "upper_bound": "-0.18913181126117706"}, {"distinct_range": 0, "num_eq": 2880156427118243006, "num_range": 910504083236778898, "upper_bound": "0.29607102274894714"}, {"distinct_range": 188.3805364239156, "num_eq": 500000, "num_range": 400, "upper_bound": "0.32463300228118896"}, {"distinct_range": 5516868238732898000, "num_eq": 4000000000, "num_range": 5516868238732898758, "upper_bound": "0.6302357316017151"}, {"distinct_range": 0, "num_eq": 3595418654558632895, "num_range": 0, "upper_bound": "0.960870087146759"}, {"distinct_range": 0, "num_eq": 7428777218583310386, "num_range": 1730699139069329706, "upper_bound": "1.12998628616333"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 180585, "histo_buckets": [{"distinct_range": 0, "num_eq": 3488253044898218778, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 2808175091232007288, "num_range": 0, "upper_bound": "2144874871"}, {"distinct_range": 2657687163077443600, "num_eq": 5000000, "num_range": 4075186810338988273, "upper_bound": "2203039788"}, {"distinct_range": 0, "num_eq": 0, "num_range": 700, "upper_bound": "2940105977"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 818406426379647591, "upper_bound": "3786648853"}, {"distinct_range": 1525257429621542400, "num_eq": 1636123901351561997, "num_range": 4155284832703844670, "upper_bound": "3962871045"}, {"distinct_range": 0, "num_eq": 7331937766064961954, "num_range": 300000000, "upper_bound": "4258249321"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 200000}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "nan" does not exist (SQLSTATE 42703)
        test_1        |     --- FAIL: TestCompare/postgres (3.33s)
        test_1        |         compare_test.go:129: CREATE TABLE table1 (col1_0 BYTEA, col1_1 "char" NOT NULL, col1_2 FLOAT8 NOT NULL, col1_3 BOX2D NOT NULL, col1_4 INTERVAL NULL, col1_5 INET, col1_6 VARCHAR, col1_7 REGPROC, col1_8 INTERVAL, col1_9 CHAR, col1_10 VARCHAR, PRIMARY KEY (col1_1));
        test_1        |             CREATE INDEX ON table1 (col1_10, col1_4 DESC, col1_6 ASC, col1_7 ASC, col1_5, col1_8 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_7 ASC, col1_2 ASC, col1_0 ASC, col1_5 DESC, col1_9 DESC, col1_8 ASC, col1_3 DESC, col1_4 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_8 ASC, col1_0 ASC, col1_5 ASC, col1_10 ASC) INCLUDE (col1_2, col1_7, col1_9);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_7 ASC, col1_3 ASC);
        test_1        |             CREATE INDEX ON table1 (col1_6 ASC, col1_8 ASC, col1_5, col1_7 DESC, col1_2 ASC, col1_3 DESC, col1_9 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_3 DESC, col1_7 ASC, col1_1 ASC, col1_9 ASC) INCLUDE (col1_5, col1_8, col1_10);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_9, col1_3, col1_1, col1_7 DESC, col1_4 DESC) INCLUDE (col1_0, col1_2, col1_5, col1_6);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_0, col1_3, col1_1 ASC, col1_6 ASC, col1_7, col1_10 ASC, col1_8 DESC, col1_9 ASC) INCLUDE (col1_2);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_5 ASC, col1_7, col1_8, col1_2 ASC, col1_6, col1_3 ASC);
        test_1        |             CREATE TABLE table2 (col2_0 VARCHAR, col2_1 BOX2D NULL, col2_2 OID, col2_3 CHAR NOT NULL, col2_4 INET NOT NULL, col2_5 DECIMAL NOT NULL, col2_6 INT8 NOT NULL, col2_7 TIMESTAMPTZ NOT NULL, col2_8 INT8, col2_9 VARCHAR NOT NULL, col2_10 CHAR NOT NULL, col2_11 BIT(20) NOT NULL);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_4);
        test_1        |             CREATE TABLE table3 (col3_0 INET NOT NULL, col3_1 TIMESTAMP NOT NULL, col3_2 OID NOT NULL, col3_3 BOOL NOT NULL, col3_4 INT8 NOT NULL, col3_5 NAME NOT NULL, col3_6 BYTEA NOT NULL, col3_7 REGPROC NOT NULL, col3_8 VARCHAR NULL, col3_9 INT8 NOT NULL, col3_10 TEXT NOT NULL, col3_11 TIMESTAMPTZ NULL, PRIMARY KEY (col3_5, col3_0, col3_9, col3_6, col3_3, col3_1, col3_4, col3_10));
        test_1        |             CREATE INDEX ON table3 (col3_2 DESC, col3_11, col3_1 DESC, col3_6 DESC, col3_8, col3_10, col3_4, col3_7 DESC, col3_0 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_7, col3_6 DESC, col3_2, col3_5 DESC, col3_1, col3_9);
        test_1        |             CREATE INDEX ON table3 (col3_4 DESC, col3_6, col3_8, col3_2, col3_11 ASC, col3_5 ASC, col3_10, col3_0 ASC, col3_1 ASC);
        test_1        |             CREATE TABLE table4 (col4_0 INT8, col4_1 REGTYPE NOT NULL, col4_2 DECIMAL NOT NULL, col4_3 INT8 NULL, col4_4 FLOAT8, col4_5 TIMESTAMP NULL, col4_6 REGPROCEDURE NOT NULL, col4_7 VARCHAR NOT NULL, PRIMARY KEY (col4_2, col4_7));
        test_1        |             CREATE INDEX ON table4 (col4_4 DESC, col4_1 DESC, col4_7 ASC, col4_5 DESC, col4_6, col4_0);
        test_1        |             CREATE INDEX ON table4 (col4_2, col4_7 DESC, col4_3, col4_6 ASC) INCLUDE (col4_1, col4_4, col4_5);
        test_1        |             CREATE INDEX ON table4 (col4_1 DESC, col4_5 DESC, col4_7 ASC, col4_0 DESC, col4_3, col4_4 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_3 DESC, col4_7 DESC) INCLUDE (col4_0, col4_1, col4_4, col4_6);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_6 ASC, col4_7 DESC, col4_0 ASC, col4_2 ASC, col4_3 ASC) INCLUDE (col4_4);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_3, col4_1, col4_2 DESC, col4_4 DESC, col4_6 ASC, col4_0 DESC, col4_5 DESC, col4_7);
        test_1        |             CREATE INDEX ON table4 (col4_1 DESC);
        test_1        |             CREATE TABLE table5 (col1_1 "char" NOT NULL, col5_1 CHAR NOT NULL, col5_2 INET NULL, col5_3 BIT(12) NOT NULL, col5_4 TIMETZ NULL, col5_5 FLOAT8, col5_6 TIMETZ NOT NULL, col5_7 DECIMAL, col5_8 OID NOT NULL, col5_9 INT8 NULL, col5_10 UUID NULL, col5_11 DECIMAL NOT NULL, col5_12 VARCHAR NULL, col5_13 INT8 NOT NULL, col5_14 BIT(39) NOT NULL, col5_15 BOOL, PRIMARY KEY (col1_1, col5_8, col5_11, col5_3, col5_6));
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_5) REFERENCES table2 (col2_4);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('mupv');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('ip', 'ne', 'eym', 'clllo', 'mecnj', 'hkpxnp');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('cve', 'ruh', 'wtdfh', 'owz');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('cfv', 'wf', 'uzzvzm');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('j', 'hixlk');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2726610&tab=buildLog) on [master@7d1324fa42732f482329a524b0166db8dd7365e6](https://github.com/cockroachdb/cockroach/commits/7d1324fa42732f482329a524b0166db8dd7365e6):

```
        test_1        |             CREATE INDEX ON table4 (col4_7 ASC, col4_5, col4_9, col4_12, col4_11 ASC, col4_6);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_6, col4_11, col4_0 ASC, col4_2 ASC, col4_10 DESC, col4_3 ASC, col4_12 ASC, col4_5 DESC, col4_13, col4_1 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_0 DESC, col4_3 DESC, col4_4 ASC, col4_13 DESC, col4_10 ASC, col4_5 DESC);
        test_1        |             CREATE INDEX ON table4 (col4_2 DESC, col4_11, col4_7 ASC, col4_9, col4_10 DESC, col4_3 DESC, col4_0 ASC, col4_8, col4_5, col4_1 ASC, col4_4 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_9 DESC, col4_12 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_11, col4_1 ASC, col4_3 DESC, col4_12 ASC, col4_7 ASC, col4_4 ASC, col4_6 DESC, col4_8);
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_1) REFERENCES table2 (col1_1);
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_4) REFERENCES table2 (col1_5) ON UPDATE RESTRICT;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('wktgmc', 'vrz', 'skoqgj', 'wmr', 'ktdmym', 'pzdffa');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('hx', 'gakp', 'ufjkno', 'pmg', 'zsx');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('wxhzg');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('jm', 'lcj', 'xsxhkg', 'sqks', 'mn', 'qobs');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (1.03s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 INT4 NULL, col1_1 DATE[] NOT NULL, col1_2 REGCLASS, col1_3 REGCLASS NOT NULL, col1_4 CHAR NULL, col1_5 INTERVAL NOT NULL, col1_6 TIMESTAMP NULL, col1_7 REGNAMESPACE, col1_8 REGPROCEDURE NOT NULL, col1_9 INTERVAL NULL, UNIQUE (col1_9 DESC, col1_4 DESC) STORING (col1_0, col1_1) WHERE ((table1.col1_0 >= -128) OR (table1.col1_4 <= '"')) AND (table1.col1_6 != '-2000-01-01 00:00:00'), UNIQUE (col1_5) STORING (col1_0, col1_1, col1_2, col1_3, col1_4, col1_6, col1_8, col1_9), INDEX (col1_4 ASC, col1_2, col1_8 DESC, col1_0) WHERE ((table1.col1_6 != '294276-12-31 23:59:59.999999') AND (table1.col1_4 = e'\U00002603')) OR (table1.col1_0 <= 0), UNIQUE (col1_8, col1_5 ASC, col1_2 ASC) STORING (col1_1, col1_3), INDEX (col1_4, col1_6 ASC) STORING (col1_0, col1_1, col1_2, col1_3, col1_8, col1_9), UNIQUE (col1_8 DESC, col1_6, col1_7) STORING (col1_0, col1_3), FAMILY (col1_4), FAMILY (col1_8), FAMILY (col1_2, col1_1), FAMILY (col1_9), FAMILY (col1_3), FAMILY (col1_0, col1_5, col1_7, col1_6));
        test_1        |             CREATE TABLE table2 (col2_0 BOOL, col2_1 REGTYPE NOT NULL, col2_2 FLOAT4 NULL, col2_3 BOOL NULL, col2_4 REGPROC NOT NULL, col2_5 INT4 NOT NULL, col2_6 INET NULL, PRIMARY KEY (col2_5 ASC), INDEX (col2_5 DESC, col2_2 DESC) STORING (col2_0, col2_4), UNIQUE (col2_1 ASC, col2_2 DESC, col2_5 DESC) STORING (col2_3, col2_4) WHERE (((table2.col2_5 = 1:::INT8) OR (table2.col2_2 > 0.0:::FLOAT8)) OR (NOT table2.col2_0)) OR table2.col2_3, UNIQUE (col2_0, col2_1 ASC, col2_5 DESC, col2_2 ASC, col2_3, col2_4 ASC, col2_6 DESC) WHERE (NOT table2.col2_3) AND (table2.col2_2 = '-Inf':::FLOAT8), FAMILY (col2_6), FAMILY (col2_0), FAMILY (col2_2), FAMILY (col2_5, col2_3), FAMILY (col2_1, col2_4));
        test_1        |             CREATE TABLE table3 (col2_5 INT4 NOT NULL, col3_1 REGPROC NULL, col3_2 REGPROC, col3_3 REGPROCEDURE NOT NULL, col3_4 REGNAMESPACE NULL, col3_5 "char" NOT NULL, col3_6 INT2 NOT NULL, col3_7 STRING NOT NULL, col3_8 INT4 NOT NULL, col3_9 BOOL NULL, col3_10 BIT(19), col3_11 INT4, col3_12 TIMESTAMP, col3_13 REGPROCEDURE NULL, PRIMARY KEY (col2_5 ASC, col3_13, col3_12 DESC, col3_3 ASC), UNIQUE (col3_1, col3_5, col3_2), FAMILY (col3_7), FAMILY (col3_6, col3_1, col3_11), FAMILY (col3_8, col3_12, col3_3, col3_5, col3_9), FAMILY (col3_10), FAMILY (col3_13, col3_2, col3_4), FAMILY (col2_5)) INTERLEAVE IN PARENT table2 (col2_5);
        test_1        |             CREATE TABLE table4 (col2_5 INT4 NOT NULL, col4_1 INET NULL, col4_2 INTERVAL, col4_3 TIMESTAMPTZ NOT NULL, col4_4 REGTYPE NOT NULL, col4_5 TIMESTAMPTZ NOT NULL, col4_6 NAME, col4_7 FLOAT8, col4_8 INT8 NOT NULL, col4_9 REGTYPE NULL, col4_10 INT4 NULL, col4_11 TIMESTAMPTZ NULL, col4_12 INTERVAL, col4_13 BYTES NULL, col4_14 INET NOT NULL, col4_15 INT4 NOT NULL, PRIMARY KEY (col2_5 ASC, col4_13, col4_14), UNIQUE (col4_2, col4_1, col4_15 DESC, col4_12 DESC, col4_14 ASC, col4_4 ASC, col4_3 DESC, col4_9, col4_11 DESC, col4_6 ASC, col4_8, col4_5 ASC) WHERE (((((table4.col4_5 <= '3000-01-01 00:00:00+00:00') AND (table4.col4_8 <= 9223372036854775807)) OR (table4.col4_11 = '3000-01-01 00:00:00+00:00')) OR (table4.col2_5 < 32767)) OR (table4.col4_13 <= '\x27')) AND (table4.col4_7 != 0.0), UNIQUE (col4_15 DESC, col4_5 ASC, col4_12 ASC, col4_14 DESC, col2_5 ASC, col4_13 DESC, col4_7 DESC, col4_8 ASC, col4_10, col4_2 DESC, col4_1, col4_4 DESC, col4_11, col4_3 ASC) STORING (col4_9) WHERE ((((((table4.col4_10 >= 127) OR (table4.col4_11 = '294276-12-31 23:59:59.999999+00:00')) AND (table4.col4_13 > '\x22')) OR (table4.col4_6 = e'\U00002603')) AND (table4.col4_8 <= 127)) AND (table4.col4_3 > '3000-01-01 00:00:00+00:00')) AND (table4.col4_15 = 1), INDEX (col4_6 ASC) STORING (col4_1, col4_2, col4_7, col4_8, col4_9, col4_10), INDEX (col4_8 DESC, col4_3 DESC, col4_4 ASC, col4_7 DESC, col4_5 DESC) STORING (col4_1, col4_2, col4_9, col4_10, col4_12, col4_15) WHERE ((((((((table4.col4_6 >= '':::STRING) AND (table4.col4_11 != '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table4.col4_8 = (-9223372036854775807):::INT8)) OR (table4.col4_7 <= '-Inf':::FLOAT8)) AND (table4.col4_5 = '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table4.col4_10 >= (-32768):::INT8)) OR (table4.col4_13 < e'\\xff':::BYTES)) OR (table4.col4_3 >= '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) OR (table4.col2_5 < 2147483647:::INT8), UNIQUE (col4_6 ASC, col4_11 DESC, col2_5 DESC, col4_5, col4_7 DESC, col4_2 ASC, col4_4, col4_8 DESC, col4_1, col4_12, col4_15 ASC) STORING (col4_3, col4_10) WHERE (((((((((table4.col4_6 > e'\'') OR (table4.col4_7 >= NaN)) OR (table4.col2_5 > 2147483647)) AND (table4.col4_8 >= -2147483648)) AND (table4.col4_3 < '3000-01-01 00:00:00+00:00')) AND (table4.col4_10 <= 32767)) OR (table4.col4_5 > '3000-01-01 00:00:00+00:00')) OR (table4.col4_15 = 2147483647)) AND (table4.col4_13 >= '\x')) AND (table4.col4_11 > '3000-01-01 00:00:00+00:00'), UNIQUE (col2_5 ASC, col4_8, col4_1 DESC, col4_14, col4_12, col4_10 DESC, col4_6 DESC, col4_15 ASC, col4_11 DESC, col4_7 ASC, col4_9 DESC, col4_4, col4_5 ASC), INDEX (col4_3 DESC, col4_4 DESC, col4_10 ASC, col2_5 ASC, col4_12, col4_14 DESC, col4_6 DESC, col4_5, col4_13 DESC, col4_2 ASC, col4_9 ASC, col4_7 ASC) STORING (col4_8, col4_11, col4_15) WHERE (((((((((table4.col4_6 = '') OR (table4.col4_13 != '\x00')) AND (table4.col4_3 < '0001-01-01 00:00:00+00:00')) OR (table4.col4_11 <= '294276-12-31 23:59:59.999999+00:00')) AND (table4.col4_5 <= '3000-01-01 00:00:00+00:00')) AND (table4.col4_8 > 32767)) OR (table4.col4_10 = 2147483647)) AND (table4.col2_5 > 1)) OR (table4.col4_7 != NaN)) AND (table4.col4_15 = 2147483647), UNIQUE (col4_4 ASC, col4_2 DESC, col4_8, col4_14 DESC, col4_1 DESC, col4_11, col4_5 ASC, col4_3, col4_6, col4_13 ASC, col4_10 DESC, col4_12, col4_7, col2_5 DESC, col4_9) STORING (col4_15), FAMILY (col4_2), FAMILY (col4_8, col4_6, col4_4, col4_9, col4_11), FAMILY (col4_10), FAMILY (col4_15, col4_1, col4_14, col4_7, col4_3, col4_12, col4_13, col4_5), FAMILY (col2_5)) INTERLEAVE IN PARENT table2 (col2_5);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 63629986927, "histo_col_type": "", "name": "__auto__", "null_count": 83953022836, "row_count": 90000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 58127754849, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 68320599247, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "-73 years -6 mons -58 days -05:38:55.046097"}, {"distinct_range": 0, "num_eq": 3209548497056143637, "num_range": 4000000000, "upper_bound": "-52 years -10 mons -90 days -17:05:09.327499"}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 300000, "upper_bound": "-44 years -10 mons -594 days -01:34:24.091973"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23393478096, "histo_col_type": "", "name": "__auto__", "null_count": 55501908153, "row_count": 90000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 40744844543, "histo_col_type": "", "name": "__auto__", "null_count": 19996899278, "row_count": 90000000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6618656725, "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 79080676682, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000, "num_range": 0, "upper_bound": "-28 years -11 mons -744 days -23:10:13.410282"}, {"distinct_range": 0, "num_eq": 2824959285466011769, "num_range": 5000, "upper_bound": "60 years 1 mon 368 days 21:06:19.02223"}, {"distinct_range": 0, "num_eq": 400000, "num_range": 5274602965333645436, "upper_bound": "290 years"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 71374150420, "row_count": 90000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9284298835, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 76308646005, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "$"}, {"distinct_range": 432983871066941630, "num_eq": 0, "num_range": 432983871066941654, "upper_bound": "s"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 61061017470, "row_count": 90000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 88068340678, "histo_col_type": "", "name": "__auto__", "null_count": 15072472469, "row_count": 90000000000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 59024, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000, "num_range": 0, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 1550, "row_count": 100000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 97264, "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 100000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5213, "histo_col_type": "", "name": "__auto__", "null_count": 52900, "row_count": 100000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 52964, "histo_col_type": "", "name": "__auto__", "null_count": 88990, "row_count": 100000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67488, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11045, "histo_buckets": [{"distinct_range": 0, "num_eq": 7084739292006993589, "num_range": 0, "upper_bound": "-1313706810"}, {"distinct_range": 600000000, "num_eq": 900000000, "num_range": 600000000, "upper_bound": "-433640993"}, {"distinct_range": 0, "num_eq": 363997746736538614, "num_range": 2000000, "upper_bound": "-1"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 3150269209941623744, "upper_bound": "514589812"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 100000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 89976, "histo_col_type": "", "name": "__auto__", "null_count": 74810, "row_count": 100000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6693, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 620, "histo_col_type": "", "name": "__auto__", "null_count": 11716, "row_count": 20000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12705, "histo_col_type": "", "name": "__auto__", "null_count": 19281, "row_count": 20000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1801, "histo_col_type": "", "name": "__auto__", "null_count": 6158, "row_count": 20000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18168, "histo_col_type": "", "name": "__auto__", "null_count": 3004, "row_count": 20000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1393, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "-1008042521"}, {"distinct_range": 0, "num_eq": 7312229985198825451, "num_range": 800000000, "upper_bound": "-973142323"}, {"distinct_range": 5465737037298620000, "num_eq": 500, "num_range": 8003199540710766149, "upper_bound": "-494019921"}, {"distinct_range": 5743521985868851000, "num_eq": 2927357497770290200, "num_range": 5743521985868851673, "upper_bound": "463874210"}, {"distinct_range": 600000000, "num_eq": 100, "num_range": 600000000, "upper_bound": "635858682"}, {"distinct_range": 11.57352950161235, "num_eq": 8921343160585693115, "num_range": 50, "upper_bound": "2097437154"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3035, "histo_col_type": "", "name": "__auto__", "null_count": 17338, "row_count": 20000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16756, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 277, "histo_col_type": "", "name": "__auto__", "null_count": 1301, "row_count": 20000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1751, "histo_col_type": "", "name": "__auto__", "null_count": 13732, "row_count": 20000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13162, "histo_col_type": "", "name": "__auto__", "null_count": 868, "row_count": 20000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13876, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8502, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1347, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18059043594, "histo_col_type": "", "name": "__auto__", "null_count": 7932429290, "row_count": 20000000000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8720511802, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000000}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 977478419, "histo_col_type": "", "name": "__auto__", "null_count": 19799216805, "row_count": 20000000000}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12037338313, "histo_col_type": "", "name": "__auto__", "null_count": 19769810719, "row_count": 20000000000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2387971799, "histo_col_type": "", "name": "__auto__", "null_count": 10694582385, "row_count": 20000000000}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3717253715, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000000, "num_range": 0, "upper_bound": "-48 years -6 mons -425 days -12:30:54.887635"}, {"distinct_range": 0, "num_eq": 5163115196831186192, "num_range": 0, "upper_bound": "-28 years -11 mons -973 days -01:09:21.791767"}, {"distinct_range": 0, "num_eq": 2912667226678259912, "num_range": 10000000000, "upper_bound": "-22 years -3 mons -742 days -19:14:20.997392"}, {"distinct_range": 0, "num_eq": 1234017723888447334, "num_range": 4653002488360437004, "upper_bound": "-2 years -3 mons -368 days -20:04:19.530494"}, {"distinct_range": 0, "num_eq": 4504228330495979953, "num_range": 4000, "upper_bound": "47 years 555 days 17:07:50.170144"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 5296644158, "row_count": 20000000000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19146121761, "histo_col_type": "", "name": "__auto__", "null_count": 18739990961, "row_count": 20000000000}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8936852214, "histo_col_type": "NAME", "name": "__auto__", "null_count": 1846538202, "row_count": 20000000000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17941049303, "histo_buckets": [{"distinct_range": 0, "num_eq": 700, "num_range": 0, "upper_bound": "-7762441756161473807"}, {"distinct_range": 583301355185805600, "num_eq": 7000000000, "num_range": 1409901087529499895, "upper_bound": "-7733794427611015689"}, {"distinct_range": 40000000, "num_eq": 100, "num_range": 40000000, "upper_bound": "-6970780840870840396"}, {"distinct_range": 0, "num_eq": 0, "num_range": 50000000, "upper_bound": "-4780179256763096537"}, {"distinct_range": 0, "num_eq": 386452549149541737, "num_range": 60, "upper_bound": "743735348973086728"}, {"distinct_range": 0, "num_eq": 5970929623388517259, "num_range": 531847332759860459, "upper_bound": "2151225953490143302"}, {"distinct_range": 9110264101178938000, "num_eq": 4776502808285944543, "num_range": 9110264101178938191, "upper_bound": "5468844381424979066"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 900000000, "upper_bound": "9068299813348928026"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 20000000000}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10367530108, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000000}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18360633632, "histo_col_type": "", "name": "__auto__", "null_count": 2438688202, "row_count": 20000000000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10881835094, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000000, "num_range": 0, "upper_bound": "373455560"}, {"distinct_range": 0, "num_eq": 5038177952283495602, "num_range": 6460919285135499282, "upper_bound": "1037742473"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 2934834241920384065, "upper_bound": "1264275559"}, {"distinct_range": 4721865160284337000, "num_eq": 0, "num_range": 4721865160284336749, "upper_bound": "1542176868"}, {"distinct_range": 0, "num_eq": 4306954753714307518, "num_range": 6333065236936459984, "upper_bound": "2277602526"}, {"distinct_range": 0, "num_eq": 8265008754413405259, "num_range": 8619323406227609593, "upper_bound": "2573396274"}, {"distinct_range": 0, "num_eq": 3291908471417424970, "num_range": 40000, "upper_bound": "3924700762"}, {"distinct_range": 338.73330851278155, "num_eq": 30, "num_range": 500, "upper_bound": "3990162902"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 20000000000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11442978251, "histo_col_type": "", "name": "__auto__", "null_count": 4204838399, "row_count": 20000000000}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8250435270, "histo_buckets": [{"distinct_range": 0, "num_eq": 4571696093698023186, "num_range": 0, "upper_bound": "-1905809454"}, {"distinct_range": 1275725093481804800, "num_eq": 8922044394536696260, "num_range": 1275725093481804850, "upper_bound": "-515037262"}, {"distinct_range": 4479422599821413400, "num_eq": 700000, "num_range": 5680104073097330235, "upper_bound": "-374292715"}, {"distinct_range": 5074155919287339000, "num_eq": 5075771854717267469, "num_range": 5074155919287339078, "upper_bound": "175598761"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 20000000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6473349171, "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 20000000000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2381200790, "histo_buckets": [{"distinct_range": 0, "num_eq": 5455183716839077318, "num_range": 0, "upper_bound": "2007-01-31 20:02:21.000497+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 20000000000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('hfq', 'gavanj');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('y', 'fevg');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1171811326323868580, "histo_col_type": "", "name": "__auto__", "null_count": 2928057311492971142, "row_count": 4042932389580150764}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3025152635009838769, "histo_col_type": "", "name": "__auto__", "null_count": 1171969294348346418, "row_count": 4042932389580150764}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2476394078680381752, "histo_col_type": "", "name": "__auto__", "null_count": 3088437836990844013, "row_count": 4042932389580150764}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3592219668598985773, "histo_col_type": "", "name": "__auto__", "null_count": 641697817455568170, "row_count": 4042932389580150764}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2481188544096859233, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 802674.9034608711, "num_eq": 90, "num_range": 1000000, "upper_bound": "662121184"}, {"distinct_range": 0, "num_eq": 500000, "num_range": 40000, "upper_bound": "1002393175"}, {"distinct_range": 2069682583555133700, "num_eq": 7522152893280134140, "num_range": 2069682583555133569, "upper_bound": "2124754373"}, {"distinct_range": 8561623801959308000, "num_eq": 20000, "num_range": 8561623801959308144, "upper_bound": "2775543207"}, {"distinct_range": 2615324.659366471, "num_eq": 3780219139945481370, "num_range": 5000000, "upper_bound": "3053320403"}, {"distinct_range": 7274371.702546524, "num_eq": 5000, "num_range": 20000000, "upper_bound": "3240937728"}, {"distinct_range": 374.3708668738215, "num_eq": 40000, "num_range": 4000, "upper_bound": "3985500231"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 4042932389580150764}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3946633379046235147, "histo_buckets": [{"distinct_range": 0, "num_eq": 2985818031758855383, "num_range": 0, "upper_bound": "-53 years -4 mons -887 days -13:06:01.41576"}, {"distinct_range": 100000, "num_eq": 3414054031377398401, "num_range": 100000, "upper_bound": "-23 years -5 mons -946 days -11:21:16.305495"}, {"distinct_range": 0, "num_eq": 20000000, "num_range": 4644787736699960044, "upper_bound": "27 years 6 mons 842 days 21:30:32.749374"}, {"distinct_range": 0, "num_eq": 500000000, "num_range": 90000, "upper_bound": "32 years 7 mons 959 days 00:07:54.128077"}, {"distinct_range": 6069301506470798000, "num_eq": 7681457773638473191, "num_range": 6069301506470798069, "upper_bound": "55 years 1 mon 634 days 01:19:52.36551"}, {"distinct_range": 0, "num_eq": 5847256734771437169, "num_range": 3000000, "upper_bound": "59 years 1 mon 576 days 18:36:59.904207"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 3436102994325600555, "row_count": 4042932389580150764}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 819676951682569211, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4042932389580150764}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3020106557152291310, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4042932389580150764}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 739029527019685052, "histo_buckets": [{"distinct_range": 0, "num_eq": 6986545723726473853, "num_range": 0, "upper_bound": "("}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 100000000, "upper_bound": "g"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 622618316313148797, "row_count": 4042932389580150764}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1919876918408829878, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000, "num_range": 0, "upper_bound": "56 years 7 mons 542 days 21:57:14.077766"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 4042932389580150764}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1854337548497497471, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2969276521457742933}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1171691546900170852, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-1847168128"}, {"distinct_range": 82656.2384266126, "num_eq": 700000000, "num_range": 90000, "upper_bound": "-271459321"}, {"distinct_range": 0, "num_eq": 601961070443968461, "num_range": 70000000, "upper_bound": "-95292058"}, {"distinct_range": 575755428577563500, "num_eq": 4013646954496384347, "num_range": 4185856886270687844, "upper_bound": "1142687696"}, {"distinct_range": 2578135686464608000, "num_eq": 100000000000, "num_range": 6105281633492127875, "upper_bound": "2147483647"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 2969276521457742933}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22226228945558057, "histo_col_type": "", "name": "__auto__", "null_count": 1490642710421132193, "row_count": 2969276521457742933}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1569049295736173011, "histo_buckets": [{"distinct_range": 0, "num_eq": 7467665461756158747, "num_range": 0, "upper_bound": "false"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 412280527293062614, "row_count": 2969276521457742933}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2866460474688850826, "histo_buckets": [{"distinct_range": 0, "num_eq": 3122333899304022078, "num_range": 0, "upper_bound": "0"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 2969276521457742933}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2165849983386117399, "histo_col_type": "", "name": "__auto__", "null_count": 2578100318277473068, "row_count": 2969276521457742933}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2578899811274459507, "histo_col_type": "", "name": "__auto__", "null_count": 852034370697805458, "row_count": 2969276521457742933}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7054362469492927992, "histo_col_type": "", "name": "__auto__", "null_count": 7337582260907141404, "row_count": 8614918707333446541}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8265380272352693928, "histo_col_type": "", "name": "__auto__", "null_count": 6288507983100211604, "row_count": 8614918707333446541}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2991459324267837840, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8614918707333446541}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2507163239138778247, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8614918707333446541}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2626346709324619975, "histo_col_type": "", "name": "__auto__", "null_count": 7592569793309641868, "row_count": 8614918707333446541}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3158929700862287724, "histo_col_type": "", "name": "__auto__", "null_count": 8239676191167177618, "row_count": 8614918707333446541}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6516906886876871596, "histo_col_type": "", "name": "__auto__", "null_count": 6013111432420323397, "row_count": 8614918707333446541}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2162329544617706615, "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 8614918707333446541}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 772230983016907638, "histo_col_type": "", "name": "__auto__", "null_count": 467445199741999914, "row_count": 8614918707333446541}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6739666954886443495, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8614918707333446541}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6487550279297642121, "histo_col_type": "", "name": "__auto__", "null_count": 7643800860546400661, "row_count": 8614918707333446541}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8326686442997009903, "histo_col_type": "", "name": "__auto__", "null_count": 267726574860415652, "row_count": 8614918707333446541}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8448016776388827291, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8614918707333446541}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3450333917437899606, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8614918707333446541}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6290117, "histo_buckets": [{"distinct_range": 0, "num_eq": 4921610309386801724, "num_range": 0, "upper_bound": "iF\\u0019p"}, {"distinct_range": 3652396174234009600, "num_eq": 9000, "num_range": 6472955713767860842, "upper_bound": "k\\u000egY\\u0005\\n"}, {"distinct_range": 0, "num_eq": 50, "num_range": 8000, "upper_bound": "}\\\\|"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 1227620, "row_count": 10000000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2565174, "histo_col_type": "", "name": "__auto__", "null_count": 4487958, "row_count": 10000000}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4238189, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000, "num_range": 0, "upper_bound": "-60 years -4 mons -841 days -17:01:24.772977"}, {"distinct_range": 28253.387250884647, "num_eq": 5280344426577472485, "num_range": 40000, "upper_bound": "-54 years -9 mons -213 days -04:21:29.713262"}, {"distinct_range": 0, "num_eq": 6873274632965494276, "num_range": 4651750121012581850, "upper_bound": "-34 years -449 days -05:25:29.564971"}, {"distinct_range": 3293656.1016914677, "num_eq": 600000, "num_range": 8000000, "upper_bound": "16 years 7 mons 903 days 19:41:02.921205"}, {"distinct_range": 6207.3043926756, "num_eq": 5943908562115441283, "num_range": 10000, "upper_bound": "39 years 4 mons 428 days 19:54:33.996141"}, {"distinct_range": 70, "num_eq": 2519758262509858417, "num_range": 70, "upper_bound": "75 years 7 mons 754 days 18:41:07.176578"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 2124681, "row_count": 10000000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7972501, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000, "num_range": 0, "upper_bound": "1998-02-02 04:16:19.000207+00:00"}, {"distinct_range": 0, "num_eq": 8853936114187373055, "num_range": 1872173786579472344, "upper_bound": "1999-08-23 14:30:51.0007+00:00"}, {"distinct_range": 7002034211586834000, "num_eq": 80000000000, "num_range": 7002034211586834216, "upper_bound": "2006-03-04 23:16:56.000625+00:00"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 8059883740011334814, "upper_bound": "2009-02-21 04:36:08.000737+00:00"}, {"distinct_range": 432982590808680900, "num_eq": 7868841105911401634, "num_range": 1462062724109349345, "upper_bound": "3000-01-01 00:00:00+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2819554, "histo_buckets": [{"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "-2012389311"}, {"distinct_range": 90000, "num_eq": 6304313614460672241, "num_range": 90000, "upper_bound": "-1383214141"}, {"distinct_range": 0, "num_eq": 2857720161440809884, "num_range": 9030260501840132656, "upper_bound": "2101253639"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1421429, "histo_col_type": "", "name": "__auto__", "null_count": 4686209, "row_count": 10000000}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8046881, "histo_col_type": "", "name": "__auto__", "null_count": 3554222, "row_count": 10000000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8671831, "histo_col_type": "", "name": "__auto__", "null_count": 2338368, "row_count": 10000000}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8364153, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8044016, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000, "num_range": 0, "upper_bound": "-719498075"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 159081516001568915, "upper_bound": "-444538957"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8208494, "histo_col_type": "", "name": "__auto__", "null_count": 3266659, "row_count": 10000000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 510987, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5368881, "histo_col_type": "", "name": "__auto__", "null_count": 8283792, "row_count": 10000000}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3294181, "histo_col_type": "", "name": "__auto__", "null_count": 9593617, "row_count": 10000000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8498574, "histo_buckets": [{"distinct_range": 0, "num_eq": 900, "num_range": 0, "upper_bound": "602920051"}, {"distinct_range": 700000000, "num_eq": 20000, "num_range": 700000000, "upper_bound": "1911261863"}, {"distinct_range": 4008322551658256000, "num_eq": 50000000, "num_range": 8533380154628684195, "upper_bound": "4153893230"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 10000000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7902197, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000, "num_range": 0, "upper_bound": "-3101174169706047019"}, {"distinct_range": 4357530280762066400, "num_eq": 2000000000, "num_range": 7788929229665026013, "upper_bound": "-1195620412533998242"}, {"distinct_range": 0, "num_eq": 6542733875807956537, "num_range": 20000000000, "upper_bound": "-128"}, {"distinct_range": 7158370426525529000, "num_eq": 0, "num_range": 7158370426525528953, "upper_bound": "3246227088378290531"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 4000000, "upper_bound": "5125107336468453686"}, {"distinct_range": 549751871053996740, "num_eq": 2028854036406110119, "num_range": 1825179225377605980, "upper_bound": "8327036308095309223"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 10000000}]';
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_2, col1_4, col1_3) REFERENCES table3 (col3_1, col3_5, col3_2);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16501065, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8109514, "histo_col_type": "", "name": "__auto__", "null_count": 4968502, "row_count": 20000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7059381, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": "197858050"}, {"distinct_range": 633.0187637631892, "num_eq": 4428663853923521525, "num_range": 700, "upper_bound": "2148718692"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4563686, "histo_buckets": [{"distinct_range": 0, "num_eq": 4961620350130986529, "num_range": 0, "upper_bound": "-51 years -10 mons -812 days -07:27:07.85151"}, {"distinct_range": 0, "num_eq": 8871833618280318611, "num_range": 0, "upper_bound": "00:00:00"}, {"distinct_range": 0, "num_eq": 6961295235544883858, "num_range": 50000000, "upper_bound": "28 years 9 mons 752 days 08:45:41.165885"}, {"distinct_range": 0, "num_eq": 2077750762342488192, "num_range": 50000000000, "upper_bound": "35 years 11 mons 406 days 21:04:55.491953"}, {"distinct_range": 387044819087950300, "num_eq": 405008884056678383, "num_range": 793330258956057667, "upper_bound": "54 years 10 mons 697 days 01:19:55.303923"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 3479009, "row_count": 20000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3811528, "histo_col_type": "", "name": "__auto__", "null_count": 529092, "row_count": 20000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11135591, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14980360, "histo_col_type": "", "name": "__auto__", "null_count": 7081940, "row_count": 20000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 810227, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "\\u000c"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 6000000000, "upper_bound": "\\u001d"}, {"distinct_range": 6078041661623152000, "num_eq": 0, "num_range": 6078041661623151120, "upper_bound": "3"}, {"distinct_range": 0, "num_eq": 320261481822529110, "num_range": 5000000, "upper_bound": "E"}, {"distinct_range": 60000, "num_eq": 70000, "num_range": 60000, "upper_bound": "M"}, {"distinct_range": 6856860895458576000, "num_eq": 5565252461046989034, "num_range": 6856860895458576780, "upper_bound": "V"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 5988018759275159413, "upper_bound": "]"}, {"distinct_range": 100, "num_eq": 479049093368233594, "num_range": 100, "upper_bound": "`"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 10944671, "row_count": 20000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14247061, "histo_buckets": [{"distinct_range": 0, "num_eq": 7511045665186587284, "num_range": 0, "upper_bound": "-72 years -9 mons -521 days -05:15:44.817561"}, {"distinct_range": 3684553371588513300, "num_eq": 10000, "num_range": 8835825272933073800, "upper_bound": "-35 years -2 mons -476 days -04:04:39.866061"}, {"distinct_range": 3000, "num_eq": 900000, "num_range": 3000, "upper_bound": "-3 years -1 mons -498 days -16:54:11.048878"}, {"distinct_range": 4596213826319688700, "num_eq": 2022477842663714135, "num_range": 4596213826319688624, "upper_bound": "00:00:00"}, {"distinct_range": 0, "num_eq": 3000000, "num_range": 5569860667868884204, "upper_bound": "30 years 6 mons 7 days 19:30:18.292131"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 30000, "upper_bound": "51 years 10 mons 821 days 16:51:45.004173"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12828116, "histo_col_type": "", "name": "__auto__", "null_count": 7719346, "row_count": 20000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7532718771, "histo_col_type": "", "name": "__auto__", "null_count": 9554839091, "row_count": 10000000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4990203591, "histo_buckets": [{"distinct_range": 0, "num_eq": 27735851815217927, "num_range": 0, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 4027580009, "row_count": 10000000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5163932067, "histo_buckets": [{"distinct_range": 0, "num_eq": 8651037768088073758, "num_range": 0, "upper_bound": "2318857144"}, {"distinct_range": 1952511218827760600, "num_eq": 500, "num_range": 1952511218827760623, "upper_bound": "3107155745"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3524744098, "histo_col_type": "", "name": "__auto__", "null_count": 630868096, "row_count": 10000000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1571320678, "histo_col_type": "", "name": "__auto__", "null_count": 3621919238, "row_count": 10000000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5675717091, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3979330068, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000, "num_range": 0, "upper_bound": "-1916897759"}, {"distinct_range": 4080636946813570600, "num_eq": 700, "num_range": 4080636946813570541, "upper_bound": "-1679045093"}, {"distinct_range": 932805704.454722, "num_eq": 10000000, "num_range": 5000000000, "upper_bound": "-1328439291"}, {"distinct_range": 2169678239.8180366, "num_eq": 4871939636437469654, "num_range": 10000000000, "upper_bound": "-432003006"}, {"distinct_range": 0, "num_eq": 2964710515010902103, "num_range": 6895527611454015228, "upper_bound": "838787435"}, {"distinct_range": 6823205517230280000, "num_eq": 0, "num_range": 6823205517230279565, "upper_bound": "1564539834"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 10000000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 633508453208823463, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1386527042595677063}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 353023554424097294, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1386527042595677063}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 373037518418648372, "histo_col_type": "", "name": "__auto__", "null_count": 344941360921255584, "row_count": 1386527042595677063}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 624306082532859340, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1386527042595677063}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 185416377465904364, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1386527042595677063}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 180888557728131451, "histo_col_type": "", "name": "__auto__", "null_count": 1120563107077599707, "row_count": 1386527042595677063}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 287148840638944013, "histo_col_type": "", "name": "__auto__", "null_count": 927812554868580162, "row_count": 1386527042595677063}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1098879743812106038, "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 1386527042595677063}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 269080702250851475, "histo_buckets": [{"distinct_range": 0, "num_eq": 8496742905080146532, "num_range": 0, "upper_bound": "42233568"}, {"distinct_range": 67.79854297052601, "num_eq": 60000000, "num_range": 90, "upper_bound": "2441740022"}, {"distinct_range": 0, "num_eq": 50, "num_range": 0, "upper_bound": "2865337974"}, {"distinct_range": 0, "num_eq": 8638944892375135788, "num_range": 8201204167228071832, "upper_bound": "2893690511"}, {"distinct_range": 448138862.4049185, "num_eq": 2258716717200089588, "num_range": 6000000000, "upper_bound": "2972095836"}, {"distinct_range": 57550.68751613368, "num_eq": 70000, "num_range": 200000, "upper_bound": "4161552051"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 851392223250349653, "row_count": 1386527042595677063}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 60351781744954280, "histo_col_type": "", "name": "__auto__", "null_count": 600369970980517566, "row_count": 1386527042595677063}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 489505911166718324, "histo_col_type": "", "name": "__auto__", "null_count": 1335463320732314258, "row_count": 1386527042595677063}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1147195420158067853, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1386527042595677063}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1337211581942222069, "histo_col_type": "", "name": "__auto__", "null_count": 101769476914596309, "row_count": 1386527042595677063}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 881910184571715381, "histo_col_type": "", "name": "__auto__", "null_count": 1326031920900719922, "row_count": 1386527042595677063}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1014373135173329483, "histo_buckets": [{"distinct_range": 0, "num_eq": 4123412126124255142, "num_range": 0, "upper_bound": "-2 years -2 mons -847 days -16:25:17.383854"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 760527293547886334, "row_count": 1893668612124689460}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 977388538701839942, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000000, "num_range": 0, "upper_bound": "1987-09-24 18:14:14.000868+00:00"}, {"distinct_range": 239.72096921307818, "num_eq": 4569723339191767759, "num_range": 400, "upper_bound": "1991-10-01 18:10:41.000659+00:00"}, {"distinct_range": 926723268.0618535, "num_eq": 2626752994218837320, "num_range": 1000000000, "upper_bound": "1993-12-20 04:42:30.000089+00:00"}, {"distinct_range": 63.302805540991855, "num_eq": 20000, "num_range": 90, "upper_bound": "2000-12-12 19:39:50.00042+00:00"}, {"distinct_range": 5092848587601960000, "num_eq": 800, "num_range": 5092848587601959656, "upper_bound": "2025-07-13 10:00:32.000657+00:00"}, {"distinct_range": 1835278642271254300, "num_eq": 5266539403569654315, "num_range": 8440917818695413606, "upper_bound": "2032-03-26 12:43:21.000167+00:00"}, {"distinct_range": 0, "num_eq": 30000000000, "num_range": 870323407513753037, "upper_bound": "3000-01-01 00:00:00+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 0, "row_count": 1893668612124689460}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 62141066189992881, "histo_buckets": [{"distinct_range": 0, "num_eq": 3223616355973904238, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 849910696925796681, "num_range": 0, "upper_bound": "390489397"}, {"distinct_range": 10000000, "num_eq": 0, "num_range": 10000000, "upper_bound": "754379634"}, {"distinct_range": 5162220300300748000, "num_eq": 3865943340516271333, "num_range": 6918416456692931119, "upper_bound": "2397824022"}, {"distinct_range": 8991590036798921000, "num_eq": 2521982562019692694, "num_range": 8991590036798920388, "upper_bound": "2401208860"}, {"distinct_range": 4000, "num_eq": 3000000, "num_range": 4000, "upper_bound": "2773842358"}, {"distinct_range": 8250789283936558000, "num_eq": 6901706768248159944, "num_range": 8250789283936557918, "upper_bound": "3615939344"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 1893668612124689460}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 486493981619278017, "histo_buckets": [{"distinct_range": 0, "num_eq": 6194559685143215541, "num_range": 0, "upper_bound": "-6921510594058837315"}, {"distinct_range": 401731133354588000, "num_eq": 4000000000, "num_range": 401731133354588036, "upper_bound": "-3746397999034932845"}, {"distinct_range": 8813807571675326000, "num_eq": 75492618881592228, "num_range": 8813807571675326245, "upper_bound": "-2091816094630684761"}, {"distinct_range": 203919216081119070, "num_eq": 7000000000, "num_range": 203919216081119083, "upper_bound": "-778620727683553282"}, {"distinct_range": 0, "num_eq": 6005393492635110898, "num_range": 0, "upper_bound": "396587664122148805"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 700000000, "upper_bound": "1911444761177444091"}, {"distinct_range": 4770430178714342000, "num_eq": 7000000000, "num_range": 4770430178714342101, "upper_bound": "2841669436683537349"}, {"distinct_range": 6686567780545929000, "num_eq": 1804477987986179504, "num_range": 6686567780545928989, "upper_bound": "6422469776472720484"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 1893668612124689460}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1245116507162896827, "histo_col_type": "", "name": "__auto__", "null_count": 1732887738600414573, "row_count": 1893668612124689460}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 940686834187166657, "histo_col_type": "", "name": "__auto__", "null_count": 1534308247765585369, "row_count": 1893668612124689460}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 222466265909322858, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1893668612124689460}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 797563348474176186, "histo_col_type": "NAME", "name": "__auto__", "null_count": 1587593205763765399, "row_count": 1893668612124689460}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 460163213287991356, "histo_col_type": "", "name": "__auto__", "null_count": 601043874485887903, "row_count": 1893668612124689460}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1717282843658113890, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1347065442"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 1893668612124689460}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1623519770003871252, "histo_col_type": "", "name": "__auto__", "null_count": 1437756157591435387, "row_count": 1893668612124689460}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1864531722676747098, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1893668612124689460}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1748117325008642186, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "-1043269014"}, {"distinct_range": 7131350847476953000, "num_eq": 3000000, "num_range": 7131350847476953183, "upper_bound": "-890551137"}, {"distinct_range": 0, "num_eq": 5296339228722847364, "num_range": 5000000000, "upper_bound": "-838225722"}, {"distinct_range": 0, "num_eq": 4000000000, "num_range": 9000, "upper_bound": "-128"}, {"distinct_range": 20000, "num_eq": 900000, "num_range": 20000, "upper_bound": "676035848"}, {"distinct_range": 36074998205008136, "num_eq": 70, "num_range": 83525517330781377, "upper_bound": "855930410"}, {"distinct_range": 1000000, "num_eq": 3272552378294962129, "num_range": 1000000, "upper_bound": "1228711088"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 1893668612124689460}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 395876117497919386, "histo_col_type": "", "name": "__auto__", "null_count": 1665595330982323522, "row_count": 1893668612124689460}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 93877944822738459, "histo_col_type": "", "name": "__auto__", "null_count": 1375060762706861763, "row_count": 1893668612124689460}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 529647442080794009, "histo_col_type": "", "name": "__auto__", "null_count": 503861960588094476, "row_count": 1893668612124689460}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "nan" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2728769&tab=buildLog) on [master@206e8178de9ac124bfde5529d8a9b858734038bb](https://github.com/cockroachdb/cockroach/commits/206e8178de9ac124bfde5529d8a9b858734038bb):

```
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        test_1        | --- FAIL: TestCompare (1.60s)
        test_1        |     --- FAIL: TestCompare/postgres (0.56s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (1.04s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 "char" NOT NULL, col1_1 DECIMAL NULL, col1_2 INT2, col1_3 INT2, col1_4 TIMETZ, col1_5 BIT(49), col1_6 NAME, col1_7 INT2, PRIMARY KEY (col1_0), UNIQUE (col1_2, col1_5, col1_6 ASC, col1_3 ASC, col1_1 DESC) STORING (col1_4, col1_7) WHERE ((((((table1.col1_0 <= e'\'') AND (table1.col1_2 != -1)) AND (table1.col1_6 < e'\'')) AND (table1.col1_7 != 1)) AND (table1.col1_4 != '24:00:00-15:59:00')) OR (table1.col1_1 != -1.234E+401)) AND (table1.col1_3 >= -1), UNIQUE (col1_0 ASC, col1_4, col1_1 ASC) STORING (col1_6) WHERE ((((((table1.col1_0 != 'X':::STRING) AND (table1.col1_4 = '00:00:00+15:59:00':::TIMETZ)) OR (table1.col1_7 > (-128):::INT8)) AND (table1.col1_1 > 0:::DECIMAL)) AND (table1.col1_3 != (-1):::INT8)) AND (table1.col1_6 = '':::STRING)) OR (table1.col1_2 >= 0:::INT8), UNIQUE (col1_0 DESC, col1_1 ASC, col1_4 DESC, col1_2 DESC, col1_7 DESC, col1_5, col1_3 ASC, col1_6 DESC) WHERE (((((table1.col1_1 > '-Infinity':::DECIMAL) AND (table1.col1_2 <= 1:::INT8)) AND (table1.col1_6 != '':::STRING)) AND (table1.col1_7 = 32767:::INT8)) OR (table1.col1_0 = '':::STRING)) AND (table1.col1_3 != 1:::INT8), FAMILY (col1_5), FAMILY (col1_4), FAMILY (col1_1, col1_0), FAMILY (col1_6), FAMILY (col1_7), FAMILY (col1_3), FAMILY (col1_2));
        test_1        |             CREATE TABLE table2 (col2_0 INT4 NOT NULL, col2_1 "char" NOT NULL, col2_2 NAME, col2_3 REGCLASS NOT NULL, col2_4 FLOAT8 NOT NULL, col2_5 BOOL NOT NULL, PRIMARY KEY (col2_4 ASC, col2_5 ASC), UNIQUE (col2_1, col2_0, col2_4 ASC, col2_3, col2_5 DESC, col2_2 DESC) WHERE (table2.col2_2 != '"':::STRING) OR (NOT table2.col2_5), FAMILY (col2_5, col2_0, col2_2, col2_3), FAMILY (col2_4), FAMILY (col2_1));
        test_1        |             CREATE TABLE table3 (col1_0 "char" NOT NULL, col3_1 VARCHAR, col3_2 NAME NULL, col3_3 VARCHAR, col3_4 STRING, col3_5 OID, col3_6 BIT(21) NULL, col3_7 FLOAT8 NOT NULL, col3_8 TIMESTAMPTZ NOT NULL, PRIMARY KEY (col1_0, col3_7 ASC, col3_6, col3_8 ASC, col3_2 DESC, col3_5 ASC), UNIQUE (col3_6 ASC, col3_4 ASC, col3_1 DESC, col3_8, col1_0, col3_7 DESC, col3_5 ASC, col3_3 ASC, col3_2) WHERE (((((table3.col3_3 < '"':::STRING) AND (table3.col1_0 >= e'\'':::STRING)) AND (table3.col3_2 > '':::STRING)) OR (table3.col3_1 > e'\U00002603':::STRING)) OR (table3.col3_8 != '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table3.col3_7 <= '-Inf':::FLOAT8), INDEX (col3_3, col1_0 DESC, col3_6 ASC, col3_5, col3_1 ASC) STORING (col3_4) WHERE ((((((table3.col3_4 >= e'\U00002603') AND (table3.col3_7 > 3.4028234663852886e+38)) AND (table3.col3_8 > '-2000-01-01 00:00:00+00:00')) AND (table3.col3_2 >= e'\'')) AND (table3.col3_1 = 'X')) AND (table3.col1_0 <= '')) OR (table3.col3_3 <= e'\x00'), UNIQUE (col3_7, col1_0 DESC, col3_6, col3_4, col3_5 ASC) STORING (col3_3) WHERE ((((table3.col3_2 > e'\'':::STRING) AND (table3.col1_0 < e'\'':::STRING)) AND (table3.col3_3 >= '':::STRING)) AND (table3.col3_4 != '':::STRING)) OR (table3.col3_1 = '':::STRING), UNIQUE (col3_1 ASC, col3_7 ASC, col3_8 ASC) WHERE table3.col3_3 <= 'X', UNIQUE (col3_1 ASC, col1_0 ASC, col3_3, col3_4 DESC, col3_6 DESC, col3_5 DESC) WHERE table3.col3_1 = e'\x00':::STRING, INDEX (col3_6 ASC, col3_2 DESC, col3_3 ASC, col3_1 DESC, col3_4 ASC, col3_5 ASC) WHERE (((((table3.col3_3 > '') OR (table3.col3_2 >= e'\U00002603')) OR (table3.col3_4 != '')) OR (table3.col3_7 != -1.0)) OR (table3.col1_0 >= '"')) OR (table3.col3_1 > ''), UNIQUE (col3_6 DESC, col3_7 DESC, col3_4 ASC, col3_2, col3_5 DESC, col1_0 DESC, col3_1), UNIQUE (col3_5 ASC, col3_6 ASC, col3_1 DESC, col3_8, col3_7, col3_2, col1_0) WHERE ((((((table3.col3_4 <= e'\'':::STRING) AND (table3.col1_0 != e'\U00002603':::STRING)) AND (table3.col3_2 > e'\U00002603':::STRING)) AND (table3.col3_8 <= '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (table3.col3_7 < 'NaN':::FLOAT8)) OR (table3.col3_3 >= '"':::STRING)) AND (table3.col3_1 >= e'\U00002603':::STRING), UNIQUE (col1_0, col3_7) STORING (col3_1) WHERE (((((table3.col3_1 <= e'\x00') OR (table3.col3_4 <= e'\x00')) AND (table3.col3_7 < 1.7976931348623157e+308)) AND (table3.col3_8 <= '-4713-11-24 00:00:00+00:00')) OR (table3.col3_2 != '"')) OR (table3.col1_0 <= '"'), FAMILY (col3_1, col3_4), FAMILY (col3_2, col3_7), FAMILY (col1_0, col3_6), FAMILY (col3_5, col3_3), FAMILY (col3_8)) INTERLEAVE IN PARENT table1 (col1_0);
        test_1        |             CREATE TABLE table4 (col2_4 FLOAT8 NOT NULL, col2_5 BOOL NOT NULL, col4_2 BIT(33), col4_3 TIMESTAMPTZ, col4_4 REGCLASS NOT NULL, col4_5 INTERVAL NOT NULL, col4_6 DECIMAL NOT NULL, col4_7 BOX2D, col4_8 INT4 NULL, col4_9 INT2 NULL, col4_10 STRING NULL, col4_11 REGNAMESPACE NULL, col4_12 REGNAMESPACE NOT NULL, col4_13 REGNAMESPACE, col4_14 TIME NOT NULL, col4_15 BIT(46) NOT NULL, col4_16 "char" NOT NULL, col4_17 BYTES NOT NULL, PRIMARY KEY (col2_4 ASC, col2_5 ASC, col4_10 ASC, col4_11 ASC, col4_16 DESC, col4_12 ASC, col4_15, col4_4, col4_14 ASC, col4_7, col4_6 ASC, col4_9 DESC, col4_3 DESC, col4_17 DESC, col4_8), UNIQUE (col4_10, col4_7, col4_12, col2_5 ASC, col4_16, col4_14 ASC, col4_6 DESC) STORING (col4_2) WHERE (((((((((table4.col4_9 <= 0:::INT8) AND (table4.col4_10 = e'\x00':::STRING)) OR (table4.col4_8 = (-128):::INT8)) AND (table4.col4_14 > '24:00:00':::TIME)) AND (table4.col4_3 > '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) OR (table4.col4_17 < e'\\x00':::BYTES)) OR (table4.col4_6 = '-Infinity':::DECIMAL)) AND (table4.col4_16 = 'X':::STRING)) OR table4.col2_5) OR (table4.col2_4 = 0.0:::FLOAT8), UNIQUE (col4_5, col4_6 DESC, col4_14 DESC, col4_11, col2_5 ASC, col4_15, col4_12 ASC, col4_2 ASC, col4_16 DESC, col4_7 DESC, col4_17 ASC, col4_10, col4_8, col2_4 DESC, col4_13 DESC, col4_9 ASC), UNIQUE (col2_4 ASC, col4_7 DESC, col4_17 ASC, col4_12 ASC, col4_14 DESC, col4_10 ASC) STORING (col4_5, col4_13) WHERE ((((table4.col4_10 >= 'X':::STRING) OR (table4.col4_17 > e'\\x':::BYTES)) AND (table4.col4_3 >= '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR table4.col2_5) AND (table4.col4_9 >= (-32768):::INT8), UNIQUE (col4_8, col4_3 ASC, col4_15, col4_5 ASC, col2_5 DESC) STORING (col4_2, col4_13), UNIQUE (col2_4, col4_14 ASC, col4_13 DESC, col4_9, col4_15 DESC, col4_6, col4_8 DESC, col4_16 DESC, col4_10, col4_17 DESC, col4_2 ASC) STORING (col4_5) WHERE (((((((((table4.col4_3 > '3000-01-01 00:00:00+00:00') OR (table4.col4_9 < -1)) OR (table4.col4_14 <= '24:00:00')) OR (table4.col2_4 >= 5e-324)) AND (table4.col4_8 = 1)) OR (table4.col4_17 != '\x27')) OR (NOT table4.col2_5)) AND (table4.col4_10 <= '"')) AND (table4.col4_16 <= e'\U00002603')) OR (table4.col4_6 >= -1.234E+401), INDEX (col4_4 ASC, col4_15, col4_16 ASC, col4_17 ASC, col2_4 ASC, col4_10 ASC, col4_14, col4_5 DESC, col4_6) WHERE ((((table4.col4_17 = '\x27') OR table4.col2_5) OR (table4.col4_14 != '00:00:00')) OR (table4.col4_10 > e'\U00002603')) AND (table4.col4_8 >= -128), UNIQUE (col2_5 ASC, col4_10 DESC, col4_6) STORING (col4_2, col4_13), UNIQUE (col4_4, col4_10, col2_5, col4_8) STORING (col4_5) WHERE (((((((((table4.col4_14 < '00:00:00') OR (table4.col2_4 = 5e-324)) OR (table4.col4_8 < 0)) AND (table4.col4_6 >= 0)) OR table4.col2_5) AND (table4.col4_16 = e'\U00002603')) OR (table4.col4_17 = '\xe29883')) AND (table4.col4_3 < '294276-12-31 23:59:59.999999+00:00')) AND (table4.col4_9 <= -128)) AND (table4.col4_10 = '"'), FAMILY (col4_11, col2_5, col4_3), FAMILY (col4_13, col4_7, col4_2), FAMILY (col4_15, col4_6), FAMILY (col4_4), FAMILY (col4_9, col4_17), FAMILY (col4_12), FAMILY (col2_4, col4_14), FAMILY (col4_8), FAMILY (col4_5), FAMILY (col4_16), FAMILY (col4_10)) INTERLEAVE IN PARENT table2 (col2_4, col2_5);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7263, "histo_col_type": "", "name": "__auto__", "null_count": 1150, "row_count": 10000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9956, "histo_col_type": "", "name": "__auto__", "null_count": 7438, "row_count": 10000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8600, "histo_buckets": [{"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "\\u0000"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5853, "histo_col_type": "", "name": "__auto__", "null_count": 4852, "row_count": 10000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2045, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000000, "num_range": 0, "upper_bound": "-24092"}, {"distinct_range": 3.05662393284071, "num_eq": 5608389940845148980, "num_range": 50, "upper_bound": "-18925"}, {"distinct_range": 0, "num_eq": 7940226015102446686, "num_range": 10000000, "upper_bound": "-14888"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 7157093681327096338, "upper_bound": "9347"}, {"distinct_range": 0, "num_eq": 500000000, "num_range": 60, "upper_bound": "24090"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 5465, "row_count": 10000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7508, "histo_col_type": "", "name": "__auto__", "null_count": 3556, "row_count": 10000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9584, "histo_col_type": "", "name": "__auto__", "null_count": 1305, "row_count": 10000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5623, "histo_col_type": "", "name": "__auto__", "null_count": 8277, "row_count": 10000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10342232, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15557891, "histo_buckets": [{"distinct_range": 0, "num_eq": 5699058576073733381, "num_range": 0, "upper_bound": "\\u0013"}, {"distinct_range": 0, "num_eq": 3931326524363269135, "num_range": 5047254840336244694, "upper_bound": "\\u001f"}, {"distinct_range": 296561139.872666, "num_eq": 8413904897643022164, "num_range": 300000000, "upper_bound": "-"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 50, "upper_bound": "\\\\"}, {"distinct_range": 0, "num_eq": 8413225301578938467, "num_range": 100, "upper_bound": "_"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6669247, "histo_col_type": "", "name": "__auto__", "null_count": 19216240, "row_count": 20000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16921106, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14310776, "histo_buckets": [{"distinct_range": 0, "num_eq": 5811413146948283137, "num_range": 0, "upper_bound": "-1.2772879024574149"}, {"distinct_range": 8021932193437578000, "num_eq": 3271800834765424448, "num_range": 8021932193437578097, "upper_bound": "-0.7732231398431129"}, {"distinct_range": 25040974.719389174, "num_eq": 5744815422321908208, "num_range": 200000000, "upper_bound": "-0.7707915390450545"}, {"distinct_range": 7484.961213488809, "num_eq": 4069368876102597860, "num_range": 9000, "upper_bound": "-0.28925726240109195"}, {"distinct_range": 3640818622397525500, "num_eq": 100000000, "num_range": 8516767164975554650, "upper_bound": "-0.2728278329280689"}, {"distinct_range": 4301558850237677600, "num_eq": 60000000, "num_range": 4301558850237677349, "upper_bound": "0.178244043549329"}, {"distinct_range": 100000, "num_eq": 4116922775428798153, "num_range": 100000, "upper_bound": "0.9286288617027678"}, {"distinct_range": 9237471049.039724, "num_eq": 1617350234558291989, "num_range": 10000000000, "upper_bound": "1.907828053850958"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11438814, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 134, "histo_buckets": [{"distinct_range": 0, "num_eq": 6025152800826735366, "num_range": 0, "upper_bound": "0"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 122, "row_count": 300}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 144, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "0.37099389966794394"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 300}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 199, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 47, "histo_buckets": [{"distinct_range": 0, "num_eq": 4032658474678389983, "num_range": 0, "upper_bound": ""}, {"distinct_range": 49573.97471011806, "num_eq": 8000000000, "num_range": 60000, "upper_bound": "\\u00036\\u0008eE=Ksm"}, {"distinct_range": 3713282377988383000, "num_eq": 8490616422250697650, "num_range": 3750646541612758066, "upper_bound": "X"}, {"distinct_range": 80000000000, "num_eq": 800000000, "num_range": 80000000000, "upper_bound": "l\\u0010dT8d6"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 84, "row_count": 300}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 153, "histo_col_type": "", "name": "__auto__", "null_count": 283, "row_count": 300}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 266, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "\\u000fd<\\u000b"}, {"distinct_range": 0, "num_eq": 7072097491095377779, "num_range": 100000, "upper_bound": "\\u0012!3^"}, {"distinct_range": 0, "num_eq": 2000000, "num_range": 40, "upper_bound": "\\u001f[FtA\\t k"}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 10, "upper_bound": "#)D\\u0016?z\\u0002\\t"}, {"distinct_range": 262262.6004775507, "num_eq": 9000000000, "num_range": 1000000, "upper_bound": "\'/o~n1nz"}, {"distinct_range": 1000000, "num_eq": 3624188814022955861, "num_range": 1000000, "upper_bound": "3\\u0011N\\n)]"}, {"distinct_range": 92.74379972308438, "num_eq": 5437251587397388533, "num_range": 200, "upper_bound": "9\\u00132,"}, {"distinct_range": 100000, "num_eq": 20000, "num_range": 100000, "upper_bound": "TwY"}, {"distinct_range": 25790510.662886064, "num_eq": 2000, "num_range": 60000000, "upper_bound": "r\\u0008WoM3\\u0007S"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 12, "row_count": 300}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9, "histo_col_type": "BIT(21)", "name": "__auto__", "null_count": 150, "row_count": 300}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42, "histo_buckets": [{"distinct_range": 0, "num_eq": 337018147758827177, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 1134284343827844200, "num_eq": 0, "num_range": 1134284343827844243, "upper_bound": "\\u000b"}, {"distinct_range": 4770175947488915000, "num_eq": 50000000000, "num_range": 8166110320266139150, "upper_bound": "\\u0015"}, {"distinct_range": 9000000000, "num_eq": 7274148645685251264, "num_range": 9000000000, "upper_bound": "0"}, {"distinct_range": 250136.21599182067, "num_eq": 10000000, "num_range": 1000000, "upper_bound": "P"}, {"distinct_range": 10000000, "num_eq": 0, "num_range": 10000000, "upper_bound": "Q"}, {"distinct_range": 0, "num_eq": 90000000000, "num_range": 8511282710373910007, "upper_bound": "W"}, {"distinct_range": 0, "num_eq": 3733591865084810306, "num_range": 6000, "upper_bound": "i"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 300}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 195, "histo_col_type": "", "name": "__auto__", "null_count": 264, "row_count": 300}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 830333822941949063, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 3000, "num_range": 1968825754855850094, "upper_bound": "8\\u0018N:]nK\\\\"}, {"distinct_range": 896148.9278050507, "num_eq": 4693379575529141575, "num_range": 900000, "upper_bound": "T_,P&%z\\rq"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 8928755400698635892, "num_range": 0, "upper_bound": "-49 years -1 mons -154 days -06:41:36.637545"}, {"distinct_range": 0, "num_eq": 2353481284694875106, "num_range": 10000000000, "upper_bound": "-23 years -11 mons -662 days -17:00:18.61756"}, {"distinct_range": 0, "num_eq": 100, "num_range": 10, "upper_bound": "-14 years -6 mons -519 days -18:56:51.221986"}, {"distinct_range": 7100751911772470000, "num_eq": 6283763038309754012, "num_range": 7100751911772470312, "upper_bound": "4 mons 447 days 18:49:06.010636"}, {"distinct_range": 0, "num_eq": 24789110317497635, "num_range": 10, "upper_bound": "52 years 1 mon 510 days 11:25:25.194273"}, {"distinct_range": 0, "num_eq": 651297074546207494, "num_range": 100000, "upper_bound": "54 years 7 mons 231 days 04:10:19.114309"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "-66256671"}, {"distinct_range": 9134194937640062000, "num_eq": 9216020448817197644, "num_range": 9134194937640062215, "upper_bound": "-1"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000000, "num_range": 0, "upper_bound": "0.2986387843343467"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 0, "upper_bound": "0.8853194332163723"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "1335524197"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('km');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('lk', 'fonh', 'brefam', 'f');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('rnjxu', 'fd', 'ledmme', 'b');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2483691477077173355, "histo_col_type": "", "name": "__auto__", "null_count": 2518937410818115979, "row_count": 3880281166295587568}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2903385260892617144, "histo_col_type": "", "name": "__auto__", "null_count": 2267022397442515314, "row_count": 3880281166295587568}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2624717393913842736, "histo_col_type": "", "name": "__auto__", "null_count": 3119170721471217304, "row_count": 3880281166295587568}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3588328175255491679, "histo_col_type": "", "name": "__auto__", "null_count": 733150059614152475, "row_count": 3880281166295587568}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1261182549392224516, "histo_col_type": "", "name": "__auto__", "null_count": 3869776998576979805, "row_count": 3880281166295587568}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 977685569083259236, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000, "num_range": 0, "upper_bound": "\\u0012"}, {"distinct_range": 0, "num_eq": 5305114216036248750, "num_range": 6587315166013549256, "upper_bound": ";"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 3880281166295587568}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1324959052528088119, "histo_col_type": "", "name": "__auto__", "null_count": 1874475759078824651, "row_count": 3880281166295587568}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 377553718976561846, "histo_buckets": [{"distinct_range": 0, "num_eq": 5229675457142769512, "num_range": 0, "upper_bound": "-23034"}, {"distinct_range": 8071342263326216000, "num_eq": 4469022458739075108, "num_range": 8071342263326216180, "upper_bound": "-15793"}, {"distinct_range": 35427920.842096806, "num_eq": 691280322780006452, "num_range": 60000000, "upper_bound": "-11287"}, {"distinct_range": 0, "num_eq": 4492309140130975706, "num_range": 700, "upper_bound": "-6293"}, {"distinct_range": 0, "num_eq": 200, "num_range": 70000000000, "upper_bound": "3968"}, {"distinct_range": 0, "num_eq": 8659924973272894377, "num_range": 100000, "upper_bound": "4573"}, {"distinct_range": 22916613006.62298, "num_eq": 5578949155687096490, "num_range": 30000000000, "upper_bound": "6482"}, {"distinct_range": 2551804437846910000, "num_eq": 627632886571140905, "num_range": 4361064283306652364, "upper_bound": "9460"}, {"distinct_range": 45.61448429927458, "num_eq": 5579121460326866682, "num_range": 70, "upper_bound": "18952"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 1423647197346183352, "row_count": 3880281166295587568}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 427751246995407895, "histo_col_type": "", "name": "__auto__", "null_count": 2488319311251981967, "row_count": 4294056465463067021}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2589387126329710113, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4294056465463067021}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2678043355916448987, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000, "num_range": 0, "upper_bound": "-1.2150810131540437"}, {"distinct_range": 8000, "num_eq": 6995072551228138158, "num_range": 8000, "upper_bound": "-0.6475267854680886"}, {"distinct_range": 0, "num_eq": 6310327323481099574, "num_range": 449180263509189052, "upper_bound": "-0.09709197688650334"}, {"distinct_range": 0, "num_eq": 1023226882356165670, "num_range": 10, "upper_bound": "0.19866760474257"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 0, "upper_bound": "1.5152433082860663"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 4294056465463067021}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3180357390766214552, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4294056465463067021}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1166701810842548069, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4294056465463067021}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4173449179307151557, "histo_buckets": [{"distinct_range": 0, "num_eq": 800, "num_range": 0, "upper_bound": "\\u0008"}, {"distinct_range": 0, "num_eq": 1456737191961874185, "num_range": 500, "upper_bound": "`"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 4294056465463067021}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 846, "histo_buckets": [{"distinct_range": 0, "num_eq": 1435836640416522179, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 800000, "num_range": 30000000, "upper_bound": "\\u0001\x7f %]wR"}, {"distinct_range": 9000000000, "num_eq": 100000000, "num_range": 9000000000, "upper_bound": "\\u0014H"}, {"distinct_range": 5904143171071657000, "num_eq": 10000, "num_range": 5904143171071657268, "upper_bound": "\\u0017^\\u0007h"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 27253609278312099, "upper_bound": "\\""}, {"distinct_range": 0, "num_eq": 500000, "num_range": 3706542994857662327, "upper_bound": "Zs)#Ppw\\u000c\\u000c"}, {"distinct_range": 688437971291461800, "num_eq": 367983753241358720, "num_range": 3028065561047783771, "upper_bound": "d,\\u000eZ\\u0018g<!"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 31, "row_count": 900}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 206, "histo_col_type": "", "name": "__auto__", "null_count": 180, "row_count": 900}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 260, "histo_buckets": [{"distinct_range": 0, "num_eq": 3932032413935590342, "num_range": 0, "upper_bound": "\\u001b=\\u0007L"}, {"distinct_range": 460564257.919365, "num_eq": 60, "num_range": 500000000, "upper_bound": "\\u001e\\\\iYf\\u00036\\\\T"}, {"distinct_range": 0, "num_eq": 0, "num_range": 3480557745727561320, "upper_bound": "&"}, {"distinct_range": 0, "num_eq": 0, "num_range": 828729044177207538, "upper_bound": "(%QK8\\u001eU"}, {"distinct_range": 5780334843147334000, "num_eq": 100, "num_range": 5780334843147333730, "upper_bound": ">j"}, {"distinct_range": 6052019.425452117, "num_eq": 4324766371496617873, "num_range": 8000000, "upper_bound": "Gpj"}, {"distinct_range": 6713.082080380842, "num_eq": 6420537088479230330, "num_range": 10000, "upper_bound": "Oh!4}Jb+v"}, {"distinct_range": 685222.7813865213, "num_eq": 8688488794268031372, "num_range": 700000, "upper_bound": "Ugo\\u0011"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 819, "row_count": 900}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 157, "histo_col_type": "", "name": "__auto__", "null_count": 352, "row_count": 900}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 456, "histo_buckets": [{"distinct_range": 0, "num_eq": 600, "num_range": 0, "upper_bound": "001011101000010010001"}, {"distinct_range": 7000, "num_eq": 80000, "num_range": 7000, "upper_bound": "011101001101010111010"}, {"distinct_range": 0, "num_eq": 9110062213316892697, "num_range": 648929645812681507, "upper_bound": "100110011111110010010"}, {"distinct_range": 10000, "num_eq": 2195763377104718203, "num_range": 10000, "upper_bound": "101011111100010001110"}, {"distinct_range": 0, "num_eq": 0, "num_range": 5064344386412580017, "upper_bound": "101110000010000101010"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 5776175626238139356, "upper_bound": "111011010100010000101"}], "histo_col_type": "BIT(21)", "name": "__auto__", "null_count": 719, "row_count": 900}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 108, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 357, "histo_buckets": [{"distinct_range": 0, "num_eq": 400, "num_range": 0, "upper_bound": ""}, {"distinct_range": 99.569791180178, "num_eq": 5517087057831535646, "num_range": 400, "upper_bound": ")"}, {"distinct_range": 357712.8824959677, "num_eq": 0, "num_range": 1000000, "upper_bound": "1"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 400000, "upper_bound": "3"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 70, "upper_bound": "F"}, {"distinct_range": 0, "num_eq": 300, "num_range": 6072674073437263623, "upper_bound": "y"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 788, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 8408980144035479914, "num_range": 1000000, "upper_bound": "1129306183"}, {"distinct_range": 3763462401458104000, "num_eq": 80000000000, "num_range": 3763462401458103745, "upper_bound": "2576169072"}, {"distinct_range": 0, "num_eq": 3482996388125535746, "num_range": 2889724475153535456, "upper_bound": "2947462546"}, {"distinct_range": 547260526000316860, "num_eq": 30, "num_range": 5042303626165034163, "upper_bound": "3156135032"}, {"distinct_range": 9000000000, "num_eq": 1748380717570306401, "num_range": 9000000000, "upper_bound": "3497414211"}, {"distinct_range": 90000, "num_eq": 8000, "num_range": 90000, "upper_bound": "4262665524"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 874, "row_count": 900}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 395, "histo_buckets": [{"distinct_range": 0, "num_eq": 5563910996315634517, "num_range": 0, "upper_bound": "-1.004012115272757"}, {"distinct_range": 0, "num_eq": 30000, "num_range": 6626048837735736546, "upper_bound": "-1.0"}, {"distinct_range": 647076441784547600, "num_eq": 5615784820737863251, "num_range": 743914295988307556, "upper_bound": "-0.338299149942214"}, {"distinct_range": 897598290.0684689, "num_eq": 5000000, "num_range": 1000000000, "upper_bound": "-0.22564979249940365"}, {"distinct_range": 4103017288588020700, "num_eq": 797029088982409122, "num_range": 4103017288588020803, "upper_bound": "0.36196888284063666"}, {"distinct_range": 0, "num_eq": 0, "num_range": 5251031502088251684, "upper_bound": "0.4228705180800294"}, {"distinct_range": 0, "num_eq": 2178077936221331492, "num_range": 4790379452070560559, "upper_bound": "1.0921347785093753"}, {"distinct_range": 493256893707928450, "num_eq": 3496854765352827232, "num_range": 493256893707928426, "upper_bound": "1.4163614709142127"}, {"distinct_range": 192386482.7096071, "num_eq": 900000, "num_range": 400000000, "upper_bound": "2.759085009860256"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 900}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 220802063673751129, "histo_col_type": "", "name": "__auto__", "null_count": 3979137646850307064, "row_count": 4619993946099933733}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3144778260996436075, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 9138946588566294000, "num_eq": 6703575385759375754, "num_range": 9138946588566293240, "upper_bound": "&?7k\\u0016Y5g"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 6536347790538866625, "upper_bound": "\'"}, {"distinct_range": 9000000, "num_eq": 509627642707327561, "num_range": 9000000, "upper_bound": "4f"}, {"distinct_range": 0, "num_eq": 4686940869301813335, "num_range": 337884759663148270, "upper_bound": "8\\u001b`0"}, {"distinct_range": 445772624890525000, "num_eq": 7979475242571499120, "num_range": 445772624890524970, "upper_bound": "?"}, {"distinct_range": 0, "num_eq": 1518738514313050564, "num_range": 90000000000, "upper_bound": "BR\\u0008\\\\"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 7749448681775530904, "upper_bound": "r\\u0005=\\u001ex:7"}, {"distinct_range": 1140719537498325400, "num_eq": 7212788185296749064, "num_range": 1970471620786401948, "upper_bound": "\U00002603"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 207431054392090509, "row_count": 4619993946099933733}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 935162166408941968, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4619993946099933733}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 683232856496400134, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 5126799794370764678, "num_range": 900000000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 4619993946099933733}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2605242404731375042, "histo_buckets": [{"distinct_range": 0, "num_eq": 7187671970622903157, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 1015495.2888088746, "num_eq": 800, "num_range": 3000000, "upper_bound": "444930683"}, {"distinct_range": 4989.957562017583, "num_eq": 400000000, "num_range": 7000, "upper_bound": "1366828268"}, {"distinct_range": 216740016296308480, "num_eq": 3195663207451059943, "num_range": 216740016296308470, "upper_bound": "1543354014"}, {"distinct_range": 2383463100892220000, "num_eq": 7609070173393791929, "num_range": 2383463100892220110, "upper_bound": "1743003762"}, {"distinct_range": 0, "num_eq": 2000000, "num_range": 1080641382865030662, "upper_bound": "3495836928"}, {"distinct_range": 0, "num_eq": 895861934687543500, "num_range": 5626645968999997504, "upper_bound": "3586566952"}, {"distinct_range": 0, "num_eq": 2516281641436174837, "num_range": 0, "upper_bound": "3678840930"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 100000000, "upper_bound": "3686601079"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 4619993946099933733}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2907136102829236724, "histo_buckets": [{"distinct_range": 0, "num_eq": 900, "num_range": 0, "upper_bound": "16 years 9 mons 913 days 19:09:28.31983"}, {"distinct_range": 165949522079279840, "num_eq": 100000000, "num_range": 165949522079279833, "upper_bound": "19 years 10 mons 467 days 22:03:57.365877"}, {"distinct_range": 123845879.78762351, "num_eq": 4707804880521206139, "num_range": 400000000, "upper_bound": "42 years 10 mons 290 days 06:07:50.029804"}, {"distinct_range": 6659721674485709000, "num_eq": 40000000000, "num_range": 6659721674485708855, "upper_bound": "54 years 7 mons 638 days 20:35:23.851669"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 4619993946099933733}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 437768928975747121, "histo_col_type": "", "name": "__auto__", "null_count": 3223852693265510379, "row_count": 4619993946099933733}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 913186706505338035, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4619993946099933733}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2696078783598922195, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000000, "num_range": 0, "upper_bound": "-1.447693179051198"}, {"distinct_range": 7571869569369147000, "num_eq": 90000000000, "num_range": 7571869569369147388, "upper_bound": "-1.3662433848191347"}, {"distinct_range": 266582.23096368933, "num_eq": 1028629167642472808, "num_range": 1000000, "upper_bound": "-0.3245741959800954"}, {"distinct_range": 30000000, "num_eq": 6000, "num_range": 30000000, "upper_bound": "0.01529875779583012"}, {"distinct_range": 43638658398.562996, "num_eq": 70000000, "num_range": 60000000000, "upper_bound": "0.6693058131954333"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 700, "upper_bound": "1.0882387011220644"}, {"distinct_range": 8618417162197993000, "num_eq": 7368463174604728665, "num_range": 8618417162197993566, "upper_bound": "1.2424631934769528"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 4619993946099933733}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 890932663126519460, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4619993946099933733}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4449124276439217617, "histo_buckets": [{"distinct_range": 0, "num_eq": 2759819391519475906, "num_range": 0, "upper_bound": "392643076"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 914267368352877003, "row_count": 4619993946099933733}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1287770185216736404, "histo_col_type": "", "name": "__auto__", "null_count": 2525302435542999046, "row_count": 4619993946099933733}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2563536114060869739, "histo_col_type": "", "name": "__auto__", "null_count": 2566362549773672322, "row_count": 4619993946099933733}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 598894222032060790, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4619993946099933733}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1131724280082067938, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4619993946099933733}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2979338555827002319, "histo_col_type": "", "name": "__auto__", "null_count": 4385384098486838491, "row_count": 4619993946099933733}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 620477082902042202, "histo_col_type": "", "name": "__auto__", "null_count": 2330082034542925216, "row_count": 4619993946099933733}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 633481618878866624, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4619993946099933733}]';
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_3, col2_1, col2_5, col2_0) REFERENCES table4 (col4_4, col4_10, col2_5, col4_8) ON UPDATE RESTRICT;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_2, col2_4) REFERENCES table3 (col1_0, col3_7) ON DELETE RESTRICT;
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2828622723799284058, "histo_buckets": [{"distinct_range": 0, "num_eq": 4648720849799885580, "num_range": 0, "upper_bound": "\\""}, {"distinct_range": 5707989.021015856, "num_eq": 200000000, "num_range": 7000000, "upper_bound": "\'"}, {"distinct_range": 1660042680198662400, "num_eq": 10000000000, "num_range": 1660042680198662473, "upper_bound": "/"}, {"distinct_range": 1422481334831278600, "num_eq": 40, "num_range": 2380223643694453258, "upper_bound": "4"}, {"distinct_range": 81442243.58148643, "num_eq": 5000000, "num_range": 900000000, "upper_bound": ":"}, {"distinct_range": 1131656945387086200, "num_eq": 30000000000, "num_range": 1131656945387086168, "upper_bound": "X"}, {"distinct_range": 5247041018187998000, "num_eq": 300000000, "num_range": 5247041018187997821, "upper_bound": "p"}, {"distinct_range": 23229622617.946278, "num_eq": 0, "num_range": 60000000000, "upper_bound": "z"}, {"distinct_range": 300000000, "num_eq": 30, "num_range": 300000000, "upper_bound": "~"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 3801922769226225783}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 57102167022946076, "histo_col_type": "", "name": "__auto__", "null_count": 2618655961249638525, "row_count": 3801922769226225783}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 518485994548131460, "histo_buckets": [{"distinct_range": 0, "num_eq": 3109435773896339186, "num_range": 0, "upper_bound": "-32711"}, {"distinct_range": 1088404494513648500, "num_eq": 10000000, "num_range": 1703493416248459025, "upper_bound": "-27514"}, {"distinct_range": 0, "num_eq": 10, "num_range": 5000, "upper_bound": "-10735"}, {"distinct_range": 0, "num_eq": 3705349247166381356, "num_range": 3990809695570465194, "upper_bound": "-5398"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 90000000, "upper_bound": "0"}, {"distinct_range": 416728332328344200, "num_eq": 0, "num_range": 3340090445479542738, "upper_bound": "6343"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 95095053825788265, "row_count": 3801922769226225783}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2156687142374659168, "histo_col_type": "", "name": "__auto__", "null_count": 1339106891594738910, "row_count": 3801922769226225783}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1241494239546981811, "histo_col_type": "", "name": "__auto__", "null_count": 1925135128639745172, "row_count": 3801922769226225783}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 252277256982474474, "histo_col_type": "", "name": "__auto__", "null_count": 1001253304731009520, "row_count": 3801922769226225783}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 850906479649542506, "histo_col_type": "", "name": "__auto__", "null_count": 2627882525094548951, "row_count": 3801922769226225783}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3243664139263216676, "histo_col_type": "", "name": "__auto__", "null_count": 2117847563964040798, "row_count": 3801922769226225783}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28300, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34050, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 9000000000, "num_range": 5554287251525507174, "upper_bound": "\\u001b"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 3000000, "upper_bound": "\\""}, {"distinct_range": 194047714174517540, "num_eq": 8668403233163764436, "num_range": 1451931082509146931, "upper_bound": "+"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 8807844673892821111, "upper_bound": "Q"}, {"distinct_range": 393300695.7531197, "num_eq": 50000000, "num_range": 2000000000, "upper_bound": "W"}, {"distinct_range": 0, "num_eq": 5000000000, "num_range": 30000000000, "upper_bound": "y"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15832, "histo_col_type": "", "name": "__auto__", "null_count": 9123, "row_count": 40000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14906, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33850, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000, "num_range": 0, "upper_bound": "-0.7470721096565589"}, {"distinct_range": 1113403905384481300, "num_eq": 0, "num_range": 1113403905384481269, "upper_bound": "-0.4064645686364129"}, {"distinct_range": 5789170831605593000, "num_eq": 9007873747005670491, "num_range": 5789170831605593202, "upper_bound": "0.5574507012254117"}, {"distinct_range": 600000000, "num_eq": 70000, "num_range": 600000000, "upper_bound": "0.8729319657350995"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 1000, "upper_bound": "0.9579003197795544"}, {"distinct_range": 182433.96591473903, "num_eq": 70, "num_range": 3000000, "upper_bound": "1.4507037216882588"}, {"distinct_range": 69195720.10725825, "num_eq": 300000000, "num_range": 2000000000, "upper_bound": "1.5612857309482386"}, {"distinct_range": 0, "num_eq": 4598298311467324744, "num_range": 2000000000, "upper_bound": "1.5892747092340207"}, {"distinct_range": 0, "num_eq": 7809657668275471682, "num_range": 2731044055752783375, "upper_bound": "1.7976931348623157e+308"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10612, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 167353986, "histo_col_type": "", "name": "__auto__", "null_count": 312518649, "row_count": 400000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 382613881, "histo_buckets": [{"distinct_range": 0, "num_eq": 1356259397681742415, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 4706017864079281000, "num_eq": 5795870172657273272, "num_range": 4706017864079281639, "upper_bound": "1769723361"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 124601740, "row_count": 400000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21078847, "histo_buckets": [{"distinct_range": 0, "num_eq": 2746588349857594153, "num_range": 0, "upper_bound": "\\u0003\\u000b]N\\u0001[&"}, {"distinct_range": 6000000000, "num_eq": 3112258480861071417, "num_range": 6000000000, "upper_bound": "&"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 4625802885118234580, "upper_bound": "5v%\\u000b@"}, {"distinct_range": 0, "num_eq": 10, "num_range": 7978411125743279808, "upper_bound": "Ou4"}, {"distinct_range": 2036948287467493600, "num_eq": 5640197121736751832, "num_range": 3518201734265644211, "upper_bound": "Pi!5u\\u0014g"}, {"distinct_range": 35643988.69979794, "num_eq": 300, "num_range": 100000000, "upper_bound": "]\\u0016>Jb"}, {"distinct_range": 2511244149454253000, "num_eq": 500000000, "num_range": 2511244149454252932, "upper_bound": "m"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 67347462, "row_count": 400000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27637539, "histo_col_type": "", "name": "__auto__", "null_count": 176403662, "row_count": 400000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 236098173, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "011101010001111111110"}], "histo_col_type": "BIT(21)", "name": "__auto__", "null_count": 157744321, "row_count": 400000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 324316606, "histo_buckets": [{"distinct_range": 0, "num_eq": 2301308246706411234, "num_range": 0, "upper_bound": "-1.9060319977112437"}, {"distinct_range": 100, "num_eq": 1634804816580256031, "num_range": 100, "upper_bound": "-1.6009169629931441"}, {"distinct_range": 1076963229599452000, "num_eq": 8000000, "num_range": 1076963229599452075, "upper_bound": "0.45975296911949426"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 161258145, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22760496, "histo_buckets": [{"distinct_range": 0, "num_eq": 8942207712277652237, "num_range": 0, "upper_bound": "A"}, {"distinct_range": 0, "num_eq": 600, "num_range": 9000, "upper_bound": "E"}, {"distinct_range": 26991.358066674, "num_eq": 395505779553285478, "num_range": 30000, "upper_bound": "Y"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 400000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 286889702, "histo_buckets": [{"distinct_range": 0, "num_eq": 5956589906881267698, "num_range": 0, "upper_bound": ""}, {"distinct_range": 24307828838.279896, "num_eq": 399972058400230519, "num_range": 40000000000, "upper_bound": "-Xs\'cF\\u001f"}, {"distinct_range": 14.46316288588999, "num_eq": 8000000000, "num_range": 30, "upper_bound": "WB\\\\\\u0015\\u000en"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 265403111, "row_count": 400000000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000000, "num_range": 0, "upper_bound": "\\u0007Z"}, {"distinct_range": 0, "num_eq": 80, "num_range": 0, "upper_bound": "*qk]J<\'"}, {"distinct_range": 6823251647386664000, "num_eq": 1000000000, "num_range": 6823251647386664432, "upper_bound": "X"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 7369910086432985901, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 3639940575131949600, "num_eq": 30, "num_range": 4383832852502840318, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 6625518770391638516, "num_range": 0, "upper_bound": "1 day"}, {"distinct_range": 299155.04405996075, "num_eq": 7332868042563335082, "num_range": 600000, "upper_bound": "1 year 456 days 01:42:44.486298"}, {"distinct_range": 0, "num_eq": 40000000000, "num_range": 80, "upper_bound": "9 years 7 mons 902 days 20:15:43.5721"}, {"distinct_range": 4.932632024791815, "num_eq": 1559254737910526006, "num_range": 70, "upper_bound": "20 years 11 mons 242 days 16:21:19.969728"}, {"distinct_range": 0, "num_eq": 0, "num_range": 700000000, "upper_bound": "31 years 6 mons 771 days 14:50:42.43029"}, {"distinct_range": 0, "num_eq": 3246176573093998066, "num_range": 5778664586413829417, "upper_bound": "45 years 6 mons 419 days 21:27:25.739639"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "-2.055452425212967"}, {"distinct_range": 3119816298957675000, "num_eq": 973957300894635195, "num_range": 3119816298957674952, "upper_bound": "-1.5616720099808654"}, {"distinct_range": 0, "num_eq": 5212585092587586452, "num_range": 0, "upper_bound": "-0.4500422628160562"}, {"distinct_range": 553.6006890140976, "num_eq": 4000, "num_range": 900, "upper_bound": "0.6377734752563948"}, {"distinct_range": 1136803790582031200, "num_eq": 2000000000, "num_range": 1362979078985534866, "upper_bound": "0.72724027696557"}, {"distinct_range": 25740116.83962939, "num_eq": 8000, "num_range": 400000000, "upper_bound": "0.8329969837933086"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 80000000000, "upper_bound": "1009933327"}, {"distinct_range": 0, "num_eq": 700000, "num_range": 60, "upper_bound": "1403583597"}, {"distinct_range": 438.91783036396004, "num_eq": 5693580924677532375, "num_range": 700, "upper_bound": "2127652288"}, {"distinct_range": 46249076.514899045, "num_eq": 20000000000, "num_range": 200000000, "upper_bound": "2788720372"}, {"distinct_range": 90000000000, "num_eq": 3000000, "num_range": 90000000000, "upper_bound": "2796189523"}, {"distinct_range": 0.3644121087903606, "num_eq": 500000, "num_range": 30, "upper_bound": "4289641224"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: there is no unique constraint matching given keys for referenced table table4 (SQLSTATE 23503)
        test_1        | FAIL
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down....2021-03-01 06:11:26.179 UTC [48] LOG:  received fast shutdown request
        postgres_1    | 2021-03-01 06:11:26.181 UTC [48] LOG:  aborting any active transactions
        postgres_1    | 2021-03-01 06:11:26.183 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2021-03-01 06:11:26.191 UTC [50] LOG:  shutting down
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2733651&tab=buildLog) on [master@9ba48738bc511ad6954682cab41e23b8492facd8](https://github.com/cockroachdb/cockroach/commits/9ba48738bc511ad6954682cab41e23b8492facd8):

```
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        test_1        | --- FAIL: TestCompare (1.72s)
        test_1        |     --- FAIL: TestCompare/postgres (1.02s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.70s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 REGNAMESPACE, col1_1 VARCHAR NULL, col1_2 INT4 NOT NULL, col1_3 REGPROC NOT NULL, col1_4 "char", col1_5 UUID, col1_6 FLOAT4 NOT NULL, col1_7 STRING NOT NULL, PRIMARY KEY (col1_2 ASC), UNIQUE (col1_1, col1_0, col1_2 ASC, col1_7 DESC, col1_5, col1_6, col1_4, col1_3 DESC) WHERE ((((table1.col1_4 >= e'\U00002603':::STRING) OR (table1.col1_2 >= (-32768):::INT8)) AND (table1.col1_6 = '+Inf':::FLOAT8)) OR (table1.col1_1 >= '':::STRING)) OR (table1.col1_7 <= e'\x00':::STRING), INDEX (col1_3 ASC, col1_6 DESC, col1_4 DESC, col1_7 ASC, col1_1 DESC, col1_2, col1_0 DESC, col1_5 ASC) WHERE table1.col1_6 >= 'NaN':::FLOAT8, UNIQUE (col1_6, col1_1 ASC, col1_7 DESC) WHERE ((((table1.col1_1 > '"') AND (table1.col1_6 <= -1.0)) AND (table1.col1_7 >= 'X')) AND (table1.col1_2 <= 2147483647)) AND (table1.col1_4 <= e'\''), INDEX (col1_3, col1_7 DESC, col1_0 DESC, col1_5 DESC, col1_6 DESC, col1_1 ASC, col1_2 ASC) WHERE ((((table1.col1_6 != '+Inf':::FLOAT8) OR (table1.col1_4 > e'\'':::STRING)) OR (table1.col1_7 >= 'X':::STRING)) OR (table1.col1_2 >= 32767:::INT8)) OR (table1.col1_1 < '"':::STRING), INDEX (col1_7 DESC, col1_6 ASC, col1_1 ASC, col1_5 DESC, col1_0 DESC, col1_4 DESC, col1_2) WHERE ((((table1.col1_1 != '"':::STRING) AND (table1.col1_7 > e'\x00':::STRING)) OR (table1.col1_4 > e'\'':::STRING)) AND (table1.col1_6 >= 3.4028234663852886e+38:::FLOAT8)) OR (table1.col1_2 != 32767:::INT8), UNIQUE (col1_7, col1_6, col1_1 DESC, col1_2, col1_4 DESC, col1_0 ASC) STORING (col1_3), UNIQUE (col1_4, col1_2, col1_6 ASC, col1_0 DESC) WHERE ((((table1.col1_2 != (-32768):::INT8) OR (table1.col1_7 > e'\U00002603':::STRING)) OR (table1.col1_6 < 1.0:::FLOAT8)) AND (table1.col1_1 = e'\U00002603':::STRING)) AND (table1.col1_4 > e'\U00002603':::STRING), INDEX (col1_7, col1_0 ASC, col1_1, col1_5, col1_2 DESC, col1_3, col1_4) STORING (col1_6), UNIQUE (col1_1) WHERE ((((table1.col1_7 > e'\'') OR (table1.col1_1 > '"')) AND (table1.col1_6 != 0.0)) OR (table1.col1_2 > -32768)) AND (table1.col1_4 <= ''), FAMILY (col1_2, col1_4, col1_5), FAMILY (col1_3), FAMILY (col1_7), FAMILY (col1_0, col1_1, col1_6));
        test_1        |             CREATE TABLE table2 (col1_2 INT4 NOT NULL, col2_1 FLOAT4, col2_2 DECIMAL NOT NULL, col2_3 REGTYPE, col2_4 TIMETZ, col2_5 NAME, col2_6 INT4 NULL, col2_7 CHAR, col2_8 INT8 NULL, col2_9 TIMESTAMP NOT NULL, col2_10 TIME NOT NULL, col2_11 INTERVAL, col2_12 BOX2D NOT NULL, col2_13 BIT(3) NULL, col2_14 TIMESTAMP, PRIMARY KEY (col1_2 ASC, col2_7 ASC, col2_2, col2_13, col2_9 ASC, col2_3 DESC, col2_12, col2_6 DESC, col2_10 ASC, col2_4), INDEX (col2_7, col1_2 ASC, col2_1 DESC, col2_5, col2_11 ASC, col2_4, col2_10, col2_3 ASC, col2_8 ASC, col2_12 ASC, col2_2 ASC) STORING (col2_14), UNIQUE (col2_5 DESC, col2_12 DESC, col1_2 ASC, col2_8 ASC, col2_13 DESC, col2_7 ASC) STORING (col2_11), UNIQUE (col2_6, col2_5 DESC, col2_13 ASC, col2_11, col2_1, col2_9 DESC, col1_2, col2_7, col2_2, col2_8, col2_3 ASC, col2_12 DESC, col2_10 DESC, col2_14 DESC, col2_4 DESC) WHERE table2.col2_4 != '00:00:00+15:59:00':::TIMETZ, INDEX (col2_4 DESC, col2_1 ASC, col2_5 DESC, col2_2 ASC, col2_13 ASC) STORING (col2_8) WHERE ((table2.col2_6 <= -2147483648) AND (table2.col2_1 >= 0.0)) OR (table2.col2_7 >= e'\U00002603'), INDEX (col1_2 ASC, col2_3, col2_1 ASC, col2_7 DESC, col2_6, col2_10 DESC, col2_12 ASC, col2_9 DESC, col2_4 DESC, col2_14 DESC, col2_2 DESC, col2_11 DESC), UNIQUE (col2_10 ASC, col2_3, col2_9 DESC, col2_4, col2_5 ASC, col1_2, col2_13, col2_1, col2_2, col2_12 ASC, col2_8 ASC, col2_6 ASC, col2_14 DESC) STORING (col2_11) WHERE (table2.col2_6 < 127:::INT8) AND (table2.col2_14 != '-4713-11-24 00:00:00':::TIMESTAMP), INDEX (col2_4 DESC, col2_10, col2_13 DESC, col2_14 ASC) STORING (col2_8, col2_11) WHERE (((((table2.col2_6 >= 127:::INT8) OR (table2.col2_5 != 'X':::STRING)) OR (table2.col2_7 >= e'\U00002603':::STRING)) AND (table2.col2_9 < '-4713-11-24 00:00:00':::TIMESTAMP)) AND (table2.col2_14 < '3000-01-01 00:00:00':::TIMESTAMP)) OR (table2.col2_1 < '-Inf':::FLOAT8), INDEX (col2_8 ASC, col2_3 DESC, col2_12 ASC, col2_14, col2_13 DESC, col1_2 DESC, col2_10 DESC, col2_7 DESC, col2_5, col2_2 DESC, col2_1 ASC, col2_9 ASC, col2_4 ASC, col2_11 ASC) WHERE ((table2.col2_5 > '"':::STRING) AND (table2.col2_1 != 1.401298464324817e-45:::FLOAT8)) AND (table2.col2_6 != 127:::INT8), FAMILY (col2_4, col2_13, col2_11), FAMILY (col2_3), FAMILY (col2_1, col2_8), FAMILY (col2_14), FAMILY (col2_10), FAMILY (col2_7), FAMILY (col1_2), FAMILY (col2_9), FAMILY (col2_2), FAMILY (col2_6, col2_12, col2_5)) INTERLEAVE IN PARENT table1 (col1_2);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 6963948014077834731, "num_range": 10000, "upper_bound": ";}R\\u000e7!"}, {"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "Q"}, {"distinct_range": 26050.898904795173, "num_eq": 8000000000, "num_range": 800000, "upper_bound": "a@\\"Vlbvgn"}, {"distinct_range": 18858.058495808546, "num_eq": 10000, "num_range": 100000, "upper_bound": "bUL`<"}, {"distinct_range": 0, "num_eq": 3656145396920381629, "num_range": 0, "upper_bound": "fp"}, {"distinct_range": 50, "num_eq": 2087431631964667164, "num_range": 50, "upper_bound": "l"}, {"distinct_range": 8000000, "num_eq": 3518968043784861304, "num_range": 8000000, "upper_bound": "r"}, {"distinct_range": 0, "num_eq": 300000, "num_range": 2863003819501387131, "upper_bound": "s\\u0012\\u0001"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000000, "num_range": 0, "upper_bound": "-1884549698"}, {"distinct_range": 100000000, "num_eq": 301927373441450019, "num_range": 100000000, "upper_bound": "-1503839002"}, {"distinct_range": 665680604915507100, "num_eq": 40000, "num_range": 665680604915507085, "upper_bound": "-1264426399"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 3051281608282834695, "upper_bound": "-1156889542"}, {"distinct_range": 687330029986367000, "num_eq": 2066063187391536359, "num_range": 776922586266400038, "upper_bound": "-421532395"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 9148436210305894223, "upper_bound": "1236882288"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 2362169236403194421, "num_range": 0, "upper_bound": "93099022"}, {"distinct_range": 5964870796327122000, "num_eq": 10000, "num_range": 5964870796327121482, "upper_bound": "236253881"}, {"distinct_range": 1538435552149142800, "num_eq": 70, "num_range": 1538435552149142727, "upper_bound": "723234723"}, {"distinct_range": 30, "num_eq": 600000, "num_range": 30, "upper_bound": "903517232"}, {"distinct_range": 14499.153994443186, "num_eq": 1942317847309988836, "num_range": 40000, "upper_bound": "1310530164"}, {"distinct_range": 237758.6635027549, "num_eq": 20000, "num_range": 500000, "upper_bound": "2237674584"}, {"distinct_range": 138397825290637310, "num_eq": 300, "num_range": 9203969116276391885, "upper_bound": "3940348160"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 8353996865393133974, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 1087335573764233559, "num_range": 7767807217977830105, "upper_bound": "\\r"}, {"distinct_range": 0, "num_eq": 4201921426593994382, "num_range": 0, "upper_bound": "+"}, {"distinct_range": 17.17317162196636, "num_eq": 5000000, "num_range": 50, "upper_bound": ";"}, {"distinct_range": 42.53655876215582, "num_eq": 294683028799823578, "num_range": 60, "upper_bound": "C"}, {"distinct_range": 0, "num_eq": 500, "num_range": 0, "upper_bound": "U"}, {"distinct_range": 5000000, "num_eq": 100000, "num_range": 5000000, "upper_bound": "a"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 56969002.983378306, "num_eq": 900, "num_range": 900000000, "upper_bound": "\\u000eH\\u001f"}, {"distinct_range": 5001653431590955000, "num_eq": 200000000, "num_range": 5001653431590955219, "upper_bound": "\\u0011"}, {"distinct_range": 6816442431216764000, "num_eq": 3524035138387473135, "num_range": 6816442431216763640, "upper_bound": ":."}, {"distinct_range": 0, "num_eq": 1000, "num_range": 101832541735074177, "upper_bound": "U"}, {"distinct_range": 0, "num_eq": 584467901588581041, "num_range": 7999739034866599428, "upper_bound": "o\\\\\\u001a\\u001fe\\u001a\\u0006"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1276130159261245792, "histo_buckets": [{"distinct_range": 0, "num_eq": 2605262841405942930, "num_range": 0, "upper_bound": "S"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 3396429349201978132, "row_count": 5446548090587796921}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3829550784557060700, "histo_buckets": [{"distinct_range": 0, "num_eq": 466887263062822649, "num_range": 0, "upper_bound": "06:49:57.393511"}, {"distinct_range": 8627888513010891000, "num_eq": 4087830171009086299, "num_range": 8627888513010890759, "upper_bound": "08:14:14.821047"}, {"distinct_range": 722.8435327368057, "num_eq": 100000000, "num_range": 1000, "upper_bound": "09:34:57.148784"}, {"distinct_range": 549005.4566642141, "num_eq": 60000, "num_range": 1000000, "upper_bound": "17:00:58.359131"}, {"distinct_range": 100000000000, "num_eq": 5000000, "num_range": 100000000000, "upper_bound": "23:52:04.634295"}, {"distinct_range": 0, "num_eq": 1779776346096968474, "num_range": 700000, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 5446548090587796921}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 269234133265380812, "histo_col_type": "", "name": "__auto__", "null_count": 5349920611990493764, "row_count": 5446548090587796921}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1680765584711744462, "histo_col_type": "", "name": "__auto__", "null_count": 3892456164424875930, "row_count": 5446548090587796921}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3187251597064762827, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000000, "num_range": 0, "upper_bound": "-1833484117"}, {"distinct_range": 0, "num_eq": 70000000, "num_range": 6556987124684233501, "upper_bound": "-5417000"}, {"distinct_range": 900, "num_eq": 6000, "num_range": 900, "upper_bound": "102005622"}, {"distinct_range": 33897196.34757869, "num_eq": 8645284131279515786, "num_range": 70000000, "upper_bound": "297297140"}, {"distinct_range": 2000000000, "num_eq": 50000000000, "num_range": 2000000000, "upper_bound": "751940484"}, {"distinct_range": 1800657580918613800, "num_eq": 7851528426789936392, "num_range": 1881773459101508111, "upper_bound": "1153898814"}, {"distinct_range": 8739903641154707000, "num_eq": 5000000, "num_range": 8739903641154707537, "upper_bound": "1199434039"}, {"distinct_range": 0, "num_eq": 6291052657348354611, "num_range": 5000000, "upper_bound": "1846783680"}, {"distinct_range": 145.7928280296532, "num_eq": 4000, "num_range": 900, "upper_bound": "2013747487"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 5446548090587796921}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4744519040001037873, "histo_col_type": "", "name": "__auto__", "null_count": 2450550740401890256, "row_count": 5446548090587796921}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5265293440098070031, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000, "num_range": 0, "upper_bound": "-688393082"}, {"distinct_range": 4000000000, "num_eq": 60, "num_range": 4000000000, "upper_bound": "-68210758"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 4297222607658250418, "row_count": 5446548090587796921}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2373648815011721974, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "-8711810649387156028"}, {"distinct_range": 3620083025002928000, "num_eq": 50000, "num_range": 3620083025002928186, "upper_bound": "-4502855385286162189"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 40000000, "upper_bound": "-1988786902935473008"}, {"distinct_range": 287.80475503474355, "num_eq": 9062613988713819524, "num_range": 10000, "upper_bound": "-1571388237865118743"}, {"distinct_range": 300000000, "num_eq": 40000, "num_range": 300000000, "upper_bound": "-1437496729822004155"}, {"distinct_range": 10000000, "num_eq": 3010984187048209414, "num_range": 10000000, "upper_bound": "4800690428873745927"}, {"distinct_range": 600000000, "num_eq": 1636371200669212501, "num_range": 600000000, "upper_bound": "5602802676015828668"}, {"distinct_range": 278040.7983079587, "num_eq": 5000000000, "num_range": 3000000, "upper_bound": "7477212277384922984"}, {"distinct_range": 825000958057349600, "num_eq": 30000000, "num_range": 825000958057349606, "upper_bound": "8498581827427228687"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 4912466327565536128, "row_count": 5446548090587796921}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2593777056408113239, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5446548090587796921}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5238561585712160589, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5446548090587796921}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1654315387743385125, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5446548090587796921}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4671086166978345466, "histo_col_type": "", "name": "__auto__", "null_count": 569925091505535998, "row_count": 5446548090587796921}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2015319004683663109, "histo_buckets": [{"distinct_range": 0, "num_eq": 7995253086190519812, "num_range": 0, "upper_bound": ""}, {"distinct_range": 3508317414248721000, "num_eq": 2564721760873220379, "num_range": 3508317414248721082, "upper_bound": "\\u0001w\\u000b"}, {"distinct_range": 106669830113619220, "num_eq": 10, "num_range": 106669830113619210, "upper_bound": "\\u0002V?"}, {"distinct_range": 30000, "num_eq": 90, "num_range": 30000, "upper_bound": "\\u0007@T6\\u0008{( "}, {"distinct_range": 0, "num_eq": 4115705473862241940, "num_range": 30000000, "upper_bound": "\\u0012-Lv"}, {"distinct_range": 181.86099685281619, "num_eq": 539212923512867072, "num_range": 1000, "upper_bound": "\\u001a2[}L\\u0019\\u0006."}, {"distinct_range": 3931960248520324600, "num_eq": 4000000000, "num_range": 5273359100455390437, "upper_bound": "\\""}, {"distinct_range": 0, "num_eq": 100, "num_range": 1480686782553731319, "upper_bound": "<k\\u0016tQ4+"}, {"distinct_range": 1776.3512635279349, "num_eq": 3126109012337807910, "num_range": 5000, "upper_bound": "S_4\\u0002L\\u000bE"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 404746098074539654, "row_count": 5446548090587796921}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3467552690374883086, "histo_buckets": [{"distinct_range": 0, "num_eq": 800875045111974919, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 0, "num_eq": 5073557152490350191, "num_range": 20, "upper_bound": "05:56:46.805861+11:57:00"}, {"distinct_range": 0, "num_eq": 50, "num_range": 800000000, "upper_bound": "20:34:08.716252-12:52:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 2616826194975488504, "row_count": 5446548090587796921}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4184300095469369354, "histo_col_type": "", "name": "__auto__", "null_count": 3674468133974158165, "row_count": 5446548090587796921}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('xztl', 'biq', 'b', 'ivcaf', 'lmra', 'zwlchu');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 453765115686296833, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000, "num_range": 0, "upper_bound": "-959088187"}, {"distinct_range": 0, "num_eq": 1883083775712020564, "num_range": 0, "upper_bound": "-895567303"}, {"distinct_range": 0, "num_eq": 4585969010769792967, "num_range": 1000000, "upper_bound": "-128"}, {"distinct_range": 0, "num_eq": 2992336900838123844, "num_range": 0, "upper_bound": "108136498"}, {"distinct_range": 0, "num_eq": 0, "num_range": 7352454754365342761, "upper_bound": "628833076"}, {"distinct_range": 5390478124085367000, "num_eq": 400000000, "num_range": 8714663268162491272, "upper_bound": "965197305"}, {"distinct_range": 0, "num_eq": 100, "num_range": 500000, "upper_bound": "2147483647"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 595659301100611950}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 234743748019278011, "histo_buckets": [{"distinct_range": 0, "num_eq": 6106400117913420231, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 80000000, "upper_bound": "542828976"}, {"distinct_range": 8000000, "num_eq": 5380506932409204723, "num_range": 8000000, "upper_bound": "1838688840"}, {"distinct_range": 549578370828141760, "num_eq": 3202058798273429424, "num_range": 4462214140471061482, "upper_bound": "2143000396"}, {"distinct_range": 6000, "num_eq": 3585938258552097942, "num_range": 6000, "upper_bound": "2558856587"}, {"distinct_range": 410.36141223389944, "num_eq": 5813847987723157253, "num_range": 500, "upper_bound": "4262741652"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 595659301100611950}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 217367742131864249, "histo_buckets": [{"distinct_range": 0, "num_eq": 1499963225511834813, "num_range": 0, "upper_bound": "\\n"}, {"distinct_range": 1000, "num_eq": 1000, "num_range": 1000, "upper_bound": "/"}, {"distinct_range": 0, "num_eq": 6699974595528913049, "num_range": 8041429072565234511, "upper_bound": "J"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 178481159537361233, "row_count": 595659301100611950}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 522959607752117235, "histo_col_type": "", "name": "__auto__", "null_count": 592207622734876716, "row_count": 595659301100611950}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 504092557245547065, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000, "num_range": 0, "upper_bound": "NaN"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 100000000000, "upper_bound": "-0.1406821757555008"}, {"distinct_range": 0, "num_eq": 6029147696935379079, "num_range": 6659308634669100144, "upper_bound": "0.6422064304351807"}, {"distinct_range": 7000, "num_eq": 60000000000, "num_range": 7000, "upper_bound": "1.004150629043579"}, {"distinct_range": 0, "num_eq": 2671815417618523678, "num_range": 2959150315695205737, "upper_bound": "1.0936845541000366"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 595659301100611950}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 343790246937556768, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 8830620910031892000, "num_eq": 4000000000, "num_range": 8830620910031892781, "upper_bound": "\\u0001aD\\u0014T"}, {"distinct_range": 216247974.8281513, "num_eq": 1787953109633599766, "num_range": 5000000000, "upper_bound": "\\r1.wlP"}, {"distinct_range": 0, "num_eq": 9067446944565250678, "num_range": 0, "upper_bound": "\\u000e\x7f\\u0012za\\u001c"}, {"distinct_range": 1000000000, "num_eq": 2000000000, "num_range": 1000000000, "upper_bound": "\\u0017]\\u0004O"}, {"distinct_range": 0, "num_eq": 3359564668837384795, "num_range": 2307075787646953677, "upper_bound": "\\u0019@z/`O."}, {"distinct_range": 0, "num_eq": 5887590782379881725, "num_range": 10, "upper_bound": "\\u001dz)+"}, {"distinct_range": 11049533478709796, "num_eq": 0, "num_range": 11049533478709796, "upper_bound": ">yuo"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 595659301100611950}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 352079356910233839, "histo_col_type": "", "name": "__auto__", "null_count": 525766419249662334, "row_count": 595659301100611950}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 366916194722591564, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 65540758596041430, "num_eq": 1982190036083288093, "num_range": 2132201557134307576, "upper_bound": "{eP"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 23721783982344993, "row_count": 595659301100611950}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 923, "histo_buckets": [{"distinct_range": 0, "num_eq": 643461628220151126, "num_range": 0, "upper_bound": "-721052069"}, {"distinct_range": 0, "num_eq": 200000000, "num_range": 0, "upper_bound": "-409078944"}, {"distinct_range": 533484446477919360, "num_eq": 7438485663892900445, "num_range": 777750970423224449, "upper_bound": "127"}, {"distinct_range": 2353295683680734000, "num_eq": 6331906114331872314, "num_range": 2353295683680734328, "upper_bound": "671339171"}, {"distinct_range": 0, "num_eq": 4723201493544777528, "num_range": 4928191629280947871, "upper_bound": "882924382"}, {"distinct_range": 1313587417.741514, "num_eq": 3679358356408334633, "num_range": 2000000000, "upper_bound": "1264971893"}, {"distinct_range": 5790887089377349000, "num_eq": 500000, "num_range": 5790887089377348259, "upper_bound": "1396328319"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 219, "row_count": 1000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 685, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000000, "num_range": 0, "upper_bound": "-6024460937583378532"}, {"distinct_range": 0, "num_eq": 5073564537767441726, "num_range": 9000000000, "upper_bound": "19360656911905564"}, {"distinct_range": 900, "num_eq": 5339166961687458030, "num_range": 900, "upper_bound": "472698650502073603"}, {"distinct_range": 26394.774368104107, "num_eq": 2257222899459212371, "num_range": 40000, "upper_bound": "5155588139722608806"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 103, "row_count": 1000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 110, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000000, "num_range": 0, "upper_bound": "02:02:19.867081"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 70000, "upper_bound": "11:16:02.05916"}, {"distinct_range": 0, "num_eq": 5447789393472391616, "num_range": 0, "upper_bound": "14:40:25.146801"}, {"distinct_range": 0, "num_eq": 5107054978125278819, "num_range": 7219417322810039842, "upper_bound": "18:43:17.443073"}, {"distinct_range": 0, "num_eq": 500000000, "num_range": 7035215482683048077, "upper_bound": "23:11:24.356399"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 174, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 283, "histo_col_type": "", "name": "__auto__", "null_count": 80, "row_count": 1000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 987, "histo_col_type": "", "name": "__auto__", "null_count": 350, "row_count": 1000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 411, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000, "num_range": 0, "upper_bound": "%)j\\u0005R"}, {"distinct_range": 100000, "num_eq": 900, "num_range": 100000, "upper_bound": "\'<+/t"}, {"distinct_range": 0, "num_eq": 8342843793073321177, "num_range": 6261401169813281798, "upper_bound": ",\'(KXb;zq"}, {"distinct_range": 1640830197.859966, "num_eq": 600000, "num_range": 90000000000, "upper_bound": "."}, {"distinct_range": 0, "num_eq": 0, "num_range": 4877235083155696925, "upper_bound": "9`\\u001a-L."}, {"distinct_range": 2217250.750857577, "num_eq": 100000000, "num_range": 400000000, "upper_bound": "]I\\n"}, {"distinct_range": 1239911173984271600, "num_eq": 5128383284640478552, "num_range": 1392192940480315619, "upper_bound": "sa\\u000fW"}, {"distinct_range": 7538188673061338000, "num_eq": 6116172790034112845, "num_range": 9126873334515686837, "upper_bound": "u\\u0014\\u0014"}, {"distinct_range": 8550551716968063000, "num_eq": 100000, "num_range": 8550551716968063372, "upper_bound": "\U00002603"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 729, "row_count": 1000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 73, "histo_buckets": [{"distinct_range": 0, "num_eq": 2011347902003136509, "num_range": 0, "upper_bound": "2033700976"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 598, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000, "num_range": 0, "upper_bound": "03:52:37.099851+01:40:00"}, {"distinct_range": 0, "num_eq": 6021498369529494288, "num_range": 8383050689342788302, "upper_bound": "15:50:48.206083+08:59:00"}, {"distinct_range": 0, "num_eq": 0, "num_range": 600, "upper_bound": "23:29:59.034078-09:25:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 371, "row_count": 1000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 852, "histo_col_type": "CHAR", "name": "__auto__", "null_count": 983, "row_count": 1000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 203, "histo_col_type": "", "name": "__auto__", "null_count": 601, "row_count": 1000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 918, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 849, "histo_col_type": "", "name": "__auto__", "null_count": 384, "row_count": 1000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 768, "histo_col_type": "", "name": "__auto__", "null_count": 358, "row_count": 1000}]';
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_5) REFERENCES table1 (col1_1) ON UPDATE RESTRICT;
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28056985588, "histo_buckets": [{"distinct_range": 0, "num_eq": 40, "num_range": 0, "upper_bound": "0.451307475566864"}, {"distinct_range": 1821745416589211100, "num_eq": 1422036832884094453, "num_range": 1821745416589211147, "upper_bound": "0.6060988903045654"}, {"distinct_range": 600000, "num_eq": 715976002311684972, "num_range": 600000, "upper_bound": "0.7873647809028625"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 0, "upper_bound": "1.2855576276779175"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 40000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3187657594, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 3781921332669042098, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 5827187884650905000, "num_eq": 8434056839964357056, "num_range": 5827187884650904934, "upper_bound": "\\u000b8g\'"}, {"distinct_range": 60, "num_eq": 70000000, "num_range": 60, "upper_bound": "\\u0013Vc\\u0008\\u00024`"}, {"distinct_range": 0, "num_eq": 7941383197190505105, "num_range": 80000, "upper_bound": "\\""}, {"distinct_range": 0, "num_eq": 100000, "num_range": 20000000000, "upper_bound": "D\\u000f,Kd.>"}, {"distinct_range": 2056780934.0104122, "num_eq": 2000000, "num_range": 3000000000, "upper_bound": "cvN\\\\i9"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 40000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36039175710, "histo_col_type": "", "name": "__auto__", "null_count": 38594406944, "row_count": 40000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 381201748, "histo_buckets": [{"distinct_range": 0, "num_eq": 585812649406868915, "num_range": 0, "upper_bound": "\\""}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 60000000000, "upper_bound": "H["}, {"distinct_range": 1834456.4666037038, "num_eq": 3678965389889864615, "num_range": 2000000, "upper_bound": "n2"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 7149253754, "row_count": 40000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36877729320, "histo_buckets": [{"distinct_range": 0, "num_eq": 400, "num_range": 0, "upper_bound": "-1796109989"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 40000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24245098386, "histo_buckets": [{"distinct_range": 0, "num_eq": 2208719998692002317, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 1232583525695864600, "num_eq": 100000000000, "num_range": 4667940548985733446, "upper_bound": "422383799"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 400000, "upper_bound": "561087518"}, {"distinct_range": 0, "num_eq": 2000000, "num_range": 8000, "upper_bound": "783468101"}, {"distinct_range": 300000, "num_eq": 10000, "num_range": 300000, "upper_bound": "3234174925"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 40000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32099341500, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000, "num_range": 0, "upper_bound": ";"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 30054721878, "row_count": 40000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11149965157, "histo_col_type": "", "name": "__auto__", "null_count": 7009578532, "row_count": 40000000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2667863439290683285, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "-564371020"}, {"distinct_range": 700000000, "num_eq": 50000000000, "num_range": 700000000, "upper_bound": "-512707618"}, {"distinct_range": 75.33022182840435, "num_eq": 10000, "num_range": 100, "upper_bound": "1"}, {"distinct_range": 176392586979081630, "num_eq": 9000, "num_range": 176392586979081641, "upper_bound": "34873046"}, {"distinct_range": 0, "num_eq": 8288495567370350846, "num_range": 4020703950409904118, "upper_bound": "870317587"}, {"distinct_range": 10000000000, "num_eq": 2000, "num_range": 10000000000, "upper_bound": "1234183727"}, {"distinct_range": 899604903151472800, "num_eq": 6788786079354134101, "num_range": 899604903151472768, "upper_bound": "1334500060"}, {"distinct_range": 1033.2966436011388, "num_eq": 7584032899827702618, "num_range": 8000, "upper_bound": "1574663657"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 6541916595325834499}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4819849417020995629, "histo_col_type": "", "name": "__auto__", "null_count": 309407472713067650, "row_count": 6541916595325834499}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2560579382036008361, "histo_col_type": "", "name": "__auto__", "null_count": 1545151008345289865, "row_count": 6541916595325834499}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4657651919143455578, "histo_col_type": "", "name": "__auto__", "null_count": 6481952653145453672, "row_count": 6541916595325834499}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5742885667916029713, "histo_buckets": [{"distinct_range": 0, "num_eq": 3442698165604624121, "num_range": 0, "upper_bound": "-1817090870"}, {"distinct_range": 0, "num_eq": 3477873132181959743, "num_range": 80, "upper_bound": "-1658486376"}, {"distinct_range": 0, "num_eq": 40000000000, "num_range": 0, "upper_bound": "-630486383"}, {"distinct_range": 0, "num_eq": 9039539175252669875, "num_range": 0, "upper_bound": "-142512000"}, {"distinct_range": 3161507639270331400, "num_eq": 2423046102984895041, "num_range": 3161507639270331259, "upper_bound": "888624360"}, {"distinct_range": 0, "num_eq": 30000, "num_range": 10000000000, "upper_bound": "1234630454"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 3134334109724855587, "row_count": 6541916595325834499}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2386287052975412564, "histo_buckets": [{"distinct_range": 0, "num_eq": 8077333675505123805, "num_range": 0, "upper_bound": "-9140852972823377922"}, {"distinct_range": 50000, "num_eq": 300, "num_range": 50000, "upper_bound": "-6851627343666226253"}, {"distinct_range": 193691446.0976527, "num_eq": 100000000000, "num_range": 400000000, "upper_bound": "-6789466400367879114"}, {"distinct_range": 0, "num_eq": 900, "num_range": 900, "upper_bound": "0"}, {"distinct_range": 1000000000, "num_eq": 10, "num_range": 1000000000, "upper_bound": "127"}, {"distinct_range": 139787155639111260, "num_eq": 40000000, "num_range": 1513841047008249525, "upper_bound": "4023986846972651936"}, {"distinct_range": 7675865866002665000, "num_eq": 2929537274551132811, "num_range": 7675865866002665504, "upper_bound": "6835111068703889702"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 1461767920519195451, "row_count": 6541916595325834499}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4957406547170510145, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 1738849198867220008, "num_range": 1000000, "upper_bound": "\\u0002"}, {"distinct_range": 0, "num_eq": 9053719661715015369, "num_range": 0, "upper_bound": "\\u0007"}, {"distinct_range": 0, "num_eq": 685906809426462711, "num_range": 30000000000, "upper_bound": "\\u0008"}, {"distinct_range": 3227.219750821109, "num_eq": 7017756156163875872, "num_range": 10000, "upper_bound": "7"}, {"distinct_range": 17.201209568629565, "num_eq": 3043407421534777468, "num_range": 70, "upper_bound": "G"}, {"distinct_range": 506624255135745300, "num_eq": 1000000000, "num_range": 4964728718171761127, "upper_bound": "v"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 5012409462023180020, "row_count": 6541916595325834499}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2432447563966462200, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6541916595325834499}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4999133908017760558, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6541916595325834499}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28097787609394526, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000, "num_range": 0, "upper_bound": "00:43:57.742989+12:59:00"}, {"distinct_range": 7000, "num_eq": 0, "num_range": 7000, "upper_bound": "19:02:52.963153+05:27:00"}, {"distinct_range": 0, "num_eq": 2613835428574312214, "num_range": 80000, "upper_bound": "22:35:46.8642+07:46:00"}, {"distinct_range": 800000000, "num_eq": 0, "num_range": 800000000, "upper_bound": "12:12:41.743258-09:30:00"}, {"distinct_range": 0, "num_eq": 523182799886928963, "num_range": 60, "upper_bound": "19:07:54.444013-11:34:00"}, {"distinct_range": 6938572759062972000, "num_eq": 2000000, "num_range": 6938572759062972171, "upper_bound": "18:09:38.326813-13:50:00"}, {"distinct_range": 269095448637652380, "num_eq": 4331993810197492571, "num_range": 1358083751532234332, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 1867159809326470870, "row_count": 6541916595325834499}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4902943005386290288, "histo_buckets": [{"distinct_range": 0, "num_eq": 200000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 5.360723034159286, "num_eq": 30000000, "num_range": 10, "upper_bound": "\\""}, {"distinct_range": 246884.45262271824, "num_eq": 30000000, "num_range": 4000000, "upper_bound": "U2*j"}], "histo_col_type": "NAME", "name": "__auto__", "null_count": 4555213888172110697, "row_count": 6541916595325834499}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3619609747242018073, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000000, "num_range": 0, "upper_bound": "01:32:59.269848"}, {"distinct_range": 0, "num_eq": 4864212499799258307, "num_range": 60, "upper_bound": "01:38:08.256494"}, {"distinct_range": 24.85645225728455, "num_eq": 6000, "num_range": 60, "upper_bound": "11:26:07.575385"}, {"distinct_range": 40000, "num_eq": 500, "num_range": 40000, "upper_bound": "14:14:49.90281"}, {"distinct_range": 4382037836555514400, "num_eq": 1000, "num_range": 4382037836555514532, "upper_bound": "14:50:31.671457"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 6541916595325834499}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5846623978783422250, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6541916595325834499}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4015080733720727870, "histo_col_type": "", "name": "__auto__", "null_count": 3050550504614151728, "row_count": 6541916595325834499}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4122000547658021903, "histo_col_type": "", "name": "__auto__", "null_count": 5006628183571474110, "row_count": 6541916595325834499}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: there is no unique constraint matching given keys for referenced table table1 (SQLSTATE 23503)
        test_1        | FAIL
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2021-03-02 06:10:55.376 UTC [48] LOG:  received fast shutdown request
        postgres_1    | .2021-03-02 06:10:55.378 UTC [48] LOG:  aborting any active transactions
        postgres_1    | 2021-03-02 06:10:55.380 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2021-03-02 06:10:55.388 UTC [50] LOG:  shutting down
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2738826&tab=buildLog) on [master@6de4313ec216161c79fe725fcc31fc87ef1804ea](https://github.com/cockroachdb/cockroach/commits/6de4313ec216161c79fe725fcc31fc87ef1804ea):

```
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_0 ASC, col1_13 DESC, col1_8 ASC, col1_12, col1_5, col1_3, col1_7, col1_2 DESC, col1_11 DESC, col1_15 ASC, col1_6 DESC, col1_14 DESC, col1_10 DESC) INCLUDE (col1_18);
        test_1        |             CREATE TABLE table2 (col1_2 TIMESTAMPTZ NOT NULL, col1_0 INET NOT NULL, col1_17 TIMESTAMP NOT NULL, col1_14 OID NOT NULL, col1_13 TIMETZ NOT NULL, col1_4 REGTYPE NOT NULL, col1_8 TIMETZ NOT NULL, col1_10 INTERVAL NOT NULL, col1_6 BOX2D NOT NULL, col1_9 TEXT NOT NULL, col1_12 BOOL NOT NULL, col2_11 TIMESTAMP NOT NULL, col2_12 REGPROCEDURE, col2_13 REGCLASS NULL, col2_14 INT8 NOT NULL, col2_15 BIT(45), col2_16 DATE NULL, col2_17 UUID NULL, col2_18 DATE NOT NULL, PRIMARY KEY (col1_2, col1_0, col1_17, col1_14, col1_13, col1_4, col1_8, col1_10, col1_6, col1_9, col1_12, col2_15, col2_11, col2_14, col2_12, col2_16));
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_14 ASC, col1_9 DESC, col2_17 DESC, col2_11 ASC, col2_14, col2_12 DESC, col1_0, col1_10 ASC);
        test_1        |             CREATE INDEX ON table2 (col1_14 DESC, col1_4 DESC, col1_0 ASC, col2_13 DESC) INCLUDE (col2_17);
        test_1        |             CREATE INDEX ON table2 (col2_17 ASC, col2_12, col1_17, col2_18, col2_13 ASC, col1_8 ASC, col1_12);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_13 DESC, col1_10 DESC, col2_14, col2_15, col2_16 ASC, col2_17, col1_2 ASC, col1_8 DESC, col1_14 DESC, col2_12 DESC, col1_4, col1_12 ASC, col1_17 ASC, col2_18 ASC, col1_9 ASC, col1_0);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_13);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_17, col1_9);
        test_1        |             CREATE TABLE table3 (col1_2 TIMESTAMPTZ NOT NULL, col1_0 INET NOT NULL, col1_17 TIMESTAMP NOT NULL, col1_14 OID NOT NULL, col1_13 TIMETZ NOT NULL, col1_4 REGTYPE NOT NULL, col1_8 TIMETZ NOT NULL, col1_10 INTERVAL NOT NULL, col1_6 BOX2D NOT NULL, col1_9 TEXT NOT NULL, col1_12 BOOL NOT NULL, col3_11 TEXT, col3_12 VARCHAR, col3_13 BIT(17) NOT NULL, col3_14 TEXT, col3_15 TEXT NULL, col3_16 TIMETZ NULL, col3_17 TIMETZ, col3_18 INT8 NOT NULL, col3_19 OID, col3_20 UUID NOT NULL, col3_21 FLOAT8, col3_22 CHAR, col3_23 REGNAMESPACE NULL, col3_24 DATE NULL, col3_25 INT8 NULL, col3_26 CHAR NULL, col3_27 REGTYPE NULL, col3_28 FLOAT8, PRIMARY KEY (col1_2, col1_0, col1_17, col1_14, col1_13, col1_4, col1_8, col1_10, col1_6, col1_9, col1_12, col3_18, col3_25, col3_21, col3_16, col3_28, col3_23, col3_27, col3_11, col3_20, col3_26, col3_15, col3_22, col3_24, col3_14, col3_12, col3_19));
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_28 ASC, col3_17 ASC, col3_26 ASC, col3_14, col1_17 DESC, col3_18 ASC, col1_14, col1_9, col1_6, col1_2, col3_19, col3_22 DESC, col1_4 ASC, col3_13, col1_8 DESC);
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_8) REFERENCES table2 (col1_13);
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col3_20, col1_9) REFERENCES table2 (col2_17, col1_9) ON DELETE CASCADE;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('xu', 'addrpw', 'vbin');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('mvcera', 'm');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('fes');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (0.74s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 INT4 NOT NULL, col1_1 BYTES NOT NULL, col1_2 BIT(19) NOT NULL, col1_3 DATE NOT NULL, col1_4 BOX2D NOT NULL, col1_5 FLOAT8 NOT NULL, col1_6 INT2 NOT NULL, col1_7 INT8 NOT NULL, col1_8 BOX2D NOT NULL, col1_9 INT8 NOT NULL, col1_10 INTERVAL NOT NULL, col1_11 BIT(19) NOT NULL, col1_12 INET NOT NULL, col1_13 TIMESTAMP NOT NULL, col1_14 DECIMAL NOT NULL, col1_15 "char" NOT NULL, col1_16 OID[] NOT NULL, col1_17 FLOAT4 NOT NULL, PRIMARY KEY (col1_9, col1_2 DESC, col1_1, col1_3, col1_12, col1_8 DESC, col1_7, col1_11 ASC, col1_0 ASC, col1_13 DESC, col1_14, col1_4 ASC, col1_10 DESC, col1_5, col1_15, col1_17), UNIQUE (col1_3 ASC, col1_12 DESC, col1_10, col1_7, col1_17, col1_6 DESC, col1_9 ASC, col1_15 DESC) WHERE ((((((((((table1.col1_3 <= '5874897-12-31') AND (table1.col1_9 != -9223372036854775807)) OR (table1.col1_7 < 2147483647)) AND (table1.col1_14 = Infinity)) OR (table1.col1_13 != '294276-12-31 23:59:59.999999')) AND (table1.col1_5 >= 1.401298464324817e-45)) AND (table1.col1_15 <= 'X')) OR (table1.col1_6 <= -1)) AND (table1.col1_1 <= '\xe29883')) OR (table1.col1_0 != -2147483648)) OR (table1.col1_17 <= 0.0), UNIQUE (col1_9 DESC, col1_2, col1_7 DESC, col1_1 DESC, col1_3 ASC, col1_12, col1_10 ASC, col1_4 ASC, col1_15, col1_17 DESC, col1_11 DESC, col1_8 ASC) WHERE ((((((((((table1.col1_17 <= +Inf) AND (table1.col1_1 < '\x00')) AND (table1.col1_5 <= 3.4028234663852886e+38)) AND (table1.col1_9 > 9223372036854775807)) AND (table1.col1_7 >= 9223372036854775807)) AND (table1.col1_3 = '1970-01-01')) AND (table1.col1_6 != 127)) OR (table1.col1_0 < 32767)) OR (table1.col1_15 < e'\x00')) OR (table1.col1_14 > -1)) OR (table1.col1_13 = '0001-01-01 00:00:00'), UNIQUE (col1_4 DESC, col1_15, col1_11 DESC, col1_10 DESC, col1_13, col1_12 DESC, col1_1 DESC, col1_17 ASC, col1_0 ASC, col1_8 ASC, col1_9 ASC, col1_2 ASC, col1_6 DESC, col1_14 DESC, col1_7 ASC) STORING (col1_16), UNIQUE (col1_5 ASC, col1_1, col1_7, col1_10 DESC, col1_8 ASC, col1_14 DESC, col1_0, col1_13, col1_3 DESC, col1_12) WHERE ((((((((table1.col1_1 >= '\x') OR (table1.col1_5 = +Inf)) AND (table1.col1_14 > NaN)) AND (table1.col1_3 != '1970-01-01')) AND (table1.col1_0 != 2147483647)) AND (table1.col1_9 <= -1)) OR (table1.col1_17 != -1.0)) AND (table1.col1_13 <= '0001-01-01 00:00:00')) OR (table1.col1_7 >= 32767), FAMILY (col1_3), FAMILY (col1_16), FAMILY (col1_5), FAMILY (col1_17), FAMILY (col1_7, col1_1), FAMILY (col1_8), FAMILY (col1_6, col1_15), FAMILY (col1_10, col1_14, col1_9), FAMILY (col1_0, col1_2), FAMILY (col1_13), FAMILY (col1_4, col1_12), FAMILY (col1_11));
        test_1        |             CREATE TABLE table2 (col1_9 INT8 NOT NULL, col1_2 BIT(19) NOT NULL, col1_1 BYTES NOT NULL, col1_3 DATE NOT NULL, col1_12 INET NOT NULL, col1_8 BOX2D NOT NULL, col1_7 INT8 NOT NULL, col1_11 BIT(19) NOT NULL, col1_0 INT4 NOT NULL, col1_13 TIMESTAMP NOT NULL, col1_14 DECIMAL NOT NULL, col1_4 BOX2D NOT NULL, col1_10 INTERVAL NOT NULL, col1_5 FLOAT8 NOT NULL, col1_15 "char" NOT NULL, col1_17 FLOAT4 NOT NULL, col2_16 TIMESTAMP, col2_17 DATE, col2_18 DATE, col2_19 CHAR, col2_20 REGNAMESPACE NOT NULL, col2_21 REGCLASS NULL, col2_22 UUID NOT NULL, col2_23 BIT(31) NOT NULL, col2_24 OID NULL, col2_25 CHAR NOT NULL, col2_26 BIT(21), col2_27 FLOAT8 NULL, col2_28 OID, PRIMARY KEY (col1_9, col1_2 DESC, col1_1, col1_3, col1_12, col1_8 DESC, col1_7, col1_11 ASC, col1_0 ASC, col1_13 DESC, col1_14, col1_4 ASC, col1_10 DESC, col1_5, col1_15, col1_17, col2_21 DESC), INDEX (col2_27, col1_8 DESC, col1_7 ASC, col1_5 ASC, col1_15 ASC, col2_24 ASC, col1_11 ASC, col2_18 DESC, col2_16 ASC, col1_13 DESC, col2_28, col2_22, col2_25 ASC) STORING (col2_17, col2_20, col2_26) WHERE (((((((((((((table2.col1_13 = '294276-12-31 23:59:59.999999') OR (table2.col1_7 < -9223372036854775807)) AND (table2.col1_1 != '\x')) OR (table2.col1_9 > -9223372036854775807)) OR (table2.col2_25 != e'\'')) AND (table2.col1_15 > '"')) OR (table2.col1_14 <= -Infinity)) OR (table2.col1_3 >= 'infinity')) AND (table2.col2_16 < '-2000-01-01 00:00:00')) AND (table2.col1_5 > +Inf)) OR (table2.col2_27 <= 3.4028234663852886e+38)) AND (table2.col2_18 != '-infinity')) OR (table2.col2_19 < e'\U00002603')) OR (table2.col1_17 != 1.401298464324817e-45), UNIQUE (col2_24 DESC, col1_7 ASC, col2_27, col2_26 ASC, col1_15 ASC, col1_11, col1_17 ASC, col2_25 DESC, col1_0, col2_17 ASC, col2_21 DESC, col1_1 ASC) WHERE (((((((((((((((table2.col1_14 <= '-Infinity':::DECIMAL) OR (table2.col1_0 < (-1):::INT8)) AND (table2.col1_7 != 2147483647:::INT8)) AND (table2.col2_25 < e'\'':::STRING)) AND (table2.col1_3 >= '-infinity':::DATE)) OR (table2.col2_18 = '4714-11-24 BC':::DATE)) AND (table2.col2_16 <= '0001-01-01 00:00:00':::TIMESTAMP)) OR (table2.col1_13 != '3000-01-01 00:00:00':::TIMESTAMP)) AND (table2.col1_15 >= e'\x00':::STRING)) OR (table2.col2_17 > '1970-01-01':::DATE)) AND (table2.col1_9 <= (-9223372036854775807):::INT8)) OR (table2.col2_19 > e'\'':::STRING)) OR (table2.col1_1 != e'\\x22':::BYTES)) AND (table2.col1_17 < '+Inf':::FLOAT8)) OR (table2.col1_5 >= 'NaN':::FLOAT8)) OR (table2.col2_27 = 1.401298464324817e-45:::FLOAT8), INDEX (col2_26, col1_3 DESC, col1_13 DESC, col2_22 DESC, col1_12 ASC, col2_25 ASC, col1_0 ASC, col1_10 ASC, col2_23 ASC, col2_20 DESC, col1_4 ASC, col1_17 DESC) STORING (col2_16, col2_17, col2_27, col2_28) WHERE ((((((((table2.col1_15 <= e'\x00':::STRING) AND (table2.col1_1 <= e'\\x':::BYTES)) OR (table2.col2_25 <= e'\U00002603':::STRING)) OR (table2.col1_0 > (-1):::INT8)) AND (table2.col2_19 > e'\x00':::STRING)) OR (table2.col1_5 < '+Inf':::FLOAT8)) AND (table2.col2_18 = '5874897-12-31':::DATE)) AND (table2.col1_17 < 1.401298464324817e-45:::FLOAT8)) AND (table2.col1_9 > 32767:::INT8), UNIQUE (col1_13) STORING (col2_17, col2_19, col2_20, col2_22, col2_23, col2_24, col2_25, col2_27), UNIQUE (col2_16, col1_5, col1_4, col1_7 DESC, col2_28, col1_3, col1_12, col1_10, col2_23 ASC, col2_22 ASC, col2_26 ASC, col2_19 DESC, col2_25 DESC, col2_21, col1_11, col1_9 ASC, col2_18) STORING (col2_27), UNIQUE (col1_5 DESC, col2_28 DESC, col2_18 ASC, col2_23, col1_4 ASC, col1_11 DESC, col1_7, col2_16 DESC, col2_26 ASC, col2_27 ASC, col2_21 ASC, col1_3 ASC, col2_25 ASC, col1_12 DESC, col1_10, col1_8, col2_20, col1_9, col1_2, col2_22 DESC, col2_17) STORING (col2_19) WHERE ((((((table2.col1_5 >= '-Inf':::FLOAT8) AND (table2.col1_15 > '"':::STRING)) OR (table2.col1_3 = '5874897-12-31':::DATE)) OR (table2.col1_7 = 1:::INT8)) OR (table2.col2_18 > '-infinity':::DATE)) AND (table2.col1_9 >= (-1):::INT8)) AND (table2.col1_14 < 'NaN':::DECIMAL), FAMILY (col1_8), FAMILY (col1_13, col1_1), FAMILY (col2_27, col2_28), FAMILY (col2_24), FAMILY (col2_26, col1_2), FAMILY (col1_14, col1_0, col1_15, col2_22), FAMILY (col2_21, col1_12), FAMILY (col1_11), FAMILY (col1_4), FAMILY (col2_16, col2_17), FAMILY (col2_20, col1_3), FAMILY (col2_25, col1_7, col1_9, col2_18), FAMILY (col2_19, col1_17), FAMILY (col2_23), FAMILY (col1_5), FAMILY (col1_10)) INTERLEAVE IN PARENT table1 (col1_9, col1_2, col1_1, col1_3, col1_12, col1_8, col1_7, col1_11, col1_0, col1_13, col1_14, col1_4, col1_10, col1_5, col1_15, col1_17);
        test_1        |             CREATE TABLE table3 (col1_9 INT8 NOT NULL, col1_2 BIT(19) NOT NULL, col1_1 BYTES NOT NULL, col1_3 DATE NOT NULL, col1_12 INET NOT NULL, col1_8 BOX2D NOT NULL, col1_7 INT8 NOT NULL, col1_11 BIT(19) NOT NULL, col1_0 INT4 NOT NULL, col1_13 TIMESTAMP NOT NULL, col1_14 DECIMAL NOT NULL, col1_4 BOX2D NOT NULL, col1_10 INTERVAL NOT NULL, col1_5 FLOAT8 NOT NULL, col1_15 "char" NOT NULL, col1_17 FLOAT4 NOT NULL, col2_21 REGCLASS NULL, col3_17 REGTYPE, col3_18 BIT(3) NOT NULL, col3_19 DATE, col3_20 TIMETZ NOT NULL, col3_21 TIME, PRIMARY KEY (col1_9, col1_2 DESC, col1_1, col1_3, col1_12, col1_8 DESC, col1_7, col1_11 ASC, col1_0 ASC, col1_13 DESC, col1_14, col1_4 ASC, col1_10 DESC, col1_5, col1_15, col1_17, col2_21 DESC, col3_19, col3_20 ASC, col3_21 DESC), INDEX (col3_20 ASC, col1_11 DESC, col1_9 DESC, col1_8 DESC, col1_7, col3_21 ASC) WHERE ((((((((((((table3.col1_14 <= 0:::DECIMAL) OR (table3.col3_21 = '00:00:00':::TIME)) AND (table3.col3_20 != '00:00:00+15:59:00':::TIMETZ)) OR (table3.col1_17 = 0.0:::FLOAT8)) OR (table3.col1_9 <= 9223372036854775807:::INT8)) OR (table3.col3_19 <= '1970-01-01':::DATE)) OR (table3.col1_13 = '0001-01-01 00:00:00':::TIMESTAMP)) AND (table3.col1_1 > e'\\xe29883':::BYTES)) OR (table3.col1_0 > 32767:::INT8)) OR (table3.col1_5 < (-1.0):::FLOAT8)) AND (table3.col1_3 >= '1970-01-01':::DATE)) OR (table3.col1_7 != (-32768):::INT8)) AND (table3.col1_15 > '':::STRING), INDEX (col3_18 ASC, col1_14 ASC, col1_1, col1_4) WHERE ((((((((((((table3.col1_3 > '5874897-12-31':::DATE) AND (table3.col1_17 != '+Inf':::FLOAT8)) AND (table3.col1_13 != '-2000-01-01 00:00:00':::TIMESTAMP)) AND (table3.col1_1 != e'\\x22':::BYTES)) AND (table3.col1_15 <= e'\U00002603':::STRING)) OR (table3.col1_7 <= 0:::INT8)) OR (table3.col1_5 <= '-Inf':::FLOAT8)) OR (table3.col3_19 > 'infinity':::DATE)) OR (table3.col1_0 != (-1):::INT8)) OR (table3.col3_21 > '24:00:00':::TIME)) OR (table3.col3_20 < '00:00:00+15:59:00':::TIMETZ)) AND (table3.col1_9 > (-128):::INT8)) AND (table3.col1_14 != 'Infinity':::DECIMAL), UNIQUE (col3_18 ASC, col1_3 ASC, col1_12 ASC, col1_7, col3_17 DESC, col1_17 DESC, col1_8 ASC, col1_11 DESC) WHERE ((((((((((((table3.col1_9 < 127) OR (table3.col1_17 < 0.0)) OR (table3.col1_14 <= 0)) AND (table3.col1_7 <= -32768)) AND (table3.col1_3 != '-infinity')) AND (table3.col3_19 > '5874897-12-31')) OR (table3.col1_13 > '-2000-01-01 00:00:00')) OR (table3.col1_0 = -2147483648)) OR (table3.col1_1 = '\x27')) AND (table3.col3_20 != '00:00:00+15:59:00')) AND (table3.col1_15 >= e'\U00002603')) OR (table3.col1_5 >= 0.0)) OR (table3.col3_21 = '24:00:00'), UNIQUE (col1_14 DESC, col1_15 ASC, col1_11 DESC, col1_13, col3_17 ASC, col1_8, col3_18 DESC, col2_21, col1_5 ASC, col1_1 ASC, col3_21 DESC), UNIQUE (col1_12 DESC, col2_21 ASC) STORING (col3_17) WHERE (table3.col1_13 = '294276-12-31 23:59:59.999999') OR (table3.col3_21 = '24:00:00'), INDEX (col3_17 DESC, col1_11, col1_13 ASC, col1_8, col1_17 DESC, col1_10, col1_1, col2_21, col1_12, col1_15 ASC, col1_0 ASC) STORING (col3_18) WHERE (((((((((((table3.col1_0 != 127:::INT8) AND (table3.col1_5 <= '+Inf':::FLOAT8)) OR (table3.col3_20 >= '00:00:00+15:59:00':::TIMETZ)) OR (table3.col1_7 >= 1:::INT8)) OR (table3.col3_19 <= '-infinity':::DATE)) AND (table3.col1_3 <= '1970-01-01':::DATE)) OR (table3.col1_14 != 'NaN':::DECIMAL)) AND (table3.col1_1 < e'\\x58':::BYTES)) AND (table3.col3_21 != '24:00:00':::TIME)) AND (table3.col1_15 != e'\x00':::STRING)) AND (table3.col1_17 <= '+Inf':::FLOAT8)) AND (table3.col1_9 != (-128):::INT8), UNIQUE (col1_11 DESC, col3_21, col1_3, col1_7, col3_20 DESC, col3_17 ASC, col1_2 DESC, col1_8, col1_15 ASC, col1_1, col3_19 ASC, col1_10 DESC, col1_12 ASC, col1_13 ASC, col3_18 DESC, col2_21 DESC, col1_17) WHERE (((((((((table3.col1_5 < (-1.0):::FLOAT8) AND (table3.col1_7 <= 9223372036854775807:::INT8)) OR (table3.col1_15 != '':::STRING)) OR (table3.col3_19 = '-infinity':::DATE)) AND (table3.col1_17 >= 3.4028234663852886e+38:::FLOAT8)) AND (table3.col1_13 = '-4713-11-24 00:00:00':::TIMESTAMP)) OR (table3.col1_0 < (-32768):::INT8)) OR (table3.col1_14 = '-Infinity':::DECIMAL)) OR (table3.col3_20 >= '24:00:00-15:59:00':::TIMETZ)) OR (table3.col3_21 != '00:00:00':::TIME), INDEX (col1_8, col1_1 ASC, col1_13 DESC, col1_7 DESC, col1_10 DESC, col1_14 DESC, col3_21 DESC, col3_18 ASC, col1_3, col3_17 ASC, col1_0 ASC, col1_17, col1_15 ASC, col1_4, col3_20 ASC) WHERE ((((((((((((table3.col1_1 >= e'\\xff':::BYTES) AND (table3.col3_20 <= '00:00:00+15:59:00':::TIMETZ)) AND (table3.col3_19 > '5874897-12-31':::DATE)) AND (table3.col1_7 != 2147483647:::INT8)) AND (table3.col1_17 <= 0.0:::FLOAT8)) AND (table3.col1_0 > 32767:::INT8)) AND (table3.col1_14 <= 'Infinity':::DECIMAL)) OR (table3.col1_9 < 2147483647:::INT8)) OR (table3.col3_21 > '24:00:00':::TIME)) OR (table3.col1_3 <= 'infinity':::DATE)) OR (table3.col1_5 != 1.0:::FLOAT8)) AND (table3.col1_13 > '-4713-11-24 00:00:00':::TIMESTAMP)) AND (table3.col1_15 >= '"':::STRING), INDEX (col1_7) STORING (col3_17, col3_18), FAMILY (col2_21), FAMILY (col1_10, col1_5), FAMILY (col1_3, col1_12, col1_17, col1_2, col3_19), FAMILY (col3_17), FAMILY (col1_13), FAMILY (col1_0, col1_1, col1_7, col3_20, col3_18), FAMILY (col1_15), FAMILY (col1_8, col1_9, col1_11), FAMILY (col1_14, col1_4, col3_21)) INTERLEAVE IN PARENT table2 (col1_9, col1_2, col1_1, col1_3, col1_12, col1_8, col1_7, col1_11, col1_0, col1_13, col1_14, col1_4, col1_10, col1_5, col1_15, col1_17, col2_21);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35281, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 40773, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000000000, "num_range": 0, "upper_bound": "-4919407245331086500"}, {"distinct_range": 0, "num_eq": 803889458182550627, "num_range": 20000000000, "upper_bound": "-3932888256306248610"}, {"distinct_range": 50, "num_eq": 0, "num_range": 50, "upper_bound": "-2794944032406245545"}, {"distinct_range": 0, "num_eq": 90000, "num_range": 6000000, "upper_bound": "-1650445828002601787"}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 80000000, "upper_bound": "-737163951883494714"}, {"distinct_range": 100, "num_eq": 7716423315250192703, "num_range": 100, "upper_bound": "-1"}, {"distinct_range": 0, "num_eq": 1911307308369694977, "num_range": 0, "upper_bound": "5550550777732019151"}, {"distinct_range": 0, "num_eq": 3530535936725692504, "num_range": 0, "upper_bound": "5742090174706952710"}, {"distinct_range": 500000, "num_eq": 2392159665849134859, "num_range": 500000, "upper_bound": "8150905444479398434"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 125348, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 157561, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 103524, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 159012, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27403, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000000, "num_range": 0, "upper_bound": "BOX(-2.2819714834593796 0.7596920969805334,1.2817795135092491 1.0117716330164834)"}, {"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "BOX(-1.9638960230831533 -0.8463685290442582,-0.32688517041148424 0.5793950270975303)"}, {"distinct_range": 0, "num_eq": 8470316657952059760, "num_range": 5179855529490784776, "upper_bound": "BOX(-0.4435755021328892 -1.061822260300683,0.5506800663264595 -0.019883652754192627)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 153158, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 75726, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 161526, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44911, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 117830, "histo_buckets": [{"distinct_range": 0, "num_eq": 699292462254148201, "num_range": 0, "upper_bound": "-1.2982089748787917"}, {"distinct_range": 2951890827260555000, "num_eq": 0, "num_range": 2951890827260554497, "upper_bound": "5e-324"}, {"distinct_range": 0, "num_eq": 90000, "num_range": 597234261232097874, "upper_bound": "1.2562643812309755"}, {"distinct_range": 0, "num_eq": 7000000, "num_range": 60000000, "upper_bound": "3.4028234663852886e+38"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 108870, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 95758, "histo_buckets": [{"distinct_range": 0, "num_eq": 5569819692075335267, "num_range": 0, "upper_bound": "1975-10-10"}, {"distinct_range": 17662509.516578943, "num_eq": 200000000, "num_range": 30000000, "upper_bound": "1992-05-05"}, {"distinct_range": 100, "num_eq": 90000, "num_range": 100, "upper_bound": "1993-06-14"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 112749, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 66183, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 61646, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 146150, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_26"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5226681293, "histo_buckets": [{"distinct_range": 0, "num_eq": 2253108814057562175, "num_range": 0, "upper_bound": "101010001000011010001"}, {"distinct_range": 0, "num_eq": 7263483656231634923, "num_range": 40000, "upper_bound": "101110111100100000000"}, {"distinct_range": 23432.08308981367, "num_eq": 500000000, "num_range": 80000, "upper_bound": "110000111001011000110"}, {"distinct_range": 400000000, "num_eq": 2026144370858903894, "num_range": 400000000, "upper_bound": "111010111011010010001"}], "histo_col_type": "BIT(21)", "name": "__auto__", "null_count": 2496954379, "row_count": 8000000000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5815953083, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1800425256, "histo_col_type": "", "name": "__auto__", "null_count": 4227143524, "row_count": 8000000000}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 891553908, "histo_col_type": "", "name": "__auto__", "null_count": 5006189401, "row_count": 8000000000}, {"columns": ["col2_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 111142687, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3915906131, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3553230223, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col2_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5019459220, "histo_col_type": "", "name": "__auto__", "null_count": 2894180710, "row_count": 8000000000}, {"columns": ["col2_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4181364156, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6946678483, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6663392584, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1842761509, "histo_buckets": [{"distinct_range": 0, "num_eq": 4554686290245188501, "num_range": 0, "upper_bound": "0001-01-01 00:00:00"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "2001-06-26 05:01:49.000021"}, {"distinct_range": 800000000, "num_eq": 300, "num_range": 800000000, "upper_bound": "2011-10-22 09:18:39.000632"}, {"distinct_range": 400000, "num_eq": 2598769709834036910, "num_range": 400000, "upper_bound": "2014-01-30 19:13:58.000701"}, {"distinct_range": 4000000, "num_eq": 0, "num_range": 4000000, "upper_bound": "2019-04-24 14:03:14.000638"}, {"distinct_range": 0, "num_eq": 3644756883260466832, "num_range": 5557083151538460749, "upper_bound": "2019-07-28 13:37:44.000374"}, {"distinct_range": 1667619428146467800, "num_eq": 30000000000, "num_range": 2537984680955796757, "upper_bound": "2027-09-09 18:00:05.00043"}, {"distinct_range": 0, "num_eq": 5590149104790354812, "num_range": 2145428093036514084, "upper_bound": "2029-09-26 06:19:13.000196"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 165864088, "row_count": 8000000000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 128196439, "histo_col_type": "", "name": "__auto__", "null_count": 2059368799, "row_count": 8000000000}, {"columns": ["col2_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5544966188, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000, "num_range": 0, "upper_bound": "134599175"}, {"distinct_range": 3000000, "num_eq": 158814629425584443, "num_range": 3000000, "upper_bound": "191857895"}, {"distinct_range": 0, "num_eq": 6553895278948908824, "num_range": 6278374149718091989, "upper_bound": "905023762"}, {"distinct_range": 166397018576030400, "num_eq": 40000000000, "num_range": 9024207605648709121, "upper_bound": "1634564160"}, {"distinct_range": 5000000, "num_eq": 7673058930766030141, "num_range": 5000000, "upper_bound": "2263218145"}, {"distinct_range": 0, "num_eq": 400, "num_range": 200000, "upper_bound": "2708471295"}, {"distinct_range": 52.26618031052562, "num_eq": 553174743146852011, "num_range": 90, "upper_bound": "3266909403"}, {"distinct_range": 4946039672243139000, "num_eq": 3210370528548597741, "num_range": 5113927047004429636, "upper_bound": "3701704025"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 6755463657, "row_count": 8000000000}, {"columns": ["col2_25"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 463184998, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 602741908, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2066264900, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3659065031, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 538998397, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 128619361, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col2_28"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3629571850, "histo_col_type": "", "name": "__auto__", "null_count": 5871474996, "row_count": 8000000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2532070176, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000000, "num_range": 0, "upper_bound": "-9223372036854775807"}, {"distinct_range": 20314.996455449404, "num_eq": 1873091940362671584, "num_range": 90000, "upper_bound": "-8324927838735225072"}, {"distinct_range": 2098923098708539100, "num_eq": 0, "num_range": 2321859054224214235, "upper_bound": "-7909664549919930668"}, {"distinct_range": 30000000, "num_eq": 6764001754901396426, "num_range": 30000000, "upper_bound": "-6567486746161132500"}, {"distinct_range": 605937323133233000, "num_eq": 5000000000, "num_range": 1776168637844645730, "upper_bound": "-4510223583743850426"}, {"distinct_range": 0, "num_eq": 50000000, "num_range": 5231914963278776777, "upper_bound": "-213693458712214060"}, {"distinct_range": 1480583554274876400, "num_eq": 20, "num_range": 3030400376887360469, "upper_bound": "-115048439115714946"}, {"distinct_range": 60000, "num_eq": 1000, "num_range": 60000, "upper_bound": "-128"}, {"distinct_range": 0, "num_eq": 100, "num_range": 8000000, "upper_bound": "4804135876321089307"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6621133277, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4461564369, "histo_buckets": [{"distinct_range": 0, "num_eq": 2213544139973654606, "num_range": 0, "upper_bound": "NaN"}, {"distinct_range": 2455188808924765000, "num_eq": 6886114908880325868, "num_range": 2455188808924765016, "upper_bound": "-1.2380953925737668"}, {"distinct_range": 803881693.9827782, "num_eq": 8446108230079774792, "num_range": 3000000000, "upper_bound": "-0.21866150632027748"}, {"distinct_range": 29.9577266991304, "num_eq": 800000000, "num_range": 100, "upper_bound": "0.36504301945026185"}, {"distinct_range": 0, "num_eq": 3370749895474891689, "num_range": 258337373916692654, "upper_bound": "0.41432821031825473"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 800000, "upper_bound": "0.5278426420253035"}, {"distinct_range": 40000000, "num_eq": 1926496883358191336, "num_range": 40000000, "upper_bound": "0.6566935451085546"}, {"distinct_range": 2172111529.8051863, "num_eq": 2298255941219539220, "num_range": 5000000000, "upper_bound": "1.1772262075001443"}, {"distinct_range": 1000, "num_eq": 2100580414800518288, "num_range": 1000, "upper_bound": "1.7976931348623157e+308"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3459384534, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000, "num_range": 0, "upper_bound": "1980-02-11 04:59:33.000122"}, {"distinct_range": 9218732630988609000, "num_eq": 8000, "num_range": 9218732630988608764, "upper_bound": "1984-04-06 07:43:31.000344"}, {"distinct_range": 1298927217982249500, "num_eq": 3171913976399655770, "num_range": 1298927217982249463, "upper_bound": "1986-12-11 08:06:47.000216"}, {"distinct_range": 0, "num_eq": 0, "num_range": 9077522277798909197, "upper_bound": "1991-10-03 07:31:02.000824"}, {"distinct_range": 2480.146650753722, "num_eq": 90000000000, "num_range": 300000, "upper_bound": "2001-07-18 16:46:55.000386"}, {"distinct_range": 127724627440008050, "num_eq": 90000, "num_range": 3124177100316400244, "upper_bound": "2013-03-12 08:06:34.000655"}, {"distinct_range": 10000000, "num_eq": 400000, "num_range": 10000000, "upper_bound": "2015-11-30 02:31:26.000539"}, {"distinct_range": 3105349702344086500, "num_eq": 8126108801234372570, "num_range": 8018237781501206741, "upper_bound": "2029-01-01 14:30:48.000015"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 533599951, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6475046818, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}, {"columns": ["col2_27"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2097538361, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-0.3991673935226059"}, {"distinct_range": 0, "num_eq": 4000000, "num_range": 7102494675903025234, "upper_bound": "-0.2638064774093438"}, {"distinct_range": 90000000000, "num_eq": 0, "num_range": 90000000000, "upper_bound": "-0.19006620133972307"}, {"distinct_range": 532958545.7097284, "num_eq": 90, "num_range": 4000000000, "upper_bound": "-0.16377304654297864"}, {"distinct_range": 0, "num_eq": 2191849281657881997, "num_range": 2677104319954863760, "upper_bound": "0.02185566853000881"}, {"distinct_range": 878813420.7164667, "num_eq": 20000000000, "num_range": 1000000000, "upper_bound": "1.0347791212889625"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 6732748184, "row_count": 8000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 611225861, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 48446046, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000000}, {"columns": ["col3_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5145456, "histo_col_type": "", "name": "__auto__", "null_count": 15132712, "row_count": 80000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 74546439, "histo_buckets": [{"distinct_range": 0, "num_eq": 2581822039073585377, "num_range": 0, "upper_bound": "-128"}, {"distinct_range": 67877.57230774092, "num_eq": 3989558241612934475, "num_range": 100000, "upper_bound": "7212699320929251025"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 80000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21364689, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22577573, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000000}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 78400855, "histo_buckets": [{"distinct_range": 0, "num_eq": 5292973040747085924, "num_range": 0, "upper_bound": "350268950"}, {"distinct_range": 1548729511377058000, "num_eq": 0, "num_range": 1548729511377057941, "upper_bound": "1208341295"}, {"distinct_range": 0, "num_eq": 7831431928152270462, "num_range": 5010800926920074484, "upper_bound": "3280303995"}, {"distinct_range": 3278710323869775000, "num_eq": 8634757999793517052, "num_range": 5434618093158225890, "upper_bound": "3421143720"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 54302649, "row_count": 80000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20188857, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000000}, {"columns": ["col2_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24534080, "histo_col_type": "", "name": "__auto__", "null_count": 19338843, "row_count": 80000000}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 54367006, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000000, "num_range": 0, "upper_bound": "000"}, {"distinct_range": 260610733492068450, "num_eq": 5000000000, "num_range": 260610733492068444, "upper_bound": "001"}, {"distinct_range": 2343670349097457700, "num_eq": 400, "num_range": 2343670349097457734, "upper_bound": "011"}, {"distinct_range": 695.3025025519004, "num_eq": 100000000000, "num_range": 800, "upper_bound": "100"}], "histo_col_type": "BIT(3)", "name": "__auto__", "null_count": 0, "row_count": 80000000}, {"columns": ["col3_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 75511668, "histo_col_type": "", "name": "__auto__", "null_count": 26152916, "row_count": 80000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32033609, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36754984, "histo_buckets": [{"distinct_range": 0, "num_eq": 10, "num_range": 0, "upper_bound": "0001000101101011010"}, {"distinct_range": 600, "num_eq": 8295892102244875456, "num_range": 600, "upper_bound": "0111101011101000011"}, {"distinct_range": 0, "num_eq": 30000000, "num_range": 300000, "upper_bound": "1011101101011111010"}, {"distinct_range": 0, "num_eq": 5142852116922231076, "num_range": 200000000, "upper_bound": "1111000000111101110"}], "histo_col_type": "BIT(19)", "name": "__auto__", "null_count": 0, "row_count": 80000000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10769414, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22865841, "histo_buckets": [{"distinct_range": 0, "num_eq": 4070921048440677994, "num_range": 0, "upper_bound": "-5.40317104484820758E+36"}, {"distinct_range": 30000, "num_eq": 2817915455931810304, "num_range": 30000, "upper_bound": "-8446.71852930208347"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 10000000, "upper_bound": "-0.07932038163049031321"}, {"distinct_range": 6801377842712656000, "num_eq": 4330055686354601226, "num_range": 6801377842712655659, "upper_bound": "0.6933919151481865201"}, {"distinct_range": 0, "num_eq": 7162380432324075603, "num_range": 3432477093840591480, "upper_bound": "3127120751263822649"}, {"distinct_range": 3680901768620946000, "num_eq": 7481913719813405484, "num_range": 5392585004752067850, "upper_bound": "4.311193026392852995E+22"}, {"distinct_range": 1968113600088834800, "num_eq": 400000, "num_range": 1968113600088834819, "upper_bound": "7.90392556343946788E+23"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 900000000, "upper_bound": "9.02779042813706182E+37"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 80000000}, {"columns": ["col3_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15874640, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "06:39:53.772941+12:40:00"}, {"distinct_range": 3135825992237220000, "num_eq": 5000, "num_range": 3135825992237219727, "upper_bound": "14:13:45.006346+08:03:00"}, {"distinct_range": 158525.35688855892, "num_eq": 100000000000, "num_range": 200000, "upper_bound": "09:14:36.414374+01:25:00"}, {"distinct_range": 0, "num_eq": 50, "num_range": 9000000, "upper_bound": "19:41:10.593362-07:39:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 80000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36105529, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19881133, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000000}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39705757, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9777764, "histo_buckets": [{"distinct_range": 0, "num_eq": 8528119354561301397, "num_range": 0, "upper_bound": "155.122.173.8/2"}, {"distinct_range": 300000, "num_eq": 7164155492646984834, "num_range": 300000, "upper_bound": "244.191.23.77/2"}, {"distinct_range": 16.011583966837954, "num_eq": 8000, "num_range": 60, "upper_bound": "194.60.151.9/5"}, {"distinct_range": 6639.928071571966, "num_eq": 100000000, "num_range": 8000, "upper_bound": "21.173.23.95/11"}, {"distinct_range": 1108111741690323500, "num_eq": 50000000000, "num_range": 1108111741690323491, "upper_bound": "0.230.108.116/18"}, {"distinct_range": 612.1359736782823, "num_eq": 5466185805102299960, "num_range": 800, "upper_bound": "130.154.59.119/18"}, {"distinct_range": 500, "num_eq": 0, "num_range": 500, "upper_bound": "208.192.248.40/22"}, {"distinct_range": 90, "num_eq": 8955322979490486088, "num_range": 90, "upper_bound": "3e68:588e:b2b8:43b1:1e6:58f4:66bc:bd74/42"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 80000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67446966, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000000, "num_range": 0, "upper_bound": "BOX(0.5924506349552277 0.21128471022084894,0.8699075858730057 0.6220707630456346)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 80000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67221710, "histo_buckets": [{"distinct_range": 0, "num_eq": 3653143997335845393, "num_range": 0, "upper_bound": "-7219528043391736496"}, {"distinct_range": 7000, "num_eq": 200000, "num_range": 7000, "upper_bound": "-6317364887129269252"}, {"distinct_range": 57924825233.19829, "num_eq": 6614484906675022998, "num_range": 70000000000, "upper_bound": "-3513677200877949324"}, {"distinct_range": 0, "num_eq": 5121035478722297379, "num_range": 100000000, "upper_bound": "1446165386271892595"}, {"distinct_range": 0, "num_eq": 80, "num_range": 40000000000, "upper_bound": "2180835551883778680"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 80000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6347278, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('ny');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('uezztn', 'xhrz', 'uhewvb');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('vt', 'qiez');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('tce', 'co', 'raiwag', 'ja', 'kzgsod');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 552484, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 129967, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 530207, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 290134, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 268003, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 354926, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000000, "num_range": 0, "upper_bound": "-1.8726396278061082"}, {"distinct_range": 657.760666318155, "num_eq": 7033469302143904278, "num_range": 1000, "upper_bound": "-0.46357688077487325"}, {"distinct_range": 3225201663.6550965, "num_eq": 1004648188993535618, "num_range": 10000000000, "upper_bound": "-0.39030587604663225"}, {"distinct_range": 200000000, "num_eq": 7000, "num_range": 200000000, "upper_bound": "-0.04893223917906264"}, {"distinct_range": 4623549459054172000, "num_eq": 6200866757890911613, "num_range": 5232608831554844161, "upper_bound": "0.032804179867428895"}, {"distinct_range": 2038141916518312400, "num_eq": 400, "num_range": 5912431182324312968, "upper_bound": "0.43387964144478797"}, {"distinct_range": 5000, "num_eq": 200000, "num_range": 5000, "upper_bound": "0.9429361785164867"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 438981, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 142899, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 467735, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 297387, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 422024, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 151055, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27271, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 278954, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 171489, "histo_buckets": [{"distinct_range": 0, "num_eq": 9128695257775306126, "num_range": 0, "upper_bound": "1970-12-22"}, {"distinct_range": 0, "num_eq": 0, "num_range": 2499414602007509136, "upper_bound": "1972-05-21"}, {"distinct_range": 4000000, "num_eq": 6000, "num_range": 4000000, "upper_bound": "1974-03-17"}, {"distinct_range": 7135469766641776000, "num_eq": 0, "num_range": 7135469766641775483, "upper_bound": "1978-06-23"}, {"distinct_range": 30.7403973075653, "num_eq": 1000, "num_range": 3000, "upper_bound": "1982-01-17"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 0, "upper_bound": "1988-03-18"}, {"distinct_range": 1200954687028098600, "num_eq": 8978484707145418379, "num_range": 8725176135953108757, "upper_bound": "1992-01-10"}, {"distinct_range": 379616665518719940, "num_eq": 6622290365235029152, "num_range": 7187793848510905433, "upper_bound": "1992-01-26"}, {"distinct_range": 5179624927968041000, "num_eq": 10, "num_range": 5882142809624460727, "upper_bound": "5874897-12-31"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 209277, "histo_buckets": [{"distinct_range": 0, "num_eq": 3333017503212703852, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 1.5491877777873282, "num_eq": 70000000, "num_range": 70, "upper_bound": "BOX(-1.8183073775438015 0.027288447212286515,-0.31761111858493724 0.7867815128381521)"}, {"distinct_range": 1015.2064874458293, "num_eq": 3337728274157997896, "num_range": 3000, "upper_bound": "BOX(-1.243846571019036 -2.2089758445288794,0.36425215216590767 -0.7872530537301712)"}, {"distinct_range": 1935.034764164893, "num_eq": 60, "num_range": 30000, "upper_bound": "BOX(-1.097048917184616 -2.57682875255235,1.3684386056881546 0.7988660307481159)"}, {"distinct_range": 113270170.8687512, "num_eq": 70, "num_range": 900000000, "upper_bound": "BOX(-0.8258810320160404 0.42589417082074665,0.5784850444575378 0.9455782456016508)"}, {"distinct_range": 0, "num_eq": 300, "num_range": 2328739122527437582, "upper_bound": "BOX(-0.5462361795009657 0.7632503282986045,0.8859930652412343 1.0620656740455732)"}, {"distinct_range": 20000000000, "num_eq": 6382815751868307263, "num_range": 20000000000, "upper_bound": "BOX(-0.2958958261341642 -0.5018508936313938,1.2259765924850003 0.1729329099877737)"}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 534916124493968519, "upper_bound": "BOX(-0.15050043105082078 -0.9200883221993776,0.4375138819622769 -0.07339857112783482)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 295091, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "-8684360537036032129"}, {"distinct_range": 0, "num_eq": 117049267908827031, "num_range": 30, "upper_bound": "-7947156316616041758"}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 2334466889857234778, "upper_bound": "-5383931577766932642"}, {"distinct_range": 0, "num_eq": 2000000000, "num_range": 8262750258749149703, "upper_bound": "-4176532151916848119"}, {"distinct_range": 705301348723094800, "num_eq": 0, "num_range": 1111583015572897725, "upper_bound": "-115896605284228642"}, {"distinct_range": 5161.443072843801, "num_eq": 3206338267146221658, "num_range": 6000, "upper_bound": "2798019781689238987"}, {"distinct_range": 0, "num_eq": 4562928633914980526, "num_range": 1132369745726231427, "upper_bound": "4700526633179222132"}, {"distinct_range": 158368.81050573365, "num_eq": 5237791908098125969, "num_range": 1000000, "upper_bound": "5042124086760800330"}, {"distinct_range": 0, "num_eq": 100, "num_range": 4096733810939063606, "upper_bound": "6635613699075031818"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 249945, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 250, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 467, "histo_col_type": "", "name": "__auto__", "null_count": 211, "row_count": 500}, {"columns": ["col2_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 277, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col2_25"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 285, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 299, "histo_buckets": [{"distinct_range": 0, "num_eq": 8255581183153109429, "num_range": 0, "upper_bound": "1312872220900280915"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 184, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 214, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 37, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 160, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 487, "histo_buckets": [{"distinct_range": 0, "num_eq": 1098998643229298517, "num_range": 0, "upper_bound": "-0.7705882635100838"}, {"distinct_range": 50000000, "num_eq": 60000000000, "num_range": 50000000, "upper_bound": "-0.5974223390997219"}, {"distinct_range": 201849678.80157316, "num_eq": 6000, "num_range": 300000000, "upper_bound": "-0.15503931901114473"}, {"distinct_range": 0, "num_eq": 4515913311001662385, "num_range": 90000000000, "upper_bound": "0.5509674646358222"}, {"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "0.7720694881790666"}, {"distinct_range": 3687200184712695000, "num_eq": 2039413185632243895, "num_range": 3687200184712694714, "upper_bound": "1.0822769585416943"}, {"distinct_range": 17949565371.850834, "num_eq": 40000, "num_range": 90000000000, "upper_bound": "1.1075600565867627"}, {"distinct_range": 4432725233827461600, "num_eq": 6000000000, "num_range": 4432725233827461839, "upper_bound": "1.2887211553525353"}, {"distinct_range": 46574290.0708959, "num_eq": 7530170386432839749, "num_range": 100000000, "upper_bound": "1.48346814650112"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col2_26"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16, "histo_buckets": [{"distinct_range": 0, "num_eq": 7489511931117290032, "num_range": 0, "upper_bound": "000000000000000000000"}, {"distinct_range": 0, "num_eq": 70, "num_range": 8587157545540606528, "upper_bound": "001011100000010100001"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 200000, "upper_bound": "010011001100010101101"}, {"distinct_range": 5840446843340852000, "num_eq": 1972357318870922929, "num_range": 7519562552655843944, "upper_bound": "011101000011101101001"}, {"distinct_range": 28439420923.999187, "num_eq": 60000000, "num_range": 30000000000, "upper_bound": "011101000110011101100"}, {"distinct_range": 1447809.4137270083, "num_eq": 70, "num_range": 2000000, "upper_bound": "110010100010111110000"}, {"distinct_range": 7897677223389150000, "num_eq": 100000000000, "num_range": 8415061225687477526, "upper_bound": "111000011011100001111"}, {"distinct_range": 959017243163816700, "num_eq": 6038877386954297748, "num_range": 959017243163816742, "upper_bound": "111011010100110010100"}], "histo_col_type": "BIT(21)", "name": "__auto__", "null_count": 354, "row_count": 500}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 334, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 477, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 93, "histo_buckets": [{"distinct_range": 0, "num_eq": 2452301053233721257, "num_range": 0, "upper_bound": "-4713-11-24 00:00:00"}, {"distinct_range": 9171081266394490000, "num_eq": 1000, "num_range": 9171081266394489544, "upper_bound": "1970-10-26 19:11:56.000969"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 8000000, "upper_bound": "1977-12-24 18:43:27.000179"}, {"distinct_range": 1562707486568097000, "num_eq": 2000000, "num_range": 2223020697375432417, "upper_bound": "1979-05-31 22:18:26.00056"}, {"distinct_range": 7698176722899023000, "num_eq": 262362536501969383, "num_range": 8288764824954750182, "upper_bound": "1986-03-15 06:28:18.000771"}, {"distinct_range": 0, "num_eq": 40000000000, "num_range": 6195098400928183729, "upper_bound": "1995-04-13 18:53:34.000358"}, {"distinct_range": 417486537072257500, "num_eq": 100000000, "num_range": 1176630449334114115, "upper_bound": "2009-03-23 05:42:31.00093"}, {"distinct_range": 0, "num_eq": 5451759230504288505, "num_range": 0, "upper_bound": "2032-11-16 18:43:13.000544"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 124, "row_count": 500}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 69, "histo_col_type": "", "name": "__auto__", "null_count": 305, "row_count": 500}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000, "num_range": 0, "upper_bound": "1986-02-26 02:26:37.000964"}, {"distinct_range": 4490154507102400000, "num_eq": 50000000000, "num_range": 6368567973930423664, "upper_bound": "1995-12-31 00:09:43.000431"}, {"distinct_range": 7363623950065433000, "num_eq": 600, "num_range": 7363623950065432893, "upper_bound": "1997-08-21 12:51:16.000289"}, {"distinct_range": 4213106967928637000, "num_eq": 300000, "num_range": 4488683063243076187, "upper_bound": "1998-01-18 06:26:43.000887"}, {"distinct_range": 7921767229472094000, "num_eq": 40000, "num_range": 7921767229472094354, "upper_bound": "2003-06-17 00:43:32.000662"}, {"distinct_range": 9194634693840966000, "num_eq": 1004612003988043791, "num_range": 9194634693840965292, "upper_bound": "2018-01-25 01:09:55.000317"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col2_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col2_27"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 347, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-0.8406831129328074"}, {"distinct_range": 44.002150187920286, "num_eq": 8627862064722681941, "num_range": 90, "upper_bound": "0.7480299842333052"}, {"distinct_range": 90, "num_eq": 1000000000, "num_range": 90, "upper_bound": "1.0"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 198, "row_count": 500}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 351, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 180, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 49, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 488, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 267, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 87, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500}, {"columns": ["col2_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 242, "histo_col_type": "", "name": "__auto__", "null_count": 302, "row_count": 500}, {"columns": ["col2_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 326, "histo_buckets": [{"distinct_range": 0, "num_eq": 2745781684260381018, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 100000, "num_eq": 60000000000, "num_range":

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2743160&tab=buildLog) on [master@37cdf01299861235888af3411e699480f5a2d670](https://github.com/cockroachdb/cockroach/commits/37cdf01299861235888af3411e699480f5a2d670):

```
        postgres_1    | 2021-03-04 06:11:37.122 UTC [93] ERROR:  type "geometry" does not exist at character 236
        postgres_1    | 2021-03-04 06:11:37.122 UTC [93] STATEMENT:  SELECT
        postgres_1    | 		*
        postgres_1    | 	FROM
        postgres_1    | 		(
        postgres_1    | 			SELECT
        postgres_1    | 				count(*) AS col_57177
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table3 AS tab_34616
        postgres_1    | 			WHERE
        postgres_1    | 				false
        postgres_1    | 			GROUP BY
        postgres_1    | 				tab_34616.col3_4
        postgres_1    | 			HAVING
        postgres_1    | 				_st_overlaps(st_mpolyfromwkb('\x41'::BYTEA::BYTEA, 2::INT8::INT8)::GEOMETRY::GEOMETRY, NULL::GEOMETRY)::BOOL
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_34616.col3_4
        postgres_1    | 		)
        postgres_1    | 			AS tab_34617
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_57177 NULLS FIRST
        test_1        | --- FAIL: TestCompare (641.59s)
        test_1        |     --- FAIL: TestCompare/postgres (40.71s)
        test_1        |         compare_test.go:155: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("7.0J"),
        test_1        |             + 	string("7J"),
        test_1        |               }))
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT (7.0::FLOAT8::FLOAT8 || 'J'::TEXT::TEXT)::TEXT AS col_57178) AS tab_34618
        test_1        |             ORDER BY
        test_1        |             	col_57178 NULLS FIRST;
        test_1        |             
        test_1        |             cockroach1: [same as previous]
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2747306&tab=buildLog) on [master@b703e663da8ededaee2e28fc39a24e3880ae54cf](https://github.com/cockroachdb/cockroach/commits/b703e663da8ededaee2e28fc39a24e3880ae54cf):

```
        postgres_1    | 		col_40775 NULLS FIRST
        postgres_1    | 2021-03-05 06:12:37.316 UTC [93] ERROR:  column tab_25918.crdb_internal_mvcc_timestamp does not exist at character 31
        postgres_1    | 2021-03-05 06:12:37.316 UTC [93] STATEMENT:  SELECT
        postgres_1    | 		*
        postgres_1    | 	FROM
        postgres_1    | 		(
        postgres_1    | 			SELECT
        postgres_1    | 				tab_25918.crdb_internal_mvcc_timestamp AS col_40776
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table2 AS tab_25918
        postgres_1    | 			WHERE
        postgres_1    | 				tab_25918.col2_0
        postgres_1    | 		)
        postgres_1    | 			AS tab_25919
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_40776 NULLS FIRST
        test_1        | --- FAIL: TestCompare (628.56s)
        test_1        |     --- FAIL: TestCompare/postgres (27.81s)
        test_1        |         compare_test.go:155: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string(`F0[{"E": {"E": ["F"]}}, "J", {"A": {"B": "E", "I": 5}, "E": false}]`),
        test_1        |             + 	string(`F0'[{"E": {"E": ["F"]}}, "J", {"A": {"B": "E", "I": 5}, "E": false}]'`),
        test_1        |               }))
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			(('F'::TEXT::TEXT || 0::OID::OID)::TEXT::TEXT || '[{"E": {"E": ["F"]}}, "J", {"A": {"B": "E", "I": 5}, "E": false}]'::JSONB::JSONB)::TEXT
        test_1        |             				AS col_40777
        test_1        |             	)
        test_1        |             		AS tab_25920
        test_1        |             ORDER BY
        test_1        |             	col_40777 NULLS FIRST;
        test_1        |             
        test_1        |             cockroach1: [same as previous]
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2753015&tab=buildLog) on [master@a69e6549a71f5a0e83eb13509001f4d7351050fb](https://github.com/cockroachdb/cockroach/commits/a69e6549a71f5a0e83eb13509001f4d7351050fb):

```
        test_1        |             CREATE UNIQUE INDEX ON table3 (col1_0 DESC, col2_3, col2_10, col2_11);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col2_6);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col1_0 ASC, col2_10 ASC, col2_4, col2_2 DESC, col2_3 ASC);
        test_1        |             CREATE TABLE table4 (col1_1 INET NOT NULL, col1_0 BOOL NOT NULL, col4_2 INT8, col4_3 BOOL NULL, col4_4 BOOL NOT NULL, col4_5 INT8 NOT NULL, col4_6 INET NOT NULL, col4_7 FLOAT8 NOT NULL, col4_8 NAME NOT NULL, col4_9 INT8 NOT NULL, col4_10 DECIMAL NOT NULL, col4_11 CHAR NULL, col4_12 "char", col4_13 INET NOT NULL, col4_14 BIT(26) NULL, col4_15 BIT(42), col4_16 BIT(29), PRIMARY KEY (col1_1, col1_0, col4_7, col4_16, col4_10, col4_6, col4_8, col4_12, col4_3, col4_15, col4_13, col4_5, col4_9, col4_2, col4_14));
        test_1        |             CREATE INDEX ON table4 (col4_6, col4_15 ASC, col4_5);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_15 ASC, col4_3 ASC, col4_16, col4_13 ASC, col4_12 DESC, col4_6 DESC, col4_5 DESC) INCLUDE (col4_4, col4_11);
        test_1        |             CREATE INDEX ON table4 (col4_12 DESC, col4_6, col4_5, col4_16 DESC, col1_0 ASC);
        test_1        |             CREATE INDEX ON table4 (col4_13 ASC, col4_3 ASC, col4_12 ASC, col4_16 ASC, col4_14 DESC, col4_5 ASC, col1_0 ASC, col4_7 DESC);
        test_1        |             CREATE INDEX ON table4 (col4_3 DESC, col4_10, col4_2 DESC, col4_14 DESC, col4_5 ASC, col4_8, col4_15 ASC, col4_7, col1_0 ASC, col4_12 DESC, col1_1 DESC, col4_6 DESC, col4_16 DESC, col4_11, col4_4 DESC, col4_9 ASC);
        test_1        |             CREATE TABLE table5 (col5_0 UUID[] NULL, col5_1 OID NOT NULL, col5_2 FLOAT8, col5_3 REGTYPE NOT NULL, col5_4 TIMESTAMPTZ NOT NULL, col5_5 "char" NOT NULL, col5_6 BOX2D NOT NULL, col5_7 VARCHAR NOT NULL, col5_8 BOOL NOT NULL, col5_9 REGNAMESPACE NOT NULL, PRIMARY KEY (col5_9, col5_4, col5_8, col5_5, col5_6, col5_1, col5_3), col5_10 TEXT NOT NULL AS (lower(col5_5)) STORED, col5_11 FLOAT8 NOT NULL AS (col5_2 + (-0.9375656768674862)::FLOAT8) STORED, col5_12 TEXT NULL AS (lower(CAST(col5_3 AS TEXT))) VIRTUAL);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('focd', 'gyirq', 'g', 'v', 'gmst', 'unrhkh');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('v', 'jmoxi', 'fgmsn', 'dmbes', 'laygbj');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: syntax error at or near "AS" (SQLSTATE 42601)
        test_1        |     --- FAIL: TestCompare/mutators (0.78s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 INTERVAL NOT NULL, col1_1 BOOL NOT NULL, col1_2 FLOAT4 NOT NULL, PRIMARY KEY (col1_0 ASC, col1_1, col1_2), col1_3 STRING NOT NULL AS (lower(CAST(col1_0 AS STRING))) STORED, UNIQUE (col1_2 DESC, col1_0 DESC, col1_3) WHERE NOT table1.col1_1, INDEX (col1_0 ASC, col1_1 DESC, col1_2, col1_3 ASC), INDEX (col1_0, col1_1 ASC, col1_2 ASC) STORING (col1_3) WHERE table1.col1_1, INDEX (col1_0, col1_1 ASC, col1_2 DESC) STORING (col1_3) WHERE ((table1.col1_3 >= 'X':::STRING) AND (table1.col1_2 >= (-1.0):::FLOAT8)) AND table1.col1_1, UNIQUE (col1_3, col1_0, col1_1 DESC, col1_2) WHERE table1.col1_2 > 1.0:::FLOAT8, FAMILY (col1_2, col1_1, col1_0), FAMILY (col1_3));
        test_1        |             CREATE TABLE table2 (col1_0 INTERVAL NOT NULL, col1_1 BOOL NOT NULL, col1_2 FLOAT4 NOT NULL, col2_3 CHAR, col2_4 BOOL, col2_5 NAME NULL, col2_6 INTERVAL NULL, col2_7 TIMESTAMPTZ NULL, col2_8 DECIMAL NULL, col2_9 VARCHAR NOT NULL, PRIMARY KEY (col1_0 ASC, col1_1, col1_2, col2_9, col2_4 ASC, col2_5, col2_6 ASC), INDEX (col2_3, col2_4, col1_0 DESC, col2_7, col2_6 DESC, col1_2 DESC) WHERE ((((((table2.col2_3 = e'\U00002603':::STRING) OR (table2.col1_2 >= 1.401298464324817e-45:::FLOAT8)) AND (table2.col2_7 != '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ)) AND (NOT table2.col1_1)) OR (NOT table2.col2_4)) AND (table2.col2_8 > 'Infinity':::DECIMAL)) OR (table2.col2_9 = e'\'':::STRING), UNIQUE (col2_7, col2_6, col1_1 ASC, col2_9, col2_3 ASC, col1_2 ASC, col2_5 ASC, col2_8, col2_4 DESC, col1_0 DESC) WHERE (((((table2.col2_8 < -1) OR (table2.col2_9 < '')) OR (NOT table2.col1_1)) OR (table2.col2_5 = e'\'')) AND (NOT table2.col2_4)) OR (table2.col1_2 = 3.4028234663852886e+38), UNIQUE (col1_0, col1_1) WHERE ((table2.col2_9 < '"') AND (NOT table2.col2_4)) OR (table2.col2_8 >= Infinity), UNIQUE (col1_2) WHERE (((((NOT table2.col1_1) OR (table2.col2_7 <= '0001-01-01 00:00:00+00:00')) OR (table2.col2_9 > e'\x00')) AND (NOT table2.col2_4)) OR (table2.col2_8 < Infinity)) OR (table2.col2_5 < e'\U00002603'), FAMILY (col1_0), FAMILY (col2_7, col2_8, col1_2), FAMILY (col1_1, col2_3), FAMILY (col2_6), FAMILY (col2_5, col2_4), FAMILY (col2_9)) INTERLEAVE IN PARENT table1 (col1_0, col1_1, col1_2);
        test_1        |             CREATE TABLE table3 (col3_0 REGCLASS NOT NULL, col3_1 BIT(6) NOT NULL, col3_2 INET NOT NULL, col3_3 VARCHAR, col3_4 BIT(48) NOT NULL, col3_5 REGTYPE NOT NULL, col3_6 INTERVAL, col3_7 STRING NOT NULL, col3_8 TIMESTAMPTZ[] NULL, col3_9 INET[] NOT NULL, col3_10 STRING NOT NULL, col3_11 INT2 NOT NULL, col3_12 FLOAT4 NOT NULL, col3_13 DECIMAL NOT NULL, col3_14 INTERVAL NOT NULL, PRIMARY KEY (col3_1 DESC, col3_2, col3_13 ASC, col3_0 ASC, col3_12, col3_4 DESC, col3_10 DESC, col3_5 DESC, col3_14, col3_7 DESC, col3_11 DESC), FAMILY (col3_12), FAMILY (col3_13, col3_3), FAMILY (col3_2), FAMILY (col3_11, col3_9, col3_4, col3_5, col3_8, col3_6, col3_1, col3_14), FAMILY (col3_0, col3_10), FAMILY (col3_7));
        test_1        |             CREATE TABLE table4 (col4_0 BOOL NOT NULL, PRIMARY KEY (col4_0 DESC), UNIQUE (col4_0 ASC) WHERE table4.col4_0, INDEX (col4_0 DESC) WHERE table4.col4_0, INDEX (col4_0 DESC), UNIQUE (col4_0) WHERE NOT table4.col4_0, INDEX (col4_0 DESC) WHERE NOT table4.col4_0);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 79196605960251280, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-79 years -2 mons -70 days -12:20:08.146321"}, {"distinct_range": 5490.168981520715, "num_eq": 0, "num_range": 20000, "upper_bound": "-43 years -6 mons -453 days -18:38:18.542467"}, {"distinct_range": 730594.3319651493, "num_eq": 30000000000, "num_range": 1000000, "upper_bound": "-5 years -9 mons -433 days -10:28:26.364534"}, {"distinct_range": 10262335788141418, "num_eq": 0, "num_range": 245729826884324308, "upper_bound": "3 mons 331 days 07:11:21.610326"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 5400899098657598663}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1706118621225121754, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5400899098657598663}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1069943805476436752, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "-0.11048372834920883"}, {"distinct_range": 576860859093206300, "num_eq": 6004615879563793583, "num_range": 4517388546945580816, "upper_bound": "0.251321405172348"}, {"distinct_range": 6308861377282448000, "num_eq": 6867087207420249425, "num_range": 6308861377282448776, "upper_bound": "0.6030879616737366"}, {"distinct_range": 0, "num_eq": 8784916515399704155, "num_range": 0, "upper_bound": "1.1463569402694702"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 5400899098657598663}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1886709535182471880, "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 5400899098657598663}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45, "histo_col_type": "", "name": "__auto__", "null_count": 33, "row_count": 60}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39, "histo_col_type": "", "name": "__auto__", "null_count": 52, "row_count": 60}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 41, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "-22 years -7 mons -510 days -01:36:07.530209"}, {"distinct_range": 4830054171591930000, "num_eq": 7000000, "num_range": 7933181000071776532, "upper_bound": "-2 years -8 mons -820 days -12:57:27.648054"}, {"distinct_range": 171.0332413170249, "num_eq": 10000000, "num_range": 7000, "upper_bound": "-1 years -7 mons -808 days -08:22:20.601554"}, {"distinct_range": 0, "num_eq": 800, "num_range": 80000000000, "upper_bound": "44 years 7 mons 55 days 06:23:59.676042"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 60000, "upper_bound": "72 years 10 mons 463 days 08:43:48.603624"}, {"distinct_range": 0, "num_eq": 1202812101624244714, "num_range": 40000, "upper_bound": "79 years 2 mons 318 days 09:07:51.697808"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 52, "histo_col_type": "", "name": "__auto__", "null_count": 25, "row_count": 60}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42, "histo_col_type": "", "name": "__auto__", "null_count": 5, "row_count": 60}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 50, "histo_col_type": "CHAR", "name": "__auto__", "null_count": 49, "row_count": 60}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 55, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000, "num_range": 0, "upper_bound": "1988-02-02 20:28:37.00044+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 35, "row_count": 60}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 2954739137700684910, "num_range": 0, "upper_bound": "110011"}], "histo_col_type": "BIT(6)", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 836622769660801694, "histo_buckets": [{"distinct_range": 0, "num_eq": 737901937211062501, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 7078762953490225305, "num_range": 20000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 1830916667604516163}]':::JSONB;
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_0, col1_1) REFERENCES table2 (col1_0, col1_1) ON DELETE CASCADE;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('hwgn');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 2395017136048865782, "num_range": 0, "upper_bound": "-Inf"}, {"distinct_range": 8391037619156851000, "num_eq": 90000000, "num_range": 8391037619156850713, "upper_bound": "-2.5504941940307617"}, {"distinct_range": 5787000767511247000, "num_eq": 2000000, "num_range": 5787000767511246972, "upper_bound": "-0.7717456817626953"}, {"distinct_range": 3626018756685432300, "num_eq": 6232691170618239509, "num_range": 3626018756685432143, "upper_bound": "-0.5310457944869995"}, {"distinct_range": 800000, "num_eq": 9000000000, "num_range": 800000, "upper_bound": "-0.09209328144788742"}, {"distinct_range": 6210568423128548000, "num_eq": 4000000000, "num_range": 6210568423128548757, "upper_bound": "0.7523565292358398"}, {"distinct_range": 3595673695237039600, "num_eq": 6793268175451204777, "num_range": 3595673695237039815, "upper_bound": "0.7801294922828674"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 10000000000, "upper_bound": "1.0"}, {"distinct_range": 2311644943439833000, "num_eq": 100000000000, "num_range": 2311644943439832905, "upper_bound": "3.4028234663852886e+38"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 8781041706096683049, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 400000, "upper_bound": "4T;"}, {"distinct_range": 50000000, "num_eq": 7989942209939188960, "num_range": 50000000, "upper_bound": "?<T\\u0001@"}, {"distinct_range": 0, "num_eq": 50, "num_range": 10000000, "upper_bound": "?L\\u0014D\\u0005rj\\t"}, {"distinct_range": 0, "num_eq": 30000000000, "num_range": 6526772658997115921, "upper_bound": "mM%h\\u001b"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 1557100043075267422, "num_range": 0, "upper_bound": "-4 years -2 mons -820 days -06:11:34.760047"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2541031878823357007, "histo_col_type": "", "name": "__auto__", "null_count": 1480036483051995473, "row_count": 3229464096032393642}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 424932914300793482, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3229464096032393642}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 945596207051162046, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3229464096032393642}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3114840222157071339, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3229464096032393642}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1671876939718463799, "histo_col_type": "", "name": "__auto__", "null_count": 2277133264495307928, "row_count": 3229464096032393642}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2335202610600019860, "histo_col_type": "", "name": "__auto__", "null_count": 1271340686666362990, "row_count": 3229464096032393642}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 891955771278776408, "histo_col_type": "", "name": "__auto__", "null_count": 963836975788137455, "row_count": 3229464096032393642}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 208363745296069659, "histo_buckets": [{"distinct_range": 0, "num_eq": 7454198047156199177, "num_range": 0, "upper_bound": "1970-03-06 03:10:19.000701+00:00"}, {"distinct_range": 0, "num_eq": 645090224221304543, "num_range": 90000, "upper_bound": "1993-07-11 02:39:08.000822+00:00"}, {"distinct_range": 471940091.11812854, "num_eq": 6000000, "num_range": 10000000000, "upper_bound": "2007-09-08 16:31:34.000875+00:00"}, {"distinct_range": 87090474392629860, "num_eq": 5374895985719650457, "num_range": 413399340578559091, "upper_bound": "2013-09-20 12:51:52.000759+00:00"}, {"distinct_range": 0, "num_eq": 1633496563595427427, "num_range": 0, "upper_bound": "2016-02-17 10:13:25.000908+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 2974062905372334092, "row_count": 3229464096032393642}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1879760386620282515, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000, "num_range": 0, "upper_bound": "-51 years -3 mons -441 days -12:06:12.157988"}, {"distinct_range": 70000000000, "num_eq": 10, "num_range": 70000000000, "upper_bound": "-12 years -7 mons -383 days -22:40:34.626616"}, {"distinct_range": 400000, "num_eq": 4296706200270010273, "num_range": 400000, "upper_bound": "00:00:00"}, {"distinct_range": 2241105668520076000, "num_eq": 6000000, "num_range": 3742220859340887480, "upper_bound": "21 years 11 mons 855 days 19:58:32.879818"}, {"distinct_range": 7397976534124131000, "num_eq": 1000000000, "num_range": 7397976534124131398, "upper_bound": "25 years 8 mons 535 days 24:28:45.283109"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 3229464096032393642}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1077991666163390382, "histo_col_type": "CHAR", "name": "__auto__", "null_count": 2828121979292986338, "row_count": 3229464096032393642}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 3183210051150806160, "num_range": 0, "upper_bound": "000000"}, {"distinct_range": 4844603359049065000, "num_eq": 1856479117989023769, "num_range": 4844603359049065239, "upper_bound": "000011"}, {"distinct_range": 2367412391730451000, "num_eq": 900000000, "num_range": 5517265684909587897, "upper_bound": "011111"}, {"distinct_range": 0, "num_eq": 3820648922678630887, "num_range": 10000000, "upper_bound": "100000"}, {"distinct_range": 90724.76617129614, "num_eq": 6098140784170501418, "num_range": 100000, "upper_bound": "101101"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 200, "upper_bound": "110011"}, {"distinct_range": 0, "num_eq": 800, "num_range": 50000000, "upper_bound": "111001"}], "histo_col_type": "BIT(6)", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4197107610453098901, "histo_buckets": [{"distinct_range": 0, "num_eq": 6473746900697992558, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 2728482381054190000, "num_eq": 8320242966751217870, "num_range": 3304752460157119285, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 8878175821411329597}]';
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_2) REFERENCES table2 (col1_2) ON DELETE RESTRICT;
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 62959300, "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27534574, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 74176382, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000, "num_range": 0, "upper_bound": "-1.7007179260253906"}, {"distinct_range": 2539782.400747123, "num_eq": 8795903201563189307, "num_range": 5000000, "upper_bound": "-1.6839162111282349"}, {"distinct_range": 935305087777705100, "num_eq": 7096226057792065168, "num_range": 1067706902804988559, "upper_bound": "0.7313703298568726"}, {"distinct_range": 7746960999937702000, "num_eq": 118513250149217825, "num_range": 7746960999937701526, "upper_bound": "3.4028234663852886e+38"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36830887, "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 100000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4072345534492998757, "histo_col_type": "", "name": "__auto__", "null_count": 4582388041731040201, "row_count": 5141315667969328798}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2941713012820182487, "histo_buckets": [{"distinct_range": 0, "num_eq": 6122867518441497528, "num_range": 0, "upper_bound": "1977-05-31 13:19:16.000527+00:00"}, {"distinct_range": 0, "num_eq": 60000000000, "num_range": 700000, "upper_bound": "1988-03-20 05:45:32.00068+00:00"}, {"distinct_range": 1000000, "num_eq": 20000000000, "num_range": 1000000, "upper_bound": "2021-10-30 11:05:30.000593+00:00"}, {"distinct_range": 1000000, "num_eq": 7583682034784661835, "num_range": 1000000, "upper_bound": "2029-02-14 02:42:48.000781+00:00"}], "histo_col_type": "TIMESTAMPTZ", "name": "__auto__", "null_count": 4425667780681353674, "row_count": 5141315667969328798}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3397492705360676399, "histo_col_type": "", "name": "__auto__", "null_count": 4020031807629497382, "row_count": 5141315667969328798}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3551530664477815019, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5141315667969328798}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3003437755954208442, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000000, "num_range": 0, "upper_bound": "-1.0"}, {"distinct_range": 404086.6780858905, "num_eq": 6923529822235801820, "num_range": 900000, "upper_bound": "-0.8853101134300232"}, {"distinct_range": 0, "num_eq": 1515843247179626349, "num_range": 6625873628972848168, "upper_bound": "-0.23322097957134247"}, {"distinct_range": 4075222034955150300, "num_eq": 5000000000, "num_range": 4075222034955150441, "upper_bound": "0.0"}, {"distinct_range": 3351351597537947000, "num_eq": 100, "num_range": 3351351597537947126, "upper_bound": "0.15363696217536926"}, {"distinct_range": 1111553834111595600, "num_eq": 7028838145628854118, "num_range": 1111553834111595608, "upper_bound": "0.22961659729480743"}, {"distinct_range": 35.46268899808339, "num_eq": 500000, "num_range": 70, "upper_bound": "0.6954238414764404"}, {"distinct_range": 0, "num_eq": 800, "num_range": 6759933977274774757, "upper_bound": "1.2324718236923218"}, {"distinct_range": 4248.629125590746, "num_eq": 0, "num_range": 9000, "upper_bound": "1.3746955394744873"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 5141315667969328798}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2983352975473517148, "histo_col_type": "", "name": "__auto__", "null_count": 32889237044547295, "row_count": 5141315667969328798}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2376252830467856394, "histo_col_type": "", "name": "__auto__", "null_count": 860923401793558169, "row_count": 5141315667969328798}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4080221331797555643, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5141315667969328798}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3022604837192393244, "histo_buckets": [{"distinct_range": 0, "num_eq": 2925801904599908437, "num_range": 0, "upper_bound": "-82 years -7 mons -999 days -02:51:45.992131"}, {"distinct_range": 89.6074881874809, "num_eq": 4107273756695146805, "num_range": 200, "upper_bound": "-57 years -9 mons -390 days -04:16:26.660642"}, {"distinct_range": 200, "num_eq": 5250374149354888275, "num_range": 200, "upper_bound": "-55 years -6 mons -561 days -23:08:02.780003"}, {"distinct_range": 0, "num_eq": 2439821858188717797, "num_range": 7000000000, "upper_bound": "-35 years -654 days -20:18:13.404268"}, {"distinct_range": 5000, "num_eq": 1873657974556246704, "num_range": 5000, "upper_bound": "37 years 480 days 21:17:42.542332"}, {"distinct_range": 0, "num_eq": 8473007387484396186, "num_range": 1844831682305702557, "upper_bound": "62 years 9 mons 20 days 10:59:03.751163"}, {"distinct_range": 4553373879977798700, "num_eq": 2000000000, "num_range": 4553373879977798836, "upper_bound": "75 years 6 mons 514 days 08:11:32.900584"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 5141315667969328798}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2771603938569754250, "histo_buckets": [{"distinct_range": 0, "num_eq": 7566551585304918596, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 1408166417072780011, "upper_bound": "."}, {"distinct_range": 7126793280233996000, "num_eq": 3165376883821717488, "num_range": 7126793280233996492, "upper_bound": "?"}, {"distinct_range": 0, "num_eq": 5676068741830539600, "num_range": 9000000000, "upper_bound": "V"}, {"distinct_range": 927288661532070500, "num_eq": 6306062897255132876, "num_range": 955618412645999624, "upper_bound": "W"}, {"distinct_range": 0, "num_eq": 200, "num_range": 3000000, "upper_bound": "["}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 300000000, "upper_bound": "\\\\"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 2153343309800370233, "row_count": 5141315667969328798}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 62, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15, "histo_col_type": "", "name": "__auto__", "null_count": 13, "row_count": 80}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27, "histo_col_type": "", "name": "__auto__", "null_count": 24, "row_count": 80}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 75, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 54, "histo_buckets": [{"distinct_range": 0, "num_eq": 4732585123317967903, "num_range": 0, "upper_bound": "101110"}, {"distinct_range": 4891400.879335192, "num_eq": 2541846629406309443, "num_range": 20000000, "upper_bound": "110110"}, {"distinct_range": 10000000, "num_eq": 2639572369452717126, "num_range": 10000000, "upper_bound": "111010"}, {"distinct_range": 7876629630513914000, "num_eq": 5000000, "num_range": 8389942395163691077, "upper_bound": "111100"}], "histo_col_type": "BIT(6)", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 77, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 50, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 63, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25, "histo_col_type": "", "name": "__auto__", "null_count": 69, "row_count": 80}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 63089512, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 20000000, "num_eq": 0, "num_range": 20000000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 800000000}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "infinity" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```


[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2757390&tab=buildLog) on [master@89c379950cb4ca77c5d25e45aac4008b1f1758d9](https://github.com/cockroachdb/cockroach/commits/89c379950cb4ca77c5d25e45aac4008b1f1758d9):

```
        postgres_1    | You can change this by editing pg_hba.conf or using the option -A, or
        postgres_1    | --auth-local and --auth-host, the next time you run initdb.
        postgres_1    | waiting for server to start....2021-03-09 06:11:23.311 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-03-09 06:11:23.329 UTC [49] LOG:  database system was shut down at 2021-03-09 06:11:23 UTC
        postgres_1    | 2021-03-09 06:11:23.334 UTC [48] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        test_1        | --- FAIL: TestCompare (1.46s)
        test_1        |     --- FAIL: TestCompare/postgres (0.42s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (1.04s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 UUID NOT NULL, col1_1 INTERVAL NOT NULL, col1_2 INET NOT NULL, col1_3 REGCLASS, col1_4 INT8 NOT NULL, col1_5 BOOL NOT NULL, col1_6 TIMESTAMPTZ NOT NULL, col1_7 STRING NOT NULL, col1_8 TIMETZ, col1_9 UUID, col1_10 BOX2D, col1_11 NAME, col1_12 TIMESTAMPTZ, col1_13 REGCLASS NOT NULL, col1_14 REGPROC NULL, PRIMARY KEY (col1_7, col1_5 DESC), col1_15 STRING NULL AS (lower(CAST(col1_5 AS STRING))) VIRTUAL, col1_16 STRING NULL AS (lower(CAST(col1_1 AS STRING))) STORED, col1_17 STRING NOT NULL AS (lower(CAST(col1_13 AS STRING))) VIRTUAL, INDEX (col1_13, col1_10 DESC, col1_7 ASC, col1_8 ASC, col1_4 DESC) STORING (col1_2, col1_3, col1_6, col1_9, col1_11, col1_12) WHERE (table1.col1_11 = '':::STRING) AND (table1.col1_7 != e'\x00':::STRING), INDEX (col1_3 DESC, col1_1, col1_0, col1_17 ASC, col1_8 DESC, col1_12 DESC, col1_6 DESC, col1_10 ASC, col1_4 ASC, col1_5 ASC, col1_9, col1_16, col1_2 ASC, col1_14 DESC, col1_15, col1_13) STORING (col1_11) WHERE (((table1.col1_8 > '24:00:00-15:59:00':::TIMETZ) AND (table1.col1_15 <= 'X':::STRING)) AND (table1.col1_16 < e'\x00':::STRING)) OR table1.col1_5, INDEX (col1_0) STORING (col1_2, col1_3, col1_4, col1_8, col1_10, col1_11, col1_13, col1_14, col1_16) WHERE (((((table1.col1_11 <= 'X':::STRING) OR (table1.col1_6 != '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) AND (table1.col1_17 > '':::STRING)) OR (table1.col1_4 = 2147483647:::INT8)) OR (table1.col1_7 > e'\U00002603':::STRING)) AND (table1.col1_16 > '"':::STRING), INDEX (col1_7, col1_0 DESC, col1_11 ASC, col1_14, col1_16 ASC, col1_15 DESC, col1_8, col1_13 ASC, col1_12, col1_3 ASC, col1_2 ASC, col1_4, col1_6) STORING (col1_10), FAMILY (col1_4, col1_14), FAMILY (col1_0, col1_6), FAMILY (col1_2), FAMILY (col1_13, col1_3, col1_12, col1_9), FAMILY (col1_10), FAMILY (col1_11, col1_1), FAMILY (col1_8), FAMILY (col1_7), FAMILY (col1_5, col1_16));
        test_1        |             CREATE TABLE table2 (col1_7 STRING NOT NULL, col1_5 BOOL NOT NULL, col2_2 INT8 NULL, col2_3 INT4, col2_4 BOOL NULL, col2_5 INT4 NOT NULL, col2_6 DECIMAL NULL, col2_7 INT2 NOT NULL, col2_8 OID, col2_9 OID, col2_10 FLOAT8 NOT NULL, col2_11 UUID NULL, col2_12 TIME, col2_13 BYTES NULL, col2_14 DATE NOT NULL, col2_15 REGTYPE NULL, col2_16 BOX2D NOT NULL, col2_17 REGPROCEDURE NOT NULL, col2_18 BYTES NULL, col2_19 BOOL, col2_20 UUID, PRIMARY KEY (col1_7, col1_5 DESC, col2_2 ASC, col2_16 ASC, col2_12 DESC, col2_4 ASC, col2_11), FAMILY (col1_7, col2_8), FAMILY (col2_18, col2_17), FAMILY (col2_19), FAMILY (col2_10), FAMILY (col2_3), FAMILY (col1_5, col2_9, col2_12, col2_5), FAMILY (col2_11, col2_14), FAMILY (col2_6), FAMILY (col2_20, col2_15), FAMILY (col2_13, col2_2, col2_16), FAMILY (col2_7), FAMILY (col2_4)) INTERLEAVE IN PARENT table1 (col1_7, col1_5);
        test_1        |             CREATE TABLE table3 (col3_0 INT2 NULL, col3_1 INT2 NOT NULL, col3_2 INT4 NOT NULL, col3_3 BOX2D NOT NULL, col3_4 DATE NOT NULL, col3_5 DECIMAL, col3_6 DECIMAL NOT NULL, col3_7 CHAR NOT NULL, col3_8 OID NOT NULL, col3_9 BIT(39) NOT NULL, col3_10 TIME NOT NULL, col3_11 TIME, PRIMARY KEY (col3_7 ASC, col3_3 DESC, col3_6, col3_9 DESC, col3_4 ASC, col3_8, col3_2 ASC, col3_10 ASC), col3_12 INT2 AS (col3_0 + (-5432):::INT8) VIRTUAL, UNIQUE (col3_4, col3_10, col3_5 DESC, col3_1 ASC, col3_7 DESC, col3_11 ASC) WHERE ((table3.col3_5 >= 'NaN':::DECIMAL) AND (table3.col3_0 > 0:::INT8)) AND (table3.col3_12 > (-1):::INT8), UNIQUE (col3_4 DESC, col3_9, col3_10 DESC, col3_3, col3_12 DESC, col3_5, col3_0, col3_11 ASC, col3_6 ASC) STORING (col3_1) WHERE (((((((((table3.col3_12 >= (-1):::INT8) AND (table3.col3_11 != '24:00:00':::TIME)) OR (table3.col3_1 >= 0:::INT8)) OR (table3.col3_7 = '':::STRING)) AND (table3.col3_0 >= (-1):::INT8)) AND (table3.col3_5 = 1:::DECIMAL)) OR (table3.col3_2 <= (-1):::INT8)) OR (table3.col3_10 = '00:00:00':::TIME)) OR (table3.col3_6 <= 1:::DECIMAL)) OR (table3.col3_4 <= '-infinity':::DATE), INDEX (col3_2 ASC, col3_9 ASC, col3_11, col3_10 ASC, col3_12 ASC, col3_3 ASC, col3_5, col3_6, col3_1, col3_4 DESC, col3_8) WHERE (((table3.col3_7 != e'\'':::STRING) AND (table3.col3_1 < 1:::INT8)) AND (table3.col3_12 > (-1):::INT8)) OR (table3.col3_6 < '-Infinity':::DECIMAL), INDEX (col3_0 ASC, col3_6 ASC, col3_9 DESC, col3_8, col3_7 DESC, col3_11 DESC, col3_5, col3_2 ASC, col3_10 ASC), UNIQUE (col3_3 DESC, col3_11 ASC, col3_9 ASC, col3_12 DESC, col3_7 DESC) WHERE (((((table3.col3_1 = (-32768):::INT8) AND (table3.col3_7 > '"':::STRING)) OR (table3.col3_11 >= '24:00:00':::TIME)) OR (table3.col3_6 <= (-1.234E+401):::DECIMAL)) AND (table3.col3_4 >= '1970-01-01':::DATE)) AND (table3.col3_0 < 127:::INT8), UNIQUE (col3_0) STORING (col3_1) WHERE (((((((table3.col3_2 != 2147483647) OR (table3.col3_5 <= NaN)) OR (table3.col3_1 < 0)) AND (table3.col3_7 >= '"')) AND (table3.col3_11 > '24:00:00')) OR (table3.col3_12 < -128)) AND (table3.col3_4 = 'infinity')) OR (table3.col3_6 < 1), FAMILY (col3_10), FAMILY (col3_4), FAMILY (col3_3), FAMILY (col3_8, col3_7, col3_2), FAMILY (col3_5, col3_9), FAMILY (col3_6, col3_0), FAMILY (col3_1), FAMILY (col3_11));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5252522633127418678, "histo_buckets": [{"distinct_range": 0, "num_eq": 1566826249963425442, "num_range": 0, "upper_bound": "00000000-0000-0000-0000-000000000000"}, {"distinct_range": 54922.3281750524, "num_eq": 2603343692948982412, "num_range": 90000, "upper_bound": "0bce971d-8a2f-4174-903e-e329c4594a1f"}, {"distinct_range": 0, "num_eq": 3229207734588124075, "num_range": 70000000000, "upper_bound": "27aa06b2-23b0-4657-bf1a-868c2caa4267"}, {"distinct_range": 6250490.203857712, "num_eq": 9000, "num_range": 8000000, "upper_bound": "2adf3522-a923-4e7f-b195-0d87f94f6db3"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 1000000, "upper_bound": "2b6bdabd-4d3d-4398-9e2b-e91c00db922d"}, {"distinct_range": 0, "num_eq": 6901762048339736075, "num_range": 300000000, "upper_bound": "5e9314c2-3e6e-4ada-99f4-de63d53539b3"}, {"distinct_range": 0, "num_eq": 700, "num_range": 0, "upper_bound": "7637fe0c-fc1a-4100-8d38-ce60fee73f4a"}, {"distinct_range": 5452118470231646000, "num_eq": 30, "num_range": 5950706832199024340, "upper_bound": "db9c01ee-ca83-4cd9-a8e4-fb63545fbeaf"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 5264773068629512561}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1617510120016612513, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5264773068629512561}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2373548466731433415, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 1170562097425503500, "num_eq": 60000000000, "num_range": 3374604166267251746, "upper_bound": "2408116281"}, {"distinct_range": 7053.003687852531, "num_eq": 90000, "num_range": 9000, "upper_bound": "2697176809"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 307390625408134312, "row_count": 5264773068629512561}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3235456448938197027, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5264773068629512561}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4146689860839378297, "histo_col_type": "", "name": "__auto__", "null_count": 1575928143738560249, "row_count": 5264773068629512561}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1077391898672556377, "histo_col_type": "", "name": "__auto__", "null_count": 3888091711992327439, "row_count": 5264773068629512561}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4694106640063611410, "histo_col_type": "", "name": "__auto__", "null_count": 3536021012379154036, "row_count": 5264773068629512561}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 150630480523201899, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5264773068629512561}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3140973219383891661, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5264773068629512561}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2761215638782107736, "histo_col_type": "", "name": "__auto__", "null_count": 1716409782566935870, "row_count": 5264773068629512561}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2503032092598807648, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5264773068629512561}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3466774420260489958, "histo_buckets": [{"distinct_range": 0, "num_eq": 6167840909367221526, "num_range": 0, "upper_bound": "\\u0010+\\\\\\u00119\\u001fx\\u001a\\u0013"}, {"distinct_range": 109655612631351250, "num_eq": 200000000, "num_range": 109655612631351251, "upper_bound": "\\u0013\\"XXx"}, {"distinct_range": 2782593633653293000, "num_eq": 10, "num_range": 8013407933529129428, "upper_bound": "("}, {"distinct_range": 5000000, "num_eq": 600000000, "num_range": 5000000, "upper_bound": ",g"}, {"distinct_range": 7000, "num_eq": 5000000000, "num_range": 7000, "upper_bound": "-wGW\\u0013"}, {"distinct_range": 5191.7051612499, "num_eq": 1594205357511693941, "num_range": 10000, "upper_bound": "^8V|"}, {"distinct_range": 8636596115089288000, "num_eq": 3000000000, "num_range": 8636596115089287753, "upper_bound": "al\\u0011G&"}, {"distinct_range": 5855712511529231000, "num_eq": 8660650487161941817, "num_range": 5855712511529231142, "upper_bound": "h;h]l"}, {"distinct_range": 80000000000, "num_eq": 6419716486793675035, "num_range": 80000000000, "upper_bound": "i"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 5264773068629512561}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3803773540353696747, "histo_col_type": "", "name": "__auto__", "null_count": 2047662578572271410, "row_count": 5264773068629512561}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3451049609541250519, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000000, "num_range": 0, "upper_bound": "514561561"}, {"distinct_range": 0, "num_eq": 3741918705345740299, "num_range": 6232716396626965961, "upper_bound": "2580603870"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 5264773068629512561}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1481180118622691609, "histo_col_type": "", "name": "__auto__", "null_count": 3064988769034632803, "row_count": 5264773068629512561}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2678455511722050700, "histo_col_type": "", "name": "__auto__", "null_count": 4346267631527407850, "row_count": 5264773068629512561}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 626341303612084165, "histo_col_type": "", "name": "__auto__", "null_count": 1519484172729407089, "row_count": 5264773068629512561}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 617440104189040949, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5264773068629512561}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 531089262, "histo_col_type": "", "name": "__auto__", "null_count": 2269638, "row_count": 800000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 271861981, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 536033038, "histo_col_type": "", "name": "__auto__", "null_count": 229706442, "row_count": 800000000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 188291910, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 199445986, "histo_col_type": "", "name": "__auto__", "null_count": 566394165, "row_count": 800000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 660007258, "histo_col_type": "", "name": "__auto__", "null_count": 739896283, "row_count": 800000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 550818419, "histo_col_type": "", "name": "__auto__", "null_count": 609868552, "row_count": 800000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 294267466, "histo_col_type": "", "name": "__auto__", "null_count": 731088392, "row_count": 800000000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30885417, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 92983406, "histo_col_type": "", "name": "__auto__", "null_count": 609807474, "row_count": 800000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9314345, "histo_col_type": "", "name": "__auto__", "null_count": 31679545, "row_count": 800000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 535578442, "histo_col_type": "", "name": "__auto__", "null_count": 18626456, "row_count": 800000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 140017543, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 693771245, "histo_col_type": "", "name": "__auto__", "null_count": 772411988, "row_count": 800000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 640022028, "histo_col_type": "", "name": "__auto__", "null_count": 592275350, "row_count": 800000000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 350207829, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 521067760, "histo_buckets": [{"distinct_range": 0, "num_eq": 5323064433386738059, "num_range": 0, "upper_bound": "#\' \\u000f/"}, {"distinct_range": 123962575015661950, "num_eq": 4027352301738067254, "num_range": 5165920094039635167, "upper_bound": "\'"}, {"distinct_range": 0, "num_eq": 1281779291145931977, "num_range": 1288796926818053314, "upper_bound": "X"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 268215915, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 703864061, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 602795172, "histo_col_type": "", "name": "__auto__", "null_count": 129285361, "row_count": 800000000}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 105508335, "histo_col_type": "", "name": "__auto__", "null_count": 281705123, "row_count": 800000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 106485118579108618, "histo_buckets": [{"distinct_range": 0, "num_eq": 4295214875267173268, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 5451331042291738000, "num_eq": 0, "num_range": 5451331042291738037, "upper_bound": "BOX(-1.1560556094792798 0.5554612352952838,-1.0813831601844792 1.127071637161125)"}, {"distinct_range": 800000, "num_eq": 2730370645043636915, "num_range": 800000, "upper_bound": "BOX(-0.7718028655548825 0.8603337445586313,-0.5511990021736679 1.3867727756854478)"}, {"distinct_range": 300, "num_eq": 1000000000, "num_range": 300, "upper_bound": "BOX(0.39466094932671947 0.8112609160356898,1.797528663129175 1.167319836345842)"}, {"distinct_range": 1708445049.2811863, "num_eq": 6597420025174528477, "num_range": 5000000000, "upper_bound": "BOX(0.5871827925238902 0.23570585999199672,2.638185941921372 2.077831238724916)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 1090549937514315425}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1042821497425230013, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1090549937514315425}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 376764956841085688, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000, "num_range": 0, "upper_bound": "!"}, {"distinct_range": 0, "num_eq": 900000000, "num_range": 0, "upper_bound": "X"}, {"distinct_range": 374857415025553900, "num_eq": 4000, "num_range": 374857415025553920, "upper_bound": "a"}, {"distinct_range": 0, "num_eq": 300000000, "num_range": 40000000000, "upper_bound": "w"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 1090549937514315425}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 310162863140125791, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1090549937514315425}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 666583995914946756, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1090549937514315425}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 808149738396816397, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1090549937514315425}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 475604084811959176, "histo_buckets": [{"distinct_range": 0, "num_eq": 4255230894560249795, "num_range": 0, "upper_bound": "-1515982082"}, {"distinct_range": 100000000, "num_eq": 40000000, "num_range": 100000000, "upper_bound": "-129513251"}, {"distinct_range": 0, "num_eq": 800000, "num_range": 4000000, "upper_bound": "-8911161"}, {"distinct_range": 728878782880375200, "num_eq": 900000000, "num_range": 728878782880375215, "upper_bound": "32767"}, {"distinct_range": 997234692213603600, "num_eq": 0, "num_range": 997234692213603538, "upper_bound": "1400708496"}, {"distinct_range": 900000000, "num_eq": 90, "num_range": 900000000, "upper_bound": "1500822183"}, {"distinct_range": 134026.87573663535, "num_eq": 8000, "num_range": 1000000, "upper_bound": "2081425215"}, {"distinct_range": 70000000, "num_eq": 3689024650722625091, "num_range": 70000000, "upper_bound": "2101606374"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 1090549937514315425}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 968798266367716372, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000, "num_range": 0, "upper_bound": "-infinity"}, {"distinct_range": 0, "num_eq": 7000000, "num_range": 20000000000, "upper_bound": "1973-04-05"}, {"distinct_range": 0, "num_eq": 175214908726920441, "num_range": 2000000, "upper_bound": "1973-12-12"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 1090549937514315425}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 93827233301117453, "histo_col_type": "", "name": "__auto__", "null_count": 1083965211798778518, "row_count": 1090549937514315425}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 476069407410322681, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1090549937514315425}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21183968205422253, "histo_col_type": "", "name": "__auto__", "null_count": 27691542826758430, "row_count": 1090549937514315425}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 711691957902665380, "histo_col_type": "", "name": "__auto__", "null_count": 474329234129622264, "row_count": 1090549937514315425}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 501284723197204143, "histo_buckets": [{"distinct_range": 0, "num_eq": 6817406700735719482, "num_range": 0, "upper_bound": "-23967"}, {"distinct_range": 0, "num_eq": 700000000, "num_range": 600000000, "upper_bound": "-8212"}, {"distinct_range": 717307880.2815169, "num_eq": 7084754744759234073, "num_range": 800000000, "upper_bound": "-2442"}, {"distinct_range": 8356559417012031000, "num_eq": 9000000, "num_range": 8356559417012031887, "upper_bound": "30577"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 770566015322324505, "row_count": 1090549937514315425}]':::JSONB;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_3) REFERENCES table3 (col3_0) ON UPDATE RESTRICT;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('bzkh', 'fglpoz', 'gqvbi', 'ild');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('a', 'zdnsg', 'gbsinb', 'ocd');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('hlgln', 'yroqd', 'arlsr', 'dcsrah', 'vc');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 426004, "histo_buckets": [{"distinct_range": 0, "num_eq": 3382444488765919265, "num_range": 0, "upper_bound": "01a9cc68-507c-4e96-ad07-e30bf80fbdab"}, {"distinct_range": 169692.90066975378, "num_eq": 400000000, "num_range": 200000, "upper_bound": "24ed2fa8-56b0-4013-bbaa-e57b6eeda3c7"}, {"distinct_range": 146.66512579643683, "num_eq": 4751721683157134156, "num_range": 200, "upper_bound": "365f63a7-b56d-4da0-9fcb-935d5188e4fe"}, {"distinct_range": 2863705922311953400, "num_eq": 90, "num_range": 2863705922311953497, "upper_bound": "56b05ace-f338-495c-953b-7a70359572b5"}, {"distinct_range": 900000000, "num_eq": 8882809679768753170, "num_range": 900000000, "upper_bound": "56bb5cf0-c982-4810-b24a-806b5527d462"}, {"distinct_range": 3140256248969680400, "num_eq": 100000000, "num_range": 6766848730377890536, "upper_bound": "666b6c1a-26c9-4ead-a6f7-c23a45319acc"}, {"distinct_range": 0, "num_eq": 40, "num_range": 100000, "upper_bound": "7542b57d-d608-4749-8d04-4cf0d907b6d0"}, {"distinct_range": 4617328793388384000, "num_eq": 7425501027677313569, "num_range": 7180578257923336726, "upper_bound": "c45c19fd-05c6-4f26-87b2-edab54eeb71f"}, {"distinct_range": 1139981032069843500, "num_eq": 9000000, "num_range": 1665121001851189577, "upper_bound": "fdf213d4-4d0d-4374-a947-5b7d38708200"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 280247, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33226, "histo_buckets": [{"distinct_range": 0, "num_eq": 60, "num_range": 0, "upper_bound": "\\u0001\\u0006]D\\u0006k\\u001dm!"}, {"distinct_range": 202.46500730435213, "num_eq": 50, "num_range": 300, "upper_bound": "\\u0006N$)@1"}, {"distinct_range": 413534.4517663274, "num_eq": 50000, "num_range": 500000, "upper_bound": "1"}, {"distinct_range": 0, "num_eq": 7366843779918102567, "num_range": 125058300274987810, "upper_bound": "GF\\u0001_V)Z"}, {"distinct_range": 0, "num_eq": 1849759022243482064, "num_range": 0, "upper_bound": "KEs`S"}, {"distinct_range": 20.06922565075464, "num_eq": 4621381862321914516, "num_range": 70, "upper_bound": "kcH"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 604743, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 157055, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 613597, "histo_col_type": "", "name": "__auto__", "null_count": 252540, "row_count": 700000}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 449850, "histo_col_type": "", "name": "__auto__", "null_count": 429352, "row_count": 700000}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 513624, "histo_col_type": "", "name": "__auto__", "null_count": 36982, "row_count": 700000}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 402022, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30744, "histo_col_type": "", "name": "__auto__", "null_count": 324887, "row_count": 700000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 136886, "histo_col_type": "", "name": "__auto__", "null_count": 76749, "row_count": 700000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 163606, "histo_col_type": "", "name": "__auto__", "null_count": 444150, "row_count": 700000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34274, "histo_col_type": "", "name": "__auto__", "null_count": 460428, "row_count": 700000}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 607066, "histo_buckets": [{"distinct_range": 0, "num_eq": 1881295185818142844, "num_range": 0, "upper_bound": "571379023"}, {"distinct_range": 5517947237039390000, "num_eq": 10000000000, "num_range": 5517947237039390163, "upper_bound": "588899118"}, {"distinct_range": 517164566306246500, "num_eq": 1705159521765596740, "num_range": 7380226828250081984, "upper_bound": "606089268"}, {"distinct_range": 8000000, "num_eq": 6090412277292793048, "num_range": 8000000, "upper_bound": "3237715084"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 549050, "histo_col_type": "", "name": "__auto__", "null_count": 124766, "row_count": 700000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 266030, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 181163, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 700000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 160989, "histo_buckets": [{"distinct_range": 0, "num_eq": 2435450366863008153, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 389343980995123400, "num_eq": 6205187655087112057, "num_range": 2152389883637377381, "upper_bound": "1188769870"}, {"distinct_range": 0, "num_eq": 0, "num_range": 20000000000, "upper_bound": "1400962437"}, {"distinct_range": 4782548267466028000, "num_eq": 10000000, "num_range": 4782548267466028091, "upper_bound": "1543871154"}, {"distinct_range": 70736806217816010, "num_eq": 0, "num_range": 5214023905636213078, "upper_bound": "1899698790"}, {"distinct_range": 474335585902644100, "num_eq": 40000, "num_range": 4043942782539384481, "upper_bound": "4269574496"}, {"distinct_range": 0, "num_eq": 5107801431075877468, "num_range": 10000000, "upper_bound": "4294333048"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 658003, "row_count": 700000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 872468022778873836, "histo_buckets": [{"distinct_range": 0, "num_eq": 1966195584511315207, "num_range": 0, "upper_bound": ""}, {"distinct_range": 4815339145108044000, "num_eq": 1363924749826115120, "num_range": 4815339145108044209, "upper_bound": "\\u0006v\\u001d"}, {"distinct_range": 700, "num_eq": 6000000, "num_range": 700, "upper_bound": "\'"}, {"distinct_range": 0, "num_eq": 3925721276046262476, "num_range": 2440506595738460006, "upper_bound": "3\\u001d\\u0001)(="}, {"distinct_range": 2611447829.393381, "num_eq": 6375282633033861916, "num_range": 10000000000, "upper_bound": ">D"}, {"distinct_range": 0, "num_eq": 7430726011271457020, "num_range": 234215950594991989, "upper_bound": "E"}, {"distinct_range": 1983875680941752000, "num_eq": 800000, "num_range": 7734958462715336938, "upper_bound": "H("}, {"distinct_range": 8027266122705661000, "num_eq": 30000000, "num_range": 8027266122705661328, "upper_bound": "J%\\u0016@J>~\\u0010"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 1323678561550484512}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 257029671881995510, "histo_col_type": "", "name": "__auto__", "null_count": 783383180789221471, "row_count": 1323678561550484512}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 286846226650547868, "histo_col_type": "", "name": "__auto__", "null_count": 1087244624979969041, "row_count": 1323678561550484512}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 347398917641345222, "histo_col_type": "", "name": "__auto__", "null_count": 607504919381385762, "row_count": 1323678561550484512}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1302723782286317954, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1323678561550484512}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1077653085753666305, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1323678561550484512}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 804710074556017480, "histo_col_type": "", "name": "__auto__", "null_count": 278579946435981407, "row_count": 1323678561550484512}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 751531238569614692, "histo_col_type": "", "name": "__auto__", "null_count": 751817266361118660, "row_count": 1323678561550484512}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 304448975003032246, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1323678561550484512}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 83059344709546157, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1323678561550484512}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 397091310782812777, "histo_col_type": "", "name": "__auto__", "null_count": 24892413009243577, "row_count": 1323678561550484512}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 820593997933422695, "histo_col_type": "", "name": "__auto__", "null_count": 1279603701973992412, "row_count": 1323678561550484512}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1182627181836164043, "histo_col_type": "", "name": "__auto__", "null_count": 549481653734402478, "row_count": 1323678561550484512}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 428122249000195612, "histo_col_type": "", "name": "__auto__", "null_count": 486303518930046368, "row_count": 1323678561550484512}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 201112583813925674, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1323678561550484512}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 948016955261094601, "histo_col_type": "", "name": "__auto__", "null_count": 773962716552022616, "row_count": 1323678561550484512}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 826262456430552930, "histo_col_type": "", "name": "__auto__", "null_count": 1176544853907403414, "row_count": 1323678561550484512}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 709823785733567980, "histo_col_type": "", "name": "__auto__", "null_count": 485889156970972139, "row_count": 1323678561550484512}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 836670731699766424, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1323678561550484512}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 468728709433940270, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1323678561550484512}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 887903098004445525, "histo_col_type": "", "name": "__auto__", "null_count": 230569023965545209, "row_count": 1323678561550484512}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5, "histo_col_type": "", "name": "__auto__", "null_count": 4, "row_count": 10}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 9, "row_count": 10}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5, "histo_col_type": "", "name": "__auto__", "null_count": 4, "row_count": 10}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5, "histo_buckets": [{"distinct_range": 0, "num_eq": 5932628388188399925, "num_range": 0, "upper_bound": "19491"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 8, "row_count": 10}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5, "histo_buckets": [{"distinct_range": 0, "num_eq": 20, "num_range": 0, "upper_bound": "-2113780066"}, {"distinct_range": 0, "num_eq": 30000000000, "num_range": 100, "upper_bound": "-2044825271"}, {"distinct_range": 42347.56566870703, "num_eq": 4115252640736049529, "num_range": 50000, "upper_bound": "-1990249582"}, {"distinct_range": 0, "num_eq": 3992199889491903704, "num_range": 5554098136404371134, "upper_bound": "-1532769302"}, {"distinct_range": 3000000, "num_eq": 2021148089167473169, "num_range": 3000000, "upper_bound": "-1474935780"}, {"distinct_range": 4703.275478724117, "num_eq": 5540744886541192291, "num_range": 5000, "upper_bound": "-1432603466"}, {"distinct_range": 200000, "num_eq": 10000000, "num_range": 200000, "upper_bound": "-1090148609"}, {"distinct_range": 7277345139136295000, "num_eq": 4464728382761925420, "num_range": 7277345139136295259, "upper_bound": "1"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 10}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 10}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000, "num_range": 0, "upper_bound": "BOX(-2.2846407862085845 0.19474031957669347,-0.37682776482752 2.5117600233612474)"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 300000000, "upper_bound": "BOX(-1.6746664837787995 1.0568925048406288,-0.2080320132309085 1.2917486611057685)"}, {"distinct_range": 8000000000, "num_eq": 20, "num_range": 8000000000, "upper_bound": "BOX(-0.2566329151471063 0.8178470023938447,0.5722267281205795 1.2233775508869438)"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 25823384120326362, "upper_bound": "BOX(1.1423731531237546 0.5836297803240313,1.1466489809394076 0.9390560337724602)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 10}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8, "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 10}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "00000000-0000-0000-0000-000000000000"}, {"distinct_range": 7307706540470971000, "num_eq": 40000000, "num_range": 7307706540470971496, "upper_bound": "18512cf6-c4c5-4250-a223-6e710c1f351c"}, {"distinct_range": 5215913.639961826, "num_eq": 20000, "num_range": 20000000, "upper_bound": "852a0c30-89fc-47e1-9d2e-4dc9128e9a1c"}, {"distinct_range": 0, "num_eq": 5558854056798136852, "num_range": 0, "upper_bound": "ac070a06-2cb6-4f56-b88d-09bdef890576"}, {"distinct_range": 290.8731961476783, "num_eq": 0, "num_range": 300, "upper_bound": "b4877642-a10d-4849-9295-1a6cb1298ed3"}, {"distinct_range": 0, "num_eq": 5000000, "num_range": 0, "upper_bound": "e9c48676-697a-4b66-b95d-f106b27685b2"}, {"distinct_range": 0, "num_eq": 4975097636351845490, "num_range": 8804777811395790123, "upper_bound": "ebc79460-0c9b-40db-847e-ef25c93192c7"}, {"distinct_range": 200344690.87579966, "num_eq": 70000, "num_range": 600000000, "upper_bound": "fb8cdb83-5e9b-4b56-b6ab-148b76d1b59d"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000000, "num_range": 0, "upper_bound": "15617840"}, {"distinct_range": 0, "num_eq": 2622348327210885604, "num_range": 5000000000, "upper_bound": "55603561"}, {"distinct_range": 0, "num_eq": 451272090181646398, "num_range": 1000, "upper_bound": "667744454"}, {"distinct_range": 0, "num_eq": 200, "num_range": 7710904950572256703, "upper_bound": "707417430"}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 20000000, "upper_bound": "3587153437"}, {"distinct_range": 173869543.84134007, "num_eq": 5538454736856108353, "num_range": 1000000000, "upper_bound": "4050614957"}, {"distinct_range": 339305382000619400, "num_eq": 2000000000, "num_range": 3363983503310453134, "upper_bound": "4095221558"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 1369683240850839637, "num_range": 0, "upper_bound": "\\r\'\\"\\u000f\\u0018"}, {"distinct_range": 68.73364619508057, "num_eq": 600000, "num_range": 80, "upper_bound": " \\u001d\\u0013&\\\\q]"}, {"distinct_range": 26105775257786116, "num_eq": 2823837873752997562, "num_range": 2246530268041301071, "upper_bound": "$CR\'dr\\u0007"}, {"distinct_range": 5953442203138612000, "num_eq": 8951604878274315010, "num_range": 5953442203138612701, "upper_bound": "\'"}, {"distinct_range": 2287730449886233000, "num_eq": 493587146170651277, "num_range": 8206876941010919958, "upper_bound": "WZX"}, {"distinct_range": 1268360009120247600, "num_eq": 0, "num_range": 1268360009120247509, "upper_bound": "]&/<IB\\"\\t\\\\"}, {"distinct_range": 179372487.00942084, "num_eq": 7100634498329853529, "num_range": 200000000, "upper_bound": "a9.\\u00162=\\u000e"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "1241278531"}, {"distinct_range": 8907992604018941000, "num_eq": 8428029467629366274, "num_range": 8907992604018940672, "upper_bound": "1831791674"}, {"distinct_range": 0, "num_eq": 30000000000, "num_range": 4781423049920232006, "upper_bound": "2812928161"}, {"distinct_range": 318035053789377860, "num_eq": 600, "num_range": 318035053789377868, "upper_bound": "2982143431"}, {"distinct_range": 2105497018831626200, "num_eq": 50000000, "num_range": 2105497018831626348, "upper_bound": "3924008383"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27418, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 744176, "histo_col_type": "", "name": "__auto__", "null_count": 472626, "row_count": 800000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 99037, "histo_col_type": "", "name": "__auto__", "null_count": 20337, "row_count": 800000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 690282, "histo_col_type": "", "name": "__auto__", "null_count": 461445, "row_count": 800000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 108359, "histo_col_type": "", "name": "__auto__", "null_count": 46870, "row_count": 800000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 349914, "histo_col_type": "", "name": "__auto__", "null_count": 355887, "row_count": 800000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 231521, "histo_col_type": "", "name": "__auto__", "null_count": 243551, "row_count": 800000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 350611, "histo_col_type": "", "name": "__auto__", "null_count": 344769, "row_count": 800000}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 689237, "histo_col_type": "", "name": "__auto__", "null_count": 376293, "row_count": 800000}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 69300, "histo_col_type": "", "name": "__auto__", "null_count": 303188, "row_count": 800000}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 558840, "histo_col_type": "", "name": "__auto__", "null_count": 472260, "row_count": 800000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 514712, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000, "num_range": 0, "upper_bound": "\\u0018qT\\u0014"}, {"distinct_range": 12068616471.459253, "num_eq": 7409861834614368316, "num_range": 50000000000, "upper_bound": "5"}, {"distinct_range": 0, "num_eq": 9000000000, "num_range": 5038002525183865602, "upper_bound": "F\\u0017u.G5="}, {"distinct_range": 9773.956387953367, "num_eq": 10000, "num_range": 10000, "upper_bound": "SEvSL"}, {"distinct_range": 39.52746847091306, "num_eq": 7210075054460087553, "num_range": 100, "upper_bound": "{>"}, {"distinct_range": 507.0288794819432, "num_eq": 30000000000, "num_range": 10000, "upper_bound": "\U00002603"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29515, "histo_col_type": "", "name": "__auto__", "null_count": 94544, "row_count": 800000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 379785, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 502826, "histo_col_type": "", "name": "__auto__", "null_count": 680491, "row_count": 800000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 752199, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 365119, "histo_col_type": "", "name": "__auto__", "null_count": 361068, "row_count": 800000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36563, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 126032, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 256599, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 329222, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2886770669, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000000000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2419362460, "histo_buckets": [{"distinct_range": 0, "num_eq": 4335322686129013613, "num_range": 0, "upper_bound": "-10730"}, {"distinct_range": 2791340093643325000, "num_eq": 8173522266466955453, "num_range": 2791340093643324686, "upper_bound": "1"}, {"distinct_range": 8112374419516151000, "num_eq": 5123249980787520558, "num_range": 8343013248614723004, "upper_bound": "7347"}, {"distinct_range": 6894661663.882664, "num_eq": 3827909458329179610, "num_range": 10000000000, "upper_bound": "23132"}, {"distinct_range": 2284502335939307500, "num_eq": 5889826207702530592, "num_range": 2284502335939307562, "upper_bound": "30015"}, {"distinct_range": 0, "num_eq": 7015479025848403723, "num_range": 4000000, "upper_bound": "31631"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 409386635, "row_count": 5000000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 889449164, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": "-1582275995"}, {"distinct_range": 4.517892674659306, "num_eq": 3231978976451520882, "num_range": 70, "upper_bound": "-1264658024"}, {"distinct_range": 511524290.7951955, "num_eq": 9000000, "num_range": 700000000, "upper_bound": "-369330645"}, {"distinct_range": 6989876.698913743, "num_eq": 90, "num_range": 400000000, "upper_bound": "0"}, {"distinct_range": 3848730378067843600, "num_eq": 300000000, "num_range": 4864331594071624162, "upper_bound": "1035692445"}, {"distinct_range": 0, "num_eq": 60000, "num_range": 10, "upper_bound": "1207754670"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 0, "row_count": 5000000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4333019819, "histo_buckets": [{"distinct_range": 0, "num_eq": 2815644812641322167, "num_range": 0, "upper_bound": "BOX(-10 -10,10 10)"}, {"distinct_range": 68934.01758273788, "num_eq": 90, "num_range": 7000000, "upper_bound": "BOX(-1.1027617627555455 -0.5825046025802,-0.7502101940392464 -0.009010387274344389)"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 4744080144226022410, "upper_bound": "BOX(-0.5010530313251625 -0.2900578339808031,-0.2299315115964978 0.8661248653853795)"}, {"distinct_range": 2322551984390779400, "num_eq": 8000, "num_range": 9118391946680557679, "upper_bound": "BOX(-0.23343913268632174 -0.41941819099833444,0.7926274405375598 2.2243137477947283)"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 70, "upper_bound": "BOX(-0.16724035445347216 -0.42974522759144995,0.201550196650498 -0.009211708730002321)"}, {"distinct_range": 50, "num_eq": 5000, "num_range": 50, "upper_bound": "BOX(0.20794066631961228 0.4505853262135555,1.043126007625896 0.5247073060067733)"}, {"distinct_range": 6379229080940591000, "num_eq": 4000000000, "num_range": 7197899241021107095, "upper_bound": "BOX(0.3424108031451659 -0.9954818783367446,1.4190264056139323 1.2756574312847548)"}, {"distinct_range": 0, "num_eq": 2000000000, "num_range": 1000000000, "upper_bound": "BOX(0.5479753230963027 -0.1858023417962869,0.74314778866965 1.0080039206094595)"}, {"distinct_range": 568153376325433800, "num_eq": 700, "num_range": 568153376325433766, "upper_bound": "BOX(0.9997507910863201 -0.041188765395610005,1.509640608092882 2.3511877185465346)"}], "histo_col_type": "BOX2D", "name": "__auto__", "null_count": 0, "row_count": 5000000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1035547830, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3331422419, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000000000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3343997387, "histo_col_type": "", "name": "__auto__", "null_count": 455757341, "row_count": 5000000000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1180243885, "histo_col_type": "", "name": "__auto__", "null_count": 1766027436, "row_count": 5000000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1685711784, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2247001554, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000000, "num_range": 0, "upper_bound": "1976-09-18"}, {"distinct_range": 770841701.1784762, "num_eq": 308721326854641563, "num_range": 1000000000, "upper_bound": "1983-08-27"}, {"distinct_range": 1000000000, "num_eq": 70, "num_range": 1000000000, "upper_bound": "1992-12-25"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 5000000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 660163853, "histo_col_type": "", "name": "__auto__", "null_count": 741463140, "row_count": 5000000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1484341185, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5000000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3195337586, "histo_buckets": [{"distinct_range": 0, "num_eq": 40, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "\\u0001"}, {"distinct_range": 185852941414546720, "num_eq": 100, "num_range": 299292903024414318, "upper_bound": "\\u000c"}, {"distinct_range": 0, "num_eq": 20000000000, "num_range": 300, "upper_bound": "\\""}, {"distinct_range": 723931194966117900, "num_eq": 8000000, "num_range": 7036741686676286919, "upper_bound": "`"}, {"distinct_range": 200000000, "num_eq": 10, "num_range": 200000000, "upper_bound": "u"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 5000000000}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "nan" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61683 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2761750&tab=buildLog) on [master@3bae381e45397225ff926f3f8f60d52b70fdc661](https://github.com/cockroachdb/cockroach/commits/3bae381e45397225ff926f3f8f60d52b70fdc661):

```
        postgres_1    | You can change this by editing pg_hba.conf or using the option -A, or
        postgres_1    | --auth-local and --auth-host, the next time you run initdb.
        postgres_1    | waiting for server to start....2021-03-10 06:13:15.211 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-03-10 06:13:15.231 UTC [49] LOG:  database system was shut down at 2021-03-10 06:13:15 UTC
        postgres_1    | 2021-03-10 06:13:15.236 UTC [48] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2021-03-10 06:13:18.130 UTC [48] LOG:  received fast shutdown request
        postgres_1    | .2021-03-10 06:13:18.132 UTC [48] LOG:  aborting any active transactions
        postgres_1    | 2021-03-10 06:13:18.134 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2021-03-10 06:13:18.141 UTC [50] LOG:  shutting down
        postgres_1    | 2021-03-10 06:13:18.261 UTC [48] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-03-10 06:13:18.342 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-03-10 06:13:18.342 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-03-10 06:13:18.346 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-03-10 06:13:18.362 UTC [85] LOG:  database system was shut down at 2021-03-10 06:13:18 UTC
        postgres_1    | 2021-03-10 06:13:18.368 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (603.15s)
        test_1        |     --- FAIL: TestCompare/postgres (2.24s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61683 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2765661&tab=buildLog) on [master@fc68d927f946103a757888fcbd3ac20904c57a6b](https://github.com/cockroachdb/cockroach/commits/fc68d927f946103a757888fcbd3ac20904c57a6b):

```
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-03-11 06:12:40.552 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-03-11 06:12:40.552 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-03-11 06:12:40.556 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-03-11 06:12:40.571 UTC [85] LOG:  database system was shut down at 2021-03-11 06:12:40 UTC
        postgres_1    | 2021-03-11 06:12:40.577 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (3.73s)
        test_1        |     --- FAIL: TestCompare/postgres (2.59s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.3:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (1.14s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 BIT(10), col1_1 VARCHAR NOT NULL, col1_2 INET NOT NULL, col1_3 BIT(36) NOT NULL, col1_4 REGTYPE NOT NULL, col1_5 REGPROCEDURE NOT NULL, col1_6 REGCLASS NOT NULL, col1_7 OID NOT NULL, col1_8 REGPROC NOT NULL, col1_9 TIME NOT NULL, col1_10 INTERVAL NOT NULL, PRIMARY KEY (col1_8, col1_4, col1_9 DESC, col1_10, col1_2 DESC, col1_1, col1_7 DESC, col1_5 ASC, col1_3 ASC, col1_6 ASC), INDEX (col1_4 ASC, col1_7 ASC, col1_3, col1_6 ASC, col1_8 ASC, col1_1 DESC, col1_0, col1_9 DESC, col1_10 ASC, col1_5 ASC) WHERE (table1.col1_1 != 'X':::STRING) AND (table1.col1_9 != '00:00:00':::TIME), INDEX (col1_1 ASC, col1_4 ASC, col1_3 DESC, col1_6 DESC) WHERE (table1.col1_9 = '24:00:00':::TIME) AND (table1.col1_1 > '"':::STRING), UNIQUE (col1_5 DESC, col1_9, col1_1, col1_7 ASC, col1_10 ASC) WHERE (table1.col1_9 <= '00:00:00') AND (table1.col1_1 > ''), INDEX (col1_10 DESC, col1_5 DESC, col1_7 DESC, col1_8, col1_6 ASC, col1_9 DESC, col1_4 ASC, col1_1 DESC, col1_2 ASC, col1_0 ASC), UNIQUE (col1_6 DESC, col1_10 ASC, col1_2 DESC, col1_8 DESC) STORING (col1_0) WHERE (table1.col1_9 < '24:00:00':::TIME) OR (table1.col1_1 < '"':::STRING), UNIQUE (col1_5 DESC, col1_9 ASC, col1_8 DESC, col1_7, col1_1 ASC, col1_2, col1_3, col1_6, col1_4 DESC, col1_10, col1_0 DESC) WHERE (table1.col1_9 = '00:00:00') AND (table1.col1_1 != ''), UNIQUE (col1_10, col1_9 DESC, col1_1, col1_8 ASC, col1_5 ASC, col1_0 ASC, col1_3 DESC, col1_2, col1_6 DESC, col1_7 ASC) WHERE (table1.col1_1 >= 'X':::STRING) AND (table1.col1_9 = '24:00:00':::TIME), INDEX (col1_9 ASC) STORING (col1_0) WHERE table1.col1_1 <= '':::STRING, FAMILY (col1_1), FAMILY (col1_2, col1_9, col1_10, col1_8), FAMILY (col1_4, col1_0, col1_7), FAMILY (col1_3), FAMILY (col1_5, col1_6));
        test_1        |             CREATE TABLE table2 (col1_8 REGPROC NOT NULL, col1_4 REGTYPE NOT NULL, col1_9 TIME NOT NULL, col1_10 INTERVAL NOT NULL, col1_2 INET NOT NULL, col1_1 VARCHAR NOT NULL, col1_7 OID NOT NULL, col1_5 REGPROCEDURE NOT NULL, col1_3 BIT(36) NOT NULL, col1_6 REGCLASS NOT NULL, col2_10 INT8 NOT NULL, col2_11 TIMESTAMPTZ NULL, col2_12 BYTES, col2_13 REGPROC NOT NULL, col2_14 FLOAT8 NULL, col2_15 FLOAT8 NOT NULL, col2_16 TIMETZ NOT NULL, col2_17 TIMESTAMP NOT NULL, PRIMARY KEY (col1_8, col1_4, col1_9 DESC, col1_10, col1_2 DESC, col1_1, col1_7 DESC, col1_5 ASC, col1_3 ASC, col1_6 ASC, col2_10 ASC, col2_17, col2_12 DESC), INDEX (col2_16, col1_4 DESC, col1_3, col1_7 ASC, col2_15, col1_10, col2_12 ASC) STORING (col2_11) WHERE ((((((((table2.col2_11 > '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ) AND (table2.col2_16 = '24:00:00-15:59:00':::TIMETZ)) AND (table2.col2_14 != 'NaN':::FLOAT8)) AND (table2.col1_9 > '24:00:00':::TIME)) AND (table2.col2_17 >= '-4713-11-24 00:00:00':::TIMESTAMP)) OR (table2.col2_10 < 32767:::INT8)) AND (table2.col2_15 <= '+Inf':::FLOAT8)) OR (table2.col2_12 != e'\\x':::BYTES)) OR (table2.col1_1 > 'X':::STRING), INDEX (col1_5 DESC, col2_16 DESC, col2_13 DESC, col2_17 DESC, col1_9 ASC) WHERE (((table2.col2_12 <= e'\\xe29883':::BYTES) OR (table2.col2_11 < '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) OR (table2.col2_17 != '-4713-11-24 00:00:00':::TIMESTAMP)) AND (table2.col2_14 = 3.4028234663852886e+38:::FLOAT8), INDEX (col2_17, col1_1, col2_15 DESC, col2_11 ASC, col2_16 ASC, col1_10) WHERE ((((((((table2.col1_1 >= e'\U00002603') OR (table2.col1_9 <= '00:00:00')) AND (table2.col2_10 >= 9223372036854775807)) AND (table2.col2_14 != 1.0)) OR (table2.col2_15 != 1.401298464324817e-45)) OR (table2.col2_12 < '\x22')) AND (table2.col2_17 = '0001-01-01 00:00:00')) AND (table2.col2_16 < '00:00:00+15:59:00')) OR (table2.col2_11 >= '294276-12-31 23:59:59.999999+00:00'), INDEX (col1_2, col2_15 ASC, col2_13 DESC) STORING (col2_11, col2_16) WHERE (table2.col1_9 != '24:00:00') OR (table2.col1_1 >= e'\x00'), UNIQUE (col2_16, col2_10 DESC, col2_11 ASC, col1_9 DESC, col1_5 DESC, col1_3, col1_6, col1_7 DESC, col2_15 ASC, col2_13 ASC, col1_4, col2_17, col1_10 DESC, col1_2 ASC) STORING (col2_14) WHERE ((((((((table2.col2_17 >= '0001-01-01 00:00:00':::TIMESTAMP) AND (table2.col2_12 = e'\\xe29883':::BYTES)) OR (table2.col1_1 >= e'\x00':::STRING)) AND (table2.col2_16 < '00:00:00+15:59:00':::TIMETZ)) OR (table2.col2_15 >= 1.401298464324817e-45:::FLOAT8)) OR (table2.col2_14 > 5e-324:::FLOAT8)) OR (table2.col1_9 = '00:00:00':::TIME)) AND (table2.col2_10 = (-1):::INT8)) OR (table2.col2_11 > '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ), FAMILY (col2_17), FAMILY (col1_6, col2_11), FAMILY (col1_1), FAMILY (col2_14), FAMILY (col2_15), FAMILY (col1_8), FAMILY (col1_2), FAMILY (col1_7), FAMILY (col1_9, col2_13, col1_10, col1_3, col2_10, col1_5, col2_12, col2_16, col1_4)) INTERLEAVE IN PARENT table1 (col1_8, col1_4, col1_9, col1_10, col1_2, col1_1, col1_7, col1_5, col1_3, col1_6);
        test_1        |             CREATE TABLE table3 (col3_0 BOOL NOT NULL, col3_1 INT8 NOT NULL, col3_2 REGCLASS NOT NULL, col3_3 TIME NULL, col3_4 REGPROCEDURE NOT NULL, col3_5 VARCHAR NOT NULL, col3_6 BYTES NOT NULL, col3_7 INT2 NOT NULL, col3_8 BYTES NOT NULL, col3_9 FLOAT4 NOT NULL, col3_10 STRING[] NULL, col3_11 VARCHAR NOT NULL, col3_12 STRING NOT NULL, col3_13 INT4 NOT NULL, col3_14 INTERVAL, PRIMARY KEY (col3_0 ASC, col3_2 ASC, col3_5 ASC, col3_1 DESC, col3_4 ASC, col3_9, col3_11, col3_6 ASC, col3_8 DESC, col3_13, col3_12 DESC), col3_15 STRING NOT NULL AS (lower(CAST(col3_0 AS STRING))) VIRTUAL, col3_16 STRING NOT NULL AS (lower(col3_12)) VIRTUAL, col3_17 INT8 AS ((col3_1 + col3_13) + col3_7) STORED, col3_18 INT2 NOT NULL AS (col3_7 + 2307:::INT8) VIRTUAL, FAMILY (col3_17), FAMILY (col3_13), FAMILY (col3_0, col3_2, col3_1, col3_7), FAMILY (col3_9), FAMILY (col3_3), FAMILY (col3_4, col3_6, col3_8, col3_5, col3_11, col3_12, col3_10, col3_14));
        test_1        |             CREATE TABLE table4 (col1_8 REGPROC NOT NULL, col1_4 REGTYPE NOT NULL, col1_9 TIME NOT NULL, col1_10 INTERVAL NOT NULL, col1_2 INET NOT NULL, col1_1 VARCHAR NOT NULL, col1_7 OID NOT NULL, col1_5 REGPROCEDURE NOT NULL, col1_3 BIT(36) NOT NULL, col1_6 REGCLASS NOT NULL, col2_10 INT8 NOT NULL, col2_17 TIMESTAMP NOT NULL, col2_12 BYTES, col4_13 INT4 NOT NULL, col4_14 INET NULL, col4_15 REGTYPE NULL, col4_16 DATE NOT NULL, col4_17 BYTES, col4_18 INET NOT NULL, col4_19 REGCLASS NOT NULL, col4_20 TIMESTAMPTZ, col4_21 DECIMAL NULL, col4_22 TIMETZ, col4_23 INET, col4_24 INT8, col4_25 INT2 NULL, PRIMARY KEY (col1_8, col1_4, col1_9 DESC, col1_10, col1_2 DESC, col1_1, col1_7 DESC, col1_5 ASC, col1_3 ASC, col1_6 ASC, col2_10 ASC, col2_17, col2_12 DESC), UNIQUE (col1_4 ASC, col1_3, col4_15 ASC, col2_17, col4_22, col1_8, col1_2, col4_25, col4_17 DESC, col4_23 DESC, col4_21 DESC, col1_1 DESC, col4_20 ASC, col2_10 ASC, col4_14 ASC, col4_18, col4_24, col1_10 DESC, col1_6 DESC, col2_12 ASC) STORING (col4_13, col4_16, col4_19), FAMILY (col4_17), FAMILY (col4_23, col4_16), FAMILY (col2_17, col2_12, col4_22, col1_3, col1_1), FAMILY (col4_24), FAMILY (col4_21, col1_9, col1_10), FAMILY (col4_13), FAMILY (col1_8, col2_10, col1_6, col1_2), FAMILY (col4_20, col4_25), FAMILY (col4_14, col4_15, col1_7, col1_5), FAMILY (col4_18, col4_19), FAMILY (col1_4)) INTERLEAVE IN PARENT table2 (col1_8, col1_4, col1_9, col1_10, col1_2, col1_1, col1_7, col1_5, col1_3, col1_6, col2_10, col2_17, col2_12);
        test_1        |             CREATE TABLE table5 (col5_0 UUID, col5_1 CHAR NOT NULL, col5_2 FLOAT4, PRIMARY KEY (col5_1), INDEX (col5_2, col5_1, col5_0 DESC) WHERE (table5.col5_2 = '+Inf':::FLOAT8) AND (table5.col5_1 = '':::STRING), UNIQUE (col5_0 ASC) STORING (col5_2) WHERE (table5.col5_2 <= -1.0) OR (table5.col5_1 >= 'X'), INDEX (col5_1, col5_0 ASC) STORING (col5_2) WHERE (table5.col5_2 = '+Inf':::FLOAT8) AND (table5.col5_1 > '"':::STRING), UNIQUE (col5_0 ASC, col5_1 ASC, col5_2) WHERE (table5.col5_2 > 3.4028234663852886e+38) AND (table5.col5_1 = e'\''), INDEX (col5_0 ASC, col5_1 DESC) STORING (col5_2), INDEX (col5_1 DESC, col5_0 ASC) STORING (col5_2) WHERE (table5.col5_2 > -Inf) AND (table5.col5_1 = '"'), FAMILY (col5_0, col5_2, col5_1));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2382259808, "histo_buckets": [{"distinct_range": 0, "num_eq": 6024713936427345650, "num_range": 0, "upper_bound": "-44 years -10 mons -272 days -03:16:52.022711"}, {"distinct_range": 0, "num_eq": 2007159243788448099, "num_range": 2012824153765138647, "upper_bound": "-30 years -9 mons -291 days -09:47:14.011836"}, {"distinct_range": 4128136994000071700, "num_eq": 400, "num_range": 4128136994000071798, "upper_bound": "-20 years -4 mons -385 days -14:54:24.083562"}, {"distinct_range": 700000, "num_eq": 9000000, "num_range": 700000, "upper_bound": "-3 years -5 mons -143 days -07:37:41.220858"}, {"distinct_range": 6000, "num_eq": 70000000000, "num_range": 6000, "upper_bound": "1 mon 1 day"}, {"distinct_range": 10373226464861322, "num_eq": 50000000000, "num_range": 7978328399708637518, "upper_bound": "6 years 6 mons 901 days 05:04:05.883046"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3407963330, "histo_col_type": "", "name": "__auto__", "null_count": 45799853739, "row_count": 50000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30774719109, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000, "num_range": 0, "upper_bound": "\\u0003?)"}, {"distinct_range": 4366897861445163500, "num_eq": 1000000000, "num_range": 4366897861445163554, "upper_bound": "\\rW\\u0017"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45752332549, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 47912305426, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24123271254, "histo_buckets": [{"distinct_range": 0, "num_eq": 9049739216733018019, "num_range": 0, "upper_bound": "1586873222"}, {"distinct_range": 347520196.81674224, "num_eq": 2385824740047558761, "num_range": 800000000, "upper_bound": "2060096046"}, {"distinct_range": 10.008859619727966, "num_eq": 5340048738847363823, "num_range": 30, "upper_bound": "3490553437"}, {"distinct_range": 0, "num_eq": 4290488945209787457, "num_range": 500, "upper_bound": "4217983977"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36747245839, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000, "num_range": 0, "upper_bound": "653700127"}, {"distinct_range": 300000000, "num_eq": 7165282289936305521, "num_range": 300000000, "upper_bound": "3709361188"}, {"distinct_range": 5000, "num_eq": 0, "num_range": 5000, "upper_bound": "4235777558"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5018161504, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 37757083998, "histo_buckets": [{"distinct_range": 0, "num_eq": 6594054288236700233, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 4330.40021381119, "num_eq": 951136940783917308, "num_range": 10000, "upper_bound": "1849351417"}, {"distinct_range": 3584565222639841300, "num_eq": 1000000000, "num_range": 6576640361624972185, "upper_bound": "1929720098"}, {"distinct_range": 80000, "num_eq": 50000, "num_range": 80000, "upper_bound": "4169728611"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 47820022499, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": "2629001799"}, {"distinct_range": 0, "num_eq": 300000, "num_range": 4675249760316608984, "upper_bound": "2678941177"}, {"distinct_range": 3798299506351765500, "num_eq": 30, "num_range": 3798299506351765349, "upper_bound": "3502470716"}, {"distinct_range": 26140434490.695175, "num_eq": 400000, "num_range": 50000000000, "upper_bound": "3574455736"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 50000000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28032526358, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "03:18:34.018402"}, {"distinct_range": 656477608.9426129, "num_eq": 1238201285334034949, "num_range": 700000000, "upper_bound": "06:29:30.4987"}, {"distinct_range": 6173358559371512000, "num_eq": 0, "num_range": 6173358559371511995, "upper_bound": "11:37:57.207288"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 30, "upper_bound": "18:28:09.114586"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 90000, "upper_bound": "21:54:48.915345"}, {"distinct_range": 2079660192432617200, "num_eq": 2853825792052315765, "num_range": 4366216996246192010, "upper_bound": "23:47:13.865717"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 50000000000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34564, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35184, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33162, "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24361, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7342, "histo_col_type": "", "name": "__auto__", "null_count": 36710, "row_count": 40000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3620, "histo_col_type": "", "name": "__auto__", "null_count": 15393, "row_count": 40000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27974, "histo_buckets": [{"distinct_range": 0, "num_eq": 7190371640659206860, "num_range": 0, "upper_bound": "275086191"}, {"distinct_range": 695282875071514500, "num_eq": 4900610039073389570, "num_range": 695282875071514501, "upper_bound": "495186042"}, {"distinct_range": 7045811511803270000, "num_eq": 10000000000, "num_range": 7045811511803270124, "upper_bound": "598696940"}, {"distinct_range": 0, "num_eq": 3000000, "num_range": 0, "upper_bound": "709876273"}, {"distinct_range": 960285838369850500, "num_eq": 40, "num_range": 2232637783527432163, "upper_bound": "2051825733"}, {"distinct_range": 3712630028132858400, "num_eq": 8713740195729432262, "num_range": 5474311614954975823, "upper_bound": "2079207911"}, {"distinct_range": 229601226412153380, "num_eq": 3000000, "num_range": 1554040463409280244, "upper_bound": "3412812682"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13997, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13151, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12755, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27646, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15521, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000, "num_range": 0, "upper_bound": "0001-01-01 00:00:00"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 742424259924910200, "upper_bound": "1973-05-12 10:12:22.000881"}, {"distinct_range": 57169.08852886013, "num_eq": 5000, "num_range": 1000000, "upper_bound": "1975-12-07 03:39:04.000261"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 2337254773530585491, "upper_bound": "1980-06-24 05:48:59.000412"}, {"distinct_range": 3256694462549120000, "num_eq": 152895896327259539, "num_range": 5789146239253494950, "upper_bound": "1998-11-27 01:25:52.000117"}, {"distinct_range": 60, "num_eq": 734281973345389810, "num_range": 60, "upper_bound": "2003-07-16 23:09:58.000518"}, {"distinct_range": 700000, "num_eq": 7000000, "num_range": 700000, "upper_bound": "2004-03-24 07:23:49.000798"}, {"distinct_range": 1000, "num_eq": 50, "num_range": 1000, "upper_bound": "2032-06-26 03:21:41.000908"}], "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26343, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27279, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "183.208.224.178/9"}, {"distinct_range": 10, "num_eq": 8545838550594673888, "num_range": 10, "upper_bound": "147.36.74.239/26"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 6768600382657173564, "upper_bound": "f1dc:1ea9:a9b4:2ea8:8095:7399:8552:f3ec/44"}, {"distinct_range": 0, "num_eq": 1685939142994645949, "num_range": 7106840213731173536, "upper_bound": "449f:d976:432c:7e92:8c4a:8bfc:63c0:c9f1/118"}, {"distinct_range": 0, "num_eq": 0, "num_range": 2921680413173807048, "upper_bound": "2f3c:aa9d:b6d:3729:32c2:cdb9:8f29:efef"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16094, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20024, "histo_col_type": "", "name": "__auto__", "null_count": 15444, "row_count": 40000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28475, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3280, "histo_buckets": [{"distinct_range": 0, "num_eq": 7460378915500203208, "num_range": 0, "upper_bound": "07:09:04.414471+10:49:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 40000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7474, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4774, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 661, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3701, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5074, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 625, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7160, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5955, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4028, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 589, "histo_col_type": "", "name": "__auto__", "null_count": 7624, "row_count": 8000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2331, "histo_col_type": "", "name": "__auto__", "null_count": 3969, "row_count": 8000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3225, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6965, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7458, "histo_col_type": "", "name": "__auto__", "null_count": 7541, "row_count": 8000}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2762, "histo_col_type": "", "name": "__auto__", "null_count": 4345, "row_count": 8000}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3610, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4630, "histo_buckets": [{"distinct_range": 0, "num_eq": 900, "num_range": 0, "upper_bound": "false"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3955, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3259, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8000}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col4_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9489526002265310, "histo_col_type": "", "name": "__auto__", "null_count": 1612552839332481245, "row_count": 6637094878715822968}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1091797739881850570, "histo_buckets": [{"distinct_range": 0, "num_eq": 5336442050335627327, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 80000000, "num_eq": 481577969003913790, "num_range": 80000000, "upper_bound": "49584715"}, {"distinct_range": 2024024.2823845968, "num_eq": 100, "num_range": 4000000, "upper_bound": "791183174"}, {"distinct_range": 5324675499808080000, "num_eq": 40000000, "num_range": 5324675499808079893, "upper_bound": "897236321"}, {"distinct_range": 4205883785817064000, "num_eq": 7750768435259732379, "num_range": 4205883785817063747, "upper_bound": "1199620600"}, {"distinct_range": 0, "num_eq": 0, "num_range": 4091686845794401085, "upper_bound": "1216026534"}, {"distinct_range": 887.214769014523, "num_eq": 7000000000, "num_range": 1000, "upper_bound": "1485865735"}, {"distinct_range": 12101.810175157985, "num_eq": 3509432784206430911, "num_range": 60000, "upper_bound": "3462101326"}, {"distinct_range": 0, "num_eq": 1168906922932462557, "num_range": 0, "upper_bound": "4269969499"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 6637094878715822968}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 903358688536407394, "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 6637094878715822968}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2525977918269512249, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6637094878715822968}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1933197455061253282, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6637094878715822968}, {"columns": ["col4_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1349218718572646949, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6637094878715822968}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4074358449739622850, "histo_col_type": "", "name": "__auto__", "null_count": 1013855661190172448, "row_count": 6637094878715822968}, {"columns": ["col4_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4031790167826584330, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6637094878715822968}, {"columns": ["col4_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2272420269028688133, "histo_col_type": "", "name": "__auto__", "null_count": 6122098777418668927, "row_count": 6637094878715822968}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2872796771563766081, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6637094878715822968}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3510068528585193287, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6637094878715822968}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1965028459436351012, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6637094878715822968}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2906672691229379546, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6637094878715822968}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4767406776789147308, "histo_col_type": "", "name": "__auto__", "null_count": 1926343979697836505, "row_count": 6637094878715822968}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 190179201516155936, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6637094878715822968}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3759154425492419457, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6637094878715822968}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4188278354629387931, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6637094878715822968}, {"columns": ["col4_25"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4492945496148283970, "histo_col_type": "", "name": "__auto__", "null_count": 1002879769180149428, "row_count": 6637094878715822968}, {"columns": ["col4_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5250815612737891339, "histo_col_type": "", "name": "__auto__", "null_count": 5813320745053525915, "row_count": 6637094878715822968}, {"columns": ["col4_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6383959935747041163, "histo_col_type": "", "name": "__auto__", "null_count": 2880131121032991979, "row_count": 6637094878715822968}, {"columns": ["col4_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1705479414338068451, "histo_col_type": "", "name": "__auto__", "null_count": 3970359972325913461, "row_count": 6637094878715822968}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5247067852642622221, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6637094878715822968}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1823073801548751773, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6637094878715822968}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5459576152823025539, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6637094878715822968}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1658695796599692672, "histo_col_type": "", "name": "__auto__", "null_count": 3513881889669208062, "row_count": 6637094878715822968}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1321463779821722139, "histo_col_type": "", "name": "__auto__", "null_count": 1653170426775474897, "row_count": 6637094878715822968}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col5_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1780207658021891716, "histo_buckets": [{"distinct_range": 0, "num_eq": 5477881459889265937, "num_range": 0, "upper_bound": "<"}, {"distinct_range": 690021795.8068715, "num_eq": 100000000, "num_range": 5000000000, "upper_bound": "U"}, {"distinct_range": 0, "num_eq": 5417914871275801266, "num_range": 278035053990471177, "upper_bound": "a"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 7949079916902405658}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5466104041730562914, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000, "num_range": 0, "upper_bound": "-0.8782202005386353"}, {"distinct_range": 10000000000, "num_eq": 100000, "num_range": 10000000000, "upper_bound": "-0.6014907360076904"}, {"distinct_range": 0, "num_eq": 7307575333155758929, "num_range": 2000000, "upper_bound": "0.8163091540336609"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 817290947074585102, "row_count": 7949079916902405658}, {"columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4805711545933021042, "histo_buckets": [{"distinct_range": 0, "num_eq": 2913356440837255660, "num_range": 0, "upper_bound": "3c5c6183-5bc0-4f99-a6a3-ce35f0201c12"}, {"distinct_range": 3661954757850826000, "num_eq": 60000000, "num_range": 4087616114428241574, "upper_bound": "3faaeeaf-6955-45ea-968c-5fd48e389db6"}, {"distinct_range": 709980401124272300, "num_eq": 8230151382333210935, "num_range": 709980401124272282, "upper_bound": "6948c32e-2747-4710-a71a-f034642f54c0"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 927161915329805637, "row_count": 7949079916902405658}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('yopa', 'bwfyt', 'h', 'hazqqp', 'jdwk', 'eykc');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('pjhw', 'ed', 'jucp', 'p', 'gh', 'cgoo');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('yy', 'sdj', 'apdk', 'sdhpp', 'nwgvhk', 'glw');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1487307765429815554, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 600000, "num_eq": 7100481461066847191, "num_range": 600000, "upper_bound": "1583318060"}, {"distinct_range": 33652615889183348, "num_eq": 30000000, "num_range": 1227389382669363326, "upper_bound": "1667124905"}, {"distinct_range": 7000000000, "num_eq": 8000000, "num_range": 7000000000, "upper_bound": "2770614782"}, {"distinct_range": 4000000, "num_eq": 200000000, "num_range": 4000000, "upper_bound": "3166753118"}, {"distinct_range": 1209333643396059100, "num_eq": 10000000000, "num_range": 3278362028296557425, "upper_bound": "3877625622"}, {"distinct_range": 1000, "num_eq": 700000000, "num_range": 1000, "upper_bound": "4073670299"}, {"distinct_range": 5340709556624051000, "num_eq": 8100896716957737014, "num_range": 5340709556624051159, "upper_bound": "4199912727"}, {"distinct_range": 500, "num_eq": 200, "num_range": 500, "upper_bound": "4259266694"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 4400548760609426547}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3421481481688437163, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4400548760609426547}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1751412982797859685, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": "687045537"}, {"distinct_range": 432803543805243650, "num_eq": 4736677819503345608, "num_range": 1027678490949289066, "upper_bound": "718890238"}, {"distinct_range": 690892.514217788, "num_eq": 70000, "num_range": 1000000, "upper_bound": "2382938868"}, {"distinct_range": 2893622788.9445186, "num_eq": 1798678912580739615, "num_range": 6000000000, "upper_bound": "2463102259"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 4400548760609426547}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4208602572826823115, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "08:26:27.841052"}, {"distinct_range": 439021450492579800, "num_eq": 7000000000, "num_range": 1334152234760710661, "upper_bound": "09:15:24.57077"}, {"distinct_range": 3237997820397284400, "num_eq": 7000000000, "num_range": 5130012089909525572, "upper_bound": "10:32:16.696531"}, {"distinct_range": 50787582903.91109, "num_eq": 60000, "num_range": 60000000000, "upper_bound": "10:38:44.995344"}, {"distinct_range": 30, "num_eq": 714980148310551615, "num_range": 30, "upper_bound": "11:40:36.69094"}, {"distinct_range": 30.37406944665162, "num_eq": 2373337128356532946, "num_range": 60, "upper_bound": "13:25:13.787084"}, {"distinct_range": 0, "num_eq": 900000, "num_range": 1217898191837456869, "upper_bound": "20:34:17.9147"}, {"distinct_range": 4734287013064867000, "num_eq": 900, "num_range": 8391442984807340256, "upper_bound": "23:19:45.700378"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 4400548760609426547}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3395289120669640999, "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 4400548760609426547}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 851626039810740522, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4400548760609426547}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4003375704287596944, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4400548760609426547}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 778639507601402286, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 3279086364992649341, "num_range": 4945157504039040888, "upper_bound": "821830784"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 40000000000, "upper_bound": "4079957810"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 4400548760609426547}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 531935069870681462, "histo_buckets": [{"distinct_range": 0, "num_eq": 7271242592688429044, "num_range": 0, "upper_bound": "3702646983"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 4400548760609426547}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3246931962033311268, "histo_buckets": [{"distinct_range": 0, "num_eq": 1497767008618970086, "num_range": 0, "upper_bound": "8 years 5 mons 902 days 09:47:22.550629"}, {"distinct_range": 0, "num_eq": 50000, "num_range": 10000000000, "upper_bound": "28 years 11 mons 13 days 06:35:57.189754"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 4400548760609426547}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3262955815509509341, "histo_col_type": "", "name": "__auto__", "null_count": 2494184595106112387, "row_count": 4400548760609426547}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 671613804802688768, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1106121114361890086}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 473317589228144125, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000000, "num_range": 0, "upper_bound": "696850959"}, {"distinct_range": 2914733010787521500, "num_eq": 2472581714527382603, "num_range": 2914733010787521637, "upper_bound": "1134848006"}, {"distinct_range": 0, "num_eq": 300, "num_range": 864943436132235170, "upper_bound": "2335685293"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 3408234229381111118, "upper_bound": "3494666643"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 1106121114361890086}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 102195572383285958, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1106121114361890086}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 417920401541584729, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1106121114361890086}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 760779030404902330, "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 1106121114361890086}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 504019801119262043, "histo_buckets": [{"distinct_range": 0, "num_eq": 60, "num_range": 0, "upper_bound": "123.64.217.235/19"}, {"distinct_range": 401005051193406200, "num_eq": 5127493986631621417, "num_range": 8173688383739098072, "upper_bound": "178.142.201.212/19"}, {"distinct_range": 2746856049936383500, "num_eq": 2782688127769457065, "num_range": 5291961984250053391, "upper_bound": "40.40.25.140/22"}, {"distinct_range": 1229571293381128000, "num_eq": 937612200946746622, "num_range": 3642816416327453318, "upper_bound": "197.201.160.219/28"}, {"distinct_range": 0, "num_eq": 50000, "num_range": 600000, "upper_bound": "11b2:ba7f:6456:4318:14a3:177c:2a13:ef9c/8"}, {"distinct_range": 78.64408042753494, "num_eq": 300000, "num_range": 900, "upper_bound": "3953:acd6:409d:f729:77fa:8114:b4f1:92eb/92"}, {"distinct_range": 40000, "num_eq": 400000, "num_range": 40000, "upper_bound": "1406:da4b:a57d:ad54:3169:9b6a:5c6e:6721/101"}, {"distinct_range": 0, "num_eq": 50000, "num_range": 1000000, "upper_bound": "bb79:701:7266:3ed7:518c:5173:ac52:9bd8/101"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 0, "row_count": 1106121114361890086}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35098446864689059, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1106121114361890086}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 777790049098375149, "histo_col_type": "", "name": "__auto__", "null_count": 155989205216475439, "row_count": 1106121114361890086}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 644242856834278953, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "00:36:43.988892-09:25:00"}, {"distinct_range": 1529893058551964700, "num_eq": 8898312994199100553, "num_range": 1576177163630135155, "upper_bound": "17:55:03.677841+00:50:00"}, {"distinct_range": 40000, "num_eq": 60, "num_range": 40000, "upper_bound": "23:55:58.582727+06:05:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 1106121114361890086}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 287835628139183543, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1106121114361890086}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2896818571019301, "histo_col_type": "", "name": "__auto__", "null_count": 917013603086408973, "row_count": 1106121114361890086}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 950761787523237008, "histo_col_type": "", "name": "__auto__", "null_count": 28118056444332449, "row_count": 1106121114361890086}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 878734152999568642, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 5557777741849368000, "num_eq": 200, "num_range": 5557777741849367970, "upper_bound": "75397183"}, {"distinct_range": 400000000, "num_eq": 900000000, "num_range": 400000000, "upper_bound": "1052708564"}, {"distinct_range": 100, "num_eq": 30, "num_range": 100, "upper_bound": "1291877703"}, {"distinct_range": 6532445600595436000, "num_eq": 7403504550104868464, "num_range": 6532445600595435278, "upper_bound": "1869854084"}, {"distinct_range": 1906873.6670691478, "num_eq": 3000000, "num_range": 100000000, "upper_bound": "3017086283"}, {"distinct_range": 600000000, "num_eq": 900000000, "num_range": 600000000, "upper_bound": "3635203404"}, {"distinct_range": 0, "num_eq": 5000000, "num_range": 8000000, "upper_bound": "4012933332"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 1106121114361890086}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 518975145513352988, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1106121114361890086}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 913940424075473152, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1106121114361890086}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 934578828309417115, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1106121114361890086}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1061253637491924155, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1106121114361890086}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 723970790066834005, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1106121114361890086}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 633516, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 867483, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 749731, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 618340, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 834099, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 242381, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 755829, "histo_buckets": [{"distinct_range": 0, "num_eq": 8195997722075370873, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 0, "num_eq": 7330572189330308145, "num_range": 80000000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 645675, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 614075, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 186930, "histo_col_type": "", "name": "__auto__", "null_count": 245383, "row_count": 900000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 147872, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 432273, "histo_col_type": "", "name": "__auto__", "null_count": 318697, "row_count": 900000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 753214, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 47087, "histo_col_type": "", "name": "__auto__", "null_count": 9790, "row_count": 900000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 432796, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 852743, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 283898, "histo_col_type": "", "name": "__auto__", "null_count": 132338, "row_count": 900000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 588638, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 896175, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS '[{"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3251742996786721816, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3618208370813175303}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 447740456657885658, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3618208370813175303}, {"columns": ["col4_23"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2367641957339517678, "histo_col_type": "", "name": "__auto__", "null_count": 2032887717791546937, "row_count": 3618208370813175303}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1449172706658600717, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3618208370813175303}, {"columns": ["col4_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2002119476193250844, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3618208370813175303}, {"columns": ["col4_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 720208070695951708, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3618208370813175303}, {"columns": ["col4_25"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 772229033040463502, "histo_col_type": "", "name": "__auto__", "null_count": 820588731035772115, "row_count": 3618208370813175303}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2146851953721720050, "histo_col_type": "", "name": "__auto__", "null_count": 2065796620431666048, "row_count": 3618208370813175303}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 241990806097366849, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3618208370813175303}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 858697905187808444, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3618208370813175303}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3180825611026898015, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3618208370813175303}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1535380085905978699, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3618208370813175303}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2236807418639578817, "histo_col_type": "", "name": "__auto__", "null_count": 693870170773542259, "row_count": 3618208370813175303}, {"columns": ["col4_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 665320718164552746, "histo_col_type": "", "name": "__auto__", "null_count": 1214043570914071429, "row_count": 3618208370813175303}, {"columns": ["col4_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 374583823352446587, "histo_col_type": "", "name": "__auto__", "null_count": 1689537578765419578, "row_count": 3618208370813175303}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 372162428546150851, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 90000, "num_range": 200000000, "upper_bound": "49922298"}, {"distinct_range": 4000000, "num_eq": 10000000, "num_range": 4000000, "upper_bound": "1252352370"}, {"distinct_range": 600000000, "num_eq": 90000000, "num_range": 600000000, "upper_bound": "1416011015"}, {"distinct_range": 0, "num_eq": 2000000000, "num_range": 5762896344551758581, "upper_bound": "1532991471"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 3618208370813175303}, {"columns": ["col4_22"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2261060529576485057, "histo_col_type": "", "name": "__auto__", "null_count": 65607576849070651, "row_count": 3618208370813175303}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1090551538464049374, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3618208370813175303}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3398592769267257065, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3618208370813175303}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 264880565876148154, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3618208370813175303}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2066368144298720886, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3618208370813175303}, {"columns": ["col4_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2006706908637933102, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3618208370813175303}, {"columns": ["col4_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3183546789484238857, "histo_col_type": "", "name": "__auto__", "null_count": 625479359224430957, "row_count": 3618208370813175303}, {"columns": ["col4_21"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2774808222788855241, "histo_col_type": "", "name": "__auto__", "null_count": 261518120324136430, "row_count": 3618208370813175303}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2127250494491320629, "histo_buckets": [{"distinct_range": 0, "num_eq": 5724506772662779698, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 1582679.6563547782, "num_eq": 4239274689383323127, "num_range": 5000000, "upper_bound": "424510799"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 5960793762958558007, "upper_bound": "1153643885"}, {"distinct_range": 7000000000, "num_eq": 800000000, "num_range": 7000000000, "upper_bound": "1561340717"}, {"distinct_range": 891720101952732700, "num_eq": 6000000000, "num_range": 6361396234111764377, "upper_bound": "2014865120"}, {"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": "2164326493"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 3618208370813175303}, {"columns": ["col4_24"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 86524184899147637, "histo_col_type": "", "name": "__auto__", "null_count": 449501930558771863, "row_count": 3618208370813175303}]';
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 666463961892768360, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "0b0fe460-996d-4dce-8ac7-edd2d769f641"}, {"distinct_range": 1522733.302426114, "num_eq": 2000000, "num_range": 10000000, "upper_bound": "3003efc0-d6fe-4002-af88-4a9bd4aa7591"}, {"distinct_range": 191845618249892320, "num_eq": 700, "num_range": 1208545344889387356, "upper_bound": "4dc9749a-0fbf-478e-8385-7cff2b7246c7"}, {"distinct_range": 22.513283913059794, "num_eq": 20000000000, "num_range": 30, "upper_bound": "a07312da-7929-42c9-88dd-3e031f52f9f2"}, {"distinct_range": 4862922864188277000, "num_eq": 379720277507593358, "num_range": 4862922864188276378, "upper_bound": "d97c18f8-b0b0-4895-8076-3b649adaf169"}, {"distinct_range": 1410024227.0887215, "num_eq": 90, "num_range": 6000000000, "upper_bound": "fd93007c-2579-48ea-ae30-2b50b82ab729"}], "histo_col_type": "UUID", "name": "__auto__", "null_count": 1140124254759865120, "row_count": 1609839681479110291}, {"columns": ["col5_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24449351205429329, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": ","}, {"distinct_range": 5239367423516920000, "num_eq": 90000000, "num_range": 7674292382640496967, "upper_bound": "O"}, {"distinct_range": 3222593220047177000, "num_eq": 8096248791406671554, "num_range": 3222593220047177074, "upper_bound": "f"}, {"distinct_range": 0, "num_eq": 6262473583228157268, "num_range": 0, "upper_bound": "j"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 0, "row_count": 1609839681479110291}, {"columns": ["col5_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1497153148509118705, "histo_buckets": [{"distinct_range": 0, "num_eq": 6609328055888972662, "num_range": 0, "upper_bound": "0.38168764114379883"}, {"distinct_range": 0, "num_eq": 60000000000, "num_range": 5000000000, "upper_bound": "0.7529169917106628"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 2659734531660666921, "upper_bound": "2.1964948177337646"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 579478013841379748, "row_count": 1609839681479110291}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 136586623, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000, "num_range": 0, "upper_bound": "09:44:58.546145"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 104913559, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "6 years 6 mons 592 days 08:30:51.098195"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 288925734, "histo_buckets": [{"distinct_range": 0, "num_eq": 600, "num_range": 0, "upper_bound": ""}, {"distinct_range": 8198415422716360000, "num_eq": 100, "num_range": 8198415422716359882, "upper_bound": "!N "}, {"distinct_range": 3928562626188969000, "num_eq": 700000, "num_range": 3928562626188968767, "upper_bound": "&AW\\r\\u0005*\\u0010"}, {"distinct_range": 206882261490186100, "num_eq": 20, "num_range": 2936977205166490927, "upper_bound": ",4-"}, {"distinct_range": 50000000000, "num_eq": 104687734056143970, "num_range": 50000000000, "upper_bound": ".\\u0019F"}, {"distinct_range": 6000000, "num_eq": 0, "num_range": 6000000, "upper_bound": "g\\u0006>>"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 62696933, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 133880059, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 197115688, "histo_buckets": [{"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "433929116"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 231237737, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000, "num_range": 0

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2769710&tab=buildLog) on [master@4b98115dfda02a9498f566958bd915c45ec7e449](https://github.com/cockroachdb/cockroach/commits/4b98115dfda02a9498f566958bd915c45ec7e449):

```
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT (5.0::FLOAT8::FLOAT8 || 'J'::TEXT::TEXT)::TEXT AS col_77081) AS tab_49099
        test_1        |             ORDER BY
        test_1        |             	col_77081 NULLS FIRST;
        test_1        |             
        test_1        |             postgres: [same as previous]
        test_1        |             
        test_1        |     --- FAIL: TestCompare/mutators (1.08s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 DATE, col1_1 DATE NOT NULL, col1_2 BIT(13), col1_3 REGTYPE NOT NULL, col1_4 STRING NOT NULL, col1_5 VARCHAR[], col1_6 INET NOT NULL, PRIMARY KEY (col1_3 DESC, col1_4 DESC, col1_1 DESC), col1_7 STRING AS (lower(CAST(col1_2 AS STRING))) STORED, INDEX (col1_1, col1_0, col1_7, col1_6 DESC, col1_3, col1_4) WHERE (((table1.col1_1 > '4714-11-24 BC':::DATE) AND (table1.col1_7 < '"':::STRING)) OR (table1.col1_4 = e'\U00002603':::STRING)) AND (table1.col1_0 >= '4714-11-24 BC':::DATE), UNIQUE (col1_1 DESC, col1_7, col1_4 ASC, col1_6, col1_2), FAMILY (col1_4, col1_7, col1_2, col1_0, col1_3), FAMILY (col1_5), FAMILY (col1_1), FAMILY (col1_6), UNIQUE (col1_3, col1_4) STORING (col1_5, col1_6) WHERE table1.col1_7 != e'\'');
        test_1        |             CREATE TABLE table2 (col2_0 REGCLASS NOT NULL, col2_1 INT4 NOT NULL, col2_2 OID NOT NULL, col2_3 REGPROC NOT NULL, col2_4 STRING NOT NULL, col2_5 REGTYPE NOT NULL, col2_6 FLOAT8 NOT NULL, col2_7 BIT(6) NULL, col2_8 FLOAT8 NOT NULL, col2_9 BOX2D NULL, col2_10 TIMESTAMPTZ, PRIMARY KEY (col2_5 DESC, col2_8 ASC, col2_2, col2_4 ASC, col2_1 ASC, col2_6 DESC, col2_3 ASC), col2_11 STRING NULL AS (lower(CAST(col2_7 AS STRING))) STORED, col2_12 STRING NOT NULL AS (lower(CAST(col2_0 AS STRING))) VIRTUAL, col2_13 FLOAT8 AS (col2_8 + col2_6) STORED, col2_14 FLOAT8 NOT NULL AS (col2_8 + col2_6) VIRTUAL, UNIQUE (col2_13, col2_2 DESC, col2_1, col2_0 DESC) STORING (col2_7, col2_9, col2_10) WHERE ((((((((table2.col2_8 > 1.401298464324817e-45) AND (table2.col2_6 != -Inf)) OR (table2.col2_11 < e'\x00')) OR (table2.col2_13 < -Inf)) OR (table2.col2_10 != '-2000-01-01 00:00:00+00:00')) AND (table2.col2_1 >= -1)) OR (table2.col2_4 >= '')) AND (table2.col2_12 = '"')) AND (table2.col2_14 != 1.401298464324817e-45), INDEX (col2_11 ASC, col2_3 DESC, col2_0 ASC, col2_4, col2_1 ASC) WHERE (((((table2.col2_13 >= (-1.0):::FLOAT8) AND (table2.col2_14 >= 'NaN':::FLOAT8)) OR (table2.col2_6 <= (-1.0):::FLOAT8)) OR (table2.col2_10 = '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table2.col2_12 < e'\U00002603':::STRING)) OR (table2.col2_8 >= 0.0:::FLOAT8), INDEX (col2_5, col2_12 DESC, col2_6 ASC, col2_4 DESC, col2_1) WHERE ((((((((table2.col2_10 != '294276-12-31 23:59:59.999999+00:00') OR (table2.col2_1 != 127)) OR (table2.col2_11 <= e'\x00')) OR (table2.col2_14 = 1.0)) OR (table2.col2_4 >= '"')) OR (table2.col2_12 < '')) OR (table2.col2_8 = 1.0)) AND (table2.col2_13 >= NaN)) OR (table2.col2_6 >= 1.401298464324817e-45), FAMILY (col2_7, col2_9, col2_1, col2_6), FAMILY (col2_11, col2_4), FAMILY (col2_13), FAMILY (col2_8), FAMILY (col2_3, col2_5, col2_2), FAMILY (col2_0, col2_10), UNIQUE (col2_7) WHERE ((((((((table2.col2_4 > '"') AND (table2.col2_1 > -2147483648)) AND (table2.col2_6 != +Inf)) AND (table2.col2_8 < 1.7976931348623157e+308)) OR (table2.col2_10 = '-2000-01-01 00:00:00+00:00')) AND (table2.col2_13 != 1.7976931348623157e+308)) AND (table2.col2_12 < '')) AND (table2.col2_11 != e'\x00')) OR (table2.col2_14 != 0.0));
        test_1        |             CREATE TABLE table3 (col3_0 BOX2D NOT NULL, col3_1 TIMESTAMP NOT NULL, col3_2 BOOL NULL, col3_3 TIMESTAMPTZ NOT NULL, col3_4 BOX2D NOT NULL, col3_5 TIMETZ NOT NULL, col3_6 INT2 NOT NULL, col3_7 VARCHAR NOT NULL, col3_8 INTERVAL NOT NULL, col3_9 TIMETZ NOT NULL, col3_10 BIT(23) NOT NULL, col3_11 FLOAT4 NOT NULL, col3_12 REGPROC, col3_13 TIMESTAMP NULL, PRIMARY KEY (col3_5, col3_7, col3_4 ASC, col3_6, col3_1, col3_3 ASC, col3_0 ASC, col3_10 DESC, col3_8 ASC, col3_11 DESC), col3_14 STRING AS (lower(CAST(col3_10 AS STRING))) VIRTUAL, col3_15 INT2 NULL AS (col3_6 + 4120:::INT8) STORED, col3_16 STRING NULL AS (lower(CAST(col3_0 AS STRING))) STORED, col3_17 STRING NOT NULL AS (lower(CAST(col3_8 AS STRING))) VIRTUAL, col3_18 STRING NOT NULL AS (lower(CAST(col3_4 AS STRING))) VIRTUAL, INDEX (col3_17 DESC, col3_14 ASC, col3_18 ASC, col3_13, col3_16 ASC, col3_2 ASC, col3_0 ASC, col3_1, col3_3 DESC) STORING (col3_9, col3_15) WHERE ((((table3.col3_3 < '3000-01-01 00:00:00+00:00':::TIMESTAMPTZ) OR (table3.col3_6 < 0:::INT8)) AND (table3.col3_9 = '24:00:00-15:59:00':::TIMETZ)) OR (table3.col3_14 <= e'\'':::STRING)) AND (table3.col3_11 = '+Inf':::FLOAT8), FAMILY (col3_2, col3_0), FAMILY (col3_5), FAMILY (col3_6, col3_11), FAMILY (col3_4, col3_1), FAMILY (col3_15, col3_12), FAMILY (col3_13, col3_10), FAMILY (col3_16), FAMILY (col3_3, col3_9), FAMILY (col3_7, col3_8), UNIQUE (col3_11));
        test_1        |             CREATE TABLE table4 (col4_0 INT2 NOT NULL, col4_1 INET NOT NULL, col4_2 BOOL, col4_3 VARCHAR NULL, col4_4 REGPROC NOT NULL, col4_5 TIMESTAMPTZ NOT NULL, col4_6 REGTYPE NOT NULL, col4_7 INT8, col4_8 INT2 NULL, col4_9 REGPROCEDURE NOT NULL, col4_10 "char" NOT NULL, PRIMARY KEY (col4_4 ASC, col4_5 ASC, col4_9 ASC, col4_1 ASC, col4_0 DESC, col4_10 DESC, col4_6), col4_11 INT2 AS (col4_8 + col4_7) STORED, col4_12 STRING AS (lower(CAST(col4_1 AS STRING))) VIRTUAL, col4_13 STRING NULL AS (lower(CAST(col4_9 AS STRING))) VIRTUAL, UNIQUE (col4_10 ASC, col4_6 DESC, col4_12 ASC, col4_13 ASC, col4_7, col4_4 ASC, col4_5, col4_3 ASC, col4_8, col4_9 DESC, col4_1, col4_0 ASC) STORING (col4_2, col4_11) WHERE ((((((((table4.col4_2 OR (table4.col4_13 = 'X')) AND (table4.col4_8 >= -1)) OR (table4.col4_0 != 1)) AND (table4.col4_3 = e'\'')) AND (table4.col4_7 != 127)) OR (table4.col4_10 <= e'\'')) AND (table4.col4_5 >= '0001-01-01 00:00:00+00:00')) OR (table4.col4_11 < 0)) AND (table4.col4_12 != 'X'), UNIQUE (col4_3, col4_11 ASC, col4_1, col4_0 ASC, col4_6 ASC, col4_13, col4_2 ASC, col4_7 DESC, col4_12 DESC, col4_10 ASC, col4_4, col4_9 DESC, col4_5 DESC, col4_8 DESC) WHERE (((((((((table4.col4_8 < 0:::INT8) OR (table4.col4_0 = (-32768):::INT8)) OR (table4.col4_12 > e'\x00':::STRING)) AND (table4.col4_3 > e'\U00002603':::STRING)) AND (table4.col4_13 > '':::STRING)) AND (table4.col4_11 >= (-1):::INT8)) AND (table4.col4_10 < 'X':::STRING)) AND (table4.col4_7 <= 2147483647:::INT8)) AND (NOT table4.col4_2)) OR (table4.col4_5 < '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ), INDEX (col4_3 DESC, col4_0 ASC, col4_7, col4_10 ASC, col4_5 ASC, col4_2 ASC, col4_12, col4_6 ASC, col4_4, col4_13 DESC, col4_9, col4_1 ASC) WHERE ((((((((table4.col4_12 > '"') AND (table4.col4_13 >= '"')) AND (table4.col4_3 >= e'\U00002603')) OR (table4.col4_7 <= 9223372036854775807)) OR (table4.col4_10 > '"')) OR (table4.col4_11 < -1)) AND (NOT table4.col4_2)) OR (table4.col4_0 != -128)) OR (table4.col4_8 > 127), UNIQUE (col4_10, col4_7 DESC, col4_1, col4_8, col4_11 DESC) STORING (col4_2, col4_3) WHERE (table4.col4_5 = '294276-12-31 23:59:59.999999+00:00':::TIMESTAMPTZ) AND (table4.col4_3 >= e'\'':::STRING), INDEX (col4_11, col4_1 ASC, col4_0, col4_8 DESC, col4_9 ASC, col4_7, col4_6, col4_10 ASC), INDEX (col4_6) STORING (col4_3, col4_7, col4_8, col4_11) WHERE ((((((table4.col4_10 >= e'\U00002603') AND (table4.col4_11 <= -1)) AND table4.col4_2) OR (table4.col4_3 > '')) OR (table4.col4_0 < -32768)) AND (table4.col4_5 <= '3000-01-01 00:00:00+00:00')) OR (table4.col4_8 = 32767), INDEX (col4_3, col4_12 ASC, col4_8, col4_9, col4_13 ASC) STORING (col4_7) WHERE (((table4.col4_12 < e'\'') AND (table4.col4_0 < 1)) OR (table4.col4_7 > 127)) OR (table4.col4_13 >= e'\''), UNIQUE (col4_12 DESC) STORING (col4_2, col4_7, col4_11), UNIQUE (col4_10 DESC, col4_3 DESC, col4_9 ASC, col4_5 DESC, col4_2 ASC, col4_13 ASC, col4_0, col4_1 ASC, col4_12 DESC, col4_8 ASC, col4_6 DESC, col4_11, col4_4, col4_7) WHERE (((table4.col4_5 < '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ) OR (table4.col4_3 = e'\'':::STRING)) OR (table4.col4_12 >= e'\'':::STRING)) AND (table4.col4_10 = '"':::STRING), FAMILY (col4_9, col4_8, col4_1, col4_4, col4_5, col4_0, col4_6), FAMILY (col4_11), FAMILY (col4_2), FAMILY (col4_10), FAMILY (col4_7, col4_3));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4192, "histo_buckets": [{"distinct_range": 0, "num_eq": 4131252925157609782, "num_range": 0, "upper_bound": "1970-01-01"}, {"distinct_range": 0, "num_eq": 4270837879713382444, "num_range": 0, "upper_bound": "1980-07-02"}, {"distinct_range": 5189353454843032000, "num_eq": 8613873404811537143, "num_range": 5189353454843031096, "upper_bound": "1982-08-22"}, {"distinct_range": 499521361029699000, "num_eq": 1830586615824704153, "num_range": 4674908922767749724, "upper_bound": "1983-11-15"}, {"distinct_range": 0, "num_eq": 0, "num_range": 7411653773882919440, "upper_bound": "1987-05-23"}, {"distinct_range": 54.08678158298842, "num_eq": 6108277789360194607, "num_range": 80, "upper_bound": "1993-04-28"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 7000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2448, "histo_col_type": "", "name": "__auto__", "null_count": 2477, "row_count": 7000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5707, "histo_buckets": [{"distinct_range": 0, "num_eq": 4203899087758496765, "num_range": 0, "upper_bound": "2389805374"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 7000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5631, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5413, "histo_col_type": "", "name": "__auto__", "null_count": 4562, "row_count": 7000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 311, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2222, "histo_col_type": "", "name": "__auto__", "null_count": 1939, "row_count": 7000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1542, "histo_col_type": "", "name": "__auto__", "null_count": 3085, "row_count": 7000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 5629236026622743383, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 0, "num_range": 7198318901177719801, "upper_bound": "\\u0018KAn"}, {"distinct_range": 71166.95834610666, "num_eq": 10000000, "num_range": 100000, "upper_bound": "9<-\\u0016\\u0014NKW?"}, {"distinct_range": 0, "num_eq": 80000000, "num_range": 70000000, "upper_bound": "`\\u000e"}, {"distinct_range": 0, "num_eq": 100, "num_range": 70000000, "upper_bound": "l"}, {"distinct_range": 1219215069896596200, "num_eq": 4088288633431730524, "num_range": 1219215069896596303, "upper_bound": "z"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000, "num_range": 0, "upper_bound": "-1.160431603072452"}, {"distinct_range": 4000000, "num_eq": 60000000, "num_range": 4000000, "upper_bound": "-1.0"}, {"distinct_range": 0, "num_eq": 20, "num_range": 861799295091912765, "upper_bound": "-0.31907853159847865"}, {"distinct_range": 0, "num_eq": 0, "num_range": 8957152867299492132, "upper_bound": "-0.15731455031754282"}, {"distinct_range": 0, "num_eq": 200000, "num_range": 90000, "upper_bound": "0.32813934479805607"}, {"distinct_range": 1195987705267999000, "num_eq": 200, "num_range": 4250169919172063750, "upper_bound": "0.5493343308262645"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "1102133838"}, {"distinct_range": 0, "num_eq": 4066956724770897583, "num_range": 8955893599736723705, "upper_bound": "2529675313"}, {"distinct_range": 5743255353116006000, "num_eq": 7759993462373400093, "num_range": 5743255353116006198, "upper_bound": "2575477843"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 4310160556936938462, "upper_bound": "4071205584"}, {"distinct_range": 8281137516975501000, "num_eq": 5685079655474746298, "num_range": 8281137516975501241, "upper_bound": "4076023233"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 432284558100544715, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 525141955323975200}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 105022238337302281, "histo_col_type": "", "name": "__auto__", "null_count": 458750140906915533, "row_count": 525141955323975200}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 306251982730082793, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 525141955323975200}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 373763729129763370, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 525141955323975200}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 510829409176716912, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 525141955323975200}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 447930018620907049, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 525141955323975200}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 487285397385947279, "histo_col_type": "", "name": "__auto__", "null_count": 30128751974373437, "row_count": 525141955323975200}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38918090077015321, "histo_col_type": "", "name": "__auto__", "null_count": 334697253749874915, "row_count": 525141955323975200}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 57724636942694026, "histo_buckets": [{"distinct_range": 0, "num_eq": 3184729424011738438, "num_range": 0, "upper_bound": "b"}, {"distinct_range": 9.948744505965335, "num_eq": 6197064372166037975, "num_range": 10, "upper_bound": "hO"}, {"distinct_range": 2176.5204946254894, "num_eq": 10000, "num_range": 4000, "upper_bound": "v^!\\r\\u0014Sv"}, {"distinct_range": 30560058689885308, "num_eq": 60000000000, "num_range": 2903208730193807418, "upper_bound": "}\\u0001"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 525141955323975200}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 497551308755033208, "histo_col_type": "", "name": "__auto__", "null_count": 33538882516530872, "row_count": 525141955323975200}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 339268805222716715, "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 525141955323975200}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 80950588769207595, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 525141955323975200}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 281665458496447762, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 525141955323975200}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 758437889262638, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 525141955323975200}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 424383066907733579, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 525141955323975200}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 205634205683027295, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 525141955323975200}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 94841949652500365, "histo_col_type": "", "name": "__auto__", "null_count": 76259985700193554, "row_count": 525141955323975200}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 364600273680608612, "histo_col_type": "", "name": "__auto__", "null_count": 436454090315762632, "row_count": 525141955323975200}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 405834442139649269, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 525141955323975200}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 87964864, "histo_buckets": [{"distinct_range": 0, "num_eq": 300000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 19943.16607667769, "num_eq": 5559829511611418390, "num_range": 20000, "upper_bound": "\\n"}, {"distinct_range": 7234736832904143000, "num_eq": 300, "num_range": 7234736832904142464, "upper_bound": "\\u001dsy!\\u0018EcxA"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 916745224383613400, "num_eq": 348383591292267516, "num_range": 2874760615908228763, "upper_bound": "[\\\\%gu\\u0019o"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 56964297, "row_count": 100000000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9958599, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45570194, "histo_buckets": [{"distinct_range": 0, "num_eq": 6701948608707059239, "num_range": 0, "upper_bound": "\\u0016"}, {"distinct_range": 0, "num_eq": 300000000, "num_range": 0, "upper_bound": ","}, {"distinct_range": 6913618965969808000, "num_eq": 2316760824993917018, "num_range": 6913618965969808420, "upper_bound": "N"}, {"distinct_range": 0, "num_eq": 5010896556297851723, "num_range": 8107926322398999818, "upper_bound": "f"}, {"distinct_range": 2055603014579214300, "num_eq": 60000, "num_range": 2397676954507609141, "upper_bound": "g"}, {"distinct_range": 6557754915088157000, "num_eq": 4072300334295570939, "num_range": 6557754915088156429, "upper_bound": "k"}, {"distinct_range": 2188680.291423652, "num_eq": 2215738630516128144, "num_range": 3000000, "upper_bound": "x"}, {"distinct_range": 2373391686836549600, "num_eq": 8465752745524105256, "num_range": 2373391686836549770, "upper_bound": "y"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 88635251, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "-22706"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 200, "upper_bound": "-13303"}, {"distinct_range": 774103664552305300, "num_eq": 300000, "num_range": 1153122072461979134, "upper_bound": "20595"}, {"distinct_range": 7000, "num_eq": 1000000, "num_range": 7000, "upper_bound": "26341"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 50793747, "row_count": 100000000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12281298, "histo_col_type": "", "name": "__auto__", "null_count": 1811892, "row_count": 100000000}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10168829, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30805346, "histo_buckets": [{"distinct_range": 0, "num_eq": 413117099627621946, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 3809024830001037300, "num_eq": 1436619767321979219, "num_range": 5873596284355824572, "upper_bound": "399880118"}, {"distinct_range": 0, "num_eq": 60, "num_range": 500000000, "upper_bound": "1249833862"}, {"distinct_range": 969.1550485422466, "num_eq": 841739357572616842, "num_range": 2000, "upper_bound": "1268029521"}, {"distinct_range": 2716379457959671000, "num_eq": 20000, "num_range": 4007071648092649027, "upper_bound": "3006177416"}, {"distinct_range": 9000000, "num_eq": 50000000, "num_range": 9000000, "upper_bound": "3111931698"}, {"distinct_range": 1870045041690692900, "num_eq": 1488248780256552490, "num_range": 9079192956358621004, "upper_bound": "4151179440"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19642098, "histo_col_type": "", "name": "__auto__", "null_count": 98731758, "row_count": 100000000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 82316527, "histo_col_type": "", "name": "__auto__", "null_count": 18069989, "row_count": 100000000}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 89425240, "histo_col_type": "", "name": "__auto__", "null_count": 61981654, "row_count": 100000000}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34315623, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12902465, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 100000, "upper_bound": "\\u0002\\u0002"}, {"distinct_range": 0, "num_eq": 1787599775631665788, "num_range": 7000, "upper_bound": "Hk;5`t$}"}, {"distinct_range": 2514355047504638000, "num_eq": 1000000, "num_range": 5150810769456246179, "upper_bound": "N9\\u0002>KC\\n"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 57389550, "row_count": 100000000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 91743190, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000000, "num_range": 0, "upper_bound": "43583720"}, {"distinct_range": 400000, "num_eq": 200000000, "num_range": 400000, "upper_bound": "1048884595"}, {"distinct_range": 1471037347589882600, "num_eq": 0, "num_range": 1471037347589882716, "upper_bound": "3983364627"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53539545, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}]':::JSONB;
        test_1        |             ALTER TABLE table4 ADD FOREIGN KEY (col4_9, col4_10) REFERENCES table1 (col1_3, col1_4) ON DELETE CASCADE ON UPDATE CASCADE;
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_2) REFERENCES table2 (col2_7) ON DELETE RESTRICT;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_8) REFERENCES table3 (col3_11);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('hn', 'o', 'ket', 'lwgek', 'ntuhy');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('meubbe');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('ild', 'owynsp', 'u', 'cog');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 64755330817, "histo_buckets": [{"distinct_range": 0, "num_eq": 4208399908954927830, "num_range": 0, "upper_bound": "4714-11-24 BC"}, {"distinct_range": 0, "num_eq": 50, "num_range": 500000, "upper_bound": "1983-04-05"}, {"distinct_range": 4941647452951362000, "num_eq": 6773454521891133812, "num_range": 8098885661347988027, "upper_bound": "1985-04-06"}, {"distinct_range": 0, "num_eq": 5848305695767567354, "num_range": 400, "upper_bound": "1993-04-29"}, {"distinct_range": 307159889743210430, "num_eq": 70000, "num_range": 397574568998647338, "upper_bound": "5874897-12-31"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 70000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39602694243, "histo_col_type": "", "name": "__auto__", "null_count": 45681048339, "row_count": 70000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44098682378, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "713079178"}, {"distinct_range": 0, "num_eq": 900, "num_range": 0, "upper_bound": "1009388128"}, {"distinct_range": 6000000000, "num_eq": 6408498772114974081, "num_range": 6000000000, "upper_bound": "2514997170"}, {"distinct_range": 0, "num_eq": 7649261465957433656, "num_range": 30000000000, "upper_bound": "2615892661"}, {"distinct_range": 4931.218488336773, "num_eq": 90000000000, "num_range": 8000, "upper_bound": "3555491659"}, {"distinct_range": 178781923747222340, "num_eq": 7985865003041223805, "num_range": 3124271025404112442, "upper_bound": "4177956706"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 70000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 55329694706, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 67691434409, "histo_col_type": "", "name": "__auto__", "null_count": 31398130527, "row_count": 70000000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44598137073, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 70000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 41510808091, "histo_col_type": "", "name": "__auto__", "null_count": 42858903486, "row_count": 70000000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 61459483739, "histo_col_type": "", "name": "__auto__", "null_count": 55452807059, "row_count": 70000000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 5985867505998650125, "num_range": 800000, "upper_bound": "135382447"}, {"distinct_range": 1000000000, "num_eq": 3000000, "num_range": 1000000000, "upper_bound": "1416598776"}, {"distinct_range": 0, "num_eq": 8128753612556157092, "num_range": 100, "upper_bound": "2061791785"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 1797259664797363932, "num_range": 0, "upper_bound": "F9p\\u001d\\u0006"}, {"distinct_range": 1437867675091528700, "num_eq": 162312723311439843, "num_range": 3581929995719381904, "upper_bound": "MM"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 6244757020352349968, "num_range": 0, "upper_bound": "0.6770646155907134"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 5460523764913306447, "num_range": 0, "upper_bound": "000010"}, {"distinct_range": 3796235798205716500, "num_eq": 5898732227857270780, "num_range": 5398149842249194557, "upper_bound": "001011"}, {"distinct_range": 0.0019238024809576152, "num_eq": 8389643901159333081, "num_range": 30, "upper_bound": "011111"}, {"distinct_range": 8321370831.682529, "num_eq": 8863497773491572871, "num_range": 30000000000, "upper_bound": "100100"}], "histo_col_type": "BIT(6)", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2482, "histo_col_type": "", "name": "__auto__", "null_count": 726, "row_count": 4000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2634, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 737, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2686, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2458, "histo_buckets": [{"distinct_range": 0, "num_eq": 5306330038132976631, "num_range": 0, "upper_bound": "-Inf"}, {"distinct_range": 0, "num_eq": 800000, "num_range": 0, "upper_bound": "0.34913063049316406"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3683, "histo_col_type": "", "name": "__auto__", "null_count": 800, "row_count": 4000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3108, "histo_col_type": "", "name": "__auto__", "null_count": 3835, "row_count": 4000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2691, "histo_col_type": "", "name": "__auto__", "null_count": 3658, "row_count": 4000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 54, "histo_col_type": "", "name": "__auto__", "null_count": 1217, "row_count": 4000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 494, "histo_col_type": "", "name": "__auto__", "null_count": 1163, "row_count": 4000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1168, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2485, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3277, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": "6*\\u0013"}, {"distinct_range": 0, "num_eq": 80, "num_range": 100, "upper_bound": "B,(P\\u0017\\u0014\\u000e*1"}, {"distinct_range": 0, "num_eq": 3665797631201280071, "num_range": 0, "upper_bound": "BL"}, {"distinct_range": 928857619435671000, "num_eq": 1232040236919796591, "num_range": 1497116256190592470, "upper_bound": "E"}, {"distinct_range": 4111009.5944868964, "num_eq": 9000000000, "num_range": 5000000, "upper_bound": "MU4<\\u00044"}, {"distinct_range": 32979.23825925997, "num_eq": 1804537705064709214, "num_range": 100000, "upper_bound": "X"}, {"distinct_range": 4000000000, "num_eq": 2626035318088669662, "num_range": 4000000000, "upper_bound": "w[\\u000c\\u001d"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 118, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "01:32:37.791484+11:56:00"}, {"distinct_range": 0, "num_eq": 9168336347812820140, "num_range": 50000000, "upper_bound": "05:46:33.217784+05:52:00"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 30000000, "upper_bound": "01:53:37.851135-04:28:00"}, {"distinct_range": 0, "num_eq": 3204783177570313194, "num_range": 80, "upper_bound": "23:20:45.800491+13:19:00"}, {"distinct_range": 0, "num_eq": 200000000, "num_range": 3205769940193636614, "upper_bound": "08:22:44.30156-02:21:00"}, {"distinct_range": 3889912962728484000, "num_eq": 40000000000, "num_range": 4863118214259044608, "upper_bound": "16:19:56.553307+00:02:00"}, {"distinct_range": 4504255188066634000, "num_eq": 632057011234022470, "num_range": 4504255188066634114, "upper_bound": "11:12:19.061173-13:36:00"}, {"distinct_range": 6000, "num_eq": 1335698612175752715, "num_range": 6000, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 426, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 882, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 544, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 339, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3306, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 104699480, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 104401827, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 192480461, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 127749443, "histo_col_type": "", "name": "__auto__", "null_count": 25341881, "row_count": 300000000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 186171058, "histo_col_type": "", "name": "__auto__", "null_count": 126219806, "row_count": 300000000}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 223344926, "histo_col_type": "", "name": "__auto__", "null_count": 78823442, "row_count": 300000000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 209790726, "histo_buckets": [{"distinct_range": 0, "num_eq": 6279327373497123224, "num_range": 0, "upper_bound": "\\u001d"}, {"distinct_range": 60000000, "num_eq": 473545721948114692, "num_range": 60000000, "upper_bound": ".uTl\\u0012"}, {"distinct_range": 60000000, "num_eq": 3046719920255426294, "num_range": 60000000, "upper_bound": "Ks"}, {"distinct_range": 2837101542276730400, "num_eq": 10000000000, "num_range": 2837101542276730328, "upper_bound": "\x7fw\\u0017\\u001c(n!"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 249328607, "row_count": 300000000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 126705458, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "475060841"}, {"distinct_range": 0, "num_eq": 9094200638236924951, "num_range": 0, "upper_bound": "3275087941"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 292622923, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000, "num_range": 0, "upper_bound": "273181983"}, {"distinct_range": 0, "num_eq": 3000, "num_range": 900000000, "upper_bound": "2334278167"}, {"distinct_range": 881930625.9636194, "num_eq": 300, "num_range": 900000000, "upper_bound": "2523415190"}, {"distinct_range": 3514.313514321009, "num_eq": 2610169132160276423, "num_range": 6000, "upper_bound": "3785936650"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28394726, "histo_col_type": "", "name": "__auto__", "null_count": 237264063, "row_count": 300000000}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 207739792, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": ""}, {"distinct_range": 196.42291724521112, "num_eq": 0, "num_range": 2000, "upper_bound": "\\n\\u0001|!939y"}, {"distinct_range": 0, "num_eq": 3681249735887213503, "num_range": 10000, "upper_bound": "a"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 219485266, "row_count": 300000000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 136518131, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46488, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "\\u0007"}, {"distinct_range": 6560325716.279718, "num_eq": 20000000000, "num_range": 8000000000, "upper_bound": "\\u0019"}, {"distinct_range": 0, "num_eq": 0, "num_range": 2000000, "upper_bound": "!"}, {"distinct_range": 0, "num_eq": 7178509103082075941, "num_range": 0, "upper_bound": "@"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 838250554660477298, "upper_bound": "M"}, {"distinct_range": 8377.48627835703, "num_eq": 80000, "num_range": 50000, "upper_bound": "T"}, {"distinct_range": 0, "num_eq": 7000, "num_range": 90, "upper_bound": "\\\\"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 4000, "upper_bound": "m"}, {"distinct_range": 2616.888532642831, "num_eq": 800000, "num_range": 50000, "upper_bound": "z"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 300000000}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 69238553, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "-26853"}, {"distinct_range": 6053534643415354000, "num_eq": 8000000, "num_range": 6053534643415354617, "upper_bound": "-23294"}, {"distinct_range": 469128036.31267256, "num_eq": 8940913584919515294, "num_range": 4000000000, "upper_bound": "2763"}, {"distinct_range": 0, "num_eq": 8930661053580184250, "num_range": 70000, "upper_bound": "9210"}, {"distinct_range": 200000000, "num_eq": 4561568445368699367, "num_range": 200000000, "upper_bound": "9637"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 94105988, "row_count": 300000000}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3324624470075418032, "histo_col_type": "", "name": "__auto__", "null_count": 1733097738461302417, "row_count": 6848276953201412194}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4924486162363580723, "histo_buckets": [{"distinct_range": 0, "num_eq": 6753631424574212189, "num_range": 0, "upper_bound": "19210723"}, {"distinct_range": 1379599067067576000, "num_eq": 40, "num_range": 2149823601869609820, "upper_bound": "639406999"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "917083692"}, {"distinct_range": 800000, "num_eq": 80000, "num_range": 800000, "upper_bound": "1229959402"}, {"distinct_range": 566987.2793921302, "num_eq": 419578901655907944, "num_range": 900000, "upper_bound": "1458083885"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 0, "upper_bound": "1950719300"}, {"distinct_range": 3340.1179402711214, "num_eq": 6258892449980433310, "num_range": 6000, "upper_bound": "3596464552"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 6848276953201412194}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4811337992112032623, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6848276953201412194}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1300259426227342918, "histo_col_type": "", "name": "__auto__", "null_count": 4532219108902904860, "row_count": 6848276953201412194}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3098994642264416176, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6848276953201412194}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5917462091661390815, "histo_col_type": "", "name": "__auto__", "null_count": 2366956130013303433, "row_count": 6848276953201412194}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4873811353337110804, "histo_col_type": "", "name": "__auto__", "null_count": 382083101369951215, "row_count": 6848276953201412194}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3004418702513968116, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000, "num_range": 0, "upper_bound": "4714-11-24 BC"}, {"distinct_range": 2470229629553004500, "num_eq": 4520951505751754949, "num_range": 7977857641990348855, "upper_bound": "1975-09-05"}, {"distinct_range": 1000000, "num_eq": 90, "num_range": 1000000, "upper_bound": "1977-09-07"}, {"distinct_range": 8599293070485541000, "num_eq": 0, "num_range": 8599293070485540497, "upper_bound": "1979-05-22"}, {"distinct_range": 110760347.59233949, "num_eq": 200, "num_range": 200000000, "upper_bound": "1981-08-23"}, {"distinct_range": 7362063755455572000, "num_eq": 1000000, "num_range": 7362063755455571988, "upper_bound": "1989-03-19"}, {"distinct_range": 1789311507780569.8, "num_eq": 20000, "num_range": 7375284756577479368, "upper_bound": "1993-02-07"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 6848276953201412194}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1891944983711839174, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9104320548372190160}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6617935830343602220, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9104320548372190160}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3422724574038144788, "histo_col_type": "", "name": "__auto__", "null_count": 5083854644253721253, "row_count": 9104320548372190160}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2534046557217876651, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9104320548372190160}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2110655674952343666, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9104320548372190160}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6333448067419878119, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9104320548372190160}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3113282581469924096, "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 7175348350355103736, "row_count": 9104320548372190160}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3846456693559326210, "histo_buckets": [{"distinct_range": 0, "num_eq": 3820894805406049338, "num_range": 0, "upper_bound": "880598735"}, {"distinct_range": 0, "num_eq": 2438694971571164086, "num_range": 0, "upper_bound": "2135973191"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 0, "upper_bound": "2445076914"}, {"distinct_range": 842617237419044400, "num_eq": 9000, "num_range": 848352556829563060, "upper_bound": "2492008806"}, {"distinct_range": 0, "num_eq": 2000000000, "num_range": 90000, "upper_bound": "3005242799"}, {"distinct_range": 200, "num_eq": 0, "num_range": 200, "upper_bound": "3198934221"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 9104320548372190160}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8078106230713594814, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000, "num_range": 0, "upper_bound": "000111"}], "histo_col_type": "BIT(6)", "name": "__auto__", "null_count": 5321490514965037014, "row_count": 9104320548372190160}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 568991430840065150, "histo_col_type": "", "name": "__auto__", "null_count": 736093865930534044, "row_count": 9104320548372190160}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3773743923106845735, "histo_buckets": [{"distinct_range": 0, "num_eq": 4414370802771061385, "num_range": 0, "upper_bound": ""}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 1608765025398308631, "row_count": 9104320548372190160}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8192957724450644880, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9104320548372190160}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 110848100644404085, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9104320548372190160}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4069725942000609516, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9104320548372190160}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4062210843768270598, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 9104320548372190160}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1912, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1921, "histo_col_type": "", "name": "__auto__", "null_count": 2405, "row_count": 10000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3545, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6836, "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3683, "histo_col_type": "", "name": "__auto__", "null_count": 7220, "row_count": 10000}, {"columns": ["col3_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9305, "histo_col_type": "", "name": "__auto__", "null_count": 151, "row_count": 10000}, {"columns": ["col3_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8234, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6786, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9705, "histo_col_type": "", "name": "__auto__", "null_count": 3946, "row_count": 10000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9636, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6146, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9900, "histo_buckets": [{"distinct_range": 0, "num_eq": 9195781248288591606, "num_range": 0, "upper_bound": "X"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1095, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9919, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 737, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 480, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000, "num_range": 0, "upper_bound": "-2.327049493789673"}, {"distinct_range": 0, "num_eq": 359452281337434062, "num_range": 700000000, "upper_bound": "-0.34401383996009827"}, {"distinct_range": 0, "num_eq": 300, "num_range": 6709661816229123521, "upper_bound": "0.46478497982025146"}, {"distinct_range": 30000000000, "num_eq": 10000000, "num_range": 30000000000, "upper_bound": "0.5096845626831055"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 10000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9331, "histo_col_type": "", "name": "__auto__", "null_count": 5726, "row_count": 10000}, {"columns": ["col3_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7160, "histo_col_type": "", "name": "__auto__", "null_count": 8520, "row_count": 10000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 777, "histo_buckets": [{"distinct_range": 0, "num_eq": 90, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 1246464220239121367, "num_range": 0, "upper_bound": "403530861"}, {"distinct_range": 6634585034363296000, "num_eq": 90000000000, "num_range": 6634585034363295547, "upper_bound": "1138177082"}, {"distinct_range": 453278420664890400, "num_eq": 1000000000, "num_range": 453278420664890391, "upper_bound": "2178090379"}, {"distinct_range": 0, "num_eq": 4630129860908006551, "num_range": 4000000, "upper_bound": "3587516679"}, {"distinct_range": 739142814.6796861, "num_eq": 1813516131065822408, "num_range": 1000000000, "upper_bound": "4278687459"}], "histo_col_type": "REGPROC", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 595, "histo_buckets": [{"distinct_range": 0, "num_eq": 1860241226806138984, "num_range": 0, "upper_bound": "1748282073"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 322, "histo_col_type": "", "name": "__auto__", "null_count": 222, "row_count": 900}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 757, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000000000, "num_range": 0, "upper_bound": "-32691"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 6414438199392270786, "upper_bound": "-9768"}, {"distinct_range": 20, "num_eq": 6000000, "num_range": 20, "upper_bound": "26277"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 886, "row_count": 900}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 86, "histo_col_type": "", "name": "__auto__", "null_count": 99, "row_count": 900}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 165, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 845, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000, "num_range": 0, "upper_bound": "#"}, {"distinct_range": 276380821539846460, "num_eq": 3000000, "num_range": 924773959143693885, "upper_bound": "\'"}, {"distinct_range": 0, "num_eq": 30000000, "num_range": 100000, "upper_bound": ">"}, {"distinct_range": 2822039847813387300, "num_eq": 9000000000, "num_range": 5666663202821467693, "upper_bound": "M"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 437, "histo_buckets": [{"distinct_range": 0, "num_eq": 6464383634128599897, "num_range": 0, "upper_bound": ""}, {"distinct_range": 368723208897867260, "num_eq": 1134081454653718846, "num_range": 368723208897867249, "upper_bound": "\\u0003-YOIZ+"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 90000000, "upper_bound": "\\u0015];_*"}, {"distinct_range": 886508095685316200, "num_eq": 8787738005991391446, "num_range": 1790732049785481611, "upper_bound": "\\u0017POz,\\ra\\u0001W"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 6622938024270137059, "upper_bound": "xc"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 749, "row_count": 900}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 709, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 624, "histo_buckets": [{"distinct_range": 0, "num_eq": 1091695221310724477, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 50000000, "num_range": 4245229551824301859, "upper_bound": "a?"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 794, "row_count": 900}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 442, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 470, "histo_col_type": "", "name"

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2771391&tab=buildLog) on [master@4b98115dfda02a9498f566958bd915c45ec7e449](https://github.com/cockroachdb/cockroach/commits/4b98115dfda02a9498f566958bd915c45ec7e449):

```
        test_1        |             CREATE INDEX ON table1 (col1_0 DESC, col1_2, col1_1 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_2 DESC, col1_3 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_3 DESC, col1_0, col1_2);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_3 DESC, col1_2 ASC, col1_0);
        test_1        |             CREATE INDEX ON table1 (col1_1 ASC);
        test_1        |             CREATE INDEX ON table1 (col1_3 ASC, col1_0, col1_2, col1_1);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_1 DESC, col1_0 DESC, col1_3 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_3 ASC, col1_0 ASC, col1_2);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_2 DESC, col1_0, col1_3 DESC);
        test_1        |             CREATE TABLE table2 (col1_1 CHAR NOT NULL, col1_2 INT8 NOT NULL, col1_3 BOX2D NOT NULL, col1_0 REGTYPE NOT NULL, col2_4 FLOAT8 NULL, col2_5 REGPROC NULL, col2_6 "char" NULL, PRIMARY KEY (col1_1, col1_2, col1_3, col1_0));
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_5 ASC, col1_0 ASC, col2_6);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_3, col1_2);
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_3, col1_2) REFERENCES table2 (col1_3, col1_2);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('wbv', 'fcrfo', 'lrfdj', 'zwbtgd', 'rnbpu', 'wyjswt');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('tcwbjf', 'qtf');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('ea', 'tyc', 'v', 'io', 'xewli');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('rvo', 'oike', 'yvf', 'r', 'mooasl', 'j');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('ajycm', 'cnau');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: type "box2d" does not exist (SQLSTATE 42704)
        test_1        |     --- FAIL: TestCompare/mutators (0.91s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 BIT(13) NOT NULL, col1_1 REGTYPE NOT NULL, col1_2 REGCLASS NOT NULL, col1_3 TIMETZ NOT NULL, PRIMARY KEY (col1_3 DESC, col1_0 ASC, col1_2, col1_1 ASC), col1_4 STRING NULL AS (lower(CAST(col1_0 AS STRING))) STORED, col1_5 STRING AS (lower(CAST(col1_1 AS STRING))) STORED, col1_6 STRING AS (lower(CAST(col1_1 AS STRING))) VIRTUAL, UNIQUE (col1_1 ASC, col1_3 DESC, col1_4 DESC, col1_6, col1_2 ASC) STORING (col1_5) WHERE (((table1.col1_6 != e'\x00':::STRING) AND (table1.col1_4 > e'\x00':::STRING)) AND (table1.col1_5 != e'\U00002603':::STRING)) AND (table1.col1_3 > '24:00:00-15:59:00':::TIMETZ), UNIQUE (col1_6, col1_5, col1_4, col1_3 DESC, col1_2 DESC, col1_0 ASC, col1_1) WHERE (((table1.col1_4 = e'\'':::STRING) OR (table1.col1_6 != e'\U00002603':::STRING)) AND (table1.col1_5 <= '"':::STRING)) AND (table1.col1_3 > '00:00:00+15:59:00':::TIMETZ), UNIQUE (col1_1, col1_3, col1_0 DESC, col1_2 DESC, col1_4 DESC, col1_5 ASC) WHERE (((table1.col1_6 = '"':::STRING) OR (table1.col1_5 >= '"':::STRING)) AND (table1.col1_3 >= '24:00:00-15:59:00':::TIMETZ)) AND (table1.col1_4 <= '"':::STRING), UNIQUE (col1_4 ASC, col1_2, col1_1 ASC, col1_0 DESC, col1_6 DESC, col1_3 DESC, col1_5 DESC) WHERE (((table1.col1_4 != '"') OR (table1.col1_6 < '"')) AND (table1.col1_5 >= e'\U00002603')) AND (table1.col1_3 >= '00:00:00+15:59:00'), UNIQUE (col1_4, col1_5 ASC, col1_0 DESC, col1_2 DESC, col1_3, col1_1) WHERE (((table1.col1_6 <= 'X') OR (table1.col1_5 <= 'X')) OR (table1.col1_3 < '00:00:00+15:59:00')) AND (table1.col1_4 != e'\x00'), FAMILY (col1_2), FAMILY (col1_0), FAMILY (col1_1, col1_3, col1_5, col1_4));
        test_1        |             CREATE TABLE table2 (col1_3 TIMETZ NOT NULL, col1_0 BIT(13) NOT NULL, col1_2 REGCLASS NOT NULL, col1_1 REGTYPE NOT NULL, col2_4 BIT(42), col2_5 BIT(21), col2_6 DATE, col2_7 NAME, col2_8 BYTES NOT NULL, col2_9 BIT(24) NOT NULL, col2_10 NAME NOT NULL, col2_11 INET NULL, col2_12 TIMETZ, col2_13 FLOAT8, col2_14 INT2, col2_15 DECIMAL NOT NULL, col2_16 FLOAT8 NULL, col2_17 TIMETZ NOT NULL, col2_18 BIT NOT NULL, col2_19 UUID NULL, col2_20 REGPROCEDURE NOT NULL, PRIMARY KEY (col1_3 DESC, col1_0 ASC, col1_2, col1_1 ASC, col2_6 DESC, col2_17 DESC, col2_7 ASC, col2_16, col2_18 ASC, col2_10 DESC, col2_20 ASC, col2_4, col2_12 DESC, col2_14, col2_5, col2_13, col2_15 ASC, col2_19 DESC, col2_11, col2_9), INDEX (col2_5 ASC, col2_15 DESC, col2_7, col1_3 DESC) STORING (col2_8), UNIQUE (col2_14, col2_7, col2_16 ASC, col1_0 ASC, col2_15 DESC, col1_2 ASC, col2_17, col2_13, col2_11 DESC, col1_3 DESC, col2_4, col2_12 ASC, col2_18 ASC, col2_9, col2_19 DESC, col2_8 ASC, col2_6 DESC, col2_5, col2_10 ASC, col2_20) WHERE ((((((((((table2.col2_16 < 3.4028234663852886e+38:::FLOAT8) OR (table2.col2_17 < '00:00:00+15:59:00':::TIMETZ)) OR (table2.col2_13 = 1.401298464324817e-45:::FLOAT8)) OR (table2.col2_6 > '5874897-12-31':::DATE)) AND (table2.col2_8 < e'\\xe29883':::BYTES)) AND (table2.col1_3 < '00:00:00+15:59:00':::TIMETZ)) AND (table2.col2_10 > '"':::STRING)) OR (table2.col2_7 = '"':::STRING)) AND (table2.col2_14 = 32767:::INT8)) AND (table2.col2_12 <= '24:00:00-15:59:00':::TIMETZ)) OR (table2.col2_15 <= 'NaN':::DECIMAL), INDEX (col2_8, col2_14, col1_1 ASC, col2_4 DESC, col2_5, col2_13 ASC, col1_3 DESC, col2_12 DESC, col1_0 DESC, col2_7 DESC, col2_16 ASC, col2_17, col2_15 DESC, col2_19 DESC, col2_9 DESC, col2_20 ASC, col2_10 DESC), INDEX (col2_11, col2_18 DESC, col2_17, col2_20 ASC, col2_4, col2_5 DESC, col2_12 DESC, col2_13 DESC, col2_15 DESC, col2_14, col2_6 DESC, col1_2 ASC) STORING (col2_8), INDEX (col2_16 ASC, col2_5 DESC, col2_4 DESC, col1_1 ASC, col2_14 ASC, col2_8 DESC, col2_17 ASC, col1_3, col2_9 DESC, col2_19 DESC, col2_10, col1_2 ASC, col2_13 ASC) WHERE ((((((((((table2.col2_14 = -1) OR (table2.col2_17 > '24:00:00-15:59:00')) OR (table2.col2_16 < 1.401298464324817e-45)) AND (table2.col2_8 = '\x00')) OR (table2.col2_15 != -1.234E+401)) AND (table2.col2_10 != '"')) AND (table2.col2_6 >= '-infinity')) AND (table2.col2_12 > '24:00:00-15:59:00')) OR (table2.col2_7 > '"')) OR (table2.col2_13 != 1.0)) OR (table2.col1_3 >= '00:00:00+15:59:00'), INDEX (col2_13 DESC, col2_19 DESC, col2_15 DESC, col2_17 ASC, col1_2 ASC, col2_14, col2_10 ASC, col2_8 ASC, col1_1 ASC, col2_12 ASC, col2_18 DESC, col2_16, col2_5 ASC, col2_20, col2_4 ASC, col2_7 DESC) WHERE (((table2.col2_15 < -Infinity) OR (table2.col2_16 < +Inf)) OR (table2.col2_8 <= '\x27')) OR (table2.col2_7 < e'\U00002603'), UNIQUE (col2_11, col2_10 DESC, col2_15 DESC, col1_3 DESC, col2_17 DESC, col2_20 DESC, col2_5, col2_16), UNIQUE (col1_2, col1_1 ASC, col2_10 ASC, col2_7 ASC, col2_4, col1_3 ASC, col2_13 ASC, col2_14 ASC, col1_0, col2_9 ASC), INDEX (col2_15 ASC, col1_3 DESC, col2_11 ASC, col1_2 DESC, col2_12, col2_9 DESC, col1_0 DESC, col1_1, col2_18 ASC, col2_5 ASC, col2_6), FAMILY (col2_10), FAMILY (col2_8, col1_3), FAMILY (col2_16), FAMILY (col2_12), FAMILY (col2_18, col2_11), FAMILY (col2_7), FAMILY (col1_1), FAMILY (col2_19, col1_0, col2_14, col2_20), FAMILY (col2_4), FAMILY (col2_15, col2_5, col2_13, col2_9, col1_2, col2_17, col2_6)) INTERLEAVE IN PARENT table1 (col1_3, col1_0, col1_2, col1_1);
        test_1        |             CREATE TABLE table3 (col3_0 INT2 NOT NULL, col3_1 REGPROCEDURE NOT NULL, col3_2 BIT(25) NOT NULL, col3_3 INET NOT NULL, col3_4 "char" NOT NULL, col3_5 FLOAT8 NOT NULL, col3_6 DECIMAL NOT NULL, col3_7 REGPROC NULL, PRIMARY KEY (col3_4, col3_3 ASC, col3_6 DESC, col3_1 ASC, col3_2 ASC, col3_0 ASC, col3_5 ASC), col3_8 STRING NOT NULL AS (lower(CAST(col3_7 AS STRING))) VIRTUAL, col3_9 STRING AS (lower(col3_4)) VIRTUAL, col3_10 INT2 NOT NULL AS (col3_0 + (-22022):::INT8) VIRTUAL, col3_11 STRING AS (lower(CAST(col3_7 AS STRING))) STORED, col3_12 FLOAT8 NOT NULL AS (col3_5 + (-0.7644362350987426):::FLOAT8) STORED, col3_13 INT2 AS (col3_0 + 1764:::INT8) STORED, col3_14 FLOAT8 AS (col3_5 + 1.4048502847182405:::FLOAT8) STORED, UNIQUE (col3_0, col3_12, col3_8 DESC, col3_13 ASC, col3_7 ASC, col3_4 DESC, col3_6 DESC, col3_5, col3_9, col3_2 ASC, col3_3, col3_10 ASC, col3_11 ASC, col3_14, col3_1) WHERE ((((table3.col3_11 > e'\'':::STRING) OR (table3.col3_6 = (-1):::DECIMAL)) OR (table3.col3_4 < e'\U00002603':::STRING)) OR (table3.col3_12 < 1.401298464324817e-45:::FLOAT8)) AND (table3.col3_14 = 0.0:::FLOAT8), INDEX (col3_12 ASC, col3_10, col3_0 ASC, col3_5 ASC) STORING (col3_11, col3_13), INDEX (col3_1 ASC, col3_5, col3_13) STORING (col3_7, col3_11) WHERE ((((((table3.col3_4 != '"':::STRING) OR (table3.col3_12 < '-Inf':::FLOAT8)) OR (table3.col3_9 < e'\U00002603':::STRING)) OR (table3.col3_11 < '':::STRING)) AND (table3.col3_0 > (-1):::INT8)) AND (table3.col3_13 > 1:::INT8)) OR (table3.col3_5 != 'NaN':::FLOAT8), UNIQUE (col3_4 DESC, col3_2 DESC, col3_1 DESC, col3_14 ASC) STORING (col3_11, col3_12) WHERE ((((((((((table3.col3_11 > '"') AND (table3.col3_6 <= 0)) AND (table3.col3_0 != -1)) OR (table3.col3_12 >= 3.4028234663852886e+38)) AND (table3.col3_8 != '"')) OR (table3.col3_5 != 5e-324)) AND (table3.col3_9 != e'\'')) OR (table3.col3_14 > -1.0)) OR (table3.col3_4 = '')) OR (table3.col3_13 <= -128)) AND (table3.col3_10 != -1), INDEX (col3_5 DESC, col3_7, col3_11 ASC, col3_1 ASC, col3_3 DESC, col3_14 DESC, col3_6, col3_12 DESC, col3_0 DESC, col3_8 ASC, col3_10 ASC) STORING (col3_13) WHERE (((((table3.col3_8 < 'X') OR (table3.col3_12 < 0.0)) AND (table3.col3_13 > 0)) OR (table3.col3_11 != '')) AND (table3.col3_5 > NaN)) AND (table3.col3_10 >= -32768), INDEX (col3_1, col3_12 DESC, col3_5 ASC, col3_10 ASC, col3_3, col3_7 DESC, col3_6, col3_9 ASC, col3_4, col3_14 ASC, col3_0 DESC, col3_2 ASC, col3_8, col3_13 DESC, col3_11 ASC) WHERE ((((table3.col3_13 > (-1):::INT8) OR (table3.col3_14 = 1.0:::FLOAT8)) OR (table3.col3_0 <= (-32768):::INT8)) AND (table3.col3_5 < 1.401298464324817e-45:::FLOAT8)) AND (table3.col3_6 <= (-1):::DECIMAL), INDEX (col3_2 DESC) STORING (col3_11, col3_12) WHERE (((table3.col3_10 > 0:::INT8) AND (table3.col3_9 > 'X':::STRING)) AND (table3.col3_6 > (-1.234E+401):::DECIMAL)) OR (table3.col3_12 > (-1.0):::FLOAT8), FAMILY (col3_1), FAMILY (col3_6, col3_12, col3_14, col3_13, col3_3, col3_11), FAMILY (col3_2, col3_0, col3_5, col3_7, col3_4));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5088385668135298393, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000000, "num_range": 0, "upper_bound": "553602643"}, {"distinct_range": 1576913838059578000, "num_eq": 400000000, "num_range": 1576913838059578159, "upper_bound": "1285838409"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 7551934018090086032}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2845842341218085810, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7551934018090086032}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6061279060639465909, "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 7551934018090086032}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 355333415505627394, "histo_buckets": [{"distinct_range": 0, "num_eq": 6404225556081494725, "num_range": 0, "upper_bound": "\'"}, {"distinct_range": 1083253891750430500, "num_eq": 7918639012950130149, "num_range": 5876210039389919079, "upper_bound": "Bo\\u001fE\\u0015O"}, {"distinct_range": 0, "num_eq": 8998920408624633984, "num_range": 5607695958185555106, "upper_bound": "IP\x7f"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 2017420159641537478, "row_count": 7551934018090086032}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2970330418786360842, "histo_col_type": "", "name": "__auto__", "null_count": 6545403703983566488, "row_count": 7551934018090086032}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6075677619436028472, "histo_buckets": [{"distinct_range": 0, "num_eq": 3714740968909014701, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 346068087162165799, "num_range": 2601489586177475125, "upper_bound": "\\u00017"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "\\u000cU\\u0002c\\u001211"}, {"distinct_range": 0, "num_eq": 30, "num_range": 1514835569768785164, "upper_bound": "\\u0013`FA\\u0017D7"}, {"distinct_range": 0, "num_eq": 8532101584108989547, "num_range": 0, "upper_bound": "-/"}, {"distinct_range": 0, "num_eq": 3316921854834821936, "num_range": 8736684046097332691, "upper_bound": "F~"}, {"distinct_range": 8026852622929953000, "num_eq": 5868666006193591163, "num_range": 8026852622929953270, "upper_bound": "X"}, {"distinct_range": 200000, "num_eq": 3561579553604992475, "num_range": 200000, "upper_bound": "\\\\3\\u0002MPna"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 4530199038503003528, "row_count": 7551934018090086032}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6145266223952748769, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7551934018090086032}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 338, "histo_col_type": "", "name": "__auto__", "null_count": 776, "row_count": 800}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 598, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 380, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "-8.329617127620624359E+30"}, {"distinct_range": 1073126202294674000, "num_eq": 500000000, "num_range": 1073126202294674092, "upper_bound": "-7199.4798479736102"}, {"distinct_range": 563962011477114300, "num_eq": 800000000, "num_range": 3873614380512469018, "upper_bound": "-0.1297851463042292401"}, {"distinct_range": 253990077568168540, "num_eq": 0, "num_range": 7563327790815447197, "upper_bound": "758.1009895518995232"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 485, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 540, "histo_buckets": [{"distinct_range": 0, "num_eq": 2614115037519645823, "num_range": 0, "upper_bound": "101000110100000101001"}, {"distinct_range": 0, "num_eq": 8424136259306676693, "num_range": 9000, "upper_bound": "101010110110100111011"}, {"distinct_range": 0, "num_eq": 1118458028353866473, "num_range": 40000, "upper_bound": "111011110101000110000"}], "histo_col_type": "BIT(21)", "name": "__auto__", "null_count": 180, "row_count": 800}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 57, "histo_col_type": "", "name": "__auto__", "null_count": 370, "row_count": 800}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 432, "histo_buckets": [{"distinct_range": 0, "num_eq": 4692596659116825460, "num_range": 0, "upper_bound": "-1.64808321518719"}, {"distinct_range": 3644297107837375500, "num_eq": 6366077903060263667, "num_range": 3644297107837375302, "upper_bound": "-0.024971920427203115"}, {"distinct_range": 1416161058869230600, "num_eq": 8357235441679110351, "num_range": 1416161058869230465, "upper_bound": "0.008526257185142994"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 456, "row_count": 800}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 252, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 413, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 5215758494511283000, "num_eq": 779093743435226595, "num_range": 5215758494511283021, "upper_bound": "\\\\x0a7f92"}, {"distinct_range": 82250984611455410, "num_eq": 900000, "num_range": 1398288941463671756, "upper_bound": "\\\\x113f"}, {"distinct_range": 14472230.819418041, "num_eq": 20000000, "num_range": 30000000, "upper_bound": "\\\\x885e39"}, {"distinct_range": 2629469491987290000, "num_eq": 10000, "num_range": 2629469491987290093, "upper_bound": "\\\\x92"}, {"distinct_range": 3307778412725524000, "num_eq": 100000, "num_range": 3307778412725524160, "upper_bound": "\\\\xd2f1"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 7000, "upper_bound": "\\\\xf77b5aa1efe6b106"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 775, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "150.2.87.195/6"}, {"distinct_range": 0, "num_eq": 60, "num_range": 80, "upper_bound": "71.247.225.211/12"}, {"distinct_range": 0, "num_eq": 800, "num_range": 40000000, "upper_bound": "1f0a:e648:31ba:1557:1bb0:a202:ba4e:40e4/72"}, {"distinct_range": 2417948398559941000, "num_eq": 90000000, "num_range": 2417948398559940908, "upper_bound": "842f:a10b:822b:54a0:96ea:4ec8:88fb:625e/73"}, {"distinct_range": 5000000, "num_eq": 8000, "num_range": 5000000, "upper_bound": "3031:d934:73b0:26b2:207c:ddd9:ee04:28d5/94"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 244, "row_count": 800}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 755, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 463, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 333, "histo_buckets": [{"distinct_range": 0, "num_eq": 8435380362203420643, "num_range": 0, "upper_bound": "-0.8243595015897504"}, {"distinct_range": 7181.059856988497, "num_eq": 700, "num_range": 10000, "upper_bound": "-0.386861514591485"}, {"distinct_range": 7604809.898719398, "num_eq": 6000000, "num_range": 90000000, "upper_bound": "0.15427057513746345"}, {"distinct_range": 360.4434948038104, "num_eq": 1659572375328483293, "num_range": 800, "upper_bound": "0.975653454769318"}, {"distinct_range": 0, "num_eq": 90000, "num_range": 0, "upper_bound": "1.0183358021408468"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 308, "row_count": 800}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 350, "histo_col_type": "INT2", "name": "__auto__", "null_count": 610, "row_count": 800}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 714, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 128, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11, "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4, "histo_buckets": [{"distinct_range": 0, "num_eq": 125461765876298205, "num_range": 0, "upper_bound": "1476460676"}, {"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": "1700465565"}, {"distinct_range": 2412284781906427400, "num_eq": 669876341187702610, "num_range": 3601911510268686204, "upper_bound": "1980998947"}, {"distinct_range": 2446624.97257128, "num_eq": 40000000000, "num_range": 3000000, "upper_bound": "2358473515"}, {"distinct_range": 400000, "num_eq": 200, "num_range": 400000, "upper_bound": "2829233919"}, {"distinct_range": 18870816226.282406, "num_eq": 8416325889737751658, "num_range": 40000000000, "upper_bound": "2841355800"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 800}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 425, "histo_col_type": "", "name": "__auto__", "null_count": 640, "row_count": 800}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 580, "histo_col_type": "", "name": "__auto__", "null_count": 46, "row_count": 800}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30, "histo_col_type": "", "name": "__auto__", "null_count": 515, "row_count": 800}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": "-1.5205944768443436"}, {"distinct_range": 1704307.0296773177, "num_eq": 0, "num_range": 7000000, "upper_bound": "-0.5000328018907141"}, {"distinct_range": 3294.61302707298, "num_eq": 800000000, "num_range": 6000, "upper_bound": "-0.46217290144701634"}, {"distinct_range": 90000000000, "num_eq": 7000000, "num_range": 90000000000, "upper_bound": "0.9583608424294037"}, {"distinct_range": 0, "num_eq": 30000000000, "num_range": 0, "upper_bound": "1.3860398776795384"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 500000, "num_range": 0, "upper_bound": "-1"}, {"distinct_range": 0, "num_eq": 0, "num_range": 4000000000, "upper_bound": "0"}, {"distinct_range": 7380026413230071000, "num_eq": 600000000, "num_range": 7380026413230071190, "upper_bound": "1275"}, {"distinct_range": 8279834879057931000, "num_eq": 1209244679396497084, "num_range": 8279834879057931740, "upper_bound": "29713"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000000000, "num_range": 0, "upper_bound": "0010100110010001100111010"}, {"distinct_range": 4307548300041997000, "num_eq": 9000000, "num_range": 4307548300041996821, "upper_bound": "0110100010010110111100011"}, {"distinct_range": 889957801682015000, "num_eq": 2114056672912168560, "num_range": 3274458803281497517, "upper_bound": "0111110010110100000000000"}, {"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": "1000010111011001110100011"}, {"distinct_range": 8825987485677081000, "num_eq": 107201204770415103, "num_range": 8825987485677081008, "upper_bound": "1011111111001000110011010"}, {"distinct_range": 157357835380005730, "num_eq": 8606511610080655134, "num_range": 157357835380005732, "upper_bound": "1100110010011111010110111"}], "histo_col_type": "BIT(25)", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 4272240940451257319, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 3489526897373834056, "num_range": 5206360602797274122, "upper_bound": "771402219"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 7352022398325608205, "upper_bound": "1076398011"}, {"distinct_range": 0, "num_eq": 20000000000, "num_range": 100, "upper_bound": "1289978120"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 5469551932256598418, "upper_bound": "1906770324"}, {"distinct_range": 700, "num_eq": 7688507622906669905, "num_range": 700, "upper_bound": "2113425104"}, {"distinct_range": 2446317681772603400, "num_eq": 8715499900829148674, "num_range": 2446317681772603409, "upper_bound": "3067121325"}, {"distinct_range": 0, "num_eq": 4339429975217889188, "num_range": 2999421808494818909, "upper_bound": "3851459763"}, {"distinct_range": 66380345.08622871, "num_eq": 6346245710766773945, "num_range": 70000000, "upper_bound": "3897175329"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 0}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000, "num_range": 0, "upper_bound": "-1.4502381033828493"}, {"distinct_range": 8226975441927203000, "num_eq": 30000000000, "num_range": 8226975441927202392, "upper_bound": "-0.8081211682118512"}, {"distinct_range": 0, "num_eq": 100, "num_range": 1145243711702031123, "upper_bound": "-0.7830251199289817"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 8233832900971212840, "upper_bound": "-0.500280327548998"}, {"distinct_range": 4305015156361068500, "num_eq": 60, "num_range": 4305015156361068782, "upper_bound": "0.1286076504932815"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 0}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('fpkuoa', 'wcie', 'ualjb', 'rveicu');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 558893074761147654, "histo_buckets": [{"distinct_range": 0, "num_eq": 2704644119361420205, "num_range": 0, "upper_bound": "\\u0004Y\'\'"}, {"distinct_range": 0, "num_eq": 3928822577305835383, "num_range": 661932256278635495, "upper_bound": "j$\\u0018\\u000f+T\\u0018\\t"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 2997347411973098907, "row_count": 7921675728258745504}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6634679347756894923, "histo_col_type": "", "name": "__auto__", "null_count": 7845486274121603546, "row_count": 7921675728258745504}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5454596141370263137, "histo_buckets": [{"distinct_range": 0, "num_eq": 5144325898381254152, "num_range": 0, "upper_bound": "0eK1\\u001c\\u0010\'j%"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 1305561629945452688, "row_count": 7921675728258745504}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5224160743464068891, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7921675728258745504}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5611811561266922361, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000, "num_range": 0, "upper_bound": "651688235"}, {"distinct_range": 1000000000, "num_eq": 7753448804588428097, "num_range": 1000000000, "upper_bound": "1140473369"}, {"distinct_range": 0, "num_eq": 4000000000, "num_range": 6790998119444030927, "upper_bound": "2593557532"}, {"distinct_range": 7205816884922486000, "num_eq": 5747146644508051586, "num_range": 8112699545915222258, "upper_bound": "2886385799"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 7921675728258745504}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7257357839536175239, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7921675728258745504}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7253175745299274131, "histo_buckets": [{"distinct_range": 0, "num_eq": 3367180522787871289, "num_range": 0, "upper_bound": "00:00:00+15:59:00"}, {"distinct_range": 20000000000, "num_eq": 70000000000, "num_range": 20000000000, "upper_bound": "06:53:31.74983+08:22:00"}, {"distinct_range": 0, "num_eq": 20000000000, "num_range": 90000000000, "upper_bound": "17:33:43.637567+11:45:00"}, {"distinct_range": 0, "num_eq": 9000, "num_range": 400000, "upper_bound": "10:08:33.308928+01:52:00"}, {"distinct_range": 1000000000, "num_eq": 700000, "num_range": 1000000000, "upper_bound": "14:03:23.418952-00:20:00"}, {"distinct_range": 379977.2160050746, "num_eq": 40, "num_range": 1000000, "upper_bound": "21:52:42.31636+02:19:00"}, {"distinct_range": 791347056158816500, "num_eq": 8569693164741768569, "num_range": 791347056158816527, "upper_bound": "08:46:11.662313-12:54:00"}, {"distinct_range": 3163414135371973600, "num_eq": 9000, "num_range": 3163414135371973616, "upper_bound": "14:53:20.476739-09:56:00"}, {"distinct_range": 129087013.39259352, "num_eq": 8000, "num_range": 800000000, "upper_bound": "20:09:26.225096-07:03:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 7921675728258745504}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13631739, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35614102, "histo_col_type": "", "name": "__auto__", "null_count": 16119096, "row_count": 40000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20360730, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2577911, "histo_col_type": "", "name": "__auto__", "null_count": 11769304, "row_count": 40000000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8551673, "histo_buckets": [{"distinct_range": 0, "num_eq": 8253722699824041695, "num_range": 0, "upper_bound": "-14999"}, {"distinct_range": 19165834287.853436, "num_eq": 8909464091260434892, "num_range": 30000000000, "upper_bound": "-2823"}, {"distinct_range": 6000, "num_eq": 1093416656328651008, "num_range": 6000, "upper_bound": "16547"}, {"distinct_range": 472058975058194370, "num_eq": 4542515514549141888, "num_range": 2629100778744093389, "upper_bound": "20355"}, {"distinct_range": 0, "num_eq": 6964757223244169680, "num_range": 200000, "upper_bound": "22646"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 33972229, "row_count": 40000000}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30661180, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12800947, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14899076, "histo_buckets": [{"distinct_range": 0, "num_eq": 852143091078934748, "num_range": 0, "upper_bound": "000100000011111100011"}, {"distinct_range": 300, "num_eq": 5859609964835341757, "num_range": 300, "upper_bound": "011000101010001001110"}, {"distinct_range": 1377731422357882000, "num_eq": 800000000, "num_range": 4071585182083625710, "upper_bound": "011100000100101101001"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 100000, "upper_bound": "011100101001100110000"}, {"distinct_range": 100, "num_eq": 20000, "num_range": 100, "upper_bound": "011110110010011111110"}, {"distinct_range": 0, "num_eq": 200, "num_range": 200000, "upper_bound": "100101010110001101000"}], "histo_col_type": "BIT(21)", "name": "__auto__", "null_count": 16576543, "row_count": 40000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24013982, "histo_col_type": "", "name": "__auto__", "null_count": 33084574, "row_count": 40000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25065080, "histo_buckets": [{"distinct_range": 0, "num_eq": 656882985759694816, "num_range": 0, "upper_bound": "-1.6848869804547542"}, {"distinct_range": 70000000000, "num_eq": 1000000000, "num_range": 70000000000, "upper_bound": "0.04665707891490256"}, {"distinct_range": 800, "num_eq": 60000000, "num_range": 800, "upper_bound": "0.2827550931831882"}, {"distinct_range": 8376300900363998000, "num_eq": 9000000000, "num_range": 8376300900363998428, "upper_bound": "0.3989702475175072"}, {"distinct_range": 30, "num_eq": 8000000, "num_range": 30, "upper_bound": "1.038069373488618"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 7941480, "row_count": 40000000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24027715, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4162777, "histo_col_type": "", "name": "__auto__", "null_count": 5599335, "row_count": 40000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 171474, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000000000, "num_range": 0, "upper_bound": "\\\\xd83bf37da9"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 40000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13875745, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19204090, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "NaN"}, {"distinct_range": 1060331912475767300, "num_eq": 6000, "num_range": 1615348356249013417, "upper_bound": "-1.2685989241282574"}, {"distinct_range": 0, "num_eq": 6864662474295395706, "num_range": 0, "upper_bound": "0.1494279405331097"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 11293551, "row_count": 40000000}, {"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16996036, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27808937, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000, "num_range": 0, "upper_bound": "10:01:12.337088+13:34:00"}, {"distinct_range": 4665510858822665000, "num_eq": 40000, "num_range": 4665510858822664918, "upper_bound": "02:16:17.207016+03:10:00"}, {"distinct_range": 259.9085879294443, "num_eq": 9000, "num_range": 300, "upper_bound": "14:16:46.124547+09:39:00"}, {"distinct_range": 782181871.121202, "num_eq": 8000000000, "num_range": 800000000, "upper_bound": "02:51:59.632632-10:28:00"}, {"distinct_range": 6388500736036298000, "num_eq": 5542186386379032384, "num_range": 6388500736036297360, "upper_bound": "24:00:00-15:59:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 40000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36047387, "histo_buckets": [{"distinct_range": 0, "num_eq": 6682553727282554627, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 2243108667830582300, "num_eq": 100000000, "num_range": 5745331530332461613, "upper_bound": "539808443"}, {"distinct_range": 6656910945959627000, "num_eq": 7649128837833341442, "num_range": 7808069925954567946, "upper_bound": "1436046055"}, {"distinct_range": 6744811441184467000, "num_eq": 30000000, "num_range": 6744811441184466582, "upper_bound": "2827142291"}, {"distinct_range": 994702.7141028754, "num_eq": 7336557621454775749, "num_range": 6000000, "upper_bound": "3181214611"}, {"distinct_range": 4986927292393784000, "num_eq": 6000, "num_range": 4986927292393783832, "upper_bound": "3794647651"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 40000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11906230, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000, "num_range": 0, "upper_bound": "249.150.50.40/5"}, {"distinct_range": 35129.496617698605, "num_eq": 100, "num_range": 50000, "upper_bound": "35.76.194.146/12"}, {"distinct_range": 3817924829953736000, "num_eq": 643047454696949505, "num_range": 8956867839524572142, "upper_bound": "400b:6c8e:56e:3b79:335d:6d47:de8c:dc29/41"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 90, "upper_bound": "af2a:45f1:d5d6:727a:59b0:464d:5d19:a2f9/71"}, {"distinct_range": 0, "num_eq": 499980138083078404, "num_range": 4725892527473150240, "upper_bound": "f06a:ab6f:e401:f7d2:7ca4:e1:7fed:642a/76"}, {"distinct_range": 9000, "num_eq": 100000, "num_range": 9000, "upper_bound": "9f66:ebf6:cb80:797f:e800:4919:835f:2dd2/77"}, {"distinct_range": 5358290466397328000, "num_eq": 100, "num_range": 5358290466397328004, "upper_bound": "aeb8:b526:79f1:408c:f204:4c71:407f:46a7/114"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 7880899, "row_count": 40000000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23006676, "histo_buckets": [{"distinct_range": 0, "num_eq": 4592620196332385991, "num_range": 0, "upper_bound": "-Infinity"}, {"distinct_range": 0, "num_eq": 2000, "num_range": 40000, "upper_bound": "-14449707.10304214258"}, {"distinct_range": 0, "num_eq": 3410075420318681725, "num_range": 50000, "upper_bound": "-0.07857016021736461721"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 40000000}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10704625, "histo_col_type": "", "name": "__auto__", "null_count": 27968630, "row_count": 40000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 994, "histo_col_type": "", "name": "__auto__", "null_count": 892, "row_count": 1000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 439, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 125, "histo_col_type": "", "name": "__auto__", "null_count": 992, "row_count": 1000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 639, "histo_col_type": "", "name": "__auto__", "null_count": 500, "row_count": 1000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 824, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 514, "histo_buckets": [{"distinct_range": 0, "num_eq": 60, "num_range": 0, "upper_bound": "-2.285245362098053"}, {"distinct_range": 1307456056057218800, "num_eq": 0, "num_range": 2344376311546653119, "upper_bound": "-1.162475613876743"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 164436192684685500, "upper_bound": "0.31754093406463624"}, {"distinct_range": 316194686818425540, "num_eq": 5260102719982028570, "num_range": 4371630781232264464, "upper_bound": "0.3720704013357526"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 525, "histo_col_type": "", "name": "__auto__", "null_count": 216, "row_count": 1000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 504, "histo_buckets": [{"distinct_range": 0, "num_eq": 7900635045609099772, "num_range": 0, "upper_bound": "3"}, {"distinct_range": 6000000, "num_eq": 800, "num_range": 6000000, "upper_bound": "7"}, {"distinct_range": 0, "num_eq": 0, "num_range": 1974975920546594654, "upper_bound": "@"}, {"distinct_range": 0, "num_eq": 0, "num_range": 7541429787732889626, "upper_bound": "F"}, {"distinct_range": 60, "num_eq": 4089247718195591133, "num_range": 60, "upper_bound": "X"}, {"distinct_range": 0, "num_eq": 3433739687864515942, "num_range": 6733174098360986475, "upper_bound": "p"}, {"distinct_range": 1085075066454075500, "num_eq": 562048101362249076, "num_range": 1085075066454075520, "upper_bound": "t"}, {"distinct_range": 100, "num_eq": 10000000, "num_range": 100, "upper_bound": "y"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 964, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000000, "num_range": 0, "upper_bound": "-0.41437069729060483"}, {"distinct_range": 3016021042627969.5, "num_eq": 6000000000, "num_range": 5752177716684649, "upper_bound": "1.3054357829992758"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 309, "histo_col_type": "BIT(25)", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 443, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 219, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 152, "histo_col_type": "", "name": "__auto__", "null_count": 436, "row_count": 1000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 992, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "-27111"}, {"distinct_range": 4161554214502827000, "num_eq": 10000, "num_range": 4161554214502827197, "upper_bound": "-20334"}, {"distinct_range": 2406.3239651810004, "num_eq": 7167043073062178051, "num_range": 9000, "upper_bound": "-10036"}, {"distinct_range": 3803130117743307300, "num_eq": 100000000000, "num_range": 3803130117743307284, "upper_bound": "31937"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 1000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 447, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000000, "num_range": 0, "upper_bound": "25961970"}, {"distinct_range": 0, "num_eq": 4300268200239065875, "num_range": 10, "upper_bound": "595410846"}, {"distinct_range": 700000000, "num_eq": 0, "num_range": 700000000, "upper_bound": "1605297290"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 1000}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1526, "histo_buckets": [{"distinct_range": 0, "num_eq": 7309492247331484134, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 5005650125147817000, "num_eq": 500000000, "num_range": 6619827248930431290, "upper_bound": "2572215307"}, {"distinct_range": 0, "num_eq": 7667597773298461324, "num_range": 1000, "upper_bound": "3018911860"}, {"distinct_range": 1045837579239698700, "num_eq": 305692421021541686, "num_range": 1045837579239698689, "upper_bound": "4171174058"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34490, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22259, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "10:22:23.716953+13:55:00"}, {"distinct_range": 2206150439390392800, "num_eq": 8450611987927975864, "num_range": 6767787807274397037, "upper_bound": "02:47:19.204597+04:46:00"}, {"distinct_range": 8161908731877899000, "num_eq": 7308694482507539883, "num_range": 8161908731877899204, "upper_bound": "05:48:27.284862+00:18:00"}, {"distinct_range": 21852213.618563358, "num_eq": 20, "num_range": 30000000, "upper_bound": "10:37:30.939366+04:27:00"}, {"distinct_range": 16.996774760004747, "num_eq": 6802155621789259133, "num_range": 90, "upper_bound": "18:19:20.559021+05:44:00"}, {"distinct_range": 50000000000, "num_eq": 9127147795438075464, "num_range": 50000000000, "upper_bound": "21:23:46.266191+04:16:00"}, {"distinct_range": 0, "num_eq": 80, "num_range": 0, "upper_bound": "06:13:06.16122-13:35:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 49381, "histo_col_type": "STRING", "name": "__auto__", "null_count": 48974, "row_count": 60000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7999, "histo_col_type": "", "name": "__auto__", "null_count": 31051, "row_count": 60000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5215, "histo_buckets": [{"distinct_range": 0, "num_eq": 80, "num_range": 0, "upper_bound": "\\t=@+2*"}, {"distinct_range": 0, "num_eq": 400, "num_range": 1141635737964921319, "upper_bound": "-\\u0015\'"}, {"distinct_range": 0, "num_eq": 400000, "num_range": 5000000000, "upper_bound": "Q"}, {"distinct_range": 0, "num_eq": 4349225377345759327, "num_range": 0, "upper_bound": "X"}, {"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "c"}, {"distinct_range": 17118396.536446523, "num_eq": 4487547643786706177, "num_range": 100000000, "upper_bound": "i:\\u001an\\u000bs\'B"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 3751, "row_count": 60000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13787, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_20"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 69, "histo_col_type": "", "name": "__auto__", "null_count": 14, "row_count": 80}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30, "histo_col_type": "", "name": "__auto__", "null_count": 12, "row_count": 80}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53, "histo_buckets": [{"distinct_range": 0, "num_eq": 8121177195292396596, "num_range": 0, "upper_bound": "-0.8933577066256579"}, {"distinct_range": 5535667330693692000, "num_eq": 90000000, "num_range": 5535667330693692314, "upper_bound": "-0.7672583530856659"}, {"distinct_range": 1122468127835640300, "num_eq": 1861830131833031524, "num_range": 1957031578699069567, "upper_bound": "-0.6546733920007677"}, {"distinct_range": 1714008093564145.5, "num_eq": 20000, "num_range": 7158900076976422, "upper_bound": "-0.527674729097699"}, {"distinct_range": 0, "num_eq": 5141163176795272146, "num_range": 300, "upper_bound": "0.12401932398947324"}, {"distinct_range": 2466480836832463000, "num_eq": 8171990591761898672, "num_range": 9223180162099435527, "upper_bound": "0.9229857186028299"}, {"distinct_range": 2403556123466654700, "num_eq": 3098088282584228739, "num_range": 2403556123466654499, "upper_bound": "1.3943233712991756"}, {"distinct_range": 1182355520085197800, "num_eq": 100, "num_range": 1182355520085197701, "upper_bound": "1.6153355094710202"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 17, "row_count": 80}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 73, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000, "num_range": 0, "upper_bound": "-7.32312802387362008E+32"}, {"distinct_range": 0, "num_eq": 6873566467098742777, "num_range": 8098378913171020453, "upper_bound": "-7.735981598548655345E+27"}, {"distinct_range": 61162198.252695985, "num_eq": 5000000000, "num_range": 300000000, "upper_bound": "-1"}, {"distinct_range": 6365303416957702000, "num_eq": 700000000, "num_range": 6365303416957702456, "upper_bound": "71.30080206046628718"}, {"distinct_range": 3000000, "num_eq": 50000000, "num_range": 3000000, "upper_bound": "8.134458014046255079E+27"}, {"distinct_range": 300, "num_eq": 100000000000, "num_range": 300, "upper_bound": "3.736163975995191888E+37"}, {"distinct_range": 3569192259758132700, "num_eq": 2699927322923441479, "num_range": 6224401091313588278, "upper_bound": "Infinity"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 73, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000000, "num_range": 0, "upper_bound": "-24934"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 23, "row_count": 80}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col2_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 68, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col2_19"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 68, "histo_col_type": "", "name": "__auto__", "null_count": 25, "row_count": 80}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 47, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 68, "histo_buckets": [{"distinct_range": 0, "num_eq": 7228030759146726441, "num_range": 0, "upper_bound": "011101011001010110100"}, {"distinct_range": 0, "num_eq": 40, "num_range": 6339311277871119141, "upper_bound": "110101101110110010100"}, {"distinct_range": 4793539.761471434, "num_eq": 40, "num_range": 5000000, "upper_bound": "111000101110111111100"}], "histo_col_type": "BIT(21)", "name": "__auto__", "null_count": 71, "row_count": 80}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 72, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 65, "histo_buckets": [{"distinct_range": 0, "num_eq": 3055495190001046866, "num_range": 0, "upper_bound": "225.141.139.108/8"}, {"distinct_range": 80, "num_eq": 5000000000, "num_range": 80, "upper_bound": "139.53.237.45/22"}, {"distinct_range": 2384380072155726300, "num_eq": 1000000, "num_range": 2384380072155726144, "upper_bound": "114.135.53.238/27"}, {"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "8d06:3f5e:99f6:8ccd:50a4:42c0:f9d2:c302/20"}, {"distinct_range": 2210325.111919033, "num_eq": 4511416548246259660, "num_range": 6000000, "upper_bound": "9752:e90a:e757:93f9:9020:76ef:a8a4:28a4/51"}, {"distinct_range": 0, "num_eq": 4036919126472948348, "num_range": 400000, "upper_bound": "98d0:ef43:f340:e643:5dfa:d8a6:bdb4:bdfb/77"}, {"distinct_range": 0, "num_eq": 200, "num_range": 80000000, "upper_bound": "de81:f66a:9553:a87a:fb8a:2322:26f3:8b95/103"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 70, "row_count": 80}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 61, "histo_buckets": [{"distinct_range": 0, "num_eq": 4340034204122046181, "num_range": 0, "upper_bound": "13:27:37.622196+13:24:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5, "histo_buckets": [{"distinct_range": 0, "num_eq": 8725489217973725166, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 726992815013067800, "num_eq": 1835200876966606861, "num_range": 4611451906585952957, "upper_bound": "461338504"}, {"distinct_range": 249.53843023392773, "num_eq": 100000, "num_range": 300, "upper_bound": "879979367"}, {"distinct_range": 5897423202877609000, "num_eq": 257759658080328419, "num_range": 6832610022209516581, "upper_bound": "1429158470"}, {"distinct_range": 4411772676689562000, "num_eq": 5000000, "num_range": 6236971823441178224, "upper_bound": "1846474023"}, {"distinct_range": 0, "num_eq": 20, "num_range": 10, "upper_bound": "1915266129"}, {"distinct_range": 0, "num_eq": 9000, "num_range": 80, "upper_bound": "2769177057"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9, "histo_col_type": "", "name": "__auto__", "null_count": 77, "row_count": 80}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24, "histo_col_type": "", "name": "__auto__", "null_count": 68, "row_count": 80}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26, "histo_buckets": [{"distinct_range": 0, "num_eq": 4948974675331054047, "num_range": 0, "upper_bound": "\\\\x0365c2ed8ea2e1"}, {"distinct_range": 0, "num_eq": 0, "num_range": 60, "upper_bound": "\\\\x3e9634b74f7e7cad"}, {"distinct_range": 218.15356598497652, "num_eq": 6208255698205803038, "num_range": 300, "upper_bound": "\\\\x58"}, {"distinct_range": 0, "num_eq": 5999919521620019697, "num_range": 3876723172730763585, "upper_bound": "\\\\xff"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25, "histo_buckets": [{"distinct_range": 0, "num_eq": 21961343674192697, "num_range": 0, "upper_bound": "-1.7338079944931393"}, {"distinct_range": 0, "num_eq": 7744707267965172000, "num_range": 1782122348213212633, "upper_bound": "-1.0427740450939331"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 0, "upper_bound": "-0.726522922982085"}, {"distinct_range": 200185546272289020, "num_eq": 30, "num_range": 4130103271457280883, "upper_bound": "-0.3214875462339668"}, {"distinct_range": 3070174786380525000, "num_eq": 4161239773364535688, "num_range": 3070174786380525120, "upper_bound": "-0.01591105930986883"}, {"distinct_range": 0, "num_eq": 1000, "num_range": 25681779254316731, "upper_bound": "1.4449078498619878"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 10, "row_count": 80}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 670933627627577188, "histo_buckets": [{"distinct_range": 0, "num_eq": 7302926235058709402, "num_range": 0, "upper_bound": "E"}, {"distinct_range": 5500555335.655115, "num_eq": 600000, "num_range": 10000000000, "upper_bound": "T"}, {"distinct_range": 20003.190742052393, "num_eq": 7000, "num_range": 100000, "upper_bound": "["}, {"distinct_range": 2945657635880854500, "num_eq": 2719033254605188836, "num_range": 8518453463372637765, "upper_bound": "p"}], "histo_col_type": "\\"char\\"", "name": "__auto__", "null_count": 0, "row_count": 722149410048955257}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 423570142137416157, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 722149410048955257}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33667552061579941, "histo_col_type": "", "name": "__auto__", "null_count": 422850986932379573, "row_count": 722149410048955257}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 443636466796278196, "histo_buckets": [{"distinct_range": 0, "num_eq": 6599062337513822501, "num_range": 0, "upper_bound": "-1.6819521965584578"}, {"distinct_range": 60000, "num_eq": 800, "num_range": 60000, "upper_bound": "-1.3369000236617912"}, {"distinct_range": 30000, "num_eq": 700000, "num_range": 30000, "upper_bound": "-0.887362563202269"}, {"distinct_range": 2000000, "num_eq": 40000000000, "num_range": 2000000, "upper_bound": "0.16756355357765373"}, {"distinct_range": 0, "num_eq": 8714652819833579564, "num_range": 1421452652093122726, "upper_bound": "0.25758143173901515"}, {"distinct_range": 10000000000, "num_eq": 747918517713776793, "num_range": 10000000000, "upper_bound": "0.3370145849681968"}, {"distinct_range": 0, "num_eq": 487536355449858154, "num_range": 1085171679473660002, "upper_bound": "0.3821705753331886"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 722149410048955257}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 105522536265194822, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 722149410048955257}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 436668692300206812, "histo_col_type": "", "name": "__auto__", "null_count": 560641680762709045, "row_count": 722149410048955257}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 688258974678853472, "histo_col_type": "", "name": "__auto__", "null_coun

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2773663&tab=buildLog) on [master@4d44ddf24153d8ef8e0a996fdbe75ac5607f9574](https://github.com/cockroachdb/cockroach/commits/4d44ddf24153d8ef8e0a996fdbe75ac5607f9574):

```
        test_1        |             CREATE INDEX ON table3 (col3_2 ASC, col3_9 ASC, col3_8 ASC, col3_4, col3_6 DESC, col3_1 ASC, col3_5 DESC);
        test_1        |             CREATE TABLE table4 (col1_6 INET NOT NULL, col1_8 TIMETZ NOT NULL, col1_4 VARCHAR NOT NULL, col1_3 NAME NOT NULL, col1_2 BIT(15) NOT NULL, col4_5 TIMETZ NULL, col4_6 INET NULL, col4_7 "char", col4_8 CHAR NOT NULL, col4_9 DATE NULL, col4_10 INT8, col4_11 CHAR NOT NULL, col4_12 BIT NOT NULL, col4_13 INET NULL, col4_14 BYTEA, col4_15 BYTEA, col4_16 INTERVAL NOT NULL, col4_17 BIT(3), col4_18 TIMETZ NOT NULL, PRIMARY KEY (col1_6, col1_8, col1_4, col1_3, col1_2, col4_5, col4_18, col4_14, col4_7, col4_11, col4_8, col4_9, col4_6, col4_16, col4_13));
        test_1        |             CREATE UNIQUE INDEX ON table4 (col1_6 ASC, col4_13 ASC, col4_16, col1_3, col4_17 DESC, col1_2) INCLUDE (col4_12);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_18, col4_13, col4_15 DESC, col4_6, col4_7, col1_3 DESC) INCLUDE (col4_10);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_15 ASC, col4_12 DESC, col4_5 ASC, col4_13 DESC) INCLUDE (col4_17);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_5 ASC);
        test_1        |             CREATE INDEX ON table4 (col4_18, col4_8 DESC, col1_3, col1_6 DESC, col4_9 ASC, col1_8, col4_11 DESC, col4_15, col4_5, col4_13, col4_7 DESC) INCLUDE (col4_10, col4_17);
        test_1        |             CREATE INDEX ON table4 (col1_6 ASC, col4_10 ASC, col4_8, col4_12 ASC, col4_9 ASC, col4_7 ASC, col4_17 DESC, col4_14, col4_16 ASC, col4_18);
        test_1        |             CREATE INDEX ON table4 (col1_2 ASC, col4_18 ASC, col4_7 DESC, col4_16 DESC, col1_3 DESC, col1_4, col4_11, col4_6 ASC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('lkbs', 'kmzvka', 'xvrba');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: syntax error at or near "AS" (SQLSTATE 42601)
        test_1        |     --- FAIL: TestCompare/mutators (0.78s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 BYTES NOT NULL, col1_1 REGNAMESPACE NULL, col1_2 INT2, col1_3 DECIMAL NULL, PRIMARY KEY (col1_0), col1_4 DECIMAL NULL AS (col1_3 + 8.765836736107960631:::DECIMAL) STORED, col1_5 STRING AS (lower(CAST(col1_1 AS STRING))) VIRTUAL, col1_6 STRING NULL AS (IF(col1_0 IS NULL, '/[':::STRING, '':::STRING)) STORED, INDEX (col1_5, col1_4 DESC, col1_1 DESC) STORING (col1_2, col1_6) WHERE (table1.col1_4 > 1) AND (table1.col1_2 = -32768), INDEX (col1_6 ASC, col1_5 ASC, col1_4, col1_0 ASC, col1_1 DESC, col1_2 ASC, col1_3 ASC) WHERE (table1.col1_2 = 127) OR (table1.col1_4 <= -1.234E+401), UNIQUE (col1_0 ASC) STORING (col1_1, col1_2) WHERE (((((table1.col1_2 >= (-32768):::INT8) OR (table1.col1_3 < 'NaN':::DECIMAL)) AND (table1.col1_5 < 'X':::STRING)) AND (table1.col1_6 < e'\U00002603':::STRING)) OR (table1.col1_4 >= 1:::DECIMAL)) OR (table1.col1_0 != e'\\x22':::BYTES), UNIQUE (col1_0, col1_2 DESC, col1_3 DESC, col1_4, col1_1 DESC, col1_5 DESC) WHERE table1.col1_5 <= '"', UNIQUE (col1_4) WHERE (((((table1.col1_5 >= '"':::STRING) OR (table1.col1_6 != e'\x00':::STRING)) AND (table1.col1_2 != (-1):::INT8)) AND (table1.col1_4 = 0:::DECIMAL)) OR (table1.col1_3 != (-1):::DECIMAL)) AND (table1.col1_0 < e'\\x':::BYTES), UNIQUE (col1_2, col1_6 ASC, col1_1 ASC, col1_5 DESC, col1_0 ASC, col1_3 DESC, col1_4) WHERE table1.col1_0 = '\xff', INDEX (col1_6 DESC, col1_3, col1_1 ASC, col1_0 DESC) STORING (col1_2) WHERE ((((table1.col1_0 > '\xff') AND (table1.col1_2 != 1)) AND (table1.col1_3 < NaN)) AND (table1.col1_4 < -1.234E+401)) AND (table1.col1_6 < '"'), UNIQUE (col1_1, col1_5 DESC, col1_3 DESC, col1_2 DESC, col1_0 ASC) STORING (col1_4) WHERE ((((table1.col1_2 >= 127:::INT8) OR (table1.col1_3 < (-1.234E+401):::DECIMAL)) AND (table1.col1_6 < 'X':::STRING)) OR (table1.col1_5 > '"':::STRING)) AND (table1.col1_0 >= e'\\x00':::BYTES), INDEX (col1_1 ASC, col1_4 ASC, col1_2 DESC, col1_5, col1_0) STORING (col1_6) WHERE ((table1.col1_4 = '-Infinity':::DECIMAL) AND (table1.col1_3 >= 'NaN':::DECIMAL)) OR (table1.col1_5 != e'\'':::STRING), FAMILY (col1_2), FAMILY (col1_3), FAMILY (col1_1, col1_4), FAMILY (col1_6), FAMILY (col1_0));
        test_1        |             CREATE TABLE table2 (col2_0 REGTYPE NOT NULL, col2_1 OID NULL, col2_2 DECIMAL, col2_3 INT2 NULL, col2_4 BOX2D NULL, col2_5 BOX2D NOT NULL, col2_6 VARCHAR NULL, col2_7 TIMESTAMPTZ NOT NULL, col2_8 TIMESTAMP NOT NULL, col2_9 BYTES NOT NULL, col2_10 DATE NOT NULL, col2_11 INT4 NOT NULL, col2_12 BOOL NULL, col2_13 INTERVAL, col2_14 OID NULL, col2_15 "char" NULL, col2_16 STRING NULL AS (lower(CAST(col2_8 AS STRING))) VIRTUAL, col2_17 STRING NULL AS (lower(col2_6)) VIRTUAL, INDEX (col2_10, col2_5 ASC, col2_13, col2_14 DESC, col2_1 ASC, col2_7 DESC, col2_6 ASC, col2_9, col2_0 DESC, col2_17, col2_11) STORING (col2_4, col2_15) WHERE (((((((((((table2.col2_15 > e'\U00002603':::STRING) AND (table2.col2_17 >= '"':::STRING)) AND (table2.col2_8 != '294276-12-31 23:59:59.999999':::TIMESTAMP)) AND (table2.col2_3 < (-1):::INT8)) OR (table2.col2_10 <= '1970-01-01':::DATE)) OR (table2.col2_6 = e'\'':::STRING)) OR (table2.col2_2 = '-Infinity':::DECIMAL)) AND (table2.col2_7 <= '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table2.col2_11 >= (-1):::INT8)) AND table2.col2_12) AND (table2.col2_9 = e'\\x58':::BYTES)) OR (table2.col2_16 <= '"':::STRING), FAMILY (col2_12, col2_7, col2_0, col2_15, col2_6, col2_2, col2_3), FAMILY (col2_9), FAMILY (col2_8), FAMILY (col2_4, col2_1), FAMILY (col2_10), FAMILY (col2_11), FAMILY (col2_13, col2_5, col2_14));
        test_1        |             CREATE TABLE table3 (col3_0 BIT(27) NULL, col3_1 CHAR NULL, col3_2 INT8 NOT NULL, col3_3 CHAR, col3_4 FLOAT4 NOT NULL, col3_5 TIME NOT NULL, col3_6 FLOAT8 NOT NULL, col3_7 REGTYPE NOT NULL, col3_8 FLOAT4 NOT NULL, col3_9 "char", PRIMARY KEY (col3_7, col3_2 DESC, col3_5 DESC, col3_4 ASC, col3_8 DESC), col3_10 FLOAT8 NOT NULL AS ((col3_6 + col3_4) + col3_8) STORED, col3_11 FLOAT8 NULL AS ((col3_6 + col3_8) + col3_4) STORED, col3_12 STRING AS (lower(col3_3)) STORED, INDEX (col3_8) STORING (col3_3, col3_10, col3_11) WHERE table3.col3_10 <= 3.4028234663852886e+38:::FLOAT8, UNIQUE (col3_4 DESC, col3_7, col3_11, col3_12, col3_8 ASC, col3_10 ASC, col3_0, col3_9, col3_2 ASC, col3_1 DESC, col3_6 DESC, col3_5 DESC), INDEX (col3_0 ASC, col3_10, col3_2 DESC, col3_12 DESC, col3_8 ASC, col3_4 DESC, col3_11 DESC, col3_1 ASC, col3_5 DESC, col3_6 DESC, col3_9 DESC, col3_7 ASC) WHERE ((((((((table3.col3_2 = -1) AND (table3.col3_3 >= e'\'')) OR (table3.col3_12 > '')) OR (table3.col3_4 <= NaN)) OR (table3.col3_9 = 'X')) AND (table3.col3_11 > 3.4028234663852886e+38)) AND (table3.col3_6 > 1.401298464324817e-45)) AND (table3.col3_1 < '')) AND (table3.col3_5 = '24:00:00'), FAMILY (col3_4, col3_2, col3_12, col3_1), FAMILY (col3_5, col3_7), FAMILY (col3_10), FAMILY (col3_6, col3_11, col3_9), FAMILY (col3_0), FAMILY (col3_3, col3_8));
        test_1        |             CREATE TABLE table4 (col4_0 BYTES NOT NULL, col4_1 "char" NOT NULL, col4_2 REGNAMESPACE NOT NULL, col4_3 INT8 NOT NULL, col4_4 BIT(5) NOT NULL, col4_5 REGPROCEDURE, col4_6 BYTES NULL, col4_7 TIME, col4_8 DATE NOT NULL, col4_9 INET NULL, col4_10 REGPROCEDURE NOT NULL, col4_11 BYTES NOT NULL, col4_12 FLOAT4 NOT NULL, col4_13 BYTES NOT NULL, PRIMARY KEY (col4_11 DESC, col4_3, col4_13 DESC, col4_4, col4_10, col4_1 DESC, col4_12 ASC), col4_14 STRING NULL AS (lower(CAST(col4_9 AS STRING))) VIRTUAL, INDEX (col4_11 ASC, col4_0 DESC, col4_10 DESC, col4_14 DESC, col4_7 ASC) STORING (col4_5, col4_6, col4_8, col4_9) WHERE table4.col4_8 != '5874897-12-31':::DATE, UNIQUE (col4_10 DESC, col4_9 ASC, col4_4, col4_0 DESC, col4_3 ASC, col4_12 ASC, col4_11, col4_8 ASC, col4_2 ASC) STORING (col4_6, col4_7) WHERE table4.col4_3 < -9223372036854775807, INDEX (col4_14 DESC, col4_12, col4_7 ASC, col4_10 ASC, col4_6 ASC, col4_11, col4_3 ASC, col4_4 DESC, col4_8, col4_1 DESC, col4_2 DESC, col4_0, col4_5 DESC, col4_13, col4_9 ASC) WHERE (((((((((table4.col4_6 <= e'\\xe29883':::BYTES) OR (table4.col4_0 != e'\\x00':::BYTES)) OR (table4.col4_12 = 0.0:::FLOAT8)) OR (table4.col4_1 > e'\'':::STRING)) AND (table4.col4_7 > '24:00:00':::TIME)) OR (table4.col4_11 <= e'\\x00':::BYTES)) OR (table4.col4_14 <= 'X':::STRING)) OR (table4.col4_3 > (-32768):::INT8)) OR (table4.col4_13 <= e'\\x':::BYTES)) OR (table4.col4_8 > '1970-01-01':::DATE), INDEX (col4_7, col4_5 DESC, col4_1 ASC, col4_0, col4_2 ASC, col4_9 DESC, col4_14 ASC, col4_3, col4_6, col4_4, col4_12 DESC, col4_11 ASC, col4_8, col4_13 ASC) WHERE ((((table4.col4_12 <= -Inf) OR (table4.col4_1 != 'X')) OR (table4.col4_11 != '\xff')) AND (table4.col4_13 != '\x58')) AND (table4.col4_0 > '\x22'), UNIQUE (col4_9 DESC, col4_4 DESC, col4_11 DESC, col4_10, col4_12 ASC, col4_2 DESC, col4_1 DESC, col4_14, col4_0 DESC, col4_3 ASC, col4_7 ASC, col4_8 DESC, col4_6 DESC, col4_5 DESC) WHERE (((((((((table4.col4_13 >= '\x') OR (table4.col4_8 != '1970-01-01')) AND (table4.col4_7 > '24:00:00')) OR (table4.col4_1 < e'\'')) OR (table4.col4_0 != '\x58')) AND (table4.col4_12 < 0.0)) AND (table4.col4_14 = 'X')) OR (table4.col4_6 >= '\x27')) OR (table4.col4_3 < -128)) AND (table4.col4_11 <= '\x'), FAMILY (col4_13), FAMILY (col4_4, col4_3, col4_9), FAMILY (col4_10, col4_12), FAMILY (col4_11, col4_5, col4_0), FAMILY (col4_2), FAMILY (col4_1, col4_7), FAMILY (col4_8, col4_6));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 0, "num_range": 4000000000, "upper_bound": "\\u0015IS\\u0004\\u000e"}, {"distinct_range": 0, "num_eq": 500, "num_range": 9072909136103459128, "upper_bound": "*CL0K"}, {"distinct_range": 3514968508620549600, "num_eq": 9027938938387213083, "num_range": 7950940319947864923, "upper_bound": "^\\u0019r-\'.C#\\u001f"}, {"distinct_range": 3011199970.4004054, "num_eq": 4555324552357049333, "num_range": 10000000000, "upper_bound": "_`iz"}, {"distinct_range": 21164178.782399885, "num_eq": 100000000, "num_range": 50000000, "upper_bound": "g=\\\\#Dm-|)"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 254, "row_count": 400}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 372, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 954.8557667518138, "num_eq": 6000000000, "num_range": 1000, "upper_bound": "\\\\x16421146a0e31e"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 40000000, "upper_bound": "\\\\x1aa58376"}, {"distinct_range": 21623.26756536971, "num_eq": 0, "num_range": 90000, "upper_bound": "\\\\x1b85c29a94e3"}, {"distinct_range": 10000000, "num_eq": 900, "num_range": 10000000, "upper_bound": "\\\\x37d6fedb9c4b672a46"}, {"distinct_range": 0, "num_eq": 7761807948439272282, "num_range": 50, "upper_bound": "\\\\x90"}, {"distinct_range": 0, "num_eq": 900, "num_range": 1506763095846232347, "upper_bound": "\\\\xbfce7d854523"}, {"distinct_range": 4910146613967080000, "num_eq": 80000000, "num_range": 4910146613967080895, "upper_bound": "\\\\xf429216f12a6b7ebff"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 400}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 253, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "280708463"}, {"distinct_range": 30000, "num_eq": 5000, "num_range": 30000, "upper_bound": "1994732923"}, {"distinct_range": 0, "num_eq": 200, "num_range": 4000000, "upper_bound": "2775589222"}, {"distinct_range": 4837163684915879000, "num_eq": 90000000000, "num_range": 4837163684915878877, "upper_bound": "3670879920"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 364, "row_count": 400}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 264, "histo_buckets": [{"distinct_range": 0, "num_eq": 4966064896893431787, "num_range": 0, "upper_bound": "-3282"}, {"distinct_range": 0, "num_eq": 400000000, "num_range": 4000000000, "upper_bound": "0"}, {"distinct_range": 4469630779811416000, "num_eq": 100000, "num_range": 8790286688145831745, "upper_bound": "2604"}, {"distinct_range": 1181841.117185307, "num_eq": 4000000, "num_range": 4000000, "upper_bound": "32767"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 371, "row_count": 400}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 124, "histo_col_type": "", "name": "__auto__", "null_count": 69, "row_count": 400}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 107, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000000, "num_range": 0, "upper_bound": "-4.18217834941671707E+27"}, {"distinct_range": 800, "num_eq": 40000000, "num_range": 800, "upper_bound": "-1.456221974923676964E+22"}, {"distinct_range": 5036940119058742000, "num_eq": 100000000000, "num_range": 5036940119058741841, "upper_bound": "-74951077.3614728"}, {"distinct_range": 2825793788074166000, "num_eq": 8903836688784215307, "num_range": 3484053439824274859, "upper_bound": "2114653.747923316956"}, {"distinct_range": 300, "num_eq": 7272720456309484605, "num_range": 300, "upper_bound": "3600649654.014537572"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 0, "upper_bound": "1.316787657281646365E+24"}, {"distinct_range": 190293213.10536513, "num_eq": 9000000, "num_range": 400000000, "upper_bound": "8.214271893981910648E+29"}, {"distinct_range": 20000000, "num_eq": 3156276678841127307, "num_range": 20000000, "upper_bound": "6.49484658115053809E+31"}, {"distinct_range": 0, "num_eq": 70000000000, "num_range": 6000000, "upper_bound": "Infinity"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 114, "row_count": 400}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 338, "histo_buckets": [{"distinct_range": 0, "num_eq": 3358722768531249187, "num_range": 0, "upper_bound": "\\u0001F=\\u001eAp\\u001f"}, {"distinct_range": 25424993773.392815, "num_eq": 6000, "num_range": 30000000000, "upper_bound": "\\u0014P(`\\u0013\\u0014\\u000bN"}, {"distinct_range": 774845032.729786, "num_eq": 60000000000, "num_range": 800000000, "upper_bound": "\\u001f[S"}, {"distinct_range": 2000, "num_eq": 80, "num_range": 2000, "upper_bound": "\'"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 1000000, "upper_bound": "0in"}, {"distinct_range": 70, "num_eq": 3000000000, "num_range": 70, "upper_bound": "S\\u0008$\\\\"}, {"distinct_range": 80000000, "num_eq": 6000000000, "num_range": 80000000, "upper_bound": "Y\\u0019c|\\u000bR"}, {"distinct_range": 0, "num_eq": 7091205388900560530, "num_range": 2108610408399935477, "upper_bound": "\\\\\\u0015Lht=\\u0005"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 366, "row_count": 400}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 794915490319056615, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4042951563024380815}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2868656003312221060, "histo_col_type": "", "name": "__auto__", "null_count": 2550841593236037971, "row_count": 4042951563024380815}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 256123865551547035, "histo_col_type": "", "name": "__auto__", "null_count": 2167969348688261492, "row_count": 4042951563024380815}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 987341528007823618, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4042951563024380815}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3224226980538365767, "histo_col_type": "", "name": "__auto__", "null_count": 1492263087826465690, "row_count": 4042951563024380815}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3652180002272427541, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4042951563024380815}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3682573471637317807, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4042951563024380815}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3394227967511416459, "histo_col_type": "", "name": "__auto__", "null_count": 490493325292606170, "row_count": 4042951563024380815}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 767842175559155385, "histo_buckets": [{"distinct_range": 0, "num_eq": 20, "num_range": 0, "upper_bound": "1979-08-03"}, {"distinct_range": 30000, "num_eq": 100000000, "num_range": 30000, "upper_bound": "1986-09-10"}, {"distinct_range": 246.91043963631176, "num_eq": 7170438661022260451, "num_range": 400, "upper_bound": "5874897-12-31"}, {"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "infinity"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 4042951563024380815}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1739215833631582747, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4042951563024380815}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1677582311479021115, "histo_col_type": "", "name": "__auto__", "null_count": 2507400670641532439, "row_count": 4042951563024380815}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1458553944664293037, "histo_col_type": "", "name": "__auto__", "null_count": 242226532499371452, "row_count": 4042951563024380815}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2478810771641225946, "histo_col_type": "", "name": "__auto__", "null_count": 3120411687581371205, "row_count": 4042951563024380815}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2854234906719480764, "histo_col_type": "", "name": "__auto__", "null_count": 2700199900367731368, "row_count": 4042951563024380815}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2471582044915977391, "histo_col_type": "", "name": "__auto__", "null_count": 1698106429229375613, "row_count": 4042951563024380815}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3372632195493123621, "histo_col_type": "", "name": "__auto__", "null_count": 454712183253255949, "row_count": 4042951563024380815}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3518931492990376882, "histo_col_type": "", "name": "__auto__", "null_count": 2630032649209247778, "row_count": 4042951563024380815}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1867445218593266538, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4042951563024380815}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46907, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29438, "histo_col_type": "", "name": "__auto__", "null_count": 53976, "row_count": 60000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44579, "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 58664, "histo_buckets": [{"distinct_range": 0, "num_eq": 3283277197826997961, "num_range": 0, "upper_bound": "-1.8143949508666992"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 700, "upper_bound": "-1.4907270669937134"}, {"distinct_range": 49785543660999470, "num_eq": 2539368669899378267, "num_range": 767497550489121496, "upper_bound": "-0.7333988547325134"}, {"distinct_range": 7158010476031914000, "num_eq": 10000000000, "num_range": 8282526115364059692, "upper_bound": "-0.5368011593818665"}, {"distinct_range": 0, "num_eq": 801332214438487192, "num_range": 300076671455797036, "upper_bound": "0.046539515256881714"}, {"distinct_range": 1038622759910173800, "num_eq": 4853899444885450832, "num_range": 9099735916203225836, "upper_bound": "1.6050975322723389"}, {"distinct_range": 5331508835740809000, "num_eq": 6000000, "num_range": 8236297277008226257, "upper_bound": "3.4028234663852886e+38"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 52822, "histo_col_type": "", "name": "__auto__", "null_count": 40103, "row_count": 60000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 40874, "histo_col_type": "", "name": "__auto__", "null_count": 4972, "row_count": 60000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33790, "histo_buckets": [{"distinct_range": 0, "num_eq": 1738705656286251538, "num_range": 0, "upper_bound": "-1.0556952953338623"}, {"distinct_range": 0, "num_eq": 7106420366450326724, "num_range": 90000, "upper_bound": "-0.8278659582138062"}, {"distinct_range": 56536.93173134727, "num_eq": 8469355153751803240, "num_range": 70000, "upper_bound": "-0.8052799701690674"}, {"distinct_range": 1403380557194546400, "num_eq": 9101195429461378601, "num_range": 3991660050054395324, "upper_bound": "-0.49376553297042847"}, {"distinct_range": 8461450.609192502, "num_eq": 80000000000, "num_range": 50000000, "upper_bound": "-0.09130457788705826"}, {"distinct_range": 6248096663271085000, "num_eq": 20000000000, "num_range": 6248096663271084745, "upper_bound": "0.2315259873867035"}, {"distinct_range": 27703261811.89451, "num_eq": 50000000000, "num_range": 30000000000, "upper_bound": "0.3607410490512848"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 1000000, "upper_bound": "0.36699944734573364"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 59037, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 24564, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 49525, "histo_col_type": "", "name": "__auto__", "null_count": 20048, "row_count": 60000}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 28515, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 60000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20313, "histo_col_type": "", "name": "__auto__", "null_count": 47388, "row_count": 60000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26016, "histo_col_type": "BIT(27)", "name": "__auto__", "null_count": 47990, "row_count": 60000}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5369752233206469067, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5979195382802790307}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3181516788351511964, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5979195382802790307}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2494110863891186651, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000000, "num_range": 0, "upper_bound": "00:00:00"}, {"distinct_range": 1494062165374449400, "num_eq": 7306009604144422462, "num_range": 2632429597508100565, "upper_bound": "02:43:31.553015"}, {"distinct_range": 172165103094237800, "num_eq": 1247834470861865900, "num_range": 172165103094237797, "upper_bound": "04:43:23.951554"}, {"distinct_range": 357253776352760600, "num_eq": 6022538988827810930, "num_range": 4530516125611950247, "upper_bound": "23:31:09.293972"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 4955208239343916899, "row_count": 5979195382802790307}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2342001771043133457, "histo_buckets": [{"distinct_range": 0, "num_eq": 2213517711242456552, "num_range": 0, "upper_bound": "0.0.0.0/0"}, {"distinct_range": 0, "num_eq": 9000, "num_range": 10, "upper_bound": "205.194.241.71/5"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 313264784499789509, "upper_bound": "45.107.142.82/8"}, {"distinct_range": 48160712561.34158, "num_eq": 700000000, "num_range": 50000000000, "upper_bound": "21.96.33.248/13"}, {"distinct_range": 0, "num_eq": 105812615184636987, "num_range": 100, "upper_bound": "9bf1:dcca:abd3:ef81:5135:cb82:8624:964c/10"}, {"distinct_range": 1329849454152488000, "num_eq": 100000000, "num_range": 1929850429216242406, "upper_bound": "770f:99d5:4c1b:3bce:a051:1665:d5a1:4df7/19"}, {"distinct_range": 4389576293296488400, "num_eq": 7767702193159807950, "num_range": 4389576293296488649, "upper_bound": "380a:33c5:9867:2396:ed7d:e13f:9562:7007/23"}, {"distinct_range": 0, "num_eq": 3030508046808067393, "num_range": 0, "upper_bound": "48fb:ee3c:866b:88b7:615f:1dfe:f2bc:66c6/49"}, {"distinct_range": 0, "num_eq": 2730566423932506802, "num_range": 6807958446414186775, "upper_bound": "5c12:e33c:b4f8:9179:864d:4316:bb7e:3f22/120"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 4677851778669220167, "row_count": 5979195382802790307}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5509955343461695717, "histo_buckets": [{"distinct_range": 0, "num_eq": 1217206050580605682, "num_range": 0, "upper_bound": "\\"ne\\u0012_<TM;"}, {"distinct_range": 7445507164337020000, "num_eq": 20000000, "num_range": 7445507164337020198, "upper_bound": "%F"}, {"distinct_range": 243557507.3091129, "num_eq": 6000000, "num_range": 400000000, "upper_bound": "2?+hxe\\u001c"}, {"distinct_range": 20000000000, "num_eq": 6346673283975388889, "num_range": 20000000000, "upper_bound": "rt\\n"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 3779685867516148248, "row_count": 5979195382802790307}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 319981390999098832, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5979195382802790307}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3710968307997375996, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5979195382802790307}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4467875461836646144, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5979195382802790307}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 656101208330279068, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5979195382802790307}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 402088544058946011, "histo_buckets": [{"distinct_range": 0, "num_eq": 200, "num_range": 0, "upper_bound": "251901055"}, {"distinct_range": 431885872542077600, "num_eq": 3383694907303188962, "num_range": 431885872542077549, "upper_bound": "363559773"}, {"distinct_range": 5000, "num_eq": 7015407493414601040, "num_range": 5000, "upper_bound": "842275781"}, {"distinct_range": 0, "num_eq": 200000, "num_range": 8504657408569676670, "upper_bound": "1189095794"}, {"distinct_range": 0, "num_eq": 6936728129341979863, "num_range": 300000, "upper_bound": "1370894500"}, {"distinct_range": 9000000000, "num_eq": 900000, "num_range": 9000000000, "upper_bound": "1858602978"}, {"distinct_range": 70.30948293356622, "num_eq": 50, "num_range": 80, "upper_bound": "3164635516"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 5979195382802790307}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1566547651529166591, "histo_buckets": [{"distinct_range": 0, "num_eq": 9071171245919987472, "num_range": 0, "upper_bound": "\\\\x07ed"}, {"distinct_range": 0, "num_eq": 9118826060965796145, "num_range": 20000000, "upper_bound": "\\\\x080688a33696477b"}, {"distinct_range": 233597513.8544522, "num_eq": 4632945579008294952, "num_range": 500000000, "upper_bound": "\\\\x324e11a006e1f984fc"}, {"distinct_range": 3375834834442365000, "num_eq": 3873603698621201618, "num_range": 3375834834442364796, "upper_bound": "\\\\xb83e4423e7da"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 8456365781042672215, "upper_bound": "\\\\xd3a8d3"}, {"distinct_range": 12335956249.837847, "num_eq": 40000000, "num_range": 30000000000, "upper_bound": "\\\\xd9"}, {"distinct_range": 0, "num_eq": 7586873793975762073, "num_range": 50000000000, "upper_bound": "\\\\xf9"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 5979195382802790307}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5674276497962191041, "histo_col_type": "", "name": "__auto__", "null_count": 2331211332956567077, "row_count": 5979195382802790307}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2340112217163963088, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5979195382802790307}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2301789640297932566, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5979195382802790307}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4880156274713711732, "histo_col_type": "", "name": "__auto__", "null_count": 2966685894003827245, "row_count": 5979195382802790307}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('vxqfq', 'rzcx', 'f', 'ujfsj');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('jfvyq', 'wvhrxe', 'ucab', 'pdnqsg', 'pacun', 'gq');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('dtt', 'bu', 'mv', 'w', 'xprj');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('atf', 'ghret', 'huiux', 'uwzf', 'rvwt');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('fei', 'd', 'hl', 'yiwl', 'zbvoz');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3085289052131490288, "histo_col_type": "STRING", "name": "__auto__", "null_count": 1893391207229065234, "row_count": 5804040007622195843}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5206897446346322120, "histo_buckets": [{"distinct_range": 0, "num_eq": 910487446986572166, "num_range": 0, "upper_bound": "\\\\x2ed06c9def593a8612"}, {"distinct_range": 3243.8612899705668, "num_eq": 0, "num_range": 90000, "upper_bound": "\\\\x7e20be85"}, {"distinct_range": 0, "num_eq": 4325074942396140169, "num_range": 0, "upper_bound": "\\\\xb8a025c1"}, {"distinct_range": 1974879332.406857, "num_eq": 5000000000, "num_range": 6000000000, "upper_bound": "\\\\xe29883"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 5804040007622195843}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 977890472848486700, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": "88507919"}, {"distinct_range": 2131405172539977200, "num_eq": 10000000, "num_range": 7264006067205047196, "upper_bound": "1029072233"}, {"distinct_range": 20, "num_eq": 20, "num_range": 20, "upper_bound": "1463003236"}, {"distinct_range": 6626008916255964000, "num_eq": 659980867735537824, "num_range": 6626008916255964448, "upper_bound": "1467316977"}, {"distinct_range": 7632111201430413000, "num_eq": 200000000, "num_range": 9133396105712628274, "upper_bound": "1582090974"}, {"distinct_range": 0, "num_eq": 9153480022114497969, "num_range": 50000, "upper_bound": "2070007750"}, {"distinct_range": 0, "num_eq": 8152745209798967399, "num_range": 30000, "upper_bound": "2702822530"}, {"distinct_range": 0, "num_eq": 70000000, "num_range": 3372617532667297161, "upper_bound": "2894424186"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 2429592227584301578, "row_count": 5804040007622195843}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1417657063234305728, "histo_buckets": [{"distinct_range": 0, "num_eq": 6541366710694610235, "num_range": 0, "upper_bound": "-19223"}, {"distinct_range": 0, "num_eq": 2222956885031306336, "num_range": 2000000, "upper_bound": "-11803"}, {"distinct_range": 1629604756745663700, "num_eq": 6263029509245498574, "num_range": 5738421079512961813, "upper_bound": "-10669"}, {"distinct_range": 0, "num_eq": 449812654400190110, "num_range": 900000, "upper_bound": "10349"}, {"distinct_range": 0, "num_eq": 100, "num_range": 20000000000, "upper_bound": "22335"}, {"distinct_range": 10000, "num_eq": 200000, "num_range": 10000, "upper_bound": "26509"}, {"distinct_range": 6590892412036755000, "num_eq": 1000000000, "num_range": 6590892412036755765, "upper_bound": "28259"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 2586241042062840113, "row_count": 5804040007622195843}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5355143164945859583, "histo_col_type": "", "name": "__auto__", "null_count": 1708747560456943880, "row_count": 5804040007622195843}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5585280161121808336, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000, "num_range": 0, "upper_bound": "-4.7707934156830323E+18"}, {"distinct_range": 60000000000, "num_eq": 4592723278964893489, "num_range": 60000000000, "upper_bound": "854128916.407746129"}, {"distinct_range": 190.61392640327625, "num_eq": 1000, "num_range": 800, "upper_bound": "3.84469332021936553E+23"}, {"distinct_range": 75076831073924780, "num_eq": 60, "num_range": 262643376886212116, "upper_bound": "6.952497067407728263E+36"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 2255168534501548619, "row_count": 5804040007622195843}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4062024131468189672, "histo_buckets": [{"distinct_range": 0, "num_eq": 6421885086391983654, "num_range": 0, "upper_bound": ".g\\u001c"}, {"distinct_range": 20, "num_eq": 9000, "num_range": 20, "upper_bound": "~*["}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 3956732639790290742, "row_count": 5804040007622195843}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 154438, "histo_col_type": "", "name": "__auto__", "null_count": 13557, "row_count": 500000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 461155, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 127853, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 159152, "histo_col_type": "", "name": "__auto__", "null_count": 393627, "row_count": 500000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 235864, "histo_col_type": "", "name": "__auto__", "null_count": 375735, "row_count": 500000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 496213, "histo_col_type": "", "name": "__auto__", "null_count": 399898, "row_count": 500000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 49198, "histo_col_type": "", "name": "__auto__", "null_count": 362295, "row_count": 500000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 377250, "histo_col_type": "", "name": "__auto__", "null_count": 268632, "row_count": 500000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 63262, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 261109, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 82191, "histo_col_type": "", "name": "__auto__", "null_count": 621, "row_count": 500000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 489721, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 164919, "histo_col_type": "", "name": "__auto__", "null_count": 333319, "row_count": 500000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 247990, "histo_col_type": "", "name": "__auto__", "null_count": 18998, "row_count": 500000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 253544, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000, "num_range": 0, "upper_bound": "-infinity"}, {"distinct_range": 4231430771030511000, "num_eq": 1000000000, "num_range": 4231430771030510966, "upper_bound": "1970-10-13"}, {"distinct_range": 0, "num_eq": 0, "num_range": 8313027022598527053, "upper_bound": "1975-09-11"}, {"distinct_range": 40829523798.192665, "num_eq": 20000, "num_range": 100000000000, "upper_bound": "1984-08-22"}, {"distinct_range": 6892149842150236000, "num_eq": 8834157685906439019, "num_range": 6892149842150235863, "upper_bound": "1986-12-21"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 284347, "histo_col_type": "", "name": "__auto__", "null_count": 134991, "row_count": 500000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 311222, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 258450, "histo_col_type": "", "name": "__auto__", "null_count": 201387, "row_count": 500000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13, "histo_buckets": [{"distinct_range": 0, "num_eq": 8677895378397952634, "num_range": 0, "upper_bound": "319852780"}, {"distinct_range": 4146054338593024500, "num_eq": 2000, "num_range": 4146054338593024360, "upper_bound": "638464209"}, {"distinct_range": 10000000, "num_eq": 4000000000, "num_range": 10000000, "upper_bound": "1664192070"}, {"distinct_range": 80, "num_eq": 2000, "num_range": 80, "upper_bound": "1986153101"}, {"distinct_range": 50544022.546988264, "num_eq": 70000, "num_range": 70000000, "upper_bound": "3542853986"}, {"distinct_range": 11.890527326250503, "num_eq": 40000000, "num_range": 30, "upper_bound": "3867369556"}, {"distinct_range": 8999.890362512371, "num_eq": 7214894039078263007, "num_range": 9000, "upper_bound": "4069997774"}, {"distinct_range": 0, "num_eq": 7889166045730597393, "num_range": 8000000000, "upper_bound": "4076305833"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 43, "histo_col_type": "", "name": "__auto__", "null_count": 48, "row_count": 50}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 37, "histo_col_type": "", "name": "__auto__", "null_count": 48, "row_count": 50}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20, "histo_buckets": [{"distinct_range": 0, "num_eq": 5160054705769377533, "num_range": 0, "upper_bound": "001101010011111100100011101"}, {"distinct_range": 0, "num_eq": 30000000, "num_range": 6292584573468658082, "upper_bound": "010100101111000000011010100"}, {"distinct_range": 0, "num_eq": 4217313534678014502, "num_range": 1557661951061312311, "upper_bound": "100010010101000001101010101"}, {"distinct_range": 450150941.2952478, "num_eq": 8000000, "num_range": 700000000, "upper_bound": "101001000001100100001001011"}, {"distinct_range": 5648157311557624000, "num_eq": 5226847475700099628, "num_range": 5648157311557623307, "upper_bound": "101011110011001010111011111"}, {"distinct_range": 0, "num_eq": 7277653964135363624, "num_range": 920630655700224640, "upper_bound": "101101100110111111000001000"}, {"distinct_range": 50000000, "num_eq": 1000, "num_range": 50000000, "upper_bound": "110011111000111111001111001"}, {"distinct_range": 2095559950523119000, "num_eq": 5452012107292434620, "num_range": 2095559950523119132, "upper_bound": "110111001111000001000001100"}, {"distinct_range": 6063624968643295000, "num_eq": 10000000000, "num_range": 7008935099467676683, "upper_bound": "111001011101110011111000000"}], "histo_col_type": "BIT(27)", "name": "__auto__", "null_count": 10, "row_count": 50}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21, "histo_col_type": "", "name": "__auto__", "null_count": 46, "row_count": 50}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 0, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "0.15343059599399567"}, {"distinct_range": 0, "num_eq": 2458751930201863116, "num_range": 9042508722072016903, "upper_bound": "0.8160967826843262"}, {"distinct_range": 5474684030405661000, "num_eq": 568434833940423451, "num_range": 7006990429919306493, "upper_bound": "1.5654160976409912"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29, "histo_col_type": "", "name": "__auto__", "null_count": 35, "row_count": 50}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33, "histo_col_type": "", "name": "__auto__", "null_count": 9, "row_count": 50}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 50}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38, "histo_buckets": [{"distinct_range": 0, "num_eq": 6372883863548960483, "num_range": 0, "upper_bound": "-0.8660390377044678"}, {"distinct_range": 20, "num_eq": 30000000, "num_range": 20, "upper_bound": "-0.6278088688850403"}, {"distinct_range": 0, "num_eq": 40000000, "num_range": 3997674346633060749, "upper_bound": "-0.5251173377037048"}, {"distinct_range": 2381147010416133000, "num_eq": 7271797804734188268, "num_range": 2381147010416132969, "upper_bound": "-0.08433402329683304"}, {"distinct_range": 7037883796017120000, "num_eq": 30000, "num_range": 8292500926604643865, "upper_bound": "0.3594124913215637"}, {"distinct_range": 200000000, "num_eq": 2928116485063448935, "num_range": 200000000, "upper_bound": "0.6917569637298584"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "0.8239467144012451"}, {"distinct_range": 200, "num_eq": 900000000, "num_range": 200, "upper_bound": "1.2386975288391113"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 50}]';
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 210791614, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9327583330, "histo_col_type": "", "name": "__auto__", "null_count": 6511440895, "row_count": 10000000000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3470281096, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7472784288, "histo_buckets": [{"distinct_range": 0, "num_eq": 3686248608457900624, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 0, "num_eq": 5118010598724919257, "num_range": 10000, "upper_bound": "\\\\x43f80e65c8c3f4"}, {"distinct_range": 0, "num_eq": 50000000, "num_range": 1654499247088331120, "upper_bound": "\\\\x44f225d9af9512c4ce"}, {"distinct_range": 66648989.56680272, "num_eq": 2000, "num_range": 100000000, "upper_bound": "\\\\x77cf286b"}, {"distinct_range": 0, "num_eq": 6085034233801468146, "num_range": 300000, "upper_bound": "\\\\xf6240013fd"}, {"distinct_range": 0, "num_eq": 8142640758719644930, "num_range": 1665741418017625086, "upper_bound": "\\\\xfbabedeb80fe0ec0"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2566856831, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2389357434, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2713637025, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7729240247, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "151.83.202.212/31"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 50000000, "upper_bound": "f72f:56cd:80f7:bae7:37ff:c768:1f29:6ea7/68"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 2541429423, "row_count": 10000000000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6605884989, "histo_col_type": "", "name": "__auto__", "null_count": 4470442843, "row_count": 10000000000}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3822044786, "histo_buckets": [{"distinct_range": 0, "num_eq": 5815023669926223913, "num_range": 0, "upper_bound": "1358398815"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 884583581, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5042265718, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8300084201, "histo_buckets": [{"distinct_range": 0, "num_eq": 2120265186173368768, "num_range": 0, "upper_bound": "00:44:14.763017"}, {"distinct_range": 0, "num_eq": 3707876156565227310, "num_range": 1856647901313233225, "upper_bound": "07:05:13.627509"}, {"distinct_range": 0, "num_eq": 2000000, "num_range": 10000000, "upper_bound": "14:25:48.450456"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 5887747034, "row_count": 10000000000}, {"columns": ["col4_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8446974152, "histo_buckets": [{"distinct_range": 0, "num_eq": 3884293901525927724, "num_range": 0, "upper_bound": ""}, {"distinct_range": 3681814951170706400, "num_eq": 3474198699555294053, "num_range": 5155174509226139949, "upper_bound": "\\u0006O"}, {"distinct_range": 0, "num_eq": 20000, "num_range": 50000, "upper_bound": ".}\\u001e\\u0019Gf\\u000c"}, {"distinct_range": 4000000, "num_eq": 8000000000, "num_range": 4000000, "upper_bound": ":o9~H#\\u0007"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 2630296592, "row_count": 10000000000}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7666778061, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 10000000000}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 41160366, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000, "num_range": 0, "upper_bound": "-16338"}, {"distinct_range": 0, "num_eq": 40000000000, "num_range": 1432992586875038907, "upper_bound": "-10547"}, {"distinct_range": 2678370460888047600, "num_eq": 7922521439196538441, "num_range": 2678370460888047584, "upper_bound": "1"}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 4778324521383464244, "upper_bound": "21451"}, {"distinct_range": 3752491960967227000, "num_eq": 20, "num_range": 3752491960967227031, "upper_bound": "22087"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 57522742, "row_count": 70000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3218414, "histo_col_type": "", "name": "__auto__", "null_count": 6197217, "row_count": 70000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44400079, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000, "num_range": 0, "upper_bound": "59229977110.80431516"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 14617631, "row_count": 70000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 42801561, "histo_buckets": [{"distinct_range": 0, "num_eq": 5320976146065234128, "num_range": 0, "upper_bound": ""}, {"distinct_range": 10000000, "num_eq": 6361739011715453924, "num_range": 10000000, "upper_bound": "\\u0000"}, {"distinct_range": 4990137459296767000, "num_eq": 0, "num_range": 4990137459296767217, "upper_bound": "\\u0018P$"}, {"distinct_range": 14981.965785858474, "num_eq": 1000, "num_range": 300000, "upper_bound": "6\\u0005ULP-Lt"}, {"distinct_range": 0, "num_eq": 8631678864640124073, "num_range": 1000000000, "upper_bound": "Dg\\u001b\\nJvW\\u000f\\u000c"}, {"distinct_range": 0, "num_eq": 5305100677396834094, "num_range": 0, "upper_bound": "F"}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 800, "upper_bound": "L9?"}, {"distinct_range": 0, "num_eq": 4354090426257507307, "num_range": 5000000, "upper_bound": "LQZt\\u00060\\u0001"}, {"distinct_range": 7821361.89019925, "num_eq": 60000, "num_range": 20000000, "upper_bound": "Mie\\u0002"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 52202963, "row_count": 70000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7945998, "histo_buckets": [{"distinct_range": 0, "num_eq": 9015381571037415890, "num_range": 0, "upper_bound": "C"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 48438910, "row_count": 70000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10851397, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000000, "num_range": 0, "upper_bound": "\\\\x1867b0c040ca8686"}, {"distinct_range": 5182804821267704000, "num_eq": 1000000, "num_range": 5182804821267704044, "upper_bound": "\\\\x1c26a5"}, {"distinct_range": 0, "num_eq": 40, "num_range": 3446553944239413190, "upper_bound": "\\\\x1cc2f4"}, {"distinct_range": 0, "num_eq": 3782120284012116178, "num_range": 98319928542016991, "upper_bound": "\\\\x31ab7a16"}, {"distinct_range": 1260056.686870678, "num_eq": 8346532900142316301, "num_range": 3000000, "upper_bound": "\\\\x4ea01bae8fac49fe9e"}, {"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "\\\\x520931ce6087ee"}, {"distinct_range": 4952207248838295000, "num_eq": 0, "num_range": 7394517885036505398, "upper_bound": "\\\\x66b8b6"}, {"distinct_range": 88.03815215723064, "num_eq": 100000, "num_range": 300, "upper_bound": "\\\\x767ad3f0"}, {"distinct_range": 8844790264392564000, "num_eq": 2464111059425645356, "num_range": 8844790264392563992, "upper_bound": "\\\\x9618"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 70000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9654503, "histo_buckets": [{"distinct_range": 0, "num_eq": 7351114102399023757, "num_range": 0, "upper_bound": "1291618118"}, {"distinct_range": 20000000, "num_eq": 7162500728575405643, "num_range": 20000000, "upper_bound": "1841010211"}, {"distinct_range": 1878606437664039200, "num_eq": 3014176348873988174, "num_range": 1878606437664039253, "upper_bound": "3170485391"}], "histo_col_type": "REGNAMESPACE", "name": "__auto__", "null_count": 4955195, "row_count": 70000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 856991839, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 68251097, "histo_col_type": "", "name": "__auto__", "null_count": 518261548, "row_count": 900000000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 566352944, "histo_col_type": "", "name": "__auto__", "null_count": 28216284, "row_count": 900000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 572894733, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000000}, {"columns": ["col2_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 376089593, "histo_col_type": "", "name": "__auto__", "null_count": 681019045, "row_count": 900000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13146874, "histo_col_type": "", "name": "__auto__", "null_count": 683108640, "row_count": 900000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 533039517, "histo_col_type": "", "name": "__auto__", "null_count": 536916274, "row_count": 900000000}, {"columns": ["col2_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 419736699, "histo_col_type": "", "name": "__auto__", "null_count": 663458917, "row_count": 900000000}, {"columns": ["col2_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 199119850, "histo_col_type": "", "name": "__auto__", "null_count": 153537167, "row_count": 900000000}, {"columns": ["col2_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 175286877, "histo_col_type": "", "name": "__auto__", "null_count": 327921741, "row_count": 900000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 140812948, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 98183762, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46189087, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 196538371, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 900000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 804614500, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000, "num_range": 0, "upper_bound": "1978-09-25"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 50000000, "upper_bound": "1983-07-01"}, {"distinct_range": 400000, "num_eq": 172204723616504901, "num_range": 400000, "upper_bound": "1988-10-23"}, {"distinct_range": 200000, "num_eq": 700000000, "num_range": 200000, "upper_bound": "1993-02-10"}, {"distinct_range": 0, "num_eq": 414980875621359361, "num_range": 9000000, "upper_bound": "1994-02-17"}, {"distinct_range": 3433965801094467000, "num_eq": 1000, "num_range": 4077764923646961182, "upper_bound": "1996-10-20"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 10000000, "upper_bound": "5874897-12-31"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 900000000}, {"columns": ["col2_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 658894282, "histo_col_type": "", "name": "__auto__", "null_count": 522122731, "row_count": 900000000}, {"columns": ["col2_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 266197876, "histo_col_type": "", "name": "__auto__", "null_count": 257569419, "row_count": 900000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 168950396, "histo_col_type": "", "name": "__auto__", "null_count": 592813795, "row_count": 900000000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 577568481364274775, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 5440406436472636000, "num_eq": 50000000, "num_range": 7323104119853265510, "upper_bound": "393250909"}, {"distinct_range": 0, "num_eq": 80, "num_range": 0, "upper_bound": "763736624"}, {"distinct_range": 590998026807678000, "num_eq": 8898120265330853543, "num_range": 1411573853419080379, "upper_bound": "2086953388"}, {"distinct_range": 84259446.00206532, "num_eq": 3270893930994083520, "num_range": 100000000, "upper_bound": "2536507713"}, {"distinct_range": 5148630206634764000, "num_eq": 50, "num_range": 5148630206634763878, "upper_bound": "2771582357"}, {"distinct_range": 0, "

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2775166&tab=buildLog) on [master@bdff5338ca725bf1cfddf7e3f648bbf02ab42999](https://github.com/cockroachdb/cockroach/commits/bdff5338ca725bf1cfddf7e3f648bbf02ab42999):

```
        postgres_1    |     pg_ctl -D /var/lib/postgresql/data -l logfile start
        postgres_1    | 
        postgres_1    | waiting for server to start....2021-03-14 06:11:23.101 UTC [47] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-03-14 06:11:23.119 UTC [48] LOG:  database system was shut down at 2021-03-14 06:11:22 UTC
        postgres_1    | 2021-03-14 06:11:23.124 UTC [47] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2021-03-14 06:11:25.996 UTC [47] LOG:  received fast shutdown request
        postgres_1    | .2021-03-14 06:11:25.998 UTC [47] LOG:  aborting any active transactions
        postgres_1    | 2021-03-14 06:11:26.000 UTC [47] LOG:  background worker "logical replication launcher" (PID 54) exited with exit code 1
        postgres_1    | 2021-03-14 06:11:26.007 UTC [49] LOG:  shutting down
        postgres_1    | 2021-03-14 06:11:26.118 UTC [47] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-03-14 06:11:26.209 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-03-14 06:11:26.209 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-03-14 06:11:26.213 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-03-14 06:11:26.229 UTC [84] LOG:  database system was shut down at 2021-03-14 06:11:26 UTC
        postgres_1    | 2021-03-14 06:11:26.235 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (601.78s)
        test_1        |     --- FAIL: TestCompare/postgres (0.96s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.3:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61683 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2776623&tab=buildLog) on [master@e09b93fe62541c3a94f32a723778660b528a0792](https://github.com/cockroachdb/cockroach/commits/e09b93fe62541c3a94f32a723778660b528a0792):

```
        postgres_1    | 	CREATE UNIQUE INDEX ON table5 (col5_3 ASC) INCLUDE (col5_0, col5_2, col5_4);
        postgres_1    | 	CREATE UNIQUE INDEX ON table5 (col5_0 DESC);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('flqfc');
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('vi', 'jywe', 'xlmfdp');
        postgres_1    | 	CREATE TYPE rand_typ_2 AS ENUM ('rials', 'uewkxs', 'qfahwx', 'knqw');
        postgres_1    | 	CREATE TYPE rand_typ_3 AS ENUM ('ds', 's', 'ddt', 'dwjn', 'xmilsy', 'olzi');
        postgres_1    | 	CREATE TYPE rand_typ_4 AS ENUM ('zwht', 'mudw', 'aso', 'aaglq');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (604.44s)
        test_1        |     --- FAIL: TestCompare/postgres (3.22s)
        test_1        |         compare_test.go:129: CREATE TABLE table1 (col1_0 REGPROC NOT NULL, col1_1 TIMESTAMPTZ NOT NULL, col1_2 TIMESTAMPTZ, col1_3 DECIMAL NOT NULL, col1_4 "char" NULL, col1_5 BOX2D NOT NULL, PRIMARY KEY (col1_0), col1_6 TEXT AS (lower(col1_4)) VIRTUAL, col1_7 TEXT AS (lower(CAST(col1_0 AS TEXT))) STORED);
        test_1        |             CREATE INDEX ON table1 (col1_6 ASC, col1_0 DESC, col1_3 ASC, col1_7 ASC, col1_1 ASC, col1_2, col1_5 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_2, col1_5 ASC, col1_7, col1_1 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_6 DESC, col1_3 ASC, col1_5 DESC, col1_1 DESC, col1_4 DESC, col1_0 DESC, col1_7 DESC, col1_2 DESC);
        test_1        |             CREATE TABLE table2 (col2_0 TIME NOT NULL, col2_1 TIME NOT NULL, col2_2 FLOAT8 NOT NULL, col2_3 BOOL NULL, col2_4 TIMESTAMPTZ[] NULL, col2_5 CHAR NOT NULL, col2_6 TIMESTAMP NOT NULL, col2_7 BOX2D NOT NULL, col2_8 BIT(16) NOT NULL, col2_9 TIMETZ NOT NULL, col2_10 INT8 NULL, PRIMARY KEY (col2_0, col2_9, col2_7, col2_6, col2_5), col2_11 TEXT NOT NULL AS (lower(col2_5)) STORED, col2_12 TEXT NULL AS (lower(CAST(col2_3 AS TEXT))) STORED, col2_13 TEXT AS (lower(CAST(col2_0 AS TEXT))) VIRTUAL, col2_14 TEXT NULL AS (lower(CAST(col2_6 AS TEXT))) STORED, col2_15 TEXT NULL AS (lower(CAST(col2_6 AS TEXT))) VIRTUAL, col2_16 TEXT AS (lower(CAST(col2_8 AS TEXT))) VIRTUAL, col2_17 FLOAT8 NULL AS (col2_2 + (-0.9547609090805054)::FLOAT8) STORED, col2_18 FLOAT8 AS (col2_2 + 1.286890983581543::FLOAT8) VIRTUAL);
        test_1        |             CREATE INDEX ON table2 (col2_14, col2_15, col2_0 DESC, col2_5, col2_10 DESC, col2_11 DESC, col2_8 ASC, col2_1 ASC, col2_9 ASC, col2_18 DESC, col2_3 DESC, col2_16 DESC, col2_7 DESC, col2_6 DESC, col2_12);
        test_1        |             CREATE INDEX ON table2 (col2_8, col2_13 DESC, col2_16 ASC, col2_15, col2_1 ASC, col2_2 DESC, col2_3, col2_14 DESC) INCLUDE (col2_4, col2_10, col2_11, col2_17);
        test_1        |             CREATE INDEX ON table2 (col2_10, col2_17 ASC, col2_7 ASC, col2_5, col2_2 DESC, col2_15, col2_12 DESC, col2_16 DESC, col2_18 DESC, col2_13 DESC);
        test_1        |             CREATE INDEX ON table2 (col2_7 ASC, col2_9 DESC, col2_3, col2_6 ASC, col2_8, col2_16 DESC, col2_13 ASC, col2_15 DESC, col2_11 ASC, col2_14 DESC, col2_0, col2_2 ASC, col2_12, col2_1, col2_18, col2_5, col2_17 ASC);
        test_1        |             CREATE INDEX ON table2 (col2_14 ASC, col2_2 ASC, col2_11, col2_8 DESC, col2_1 ASC, col2_9 DESC, col2_17, col2_7 ASC, col2_6 DESC, col2_10 DESC, col2_5 DESC, col2_0, col2_13 ASC, col2_3, col2_16 DESC, col2_18 DESC, col2_12 DESC, col2_15 DESC);
        test_1        |             CREATE TABLE table3 (col2_0 TIME NOT NULL, col2_9 TIMETZ NOT NULL, col2_7 BOX2D NOT NULL, col2_6 TIMESTAMP NOT NULL, col2_5 CHAR NOT NULL, col3_5 INT8 NOT NULL, col3_6 REGCLASS, col3_7 OID, col3_8 REGTYPE NOT NULL, col3_9 REGNAMESPACE NULL, PRIMARY KEY (col2_0, col2_9, col2_7, col2_6, col2_5, col3_8, col3_9, col3_5, col3_6));
        test_1        |             CREATE TABLE table4 (col4_0 REGPROCEDURE NULL, col4_1 INT8 NOT NULL, col4_2 DECIMAL NULL, col4_3 INT8, col4_4 TIMESTAMP NULL, col4_5 UUID, col4_6 TIMESTAMPTZ NULL, col4_7 TEXT NULL, col4_8 BYTEA NOT NULL, PRIMARY KEY (col4_8));
        test_1        |             CREATE INDEX ON table4 (col4_3) INCLUDE (col4_1, col4_2, col4_4, col4_6, col4_7);
        test_1        |             CREATE INDEX ON table4 (col4_3, col4_7 ASC, col4_0, col4_2, col4_6) INCLUDE (col4_1);
        test_1        |             CREATE INDEX ON table4 (col4_0, col4_4) INCLUDE (col4_1, col4_2, col4_5, col4_7);
        test_1        |             CREATE TABLE table5 (col5_0 NAME, col5_1 INT8 NOT NULL, col5_2 CHAR NOT NULL, col5_3 INT8 NOT NULL, col5_4 "char", PRIMARY KEY (col5_1, col5_3), col5_5 TEXT NULL AS (lower(col5_2)) VIRTUAL, col5_6 TEXT NOT NULL AS (lower(col5_0)) VIRTUAL, col5_7 INT8 NULL AS (col5_3 + col5_1) VIRTUAL);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_6 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_6 DESC, col5_7 ASC, col5_0, col5_2 DESC);
        test_1        |             CREATE INDEX ON table5 (col5_4, col5_7, col5_3 DESC, col5_5 DESC, col5_6 DESC, col5_1);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_5, col5_1 DESC, col5_3, col5_0, col5_2 DESC, col5_4 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_3 ASC) INCLUDE (col5_0, col5_2, col5_4);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_0 DESC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('flqfc');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('vi', 'jywe', 'xlmfdp');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('rials', 'uewkxs', 'qfahwx', 'knqw');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('ds', 's', 'ddt', 'dwjn', 'xmilsy', 'olzi');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('zwht', 'mudw', 'aso', 'aaglq');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: syntax error at or near "AS" (SQLSTATE 42601)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61683 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2780697&tab=buildLog) on [master@e9387a6e5dfdad71c74ccd0a07c907632613fa3e](https://github.com/cockroachdb/cockroach/commits/e9387a6e5dfdad71c74ccd0a07c907632613fa3e):

```
        postgres_1    | 	CREATE UNIQUE INDEX ON table1 (col1_9 DESC, col1_3 ASC, col1_4 ASC, col1_0 DESC, col1_8) INCLUDE (col1_5, col1_6, col1_10);
        postgres_1    | 	CREATE UNIQUE INDEX ON table1 (col1_3);
        postgres_1    | 	CREATE TABLE table2 (col2_0 INTERVAL NOT NULL, PRIMARY KEY (col2_0));
        postgres_1    | 	CREATE TABLE table3 (col3_0 TEXT, col3_1 BOOL NOT NULL, col3_2 BYTEA NOT NULL, col3_3 DATE, col3_4 REGTYPE, col3_5 TIME[] NOT NULL, col3_6 BIT(32) NULL, col3_7 TEXT NOT NULL AS (lower(CAST(col3_1 AS TEXT))) VIRTUAL, col3_8 TEXT NOT NULL AS (lower(CAST(col3_3 AS TEXT))) STORED, col3_9 TEXT AS (lower(CAST(col3_4 AS TEXT))) STORED);
        postgres_1    | 	CREATE INVERTED INDEX ON table3 (col3_0 ASC, col3_2 DESC, col3_9, col3_7 ASC, col3_6 DESC, col3_8 ASC, col3_3 DESC, col3_5 ASC);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_0 DESC, col3_9 DESC, col3_1) INCLUDE (col3_8);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_2 ASC, col3_0 ASC, col3_8 DESC, col3_4 DESC, col3_9 ASC);
        postgres_1    | 	CREATE TABLE table4 (col4_0 "char" NOT NULL, col4_1 GEOMETRY NULL, PRIMARY KEY (col4_0));
        postgres_1    | 	CREATE UNIQUE INDEX ON table4 (col4_0 DESC);
        postgres_1    | 	CREATE INVERTED INDEX ON table4 (col4_0 DESC, col4_1 ASC);
        postgres_1    | 	CREATE UNIQUE INDEX ON table4 (col4_0 DESC);
        postgres_1    | 	CREATE INVERTED INDEX ON table4 (col4_1);
        postgres_1    | 	CREATE INVERTED INDEX ON table4 (col4_1);
        postgres_1    | 	CREATE INDEX ON table4 (col4_0 DESC) INCLUDE (col4_1);
        postgres_1    | 	ALTER TABLE table4 ADD FOREIGN KEY (col4_0) REFERENCES table1 (col1_3) ON UPDATE CASCADE;
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('xaa', 'sckm', 'fxycic', 'fqcdur');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (603.25s)
        test_1        |     --- FAIL: TestCompare/postgres (2.25s)
        test_1        |         compare_test.go:129: CREATE TABLE table1 (col1_0 UUID NOT NULL, col1_1 TIME NULL, col1_2 BIT(8) NOT NULL, col1_3 CHAR NOT NULL, col1_4 REGTYPE NOT NULL, col1_5 GEOGRAPHY NOT NULL, col1_6 REGCLASS NOT NULL, col1_7 INET NOT NULL, col1_8 TEXT NOT NULL AS (lower(CAST(col1_4 AS TEXT))) STORED, col1_9 TEXT NOT NULL AS (lower(col1_3)) STORED, col1_10 TEXT NULL AS (lower(CAST(col1_0 AS TEXT))) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_8, col1_6, col1_7, col1_9 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_9 DESC, col1_3 ASC, col1_4 ASC, col1_0 DESC, col1_8) INCLUDE (col1_5, col1_6, col1_10);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_3);
        test_1        |             CREATE TABLE table2 (col2_0 INTERVAL NOT NULL, PRIMARY KEY (col2_0));
        test_1        |             CREATE TABLE table3 (col3_0 TEXT, col3_1 BOOL NOT NULL, col3_2 BYTEA NOT NULL, col3_3 DATE, col3_4 REGTYPE, col3_5 TIME[] NOT NULL, col3_6 BIT(32) NULL, col3_7 TEXT NOT NULL AS (lower(CAST(col3_1 AS TEXT))) VIRTUAL, col3_8 TEXT NOT NULL AS (lower(CAST(col3_3 AS TEXT))) STORED, col3_9 TEXT AS (lower(CAST(col3_4 AS TEXT))) STORED);
        test_1        |             CREATE INVERTED INDEX ON table3 (col3_0 ASC, col3_2 DESC, col3_9, col3_7 ASC, col3_6 DESC, col3_8 ASC, col3_3 DESC, col3_5 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 DESC, col3_9 DESC, col3_1) INCLUDE (col3_8);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_2 ASC, col3_0 ASC, col3_8 DESC, col3_4 DESC, col3_9 ASC);
        test_1        |             CREATE TABLE table4 (col4_0 "char" NOT NULL, col4_1 GEOMETRY NULL, PRIMARY KEY (col4_0));
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_0 DESC);
        test_1        |             CREATE INVERTED INDEX ON table4 (col4_0 DESC, col4_1 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_0 DESC);
        test_1        |             CREATE INVERTED INDEX ON table4 (col4_1);
        test_1        |             CREATE INVERTED INDEX ON table4 (col4_1);
        test_1        |             CREATE INDEX ON table4 (col4_0 DESC) INCLUDE (col4_1);
        test_1        |             ALTER TABLE table4 ADD FOREIGN KEY (col4_0) REFERENCES table1 (col1_3) ON UPDATE CASCADE;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('xaa', 'sckm', 'fxycic', 'fqcdur');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: syntax error at or near "AS" (SQLSTATE 42601)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61683 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2785151&tab=buildLog) on [master@597e4a8c487e3c23d64885563d608a692b59055c](https://github.com/cockroachdb/cockroach/commits/597e4a8c487e3c23d64885563d608a692b59055c):

```
        postgres_1    | 2021-03-17 05:13:25.095 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2021-03-17 05:13:25.099 UTC [50] LOG:  shutting down
        postgres_1    | 2021-03-17 05:13:25.217 UTC [48] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-03-17 05:13:25.305 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-03-17 05:13:25.305 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-03-17 05:13:25.309 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-03-17 05:13:25.326 UTC [85] LOG:  database system was shut down at 2021-03-17 05:13:25 UTC
        postgres_1    | 2021-03-17 05:13:25.333 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (600.99s)
        test_1        |     --- FAIL: TestCompare/postgres (0.32s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 TIMESTAMPTZ NOT NULL, col1_1 FLOAT8 NOT NULL, col1_2 BYTES NOT NULL, col1_3 INET NOT NULL, col1_4 UUID NOT NULL, col1_5 STRING NOT NULL, col1_6 TIMETZ NOT NULL, col1_7 TIMESTAMP NOT NULL, col1_8 DATE NOT NULL, col1_9 FLOAT8 NOT NULL, PRIMARY KEY (col1_7, col1_2, col1_1, col1_4, col1_0, col1_8, col1_6, col1_3, col1_9), col1_10 STRING NOT NULL AS (IF(col1_0 IS NULL, NULL, e'\x0b*QIJ':::STRING)) STORED, col1_11 FLOAT8 NOT NULL AS (col1_9 + col1_1) VIRTUAL, col1_12 FLOAT8 AS (col1_9 + col1_1) VIRTUAL, col1_13 FLOAT8 AS (col1_9 + col1_1) STORED, col1_14 FLOAT8 AS (col1_1 + col1_9) STORED, col1_15 FLOAT8 AS (col1_9 + (-0.8331368931048742):::FLOAT8) VIRTUAL, col1_16 FLOAT8 NULL AS (col1_9 + col1_1) STORED, col1_17 FLOAT8 AS (col1_9 + (-0.23483122447101767):::FLOAT8) VIRTUAL, col1_18 FLOAT8 NULL AS (col1_9 + col1_1) VIRTUAL, FAMILY (col1_4, col1_9, col1_0), FAMILY (col1_6), FAMILY (col1_2, col1_7), FAMILY (col1_13), FAMILY (col1_3), FAMILY (col1_16), FAMILY (col1_1, col1_8), FAMILY (col1_5, col1_14, col1_10));
        test_1        |             ALTER TABLE table1 ALTER COLUMN col1_5 SET DATA TYPE STRING;
        test_1        |             CREATE INVERTED INDEX ON table1 (col1_15 ASC, col1_7, col1_12 DESC, col1_14 ASC, col1_1 DESC, col1_16 DESC, col1_9 ASC, col1_13 ASC, col1_0 ASC, col1_6 ASC, col1_18 DESC, col1_10, col1_3 DESC, col1_17 DESC, col1_5);
        test_1        |             CREATE INDEX ON table1 (col1_14 ASC, col1_4 DESC, col1_15 ASC, col1_13 ASC, col1_0 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_14 ASC, col1_2, col1_13 ASC, col1_15, col1_3, col1_16);
        test_1        |             CREATE TABLE table2 (col2_0 "char" NULL, col2_1 REGPROCEDURE NULL, col2_2 BOOL NOT NULL, col2_3 REGPROC NOT NULL, col2_4 TIMESTAMP NULL, col2_5 FLOAT8 NOT NULL, col2_6 TIMETZ NOT NULL, PRIMARY KEY (col2_6), col2_7 FLOAT8 AS (col2_5 + 0.8667200053324464:::FLOAT8) STORED, col2_8 STRING NOT NULL AS (lower(CAST(col2_3 AS STRING))) STORED, col2_9 STRING NULL AS (lower(CAST(col2_4 AS STRING))) STORED, col2_10 STRING NULL AS (lower(CAST(col2_6 AS STRING))) VIRTUAL, col2_11 STRING NULL AS (lower(CAST(col2_4 AS STRING))) VIRTUAL, FAMILY (col2_0, col2_2, col2_1), FAMILY (col2_8, col2_6, col2_7), FAMILY (col2_4), FAMILY (col2_5, col2_3), FAMILY (col2_9));
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_11 ASC, col2_10 ASC, col2_0 DESC, col2_1, col2_5 DESC, col2_3 ASC, col2_9 DESC) STORING (col2_2, col2_4, col2_7, col2_8);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_2 DESC, col2_5 DESC, col2_3, col2_10 DESC, col2_7 ASC, col2_1, col2_11 DESC, col2_8, col2_9 DESC, col2_0 ASC);
        test_1        |             CREATE TABLE table3 (col3_0 INTERVAL NOT NULL, col3_1 INTERVAL NOT NULL, col3_2 INET NOT NULL, col3_3 REGPROC NOT NULL, col3_4 BIT(17) NOT NULL, col3_5 DECIMAL NOT NULL, col3_6 TIMETZ NOT NULL, col3_7 BIT(15) NULL, col3_8 CHAR NOT NULL, col3_9 BYTES NOT NULL, col3_10 NAME NOT NULL, col3_11 BYTES NOT NULL, PRIMARY KEY (col3_4, col3_8, col3_5, col3_0, col3_6, col3_10, col3_11, col3_3, col3_9), col3_12 STRING NULL AS (lower(CAST(col3_0 AS STRING))) STORED, col3_13 STRING NOT NULL AS (lower(col3_10)) STORED, FAMILY (col3_5), FAMILY (col3_12), FAMILY (col3_0), FAMILY (col3_8), FAMILY (col3_1, col3_6, col3_4), FAMILY (col3_13), FAMILY (col3_9, col3_3, col3_11), FAMILY (col3_2), FAMILY (col3_7), FAMILY (col3_10));
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 DESC, col3_10 ASC, col3_2 ASC, col3_12 ASC);
        test_1        |             CREATE INDEX ON table3 (col3_12 DESC, col3_1 DESC, col3_4);
        test_1        |             CREATE INDEX ON table3 (col3_6 DESC, col3_3, col3_12 ASC, col3_9, col3_4, col3_13, col3_11 DESC, col3_1, col3_5, col3_8 ASC, col3_7 ASC, col3_10 DESC, col3_0 DESC, col3_2 ASC);
        test_1        |             CREATE INDEX ON table3 (col3_12 ASC, col3_7 DESC, col3_1);
        test_1        |             CREATE INDEX ON table3 (col3_12, col3_4 DESC, col3_6 ASC, col3_13 ASC, col3_5 DESC, col3_11, col3_9 ASC, col3_0) STORING (col3_1);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS '[{"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3626882857286916868, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3867142633530577182}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2554593311891272156, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3867142633530577182}, {"columns": ["col1_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1275873241161957594, "histo_col_type": "", "name": "__auto__", "null_count": 1589605813412008000, "row_count": 3867142633530577182}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 487236262664608244, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3867142633530577182}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 560143325671087205, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3867142633530577182}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3439003722995067534, "histo_col_type": "TIMESTAMP", "name": "__auto__", "null_count": 0, "row_count": 3867142633530577182}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1512597692871317143, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3867142633530577182}, {"columns": ["col1_17"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 580449775542332182, "histo_col_type": "", "name": "__auto__", "null_count": 1837906489451731034, "row_count": 3867142633530577182}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1967882600191659270, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3867142633530577182}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2425886152443813937, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3867142633530577182}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3647750538879095509, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3867142633530577182}, {"columns": ["col1_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2814876644435195555, "histo_col_type": "", "name": "__auto__", "null_count": 609082791380960091, "row_count": 3867142633530577182}, {"columns": ["col1_18"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1570071840881923429, "histo_col_type": "", "name": "__auto__", "null_count": 1592228881158950145, "row_count": 3867142633530577182}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3212929181585534042, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3867142633530577182}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1010553774401060475, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3867142633530577182}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1010705117898190563, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3867142633530577182}, {"columns": ["col1_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 436864498473504888, "histo_buckets": [{"distinct_range": 0, "num_eq": 1358713436508140043, "num_range": 0, "upper_bound": "-1.2528695325301444"}, {"distinct_range": 0, "num_eq": 1000000000, "num_range": 2000, "upper_bound": "-0.9162843587866182"}, {"distinct_range": 120472515444841330, "num_eq": 747833636893653298, "num_range": 122326910249041907, "upper_bound": "-0.8770105865313984"}, {"distinct_range": 0, "num_eq": 500, "num_range": 0, "upper_bound": "-0.6194571906574261"}, {"distinct_range": 5619580108612856000, "num_eq": 935692091602572105, "num_range": 5619580108612856106, "upper_bound": "1.401298464324817e-45"}, {"distinct_range": 9169563959563322000, "num_eq": 60000000000, "num_range": 9169563959563322145, "upper_bound": "0.5780079106866727"}, {"distinct_range": 5583984408123052, "num_eq": 5529008986010908623, "num_range": 11426778313993257, "upper_bound": "0.6830746924669912"}, {"distinct_range": 7674682411059880000, "num_eq": 30000, "num_range": 7674682411059880371, "upper_bound": "0.7237534003139693"}, {"distinct_range": 900000000, "num_eq": 100, "num_range": 900000000, "upper_bound": "0.8973288272333954"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 1979953053030031205, "row_count": 3867142633530577182}, {"columns": ["col1_15"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1252710057614954309, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-1.3070111315737623"}, {"distinct_range": 0, "num_eq": 1579860588556861053, "num_range": 7477475251096205374, "upper_bound": "-0.4621356852388848"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 1536507744252288725, "row_count": 3867142633530577182}, {"columns": ["col1_16"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1855876402687401670, "histo_col_type": "", "name": "__auto__", "null_count": 3158145061417084343, "row_count": 3867142633530577182}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22409014, "histo_col_type": "", "name": "__auto__", "null_count": 95420155, "row_count": 100000000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 57557316, "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17487615, "histo_col_type": "", "name": "__auto__", "null_count": 69147795, "row_count": 100000000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 50357837, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 33758246, "histo_col_type": "", "name": "__auto__", "null_count": 13254360, "row_count": 100000000}, {"columns": ["col2_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5516682, "histo_col_type": "", "name": "__auto__", "null_count": 26716872, "row_count": 100000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 38337162, "histo_col_type": "", "name": "__auto__", "null_count": 50187746, "row_count": 100000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 64950381, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 30000000, "num_eq": 112206248272518005, "num_range": 30000000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23140153, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}, {"columns": ["col2_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5009037, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "\\u0005\\u0013\x7f"}, {"distinct_range": 0, "num_eq": 2060026361855169867, "num_range": 900000000, "upper_bound": "4&wjv"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 30855739, "row_count": 100000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 79988875, "histo_col_type": "", "name": "__auto__", "null_count": 31952065, "row_count": 100000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36912089, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 41388840621, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34152026216, "histo_buckets": [{"distinct_range": 0, "num_eq": 2061447953858448505, "num_range": 0, "upper_bound": ""}, {"distinct_range": 442.83181712066647, "num_eq": 280238498561841815, "num_range": 900, "upper_bound": "\\u0002\\u0002\\u0017ipV\\u0012\\u0004"}, {"distinct_range": 5531767220401530000, "num_eq": 900000000, "num_range": 5531767220401530011, "upper_bound": "\\u000b\\u0002)WO"}, {"distinct_range": 20380.942532576042, "num_eq": 8798524511599196212, "num_range": 60000, "upper_bound": "0A89"}, {"distinct_range": 6876607761520758000, "num_eq": 6000000000, "num_range": 8517212218902520505, "upper_bound": "1"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 60564904746, "row_count": 90000000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14445401848, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": "00101101110110001"}, {"distinct_range": 0, "num_eq": 0, "num_range": 7000, "upper_bound": "01111001100010101"}, {"distinct_range": 0, "num_eq": 5854613774065586871, "num_range": 80000, "upper_bound": "01111100100001000"}, {"distinct_range": 0, "num_eq": 4704847331768170351, "num_range": 4075137710096632927, "upper_bound": "10100101101100000"}, {"distinct_range": 0, "num_eq": 6000000, "num_range": 8319778440764356310, "upper_bound": "11000000001000110"}], "histo_col_type": "BIT(17)", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29721855093, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000000, "num_range": 0, "upper_bound": "21 years 4 mons 80 days 15:55:16.568501"}, {"distinct_range": 9000000, "num_eq": 3000000000, "num_range": 9000000, "upper_bound": "24 years 10 mons 746 days 21:24:07.732894"}, {"distinct_range": 73.81662356600928, "num_eq": 500, "num_range": 600, "upper_bound": "290 years"}], "histo_col_type": "INTERVAL", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 55972865700, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53442627129, "histo_buckets": [{"distinct_range": 0, "num_eq": 7976168386776267246, "num_range": 0, "upper_bound": "22:33:30.138401+02:28:00"}], "histo_col_type": "TIMETZ", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 65048672730, "histo_col_type": "", "name": "__auto__", "null_count": 20614858180, "row_count": 90000000000}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 69329335633, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 48726921555, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45287102666, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 40732787535, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 64445827881, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5247008145, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 61159947126, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 90000000000}]':::JSONB;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('s', 'aerjtv');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('msieti', 'rxwjla', 'zurgky');
        test_1        |             
        test_1        |         compare_test.go:130: cockroach1: exec: ERROR: column col1_5 of type string is not allowed as the last column in an inverted index (SQLSTATE 0A000)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61683 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2788997&tab=buildLog) on [master@36dea46f8cedf42df31b57dd70db7e0f1fd7a453](https://github.com/cockroachdb/cockroach/commits/36dea46f8cedf42df31b57dd70db7e0f1fd7a453):

```
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-03-18 05:13:04.966 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-03-18 05:13:04.966 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-03-18 05:13:04.971 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-03-18 05:13:04.989 UTC [85] LOG:  database system was shut down at 2021-03-18 05:13:04 UTC
        postgres_1    | 2021-03-18 05:13:04.994 UTC [1] LOG:  database system is ready to accept connections
        postgres_1    | 2021-03-18 05:23:04.098 UTC [112] ERROR:  syntax error at or near "AS" at character 186
        postgres_1    | 2021-03-18 05:23:04.098 UTC [112] STATEMENT:  CREATE TABLE table1 (col1_0 REGCLASS NULL, col1_1 TIMETZ, col1_2 REGPROCEDURE NOT NULL, col1_3 BOOL, col1_4 INTERVAL NOT NULL, col1_5 "char" NULL, col1_6 REGPROCEDURE, col1_7 TEXT NULL AS (lower(col1_5)) STORED);
        postgres_1    | 	CREATE TABLE table2 (col2_0 REGTYPE NULL, col2_1 TIME NOT NULL, col2_2 TEXT NOT NULL, col2_3 TEXT, col2_4 FLOAT8, col2_5 VARCHAR NOT NULL, col2_6 NAME NOT NULL, col2_7 NAME NOT NULL, col2_8 INT8, col2_9 DECIMAL NOT NULL, col2_10 REGCLASS NOT NULL, col2_11 NAME NOT NULL, col2_12 TEXT AS (lower(col2_6)) STORED);
        postgres_1    | 	ALTER TABLE table2 ALTER COLUMN col2_2 SET DATA TYPE TEXT;
        postgres_1    | 	CREATE UNIQUE INDEX ON table2 (col2_6 DESC, col2_10 ASC, col2_8, col2_3 DESC, col2_0 DESC);
        postgres_1    | 	CREATE INDEX ON table2 (col2_8, col2_3, col2_9, col2_0 DESC, col2_10, col2_6 DESC, col2_11 DESC);
        postgres_1    | 	CREATE TABLE table3 (col3_0 BYTEA, col3_1 BIT(11), col3_2 TIME NOT NULL, col3_3 TEXT NOT NULL, col3_4 TIMESTAMPTZ NOT NULL, col3_5 INTERVAL NULL, col3_6 "char" NOT NULL, col3_7 OID NOT NULL, PRIMARY KEY (col3_2, col3_3, col3_6, col3_4), col3_8 TEXT NULL AS (IF(col3_0 IS NULL, ''::TEXT, e'+\x14(?'::TEXT)) STORED, col3_9 TEXT NULL AS (lower(CAST(col3_2 AS TEXT))) STORED, col3_10 TEXT AS (lower(CAST(col3_1 AS TEXT))) STORED, col3_11 TEXT NULL AS (lower(CAST(col3_7 AS TEXT))) STORED);
        postgres_1    | 	CREATE INDEX ON table3 (col3_5 ASC, col3_10, col3_3 DESC) INCLUDE (col3_1, col3_8, col3_11);
        postgres_1    | 	CREATE INDEX ON table3 (col3_6 ASC, col3_5 ASC, col3_11) INCLUDE (col3_1, col3_7, col3_8);
        postgres_1    | 	CREATE INDEX ON table3 (col3_2 ASC, col3_9, col3_4 ASC, col3_11 ASC, col3_6 DESC, col3_1 ASC, col3_0, col3_8 ASC, col3_5);
        postgres_1    | 	CREATE INDEX ON table3 (col3_8 DESC, col3_7 ASC, col3_6 ASC, col3_2 ASC, col3_11 DESC, col3_0 ASC, col3_10 ASC);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('kcnq', 'bhrsbj', 'am', 'aczel');
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('cy', 'yicp', 'xuojdf', 'giy', 'tf');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (602.39s)
        test_1        |     --- FAIL: TestCompare/postgres (1.48s)
        test_1        |         compare_test.go:129: CREATE TABLE table1 (col1_0 REGCLASS NULL, col1_1 TIMETZ, col1_2 REGPROCEDURE NOT NULL, col1_3 BOOL, col1_4 INTERVAL NOT NULL, col1_5 "char" NULL, col1_6 REGPROCEDURE, col1_7 TEXT NULL AS (lower(col1_5)) STORED);
        test_1        |             CREATE TABLE table2 (col2_0 REGTYPE NULL, col2_1 TIME NOT NULL, col2_2 TEXT NOT NULL, col2_3 TEXT, col2_4 FLOAT8, col2_5 VARCHAR NOT NULL, col2_6 NAME NOT NULL, col2_7 NAME NOT NULL, col2_8 INT8, col2_9 DECIMAL NOT NULL, col2_10 REGCLASS NOT NULL, col2_11 NAME NOT NULL, col2_12 TEXT AS (lower(col2_6)) STORED);
        test_1        |             ALTER TABLE table2 ALTER COLUMN col2_2 SET DATA TYPE TEXT;
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_6 DESC, col2_10 ASC, col2_8, col2_3 DESC, col2_0 DESC);
        test_1        |             CREATE INDEX ON table2 (col2_8, col2_3, col2_9, col2_0 DESC, col2_10, col2_6 DESC, col2_11 DESC);
        test_1        |             CREATE TABLE table3 (col3_0 BYTEA, col3_1 BIT(11), col3_2 TIME NOT NULL, col3_3 TEXT NOT NULL, col3_4 TIMESTAMPTZ NOT NULL, col3_5 INTERVAL NULL, col3_6 "char" NOT NULL, col3_7 OID NOT NULL, PRIMARY KEY (col3_2, col3_3, col3_6, col3_4), col3_8 TEXT NULL AS (IF(col3_0 IS NULL, ''::TEXT, e'+\x14(?'::TEXT)) STORED, col3_9 TEXT NULL AS (lower(CAST(col3_2 AS TEXT))) STORED, col3_10 TEXT AS (lower(CAST(col3_1 AS TEXT))) STORED, col3_11 TEXT NULL AS (lower(CAST(col3_7 AS TEXT))) STORED);
        test_1        |             CREATE INDEX ON table3 (col3_5 ASC, col3_10, col3_3 DESC) INCLUDE (col3_1, col3_8, col3_11);
        test_1        |             CREATE INDEX ON table3 (col3_6 ASC, col3_5 ASC, col3_11) INCLUDE (col3_1, col3_7, col3_8);
        test_1        |             CREATE INDEX ON table3 (col3_2 ASC, col3_9, col3_4 ASC, col3_11 ASC, col3_6 DESC, col3_1 ASC, col3_0, col3_8 ASC, col3_5);
        test_1        |             CREATE INDEX ON table3 (col3_8 DESC, col3_7 ASC, col3_6 ASC, col3_2 ASC, col3_11 DESC, col3_0 ASC, col3_10 ASC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('kcnq', 'bhrsbj', 'am', 'aczel');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('cy', 'yicp', 'xuojdf', 'giy', 'tf');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: syntax error at or near "AS" (SQLSTATE 42601)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61683 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2792394&tab=buildLog) on [master@ee9f47b9ec9476a693464e2dcd09a01bf9d39ad2](https://github.com/cockroachdb/cockroach/commits/ee9f47b9ec9476a693464e2dcd09a01bf9d39ad2):

```
        postgres_1    | WARNING: enabling "trust" authentication for local connections
        postgres_1    | You can change this by editing pg_hba.conf or using the option -A, or
        postgres_1    | --auth-local and --auth-host, the next time you run initdb.
        postgres_1    | waiting for server to start....2021-03-19 05:13:02.813 UTC [49] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-03-19 05:13:02.832 UTC [50] LOG:  database system was shut down at 2021-03-19 05:13:02 UTC
        postgres_1    | 2021-03-19 05:13:02.837 UTC [49] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        test_1        | --- FAIL: TestCompare (1.02s)
        test_1        |     --- FAIL: TestCompare/postgres (0.31s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.3:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.71s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 BOX2D NOT NULL, col1_1 REGCLASS NOT NULL, col1_2 OID NOT NULL, col1_3 REGCLASS NOT NULL, col1_4 FLOAT4 NOT NULL, col1_5 REGPROCEDURE, col1_6 "char", PRIMARY KEY (col1_2, col1_3, col1_4 DESC, col1_0 ASC, col1_1 ASC), col1_7 STRING NOT NULL AS (lower(CAST(col1_5 AS STRING))) VIRTUAL, col1_8 STRING NULL AS (lower(CAST(col1_0 AS STRING))) STORED, col1_9 STRING NOT NULL AS (lower(CAST(col1_1 AS STRING))) VIRTUAL, col1_10 STRING NOT NULL AS (lower(CAST(col1_1 AS STRING))) STORED, col1_11 STRING NOT NULL AS (lower(CAST(col1_0 AS STRING))) VIRTUAL, UNIQUE (col1_10 DESC, col1_6 DESC, col1_2, col1_11 DESC, col1_1 DESC, col1_4, col1_8 ASC, col1_3 DESC, col1_0 DESC, col1_7 DESC, col1_9 DESC, col1_5 DESC) WHERE ((((((table1.col1_8 < e'\'') OR (table1.col1_4 > +Inf)) OR (table1.col1_6 >= 'X')) AND (table1.col1_9 < e'\U00002603')) OR (table1.col1_11 < '')) OR (table1.col1_7 != '"')) OR (table1.col1_10 <= ''), UNIQUE (col1_11 DESC, col1_10, col1_4 ASC, col1_6 DESC, col1_5 DESC), INDEX (col1_7 ASC, col1_9, col1_3 ASC, col1_0 DESC) STORING (col1_5, col1_8), INDEX (col1_11, col1_2 DESC, col1_8 DESC, col1_6 DESC, col1_3 ASC, col1_10), UNIQUE (col1_11 ASC, col1_1 ASC, col1_10, col1_9, col1_5 DESC, col1_7 ASC, col1_6, col1_8, col1_4, col1_3) WHERE ((((((table1.col1_6 > '') OR (table1.col1_4 > 3.4028234663852886e+38)) OR (table1.col1_9 < e'\U00002603')) OR (table1.col1_7 <= e'\'')) AND (table1.col1_11 = '')) AND (table1.col1_8 >= '"')) OR (table1.col1_10 > e'\U00002603'), INDEX (col1_2 ASC, col1_4 DESC, col1_10 ASC, col1_11, col1_0, col1_1 ASC, col1_8 ASC, col1_5 ASC, col1_6 DESC, col1_7 ASC, col1_9 ASC) WHERE ((((((table1.col1_11 != e'\U00002603') AND (table1.col1_8 = e'\x00')) OR (table1.col1_9 != '"')) AND (table1.col1_6 != '')) AND (table1.col1_4 >= -Inf)) OR (table1.col1_10 = e'\x00')) AND (table1.col1_7 = e'\U00002603'), INDEX (col1_2 DESC, col1_10, col1_4, col1_6, col1_5, col1_3 DESC, col1_7 ASC, col1_11, col1_9 DESC, col1_1 DESC, col1_0) STORING (col1_8), FAMILY (col1_8, col1_10), FAMILY (col1_6, col1_2), FAMILY (col1_4), FAMILY (col1_0, col1_1, col1_3, col1_5));
        test_1        |             CREATE TABLE table2 (col2_0 CHAR[], col2_1 DATE NOT NULL, col2_2 GEOGRAPHY, col2_3 REGCLASS NOT NULL, col2_4 DATE NULL, col2_5 BOX2D NOT NULL, col2_6 INT4 NULL, col2_7 OID, col2_8 TIME, col2_9 DATE NULL, INVERTED INDEX (col2_2), INVERTED INDEX (col2_8 DESC, col2_3, col2_7 ASC, col2_1 DESC, col2_6 ASC, col2_5 ASC, col2_4, col2_9 ASC, col2_0) WHERE ((((table2.col2_8 = '24:00:00':::TIME) OR (table2.col2_1 <= 'infinity':::DATE)) AND (table2.col2_9 <= '5874897-12-31':::DATE)) OR (table2.col2_6 = 32767:::INT8)) AND (table2.col2_4 > '1970-01-01':::DATE), UNIQUE (col2_3) STORING (col2_1, col2_4, col2_6, col2_7) WHERE ((((table2.col2_9 <= '1970-01-01') AND (table2.col2_6 != -128)) AND (table2.col2_4 >= '4714-11-24 BC')) AND (table2.col2_1 = '1970-01-01')) AND (table2.col2_8 >= '24:00:00'), UNIQUE (col2_3) WHERE ((((table2.col2_4 >= '-infinity') OR (table2.col2_8 <= '24:00:00')) AND (table2.col2_6 >= 2147483647)) OR (table2.col2_9 < 'infinity')) OR (table2.col2_1 >= '1970-01-01'), FAMILY (col2_6), FAMILY (col2_9), FAMILY (col2_7, col2_4, col2_2, col2_5, col2_1), FAMILY (col2_8), FAMILY (col2_0, col2_3));
        test_1        |             CREATE TABLE table3 (col3_0 VARCHAR NOT NULL, col3_1 TIME NOT NULL, col3_2 UUID[], col3_3 TIME, col3_4 STRING NULL AS (lower(CAST(col3_1 AS STRING))) STORED, col3_5 STRING AS (lower(CAST(col3_3 AS STRING))) STORED, INVERTED INDEX (col3_5 ASC, col3_3 ASC, col3_4, col3_0, col3_1, col3_2 ASC) WHERE (table3.col3_0 <= 'X':::STRING) AND (table3.col3_3 <= '24:00:00':::TIME), INDEX (col3_3) STORING (col3_1), FAMILY (col3_3), FAMILY (col3_2, col3_0), FAMILY (col3_1, col3_5), FAMILY (col3_4));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19078, "histo_col_type": "", "name": "__auto__", "null_count": 149991, "row_count": 600000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 472118, "histo_col_type": "", "name": "__auto__", "null_count": 536391, "row_count": 600000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 66861, "histo_col_type": "", "name": "__auto__", "null_count": 465149, "row_count": 600000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 565575, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 562565, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 2265578918805300200, "num_eq": 2984102263896677597, "num_range": 2265578918805300108, "upper_bound": "\\u000en[\\u0018\\u001f@H\\u0005"}, {"distinct_range": 0, "num_eq": 6780185593765722631, "num_range": 6487000931496483476, "upper_bound": ",W\\u0019*\\u0015S"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 207642, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 222040, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 159879, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 556536, "histo_buckets": [{"distinct_range": 0, "num_eq": 30, "num_range": 0, "upper_bound": "\\u001b\x7f"}, {"distinct_range": 6263494951067086000, "num_eq": 4694000054427500801, "num_range": 6263494951067086142, "upper_bound": "\'"}, {"distinct_range": 1464160186809431800, "num_eq": 59948309003655099, "num_range": 1520493058531100576, "upper_bound": "/\\u0010zp-` $d"}, {"distinct_range": 10, "num_eq": 7000000000, "num_range": 10, "upper_bound": "<!\\u0013y76"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 506208, "histo_buckets": [{"distinct_range": 0, "num_eq": 4009220576292888675, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 3634296397011008598, "num_range": 1000, "upper_bound": "377987580"}, {"distinct_range": 0, "num_eq": 3000000, "num_range": 8000000, "upper_bound": "786645204"}, {"distinct_range": 784.7955094356664, "num_eq": 6527919621397994432, "num_range": 800, "upper_bound": "977084767"}, {"distinct_range": 80000000, "num_eq": 900, "num_range": 80000000, "upper_bound": "1725320838"}, {"distinct_range": 217536322596459230, "num_eq": 39084350417310786, "num_range": 2246184325667565663, "upper_bound": "4201002056"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 546853, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 600000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 300928, "histo_buckets": [{"distinct_range": 0, "num_eq": 1304066729870354027, "num_range": 0, "upper_bound": "UQV\\u001em"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 600000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17281, "histo_col_type": "", "name": "__auto__", "null_count": 28398, "row_count": 30000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10194, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "0106000020E610000000000000"}, {"distinct_range": 60, "num_eq": 5336222085421615313, "num_range": 60, "upper_bound": "0102000020E61000000300000080ADF8A780345340D1954CDD638F43C004653FF1D3545A409AC54DDFDD6054400D47AFE04DFC62C01414B6A692F35040"}], "histo_col_type": "GEOGRAPHY", "name": "__auto__", "null_count": 21115, "row_count": 30000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13278, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 17587, "histo_col_type": "", "name": "__auto__", "null_count": 27358, "row_count": 30000}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 29391, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4929, "histo_col_type": "", "name": "__auto__", "null_count": 6574, "row_count": 30000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16820, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 30000}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11031, "histo_col_type": "", "name": "__auto__", "null_count": 6592, "row_count": 30000}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25025, "histo_buckets": [{"distinct_range": 0, "num_eq": 1539572177063022560, "num_range": 0, "upper_bound": "08:08:39.182473"}, {"distinct_range": 0, "num_eq": 5392137546883372536, "num_range": 8672932037159110640, "upper_bound": "10:33:10.407106"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 700, "upper_bound": "12:11:32.250632"}, {"distinct_range": 4219820255037962000, "num_eq": 400, "num_range": 4219820255037962486, "upper_bound": "15:22:10.399601"}, {"distinct_range": 1419725080620183800, "num_eq": 0, "num_range": 1656808656563664750, "upper_bound": "15:45:11.563691"}, {"distinct_range": 2202002721756859600, "num_eq": 1000000000, "num_range": 2202002721756859585, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 2133, "row_count": 30000}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25691, "histo_col_type": "", "name": "__auto__", "null_count": 25468, "row_count": 30000}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 951559113678302862, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3915711196905719420}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2911444813526790820, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 3915711196905719420}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1946489410742031882, "histo_col_type": "", "name": "__auto__", "null_count": 2913563911457014571, "row_count": 3915711196905719420}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 51364794160395768, "histo_buckets": [{"distinct_range": 0, "num_eq": 8907097897543845985, "num_range": 0, "upper_bound": "14:04:26.942835"}, {"distinct_range": 49343.29662844447, "num_eq": 80, "num_range": 70000, "upper_bound": "14:45:15.664429"}, {"distinct_range": 9160997093152601000, "num_eq": 50000, "num_range": 9160997093152600992, "upper_bound": "18:50:04.90251"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 2929329798020765780, "row_count": 3915711196905719420}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3511703979436575385, "histo_col_type": "", "name": "__auto__", "null_count": 184862659861439446, "row_count": 3915711196905719420}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1266811980343122914, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 196989248.70962623, "num_eq": 4000000, "num_range": 300000000, "upper_bound": "\\u0013J"}, {"distinct_range": 700, "num_eq": 4000000, "num_range": 700, "upper_bound": "\'"}, {"distinct_range": 8408930187716308000, "num_eq": 1752209279048651297, "num_range": 8408930187716308449, "upper_bound": "g@B\\nOS"}, {"distinct_range": 8000, "num_eq": 1923541240732917621, "num_range": 8000, "upper_bound": "n\\u0018gf"}, {"distinct_range": 30000, "num_eq": 100000, "num_range": 30000, "upper_bound": "qh@K7"}, {"distinct_range": 1661575677097038300, "num_eq": 3000, "num_range": 1706882962821256453, "upper_bound": "|dX+"}, {"distinct_range": 4000, "num_eq": 707613427587963801, "num_range": 4000, "upper_bound": "~+\\u0013"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 2273168854170679438, "row_count": 3915711196905719420}]':::JSONB;
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_1) REFERENCES table2 (col2_3);
        test_1        |             ALTER TABLE table1 ADD FOREIGN KEY (col1_2) REFERENCES table2 (col2_3);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('nsy');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1726258763782129759, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4682491088377980539}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3310551042413427975, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4682491088377980539}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3254857892965190714, "histo_col_type": "", "name": "__auto__", "null_count": 1308126457632719283, "row_count": 4682491088377980539}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3299673644727655053, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "\\u0015PRYz"}, {"distinct_range": 1957.2314340694481, "num_eq": 100000, "num_range": 2000, "upper_bound": "\'"}, {"distinct_range": 0, "num_eq": 0, "num_range": 6960914786100836641, "upper_bound": "X8\\":"}, {"distinct_range": 0, "num_eq": 90, "num_range": 3260258607307688207, "upper_bound": "]g\\u0003M"}, {"distinct_range": 3708374351486067000, "num_eq": 8781412125021406842, "num_range": 3708374351486067314, "upper_bound": "r,"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 4682491088377980539}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2002569403499177305, "histo_col_type": "", "name": "__auto__", "null_count": 3104828746493652668, "row_count": 4682491088377980539}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2875528024120894844, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4682491088377980539}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4534087910536080795, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 2772164308196809700, "num_eq": 300, "num_range": 4028566515487184192, "upper_bound": "\\u0000"}, {"distinct_range": 513192660828287400, "num_eq": 100000000000, "num_range": 513192660828287410, "upper_bound": "\\u0012M"}, {"distinct_range": 2743325881633964500, "num_eq": 20000000000, "num_range": 3247724115582127310, "upper_bound": ";&"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 4682491088377980539}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 191751579218538353, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 500, "num_range": 9000, "upper_bound": "66419571"}, {"distinct_range": 360132406192357600, "num_eq": 1207144362164545511, "num_range": 4452057935012403293, "upper_bound": "391483032"}, {"distinct_range": 596.640951458087, "num_eq": 3000, "num_range": 2000, "upper_bound": "1925472025"}, {"distinct_range": 7327979732329603000, "num_eq": 800, "num_range": 8687805021168424898, "upper_bound": "2285996167"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 4682491088377980539}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2233000121664787411, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4682491088377980539}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 685176144309616915, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4682491088377980539}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3821899255228664649, "histo_col_type": "", "name": "__auto__", "null_count": 118633541123577322, "row_count": 4682491088377980539}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 958262474618480596, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000, "num_range": 0, "upper_bound": "S:\\u0010W"}, {"distinct_range": 455.4690752899315, "num_eq": 5599276252483376050, "num_range": 500, "upper_bound": "f\\u00146"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 4682491088377980539}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 185, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 151, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200}, {"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 118, "histo_col_type": "", "name": "__auto__", "null_count": 111, "row_count": 200}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 87, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000, "num_range": 0, "upper_bound": "00:16:38.874249"}, {"distinct_range": 50000, "num_eq": 8902104844624131785, "num_range": 50000, "upper_bound": "07:26:28.808392"}, {"distinct_range": 398687875.2004357, "num_eq": 3722873035049833180, "num_range": 400000000, "upper_bound": "10:46:27.861789"}, {"distinct_range": 0, "num_eq": 7385441055184039639, "num_range": 8065252759351681329, "upper_bound": "14:25:18.152621"}, {"distinct_range": 5745935.813000884, "num_eq": 0, "num_range": 20000000, "upper_bound": "23:41:20.890079"}, {"distinct_range": 4.625499818401395, "num_eq": 5202431121156734223, "num_range": 60, "upper_bound": "23:44:14.814091"}, {"distinct_range": 50, "num_eq": 9000, "num_range": 50, "upper_bound": "24:00:00"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 105, "row_count": 200}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36, "histo_col_type": "", "name": "__auto__", "null_count": 120, "row_count": 200}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 109, "histo_buckets": [{"distinct_range": 0, "num_eq": 300000000, "num_range": 0, "upper_bound": "01010000A0E6100000000000000000F87F000000000000F87F000000000000F87F"}, {"distinct_range": 23920683.389420003, "num_eq": 1233674247756763985, "num_range": 800000000, "upper_bound": "01050000E0E61000000800000001020000C007000000C8494F16F9DC32C0108701D953032AC02BF20C31ACAFFDC19CA04F4558CFF5C1349AB51EEE805F40278430F3980040C02D34A58EF0EAF1C15EB08165FAB3F941EC339F60AC1A57404002896F40E42BC0104AB9E3024EFD41D4241075C575FB4144688A52F0BD5640C0C9816326770F404050A3337F4BB8C1B4252CE6E7D8F84184D269DAF228654000710F145999064002C3A7555D36E0C104CEFF52F7E5FCC1A0AE49F181566140A07DA5EE8E5046402D6CD9867EBCF0C1E45817DD6BF5E04140DCC198375A654030D4201659F75540CE4691BF24EEF1414A77590F0A6CFDC101020000C004000000B61A322F9CD859C0964555662FFA40C0E057A742135BC641CCFDF0206616F3C14CB153B5F7F550C01CEEDCBE3A1F3AC05C249740B8390042807349FEC2C1D041B00F618CB8EE2C40DC7D4F28F38349C0EB9CACEAFE3801C21C059294157E01C24A610EF5A50261402CA191138FE24EC05CCE46851B09E2C154B63BAA545D014201020000C002000000EC8E68BEDF8F40C07E2C5E4DC2A341C0E4470DC6F956E1412055D2679596EF418644ADBBB72160C01F201753367243C04C1279B91B6FE74122DFE15AAFA4004201020000C0090000006B7F6EAAA06B64C090BBFA4F4AA54EC0E0E3462A6F30E6415EDC57B175A1F14140624971172720405B772E70095341C08A27663068BCF241AC0F6FD5E41BDFC1A0E213E8BBB033405864EC1C55E322C050D98A3A578CE64130588071DC36D1C1C822524C996154405CF238FC320A47C02CBBF8B2CE4DE641001611541C51E3C1D8C766C4B839664044CE9C5A503D53408C47019623DFF84176D41EAE54E4F441729D20EE7A51504078DB5152804A434088D29FB85A74FBC112FE728754F5F8C130064B6638555540889EAD09ED9548408C9373252115D8C11BAD1BB6ECB801C220A6A612D85E2F409E4BC3606C50554002D70246CC4B02C2745A7D85880AFF41E360C08D5C1553C0FCE141B8E7844640E018323C95AE004240D236C3835BAAC101020000C0050000004C7C294222C665C0EF62B574C0FE40C080C303DF002B9E4116265D87E4BFFEC160F7413A21134AC0B5D91AC68F5550C0A8B834A334EEF14130AF90F2E5C1C74100272BCDDDA229408FD9A056458A54C0684766D493D9EE4168A8F2C256C8FC418890BEEF9590584017D76CE7157D52C0D8FC29D4394CDF41C85D3F6DEE52D641D0CC61C0064C5C40FAC68964FF7C4EC057B41B5F52EFFCC15C68BF8A9B5AE2C101020000C0030000009C4C3EA59A73654027086D9C34CD41C09AF9C4FC04B7E4C185670EA9DAB9F3C1205AD77AEC005A4000D10833709507C08F8D1C14FCD0F3C1FA8EEFE7E66B004230BD44B5BBCE4EC0D0FF38C1911D504020151B098958BD41F212C95C8DA1F64101020000C00600000004894E39348A50C029F97A4E276E51C0AB75B6B76550F0C1B0D534B06EC1E5C1366091C6F55650403FEBCBE8AC3955C02A647C0452A5E5C18CEC14DE5655E641C497BBF68A1F5A40C45E133A267442C07C3145D2F81EF441A84DAFCC692CD541709996F9BF4F62400EB8D98170FC5540544035A6C4860142D831E4C84938F041928B0F92378354C00A4684284D2E56400840D5686594E1C198FB9DFD84CAEBC1D2471FA927B353C0D0E894B1A02A4C400071B2CE692A9AC1F699ACDD9E64014201020000C0070000005C8AAB12AAFE56409EAAF9B40E1754C0EC3C3ADCADFAFBC10C5F3B61BCFEFA4198BD07BE5A8A56404320057A20894FC00C85269F10F7F64178ED568C78B0F2C12C2A1E67806B5540462E035C30BB4BC03C5D17BFAEE9FC41DC9BA7FE00F3FB411060A42744203340A0F71C77A48814C078A5ACABE6D1FA41246B0FABEC70E7C1243ADBACAC615D40287F7C455E514F40827643A3894DF4C1217C6584936DFDC1970FD41C96BD60C028819E5254DC3240373DBECB84A900C2D8E3A9C1D6C6E5416AAD1A41B2A962C030CAC7CE4B5B2140303F4A4654B7DB4148C6EFB30B24FF41"}], "histo_col_type": "GEOGRAPHY", "name": "__auto__", "null_count": 101, "row_count": 200}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 139, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1813320872"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 200}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 25, "histo_col_type": "", "name": "__auto__", "null_count": 51, "row_count": 200}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 181, "histo_col_type": "", "name": "__auto__", "null_count": 15, "row_count": 200}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 13, "histo_col_type": "", "name": "__auto__", "null_count": 40, "row_count": 200}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6505836, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 18767573, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9364690, "histo_col_type": "", "name": "__auto__", "null_count": 463291, "row_count": 20000000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6318001, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000000, "num_range": 0, "upper_bound": "01:03:58.535121"}, {"distinct_range": 0, "num_eq": 3927091389992983300, "num_range": 3366069947377799684, "upper_bound": "02:05:19.687258"}, {"distinct_range": 4815175301136683000, "num_eq": 2000, "num_range": 7327899603993118690, "upper_bound": "05:11:44.128215"}, {"distinct_range": 1706539187892526300, "num_eq": 300000, "num_range": 1706539187892526277, "upper_bound": "16:06:14.927439"}, {"distinct_range": 0, "num_eq": 0, "num_range": 3673752827275726688, "upper_bound": "16:40:38.24774"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "16:58:53.543343"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 14074473, "row_count": 20000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4921589, "histo_col_type": "", "name": "__auto__", "null_count": 18855150, "row_count": 20000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7272669, "histo_buckets": [{"distinct_range": 0, "num_eq": 8275672614107642973, "num_range": 0, "upper_bound": ","}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 6568256, "row_count": 20000000}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2317584371811571408, "histo_buckets": [{"distinct_range": 0, "num_eq": 7000000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 96201528.63811643, "num_eq": 600000, "num_range": 100000000, "upper_bound": "\\u0001u%.l\\u0010"}, {"distinct_range": 0, "num_eq": 4615623245264725361, "num_range": 300, "upper_bound": "i+/BJ\\u0008"}, {"distinct_range": 4146441920221671000, "num_eq": 6801435453640674558, "num_range": 4146441920221671085, "upper_bound": "}z3\\u001c,*"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 8496429277935723006}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 636583786423207783, "histo_buckets": [{"distinct_range": 0, "num_eq": 80000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 4068008290862576540, "num_range": 3930129047946724995, "upper_bound": "\\u0000"}, {"distinct_range": 1412580300860919600, "num_eq": 6968373722108213921, "num_range": 7809494863393874017, "upper_bound": "\\u000cexV\\u0010Ky"}, {"distinct_range": 6835606918891278000, "num_eq": 200000000, "num_range": 6987101322960165249, "upper_bound": "\\""}, {"distinct_range": 0, "num_eq": 1000, "num_range": 900000, "upper_bound": ")"}, {"distinct_range": 2122859033965129700, "num_eq": 50000000000, "num_range": 4423527561967186628, "upper_bound": ")&\\neZDQ*"}, {"distinct_range": 2235777365304554200, "num_eq": 4000, "num_range": 2235777365304554287, "upper_bound": "3f!nb\\u000c#"}, {"distinct_range": 0, "num_eq": 5000000000, "num_range": 200000000, "upper_bound": "BTy"}, {"distinct_range": 3793511.1063024495, "num_eq": 1229838104986701791, "num_range": 40000000, "upper_bound": "]yD\\u0001C"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 8496429277935723006}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7352872878285854898, "histo_buckets": [{"distinct_range": 0, "num_eq": 5957511998270676030, "num_range": 0, "upper_bound": "4128603650"}], "histo_col_type": "OID", "name": "__auto__", "null_count": 0, "row_count": 8496429277935723006}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8465158253503160288, "histo_col_type": "", "name": "__auto__", "null_count": 7615899625691171394, "row_count": 8496429277935723006}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1066100521726213370, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8496429277935723006}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3408782760143409838, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8496429277935723006}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5265587628100738638, "histo_col_type": "", "name": "__auto__", "null_count": 6424690194989139307, "row_count": 8496429277935723006}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4153288536535857025, "histo_col_type": "", "name": "__auto__", "null_count": 8081416164607369694, "row_count": 8496429277935723006}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8352562240747954188, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8496429277935723006}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 776213816728419896, "histo_buckets": [{"distinct_range": 0, "num_eq": 4544847657997996812, "num_range": 0, "upper_bound": "\\u0007^Ds*"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 8496429277935723006}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5784842270948893810, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8496429277935723006}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 430639270049319457, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8496429277935723006}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS '[{"columns": ["col2_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 253883919536897370, "histo_col_type": "", "name": "__auto__", "null_count": 5332194695397496942, "row_count": 5785580152991512990}, {"columns": ["col2_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3418312630381156748, "histo_col_type": "", "name": "__auto__", "null_count": 3516271696557681583, "row_count": 5785580152991512990}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5295405467358786127, "histo_buckets": [{"distinct_range": 0, "num_eq": 20000000000, "num_range": 0, "upper_bound": "0105000060E61000000200000001020000400300000020D0B3FCFF9C3140CC325B3CCF4B4DC09004AE8207F301C28845A5BD0EAF5F40FCA1DFC48D535140B0054E67696CE1411866C7FF2F1B3C40A625D9CB5A755240E0371FEA7C21EA41010200004006000000951292ECF3DD61C07A41A5F54BB74CC0D497770A8B1FE741EC2EC7E52BCD45C0957EE63A668250C0023706C84CDEF341B8A4B1909ED74AC0E1191A0B56B341C0F2C733E1553BFEC108053F48ECF74A40843147C3D1713F407A508B913B4CFDC1C22132D486685DC0BC872B39EBDF54403C22CAED7BF2E1C133C2AC8B44F45DC088CCC32481A953404461D308F7DAE641"}, {"distinct_range": 0, "num_eq": 2472963222080237073, "num_range": 0, "upper_bound": "0105000060E610000007000000010200004008000000EA080D0535EC53C0DE991971A2AA47C09882B75B1982FA41381E938A659C31C017D47EB98D2353C00E83BC3E9B49FAC1787769F979C14140742C05C0EB6E40C0A0A67BB0AA73BBC1C0237792976316C09885E55ECE072DC00DC6E2A70D11FCC198042C7642575F4010EC0510C6E023404EC4AA87D839F741202E5F93792022C090B80AF9CD7C49404EC062980704F841C0047DCDE0382AC074080D476B4F4F4000124ECF943DB841A4E2895640AC5BC054AC7D2983DD3F406CA839F73FE6E54101020000400500000094A4F453341A5940C6F4632E50784BC0F0072B390195F8413C39ED8932A761400ECEADE9BF8942C0849E52B44380E9C198E820FDA0DC5040D84B38C32DCC36C0B05FC5465123DF416C510BAE75A45540B4384CA17941344000BA2BAEEEAF79C13816D98B95D257C00C6FE8B150D05240009165767BC9E241010200004002000000605A3C5FA1AA5C40908CD70271561EC08893184105D6D0411083D2D4BC4325C080D718B063542040188DD0093E02EFC1010200004003000000D0C4BE86927B3BC04DBBD1D25F8048C060F46041678B01426006474ACC8554401154D9A9BC4044C08E67D7E23E590042306A26D549155F40B31F84FA727D4CC0D494857AF227E5C10102000040040000006EC82F24C48F61C076A634B6BEAB47C03A26CA8E7271FAC1F623E4AF26DB52C0647F985B4F814EC050C14551E188C941C0C0DFC8F2331A40802BEA99051D1BC020D2A8085693B34124D9069062E3644080802DC291F21D40A84E9598BE9BD5C1010200004003000000F2F0FEF67B9560C071F91947EF6546C06C8BF89FA733E94110719ABD86AA3FC08117A8B8A1DB4FC0C5BE6483F345F3C168094982FC3D34C03FF012B2155841C00E913D646CB0EFC101020000400200000010E5623ACA5A5840A0239601DD1523C0CCE8FFA32D98FD4174E239C6B7995C40003D63E6B6282D408851306EB2A1D241"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 6352294230222290283, "upper_bound": "0107000060E6100000040000000103000060E6100000010000000D0000000E84E695EEA54EC093761887ACA14DC0AE654B3ED2C4FAC198495CC7707B47C07B22B6445EA749C038D9E034035EF641C00A79F1587A2AC08690D1C8CCF13BC090FA73F2A6E2C141A4E9735B37184C40BF4098D1B59F41C0640E8316D2110242F85C56DD98D165404CF7AB2F522953C020493CF83CF8B5C198A3FD05FA5637402CCF6D91610E36C0CEDDB663C6FBF8C128976A1E5AD55B408C45AAB21C3236C03C70F61CEB2002C2125C046D48156340407C59AAB1FA0C402079BA99F185DF41505A093EFE3240C0C8843E8EF1E94740CC762E370700FEC1F82DC8801DF43CC010484FFF90A41240E46C11040C9D01422E80F101FFD764C01463C87327954640787D90CACFA2DC41155B672C705462C0E8717BFE86B726407C72BC1DB11CF0410E84E695EEA54EC093761887ACA14DC0AE654B3ED2C4FAC10102000060E610000002000000C0926CEA24C632C098B2D52D68EE4040CC1167E863A3EE4185E533B6094462C000107F828D0FD3BF1C0E665BB91000420104000060E61000000300000001010000404293D0E2DA45664049B6C711755456C05CB15D431D73EE41010100004048F40B682D4D31402903583ACEE952C080B8F67AFC77BDC101010000407C73256D7F965440A3412AEFF7024BC048C80AFEA857D2C10101000060E6100000AF72ABF5F71659C0EC00812FA4CC55C0F8F34FE11A1DECC1"}, {"distinct_range": 200, "num_eq": 7000000, "num_range": 200, "upper_bound": "0103000020E61000000100000009000000C5DF2194082D55C07F5656436A1A49C0C35BF3A2E23361C0D6490FFC0DB953C03C3E25442DA75240F88A13CA777850C0A8F574F861F75F40B72C51CA3F4F52C0866BB00A3A5F644020E9101FCB5D37C06CD93B10180144C0A874069260E335C0844A8B7CDA2E42C07BAB072C1BEE42C0661AB2B8DF0465C02261B816C45746C0C5DF2194082D55C07F5656436A1A49C0"}], "histo_col_type": "GEOGRAPHY", "name": "__auto__", "null_count": 1636265025642710799, "row_count": 5785580152991512990}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2758184750412514959, "histo_col_type": "", "name": "__auto__", "null_count": 3283826083482377754, "row_count": 5785580152991512990}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3808756472061080175, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5785580152991512990}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5366720341853615307, "histo_col_type": "", "name": "__auto__", "null_count": 5115255574547522564, "row_count": 5785580152991512990}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5350915047767704992, "histo_col_type": "", "name": "__auto__", "null_count": 371816093877999288, "row_count": 5785580152991512990}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 348260685387457565, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 5785580152991512990}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5104080844167323681, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000000, "num_range": 0, "upper_bound": "2278334439"}], "histo_col_type": "REGCLASS", "name": "__auto__", "null_count": 0, "row_count": 5785580152991512990}, {"columns": ["col2_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3512398390214645354, "histo_col_type": "TIME", "name": "__auto__", "null_count": 3607941697053371862, "row_count": 5785580152991512990}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 140629, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "03:16:27.999019"}, {"distinct_range": 1260261423298682000, "num_eq": 600000, "num_range": 7640599258885947513, "upper_bound": "13:32:03.505388"}, {"distinct_range": 889644279044441500, "num_eq": 400, "num_range": 2677299064111968703, "upper_bound": "19:40:11.162014"}, {"distinct_range": 4305961944538381300, "num_eq": 50000000000, "num_range": 8623623481372671658, "upper_bound": "22:08:09.223474"}, {"distinct_range": 0, "num_eq": 700, "num_range": 7000000000, "upper_bound": "23:32:56.55535"}, {"distinct_range": 1622969044394016800, "num_eq": 6362298255417668164, "num_range": 7456121741513172943, "upper_bound": "23:37:13.635829"}], "histo_col_type": "TIME", "name": "__auto__", "null_count": 203621, "row_count": 500000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 464238, "histo_col_type": "", "name": "__auto__", "null_count": 48227, "row_count": 500000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 109377, "histo_buckets": [{"distinct_range": 0, "num_eq": 8770514542990396178, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0.5743617851271328, "num_eq": 4148589326467159348, "num_range": 10, "upper_bound": "CSKJ\\u0012"}, {"distinct_range": 7000, "num_eq": 0, "num_range": 7000, "upper_bound": "LR9{=b7"}, {"distinct_range": 0, "num_eq": 9000000, "num_range": 4906430322998385199, "upper_bound": "gl"}, {"distinct_range": 1655628255044250000, "num_eq": 1000000, "num_range": 7782808323446800424, "upper_bound": "j6\\u00046r[&"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 225859, "row_count": 500000}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 272358, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 159876, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7612, "histo_col_type": "", "name": "__auto__", "null_count": 442886, "row_count": 500000}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "inf" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_postgres_1   ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61683 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2795643&tab=buildLog) on [master@3d19b2cf6b290a152b23722fc32e995eed3b437b](https://github.com/cockroachdb/cockroach/commits/3d19b2cf6b290a152b23722fc32e995eed3b437b):

```
        postgres_1    | 				'::3/0'::INET AS col_67010,
        postgres_1    | 				6::OID AS col_67011,
        postgres_1    | 				tab_39672.col1_4 AS col_67012,
        postgres_1    | 				tab_39672.col1_2 AS col_67013,
        postgres_1    | 				NULL AS col_67014
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table1 AS tab_39672
        postgres_1    | 			WHERE
        postgres_1    | 				(false AND true)
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_39672.crdb_internal_mvcc_timestamp, tab_39672.col1_2 DESC
        postgres_1    | 		)
        postgres_1    | 			AS tab_39673
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_67009 NULLS FIRST,
        postgres_1    | 		col_67010 NULLS FIRST,
        postgres_1    | 		col_67011 NULLS FIRST,
        postgres_1    | 		col_67012 NULLS FIRST,
        postgres_1    | 		col_67013 NULLS FIRST,
        postgres_1    | 		col_67014 NULLS FIRST
        test_1        | --- FAIL: TestCompare (642.79s)
        test_1        |     --- FAIL: TestCompare/postgres (42.09s)
        test_1        |         compare_test.go:155: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |               	nil,
        test_1        |             - 	string("G6.0"),
        test_1        |             + 	string("G6"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT NULL AS col_67015, ('G'::TEXT::TEXT || 6.0::FLOAT8::FLOAT8)::TEXT AS col_67016) AS tab_39674
        test_1        |             ORDER BY
        test_1        |             	col_67015 NULLS FIRST, col_67016 NULLS FIRST;
        test_1        |             
        test_1        |             postgres: [same as previous]
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61683 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2797329&tab=buildLog) on [master@893643b63ea0b1cfa4888c6b73b5c68a9c100c3a](https://github.com/cockroachdb/cockroach/commits/893643b63ea0b1cfa4888c6b73b5c68a9c100c3a):

```
        postgres_1    | 2021-03-21 05:12:19.623 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2021-03-21 05:12:19.631 UTC [50] LOG:  shutting down
        postgres_1    | 2021-03-21 05:12:19.733 UTC [48] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-03-21 05:12:19.831 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-03-21 05:12:19.831 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-03-21 05:12:19.834 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-03-21 05:12:19.849 UTC [85] LOG:  database system was shut down at 2021-03-21 05:12:19 UTC
        postgres_1    | 2021-03-21 05:12:19.854 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (1.72s)
        test_1        |     --- FAIL: TestCompare/postgres (0.95s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.4:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (0.78s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 NAME NOT NULL, col1_1 FLOAT4 NOT NULL, col1_2 REGPROCEDURE, col1_3 UUID[] NOT NULL, col1_4 FLOAT8 NOT NULL, col1_5 INT2, col1_6 STRING NOT NULL, col1_7 STRING NOT NULL AS (lower(col1_0)) VIRTUAL, col1_8 FLOAT4 NULL AS (col1_1 + col1_4) STORED, col1_9 STRING NULL AS (IF(col1_3 IS NULL, NULL, ';0':::STRING)) VIRTUAL, INDEX (col1_2 DESC), UNIQUE (col1_6, col1_5 DESC) STORING (col1_0, col1_1, col1_2, col1_8), INVERTED INDEX (col1_4 DESC, col1_6 DESC, col1_9 ASC, col1_8, col1_2 ASC, col1_3) WHERE ((((table1.col1_7 < e'\x00':::STRING) OR (table1.col1_5 != (-32768):::INT8)) OR (table1.col1_9 >= e'\'':::STRING)) OR (table1.col1_6 > 'X':::STRING)) AND (table1.col1_8 < 'NaN':::FLOAT8), FAMILY (col1_8, col1_0), FAMILY (col1_4, col1_6), FAMILY (col1_2, col1_5), FAMILY (col1_3, col1_1));
        test_1        |             CREATE TABLE table2 (col2_0 BYTES NOT NULL, col2_1 INT2 NOT NULL, col2_2 FLOAT8 NOT NULL, col2_3 INT8 NOT NULL, PRIMARY KEY (col2_0 DESC, col2_1 ASC, col2_3, col2_2 ASC), INDEX (col2_3 ASC) WHERE ((table2.col2_0 >= '\xff') OR (table2.col2_1 = 32767)) AND (table2.col2_2 >= -1.0), INDEX (col2_3 ASC, col2_0 DESC) WHERE ((table2.col2_2 = 1.7976931348623157e+308:::FLOAT8) AND (table2.col2_0 < e'\\x22':::BYTES)) OR (table2.col2_1 < (-1):::INT8), FAMILY (col2_0, col2_2, col2_1), FAMILY (col2_3));
        test_1        |             CREATE TABLE table3 (col2_0 BYTES NOT NULL, col2_1 INT2 NOT NULL, col2_3 INT8 NOT NULL, col2_2 FLOAT8 NOT NULL, col3_4 INET NULL, col3_5 BIT(49) NOT NULL, col3_6 TIMESTAMPTZ NULL, col3_7 TIMETZ, PRIMARY KEY (col2_0 DESC, col2_1 ASC, col2_3, col2_2 ASC, col3_4 ASC, col3_7 ASC), INDEX (col2_2 ASC, col2_0 DESC), INDEX (col2_0 ASC, col2_3, col2_1, col3_5 ASC, col3_7 ASC) WHERE (((((table3.col2_2 > 1.0:::FLOAT8) AND (table3.col2_0 != e'\\xe29883':::BYTES)) AND (table3.col3_6 >= '-2000-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table3.col2_1 != (-1):::INT8)) OR (table3.col3_7 < '00:00:00+15:59:00':::TIMETZ)) AND (table3.col2_3 < (-1):::INT8), INDEX (col2_3, col3_7, col2_0 ASC, col2_2, col3_6, col3_5, col3_4 ASC) WHERE (((((table3.col2_0 <= e'\\x':::BYTES) AND (table3.col3_6 >= '-4713-11-24 00:00:00+00:00':::TIMESTAMPTZ)) OR (table3.col2_3 != 9223372036854775807:::INT8)) OR (table3.col2_2 != 1.0:::FLOAT8)) AND (table3.col2_1 > 127:::INT8)) AND (table3.col3_7 != '00:00:00+15:59:00':::TIMETZ), UNIQUE (col3_4, col3_5 DESC, col3_6 ASC, col3_7 DESC, col2_3 ASC, col2_1 DESC) WHERE table3.col2_3 = 2147483647:::INT8, INDEX (col2_2 DESC, col2_1 DESC, col2_0 ASC, col3_7 DESC) STORING (col3_6) WHERE (((((table3.col2_1 != 1:::INT8) AND (table3.col3_6 > '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) AND (table3.col2_0 <= e'\\x58':::BYTES)) AND (table3.col2_2 >= 1.0:::FLOAT8)) OR (table3.col2_3 > 32767:::INT8)) OR (table3.col3_7 < '00:00:00+15:59:00':::TIMETZ), UNIQUE (col3_4, col2_0, col3_6, col2_3 ASC, col2_1 ASC, col3_7, col3_5, col2_2) WHERE table3.col2_0 != '\x58', UNIQUE (col3_5, col3_7, col3_4, col2_3 ASC) WHERE ((table3.col3_7 >= '24:00:00-15:59:00':::TIMETZ) AND (table3.col2_1 = 1:::INT8)) AND (table3.col3_6 = '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ), UNIQUE (col3_5 ASC, col3_4 ASC, col3_6, col3_7 ASC, col2_3 ASC) WHERE (((((table3.col2_2 != 1.401298464324817e-45:::FLOAT8) AND (table3.col2_3 < (-9223372036854775807):::INT8)) OR (table3.col3_7 < '24:00:00-15:59:00':::TIMETZ)) AND (table3.col2_1 != 32767:::INT8)) OR (table3.col3_6 >= '0001-01-01 00:00:00+00:00':::TIMESTAMPTZ)) OR (table3.col2_0 < e'\\x22':::BYTES), INDEX (col2_1, col2_2 DESC, col3_4 ASC, col2_0, col3_5, col3_7, col3_6 DESC, col2_3 ASC) WHERE table3.col3_7 <= '24:00:00-15:59:00', UNIQUE (col2_2) WHERE (table3.col2_2 = -Inf) OR (table3.col2_1 < -1), FAMILY (col2_1, col3_6, col2_0), FAMILY (col2_3), FAMILY (col3_4, col3_5), FAMILY (col2_2, col3_7)) INTERLEAVE IN PARENT table2 (col2_0, col2_1, col2_3, col2_2);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1371927413860205467, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1649059330686715014}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1099765467132867481, "histo_buckets": [{"distinct_range": 0, "num_eq": 2860101567391108218, "num_range": 0, "upper_bound": "-3.003465567806245"}, {"distinct_range": 13364794085.387299, "num_eq": 1404422422437835993, "num_range": 30000000000, "upper_bound": "-1.279440230619051"}, {"distinct_range": 1923301342.9730532, "num_eq": 100, "num_range": 7000000000, "upper_bound": "-0.9762593871941412"}, {"distinct_range": 3012582520061794000, "num_eq": 1000, "num_range": 3285700383192136574, "upper_bound": "-0.4794153961318339"}, {"distinct_range": 8738188307788849000, "num_eq": 8063377229653400034, "num_range": 8738188307788848799, "upper_bound": "0.0"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 30000, "upper_bound": "0.6676904809949273"}, {"distinct_range": 0, "num_eq": 665320969157277148, "num_range": 30000000000, "upper_bound": "0.8908938576717882"}, {"distinct_range": 0, "num_eq": 687611603042000106, "num_range": 8494347656524140664, "upper_bound": "1.0"}, {"distinct_range": 0, "num_eq": 3216510594855802926, "num_range": 0, "upper_bound": "1.1747661558377012"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 1649059330686715014}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1250086200553121358, "histo_col_type": "", "name": "__auto__", "null_count": 382961210424117180, "row_count": 1649059330686715014}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1150931832786904131, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 100000, "num_eq": 4907772595661410175, "num_range": 100000, "upper_bound": "e\\u0019Wp\\u0015\\te"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 1649059330686715014}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 642518985024190098, "histo_col_type": "", "name": "__auto__", "null_count": 1073886717610259194, "row_count": 1649059330686715014}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1331992643546810614, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1649059330686715014}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 777561117827076228, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "1298819519"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 181407236257517208, "row_count": 1649059330686715014}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 472546274241420830, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1649059330686715014}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1152680536024783769, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1649059330686715014}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 299194702077769167, "histo_col_type": "", "name": "__auto__", "null_count": 675941247653784836, "row_count": 1649059330686715014}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1513078535355699126, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000000, "num_range": 0, "upper_bound": "\\\\x208de37d"}, {"distinct_range": 0, "num_eq": 1894333635459233715, "num_range": 1846112663272679228, "upper_bound": "\\\\x56a8d30454cd35ca"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 1823464991852991684}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1161676528129480144, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1823464991852991684}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1671809150363862222, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1823464991852991684}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1441008979284468505, "histo_buckets": [{"distinct_range": 0, "num_eq": 800, "num_range": 0, "upper_bound": "-3724495250348118417"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 1823464991852991684}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11346259, "histo_col_type": "", "name": "__auto__", "null_count": 17070552, "row_count": 20000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 14233188, "histo_buckets": [{"distinct_range": 0, "num_eq": 30000, "num_range": 0, "upper_bound": "\\\\x22"}, {"distinct_range": 271727965781523550, "num_eq": 6308120181374875862, "num_range": 4957054407686655745, "upper_bound": "\\\\x7bce5d982b1450401b"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8099303, "histo_buckets": [{"distinct_range": 0, "num_eq": 7882391782352408236, "num_range": 0, "upper_bound": "-29038"}, {"distinct_range": 0, "num_eq": 70000000, "num_range": 800, "upper_bound": "-10176"}, {"distinct_range": 1146118757478645100, "num_eq": 2844853847925122726, "num_range": 1146118757478645096, "upper_bound": "-7583"}, {"distinct_range": 0, "num_eq": 50, "num_range": 1182080877696877159, "upper_bound": "0"}, {"distinct_range": 4456315181204273000, "num_eq": 2423597341482081509, "num_range": 4456315181204272980, "upper_bound": "8650"}, {"distinct_range": 42343196230.7407, "num_eq": 60000000, "num_range": 90000000000, "upper_bound": "15676"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5776496, "histo_buckets": [{"distinct_range": 0, "num_eq": 717216604280214067, "num_range": 0, "upper_bound": "-7081245513125172955"}, {"distinct_range": 900000, "num_eq": 9000000000, "num_range": 900000, "upper_bound": "-3440006974660807525"}, {"distinct_range": 400, "num_eq": 2165003308957960121, "num_range": 400, "upper_bound": "-501837385583230833"}, {"distinct_range": 1200247697668970500, "num_eq": 4136434201836200571, "num_range": 8244902146213579677, "upper_bound": "6691183154338944056"}, {"distinct_range": 0, "num_eq": 1798413406239519981, "num_range": 0, "upper_bound": "7700139545817781771"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6430003, "histo_buckets": [{"distinct_range": 0, "num_eq": 400000, "num_range": 0, "upper_bound": "-1.8669907910262111"}, {"distinct_range": 3478471.256170492, "num_eq": 4826631813361462506, "num_range": 4000000, "upper_bound": "-1.1698950871736118"}, {"distinct_range": 10000, "num_eq": 900000, "num_range": 10000, "upper_bound": "-0.8139321382924223"}, {"distinct_range": 58935915.79470498, "num_eq": 2889747508627978049, "num_range": 60000000, "upper_bound": "-0.5496425730430364"}, {"distinct_range": 0, "num_eq": 50000000000, "num_range": 0, "upper_bound": "-0.4249283981774096"}, {"distinct_range": 0, "num_eq": 7000000000, "num_range": 3714308250052305557, "upper_bound": "0.5313513374845835"}, {"distinct_range": 64404498433695550, "num_eq": 50000000, "num_range": 5188172137618478265, "upper_bound": "0.789445098951032"}, {"distinct_range": 0, "num_eq": 20000000, "num_range": 3000, "upper_bound": "0.790579628228531"}, {"distinct_range": 51284.05012497127, "num_eq": 0, "num_range": 80000, "upper_bound": "0.9350820209261637"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 8991348, "histo_buckets": [{"distinct_range": 0, "num_eq": 50000000000, "num_range": 0, "upper_bound": "225.30.144.49/15"}, {"distinct_range": 415641384780588000, "num_eq": 700, "num_range": 3224464614227640258, "upper_bound": "238.110.139.62/31"}, {"distinct_range": 0, "num_eq": 4618366787619969634, "num_range": 800, "upper_bound": "105.237.83.10"}, {"distinct_range": 945825807903557200, "num_eq": 40000000000, "num_range": 2767524254385426607, "upper_bound": "125.217.233.115"}, {"distinct_range": 0, "num_eq": 8000, "num_range": 1509894699743413838, "upper_bound": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 16525562, "row_count": 20000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 11753008, "histo_buckets": [{"distinct_range": 0, "num_eq": 5552276870469382638, "num_range": 0, "upper_bound": "1100001110101100011010111110100011100110111111110"}], "histo_col_type": "BIT(49)", "name": "__auto__", "null_count": 0, "row_count": 20000000}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 15436541, "histo_col_type": "", "name": "__auto__", "null_count": 8198151, "row_count": 20000000}]':::JSONB;
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col2_2) REFERENCES table3 (col2_2);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('ktok');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('drf', 'nkchsg', 'sebg', 'of');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('psxh');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('ailyrw', 'eq', 'ij', 'tod');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5978425878514380255, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8533939271536059249}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4900797643901182179, "histo_buckets": [{"distinct_range": 0, "num_eq": 1240904493295154991, "num_range": 0, "upper_bound": "-0.6373718398686561"}, {"distinct_range": 900, "num_eq": 1000, "num_range": 900, "upper_bound": "-0.19572188328198034"}, {"distinct_range": 8582523545648923000, "num_eq": 8165762757523820047, "num_range": 8582523545648922504, "upper_bound": "0.3151628771169994"}, {"distinct_range": 0, "num_eq": 1348428583878716636, "num_range": 2442571241790230354, "upper_bound": "0.3914547284661107"}, {"distinct_range": 0, "num_eq": 80000000000, "num_range": 300, "upper_bound": "1.1927192503022368"}, {"distinct_range": 726095639489484900, "num_eq": 5000, "num_range": 6821827942142237974, "upper_bound": "1.5871597550157612"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 8533939271536059249}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7742015564396409554, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8533939271536059249}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1188108773798878492, "histo_col_type": "", "name": "__auto__", "null_count": 3340442220371393997, "row_count": 8533939271536059249}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4216018553080707047, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8533939271536059249}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1319395663674619833, "histo_buckets": [{"distinct_range": 0, "num_eq": 60000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 3015967736729674677, "num_range": 20000000000, "upper_bound": "306640969"}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 0, "upper_bound": "2123641504"}, {"distinct_range": 4057890290.7353215, "num_eq": 6000, "num_range": 5000000000, "upper_bound": "2375642943"}, {"distinct_range": 0, "num_eq": 60, "num_range": 0, "upper_bound": "2475296751"}, {"distinct_range": 0, "num_eq": 5861357831911079936, "num_range": 7844946586160834025, "upper_bound": "2645559597"}, {"distinct_range": 3119472677971241000, "num_eq": 0, "num_range": 3119472677971240717, "upper_bound": "3351881744"}, {"distinct_range": 0, "num_eq": 8417900811793525751, "num_range": 3286161691108086352, "upper_bound": "3707953854"}, {"distinct_range": 6573180557056173000, "num_eq": 7985770899191097370, "num_range": 7274427703005786165, "upper_bound": "3713998083"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 6777533583017681442, "row_count": 8533939271536059249}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4778403097583687789, "histo_col_type": "", "name": "__auto__", "null_count": 3253309431014368967, "row_count": 8533939271536059249}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4077768080780036295, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 7556174728359671000, "num_eq": 9177653208729667783, "num_range": 7556174728359670348, "upper_bound": "\\u001bL\\u0006e-A"}, {"distinct_range": 0, "num_eq": 7282816913003098262, "num_range": 40000000, "upper_bound": ")9zKxa"}, {"distinct_range": 515637033769250900, "num_eq": 2876255689747231026, "num_range": 3412990064272143201, "upper_bound": "q6DLg\\u0019"}, {"distinct_range": 800, "num_eq": 7840461551497595815, "num_range": 800, "upper_bound": "\U00002603"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 8533939271536059249}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6513564388060279443, "histo_col_type": "", "name": "__auto__", "null_count": 789578319172419186, "row_count": 8533939271536059249}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6241867522576801339, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 8533939271536059249}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 308710142030071051, "histo_buckets": [{"distinct_range": 0, "num_eq": 9186192231181532943, "num_range": 0, "upper_bound": "\\\\x165500a1ac8071"}, {"distinct_range": 10000, "num_eq": 10000000000, "num_range": 10000, "upper_bound": "\\\\x1e98b3"}, {"distinct_range": 166264592632908600, "num_eq": 3408818322293259519, "num_range": 8462700407586053886, "upper_bound": "\\\\x3d2a"}, {"distinct_range": 11381880.493956188, "num_eq": 0, "num_range": 70000000, "upper_bound": "\\\\x5bca25e38fb55eed"}, {"distinct_range": 576.3421022928126, "num_eq": 40000000, "num_range": 800, "upper_bound": "\\\\x8560c18e5acb"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 2000000000, "upper_bound": "\\\\xa78c2c7dc676ec42f6"}, {"distinct_range": 5202306125364120000, "num_eq": 30000000000, "num_range": 7363168868268064243, "upper_bound": "\\\\xf856f85996"}, {"distinct_range": 0, "num_eq": 6000000, "num_range": 200000, "upper_bound": "\\\\xfc7ef57d1fc32b33a3"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 1178215245476813896}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 830831385481462329, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1178215245476813896}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 507360058328141334, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1178215245476813896}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 301378127812812446, "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 1178215245476813896}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 724931, "histo_col_type": "", "name": "__auto__", "null_count": 7695, "row_count": 3000000}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 142934, "histo_col_type": "", "name": "__auto__", "null_count": 605186, "row_count": 3000000}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1541433, "histo_buckets": [{"distinct_range": 0, "num_eq": 3824405285108754499, "num_range": 0, "upper_bound": "\\\\xdb"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2518750, "histo_buckets": [{"distinct_range": 0, "num_eq": 20, "num_range": 0, "upper_bound": "-32768"}, {"distinct_range": 3411097466540650500, "num_eq": 5000000, "num_range": 6249486281054342442, "upper_bound": "-31457"}, {"distinct_range": 2364858527255776000, "num_eq": 708178884154349242, "num_range": 2364858527255776213, "upper_bound": "-25749"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 50000000, "upper_bound": "-25497"}, {"distinct_range": 22950907.34411686, "num_eq": 50000000000, "num_range": 30000000, "upper_bound": "-20499"}, {"distinct_range": 767735.6517315182, "num_eq": 2286845546987070877, "num_range": 1000000, "upper_bound": "-14288"}, {"distinct_range": 0, "num_eq": 20, "num_range": 0, "upper_bound": "-128"}, {"distinct_range": 3287906865.8981295, "num_eq": 40000, "num_range": 4000000000, "upper_bound": "12622"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1574069, "histo_buckets": [{"distinct_range": 0, "num_eq": 1000000, "num_range": 0, "upper_bound": "-6690218403783549726"}, {"distinct_range": 3443202649176304000, "num_eq": 1882400064629006722, "num_range": 5719008496537740500, "upper_bound": "-6098412379869072407"}, {"distinct_range": 0, "num_eq": 800, "num_range": 5796317175422881360, "upper_bound": "-2298196063319382584"}, {"distinct_range": 90, "num_eq": 1000, "num_range": 90, "upper_bound": "71641939999061640"}, {"distinct_range": 33094139.001240335, "num_eq": 600000, "num_range": 200000000, "upper_bound": "2035146336709592494"}, {"distinct_range": 0, "num_eq": 50000000, "num_range": 8000, "upper_bound": "2483477433478315680"}, {"distinct_range": 69677727413.5014, "num_eq": 300, "num_range": 90000000000, "upper_bound": "4720545137883341882"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1667744, "histo_buckets": [{"distinct_range": 0, "num_eq": 703475883672542835, "num_range": 0, "upper_bound": "-2.0355662919705577"}, {"distinct_range": 0, "num_eq": 0, "num_range": 20000000000, "upper_bound": "-1.084585845088454"}, {"distinct_range": 10000000, "num_eq": 70000000000, "num_range": 10000000, "upper_bound": "-1.0794709212085563"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 3000000}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1475159, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "110.154.221.55/2"}, {"distinct_range": 2700932042517652000, "num_eq": 700, "num_range": 2700932042517652084, "upper_bound": "127.219.189.123/2"}, {"distinct_range": 10000000, "num_eq": 9022172910293783603, "num_range": 10000000, "upper_bound": "e8dd:e2c7:ad2b:3806:2910:6eac:6189:6738/26"}, {"distinct_range": 5883338654643310000, "num_eq": 3650169055372855247, "num_range": 5883338654643309065, "upper_bound": "fbdb:ccfb:fe6d:5691:f4d3:5846:5ab7:7b2f/34"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 2837372, "row_count": 3000000}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2725186, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000, "num_range": 0, "upper_bound": "0011111101100100101101000100000111100100100100101"}, {"distinct_range": 5690245822944830000, "num_eq": 30000, "num_range": 8381356810302917273, "upper_bound": "1000000011000011101010111101100010110010000000011"}, {"distinct_range": 80000000, "num_eq": 2927322091661966139, "num_range": 80000000, "upper_bound": "1000100011100010110011011100110001110100010111011"}, {"distinct_range": 6989812935.895555, "num_eq": 300000, "num_range": 9000000000, "upper_bound": "1000111010011001011001111010010010101001110000000"}, {"distinct_range": 2602396804361542700, "num_eq": 6954513228702913284, "num_range": 3737283283660688988, "upper_bound": "1100101010000110100010000111100010000001000111110"}, {"distinct_range": 30000000, "num_eq": 90000000000, "num_range": 30000000, "upper_bound": "1101100000001110111111111101100010111001100111111"}, {"distinct_range": 0, "num_eq": 500000, "num_range": 0, "upper_bound": "1101100010010110110010101100011011011001100101010"}], "histo_col_type": "BIT(49)", "name": "__auto__", "null_count": 0, "row_count": 3000000}]';
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 290162090, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 326940360, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 400180924, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 5231.509517878152, "num_eq": 1755106960931814906, "num_range": 6000, "upper_bound": "1367529703"}, {"distinct_range": 240101050779655400, "num_eq": 3063693406807900831, "num_range": 240101050779655389, "upper_bound": "1733835554"}, {"distinct_range": 793.950709380087, "num_eq": 10000, "num_range": 1000, "upper_bound": "2581517031"}, {"distinct_range": 7266115699426605000, "num_eq": 20, "num_range": 7266115699426604854, "upper_bound": "3108311667"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 900000, "upper_bound": "3412032583"}, {"distinct_range": 0, "num_eq": 8171341020273358499, "num_range": 0, "upper_bound": "3903415179"}], "histo_col_type": "REGPROCEDURE", "name": "__auto__", "null_count": 45204942, "row_count": 500000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 316194220, "histo_buckets": [{"distinct_range": 0, "num_eq": 70, "num_range": 0, "upper_bound": "0.13081271057119648"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 183351649, "histo_col_type": "", "name": "__auto__", "null_count": 109625027, "row_count": 500000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 162039207, "histo_buckets": [{"distinct_range": 0, "num_eq": 5345842968506065827, "num_range": 0, "upper_bound": "\\"\\u000b\\u001e"}, {"distinct_range": 5761266591720452000, "num_eq": 2666005468240379593, "num_range": 5761266591720451742, "upper_bound": "-/4j\\u0011D"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 70000, "upper_bound": "R$_7#U^_"}, {"distinct_range": 3000000000, "num_eq": 2643313777039180717, "num_range": 3000000000, "upper_bound": "Z\\u00081\\u001d5m[y"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 0, "upper_bound": "|r/]"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 408973231, "histo_col_type": "", "name": "__auto__", "null_count": 43445958, "row_count": 500000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 271904514, "histo_col_type": "", "name": "__auto__", "null_count": 423069251, "row_count": 500000000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 400847302, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 498225790, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 500000000}]';
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 216, "histo_buckets": [{"distinct_range": 0, "num_eq": 8429351632870793566, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 60000000, "num_eq": 917862570813749132, "num_range": 60000000, "upper_bound": "\\\\x8c3b1c1e80b2"}, {"distinct_range": 0, "num_eq": 601473866517682858, "num_range": 2091998390818404327, "upper_bound": "\\\\xa383ba42"}, {"distinct_range": 6408320006155480000, "num_eq": 60000000, "num_range": 6698652429942423875, "upper_bound": "\\\\xb11ec0bd70be7ea5"}, {"distinct_range": 1886532487423806200, "num_eq": 7456591442893910838, "num_range": 9113923656236054540, "upper_bound": "\\\\xe9fc"}, {"distinct_range": 2407965905185751600, "num_eq": 9000, "num_range": 4431155361015587465, "upper_bound": "\\\\xfe7b"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 100000}, {"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 34618, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22087, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 100000}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 95064, "histo_buckets": [{"distinct_range": 0, "num_eq": 8000, "num_range": 0, "upper_bound": "-4614385875194637122"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 100000}]';
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 554277631452401739, "histo_buckets": [{"distinct_range": 0, "num_eq": 7750399943066773992, "num_range": 0, "upper_bound": "-20808"}, {"distinct_range": 6809657579586055000, "num_eq": 5276146578622780983, "num_range": 7085499144280426453, "upper_bound": "-6082"}, {"distinct_range": 26364.023668931128, "num_eq": 60000000, "num_range": 90000, "upper_bound": "1"}, {"distinct_range": 0, "num_eq": 300, "num_range": 3444482290618945992, "upper_bound": "7185"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 0, "row_count": 6316801580309734169}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4286497517170804527, "histo_buckets": [{"distinct_range": 0, "num_eq": 7461588807769650509, "num_range": 0, "upper_bound": "-8868011641740083252"}, {"distinct_range": 286901105363603700, "num_eq": 400000000, "num_range": 286901105363603718, "upper_bound": "-7710830871695987973"}], "histo_col_type": "INT8", "name": "__auto__", "null_count": 0, "row_count": 6316801580309734169}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5456802248152192542, "histo_buckets": [{"distinct_range": 0, "num_eq": 40000000000, "num_range": 0, "upper_bound": "-1.9001002387812902"}, {"distinct_range": 0, "num_eq": 0, "num_range": 400000000, "upper_bound": "0.9689524830888648"}], "histo_col_type": "FLOAT8", "name": "__auto__", "null_count": 0, "row_count": 6316801580309734169}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5173343430608218771, "histo_buckets": [{"distinct_range": 0, "num_eq": 3022396200400817472, "num_range": 0, "upper_bound": "0.0.0.0/0"}, {"distinct_range": 14572.755430128722, "num_eq": 3995600957505183831, "num_range": 100000, "upper_bound": "653f:2f70:c636:8b96:aae2:dd69:3043:d328/83"}, {"distinct_range": 0, "num_eq": 300000, "num_range": 300000, "upper_bound": "560:2e09:ee4d:78f5:54:cf17:5c2a:706b/97"}, {"distinct_range": 0, "num_eq": 3550732107070622558, "num_range": 4563862843208331694, "upper_bound": "921f:a6ef:5781:d306:1246:e582:1fbb:78d6/127"}, {"distinct_range": 2820383772037918700, "num_eq": 6000000, "num_range": 2820383772037918597, "upper_bound": "ffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff"}], "histo_col_type": "INET", "name": "__auto__", "null_count": 3577134908724848709, "row_count": 6316801580309734169}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1439142000806986835, "histo_col_type": "BIT(49)", "name": "__auto__", "null_count": 0, "row_count": 6316801580309734169}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4588335493545432510, "histo_col_type": "", "name": "__auto__", "null_count": 155733828050096513, "row_count": 6316801580309734169}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4908878500040185963, "histo_col_type": "", "name": "__auto__", "null_count": 6080206996597472004, "row_count": 6316801580309734169}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 887497682131204110, "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 6316801580309734169}]';
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: column "inf" does not exist (SQLSTATE 42703)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61683 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2802938&tab=buildLog) on [master@53bf501e233c337b9863755914d9c00010517329](https://github.com/cockroachdb/cockroach/commits/53bf501e233c337b9863755914d9c00010517329):

```
        postgres_1    | 2021-03-23 05:11:38.279 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-03-23 05:11:38.279 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-03-23 05:11:38.284 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        test_1        | --- FAIL: TestCompare (2.08s)
        test_1        |     --- FAIL: TestCompare/postgres (0.69s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.3:5432: connect: connection refused
        test_1        |     --- FAIL: TestCompare/mutators (1.39s)
        test_1        |         compare_test.go:129: SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 BYTES NOT NULL, col1_1 INT4, col1_2 DECIMAL NOT NULL, col1_3 TIMESTAMP NOT NULL, col1_4 TIMETZ, col1_5 BIT(41) NOT NULL, col1_6 REGTYPE NOT NULL, PRIMARY KEY (col1_6 DESC, col1_2, col1_0 ASC), col1_7 STRING NOT NULL AS (IF(col1_0 IS NULL, e'z\x18:@ /':::STRING, '':::STRING)) STORED, col1_8 STRING NULL AS (IF(col1_0 IS NULL, e'C7!\x19O\f':::STRING, ':':::STRING)) STORED, col1_9 STRING NOT NULL AS (IF(col1_0 IS NULL, e'\x01vW/{\n<':::STRING, e'}88\x06g\x05%':::STRING)) STORED, col1_10 DECIMAL NOT NULL AS (col1_2 + (-421124002293.4931155):::DECIMAL) VIRTUAL, INDEX (col1_8, col1_10, col1_2, col1_4 DESC, col1_0 ASC, col1_1 DESC, col1_9 ASC, col1_3, col1_5 ASC, col1_7) WHERE (((((table1.col1_9 < '':::STRING) AND (table1.col1_8 > 'X':::STRING)) AND (table1.col1_4 > '00:00:00+15:59:00':::TIMETZ)) OR (table1.col1_10 < 0:::DECIMAL)) OR (table1.col1_2 != (-1.234E+401):::DECIMAL)) AND (table1.col1_3 = '-4713-11-24 00:00:00':::TIMESTAMP), UNIQUE (col1_2 DESC) STORING (col1_1, col1_7) WHERE (table1.col1_0 = e'\\x27':::BYTES) OR (table1.col1_10 >= 'NaN':::DECIMAL), INDEX (col1_5 DESC, col1_4, col1_6, col1_10, col1_8, col1_7, col1_0 DESC, col1_2, col1_9 DESC, col1_1), UNIQUE (col1_8, col1_9, col1_4 DESC, col1_0 ASC, col1_10 ASC) STORING (col1_1, col1_3, col1_5) WHERE ((((((((table1.col1_8 < 'X':::STRING) AND (table1.col1_10 <= '-Infinity':::DECIMAL)) OR (table1.col1_7 != e'\x00':::STRING)) OR (table1.col1_9 < '':::STRING)) AND (table1.col1_0 > '\xe29883':::BYTES)) AND (table1.col1_4 = '24:00:00-15:59:00':::TIMETZ)) AND (table1.col1_1 <= (-1):::INT8)) OR (table1.col1_2 = 'Infinity':::DECIMAL)) AND (table1.col1_3 <= '-2000-01-01 00:00:00':::TIMESTAMP), UNIQUE (col1_1 DESC, col1_10 ASC, col1_8 DESC, col1_6 DESC, col1_9 ASC, col1_3 DESC, col1_5 DESC) WHERE (((table1.col1_3 < '-4713-11-24 00:00:00':::TIMESTAMP) OR (table1.col1_1 = 1:::INT8)) OR (table1.col1_7 != '"':::STRING)) OR (table1.col1_8 <= e'\x00':::STRING), UNIQUE (col1_2 DESC, col1_4, col1_6, col1_1, col1_3 ASC) STORING (col1_9), INDEX (col1_0, col1_9 DESC, col1_6, col1_4 ASC, col1_3 DESC, col1_5 DESC, col1_8 DESC) STORING (col1_1, col1_7), FAMILY (col1_0, col1_8), FAMILY (col1_3, col1_2, col1_6, col1_1, col1_4, col1_9), FAMILY (col1_5), FAMILY (col1_7), UNIQUE (col1_7) WHERE (table1.col1_1 < 2147483647:::INT8) OR (table1.col1_3 >= '294276-12-31 23:59:59.999999':::TIMESTAMP));
        test_1        |             CREATE TABLE table2 (col1_6 REGTYPE NOT NULL, col1_2 DECIMAL NOT NULL, col1_0 BYTES NOT NULL, col2_3 STRING NOT NULL, col2_4 BOOL, PRIMARY KEY (col1_6 DESC, col1_2, col1_0 ASC, col2_4), UNIQUE (col1_0, col1_6, col2_4 DESC) STORING (col2_3), INDEX (col2_4 ASC, col1_0 DESC, col1_6) STORING (col2_3) WHERE ((table2.col1_2 = 0:::DECIMAL) OR (NOT table2.col2_4)) OR (table2.col1_0 = e'\\x58':::BYTES), INDEX (col1_6, col1_2 ASC, col2_4 DESC, col2_3 ASC, col1_0 DESC), UNIQUE (col2_4 ASC, col1_6 DESC, col2_3 DESC) WHERE (NOT table2.col2_4) AND (table2.col2_3 <= '':::STRING), UNIQUE (col1_2, col1_6, col2_3 DESC, col2_4 ASC, col1_0 ASC) WHERE (table2.col2_3 > '"':::STRING) AND (table2.col1_0 != e'\\x00':::BYTES), INDEX (col2_4 ASC, col1_6 DESC, col1_0 DESC) STORING (col2_3) WHERE (((table2.col2_3 != '"':::STRING) OR (table2.col1_0 < '\xff':::BYTES)) AND (table2.col1_2 >= 'NaN':::DECIMAL)) AND table2.col2_4, INDEX (col2_3 ASC, col1_6, col1_2 ASC) WHERE table2.col2_3 = '"':::STRING, INDEX (col1_6, col2_3 ASC, col1_0 ASC, col2_4 DESC, col1_2 DESC) WHERE (table2.col2_3 < e'\x00':::STRING) AND (table2.col1_2 != '-Infinity':::DECIMAL), FAMILY (col1_2), FAMILY (col2_3), FAMILY (col1_6, col2_4), FAMILY (col1_0)) INTERLEAVE IN PARENT table1 (col1_6, col1_2, col1_0);
        test_1        |             CREATE TABLE table3 (col1_6 REGTYPE NOT NULL, col1_2 DECIMAL NOT NULL, col1_0 BYTES NOT NULL, col3_3 VARCHAR, PRIMARY KEY (col1_6 DESC, col1_2, col1_0 ASC), UNIQUE (col3_3 ASC, col1_6 DESC, col1_0, col1_2 DESC), UNIQUE (col1_6 ASC, col3_3, col1_0 DESC) WHERE ((table3.col3_3 >= e'\x00':::STRING) AND (table3.col1_2 <= '-Infinity':::DECIMAL)) AND (table3.col1_0 < e'\\xe29883':::BYTES), INDEX (col3_3 DESC, col1_2, col1_6 ASC, col1_0) WHERE ((table3.col1_0 < e'\\x00':::BYTES) OR (table3.col3_3 <= '"':::STRING)) OR (table3.col1_2 = 0:::DECIMAL), INDEX (col1_6 DESC, col3_3 DESC, col1_0, col1_2), INDEX (col1_0) WHERE table3.col3_3 != e'\x00':::STRING, INDEX (col1_0 ASC, col3_3 DESC, col1_2 DESC) WHERE table3.col3_3 != e'\U00002603':::STRING, INDEX (col3_3 ASC, col1_2 ASC), FAMILY (col1_0, col1_2, col3_3, col1_6)) INTERLEAVE IN PARENT table1 (col1_6, col1_2, col1_0);
        test_1        |             CREATE TABLE table4 (col4_0 INT2 NULL, col4_1 STRING, col4_2 STRING NULL, col4_3 DECIMAL NOT NULL, col4_4 REGCLASS, col4_5 STRING NOT NULL, col4_6 FLOAT8 NOT NULL, col4_7 CHAR NOT NULL, PRIMARY KEY (col4_5 ASC, col4_7 DESC), col4_8 DECIMAL AS (col4_3 + (-716878506234.4252335):::DECIMAL) VIRTUAL, col4_9 FLOAT8 NULL AS (col4_6 + (-0.7957448627807059):::FLOAT8) STORED, col4_10 STRING NOT NULL AS (lower(col4_5)) VIRTUAL, col4_11 STRING NOT NULL AS (lower(col4_5)) STORED, col4_12 STRING AS (lower(CAST(col4_4 AS STRING))) VIRTUAL, col4_13 DECIMAL NULL AS (col4_3 + 728.0703121431200208:::DECIMAL) STORED, UNIQUE (col4_8 ASC, col4_13 ASC, col4_2 DESC, col4_11 ASC, col4_5 ASC, col4_12 ASC) WHERE (((((((((((table4.col4_13 < 0:::DECIMAL) OR (table4.col4_9 = 3.4028234663852886e+38:::FLOAT8)) AND (table4.col4_12 = e'\'':::STRING)) OR (table4.col4_1 > '"':::STRING)) AND (table4.col4_5 = e'\'':::STRING)) OR (table4.col4_7 != '':::STRING)) AND (table4.col4_10 = e'\'':::STRING)) OR (table4.col4_2 = e'\x00':::STRING)) AND (table4.col4_11 != 'X':::STRING)) AND (table4.col4_8 < 'NaN':::DECIMAL)) OR (table4.col4_0 >= 127:::INT8)) AND (table4.col4_6 != 1.401298464324817e-45:::FLOAT8), INDEX (col4_8, col4_5, col4_3 ASC, col4_4 ASC, col4_13 ASC, col4_0, col4_9) STORING (col4_1, col4_6), UNIQUE (col4_11 DESC, col4_4 DESC, col4_9 ASC, col4_8 ASC, col4_1 ASC, col4_12 ASC, col4_5 ASC, col4_10 DESC, col4_6) STORING (col4_0, col4_13) WHERE table4.col4_8 < 'NaN':::DECIMAL, INDEX (col4_1, col4_9 ASC, col4_10 DESC, col4_12, col4_5 ASC, col4_8 DESC, col4_11 ASC, col4_2 ASC, col4_0, col4_13, col4_6), FAMILY (col4_4, col4_9, col4_1, col4_5), FAMILY (col4_13, col4_0), FAMILY (col4_7, col4_3, col4_11, col4_2), FAMILY (col4_6));
        test_1        |             CREATE TABLE table5 (col5_0 DATE NOT NULL, PRIMARY KEY (col5_0));
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 779341759365265118, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1409999083362808734}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 980391586675773646, "histo_buckets": [{"distinct_range": 0, "num_eq": 700000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 145645790.2085263, "num_eq": 500, "num_range": 400000000, "upper_bound": "161652137"}, {"distinct_range": 372802.43722272577, "num_eq": 8445389699874004654, "num_range": 2000000, "upper_bound": "986491521"}, {"distinct_range": 2505323377913569300, "num_eq": 87059024437829036, "num_range": 2864137525348306715, "upper_bound": "1041071440"}, {"distinct_range": 1000, "num_eq": 2880232175876447430, "num_range": 1000, "upper_bound": "1111104415"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 1394610814978723619, "upper_bound": "2711948087"}, {"distinct_range": 2257272721186095900, "num_eq": 100, "num_range": 2257272721186095906, "upper_bound": "3847294542"}, {"distinct_range": 0, "num_eq": 70, "num_range": 50000000, "upper_bound": "4038937729"}, {"distinct_range": 91160661127.48825, "num_eq": 0, "num_range": 100000000000, "upper_bound": "4274177085"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 1409999083362808734}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1205985078961360820, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1409999083362808734}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 542412268248156611, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1409999083362808734}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 292180495046533169, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1409999083362808734}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 249004828643400622, "histo_buckets": [{"distinct_range": 0, "num_eq": 4759613940828711515, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 50000000, "num_eq": 80000000000, "num_range": 50000000, "upper_bound": "\\\\x22"}, {"distinct_range": 40000, "num_eq": 7374275842161344899, "num_range": 40000, "upper_bound": "\\\\x2c"}, {"distinct_range": 5363143179290103000, "num_eq": 80, "num_range": 7553743790288963798, "upper_bound": "\\\\x37"}, {"distinct_range": 0, "num_eq": 7318198017690148659, "num_range": 274533006056245076, "upper_bound": "\\\\x67"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 1409999083362808734}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 681749539447231978, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "-2147483648"}, {"distinct_range": 0, "num_eq": 5722156117720425545, "num_range": 100, "upper_bound": "-1718354478"}, {"distinct_range": 0, "num_eq": 2072783271708963102, "num_range": 4499850942567335254, "upper_bound": "-1238991256"}, {"distinct_range": 5864491056443193000, "num_eq": 1518777508698428073, "num_range": 8743946821661410935, "upper_bound": "-1105008590"}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 20, "upper_bound": "-620731886"}, {"distinct_range": 0, "num_eq": 50, "num_range": 5734630631145458387, "upper_bound": "-51719463"}, {"distinct_range": 0, "num_eq": 6000, "num_range": 9000000000, "upper_bound": "-128"}, {"distinct_range": 2396693726574920700, "num_eq": 60000000, "num_range": 5044492047372138534, "upper_bound": "2114570251"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 1217500760304129135, "row_count": 1409999083362808734}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 670010434590977581, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000, "num_range": 0, "upper_bound": "-1.234E+401"}, {"distinct_range": 8172040002096062000, "num_eq": 1494408193822028016, "num_range": 8172040002096062726, "upper_bound": "-3.743434283429810537E+20"}, {"distinct_range": 28300992074.994263, "num_eq": 10000000000, "num_range": 50000000000, "upper_bound": "-678057.7232799266078"}, {"distinct_range": 1922645808211344600, "num_eq": 7403554394143220573, "num_range": 2113108038779833093, "upper_bound": "4.833308489183698388E+36"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 1409999083362808734}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1060985529067522075, "histo_col_type": "", "name": "__auto__", "null_count": 743627104691973842, "row_count": 1409999083362808734}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1356273888542098702, "histo_buckets": [{"distinct_range": 0, "num_eq": 4644608350943246306, "num_range": 0, "upper_bound": "00111101000111111101001101000000100011011"}, {"distinct_range": 0, "num_eq": 4598517902279085996, "num_range": 5000000000, "upper_bound": "10010110101001100111010111010101101110100"}, {"distinct_range": 9047426321077150000, "num_eq": 6000, "num_range": 9047426321077149448, "upper_bound": "10110101100001010000111111000111001011010"}, {"distinct_range": 6189771382738620000, "num_eq": 7462193085404250126, "num_range": 6281777337495193948, "upper_bound": "11011001100001000001000000001011001100001"}, {"distinct_range": 3000, "num_eq": 8000000000, "num_range": 3000, "upper_bound": "11101010100101111110010000001001000001011"}], "histo_col_type": "BIT(41)", "name": "__auto__", "null_count": 0, "row_count": 1409999083362808734}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 434625593484024756, "histo_buckets": [{"distinct_range": 0, "num_eq": 90000000, "num_range": 0, "upper_bound": "\\u0016Oi\\n["}, {"distinct_range": 884.9969313640822, "num_eq": 10000000000, "num_range": 1000, "upper_bound": "%F\\u0012M\\u000b-b["}, {"distinct_range": 600000, "num_eq": 70000000000, "num_range": 600000, "upper_bound": ",oA]"}, {"distinct_range": 577728.4047516389, "num_eq": 100000, "num_range": 600000, "upper_bound": "7R\\u0006I\\u000fB"}, {"distinct_range": 0, "num_eq": 40000, "num_range": 400000000, "upper_bound": "S#]"}, {"distinct_range": 0, "num_eq": 0, "num_range": 8856032306280832641, "upper_bound": "_\\"8>"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 1132935580772309713, "row_count": 1409999083362808734}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 209592518880166295, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 484315.8014510311, "num_eq": 2071727046031368016, "num_range": 500000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 119256340190493119, "row_count": 739610630242558629}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 129842723690099075, "histo_buckets": [{"distinct_range": 0, "num_eq": 857469382596427107, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 2195917352700603263, "num_range": 400000, "upper_bound": "586925094"}, {"distinct_range": 0, "num_eq": 10000000, "num_range": 3404460231401028600, "upper_bound": "589904396"}, {"distinct_range": 0, "num_eq": 7928878644191082640, "num_range": 70000, "upper_bound": "1475043786"}, {"distinct_range": 0, "num_eq": 8000000000, "num_range": 8048004005463204255, "upper_bound": "2288280445"}, {"distinct_range": 3045350983889656300, "num_eq": 90000, "num_range": 7472901010330302853, "upper_bound": "2363209619"}, {"distinct_range": 0, "num_eq": 2358057338121408553, "num_range": 60000000, "upper_bound": "2922394410"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 739610630242558629}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 185130948947624258, "histo_buckets": [{"distinct_range": 0, "num_eq": 7359568289417843585, "num_range": 0, "upper_bound": "-6258687180946631874"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 739610630242558629}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 679139604304638553, "histo_buckets": [{"distinct_range": 0, "num_eq": 5615542985495751391, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 800.1803747638969, "num_eq": 200, "num_range": 900, "upper_bound": "\\\\x14d3"}, {"distinct_range": 824223664268894600, "num_eq": 5151740001760798965, "num_range": 5552534502210485509, "upper_bound": "\\\\xb60be3"}, {"distinct_range": 4220047624118884400, "num_eq": 7902640731140746817, "num_range": 4220047624118884600, "upper_bound": "\\\\xd3"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 739610630242558629}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 688683074591840825, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000, "num_range": 0, "upper_bound": "de\\n\\n{"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 739610630242558629}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 170104, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000, "num_range": 0, "upper_bound": "\\\\x026f"}, {"distinct_range": 8000, "num_eq": 90000000000, "num_range": 8000, "upper_bound": "\\\\x58"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 9118890954506919694, "upper_bound": "\\\\x5cdd67"}, {"distinct_range": 0, "num_eq": 4660858135318316289, "num_range": 40000000, "upper_bound": "\\\\x5e809aba307111ff1a"}, {"distinct_range": 196518273.16793475, "num_eq": 2000000000, "num_range": 400000000, "upper_bound": "\\\\x74b13a8d4c51"}, {"distinct_range": 7513554559714002000, "num_eq": 50000, "num_range": 7779969205124931278, "upper_bound": "\\\\xa93b28577b"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 89002, "histo_buckets": [{"distinct_range": 0, "num_eq": 1106714854605818912, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "X"}, {"distinct_range": 0, "num_eq": 300, "num_range": 8000, "upper_bound": "`"}, {"distinct_range": 0, "num_eq": 3820050879664551013, "num_range": 8664406818268105316, "upper_bound": "b}\\u000e\\u0010z\\u0007-"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 118264, "row_count": 200000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 37809, "histo_buckets": [{"distinct_range": 0, "num_eq": 5086270464633450428, "num_range": 0, "upper_bound": "297885721"}, {"distinct_range": 0, "num_eq": 1940985715253961830, "num_range": 1647502638697023898, "upper_bound": "1046089128"}, {"distinct_range": 2514985483604681700, "num_eq": 3128139436885990371, "num_range": 5981080090893512820, "upper_bound": "1269263721"}, {"distinct_range": 3331597703707675000, "num_eq": 7308798165652208456, "num_range": 5007524082008523945, "upper_bound": "1318807325"}, {"distinct_range": 0, "num_eq": 3877667443784020200, "num_range": 700000000, "upper_bound": "2342983607"}, {"distinct_range": 0, "num_eq": 70000, "num_range": 5266523240980789103, "upper_bound": "3617524619"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 200000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 123606, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 200000}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 10992, "histo_col_type": "", "name": "__auto__", "null_count": 20475, "row_count": 40000}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6151, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 23232, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 7294, "histo_col_type": "", "name": "__auto__", "null_count": 30381, "row_count": 40000}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35197, "histo_col_type": "", "name": "__auto__", "null_count": 21892, "row_count": 40000}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 35254, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36385, "histo_col_type": "", "name": "__auto__", "null_count": 8844, "row_count": 40000}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 32567, "histo_buckets": [{"distinct_range": 0, "num_eq": 2061621040239731109, "num_range": 0, "upper_bound": ""}, {"distinct_range": 2334123435713576000, "num_eq": 4000000000, "num_range": 3022677573332793858, "upper_bound": "OP\\u0001\'\'wv"}, {"distinct_range": 0, "num_eq": 9000, "num_range": 60000, "upper_bound": "[oK\\u0013\\u000f$Suy"}, {"distinct_range": 634579557727469400, "num_eq": 70000000000, "num_range": 5690340125949808402, "upper_bound": "k"}, {"distinct_range": 196281779421584500, "num_eq": 5000000000, "num_range": 676839405212139208, "upper_bound": "o\\u001f>.uz\\u0012"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9301, "histo_buckets": [{"distinct_range": 0, "num_eq": 2666849965165485488, "num_range": 0, "upper_bound": "-8.180875092718093719E+37"}, {"distinct_range": 0, "num_eq": 360190223666144539, "num_range": 5000000000, "upper_bound": "-2.281840029084504597E+35"}, {"distinct_range": 20000000, "num_eq": 100, "num_range": 20000000, "upper_bound": "-68354137769565.68337"}, {"distinct_range": 200000, "num_eq": 1000, "num_range": 200000, "upper_bound": "-400.7829224502864169"}, {"distinct_range": 0, "num_eq": 60000000, "num_range": 0, "upper_bound": "0.0207222342754315736"}, {"distinct_range": 0, "num_eq": 6248221119053856773, "num_range": 7487167482785038698, "upper_bound": "2937.281507158415795"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 16994, "row_count": 40000}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12299, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 10000000000, "num_range": 1000000000, "upper_bound": "\\u0007\\\\\\u00158o\\u000c}\\u000b\\u001f"}, {"distinct_range": 0, "num_eq": 2219310523133483880, "num_range": 0, "upper_bound": "\'"}, {"distinct_range": 1014118900181196800, "num_eq": 300000, "num_range": 3749752448853966598, "upper_bound": "2\\u0013\\u0007\\n~*\\u0012B"}, {"distinct_range": 261137696.81501117, "num_eq": 20000, "num_range": 700000000, "upper_bound": ";`F"}, {"distinct_range": 0, "num_eq": 100000, "num_range": 5000000000, "upper_bound": "nQP\\u00056H1.*"}, {"distinct_range": 0, "num_eq": 2270403562616972809, "num_range": 40000000000, "upper_bound": "u\\u000b+0\\u0003AM="}, {"distinct_range": 2648300107120143400, "num_eq": 6342987339138732759, "num_range": 2648300107120143223, "upper_bound": "z\\u0010xO\\u001eJ\\u0010\\u0015g"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 36428, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 300, "num_eq": 800000000, "num_range": 300, "upper_bound": "2*l\\u001az\\u0016B"}, {"distinct_range": 0, "num_eq": 80000, "num_range": 900000, "upper_bound": "H[y"}, {"distinct_range": 10000000000, "num_eq": 400, "num_range": 10000000000, "upper_bound": "[3A\\u0016~"}, {"distinct_range": 0, "num_eq": 100, "num_range": 3307703810233927131, "upper_bound": "]"}, {"distinct_range": 45.89680985149489, "num_eq": 2753140476647637830, "num_range": 100, "upper_bound": "d0>qX\\u0019|"}, {"distinct_range": 0, "num_eq": 4355737464185786815, "num_range": 6000000, "upper_bound": "hpK _\\u0019\\t\\u000bu"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 5955853508073429988, "upper_bound": "|\\u0016\x7fN"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 14357, "row_count": 40000}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4172, "histo_col_type": "", "name": "__auto__", "null_count": 25168, "row_count": 40000}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 39976, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 40000}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2090, "histo_col_type": "", "name": "__auto__", "null_count": 16314, "row_count": 40000}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 57076, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "1970-05-11"}, {"distinct_range": 0, "num_eq": 1000000, "num_range": 8146355200605621958, "upper_bound": "1972-10-23"}, {"distinct_range": 0, "num_eq": 4610919229611449883, "num_range": 50000, "upper_bound": "1979-12-14"}, {"distinct_range": 5299222817694684000, "num_eq": 10000, "num_range": 5299222817694684062, "upper_bound": "1988-10-03"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 60000}]':::JSONB;
        test_1        |             ALTER TABLE table4 ADD FOREIGN KEY (col4_2) REFERENCES table1 (col1_7);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('xwxdw', 'ulxqu', 'yi');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('qtlw', 'itykcl', 'camo', 'vorvbr', 'c');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('pnfoc', 'hc', 't', 'y', 'twav');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('rhgn', 'momnt', 'mu', 'mujy');
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19, "histo_buckets": [{"distinct_range": 0, "num_eq": 100, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 0, "num_eq": 60, "num_range": 30000000, "upper_bound": "\\\\x422ef6cc"}, {"distinct_range": 6963788515327767000, "num_eq": 2000000, "num_range": 7398475594286487831, "upper_bound": "\\\\x624c8f"}, {"distinct_range": 915227250037148800, "num_eq": 800000, "num_range": 6312165259595701767, "upper_bound": "\\\\xda"}, {"distinct_range": 3180664239128397300, "num_eq": 8844459467876802922, "num_range": 6897213599232990235, "upper_bound": "\\\\xecc2df674b0e"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 53, "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 61, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 51, "histo_buckets": [{"distinct_range": 0, "num_eq": 4000000000, "num_range": 0, "upper_bound": "6),?CwR#"}, {"distinct_range": 0, "num_eq": 1129077963309756654, "num_range": 10, "upper_bound": "N"}, {"distinct_range": 400, "num_eq": 8208563994175957275, "num_range": 400, "upper_bound": "X"}, {"distinct_range": 0, "num_eq": 1454636606061042083, "num_range": 5000, "upper_bound": "pOX_p"}, {"distinct_range": 493680684.47330856, "num_eq": 2627839470954049419, "num_range": 600000000, "upper_bound": "vn5"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 45, "histo_buckets": [{"distinct_range": 0, "num_eq": 4991160823807639506, "num_range": 0, "upper_bound": "\\""}, {"distinct_range": 0, "num_eq": 3153081281104032999, "num_range": 0, "upper_bound": "c\\u00183P\'P"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 43, "row_count": 80}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 19, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000, "num_range": 0, "upper_bound": "-379678270"}, {"distinct_range": 15613173742.120464, "num_eq": 3632701708318089549, "num_range": 50000000000, "upper_bound": "-54537946"}, {"distinct_range": 0, "num_eq": 3187873324599849427, "num_range": 60000, "upper_bound": "-128"}, {"distinct_range": 136.70409730117856, "num_eq": 100000000, "num_range": 200, "upper_bound": "385603477"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 55, "row_count": 80}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 30, "histo_col_type": "", "name": "__auto__", "null_count": 11, "row_count": 80}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5, "histo_col_type": "BIT(41)", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 9, "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 16, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 70, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6, "histo_buckets": [{"distinct_range": 0, "num_eq": 100000000, "num_range": 0, "upper_bound": "235211560"}, {"distinct_range": 0, "num_eq": 900000000, "num_range": 683795314978043882, "upper_bound": "1168733087"}, {"distinct_range": 5617086078336019000, "num_eq": 7552076013102831210, "num_range": 5617086078336019276, "upper_bound": "1711768102"}, {"distinct_range": 6537793871163855000, "num_eq": 5000000000, "num_range": 6537793871163854755, "upper_bound": "2878214313"}, {"distinct_range": 2232140598943398700, "num_eq": 200000000, "num_range": 2995929232051241486, "upper_bound": "3540686302"}, {"distinct_range": 0, "num_eq": 1304786902446155145, "num_range": 200, "upper_bound": "3748236380"}, {"distinct_range": 8821.124342808367, "num_eq": 5000, "num_range": 10000, "upper_bound": "4142370222"}, {"distinct_range": 1653941412986068200, "num_eq": 700000000, "num_range": 4291149449753764323, "upper_bound": "4234843578"}, {"distinct_range": 0, "num_eq": 3238247779790664859, "num_range": 1000, "upper_bound": "4267107626"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20, "histo_buckets": [{"distinct_range": 0, "num_eq": 700, "num_range": 0, "upper_bound": "-8.443938778824645968E+33"}, {"distinct_range": 5311769539511103000, "num_eq": 6000000000, "num_range": 7803868863615742079, "upper_bound": "-6.123723405643702989E+27"}, {"distinct_range": 0, "num_eq": 500000000, "num_range": 10, "upper_bound": "-1"}, {"distinct_range": 0, "num_eq": 70000000, "num_range": 50000000, "upper_bound": "3.751026448740773647E+35"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 50, "histo_buckets": [{"distinct_range": 0, "num_eq": 4989834150223648680, "num_range": 0, "upper_bound": "\\\\x"}, {"distinct_range": 1208016616962290000, "num_eq": 100, "num_range": 4069670534903126472, "upper_bound": "\\\\x0a5d5c3326"}, {"distinct_range": 5148379884980219000, "num_eq": 680389541230081070, "num_range": 5148379884980219219, "upper_bound": "\\\\x1db8c3e05451c2"}, {"distinct_range": 70000000, "num_eq": 0, "num_range": 70000000, "upper_bound": "\\\\x35f4"}, {"distinct_range": 200000, "num_eq": 300000000, "num_range": 200000, "upper_bound": "\\\\x4c5facd220b643bc7f"}, {"distinct_range": 4326324944041409000, "num_eq": 3799852017387979745, "num_range": 4326324944041409115, "upper_bound": "\\\\x5320f52ef6"}, {"distinct_range": 3244350613527728000, "num_eq": 60000, "num_range": 6324575971212999762, "upper_bound": "\\\\x5b8afe168ed1e7"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 22, "histo_buckets": [{"distinct_range": 0, "num_eq": 4339003749723429657, "num_range": 0, "upper_bound": ""}, {"distinct_range": 0, "num_eq": 7263068865101934401, "num_range": 300000, "upper_bound": "\\u0017\\u0015\\u0005[BB*`*"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 80}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 57, "histo_buckets": [{"distinct_range": 0, "num_eq": 6253869502689465968, "num_range": 0, "upper_bound": "false"}, {"distinct_range": 800000, "num_eq": 100000000000, "num_range": 800000, "upper_bound": "true"}], "histo_col_type": "BOOL", "name": "__auto__", "null_count": 32, "row_count": 80}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 44602, "histo_buckets": [{"distinct_range": 0, "num_eq": 10000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 600000000, "num_range": 3222965589713325416, "upper_bound": "110317825"}, {"distinct_range": 4300405914805141500, "num_eq": 5288439065855558051, "num_range": 7508653711865391401, "upper_bound": "140193054"}, {"distinct_range": 0, "num_eq": 4035492236953940998, "num_range": 90, "upper_bound": "1310729358"}, {"distinct_range": 400000, "num_eq": 0, "num_range": 400000, "upper_bound": "2341100119"}, {"distinct_range": 1038609416374270800, "num_eq": 50, "num_range": 3047823038167891688, "upper_bound": "2592453595"}, {"distinct_range": 1449567.6596684398, "num_eq": 10000000000, "num_range": 4000000, "upper_bound": "3148942111"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 27200, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 49746, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "\\\\x139de0"}, {"distinct_range": 0, "num_eq": 60, "num_range": 0, "upper_bound": "\\\\x2a8601"}, {"distinct_range": 0, "num_eq": 8945877060856131909, "num_range": 4124262286450667790, "upper_bound": "\\\\x6bfd7df1439c"}, {"distinct_range": 0, "num_eq": 9000, "num_range": 600, "upper_bound": "\\\\x796e6e6d"}, {"distinct_range": 4414774577149548000, "num_eq": 4227653815662267247, "num_range": 8098498345169228897, "upper_bound": "\\\\x8fd5"}, {"distinct_range": 8120330060364588000, "num_eq": 4212575290382902980, "num_range": 9073774602586964769, "upper_bound": "\\\\xdd079024b7"}, {"distinct_range": 49485.35214133558, "num_eq": 200, "num_range": 60000, "upper_bound": "\\\\xdd4f"}, {"distinct_range": 10000, "num_eq": 9000000000, "num_range": 10000, "upper_bound": "\\\\xe29883"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 80000}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46948, "histo_buckets": [{"distinct_range": 0, "num_eq": 6413486272802985788, "num_range": 0, "upper_bound": ""}, {"distinct_range": 500000, "num_eq": 1262698042411522472, "num_range": 500000, "upper_bound": "\\u0000"}, {"distinct_range": 0, "num_eq": 718486743089507343, "num_range": 10000, "upper_bound": "\\u0005 t8"}, {"distinct_range": 3736001196954237000, "num_eq": 1000000000, "num_range": 3736001196954236870, "upper_bound": "\\u00191l\\u0010"}, {"distinct_range": 0, "num_eq": 5705912971275120641, "num_range": 8000000, "upper_bound": "\\u001d\\\\o"}, {"distinct_range": 0, "num_eq": 5000, "num_range": 900, "upper_bound": "@\\\\I"}, {"distinct_range": 100, "num_eq": 7989842580194316545, "num_range": 100, "upper_bound": "\U00002603"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 49967, "row_count": 80000}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1956128801505669571, "histo_col_type": "", "name": "__auto__", "null_count": 3618654319011199478, "row_count": 7094810115744551056}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1273258375755459932, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7094810115744551056}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2096114597021473062, "histo_col_type": "", "name": "__auto__", "null_count": 384252982443888101, "row_count": 7094810115744551056}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1303407404809874091, "histo_buckets": [{"distinct_range": 0, "num_eq": 20, "num_range": 0, "upper_bound": "-4.943058176305935487E+34"}, {"distinct_range": 2431607119323173000, "num_eq": 7019034798332768244, "num_range": 3801674372424721108, "upper_bound": "73032.81950533030301"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 1358456731495851178, "row_count": 7094810115744551056}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4870139374007076500, "histo_col_type": "", "name": "__auto__", "null_count": 5612354704551398859, "row_count": 7094810115744551056}, {"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 686291831296498250, "histo_buckets": [{"distinct_range": 0, "num_eq": 2000000000, "num_range": 0, "upper_bound": "*"}, {"distinct_range": 9055657420014122000, "num_eq": 7529655449021661876, "num_range": 9055657420014122051, "upper_bound": "c"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 7094810115744551056}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2909425073325186950, "histo_col_type": "", "name": "__auto__", "null_count": 1212233021964848412, "row_count": 7094810115744551056}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3004919144524541116, "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 7094810115744551056}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5676271742844430710, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7094810115744551056}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 20514309559071773, "histo_col_type": "", "name": "__auto__", "null_count": 3363441722358524102, "row_count": 7094810115744551056}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 5499530383739234194, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7094810115744551056}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4957990722108338746, "histo_buckets": [{"distinct_range": 0, "num_eq": 600, "num_range": 0, "upper_bound": "\\u0006i"}, {"distinct_range": 555552782134151300, "num_eq": 8237462184123062632, "num_range": 2400202572250926824, "upper_bound": "\\u0010\\u0012>"}, {"distinct_range": 0, "num_eq": 1065125098352572314, "num_range": 7604111812984966624, "upper_bound": "&Nm\\u0004l\\\\\\u000b4$"}, {"distinct_range": 0, "num_eq": 3715610906094453588, "num_range": 70000, "upper_bound": "BL~D8<n>p"}, {"distinct_range": 0, "num_eq": 800000, "num_range": 0, "upper_bound": "_.w(4@|x"}, {"distinct_range": 0, "num_eq": 726819516047797424, "num_range": 4413888249351645786, "upper_bound": "w.f"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 2117827813578855991, "row_count": 7094810115744551056}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6001404746414841281, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 7094810115744551056}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 748995404173566647, "histo_col_type": "", "name": "__auto__", "null_count": 2291176706989109576, "row_count": 7094810115744551056}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 602499856, "histo_buckets": [{"distinct_range": 0, "num_eq": 5081862078371951905, "num_range": 0, "upper_bound": "1985-11-11"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 1000000000}]':::JSONB;
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 46800343379581664, "histo_buckets": [{"distinct_range": 0, "num_eq": 632914315060790138, "num_range": 0, "upper_bound": "\\u0004/b}\\u0003Y1"}, {"distinct_range": 2587365750982881300, "num_eq": 700000, "num_range": 2587365750982881191, "upper_bound": "\\""}, {"distinct_range": 1156592438275368700, "num_eq": 2000000, "num_range": 5814541564314739569, "upper_bound": "1id\x7f\\u001b6Jg"}, {"distinct_range": 2742384438720800000, "num_eq": 0, "num_range": 2742384438720799651, "upper_bound": "Ye0"}, {"distinct_range": 0, "num_eq": 4370387265350026438, "num_range": 8826268156047547215, "upper_bound": "v0\\"N\\u0003<u"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 123793581170984864, "row_count": 206084174004320133}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 204265791153551180, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 206084174004320133}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 91339217709705782, "histo_col_type": "", "name": "__auto__", "null_count": 4818490841738756, "row_count": 206084174004320133}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 90428532892772059, "histo_buckets": [{"distinct_range": 0, "num_eq": 7327988319405077265, "num_range": 0, "upper_bound": "01001101100100001001111000111111010011101"}, {"distinct_range": 0, "num_eq": 8000000, "num_range": 6000000, "upper_bound": "01011100101101111111111011001000011001001"}, {"distinct_range": 0, "num_eq": 90000000, "num_range": 100000000, "upper_bound": "01111111111111111111111111111111111111111"}, {"distinct_range": 8455517103795200000, "num_eq": 100000, "num_range": 8455517103795200016, "upper_bound": "10000001100011100100100000011111010001001"}, {"distinct_range": 1906166423641179400, "num_eq": 4000000000, "num_range": 1906166423641179440, "upper_bound": "10100010001111011000110011010100101010110"}], "histo_col_type": "BIT(41)", "name": "__auto__", "null_count": 0, "row_count": 206084174004320133}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 185337998297552193, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-Infinity"}, {"distinct_range": 2431711562730845000, "num_eq": 7073820652803213949, "num_range": 8207164823563999535, "upper_bound": "-1.234E+401"}, {"distinct_range": 3995065441312332300, "num_eq": 1840553980918629873, "num_range": 3995065441312332414, "upper_bound": "-2.23183034222884054E+36"}, {"distinct_range": 400, "num_eq": 2317667602788241684, "num_range": 400, "upper_bound": "-2.6368525433062396E+21"}, {"distinct_range": 400000000, "num_eq": 3737658998235099094, "num_range": 400000000, "upper_bound": "-3009794941537748.338"}, {"distinct_range": 3000000, "num_eq": 6668785980810417771, "num_range": 3000000, "upper_bound": "-3895.98022664188252"}, {"distinct_range": 13795.060719645036, "num_eq": 1000, "num_range": 20000, "upper_bound": "-1500.212514135593051"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 206084174004320133}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 94575892473639446, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 206084174004320133}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 111986741003048083, "histo_buckets": [{"distinct_range": 0, "num_eq": 6000000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 10000, "num_eq": 80000000, "num_range": 10000, "upper_bound": "62969008"}, {"distinct_range": 0, "num_eq": 118537875071834504, "num_range": 50000, "upper_bound": "2217864403"}, {"distinct_range": 0, "num_eq": 0, "num_range": 200000, "upper_bound": "4212506580"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 206084174004320133}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 114880461903445775, "histo_buckets": [{"distinct_range": 0, "num_eq": 20, "num_range": 0, "upper_bound": "R:K\\u001f+\\u001f"}, {"distinct_range": 2569645030186360300, "num_eq": 400000, "num_range": 2569645030186360571, "upper_bound": "jqP\\u001d"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 206084174004320133}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 139374908703871477, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 206084174004320133}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 12635062686388451, "histo_buckets": [{"distinct_range": 0, "num_eq": 10, "num_range": 0, "upper_bound": "\\\\xda"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 206084174004320133}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 174131557216991048, "histo_buckets": [{"distinct_range": 0, "num_eq": 8732534017349116963, "num_range": 0, "upper_bound": "-2047889531"}, {"distinct_range": 6081948.153016019, "num_eq": 200000, "num_range": 20000000, "upper_bound": "-1423524514"}, {"distinct_range": 0, "num_eq": 200000, "num_range": 3477759477231793931, "upper_bound": "326898494"}, {"distinct_range": 18.35161616844308, "num_eq": 2147273491010804761, "num_range": 30, "upper_bound": "342169235"}, {"distinct_range": 4469129423451180000, "num_eq": 6597837343549466746, "num_range": 4469129423451179908, "upper_bound": "1149206587"}, {"distinct_range": 0, "num_eq": 800, "num_range": 3000000000, "upper_bound": "1689574083"}], "histo_col_type": "INT4", "name": "__auto__", "null_count": 87047057234334408, "row_count": 206084174004320133}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2476510381835691132, "histo_buckets": [{"distinct_range": 0, "num_eq": 2411128997337931669, "num_range": 0, "upper_bound": "-1.540178216586091968E+36"}, {"distinct_range": 0, "num_eq": 6000000000, "num_range": 1518093582134766627, "upper_bound": "-4.74647389244868542E+27"}, {"distinct_range": 635542.4050378415, "num_eq": 8847078888542220100, "num_range": 40000000, "upper_bound": "1"}, {"distinct_range": 1187927718473232600, "num_eq": 40, "num_range": 1775480243278065551, "upper_bound": "4358.957796940943682"}, {"distinct_range": 0, "num_eq": 20000000000, "num_range": 9000000, "upper_bound": "1721981820418.491482"}, {"distinct_range": 0, "num_eq": 4606486202883880663, "num_range": 40, "upper_bound": "3929422692562661533"}, {"distinct_range": 1641653236.6875527, "num_eq": 586074534654823086, "num_range": 7000000000, "upper_bound": "2.373725479133311078E+19"}, {"distinct_range": 0, "num_eq": 921205380231531562, "num_range": 7592853224461924775, "upper_bound": "6.104017854230609104E+37"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 0, "row_count": 3565734306809611351}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2275883592995850645, "histo_buckets": [{"distinct_range": 0, "num_eq": 60, "num_range": 0, "upper_bound": "\\\\x32b9bacc2c"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 3565734306809611351}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2579564077650313062, "histo_buckets": [{"distinct_range": 0, "num_eq": 3192902640791950313, "num_range": 0, "upper_bound": "\\"x-y\\u0005/{"}, {"distinct_range": 6992800620587440000, "num_eq": 1170591967652479939, "num_range": 6992800620587440558, "upper_bound": "EHLAr\\u0015"}, {"distinct_range": 889680971509936600, "num_eq": 9000, "num_range": 896420210622595066, "upper_bound": "\\\\"}, {"distinct_range": 2579948415930332000, "num_eq": 6108418520822922973, "num_range": 3492894692328743803, "upper_bound": "t\\u0008-"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 3565734306809611351}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1288956359039924883, "histo_col_type": "BOOL", "name": "__auto__", "null_count": 3285587844079388361, "row_count": 3565734306809611351}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2705849966502180065, "histo_buckets": [{"distinct_range": 0, "num_eq": 3000000, "num_range": 0, "upper_bound": "0"}, {"distinct_range": 0, "num_eq": 598380259610731584, "num_range": 0, "upper_bound": "195108421"}, {"distinct_range": 800, "num_eq": 8290357182743977635, "num_range": 800, "upper_bound": "376114739"}, {"distinct_range": 11414367643.4194, "num_eq": 700, "num_range": 30000000000, "upper_bound": "816721882"}, {"distinct_range": 2408628.021309712, "num_eq": 2481233103945145475, "num_range": 7000000, "upper_bound": "2485694544"}, {"distinct_range": 45053.32583407228, "num_eq": 7801480431458689711, "num_range": 80000, "upper_bound": "4128188060"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 3565734306809611351}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS e'[{"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2748115206749005316, "histo_buckets": [{"distinct_range": 0, "num_eq": 8225670561652671238, "num_range": 0, "upper_bound": "628761282"}, {"distinct_range": 0, "num_eq": 6337573205029848081, "num_range": 0, "upper_bound": "904124577"}, {"distinct_range": 4337841477538937000, "num_eq": 6198581275835850552, "num_range": 4337841477538936582, "upper_bound": "1598103909"}, {"distinct_range": 290775983998738500, "num_eq": 500000000, "num_range": 722385066039427581, "upper_bound": "1656188448"}, {"distinct_range": 86092475803110530, "num_eq": 6000, "num_range": 341602083542031194, "upper_bound": "1944769077"}, {"distinct_range": 40, "num_eq": 5073673139784535023, "num_range": 40, "upper_bound": "2098466610"}, {"distinct_range": 5773668990717368000, "num_eq": 2254158437211466942, "num_range": 6679547571810441447, "upper_bound": "3988546809"}, {"distinct_range": 0, "num_eq": 3606161653822999947, "num_range": 100, "upper_bound": "4077762057"}], "histo_col_type": "REGTYPE", "name": "__auto__", "null_count": 0, "row_count": 6450419061145576383}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1620327590561277016, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 6450419061145576383}, {"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 6339458860127037333, "histo_buckets": [{"distinct_range": 0, "num_eq": 600000, "num_range": 0, "upper_bound": "\\\\x192701bfa6c3"}, {"distinct_range": 7256604169657637000, "num_eq": 312837204020687976, "num_range": 7256604169657636700, "upper_bound": "\\\\x4588fd67b7"}, {"distinct_range": 0, "num_eq": 5735478775806039032, "num_range": 0, "upper_bound": "\\\\x58"}, {"distinct_range": 7331270965.740283, "num_eq": 7000, "num_range": 100000000000, "upper_bound": "\\\\x67d7"}, {"distinct_range": 5.6242616985449505, "num_eq": 300, "num_range": 20, "upper_bound": "\\\\x6f70c3a907"}, {"distinct_range": 0, "num_eq": 4604652289795057498, "num_range": 800, "upper_bound": "\\\\x7340af2c37fd06fd73"}, {"distinct_range": 1055155585865518700, "num_eq": 6460523512373995803, "num_range": 1055155585865518681, "upper_bound": "\\\\xa63c31"}, {"distinct_range": 0, "num_eq": 5311751572576006921, "num_range": 5631846349491816900, "upper_bound": "\\\\xddeefccf"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 6450419061145576383}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4244564396661516436, "histo_buckets": [{"distinct_range": 0, "num_eq": 3720322189652175546, "num_range": 0, "upper_bound": ""}, {"distinct_range": 4053149504248591000, "num_eq": 1000000, "num_range": 4053149504248590771, "upper_bound": "rE.5Skak"}], "histo_col_type": "VARCHAR", "name": "__auto__", "null_count": 3467913848254061068, "row_count": 6450419061145576383}]':::JSONB;
        test_1        |             ALTER TABLE table4 INJECT STATISTICS e'[{"columns": ["col4_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1179679882436923761, "histo_buckets": [{"distinct_range": 0, "num_eq": 751619261562019957, "num_range": 0, "upper_bound": "\\u001bc%_"}, {"distinct_range": 0, "num_eq": 5436283273850806521, "num_range": 100000, "upper_bound": "("}, {"distinct_range": 4389480019391916000, "num_eq": 40000000000, "num_range": 4389480019391916149, "upper_bound": "Ds\\u0006\\u0013\\u000eOK!j"}, {"distinct_range": 2079062654180926000, "num_eq": 5460135719235028177, "num_range": 2079062654180925837, "upper_bound": "oY_Nf"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 1372828787278813890}, {"columns": ["col4_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 914262474863460063, "histo_col_type": "", "name": "__auto__", "null_count": 246490370715599316, "row_count": 1372828787278813890}, {"columns": ["col4_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 392778459400860268, "histo_buckets": [{"distinct_range": 0, "num_eq": 800000000, "num_range": 0, "upper_bound": ""}, {"distinct_range": 557062178156981900, "num_eq": 1000000000, "num_range": 557062178156981909, "upper_bound": "\\""}, {"distinct_range": 0, "num_eq": 3000000000, "num_range": 4974495770579833777, "upper_bound": "9\\u0015\\u0001"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 65391957016691266, "row_count": 1372828787278813890}, {"columns": ["col4_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 501368130974200809, "histo_col_type": "", "name": "__auto__", "null_count": 916158738453022556, "row_count": 1372828787278813890}, {"columns": ["col4_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 533070420523749928, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1372828787278813890}, {"columns": ["col4_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 66641581480858552, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1372828787278813890}, {"columns": ["col4_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 939928241980449584, "histo_buckets": [{"distinct_range": 0, "num_eq": 9000, "num_range": 0, "upper_bound": "\\u0000"}, {"distinct_range": 12194815189.305115, "num_eq": 0, "num_range": 30000000000, "upper_bound": "7\\u0001)A\\u0011=n90"}, {"distinct_range": 6136813677985244000, "num_eq": 400000, "num_range": 6426860579860204689, "upper_bound": "C"}, {"distinct_range": 0, "num_eq": 5000000000, "num_range": 4686442447129870962, "upper_bound": "a4\\u0001nR\\u000bB\\u001c"}, {"distinct_range": 740971996600175600, "num_eq": 100000, "num_range": 4858406065687465012, "upper_bound": "z\\u0008o\\u0015K\\u0004\\u0007c"}, {"distinct_range": 6231698268239826000, "num_eq": 300000000, "num_range": 6231698268239826263, "upper_bound": "|.\\u00105~\\u0017"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 0, "row_count": 1372828787278813890}, {"columns": ["col4_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1201617226366959741, "histo_col_type": "", "name": "__auto__", "null_count": 520507447402007537, "row_count": 1372828787278813890}, {"columns": ["col4_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1065792994515063346, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1372828787278813890}, {"columns": ["col4_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 784642903962837112, "histo_col_type": "", "name": "__auto__", "null_count": 1268538104019008437, "row_count": 1372828787278813890}, {"columns": ["col4_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 624306669034287787, "histo_col_type": "", "name": "__auto__", "null_count": 359614931714187026, "row_count": 1372828787278813890}, {"columns": ["col4_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 26685093685780711, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 1372828787278813890}, {"columns": ["col4_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1357101002251046705, "histo_buckets": [{"distinct_range": 0, "num_eq": 0, "num_range": 0, "upper_bound": "-86393.60008341344512"}, {"distinct_range": 0, "num_eq": 1701356523297893570, "num_range": 1584689772871091620, "upper_bound": "4559.24597242337213"}, {"distinct_range": 500, "num_eq": 10000, "num_range": 500, "upper_bound": "4292142.047113299783"}, {"distinct_range": 0, "num_eq": 4000, "num_range": 6261375532396745956, "upper_bound": "74922414604663.51132"}], "histo_col_type": "DECIMAL", "name": "__auto__", "null_count": 996583186537356682, "row_count": 1372828787278813890}, {"columns": ["col4_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1041917188758413858, "histo_col_type": "", "name": "__auto__", "null_count": 1345058368251322651, "row_count": 1372828787278813890}]':::JSONB;
        test_1        |             ALTER TABLE table5 INJECT STATISTICS '[{"columns": ["col5_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 21952569643878281, "histo_buckets": [{"distinct_range": 0, "num_eq": 2269862748521664587, "num_range": 0, "upper_bound": "1975-11-22"}, {"distinct_range": 0, "num_eq": 7707498981893055956, "num_range": 4546486472610191030, "upper_bound": "1977-03-19"}, {"distinct_range": 207430062795815360, "num_eq": 6475260529743481568, "num_range": 207430062795815346, "upper_bound": "1977-05-24"}], "histo_col_type": "DATE", "name": "__auto__", "null_count": 0, "row_count": 24924674112592355}]':::JSONB;
        test_1        |             
        test_1        |         compare_test.go:130: cockroach2: exec: ERROR: there is no unique constraint matching given keys for referenced table table1 (SQLSTATE 23503)
        test_1        | FAIL
        postgres_1    | 2021-03-23 05:11:38.299 UTC [85] LO

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2812259&tab=buildLog) on [master@cbebc6e05491c6951216993ed5e12e22504624f2](https://github.com/cockroachdb/cockroach/commits/cbebc6e05491c6951216993ed5e12e22504624f2):

```
        postgres_1    | You can change this by editing pg_hba.conf or using the option -A, or
        postgres_1    | --auth-local and --auth-host, the next time you run initdb.
        postgres_1    | waiting for server to start....2021-03-25 05:13:13.340 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-03-25 05:13:13.356 UTC [49] LOG:  database system was shut down at 2021-03-25 05:13:13 UTC
        postgres_1    | 2021-03-25 05:13:13.361 UTC [48] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2021-03-25 05:13:16.263 UTC [48] LOG:  received fast shutdown request
        postgres_1    | .2021-03-25 05:13:16.265 UTC [48] LOG:  aborting any active transactions
        postgres_1    | 2021-03-25 05:13:16.267 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2021-03-25 05:13:16.273 UTC [50] LOG:  shutting down
        postgres_1    | 2021-03-25 05:13:16.399 UTC [48] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-03-25 05:13:16.477 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-03-25 05:13:16.477 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-03-25 05:13:16.480 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-03-25 05:13:16.496 UTC [85] LOG:  database system was shut down at 2021-03-25 05:13:16 UTC
        postgres_1    | 2021-03-25 05:13:16.502 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (602.07s)
        test_1        |     --- FAIL: TestCompare/postgres (0.95s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.3:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #61683 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2816342&tab=buildLog) on [master@8b137b4f068a0d590a3e86ae91fd60eb84f2750a](https://github.com/cockroachdb/cockroach/commits/8b137b4f068a0d590a3e86ae91fd60eb84f2750a):

```
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_1 DESC);
        postgres_1    | 	CREATE INDEX ON table3 (col3_3 DESC, col3_0, col3_5 ASC, col3_2 ASC, col3_1, col3_4 ASC);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_0, col3_4, col3_1 ASC, col3_2 DESC, col3_5 DESC, col3_3 ASC);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_3 DESC, col3_5 ASC, col3_1, col3_0, col3_2 ASC);
        postgres_1    | 	CREATE INDEX ON table3 (col3_4 DESC, col3_3 ASC, col3_1 ASC, col3_5, col3_0 ASC, col3_2);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_3 ASC, col3_4 DESC);
        postgres_1    | 	CREATE TABLE table4 (col4_0 REGCLASS NULL, col4_1 BYTEA, col4_2 TEXT NOT NULL, col4_3 INET NOT NULL, col4_4 INT8[], col4_5 "char" NOT NULL, col4_6 INET NULL, col4_7 NAME NOT NULL, col4_8 TEXT NULL GENERATED ALWAYS AS (IF(col4_4 IS NULL, e'\x15l'::TEXT, ''::TEXT)) STORED);
        postgres_1    | 	CREATE INDEX ON table4 (col4_3 DESC, col4_6 DESC);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('zivmw');
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('ylpye', 'tmstgh', 'vvk');
        postgres_1    | 	CREATE TYPE rand_typ_2 AS ENUM ('ru', 'iopf', 'mtnzpp');
        postgres_1    | 	CREATE TYPE rand_typ_3 AS ENUM ('q');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (603.89s)
        test_1        |     --- FAIL: TestCompare/postgres (2.42s)
        test_1        |         compare_test.go:129: CREATE TABLE table1 (col1_0 REGNAMESPACE NOT NULL, col1_1 JSONB NOT NULL);
        test_1        |             CREATE TABLE table2 (col2_0 TIMESTAMPTZ NOT NULL, col2_1 BIT(37) NOT NULL, PRIMARY KEY (col2_1, col2_0));
        test_1        |             CREATE INDEX ON table2 (col2_0 DESC, col2_1 ASC);
        test_1        |             CREATE INDEX ON table2 (col2_1 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_1);
        test_1        |             CREATE INDEX ON table2 (col2_1 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_0, col2_1 ASC);
        test_1        |             CREATE TABLE table3 (col3_0 DATE, col3_1 TIME NOT NULL, col3_2 NAME NULL, col3_3 OID NULL, col3_4 OID NULL, col3_5 INT8 NOT NULL, PRIMARY KEY (col3_5));
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 ASC, col3_4 ASC, col3_1 DESC) INCLUDE (col3_3);
        test_1        |             CREATE INDEX ON table3 (col3_5, col3_0 DESC, col3_3 ASC, col3_4 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_1 DESC);
        test_1        |             CREATE INDEX ON table3 (col3_3 DESC, col3_0, col3_5 ASC, col3_2 ASC, col3_1, col3_4 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0, col3_4, col3_1 ASC, col3_2 DESC, col3_5 DESC, col3_3 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_3 DESC, col3_5 ASC, col3_1, col3_0, col3_2 ASC);
        test_1        |             CREATE INDEX ON table3 (col3_4 DESC, col3_3 ASC, col3_1 ASC, col3_5, col3_0 ASC, col3_2);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_3 ASC, col3_4 DESC);
        test_1        |             CREATE TABLE table4 (col4_0 REGCLASS NULL, col4_1 BYTEA, col4_2 TEXT NOT NULL, col4_3 INET NOT NULL, col4_4 INT8[], col4_5 "char" NOT NULL, col4_6 INET NULL, col4_7 NAME NOT NULL, col4_8 TEXT NULL GENERATED ALWAYS AS (IF(col4_4 IS NULL, e'\x15l'::TEXT, ''::TEXT)) STORED);
        test_1        |             CREATE INDEX ON table4 (col4_3 DESC, col4_6 DESC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('zivmw');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('ylpye', 'tmstgh', 'vvk');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('ru', 'iopf', 'mtnzpp');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('q');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: syntax error at or near "(" (SQLSTATE 42601)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #62602 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2819726&tab=buildLog) on [master@3cfe2a38044b9e0d47b09815658e8634e4f4bfda](https://github.com/cockroachdb/cockroach/commits/3cfe2a38044b9e0d47b09815658e8634e4f4bfda):

```
        postgres_1    | You can change this by editing pg_hba.conf or using the option -A, or
        postgres_1    | --auth-local and --auth-host, the next time you run initdb.
        postgres_1    | waiting for server to start....2021-03-27 05:11:46.928 UTC [48] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-03-27 05:11:46.944 UTC [49] LOG:  database system was shut down at 2021-03-27 05:11:46 UTC
        postgres_1    | 2021-03-27 05:11:46.950 UTC [48] LOG:  database system is ready to accept connections
        postgres_1    |  done
        postgres_1    | server started
        postgres_1    | 
        postgres_1    | /usr/local/bin/docker-entrypoint.sh: sourcing /docker-entrypoint-initdb.d/10_postgis.sh
        postgres_1    | CREATE DATABASE
        postgres_1    | Loading PostGIS extensions into template_postgis
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | Loading PostGIS extensions into postgres
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | waiting for server to shut down...2021-03-27 05:11:49.872 UTC [48] LOG:  received fast shutdown request
        postgres_1    | .2021-03-27 05:11:49.874 UTC [48] LOG:  aborting any active transactions
        postgres_1    | 2021-03-27 05:11:49.876 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2021-03-27 05:11:49.885 UTC [50] LOG:  shutting down
        postgres_1    | 2021-03-27 05:11:49.996 UTC [48] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-03-27 05:11:50.085 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-03-27 05:11:50.085 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-03-27 05:11:50.089 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-03-27 05:11:50.105 UTC [85] LOG:  database system was shut down at 2021-03-27 05:11:49 UTC
        postgres_1    | 2021-03-27 05:11:50.111 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (601.53s)
        test_1        |     --- FAIL: TestCompare/postgres (0.44s)
        test_1        |         compare_test.go:119: pgx conn: dial tcp 172.18.0.3:5432: connect: connection refused
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #62602 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2821321&tab=buildLog) on [master@4f0629742ff665f8f6a60be60a5ecc5e4c248085](https://github.com/cockroachdb/cockroach/commits/4f0629742ff665f8f6a60be60a5ecc5e4c248085):

```
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col2_8 ASC, col3_25 ASC, col2_17, col3_19 DESC, col3_26 DESC, col2_5, col3_18 DESC, col2_12 DESC, col3_21 DESC);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col2_12 DESC);
        postgres_1    | 	CREATE INDEX ON table3 (col3_25, col2_17, col3_23 DESC, col3_24 DESC, col3_20 ASC, col2_9 DESC, col2_11 DESC, col3_21 DESC, col2_12 ASC, col2_7 ASC, col2_8 DESC, col3_19 ASC, col1_2 DESC, col2_3 DESC, col2_14, col2_13 ASC, col3_26, col3_18 DESC, col2_15, col2_5, col2_2 DESC, col2_16 DESC, col2_6, col3_22 DESC, col2_18 ASC, col1_3 ASC, col2_10 ASC);
        postgres_1    | 	CREATE INDEX ON table3 (col2_8 DESC, col2_2, col2_13 ASC, col3_23 DESC, col2_15 ASC, col2_3 DESC, col2_14 ASC, col2_16 DESC, col2_5 ASC, col1_2, col3_25, col2_11 ASC);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('vvgndk', 'ra', 'bpl');
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('ow');
        postgres_1    | 	CREATE TYPE rand_typ_2 AS ENUM ('qkg', 'o', 'jfl', 'ms', 'rgqzzm');
        postgres_1    | 	CREATE TYPE rand_typ_3 AS ENUM ('hu', 'n', 'bu', 'pmoxvs', 'fs');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (603.58s)
        test_1        |     --- FAIL: TestCompare/postgres (2.62s)
        test_1        |         compare_test.go:129: CREATE TABLE table1 (col1_0 TIME, col1_1 FLOAT8[] NOT NULL, col1_2 INT8 NOT NULL, col1_3 INT8 NOT NULL, col1_4 VARCHAR NULL, col1_5 TIMESTAMP, col1_6 DECIMAL NULL, col1_7 TEXT, col1_8 DATE, PRIMARY KEY (col1_2, col1_3), col1_9 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col1_7 AS TEXT))) STORED, col1_10 TEXT NOT NULL GENERATED ALWAYS AS (lower(CAST(col1_5 AS TEXT))) STORED, col1_11 TEXT NOT NULL GENERATED ALWAYS AS (lower(CAST(col1_5 AS TEXT))) STORED, col1_12 TEXT GENERATED ALWAYS AS (lower(CAST(col1_5 AS TEXT))) STORED, col1_13 INT8 GENERATED ALWAYS AS (col1_3 + col1_2) STORED, col1_14 INT8 NOT NULL GENERATED ALWAYS AS (col1_3 + 307757417::INT8) STORED);
        test_1        |             ALTER TABLE table1 ALTER COLUMN col1_7 SET DATA TYPE TEXT;
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_8, col1_12 ASC, col1_6 DESC);
        test_1        |             CREATE TABLE table2 (col1_2 INT8 NOT NULL, col1_3 INT8 NOT NULL, col2_2 INT8, col2_3 INT8 NOT NULL, col2_4 BYTEA NULL, col2_5 REGTYPE, col2_6 TIMETZ NOT NULL, col2_7 TEXT NOT NULL, col2_8 BIT(49), col2_9 FLOAT8 NULL, col2_10 REGTYPE NULL, col2_11 REGPROC NOT NULL, col2_12 INET NULL, col2_13 REGTYPE NOT NULL, col2_14 DECIMAL NULL, col2_15 NAME, col2_16 INT8, col2_17 TEXT, col2_18 BOOL NOT NULL, col2_19 FLOAT8, col2_20 TIMETZ NOT NULL, PRIMARY KEY (col1_2, col1_3, col2_17, col2_18, col2_13, col2_8, col2_10, col2_7, col2_9, col2_3, col2_15, col2_2, col2_16, col2_6, col2_12, col2_14, col2_5, col2_11));
        test_1        |             ALTER TABLE table2 ALTER COLUMN col2_7 SET DATA TYPE TEXT;
        test_1        |             ALTER TABLE table2 ALTER COLUMN col2_17 SET DATA TYPE TEXT;
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_3 ASC, col2_15, col2_20 DESC, col2_19 DESC, col1_2 DESC);
        test_1        |             CREATE INDEX ON table2 (col2_2 DESC, col1_3 DESC, col2_6, col2_14 ASC, col2_9 ASC, col2_16, col2_12 DESC, col2_5 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_3 DESC, col2_20, col2_17 DESC, col2_15 ASC, col2_13 DESC, col2_16 ASC, col2_7, col2_3, col2_11 DESC, col2_18 ASC, col2_9 ASC, col2_8 ASC, col2_19, col2_14 DESC, col2_10 ASC, col2_5 ASC, col2_12, col2_4, col2_2, col2_6, col1_2 DESC);
        test_1        |             CREATE INDEX ON table2 (col2_5 DESC, col2_6, col2_17 DESC, col2_16 ASC, col1_2, col2_3, col2_14 DESC, col2_10 ASC, col2_4 ASC, col2_20, col2_2 DESC, col2_9 DESC, col2_11, col2_7 DESC, col2_13 ASC, col2_12 ASC, col2_18 ASC, col1_3 ASC);
        test_1        |             CREATE INDEX ON table2 (col2_10, col1_3 ASC, col2_6 ASC, col2_18 DESC, col2_12, col2_3 DESC, col2_20, col2_4, col2_9 DESC, col2_11, col2_2 DESC, col2_15 ASC, col2_5 DESC, col2_17 ASC, col2_16 DESC);
        test_1        |             CREATE TABLE table3 (col1_2 INT8 NOT NULL, col1_3 INT8 NOT NULL, col2_17 TEXT, col2_18 BOOL NOT NULL, col2_13 REGTYPE NOT NULL, col2_8 BIT(49), col2_10 REGTYPE NULL, col2_7 TEXT NOT NULL, col2_9 FLOAT8 NULL, col2_3 INT8 NOT NULL, col2_15 NAME, col2_2 INT8, col2_16 INT8, col2_6 TIMETZ NOT NULL, col2_12 INET NULL, col2_14 DECIMAL NULL, col2_5 REGTYPE, col2_11 REGPROC NOT NULL, col3_18 DECIMAL, col3_19 BIT(30) NOT NULL, col3_20 UUID NULL, col3_21 TIMESTAMP NULL, col3_22 VARCHAR NOT NULL, col3_23 REGCLASS, col3_24 REGTYPE NULL, col3_25 BOOL, col3_26 VARCHAR NULL, PRIMARY KEY (col1_2, col1_3, col2_17, col2_18, col2_13, col2_8, col2_10, col2_7, col2_9, col2_3, col2_15, col2_2, col2_16, col2_6, col2_12, col2_14, col2_5, col2_11, col3_20, col3_19, col3_22, col3_21, col3_24, col3_25, col3_18));
        test_1        |             ALTER TABLE table3 ALTER COLUMN col2_17 SET DATA TYPE TEXT;
        test_1        |             ALTER TABLE table3 ALTER COLUMN col2_7 SET DATA TYPE TEXT;
        test_1        |             CREATE UNIQUE INDEX ON table3 (col2_6 DESC, col2_2, col2_18 DESC, col2_14, col2_10 ASC, col3_19 ASC, col2_13 ASC, col3_26 ASC, col2_8 DESC, col3_18 DESC, col3_23 ASC, col3_20 ASC, col1_2, col2_15 ASC, col3_21 ASC, col2_7 DESC, col2_5 ASC, col3_24 ASC, col3_25 ASC, col2_11, col2_3 ASC);
        test_1        |             CREATE INDEX ON table3 (col3_18 ASC, col2_17 DESC, col2_8, col2_16, col2_7 ASC, col3_25 ASC, col1_3, col3_26 DESC, col3_20 DESC, col3_24 DESC, col2_9 DESC, col2_6 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col2_14 DESC, col2_18 DESC, col3_24 DESC, col3_19 DESC, col2_11 ASC, col2_12, col3_22 ASC, col2_17 DESC, col1_2 ASC, col3_21 DESC, col2_5 DESC, col2_3 ASC, col2_15, col2_8 DESC, col2_7, col3_23, col2_9 ASC, col3_26 ASC, col2_2 ASC, col3_20 DESC, col1_3 ASC, col3_25, col2_16 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col2_9, col3_26, col2_12, col3_21 DESC, col2_7, col2_8 DESC, col2_10 DESC, col3_24 DESC, col2_16 DESC, col2_14 ASC, col3_20, col2_17 DESC, col2_5, col2_2 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col2_8 ASC, col3_25 ASC, col2_17, col3_19 DESC, col3_26 DESC, col2_5, col3_18 DESC, col2_12 DESC, col3_21 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col2_12 DESC);
        test_1        |             CREATE INDEX ON table3 (col3_25, col2_17, col3_23 DESC, col3_24 DESC, col3_20 ASC, col2_9 DESC, col2_11 DESC, col3_21 DESC, col2_12 ASC, col2_7 ASC, col2_8 DESC, col3_19 ASC, col1_2 DESC, col2_3 DESC, col2_14, col2_13 ASC, col3_26, col3_18 DESC, col2_15, col2_5, col2_2 DESC, col2_16 DESC, col2_6, col3_22 DESC, col2_18 ASC, col1_3 ASC, col2_10 ASC);
        test_1        |             CREATE INDEX ON table3 (col2_8 DESC, col2_2, col2_13 ASC, col3_23 DESC, col2_15 ASC, col2_3 DESC, col2_14 ASC, col2_16 DESC, col2_5 ASC, col1_2, col3_25, col2_11 ASC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('vvgndk', 'ra', 'bpl');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('ow');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('qkg', 'o', 'jfl', 'ms', 'rgqzzm');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('hu', 'n', 'bu', 'pmoxvs', 'fs');
        test_1        |             
        test_1        |         compare_test.go:130: postgres: exec: ERROR: syntax error at or near "(" (SQLSTATE 42601)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #62602 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2841672&tab=buildLog) on [master@9891b8db5ea2e9339a04fd2731a16658103e969f](https://github.com/cockroachdb/cockroach/commits/9891b8db5ea2e9339a04fd2731a16658103e969f):

```
        postgres_1    | 						AND (tab_50170.col1_0) = (tab_50171.col1_0)
        postgres_1    | 						AND (tab_50170.col1_8) = (tab_50171.col1_8)
        postgres_1    | 			WHERE
        postgres_1    | 				NULL
        postgres_1    | 		)
        postgres_1    | 			AS tab_50172
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_93257 NULLS FIRST,
        postgres_1    | 		col_93258 NULLS FIRST,
        postgres_1    | 		col_93259 NULLS FIRST,
        postgres_1    | 		col_93260 NULLS FIRST,
        postgres_1    | 		col_93261 NULLS FIRST,
        postgres_1    | 		col_93262 NULLS FIRST,
        postgres_1    | 		col_93263 NULLS FIRST,
        postgres_1    | 		col_93264 NULLS FIRST,
        postgres_1    | 		col_93265 NULLS FIRST,
        postgres_1    | 		col_93266 NULLS FIRST,
        postgres_1    | 		col_93267 NULLS FIRST,
        postgres_1    | 		col_93268 NULLS FIRST
        test_1        | --- FAIL: TestCompare (666.87s)
        test_1        |     compare_test.go:117: Checking connection to: postgres
        test_1        |     compare_test.go:117: Checking connection to: cockroach1
        test_1        |     compare_test.go:117: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (63.75s)
        test_1        |         compare_test.go:126: starting test: postgres
        test_1        |         compare_test.go:133: initializing connection: cockroach1
        test_1        |         compare_test.go:133: initializing connection: postgres
        test_1        |         compare_test.go:176: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("0.5.0 (Internal)"),
        test_1        |             + 	string("0.4.3 (Internal)"),
        test_1        |               }))
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT postgis_wagyu_version()::TEXT AS col_93482) AS tab_50285 ORDER BY col_93482 NULLS FIRST;
        test_1        |             
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT postgis_wagyu_version()::STRING AS col_93482) AS tab_50285 ORDER BY col_93482 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #62602 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2844890&tab=buildLog) on [master@df07592e26735be7362ef2b815ee163ead00739e](https://github.com/cockroachdb/cockroach/commits/df07592e26735be7362ef2b815ee163ead00739e):

```
        postgres_1    | 	CREATE INDEX ON table2 (col2_13, col2_0 ASC, col2_17 ASC, col2_4, col2_3 ASC, col2_1 DESC, col2_16, col2_8 ASC, col2_6, col2_2 DESC, col2_7 DESC, col2_12 DESC) INCLUDE (col2_9, col2_11, col2_15);
        postgres_1    | 	CREATE INDEX ON table2 (col2_0 ASC) INCLUDE (col2_1, col2_2, col2_3, col2_5, col2_6, col2_9, col2_10, col2_11, col2_12, col2_15);
        postgres_1    | 	CREATE INDEX ON table2 (col2_13 ASC, col2_11 ASC, col2_12 DESC, col2_3, col2_17 DESC, col2_6, col2_1 DESC, col2_7 DESC) INCLUDE (col2_2, col2_4, col2_9, col2_16);
        postgres_1    | 	CREATE TABLE table3 (col1_1 BOOL NOT NULL, col3_1 TIMETZ NOT NULL, col3_2 TEXT NOT NULL, col3_3 TIMESTAMPTZ, col3_4 BIT(44) NOT NULL, col3_5 DECIMAL, col3_6 BIT(42), col3_7 OID NULL, col3_8 INT8 NULL, col3_9 TIME, col3_10 REGPROC, col3_11 INT8 NULL, col3_12 VARCHAR, col3_13 "char" NOT NULL, col3_14 INT8, col3_15 REGPROCEDURE NOT NULL, col3_16 REGPROCEDURE NULL, col3_17 CHAR, col3_18 TIMETZ NULL, PRIMARY KEY (col1_1, col3_1));
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_6 ASC, col3_15 DESC, col3_3, col3_16 DESC, col3_14 DESC, col1_1 DESC, col3_17 DESC, col3_13 ASC, col3_8, col3_10 DESC, col3_9 ASC, col3_2, col3_11 DESC, col3_12 ASC) INCLUDE (col3_4, col3_7, col3_18);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_3, col3_13, col3_11, col3_8 DESC, col3_7, col3_16 DESC, col1_1, col3_10 ASC, col3_17, col3_5 DESC, col3_15 DESC, col3_2 DESC, col3_4, col3_6, col3_18 ASC, col3_14, col3_1, col3_12);
        postgres_1    | 	CREATE INDEX ON table3 (col3_4 DESC, col3_5 ASC, col3_10 ASC, col3_1, col3_14 DESC, col3_12 ASC);
        postgres_1    | 	CREATE TABLE table4 (col4_0 OID NOT NULL, col4_1 BIT(13) NOT NULL, col4_2 DATE NOT NULL, col4_3 BIT(42) NOT NULL, col4_4 GEOMETRY NULL, PRIMARY KEY (col4_0, col4_3, col4_2), col4_5 TEXT NOT NULL GENERATED ALWAYS AS (lower(CAST(col4_3 AS TEXT))) STORED, col4_6 TEXT GENERATED ALWAYS AS (lower(CAST(col4_3 AS TEXT))) STORED, col4_7 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col4_2 AS TEXT))) STORED, col4_8 TEXT GENERATED ALWAYS AS (lower(CAST(col4_3 AS TEXT))) STORED);
        postgres_1    | 	CREATE INDEX ON table4 (col4_0, col4_3 ASC, col4_6 ASC, col4_5, col4_2, col4_8);
        postgres_1    | 	CREATE UNIQUE INDEX ON table4 (col4_5 ASC, col4_2 DESC, col4_1 ASC, col4_6, col4_8 ASC);
        postgres_1    | 	CREATE INDEX ON table4 (col4_7 ASC, col4_1, col4_2, col4_8) INCLUDE (col4_6);
        postgres_1    | 	CREATE UNIQUE INDEX ON table4 (col4_7 ASC) INCLUDE (col4_1, col4_4, col4_6);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('umb', 'td', 'wvxbpk', 'cgnv');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (606.30s)
        test_1        |     compare_test.go:117: Checking connection to: postgres
        test_1        |     compare_test.go:117: Checking connection to: cockroach1
        test_1        |     compare_test.go:117: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (2.38s)
        test_1        |         compare_test.go:126: starting test: postgres
        test_1        |         compare_test.go:133: initializing connection: cockroach1
        test_1        |         compare_test.go:133: initializing connection: postgres
        test_1        |         compare_test.go:150: CREATE TABLE table1 (col1_0 INT8, col1_1 BOOL NOT NULL, PRIMARY KEY (col1_1), col1_2 INT8 NOT NULL GENERATED ALWAYS AS (col1_0 + (-1834074160)::INT8) STORED);
        test_1        |             CREATE TABLE table2 (col2_0 NAME NULL, col2_1 BOOL NOT NULL, col2_2 DATE NULL, col2_3 INT8 NOT NULL, col2_4 REGPROC, col2_5 BOX2D NOT NULL, col2_6 REGPROCEDURE, col2_7 FLOAT8 NOT NULL, col2_8 BYTEA, col2_9 GEOGRAPHY NOT NULL, col2_10 INT8, col2_11 INT8 NOT NULL GENERATED ALWAYS AS (col2_10 + col2_3) STORED, col2_12 INT8 GENERATED ALWAYS AS (col2_10 + col2_3) STORED, col2_13 TEXT GENERATED ALWAYS AS (lower(CAST(col2_1 AS TEXT))) STORED, col2_14 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col2_5 AS TEXT))) STORED, col2_15 INT8 GENERATED ALWAYS AS (col2_3 + col2_10) STORED, col2_16 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col2_4 IS NULL THEN NULL ELSE NULL END) STORED, col2_17 INT8 GENERATED ALWAYS AS (col2_10 + (-1247273749)::INT8) STORED);
        test_1        |             CREATE INDEX ON table2 (col2_13, col2_0 ASC, col2_17 ASC, col2_4, col2_3 ASC, col2_1 DESC, col2_16, col2_8 ASC, col2_6, col2_2 DESC, col2_7 DESC, col2_12 DESC) INCLUDE (col2_9, col2_11, col2_15);
        test_1        |             CREATE INDEX ON table2 (col2_0 ASC) INCLUDE (col2_1, col2_2, col2_3, col2_5, col2_6, col2_9, col2_10, col2_11, col2_12, col2_15);
        test_1        |             CREATE INDEX ON table2 (col2_13 ASC, col2_11 ASC, col2_12 DESC, col2_3, col2_17 DESC, col2_6, col2_1 DESC, col2_7 DESC) INCLUDE (col2_2, col2_4, col2_9, col2_16);
        test_1        |             CREATE TABLE table3 (col1_1 BOOL NOT NULL, col3_1 TIMETZ NOT NULL, col3_2 TEXT NOT NULL, col3_3 TIMESTAMPTZ, col3_4 BIT(44) NOT NULL, col3_5 DECIMAL, col3_6 BIT(42), col3_7 OID NULL, col3_8 INT8 NULL, col3_9 TIME, col3_10 REGPROC, col3_11 INT8 NULL, col3_12 VARCHAR, col3_13 "char" NOT NULL, col3_14 INT8, col3_15 REGPROCEDURE NOT NULL, col3_16 REGPROCEDURE NULL, col3_17 CHAR, col3_18 TIMETZ NULL, PRIMARY KEY (col1_1, col3_1));
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_6 ASC, col3_15 DESC, col3_3, col3_16 DESC, col3_14 DESC, col1_1 DESC, col3_17 DESC, col3_13 ASC, col3_8, col3_10 DESC, col3_9 ASC, col3_2, col3_11 DESC, col3_12 ASC) INCLUDE (col3_4, col3_7, col3_18);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_3, col3_13, col3_11, col3_8 DESC, col3_7, col3_16 DESC, col1_1, col3_10 ASC, col3_17, col3_5 DESC, col3_15 DESC, col3_2 DESC, col3_4, col3_6, col3_18 ASC, col3_14, col3_1, col3_12);
        test_1        |             CREATE INDEX ON table3 (col3_4 DESC, col3_5 ASC, col3_10 ASC, col3_1, col3_14 DESC, col3_12 ASC);
        test_1        |             CREATE TABLE table4 (col4_0 OID NOT NULL, col4_1 BIT(13) NOT NULL, col4_2 DATE NOT NULL, col4_3 BIT(42) NOT NULL, col4_4 GEOMETRY NULL, PRIMARY KEY (col4_0, col4_3, col4_2), col4_5 TEXT NOT NULL GENERATED ALWAYS AS (lower(CAST(col4_3 AS TEXT))) STORED, col4_6 TEXT GENERATED ALWAYS AS (lower(CAST(col4_3 AS TEXT))) STORED, col4_7 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col4_2 AS TEXT))) STORED, col4_8 TEXT GENERATED ALWAYS AS (lower(CAST(col4_3 AS TEXT))) STORED);
        test_1        |             CREATE INDEX ON table4 (col4_0, col4_3 ASC, col4_6 ASC, col4_5, col4_2, col4_8);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_5 ASC, col4_2 DESC, col4_1 ASC, col4_6, col4_8 ASC);
        test_1        |             CREATE INDEX ON table4 (col4_7 ASC, col4_1, col4_2, col4_8) INCLUDE (col4_6);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_7 ASC) INCLUDE (col4_1, col4_4, col4_6);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('umb', 'td', 'wvxbpk', 'cgnv');
        test_1        |             
        test_1        |         compare_test.go:151: postgres: exec: ERROR: generation expression is not immutable (SQLSTATE 42P17)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #62602 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2846455&tab=buildLog) on [master@6e5dfc46eab59f7fd6e77ee5dbab5bac715330de](https://github.com/cockroachdb/cockroach/commits/6e5dfc46eab59f7fd6e77ee5dbab5bac715330de):

```
        postgres_1    | 2021-04-04 05:26:16.766 UTC [125] STATEMENT:  SELECT
        postgres_1    | 		*
        postgres_1    | 	FROM
        postgres_1    | 		(
        postgres_1    | 			SELECT
        postgres_1    | 				extract('E'::TEXT::TEXT, '2000-01-01 04:00:00'::TIMESTAMP::TIMESTAMP)::FLOAT8 AS col_28443,
        postgres_1    | 				tab_17540.col3_3 AS col_28444
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table2 AS tab_17539
        postgres_1    | 				JOIN postgres.public.table3 AS tab_17540 ON
        postgres_1    | 						(tab_17539.col2_0) = (tab_17540.col3_5),
        postgres_1    | 				postgres.public.table3 AS tab_17541
        postgres_1    | 				JOIN postgres.public.table3 AS tab_17542 ON (tab_17541.col3_5) = (tab_17542.col3_2)
        postgres_1    | 			WHERE
        postgres_1    | 				true
        postgres_1    | 		)
        postgres_1    | 			AS tab_17543
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_28443 NULLS FIRST, col_28444 NULLS FIRST
        test_1        | --- FAIL: TestCompare (622.89s)
        test_1        |     compare_test.go:117: Checking connection to: cockroach1
        test_1        |     compare_test.go:117: Checking connection to: cockroach2
        test_1        |     compare_test.go:117: Checking connection to: postgres
        test_1        |     --- FAIL: TestCompare/postgres (19.33s)
        test_1        |         compare_test.go:126: starting test: postgres
        test_1        |         compare_test.go:133: initializing connection: cockroach1
        test_1        |         compare_test.go:133: initializing connection: postgres
        test_1        |         compare_test.go:176: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string(`POSTGIS="3.1.1 aaf4c79" [EXTENSION] PGSQL="130" GEOS="3.7.1-CAPI-1.11.1 27a5e771" PROJ="Rel. 5.2.0, September 15th, 2018" LIBXML="2.9.4" LIBJSON="0.12.1" LIBPROTOBUF="1.3.1" WAGYU="0.5.0 (Internal)" TOPOLOGY`),
        test_1        |             + 	string(`POSTGIS="3.0.1 ec2a9aa" [EXTENSION] PGSQL="120" GEOS="3.8.1-CAPI-1.13.3" PROJ="4.9.3" LIBXML="2.9.10" LIBJSON="0.13.1" LIBPROTOBUF="1.4.2" WAGYU="0.4.3 (Internal)"`),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT postgis_full_version()::STRING AS col_28490) AS tab_17562 ORDER BY col_28490 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT postgis_full_version()::TEXT AS col_28490) AS tab_17562 ORDER BY col_28490 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #62602 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2851134&tab=buildLog) on [master@f676f07152b8d0ca203831bb9e75d9b7d3ed5b79](https://github.com/cockroachdb/cockroach/commits/f676f07152b8d0ca203831bb9e75d9b7d3ed5b79):

```
        postgres_1    | 				tab_91634.col2_8 ASC, tab_91634.tableoid DESC, tab_91634.col2_6
        postgres_1    | 		)
        postgres_1    | 			AS tab_91635
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_163757 NULLS FIRST,
        postgres_1    | 		col_163758 NULLS FIRST,
        postgres_1    | 		col_163759 NULLS FIRST,
        postgres_1    | 		col_163760 NULLS FIRST,
        postgres_1    | 		col_163761 NULLS FIRST
        test_1        | --- FAIL: TestCompare (696.61s)
        test_1        |     compare_test.go:117: Checking connection to: postgres
        test_1        |     compare_test.go:117: Checking connection to: cockroach1
        test_1        |     compare_test.go:117: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (93.46s)
        test_1        |         compare_test.go:126: starting test: postgres
        test_1        |         compare_test.go:133: initializing connection: cockroach1
        test_1        |         compare_test.go:133: initializing connection: postgres
        test_1        |         compare_test.go:176: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("G8"),
        test_1        |             + 	string("G8.0"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('G':::STRING::STRING || 8.0:::FLOAT8::FLOAT8)::STRING AS col_163937) AS tab_91720
        test_1        |             ORDER BY
        test_1        |             	col_163937 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('G'::TEXT::TEXT || 8.0::FLOAT8::FLOAT8)::TEXT AS col_163937) AS tab_91720
        test_1        |             ORDER BY
        test_1        |             	col_163937 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #62602 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(compose).TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2854951&tab=buildLog) on [master@651184b3cfbd7c6d336bb49e7152e3d48ea06416](https://github.com/cockroachdb/cockroach/commits/651184b3cfbd7c6d336bb49e7152e3d48ea06416):

```
        postgres_1    | 	CREATE TABLE table2 (col2_0 BOX2D, col2_1 FLOAT8, col2_2 FLOAT8 NOT NULL, col2_3 GEOMETRY NULL, col2_4 BYTEA NOT NULL, col2_5 GEOMETRY NOT NULL, col2_6 CHAR NULL, col2_7 INTERVAL NULL, col2_8 BIT(18) NOT NULL, col2_9 INET, PRIMARY KEY (col2_2, col2_8, col2_4), col2_10 FLOAT8 GENERATED ALWAYS AS (col2_2 + col2_1) STORED);
        postgres_1    | 	CREATE INDEX ON table2 (col2_6, col2_10 ASC) INCLUDE (col2_0, col2_1, col2_3, col2_5, col2_7);
        postgres_1    | 	CREATE TABLE table3 (col3_0 GEOMETRY NULL, col3_1 DECIMAL, col3_2 FLOAT8 NOT NULL, col3_3 TEXT[], col3_4 TIMESTAMP NULL, col3_5 "char", col3_6 "char" NOT NULL, col3_7 OID[] NOT NULL, PRIMARY KEY (col3_6), col3_8 DECIMAL NOT NULL GENERATED ALWAYS AS (col3_1 + 'NaN'::DECIMAL) STORED, col3_9 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col3_3 IS NULL THEN ''::TEXT ELSE 'X'::TEXT END) STORED, col3_10 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col3_7 IS NULL THEN NULL ELSE e'i[\x14h9\b`\x12'::TEXT END) STORED, col3_11 TEXT NULL GENERATED ALWAYS AS (lower(col3_6)) STORED);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_2 DESC, col3_11, col3_6, col3_1 DESC);
        postgres_1    | 	CREATE INDEX ON table3 (col3_6 ASC, col3_9 DESC);
        postgres_1    | 	CREATE TABLE table4 (col4_0 GEOMETRY NOT NULL, col4_1 OID, col4_2 FLOAT8 NOT NULL, col4_3 REGCLASS NULL, col4_4 FLOAT8 NOT NULL, col4_5 NAME NOT NULL, col4_6 REGTYPE NOT NULL, col4_7 INT8 NOT NULL, col4_8 INTERVAL, PRIMARY KEY (col4_4, col4_6, col4_7), col4_9 FLOAT8 GENERATED ALWAYS AS (col4_4 + col4_2) STORED, col4_10 INT8 GENERATED ALWAYS AS (col4_7 + 2052586798::INT8) STORED, col4_11 FLOAT8 GENERATED ALWAYS AS (col4_4 + (-0.7612022757530212)::FLOAT8) STORED, col4_12 INT8 NULL GENERATED ALWAYS AS (col4_7 + 843980844::INT8) STORED, col4_13 FLOAT8 NOT NULL GENERATED ALWAYS AS (col4_4 + col4_2) STORED, col4_14 INT8 NOT NULL GENERATED ALWAYS AS (col4_7 + (-414366783)::INT8) STORED, col4_15 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col4_3 IS NULL THEN e'\t\x043('::TEXT ELSE e'S\x1dbk'::TEXT END) STORED, col4_16 TEXT GENERATED ALWAYS AS (lower(CAST(col4_8 AS TEXT))) STORED);
        postgres_1    | 	CREATE UNIQUE INDEX ON table4 (col4_10 ASC, col4_2 DESC, col4_1 DESC, col4_7, col4_6 DESC, col4_14, col4_4 DESC, col4_5, col4_11 DESC, col4_3, col4_15 DESC, col4_12);
        postgres_1    | 	CREATE UNIQUE INDEX ON table4 (col4_12);
        postgres_1    | 	CREATE UNIQUE INDEX ON table4 (col4_11, col4_12 ASC, col4_9, col4_5, col4_15 DESC, col4_6 DESC, col4_16 ASC, col4_10, col4_4 ASC) INCLUDE (col4_1, col4_2, col4_8, col4_13);
        postgres_1    | 	CREATE TABLE table5 (col5_0 INTERVAL, col5_1 GEOGRAPHY NULL, col5_2 VARCHAR NULL, col5_3 FLOAT8 NULL, col5_4 BOOL, col5_5 REGTYPE NULL, col5_6 TIMESTAMP NOT NULL, col5_7 REGNAMESPACE NULL, col5_8 GEOGRAPHY, col5_9 REGNAMESPACE NULL, col5_10 DECIMAL, col5_11 REGNAMESPACE, col5_12 TEXT GENERATED ALWAYS AS (CASE WHEN col5_9 IS NULL THEN e'\U00002603'::TEXT ELSE NULL END) STORED, col5_13 TEXT GENERATED ALWAYS AS (lower(col5_2)) STORED, col5_14 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col5_9 IS NULL THEN e'$XC\t:*A'::TEXT ELSE NULL END) STORED, col5_15 FLOAT8 GENERATED ALWAYS AS (col5_3 + (-1.2443554896369446)::FLOAT8) STORED);
        postgres_1    | 	CREATE UNIQUE INDEX ON table5 (col5_6 DESC, col5_12);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('exnll', 'r', 'snx');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (605.62s)
        test_1        |     compare_test.go:117: Checking connection to: cockroach2
        test_1        |     compare_test.go:117: Checking connection to: postgres
        test_1        |     compare_test.go:117: Checking connection to: cockroach1
        test_1        |     --- FAIL: TestCompare/postgres (2.20s)
        test_1        |         compare_test.go:126: starting test: postgres
        test_1        |         compare_test.go:133: initializing connection: cockroach1
        test_1        |         compare_test.go:133: initializing connection: postgres
        test_1        |         compare_test.go:150: CREATE TABLE table1 (col1_0 "char" NOT NULL, col1_1 GEOGRAPHY NOT NULL, col1_2 NAME, col1_3 TIMETZ, col1_4 BYTEA NULL, col1_5 FLOAT8 NOT NULL, col1_6 BOX2D, col1_7 INET NOT NULL, col1_8 "char" NOT NULL, col1_9 GEOMETRY NULL, col1_10 REGCLASS NULL, col1_11 REGNAMESPACE NULL, col1_12 TEXT NOT NULL GENERATED ALWAYS AS (lower(col1_0)) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_3 DESC);
        test_1        |             CREATE INDEX ON table1 (col1_4 DESC, col1_11, col1_2 ASC, col1_5 DESC, col1_7 ASC, col1_8) INCLUDE (col1_0, col1_3, col1_9, col1_10);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_5, col1_10, col1_11 ASC, col1_3 DESC, col1_2 ASC, col1_7 ASC) INCLUDE (col1_0, col1_4, col1_8, col1_12);
        test_1        |             CREATE TABLE table2 (col2_0 BOX2D, col2_1 FLOAT8, col2_2 FLOAT8 NOT NULL, col2_3 GEOMETRY NULL, col2_4 BYTEA NOT NULL, col2_5 GEOMETRY NOT NULL, col2_6 CHAR NULL, col2_7 INTERVAL NULL, col2_8 BIT(18) NOT NULL, col2_9 INET, PRIMARY KEY (col2_2, col2_8, col2_4), col2_10 FLOAT8 GENERATED ALWAYS AS (col2_2 + col2_1) STORED);
        test_1        |             CREATE INDEX ON table2 (col2_6, col2_10 ASC) INCLUDE (col2_0, col2_1, col2_3, col2_5, col2_7);
        test_1        |             CREATE TABLE table3 (col3_0 GEOMETRY NULL, col3_1 DECIMAL, col3_2 FLOAT8 NOT NULL, col3_3 TEXT[], col3_4 TIMESTAMP NULL, col3_5 "char", col3_6 "char" NOT NULL, col3_7 OID[] NOT NULL, PRIMARY KEY (col3_6), col3_8 DECIMAL NOT NULL GENERATED ALWAYS AS (col3_1 + 'NaN'::DECIMAL) STORED, col3_9 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col3_3 IS NULL THEN ''::TEXT ELSE 'X'::TEXT END) STORED, col3_10 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col3_7 IS NULL THEN NULL ELSE e'i[\x14h9\b`\x12'::TEXT END) STORED, col3_11 TEXT NULL GENERATED ALWAYS AS (lower(col3_6)) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_2 DESC, col3_11, col3_6, col3_1 DESC);
        test_1        |             CREATE INDEX ON table3 (col3_6 ASC, col3_9 DESC);
        test_1        |             CREATE TABLE table4 (col4_0 GEOMETRY NOT NULL, col4_1 OID, col4_2 FLOAT8 NOT NULL, col4_3 REGCLASS NULL, col4_4 FLOAT8 NOT NULL, col4_5 NAME NOT NULL, col4_6 REGTYPE NOT NULL, col4_7 INT8 NOT NULL, col4_8 INTERVAL, PRIMARY KEY (col4_4, col4_6, col4_7), col4_9 FLOAT8 GENERATED ALWAYS AS (col4_4 + col4_2) STORED, col4_10 INT8 GENERATED ALWAYS AS (col4_7 + 2052586798::INT8) STORED, col4_11 FLOAT8 GENERATED ALWAYS AS (col4_4 + (-0.7612022757530212)::FLOAT8) STORED, col4_12 INT8 NULL GENERATED ALWAYS AS (col4_7 + 843980844::INT8) STORED, col4_13 FLOAT8 NOT NULL GENERATED ALWAYS AS (col4_4 + col4_2) STORED, col4_14 INT8 NOT NULL GENERATED ALWAYS AS (col4_7 + (-414366783)::INT8) STORED, col4_15 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col4_3 IS NULL THEN e'\t\x043('::TEXT ELSE e'S\x1dbk'::TEXT END) STORED, col4_16 TEXT GENERATED ALWAYS AS (lower(CAST(col4_8 AS TEXT))) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_10 ASC, col4_2 DESC, col4_1 DESC, col4_7, col4_6 DESC, col4_14, col4_4 DESC, col4_5, col4_11 DESC, col4_3, col4_15 DESC, col4_12);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_12);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_11, col4_12 ASC, col4_9, col4_5, col4_15 DESC, col4_6 DESC, col4_16 ASC, col4_10, col4_4 ASC) INCLUDE (col4_1, col4_2, col4_8, col4_13);
        test_1        |             CREATE TABLE table5 (col5_0 INTERVAL, col5_1 GEOGRAPHY NULL, col5_2 VARCHAR NULL, col5_3 FLOAT8 NULL, col5_4 BOOL, col5_5 REGTYPE NULL, col5_6 TIMESTAMP NOT NULL, col5_7 REGNAMESPACE NULL, col5_8 GEOGRAPHY, col5_9 REGNAMESPACE NULL, col5_10 DECIMAL, col5_11 REGNAMESPACE, col5_12 TEXT GENERATED ALWAYS AS (CASE WHEN col5_9 IS NULL THEN e'\U00002603'::TEXT ELSE NULL END) STORED, col5_13 TEXT GENERATED ALWAYS AS (lower(col5_2)) STORED, col5_14 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col5_9 IS NULL THEN e'$XC\t:*A'::TEXT ELSE NULL END) STORED, col5_15 FLOAT8 GENERATED ALWAYS AS (col5_3 + (-1.2443554896369446)::FLOAT8) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_6 DESC, col5_12);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('exnll', 'r', 'snx');
        test_1        |             
        test_1        |         compare_test.go:151: postgres: exec: ERROR: conversion between UTF8 and SQL_ASCII is not supported (SQLSTATE 0A000)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1

```

<details><summary>More</summary><p>


```
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```

Related:
- #62602 compose: TestComposeCompare failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-21.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-21.1)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2ATestComposeCompare.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[compose.TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2859841&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2859841&tab=artifacts#/) on [master @ 704a9fa4ffd144a2bc977bc1a6853a853b17e70e](https://github.com/cockroachdb/cockroach/commits/704a9fa4ffd144a2bc977bc1a6853a853b17e70e):


```
        postgres_1    | 	CREATE TYPE rand_typ_4 AS ENUM ('bvvdp', 'gpoo', 'c', 'fbs');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (605.88s)
        test_1        |     compare_test.go:117: Checking connection to: postgres
        test_1        |     compare_test.go:117: Checking connection to: cockroach1
        test_1        |     compare_test.go:117: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (3.50s)
        test_1        |         compare_test.go:126: starting test: postgres
        test_1        |         compare_test.go:133: initializing connection: cockroach1
        test_1        |         compare_test.go:133: initializing connection: postgres
        test_1        |         compare_test.go:150: CREATE TABLE table1 (col1_0 REGNAMESPACE NOT NULL, col1_1 REGPROC, col1_2 BIT(6), col1_3 VARCHAR NOT NULL, col1_4 TIME NOT NULL, col1_5 VARCHAR NOT NULL, col1_6 OID NULL, col1_7 TIMETZ, col1_8 INET NULL, PRIMARY KEY (col1_0, col1_5, col1_4, col1_3), col1_9 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col1_0 IS NULL THEN 'q$f``'::TEXT ELSE e'Pc\x1f'::TEXT END) STORED, col1_10 TEXT GENERATED ALWAYS AS (lower(col1_3)) STORED, col1_11 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col1_7 AS TEXT))) STORED, col1_12 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col1_1 IS NULL THEN '!80x'::TEXT ELSE e'R\x04\x04:\x12'::TEXT END) STORED, col1_13 TEXT GENERATED ALWAYS AS (lower(col1_5)) STORED, col1_14 TEXT NOT NULL GENERATED ALWAYS AS (lower(CAST(col1_8 AS TEXT))) STORED, col1_15 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col1_1 IS NULL THEN 'w'::TEXT ELSE e'Cr\x12\n\x13\b'::TEXT END) STORED, col1_16 TEXT GENERATED ALWAYS AS (lower(CAST(col1_7 AS TEXT))) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_7);
        test_1        |             CREATE INDEX ON table1 (col1_13, col1_3 DESC, col1_1, col1_2 DESC, col1_8 DESC, col1_6 DESC, col1_5 ASC, col1_12);
        test_1        |             CREATE INDEX ON table1 (col1_12, col1_16 ASC, col1_11 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_3);
        test_1        |             CREATE TABLE table2 (col2_0 INT8[], col2_1 BOOL, col2_2 OID NOT NULL, col2_3 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col2_0 IS NULL THEN 'hAg oV$k'::TEXT ELSE ''::TEXT END) STORED, col2_4 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col2_0 IS NULL THEN '|:5'::TEXT ELSE 'Z'::TEXT END) STORED);
        test_1        |             CREATE TABLE table3 (col3_0 NAME NOT NULL);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0);
        test_1        |             CREATE INDEX ON table3 (col3_0 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 ASC);
        test_1        |             CREATE TABLE table4 (col4_0 NAME, col4_1 OID NULL, col4_2 TEXT NULL GENERATED ALWAYS AS (lower(col4_0)) STORED);
        test_1        |             CREATE INDEX ON table4 (col4_0, col4_2 DESC);
        test_1        |             CREATE INDEX ON table4 (col4_0 ASC, col4_2 ASC);
        test_1        |             CREATE INDEX ON table4 (col4_0);
        test_1        |             CREATE TABLE table5 (col5_0 TEXT, col5_1 INT8[] NOT NULL, col5_2 REGCLASS NOT NULL, col5_3 DATE NOT NULL, col5_4 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col5_2 IS NULL THEN e'\''::TEXT ELSE e'.t\x06vOKeR'::TEXT END) STORED, col5_5 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col5_3 AS TEXT))) STORED, col5_6 TEXT GENERATED ALWAYS AS (CASE WHEN col5_1 IS NULL THEN e';\x05'::TEXT ELSE '~}eCbNBQ'::TEXT END) STORED);
        test_1        |             CREATE INDEX ON table5 (col5_3 ASC, col5_4 ASC, col5_2 ASC, col5_5 DESC) INCLUDE (col5_0);
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col3_0) REFERENCES table1 (col1_3);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('zs');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('lk');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('xxhaf', 'kw', 'lsba', 'c', 'pbyu', 'qzsqtp');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('suzxp');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('bvvdp', 'gpoo', 'c', 'fbs');
        test_1        |             
        test_1        |         compare_test.go:151: postgres: exec: ERROR: generation expression is not immutable (SQLSTATE 42P17)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]</p>
</details>

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project)

<sub>[Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>

[compose.TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2867134&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2867134&tab=artifacts#/) on [master @ 909fce463817b0a165d380de6451bd6d4a43b4a5](https://github.com/cockroachdb/cockroach/commits/909fce463817b0a165d380de6451bd6d4a43b4a5):


```
        postgres_1    | 			WHERE
        postgres_1    | 				(NULL OR (NULL::GEOGRAPHY IN (SELECT NULL::GEOGRAPHY AS col_27015 FROM postgres.public.table2 AS tab_17346 WHERE st_expand(NULL::BOX2D, 6.0::FLOAT8::FLOAT8, tab_17346.col2_4::FLOAT8)::BOX2D::BOX2D IN (st_box2dfromgeohash('I'::TEXT::TEXT)::BOX2D, NULL, NULL, NULL))))
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_17343.col1_7
        postgres_1    | 		)
        postgres_1    | 			AS tab_17347
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_27016 NULLS FIRST, col_27017 NULLS FIRST, col_27018 NULLS FIRST
        test_1        | --- FAIL: TestCompare (626.62s)
        test_1        |     compare_test.go:117: Checking connection to: postgres
        test_1        |     compare_test.go:117: Checking connection to: cockroach1
        test_1        |     compare_test.go:117: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (23.54s)
        test_1        |         compare_test.go:126: starting test: postgres
        test_1        |         compare_test.go:133: initializing connection: cockroach1
        test_1        |         compare_test.go:133: initializing connection: postgres
        test_1        |         compare_test.go:176: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("tJ"),
        test_1        |             + 	string("trueJ"),
        test_1        |               	&net.IPNet{IP: s"::", Mask: {0x00, 0x00, 0x00, 0x00, ...}},
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT (true::BOOL || 'J':::STRING::STRING)::STRING AS col_27289, '::7/0':::INET AS col_27290) AS tab_17497
        test_1        |             ORDER BY
        test_1        |             	col_27289 NULLS FIRST, col_27290 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT (true::BOOL || 'J'::TEXT::TEXT)::TEXT AS col_27289, '::7/0'::INET AS col_27290) AS tab_17497
        test_1        |             ORDER BY
        test_1        |             	col_27289 NULLS FIRST, col_27290 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]</p>
</details>

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project)

<sub>[Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>

[compose.TestComposeCompare failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2870013&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2870013&tab=artifacts#/) on [master @ 71a023f9212cf1285a4475ed85d5ab6eefa9bdf7](https://github.com/cockroachdb/cockroach/commits/71a023f9212cf1285a4475ed85d5ab6eefa9bdf7):


```
        test_1        |     compare_test.go:117: Checking connection to: postgres
        test_1        |     compare_test.go:117: Checking connection to: cockroach1
        test_1        |     compare_test.go:117: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (26.01s)
        test_1        |         compare_test.go:126: starting test: postgres
        test_1        |         compare_test.go:133: initializing connection: cockroach1
        test_1        |         compare_test.go:133: initializing connection: postgres
        test_1        |         compare_test.go:176: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string(``D{"A": {"A": {"D": "I"}, "B": [{"G": true}], "H": null}}``),
        test_1        |             + 	string(``D'{"A": {"A": {"D": "I"}, "B": [{"G": true}], "H": null}}'``),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			('D':::STRING::STRING || '{"A": {"A": {"D": "I"}, "B": [{"G": true}], "H": null}}':::JSONB::JSONB)::STRING
        test_1        |             				AS col_34862
        test_1        |             	)
        test_1        |             		AS tab_17300
        test_1        |             ORDER BY
        test_1        |             	col_34862 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			('D'::TEXT::TEXT || '{"A": {"A": {"D": "I"}, "B": [{"G": true}], "H": null}}'::JSONB::JSONB)::TEXT
        test_1        |             				AS col_34862
        test_1        |             	)
        test_1        |             		AS tab_17300
        test_1        |             ORDER BY
        test_1        |             	col_34862 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]</p>
</details>

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project)

<sub>[Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2873149&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2873149&tab=artifacts#/) on master @ [c40ed790b3a19f20de30375788b0359f25dace97](https://github.com/cockroachdb/cockroach/commits/c40ed790b3a19f20de30375788b0359f25dace97):


```
        postgres_1    | 				tab_38902.col1_4 AS col_70476,
        postgres_1    | 				tab_38902.col1_0 AS col_70477
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table1 AS tab_38902
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_38902.col1_3 DESC, tab_38902.col1_2 ASC, tab_38902.col1_6 DESC, tab_38902.col1_3 DESC, tab_38902.col1_3
        postgres_1    | 		)
        postgres_1    | 			AS tab_38903
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_70468 NULLS FIRST,
        postgres_1    | 		col_70469 NULLS FIRST,
        postgres_1    | 		col_70470 NULLS FIRST,
        postgres_1    | 		col_70471 NULLS FIRST,
        postgres_1    | 		col_70472 NULLS FIRST,
        postgres_1    | 		col_70473 NULLS FIRST,
        postgres_1    | 		col_70474 NULLS FIRST,
        postgres_1    | 		col_70475 NULLS FIRST,
        postgres_1    | 		col_70476 NULLS FIRST,
        postgres_1    | 		col_70477 NULLS FIRST
        test_1        | --- FAIL: TestCompare (644.62s)
        test_1        |     compare_test.go:117: Checking connection to: postgres
        test_1        |     compare_test.go:117: Checking connection to: cockroach1
        test_1        |     compare_test.go:117: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (42.93s)
        test_1        |         compare_test.go:126: starting test: postgres
        test_1        |         compare_test.go:133: initializing connection: cockroach1
        test_1        |         compare_test.go:133: initializing connection: postgres
        test_1        |         compare_test.go:176: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("3.0.1 ec2a9aa"),
        test_1        |             + 	string("3.1.1 aaf4c79"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT postgis_scripts_released()::STRING AS col_70539) AS tab_38938 ORDER BY col_70539 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT postgis_scripts_released()::TEXT AS col_70539) AS tab_38938 ORDER BY col_70539 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2878585&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2878585&tab=artifacts#/) on master @ [7495d19e50d8fd14140220f3ba5c7ad951abafa6](https://github.com/cockroachdb/cockroach/commits/7495d19e50d8fd14140220f3ba5c7ad951abafa6):


```
        postgres_1    | 2021-04-13 18:14:14.477 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-04-13 18:14:14.483 UTC [95] LOG:  database system was shut down at 2021-04-13 18:14:14 UTC
        postgres_1    | 2021-04-13 18:14:14.489 UTC [1] LOG:  database system is ready to accept connections
        test_1        | --- FAIL: TestCompare (604.12s)
        test_1        |     compare_test.go:117: Checking connection to: postgres
        test_1        |     compare_test.go:117: Checking connection to: cockroach1
        test_1        |     compare_test.go:117: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (2.06s)
        test_1        |         compare_test.go:126: starting test: postgres
        test_1        |         compare_test.go:133: initializing connection: cockroach1
        test_1        |         compare_test.go:150: SET CLUSTER SETTING "sql.defaults.drop_enum_value.enabled" = true;
        test_1        |             SET enable_drop_enum_value = true;
        test_1        |             SET CLUSTER SETTING "sql.stats.automatic_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.stats.histogram_collection.enabled" = false;
        test_1        |             SET CLUSTER SETTING "sql.defaults.interleaved_tables.enabled" = true;
        test_1        |             CREATE TABLE table1 (col1_0 TIMESTAMP[], col1_1 "char" NOT NULL, col1_2 TIMESTAMPTZ NOT NULL, col1_3 FLOAT8, col1_4 REGPROCEDURE NULL, col1_5 UUID NOT NULL, col1_6 BOX2D, col1_7 STRING NOT NULL AS (CASE WHEN col1_2 IS NULL THEN 'a@':::STRING ELSE e'a@ZQ_d\x7fs':::STRING END) STORED, col1_8 STRING AS (CASE WHEN col1_0 IS NULL THEN e'(\x19':::STRING ELSE '':::STRING END) STORED, col1_9 STRING NOT NULL AS (CASE WHEN col1_2 IS NULL THEN e'!Q\x16':::STRING ELSE e']"\x07eeR,]b':::STRING END) STORED, col1_10 STRING NOT NULL AS (lower(CAST(col1_5 AS STRING))) STORED, col1_11 STRING AS (lower(CAST(col1_6 AS STRING))) VIRTUAL);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_11 ASC, col1_5 ASC, col1_3);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_3);
        test_1        |             CREATE TABLE table2 (col2_0 CHAR NULL, col2_1 INTERVAL NOT NULL, col2_2 INT2 NULL, col2_3 GEOMETRY NULL, col2_4 BIT(47) NULL, col2_5 BYTES NOT NULL, col2_6 INT2 NULL AS (col2_2 + (-19871):::INT8) VIRTUAL);
        test_1        |             CREATE INDEX ON table2 (col2_4 ASC, col2_0 DESC, col2_5 ASC, col2_1 DESC);
        test_1        |             CREATE INDEX ON table2 (col2_6 ASC) STORING (col2_1, col2_3, col2_4, col2_5);
        test_1        |             CREATE INDEX ON table2 (col2_0, col2_6 DESC) STORING (col2_5);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_0, col2_5, col2_2);
        test_1        |             CREATE INDEX ON table2 (col2_0 ASC, col2_2 DESC) STORING (col2_4);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_5 ASC, col2_0, col2_6 ASC, col2_2 ASC);
        test_1        |             CREATE TABLE table3 (col3_0 REGTYPE, col3_1 OID NULL, col3_2 VARCHAR NOT NULL, col3_3 FLOAT4 NOT NULL, col3_4 BOX2D, col3_5 BOX2D NOT NULL, col3_6 BIT(43), col3_7 INT8, col3_8 FLOAT8 NULL, col3_9 TIMESTAMP NOT NULL, col3_10 BIT(38)[], col3_11 GEOMETRY, col3_12 UUID, col3_13 JSONB NULL, col3_14 INT4, FAMILY (col3_9, col3_2), FAMILY (col3_14, col3_0, col3_10), FAMILY (col3_13, col3_11), FAMILY (col3_1), FAMILY (col3_4, col3_8, col3_12), FAMILY (col3_3), FAMILY (col3_5), FAMILY (col3_6, col3_7));
        test_1        |             CREATE INDEX ON table3 (col3_14, col3_12 ASC, col3_9, col3_1 ASC, col3_8, col3_7 DESC) STORING (col3_2, col3_5, col3_6, col3_13);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_3, col3_0 DESC);
        test_1        |             ALTER TABLE table1 INJECT STATISTICS e'[{"columns": ["col1_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2198506171, "histo_col_type": "", "name": "__auto__", "null_count": 450230451, "row_count": 4000000000}, {"columns": ["col1_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 567714865, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000000}, {"columns": ["col1_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3396055614, "histo_col_type": "", "name": "__auto__", "null_count": 21181829, "row_count": 4000000000}, {"columns": ["col1_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1681789669, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000000}, {"columns": ["col1_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2459372738, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000000}, {"columns": ["col1_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1529145237, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000000}, {"columns": ["col1_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 744810863, "histo_buckets": [{"distinct_range": 0, "num_eq": 5000, "num_range": 0, "upper_bound": "\\u0003L\\u0018{fw."}, {"distinct_range": 0, "num_eq": 200, "num_range": 8611470841008874446, "upper_bound": "\\n\\t\\u0004>"}, {"distinct_range": 4000000, "num_eq": 519632737399276121, "num_range": 4000000, "upper_bound": "\\u0016\\u00178w^\\u001f]"}, {"distinct_range": 1180851031183228700, "num_eq": 900000, "num_range": 1180851031183228567, "upper_bound": "#BZP-"}, {"distinct_range": 2000000000, "num_eq": 9041926978672199579, "num_range": 2000000000, "upper_bound": "+S&xH\\\\"}, {"distinct_range": 872675460730590100, "num_eq": 126306803699886236, "num_range": 872675460730590129, "upper_bound": "1.r\\u0001k)B\'T"}, {"distinct_range": 100, "num_eq": 6124296982913906190, "num_range": 100, "upper_bound": "b\\u0016;"}], "histo_col_type": "STRING", "name": "__auto__", "null_count": 1301966452, "row_count": 4000000000}, {"columns": ["col1_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 509398161, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000000}, {"columns": ["col1_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3018561229, "histo_col_type": "", "name": "__auto__", "null_count": 3724512085, "row_count": 4000000000}, {"columns": ["col1_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2658289426, "histo_col_type": "", "name": "__auto__", "null_count": 844987938, "row_count": 4000000000}, {"columns": ["col1_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2347347195, "histo_col_type": "", "name": "__auto__", "null_count": 1268066062, "row_count": 4000000000}, {"columns": ["col1_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 494695801, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4000000000}]':::JSONB;
        test_1        |             ALTER TABLE table2 INJECT STATISTICS e'[{"columns": ["col2_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 419522792796002617, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 2038479740173653464}, {"columns": ["col2_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 407120466294491956, "histo_col_type": "", "name": "__auto__", "null_count": 1878020636572358982, "row_count": 2038479740173653464}, {"columns": ["col2_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1280359833139815341, "histo_col_type": "", "name": "__auto__", "null_count": 207699152202515341, "row_count": 2038479740173653464}, {"columns": ["col2_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 97130871891470004, "histo_buckets": [{"distinct_range": 0, "num_eq": 70000000, "num_range": 0, "upper_bound": "01100001011010101110101111001111011010110000000"}], "histo_col_type": "BIT(47)", "name": "__auto__", "null_count": 2019674256912211486, "row_count": 2038479740173653464}, {"columns": ["col2_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1841156365893854678, "histo_buckets": [{"distinct_range": 0, "num_eq": 900000000, "num_range": 0, "upper_bound": "\\\\x1063def49bc4b52b04"}, {"distinct_range": 0, "num_eq": 3648056963366833227, "num_range": 300000000, "upper_bound": "\\\\x21e3e2596423"}, {"distinct_range": 10000000, "num_eq": 7320643071813314315, "num_range": 10000000, "upper_bound": "\\\\x2be5bd4155"}, {"distinct_range": 0, "num_eq": 100000000000, "num_range": 536775500669844312, "upper_bound": "\\\\x4f8da30556"}], "histo_col_type": "BYTES", "name": "__auto__", "null_count": 0, "row_count": 2038479740173653464}, {"columns": ["col2_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 124877746336551034, "histo_buckets": [{"distinct_range": 0, "num_eq": 5557592370853999765, "num_range": 0, "upper_bound": "-32364"}, {"distinct_range": 0, "num_eq": 10000, "num_range": 80000000000, "upper_bound": "-21889"}, {"distinct_range": 631515167492077300, "num_eq": 80, "num_range": 631515167492077269, "upper_bound": "-6981"}, {"distinct_range": 8621932650218097000, "num_eq": 949455508354205527, "num_range": 8988237454553159333, "upper_bound": "127"}, {"distinct_range": 50000, "num_eq": 2799709490906484370, "num_range": 50000, "upper_bound": "19222"}, {"distinct_range": 464.37585678995606, "num_eq": 70, "num_range": 600, "upper_bound": "19784"}, {"distinct_range": 8607872435802817000, "num_eq": 4321681928246554967, "num_range": 8607872435802816664, "upper_bound": "25096"}, {"distinct_range": 6467708934430932000, "num_eq": 40000000, "num_range": 6467708934430932075, "upper_bound": "29332"}, {"distinct_range": 5500087136448697000, "num_eq": 5397988536294491614, "num_range": 5500087136448697653, "upper_bound": "32767"}], "histo_col_type": "INT2", "name": "__auto__", "null_count": 1964938194616027033, "row_count": 2038479740173653464}, {"columns": ["col2_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 707460587808248952, "histo_buckets": [{"distinct_range": 0, "num_eq": 300, "num_range": 0, "upper_bound": "\\n"}, {"distinct_range": 3667690761979738000, "num_eq": 600, "num_range": 5344231781427045678, "upper_bound": "\\u001b"}, {"distinct_range": 8.99366268394994, "num_eq": 7292533098821615211, "num_range": 100, "upper_bound": "\\u001e"}, {"distinct_range": 7270604933373399000, "num_eq": 300, "num_range": 7270604933373399193, "upper_bound": "C"}, {"distinct_range": 3210127555.670375, "num_eq": 5407171580812713900, "num_range": 30000000000, "upper_bound": "^"}, {"distinct_range": 0, "num_eq": 0, "num_range": 1708494434656115845, "upper_bound": "q"}], "histo_col_type": "CHAR", "name": "__auto__", "null_count": 1092160961150520400, "row_count": 2038479740173653464}]':::JSONB;
        test_1        |             ALTER TABLE table3 INJECT STATISTICS '[{"columns": ["col3_1"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 588188245495015538, "histo_col_type": "", "name": "__auto__", "null_count": 358442278829080101, "row_count": 4815516927734463696}, {"columns": ["col3_10"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2295442078121341112, "histo_col_type": "", "name": "__auto__", "null_count": 3695975651260174768, "row_count": 4815516927734463696}, {"columns": ["col3_7"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2399271982465998092, "histo_col_type": "", "name": "__auto__", "null_count": 4133531667764285925, "row_count": 4815516927734463696}, {"columns": ["col3_8"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4505432504555558962, "histo_col_type": "", "name": "__auto__", "null_count": 1816476009216881420, "row_count": 4815516927734463696}, {"columns": ["col3_9"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4733613389791488643, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4815516927734463696}, {"columns": ["col3_11"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 954219443453043778, "histo_col_type": "", "name": "__auto__", "null_count": 3096758395471239715, "row_count": 4815516927734463696}, {"columns": ["col3_14"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 2733574787494735448, "histo_col_type": "INT4", "name": "__auto__", "null_count": 1522417216452657966, "row_count": 4815516927734463696}, {"columns": ["col3_0"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3000481833455172497, "histo_col_type": "", "name": "__auto__", "null_count": 560669254313665724, "row_count": 4815516927734463696}, {"columns": ["col3_5"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3405678883427037897, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4815516927734463696}, {"columns": ["col3_6"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 1988310047193304208, "histo_buckets": [{"distinct_range": 0, "num_eq": 860249982021336835, "num_range": 0, "upper_bound": "0000000000000000000000000000000000000000000"}, {"distinct_range": 1657623808142550500, "num_eq": 0, "num_range": 1657623808142550440, "upper_bound": "1100011111010100000101101011011110101111100"}, {"distinct_range": 0, "num_eq": 3525687318057086739, "num_range": 0, "upper_bound": "1111110001001111111001111010101100001101101"}], "histo_col_type": "BIT(43)", "name": "__auto__", "null_count": 90666835491583387, "row_count": 4815516927734463696}, {"columns": ["col3_3"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4110487456047238367, "histo_buckets": [{"distinct_range": 0, "num_eq": 10, "num_range": 0, "upper_bound": "-1.0071858167648315"}, {"distinct_range": 0, "num_eq": 3432484669696573208, "num_range": 70000000, "upper_bound": "-1.0063306093215942"}, {"distinct_range": 4000000, "num_eq": 4442544126996075816, "num_range": 4000000, "upper_bound": "0.17988602817058563"}, {"distinct_range": 100000000000, "num_eq": 0, "num_range": 100000000000, "upper_bound": "0.4088081121444702"}, {"distinct_range": 0, "num_eq": 9000, "num_range": 1040025834091568899, "upper_bound": "0.5401249527931213"}, {"distinct_range": 0, "num_eq": 33337617801367867, "num_range": 1688859400986303294, "upper_bound": "1.3012795448303223"}, {"distinct_range": 80000000, "num_eq": 7953193794151741154, "num_range": 80000000, "upper_bound": "1.5677287578582764"}], "histo_col_type": "FLOAT4", "name": "__auto__", "null_count": 0, "row_count": 4815516927734463696}, {"columns": ["col3_4"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3569397533570977016, "histo_col_type": "", "name": "__auto__", "null_count": 3454137017203836617, "row_count": 4815516927734463696}, {"columns": ["col3_13"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4578883195732562352, "histo_col_type": "", "name": "__auto__", "null_count": 2639735263004161186, "row_count": 4815516927734463696}, {"columns": ["col3_2"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 4701067823744297141, "histo_col_type": "", "name": "__auto__", "null_count": 0, "row_count": 4815516927734463696}, {"columns": ["col3_12"], "created_at": "2000-01-01 00:00:00+00:00", "distinct_count": 3832133737956935155, "histo_col_type": "", "name": "__auto__", "null_count": 514980874635768483, "row_count": 4815516927734463696}]':::JSONB;
        test_1        |             ALTER TABLE table3 ADD FOREIGN KEY (col3_8, col3_5) REFERENCES table1 (col1_3, col1_6) ON DELETE RESTRICT;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('whif', 'owiqoh', 'rixnl', 'uliqu');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('il', 'sbrfuv', 'ern', 'qe', 'gnmoq', 'bwjki');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('qcene');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('qiwsq', 'bldb');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('fiqnl');
        test_1        |             
        test_1        |         compare_test.go:151: cockroach1: exec: ERROR: there is no unique constraint matching given keys for referenced table table1 (SQLSTATE 23503)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2887680&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2887680&tab=artifacts#/) on master @ [b2253451fb2d38462c842996c6e6cfddece02802](https://github.com/cockroachdb/cockroach/commits/b2253451fb2d38462c842996c6e6cfddece02802):


```
        postgres_1    | 				true
        postgres_1    | 			GROUP BY
        postgres_1    | 				tab_94408.col1_2, tab_94408.col1_1
        postgres_1    | 		)
        postgres_1    | 			AS tab_94409
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_134886 NULLS FIRST, col_134887 NULLS FIRST
        test_1        | --- FAIL: TestCompare (704.36s)
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     --- FAIL: TestCompare/postgres (101.20s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("G2000-01-01 00:00:00"),
        test_1        |             + 	string("G2000-01-01 00:00:00+00"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('G':::STRING::STRING || '2000-01-01 00:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ)::STRING AS col_134946)
        test_1        |             		AS tab_94458
        test_1        |             ORDER BY
        test_1        |             	col_134946 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('G'::TEXT::TEXT || '2000-01-01 00:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ)::TEXT AS col_134946)
        test_1        |             		AS tab_94458
        test_1        |             ORDER BY
        test_1        |             	col_134946 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2891762&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2891762&tab=artifacts#/) on master @ [e29d694755e4800f07df139137c71cf741829dd0](https://github.com/cockroachdb/cockroach/commits/e29d694755e4800f07df139137c71cf741829dd0):


```
        postgres_1    | 	ALTER TABLE table2 ADD FOREIGN KEY (col1_5, col2_3, col2_1, col2_2) REFERENCES table3 (col1_5, col2_3, col3_2, col3_3) ON DELETE CASCADE;
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('duafen', 'ltqpq', 'trq', 'jfzq');
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('wnuwx');
        postgres_1    | 	CREATE TYPE rand_typ_2 AS ENUM ('wvi', 'o', 'rj', 'emu', 'lxmhc');
        postgres_1    | 	CREATE TYPE rand_typ_3 AS ENUM ('gfzw', 'kx', 'jb', 'wtoxv');
        postgres_1    | 	CREATE TYPE rand_typ_4 AS ENUM ('mzv', 'av', 'pjny', 'eze');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (605.83s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (2.75s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:149: CREATE TABLE table1 (col1_0 CHAR NULL, col1_1 INET NOT NULL, col1_2 REGNAMESPACE, col1_3 INT8 NOT NULL, col1_4 JSONB NULL, col1_5 INT8 NOT NULL, col1_6 REGNAMESPACE NULL, col1_7 NAME NULL, PRIMARY KEY (col1_5), col1_8 INT8 NOT NULL GENERATED ALWAYS AS (col1_3 + col1_5) STORED, col1_9 TEXT GENERATED ALWAYS AS (CASE WHEN col1_2 IS NULL THEN ''::TEXT ELSE e'hiV\x14Y(G'::TEXT END) STORED, col1_10 INT8 GENERATED ALWAYS AS (col1_3 + col1_5) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_7 DESC, col1_9, col1_6 DESC, col1_8, col1_2 DESC, col1_1 DESC, col1_10 DESC, col1_5 ASC, col1_3);
        test_1        |             CREATE INDEX ON table1 (col1_6 DESC, col1_0 DESC);
        test_1        |             CREATE TABLE table2 (col1_5 INT8 NOT NULL, col2_1 INT8, col2_2 NAME, col2_3 INTERVAL NOT NULL, PRIMARY KEY (col1_5, col2_3));
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_1, col2_3 ASC);
        test_1        |             CREATE TABLE table3 (col1_5 INT8 NOT NULL, col2_3 INTERVAL NOT NULL, col3_2 INT8 NOT NULL, col3_3 "char", PRIMARY KEY (col1_5, col2_3, col3_2));
        test_1        |             CREATE UNIQUE INDEX ON table3 (col1_5, col2_3, col3_2, col3_3);
        test_1        |             CREATE TABLE table4 (col1_5 INT8 NOT NULL, col2_3 INTERVAL NOT NULL, col3_2 INT8 NOT NULL, col4_3 TIMESTAMPTZ NULL, col4_4 DECIMAL NOT NULL, col4_5 REGPROCEDURE NOT NULL, col4_6 BIT(45), col4_7 BIT(49) NULL, col4_8 OID NOT NULL, col4_9 TIMETZ NOT NULL, col4_10 BYTEA NOT NULL, col4_11 INTERVAL NOT NULL, col4_12 INTERVAL NOT NULL, col4_13 BYTEA NULL, col4_14 TIMETZ NOT NULL, col4_15 INET NOT NULL, col4_16 TIMESTAMP, col4_17 INT8 NULL, col4_18 DATE NOT NULL, col4_19 INT8, PRIMARY KEY (col1_5, col2_3, col3_2, col4_14, col4_12, col4_18, col4_19, col4_4, col4_9, col4_6, col4_7, col4_15, col4_8, col4_13, col4_10, col4_11, col4_16, col4_3));
        test_1        |             CREATE TABLE table5 (col1_5 INT8 NOT NULL, col2_3 INTERVAL NOT NULL, col5_2 INT8 NULL, col5_3 OID NULL, col5_4 VARCHAR, col5_5 TIMESTAMPTZ NULL, col5_6 REGCLASS, col5_7 INTERVAL NULL, col5_8 FLOAT8 NOT NULL, col5_9 REGCLASS NOT NULL, col5_10 VARCHAR NULL, col5_11 VARCHAR NOT NULL, col5_12 INT8 NOT NULL, col5_13 BIT(20), col5_14 VARCHAR, PRIMARY KEY (col1_5, col2_3, col5_13));
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_4 DESC, col5_10, col5_7 ASC, col5_2, col1_5, col5_9, col5_13 ASC, col5_5 ASC, col2_3) INCLUDE (col5_8, col5_11);
        test_1        |             CREATE INDEX ON table5 (col2_3 ASC, col5_4, col5_13 DESC, col5_5 ASC, col5_8 ASC, col5_7 DESC, col5_11);
        test_1        |             CREATE INDEX ON table5 (col5_2, col5_4, col5_5 DESC, col5_9 DESC, col5_12 ASC, col2_3 DESC, col5_13, col5_11 DESC, col5_8 DESC, col5_7 ASC, col5_6 ASC, col5_3 DESC, col5_14, col5_10 ASC, col1_5 ASC);
        test_1        |             CREATE INDEX ON table5 (col5_9, col5_10 ASC, col5_11 ASC, col5_2, col5_13 DESC) INCLUDE (col5_4, col5_12, col5_14);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_14 DESC, col5_2 DESC, col5_6 ASC, col5_3, col5_4 DESC, col5_5 DESC, col1_5 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table5 (col5_12, col5_9 ASC, col5_14 ASC, col2_3 ASC, col1_5 ASC, col5_6 DESC);
        test_1        |             CREATE INDEX ON table5 (col5_7 DESC, col5_8 DESC, col5_13 ASC) INCLUDE (col5_4, col5_5, col5_6, col5_11, col5_12, col5_14);
        test_1        |             ALTER TABLE table2 ADD FOREIGN KEY (col1_5, col2_3, col2_1, col2_2) REFERENCES table3 (col1_5, col2_3, col3_2, col3_3) ON DELETE CASCADE;
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('duafen', 'ltqpq', 'trq', 'jfzq');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('wnuwx');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('wvi', 'o', 'rj', 'emu', 'lxmhc');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('gfzw', 'kx', 'jb', 'wtoxv');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('mzv', 'av', 'pjny', 'eze');
        test_1        |             
        test_1        |         compare_test.go:150: postgres: exec: ERROR: foreign key constraint "table2_col1_5_col2_3_col2_1_col2_2_fkey" cannot be implemented (SQLSTATE 42804)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2894001&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2894001&tab=artifacts#/) on master @ [3aad3f8a372090ddaedad0ad58a945fb399ddbc6](https://github.com/cockroachdb/cockroach/commits/3aad3f8a372090ddaedad0ad58a945fb399ddbc6):


```
        postgres_1    | 	CREATE UNIQUE INDEX ON table2 (col2_0, col2_5);
        postgres_1    | 	CREATE INDEX ON table2 (col2_10 DESC, col2_14 DESC, col2_12 DESC, col2_1, col2_11 DESC, col2_9 DESC) INCLUDE (col2_0, col2_4, col2_5, col2_8);
        postgres_1    | 	CREATE TABLE table3 (col3_0 FLOAT8 NOT NULL, col3_1 BOOL NULL, col3_2 REGNAMESPACE, col3_3 CHAR, col3_4 TEXT NULL, col3_5 REGPROC NOT NULL, PRIMARY KEY (col3_5));
        postgres_1    | 	CREATE INDEX ON table3 (col3_2 ASC, col3_0 DESC, col3_4 DESC, col3_5 ASC, col3_1 DESC) INCLUDE (col3_3);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_3 ASC, col3_0, col3_1 DESC, col3_4 DESC) INCLUDE (col3_2);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_4, col3_2 ASC, col3_1 DESC, col3_0 ASC);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_3 DESC, col3_0 DESC, col3_4 ASC, col3_2);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_5 ASC) INCLUDE (col3_0, col3_1, col3_4);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('jglfx');
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('rumdea', 'pes', 'el', 'rkt', 'hne', 'kluelt');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (605.10s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (2.42s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:149: CREATE TABLE table1 (col1_0 DATE NULL, col1_1 REGNAMESPACE, col1_2 TEXT NULL, col1_3 INET, col1_4 BOX2D, col1_5 GEOMETRY NOT NULL, col1_6 OID NOT NULL, col1_7 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col1_0 AS TEXT))) STORED, col1_8 TEXT NOT NULL GENERATED ALWAYS AS (lower(CAST(col1_5 AS TEXT))) STORED, col1_9 TEXT GENERATED ALWAYS AS (lower(CAST(col1_3 AS TEXT))) STORED, col1_10 TEXT GENERATED ALWAYS AS (lower(CAST(col1_4 AS TEXT))) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_0 ASC, col1_7 ASC, col1_9 DESC, col1_10 ASC, col1_3 DESC, col1_2 DESC, col1_8 ASC, col1_6 DESC);
        test_1        |             CREATE INDEX ON table1 (col1_9 DESC, col1_6, col1_0) INCLUDE (col1_2, col1_5);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_2 ASC, col1_3, col1_6 DESC, col1_10 DESC, col1_8 ASC, col1_7 ASC) INCLUDE (col1_1, col1_4);
        test_1        |             CREATE TABLE table2 (col2_0 TIME NULL, col2_1 REGNAMESPACE NOT NULL, col2_2 FLOAT8[], col2_3 BYTEA, col2_4 JSONB, col2_5 DECIMAL NOT NULL, col2_6 REGPROC NOT NULL, col2_7 BOOL, col2_8 REGPROCEDURE NULL, col2_9 "char", col2_10 DATE NULL, col2_11 BOOL NULL, PRIMARY KEY (col2_6, col2_1), col2_12 TEXT GENERATED ALWAYS AS (CASE WHEN col2_2 IS NULL THEN 'R8~2Z'::TEXT ELSE ''::TEXT END) STORED, col2_13 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col2_8 IS NULL THEN e'&\x1cB2w\x0580'::TEXT ELSE e'\x1d'::TEXT END) STORED, col2_14 TEXT GENERATED ALWAYS AS (lower(col2_9)) STORED);
        test_1        |             CREATE INDEX ON table2 (col2_12 ASC, col2_0, col2_1, col2_3 DESC) INCLUDE (col2_2, col2_9);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_3 DESC, col2_13 DESC, col2_7 ASC);
        test_1        |             CREATE INDEX ON table2 (col2_1 DESC, col2_9 ASC, col2_0 DESC, col2_13 DESC, col2_5 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_0, col2_5);
        test_1        |             CREATE INDEX ON table2 (col2_10 DESC, col2_14 DESC, col2_12 DESC, col2_1, col2_11 DESC, col2_9 DESC) INCLUDE (col2_0, col2_4, col2_5, col2_8);
        test_1        |             CREATE TABLE table3 (col3_0 FLOAT8 NOT NULL, col3_1 BOOL NULL, col3_2 REGNAMESPACE, col3_3 CHAR, col3_4 TEXT NULL, col3_5 REGPROC NOT NULL, PRIMARY KEY (col3_5));
        test_1        |             CREATE INDEX ON table3 (col3_2 ASC, col3_0 DESC, col3_4 DESC, col3_5 ASC, col3_1 DESC) INCLUDE (col3_3);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_3 ASC, col3_0, col3_1 DESC, col3_4 DESC) INCLUDE (col3_2);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_4, col3_2 ASC, col3_1 DESC, col3_0 ASC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_3 DESC, col3_0 DESC, col3_4 ASC, col3_2);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_5 ASC) INCLUDE (col3_0, col3_1, col3_4);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('jglfx');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('rumdea', 'pes', 'el', 'rkt', 'hne', 'kluelt');
        test_1        |             
        test_1        |         compare_test.go:150: postgres: exec: ERROR: generation expression is not immutable (SQLSTATE 42P17)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2898499&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2898499&tab=artifacts#/) on master @ [4dc05cceea254854317a9374a4b88df06c0946a6](https://github.com/cockroachdb/cockroach/commits/4dc05cceea254854317a9374a4b88df06c0946a6):


```
        postgres_1    | 				2::OID AS col_6442
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table1 AS tab_3314
        postgres_1    | 			GROUP BY
        postgres_1    | 				tab_3314.col1_2, tab_3314.col1_6, tab_3314.col1_4
        postgres_1    | 			HAVING
        postgres_1    | 				bool_and(false::BOOL)::BOOL
        postgres_1    | 		)
        postgres_1    | 			AS tab_3315
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_6440 NULLS FIRST, col_6441 NULLS FIRST, col_6442 NULLS FIRST
        postgres_1    | 2021-04-19 18:13:35.782 UTC [104] ERROR:  operator does not exist: date - bigint at character 38
        postgres_1    | 2021-04-19 18:13:35.782 UTC [104] HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.
        postgres_1    | 2021-04-19 18:13:35.782 UTC [104] STATEMENT:  SELECT
        postgres_1    | 		*
        postgres_1    | 	FROM
        postgres_1    | 		(SELECT ((NULL::DATE - 8::INT8::INT8)::DATE::DATE + NULL::INT8)::DATE AS col_6750) AS tab_3462
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_6750 NULLS FIRST
        test_1        | --- FAIL: TestCompare (607.64s)
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     --- FAIL: TestCompare/postgres (6.20s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("2.9.10"),
        test_1        |             + 	string("2.9.4"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT postgis_libxml_version()::STRING AS col_6898) AS tab_3555 ORDER BY col_6898 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT postgis_libxml_version()::TEXT AS col_6898) AS tab_3555 ORDER BY col_6898 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2903572&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2903572&tab=artifacts#/) on master @ [09be4f72a054cf66ef603b1e4525844c9ef547fc](https://github.com/cockroachdb/cockroach/commits/09be4f72a054cf66ef603b1e4525844c9ef547fc):


```
        postgres_1    | 		col_3287 NULLS FIRST, col_3288 NULLS FIRST, col_3289 NULLS FIRST
        postgres_1    | 2021-04-20 18:15:04.797 UTC [105] ERROR:  interval units "object" not recognized
        postgres_1    | 2021-04-20 18:15:04.797 UTC [105] STATEMENT:  SELECT
        postgres_1    | 		*
        postgres_1    | 	FROM
        postgres_1    | 		(
        postgres_1    | 			SELECT
        postgres_1    | 				date_trunc(jsonb_typeof('{"A": [true], "H": {"B": "G", "I": {"D": "I", "E": "J", "J": true}}}'::JSONB::JSONB)::TEXT::TEXT, '00:04:00'::TIME::TIME)::INTERVAL
        postgres_1    | 					AS col_3350,
        postgres_1    | 				tab_1942.col2_1 AS col_3351,
        postgres_1    | 				tab_1942.tableoid AS col_3352
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table2 AS tab_1942
        postgres_1    | 			WHERE
        postgres_1    | 				false
        postgres_1    | 		)
        postgres_1    | 			AS tab_1943
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_3350 NULLS FIRST, col_3351 NULLS FIRST, col_3352 NULLS FIRST
        test_1        | --- FAIL: TestCompare (611.04s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (4.51s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("3.8.1-CAPI-1.13.3"),
        test_1        |             + 	string("3.7.1-CAPI-1.11.1 27a5e771"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT postgis_geos_version()::STRING AS col_3433) AS tab_1993 ORDER BY col_3433 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT postgis_geos_version()::TEXT AS col_3433) AS tab_1993 ORDER BY col_3433 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2907570&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2907570&tab=artifacts#/) on master @ [66a1657247f8e1cd84ce22a32321fda6352d789f](https://github.com/cockroachdb/cockroach/commits/66a1657247f8e1cd84ce22a32321fda6352d789f):


```
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('diu', 'sdgwnc', 'lbaol', 'ftnxx', 'c', 'x');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('hifgop', 'glkwln', 'm', 'sqe');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('zs', 'kqm', 'krdo');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('a', 'mr', 'lt', 'nl');
        test_1        |             
        test_1        |         compare_test.go:150: postgres: exec: ERROR: generation expression is not immutable (SQLSTATE 42P17)
        test_1        |     --- FAIL: TestCompare/mutators (0.72s)
        test_1        |         compare_test.go:125: starting test: mutators
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: cockroach2
        test_1        | panic: geos: no locations to init GEOS [recovered]
        test_1        | 	panic: geos: no locations to init GEOS
        test_1        | 
        test_1        | goroutine 13 [running]:
        test_1        | testing.tRunner.func1.1(0x1ff3100, 0xc000d42320)
        test_1        | 	/usr/local/go/src/testing/testing.go:1072 +0x30d
        test_1        | testing.tRunner.func1(0xc000001c80)
        test_1        | 	/usr/local/go/src/testing/testing.go:1075 +0x41a
        test_1        | panic(0x1ff3100, 0xc000d42320)
        test_1        | 	/usr/local/go/src/runtime/panic.go:969 +0x1b9
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.encodeInvertedIndexHistogramUpperBounds(0x399daa0, 0x2ae8cc0, 0xc000d443c0, 0x2ae8cc0, 0xc000d443c0, 0x7f78bda1cbf8)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:368 +0x5d3
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.randHistogram(0xc000c083f0, 0x399daa0, 0x399daa0, 0x9675ce6eb5002e01, 0x6, 0x960000c000d36ff0)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:301 +0x10c
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator.func1(0xc000bff7a0)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:220 +0x9d
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator(0xc000c083f0, 0xc000bffb00, 0x11, 0x11, 0x0, 0x1f47b58, 0xc000c9d960, 0x434f5f)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:251 +0xbe6
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.MultiStatementMutation.Mutate(0x27414d8, 0xc000c083f0, 0xc000bffb00, 0x11, 0x11, 0x7f78bda27be0, 0xc000c9d990, 0x43636a, 0x1f37c40)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:97 +0x4e
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.Apply(0xc000c083f0, 0xc000bffb00, 0x11, 0x11, 0xc0006b8880, 0x6, 0x8, 0x8, 0x0, 0x0, ...)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:108 +0xa7
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.ApplyString(0xc000c083f0, 0xc000ba2000, 0x2af9, 0xc0003dc0c0, 0x6, 0x6, 0x0, 0x0, 0x0)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:159 +0x332
        test_1        | github.com/cockroachdb/cockroach/pkg/compose/compare/compare.TestCompare.func2(0xc000001c80)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/compose/compare/compare/compare_test.go:147 +0x2c5
        test_1        | testing.tRunner(0xc000001c80, 0xc000c083c0)
        test_1        | 	/usr/local/go/src/testing/testing.go:1123 +0xef
        test_1        | created by testing.(*T).Run
        test_1        | 	/usr/local/go/src/testing/testing.go:1168 +0x2b3
        compare_test_1 exited with code 2
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 2
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2917182&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2917182&tab=artifacts#/) on master @ [9e821aa456bfbe8b5ec3a8ef79ae3954d52cb675](https://github.com/cockroachdb/cockroach/commits/9e821aa456bfbe8b5ec3a8ef79ae3954d52cb675):


```
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_10120 NULLS FIRST, col_10121 NULLS FIRST, col_10122 NULLS FIRST
        postgres_1    | 2021-04-23 18:14:56.620 UTC [110] ERROR:  function st_setsrid(geometry, bigint) does not exist at character 42
        postgres_1    | 2021-04-23 18:14:56.620 UTC [110] HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
        postgres_1    | 2021-04-23 18:14:56.620 UTC [110] STATEMENT:  SELECT
        postgres_1    | 		*
        postgres_1    | 	FROM
        postgres_1    | 		(
        postgres_1    | 			SELECT
        postgres_1    | 				st_geohash(st_setsrid(NULL::GEOMETRY, 8::INT8::INT8)::GEOMETRY::GEOMETRY)::TEXT AS col_10267,
        postgres_1    | 				B'1000' AS col_10268
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table1 AS tab_5539
        postgres_1    | 			WHERE
        postgres_1    | 				tab_5539.col1_4
        postgres_1    | 		)
        postgres_1    | 			AS tab_5540
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_10267 NULLS FIRST, col_10268 NULLS FIRST
        test_1        | --- FAIL: TestCompare (610.33s)
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     --- FAIL: TestCompare/postgres (8.67s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("2.9.10"),
        test_1        |             + 	string("2.9.4"),
        test_1        |               }))
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT postgis_libxml_version()::TEXT AS col_10443) AS tab_5630 ORDER BY col_10443 NULLS FIRST;
        test_1        |             
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT postgis_libxml_version()::STRING AS col_10443) AS tab_5630 ORDER BY col_10443 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2920167&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2920167&tab=artifacts#/) on master @ [3cd29547884feb24b150b67ca2a88073798a35fd](https://github.com/cockroachdb/cockroach/commits/3cd29547884feb24b150b67ca2a88073798a35fd):


```
        postgres_1    | 
        postgres_1    | waiting for server to shut down....2021-04-24 18:15:54.560 UTC [48] LOG:  received fast shutdown request
        postgres_1    | 2021-04-24 18:15:54.562 UTC [48] LOG:  aborting any active transactions
        postgres_1    | 2021-04-24 18:15:54.564 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2021-04-24 18:15:54.565 UTC [50] LOG:  shutting down
        postgres_1    | 2021-04-24 18:15:54.673 UTC [48] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-04-24 18:15:54.782 UTC [1] LOG:  starting PostgreSQL 13.2 (Debian 13.2-1.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
        postgres_1    | 2021-04-24 18:15:54.782 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-04-24 18:15:54.782 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-04-24 18:15:54.786 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-04-24 18:15:54.792 UTC [95] LOG:  database system was shut down at 2021-04-24 18:15:54 UTC
        postgres_1    | 2021-04-24 18:15:54.797 UTC [1] LOG:  database system is ready to accept connections
        postgres_1    | 2021-04-24 18:15:56.661 UTC [105] ERROR:  conversion between UTF8 and SQL_ASCII is not supported at character 753
        postgres_1    | 2021-04-24 18:15:56.661 UTC [105] STATEMENT:  CREATE TABLE table1 (col1_0 BOX2D, col1_1 TIME NOT NULL, col1_2 DATE NOT NULL, col1_3 INTERVAL, col1_4 DATE, col1_5 "char" NOT NULL, col1_6 REGTYPE NULL, col1_7 REGNAMESPACE NOT NULL, col1_8 BYTEA, col1_9 REGNAMESPACE NOT NULL, col1_10 BOOL NOT NULL, PRIMARY KEY (col1_5, col1_7), col1_11 TEXT NOT NULL GENERATED ALWAYS AS (lower(CAST(col1_10 AS TEXT))) STORED, col1_12 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col1_9 IS NULL THEN e'k\'+n'::TEXT ELSE '('::TEXT END) STORED, col1_13 TEXT NOT NULL GENERATED ALWAYS AS (lower(CAST(col1_1 AS TEXT))) STORED, col1_14 TEXT GENERATED ALWAYS AS (CASE WHEN col1_8 IS NULL THEN '[T'::TEXT ELSE ''::TEXT END) STORED, col1_15 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col1_9 IS NULL THEN '.o&'::TEXT ELSE e'\U00002603'::TEXT END) STORED);
        postgres_1    | 	CREATE UNIQUE INDEX ON table1 (col1_7, col1_8 DESC, col1_5 ASC, col1_6, col1_13 ASC, col1_14 ASC, col1_1 DESC, col1_4, col1_3 DESC, col1_11 ASC, col1_2 ASC, col1_10 DESC) INCLUDE (col1_9);
        postgres_1    | 	CREATE UNIQUE INDEX ON table1 (col1_5 ASC, col1_3 DESC, col1_4 DESC, col1_8 ASC, col1_6, col1_11 DESC, col1_7 DESC, col1_1 ASC, col1_12 DESC, col1_9, col1_10 ASC, col1_15 DESC, col1_13 ASC, col1_14, col1_2);
        postgres_1    | 	CREATE UNIQUE INDEX ON table1 (col1_14 DESC, col1_12) INCLUDE (col1_0, col1_2, col1_4, col1_6, col1_8, col1_9, col1_10);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('pbduas', 'fjxsf', 'uk');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (603.97s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (1.24s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:149: CREATE TABLE table1 (col1_0 BOX2D, col1_1 TIME NOT NULL, col1_2 DATE NOT NULL, col1_3 INTERVAL, col1_4 DATE, col1_5 "char" NOT NULL, col1_6 REGTYPE NULL, col1_7 REGNAMESPACE NOT NULL, col1_8 BYTEA, col1_9 REGNAMESPACE NOT NULL, col1_10 BOOL NOT NULL, PRIMARY KEY (col1_5, col1_7), col1_11 TEXT NOT NULL GENERATED ALWAYS AS (lower(CAST(col1_10 AS TEXT))) STORED, col1_12 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col1_9 IS NULL THEN e'k\'+n'::TEXT ELSE '('::TEXT END) STORED, col1_13 TEXT NOT NULL GENERATED ALWAYS AS (lower(CAST(col1_1 AS TEXT))) STORED, col1_14 TEXT GENERATED ALWAYS AS (CASE WHEN col1_8 IS NULL THEN '[T'::TEXT ELSE ''::TEXT END) STORED, col1_15 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col1_9 IS NULL THEN '.o&'::TEXT ELSE e'\U00002603'::TEXT END) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_7, col1_8 DESC, col1_5 ASC, col1_6, col1_13 ASC, col1_14 ASC, col1_1 DESC, col1_4, col1_3 DESC, col1_11 ASC, col1_2 ASC, col1_10 DESC) INCLUDE (col1_9);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_5 ASC, col1_3 DESC, col1_4 DESC, col1_8 ASC, col1_6, col1_11 DESC, col1_7 DESC, col1_1 ASC, col1_12 DESC, col1_9, col1_10 ASC, col1_15 DESC, col1_13 ASC, col1_14, col1_2);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_14 DESC, col1_12) INCLUDE (col1_0, col1_2, col1_4, col1_6, col1_8, col1_9, col1_10);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('pbduas', 'fjxsf', 'uk');
        test_1        |             
        test_1        |         compare_test.go:150: postgres: exec: ERROR: conversion between UTF8 and SQL_ASCII is not supported (SQLSTATE 0A000)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2921794&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2921794&tab=artifacts#/) on master @ [2d7f9a08bac49040a6d77c5465e2b4fd8fdac1f4](https://github.com/cockroachdb/cockroach/commits/2d7f9a08bac49040a6d77c5465e2b4fd8fdac1f4):


```
        postgres_1    | 			AS tab_165
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_281 NULLS FIRST,
        postgres_1    | 		col_282 NULLS FIRST,
        postgres_1    | 		col_283 NULLS FIRST,
        postgres_1    | 		col_284 NULLS FIRST,
        postgres_1    | 		col_285 NULLS FIRST,
        postgres_1    | 		col_286 NULLS FIRST,
        postgres_1    | 		col_287 NULLS FIRST
        test_1        | --- FAIL: TestCompare (606.97s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (3.24s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("fE"),
        test_1        |             + 	string("falseE"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT (('E':::STRING ~ 'B':::STRING)::BOOL || 'E':::STRING::STRING)::STRING AS col_357) AS tab_217
        test_1        |             ORDER BY
        test_1        |             	col_357 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT (('E'::TEXT ~ 'B'::TEXT)::BOOL || 'E'::TEXT::TEXT)::TEXT AS col_357) AS tab_217
        test_1        |             ORDER BY
        test_1        |             	col_357 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2924711&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2924711&tab=artifacts#/) on master @ [2dbd33ce319439bc424a52896a03e0a7ccff07f1](https://github.com/cockroachdb/cockroach/commits/2dbd33ce319439bc424a52896a03e0a7ccff07f1):


```
        postgres_1    | 		col_92122 NULLS FIRST,
        postgres_1    | 		col_92123 NULLS FIRST,
        postgres_1    | 		col_92124 NULLS FIRST,
        postgres_1    | 		col_92125 NULLS FIRST,
        postgres_1    | 		col_92126 NULLS FIRST,
        postgres_1    | 		col_92127 NULLS FIRST,
        postgres_1    | 		col_92128 NULLS FIRST
        test_1        | --- FAIL: TestCompare (664.08s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (62.58s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("2000-01-01 08:00:00H"),
        test_1        |             + 	string("2000-01-01 08:00:00+00H"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('2000-01-01 08:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ || 'H':::STRING::STRING)::STRING AS col_92160)
        test_1        |             		AS tab_60944
        test_1        |             ORDER BY
        test_1        |             	col_92160 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('2000-01-01 08:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ || 'H'::TEXT::TEXT)::TEXT AS col_92160)
        test_1        |             		AS tab_60944
        test_1        |             ORDER BY
        test_1        |             	col_92160 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2928834&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2928834&tab=artifacts#/) on master @ [9f198e190812bf301e8a2126cf43cd78e1cfb163](https://github.com/cockroachdb/cockroach/commits/9f198e190812bf301e8a2126cf43cd78e1cfb163):


```
        postgres_1    | 				tab_40729.col1_1 AS col_61717,
        postgres_1    | 				4::OID AS col_61718,
        postgres_1    | 				tab_40729.col1_3 AS col_61719,
        postgres_1    | 				tab_40729.col1_1 AS col_61720,
        postgres_1    | 				tab_40728.col_61715 AS col_61721,
        postgres_1    | 				rpad(tab_40729.col1_4::TEXT, tab_40729.col1_2::INT8, 'D'::TEXT::TEXT)::TEXT AS col_61722
        postgres_1    | 			FROM
        postgres_1    | 				(SELECT '2000-01-01 00:00:00'::TIMESTAMP AS col_61714) AS tab_40728 (col_61715),
        postgres_1    | 				postgres.public.table1 AS tab_40729
        postgres_1    | 		)
        postgres_1    | 			AS tab_40730
        postgres_1    | 	ORDER BY
        postgres_1    | 		CAST(col_61716 AS TEXT) NULLS FIRST,
        postgres_1    | 		col_61717 NULLS FIRST,
        postgres_1    | 		col_61718 NULLS FIRST,
        postgres_1    | 		col_61719 NULLS FIRST,
        postgres_1    | 		col_61720 NULLS FIRST,
        postgres_1    | 		col_61721 NULLS FIRST,
        postgres_1    | 		col_61722 NULLS FIRST
        test_1        | --- FAIL: TestCompare (646.69s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (45.11s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("3.1.1 aaf4c79"),
        test_1        |             + 	string("3.0.1 ec2a9aa"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT postgis_scripts_released()::STRING AS col_61723) AS tab_40731 ORDER BY col_61723 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT postgis_scripts_released()::TEXT AS col_61723) AS tab_40731 ORDER BY col_61723 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2932923&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2932923&tab=artifacts#/) on master @ [852e22a2ad78b1a19bd4f504f7079274c0c95fe5](https://github.com/cockroachdb/cockroach/commits/852e22a2ad78b1a19bd4f504f7079274c0c95fe5):


```
        postgres_1    | 2021-04-28 18:18:21.808 UTC [105] ERROR:  function set_masklen(inet, bigint) does not exist at character 126
        postgres_1    | 2021-04-28 18:18:21.808 UTC [105] HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
        postgres_1    | 2021-04-28 18:18:21.808 UTC [105] STATEMENT:  SELECT
        postgres_1    | 		*
        postgres_1    | 	FROM
        postgres_1    | 		(
        postgres_1    | 			SELECT
        postgres_1    | 				tab_187329.col4_5 AS col_358862,
        postgres_1    | 				NULL AS col_358863,
        postgres_1    | 				tab_187329.col4_7 AS col_358864,
        postgres_1    | 				set_masklen('::6/0'::INET::INET, tab_187329.col4_14::INT8)::INET AS col_358865
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table4 AS tab_187329
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_187329.col4_0 DESC
        postgres_1    | 		)
        postgres_1    | 			AS tab_187330
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_358862 NULLS FIRST, CAST(col_358863 AS TEXT) NULLS FIRST, col_358864 NULLS FIRST, col_358865 NULLS FIRST
        test_1        | --- FAIL: TestCompare (834.44s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (230.05s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("2.9.10"),
        test_1        |             + 	string("2.9.4"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT postgis_libxml_version()::STRING AS col_358970) AS tab_187391 ORDER BY col_358970 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT postgis_libxml_version()::TEXT AS col_358970) AS tab_187391 ORDER BY col_358970 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2937583&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2937583&tab=artifacts#/) on master @ [445175daed0848429219755703febfb3b50a73e9](https://github.com/cockroachdb/cockroach/commits/445175daed0848429219755703febfb3b50a73e9):


```
        postgres_1    | 				false
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_37124.col1_9 ASC, tab_37124.col1_13 DESC
        postgres_1    | 		)
        postgres_1    | 			AS tab_37125
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_72052 NULLS FIRST, col_72053 NULLS FIRST, col_72054 NULLS FIRST, col_72055 NULLS FIRST, col_72056 NULLS FIRST
        test_1        | --- FAIL: TestCompare (650.25s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (49.10s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("E2000-01-01 08:00:00"),
        test_1        |             + 	string("E2000-01-01 08:00:00+00"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('E':::STRING::STRING || '2000-01-01 08:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ)::STRING AS col_72340)
        test_1        |             		AS tab_37296
        test_1        |             ORDER BY
        test_1        |             	col_72340 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('E'::TEXT::TEXT || '2000-01-01 08:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ)::TEXT AS col_72340)
        test_1        |             		AS tab_37296
        test_1        |             ORDER BY
        test_1        |             	col_72340 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2941633&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2941633&tab=artifacts#/) on master @ [42e2e4265087751695afc05fc5ad8ec534fe0121](https://github.com/cockroachdb/cockroach/commits/42e2e4265087751695afc05fc5ad8ec534fe0121):


```
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_0 DESC);
        postgres_1    | 	CREATE INDEX ON table3 (col3_2 ASC, col3_1);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_0, col3_1 DESC, col3_2 ASC);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('awy', 'tgeo', 'j');
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('h', 'zi', 'pksvat', 'qbouf', 'txhd');
        postgres_1    | 	CREATE TYPE rand_typ_2 AS ENUM ('tb');
        postgres_1    | 	CREATE TYPE rand_typ_3 AS ENUM ('oa', 'n', 'ivmn');
        postgres_1    | 	CREATE TYPE rand_typ_4 AS ENUM ('s', 'mpkrbs', 'jspc');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (605.52s)
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     --- FAIL: TestCompare/postgres (2.56s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:149: CREATE TABLE table1 (col1_0 DATE NOT NULL, col1_1 INT8 NOT NULL, col1_2 DATE NULL, col1_3 VARCHAR NULL, col1_4 REGPROC, col1_5 INT8 NULL, col1_6 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col1_2 AS TEXT))) STORED, col1_7 INT8 NULL GENERATED ALWAYS AS (col1_5 + (-1246576657)::INT8) STORED);
        test_1        |             CREATE INDEX ON table1 (col1_5 ASC, col1_3, col1_6 DESC, col1_2 DESC, col1_1, col1_0);
        test_1        |             CREATE INDEX ON table1 (col1_3, col1_4 DESC, col1_5, col1_0 ASC, col1_7 DESC, col1_6 DESC, col1_1 DESC);
        test_1        |             CREATE INDEX ON table1 (col1_5 DESC, col1_3, col1_7, col1_6 DESC, col1_2, col1_4 ASC, col1_0);
        test_1        |             CREATE INDEX ON table1 (col1_7, col1_1 ASC);
        test_1        |             CREATE INDEX ON table1 (col1_3 DESC, col1_0, col1_1 DESC);
        test_1        |             CREATE INDEX ON table1 (col1_7, col1_5, col1_0 ASC) INCLUDE (col1_6);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_1 DESC, col1_5, col1_2, col1_0, col1_4 ASC) INCLUDE (col1_3, col1_6, col1_7);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_5, col1_1 DESC, col1_6 DESC, col1_7, col1_2);
        test_1        |             CREATE TABLE table2 (col2_0 OID NOT NULL, col2_1 FLOAT8, col2_2 VARCHAR NOT NULL, col2_3 FLOAT8 NOT NULL, col2_4 INT8, col2_5 BOX2D NOT NULL, col2_6 DECIMAL, col2_7 BIT(35), col2_8 DECIMAL, col2_9 BOOL NOT NULL, col2_10 DATE NOT NULL, col2_11 INET NULL, PRIMARY KEY (col2_10, col2_0, col2_9));
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_3, col2_9, col2_7, col2_4 ASC, col2_11 ASC, col2_2 DESC, col2_1 DESC, col2_0 ASC, col2_6 DESC, col2_8, col2_10 DESC);
        test_1        |             CREATE TABLE table3 (col3_0 OID, col3_1 TIMETZ NOT NULL, col3_2 FLOAT8 NOT NULL, PRIMARY KEY (col3_2));
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 DESC);
        test_1        |             CREATE INDEX ON table3 (col3_2 ASC, col3_1);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0, col3_1 DESC, col3_2 ASC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('awy', 'tgeo', 'j');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('h', 'zi', 'pksvat', 'qbouf', 'txhd');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('tb');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('oa', 'n', 'ivmn');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('s', 'mpkrbs', 'jspc');
        test_1        |             
        test_1        |         compare_test.go:150: postgres: exec: ERROR: generation expression is not immutable (SQLSTATE 42P17)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2944552&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2944552&tab=artifacts#/) on master @ [684e753c15f3fc58df79b6ea70e7b6715eae4835](https://github.com/cockroachdb/cockroach/commits/684e753c15f3fc58df79b6ea70e7b6715eae4835):


```
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('i', 'gyl', 'o', 'e', 'yvlh', 'l');
        postgres_1    | 	CREATE TYPE rand_typ_2 AS ENUM ('haju');
        postgres_1    | 	CREATE TYPE rand_typ_3 AS ENUM ('hgigz', 'waoe', 'xryqy');
        postgres_1    | 	CREATE TYPE rand_typ_4 AS ENUM ('kxazrp');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (606.65s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (3.00s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:149: CREATE TABLE table1 (col1_0 TIMETZ NOT NULL, col1_1 TIME, col1_2 GEOGRAPHY[] NOT NULL, col1_3 BYTEA, col1_4 TIMETZ, col1_5 INT8 NOT NULL, col1_6 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col1_2 IS NULL THEN ''::TEXT ELSE e'\nv3'::TEXT END) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_6 ASC, col1_4, col1_3 DESC) INCLUDE (col1_1, col1_2, col1_5);
        test_1        |             CREATE INDEX ON table1 (col1_5 ASC, col1_1 DESC) INCLUDE (col1_0, col1_2, col1_3, col1_4);
        test_1        |             CREATE TABLE table2 (col2_0 "char" NOT NULL, col2_1 REGCLASS NOT NULL, col2_2 BOX2D NULL, col2_3 REGTYPE NOT NULL, col2_4 REGPROCEDURE NOT NULL, PRIMARY KEY (col2_1, col2_0), col2_5 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col2_3 IS NULL THEN e'|\x01s'::TEXT ELSE '^ '::TEXT END) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_3) INCLUDE (col2_4, col2_5);
        test_1        |             CREATE INDEX ON table2 (col2_0 ASC, col2_4 DESC) INCLUDE (col2_3);
        test_1        |             CREATE INDEX ON table2 (col2_0 DESC, col2_5 DESC, col2_4, col2_3 DESC, col2_1 DESC);
        test_1        |             CREATE TABLE table3 (col2_1 REGCLASS NOT NULL, col2_0 "char" NOT NULL, col3_2 CHAR NULL, PRIMARY KEY (col2_1, col2_0));
        test_1        |             CREATE INDEX ON table3 (col2_1, col2_0 ASC, col3_2 DESC);
        test_1        |             CREATE INDEX ON table3 (col2_1, col3_2 ASC);
        test_1        |             CREATE INDEX ON table3 (col3_2 ASC, col2_1, col2_0 DESC);
        test_1        |             CREATE INDEX ON table3 (col3_2 ASC);
        test_1        |             CREATE INDEX ON table3 (col2_1 DESC, col2_0 DESC);
        test_1        |             CREATE INDEX ON table3 (col2_0, col3_2 ASC, col2_1 DESC);
        test_1        |             CREATE INDEX ON table3 (col2_1 DESC) INCLUDE (col3_2);
        test_1        |             CREATE INDEX ON table3 (col3_2, col2_1 DESC, col2_0 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_2 ASC, col2_0, col2_1 DESC);
        test_1        |             CREATE TABLE table4 (col4_0 FLOAT8 NULL, col4_1 OID, col4_2 BOOL[] NOT NULL, col4_3 REGTYPE NOT NULL, col4_4 JSONB NOT NULL, col4_5 CHAR NULL, col4_6 REGCLASS NOT NULL);
        test_1        |             CREATE TABLE table5 (col5_0 INTERVAL[] NOT NULL, col5_1 UUID, col5_2 BIT(17)[], col5_3 REGPROC NULL, col5_4 DATE, col5_5 INT8, col5_6 BYTEA, col5_7 NAME NOT NULL, PRIMARY KEY (col5_7), col5_8 INT8 GENERATED ALWAYS AS (col5_5 + (-1806224584)::INT8) STORED, col5_9 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col5_3 IS NULL THEN e'\x00'::TEXT ELSE e'\'-8\x0f$ZG'::TEXT END) STORED, col5_10 TEXT GENERATED ALWAYS AS (lower(CAST(col5_4 AS TEXT))) STORED, col5_11 TEXT GENERATED ALWAYS AS (lower(CAST(col5_4 AS TEXT))) STORED, col5_12 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col5_0 IS NULL THEN e';.\\fO'::TEXT ELSE e'n\ru\x13\f'::TEXT END) STORED);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('bzil', 'd', 'lva', 'hmfrob', 'rfzrn', 'aekyoh');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('i', 'gyl', 'o', 'e', 'yvlh', 'l');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('haju');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('hgigz', 'waoe', 'xryqy');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('kxazrp');
        test_1        |             
        test_1        |         compare_test.go:150: postgres: exec: ERROR: invalid byte sequence for encoding "SQL_ASCII": 0x00 (SQLSTATE 22021)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2945605&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2945605&tab=artifacts#/) on master @ [dbf711a83969210bcf706d65f53f888856da6ad0](https://github.com/cockroachdb/cockroach/commits/dbf711a83969210bcf706d65f53f888856da6ad0):


```
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     --- FAIL: TestCompare/postgres (39.09s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("f2000-01-01 00:00:00D"),
        test_1        |             + 	string("false2000-01-01 00:00:00+00D"),
        test_1        |               }))
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			(false::BOOL || ('2000-01-01 00:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ || 'D'::TEXT::TEXT)::TEXT::TEXT)::TEXT
        test_1        |             				AS col_49158
        test_1        |             	)
        test_1        |             		AS tab_26983
        test_1        |             ORDER BY
        test_1        |             	col_49158 NULLS FIRST;
        test_1        |             
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			(false::BOOL || ('2000-01-01 00:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ || 'D':::STRING::STRING)::STRING::STRING)::STRING
        test_1        |             				AS col_49158
        test_1        |             	)
        test_1        |             		AS tab_26983
        test_1        |             ORDER BY
        test_1        |             	col_49158 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2949836&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2949836&tab=artifacts#/) on master @ [f9dca40d3afb0a605eeee76bec4a07ff65d6fec0](https://github.com/cockroachdb/cockroach/commits/f9dca40d3afb0a605eeee76bec4a07ff65d6fec0):


```
        postgres_1    | 		col_197554 NULLS FIRST,
        postgres_1    | 		col_197555 NULLS FIRST,
        postgres_1    | 		col_197556 NULLS FIRST,
        postgres_1    | 		col_197557 NULLS FIRST,
        postgres_1    | 		col_197558 NULLS FIRST,
        postgres_1    | 		col_197559 NULLS FIRST,
        postgres_1    | 		col_197560 NULLS FIRST
        postgres_1    | 2021-05-04 18:17:20.934 UTC [105] ERROR:  time zone "D" not recognized
        postgres_1    | 2021-05-04 18:17:20.934 UTC [105] STATEMENT:  SELECT
        postgres_1    | 		*
        postgres_1    | 	FROM
        postgres_1    | 		(
        postgres_1    | 			SELECT
        postgres_1    | 				timezone('D'::TEXT::TEXT, '2000-01-01 06:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ)::TIMESTAMP
        postgres_1    | 					AS col_197593
        postgres_1    | 		)
        postgres_1    | 			AS tab_125450
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_197593 NULLS FIRST
        test_1        | --- FAIL: TestCompare (750.66s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (146.38s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("3.1.1"),
        test_1        |             + 	string("3.0.1"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT postgis_lib_version()::STRING AS col_197597) AS tab_125454 ORDER BY col_197597 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT postgis_lib_version()::TEXT AS col_197597) AS tab_125454 ORDER BY col_197597 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2958001&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2958001&tab=artifacts#/) on master @ [dee0558010170d938ead781d82514fd0bb3b0b88](https://github.com/cockroachdb/cockroach/commits/dee0558010170d938ead781d82514fd0bb3b0b88):


```
        postgres_1    | 				THEN 2::DECIMAL
        postgres_1    | 				ELSE tab_4292.col2_7
        postgres_1    | 				END
        postgres_1    | 					AS col_7599,
        postgres_1    | 				tab_4292.col2_6 AS col_7600
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table2 AS tab_4292
        postgres_1    | 		)
        postgres_1    | 			AS tab_4293
        postgres_1    | 	ORDER BY
        postgres_1    | 		CAST(col_7592 AS TEXT) NULLS FIRST,
        postgres_1    | 		col_7593 NULLS FIRST,
        postgres_1    | 		col_7594 NULLS FIRST,
        postgres_1    | 		col_7595 NULLS FIRST,
        postgres_1    | 		col_7596 NULLS FIRST,
        postgres_1    | 		col_7597 NULLS FIRST,
        postgres_1    | 		col_7598 NULLS FIRST,
        postgres_1    | 		col_7599 NULLS FIRST,
        postgres_1    | 		col_7600 NULLS FIRST
        test_1        | --- FAIL: TestCompare (611.64s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (8.16s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("3.1.1 aaf4c79"),
        test_1        |             + 	string("3.0.1 ec2a9aa"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT postgis_scripts_released()::STRING AS col_7680) AS tab_4334 ORDER BY col_7680 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT postgis_scripts_released()::TEXT AS col_7680) AS tab_4334 ORDER BY col_7680 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2960945&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2960945&tab=artifacts#/) on master @ [329a636debe0d968f2f4d6fe0daef957c4c41583](https://github.com/cockroachdb/cockroach/commits/329a636debe0d968f2f4d6fe0daef957c4c41583):


```
        test_1        | --- FAIL: TestCompare (3.12s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (0.00s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        | panic: geos: no locations to init GEOS [recovered]
        test_1        | 	panic: geos: no locations to init GEOS
        test_1        | 
        test_1        | goroutine 38 [running]:
        test_1        | testing.tRunner.func1.1(0x1ffe4a0, 0xc000137d60)
        test_1        | 	/usr/local/go/src/testing/testing.go:1072 +0x30d
        test_1        | testing.tRunner.func1(0xc00030ea80)
        test_1        | 	/usr/local/go/src/testing/testing.go:1075 +0x41a
        test_1        | panic(0x1ffe4a0, 0xc000137d60)
        test_1        | 	/usr/local/go/src/runtime/panic.go:969 +0x1b9
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.encodeInvertedIndexHistogramUpperBounds(0x39b1ae0, 0x2af6f80, 0xc0009fe1b0, 0x2af6f80, 0xc0009fe1b0, 0xc00072e1c0)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:368 +0x5d3
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.randHistogram(0xc000821c20, 0x39b1ae0, 0x39b1ae0, 0x2ea5d148bc96fd0f, 0x6, 0x2e0000c000338530)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:301 +0x10c
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator.func1(0xc0002059e0)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:220 +0x9d
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator(0xc000821c20, 0xc000821c50, 0x3, 0x3, 0x0, 0x0, 0x0, 0x0)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:251 +0xbe6
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.MultiStatementMutation.Mutate(0x274dee8, 0xc000821c20, 0xc000821c50, 0x3, 0x3, 0x0, 0xc0004343c0, 0xe0, 0x0)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:97 +0x4e
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.RandCreateTables(0xc000821c20, 0x2192486, 0x5, 0x3, 0xc0009a5a40, 0x3, 0x3, 0xe0, 0x4000, 0xc000309901)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/schema.go:94 +0x224
        test_1        | github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTablesN(0xc000821c20, 0x3, 0x2, 0x3cf1200)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:91 +0x1a5
        test_1        | github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTables(0xc000821c20, 0x30, 0x2059320)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:77 +0x4f
        test_1        | github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.wrapCommonSetup.func1(0xc000821c20, 0x4a9f86fdbf726fa4, 0x11)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:43 +0x44
        test_1        | github.com/cockroachdb/cockroach/pkg/compose/compare/compare.TestCompare.func2(0xc00030ea80)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/compose/compare/compare/compare_test.go:127 +0x113
        test_1        | testing.tRunner(0xc00030ea80, 0xc000821bf0)
        test_1        | 	/usr/local/go/src/testing/testing.go:1123 +0xef
        test_1        | created by testing.(*T).Run
        test_1        | 	/usr/local/go/src/testing/testing.go:1168 +0x2b3
        compare_test_1 exited with code 2
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 2
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2963121&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2963121&tab=artifacts#/) on master @ [d302ba88430dd2438a5edcee15edf18286f81485](https://github.com/cockroachdb/cockroach/commits/d302ba88430dd2438a5edcee15edf18286f81485):


```
        postgres_1    | 		CAST(col_29182 AS TEXT) NULLS FIRST,
        postgres_1    | 		col_29183 NULLS FIRST,
        postgres_1    | 		col_29184 NULLS FIRST,
        postgres_1    | 		col_29185 NULLS FIRST,
        postgres_1    | 		col_29186 NULLS FIRST,
        postgres_1    | 		col_29187 NULLS FIRST,
        postgres_1    | 		col_29188 NULLS FIRST
        test_1        | --- FAIL: TestCompare (622.81s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (19.04s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("I2000-01-01 03:00:00"),
        test_1        |             + 	string("I2000-01-01 03:00:00+00"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('I':::STRING::STRING || '2000-01-01 03:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ)::STRING AS col_29239)
        test_1        |             		AS tab_15960
        test_1        |             ORDER BY
        test_1        |             	col_29239 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('I'::TEXT::TEXT || '2000-01-01 03:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ)::TEXT AS col_29239)
        test_1        |             		AS tab_15960
        test_1        |             ORDER BY
        test_1        |             	col_29239 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2964422&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2964422&tab=artifacts#/) on master @ [704e841bbc440deb4321447d9f36fc6ba41c39df](https://github.com/cockroachdb/cockroach/commits/704e841bbc440deb4321447d9f36fc6ba41c39df):


```
        postgres_1    | 				1::OID AS col_111699, tab_85334.col1_0 AS col_111700, 'G'::TEXT AS col_111701
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table1 AS tab_85334
        postgres_1    | 			WHERE
        postgres_1    | 				(COALESCE(st_geographyfromtext('E'::TEXT::TEXT, NULL::INT8)::GEOGRAPHY, NULL) IS NOT DISTINCT FROM NULL)
        postgres_1    | 		)
        postgres_1    | 			AS tab_85335
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_111699 NULLS FIRST, col_111700 NULLS FIRST, col_111701 NULLS FIRST
        test_1        | --- FAIL: TestCompare (682.98s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (80.03s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("tD"),
        test_1        |             + 	string("trueD"),
        test_1        |               }))
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT (true::BOOL || 'D'::TEXT::TEXT)::TEXT AS col_112031) AS tab_85588
        test_1        |             ORDER BY
        test_1        |             	col_112031 NULLS FIRST;
        test_1        |             
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT (true::BOOL || 'D':::STRING::STRING)::STRING AS col_112031) AS tab_85588
        test_1        |             ORDER BY
        test_1        |             	col_112031 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2965592&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2965592&tab=artifacts#/) on master @ [ac033403c8fbb699cff200316e432dc60b47be6e](https://github.com/cockroachdb/cockroach/commits/ac033403c8fbb699cff200316e432dc60b47be6e):


```
        postgres_1    | 2021-05-10 18:14:28.782 UTC [48] LOG:  aborting any active transactions
        postgres_1    | 2021-05-10 18:14:28.783 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2021-05-10 18:14:28.785 UTC [50] LOG:  shutting down
        postgres_1    | 2021-05-10 18:14:28.886 UTC [48] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-05-10 18:14:29.002 UTC [1] LOG:  starting PostgreSQL 13.2 (Debian 13.2-1.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
        postgres_1    | 2021-05-10 18:14:29.002 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-05-10 18:14:29.003 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-05-10 18:14:29.006 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-05-10 18:14:29.012 UTC [95] LOG:  database system was shut down at 2021-05-10 18:14:28 UTC
        postgres_1    | 2021-05-10 18:14:29.017 UTC [1] LOG:  database system is ready to accept connections
        postgres_1    | 2021-05-10 18:14:30.927 UTC [105] ERROR:  generation expression is not immutable
        postgres_1    | 2021-05-10 18:14:30.927 UTC [105] STATEMENT:  CREATE TABLE table1 (col1_0 BOOL, col1_1 TIMESTAMP, col1_2 TIMETZ NULL, col1_3 DATE NULL, col1_4 DATE NOT NULL, col1_5 BOOL, col1_6 "char" NULL, col1_7 INET NULL, col1_8 CHAR, col1_9 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col1_7 AS TEXT))) STORED, col1_10 TEXT NOT NULL GENERATED ALWAYS AS (lower(CAST(col1_4 AS TEXT))) STORED);
        postgres_1    | 	CREATE UNIQUE INDEX ON table1 (col1_6 ASC, col1_5);
        postgres_1    | 	CREATE UNIQUE INDEX ON table1 (col1_2 ASC, col1_8 DESC) INCLUDE (col1_0, col1_1, col1_3, col1_4, col1_5, col1_10);
        postgres_1    | 	CREATE INDEX ON table1 (col1_5 DESC, col1_1 ASC);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('wkmgga', 'yieqr', 's', 'ytwdpi', 'rwu', 'syyy');
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('brjdd');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (605.05s)
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     --- FAIL: TestCompare/postgres (1.32s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:149: CREATE TABLE table1 (col1_0 BOOL, col1_1 TIMESTAMP, col1_2 TIMETZ NULL, col1_3 DATE NULL, col1_4 DATE NOT NULL, col1_5 BOOL, col1_6 "char" NULL, col1_7 INET NULL, col1_8 CHAR, col1_9 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col1_7 AS TEXT))) STORED, col1_10 TEXT NOT NULL GENERATED ALWAYS AS (lower(CAST(col1_4 AS TEXT))) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_6 ASC, col1_5);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_2 ASC, col1_8 DESC) INCLUDE (col1_0, col1_1, col1_3, col1_4, col1_5, col1_10);
        test_1        |             CREATE INDEX ON table1 (col1_5 DESC, col1_1 ASC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('wkmgga', 'yieqr', 's', 'ytwdpi', 'rwu', 'syyy');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('brjdd');
        test_1        |             
        test_1        |         compare_test.go:150: postgres: exec: ERROR: generation expression is not immutable (SQLSTATE 42P17)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2973751&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2973751&tab=artifacts#/) on master @ [88754f2866fde1f37620b04c6a708eefc0b5be54](https://github.com/cockroachdb/cockroach/commits/88754f2866fde1f37620b04c6a708eefc0b5be54):


```
        postgres_1    | 			AS tab_149544
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_225091 NULLS FIRST,
        postgres_1    | 		col_225092 NULLS FIRST,
        postgres_1    | 		col_225093 NULLS FIRST,
        postgres_1    | 		col_225094 NULLS FIRST,
        postgres_1    | 		col_225095 NULLS FIRST,
        postgres_1    | 		col_225096 NULLS FIRST,
        postgres_1    | 		col_225097 NULLS FIRST
        test_1        | --- FAIL: TestCompare (765.80s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (162.51s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	int16(4),
        test_1        |             + 	s"4",
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT COALESCE(st_zmflag(NULL::GEOMETRY)::INT2, 4:::INT8) AS col_225113) AS tab_149554
        test_1        |             ORDER BY
        test_1        |             	col_225113 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT COALESCE(st_zmflag(NULL::GEOMETRY)::INT8, 4::INT8) AS col_225113) AS tab_149554
        test_1        |             ORDER BY
        test_1        |             	col_225113 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2978385&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2978385&tab=artifacts#/) on master @ [5c0b2ec40f06678c0a09c48bcca2904354765734](https://github.com/cockroachdb/cockroach/commits/5c0b2ec40f06678c0a09c48bcca2904354765734):


```
        postgres_1    | 			SELECT
        postgres_1    | 				(NULL::INET & set_masklen(netmask('::9/0'::INET::INET)::INET::INET, 5::INT8::INT8)::INET::INET)::INET
        postgres_1    | 					AS col_7141
        postgres_1    | 		)
        postgres_1    | 			AS tab_4067
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_7141 NULLS FIRST
        test_1        | --- FAIL: TestCompare (612.49s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (8.94s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("2000-01-01 08:00:00A"),
        test_1        |             + 	string("2000-01-01 08:00:00+00A"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('2000-01-01 08:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ || 'A':::STRING::STRING)::STRING AS col_7312)
        test_1        |             		AS tab_4183
        test_1        |             ORDER BY
        test_1        |             	col_7312 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('2000-01-01 08:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ || 'A'::TEXT::TEXT)::TEXT AS col_7312)
        test_1        |             		AS tab_4183
        test_1        |             ORDER BY
        test_1        |             	col_7312 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2982474&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2982474&tab=artifacts#/) on master @ [68b69b6e24c4016e724492b19a0567ad189fd8fd](https://github.com/cockroachdb/cockroach/commits/68b69b6e24c4016e724492b19a0567ad189fd8fd):


```
        postgres_1    | 2021-05-14 18:13:58.010 UTC [106] ERROR:  operator does not exist: date + bigint at character 52
        postgres_1    | 2021-05-14 18:13:58.010 UTC [106] HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.
        postgres_1    | 2021-05-14 18:13:58.010 UTC [106] STATEMENT:  SELECT
        postgres_1    | 		*
        postgres_1    | 	FROM
        postgres_1    | 		(SELECT (('2000-01-04'::DATE::DATE + 2::INT8::INT8)::DATE::DATE + 8::INT8::INT8)::DATE AS col_4648) AS tab_2470
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_4648 NULLS FIRST
        test_1        | --- FAIL: TestCompare (607.83s)
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     --- FAIL: TestCompare/postgres (6.17s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |               	uint32(1),
        test_1        |             - 	string("3.0.1 ec2a9aa"),
        test_1        |             + 	string("3.1.1 aaf4c79"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT 1:::OID AS col_4706, ltrim(postgis_scripts_released()::STRING::STRING)::STRING AS col_4707) AS tab_2509
        test_1        |             ORDER BY
        test_1        |             	col_4706 NULLS FIRST, col_4707 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT 1::OID AS col_4706, ltrim(postgis_scripts_released()::TEXT::TEXT)::TEXT AS col_4707) AS tab_2509
        test_1        |             ORDER BY
        test_1        |             	col_4706 NULLS FIRST, col_4707 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2988806&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2988806&tab=artifacts#/) on master @ [25e26a5f0361fcfbf71cee86f963b93f806604d8](https://github.com/cockroachdb/cockroach/commits/25e26a5f0361fcfbf71cee86f963b93f806604d8):


```
        postgres_1    | 2021-05-17 18:13:22.937 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-05-17 18:13:22.937 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-05-17 18:13:22.941 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-05-17 18:13:22.947 UTC [95] LOG:  database system was shut down at 2021-05-17 18:13:22 UTC
        postgres_1    | 2021-05-17 18:13:22.952 UTC [1] LOG:  database system is ready to accept connections
        postgres_1    | 2021-05-17 18:13:25.562 UTC [105] ERROR:  invalid byte sequence for encoding "SQL_ASCII": 0x00
        postgres_1    | 2021-05-17 18:13:25.562 UTC [105] STATEMENT:  CREATE TABLE table1 (col1_0 TIMESTAMP NULL, col1_1 GEOGRAPHY NOT NULL);
        postgres_1    | 	CREATE INDEX ON table1 (col1_0);
        postgres_1    | 	CREATE TABLE table2 (col2_0 BYTEA NOT NULL, col2_1 UUID NOT NULL, col2_2 TIME NOT NULL, col2_3 INET NOT NULL, PRIMARY KEY (col2_0, col2_2, col2_3, col2_1), col2_4 TEXT NOT NULL GENERATED ALWAYS AS (lower(CAST(col2_1 AS TEXT))) STORED);
        postgres_1    | 	CREATE INDEX ON table2 (col2_4 DESC, col2_2, col2_1 DESC, col2_3 ASC, col2_0);
        postgres_1    | 	CREATE INDEX ON table2 (col2_0, col2_2, col2_3, col2_1 ASC, col2_4 ASC);
        postgres_1    | 	CREATE TABLE table3 (col3_0 UUID, col3_1 REGNAMESPACE, col3_2 INT8 NULL, col3_3 INT8[] NOT NULL, col3_4 "char", col3_5 NAME, col3_6 "char" NOT NULL, col3_7 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col3_3 IS NULL THEN e'\x00'::TEXT ELSE e'\x1fL\\:&d'::TEXT END) STORED, col3_8 TEXT GENERATED ALWAYS AS (lower(col3_5)) STORED, col3_9 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col3_3 IS NULL THEN e'\x7f{.B'::TEXT ELSE 'o}QR'::TEXT END) STORED, col3_10 TEXT GENERATED ALWAYS AS (lower(col3_4)) STORED, col3_11 TEXT GENERATED ALWAYS AS (CASE WHEN col3_1 IS NULL THEN e'>\t'::TEXT ELSE ''::TEXT END) STORED, col3_12 TEXT GENERATED ALWAYS AS (lower(col3_4)) STORED);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_6 DESC, col3_8, col3_10, col3_5 ASC, col3_0) INCLUDE (col3_2, col3_4);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_12, col3_5, col3_7, col3_2 ASC, col3_6 DESC, col3_10, col3_9 DESC, col3_11) INCLUDE (col3_0, col3_4);
        postgres_1    | 	CREATE INDEX ON table3 (col3_11);
        postgres_1    | 	CREATE INDEX ON table3 (col3_8 DESC, col3_1 ASC, col3_9 ASC, col3_4 ASC);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('shb', 'ia', 'oeaygi', 'z');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (604.89s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (2.01s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:149: CREATE TABLE table1 (col1_0 TIMESTAMP NULL, col1_1 GEOGRAPHY NOT NULL);
        test_1        |             CREATE INDEX ON table1 (col1_0);
        test_1        |             CREATE TABLE table2 (col2_0 BYTEA NOT NULL, col2_1 UUID NOT NULL, col2_2 TIME NOT NULL, col2_3 INET NOT NULL, PRIMARY KEY (col2_0, col2_2, col2_3, col2_1), col2_4 TEXT NOT NULL GENERATED ALWAYS AS (lower(CAST(col2_1 AS TEXT))) STORED);
        test_1        |             CREATE INDEX ON table2 (col2_4 DESC, col2_2, col2_1 DESC, col2_3 ASC, col2_0);
        test_1        |             CREATE INDEX ON table2 (col2_0, col2_2, col2_3, col2_1 ASC, col2_4 ASC);
        test_1        |             CREATE TABLE table3 (col3_0 UUID, col3_1 REGNAMESPACE, col3_2 INT8 NULL, col3_3 INT8[] NOT NULL, col3_4 "char", col3_5 NAME, col3_6 "char" NOT NULL, col3_7 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col3_3 IS NULL THEN e'\x00'::TEXT ELSE e'\x1fL\\:&d'::TEXT END) STORED, col3_8 TEXT GENERATED ALWAYS AS (lower(col3_5)) STORED, col3_9 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col3_3 IS NULL THEN e'\x7f{.B'::TEXT ELSE 'o}QR'::TEXT END) STORED, col3_10 TEXT GENERATED ALWAYS AS (lower(col3_4)) STORED, col3_11 TEXT GENERATED ALWAYS AS (CASE WHEN col3_1 IS NULL THEN e'>\t'::TEXT ELSE ''::TEXT END) STORED, col3_12 TEXT GENERATED ALWAYS AS (lower(col3_4)) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_6 DESC, col3_8, col3_10, col3_5 ASC, col3_0) INCLUDE (col3_2, col3_4);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_12, col3_5, col3_7, col3_2 ASC, col3_6 DESC, col3_10, col3_9 DESC, col3_11) INCLUDE (col3_0, col3_4);
        test_1        |             CREATE INDEX ON table3 (col3_11);
        test_1        |             CREATE INDEX ON table3 (col3_8 DESC, col3_1 ASC, col3_9 ASC, col3_4 ASC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('shb', 'ia', 'oeaygi', 'z');
        test_1        |             
        test_1        |         compare_test.go:150: postgres: exec: ERROR: invalid byte sequence for encoding "SQL_ASCII": 0x00 (SQLSTATE 22021)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience @rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2993937&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2993937&tab=artifacts#/) on master @ [eecead5c3aa93b5718a57c47970d67ee3035e794](https://github.com/cockroachdb/cockroach/commits/eecead5c3aa93b5718a57c47970d67ee3035e794):


```
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (64.10s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	s"0",
        test_1        |             + 	s"1",
        test_1        |               }))
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			ceiling(power(COALESCE(0::DECIMAL, 4::DECIMAL)::DECIMAL, 0::DECIMAL::DECIMAL)::DECIMAL::DECIMAL)::DECIMAL
        test_1        |             				AS col_70026
        test_1        |             	)
        test_1        |             		AS tab_64080
        test_1        |             ORDER BY
        test_1        |             	col_70026 NULLS FIRST;
        test_1        |             
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			ceiling(power(COALESCE(0:::DECIMAL, 4:::DECIMAL)::DECIMAL, 0:::DECIMAL::DECIMAL)::DECIMAL::DECIMAL)::DECIMAL
        test_1        |             				AS col_70026
        test_1        |             	)
        test_1        |             		AS tab_64080
        test_1        |             ORDER BY
        test_1        |             	col_70026 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2998289&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=2998289&tab=artifacts#/) on master @ [c305dbcff1bcf621ae8edf058b220033c98d6526](https://github.com/cockroachdb/cockroach/commits/c305dbcff1bcf621ae8edf058b220033c98d6526):


```
        postgres_1    | 		col_78513 NULLS FIRST,
        postgres_1    | 		col_78514 NULLS FIRST,
        postgres_1    | 		col_78515 NULLS FIRST,
        postgres_1    | 		col_78516 NULLS FIRST
        test_1        | --- FAIL: TestCompare (653.86s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (49.19s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |               	nil,
        test_1        |               	nil,
        test_1        |             - 	string("3.0.1 ec2a9aa"),
        test_1        |             + 	string("3.1.1 aaf4c79"),
        test_1        |               	nil,
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT NULL AS col_78525, NULL AS col_78526, postgis_scripts_installed()::STRING AS col_78527, NULL AS col_78528)
        test_1        |             		AS tab_40807
        test_1        |             ORDER BY
        test_1        |             	CAST(col_78525 AS STRING) NULLS FIRST, col_78526 NULLS FIRST, col_78527 NULLS FIRST, col_78528 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT NULL AS col_78525, NULL AS col_78526, postgis_scripts_installed()::TEXT AS col_78527, NULL AS col_78528)
        test_1        |             		AS tab_40807
        test_1        |             ORDER BY
        test_1        |             	CAST(col_78525 AS TEXT) NULLS FIRST, col_78526 NULLS FIRST, col_78527 NULLS FIRST, col_78528 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3005161&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3005161&tab=artifacts#/) on master @ [d46b65000118d7f401d61f90a11d0a2438949216](https://github.com/cockroachdb/cockroach/commits/d46b65000118d7f401d61f90a11d0a2438949216):


```
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT postgis_liblwgeom_version()::TEXT AS col_83837) AS tab_44960 ORDER BY col_83837 NULLS FIRST;
        test_1        |             
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT postgis_liblwgeom_version()::STRING AS col_83837) AS tab_44960 ORDER BY col_83837 NULLS FIRST;
        test_1        |             
        test_1        |     --- FAIL: TestCompare/mutators (535.53s)
        test_1        |         compare_test.go:125: starting test: mutators
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: cockroach2
        test_1        |         compare_test.go:181: SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			'-46 years -6 mons -753 days -18:39:42.088991':::INTERVAL AS col_489580,
        test_1        |             			tab_287202.col1_15 AS col_489581,
        test_1        |             			tab_287202.col1_1 AS col_489582,
        test_1        |             			tab_287202.col1_11 AS col_489583,
        test_1        |             			264909402:::OID AS col_489584,
        test_1        |             			4259896486:::OID AS col_489585,
        test_1        |             			'1972-02-08':::DATE AS col_489586,
        test_1        |             			tab_287202.col1_5 AS col_489587,
        test_1        |             			72252622.13119571141:::DECIMAL AS col_489588
        test_1        |             		FROM
        test_1        |             			postgres.public.table1@table1_col1_7_col1_14_col1_9_col1_10_col1_3_key AS tab_287202
        test_1        |             		WHERE
        test_1        |             			true
        test_1        |             	)
        test_1        |             		AS tab_287203
        test_1        |             ORDER BY
        test_1        |             	col_489580 NULLS FIRST,
        test_1        |             	col_489581 NULLS FIRST,
        test_1        |             	col_489582 NULLS FIRST,
        test_1        |             	col_489583 NULLS FIRST,
        test_1        |             	col_489584 NULLS FIRST,
        test_1        |             	col_489585 NULLS FIRST,
        test_1        |             	col_489586 NULLS FIRST,
        test_1        |             	col_489587 NULLS FIRST,
        test_1        |             	col_489588 NULLS FIRST
        test_1        |         compare_test.go:182: cockroach1: ping: set tcp 172.18.0.5:43512: use of closed network connection
        test_1        | FAIL
        compare_cockroach1_1 exited with code 2
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3007634&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3007634&tab=artifacts#/) on master @ [73154b35a3f679861a102b7cfcbeaa9bfc1aa10f](https://github.com/cockroachdb/cockroach/commits/73154b35a3f679861a102b7cfcbeaa9bfc1aa10f):


```
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (84.35s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("3.0 USE_GEOS=1 USE_PROJ=1 USE_STATS=1"),
        test_1        |             + 	string("3.1 USE_GEOS=1 USE_PROJ=1 USE_STATS=1"),
        test_1        |               	string("nullD"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			postgis_version()::STRING AS col_115083,
        test_1        |             			('null':::JSONB::JSONB || 'D':::STRING::STRING)::STRING AS col_115084
        test_1        |             	)
        test_1        |             		AS tab_84079
        test_1        |             ORDER BY
        test_1        |             	col_115083 NULLS FIRST, col_115084 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			postgis_version()::TEXT AS col_115083,
        test_1        |             			('null'::JSONB::JSONB || 'D'::TEXT::TEXT)::TEXT AS col_115084
        test_1        |             	)
        test_1        |             		AS tab_84079
        test_1        |             ORDER BY
        test_1        |             	col_115083 NULLS FIRST, col_115084 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3008587&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3008587&tab=artifacts#/) on master @ [3eeb35f3ea834228cd724f2a0e01c27b75bb6dd5](https://github.com/cockroachdb/cockroach/commits/3eeb35f3ea834228cd724f2a0e01c27b75bb6dd5):


```
        postgres_1    | 						FROM
        postgres_1    | 							postgres.public.table1 AS tab_53744
        postgres_1    | 						WHERE
        postgres_1    | 							EXISTS(SELECT tab_53744.col1_6 AS col_101868)
        postgres_1    | 					)
        postgres_1    | 		)
        postgres_1    | 			AS tab_53745
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_101870 NULLS FIRST, col_101871 NULLS FIRST, col_101872 NULLS FIRST
        test_1        | --- FAIL: TestCompare (661.61s)
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     --- FAIL: TestCompare/postgres (58.46s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	s"1",
        test_1        |             + 	s"0",
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT get_bit('\x45':::BYTES::BYTES, 1:::INT8::INT4)::INT8 AS col_101892) AS tab_53760
        test_1        |             ORDER BY
        test_1        |             	col_101892 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT get_bit('\x45'::BYTEA::BYTEA, 1::INT8::INT8)::INT8 AS col_101892) AS tab_53760
        test_1        |             ORDER BY
        test_1        |             	col_101892 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3012754&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3012754&tab=artifacts#/) on master @ [d95eec9bf1b7b18792c9da528cb204c20804216f](https://github.com/cockroachdb/cockroach/commits/d95eec9bf1b7b18792c9da528cb204c20804216f):


```
        postgres_1    | 2021-05-25 18:15:52.508 UTC [51] LOG:  shutting down
        postgres_1    | 2021-05-25 18:15:52.617 UTC [49] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-05-25 18:15:52.725 UTC [1] LOG:  starting PostgreSQL 13.3 (Debian 13.3-1.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
        postgres_1    | 2021-05-25 18:15:52.726 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-05-25 18:15:52.726 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-05-25 18:15:52.730 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-05-25 18:15:52.736 UTC [96] LOG:  database system was shut down at 2021-05-25 18:15:52 UTC
        postgres_1    | 2021-05-25 18:15:52.742 UTC [1] LOG:  database system is ready to accept connections
        postgres_1    | 2021-05-25 18:25:55.468 UTC [126] ERROR:  invalid byte sequence for encoding "SQL_ASCII": 0x00
        postgres_1    | 2021-05-25 18:25:55.468 UTC [126] STATEMENT:  CREATE TABLE table1 (col1_0 VARCHAR NULL, col1_1 BIT(6) NULL, col1_2 OID NULL, col1_3 REGPROCEDURE, col1_4 NAME, col1_5 REGTYPE, col1_6 DATE NULL, col1_7 TIMESTAMP NOT NULL, col1_8 GEOGRAPHY NOT NULL, col1_9 TIMESTAMP NOT NULL, col1_10 JSONB NOT NULL, col1_11 VARCHAR NULL, col1_12 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col1_2 IS NULL THEN e'-\n!\f*``C'::TEXT ELSE e'DX:H\'_)'::TEXT END) STORED, col1_13 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col1_2 IS NULL THEN '7'::TEXT ELSE 'h8ll'::TEXT END) STORED, col1_14 TEXT GENERATED ALWAYS AS (CASE WHEN col1_3 IS NULL THEN e'\x00'::TEXT ELSE ''::TEXT END) STORED);
        postgres_1    | 	CREATE TABLE table2 (col2_0 REGNAMESPACE NULL, col2_1 VARCHAR, col2_2 REGCLASS NULL, col2_3 JSONB NULL, col2_4 NAME NOT NULL, col2_5 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col2_3 AS TEXT))) STORED, col2_6 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col2_0 IS NULL THEN ''::TEXT ELSE e'\U00002603'::TEXT END) STORED, col2_7 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col2_2 IS NULL THEN 'X'::TEXT ELSE e'\U00002603'::TEXT END) STORED, col2_8 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col2_0 IS NULL THEN e'``NMq#\x14'::TEXT ELSE e'\x11<U\x01nH'::TEXT END) STORED);
        postgres_1    | 	CREATE UNIQUE INDEX ON table2 (col2_5 DESC) INCLUDE (col2_1, col2_3, col2_8);
        postgres_1    | 	CREATE INDEX ON table2 (col2_5, col2_2 ASC, col2_6 ASC);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('dujng', 'tuj', 'wuxfor', 'h');
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('mmzgt', 'sfby', 'px', 'mie', 'g');
        postgres_1    | 	CREATE TYPE rand_typ_2 AS ENUM ('dtp', 'givw', 'sgbe', 'wgqo', 'jb');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (605.22s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (1.49s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:149: CREATE TABLE table1 (col1_0 VARCHAR NULL, col1_1 BIT(6) NULL, col1_2 OID NULL, col1_3 REGPROCEDURE, col1_4 NAME, col1_5 REGTYPE, col1_6 DATE NULL, col1_7 TIMESTAMP NOT NULL, col1_8 GEOGRAPHY NOT NULL, col1_9 TIMESTAMP NOT NULL, col1_10 JSONB NOT NULL, col1_11 VARCHAR NULL, col1_12 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col1_2 IS NULL THEN e'-\n!\f*``C'::TEXT ELSE e'DX:H\'_)'::TEXT END) STORED, col1_13 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col1_2 IS NULL THEN '7'::TEXT ELSE 'h8ll'::TEXT END) STORED, col1_14 TEXT GENERATED ALWAYS AS (CASE WHEN col1_3 IS NULL THEN e'\x00'::TEXT ELSE ''::TEXT END) STORED);
        test_1        |             CREATE TABLE table2 (col2_0 REGNAMESPACE NULL, col2_1 VARCHAR, col2_2 REGCLASS NULL, col2_3 JSONB NULL, col2_4 NAME NOT NULL, col2_5 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col2_3 AS TEXT))) STORED, col2_6 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col2_0 IS NULL THEN ''::TEXT ELSE e'\U00002603'::TEXT END) STORED, col2_7 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col2_2 IS NULL THEN 'X'::TEXT ELSE e'\U00002603'::TEXT END) STORED, col2_8 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col2_0 IS NULL THEN e'``NMq#\x14'::TEXT ELSE e'\x11<U\x01nH'::TEXT END) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_5 DESC) INCLUDE (col2_1, col2_3, col2_8);
        test_1        |             CREATE INDEX ON table2 (col2_5, col2_2 ASC, col2_6 ASC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('dujng', 'tuj', 'wuxfor', 'h');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('mmzgt', 'sfby', 'px', 'mie', 'g');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('dtp', 'givw', 'sgbe', 'wgqo', 'jb');
        test_1        |             
        test_1        |         compare_test.go:150: postgres: exec: ERROR: invalid byte sequence for encoding "SQL_ASCII": 0x00 (SQLSTATE 22021)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3017145&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3017145&tab=artifacts#/) on master @ [4e3437fb275f1eb7645ca04d942c82a596c3826e](https://github.com/cockroachdb/cockroach/commits/4e3437fb275f1eb7645ca04d942c82a596c3826e):


```
        test_1        |     --- FAIL: TestCompare/postgres (103.01s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |               	nil,
        test_1        |             - 	s"26",
        test_1        |             + 	s"23",
        test_1        |               }))
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			NULL AS col_139053,
        test_1        |             			length(('2000-01-01 06:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ || 'E'::TEXT::TEXT)::TEXT::TEXT)::INT8
        test_1        |             				AS col_139054
        test_1        |             	)
        test_1        |             		AS tab_74145
        test_1        |             ORDER BY
        test_1        |             	col_139053 NULLS FIRST, col_139054 NULLS FIRST;
        test_1        |             
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			NULL AS col_139053,
        test_1        |             			length(('2000-01-01 06:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ || 'E':::STRING::STRING)::STRING::STRING)::INT8
        test_1        |             				AS col_139054
        test_1        |             	)
        test_1        |             		AS tab_74145
        test_1        |             ORDER BY
        test_1        |             	col_139053 NULLS FIRST, col_139054 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3024465&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3024465&tab=artifacts#/) on master @ [53a0aee6bfe96fc25705c50f8350eeeafc1bf6b1](https://github.com/cockroachdb/cockroach/commits/53a0aee6bfe96fc25705c50f8350eeeafc1bf6b1):


```
        postgres_1    | 2021-05-28 05:15:54.334 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-05-28 05:15:54.334 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-05-28 05:15:54.338 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-05-28 05:15:54.343 UTC [95] LOG:  database system was shut down at 2021-05-28 05:15:54 UTC
        postgres_1    | 2021-05-28 05:15:54.347 UTC [1] LOG:  database system is ready to accept connections
        postgres_1    | 2021-05-28 05:25:57.181 UTC [125] ERROR:  conversion between UTF8 and SQL_ASCII is not supported at character 1012
        postgres_1    | 2021-05-28 05:25:57.181 UTC [125] STATEMENT:  CREATE TABLE table1 (col1_0 INTERVAL NOT NULL, col1_1 CHAR NOT NULL, col1_2 VARCHAR, col1_3 CHAR NOT NULL, PRIMARY KEY (col1_0, col1_3, col1_1), col1_4 TEXT NOT NULL GENERATED ALWAYS AS (lower(col1_3)) STORED, col1_5 TEXT NOT NULL GENERATED ALWAYS AS (lower(col1_3)) STORED);
        postgres_1    | 	CREATE UNIQUE INDEX ON table1 (col1_0 ASC, col1_3 DESC, col1_2 ASC, col1_1, col1_5 DESC) INCLUDE (col1_4);
        postgres_1    | 	CREATE INDEX ON table1 (col1_0 ASC, col1_1, col1_4, col1_2 ASC, col1_3 DESC);
        postgres_1    | 	CREATE TABLE table2 (col2_0 DATE[] NOT NULL, col2_1 BOOL NOT NULL, col2_2 BYTEA NOT NULL, col2_3 TIMESTAMP NULL, col2_4 TEXT NULL, col2_5 GEOGRAPHY NOT NULL, col2_6 BIT(27), col2_7 TEXT NOT NULL GENERATED ALWAYS AS (lower(CAST(col2_1 AS TEXT))) STORED, col2_8 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col2_3 IS NULL THEN e'|\x13\x0b O12Lk'::TEXT ELSE ''::TEXT END) STORED, col2_9 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col2_3 IS NULL THEN NULL ELSE NULL END) STORED, col2_10 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col2_3 IS NULL THEN NULL ELSE e'\U00002603'::TEXT END) STORED);
        postgres_1    | 	CREATE UNIQUE INDEX ON table2 (col2_10 ASC, col2_3, col2_2 DESC, col2_7, col2_8 ASC, col2_4) INCLUDE (col2_1, col2_6, col2_9);
        postgres_1    | 	CREATE TABLE table3 (col3_0 TIMESTAMP NULL, col3_1 UUID NOT NULL, col3_2 INT8 NOT NULL, col3_3 INT8 NOT NULL, col3_4 CHAR[], col3_5 FLOAT8 NOT NULL, col3_6 BIT(19) NOT NULL, col3_7 REGTYPE NULL, PRIMARY KEY (col3_6, col3_1, col3_2), col3_8 INT8 GENERATED ALWAYS AS (col3_3 + col3_2) STORED, col3_9 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col3_0 IS NULL THEN ''::TEXT ELSE e'p:C/|\x12k'::TEXT END) STORED);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_1, col3_9 ASC, col3_7 DESC, col3_0, col3_2, col3_8 ASC, col3_6 DESC);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_0 ASC, col3_9 DESC);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_6 DESC, col3_3);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('asdblo');
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('q');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (603.21s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (2.05s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:149: CREATE TABLE table1 (col1_0 INTERVAL NOT NULL, col1_1 CHAR NOT NULL, col1_2 VARCHAR, col1_3 CHAR NOT NULL, PRIMARY KEY (col1_0, col1_3, col1_1), col1_4 TEXT NOT NULL GENERATED ALWAYS AS (lower(col1_3)) STORED, col1_5 TEXT NOT NULL GENERATED ALWAYS AS (lower(col1_3)) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_0 ASC, col1_3 DESC, col1_2 ASC, col1_1, col1_5 DESC) INCLUDE (col1_4);
        test_1        |             CREATE INDEX ON table1 (col1_0 ASC, col1_1, col1_4, col1_2 ASC, col1_3 DESC);
        test_1        |             CREATE TABLE table2 (col2_0 DATE[] NOT NULL, col2_1 BOOL NOT NULL, col2_2 BYTEA NOT NULL, col2_3 TIMESTAMP NULL, col2_4 TEXT NULL, col2_5 GEOGRAPHY NOT NULL, col2_6 BIT(27), col2_7 TEXT NOT NULL GENERATED ALWAYS AS (lower(CAST(col2_1 AS TEXT))) STORED, col2_8 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col2_3 IS NULL THEN e'|\x13\x0b O12Lk'::TEXT ELSE ''::TEXT END) STORED, col2_9 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col2_3 IS NULL THEN NULL ELSE NULL END) STORED, col2_10 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col2_3 IS NULL THEN NULL ELSE e'\U00002603'::TEXT END) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col2_10 ASC, col2_3, col2_2 DESC, col2_7, col2_8 ASC, col2_4) INCLUDE (col2_1, col2_6, col2_9);
        test_1        |             CREATE TABLE table3 (col3_0 TIMESTAMP NULL, col3_1 UUID NOT NULL, col3_2 INT8 NOT NULL, col3_3 INT8 NOT NULL, col3_4 CHAR[], col3_5 FLOAT8 NOT NULL, col3_6 BIT(19) NOT NULL, col3_7 REGTYPE NULL, PRIMARY KEY (col3_6, col3_1, col3_2), col3_8 INT8 GENERATED ALWAYS AS (col3_3 + col3_2) STORED, col3_9 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col3_0 IS NULL THEN ''::TEXT ELSE e'p:C/|\x12k'::TEXT END) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_1, col3_9 ASC, col3_7 DESC, col3_0, col3_2, col3_8 ASC, col3_6 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 ASC, col3_9 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_6 DESC, col3_3);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('asdblo');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('q');
        test_1        |             
        test_1        |         compare_test.go:150: postgres: exec: ERROR: conversion between UTF8 and SQL_ASCII is not supported (SQLSTATE 0A000)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3028237&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3028237&tab=artifacts#/) on master @ [55877f386a04ed5b6c7722259ff0b3ac3ad545f6](https://github.com/cockroachdb/cockroach/commits/55877f386a04ed5b6c7722259ff0b3ac3ad545f6):


```
        postgres_1    | 		col_63307 NULLS FIRST,
        postgres_1    | 		col_63308 NULLS FIRST,
        postgres_1    | 		col_63309 NULLS FIRST,
        postgres_1    | 		col_63310 NULLS FIRST,
        postgres_1    | 		col_63311 NULLS FIRST,
        postgres_1    | 		col_63312 NULLS FIRST,
        postgres_1    | 		col_63313 NULLS FIRST
        test_1        | --- FAIL: TestCompare (639.89s)
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     --- FAIL: TestCompare/postgres (35.49s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("4.9.3"),
        test_1        |             + 	string("Rel. 5.2.0, September 15th, 2018"),
        test_1        |               	nil,
        test_1        |               	bool(true),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT postgis_proj_version()::STRING AS col_63805, NULL AS col_63806, true AS col_63807) AS tab_32789
        test_1        |             ORDER BY
        test_1        |             	col_63805 NULLS FIRST, col_63806 NULLS FIRST, col_63807 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT postgis_proj_version()::TEXT AS col_63805, NULL AS col_63806, true AS col_63807) AS tab_32789
        test_1        |             ORDER BY
        test_1        |             	col_63805 NULLS FIRST, col_63806 NULLS FIRST, col_63807 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3029988&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3029988&tab=artifacts#/) on master @ [7628160a7dfdc73d693f8018772fef8f79c8c0fe](https://github.com/cockroachdb/cockroach/commits/7628160a7dfdc73d693f8018772fef8f79c8c0fe):


```
        test_1        |             
        test_1        |     --- FAIL: TestCompare/mutators (45.16s)
        test_1        |         compare_test.go:125: starting test: mutators
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: cockroach2
        test_1        |         compare_test.go:181: SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			tab_834.col2_10 AS col_1414, tab_829.col3_7 AS col_1415, '0001-01-01 00:00:00':::TIMESTAMP AS col_1416
        test_1        |             		FROM
        test_1        |             			postgres.public.table1@[0] AS tab_828,
        test_1        |             			postgres.public.table3@table3_col3_12_col3_8_col3_13_col3_6_col3_3_col3_15_col3_7_idx AS tab_829
        test_1        |             			JOIN postgres.public.table3@[0] AS tab_830 ON (tab_829.col3_7) = (tab_830.col3_7)
        test_1        |             			JOIN postgres.public.table3 AS tab_831
        test_1        |             				JOIN postgres.public.table3 AS tab_832 ON
        test_1        |             						(tab_831.col3_11) = (tab_832.col3_11) AND (tab_831.col3_12) = (tab_832.col3_12)
        test_1        |             				JOIN postgres.public.table2@[0] AS tab_833
        test_1        |             					JOIN postgres.public.table2@[0] AS tab_834
        test_1        |             						JOIN postgres.public.table2@primary AS tab_835 ON (tab_834.col2_8) = (tab_835.col2_8) ON
        test_1        |             							(tab_833.col2_12) = (tab_835.col2_14) ON
        test_1        |             						(tab_832.col3_8) = (tab_833.tableoid)
        test_1        |             						AND (tab_832.col3_8) = (tab_835.tableoid)
        test_1        |             						AND (tab_832.col3_8) = (tab_835.tableoid)
        test_1        |             				JOIN postgres.public.table1@table1_col1_2_col1_6_idx AS tab_836 ON
        test_1        |             						(tab_835.tableoid) = (tab_836.tableoid) AND (tab_835.col2_14) = (tab_836.col1_4) ON
        test_1        |             					(tab_829.col3_15) = (tab_834.col2_10)
        test_1        |             					AND (tab_829.col3_7) = (tab_833.tableoid)
        test_1        |             					AND (tab_829.col3_6) = (tab_831.col3_8)
        test_1        |             					AND (tab_830.tableoid) = (tab_832.col3_8)
        test_1        |             					AND (tab_829.crdb_internal_mvcc_timestamp) = (tab_835.crdb_internal_mvcc_timestamp)
        test_1        |             		WHERE
        test_1        |             			tab_834.col2_7
        test_1        |             	)
        test_1        |             		AS tab_837
        test_1        |             ORDER BY
        test_1        |             	col_1414 NULLS FIRST, col_1415 NULLS FIRST, col_1416 NULLS FIRST
        test_1        |         compare_test.go:182: cockroach1: ping: context deadline exceeded
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3031657&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3031657&tab=artifacts#/) on master @ [995f0dd1512d0266e78001980aab0023eab0e70f](https://github.com/cockroachdb/cockroach/commits/995f0dd1512d0266e78001980aab0023eab0e70f):


```
        postgres_1    | 		col_501539 NULLS FIRST,
        postgres_1    | 		col_501540 NULLS FIRST,
        postgres_1    | 		col_501541 NULLS FIRST,
        postgres_1    | 		col_501542 NULLS FIRST,
        postgres_1    | 		col_501543 NULLS FIRST,
        postgres_1    | 		col_501544 NULLS FIRST,
        postgres_1    | 		col_501545 NULLS FIRST
        test_1        | --- FAIL: TestCompare (925.52s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (322.19s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("2000-01-01 07:00:00H"),
        test_1        |             + 	string("2000-01-01 07:00:00+00H"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('2000-01-01 07:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ || 'H':::STRING::STRING)::STRING AS col_501719)
        test_1        |             		AS tab_281177
        test_1        |             ORDER BY
        test_1        |             	col_501719 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('2000-01-01 07:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ || 'H'::TEXT::TEXT)::TEXT AS col_501719)
        test_1        |             		AS tab_281177
        test_1        |             ORDER BY
        test_1        |             	col_501719 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3035009&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3035009&tab=artifacts#/) on master @ [7762f00fdb5dd7a65545b684fbd4f0babe383a76](https://github.com/cockroachdb/cockroach/commits/7762f00fdb5dd7a65545b684fbd4f0babe383a76):


```
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_9871 NULLS FIRST,
        postgres_1    | 		col_9872 NULLS FIRST,
        postgres_1    | 		col_9873 NULLS FIRST,
        postgres_1    | 		col_9874 NULLS FIRST,
        postgres_1    | 		col_9875 NULLS FIRST,
        postgres_1    | 		col_9876 NULLS FIRST,
        postgres_1    | 		col_9877 NULLS FIRST
        test_1        | --- FAIL: TestCompare (612.95s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (9.42s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("0.4.3 (Internal)"),
        test_1        |             + 	string("0.5.0 (Internal)"),
        test_1        |               	nil,
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT postgis_wagyu_version()::STRING AS col_10028, NULL AS col_10029) AS tab_5708
        test_1        |             ORDER BY
        test_1        |             	col_10028 NULLS FIRST, col_10029 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT postgis_wagyu_version()::TEXT AS col_10028, NULL AS col_10029) AS tab_5708
        test_1        |             ORDER BY
        test_1        |             	col_10028 NULLS FIRST, col_10029 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3039180&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3039180&tab=artifacts#/) on master @ [91632b5a4db844ea272a1f29196d74e3835922bd](https://github.com/cockroachdb/cockroach/commits/91632b5a4db844ea272a1f29196d74e3835922bd):


```
        test_1        | --- FAIL: TestCompare (2.12s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (0.00s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        | panic: geos: no locations to init GEOS [recovered]
        test_1        | 	panic: geos: no locations to init GEOS
        test_1        | 
        test_1        | goroutine 26 [running]:
        test_1        | testing.tRunner.func1.1(0x201bd60, 0xc0004697c0)
        test_1        | 	/usr/local/go/src/testing/testing.go:1072 +0x30d
        test_1        | testing.tRunner.func1(0xc000602900)
        test_1        | 	/usr/local/go/src/testing/testing.go:1075 +0x41a
        test_1        | panic(0x201bd60, 0xc0004697c0)
        test_1        | 	/usr/local/go/src/runtime/panic.go:969 +0x1b9
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.encodeInvertedIndexHistogramUpperBounds(0x39e4a60, 0x2b1d060, 0xc000330c00, 0x2b1d060, 0xc000330c00, 0x0)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:368 +0x5d3
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.randHistogram(0xc0004b2f90, 0x39e4a60, 0x39e4a60, 0x0, 0xc000ae9780, 0x4)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:301 +0x10c
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator.func1(0xc0004517a0)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:220 +0x9d
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.statisticsMutator(0xc0004b2f90, 0xc00047b2c0, 0x4, 0x4, 0x0, 0x0, 0x0, 0x0)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:251 +0xbe6
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.MultiStatementMutation.Mutate(0x276fd88, 0xc0004b2f90, 0xc00047b2c0, 0x4, 0x4, 0x0, 0xc0000d2aa0, 0xe0, 0x0)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/mutator.go:97 +0x4e
        test_1        | github.com/cockroachdb/cockroach/pkg/sql/randgen.RandCreateTables(0xc0004b2f90, 0x21b2c74, 0x5, 0x4, 0xc0004ffa40, 0x3, 0x3, 0xe0, 0x4000, 0xc000309901)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/sql/randgen/schema.go:94 +0x224
        test_1        | github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTablesN(0xc0004b2f90, 0x4, 0x3, 0x3d2f800)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:91 +0x1a5
        test_1        | github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.randTables(0xc0004b2f90, 0x30, 0x2077660)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:77 +0x4f
        test_1        | github.com/cockroachdb/cockroach/pkg/internal/sqlsmith.wrapCommonSetup.func1(0xc0004b2f90, 0xe6f9569b94cd8b22, 0x11)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/internal/sqlsmith/setup.go:43 +0x44
        test_1        | github.com/cockroachdb/cockroach/pkg/compose/compare/compare.TestCompare.func2(0xc000602900)
        test_1        | 	/go/src/github.com/cockroachdb/cockroach/pkg/compose/compare/compare/compare_test.go:127 +0x113
        test_1        | testing.tRunner(0xc000602900, 0xc0004b2f00)
        test_1        | 	/usr/local/go/src/testing/testing.go:1123 +0xef
        test_1        | created by testing.(*T).Run
        test_1        | 	/usr/local/go/src/testing/testing.go:1168 +0x2b3
        compare_test_1 exited with code 2
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 2
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3043011&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3043011&tab=artifacts#/) on master @ [7869b781b66823c44f1b30047d0fcbcc2dbe8e0f](https://github.com/cockroachdb/cockroach/commits/7869b781b66823c44f1b30047d0fcbcc2dbe8e0f):


```
        postgres_1    | 		col_51788 NULLS FIRST,
        postgres_1    | 		col_51789 NULLS FIRST,
        postgres_1    | 		col_51790 NULLS FIRST,
        postgres_1    | 		col_51791 NULLS FIRST,
        postgres_1    | 		col_51792 NULLS FIRST,
        postgres_1    | 		col_51793 NULLS FIRST,
        postgres_1    | 		col_51794 NULLS FIRST,
        postgres_1    | 		col_51795 NULLS FIRST,
        postgres_1    | 		col_51796 NULLS FIRST,
        postgres_1    | 		col_51797 NULLS FIRST,
        postgres_1    | 		col_51798 NULLS FIRST,
        postgres_1    | 		col_51799 NULLS FIRST,
        postgres_1    | 		col_51800 NULLS FIRST,
        postgres_1    | 		col_51801 NULLS FIRST,
        postgres_1    | 		col_51802 NULLS FIRST,
        postgres_1    | 		col_51803 NULLS FIRST,
        postgres_1    | 		col_51804 NULLS FIRST,
        postgres_1    | 		col_51805 NULLS FIRST,
        postgres_1    | 		col_51806 NULLS FIRST
        test_1        | --- FAIL: TestCompare (636.24s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (32.15s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("0.5.0 (Internal)"),
        test_1        |             + 	string("0.4.3 (Internal)"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT postgis_wagyu_version()::STRING AS col_51860) AS tab_28707 ORDER BY col_51860 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT postgis_wagyu_version()::TEXT AS col_51860) AS tab_28707 ORDER BY col_51860 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3047245&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3047245&tab=artifacts#/) on master @ [63407904f8ecadbe40cce29c3d67788f859b58b3](https://github.com/cockroachdb/cockroach/commits/63407904f8ecadbe40cce29c3d67788f859b58b3):


```
        postgres_1    | 				CASE WHEN true THEN tab_885.col3_11 ELSE tab_885.col2_5 END AS col_1642,
        postgres_1    | 				tab_885.col1_1 AS col_1643,
        postgres_1    | 				tab_885.col2_11 AS col_1644
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table3 AS tab_885
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_885.col1_1 ASC
        postgres_1    | 		)
        postgres_1    | 			AS tab_886
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_1636 NULLS FIRST,
        postgres_1    | 		col_1637 NULLS FIRST,
        postgres_1    | 		col_1638 NULLS FIRST,
        postgres_1    | 		col_1639 NULLS FIRST,
        postgres_1    | 		col_1640 NULLS FIRST,
        postgres_1    | 		col_1641 NULLS FIRST,
        postgres_1    | 		col_1642 NULLS FIRST,
        postgres_1    | 		col_1643 NULLS FIRST,
        postgres_1    | 		col_1644 NULLS FIRST
        test_1        | --- FAIL: TestCompare (605.94s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (4.41s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("0.4.3 (Internal)"),
        test_1        |             + 	string("0.5.0 (Internal)"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT postgis_wagyu_version()::STRING AS col_1809) AS tab_988 ORDER BY col_1809 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT postgis_wagyu_version()::TEXT AS col_1809) AS tab_988 ORDER BY col_1809 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3051467&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3051467&tab=artifacts#/) on master @ [9a1a5eb4211dfcbe27b904fa895d570138a023ab](https://github.com/cockroachdb/cockroach/commits/9a1a5eb4211dfcbe27b904fa895d570138a023ab):


```
        postgres_1    | 			GROUP BY
        postgres_1    | 				tab_55061.col1_6
        postgres_1    | 			HAVING
        postgres_1    | 				bool_and(false::BOOL)::BOOL
        postgres_1    | 		)
        postgres_1    | 			AS tab_55065
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_106339 NULLS FIRST
        test_1        | --- FAIL: TestCompare (672.62s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (71.37s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	int16(1),
        test_1        |             + 	s"1",
        test_1        |               	s"1",
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT COALESCE(st_zmflag(NULL::GEOMETRY)::INT2, 1:::INT8) AS col_106361, 1:::INT8 AS col_106362) AS tab_55071
        test_1        |             ORDER BY
        test_1        |             	col_106361 NULLS FIRST, col_106362 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT COALESCE(st_zmflag(NULL::GEOMETRY)::INT8, 1::INT8) AS col_106361, 1::INT8 AS col_106362) AS tab_55071
        test_1        |             ORDER BY
        test_1        |             	col_106361 NULLS FIRST, col_106362 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3053609&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3053609&tab=artifacts#/) on master @ [fb5b16e4332022cdca96b946b8d88814e7bea038](https://github.com/cockroachdb/cockroach/commits/fb5b16e4332022cdca96b946b8d88814e7bea038):


```
        postgres_1    | 			SELECT
        postgres_1    | 				NULL AS col_44412,
        postgres_1    | 				tab_24761.col4_2 AS col_44413,
        postgres_1    | 				NULL AS col_44414,
        postgres_1    | 				'00:00:01'::INTERVAL AS col_44415,
        postgres_1    | 				tab_24761.col4_13 AS col_44416
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table4 AS tab_24761
        postgres_1    | 				RIGHT JOIN postgres.public.table3 AS tab_24762 ON (NOT (NOT (true OR NULL)))
        postgres_1    | 			WHERE
        postgres_1    | 				NULL::BOX2D NOT IN (NULL)
        postgres_1    | 		)
        postgres_1    | 			AS tab_24763
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_44412 NULLS FIRST,
        postgres_1    | 		col_44413 NULLS FIRST,
        postgres_1    | 		CAST(col_44414 AS TEXT) NULLS FIRST,
        postgres_1    | 		col_44415 NULLS FIRST,
        postgres_1    | 		col_44416 NULLS FIRST
        test_1        | --- FAIL: TestCompare (634.23s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (30.25s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("3.0.1 ec2a9aa"),
        test_1        |             + 	string("3.1.2 cbe925d"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT postgis_scripts_released()::STRING AS col_44455) AS tab_24787 ORDER BY col_44455 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT postgis_scripts_released()::TEXT AS col_44455) AS tab_24787 ORDER BY col_44455 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3058193&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3058193&tab=artifacts#/) on master @ [cef0bd947590218ea6ac94f4f85dffc25e16fcd0](https://github.com/cockroachdb/cockroach/commits/cef0bd947590218ea6ac94f4f85dffc25e16fcd0):


```
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | CREATE EXTENSION
        postgres_1    | 
        postgres_1    | 2021-06-08 05:15:21.763 UTC [48] LOG:  received fast shutdown request
        postgres_1    | waiting for server to shut down....2021-06-08 05:15:21.764 UTC [48] LOG:  aborting any active transactions
        postgres_1    | 2021-06-08 05:15:21.766 UTC [48] LOG:  background worker "logical replication launcher" (PID 55) exited with exit code 1
        postgres_1    | 2021-06-08 05:15:21.768 UTC [50] LOG:  shutting down
        postgres_1    | 2021-06-08 05:15:21.848 UTC [48] LOG:  database system is shut down
        postgres_1    |  done
        postgres_1    | server stopped
        postgres_1    | 
        postgres_1    | PostgreSQL init process complete; ready for start up.
        postgres_1    | 
        postgres_1    | 2021-06-08 05:15:21.884 UTC [1] LOG:  starting PostgreSQL 13.3 (Debian 13.3-1.pgdg100+1) on x86_64-pc-linux-gnu, compiled by gcc (Debian 8.3.0-6) 8.3.0, 64-bit
        postgres_1    | 2021-06-08 05:15:21.884 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
        postgres_1    | 2021-06-08 05:15:21.884 UTC [1] LOG:  listening on IPv6 address "::", port 5432
        postgres_1    | 2021-06-08 05:15:21.887 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
        postgres_1    | 2021-06-08 05:15:21.891 UTC [95] LOG:  database system was shut down at 2021-06-08 05:15:21 UTC
        postgres_1    | 2021-06-08 05:15:21.896 UTC [1] LOG:  database system is ready to accept connections
        postgres_1    | 2021-06-08 05:25:23.567 UTC [125] ERROR:  syntax error at or near "DESC" at character 492
        postgres_1    | 2021-06-08 05:25:23.567 UTC [125] STATEMENT:  CREATE TABLE table1 (col1_0 DECIMAL[] NULL, col1_1 REGTYPE, col1_2 TIMESTAMPTZ NULL, col1_3 CHAR NULL, col1_4 FLOAT8 NOT NULL, col1_5 VARCHAR[], col1_6 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col1_2 IS NULL THEN e'\x0e\x18yQ\x14m!'::TEXT ELSE '4[pk~]'::TEXT END) STORED, col1_7 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col1_0 IS NULL THEN e'\n=^\x07y\x12'::TEXT ELSE '?'::TEXT END) STORED, col1_8 TEXT NULL GENERATED ALWAYS AS (lower(col1_3)) STORED, INVERTED INDEX (col1_7, col1_6, col1_0 DESC) WHERE ((((table1.col1_6 >= e'\U00002603'::TEXT) OR (table1.col1_7 < 'X'::TEXT)) AND (table1.col1_3 = e'\U00002603'::TEXT)) AND (table1.col1_8 <= e'\U00002603'::TEXT)) OR (table1.col1_2 >= '-4713-11-24 00:00:00+00:00'::TIMESTAMPTZ));
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('r');
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('pfmj', 'd', 'eezz', 'lmx', 'pm');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (601.89s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (1.02s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:149: CREATE TABLE table1 (col1_0 DECIMAL[] NULL, col1_1 REGTYPE, col1_2 TIMESTAMPTZ NULL, col1_3 CHAR NULL, col1_4 FLOAT8 NOT NULL, col1_5 VARCHAR[], col1_6 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col1_2 IS NULL THEN e'\x0e\x18yQ\x14m!'::TEXT ELSE '4[pk~]'::TEXT END) STORED, col1_7 TEXT NULL GENERATED ALWAYS AS (CASE WHEN col1_0 IS NULL THEN e'\n=^\x07y\x12'::TEXT ELSE '?'::TEXT END) STORED, col1_8 TEXT NULL GENERATED ALWAYS AS (lower(col1_3)) STORED, INVERTED INDEX (col1_7, col1_6, col1_0 DESC) WHERE ((((table1.col1_6 >= e'\U00002603'::TEXT) OR (table1.col1_7 < 'X'::TEXT)) AND (table1.col1_3 = e'\U00002603'::TEXT)) AND (table1.col1_8 <= e'\U00002603'::TEXT)) OR (table1.col1_2 >= '-4713-11-24 00:00:00+00:00'::TIMESTAMPTZ));
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('r');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('pfmj', 'd', 'eezz', 'lmx', 'pm');
        test_1        |             
        test_1        |         compare_test.go:150: postgres: exec: ERROR: syntax error at or near "DESC" (SQLSTATE 42601)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3063275&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3063275&tab=artifacts#/) on master @ [894a486398f12dc92b2f4af8c6acac4406b0d031](https://github.com/cockroachdb/cockroach/commits/894a486398f12dc92b2f4af8c6acac4406b0d031):


```
        postgres_1    | 	CREATE INDEX ON table2 (col2_3 DESC, col2_6 DESC, col2_8 ASC, col2_2 ASC, col2_0, col2_1 DESC, col2_7 DESC) INCLUDE (col2_4);
        postgres_1    | 	CREATE TABLE table3 (col3_0 REGNAMESPACE NOT NULL, col3_1 CHAR, col3_2 DATE NOT NULL, PRIMARY KEY (col3_0, col3_2), col3_3 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col3_0 IS NULL THEN e'\\\x1eWI\x05~'::TEXT ELSE 'Pn='::TEXT END) STORED);
        postgres_1    | 	CREATE INDEX ON table3 (col3_3 ASC, col3_0);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_0 DESC, col3_1, col3_3 ASC, col3_2 DESC);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_3);
        postgres_1    | 	CREATE INDEX ON table3 (col3_0 DESC, col3_1 DESC, col3_3 DESC, col3_2 DESC);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_3 DESC) INCLUDE (col3_1);
        postgres_1    | 	CREATE INDEX ON table3 (col3_1 ASC);
        postgres_1    | 	CREATE INDEX ON table3 (col3_3 ASC);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('evk');
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('vwkpfd', 'izxnaf', 'nim', 'mcnw', 'dfmlp');
        postgres_1    | 	CREATE TYPE rand_typ_2 AS ENUM ('xrge', 'tccvuo', 'eqil', 'q', 'koxdfy', 'hlvobu');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (607.57s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (3.55s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:149: CREATE TABLE table1 (col1_0 UUID NULL, col1_1 TEXT, col1_2 REGCLASS NOT NULL, col1_3 REGPROCEDURE, col1_4 TIMESTAMPTZ NOT NULL, col1_5 TIMETZ, col1_6 UUID, col1_7 JSONB NULL, col1_8 DATE NULL, col1_9 BOOL NOT NULL, col1_10 BOX2D, col1_11 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col1_4 IS NULL THEN e'\x00'::TEXT ELSE e'w\x17k\x18T]'::TEXT END) STORED, col1_12 TEXT GENERATED ALWAYS AS (lower(col1_1)) STORED, col1_13 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col1_4 IS NULL THEN ''::TEXT ELSE e'U\x10>=\x04'::TEXT END) STORED, col1_14 TEXT GENERATED ALWAYS AS (lower(CAST(col1_5 AS TEXT))) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table1 (col1_14 DESC) INCLUDE (col1_0, col1_1, col1_3, col1_6, col1_7, col1_11);
        test_1        |             CREATE INDEX ON table1 (col1_6 ASC, col1_14 DESC, col1_1, col1_9 DESC, col1_0 ASC, col1_2, col1_8 ASC, col1_13) INCLUDE (col1_5, col1_7, col1_10, col1_11);
        test_1        |             CREATE TABLE table2 (col2_0 TIMESTAMPTZ NULL, col2_1 DECIMAL, col2_2 OID, col2_3 INT8 NOT NULL, col2_4 GEOGRAPHY NULL, col2_5 BOX2D NULL, col2_6 REGTYPE NOT NULL, col2_7 BIT(10) NOT NULL, col2_8 FLOAT8 NULL);
        test_1        |             CREATE INDEX ON table2 (col2_3 DESC, col2_6 DESC, col2_8 ASC, col2_2 ASC, col2_0, col2_1 DESC, col2_7 DESC) INCLUDE (col2_4);
        test_1        |             CREATE TABLE table3 (col3_0 REGNAMESPACE NOT NULL, col3_1 CHAR, col3_2 DATE NOT NULL, PRIMARY KEY (col3_0, col3_2), col3_3 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col3_0 IS NULL THEN e'\\\x1eWI\x05~'::TEXT ELSE 'Pn='::TEXT END) STORED);
        test_1        |             CREATE INDEX ON table3 (col3_3 ASC, col3_0);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_0 DESC, col3_1, col3_3 ASC, col3_2 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_3);
        test_1        |             CREATE INDEX ON table3 (col3_0 DESC, col3_1 DESC, col3_3 DESC, col3_2 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_3 DESC) INCLUDE (col3_1);
        test_1        |             CREATE INDEX ON table3 (col3_1 ASC);
        test_1        |             CREATE INDEX ON table3 (col3_3 ASC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('evk');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('vwkpfd', 'izxnaf', 'nim', 'mcnw', 'dfmlp');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('xrge', 'tccvuo', 'eqil', 'q', 'koxdfy', 'hlvobu');
        test_1        |             
        test_1        |         compare_test.go:150: postgres: exec: ERROR: invalid byte sequence for encoding "SQL_ASCII": 0x00 (SQLSTATE 22021)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3071073&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3071073&tab=artifacts#/) on master @ [0a61eb767e9d02a01fbbd3c8fd0ebcb7388c3acd](https://github.com/cockroachdb/cockroach/commits/0a61eb767e9d02a01fbbd3c8fd0ebcb7388c3acd):


```
        postgres_1    | 				tab_20489.col4_4
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_20489.col4_9 DESC, tab_20489.col4_15 ASC, tab_20489.col4_19 DESC
        postgres_1    | 		)
        postgres_1    | 			AS tab_20490
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_34643 NULLS FIRST, col_34644 NULLS FIRST, col_34645 NULLS FIRST
        test_1        | --- FAIL: TestCompare (630.04s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (26.64s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("2000-01-01 09:00:00+00C"),
        test_1        |             + 	string("2000-01-01 09:00:00C"),
        test_1        |               }))
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('2000-01-01 09:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ || 'C'::TEXT::TEXT)::TEXT AS col_34730)
        test_1        |             		AS tab_20534
        test_1        |             ORDER BY
        test_1        |             	col_34730 NULLS FIRST;
        test_1        |             
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('2000-01-01 09:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ || 'C':::STRING::STRING)::STRING AS col_34730)
        test_1        |             		AS tab_20534
        test_1        |             ORDER BY
        test_1        |             	col_34730 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3074978&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3074978&tab=artifacts#/) on master @ [bc03d78fb789a4cf52027228914ab80ef4596035](https://github.com/cockroachdb/cockroach/commits/bc03d78fb789a4cf52027228914ab80ef4596035):


```
        postgres_1    | 				postgres.public.table3 AS tab_115397
        postgres_1    | 			GROUP BY
        postgres_1    | 				tab_115397.col3_6, tab_115397.col3_11, tab_115397.col3_7
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_115397.col3_7, tab_115397.col3_7, tab_115397.col3_7 DESC, tab_115397.col3_11 DESC
        postgres_1    | 		)
        postgres_1    | 			AS tab_115398
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_221953 NULLS FIRST
        test_1        | --- FAIL: TestCompare (737.62s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (133.71s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	s"1",
        test_1        |             + 	int16(1),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT COALESCE(st_zmflag(NULL::GEOMETRY)::INT2, 1:::INT8) AS col_222179) AS tab_115511
        test_1        |             ORDER BY
        test_1        |             	col_222179 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT COALESCE(st_zmflag(NULL::GEOMETRY)::INT2, 1::INT8) AS col_222179) AS tab_115511
        test_1        |             ORDER BY
        test_1        |             	col_222179 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3076663&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3076663&tab=artifacts#/) on master @ [9335ade9ce91ebdab1996d7c818b16df32557ac8](https://github.com/cockroachdb/cockroach/commits/9335ade9ce91ebdab1996d7c818b16df32557ac8):


```
        postgres_1    | 				tab_194975.col3_0, tab_194975.col3_4, tab_194975.col3_7, tab_194975.col3_3, tab_194975.col3_8
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_194975.col3_8 DESC
        postgres_1    | 		)
        postgres_1    | 			AS tab_194976
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_314626 NULLS FIRST, col_314627 NULLS FIRST
        test_1        | --- FAIL: TestCompare (823.40s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (220.35s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("F2000-01-01 05:00:00"),
        test_1        |             + 	string("F2000-01-01 05:00:00+00"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('F':::STRING::STRING || '2000-01-01 05:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ)::STRING AS col_314748)
        test_1        |             		AS tab_195039
        test_1        |             ORDER BY
        test_1        |             	col_314748 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('F'::TEXT::TEXT || '2000-01-01 05:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ)::TEXT AS col_314748)
        test_1        |             		AS tab_195039
        test_1        |             ORDER BY
        test_1        |             	col_314748 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3078295&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3078295&tab=artifacts#/) on master @ [2bdeffc210b63c8268944048c758a73a2cf07bed](https://github.com/cockroachdb/cockroach/commits/2bdeffc210b63c8268944048c758a73a2cf07bed):


```
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_8, col3_4, col3_11 DESC, col3_12 DESC, col3_9, col3_3 ASC, col3_10 DESC, col3_14, col3_6 ASC, col3_13, col3_2 DESC, col3_0 DESC, col3_15 ASC, col3_7, col3_1 DESC);
        postgres_1    | 	CREATE TABLE table4 (col4_0 BYTEA, col4_1 REGPROCEDURE NOT NULL, col4_2 UUID NULL, col4_3 UUID NOT NULL, col4_4 TIMESTAMPTZ, col4_5 TIMESTAMPTZ NOT NULL, col4_6 INT8 NOT NULL, col4_7 BIT(34) NULL, col4_8 BOOL NULL, col4_9 TIMETZ NOT NULL, col4_10 GEOGRAPHY, PRIMARY KEY (col4_1, col4_3, col4_5), col4_11 TEXT GENERATED ALWAYS AS (lower(CAST(col4_10 AS TEXT))) STORED, col4_12 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col4_8 AS TEXT))) STORED, col4_13 TEXT GENERATED ALWAYS AS (CASE WHEN col4_4 IS NULL THEN NULL ELSE NULL END) STORED, col4_14 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col4_5 IS NULL THEN '?'::TEXT ELSE e'\x06G22S\x1f\x073\x04'::TEXT END) STORED, col4_15 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col4_2 AS TEXT))) STORED, col4_16 TEXT GENERATED ALWAYS AS (lower(CAST(col4_10 AS TEXT))) STORED, col4_17 TEXT GENERATED ALWAYS AS (lower(CAST(col4_10 AS TEXT))) STORED, col4_18 INT8 NOT NULL GENERATED ALWAYS AS (col4_6 + 127::INT8) STORED);
        postgres_1    | 	CREATE INDEX ON table4 (col4_18, col4_9) INCLUDE (col4_6, col4_8, col4_10, col4_11, col4_12, col4_13, col4_15, col4_16);
        postgres_1    | 	CREATE INDEX ON table4 (col4_15 DESC, col4_7 ASC, col4_12 ASC, col4_6) INCLUDE (col4_0, col4_2, col4_8, col4_9, col4_11, col4_18);
        postgres_1    | 	CREATE UNIQUE INDEX ON table4 (col4_13 ASC, col4_5, col4_12 DESC, col4_3 DESC, col4_9 DESC, col4_6 DESC, col4_0, col4_2 DESC);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('uw');
        postgres_1    | 	CREATE TYPE rand_typ_1 AS ENUM ('mid', 'xygoy', 'qjvwoh', 'n', 'b', 'ubyik');
        postgres_1    | 	CREATE TYPE rand_typ_2 AS ENUM ('hybjr', 'quq', 'xktn', 'svtd', 'tvtus', 'td');
        postgres_1    | 	CREATE TYPE rand_typ_3 AS ENUM ('mn', 'fuhczv');
        postgres_1    | 	CREATE TYPE rand_typ_4 AS ENUM ('lsxvc', 'oewfz', 'y', 'vm', 'mhus', 'nglh');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (608.33s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (3.50s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:149: CREATE TABLE table1 (col1_0 INT8 NOT NULL, col1_1 CHAR, col1_2 TIMESTAMPTZ NOT NULL, col1_3 DECIMAL NOT NULL, col1_4 GEOGRAPHY NULL, col1_5 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col1_4 AS TEXT))) STORED, col1_6 INT8 NOT NULL GENERATED ALWAYS AS (col1_0 + 9223372036854775807::INT8) STORED);
        test_1        |             CREATE INDEX ON table1 (col1_3 ASC, col1_6, col1_0 ASC, col1_2 ASC) INCLUDE (col1_1, col1_5);
        test_1        |             CREATE INDEX ON table1 (col1_2 DESC, col1_3, col1_6 ASC, col1_5 DESC);
        test_1        |             CREATE TABLE table2 (col2_0 CHAR, col2_1 JSONB, col2_2 UUID NOT NULL, col2_3 FLOAT4 NOT NULL, col2_4 REGCLASS NOT NULL, col2_5 GEOGRAPHY NOT NULL, col2_6 "char" NOT NULL, col2_7 FLOAT4 NOT NULL, PRIMARY KEY (col2_6, col2_7, col2_3, col2_4), col2_8 FLOAT4 GENERATED ALWAYS AS (col2_7 + col2_3) STORED, col2_9 FLOAT4 NOT NULL GENERATED ALWAYS AS (col2_7 + 1.8407866954803467::FLOAT8) STORED, col2_10 FLOAT4 NOT NULL GENERATED ALWAYS AS (col2_3 + col2_7) STORED);
        test_1        |             CREATE INDEX ON table2 (col2_3 DESC, col2_8 ASC, col2_9, col2_10 ASC, col2_6 ASC, col2_2 ASC) INCLUDE (col2_0, col2_5);
        test_1        |             CREATE INDEX ON table2 (col2_3, col2_0 ASC, col2_9 DESC);
        test_1        |             CREATE TABLE table3 (col3_0 REGPROCEDURE NOT NULL, col3_1 NAME NOT NULL, col3_2 BIT(10) NOT NULL, col3_3 REGPROC NOT NULL, col3_4 TIME NOT NULL, col3_5 BOX2D NOT NULL, col3_6 TIMESTAMP NOT NULL, col3_7 BYTEA NOT NULL, col3_8 BYTEA NOT NULL, col3_9 TIMESTAMPTZ NOT NULL, col3_10 REGNAMESPACE NOT NULL, col3_11 BYTEA NOT NULL, col3_12 REGTYPE NOT NULL, col3_13 BOOL NOT NULL, PRIMARY KEY (col3_1, col3_11, col3_10, col3_9, col3_6, col3_13, col3_3, col3_0, col3_8, col3_12, col3_7), col3_14 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col3_7 IS NULL THEN e'\U00002603'::TEXT ELSE e'Bb\t\rX\x11B'::TEXT END) STORED, col3_15 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col3_9 IS NULL THEN e'\x0f\x19\nD'::TEXT ELSE e'.\x17\bT\x03'::TEXT END) STORED);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_12 DESC, col3_14 ASC, col3_15 ASC, col3_6 ASC, col3_3, col3_11, col3_4 DESC, col3_0, col3_7 ASC, col3_8, col3_9 DESC, col3_2 DESC, col3_1, col3_13 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_8, col3_4, col3_11 DESC, col3_12 DESC, col3_9, col3_3 ASC, col3_10 DESC, col3_14, col3_6 ASC, col3_13, col3_2 DESC, col3_0 DESC, col3_15 ASC, col3_7, col3_1 DESC);
        test_1        |             CREATE TABLE table4 (col4_0 BYTEA, col4_1 REGPROCEDURE NOT NULL, col4_2 UUID NULL, col4_3 UUID NOT NULL, col4_4 TIMESTAMPTZ, col4_5 TIMESTAMPTZ NOT NULL, col4_6 INT8 NOT NULL, col4_7 BIT(34) NULL, col4_8 BOOL NULL, col4_9 TIMETZ NOT NULL, col4_10 GEOGRAPHY, PRIMARY KEY (col4_1, col4_3, col4_5), col4_11 TEXT GENERATED ALWAYS AS (lower(CAST(col4_10 AS TEXT))) STORED, col4_12 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col4_8 AS TEXT))) STORED, col4_13 TEXT GENERATED ALWAYS AS (CASE WHEN col4_4 IS NULL THEN NULL ELSE NULL END) STORED, col4_14 TEXT NOT NULL GENERATED ALWAYS AS (CASE WHEN col4_5 IS NULL THEN '?'::TEXT ELSE e'\x06G22S\x1f\x073\x04'::TEXT END) STORED, col4_15 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col4_2 AS TEXT))) STORED, col4_16 TEXT GENERATED ALWAYS AS (lower(CAST(col4_10 AS TEXT))) STORED, col4_17 TEXT GENERATED ALWAYS AS (lower(CAST(col4_10 AS TEXT))) STORED, col4_18 INT8 NOT NULL GENERATED ALWAYS AS (col4_6 + 127::INT8) STORED);
        test_1        |             CREATE INDEX ON table4 (col4_18, col4_9) INCLUDE (col4_6, col4_8, col4_10, col4_11, col4_12, col4_13, col4_15, col4_16);
        test_1        |             CREATE INDEX ON table4 (col4_15 DESC, col4_7 ASC, col4_12 ASC, col4_6) INCLUDE (col4_0, col4_2, col4_8, col4_9, col4_11, col4_18);
        test_1        |             CREATE UNIQUE INDEX ON table4 (col4_13 ASC, col4_5, col4_12 DESC, col4_3 DESC, col4_9 DESC, col4_6 DESC, col4_0, col4_2 DESC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('uw');
        test_1        |             CREATE TYPE rand_typ_1 AS ENUM ('mid', 'xygoy', 'qjvwoh', 'n', 'b', 'ubyik');
        test_1        |             CREATE TYPE rand_typ_2 AS ENUM ('hybjr', 'quq', 'xktn', 'svtd', 'tvtus', 'td');
        test_1        |             CREATE TYPE rand_typ_3 AS ENUM ('mn', 'fuhczv');
        test_1        |             CREATE TYPE rand_typ_4 AS ENUM ('lsxvc', 'oewfz', 'y', 'vm', 'mhus', 'nglh');
        test_1        |             
        test_1        |         compare_test.go:150: postgres: exec: ERROR: conversion between UTF8 and SQL_ASCII is not supported (SQLSTATE 0A000)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3087615&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3087615&tab=artifacts#/) on master @ [ec36bc66f5147a8703719a9c890be12dd0c08945](https://github.com/cockroachdb/cockroach/commits/ec36bc66f5147a8703719a9c890be12dd0c08945):


```
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (19.26s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |               	&net.IPNet{IP: s"::", Mask: {0x00, 0x00, 0x00, 0x00, ...}},
        test_1        |             - 	string("Dffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff"),
        test_1        |             + 	string("Dffff:ffff:ffff:ffff:ffff:ffff:ffff:ffff/128"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			'::2/0':::INET AS col_28357,
        test_1        |             			('D':::STRING::STRING || hostmask('::5/0':::INET::INET)::INET::INET)::STRING AS col_28358
        test_1        |             	)
        test_1        |             		AS tab_14004
        test_1        |             ORDER BY
        test_1        |             	col_28357 NULLS FIRST, col_28358 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			'::2/0'::INET AS col_28357,
        test_1        |             			('D'::TEXT::TEXT || hostmask('::5/0'::INET::INET)::INET::INET)::TEXT AS col_28358
        test_1        |             	)
        test_1        |             		AS tab_14004
        test_1        |             ORDER BY
        test_1        |             	col_28357 NULLS FIRST, col_28358 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3095641&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3095641&tab=artifacts#/) on master @ [9936b14065ed511b765f1c4442ed440252a2d897](https://github.com/cockroachdb/cockroach/commits/9936b14065ed511b765f1c4442ed440252a2d897):


```
        test_1        |     --- FAIL: TestCompare/postgres (191.37s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |               	string("A"),
        test_1        |             - 	string("2000-01-01 06:00:00D"),
        test_1        |             + 	string("2000-01-01 06:00:00+00D"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			'A':::STRING AS col_304987,
        test_1        |             			(('2000-01-01 06:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ + '00:00:00':::INTERVAL::INTERVAL)::TIMESTAMPTZ::TIMESTAMPTZ || 'D':::STRING::STRING)::STRING
        test_1        |             				AS col_304988
        test_1        |             	)
        test_1        |             		AS tab_168945
        test_1        |             ORDER BY
        test_1        |             	col_304987 NULLS FIRST, col_304988 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			'A'::TEXT AS col_304987,
        test_1        |             			(('2000-01-01 06:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ + '00:00:00'::INTERVAL::INTERVAL)::TIMESTAMPTZ::TIMESTAMPTZ || 'D'::TEXT::TEXT)::TEXT
        test_1        |             				AS col_304988
        test_1        |             	)
        test_1        |             		AS tab_168945
        test_1        |             ORDER BY
        test_1        |             	col_304987 NULLS FIRST, col_304988 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3098032&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3098032&tab=artifacts#/) on master @ [69308cce3bb7e660908cb3e2724eedd271ce5585](https://github.com/cockroachdb/cockroach/commits/69308cce3bb7e660908cb3e2724eedd271ce5585):


```
        postgres_1    | 				st_dwithin(st_buffer('C'::TEXT::TEXT, 1.0::FLOAT8::FLOAT8, 1::INT8::INT4)::GEOMETRY::GEOMETRY, NULL::GEOMETRY, 8.0::FLOAT8::FLOAT8)::BOOL
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_90332.col1_0 ASC, tab_90333.col1_0 DESC
        postgres_1    | 		)
        postgres_1    | 			AS tab_90334
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_131736 NULLS FIRST, col_131737 NULLS FIRST
        test_1        | --- FAIL: TestCompare (692.19s)
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     --- FAIL: TestCompare/postgres (89.22s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("J2000-01-01 03:00:00"),
        test_1        |             + 	string("J2000-01-01 03:00:00+00"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('J':::STRING::STRING || '2000-01-01 03:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ)::STRING AS col_132881)
        test_1        |             		AS tab_91126
        test_1        |             ORDER BY
        test_1        |             	col_132881 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('J'::TEXT::TEXT || '2000-01-01 03:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ)::TEXT AS col_132881)
        test_1        |             		AS tab_91126
        test_1        |             ORDER BY
        test_1        |             	col_132881 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3099740&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3099740&tab=artifacts#/) on master @ [0e747e4ac3630a9b41f2eccb7ace0e2d17258ae6](https://github.com/cockroachdb/cockroach/commits/0e747e4ac3630a9b41f2eccb7ace0e2d17258ae6):


```
        postgres_1    | 				tab_83005.col2_0
        postgres_1    | 			HAVING
        postgres_1    | 				every(NULL::BOOL)::BOOL
        postgres_1    | 		)
        postgres_1    | 			AS tab_83007
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_140658 NULLS FIRST
        test_1        | --- FAIL: TestCompare (783.39s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (180.01s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("B2000-01-01 02:00:00"),
        test_1        |             + 	string("B2000-01-01 02:00:00+00"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('B':::STRING::STRING || '2000-01-01 02:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ)::STRING AS col_140964)
        test_1        |             		AS tab_83204
        test_1        |             ORDER BY
        test_1        |             	col_140964 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('B'::TEXT::TEXT || '2000-01-01 02:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ)::TEXT AS col_140964)
        test_1        |             		AS tab_83204
        test_1        |             ORDER BY
        test_1        |             	col_140964 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3103797&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3103797&tab=artifacts#/) on master @ [1b686aef9949c1c7ef930b55bd1fbc0ed2e8268a](https://github.com/cockroachdb/cockroach/commits/1b686aef9949c1c7ef930b55bd1fbc0ed2e8268a):


```
        postgres_1    | 				postgres.public.table1 AS tab_133107
        postgres_1    | 			WHERE
        postgres_1    | 				NULL
        postgres_1    | 		)
        postgres_1    | 			AS tab_133108
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_254935 NULLS FIRST, col_254936 NULLS FIRST
        test_1        | --- FAIL: TestCompare (815.78s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (209.65s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("2000-01-01 06:00:00J"),
        test_1        |             + 	string("2000-01-01 06:00:00+00J"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('2000-01-01 06:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ || 'J':::STRING::STRING)::STRING AS col_255233)
        test_1        |             		AS tab_133260
        test_1        |             ORDER BY
        test_1        |             	col_255233 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('2000-01-01 06:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ || 'J'::TEXT::TEXT)::TEXT AS col_255233)
        test_1        |             		AS tab_133260
        test_1        |             ORDER BY
        test_1        |             	col_255233 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3108446&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3108446&tab=artifacts#/) on master @ [b35fe177c0fb6bf58ba00357e152ecaf6c1e0a2a](https://github.com/cockroachdb/cockroach/commits/b35fe177c0fb6bf58ba00357e152ecaf6c1e0a2a):


```
        postgres_1    | 		col_237839 NULLS FIRST,
        postgres_1    | 		col_237840 NULLS FIRST,
        postgres_1    | 		col_237841 NULLS FIRST,
        postgres_1    | 		col_237842 NULLS FIRST,
        postgres_1    | 		col_237843 NULLS FIRST,
        postgres_1    | 		col_237844 NULLS FIRST,
        postgres_1    | 		col_237845 NULLS FIRST
        test_1        | --- FAIL: TestCompare (766.53s)
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     --- FAIL: TestCompare/postgres (162.12s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("B2000-01-01 08:00:00"),
        test_1        |             + 	string("B2000-01-01 08:00:00+00"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('B':::STRING::STRING || '2000-01-01 08:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ)::STRING AS col_237966)
        test_1        |             		AS tab_120739
        test_1        |             ORDER BY
        test_1        |             	col_237966 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('B'::TEXT::TEXT || '2000-01-01 08:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ)::TEXT AS col_237966)
        test_1        |             		AS tab_120739
        test_1        |             ORDER BY
        test_1        |             	col_237966 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3117819&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3117819&tab=artifacts#/) on master @ [ea42b77e7b29fc97871a49b31f5b7ea951e6b7a4](https://github.com/cockroachdb/cockroach/commits/ea42b77e7b29fc97871a49b31f5b7ea951e6b7a4):


```
        postgres_1    | 2021-06-25 05:20:52.624 UTC [105] HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
        postgres_1    | 2021-06-25 05:20:52.624 UTC [105] STATEMENT:  SELECT
        postgres_1    | 		*
        postgres_1    | 	FROM
        postgres_1    | 		(SELECT 1::OID AS col_327600, ulid_to_uuid('D'::TEXT::TEXT)::UUID AS col_327601) AS tab_251903
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_327600 NULLS FIRST, col_327601 NULLS FIRST
        test_1        | --- FAIL: TestCompare (832.59s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (229.53s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("2000-01-01 04:00:00F"),
        test_1        |             + 	string("2000-01-01 04:00:00+00F"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('2000-01-01 04:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ || 'F':::STRING::STRING)::STRING AS col_327994)
        test_1        |             		AS tab_252188
        test_1        |             ORDER BY
        test_1        |             	col_327994 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('2000-01-01 04:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ || 'F'::TEXT::TEXT)::TEXT AS col_327994)
        test_1        |             		AS tab_252188
        test_1        |             ORDER BY
        test_1        |             	col_327994 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3121769&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3121769&tab=artifacts#/) on master @ [6b9d2a15f0c223c8dda04c5b2a39abe784b58bdd](https://github.com/cockroachdb/cockroach/commits/6b9d2a15f0c223c8dda04c5b2a39abe784b58bdd):


```
        postgres_1    | 	CREATE INDEX ON table2 (col2_2 DESC, col1_0 ASC, col2_1 DESC);
        postgres_1    | 	CREATE UNIQUE INDEX ON table2 (col1_0 ASC, col2_1 ASC, col2_2 DESC);
        postgres_1    | 	CREATE UNIQUE INDEX ON table2 (col1_0 DESC, col2_1, col2_2 ASC);
        postgres_1    | 	CREATE TABLE table3 (col1_0 INT4 NOT NULL, col3_1 "char" NOT NULL, col3_2 REGPROC NOT NULL, col3_3 BOOL, col3_4 FLOAT4, col3_5 INTERVAL, col3_6 REGTYPE NOT NULL, col3_7 REGPROC NOT NULL, col3_8 "char" NULL, col3_9 BIT(16) NULL, col3_10 FLOAT4 NULL, col3_11 REGCLASS NOT NULL, col3_12 INT4, col3_13 BYTEA NOT NULL, col3_14 BOOL NULL, col3_15 TIMETZ, col3_16 INT4, col3_17 FLOAT8 NOT NULL, PRIMARY KEY (col1_0, col3_1, col3_12, col3_5, col3_4, col3_2, col3_14, col3_6, col3_13, col3_16, col3_9, col3_3, col3_8));
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col1_0, col3_14 DESC, col3_16 ASC, col3_9 ASC, col3_11 ASC, col3_6 ASC, col3_15, col3_2 DESC, col3_10 ASC, col3_13 DESC);
        postgres_1    | 	CREATE INDEX ON table3 (col3_15 DESC, col3_6, col3_7 DESC, col3_8 DESC, col3_14 DESC, col3_10, col3_3 DESC, col3_2 ASC, col3_5, col3_12);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_5 DESC, col3_14 DESC, col3_15 DESC);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_12 ASC, col3_1) INCLUDE (col3_7, col3_11, col3_15);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_13 ASC, col3_1, col3_7 DESC, col1_0 DESC, col3_6 ASC, col3_8, col3_11 ASC);
        postgres_1    | 	CREATE INDEX ON table3 (col3_15 DESC, col3_12, col3_17, col3_2 DESC);
        postgres_1    | 	CREATE INDEX ON table3 (col3_14, col3_2);
        postgres_1    | 	CREATE UNIQUE INDEX ON table3 (col3_7 ASC, col3_13, col3_11, col3_1, col3_2 ASC, col3_15 DESC, col3_3 ASC);
        postgres_1    | 	CREATE TYPE rand_typ_0 AS ENUM ('h', 'xblax', 'xrasft', 'uderv', 'su');
        postgres_1    | 	
        test_1        | --- FAIL: TestCompare (608.23s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (4.05s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:149: CREATE TABLE table1 (col1_0 INT4 NOT NULL, col1_1 TEXT NOT NULL, col1_2 REGNAMESPACE, col1_3 UUID, col1_4 REGPROCEDURE, col1_5 GEOGRAPHY, col1_6 REGCLASS NOT NULL, col1_7 OID NULL, col1_8 NAME NOT NULL, col1_9 INET NULL, PRIMARY KEY (col1_0), col1_10 TEXT GENERATED ALWAYS AS (lower(CAST(col1_3 AS TEXT))) STORED, col1_11 TEXT GENERATED ALWAYS AS (CASE WHEN col1_4 IS NULL THEN e'\x1a'::TEXT ELSE e'9/*3W\x1f\x18'::TEXT END) STORED, col1_12 TEXT NOT NULL GENERATED ALWAYS AS (lower(col1_8)) STORED, col1_13 TEXT GENERATED ALWAYS AS (CASE WHEN col1_4 IS NULL THEN e'"N\n>|'::TEXT ELSE e'f\x01;'::TEXT END) STORED, col1_14 TEXT NULL GENERATED ALWAYS AS (lower(CAST(col1_9 AS TEXT))) STORED, col1_15 TEXT GENERATED ALWAYS AS (CASE WHEN col1_2 IS NULL THEN NULL ELSE e'\U00002603'::TEXT END) STORED, col1_16 TEXT NOT NULL GENERATED ALWAYS AS (lower(col1_1)) STORED);
        test_1        |             CREATE TABLE table2 (col1_0 INT4 NOT NULL, col2_1 TIMESTAMPTZ, col2_2 TIMESTAMPTZ, PRIMARY KEY (col1_0));
        test_1        |             CREATE INDEX ON table2 (col2_2 DESC, col1_0 ASC, col2_1 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_0 ASC, col2_1 ASC, col2_2 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table2 (col1_0 DESC, col2_1, col2_2 ASC);
        test_1        |             CREATE TABLE table3 (col1_0 INT4 NOT NULL, col3_1 "char" NOT NULL, col3_2 REGPROC NOT NULL, col3_3 BOOL, col3_4 FLOAT4, col3_5 INTERVAL, col3_6 REGTYPE NOT NULL, col3_7 REGPROC NOT NULL, col3_8 "char" NULL, col3_9 BIT(16) NULL, col3_10 FLOAT4 NULL, col3_11 REGCLASS NOT NULL, col3_12 INT4, col3_13 BYTEA NOT NULL, col3_14 BOOL NULL, col3_15 TIMETZ, col3_16 INT4, col3_17 FLOAT8 NOT NULL, PRIMARY KEY (col1_0, col3_1, col3_12, col3_5, col3_4, col3_2, col3_14, col3_6, col3_13, col3_16, col3_9, col3_3, col3_8));
        test_1        |             CREATE UNIQUE INDEX ON table3 (col1_0, col3_14 DESC, col3_16 ASC, col3_9 ASC, col3_11 ASC, col3_6 ASC, col3_15, col3_2 DESC, col3_10 ASC, col3_13 DESC);
        test_1        |             CREATE INDEX ON table3 (col3_15 DESC, col3_6, col3_7 DESC, col3_8 DESC, col3_14 DESC, col3_10, col3_3 DESC, col3_2 ASC, col3_5, col3_12);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_5 DESC, col3_14 DESC, col3_15 DESC);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_12 ASC, col3_1) INCLUDE (col3_7, col3_11, col3_15);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_13 ASC, col3_1, col3_7 DESC, col1_0 DESC, col3_6 ASC, col3_8, col3_11 ASC);
        test_1        |             CREATE INDEX ON table3 (col3_15 DESC, col3_12, col3_17, col3_2 DESC);
        test_1        |             CREATE INDEX ON table3 (col3_14, col3_2);
        test_1        |             CREATE UNIQUE INDEX ON table3 (col3_7 ASC, col3_13, col3_11, col3_1, col3_2 ASC, col3_15 DESC, col3_3 ASC);
        test_1        |             CREATE TYPE rand_typ_0 AS ENUM ('h', 'xblax', 'xrasft', 'uderv', 'su');
        test_1        |             
        test_1        |         compare_test.go:150: postgres: exec: ERROR: conversion between UTF8 and SQL_ASCII is not supported (SQLSTATE 0A000)
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3123387&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3123387&tab=artifacts#/) on master @ [a142e392ddebb2d8a7c575170ac168cdefa90088](https://github.com/cockroachdb/cockroach/commits/a142e392ddebb2d8a7c575170ac168cdefa90088):


```
        test_1        |     --- FAIL: TestCompare/postgres (123.85s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |               	nil,
        test_1        |             - 	string("2000-01-01 00:00:00+00D"),
        test_1        |             + 	string("2000-01-01 00:00:00D"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			NULL AS col_139698,
        test_1        |             			((('2000-01-01 06:00:00':::TIMESTAMP::TIMESTAMP - '2000-01-01 07:00:00':::TIMESTAMP::TIMESTAMP)::INTERVAL::INTERVAL + '2000-01-01 01:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ)::TIMESTAMPTZ::TIMESTAMPTZ || 'D':::STRING::STRING)::STRING
        test_1        |             				AS col_139699
        test_1        |             	)
        test_1        |             		AS tab_82540
        test_1        |             ORDER BY
        test_1        |             	col_139698 NULLS FIRST, col_139699 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			NULL AS col_139698,
        test_1        |             			((('2000-01-01 06:00:00'::TIMESTAMP::TIMESTAMP - '2000-01-01 07:00:00'::TIMESTAMP::TIMESTAMP)::INTERVAL::INTERVAL + '2000-01-01 01:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ)::TIMESTAMPTZ::TIMESTAMPTZ || 'D'::TEXT::TEXT)::TEXT
        test_1        |             				AS col_139699
        test_1        |             	)
        test_1        |             		AS tab_82540
        test_1        |             ORDER BY
        test_1        |             	col_139698 NULLS FIRST, col_139699 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3130423&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3130423&tab=artifacts#/) on master @ [085d7116cbe862f3ced540508778367ce42146f8](https://github.com/cockroachdb/cockroach/commits/085d7116cbe862f3ced540508778367ce42146f8):


```
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (9.70s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	s"0",
        test_1        |             + 	s"1",
        test_1        |               	[]uint8{0x43},
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			ceil(pow(0:::DECIMAL::DECIMAL, 0:::DECIMAL::DECIMAL)::DECIMAL::DECIMAL)::DECIMAL AS col_8087,
        test_1        |             			'\x43':::BYTES AS col_8088
        test_1        |             	)
        test_1        |             		AS tab_4600
        test_1        |             ORDER BY
        test_1        |             	col_8087 NULLS FIRST, col_8088 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			ceil(pow(0::DECIMAL::DECIMAL, 0::DECIMAL::DECIMAL)::DECIMAL::DECIMAL)::DECIMAL AS col_8087,
        test_1        |             			'\x43'::BYTEA AS col_8088
        test_1        |             	)
        test_1        |             		AS tab_4600
        test_1        |             ORDER BY
        test_1        |             	col_8087 NULLS FIRST, col_8088 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3140430&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3140430&tab=artifacts#/) on master @ [cea4a80a82676db34f9b743469e75b87b1e75a06](https://github.com/cockroachdb/cockroach/commits/cea4a80a82676db34f9b743469e75b87b1e75a06):


```
        postgres_1    | 	FROM
        postgres_1    | 		(
        postgres_1    | 			SELECT
        postgres_1    | 				(true AND NULL) AS col_41090,
        postgres_1    | 				st_polygonfromtext('J'::TEXT::TEXT, 2::INT8::INT4)::GEOMETRY AS col_41091
        postgres_1    | 		)
        postgres_1    | 			AS tab_23375
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_41090 NULLS FIRST, col_41091 NULLS FIRST
        test_1        | --- FAIL: TestCompare (633.93s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (30.81s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	s"8",
        test_1        |             + 	int16(8),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT COALESCE(st_zmflag(NULL::GEOMETRY)::INT2, 8:::INT8) AS col_41254) AS tab_23488
        test_1        |             ORDER BY
        test_1        |             	col_41254 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT COALESCE(st_zmflag(NULL::GEOMETRY)::INT2, 8::INT8) AS col_41254) AS tab_23488
        test_1        |             ORDER BY
        test_1        |             	col_41254 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3146907&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3146907&tab=artifacts#/) on master @ [7121f6d80505801cc432a178ad8b6dacbaac1448](https://github.com/cockroachdb/cockroach/commits/7121f6d80505801cc432a178ad8b6dacbaac1448):


```
        postgres_1    | 			AS tab_92066
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_143481 NULLS FIRST,
        postgres_1    | 		col_143482 NULLS FIRST,
        postgres_1    | 		col_143483 NULLS FIRST,
        postgres_1    | 		col_143484 NULLS FIRST,
        postgres_1    | 		col_143485 NULLS FIRST
        test_1        | --- FAIL: TestCompare (694.95s)
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     --- FAIL: TestCompare/postgres (93.72s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("J2000-01-01 00:00:00"),
        test_1        |             + 	string("J2000-01-01 00:00:00+00"),
        test_1        |               }))
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('J'::TEXT::TEXT || '2000-01-01 00:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ)::TEXT AS col_143772)
        test_1        |             		AS tab_92267
        test_1        |             ORDER BY
        test_1        |             	col_143772 NULLS FIRST;
        test_1        |             
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('J':::STRING::STRING || '2000-01-01 00:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ)::STRING AS col_143772)
        test_1        |             		AS tab_92267
        test_1        |             ORDER BY
        test_1        |             	col_143772 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3150360&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3150360&tab=artifacts#/) on master @ [84ec89c77841016da0b9c4c71772a4304bad45a5](https://github.com/cockroachdb/cockroach/commits/84ec89c77841016da0b9c4c71772a4304bad45a5):


```
        postgres_1    | 2021-07-06 05:15:10.706 UTC [96] LOG:  database system was shut down at 2021-07-06 05:15:10 UTC
        postgres_1    | 2021-07-06 05:15:10.711 UTC [1] LOG:  database system is ready to accept connections
        postgres_1    | 2021-07-06 05:15:17.841 UTC [106] ERROR:  cannot cast type interval to double precision at character 67
        postgres_1    | 2021-07-06 05:15:17.841 UTC [106] STATEMENT:  SELECT
        postgres_1    | 		*
        postgres_1    | 	FROM
        postgres_1    | 		(
        postgres_1    | 			SELECT
        postgres_1    | 				percentile_cont('00:00:09'::INTERVAL::FLOAT8)::INTERVAL AS col_123, 6::INT8 AS col_124, NULL AS col_125
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table1 AS tab_68
        postgres_1    | 			GROUP BY
        postgres_1    | 				tab_68.col1_3, tab_68.col1_2, tab_68.col1_0
        postgres_1    | 			HAVING
        postgres_1    | 				bool_and(NULL::BOOL)::BOOL
        postgres_1    | 		)
        postgres_1    | 			AS tab_69
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_123 NULLS FIRST, col_124 NULLS FIRST, col_125 NULLS FIRST
        test_1        | --- FAIL: TestCompare (613.50s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (6.95s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare postgres to cockroach1:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("3.1.2 cbe925d"),
        test_1        |             + 	string("3.0.1 ec2a9aa"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT postgis_scripts_installed()::STRING AS col_261) AS tab_140 ORDER BY col_261 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT postgis_scripts_installed()::TEXT AS col_261) AS tab_140 ORDER BY col_261 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3154230&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3154230&tab=artifacts#/) on master @ [ba689f91a5bbd7737a8a229522048b2ba91b2ec0](https://github.com/cockroachdb/cockroach/commits/ba689f91a5bbd7737a8a229522048b2ba91b2ec0):


```
        postgres_1    | 2021-07-07 05:16:12.249 UTC [107] STATEMENT:  SELECT
        postgres_1    | 		*
        postgres_1    | 	FROM
        postgres_1    | 		(
        postgres_1    | 			SELECT
        postgres_1    | 				count(*) AS col_80674
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table2 AS tab_48716
        postgres_1    | 			WHERE
        postgres_1    | 				parse_timestamp(tab_48716.col2_13::TEXT)::TIMESTAMP::TIMESTAMP
        postgres_1    | 				NOT IN (SELECT '2000-01-01 01:00:00'::TIMESTAMP::TIMESTAMP AS col_80675)
        postgres_1    | 			GROUP BY
        postgres_1    | 				tab_48716.col2_4
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_48716.col2_4 ASC
        postgres_1    | 		)
        postgres_1    | 			AS tab_48717
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_80674 NULLS FIRST
        test_1        | --- FAIL: TestCompare (704.29s)
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     --- FAIL: TestCompare/postgres (101.03s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("3.0.1 ec2a9aa"),
        test_1        |             + 	string("3.1.2 cbe925d"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT postgis_scripts_installed()::STRING AS col_80744) AS tab_48764 ORDER BY col_80744 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT postgis_scripts_installed()::TEXT AS col_80744) AS tab_48764 ORDER BY col_80744 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3159076&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3159076&tab=artifacts#/) on master @ [7d0fd136a538b22cbf9bfff03b2885b7783711aa](https://github.com/cockroachdb/cockroach/commits/7d0fd136a538b22cbf9bfff03b2885b7783711aa):


```
        postgres_1    | 				postgres.public.table1 AS tab_395201
        postgres_1    | 			GROUP BY
        postgres_1    | 				tab_395201.col1_2, tab_395201.tableoid, tab_395201.col1_0
        postgres_1    | 		)
        postgres_1    | 			AS tab_395202
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_510249 NULLS FIRST
        test_1        | --- FAIL: TestCompare (984.32s)
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     --- FAIL: TestCompare/postgres (380.90s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("I2000-01-01 05:00:00"),
        test_1        |             + 	string("I2000-01-01 05:00:00+00"),
        test_1        |               }))
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('I'::TEXT::TEXT || '2000-01-01 05:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ)::TEXT AS col_510279)
        test_1        |             		AS tab_395224
        test_1        |             ORDER BY
        test_1        |             	col_510279 NULLS FIRST;
        test_1        |             
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT ('I':::STRING::STRING || '2000-01-01 05:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ)::STRING AS col_510279)
        test_1        |             		AS tab_395224
        test_1        |             ORDER BY
        test_1        |             	col_510279 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3163910&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3163910&tab=artifacts#/) on master @ [c543d448266e2e1e3adefe3a55f920beacd848b3](https://github.com/cockroachdb/cockroach/commits/c543d448266e2e1e3adefe3a55f920beacd848b3):


```
        postgres_1    | 				true
        postgres_1    | 			GROUP BY
        postgres_1    | 				tab_34972.tableoid, tab_34973.col1_5, tab_34973.col1_7, tab_34972.col1_3, tab_34972.col1_1
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_34972.col1_1, tab_34973.col1_5 DESC, tab_34973.col1_7, tab_34972.col1_3
        postgres_1    | 		)
        postgres_1    | 			AS tab_34974
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_63343 NULLS FIRST
        test_1        | --- FAIL: TestCompare (644.05s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (39.64s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	int16(3),
        test_1        |             + 	s"3",
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT COALESCE(st_zmflag(NULL::GEOMETRY)::INT2, 3:::INT8) AS col_64053) AS tab_35364
        test_1        |             ORDER BY
        test_1        |             	col_64053 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT COALESCE(st_zmflag(NULL::GEOMETRY)::INT2, 3::INT8) AS col_64053) AS tab_35364
        test_1        |             ORDER BY
        test_1        |             	col_64053 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3170586&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3170586&tab=artifacts#/) on master @ [9ba3068987a973a92757db53800dfafbf8a262e7](https://github.com/cockroachdb/cockroach/commits/9ba3068987a973a92757db53800dfafbf8a262e7):


```
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (22.48s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("2000-01-01 01:00:00E"),
        test_1        |             + 	string("2000-01-01 01:00:00+00E"),
        test_1        |               	string("H"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			('2000-01-01 01:00:00+00:00':::TIMESTAMPTZ::TIMESTAMPTZ || 'E':::STRING::STRING)::STRING AS col_27620,
        test_1        |             			'H':::STRING AS col_27621
        test_1        |             	)
        test_1        |             		AS tab_14127
        test_1        |             ORDER BY
        test_1        |             	col_27620 NULLS FIRST, col_27621 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(
        test_1        |             		SELECT
        test_1        |             			('2000-01-01 01:00:00+00:00'::TIMESTAMPTZ::TIMESTAMPTZ || 'E'::TEXT::TEXT)::TEXT AS col_27620,
        test_1        |             			'H'::TEXT AS col_27621
        test_1        |             	)
        test_1        |             		AS tab_14127
        test_1        |             ORDER BY
        test_1        |             	col_27620 NULLS FIRST, col_27621 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3185659&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3185659&tab=artifacts#/) on master @ [31cdc0e4e0c5dda4cf67b53b45c36faf6a24eb33](https://github.com/cockroachdb/cockroach/commits/31cdc0e4e0c5dda4cf67b53b45c36faf6a24eb33):


```
        postgres_1    | 2021-07-15 05:16:26.540 UTC [105] ERROR:  operator does not exist: bigint >> bigint at character 68
        postgres_1    | 2021-07-15 05:16:26.540 UTC [105] HINT:  No operator matches the given name and argument types. You might need to add explicit type casts.
        postgres_1    | 2021-07-15 05:16:26.540 UTC [105] STATEMENT:  SELECT
        postgres_1    | 		*
        postgres_1    | 	FROM
        postgres_1    | 		(SELECT NULL::OID AS col_237, (0::INT8::INT8::INT8 >> 6::INT8::INT8::INT8)::INT8 AS col_238) AS tab_101
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_237 NULLS FIRST, col_238 NULLS FIRST
        test_1        | --- FAIL: TestCompare (606.54s)
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     --- FAIL: TestCompare/postgres (2.12s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |               	s"4",
        test_1        |             - 	string("0"),
        test_1        |             + 	string("-"),
        test_1        |               }))
        test_1        |             postgres:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT 4::INT8::INT8 AS col_513, 0::OID::REGCLASS AS col_514) AS tab_234
        test_1        |             ORDER BY
        test_1        |             	col_513 NULLS FIRST, col_514 NULLS FIRST;
        test_1        |             
        test_1        |             cockroach1:
        test_1        |             SELECT
        test_1        |             	*
        test_1        |             FROM
        test_1        |             	(SELECT 4:::INT8::INT8 AS col_513, 0:::OID::REGCLASS AS col_514) AS tab_234
        test_1        |             ORDER BY
        test_1        |             	col_513 NULLS FIRST, col_514 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach2_1 ... 
        Stopping compare_cockroach1_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach2_1 ... done
        Stopping compare_cockroach1_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3189964&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3189964&tab=artifacts#/) on master @ [4e2ee35288faf691d655d51e315a14fffab431ac](https://github.com/cockroachdb/cockroach/commits/4e2ee35288faf691d655d51e315a14fffab431ac):


```
        postgres_1    | 		col_206 NULLS FIRST
        postgres_1    | 2021-07-16 05:16:29.991 UTC [106] ERROR:  function xor_agg(bytea) does not exist at character 60
        postgres_1    | 2021-07-16 05:16:29.991 UTC [106] HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
        postgres_1    | 2021-07-16 05:16:29.991 UTC [106] STATEMENT:  SELECT
        postgres_1    | 		*
        postgres_1    | 	FROM
        postgres_1    | 		(
        postgres_1    | 			SELECT
        postgres_1    | 				tab_131.tableoid AS col_241, xor_agg(tab_131.col2_5::BYTEA)::BYTEA AS col_242
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table2 AS tab_131
        postgres_1    | 			GROUP BY
        postgres_1    | 				tab_131.tableoid, tab_131.col2_5
        postgres_1    | 			ORDER BY
        postgres_1    | 				tab_131.col2_5
        postgres_1    | 		)
        postgres_1    | 			AS tab_132
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_241 NULLS FIRST, col_242 NULLS FIRST
        test_1        | --- FAIL: TestCompare (605.45s)
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     --- FAIL: TestCompare/postgres (2.01s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("0"),
        test_1        |             + 	string("-"),
        test_1        |               }))
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT 0::OID::REGNAMESPACE AS col_288) AS tab_159 ORDER BY col_288 NULLS FIRST;
        test_1        |             
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT 0:::OID::REGNAMESPACE AS col_288) AS tab_159 ORDER BY col_288 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

compose.TestComposeCompare [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=3194043&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=3194043&tab=artifacts#/) on master @ [fa1da4ac2872533007e4ed4c70127f85040ecb25](https://github.com/cockroachdb/cockroach/commits/fa1da4ac2872533007e4ed4c70127f85040ecb25):


```
        postgres_1    | 2021-07-19 14:29:29.921 UTC [1] LOG:  database system is ready to accept connections
        postgres_1    | 2021-07-19 14:29:31.774 UTC [107] ERROR:  frame starting offset must not be null
        postgres_1    | 2021-07-19 14:29:31.774 UTC [107] STATEMENT:  SELECT
        postgres_1    | 		*
        postgres_1    | 	FROM
        postgres_1    | 		(
        postgres_1    | 			SELECT
        postgres_1    | 				string_agg(tab_118.col1_4::TEXT, tab_118.col1_4::TEXT) OVER (PARTITION BY tab_118.col1_4 ROWS BETWEEN NULL::INT8 PRECEDING AND CURRENT ROW)::TEXT
        postgres_1    | 					AS col_201
        postgres_1    | 			FROM
        postgres_1    | 				postgres.public.table1 AS tab_118
        postgres_1    | 			GROUP BY
        postgres_1    | 				tab_118.col1_4
        postgres_1    | 			HAVING
        postgres_1    | 				bool_or(true::BOOL::BOOL)::BOOL
        postgres_1    | 		)
        postgres_1    | 			AS tab_119
        postgres_1    | 	ORDER BY
        postgres_1    | 		col_201 NULLS FIRST
        test_1        | --- FAIL: TestCompare (605.89s)
        test_1        |     compare_test.go:116: Checking connection to: cockroach2
        test_1        |     compare_test.go:116: Checking connection to: postgres
        test_1        |     compare_test.go:116: Checking connection to: cockroach1
        test_1        |     --- FAIL: TestCompare/postgres (1.34s)
        test_1        |         compare_test.go:125: starting test: postgres
        test_1        |         compare_test.go:132: initializing connection: cockroach1
        test_1        |         compare_test.go:132: initializing connection: postgres
        test_1        |         compare_test.go:175: compare cockroach1 to postgres:
        test_1        |             unexpected diff:
        test_1        |               []interface{}(Inverse(func1, []interface{}{
        test_1        |             - 	string("0"),
        test_1        |             + 	string("-"),
        test_1        |               }))
        test_1        |             cockroach1:
        test_1        |             SELECT * FROM (SELECT 0:::OID::REGNAMESPACE AS col_310) AS tab_188 ORDER BY col_310 NULLS FIRST;
        test_1        |             
        test_1        |             postgres:
        test_1        |             SELECT * FROM (SELECT 0::OID::REGNAMESPACE AS col_310) AS tab_188 ORDER BY col_310 NULLS FIRST;
        test_1        |             
        test_1        | FAIL
        compare_test_1 exited with code 1
        Stopping compare_cockroach1_1 ... 
        Stopping compare_cockroach2_1 ... 
        Stopping compare_postgres_1   ... 
        Stopping compare_postgres_1   ... done
        Stopping compare_cockroach1_1 ... done
        Stopping compare_cockroach2_1 ... done
        Aborting on container exit...
        
    compose_test.go:49: exit status 1
```
<details><summary>Reproduce</summary>
<p>
<p>To reproduce, try:

```bash
make stressrace TESTS=TestComposeCompare PKG=./pkg/compose TESTTIMEOUT=5m STRESSFLAGS='-timeout 5m' 2>&1
```
</p>
</p>
</details>
<details><summary>Same failure on other branches</summary>
<p>

- #62602 compose: TestComposeCompare failed [C-test-failure O-robot T-sql-experience branch-release-21.1]
</p>
</details>
<details><summary>Internal log</summary>
<p>

```
mjibson marked as alumn{us/a}; resolving to rafiss instead
```
</p>
</details>

/cc @cockroachdb/sql-experience rafiss
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestComposeCompare.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

--- a/pkg/cmd/cmpconn/conn.go
+++ b/pkg/cmd/cmpconn/conn.go
@@ -174,7 +174,7 @@ func CompareConns(
 	conns map[string]Conn,
 	prep, exec string,
 	ignoreSQLErrors bool,
-) (err error) {
+) (ignoredErr bool, err error) {
 	ctx, cancel := context.WithTimeout(ctx, timeout)
 	defer cancel()
 	connRows := make(map[string]*pgx.Rows)
@@ -186,7 +186,7 @@ func CompareConns(
 		}
 		rows, err := conn.Values(ctx, prep, connExecs[name])
 		if err != nil {
-			return nil //nolint:returnerrcheck
+			return true, nil //nolint:returnerrcheck
 		}
 		defer rows.Close()
 		connRows[name] = rows
@@ -219,7 +219,9 @@ func CompareConns(
 // It always returns an error if there are any differences. Additionally,
 // ignoreSQLErrors specifies whether SQL errors should be ignored (in which
 // case the function returns nil if SQL error occurs).
-func compareRows(connRows map[string]*pgx.Rows, ignoreSQLErrors bool) error {
+func compareRows(
+	connRows map[string]*pgx.Rows, ignoreSQLErrors bool,
+) (ignoredErr bool, retErr error) {
 	var first []interface{}
 	var firstName string
 	var minCount int
@@ -240,16 +242,16 @@ ReadRows:
 					// This function can fail if, for example,
 					// a number doesn't fit into a float64. Ignore
 					// them and move along to another query.
-					err = nil
+					return true, nil
 				}
-				return err
+				return false, err
 			}
 			if firstName == "" {
 				firstName = name
 				first = vals
 			} else {
 				if err := CompareVals(first, vals); err != nil {
-					return fmt.Errorf("compare %s to %s:\n%v", firstName, name, err)
+					return false, fmt.Errorf("compare %s to %s:\n%v", firstName, name, err)
 				}
 			}
 		}
@@ -263,16 +265,16 @@ ReadRows:
 			if ignoreSQLErrors {
 				// Aww someone had a SQL error maybe, so we can't use this
 				// query.
-				err = nil
+				return true, nil
 			}
-			return err
+			return false, err
 		}
 	}
 	// Ensure each connection returned the same number of rows.
 	for name, count := range rowCounts {
 		if minCount != count {
-			return fmt.Errorf("%s had %d rows, expected %d", name, count, minCount)
+			return false, fmt.Errorf("%s had %d rows, expected %d", name, count, minCount)
 		}
 	}
-	return nil
+	return false, nil
 }

--- a/pkg/cmd/roachtest/tests/tpcdsvec.go
+++ b/pkg/cmd/roachtest/tests/tpcdsvec.go
@@ -106,7 +106,7 @@ func registerTPCDSVec(r registry.Registry) {
 			// A sanity check that we have different values of 'vectorize'
 			// session variable on two connections and that the comparator will
 			// emit an error because of that difference.
-			if err := cmpconn.CompareConns(
+			if _, err := cmpconn.CompareConns(
 				ctx, timeout, conns, "", "SHOW vectorize;", false, /* ignoreSQLErrors */
 			); err == nil {
 				t.Fatal("unexpectedly SHOW vectorize didn't trigger an error on comparison")
@@ -139,7 +139,7 @@ func registerTPCDSVec(r registry.Registry) {
 				// around issues with cancellation.
 				conns, cleanup := openNewConnections()
 				start := timeutil.Now()
-				if err := cmpconn.CompareConns(
+				if _, err := cmpconn.CompareConns(
 					ctx, 3*timeout, conns, "", query, false, /* ignoreSQLErrors */
 				); err != nil {
 					t.Status(fmt.Sprintf("encountered an error: %s\n", err))

--- a/pkg/cmd/smithcmp/main.go
+++ b/pkg/cmd/smithcmp/main.go
@@ -164,9 +164,10 @@ func main() {
 	var prep, exec string
 	ctx := context.Background()
 	done := time.After(timeout)
-	for i := 0; true; i++ {
+	for i, ignoredErrCount := 0, 0; true; i++ {
 		select {
 		case <-done:
+			fmt.Printf("executed query count: %d\nignored error count: %d\n", i, ignoredErrCount)
 			return
 		default:
 		}
@@ -197,11 +198,13 @@ func main() {
 			fmt.Println(exec)
 		}
 		if compare {
-			if err := cmpconn.CompareConns(
+			if ignoredErr, err := cmpconn.CompareConns(
 				ctx, stmtTimeout, conns, prep, exec, true, /* ignoreSQLErrors */
 			); err != nil {
 				fmt.Printf("prep:\n%s;\nexec:\n%s;\nERR: %s\n\n", prep, exec, err)
 				os.Exit(1)
+			} else if ignoredErr {
+				ignoredErrCount++
 			}
 		} else {
 			for _, conn := range conns {

--- a/pkg/compose/compare/compare/compare_test.go
+++ b/pkg/compose/compare/compare/compare_test.go
@@ -156,16 +156,20 @@ func TestCompare(t *testing.T) {
 				t.Fatal(err)
 			}
 
+			ignoredErrCount := 0
+			totalQueryCount := 0
 			until := time.After(*flagEach)
 			for {
 				select {
 				case <-until:
-					t.Logf("done with test: %s", confName)
+					t.Logf("done with test. totalQueryCount=%d ignoredErrCount=%d test=%s",
+						totalQueryCount, ignoredErrCount, confName,
+					)
 					return
 				default:
 				}
 				query := smither.Generate()
-				if err := cmpconn.CompareConns(
+				if ignoredErr, err := cmpconn.CompareConns(
 					ctx, time.Second*30, conns, "" /* prep */, query, config.ignoreSQLErrors,
 				); err != nil {
 					path := filepath.Join(*flagArtifacts, confName+".log")
@@ -173,7 +177,10 @@ func TestCompare(t *testing.T) {
 						t.Log(err)
 					}
 					t.Fatal(err)
+				} else if ignoredErr {
+					ignoredErrCount++
 				}
+				totalQueryCount++
 				// Make sure we can still ping on a connection. If we can't we may have
 				// crashed something.
 				for name, conn := range conns {

--- a/pkg/compose/compare/docker-compose.yml
+++ b/pkg/compose/compare/docker-compose.yml
@@ -3,7 +3,7 @@ services:
   postgres:
     image: postgis/postgis:13-3.1
     environment:
-      - POSTGRES_INITDB_ARGS=--locale=C
+      - POSTGRES_INITDB_ARGS=--locale=C --encoding=UTF8
       - POSTGRES_HOST_AUTH_METHOD=trust
   cockroach1:
     image: ubuntu:xenial-20170214

--- a/pkg/internal/sqlsmith/scalar.go
+++ b/pkg/internal/sqlsmith/scalar.go
@@ -152,6 +152,12 @@ func makeConstExpr(s *Smither, typ *types.T, refs colRefs) tree.TypedExpr {
 	// In Postgres mode, make sure the datum is resolved as the type we want.
 	// CockroachDB and Postgres differ in how constants are typed otherwise.
 	if s.postgres {
+		// Casts to REGTYPE, REGCLASS, etc are not deterministic since they
+		// involve OID->name resolution, and the OIDs will not match across
+		// two different databases.
+		if typ.Family() == types.OidFamily {
+			typ = types.Oid
+		}
 		expr = tree.NewTypedCastExpr(expr, typ)
 	}
 	return expr

